---
search:
  boost: 0.5
---
# read2tree

read2tree is a software tool that allows to obtain alignment matrices for tree inference.  For this purpose it makes use of the OMA database and a set of reads. Its strength lies in the fact that  it bipasses the several standard steps when obtaining such a matrix in regular analysis. These steps are  read filtereing, assembly, gene prediction, gene annotation, all vs all comparison, orthology prediction,  alignment and concatination.

*homepage*: <https://github.com/DessimozLab/read2tree>

version | toolchain
--------|----------
``0.1.5`` | ``foss/2023a``


*(quick links: [(all)](../index.md) - [0](../0/index.md) - [a](../a/index.md) - [b](../b/index.md) - [c](../c/index.md) - [d](../d/index.md) - [e](../e/index.md) - [f](../f/index.md) - [g](../g/index.md) - [h](../h/index.md) - [i](../i/index.md) - [j](../j/index.md) - [k](../k/index.md) - [l](../l/index.md) - [m](../m/index.md) - [n](../n/index.md) - [o](../o/index.md) - [p](../p/index.md) - [q](../q/index.md) - [r](../r/index.md) - [s](../s/index.md) - [t](../t/index.md) - [u](../u/index.md) - [v](../v/index.md) - [w](../w/index.md) - [x](../x/index.md) - [y](../y/index.md) - [z](../z/index.md))*

