# List of supported software

!!! note

    This page contains a lot of information, it may take a while to load.


SimpleBuild supports 3161 different software packages (incl. toolchains, bundles):

[a](#a) - [b](#b) - [c](#c) - [d](#d) - [e](#e) - [f](#f) - [g](#g) - [h](#h) - [i](#i) - [j](#j) - [k](#k) - [l](#l) - [m](#m) - [n](#n) - [o](#o) - [p](#p) - [q](#q) - [r](#r) - [s](#s) - [t](#t) - [u](#u) - [v](#v) - [w](#w) - [x](#x) - [y](#y) - [z](#z)


## 3


[3d-dna](#3d-dna) - [3to2](#3to2)


### 3d-dna

3D de novo assembly (3D DNA) pipeline

*homepage*: <https://github.com/theaidenlab/3d-dna>

version   |versionsuffix     |toolchain
----------|------------------|-----------------
``180922``|``-Python-2.7.15``|``GCCcore/8.2.0``

### 3to2

lib3to2 is a set of fixers that are intended to backport code written for Python version 3.x  into Python version 2.x.

*homepage*: <https://pypi.python.org/pypi/3to2>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.1.1``|``-Python-2.7.13``|``intel/2017a``

## 4


[4ti2](#4ti2)


### 4ti2

A software package for algebraic, geometric and combinatorial problems on linear spaces

*homepage*: <https://4ti2.github.io/>

version  |toolchain
---------|-------------------------------------
``1.6.9``|``GCC/8.2.0-2.31.1``, ``intel/2018b``

## A


[ABAQUS](#abaqus) - [ABINIT](#abinit) - [ABRA2](#abra2) - [ABRicate](#abricate) - [Abseil](#abseil) - [abTEM](#abtem) - [ABySS](#abyss) - [ack](#ack) - [ACT](#act) - [ACTC](#actc) - [ada](#ada) - [AdapterRemoval](#adapterremoval) - [ADDA](#adda) - [ADF](#adf) - [ADIOS](#adios) - [adjustText](#adjusttext) - [ADMIXTURE](#admixture) - [ADOL-C](#adol-c) - [Advisor](#advisor) - [affinity](#affinity) - [AFNI](#afni) - [AGAT](#agat) - [AGeNT](#agent) - [AGFusion](#agfusion) - [AIMAll](#aimall) - [aiohttp](#aiohttp) - [ALADIN](#aladin) - [Albacore](#albacore) - [Albumentations](#albumentations) - [alevin-fry](#alevin-fry) - [ALFA](#alfa) - [Alfred](#alfred) - [alleleCount](#allelecount) - [alleleIntegrator](#alleleintegrator) - [Allinea](#allinea) - [ALLPATHS-LG](#allpaths-lg) - [almosthere](#almosthere) - [Alpha](#alpha) - [AlphaFold](#alphafold) - [AlphaPulldown](#alphapulldown) - [ALPS](#alps) - [alsa-lib](#alsa-lib) - [AMAPVox](#amapvox) - [Amara](#amara) - [amask](#amask) - [Amber](#amber) - [AmberMini](#ambermini) - [AmberTools](#ambertools) - [AMD-LibM](#amd-libm) - [AMD-RNG](#amd-rng) - [AMD-SecureRNG](#amd-securerng) - [AMD-uProf](#amd-uprof) - [AMOS](#amos) - [AMPHORA2](#amphora2) - [AMPL-MP](#ampl-mp) - [amplimap](#amplimap) - [AMPtk](#amptk) - [AmrPlusPlus](#amrplusplus) - [AMS](#ams) - [Anaconda2](#anaconda2) - [Anaconda3](#anaconda3) - [aNCI](#anci) - [andi](#andi) - [ANGEL](#angel) - [angsd](#angsd) - [ANIcalculator](#anicalculator) - [anndata](#anndata) - [Annif](#annif) - [Annocript](#annocript) - [annovar](#annovar) - [ANSYS](#ansys) - [ANSYS_CFD](#ansys_cfd) - [ant](#ant) - [antiSMASH](#antismash) - [ANTLR](#antlr) - [ANTs](#ants) - [anvio](#anvio) - [any2fasta](#any2fasta) - [AOCC](#aocc) - [AOMP](#aomp) - [APBS](#apbs) - [apex](#apex) - [APR](#apr) - [APR-util](#apr-util) - [AptaSUITE](#aptasuite) - [ARAGORN](#aragorn) - [Arb](#arb) - [Arcade-Learning-Environment](#arcade-learning-environment) - [arcasHLA](#arcashla) - [ARCH](#arch) - [Archive-Zip](#archive-zip) - [ArchR](#archr) - [archspec](#archspec) - [ARGoS](#argos) - [argtable](#argtable) - [aria2](#aria2) - [Arlequin](#arlequin) - [Armadillo](#armadillo) - [arosics](#arosics) - [ARPACK++](#arpack++) - [arpack-ng](#arpack-ng) - [ArrayFire](#arrayfire) - [Arriba](#arriba) - [Arrow](#arrow) - [arrow-R](#arrow-r) - [ART](#art) - [artic-ncov2019](#artic-ncov2019) - [ARTS](#arts) - [ArviZ](#arviz) - [ARWEN](#arwen) - [ASAP](#asap) - [ASAP3](#asap3) - [ASCAT](#ascat) - [ASE](#ase) - [ASF-SearchAPI](#asf-searchapi) - [ASHS](#ashs) - [Aspera-CLI](#aspera-cli) - [Aspera-Connect](#aspera-connect) - [assimp](#assimp) - [Assimulo](#assimulo) - [ASTRID](#astrid) - [astro-tulips](#astro-tulips) - [astropy](#astropy) - [at-spi2-atk](#at-spi2-atk) - [at-spi2-core](#at-spi2-core) - [ATAT](#atat) - [ATK](#atk) - [ATLAS](#atlas) - [atomate](#atomate) - [AtomPAW](#atompaw) - [atools](#atools) - [atropos](#atropos) - [ATSAS](#atsas) - [attr](#attr) - [attrdict](#attrdict) - [attrdict3](#attrdict3) - [augur](#augur) - [AUGUSTUS](#augustus) - [Austin](#austin) - [AUTO-07p](#auto-07p) - [Autoconf](#autoconf) - [Autoconf-archive](#autoconf-archive) - [AutoDock](#autodock) - [AutoDock-GPU](#autodock-gpu) - [AutoDock-Vina](#autodock-vina) - [AutoDockSuite](#autodocksuite) - [AutoGeneS](#autogenes) - [AutoGrid](#autogrid) - [Automake](#automake) - [AutoMap](#automap) - [autopep8](#autopep8) - [Autotools](#autotools) - [Avogadro2](#avogadro2) - [avro-cpp](#avro-cpp) - [awscli](#awscli) - [axel](#axel)


### ABAQUS

Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit  dynamics FEA.

*homepage*: <http://www.simulia.com/products/abaqus_fea.html>

version   |versionsuffix    |toolchain
----------|-----------------|----------
``6.12.1``|``-linux-x86_64``|``system``
``6.13.5``|``-linux-x86_64``|``system``
``6.14.1``|``-linux-x86_64``|``system``
``2017``  |``-hotfix-1721`` |``system``
``2018``  |``-hotfix-1806`` |``system``
``2020``  |                 |``system``
``2021``  |``-hotfix-2132`` |``system``
``2022``  |                 |``system``
``2022``  |``-hotfix-2214`` |``system``
``2022``  |``-hotfix-2223`` |``system``

### ABINIT

ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. 

*homepage*: <https://www.abinit.org/>

version   |versionsuffix           |toolchain
----------|------------------------|-----------------------------------------------------------------
``7.0.3`` |``-x86_64_linux_gnu4.5``|``system``
``7.0.5`` |``-x86_64_linux_gnu4.5``|``system``
``7.2.1`` |``-x86_64_linux_gnu4.5``|``system``
``7.10.5``|                        |``intel/2016.02-GCC-4.9``
``7.10.5``|``-libxc``              |``intel/2016.02-GCC-4.9``
``8.0.8`` |                        |``intel/2016a``
``8.0.8b``|                        |``foss/2016b``, ``intel/2016b``
``8.2.2`` |                        |``foss/2016b``, ``intel/2016b``
``8.4.4`` |                        |``intel/2017b``
``8.6.3`` |                        |``intel/2018a``
``8.10.2``|                        |``intel/2018b``
``8.10.3``|                        |``intel/2018b``
``9.2.1`` |                        |``foss/2019b``, ``intel/2019b``, ``intel/2020a``
``9.4.1`` |                        |``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``9.4.2`` |                        |``foss/2021a``, ``intel/2021a``
``9.6.2`` |                        |``foss/2022a``, ``intel/2021a``, ``intel/2021b``, ``intel/2022a``

### ABRA2

Assembly Based ReAligner

*homepage*: <https://github.com/mozack/abra2>

version |toolchain
--------|-----------------------------
``2.22``|``iccifort/2019.5.281``
``2.23``|``GCC/10.2.0``, ``GCC/9.3.0``

### ABRicate

Mass screening of contigs for antimicrobial and virulence genes

*homepage*: <https://github.com/tseemann/abricate>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``0.9.9``|                |``gompi/2019b``
``0.9.9``|``-Perl-5.28.1``|``gompi/2019a``
``1.0.0``|                |``gompi/2021a``

### Abseil

Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives.

*homepage*: <https://abseil.io/>

version       |toolchain
--------------|--------------------------------------
``20210324.2``|``GCCcore/11.2.0``
``20230125.2``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``20230125.3``|``GCCcore/12.3.0``

### abTEM

abTEM provides a Python API for running simulations of Transmission Electron Microscopy images.

*homepage*: <https://github.com/jacobjma/abTEM>

version     |versionsuffix  |toolchain
------------|---------------|----------------------------------
``1.0.0b24``|``-ASE-3.22.0``|``fosscuda/2020b``
``1.0.0b26``|``-ASE-3.22.0``|``foss/2020b``, ``fosscuda/2020b``

### ABySS

Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler

*homepage*: <https://www.bcgsc.ca/platform/bioinfo/software/abyss>

version  |toolchain
---------|----------------------------------------------------------------
``1.9.0``|``foss/2016a``
``2.0.2``|``foss/2016b``, ``foss/2018a``, ``gompi/2019a``, ``intel/2016b``
``2.0.3``|``foss/2017b``, ``intel/2017b``
``2.1.5``|``foss/2019b``
``2.2.5``|``foss/2020b``

### ack

ack is a tool like grep, optimized for programmers

*homepage*: <https://beyondgrep.com>

version  |toolchain
---------|------------------
``2.14`` |``system``
``3.4.0``|``GCCcore/10.2.0``
``3.5.0``|``GCCcore/10.3.0``

### ACT

ACT is a Java application for displaying pairwise comparisons between two or more DNA sequences.  It can be used to identify and analyse regions of similarity and difference between genomes  and to explore conservation of synteny, in the context of the entire sequences and their annotation.  It can read complete EMBL, GENBANK and GFF entries or sequences in FASTA or raw format. 

*homepage*: <https://sanger-pathogens.github.io/Artemis/ACT/>

version   |versionsuffix|toolchain
----------|-------------|----------
``18.0.2``|``-Java-11`` |``system``
``18.0.3``|``-Java-11`` |``system``

### ACTC

ACTC converts independent triangles into triangle strips or fans.

*homepage*: <https://sourceforge.net/projects/actc>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------
``1.1``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2016b``

### ada

Performs discrete, real, and gentle boost under both exponential and logistic loss on a given data set.

*homepage*: <https://cran.r-project.org/web/packages/ada/index.html>

version  |versionsuffix|toolchain
---------|-------------|---------------
``2.0-5``|``-R-3.4.0`` |``intel/2017a``

### AdapterRemoval

AdapterRemoval searches for and removes remnant adapter sequences  from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases  from the 3' end of reads following adapter removal.

*homepage*: <https://github.com/MikkelSchubert/adapterremoval>

version  |toolchain
---------|----------------------------------------------
``2.2.0``|``foss/2016b``
``2.2.2``|``foss/2018b``
``2.3.1``|``foss/2018b``
``2.3.2``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``
``2.3.3``|``GCC/11.3.0``

### ADDA

ADDA is an open-source parallel implementation of the discrete dipole approximation, capable to simulate light scattering by particles of arbitrary shape and composition in a wide range of particle sizes.

*homepage*: <https://github.com/adda-team/adda/wiki>

version  |toolchain
---------|--------------
``1.3b4``|``foss/2019a``

### ADF

ADF is an accurate, parallelized, powerful computational chemistry program to understand and  predict chemical structure and reactivity with density functional theory (DFT).

*homepage*: <https://www.scm.com/ADF/>

version                         |versionsuffix|toolchain
--------------------------------|-------------|---------------
``2009.01a.pc64_linux.intelmpi``|             |``system``
``2014.02``                     |             |``system``
``2014.11.r48287``              |             |``intel/2016a``
``2016.101``                    |             |``system``
``2019.303``                    |``-intelmpi``|``system``

### ADIOS

The Adaptable IO System (ADIOS) provides a simple, flexible way for scientists to describe the data in their code that may need to be written, read, or processed outside of the running simulation.

*homepage*: <https://www.olcf.ornl.gov/center-projects/adios/>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``1.13.1``  |``-Python-2.7.15``|``foss/2019a``
``1.13.1``  |``-Python-3.8.2`` |``foss/2020a``
``20210804``|``-Python-3.8.2`` |``foss/2020a``

### adjustText

A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps.

*homepage*: <https://github.com/Phlya/adjustText>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.3``|                 |``foss/2021b``
``0.7.3``|``-Python-3.6.6``|``foss/2018b``
``0.7.3``|``-Python-3.7.2``|``intel/2019a``

### ADMIXTURE

ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from  multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much  more rapidly using a fast numerical optimization algorithm.

*homepage*: <https://dalexander.github.io/admixture/>

version  |toolchain
---------|----------
``1.3.0``|``system``

### ADOL-C

The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C.  

*homepage*: <https://projects.coin-or.org/ADOL-C>

version  |toolchain
---------|---------------
``2.7.0``|``gompi/2019a``
``2.7.2``|``gompi/2020a``

### Advisor

Vectorization Optimization and Thread Prototyping  - Vectorize & thread code or performance “dies”  - Simple workflow + data + tips = faster code faster  - Prioritize, Prototype & Predict performance gain  

*homepage*: <https://software.intel.com/intel-advisor-xe>

version         |toolchain
----------------|----------
``2016_update2``|``system``
``2017_update1``|``system``
``2018_update1``|``system``
``2018_update3``|``system``
``2019_update2``|``system``
``2019_update3``|``system``
``2019_update5``|``system``
``2021.2.0``    |``system``
``2021.4.0``    |``system``
``2022.1.0``    |``system``

### affinity

A small C++ wrapper for managing Linux CPU sets and CPU affinity. It also includes a tool to report binding, which is useful for testing different binding options

*homepage*: <https://github.com/vkarak/affinity>

version     |toolchain
------------|-------------------------------
``20230524``|``foss/2022a``, ``intel/2022a``

### AFNI

AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data -  a technique for mapping human brain activity.

*homepage*: <http://afni.nimh.nih.gov/>

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``18.1.09`` |``-Python-3.6.4`` |``intel/2018a``
``18.3.00`` |``-Python-3.6.6`` |``foss/2018b``
``19.0.01`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``20160329``|``-Python-2.7.11``|``intel/2016a``

### AGAT

AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations  in any GTF/GFF format.

*homepage*: <https://agat.readthedocs.io/en/latest/>

version  |toolchain
---------|--------------
``0.9.2``|``GCC/11.2.0``
``1.1.0``|``GCC/12.2.0``

### AGeNT

The Agilent Genomics NextGen Toolkit (AGeNT) is a Java-based software module that processes the read sequences from targeted high-throughput sequencing data generated by sequencing Agilent SureSelect and HaloPlex libraries.  The Trimmer utility of the AGeNT module processes the read sequences to identify and remove the adaptor sequences and extracts dual molecular barcodes (for SureSelect XT HS2).  The LocatIt utility of the AGeNT module processes the Molecular Barcode (MBC) information from HaloPlex HS, SureSelect XT HS, and SureSelect XT HS2 Illumina sequencing runs with options to either mark or merge duplicate reads and output in BAM file format. The Illumina InterOp libraries are a set of common routines used for reading InterOp metric files produced by Illumina sequencers including NextSeq 1k/2k. These libraries are backwards compatible and capable of supporting prior releases of the software, with one exception: GA systems have been excluded.

*homepage*: <https://www.agilent.com/en/product/next-generation-sequencing/hybridization-based-next-generation-sequencing-ngs/ngs-software/agent-232879>

version  |toolchain
---------|----------
``3.0.6``|``system``

### AGFusion

AGFusion is a python package for annotating gene fusions  from the human or mouse genomes.

*homepage*: <https://github.com/murphycj/AGFusion>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``1.2``|``-Python-3.7.2``|``foss/2019a``

### AIMAll

AIMAll is an simple to use, accurate, reliable and efficient quantum chemistry software package for performing comprehensive, quantitative and visual QTAIM analyses of molecular systems - starting from molecular wavefunction data.

*homepage*: <http://aim.tkgristmill.com>

version     |versionsuffix   |toolchain
------------|----------------|---------------
``19.10.12``|``-linux_64bit``|``intel/2020b``

### aiohttp

Asynchronous HTTP client/server framework for asyncio and Python.

*homepage*: <https://github.com/aio-libs/aiohttp>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``3.5.4``|``-Python-3.6.6``|``foss/2018b``
``3.8.1``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.8.3``|                 |``GCCcore/11.3.0``

### ALADIN

ALADIN was entirely built on the notion of compatibility with its mother system, IFS/ARPEG.  The latter, a joint development between the European Centre for Medium-Range Weather Forecasts (ECMWF) and  Meteo-France, was only meant to consider global Numerical Weather Prediction applications; hence the idea,  for ALADIN, to complement the IFS/ARPEGE project with a limited area model (LAM) version, while keeping the  differences between the two softwares as small as possible.

*homepage*: <http://www.cnrm.meteo.fr/aladin/>

version        |toolchain
---------------|---------------
``36t1_op2bf1``|``intel/2016a``

### Albacore

Albacore is a software project that provides an entry point to the Oxford Nanopore basecalling  algorithms.

*homepage*: <https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.0.2``|``-Python-3.6.1``|``intel/2017a``

### Albumentations

Albumentations is a Python library for fast and flexible image augmentations

*homepage*: <https://albumentations.ai>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.1.0``|                |``foss/2021b``
``1.1.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.3.0``|                |``foss/2022a``
``1.3.0``|``-CUDA-11.7.0``|``foss/2022a``

### alevin-fry

alevin-fry is an efficient and flexible tool for processing single-cell sequencing data,  currently focused on single-cell transcriptomics and feature barcoding.

*homepage*: <https://github.com/COMBINE-lab/alevin-fry>

version  |toolchain
---------|------------------
``0.4.3``|``GCCcore/11.2.0``
``0.6.0``|``GCCcore/10.3.0``

### ALFA

ALFA provides a global overview of features distribution composing NGS dataset(s). Given a set of  aligned reads (BAM files) and an annotation file (GTF format), the tool produces plots of the raw and normalized  distributions of those reads among genomic categories (stop codon, 5'-UTR, CDS, intergenic, etc.) and biotypes  (protein coding genes, miRNA, tRNA, etc.). Whatever the sequencing technique, whatever the organism.

*homepage*: <https://github.com/biocompibens/ALFA>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.1``|``-Python-3.7.2``|``foss/2019a``

### Alfred

Alfred is an efficient and versatile command-line application  that computes multi-sample quality control metrics in a read-group aware manner.  Alfred supports read counting, feature annotation and haplotype-resolved consensus  computation using multiple sequence alignments.

*homepage*: <https://www.gear-genomics.com/alfred/>

version  |toolchain
---------|--------------
``0.2.6``|``GCC/11.2.0``

### alleleCount

The alleleCount package primarily exists to prevent code duplication between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl  code wraps the C implementation of allele counting code for BAM/CRAM processing. 

*homepage*: <http://cancerit.github.io/alleleCount/>

version  |toolchain
---------|-----------------
``4.0.0``|``GCCcore/6.4.0``
``4.2.1``|``GCC/11.3.0``

### alleleIntegrator

R package to generate allele specific counts for scRNA data and use it to identify cancer cells

*homepage*: <https://github.com/constantAmateur/alleleIntegrator>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.8.8``|``-R-4.2.1`` |``foss/2022a``

### Allinea

The Allinea environment is an essential toolkit for developers and computational scientists looking to get results faster.

*homepage*: <http://www.allinea.com>

version                        |toolchain
-------------------------------|----------
``4.1-32834-Redhat-5.7-x86_64``|``system``
``4.1-32834-Redhat-6.0-x86_64``|``system``
``6.1.1-Ubuntu-14.04-x86_64``  |``system``

### ALLPATHS-LG

ALLPATHS-LG, the new short read genome assembler.

*homepage*: <http://www.broadinstitute.org/software/allpaths-lg/blog/>

version  |toolchain
---------|--------------
``52488``|``foss/2016a``

### almosthere

Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows,  Linux, and macOS. It is non-blocking as the progress update is done via a  dedicated, lightweight thread, as to not impair the performance of the calling program.

*homepage*: <https://github.com/horta/almosthere>

version   |toolchain
----------|------------------
``1.0.1`` |``GCCcore/7.3.0``
``1.0.10``|``GCCcore/9.3.0``
``2.0.2`` |``GCCcore/10.2.0``

### Alpha

Alpha is a tool designed for detailed comparative study of bacteriophage genomes.

*homepage*: <https://www.lirmm.fr/~swenson/alpha/alpha.htm>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20200430``|``-Python-2.7.16``|``foss/2019b``

### AlphaFold

AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known. This package of AlphaFold contains patches for ColabFold.

*homepage*: <https://github.com/sokrypton/alphafold>

version  |versionsuffix             |toolchain
---------|--------------------------|----------------------------------
``2.0.0``|                          |``foss/2020b``, ``fosscuda/2020b``
``2.0.1``|                          |``foss/2020b``, ``fosscuda/2020b``
``2.1.1``|                          |``fosscuda/2020b``
``2.1.2``|                          |``foss/2021a``
``2.1.2``|``-CUDA-11.3.1``          |``foss/2021a``
``2.1.2``|``-TensorFlow-2.5.0``     |``foss/2020b``, ``fosscuda/2020b``
``2.2.2``|                          |``foss/2021a``
``2.2.2``|``-CUDA-11.3.1``          |``foss/2021a``
``2.3.0``|``-CUDA-11.4.1``          |``foss/2021b``
``2.3.1``|                          |``foss/2022a``
``2.3.1``|``-CUDA-11.7.0``          |``foss/2022a``
``2.3.4``|``-CUDA-11.7.0-ColabFold``|``foss/2022a``
``2.3.4``|``-ColabFold``            |``foss/2022a``

### AlphaPulldown

AlphaPulldown is a Python package that streamlines protein-protein interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer

*homepage*: <https://github.com/KosinskiLab/AlphaPulldown>

version   |toolchain
----------|----------------------------------
``0.30.4``|``foss/2020b``, ``fosscuda/2020b``

### ALPS

The ALPS project (Algorithms and Libraries for Physics Simulations) is an open source effort  aiming at providing high-end simulation codes for strongly correlated quantum mechanical systems as well as   C++ libraries for simplifying the development of such code. 

*homepage*: <http://alps.comp-phys.org/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.b4``|``-Python-2.7.11``|``intel/2016a``
``2.3.0`` |``-Python-2.7.12``|``foss/2016b``
``2.3.0`` |``-Python-3.5.2`` |``foss/2016b``

### alsa-lib

The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality  to the Linux operating system.

*homepage*: <https://www.alsa-project.org>

version  |toolchain
---------|----------------------------------------------------------
``1.2.4``|``GCCcore/9.3.0``
``1.2.8``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### AMAPVox

LiDAR data voxelisation software

*homepage*: <https://amap-dev.cirad.fr/projects/amapvox>

version  |versionsuffix|toolchain
---------|-------------|----------
``1.9.4``|``-Java-11`` |``system``

### Amara

Library for XML processing in Python, designed to balance the native idioms of Python  with the native character of XML.

*homepage*: <https://pypi.org/project/Amara>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``1.2.0.2``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``

### amask

amask is a set of tools to to determine the affinity of MPI processes and OpenMP threads  in a parallel environment.

*homepage*: <https://github.com/TACC/amask>

version     |toolchain
------------|--------------
``20171106``|``foss/2018a``
``20190404``|``foss/2018b``

### Amber

Amber (originally Assisted Model Building with Energy Refinement) is software for performing  molecular dynamics and structure prediction.

*homepage*: <https://ambermd.org/amber.html>

version  |versionsuffix                                    |toolchain
---------|-------------------------------------------------|------------------------------------------------------------------------
``14``   |``-AmberTools-15-patchlevel-13-13``              |``intel/2016a``
``16``   |``-AmberTools-16-patchlevel-5-14``               |``iomkl/2016.07``
``16``   |``-AmberTools-16-patchlevel-5-14-CUDA``          |``iomkl/2016.09-GCC-4.9.3-2.25``
``16``   |``-AmberTools-16-patchlevel-5-14-serial``        |``iomkl/2016.07``
``16``   |``-AmberTools-17-patchlevel-10-15``              |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``16``   |``-AmberTools-17-patchlevel-10-15-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``16``   |``-AmberTools-17-patchlevel-8-12``               |``intel/2017b``
``18``   |``-AmberTools-18-patchlevel-10-8``               |``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``
``18``   |``-AmberTools-19-patchlevel-12-17-Python-2.7.16``|``foss/2019b``, ``fosscuda/2019b``
``20.11``|``-AmberTools-20.15-Python-3.8.2``               |``foss/2020a``, ``fosscuda/2020a``
``20.11``|``-AmberTools-21.3``                             |``foss/2020b``, ``fosscuda/2020b``
``22.0`` |``-AmberTools-22.3``                             |``foss/2021b``
``22.0`` |``-AmberTools-22.3-CUDA-11.4.1``                 |``foss/2021b``

### AmberMini

A stripped-down set of just antechamber, sqm, and tleap.

*homepage*: <https://github.com/choderalab/ambermini>

version    |toolchain
-----------|--------------------------------
``16.16.0``|``intel/2017b``, ``intel/2020a``

### AmberTools

AmberTools consists of several independently developed packages that work well by themselves,  and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,  with either explicit water or generalized Born solvent models.

*homepage*: <https://ambermd.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------
``17``   |                  |``intel/2017b``, ``intel/2018a``
``17``   |``-Python-2.7.14``|``foss/2018a``
``20``   |``-Python-3.8.2`` |``intel/2020a``
``21``   |                  |``foss/2021a``, ``intel/2021b``
``21.12``|                  |``foss/2021b``
``22.3`` |                  |``foss/2021b``

### AMD-LibM

AMD LibM is a software library containing a collection of basic math functions optimized for x86-64 processor based machines.

*homepage*: <https://developer.amd.com/amd-cpu-libraries/amd-math-library-libm/>

version    |toolchain
-----------|------------------
``3.2.2``  |``GCC/7.3.0-2.30``
``3.6.0-4``|``GCC/9.3.0``

### AMD-RNG

AMD Random Number Generator Library is a pseudorandom number generator library.

*homepage*: <https://developer.amd.com/amd-cpu-libraries/rng-library/>

version  |toolchain
---------|------------------
``1.0``  |``GCC/7.3.0-2.30``
``2.2-4``|``GCC/9.3.0``

### AMD-SecureRNG

The AMD Secure Random Number Generator (RNG) is a library that provides APIs to access the cryptographically secure random numbers generated by AMD’s hardware-based random number generator implementation.

*homepage*: <https://developer.amd.com/amd-cpu-libraries/rng-library/>

version  |toolchain
---------|------------------
``1.0``  |``GCC/7.3.0-2.30``
``2.2-4``|``GCC/9.3.0``

### AMD-uProf

AMD uProf is a performance analysis tool for applications running on Windows, Linux & FreeBSD  operating systems. It allows developers to better understand the runtime performance of their application and  to identify ways to improve its performance.

*homepage*: <https://developer.amd.com/amd-uprof/>

version    |toolchain
-----------|----------
``3.4.502``|``system``
``3.5.671``|``system``

### AMOS

The AMOS consortium is committed to the development of open-source whole genome assembly software

*homepage*: <http://amos.sourceforge.net>

version  |toolchain
---------|------------------------------
``3.1.0``|``foss/2018b``, ``foss/2021b``

### AMPHORA2

An Automated Phylogenomic Inference Pipeline for Bacterial and Archaeal Sequences.

*homepage*: <https://github.com/wu-lab-uva/AMPHORA2>

version     |versionsuffix             |toolchain
------------|--------------------------|---------------
``20190730``|``-Java-13-pthreads-avx2``|``gompi/2020b``

### AMPL-MP

An open-source library for mathematical programming. 

*homepage*: <https://github.com/ampl/mp>

version  |toolchain
---------|-----------------
``3.1.0``|``GCCcore/6.4.0``

### amplimap

amplimap is a command-line tool to automate the processing and analysis of data from targeted  next-generation sequencing (NGS) experiments with PCR-based amplicons or capture-based enrichment systems.

*homepage*: <https://amplimap.readthedocs.io/en/latest/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.4.16``|``-Python-3.6.6``|``foss/2018b``

### AMPtk

AMPtk is a series of scripts to process NGS amplicon data using USEARCH and VSEARCH, it can also be used to process any NGS amplicon data and includes databases setup for analysis of fungal ITS, fungal LSU, bacterial 16S, and insect COI amplicons.

*homepage*: <https://amptk.readthedocs.io>

version  |toolchain
---------|--------------
``1.5.4``|``foss/2021b``

### AmrPlusPlus

AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. 

*homepage*: <https://megares.meglab.org/amrplusplus/latest/html/v2>

version         |toolchain
----------------|-------------
``2.0-20200114``|``GCC/8.3.0``

### AMS

The Amsterdam Modeling Suite (AMS) provides a comprehensive set of modules for computational chemistry and materials science, from quantum mechanics to fluid thermodynamics. 

*homepage*: <https://www.scm.com/amsterdam-modeling-suite/>

version     |versionsuffix|toolchain
------------|-------------|---------------
``2020.102``|``-intelmpi``|``iimpi/2020b``
``2022.102``|``-intelmpi``|``iimpi/2021b``
``2023.101``|``-intelmpi``|``iimpi/2022a``

### Anaconda2

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: <https://www.anaconda.com>

version    |toolchain
-----------|----------
``4.0.0``  |``system``
``4.2.0``  |``system``
``4.4.0``  |``system``
``5.0.1``  |``system``
``5.1.0``  |``system``
``5.3.0``  |``system``
``2018.12``|``system``
``2019.03``|``system``
``2019.07``|``system``
``2019.10``|``system``

### Anaconda3

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: <https://www.anaconda.com>

version      |toolchain
-------------|----------
``4.0.0``    |``system``
``4.2.0``    |``system``
``4.4.0``    |``system``
``5.0.1``    |``system``
``5.1.0``    |``system``
``5.3.0``    |``system``
``2018.12``  |``system``
``2019.03``  |``system``
``2019.07``  |``system``
``2019.10``  |``system``
``2020.02``  |``system``
``2020.07``  |``system``
``2020.11``  |``system``
``2021.05``  |``system``
``2021.11``  |``system``
``2022.05``  |``system``
``2022.10``  |``system``
``2023.03-1``|``system``

### aNCI

Non-covalent interaction (NCI) for MD trajectories

*homepage*: <https://www.lct.jussieu.fr/pagesperso/contrera/nci-MD.html>

version|toolchain
-------|-----------------------
``2.0``|``iccifort/2019.5.281``

### andi

This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes.

*homepage*: <https://github.com/evolbioinf/andi/>

version |toolchain
--------|--------------
``0.13``|``foss/2018b``

### ANGEL

ANGEL: Robust Open Reading Frame prediction

*homepage*: <https://github.com/PacificBiosciences/ANGEL>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``3.0``|``-Python-3.7.2``|``foss/2019a``

### angsd

Program for analysing NGS data.

*homepage*: <http://www.popgen.dk/angsd>

version  |toolchain
---------|--------------------------------------
``0.910``|``foss/2016a``
``0.921``|``foss/2018a``
``0.925``|``foss/2018b``
``0.933``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.935``|``GCC/10.2.0``
``0.940``|``GCC/11.2.0``

### ANIcalculator

This tool will calculate the bidirectional average nucleotide identity (gANI) and  Alignment Fraction (AF) between two genomes. Required input is the full path to the fna file  (nucleotide sequence of genes in fasta format) of each query genome. Either the rRNA and tRNA genes  can be excluded, or provided in a list with the -ignoreList option. This is necessary as the presence of tRNA and/or rRNA genes in the fna will artificially inflate the ANI.

*homepage*: <https://ani.jgi.doe.gov/html/home.php>

version|toolchain
-------|--------------------------------------
``1.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### anndata

anndata is a Python package for handling annotated data matrices in memory and on disk,  positioned between pandas and xarray

*homepage*: <https://github.com/scverse/anndata>

version  |toolchain
---------|--------------
``0.8.0``|``foss/2022a``

### Annif

Annif is a multi-algorithm automated subject indexing tool for libraries, archives and museums.

*homepage*: <https://github.com/NatLibFi/Annif>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.40.0``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``

### Annocript

Annocript is a pipeline for the annotation of de-novo generated transcriptomes.  It executes blast analysis with UniProt, NCBI Conserved Domain Database and Nucleotide division  adding also annotations from Gene Ontology, the Enzyme Commission and UniPathways.  Annocript also gives information about the longest ORF and the non-coding potential using external software.  Annocript is also capable to identify putative long non-coding RNAs by using an heuristic based on homology  and sequence features. 

*homepage*: <https://github.com/frankMusacchia/Annocript>

version|toolchain
-------|--------------
``2.0``|``foss/2022a``

### annovar

ANNOVAR is an efficient software tool to utilize update-to-date information  to functionally annotate genetic variants detected from diverse genomes (including human  genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others).

*homepage*: <http://annovar.openbioinformatics.org/en/latest/>

version      |versionsuffix   |toolchain
-------------|----------------|------------------
``2016Feb01``|``-Perl-5.22.1``|``foss/2016a``
``20191024`` |``-Perl-5.28.1``|``GCCcore/8.2.0``
``20200607`` |``-Perl-5.34.0``|``GCCcore/11.2.0``

### ANSYS

ANSYS simulation software enables organizations to confidently predict      how their products will operate in the real world. We believe that every product is      a promise of something greater. 

*homepage*: <https://www.ansys.com>

version   |toolchain
----------|----------
``15.0``  |``system``
``2022R2``|``system``
``2023R1``|``system``

### ANSYS_CFD

ANSYS computational fluid dynamics (CFD) simulation software allows you to predict, with confidence,  the impact of fluid flows on your product throughout design and manufacturing as well as during end use.  ANSYS renowned CFD analysis tools include the widely used and well-validated ANSYS Fluent and ANSYS CFX.

*homepage*: <http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics>

version |toolchain
--------|----------
``16.2``|``system``
``17.0``|``system``

### ant

Apache Ant is a Java library and command-line tool whose mission is to drive processes described in  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of  Java applications.

*homepage*: <http://ant.apache.org/>

version    |versionsuffix      |toolchain
-----------|-------------------|----------
``1.8.4``  |``-Java-1.7.0_10`` |``system``
``1.8.4``  |``-Java-1.7.0_21`` |``system``
``1.9.0``  |``-Java-1.7.0_15`` |``system``
``1.9.0``  |``-Java-1.7.0_21`` |``system``
``1.9.3``  |``-Java-1.7.0_60`` |``system``
``1.9.3``  |``-Java-1.7.0_79`` |``system``
``1.9.6``  |``-Java-1.7.0_75`` |``system``
``1.9.6``  |``-Java-1.7.0_79`` |``system``
``1.9.6``  |``-Java-1.7.0_80`` |``system``
``1.9.6``  |``-Java-1.8.0_66`` |``system``
``1.9.6``  |``-Java-1.8.0_72`` |``system``
``1.9.6``  |``-Java-1.8.0_77`` |``system``
``1.9.7``  |``-Java-1.8.0_92`` |``system``
``1.10.0`` |``-Java-1.8.0_112``|``system``
``1.10.1`` |``-Java-1.8``      |``system``
``1.10.1`` |``-Java-1.8.0_121``|``system``
``1.10.1`` |``-Java-1.8.0_144``|``system``
``1.10.1`` |``-Java-1.8.0_152``|``system``
``1.10.1`` |``-Java-1.8.0_162``|``system``
``1.10.5`` |``-Java-1.8``      |``system``
``1.10.6`` |``-Java-1.8``      |``system``
``1.10.7`` |``-Java-11``       |``system``
``1.10.8`` |``-Java-11``       |``system``
``1.10.9`` |``-Java-11``       |``system``
``1.10.11``|``-Java-11``       |``system``
``1.10.11``|``-Java-13``       |``system``
``1.10.12``|``-Java-11``       |``system``
``1.10.12``|``-Java-17``       |``system``

### antiSMASH

antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary  metabolite biosynthesis gene clusters in bacterial and fungal genomes.

*homepage*: <https://github.com/antismash/antismash>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``5.1.2``|``-Python-3.7.4``|``foss/2019b``
``5.2.0``|                 |``foss/2020b``
``6.0.1``|                 |``foss/2020b``

### ANTLR

ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.

*homepage*: <http://www.antlr2.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------------------------------------------------------------
``2.7.7``|                  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018b``, ``foss/2019a``, ``intel/2017b``
``2.7.7``|``-Java-11``      |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.7.7``|``-Python-2.7.11``|``foss/2016a``
``2.7.7``|``-Python-2.7.13``|``intel/2017a``
``2.7.7``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``2.7.7``|``-Python-3.6.4`` |``intel/2018a``

### ANTs

ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,  interpreting and visualizing multidimensional data.

*homepage*: <https://stnava.github.io/ANTs/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.2.0``|``-Python-2.7.12``|``foss/2016b``
``2.3.0``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.3.1``|``-Python-3.6.6`` |``foss/2018b``
``2.3.2``|``-Python-3.7.4`` |``foss/2019b``
``2.3.5``|                  |``foss/2021a``

### anvio

An analysis and visualization platform for 'omics data.

*homepage*: <https://merenlab.org/software/anvio/>

version|versionsuffix    |toolchain
-------|-----------------|---------------
``6.1``|``-Python-3.7.4``|``intel/2019b``

### any2fasta

Convert various sequence formats to FASTA

*homepage*: <https://github.com/tseemann/any2fasta>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------
``0.4.2``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.4.2``|``-Perl-5.28.1``|``GCCcore/8.2.0``

### AOCC

AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 13.0

*homepage*: <https://developer.amd.com/amd-aocc/>

version  |toolchain
---------|----------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.0.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.2.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.0.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### AOMP

AMD fork of LLVM, setup for OpenMP offloading to Accelerators

*homepage*: <https://github.com/ROCm-Developer-Tools/aomp>

version   |toolchain
----------|-------------------------------------
``13.0-2``|``GCCcore/10.2.0``, ``gcccuda/2020a``

### APBS

APBS is a software package for modeling biomolecular solvation   through solution of the Poisson-Boltzmann equation (PBE), one of the most popular   continuum models for describing electrostatic interactions between molecular solutes   in salty, aqueous media. 

*homepage*: <http://www.poissonboltzmann.org/apbs>

version|versionsuffix           |toolchain
-------|------------------------|----------
``1.4``|``-linux-static-x86_64``|``system``

### apex

A PyTorch Extension: Tools for simple mixed precision and distributed training in Pytorch

*homepage*: <https://github.com/nvidia/apex>

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``20200325``|``-Python-3.7.4``|``fosscuda/2019b``
``20210420``|                 |``fosscuda/2020b``

### APR

Apache Portable Runtime (APR) libraries.

*homepage*: <https://apr.apache.org/>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``1.6.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``iomkl/2018a``
``1.7.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``

### APR-util

Apache Portable Runtime (APR) util libraries.

*homepage*: <http://apr.apache.org/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.6.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``, ``iomkl/2018a``

### AptaSUITE

A full-featured bioinformatics software collection for the  comprehensive analysis of aptamers in HT-SELEX experiments 

*homepage*: <https://drivenbyentropy.github.io/>

version  |versionsuffix|toolchain
---------|-------------|----------
``0.9.4``|``-Java-11`` |``system``

### ARAGORN

a program to detect tRNA genes and tmRNA genes in nucleotide sequences

*homepage*: <http://www.ansikte.se/ARAGORN/>

version   |toolchain
----------|---------------------------------------
``1.2.38``|``foss/2016b``, ``iccifort/2019.5.281``
``1.2.41``|``foss/2021b``

### Arb

Arb is a C library for arbitrary-precision interval arithmetic.  It has full support for both real and complex numbers. The library is thread-safe,  portable, and extensively tested.

*homepage*: <https://arblib.org/>

version   |toolchain
----------|--------------------------------------------------------------------------------
``2.16.0``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2.17.0``|``GCC/8.3.0``
``2.19.0``|``GCC/10.3.0``
``2.22.1``|``foss/2021b``
``2.23.0``|``gfbf/2022a``

### Arcade-Learning-Environment

The Arcade Learning Environment (ALE) is a simple framework that allows researchers and hobbyists to develop AI agents for Atari 2600 games. It is built on top of the Atari 2600 emulator Stella and separates the details of emulation from agent design. This video depicts over 50 games currently supported in the ALE.

*homepage*: <https://github.com/mgbellemare/Arcade-Learning-Environment>

version  |toolchain
---------|--------------
``0.7.3``|``foss/2021b``

### arcasHLA

arcasHLA performs high resolution genotyping for HLA class I and class II genes from RNA sequencing, supporting both paired and single-end samples.

*homepage*: <https://github.com/RabadanLab/arcasHLA>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.7.4``|``foss/2019b``

### ARCH

Autoregressive Conditional Heteroskedasticity (ARCH) and other tools for financial econometrics,  written in Python (with Cython and/or Numba used to improve performance).

*homepage*: <https://pypi.org/project/arch>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``4.5.0``|``-Python-3.6.4``|``intel/2018a``

### Archive-Zip

Provide an interface to ZIP archive files.

*homepage*: <https://metacpan.org/pod/Archive::Zip>

version |toolchain
--------|------------------------------------------------------------------------------
``1.68``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### ArchR

ArchR is a full-featured R package for processing and analyzing single-cell ATAC-seq data.

*homepage*: <https://www.archrproject.com>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-4.1.2`` |``foss/2021b``

### archspec

A library for detecting, labeling, and reasoning about microarchitectures

*homepage*: <https://github.com/archspec/archspec>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``0.1.0``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.1.0``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.1.2``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``0.1.3``|                 |``GCCcore/11.2.0``
``0.1.4``|                 |``GCCcore/11.3.0``
``0.2.0``|                 |``GCCcore/12.2.0``

### ARGoS

A parallel, multi-engine simulator for heterogeneous swarm robotics

*homepage*: <http://www.argos-sim.info>

version         |versionsuffix |toolchain
----------------|--------------|--------------
``3.0.0-beta53``|``-Lua-5.2.4``|``foss/2018b``
``3.0.0-beta59``|              |``GCC/11.2.0``

### argtable

Argtable is an ANSI C library for parsing GNU style  command line options with a minimum of fuss. 

*homepage*: <http://argtable.sourceforge.net/>

version |toolchain
--------|----------------------------------------------------------------------------------------------------------------------------------------------
``2.13``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``

### aria2

aria2 is a lightweight multi-protocol & multi-source command-line download utility.

*homepage*: <https://aria2.github.io>

version   |toolchain
----------|------------------
``1.35.0``|``GCCcore/10.3.0``
``1.36.0``|``GCCcore/11.3.0``

### Arlequin

Arlequin: An Integrated Software for Population Genetics Data Analysis

*homepage*: <http://cmpg.unibe.ch/software/arlequin35/Arlequin35.html>

version    |toolchain
-----------|--------------
``3.5.2.2``|``foss/2019b``

### Armadillo

Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards  a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,  as well as a subset of trigonometric and statistics functions.

*homepage*: <https://arma.sourceforge.net/>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``7.600.2``|``-Python-2.7.12``|``foss/2016b``
``7.800.0``|``-Python-2.7.12``|``intel/2016b``
``7.950.1``|``-Python-2.7.12``|``intel/2016b``
``8.300.1``|                  |``foss/2017b``, ``intel/2017b``
``8.400.0``|                  |``foss/2018a``
``9.600.5``|                  |``foss/2018b``
``9.700.2``|                  |``foss/2019a``
``9.880.1``|                  |``foss/2020a``
``9.900.1``|                  |``foss/2019b``, ``foss/2020a``
``10.5.3`` |                  |``foss/2020b``
``10.7.5`` |                  |``foss/2021a``
``11.4.3`` |                  |``foss/2022a``, ``foss/2022b``

### arosics

AROSICS is a python package to perform automatic subpixel co-registration of two satellite image datasets based on an image matching approach working in the frequency domain, combined with a multistage workflow for effective detection of false-positives.

*homepage*: <https://github.com/GFZ/arosics>

version  |toolchain
---------|--------------
``1.7.6``|``foss/2021a``

### ARPACK++

Arpackpp is a C++ interface to the ARPACK Fortran package,  which implements the implicit restarted Arnoldi method for  iteratively solving large-scale sparse eigenvalue problems.

*homepage*: <https://github.com/m-reuter/arpackpp>

version       |toolchain
--------------|--------------
``2018.03.26``|``foss/2017b``

### arpack-ng

ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.

*homepage*: <https://github.com/opencollab/arpack-ng>

version  |toolchain
---------|--------------------------------------------------------------------------------
``3.3.0``|``foss/2016a``, ``intel/2016a``
``3.4.0``|``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``
``3.5.0``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``
``3.6.2``|``intel/2018a``
``3.7.0``|``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``3.8.0``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``foss/2022b``

### ArrayFire

ArrayFire is a general-purpose library that simplifies the process of  developing software that targets parallel and massively-parallel architectures  including CPUs, GPUs, and other hardware acceleration devices. 

*homepage*: <https://arrayfire.com/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.6.4``|                |``foss/2018b``
``3.6.4``|``-CUDA-9.2.88``|``foss/2018b``

### Arriba

Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.  It was developed for the use in a clinical research setting. Therefore, short runtimes and high  sensitivity were important design criteria.

*homepage*: <https://github.com/suhrig/arriba>

version  |toolchain
---------|------------------------------
``1.1.0``|``foss/2018b``
``2.1.0``|``GCC/10.2.0``, ``GCC/10.3.0``
``2.2.1``|``GCC/11.2.0``
``2.3.0``|``GCC/11.2.0``

### Arrow

Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform  for in-memory data.

*homepage*: <https://arrow.apache.org>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.7.1`` |``-Python-3.6.3`` |``intel/2017b``
``0.12.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.12.0``|``-Python-3.6.6`` |``intel/2018b``
``0.16.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.17.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.17.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``6.0.0`` |                  |``foss/2021a``, ``foss/2021b``
``6.0.1`` |                  |``foss/2021a``
``8.0.0`` |                  |``foss/2022.05``, ``foss/2022a``
``11.0.0``|                  |``gfbf/2022b``

### arrow-R

R interface to the Apache Arrow C++ library

*homepage*: <https://cran.r-project.org/web/packages/arrow>

version     |versionsuffix|toolchain
------------|-------------|--------------
``0.17.1``  |``-R-4.0.0`` |``foss/2020a``
``6.0.0.2`` |``-R-4.1.0`` |``foss/2021a``
``6.0.0.2`` |``-R-4.1.2`` |``foss/2021b``
``6.0.0.2`` |``-R-4.2.0`` |``foss/2021b``
``8.0.0``   |``-R-4.2.1`` |``foss/2022a``
``11.0.0.3``|``-R-4.2.2`` |``foss/2022b``

### ART

ART is a set of simulation tools to generate synthetic next-generation sequencing reads

*homepage*: <http://www.niehs.nih.gov/research/resources/software/biostatistics/art/>

version       |toolchain
--------------|----------------------------------
``2016.06.05``|``GCCcore/6.4.0``, ``intel/2016b``

### artic-ncov2019

Initial implementation of an ARTIC bioinformatics platform  for nanopore sequencing of nCoV2019 novel coronavirus.

*homepage*: <https://github.com/artic-network/artic-ncov2019>

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2020.04.13``|``-Python-3.6.6``|``foss/2018b``
``2021.06.24``|                 |``foss/2020b``

### ARTS

ARTS is a radiative transfer model for the millimeter and sub-millimeter  spectral range. There are a number of models mostly developed explicitly  for the different sensors.

*homepage*: <http://www.radiativetransfer.org/>

version   |toolchain
----------|---------------
``2.2.64``|``gompi/2019a``

### ArviZ

Exploratory analysis of Bayesian models with Python

*homepage*: <https://github.com/arviz-devs/arviz>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.7.0`` |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``0.11.1``|                 |``intel/2020b``
``0.11.4``|                 |``foss/2021b``, ``intel/2021b``
``0.12.1``|                 |``foss/2022a``, ``intel/2022a``

### ARWEN

ARWEN, tRNA detection in metazoan mitochondrial sequences

*homepage*: <http://mbio-serv2.mbioekol.lu.se/ARWEN>

version  |toolchain
---------|-----------------
``1.2.3``|``GCCcore/7.3.0``

### ASAP

ASAP focuses on fast and fluid image viewing with an simple-to-use interface  for making annotations. It consists of two main components: an IO library for reading and writing  multi-resolution images and a viewer component for visualizing such images.

*homepage*: <https://computationalpathologygroup.github.io/ASAP/>

version|versionsuffix   |toolchain
-------|----------------|--------------
``2.0``|``-CUDA-11.3.1``|``foss/2021a``
``2.1``|                |``foss/2022a``

### ASAP3

ASAP is a calculator for doing large-scale classical molecular dynamics within the Campos Atomic Simulation Environment (ASE).

*homepage*: <https://wiki.fysik.dtu.dk/asap/>

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------------------------
``3.10.7`` |``-Python-3.5.2``|``foss/2016b``
``3.10.7`` |``-Python-3.6.2``|``foss/2017b``
``3.10.8`` |``-Python-3.5.2``|``foss/2016b``
``3.10.8`` |``-Python-3.6.2``|``foss/2017b``
``3.10.8`` |``-Python-3.6.3``|``foss/2017b``
``3.10.10``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.11.10``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``
``3.12.2`` |``-ASE-3.21.1``  |``foss/2020b``, ``intel/2020b``
``3.12.7`` |``-ASE-3.21.1``  |``foss/2020b``, ``intel/2020b``

### ASCAT

ASCAT is a method to derive copy number profiles of tumor cells,  accounting for normal cell admixture and tumor aneuploidy.

*homepage*: <https://github.com/VanLoo-lab/ascat>

version  |versionsuffix|toolchain
---------|-------------|--------------
``3.1.2``|``-R-4.2.1`` |``foss/2022a``

### ASE

ASE is a python package providing an open source Atomic Simulation Environment  in the Python scripting language.

*homepage*: <https://wiki.fysik.dtu.dk/ase/>

version       |versionsuffix     |toolchain
--------------|------------------|-------------------------------------------------------------------------------------------------
``3.9.1.4567``|``-Python-2.7.11``|``intel/2016a``
``3.10.0``    |``-Python-2.7.11``|``intel/2016.02-GCC-4.9``
``3.11.0``    |``-Python-2.7.12``|``intel/2016b``
``3.13.0``    |``-Python-2.7.12``|``foss/2016b``
``3.13.0``    |``-Python-2.7.13``|``intel/2017a``
``3.15.0``    |``-Python-2.7.12``|``foss/2016b``
``3.15.0``    |``-Python-2.7.14``|``intel/2017b``
``3.15.0``    |``-Python-3.5.2`` |``foss/2016b``
``3.15.0``    |``-Python-3.6.2`` |``foss/2017b``
``3.15.0``    |``-Python-3.6.3`` |``foss/2017b``
``3.16.2``    |``-Python-3.6.4`` |``iomkl/2018.02``, ``iomkl/2018a``
``3.16.2``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.18.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.18.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.19.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.19.0``    |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.19.0``    |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``3.20.1``    |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.20.1``    |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``3.20.1``    |``-Python-3.8.2`` |``intel/2020a``
``3.21.1``    |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.21.1``    |``-Python-3.8.2`` |``foss/2020a``
``3.22.0``    |                  |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021a``
``3.22.1``    |                  |``foss/2021b``, ``foss/2022a``, ``gfbf/2022b``, ``gomkl/2021a``, ``intel/2021b``, ``intel/2022a``

### ASF-SearchAPI

asf_search is a Python module for performing searches of the ASF catalog. In addition, it offers baseline functionality and download support.

*homepage*: <https://docs.asf.alaska.edu/>

version  |toolchain
---------|--------------
``6.5.0``|``foss/2022a``

### ASHS

Automatic Segmentation of Hippocampal Subfields (ASHS) 

*homepage*: <https://sites.google.com/site/hipposubfields/home>

version            |toolchain
-------------------|----------
``rev103_20140612``|``system``

### Aspera-CLI

IBM Aspera Command-Line Interface (the Aspera CLI) is a collection of Aspera tools for performing high-speed, secure data transfers from the command line. The Aspera CLI is for users and organizations who want to automate their transfer workflows.

*homepage*: <https://asperasoft.com>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``3.7.2``|``.354.010c3b8`` |``system``
``3.9.0``|``.1326.6985b21``|``system``
``3.9.6``|``.1467.159c5b1``|``system``

### Aspera-Connect

Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and  downloads with an Aspera transfer server.

*homepage*: <http://downloads.asperasoft.com/connect2/>

version  |toolchain
---------|----------
``3.6.1``|``system``
``3.9.6``|``system``

### assimp

Open Asset Import Library (assimp) is a library to import and export various  3d-model-formats including scene-post-processing to generate missing render data. 

*homepage*: <https://github.com/assimp/assimp>

version  |toolchain
---------|-----------------
``5.0.1``|``GCCcore/8.3.0``

### Assimulo

Assimulo is a simulation package for solving ordinary differential equations.

*homepage*: <https://jmodelica.org/assimulo/>

version|versionsuffix     |toolchain
-------|------------------|---------------
``2.9``|``-Python-2.7.15``|``intel/2018b``

### ASTRID

ASTRID-2 is a method for estimating species trees from gene trees.

*homepage*: <https://github.com/pranjalv123/ASTRID>

version  |toolchain
---------|---------------
``2.2.1``|``gompi/2019a``

### astro-tulips

tulips creates diagrams of the structure and evolution of stars. It creates plots and movies based on output from the MESA stellar evolution code

*homepage*: <https://astro-tulips.readthedocs.io/en/latest/installation.html>

version  |toolchain
---------|--------------
``1.0.1``|``foss/2022a``

### astropy

The Astropy Project is a community effort to develop a common  core package for Astronomy in Python and foster an ecosystem of interoperable  astronomy packages.  The Astropy community is committed to supporting diversity and inclusion.

*homepage*: <https://www.astropy.org/>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------
``2.0.12``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.0.14``|                  |``foss/2019a``
``4.0.1`` |``-Python-3.7.4`` |``foss/2019b``
``4.0.1`` |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``4.2.1`` |                  |``foss/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``4.3.1`` |                  |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``5.0.4`` |                  |``foss/2021a``
``5.1.1`` |                  |``foss/2022a``
``5.2.2`` |                  |``gfbf/2022b``

### at-spi2-atk

AT-SPI 2 toolkit bridge

*homepage*: <https://wiki.gnome.org/Accessibility>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------
``2.26.3``|``fosscuda/2018b``
``2.32.0``|``GCCcore/8.2.0``
``2.34.1``|``GCCcore/8.3.0``
``2.34.2``|``GCCcore/9.3.0``
``2.38.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### at-spi2-core

Assistive Technology Service Provider Interface. 

*homepage*: <https://wiki.gnome.org/Accessibility>

version   |toolchain
----------|------------------
``2.26.3``|``fosscuda/2018b``
``2.32.0``|``GCCcore/8.2.0``
``2.34.0``|``GCCcore/8.3.0``
``2.36.0``|``GCCcore/9.3.0``
``2.38.0``|``GCCcore/10.2.0``
``2.40.2``|``GCCcore/10.3.0``
``2.40.3``|``GCCcore/11.2.0``
``2.44.1``|``GCCcore/11.3.0``
``2.46.0``|``GCCcore/12.2.0``

### ATAT

The Alloy-Theoretic Automated Toolkit (ATAT) is a generic name that refers to a collection of alloy theory tools 

*homepage*: <https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/>

version |toolchain
--------|-----------------
``3.36``|``GCCcore/9.3.0``

### ATK

ATK provides the set of accessibility interfaces that are implemented by other  toolkits and applications. Using the ATK interfaces, accessibility tools have  full access to view and control running applications. 

*homepage*: <https://developer.gnome.org/atk/>

version   |toolchain
----------|-----------------------------------------------------------------------------
``2.18.0``|``intel/2016a``
``2.20.0``|``foss/2016a``, ``intel/2016a``
``2.22.0``|``foss/2016b``, ``intel/2016b``
``2.26.0``|``intel/2017a``
``2.26.1``|``foss/2018b``, ``intel/2017b``
``2.27.1``|``foss/2017b``, ``intel/2017b``
``2.28.1``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``
``2.32.0``|``GCCcore/8.2.0``
``2.34.1``|``GCCcore/8.3.0``
``2.36.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.38.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### ATLAS

ATLAS (Automatically Tuned Linear Algebra Software) is the application of  the AEOS (Automated Empirical Optimization of Software) paradigm, with the present emphasis   on the Basic Linear Algebra Subprograms (BLAS), a widely used, performance-critical, linear   algebra kernel library.

*homepage*: <http://math-atlas.sourceforge.net>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``3.10.2``|``-LAPACK-3.6.1``|``GCC/5.4.0-2.26``

### atomate

atomate has implementations of FireWorks workflows for Materials Science.

*homepage*: <https://pythonhosted.org/atomate/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.4.4``|``-Python-2.7.13``|``intel/2017a``

### AtomPAW

AtomPAW is a Projector-Augmented Wave Dataset Generator that  can be used both as a standalone program and a library.

*homepage*: <http://users.wfu.edu/natalie/papers/pwpaw/man.html>

version    |toolchain
-----------|---------------
``4.1.0.5``|``intel/2018b``
``4.1.0.6``|``intel/2018b``

### atools

Tools to make using job arrays a lot more convenient.

*homepage*: <https://github.com/gjbex/atools>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``1.4.2``|``-Python-2.7.12``|``intel/2016b``
``1.4.6``|``-Python-2.7.16``|``GCCcore/8.3.0``
``1.4.8``|``-Python-2.7.18``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.5.1``|                  |``GCCcore/11.2.0``

### atropos

Atropos is tool for specific, sensitive, and speedy trimming of NGS reads. 

*homepage*: <https://atropos.readthedocs.io>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.1.21``|``-Python-3.6.6``|``intel/2018b``

### ATSAS

ATSAS is a program suite for small-angle scattering data analysis from biological macromolecules.

*homepage*: <http://www.embl-hamburg.de/ExternalInfo/Research/Sax/software.html>

version    |versionsuffix  |toolchain
-----------|---------------|----------
``2.5.1-1``|``.el6.x86_64``|``system``
``2.5.1-1``|``.sl5.x86_64``|``system``
``2.7.1-1``|``.el7.x86_64``|``system``

### attr

Commands for Manipulating Filesystem Extended Attributes

*homepage*: <https://savannah.nongnu.org/projects/attr>

version   |toolchain
----------|----------------------------------------------------------
``2.4.47``|``GCCcore/8.2.0``
``2.4.48``|``GCCcore/9.3.0``
``2.5.1`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### attrdict

AttrDict is a Python library that provides mapping objects that allow their elements  to be accessed both as keys and as attributes.

*homepage*: <https://github.com/bcj/AttrDict>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``2.0.1``|``-Python-3.7.4``|``GCCcore/8.3.0``

### attrdict3

AttrDict is a Python library that provides mapping objects that allow their elements  to be accessed both as keys and as attributes.

*homepage*: <https://github.com/pirofti/AttrDict3>

version  |toolchain
---------|------------------
``2.0.2``|``GCCcore/11.2.0``

### augur

Pipeline components for real-time phylodynamic analysis

*homepage*: <https://github.com/nextstrain/augur>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``7.0.2``|``-Python-3.7.4``|``intel/2019b``

### AUGUSTUS

AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences

*homepage*: <https://bioinf.uni-greifswald.de/augustus/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------
``3.2.3``|``-Python-2.7.13``|``intel/2017a``
``3.3``  |                  |``foss/2018a``
``3.3.2``|                  |``intel/2019a``
``3.3.2``|``-Python-2.7.13``|``intel/2017a``
``3.3.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``3.3.2``|``-Python-2.7.15``|``intel/2018b``
``3.3.3``|                  |``foss/2019b``, ``intel/2019b``
``3.4.0``|                  |``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``
``3.5.0``|                  |``foss/2022a``, ``foss/2022b``

### Austin

Austin is a Python frame stack sampler for CPython written in pure C.

*homepage*: <https://github.com/P403n1x87/austin>

version  |toolchain
---------|------------------------------
``3.2.0``|``GCCcore/11.2.0``, ``system``

### AUTO-07p

AUTO is a publicly available software for continuation and bifurcation problems in ordinary differential equations originally written in 1980 and widely used in the dynamical systems community.

*homepage*: <https://github.com/auto-07p>

version  |toolchain
---------|--------------
``0.9.3``|``foss/2021a``

### Autoconf

Autoconf is an extensible package of M4 macros that produce shell scripts  to automatically configure software source code packages. These scripts can adapt the  packages to many kinds of UNIX-like systems without manual user intervention. Autoconf  creates a configuration script for a package from a template file that lists the  operating system features that the package can use, in the form of M4 macro calls.

*homepage*: <https://www.gnu.org/software/autoconf/>

version |toolchain
--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.68``|``foss/2016b``
``2.69``|``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/10.2.0``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.71``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### Autoconf-archive

The GNU Autoconf Archive is a collection of more than 500 macros for GNU Autoconf that have been contributed as free software by friendly supporters of the cause from all over the Internet. Every single one of those macros can be re-used without imposing any restrictions whatsoever on the licensing of the generated configure script. In particular, it is possible to use all those macros in configure scripts that are meant for non-free software. This policy is unusual for a Free Software Foundation project. The FSF firmly believes that software ought to be free, and software licenses like the GPL are specifically designed to ensure that derivative work based on free software must be free as well. In case of Autoconf, however, an exception has been made, because Autoconf is at such a pivotal position in the software development tool chain that the benefits from having this tool available as widely as possible outweigh the disadvantage that some authors may choose to use it, too, for proprietary software. 

*homepage*: <https://www.gnu.org/software/autoconf-archive>

version       |toolchain
--------------|--------------------
``2019.01.06``|``GCC/8.2.0-2.31.1``
``2021.02.19``|``GCCcore/10.2.0``

### AutoDock

AutoDock is a suite of automated docking tools. It is designed to  predict how small molecules, such as substrates or drug candidates, bind to  a receptor of known 3D structure.

*homepage*: <http://autodock.scripps.edu/>

version    |toolchain
-----------|--------------
``4.2.5.1``|``GCC/5.2.0``
``4.2.6``  |``GCC/10.3.0``

### AutoDock-GPU

OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure.

*homepage*: <https://github.com/ccsb-scripps/AutoDock-GPU>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.3``|``-CUDA-11.3.1``|``GCC/10.3.0``
``1.5.3``|``-CUDA-11.7.0``|``GCC/11.3.0``

### AutoDock-Vina

AutoDock Vina is an open-source program for doing molecular docking.

*homepage*: <https://vina.scripps.edu/>

version  |versionsuffix |toolchain
---------|--------------|------------------------------
``1.1.2``|``-linux_x86``|``system``
``1.2.3``|              |``foss/2021a``, ``foss/2021b``

### AutoDockSuite

AutoDock is a suite of automated docking tools. It is designed to  predict how small molecules, such as substrates or drug candidates, bind to  a receptor of known 3D structure. 

*homepage*: <https://autodock.scripps.edu/>

version  |toolchain
---------|-----------------
``4.2.6``|``GCCcore/8.3.0``

### AutoGeneS

AutoGeneS automatically extracts informative genes and reveals the cellular heterogeneity of bulk RNA samples.

*homepage*: <https://github.com/theislab/AutoGeneS>

version  |toolchain
---------|--------------
``1.0.4``|``foss/2020b``

### AutoGrid

AutoDock is a suite of automated docking tools. It is designed to   predict how small molecules, such as substrates or drug candidates, bind to   a receptor of known 3D structure.

*homepage*: <http://autodock.scripps.edu/>

version    |toolchain
-----------|-------------
``4.2.5.1``|``GCC/5.2.0``

### Automake

Automake: GNU Standards-compliant Makefile generator

*homepage*: <https://www.gnu.org/software/automake/automake.html>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.11.3``|``foss/2016b``
``1.14``  |``GCC/4.8.2``, ``intel/2016a``
``1.14.1``|``GCC/4.8.2``
``1.15``  |``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.15.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``1.16.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``1.16.2``|``GCCcore/10.2.0``
``1.16.3``|``FCC/4.5.0``, ``GCCcore/10.3.0``
``1.16.4``|``GCCcore/11.2.0``
``1.16.5``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### AutoMap

Tool to find regions of homozygosity (ROHs) from sequencing data.

*homepage*: <https://github.com/mquinodo/AutoMap>

version|versionsuffix|toolchain
-------|-------------|--------------
``1.0``|``-20200324``|``foss/2019b``

### autopep8

A tool that automatically formats Python code to conform to the PEP 8 style guide.

*homepage*: <https://github.com/hhatto/autopep8>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.4.4``|``-Python-3.6.4``|``intel/2018a``

### Autotools

This bundle collect the standard GNU build tools: Autoconf, Automake  and libtool 

*homepage*: <https://autotools.io>

version     |toolchain
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``20150119``|``GCC/4.9.2``
``20150215``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``20170619``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``20180311``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``20200321``|``GCCcore/10.2.0``
``20210128``|``FCC/4.5.0``, ``GCCcore/10.3.0``
``20210726``|``GCCcore/11.2.0``
``20220317``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### Avogadro2

Avogadro is an advanced molecule editor and visualizer designed for cross-platform      use in computational chemistry, molecular modeling, bioinformatics, materials science,      and related areas. 

*homepage*: <https://two.avogadro.cc/index.html>

version   |versionsuffix    |toolchain
----------|-----------------|----------
``1.97.0``|``-linux-x86_64``|``system``

### avro-cpp

C++ implementation of Avro data serialization system.

*homepage*: <https://avro.apache.org>

version   |toolchain
----------|--------------
``1.11.1``|``GCC/11.2.0``

### awscli

Universal Command Line Environment for AWS

*homepage*: <https://pypi.python.org/pypi/awscli>

version     |versionsuffix     |toolchain
------------|------------------|------------------
``1.11.1``  |``-Python-2.7.12``|``intel/2016b``
``1.11.56`` |``-Python-2.7.12``|``foss/2016b``
``1.16.290``|``-Python-3.6.6`` |``foss/2018b``
``1.17.7``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.18.89`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.0.55``  |``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.11.21`` |                  |``GCCcore/11.3.0``

### axel

Lightweight CLI download accelerator 

*homepage*: <https://github.com/axel-download-accelerator/axel>

version   |toolchain
----------|------------------------------------
``2.17.9``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``

## B


[BA3-SNPS-autotune](#ba3-snps-autotune) - [BabelStream](#babelstream) - [babl](#babl) - [Bader](#bader) - [BAGEL](#bagel) - [BAli-Phy](#bali-phy) - [bam-readcount](#bam-readcount) - [Bambi](#bambi) - [bamFilters](#bamfilters) - [BAMM](#bamm) - [BAMSurgeon](#bamsurgeon) - [bamtofastq](#bamtofastq) - [BamTools](#bamtools) - [BamUtil](#bamutil) - [Bandage](#bandage) - [barrnap](#barrnap) - [basemap](#basemap) - [Bash](#bash) - [bashplotlib](#bashplotlib) - [bat](#bat) - [batchgenerators](#batchgenerators) - [BatMeth2](#batmeth2) - [BayesAss](#bayesass) - [BayesAss3-SNPs](#bayesass3-snps) - [BayeScan](#bayescan) - [BayeScEnv](#bayescenv) - [BayesPrism](#bayesprism) - [BayesTraits](#bayestraits) - [Bazel](#bazel) - [bbcp](#bbcp) - [bbFTP](#bbftp) - [BBMap](#bbmap) - [bc](#bc) - [BCALM](#bcalm) - [bcbio-gff](#bcbio-gff) - [BCEL](#bcel) - [BCFtools](#bcftools) - [bcgTree](#bcgtree) - [bcl-convert](#bcl-convert) - [bcl2fastq2](#bcl2fastq2) - [bcolz](#bcolz) - [BDBag](#bdbag) - [Beagle](#beagle) - [beagle-lib](#beagle-lib) - [Beast](#beast) - [BeautifulSoup](#beautifulsoup) - [BEDOPS](#bedops) - [BEDTools](#bedtools) - [BEEF](#beef) - [behave](#behave) - [BerkeleyGW](#berkeleygw) - [BFAST](#bfast) - [BFC](#bfc) - [BGC-Bayesian-genomic-clines](#bgc-bayesian-genomic-clines) - [bgen](#bgen) - [bgen-reader](#bgen-reader) - [bibtexparser](#bibtexparser) - [BiG-SCAPE](#big-scape) - [BigDFT](#bigdft) - [BinSanity](#binsanity) - [binutils](#binutils) - [Bio-DB-HTS](#bio-db-hts) - [Bio-EUtilities](#bio-eutilities) - [Bio-SamTools](#bio-samtools) - [Bio-SearchIO-hmmer](#bio-searchio-hmmer) - [bioawk](#bioawk) - [biobambam2](#biobambam2) - [biogeme](#biogeme) - [biom-format](#biom-format) - [biomart-perl](#biomart-perl) - [BioPerl](#bioperl) - [BioPP](#biopp) - [Biopython](#biopython) - [BioServices](#bioservices) - [BirdNET](#birdnet) - [biscuit](#biscuit) - [Bismark](#bismark) - [Bison](#bison) - [bitarray](#bitarray) - [BLACS](#blacs) - [BLASR](#blasr) - [blasr_libcpp](#blasr_libcpp) - [BLAST](#blast) - [BLAST+](#blast+) - [BLAT](#blat) - [Blender](#blender) - [BLIS](#blis) - [Blitz++](#blitz++) - [BlobTools](#blobtools) - [Block](#block) - [Blosc](#blosc) - [Blosc2](#blosc2) - [BLT](#blt) - [bmtagger](#bmtagger) - [bnpy](#bnpy) - [BOINC](#boinc) - [bokeh](#bokeh) - [BoltzTraP](#boltztrap) - [BoltzTraP2](#boltztrap2) - [Bonito](#bonito) - [Bonmin](#bonmin) - [Bonnie++](#bonnie++) - [Boost](#boost) - [Boost.MPI](#boost.mpi) - [Boost.Python](#boost.python) - [boost_histogram](#boost_histogram) - [boto3](#boto3) - [Bottleneck](#bottleneck) - [Bowtie](#bowtie) - [Bowtie2](#bowtie2) - [Bpipe](#bpipe) - [bpp](#bpp) - [bpytop](#bpytop) - [Bracken](#bracken) - [Braindecode](#braindecode) - [BRAKER](#braker) - [BreakDancer](#breakdancer) - [breseq](#breseq) - [Brotli](#brotli) - [Brotli-python](#brotli-python) - [Brunsli](#brunsli) - [bsddb3](#bsddb3) - [BSMAPz](#bsmapz) - [Bsoft](#bsoft) - [BSseeker2](#bsseeker2) - [BuDDy](#buddy) - [BUFRLIB](#bufrlib) - [build](#build) - [buildenv](#buildenv) - [buildingspy](#buildingspy) - [Bullet](#bullet) - [BUSCO](#busco) - [BUStools](#bustools) - [BWA](#bwa) - [bwa-meth](#bwa-meth) - [bwakit](#bwakit) - [bwidget](#bwidget) - [BWISE](#bwise) - [bx-python](#bx-python) - [BXH_XCEDE_TOOLS](#bxh_xcede_tools) - [byacc](#byacc) - [byobu](#byobu) - [bzip2](#bzip2)


### BA3-SNPS-autotune

This program will automatically tune mixing parameters for BA3-SNPs by implementing  a binary search algorithm and conducting short exploratory runs of BA3-SNPS. 

*homepage*: <https://github.com/stevemussmann/BA3-SNPS-autotune>

version  |toolchain
---------|--------------
``2.1.2``|``GCC/11.3.0``

### BabelStream

STREAM, for lots of devices written in many programming models

*homepage*: <https://uob-hpc.github.io/BabelStream>

version|versionsuffix|toolchain
-------|-------------|--------------
``3.4``|``-omp``     |``GCC/11.2.0``

### babl

babl is pixel encoding and color space conversion engine in C

*homepage*: <https://www.gegl.org/babl>

version   |toolchain
----------|------------------
``0.1.86``|``GCCcore/10.3.0``

### Bader

A fast algorithm for doing Bader's analysis on a charge density grid.

*homepage*: <http://theory.cm.utexas.edu/henkelman/code/bader/>

version |toolchain
--------|---------------------------------------
``1.02``|``intel/2018a``
``1.03``|``intel/2018b``
``1.04``|``GCC/11.2.0``, ``iccifort/2020.4.304``

### BAGEL

BAGEL (Brilliantly Advanced General Electronic-structure Library) is a parallel electronic-structure program.

*homepage*: <http://www.nubakery.org>

version  |toolchain
---------|---------------
``1.1.1``|``intel/2016b``

### BAli-Phy

BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid,  or codon sequences.

*homepage*: <http://www.bali-phy.org/>

version  |toolchain
---------|----------
``3.6.0``|``system``

### bam-readcount

Count DNA sequence reads in BAM files

*homepage*: <https://github.com/genome/bam-readcount>

version  |toolchain
---------|---------------------------------------------
``0.8.0``|``GCC/11.2.0``, ``GCC/9.3.0``, ``foss/2018b``

### Bambi

Bambi is a high-level Bayesian model-building interface written in Python. It works with the probabilistic programming frameworks PyMC3 and is designed to make it extremely simple to fit Bayesian mixed-effects models common in biology, social sciences and other disciplines.

*homepage*: <https://bambinos.github.io/bambi>

version  |toolchain
---------|-------------------------------
``0.7.1``|``foss/2021b``, ``intel/2021b``

### bamFilters

A utility tool to filter alignments from bam files,  using identity percent, low complexity and read coverage.

*homepage*: <https://github.com/institut-de-genomique/bamFilters>

version       |toolchain
--------------|--------------
``2022-06-30``|``GCC/11.3.0``

### BAMM

BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates.  It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of  lineage diversification and trait evolution. 

*homepage*: <http://bamm-project.org/>

version  |toolchain
---------|--------------
``2.5.0``|``foss/2022a``

### BAMSurgeon

Tools for adding mutations to existing .bam files, used for testing mutation callers

*homepage*: <https://github.com/adamewing/bamsurgeon>

version|versionsuffix     |toolchain
-------|------------------|-------------
``1.2``|``-Python-2.7.16``|``GCC/8.3.0``

### bamtofastq

Convert 10x BAM files to the original FASTQs compatible with 10x pipelines.

*homepage*: <https://github.com/10XGenomics/bamtofastq>

version  |toolchain
---------|------------------
``1.4.0``|``GCCcore/10.3.0``

### BamTools

BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.

*homepage*: <https://github.com/pezmaster31/bamtools>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.0``|``foss/2016b``
``2.4.1``|``intel/2017a``
``2.5.0``|``foss/2016b``, ``intel/2017b``
``2.5.1``|``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel/2017b``, ``intel/2018b``
``2.5.2``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### BamUtil

BamUtil is a repository that contains several programs   that perform operations on SAM/BAM files. All of these programs   are built into a single executable, bam.

*homepage*: <http://genome.sph.umich.edu/wiki/BamUtil>

version   |toolchain
----------|---------------
``1.0.13``|``intel/2016b``
``1.0.14``|``intel/2018a``

### Bandage

Bandage is a program for visualising de novo assembly graphs

*homepage*: <http://rrwick.github.io/Bandage>

version  |versionsuffix|toolchain
---------|-------------|--------------------------------------
``0.8.1``|``_Centos``  |``system``
``0.8.1``|``_Ubuntu``  |``system``
``0.9.0``|             |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### barrnap

Barrnap (BAsic Rapid Ribosomal RNA Predictor) predicts the location of ribosomal RNA genes in genomes.

*homepage*: <https://github.com/tseemann/barrnap>

version|toolchain
-------|---------------------------------------------------------------------------------------
``0.9``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``gompi/2020b``, ``gompi/2021b``, ``gompi/2022a``

### basemap

The matplotlib basemap toolkit is a library for plotting  2D data on maps in Python

*homepage*: <https://matplotlib.org/basemap/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.7``|``-Python-2.7.13``|``intel/2017a``
``1.0.7``|``-Python-3.6.3`` |``intel/2017b``
``1.0.7``|``-Python-3.6.4`` |``intel/2018a``
``1.2.0``|``-Python-3.6.6`` |``intel/2018b``
``1.2.2``|``-Python-3.8.2`` |``foss/2020a``
``1.3.6``|                  |``foss/2022a``

### Bash

Bash is an sh-compatible command language interpreter that executes commands    read from the standard input or from a file.  Bash also incorporates useful features from the    Korn and C shells (ksh and csh).

*homepage*: <http://www.gnu.org/software/bash>

version|toolchain
-------|-------------
``4.3``|``GCC/4.9.2``

### bashplotlib

bashplotlib is a python package and command line tool for making basic plots in the terminal.  It's a quick way to visualize data when you don't have a GUI.

*homepage*: <https://github.com/glamp/bashplotlib>

version  |toolchain
---------|------------------
``0.6.5``|``GCCcore/10.3.0``

### bat

The BAT Python package supports the processing and analysis of Bro data with Pandas, scikit-learn, and Spark

*homepage*: <https://pypi.python.org/pypi/bat>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.3``|``-Python-3.6.3``|``intel/2017b``

### batchgenerators

Data augmentation toolkit developed at the Division of Medical Image Computing at the German Cancer Research Center (DKFZ) to suit all our deep learning data augmentation needs.

*homepage*: <https://github.com/MIC-DKFZ/batchgenerators>

version |toolchain
--------|--------------
``0.25``|``foss/2021a``

### BatMeth2

An Integrated Package for Bisulfite DNA Methylation Data Analysis with Indel-sensitive Mapping.

*homepage*: <https://github.com/GuoliangLi-HZAU/BatMeth2>

version|toolchain
-------|--------------
``2.1``|``foss/2019b``

### BayesAss

BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes

*homepage*: <http://www.rannala.org/?page_id=245>

version  |toolchain
---------|--------------
``3.0.4``|``foss/2016a``

### BayesAss3-SNPs

Modification of BayesAss 3.0.4 to allow handling of large SNP datasets generated via methods such as RADseq protocols. 

*homepage*: <https://github.com/stevemussmann/BayesAss3-SNPs>

version|toolchain
-------|--------------
``1.1``|``GCC/11.3.0``

### BayeScan

BayeScan aims at identifying candidate loci under natural selection from genetic data,  using differences in allele frequencies between populations.

*homepage*: <http://cmpg.unibe.ch/software/BayeScan/>

version|toolchain
-------|-----------------------------------------------
``2.1``|``foss/2016a``, ``foss/2018a``, ``intel/2018a``

### BayeScEnv

BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detect local adaptation.

*homepage*: <https://github.com/devillemereuil/bayescenv>

version|toolchain
-------|------------------------------------------------------
``1.1``|``GCC/8.3.0``, ``foss/2016a``, ``iccifort/2019.5.281``

### BayesPrism

Bayesian cell Proportion Reconstruction Inferred using Statistical Marginalization   (BayesPrism): A Fully Bayesian Inference of Tumor Microenvironment composition and gene expression  

*homepage*: <https://github.com/Danko-Lab/BayesPrism>

version|versionsuffix|toolchain
-------|-------------|--------------
``2.0``|``-R-4.2.1`` |``foss/2022a``

### BayesTraits

BayesTraits is a computer package for performing analyses of trait   evolution among groups of species for which a phylogeny or sample of phylogenies is   available. This new package incoporates our earlier and separate programes Multistate,   Discrete and Continuous. BayesTraits can be applied to the analysis of traits that adopt   a finite number of discrete states, or to the analysis of continuously varying traits.   Hypotheses can be tested about models of evolution, about ancestral states and about   correlations among pairs of traits. 

*homepage*: <https://github.com/AndrewPMeade/BayesTraits-Public>

version        |versionsuffix    |toolchain
---------------|-----------------|----------
``1.0-linux32``|                 |``system``
``2.0``        |``-Beta-Linux64``|``system``
``3.0.2``      |``-Linux``       |``system``

### Bazel

Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software.

*homepage*: <https://bazel.io/>

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------
``0.4.4`` |             |``system``
``0.7.0`` |             |``GCCcore/6.4.0``
``0.10.0``|             |``GCCcore/6.4.0``
``0.11.0``|             |``GCCcore/6.4.0``
``0.11.1``|             |``GCCcore/6.4.0``
``0.12.0``|             |``GCCcore/6.4.0``
``0.16.0``|             |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``0.18.0``|             |``GCCcore/7.3.0``
``0.20.0``|             |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``0.25.2``|             |``GCCcore/8.2.0``
``0.26.1``|             |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.29.1``|             |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.29.1``|``-Java-1.8``|``GCCcore/9.3.0``
``1.1.0`` |             |``GCCcore/8.3.0``
``2.0.0`` |             |``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``3.4.1`` |             |``GCCcore/8.3.0``
``3.6.0`` |             |``GCCcore/9.3.0``
``3.7.1`` |             |``GCCcore/8.3.0``
``3.7.2`` |             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``
``4.2.2`` |             |``GCCcore/11.2.0``
``5.1.1`` |             |``GCCcore/11.3.0``

### bbcp

BBCP is an alternative to Gridftp when transferring large amounts of data,  capable of breaking up your transfer into multiple simultaneous transferring streams,  thereby transferring data much faster than single-streaming utilities such as SCP and SFTP.  See details at http://pcbunn.cithep.caltech.edu/bbcp/using_bbcp.htm  or http://www.nics.tennessee.edu/computing-resources/data-transfer/bbcp

*homepage*: <http://www.slac.stanford.edu/~abh/bbcp/>

version          |versionsuffix     |toolchain
-----------------|------------------|----------
``12.01.30.00.0``|``-amd64_linux26``|``system``

### bbFTP

bbFTP is a file transfer software. It implements its own transfer protocol,   which is optimized for large files (larger than 2GB) and secure as it does not read the   password in a file and encrypts the connection information. bbFTP main features are:   * Encoded username and password at connection * SSH and Certificate authentication modules   * Multi-stream transfer * Big windows as defined in RFC1323 * On-the-fly data compression   * Automatic retry * Customizable time-outs * Transfer simulation   * AFS authentication integration * RFIO interface

*homepage*: <https://software.in2p3.fr/bbftp/>

version  |toolchain
---------|----------------------------------------------
``3.2.1``|``GCCcore/9.3.0``, ``intel/2016a``, ``system``

### BBMap

BBMap short read aligner, and other bioinformatic tools.

*homepage*: <https://sourceforge.net/projects/bbmap/>

version   |versionsuffix      |toolchain
----------|-------------------|-------------------------------
``36.62`` |``-Java-1.8.0_112``|``intel/2016b``
``37.93`` |                   |``foss/2018a``, ``intel/2018a``
``38.26`` |                   |``foss/2018b``
``38.50b``|                   |``GCC/8.2.0-2.31.1``
``38.76`` |                   |``GCC/8.2.0-2.31.1``
``38.79`` |                   |``GCC/8.3.0``
``38.87`` |                   |``iccifort/2020.1.217``
``38.90`` |                   |``GCC/10.2.0``, ``GCC/9.3.0``
``38.96`` |                   |``GCC/10.3.0``
``38.98`` |                   |``GCC/11.2.0``
``39.01`` |                   |``GCC/11.3.0``, ``GCC/12.2.0``

### bc

bc is an arbitrary precision numeric processing language.

*homepage*: <https://www.gnu.org/software/bc/>

version    |toolchain
-----------|-------------
``1.06.95``|``GCC/4.8.2``

### BCALM

de Bruijn graph compaction in low memory

*homepage*: <https://github.com/GATB/bcalm>

version  |toolchain
---------|--------------
``2.2.0``|``foss/2018a``

### bcbio-gff

Read and write Generic Feature Format (GFF) with Biopython integration. 

*homepage*: <https://github.com/chapmanb/bcbb/tree/master/gff>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``0.6.6``|``-Python-3.8.2``|``foss/2020a``
``0.6.7``|                 |``foss/2021a``
``0.7.0``|                 |``foss/2020b``, ``foss/2022a``

### BCEL

The Byte Code Engineering Library (Apache Commons BCEL™) is intended to give  users a convenient way to analyze, create, and manipulate (binary) Java class  files (those ending with .class). 

*homepage*: <https://commons.apache.org/proper/commons-bcel/>

version  |versionsuffix|toolchain
---------|-------------|----------
``5.2``  |``-Java-1.8``|``system``
``6.5.0``|``-Java-1.8``|``system``

### BCFtools

Samtools is a suite of programs for interacting with high-throughput sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence  variants

*homepage*: <http://www.htslib.org/>

version   |toolchain
----------|-----------------------------------------------------------------------------------------
``1.3``   |``foss/2016a``, ``intel/2016a``
``1.3.1`` |``foss/2016b``
``1.6``   |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``1.8``   |``GCC/6.4.0-2.28``
``1.9``   |``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10.2``|``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``  |``GCC/10.2.0``
``1.12``  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.14``  |``GCC/11.2.0``
``1.15.1``|``GCC/11.3.0``
``1.17``  |``GCC/12.2.0``

### bcgTree

Automatized phylogenetic tree building from bacterial core genomes.

*homepage*: <https://github.com/molbiodiv/bcgTree>

version   |versionsuffix   |toolchain
----------|----------------|---------------
``1.0.10``|``-Perl-5.26.1``|``intel/2018a``
``1.1.0`` |``-Perl-5.28.0``|``intel/2018b``

### bcl-convert

The Illumina BCL Convert v4.0 is a standalone local software app that converts the  Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files.

*homepage*: <https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html>

version    |versionsuffix |toolchain
-----------|--------------|----------
``4.0.3-2``|``el7.x86_64``|``system``

### bcl2fastq2

bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by  Illumina sequencing systems to standard FASTQ file formats for downstream analysis.

*homepage*: <https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------------------------------------------------
``2.19.1``|``-Python-2.7.12``|``foss/2016b``
``2.20.0``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``intel/2019a``
``2.20.0``|``-Python-2.7.12``|``foss/2016b``
``2.20.0``|``-Python-2.7.14``|``intel/2017b``

### bcolz

bcolz provides columnar, chunked data containers that can be compressed either in-memory and on-disk.  Column storage allows for efficiently querying tables, as well as for cheap column addition and removal.  It is based on NumPy, and uses it as the standard data container to communicate with bcolz objects,  but it also comes with support for import/export facilities to/from HDF5/PyTables tables and pandas dataframes.

*homepage*: <https://bcolz.blosc.org/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.1``|``-Python-2.7.13``|``foss/2017a``
``1.2.1``|                  |``foss/2020b``
``1.2.1``|``-Python-3.8.2`` |``foss/2020a``

### BDBag

The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the Bagit and Bagit/RO profiles.

*homepage*: <https://github.com/fair-research/bdbag>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.14``|``intel/2018a``
``1.4.1``|``-Python-3.6.4`` |``intel/2018a``
``1.6.3``|                  |``intel/2021b``

### Beagle

Beagle is a software package for phasing genotypes and for imputing  ungenotyped markers.

*homepage*: <https://faculty.washington.edu/browning/beagle/beagle.html>

version            |versionsuffix|toolchain
-------------------|-------------|----------
``5.4.22Jul22.46e``|``-Java-11`` |``system``

### beagle-lib

beagle-lib is a high-performance library that can perform the core calculations at the heart of most  Bayesian and Maximum Likelihood phylogenetics packages.

*homepage*: <https://github.com/beagle-dev/beagle-lib>

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------
``2.1.2``|                |``foss/2016a``, ``foss/2017a``
``3.0.1``|                |``foss/2018a``, ``intel/2018a``
``3.0.2``|                |``foss/2018b``
``3.0.2``|``-CUDA-9.2.88``|``foss/2018b``
``3.1.2``|                |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``, ``gcccuda/2019b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``4.0.0``|                |``GCC/11.3.0``

### Beast

BEAST is a cross-platform program for Bayesian MCMC analysis of molecular   sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using   strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies   but is also a framework for testing evolutionary hypotheses without conditioning on a single   tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted   proportional to its posterior probability. 

*homepage*: <https://beast2.org>

version   |toolchain
----------|------------------------------------
``1.8.4`` |``system``
``1.10.1``|``intel/2018a``
``1.10.4``|``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``
``2.4.0`` |``foss/2016a``
``2.4.7`` |``foss/2017a``
``2.5.0`` |``foss/2018a``
``2.5.1`` |``foss/2018b``
``2.5.2`` |``GCC/8.2.0-2.31.1``
``2.6.3`` |``gcccuda/2019b``
``2.6.4`` |``GCC/10.2.0``
``2.6.7`` |``GCC/10.3.0``
``2.7.3`` |``GCC/11.3.0``

### BeautifulSoup

Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.

*homepage*: <https://www.crummy.com/software/BeautifulSoup>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------
``4.6.0`` |``-Python-3.6.3``|``intel/2017b``
``4.6.3`` |``-Python-3.6.4``|``intel/2018a``
``4.7.1`` |``-Python-3.6.6``|``intel/2018b``
``4.8.0`` |                 |``GCCcore/8.2.0``
``4.9.1`` |                 |``GCCcore/8.3.0``
``4.9.1`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``4.9.3`` |                 |``GCCcore/10.2.0``
``4.10.0``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.11.1``|                 |``GCCcore/12.2.0``

### BEDOPS

BEDOPS is an open-source command-line toolkit that performs highly efficient and  scalable Boolean and other set operations, statistical calculations, archiving, conversion and  other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for  distributing whole-genome analyses across a computational cluster.

*homepage*: <http://bedops.readthedocs.io/en/latest/index.html>

version   |toolchain
----------|-------------------------------
``2.4.1`` |``GCC/4.8.4``
``2.4.2`` |``GCC/4.8.2``
``2.4.20``|``system``
``2.4.26``|``system``
``2.4.30``|``foss/2016b``
``2.4.32``|``foss/2018a``, ``intel/2018a``
``2.4.35``|``foss/2018b``
``2.4.41``|``foss/2021b``

### BEDTools

BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.

*homepage*: <https://bedtools.readthedocs.io/>

version   |toolchain
----------|-------------------------------------------------------------------------------------------------------
``2.25.0``|``foss/2016a``
``2.26.0``|``GCCcore/6.4.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``2.27.1``|``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018a``
``2.28.0``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.29.2``|``GCC/8.3.0``, ``GCC/9.3.0``
``2.30.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``iccifort/2020.4.304``

### BEEF

BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149.

*homepage*: <https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software>

version  |toolchain
---------|------------------------------------------------
``0.1.1``|``iccifort/2019.5.281``, ``iccifort/2020.4.304``

### behave

behave: Behavior-driven development (or BDD) is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.

*homepage*: <http://pythonhosted.org/behave>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.2.5``|``-Python-2.7.12``|``foss/2016b``
``1.2.6``|``-Python-3.6.4`` |``intel/2018a``

### BerkeleyGW

The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle  properties and the optical responses of a large variety of materials from bulk periodic crystals to  nanostructures such as slabs, wires and molecules.

*homepage*: <https://www.berkeleygw.org>

version      |versionsuffix    |toolchain
-------------|-----------------|----------------------------------------------------------------
``1.0.6``    |                 |``intel/2016.02-GCC-4.9``
``1.1-beta2``|                 |``intel/2016.02-GCC-4.9``
``1.2.0``    |                 |``intel/2017a``, ``intel/2018a``
``2.0.0``    |                 |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``2.1.0``    |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``3.0.1``    |                 |``foss/2020b``, ``foss/2021a``

### BFAST

BFAST facilitates the fast and accurate mapping of short reads to reference sequences.  Some advantages of BFAST include:  1) Speed: enables billions of short reads to be mapped quickly.   2) Accuracy: A priori probabilities for mapping reads with defined set of variants.   3) An simple way to measurably tune accuracy at the expense of speed.

*homepage*: <http://bfast.sourceforge.net/>

version   |toolchain
----------|--------------
``0.7.0a``|``foss/2016b``

### BFC

BFC is a standalone high-performance tool for correcting sequencing errors from Illumina sequencing data. It is specifically designed for high-coverage whole-genome human data, though also performs well for small genomes.

*homepage*: <https://github.com/lh3/bfc>

version|toolchain
-------|--------------
``1``  |``foss/2018a``

### BGC-Bayesian-genomic-clines

Collection of code for Bayesian genomic cline analyses.

*homepage*: <https://github.com/zgompert/BGC-Bayesian-genomic-clines>

version |toolchain
--------|---------------
``1.03``|``gompi/2021a``

### bgen

A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3.

*homepage*: <https://github.com/limix/bgen>

version  |toolchain
---------|------------------
``3.0.2``|``GCCcore/7.3.0``
``3.0.3``|``GCCcore/9.3.0``
``4.1.3``|``GCCcore/10.2.0``

### bgen-reader

A bgen file format reader. This python package is a wrapper around the bgen library, a low-memory footprint reader that efficiently reads bgen files. It fully supports the bgen format specifications: 1.2 and 1.3; as well as their optional compressed formats.

*homepage*: <https://github.com/limix/bgen-reader-py>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.2``|``-Python-3.6.6``|``foss/2018b``

### bibtexparser

Bibtex parser in Python 2.7 and 3.x

*homepage*: <https://github.com/sciunto-org/python-bibtexparser>

version  |toolchain
---------|-----------------
``1.1.0``|``GCCcore/8.2.0``

### BiG-SCAPE

BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters  (BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene  cluster families, and exploring gene cluster diversity linked to enzyme phylogenies.

*homepage*: <https://bigscape-corason.secondarymetabolites.org/index.html>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.0.1``|``-Python-3.7.4``|``intel/2019b``

### BigDFT

BigDFT: electronic structure calculation based on Daubechies wavelets. bigdft-suite is a set of different packages to run bigdft.

*homepage*: <https://gitlab.com/l_sim/bigdft-suite>

version  |toolchain
---------|--------------
``1.9.1``|``foss/2021b``

### BinSanity

BinSanity contains a suite a scripts designed to cluster contigs generated from metagenomic assembly into putative genomes.

*homepage*: <https://github.com/edgraham/BinSanity/wiki>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.5``|``-Python-3.7.4``|``foss/2019b``

### binutils

binutils: GNU binary utilities

*homepage*: <https://directory.fsf.org/project/binutils/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.25``  |``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``system``
``2.25.1``|``system``
``2.26``  |``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``system``
``2.27``  |``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``system``
``2.28``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``system``
``2.29``  |``GCCcore/7.2.0``, ``GCCcore/system``, ``system``
``2.30``  |``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``system``
``2.31.1``|``GCCcore/7.4.0``, ``GCCcore/8.2.0``, ``system``
``2.32``  |``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``
``2.34``  |``GCCcore/10.1.0``, ``GCCcore/9.3.0``, ``system``
``2.35``  |``GCCcore/10.2.0``, ``system``
``2.36.1``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/8.4.0``, ``GCCcore/9.4.0``, ``system``
``2.37``  |``GCCcore/11.2.0``, ``system``
``2.38``  |``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/9.5.0``, ``system``
``2.39``  |``GCCcore/12.2.0``, ``system``
``2.40``  |``GCCcore/11.4.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### Bio-DB-HTS

Read files using HTSlib including BAM/CRAM, Tabix and BCF database files

*homepage*: <https://metacpan.org/release/Bio-DB-HTS>

version |versionsuffix   |toolchain
--------|----------------|----------------------------------------------
``2.11``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``2.11``|``-Perl-5.26.1``|``intel/2018a``
``2.11``|``-Perl-5.28.0``|``foss/2018b``
``3.01``|                |``GCC/10.2.0``, ``GCC/11.2.0``, ``GCC/11.3.0``
``3.01``|``-Perl-5.28.1``|``GCC/8.2.0-2.31.1``

### Bio-EUtilities

BioPerl low-level API for retrieving and storing data from NCBI eUtils

*homepage*: <https://github.com/bioperl/bio-eutilities>

version |toolchain
--------|-----------------
``1.76``|``GCCcore/8.3.0``

### Bio-SamTools

This is a Perl interface to the SAMtools sequence alignment interface.

*homepage*: <http://search.cpan.org/~lds/Bio-SamTools/>

version |versionsuffix   |toolchain
--------|----------------|---------------
``1.43``|``-Perl-5.24.1``|``intel/2017a``

### Bio-SearchIO-hmmer

Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org.

*homepage*: <https://metacpan.org/pod/Bio::SearchIO::hmmer3>

version  |toolchain
---------|--------------------------------------------------------------
``1.7.3``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

### bioawk

Bioawk is an extension to Brian Kernighan's awk,  adding the support of several common biological data formats,  including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. 

*homepage*: <https://github.com/lh3/bioawk>

version|toolchain
-------|----------------------------------------------
``1.0``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2018b``

### biobambam2

Tools for processing BAM files

*homepage*: <https://github.com/gt1/biobambam2>

version   |toolchain
----------|-------------------------------
``2.0.87``|``GCC/11.3.0``, ``intel/2018a``

### biogeme

Biogeme is a open source Python package designed for the maximum likelihood estimation of parametric models in general, with a special emphasis on discrete choice models.

*homepage*: <https://pypi.python.org/pypi/biogeme>

version   |toolchain
----------|--------------
``3.1.2`` |``foss/2021a``
``3.2.6`` |``foss/2022a``
``3.2.8`` |``foss/2021a``
``3.2.10``|``foss/2022a``

### biom-format

The BIOM file format (canonically pronounced biome) is designed to be  a general-use format for representing biological sample by observation  contingency tables. BIOM is a recognized standard for the Earth Microbiome  Project and is a Genomics Standards Consortium supported project. 

*homepage*: <https://biom-format.org>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2.1.10``|``-Python-3.8.2``|``foss/2020a``
``2.1.12``|                 |``foss/2021b``
``2.1.14``|                 |``foss/2022a``

### biomart-perl

The BioMart Perl API allows you to go a step further with BioMart   and integrate BioMart Perl Code into custom Perl scripts.

*homepage*: <https://useast.ensembl.org/info/data/biomart/biomart_perl_api.html>

version        |versionsuffix   |toolchain
---------------|----------------|-----------------
``0.7_e6db561``|``-Perl-5.26.0``|``GCCcore/6.4.0``

### BioPerl

Bioperl is the product of a community effort to produce Perl code which is useful in biology.  Examples include Sequence objects, Alignment objects and database searching objects.

*homepage*: <http://www.bioperl.org/>

version    |versionsuffix   |toolchain
-----------|----------------|--------------------------------------------------------------------------------------------------
``1.6.924``|``-Perl-5.20.3``|``intel/2016a``
``1.6.924``|``-Perl-5.22.1``|``foss/2016a``
``1.7.0``  |``-Perl-5.24.0``|``foss/2016b``
``1.7.1``  |``-Perl-5.24.0``|``intel/2016b``
``1.7.1``  |``-Perl-5.24.1``|``intel/2017a``
``1.7.2``  |                |``GCCcore/8.3.0``
``1.7.2``  |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``1.7.2``  |``-Perl-5.26.1``|``intel/2018a``
``1.7.2``  |``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``
``1.7.2``  |``-Perl-5.28.1``|``GCCcore/8.2.0``
``1.7.7``  |                |``GCCcore/9.3.0``
``1.7.8``  |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### BioPP

Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics. Bio++ is Object Oriented and is designed to be both simple to use and computer efficient. Bio++ intends to help programmers to write computer expensive programs, by providing them a set of re-usable tools. 

*homepage*: <https://github.com/BioPP/bpp-core>

version  |toolchain
---------|-----------------------------------
``2.4.1``|``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``

### Biopython

Biopython is a set of freely available tools for biological  computation written in Python by an international team of developers. It is  a distributed collaborative effort to develop Python libraries and  applications which address the needs of current and future work in  bioinformatics. 

*homepage*: <https://www.biopython.org>

version |versionsuffix     |toolchain
--------|------------------|---------------------------------------------------------------
``1.65``|``-Python-2.7.11``|``foss/2016a``
``1.68``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.68``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``1.70``|``-Python-2.7.13``|``foss/2017a``
``1.70``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.70``|``-Python-3.6.1`` |``intel/2017a``
``1.70``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.71``|``-Python-2.7.14``|``intel/2018a``
``1.71``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.72``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.72``|``-Python-3.6.6`` |``foss/2018b``
``1.73``|                  |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``1.73``|``-Python-3.6.6`` |``foss/2018b``
``1.74``|                  |``foss/2019a``
``1.75``|``-Python-2.7.16``|``foss/2019b``
``1.75``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.76``|``-Python-2.7.18``|``foss/2020b``
``1.78``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.78``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.79``|                  |``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2021b``
``1.81``|                  |``foss/2022b``

### BioServices

Bioservices is a Python package that provides access to many Bioinformatices Web Services (e.g.,  UniProt) and a framework to easily implement Web Services wrappers (based on WSDL/SOAP or REST protocols).

*homepage*: <https://bioservices.readthedocs.io/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.7.9``|``-Python-3.8.2``|``foss/2020a``

### BirdNET

BirdNET is a research platform that aims at recognizing birds by sound at scale. We support various hardware and operating systems such as Arduino microcontrollers, the Raspberry Pi,  smartphones, web browsers, workstation PCs, and even cloud services. BirdNET is a citizen science  platform as well as an analysis software for extremely large collections of audio. BirdNET aims to provide innovative tools for conservationists, biologists, and birders alike.

*homepage*: <https://birdnet.cornell.edu/>

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``20201214``|``-Python-3.7.4``|``fosscuda/2019b``

### biscuit

Utilities to help analyze bisulfite-treated sequence data  

*homepage*: <https://github.com/zwdzwd/biscuit>

version  |toolchain
---------|--------------
``0.1.4``|``foss/2016a``

### Bismark

A tool to map bisulfite converted sequence reads and determine cytosine methylation states

*homepage*: <https://www.bioinformatics.babraham.ac.uk/projects/bismark/>

version   |toolchain
----------|-------------------------------
``0.19.0``|``intel/2017b``
``0.20.1``|``foss/2018b``, ``intel/2018b``
``0.23.1``|``foss/2021b``
``0.24.0``|``GCC/11.3.0``
``0.24.1``|``GCC/12.2.0``

### Bison

Bison is a general-purpose parser generator that converts an annotated  context-free grammar into a deterministic LR or generalized LR (GLR) parser  employing LALR(1) parser tables. 

*homepage*: <https://www.gnu.org/software/bison>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.7``  |``GCC/4.8.1``, ``GCC/4.8.4``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``
``3.0.2``|``GCC/4.8.2``
``3.0.3``|``GCC/4.9.2``
``3.0.4``|``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/system``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``3.0.5``|``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/system``, ``system``
``3.2.2``|``GCCcore/7.4.0``
``3.3.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``
``3.5.2``|``system``
``3.5.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``, ``system``
``3.6.1``|``GCCcore/10.1.0``
``3.7.1``|``GCCcore/10.2.0``, ``system``
``3.7.6``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/9.4.0``, ``system``
``3.8.2``|``GCCcore/11.3.0``, ``GCCcore/11.4.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``GCCcore/9.5.0``, ``system``

### bitarray

bitarray provides an object type which efficiently represents an array of booleans

*homepage*: <https://github.com/ilanschnell/bitarray>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.3``|``-Python-2.7.15``|``intel/2018b``
``0.8.3``|``-Python-3.6.6`` |``intel/2018b``
``1.2.1``|``-Python-3.7.4`` |``foss/2019b``
``1.5.3``|``-Python-2.7.16``|``GCC/8.3.0``

### BLACS

The BLACS (Basic Linear Algebra Communication Subprograms) project is  an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface  that may be implemented efficiently and uniformly across a large range of distributed memory platforms.

*homepage*: <http://www.netlib.org/blacs/>

version|toolchain
-------|-------------------
``1.1``|``gmvapich2/2016a``

### BLASR

The PacBio® long read aligner

*homepage*: <https://github.com/PacificBiosciences/blasr>

version     |toolchain
------------|---------------
``2.2``     |``intel/2016b``
``5.3.3``   |``gompi/2019a``
``20170330``|``intel/2017a``

### blasr_libcpp

Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for  analyzing PacBio sequences

*homepage*: <https://github.com/PacificBiosciences/blasr_libcpp>

version     |toolchain
------------|---------------
``20170426``|``intel/2017a``

### BLAST

Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological  sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: <http://blast.ncbi.nlm.nih.gov/>

version   |versionsuffix    |toolchain
----------|-----------------|----------
``2.2.26``|``-Linux_x86_64``|``system``
``2.8.1`` |``-Linux_x86_64``|``system``
``2.10.0``|``-Linux_x86_64``|``system``
``2.10.1``|``-Linux_x86_64``|``system``
``2.11.0``|``-Linux_x86_64``|``system``

### BLAST+

Basic Local Alignment Search Tool, or BLAST, is an algorithm  for comparing primary biological sequence information, such as the amino-acid  sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: <https://blast.ncbi.nlm.nih.gov/>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------
``2.2.31``|                  |``system``
``2.3.0`` |``-Python-2.7.11``|``foss/2016a``
``2.6.0`` |``-Python-2.7.12``|``foss/2016b``
``2.6.0`` |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``2.6.0`` |``-Python-2.7.14``|``intel/2017b``
``2.7.1`` |                  |``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2.7.1`` |``-Python-2.7.14``|``intel/2017b``
``2.8.1`` |                  |``foss/2018b``
``2.9.0`` |                  |``gompi/2019a``, ``gompi/2019b``, ``gompi/2021b``, ``iimpi/2019a``, ``iimpi/2019b``
``2.10.1``|                  |``gompi/2020a``, ``iimpi/2020a``
``2.11.0``|                  |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``
``2.12.0``|                  |``gompi/2021b``
``2.13.0``|                  |``gompi/2022a``
``2.14.0``|                  |``gompi/2022b``

### BLAT

BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more.

*homepage*: <https://genome.ucsc.edu/goldenPath/help/blatSpec.html>

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------
``3.5``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2016b``, ``intel/2017a``
``3.7``|``GCC/11.2.0``, ``GCC/11.3.0``

### Blender

Blender is the free and open source 3D creation suite. It supports  the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,  compositing and motion tracking, even video editing and game creation.

*homepage*: <https://www.blender.org/>

version  |versionsuffix                |toolchain
---------|-----------------------------|-------------------------------
``2.77a``|``-Python-3.5.2``            |``intel/2016b``
``2.79`` |``-Python-3.6.1``            |``intel/2017a``
``2.79b``|``-Python-3.6.6``            |``intel/2018b``
``2.79b``|``-Python-3.6.6-CUDA-9.2.88``|``foss/2018b``
``2.81`` |``-Python-3.7.4``            |``foss/2019b``, ``intel/2019b``
``3.0.0``|``-linux-x64``               |``system``
``3.1.2``|``-linux-x64``               |``system``
``3.2.2``|``-linux-x64``               |``system``
``3.3.1``|``-linux-x86_64-CUDA-11.7.0``|``system``
``3.4.1``|``-linux-x86_64-CUDA-11.7.0``|``system``
``3.5.0``|``-linux-x86_64-CUDA-11.7.0``|``system``

### BLIS

AMD's fork of BLIS. BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries.

*homepage*: <https://developer.amd.com/amd-cpu-libraries/blas-library/>

version  |versionsuffix|toolchain
---------|-------------|----------------------------------------------------------------------------
``0.3.2``|             |``GCC/7.3.0-2.30``
``0.6.0``|             |``GCC/8.3.0-2.32``
``0.8.0``|             |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``0.8.1``|             |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCCcore/10.3.0``
``0.9.0``|             |``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/12.3.0``, ``intel-compilers/2022.1.0``
``1.2``  |``-amd``     |``GCC/7.3.0-2.30``
``2.2``  |``-amd``     |``GCCcore/9.3.0``
``3.0``  |``-amd``     |``GCCcore/10.3.0``
``3.0.1``|``-amd``     |``GCC/11.2.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.1``  |``-amd``     |``GCCcore/11.2.0``

### Blitz++

Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation  in C++ with a speed comparable to Fortran implementations, while preserving an  object-oriented interface 

*homepage*: <https://github.com/blitzpp/blitz>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``0.10`` |``GCCcore/6.4.0``, ``foss/2016a``
``1.0.2``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/9.3.0``

### BlobTools

A modular command-line solution for visualisation,  quality control and taxonomic partitioning of genome datasets. 

*homepage*: <https://blobtools.readme.io/docs>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20180528``|``-Python-2.7.15``|``foss/2018b``

### Block

Block implements the density matrix renormalization group (DMRG) algorithm for quantum chemistry.

*homepage*: <https://sanshar.github.io/Block/>

version           |toolchain
------------------|--------------
``1.5.3-20200525``|``foss/2022a``

### Blosc

Blosc, an extremely fast, multi-threaded, meta-compressor library

*homepage*: <https://www.blosc.org/>

version   |toolchain
----------|------------------------------------------------------------------
``1.11.1``|``intel/2016b``
``1.12.1``|``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2017a``
``1.14.2``|``GCCcore/6.4.0``, ``foss/2016a``
``1.14.4``|``GCCcore/7.3.0``
``1.17.0``|``GCCcore/8.2.0``
``1.17.1``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.21.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.21.1``|``GCCcore/11.2.0``
``1.21.3``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### Blosc2

Blosc, an extremely fast, multi-threaded, meta-compressor library

*homepage*: <https://www.blosc.org/>

version  |toolchain
---------|------------------
``2.0.3``|``GCCcore/10.2.0``
``2.0.4``|``GCCcore/10.3.0``
``2.4.3``|``GCCcore/11.3.0``
``2.6.1``|``GCCcore/11.3.0``
``2.8.0``|``GCCcore/12.2.0``

### BLT

BLT is an extension to the Tk toolkit, adding new widgets,  geometry managers, and miscellaneous commands.

*homepage*: <http://blt.sourceforge.net/>

version     |toolchain
------------|------------------
``20181223``|``GCCcore/11.2.0``

### bmtagger

Best Match Tagger for removing human reads from metagenomics datasets

*homepage*: <ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/>

version  |toolchain
---------|------------------------------------------------
``3.101``|``foss/2018b``, ``gompi/2019a``, ``gompi/2020b``

### bnpy

Bayesian nonparametric machine learning for python provides  code for training popular clustering models on large datasets. The focus is  on Bayesian nonparametric models based on the Dirichlet process, but it also  provides parametric counterparts.

*homepage*: <https://github.com/bnpy/bnpy>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.1.6``|``-Python-2.7.15``|``foss/2018b``

### BOINC

BOINC is a program that lets you donate your idle computer time to science projects  like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others.

*homepage*: <https://boinc.berkeley.edu>

version   |versionsuffix|toolchain
----------|-------------|-------------
``7.2.42``|``-client``  |``GCC/4.8.2``

### bokeh

Statistical and novel interactive HTML plots for Python

*homepage*: <https://github.com/bokeh/bokeh>

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------------------------------
``0.12.3`` |``-Python-2.7.12``|``intel/2016b``
``0.12.3`` |``-Python-3.5.2`` |``intel/2016b``
``0.12.15``|``-Python-3.6.4`` |``intel/2018a``
``1.0.4``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.3.4``  |``-Python-3.7.2`` |``foss/2019a``
``1.4.0``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.0.2``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.2.3``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2.4.1``  |                  |``foss/2021a``
``2.4.2``  |                  |``foss/2021b``
``2.4.3``  |                  |``foss/2022a``

### BoltzTraP

Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport  coefficients.

*homepage*: <http://www.icams.de/content/departments/cmat/boltztrap/>

version  |toolchain
---------|---------------
``1.2.5``|``intel/2016a``

### BoltzTraP2

BoltzTraP2 provides a numerically stable and efficient method for obtaining analytic representations of electronic bands based on density-functional-theory results for relatively sparse grids. It achieves this goal by using smoothed Fourier interpolation.

*homepage*: <https://www.boltztrap.org>

version    |toolchain
-----------|--------------
``22.12.1``|``foss/2022a``

### Bonito

Convolution Basecaller for Oxford Nanopore Reads

*homepage*: <https://github.com/nanoporetech/bonito>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.1.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.1.4``|``-Python-3.7.4``|``fosscuda/2019b``
``0.2.0``|``-Python-3.7.4``|``fosscuda/2019b``
``0.2.2``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.2``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.5``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.8``|                 |``fosscuda/2020b``
``0.4.0``|                 |``fosscuda/2020b``

### Bonmin

Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for  large-scale nonlinear optimization.

*homepage*: <https://coin-or.github.io/Ipopt>

version  |toolchain
---------|---------------
``1.8.7``|``intel/2019a``

### Bonnie++

Enhanced performance Test of Filesystem I/O

*homepage*: <https://www.coker.com.au/bonnie++>

version  |toolchain
---------|--------------
``1.97`` |``foss/2016a``
``2.00a``|``GCC/10.3.0``

### Boost

Boost provides free peer-reviewed portable C++ source libraries.

*homepage*: <https://www.boost.org/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------
``1.54.0``|``-Python-2.7.12``|``foss/2016b``
``1.55.0``|                  |``system``
``1.55.0``|``-Python-2.7.11``|``foss/2016a``
``1.57.0``|``-Python-2.7.10``|``gimkl/2.11.5``
``1.58.0``|                  |``intel/2017a``
``1.58.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.58.0``|``-serial``       |``GCC/4.9.2``
``1.59.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.60.0``|                  |``foss/2016a``, ``intel/2016a``
``1.60.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.60.0``|``-Python-3.5.1`` |``foss/2016a``
``1.61.0``|                  |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.61.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.61.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.61.0``|``-Python-3.5.1`` |``foss/2016a``
``1.62.0``|``-Python-2.7.12``|``intel/2016b``
``1.63.0``|                  |``foss/2017a``
``1.63.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.63.0``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.63.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``1.63.0``|``-Python-3.5.2`` |``foss/2016b``
``1.64.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``intel/2017a``
``1.64.0``|``-Python-2.7.13``|``intel/2017a``
``1.65.0``|``-Python-2.7.13``|``intel/2017a``
``1.65.1``|                  |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``1.65.1``|``-Python-2.7.13``|``intel/2017a``
``1.65.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.65.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.66.0``|                  |``foss/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``
``1.66.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``1.66.0``|``-Python-3.6.3`` |``intel/2018.01``
``1.66.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.66.0``|``-no_mpi``       |``GCCcore/6.4.0``
``1.67.0``|                  |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``1.67.0``|``-Python-2.7.14``|``foss/2018a``
``1.68.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.68.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.69.0``|                  |``intel/2019.01``
``1.70.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``iimpi/2019a``, ``iimpic/2019a``
``1.71.0``|                  |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``1.72.0``|                  |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``
``1.72.0``|``-no_mpi``       |``GCCcore/9.3.0``
``1.74.0``|                  |``GCC/10.2.0``, ``iccifort/2020.4.304``
``1.75.0``|                  |``GCC/11.2.0``
``1.76.0``|                  |``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``1.77.0``|                  |``GCC/11.2.0``, ``intel-compilers/2021.4.0``
``1.79.0``|                  |``GCC/11.2.0``, ``GCC/11.3.0``
``1.81.0``|                  |``GCC/12.2.0``

### Boost.MPI

Boost provides free peer-reviewed portable C++ source libraries.

*homepage*: <https://www.boost.org/>

version   |toolchain
----------|---------------
``1.76.0``|``gompi/2021a``
``1.77.0``|``gompi/2021b``
``1.79.0``|``gompi/2022a``

### Boost.Python

Boost.Python is a C++ library which enables seamless interoperability between C++  and the Python programming language.

*homepage*: <https://boostorg.github.io/python>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------
``1.64.0``|                  |``gompi/2019a``, ``gompic/2019a``
``1.65.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.66.0``|``-Python-2.7.14``|``intel/2018a``
``1.66.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.67.0``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.67.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.70.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``iimpi/2019a``, ``iimpic/2019a``
``1.71.0``|                  |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``1.72.0``|                  |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``
``1.74.0``|                  |``GCC/10.2.0``
``1.76.0``|                  |``GCC/10.3.0``
``1.77.0``|                  |``GCC/11.2.0``
``1.79.0``|                  |``GCC/11.3.0``

### boost_histogram

Boost-histogram is a Python package providing Python bindings for Boost.Histogram.

*homepage*: <https://boost-histogram.readthedocs.io>

version  |toolchain
---------|--------------
``1.2.1``|``foss/2021a``

### boto3

Boto3 is the Amazon Web Services (AWS) Software Development Kit (SDK) for Python, which allows Python developers to write software that makes use of services like Amazon S3 and Amazon EC2.

*homepage*: <https://github.com/boto/boto3>

version     |toolchain
------------|--------------------------------------
``1.20.13`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.26.37`` |``GCCcore/11.3.0``
``1.26.163``|``GCCcore/12.2.0``

### Bottleneck

Fast NumPy array functions written in C

*homepage*: <https://kwgoodman.github.io/bottleneck-doc>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.2.1``|``-Python-3.6.4``|``intel/2018a``
``1.3.2``|``-Python-3.8.2``|``foss/2020a``
``1.3.6``|                 |``foss/2022a``
``1.3.7``|                 |``foss/2022a``

### Bowtie

Bowtie is an ultrafast, memory-efficient short read aligner.  It aligns short DNA sequences (reads) to the human genome.

*homepage*: <http://bowtie-bio.sourceforge.net/index.shtml>

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------------------------------------------
``1.1.2``  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``
``1.2.1.1``|``foss/2016b``, ``intel/2017b``
``1.2.2``  |``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``1.2.3``  |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``
``1.3.0``  |``GCC/10.2.0``, ``GCC/9.3.0``
``1.3.1``  |``GCC/11.2.0``, ``GCC/11.3.0``

### Bowtie2

Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads  to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s  of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.  Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,  its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.

*homepage*: <https://bowtie-bio.sourceforge.net/bowtie2/index.shtml>

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------
``2.2.8``  |``foss/2016a``
``2.2.9``  |``foss/2016a``, ``intel/2016b``
``2.3.2``  |``foss/2016b``, ``foss/2017a``, ``intel/2017a``
``2.3.3.1``|``intel/2017b``
``2.3.4``  |``intel/2017b``
``2.3.4.1``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.3.4.2``|``foss/2018b``, ``intel/2018b``
``2.3.4.3``|``foss/2017b``
``2.3.5.1``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``
``2.4.1``  |``GCC/9.3.0``
``2.4.2``  |``GCC/10.2.0``, ``GCC/9.3.0``
``2.4.4``  |``GCC/10.3.0``, ``GCC/11.2.0``
``2.4.5``  |``GCC/11.3.0``
``2.5.1``  |``GCC/12.2.0``

### Bpipe

Bpipe - a tool for running and managing bioinformatics pipelines

*homepage*: <http://docs.bpipe.org/>

version    |toolchain
-----------|---------------
``0.9.9.2``|``intel/2017a``

### bpp

The aim of this project is to implement a versatile high-performance version of the BPP software. 

*homepage*: <https://github.com/bpp/bpp>

version  |toolchain
---------|--------------
``4.3.8``|``GCC/8.3.0``
``4.4.0``|``GCC/10.3.0``

### bpytop

Resource monitor that shows usage and stats for processor, memory, disks, network and processes.

*homepage*: <https://github.com/aristocratos/bpytop/>

version   |toolchain
----------|------------------
``1.0.60``|``GCCcore/10.2.0``
``1.0.67``|``GCCcore/10.3.0``

### Bracken

Bracken (Bayesian Reestimation of Abundance with KrakEN)  is a highly accurate statistical method that computes the abundance of  species in DNA sequences from a metagenomics sample. Braken uses the  taxonomy labels assigned by Kraken, a highly accurate metagenomics  classification algorithm, to estimate the number of reads originating  from each species present in a sample. Kraken classifies reads to the  best matching location in the taxonomic tree, but does not estimate  abundances of species. We use the Kraken database itself to derive  probabilities that describe how much sequence from each genome is  identical to other genomes in the database, and combine this information  with the assignments for a particular sample to estimate abundance at  the species level, the genus level, or above. Combined with the Kraken  classifier, Bracken produces accurate species- and genus-level abundance  estimates even when a sample contains two or more near-identical species.  NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the  default kmer length is different depending on the version of Kraken used.  If you use Kraken 1 defaults, specify 31 as the kmer length. If you use  Kraken 2 defaults, specify 35 as the kmer length.

*homepage*: <https://ccb.jhu.edu/software/bracken/>

version  |toolchain
---------|------------------
``2.6.0``|``GCCcore/9.3.0``
``2.6.2``|``GCCcore/11.2.0``
``2.7``  |``GCCcore/11.2.0``

### Braindecode

Braindecode is an open-source Python toolbox for decoding raw electrophysiological brain data with deep learning models. It includes dataset fetchers, data preprocessing and visualization tools, as well as implementations of several deep learning architectures and data augmentations for analysis of EEG, ECoG and MEG.

*homepage*: <https://braindecode.org/>

version|versionsuffix                  |toolchain
-------|-------------------------------|--------------
``0.7``|``-PyTorch-1.10.0``            |``foss/2021a``
``0.7``|``-PyTorch-1.10.0-CUDA-11.3.1``|``foss/2021a``

### BRAKER

BRAKER is a pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET  and AUGUSTUS in novel eukaryotic genomes.

*homepage*: <https://github.com/Gaius-Augustus/BRAKER>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``2.1.2``|                 |``intel/2019a``
``2.1.5``|``-Python-3.7.4``|``intel/2019b``
``2.1.6``|                 |``foss/2021b``, ``foss/2022a``

### BreakDancer

BreakDancer is a Perl/C++ package that provides genome-wide detection of structural variants from  next generation paired-end sequencing reads

*homepage*: <http://gmt.genome.wustl.edu/packages/breakdancer>

version  |toolchain
---------|---------------
``1.4.5``|``intel/2017a``

### breseq

breseq is a computational pipeline for the analysis of short-read re-sequencing data

*homepage*: <https://barricklab.org/breseq>

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.35.0``|``-R-3.6.0`` |``intel/2019a``
``0.35.4``|``-R-4.0.0`` |``foss/2020a``
``0.36.1``|             |``foss/2021b``
``0.38.1``|             |``foss/2022a``

### Brotli

Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.

*homepage*: <https://github.com/google/brotli>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------
``1.0.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/8.3.0``

### Brotli-python

Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.

*homepage*: <https://github.com/google/brotli>

version  |toolchain
---------|------------------------------------------------------------------------------
``1.0.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### Brunsli

Brunsli is a lossless JPEG repacking library.

*homepage*: <https://github.com/google/brunsli/>

version|toolchain
-------|--------------------------------------------------------------------------------------------------
``0.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### bsddb3

bsddb3 is a nearly complete Python binding of the Oracle/Sleepycat C API for the Database Environment, Database, Cursor, Log Cursor, Sequence and Transaction objects.

*homepage*: <https://pypi.org/project/bsddb3/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``6.2.6``|                  |``GCCcore/8.2.0``
``6.2.6``|``-Python-2.7.15``|``fosscuda/2018b``
``6.2.9``|                  |``GCCcore/10.2.0``, ``GCCcore/11.3.0``

### BSMAPz

Updated and optimized fork of BSMAP. BSMAPz is a short reads mapping program for bisulfite sequencing in DNA methylation study.

*homepage*: <https://github.com/zyndagj/BSMAPz>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.1``|``-Python-2.7.16``|``intel/2019b``

### Bsoft

Bsoft is a collection of programs and a platform for development of software  for image and molecular processing in structural biology. Problems in  structural biology are approached with a highly modular design, allowing fast  development of new algorithms without the burden of issues such as file I/O.  It provides an easily accessible interface, a resource that can be and has  been used in other packages. 

*homepage*: <https://lsbr.niams.nih.gov/bsoft/>

version  |toolchain
---------|--------------
``2.0.2``|``foss/2017b``
``2.0.7``|``GCC/9.3.0``

### BSseeker2

BS-Seeker2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated reads.

*homepage*: <http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``2.1.8``|``-Python-2.7.16``|``GCC/8.3.0``, ``iccifort/2019.5.281``

### BuDDy

A Binary Decision Diagram library, with many highly efficient vectorized BDD operations, dynamic  variable reordering, automated garbage collection, a C++ interface with automatic reference counting, and much  more.

*homepage*: <http://buddy.sourceforge.net/>

version|toolchain
-------|------------------
``2.4``|``GCCcore/10.2.0``

### BUFRLIB

NCEP BUFRLIB software to encode or decode BUFR messages. It is not intended to be a primer on the BUFR code form itself.

*homepage*: <https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB>

version     |toolchain
------------|-----------------------
``11.3.0.2``|``iccifort/2020.1.217``

### build

A simple, correct Python build frontend.

*homepage*: <https://github.com/pypa/build>

version   |toolchain
----------|------------------------------
``0.10.0``|``foss/2022a``, ``foss/2022b``

### buildenv

This module sets a group of environment variables for compilers, linkers, maths libraries, etc., that  you can use to easily transition between toolchains when building your software. To query the variables being set  please use: module show <this module name>

*homepage*: <None>

version    |versionsuffix   |toolchain
-----------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``default``|                |``FCC/4.5.0``, ``Fujitsu/21.05``, ``foss/2017b``, ``foss/2018b``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``foss/2022b``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``fosscuda/2020b``, ``intel/2016b``, ``intel/2017a``, ``intel/2019b``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022a``, ``intel/2022b``, ``intelcuda/2019b``, ``intelcuda/2020a``, ``intelcuda/2020b``
``default``|``-CUDA-11.3.1``|``foss/2021a``
``default``|``-CUDA-11.4.1``|``foss/2021b``
``default``|``-CUDA-11.7.0``|``foss/2022a``
``default``|``-CUDA-12.0.0``|``foss/2022b``

### buildingspy

Python modules for automating Modelica simulations and for running unit test for the Buildings library

*homepage*: <https://simulationresearch.lbl.gov/modelica/buildingspy>

version  |toolchain
---------|--------------
``4.0.0``|``foss/2022a``

### Bullet

Bullet professional 3D Game Multiphysics Library provides state  of the art collision detection, soft body and rigid body dynamics.

*homepage*: <http://bulletphysics.org/>

version   |toolchain
----------|-------------------------------
``2.83.7``|``foss/2016a``, ``intel/2016a``

### BUSCO

BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs

*homepage*: <https://busco.ezlab.org/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.22`` |``-Python-2.7.13``|``intel/2017a``
``2.0.1``|``-Python-2.7.13``|``intel/2017a``
``3.0.2``|``-Python-2.7.15``|``intel/2018b``
``4.0.5``|``-Python-3.7.4`` |``foss/2019b``
``4.0.6``|                  |``foss/2020b``
``5.0.0``|                  |``foss/2020b``
``5.1.2``|                  |``foss/2020b``
``5.4.2``|                  |``foss/2021a``
``5.4.3``|                  |``foss/2021b``
``5.4.5``|                  |``foss/2022a``
``5.4.7``|                  |``foss/2022b``

### BUStools

bustools is a program for manipulating BUS files for single cell RNA-Seq datasets.  It can be used to error correct barcodes, collapse  UMIs, produce gene count or transcript compatibility  count matrices, and is useful for many other tasks. See the kallisto | bustools  website for examples  and instructions on how to use bustools as part of a single-cell RNA-seq workflow.

*homepage*: <https://github.com/BUStools/bustools>

version   |toolchain
----------|---------------------------------
``0.40.0``|``GCCcore/9.3.0``, ``foss/2018b``

### BWA

Burrows-Wheeler Aligner (BWA) is an efficient program that aligns  relatively short nucleotide sequences against a long reference sequence such as the human genome.

*homepage*: <http://bio-bwa.sourceforge.net/>

version    |toolchain
-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.7.12`` |``foss/2016b``
``0.7.13`` |``foss/2016a``, ``intel/2016a``
``0.7.15`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``0.7.16a``|``foss/2016b``, ``intel/2017a``
``0.7.17`` |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

### bwa-meth

Fast and accurante alignment of BS-Seq reads.

*homepage*: <https://github.com/brentp/bwa-meth>

version  |toolchain
---------|-----------------------------------------------------------------
``0.2.2``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``
``0.2.6``|``GCC/11.3.0``

### bwakit

Bwakit is a self-consistent installation-free package of scripts and precompiled binaries, providing an end-to-end solution to read mapping.

*homepage*: <https://github.com/lh3/bwa/tree/master/bwakit>

version   |versionsuffix |toolchain
----------|--------------|----------
``0.7.15``|``_x64-linux``|``system``

### bwidget

The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.

*homepage*: <https://core.tcl-lang.org/bwidget/home>

version   |toolchain
----------|--------------------------------------------------------
``1.9.13``|``GCCcore/8.2.0``
``1.9.14``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.9.15``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### BWISE

de Bruijn Workflow using Integral information of Short pair End reads

*homepage*: <https://github.com/Malfoy/BWISE>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180820``|``-Python-3.6.4``|``foss/2018a``

### bx-python

The bx-python project is a Python library and associated set of scripts to allow for rapid  implementation of genome scale analyses.

*homepage*: <https://github.com/bxlab/bx-python>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.7.4`` |``-Python-2.7.12``|``foss/2016b``
``0.7.4`` |``-Python-2.7.13``|``intel/2017a``
``0.8.1`` |``-Python-2.7.14``|``intel/2018a``
``0.8.2`` |``-Python-3.6.6`` |``foss/2018b``
``0.8.4`` |                  |``foss/2019a``
``0.8.8`` |``-Python-3.7.4`` |``foss/2019b``
``0.8.9`` |``-Python-3.8.2`` |``foss/2020a``
``0.8.11``|                  |``foss/2021a``
``0.8.13``|                  |``foss/2021b``
``0.9.0`` |                  |``foss/2022a``

### BXH_XCEDE_TOOLS

A collection of data processing and image analysis  tools for data in BXH or XCEDE format. This includes data format  encapsulation/conversion, event-related analysis, QA tools, and more.  These tools form the basis of the fBIRN QA procedures and are also  distributed as part of the fBIRN Data Upload Scripts.

*homepage*: <http://www.nitrc.org/projects/bxh_xcede_tools/>

version   |toolchain
----------|----------
``1.11.1``|``system``

### byacc

Berkeley Yacc (byacc) is generally conceded to be the best yacc variant  available.  In contrast to bison, it is written to avoid dependencies  upon a particular compiler. 

*homepage*: <http://invisible-island.net/byacc/byacc.html>

version     |toolchain
------------|-------------------------------
``20160324``|``intel/2016a``
``20160606``|``foss/2016b``, ``intel/2016b``
``20170709``|``GCCcore/6.4.0``

### byobu

Byobu is an elegant enhancement of the otherwise functional, plain, practical GNU Screen. Byobu includes an enhanced profile, configuration utilities, and system status notifications for the GNU screen window manager as well as the Tmux terminal multiplexer

*homepage*: <https://byobu.org>

version  |toolchain
---------|-------------
``5.133``|``GCC/8.3.0``

### bzip2

bzip2 is a freely available, patent free, high-quality data compressor. It  typically compresses files to within 10% to 15% of the best available  techniques (the PPM family of statistical compressors), whilst being around  twice as fast at compression and six times faster at decompression. 

*homepage*: <https://sourceware.org/bzip2>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.6``|``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.0.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``

## C


[c-ares](#c-ares) - [C3D](#c3d) - [cadaver](#cadaver) - [CaDiCaL](#cadical) - [CAFE5](#cafe5) - [Caffe](#caffe) - [cairo](#cairo) - [cairomm](#cairomm) - [Calcam](#calcam) - [CalculiX-CrunchiX](#calculix-crunchix) - [Calendrical](#calendrical) - [Calib](#calib) - [CAMPARI](#campari) - [Cantera](#cantera) - [canu](#canu) - [Canvas](#canvas) - [CAP3](#cap3) - [CapnProto](#capnproto) - [captum](#captum) - [Cargo](#cargo) - [Carma](#carma) - [carputils](#carputils) - [Cartopy](#cartopy) - [CASA](#casa) - [casacore](#casacore) - [Casanovo](#casanovo) - [CaSpER](#casper) - [CASPR](#caspr) - [CASTEP](#castep) - [CastXML](#castxml) - [CAT-BAT](#cat-bat) - [Catch2](#catch2) - [category_encoders](#category_encoders) - [CatLearn](#catlearn) - [CatMAP](#catmap) - [causallift](#causallift) - [causalml](#causalml) - [CaVEMan](#caveman) - [CAVIAR](#caviar) - [Cbc](#cbc) - [CBLAS](#cblas) - [ccache](#ccache) - [CCfits](#ccfits) - [CCL](#ccl) - [cclib](#cclib) - [cctbx-base](#cctbx-base) - [cctools](#cctools) - [CD-HIT](#cd-hit) - [CDAT](#cdat) - [cdbfasta](#cdbfasta) - [CDBtools](#cdbtools) - [cddlib](#cddlib) - [CDFlib](#cdflib) - [cDNA_Cupcake](#cdna_cupcake) - [CDO](#cdo) - [cdo-bindings](#cdo-bindings) - [cdsapi](#cdsapi) - [cell2location](#cell2location) - [CellChat](#cellchat) - [CellMix](#cellmix) - [CellOracle](#celloracle) - [Cellpose](#cellpose) - [CellRanger](#cellranger) - [CellRanger-ARC](#cellranger-arc) - [CellRanger-ATAC](#cellranger-atac) - [CellRank](#cellrank) - [CellTypist](#celltypist) - [Centrifuge](#centrifuge) - [Cereal](#cereal) - [CESM-deps](#cesm-deps) - [CFDEMcoupling](#cfdemcoupling) - [CFITSIO](#cfitsio) - [cftime](#cftime) - [CGAL](#cgal) - [cget](#cget) - [Cgl](#cgl) - [CGmapTools](#cgmaptools) - [CGNS](#cgns) - [CharLS](#charls) - [CHASE](#chase) - [Check](#check) - [CheckM](#checkm) - [CheckM-Database](#checkm-database) - [Cheetah](#cheetah) - [Chemaxon-Marvin](#chemaxon-marvin) - [chemprop](#chemprop) - [CheMPS2](#chemps2) - [CHERAB](#cherab) - [chewBBACA](#chewbbaca) - [chi2comb](#chi2comb) - [Chimera](#chimera) - [ChimPipe](#chimpipe) - [Chromaprint](#chromaprint) - [cicero](#cicero) - [CIF2Cell](#cif2cell) - [cimfomfa](#cimfomfa) - [CIRCexplorer](#circexplorer) - [CIRCexplorer2](#circexplorer2) - [Circos](#circos) - [Circuitscape](#circuitscape) - [CIRI](#ciri) - [CIRI-long](#ciri-long) - [cisTEM](#cistem) - [CITE-seq-Count](#cite-seq-count) - [Clang](#clang) - [Clang-AOMP](#clang-aomp) - [Clang-Python-bindings](#clang-python-bindings) - [CLAPACK](#clapack) - [CLEASE](#clease) - [CLHEP](#clhep) - [CliMetLab](#climetlab) - [CLIP](#clip) - [CLISP](#clisp) - [ClonalFrameML](#clonalframeml) - [CLooG](#cloog) - [CloudCompare](#cloudcompare) - [Clp](#clp) - [Clustal-Omega](#clustal-omega) - [ClustalW2](#clustalw2) - [Cluster-Buster](#cluster-buster) - [ClusterShell](#clustershell) - [CMake](#cmake) - [CMAverse](#cmaverse) - [CmdStanR](#cmdstanr) - [cmocean](#cmocean) - [CMSeq](#cmseq) - [CNT-ILP](#cnt-ilp) - [CNVkit](#cnvkit) - [CNVnator](#cnvnator) - [Co-phylog](#co-phylog) - [COBRApy](#cobrapy) - [CoCoALib](#cocoalib) - [CodAn](#codan) - [code-server](#code-server) - [CODEX2](#codex2) - [CodingQuarry](#codingquarry) - [Cogent](#cogent) - [Coin](#coin) - [CoinUtils](#coinutils) - [ColabFold](#colabfold) - [colossalai](#colossalai) - [Commet](#commet) - [CompareM](#comparem) - [Compress-Raw-Zlib](#compress-raw-zlib) - [COMSOL](#comsol) - [Con3F](#con3f) - [conan](#conan) - [CONCOCT](#concoct) - [configparser](#configparser) - [configurable-http-proxy](#configurable-http-proxy) - [CONN](#conn) - [ConnectomeWorkbench](#connectomeworkbench) - [Control-FREEC](#control-freec) - [cooler](#cooler) - [CoordgenLibs](#coordgenlibs) - [Coot](#coot) - [CopyKAT](#copykat) - [core-counter](#core-counter) - [Coreutils](#coreutils) - [corner](#corner) - [CoSymLib](#cosymlib) - [coverage](#coverage) - [cowsay](#cowsay) - [CP2K](#cp2k) - [CPB](#cpb) - [CPC2](#cpc2) - [CPLEX](#cplex) - [CPPE](#cppe) - [CppHeaderParser](#cppheaderparser) - [CppUnit](#cppunit) - [cppy](#cppy) - [cppzmq](#cppzmq) - [cpu_features](#cpu_features) - [cram](#cram) - [cramtools](#cramtools) - [CrayCCE](#craycce) - [CrayGNU](#craygnu) - [CrayIntel](#crayintel) - [CrayPGI](#craypgi) - [crb-blast](#crb-blast) - [CREST](#crest) - [CRF++](#crf++) - [CRISPR-DAV](#crispr-dav) - [CRISPResso2](#crispresso2) - [cromwell](#cromwell) - [crossguid](#crossguid) - [CrossMap](#crossmap) - [CrossTalkZ](#crosstalkz) - [CRPropa](#crpropa) - [Crumble](#crumble) - [cryoCARE](#cryocare) - [cryoDRGN](#cryodrgn) - [CryptoMiniSat](#cryptominisat) - [CrystFEL](#crystfel) - [CSB](#csb) - [CSBDeep](#csbdeep) - [CSBLAST](#csblast) - [cscope](#cscope) - [csvkit](#csvkit) - [ctags](#ctags) - [ctffind](#ctffind) - [CTPL](#ctpl) - [Cube](#cube) - [CubeGUI](#cubegui) - [CubeLib](#cubelib) - [CubeWriter](#cubewriter) - [CuCLARK](#cuclark) - [CUDA](#cuda) - [CUDA-Samples](#cuda-samples) - [CUDAcompat](#cudacompat) - [CUDAcore](#cudacore) - [cuDNN](#cudnn) - [Cufflinks](#cufflinks) - [CUnit](#cunit) - [CuPy](#cupy) - [cURL](#curl) - [cuSPARSELt](#cusparselt) - [custodian](#custodian) - [cutadapt](#cutadapt) - [cuTENSOR](#cutensor) - [cuteSV](#cutesv) - [CVS](#cvs) - [CVXOPT](#cvxopt) - [CVXPY](#cvxpy) - [CWIPI](#cwipi) - [cwltool](#cwltool) - [cxxopts](#cxxopts) - [cysignals](#cysignals) - [Cython](#cython) - [cython-blis](#cython-blis) - [cytoolz](#cytoolz) - [Cytoscape](#cytoscape) - [cytosim](#cytosim) - [cyvcf2](#cyvcf2)


### c-ares

c-ares is a C library for asynchronous DNS requests (including name resolves)

*homepage*: <https://c-ares.org/>

version   |toolchain
----------|--------------------------------------
``1.17.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.18.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### C3D

Convert3D Medical Image Processing Tool

*homepage*: <https://sourceforge.net/projects/c3d/>

version  |toolchain
---------|----------
``1.0.0``|``system``

### cadaver

cadaver is a command-line WebDAV client for Unix.

*homepage*: <http://www.webdav.org/cadaver/>

version   |toolchain
----------|---------------
``0.23.3``|``intel/2017a``

### CaDiCaL

CaDiCaL is a simplified satisfiability solver. The original goal of the development of CaDiCaL was to obtain a CDCL solver, which is simple to understand and change, while at the same time not being much slower than other state-of-the-art CDCL solvers.

*homepage*: <https://github.com/arminbiere/cadical>

version  |toolchain
---------|-------------
``1.3.0``|``GCC/9.3.0``

### CAFE5

Software for Computational Analysis of gene Family Evolution  The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa.

*homepage*: <https://github.com/hahnlab/CAFE5>

version  |toolchain
---------|--------------
``5.0.0``|``GCC/10.2.0``

### Caffe

Caffe is a deep learning framework made with expression, speed,  and modularity in mind. It is developed by the Berkeley Vision  and Learning Center (BVLC) and community contributors. 

*homepage*: <https://github.com/BVLC/caffe>

version|versionsuffix                 |toolchain
-------|------------------------------|---------------
``1.0``|``-CUDA-9.1.85-Python-2.7.14``|``intel/2017b``
``1.0``|``-Python-2.7.13``            |``intel/2017a``
``1.0``|``-Python-2.7.14``            |``intel/2017b``
``rc3``|``-CUDA-7.5.18-Python-2.7.11``|``foss/2016a``

### cairo

Cairo is a 2D graphics library with support for multiple output devices.  Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,  PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB

*homepage*: <https://cairographics.org>

version    |versionsuffix   |toolchain
-----------|----------------|-------------------------------------------------------------------------------------------------------------------
``1.14.6`` |                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.14.6`` |``-GLib-2.48.0``|``foss/2016a``, ``intel/2016a``
``1.14.8`` |                |``intel/2017a``
``1.14.10``|                |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017b``
``1.14.12``|                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.16.0`` |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.17.4`` |                |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### cairomm

The Cairomm package provides a C++ interface to Cairo.

*homepage*: <http://cairographics.org>

version   |toolchain
----------|------------------------------------
``1.12.2``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``

### Calcam

Calcam is a Python package providing tools for spatial calibration  of cameras, i.e. determining the mapping between pixel coordinates in an image and  real-world 3D sight lines & coordinates.

*homepage*: <https://euratom-software.github.io/calcam>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.0``|``-Python-2.7.14``|``intel/2018a``
``2.1.0``|``-Python-3.6.4`` |``intel/2018a``

### CalculiX-CrunchiX

A Free Software Three-Dimensional Structural Finite Element Program

*homepage*: <http://www.calculix.de>

version |toolchain
--------|--------------
``2.20``|``foss/2021a``

### Calendrical

Calendrical module is for calendrical calculations.

*homepage*: <https://www.funaba.org/code#calendrical>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``2.0.1`` |``-Python-3.6.3``|``intel/2017b``
``2.0.2a``|``-Python-3.6.4``|``intel/2018a``
``2.0.2a``|``-Python-3.6.6``|``intel/2018b``

### Calib

Calib clusters paired-end reads using their barcodes and sequences. Calib is suitable for amplicon sequencing where a molecule is tagged, then PCR amplified with high depth, also known as Unique Molecule Identifier (UMI) sequencing.

*homepage*: <https://github.com/vpc-ccg/calib>

version  |toolchain
---------|-------------
``0.3.4``|``GCC/9.3.0``

### CAMPARI

CAMPARI is a joint package for performing and analyzing molecular simulations, in particular of systems of biological relevance. It focuses on a wide availability of algorithms for (advanced) sampling and is capable of combining Monte Carlo and molecular dynamics in seamless fashion.

*homepage*: <http://campari.sourceforge.net/V4/index.html>

version|toolchain
-------|---------------
``4.0``|``intel/2020b``

### Cantera

Chemical kinetics, thermodynamics, and transport tool suite

*homepage*: <https://github.com/Cantera/cantera>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``2.2.1``|``-Python-2.7.12``|``intel/2016b``
``2.3.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.3.0``|``-Python-2.7.13``|``intel/2017a``
``2.3.0``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.4.0``|``-Python-2.7.14``|``intel/2018a``
``2.6.0``|                  |``foss/2022a``

### canu

Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing

*homepage*: <https://canu.readthedocs.io>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------
``1.4``  |                |``foss/2016b``
``1.7``  |                |``intel/2018a``
``1.8``  |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``1.8``  |``-Perl-5.28.0``|``foss/2018b``
``1.8``  |``-Perl-5.28.1``|``GCCcore/8.2.0``
``1.9``  |``-Java-11``    |``GCCcore/8.3.0``
``2.1.1``|                |``GCCcore/10.2.0``
``2.1.1``|``-Java-11``    |``GCCcore/9.3.0``
``2.2``  |                |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``2.2``  |``-Java-11``    |``GCCcore/10.3.0``

### Canvas

Copy number variant (CNV) calling from DNA sequencing data

*homepage*: <https://github.com/Illumina/canvas>

version        |toolchain
---------------|----------
``1.39.0.1598``|``system``

### CAP3

CAP3 assembly program 

*homepage*: <http://seq.cs.iastate.edu/>

version                  |toolchain
-------------------------|----------
``20071221-intel-x86``   |``system``
``20071221-intel-x86_64``|``system``
``20071221-opteron``     |``system``

### CapnProto

Cap’n Proto is an insanely fast data interchange format and capability-based RPC system.

*homepage*: <https://capnproto.org>

version   |toolchain
----------|----------------------------------------------------------
``0.6.1`` |``GCCcore/6.4.0``
``0.7.0`` |``GCCcore/7.3.0``
``0.8.0`` |``GCCcore/9.3.0``
``0.9.1`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.10.2``|``GCCcore/11.3.0``
``0.10.3``|``GCCcore/12.2.0``

### captum

Captum is a model interpretability and understanding library for PyTorch. Captum means comprehension in Latin and contains general purpose implementations of integrated gradients, saliency maps, smoothgrad, vargrad and others for PyTorch models. It has quick integration for models built with domain-specific libraries such as torchvision, torchtext, and others.

*homepage*: <https://captum.ai/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.5.0``|                |``foss/2022a``
``0.5.0``|``-CUDA-11.7.0``|``foss/2022a``

### Cargo

The Rust package manager

*homepage*: <https://crates.io/>

version   |toolchain
----------|--------------
``0.13.0``|``foss/2016b``

### Carma

Carma - A molecular dynamics analysis program

*homepage*: <http://utopia.duth.gr/~glykos/Carma.html>

version |toolchain
--------|--------------
``2.01``|``foss/2019b``

### carputils

carputils is a Python framework for generating and running openCARP examples.

*homepage*: <https://git.opencarp.org/openCARP/carputils>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20200915``|``-Python-3.8.2``|``foss/2020a``
``20210513``|                 |``foss/2020b``

### Cartopy

Cartopy is a Python package designed to make drawing maps for data analysis and visualisation simple.

*homepage*: <https://scitools.org.uk/cartopy/docs/latest/>

version         |versionsuffix    |toolchain
----------------|-----------------|-------------------------------
``0.18.0``      |``-Python-3.7.4``|``foss/2019b``
``0.18.0``      |``-Python-3.8.2``|``foss/2020a``
``0.19.0.post1``|                 |``foss/2020b``, ``intel/2020b``
``0.20.0``      |                 |``foss/2021a``
``0.20.3``      |                 |``foss/2021b``, ``foss/2022a``

### CASA

CASA, the Common Astronomy Software Applications package, is the primary data processing software for the Atacama Large Millimeter/submillimeter Array (ALMA) and NSF's Karl G. Jansky Very Large Array (VLA), and is frequently used also for other radio telescopes. The CASA software can process data from both single-dish and aperture-synthesis telescopes, and one of its core functionalities is to support the data reduction and imaging pipelines for ALMA, VLA and the VLA Sky Survey (VLASS). 

*homepage*: <https://casa.nrao.edu/>

version     |versionsuffix|toolchain
------------|-------------|----------
``6.5.5-21``|``-py3.8``   |``system``

### casacore

A suite of C++ libraries for radio astronomy data processing. The ephemerides data needs to be in DATA_DIR and the location must be specified at runtime. Thus user's can update them. 

*homepage*: <https://github.com/casacore/casacore>

version  |toolchain
---------|--------------
``3.4.0``|``foss/2021b``

### Casanovo

De Novo Mass Spectrometry Peptide Sequencing with a Transformer Model

*homepage*: <https://casanovo.readthedocs.io>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.3.0``|                |``foss/2022a``
``3.3.0``|``-CUDA-11.7.0``|``foss/2022a``

### CaSpER

CaSpER is a signal processing approach for identification, visualization, and integrative analysis  of focal and large-scale CNV events in multiscale resolution using either bulk or single-cell RNA sequencing data. 

*homepage*: <https://github.com/akdess/CaSpER>

version|toolchain
-------|--------------
``2.0``|``foss/2019b``

### CASPR

Running CASPR is extremely simple and convenient to analyze CRIPR-Cas9 screens using pgRNAs.

*homepage*: <https://judithbergada.github.io/CASPR>

version     |toolchain
------------|--------------
``20200730``|``foss/2022a``

### CASTEP

CASTEP is an electronic structure materials modelling code based on density functional theory (DFT), with functionality including geometry optimization molecular dynamics, phonons, NMR chemical shifts and much more. 

*homepage*: <http://www.castep.org>

version   |toolchain
----------|--------------------------------------------------------------------------------------
``21.1.1``|``CrayCCE/19.06``, ``CrayGNU/19.06``, ``foss/2019b``, ``intel/2019b``, ``iomkl/2019b``
``22.11`` |``foss/2022a``

### CastXML

CastXML is a C-family abstract syntax tree XML output tool.

*homepage*: <https://github.com/CastXML/CastXML>

version     |toolchain
------------|-----------------
``0.4.3``   |``GCCcore/8.3.0``
``20160617``|``foss/2016a``
``20180806``|``foss/2018a``

### CAT-BAT

Tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs).

*homepage*: <https://github.com/dutilh/CAT>

version  |toolchain
---------|--------------
``5.2.3``|``GCC/10.3.0``

### Catch2

A modern, C++-native, header-only,   test framework for unit-tests, TDD and BDD   - using C++11, C++14, C++17 and later     (or C++03 on the Catch1.x branch) 

*homepage*: <https://github.com/catchorg/Catch2>

version   |toolchain
----------|----------
``2.9.1`` |``system``
``2.11.0``|``system``
``2.13.4``|``system``

### category_encoders

A set of scikit-learn-style transformers for encoding categorical variables into numeric by means of different techniques.

*homepage*: <https://github.com/scikit-learn-contrib/category_encoders>

version  |toolchain
---------|--------------
``2.4.1``|``foss/2021b``

### CatLearn

An environment for atomistic machine learning in Python for applications in catalysis

*homepage*: <https://catlearn.readthedocs.io>

version  |toolchain
---------|---------------
``0.6.2``|``intel/2022a``

### CatMAP

Catalyst Micro-kinetic Analysis Package for automated creation of micro-kinetic models used  in catalyst screening.

*homepage*: <https://catmap.readthedocs.io>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170927``|``-Python-2.7.14``|``intel/2017b``
``20220519``|                  |``foss/2022a``

### causallift

CausalLift: Python package for Uplift Modeling in real-world business; applicable for both A/B testing and observational data 

*homepage*: <https://github.com/Minyus/causallift>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.6``|``-Python-3.8.2``|``foss/2020a``

### causalml

Causal ML: A Python Package for Uplift Modeling and Causal Inference with ML 

*homepage*: <https://github.com/uber/causalml>

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``0.3.0-20180610``|``-Python-3.7.2``|``foss/2019a``
``0.8.0-20200909``|``-Python-3.8.2``|``foss/2020a``

### CaVEMan

SNV expectation maximisation based mutation calling algorithm   aimed at detecting somatic mutations in paired (tumour/normal) cancer samples.  Supports both bam and cram format via htslib

*homepage*: <http://cancerit.github.io/CaVEMan/>

version   |toolchain
----------|--------------
``1.13.2``|``foss/2018a``

### CAVIAR

CAusal Variants Identication in Associated Regions. A statistical framework  that quantifies the probability of each variant to be causal while allowing an arbitrary  number of causal variants.

*homepage*: <https://github.com/fhormoz/caviar>

version         |toolchain
----------------|--------------
``2.2-20190419``|``foss/2019b``

### Cbc

Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming solver written in C++. It can be used as a callable library or using a stand-alone executable.

*homepage*: <https://github.com/coin-or/Cbc>

version   |toolchain
----------|------------------------------
``2.10.3``|``foss/2018b``
``2.10.5``|``foss/2020b``, ``foss/2021a``

### CBLAS

C interface to the BLAS

*homepage*: <https://www.netlib.org/blas/>

version     |toolchain
------------|------------------------------------------------
``20110120``|``foss/2016b``, ``intel/2019b``, ``intel/2020a``

### ccache

Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by caching previous compilations and detecting when the same compilation is being done again

*homepage*: <https://ccache.dev/>

version   |versionsuffix|toolchain
----------|-------------|------------------------------
``3.2.5`` |             |``system``
``3.3.1`` |             |``system``
``3.3.3`` |             |``system``
``3.3.4`` |``-f90``     |``system``
``3.7.11``|             |``system``
``4.2.1`` |             |``system``
``4.6.1`` |             |``GCCcore/11.2.0``
``4.6.3`` |             |``GCCcore/11.3.0``, ``system``

### CCfits

CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++.

*homepage*: <https://heasarc.gsfc.nasa.gov/fitsio/CCfits/>

version|toolchain
-------|-----------------
``2.5``|``GCCcore/9.3.0``

### CCL

Clozure CL (often called CCL for short) is a free Common Lisp  implementation with a long history. Some distinguishing features of the implementation  include fast compilation speed, native threads, a precise, generational, compacting  garbage collector, and a convenient foreign-function interface.

*homepage*: <https://ccl.clozure.com/>

version   |toolchain
----------|------------------
``1.11.5``|``system``
``1.12``  |``GCCcore/9.3.0``
``1.12.1``|``GCCcore/10.3.0``

### cclib

cclib is a Python library that provides parsers for computational chemistry log files.  It also provides a platform to implement algorithms in a package-independent manner. 

*homepage*: <https://cclib.github.io/>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.5``  |``-Python-3.5.2``|``foss/2016b``
``1.6.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``1.7.2``|                 |``foss/2021b``

### cctbx-base

The Computational Crystallography Toolbox (cctbx) is being developed as the open source component of the Phenix project. The goal of the Phenix project is to advance automation of macromolecular structure determination. Phenix depends on the cctbx, but not vice versa. This hierarchical approach enforces a clean design as a reusable library. The cctbx is therefore also useful for small-molecule crystallography and even general scientific applications. 

*homepage*: <https://github.com/cctbx/cctbx_project>

version   |toolchain
----------|----------------------------------
``2020.8``|``foss/2020b``, ``fosscuda/2020b``

### cctools

The Cooperating Computing Tools (CCTools) help you to design and deploy  scalable applications that run on hundreds or thousands of machines at once. 

*homepage*: <https://ccl.cse.nd.edu/>

version   |toolchain
----------|-----------------
``7.0.22``|``GCCcore/8.3.0``

### CD-HIT

CD-HIT is a very widely used program for clustering and  comparing protein or nucleotide sequences.

*homepage*: <https://github.com/weizhongli/cdhit>

version  |versionsuffix |toolchain
---------|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------
``4.6.4``|``-2015-0603``|``GNU/4.9.3-2.25``
``4.6.6``|              |``foss/2016b``
``4.6.8``|              |``foss/2018b``, ``intel/2017a``, ``intel/2018a``
``4.8.1``|              |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``iccifort/2019.5.281``

### CDAT

CDAT is a powerful and complete front-end to a rich set of  visual-data exploration and analysis capabilities well suited for data analysis problems.

*homepage*: <https://github.com/CDAT/cdat>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``8.2.1``|``-Python-3.8.2``|``foss/2020a``

### cdbfasta

Fasta file indexing and retrival tool

*homepage*: <https://sourceforge.net/projects/cdbfasta>

version |toolchain
--------|--------------------------------------------------------------------------------
``0.99``|``GCC/8.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``

### CDBtools

CDB (Constant DataBase) indexing and retrieval tools for FASTA files

*homepage*: <http://compbio.dfci.harvard.edu/tgi>

version |toolchain
--------|------------------------------
``0.99``|``GCC/10.2.0``, ``GCC/11.3.0``

### cddlib

An efficient implementation of the Double Description Method

*homepage*: <https://github.com/cddlib/cddlib>

version  |toolchain
---------|-----------------
``0.94i``|``GCCcore/8.2.0``
``0.94j``|``GCCcore/8.3.0``

### CDFlib

cdflib is a python module to read/write CDF (Common Data Format .cdf) files without needing to install the CDF NASA library. 

*homepage*: <https://github.com/MAVENSDC/cdflib>

version  |toolchain
---------|--------------
``0.4.9``|``foss/2022a``

### cDNA_Cupcake

cDNA_Cupcake is a miscellaneous collection of Python and R scripts used for analyzing sequencing data.

*homepage*: <https://github.com/Magdoll/cDNA_Cupcake>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``5.8``   |``-Python-2.7.14``|``intel/2018a``
``24.2.0``|``-Python-3.8.2`` |``foss/2020a``
``26.0.0``|                  |``foss/2021a``

### CDO

CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.

*homepage*: <https://code.zmaw.de/projects/cdo>

version   |toolchain
----------|------------------------------------------------------------------
``1.7.2`` |``intel/2016b``
``1.8.1`` |``intel/2017a``
``1.9.1`` |``intel/2017b``
``1.9.2`` |``intel/2017b``
``1.9.5`` |``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``1.9.8`` |``intel/2019b``
``1.9.10``|``gompi/2019b``, ``gompi/2020b``, ``gompi/2021a``, ``iimpi/2021b``
``2.0.5`` |``gompi/2021b``
``2.0.6`` |``gompi/2022a``
``2.1.1`` |``gompi/2021a``

### cdo-bindings

Python interface to CDO.

*homepage*: <https://github.com/Try2Code/cdo-bindings>

version  |toolchain
---------|--------------
``1.5.7``|``foss/2021b``

### cdsapi

Climate Data Store API

*homepage*: <https://pypi.org/project/cdsapi>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.1.4``|                 |``foss/2019a``
``0.1.4``|``-Python-3.6.6``|``foss/2018b``
``0.3.0``|                 |``GCCcore/9.3.0``
``0.5.1``|                 |``GCCcore/11.3.0``

### cell2location

Comprehensive mapping of tissue cell architecture via integrated  single cell and spatial transcriptomics (cell2location model)

*homepage*: <https://github.com/BayraktarLab/cell2location/>

version       |toolchain
--------------|------------------
``0.05-alpha``|``fosscuda/2020b``

### CellChat

" R toolkit for inference, visualization and analysis of cell-cell communication from single-cell data

*homepage*: <https://github.com/sqjin/CellChat>

version  |toolchain
---------|--------------
``1.5.0``|``foss/2022a``

### CellMix

A Comprehensive Toolbox for Gene Expression Deconvolution

*homepage*: <http://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.6.2``|``-R-3.5.1`` |``foss/2018b``

### CellOracle

CellOracle is a Python library for in silico gene perturbation analyses using single-cell omics data and Gene Regulatory Network models.

*homepage*: <https://github.com/morris-lab/CellOracle>

version   |toolchain
----------|--------------
``0.12.0``|``foss/2022a``

### Cellpose

a generalist algorithm for cellular segmentation

*homepage*: <https://www.cellpose.org>

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------
``0.6.5``|                |``foss/2020b``, ``fosscuda/2020b``
``2.2.2``|                |``foss/2022a``
``2.2.2``|``-CUDA-11.7.0``|``foss/2022a``

### CellRanger

Cell Ranger is a set of analysis pipelines that process Chromium  single-cell RNA-seq output to align reads, generate gene-cell matrices and perform  clustering and gene expression analysis.

*homepage*: <https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger>

version  |toolchain
---------|----------
``3.0.0``|``system``
``3.0.2``|``system``
``3.1.0``|``system``
``4.0.0``|``system``
``5.0.0``|``system``
``5.0.1``|``system``
``6.0.0``|``system``
``6.0.1``|``system``
``6.0.2``|``system``
``6.1.2``|``system``
``7.0.0``|``system``
``7.1.0``|``system``

### CellRanger-ARC

Cell Ranger ARC is a set of analysis pipelines that process  Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a  variety of analyses pertaining to gene expression, chromatin accessibility and  their linkage. Furthermore, since the ATAC and gene expression measurements are on  the very same cell, we are able to perform analyses that link chromatin  accessibility and gene expression.

*homepage*: <https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc>

version  |toolchain
---------|----------
``1.0.1``|``system``
``2.0.0``|``system``
``2.0.1``|``system``

### CellRanger-ATAC

Cell Ranger ATAC is a set of analysis pipelines that process  Chromium Single Cell ATAC data.

*homepage*: <https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac>

version  |toolchain
---------|----------
``1.2.0``|``system``
``2.0.0``|``system``
``2.1.0``|``system``

### CellRank

CellRank is a toolkit to uncover cellular dynamics based on  Markov state modeling of single-cell data.  It contains two main modules: kernels compute cell-cell transition probabilities and estimators generate hypothesis based on these. 

*homepage*: <https://cellrank.readthedocs.io/en/stable/>

version  |toolchain
---------|--------------
``1.4.0``|``foss/2021a``

### CellTypist

A tool for semi-automatic cell type annotation

*homepage*: <https://www.celltypist.org/>

version  |toolchain
---------|--------------
``1.0.0``|``foss/2021b``

### Centrifuge

Classifier for metagenomic sequences

*homepage*: <https://ccb.jhu.edu/software/centrifuge>

version       |toolchain
--------------|--------------------------------
``1.0.3``     |``foss/2018b``
``1.0.4``     |``gompi/2020b``, ``gompi/2021a``
``1.0.4-beta``|``foss/2018b``, ``gompi/2020a``

### Cereal

cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly  turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and simple to extend - it has no external dependencies and can be easily bundled with other code or  used standalone.

*homepage*: <https://uscilab.github.io/cereal/>

version  |toolchain
---------|----------
``1.3.0``|``system``
``1.3.2``|``system``

### CESM-deps

CESM is a fully-coupled, community, global climate model that provides state-of-the-art  computer simulations of the Earth's past, present, and future climate states.

*homepage*: <https://www.cesm.ucar.edu/models/cesm2/>

version|toolchain
-------|----------------------------------------------------------------
``2``  |``foss/2018b``, ``foss/2021b``, ``intel/2018b``, ``iomkl/2018b``

### CFDEMcoupling

CFDEMcoupling is an open source CFD-DEM engine. It provides the possibility to couple  the DEM engine LIGGGHTS to a CFD framework.

*homepage*: <https://www.cfdem.com/cfdemrcoupling-open-source-cfd-dem-framework>

version  |toolchain
---------|--------------
``3.8.0``|``foss/2018a``

### CFITSIO

CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.

*homepage*: <https://heasarc.gsfc.nasa.gov/fitsio/>

version  |toolchain
---------|----------------------------------------------------------
``3.38`` |``foss/2016a``, ``intel/2016a``
``3.41`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``intel/2016b``
``3.42`` |``GCCcore/6.4.0``, ``intel/2017b``
``3.45`` |``GCCcore/7.3.0``, ``intel/2018b``
``3.47`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.48`` |``GCCcore/9.3.0``
``3.49`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``4.2.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### cftime

Time-handling functionality from netcdf4-python

*homepage*: <https://github.com/Unidata/cftime>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``1.0.0``  |``-Python-3.6.4`` |``intel/2018a``
``1.0.0b1``|``-Python-3.6.2`` |``foss/2017b``
``1.0.1``  |``-Python-2.7.15``|``intel/2018b``
``1.0.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``

### CGAL

The goal of the CGAL Open Source Project is to provide simple access to efficient  and reliable geometric algorithms in the form of a C++ library.

*homepage*: <https://www.cgal.org/>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------
``4.8``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.8.1`` |                  |``foss/2016b``, ``intel/2016b``
``4.8.1`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.9``   |``-Python-2.7.12``|``intel/2016b``
``4.9``   |``-Python-2.7.13``|``intel/2017a``
``4.11``  |``-Python-2.7.13``|``intel/2017a``
``4.11``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``4.11``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``4.11.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``4.11.1``|``-Python-2.7.15``|``foss/2018b``
``4.11.1``|``-Python-3.6.4`` |``foss/2018a``
``4.11.1``|``-Python-3.6.6`` |``foss/2018b``
``4.14``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``4.14.1``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``4.14.3``|                  |``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``iimpi/2021a``
``4.14.3``|``-Python-3.8.2`` |``gompi/2020a``, ``iimpi/2020a``
``5.2``   |                  |``gompi/2020b``

### cget

Cmake package retrieval. This can be used to download and install cmake packages

*homepage*: <https://cget.readthedocs.io/en/latest/index.html>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.6``|``-Python-3.6.4``|``foss/2018a``

### Cgl

The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that can be used with other COIN-OR packages that make use of cuts, such as, among others, the linear solver Clp or the mixed integer linear programming solvers Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or communicate with a solver. It does not directly call a solver.

*homepage*: <https://github.com/coin-or/Cgl>

version   |toolchain
----------|------------------------------
``0.60.2``|``foss/2018b``
``0.60.3``|``foss/2020b``, ``foss/2021a``

### CGmapTools

Command-line Toolset for Bisulfite Sequencing Data Analysis

*homepage*: <https://cgmaptools.github.io/>

version  |toolchain
---------|---------------
``0.1.2``|``intel/2019b``

### CGNS

The CGNS system is designed to facilitate the exchange of data between sites and applications,  and to help stabilize the archiving of aerodynamic data.

*homepage*: <https://cgns.github.io/>

version  |toolchain
---------|-------------------
``3.3.1``|``foss/2016b``
``4.1.0``|``intelcuda/2019b``

### CharLS

CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000 compression ratios.

*homepage*: <https://github.com/team-charls/charls>

version  |toolchain
---------|------------------------------------
``2.0.0``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.1.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.2.0``|``GCCcore/10.2.0``
``2.3.4``|``GCCcore/10.3.0``
``2.4.1``|``GCCcore/11.3.0``

### CHASE

Case-control HAplotype Sharing analyses.   Haplotype sharing analyses for genome-wide association studies.

*homepage*: <http://people.duke.edu/~asallen/Software.html>

version     |toolchain
------------|----------
``20130626``|``system``

### Check

Check is a unit testing framework for C. It features a simple interface for defining unit tests, putting little in the way of the developer. Tests are run in a separate address space, so both assertion failures and code errors that cause segmentation faults or other signals can be caught. Test results are reportable in the following: Subunit, TAP, XML, and a generic logging format.

*homepage*: <https://libcheck.github.io/check/>

version   |toolchain
----------|-------------------------------------------------------------------------------------------------
``0.12.0``|``GCCcore/6.4.0``
``0.15.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### CheckM

CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,  single cells, or metagenomes.

*homepage*: <https://github.com/Ecogenomics/CheckM>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0.13``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.0.13``|``-Python-2.7.15``|``foss/2018b``
``1.0.13``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.0.18``|``-Python-2.7.15``|``foss/2019a``
``1.0.18``|``-Python-2.7.18``|``foss/2020b``
``1.1.2`` |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.1.3`` |                  |``foss/2021a``, ``foss/2021b``
``1.1.3`` |``-Python-3.8.2`` |``intel/2020a``
``1.2.2`` |                  |``foss/2022a``

### CheckM-Database

CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.  This is the corresponding database. 

*homepage*: <https://github.com/Ecogenomics/CheckM>

version       |toolchain
--------------|----------
``2015_01_16``|``system``

### Cheetah

Cheetah is an open source template engine and code generation tool.

*homepage*: <https://cheetahtemplate.org>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.4.4``|``-Python-2.7.15``|``intel/2018b``

### Chemaxon-Marvin

Marvin suite is a chemically intelligent desktop toolkit built to help you draw, edit, publish, render, import and export your chemical structures and as well as allowing you to convert between various chemical and graphical file formats. It is free for individual, academic and non-commercial use.

*homepage*: <https://chemaxon.com/products/marvin>

version  |toolchain
---------|----------
``21.14``|``system``

### chemprop

Message Passing Neural Networks for Molecule Property Prediction

*homepage*: <https://github.com/chemprop/chemprop>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.2``|                |``foss/2022a``
``1.5.2``|``-CUDA-11.7.0``|``foss/2022a``

### CheMPS2

CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.

*homepage*: <https://github.com/SebWouters/CheMPS2>

version    |toolchain
-----------|---------------------------------------------------------------------------------
``1.6``    |``intel/2016a``
``1.7-rc2``|``intel/2016a``
``1.7.1``  |``intel/2016a``
``1.7.2``  |``intel/2016a``
``1.8``    |``intel/2016b``
``1.8.8``  |``intel/2018b``
``1.8.9``  |``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``, ``intel/2019b``
``1.8.11`` |``foss/2021b``
``1.8.12`` |``foss/2022a``

### CHERAB

CHERAB is a python library for forward modelling diagnostics  based on spectroscopic plasma emission.

*homepage*: <https://cherab.github.io/documentation/index.html>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.2.0``|``-Python-3.6.6``|``intel/2018b``
``1.3.0``|                 |``foss/2020b``, ``intel/2020b``
``1.4.0``|                 |``foss/2020b``, ``intel/2020b``

### chewBBACA

chewBBACA stands for "BSR-Based Allele Calling Algorithm". chewBBACA is a comprehensive pipeline including a set of functions for the creation and validation of whole genome and core genome MultiLocus Sequence Typing (wg/cgMLST) schemas, providing an allele calling algorithm based on Blast Score Ratio that can be run in multiprocessor settings and a set of functions to visualize and validate allele variation in the loci.

*homepage*: <https://github.com/B-UMMI/chewBBACA>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.5``|``-Python-3.8.2``|``intel/2020a``

### chi2comb

Cumulative density function of linear combinations of independent chi-square random variables and a standard Normal distribution. As of now, this is basically a repackaging of the davies function implemented in the CompQuadForm library for R.

*homepage*: <https://github.com/limix/chi2comb>

version  |toolchain
---------|-----------------
``0.0.3``|``GCCcore/7.3.0``

### Chimera

UCSF Chimera is a highly extensible program for interactive visualization   and analysis of molecular structures and related data, including density maps, supramolecular   assemblies, sequence alignments, docking results, trajectories, and conformational ensembles. 

*homepage*: <https://www.cgl.ucsf.edu/chimera/>

version |versionsuffix    |toolchain
--------|-----------------|----------
``1.10``|``-linux_x86_64``|``system``
``1.16``|``-linux_x86_64``|``system``

### ChimPipe

ChimPipe is a computational method for the detection of novel transcription-induced  chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction  spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair  resolution.

*homepage*: <https://chimpipe.readthedocs.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.9.5``|                  |``foss/2018b``
``0.9.5``|``-Python-2.7.12``|``foss/2016b``

### Chromaprint

Chromaprint is the core component of the AcoustID project. It's a client-side library  that implements a custom algorithm for extracting fingerprints from any audio source.

*homepage*: <https://acoustid.org/chromaprint>

version  |toolchain
---------|-----------------
``1.4.3``|``GCCcore/8.2.0``

### cicero

Cicero is an R package that provides tools for analyzing single-cell chromatin accessibility  experiments.

*homepage*: <https://cole-trapnell-lab.github.io/cicero-release>

version     |versionsuffix        |toolchain
------------|---------------------|--------------
``1.3.4.11``|``-R-4.0.3-Monocle3``|``foss/2020b``

### CIF2Cell

CIF2Cell is a tool to generate the geometrical setup for various electronic structure codes from a CIF (Crystallographic Information Framework) file. The program currently supports output for a number of popular electronic structure programs, including ABINIT, ASE, CASTEP, CP2K, CPMD, CRYSTAL09, Elk, EMTO, Exciting, Fleur, FHI-aims, Hutsepot, MOPAC, Quantum Espresso, RSPt, Siesta, SPR-KKR, VASP. Also exports some related formats like .coo, .cfg and .xyz-files.

*homepage*: <https://sourceforge.net/projects/cif2cell>

version   |versionsuffix     |toolchain
----------|------------------|-----------------
``1.2.10``|``-Python-2.7.16``|``GCCcore/8.3.0``

### cimfomfa

This library supports both MCL, a cluster algorithm for graphs, and zoem, a macro/DSL language. It supplies abstractions for memory management, I/O, associative arrays, strings, heaps, and a few other things. The string library has had heavy testing as part of zoem. Both understandably and regrettably I chose long ago to make it C-string-compatible, hence nul bytes may not be part of a string. At some point I hope to rectify this, perhaps unrealistically.

*homepage*: <https://github.com/micans/cimfomfa>

version   |toolchain
----------|------------------
``22.273``|``GCCcore/11.3.0``

### CIRCexplorer

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: <https://circexplorer2.readthedocs.io/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.1.10``|``-Python-2.7.14``|``intel/2017b``

### CIRCexplorer2

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: <http://circexplorer2.readthedocs.io/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.3.2``|``-Python-2.7.14``|``intel/2017b``
``2.3.3``|``-Python-2.7.14``|``intel/2018a``
``2.3.8``|``-Python-2.7.18``|``foss/2020b``

### Circos

Circos is a software package for visualizing data and information.  It visualizes data in a circular layout - this makes Circos ideal for exploring  relationships between objects or positions.

*homepage*: <http://www.circos.ca/>

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------
``0.69-5``|``-Perl-5.24.0``|``foss/2016b``
``0.69-6``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``0.69-6``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``0.69-9``|                |``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### Circuitscape

Algorithms from circuit theory to predict connectivity in heterogeneous landscapes

*homepage*: <https://github.com/Circuitscape/Circuitscape.jl>

version   |versionsuffix   |toolchain
----------|----------------|----------
``5.12.3``|``-Julia-1.7.2``|``system``

### CIRI

CircRNA Identifier. A de novo circular RNA identification tool

*homepage*: <https://sourceforge.net/projects/ciri/>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.0.6``|``-Perl-5.26.0``|``intel/2017b``

### CIRI-long

Circular RNA Identification for Long-Reads Nanopore Sequencing Data

*homepage*: <https://github.com/bioinfo-biols/CIRI-long/>

version  |toolchain
---------|--------------
``1.0.2``|``foss/2020b``

### cisTEM

cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes  and obtain high-resolution 3D reconstructions from them. 

*homepage*: <https://cistem.org/>

version       |toolchain
--------------|--------------
``1.0.0-beta``|``foss/2018a``

### CITE-seq-Count

A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.

*homepage*: <https://github.com/Hoohm/CITE-seq-Count>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.4.3``|``-Python-3.6.6``|``foss/2018b``
``1.4.3``|``-Python-3.7.4``|``foss/2019b``

### Clang

C, C++, Objective-C compiler, based on LLVM.  Does not  include C++ standard library -- use libstdc++ from GCC.

*homepage*: <https://clang.llvm.org/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``3.3``   |                  |``GCC/4.8.1``
``3.4``   |                  |``GCC/4.8.2``
``3.4.1`` |                  |``GCC/4.8.2``
``3.4.2`` |                  |``GCC/4.8.2``
``3.6.0`` |                  |``GCC/4.9.2``
``3.6.1`` |                  |``GCC/4.9.2``
``3.7.0`` |                  |``GNU/4.9.3-2.25``
``3.7.1`` |                  |``GCC/4.9.3-2.25``, ``foss/2016a``
``3.8.0`` |                  |``GCC/4.9.3-2.25``
``3.8.1`` |                  |``GCC/5.4.0-2.26``, ``foss/2016b``
``5.0.0`` |                  |``GCC/6.4.0-2.28``
``5.0.1`` |                  |``GCC/6.4.0-2.28``
``6.0.1`` |                  |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``
``7.0.0`` |                  |``GCC/6.4.0-2.28``
``7.0.1`` |                  |``GCC/7.3.0-2.30``
``8.0.0`` |                  |``GCCcore/8.2.0``
``8.0.0`` |``-CUDA-10.1.105``|``GCCcore/8.2.0``
``8.0.1`` |``-CUDA-10.1.105``|``GCC/8.2.0-2.31.1``
``8.0.1`` |``-CUDA-10.1.243``|``GCC/8.3.0``
``9.0.1`` |                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0.1`` |``-CUDA-10.1.243``|``GCC/8.3.0``
``10.0.0``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``10.0.1``|                  |``GCCcore/9.3.0``
``11.0.0``|                  |``GCCcore/9.3.0``
``11.0.1``|                  |``GCCcore/10.2.0``, ``gcccuda/2020b``
``12.0.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``12.0.1``|``-CUDA-11.3.1``  |``GCCcore/10.3.0``
``13.0.1``|                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``13.0.1``|``-CUDA-11.4.1``  |``GCCcore/11.2.0``
``15.0.5``|                  |``GCCcore/11.3.0``

### Clang-AOMP

AOMP is an open source Clang/LLVM based compiler with added support for the OpenMP® API on Radeon™ GPUs.

*homepage*: <https://github.com/RadeonOpenCompute/llvm-project>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### Clang-Python-bindings

Python bindings for libclang

*homepage*: <https://clang.llvm.org>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``8.0.0`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``10.0.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``13.0.1``|                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### CLAPACK

C version of LAPACK

*homepage*: <http://www.netlib.org/clapack>

version  |toolchain
---------|---------------------------------------------------------------------------
``3.2.1``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``

### CLEASE

CLuster Expansion in Atomic Simulation Environment (CLEASE) is a package that automates the cumbersome setup and construction procedure of cluster expansion (CE). It provides a comprehensive list of tools for specifying parameters for CE, generating training structures, fitting effective cluster interaction (ECI) values and running Monte Carlo simulations.

*homepage*: <https://gitlab.com/computationalmaterials/clease>

version   |toolchain
----------|---------------
``0.10.6``|``intel/2021a``

### CLHEP

The CLHEP project is intended to be a set of HEP-specific foundation and  utility classes such as random generators, physics vectors, geometry and linear algebra.  CLHEP is structured in a set of packages independent of any external package.

*homepage*: <https://proj-clhep.web.cern.ch/proj-clhep/>

version    |toolchain
-----------|----------------------------------------------------------------
``2.1.1.0``|``intel/2016a``
``2.1.3.1``|``intel/2016a``
``2.2.0.8``|``intel/2016a``
``2.3.1.1``|``intel/2016a``
``2.3.4.3``|``foss/2017b``, ``intel/2017b``
``2.4.0.0``|``intel/2017b``
``2.4.1.0``|``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
``2.4.1.3``|``foss/2019b``, ``foss/2020a``
``2.4.4.0``|``GCC/10.2.0``, ``GCC/11.2.0``
``2.4.5.1``|``GCC/11.2.0``
``2.4.5.3``|``GCC/11.3.0``
``2.4.6.2``|``GCC/11.3.0``
``2.4.6.4``|``GCC/12.2.0``

### CliMetLab

CliMetLab is a Python package aiming at simplifying access to climate and meteorological datasets, allowing users to focus on science instead of technical issues such as data access and data formats. It is mostly intended to be used in Jupyter notebooks, and be interoperable with all popular data analytic packages, such as Numpy, Pandas, Xarray, SciPy, Matplotlib, etc. as well as machine learning frameworks, such as Tensorflow, Keras or PyTorch.

*homepage*: <https://climetlab.readthedocs.io/>

version   |toolchain
----------|--------------
``0.12.6``|``foss/2022a``

### CLIP

CLIP (Contrastive Language-Image Pre-Training) is a neural network trained on a variety of (image, text) pairs. It can be instructed in natural language to predict the most relevant text snippet, given an image, without directly optimizing for the task, similarly to the zero-shot capabilities of GPT-2 and 3.

*homepage*: <https://github.com/openai/CLIP>

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20230220``|``-CUDA-11.7.0``|``foss/2022a``

### CLISP

Common Lisp is a high-level, general-purpose, object-oriented, dynamic,  functional programming language. 

*homepage*: <https://clisp.sourceforge.io/>

version |toolchain
--------|------------------------------------
``2.49``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

### ClonalFrameML

Efficient Inference of Recombination in Whole Bacterial Genomes

*homepage*: <https://github.com/xavierdidelot/ClonalFrameML>

version |toolchain
--------|--------------
``1.11``|``foss/2016b``

### CLooG

CLooG is a free software and library to generate code for scanning Z-polyhedra. That is, it finds a  code (e.g. in C, FORTRAN...) that reaches each integral point of one or more parameterized polyhedra. CLooG has been  originally written to solve the code generation problem for optimizing compilers based on the polytope model.  Nevertheless it is used now in various area e.g. to build control automata for high-level synthesis or to find the  best polynomial approximation of a function. CLooG may help in any situation where scanning polyhedra matters. While  the user has full control on generated code quality, CLooG is designed to avoid control overhead and to produce a  very effective code.

*homepage*: <http://www.bastoul.net/cloog/index.php>

version   |toolchain
----------|-------------
``0.18.1``|``GCC/4.8.2``

### CloudCompare

3D point cloud and mesh processing software

*homepage*: <https://www.cloudcompare.org/>

version   |toolchain
----------|--------------
``2.12.4``|``foss/2021b``

### Clp

Clp (Coin-or linear programming) is an open-source linear programming solver. It is primarily meant to be used as a callable library, but a basic, stand-alone executable version is also available.

*homepage*: <https://github.com/coin-or/Clp>

version   |toolchain
----------|------------------------------
``1.17.3``|``foss/2018b``
``1.17.6``|``foss/2020b``, ``foss/2021a``
``1.17.7``|``foss/2021b``

### Clustal-Omega

Clustal Omega is a multiple sequence alignment  program for proteins. It produces biologically meaningful multiple  sequence alignments of divergent sequences. Evolutionary relationships  can be seen via viewing Cladograms or Phylograms 

*homepage*: <http://www.clustal.org/omega/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------
``1.2.0``|``foss/2016b``
``1.2.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.3.0``, ``foss/2018b``, ``intel-compilers/2021.2.0``, ``intel/2018a``, ``intel/2018b``

### ClustalW2

ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.

*homepage*: <https://www.ebi.ac.uk/Tools/msa/clustalw2/>

version|toolchain
-------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2016b``, ``foss/2018b``, ``foss/2021a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017b``, ``intel/2018b``, ``intel/2020a``

### Cluster-Buster

Cluster-Buster: Finding dense clusters of motifs in DNA sequences

*homepage*: <https://github.com/weng-lab/cluster-buster/>

version     |toolchain
------------|---------------
``20160106``|``intel/2016a``

### ClusterShell

ClusterShell is an event-driven open source Python library, designed to run local or distant commands  in parallel on server farms or on large Linux clusters.

*homepage*: <https://cea-hpc.github.io/clustershell/>

version  |toolchain
---------|----------
``1.7.3``|``system``

### CMake

CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.

*homepage*: <https://www.cmake.org>

version   |toolchain
----------|-------------------------------------------------------------------------------------------------------------------------------------------------
``2.8.11``|``GCC/4.8.1``
``2.8.12``|``GCC/4.8.1``, ``GCC/4.8.2``
``3.0.0`` |``GCC/4.8.3``
``3.1.0`` |``GCC/4.9.2``
``3.1.3`` |``GCC/4.9.2``, ``system``
``3.2.1`` |``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``3.3.1`` |``system``
``3.3.2`` |``GNU/4.9.3-2.25``, ``gimkl/2.11.5``
``3.4.1`` |``GCC/4.9.2``, ``GCCcore/4.9.3``, ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``3.4.3`` |``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``
``3.5.1`` |``intel/2016a``
``3.5.2`` |``GCC/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``system``
``3.6.1`` |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``, ``system``
``3.6.2`` |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``3.7.1`` |``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``foss/2016b``, ``intel/2016b``
``3.7.2`` |``GCCcore/6.3.0``, ``foss/2016b``, ``intel/2016b``
``3.8.0`` |``GCCcore/6.3.0``
``3.8.1`` |``GCCcore/6.3.0``
``3.8.2`` |``GCCcore/6.3.0``
``3.9.1`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``
``3.9.4`` |``GCCcore/6.4.0``
``3.9.5`` |``GCCcore/6.4.0``
``3.9.6`` |``system``
``3.10.0``|``GCCcore/6.4.0``
``3.10.1``|``GCCcore/6.4.0``
``3.10.2``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``3.10.3``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``3.11.1``|``GCCcore/6.4.0``
``3.11.4``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.12.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``system``
``3.13.3``|``GCCcore/8.2.0``
``3.15.1``|``system``
``3.15.3``|``GCCcore/8.3.0``
``3.16.4``|``GCCcore/9.3.0``
``3.18.4``|``GCCcore/10.2.0``, ``system``
``3.20.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.21.1``|``GCCcore/11.2.0``
``3.22.1``|``GCCcore/11.2.0``
``3.23.1``|``GCCcore/11.3.0``
``3.24.3``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``3.26.3``|``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### CMAverse

The R package CMAverse provides a suite of functions for reproducible causal mediation analysis including cmdag for DAG visualization, cmest for statistical modeling and cmsens for sensitivity analysis.

*homepage*: <https://bs1125.github.io/CMAverse/>

version     |toolchain
------------|--------------
``20220112``|``foss/2021b``

### CmdStanR

CmdStanR is a lightweight interface to Stan for R users

*homepage*: <https://mc-stan.org/cmdstanr>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.5.0``|``-R-4.1.2`` |``foss/2021b``
``0.5.2``|``-R-4.2.1`` |``foss/2022a``

### cmocean

This package contains colormaps for commonly-used  oceanographic variables. Most of the colormaps started from  matplotlib colormaps, but have now been adjusted using the viscm  tool to be perceptually uniform.

*homepage*: <https://github.com/matplotlib/cmocean>

version|toolchain
-------|--------------
``2.0``|``foss/2022a``

### CMSeq

CMSeq is a set of commands to provide an interface to .bam files for coverage and sequence consensus.

*homepage*: <https://github.com/SegataLab/cmseq/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.8.2``|``foss/2020a``
``1.0.4``|                 |``foss/2022a``

### CNT-ILP

Integer Linear Program for the Copy-Number Tree Problem 

*homepage*: <https://compbio.cs.brown.edu/projects/cnt-ilp/>

version     |toolchain
------------|--------------------
``20171031``|``GCC/8.2.0-2.31.1``

### CNVkit

A command-line toolkit and Python library for detecting copy  number variants and alterations genome-wide from high-throughput sequencing.

*homepage*: <https://github.com/etal/cnvkit>

version   |versionsuffix            |toolchain
----------|-------------------------|---------------
``0.9.2`` |``-Python-2.7.14``       |``intel/2017b``
``0.9.3`` |``-Python-3.6.4``        |``intel/2018a``
``0.9.6`` |``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``0.9.8`` |``-R-4.0.3``             |``foss/2020b``
``0.9.10``|``-R-4.2.2``             |``foss/2022b``

### CNVnator

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads 

*homepage*: <https://github.com/abyzovlab/CNVnator>

version  |toolchain
---------|--------------
``0.3.3``|``foss/2016b``

### Co-phylog

Co-phylog: an assembly-free phylogenomic approach for closely related organisms H Yi, L Jin Nucleic acids research 41 (7), e75-e75

*homepage*: <https://github.com/yhg926/co-phylog>

version     |toolchain
------------|------------------
``20201012``|``GCC/7.3.0-2.30``

### COBRApy

COBRApy is a package for constraint-based modeling of metabolic networks.

*homepage*: <https://opencobra.github.io/cobrapy/>

version   |toolchain
----------|--------------
``0.26.0``|``foss/2021a``

### CoCoALib

CoCoALib is a free GPL3 C++ library for doing Computations in Commutative Algebra.

*homepage*: <http://cocoa.dima.unige.it/cocoalib >

version    |toolchain
-----------|--------------------
``0.99601``|``GCC/8.2.0-2.31.1``
``0.99700``|``GCC/8.3.0``

### CodAn

CodAn (Coding sequence Annotator) is a computational tool designed to characterize the CDS and UTR regions on transcripts from any Eukaryote species.

*homepage*: <https://github.com/pedronachtigall/CodAn>

version|toolchain
-------|--------------
``1.2``|``foss/2021b``

### code-server

Run VS Code on any machine anywhere and access it in the browser.

*homepage*: <https://github.com/cdr/code-server>

version  |toolchain
---------|----------
``3.7.3``|``system``
``4.9.1``|``system``

### CODEX2

Full-spectrum copy number variation detection by high-throughput DNA sequencing

*homepage*: <https://github.com/yuchaojiang/CODEX2>

version     |versionsuffix|toolchain
------------|-------------|---------------
``20180227``|``-R-3.4.3`` |``intel/2017b``

### CodingQuarry

Highly accurate hidden Markov model gene prediction in fungal genomes using RNA-seq transcripts

*homepage*: <https://sourceforge.net/p/codingquarry>

version|toolchain
-------|--------------
``2.0``|``foss/2021b``

### Cogent

Cogent is a tool for reconstructing the coding genome using high-quality full-length transcriptome sequences. It is designed to be used on Iso-Seq data and in cases where there is no reference genome or the ref genome is highly incomplete. 

*homepage*: <https://github.com/Magdoll/Cogent>

version  |toolchain
---------|--------------
``8.0.0``|``foss/2021a``

### Coin

Coin is an OpenGL-based, 3D graphics library that has its roots in the Open Inventor 2.1 API, which Coin still is compatible with.

*homepage*: <https://coin3d.github.io>

version  |toolchain
---------|------------------------------
``4.0.0``|``GCC/10.3.0``, ``GCC/11.2.0``

### CoinUtils

CoinUtils (Coin-OR Utilities) is an open-source collection of classes and functions that are generally useful to more than one COIN-OR project.

*homepage*: <https://github.com/coin-or/CoinUtils>

version   |toolchain
----------|----------------------------------
``2.11.3``|``GCCcore/7.3.0``, ``foss/2018b``
``2.11.4``|``GCC/10.3.0``, ``GCCcore/10.2.0``
``2.11.6``|``GCC/11.2.0``

### ColabFold

Making protein folding accessible to all. Predict proteins structures both in google colab and on your machine.

*homepage*: <https://github.com/sokrypton/ColabFold>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.2``|                |``foss/2022a``
``1.5.2``|``-CUDA-11.7.0``|``foss/2022a``

### colossalai

Colossal-AI: A Unified Deep Learning System for Big Model Era

*homepage*: <https://colossalai.org/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.8``|``-CUDA-11.3.1``|``foss/2021a``

### Commet

COMMET ("COmpare Multiple METagenomes") provides a global similarity overview between all datasets of  a large metagenomic project. 

*homepage*: <https://colibread.inria.fr/software/commet/>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20150415``|``-Python-2.7.11``|``foss/2016a``

### CompareM

A toolbox for comparative genomics.

*homepage*: <https://github.com/dparks1134/CompareM>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``0.0.23``|``-Python-2.7.15``|``foss/2018b``
``0.1.2`` |                  |``foss/2021b``

### Compress-Raw-Zlib

Low-Level Interface to zlib or zlib-ng compression library

*homepage*: <https://metacpan.org/pod/Compress::Raw::Zlib>

version  |toolchain
---------|------------------
``2.202``|``GCCcore/11.3.0``

### COMSOL

COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems.  

*homepage*: <https://www.comsol.com>

version      |toolchain
-------------|----------
``5.4.0.225``|``system``

### Con3F

Con3F is a Python package to read, manipulate and convert force field files

*homepage*: <https://github.ugent.be/CMM/con3f>

version         |versionsuffix    |toolchain
----------------|-----------------|---------------
``1.0-20190329``|``-Python-3.7.2``|``intel/2019a``

### conan

Decentralized, open-source (MIT), C/C++ package manager.

*homepage*: <https://conan.io>

version   |toolchain
----------|------------------
``1.58.0``|``GCCcore/11.3.0``

### CONCOCT

Clustering cONtigs with COverage and ComposiTion (CONCOCT) is a program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.

*homepage*: <https://concoct.readthedocs.io>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.0.0``|``-Python-2.7.14``|``foss/2017b``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.0.0``|``-Python-3.6.3`` |``foss/2017b``
``1.1.0``|``-Python-2.7.15``|``foss/2019a``
``1.1.0``|``-Python-2.7.18``|``foss/2020b``

### configparser

configparser is a Python library that brings the updated configparser from Python 3.5 to Python 2.6-3.5

*homepage*: <https://docs.python.org/3/library/configparser.html>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.5.0``|``-Python-2.7.11``|``foss/2016a``
``3.5.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.5.0``|``-Python-3.5.2`` |``intel/2016b``
``3.5.0``|``-Python-3.6.3`` |``intel/2017b``

### configurable-http-proxy

HTTP proxy for node.js including a REST API for updating the routing table.  Developed as a part of the Jupyter Hub multi-user server.

*homepage*: <https://github.com/jupyterhub/configurable-http-proxy>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.3.0``|``-nodejs-4.4.7``|``foss/2016a``
``3.1.1``|``-nodejs-8.9.4``|``foss/2017a``
``4.2.1``|                 |``GCCcore/10.2.0``
``4.4.0``|                 |``GCCcore/10.3.0``
``4.5.3``|                 |``GCCcore/11.3.0``

### CONN

CONN is an open-source Matlab/SPM-based cross-platform software  for the computation, display, and analysis of functional connectivity Magnetic  Resonance Imaging (fcMRI). CONN is used to analyze resting state data (rsfMRI)  as well as task-related designs. 

*homepage*: <https://web.conn-toolbox.org/home>

version|versionsuffix    |toolchain
-------|-----------------|----------
``21a``|``-MATLAB-2021a``|``system``

### ConnectomeWorkbench

Connectome Workbench is an open source, freely available visualization  and discovery tool used to map neuroimaging data, especially data generated by the  Human Connectome Project.

*homepage*: <https://www.humanconnectome.org/software/connectome-workbench>

version  |versionsuffix  |toolchain
---------|---------------|--------------------------------------------------
``1.2.2``|               |``system``
``1.3.2``|               |``GCCcore/8.2.0``, ``foss/2017b``, ``intel/2017b``
``1.4.2``|``-rh_linux64``|``system``
``1.5.0``|               |``GCCcore/10.3.0``

### Control-FREEC

Copy number and genotype annotation from whole  genome and whole exome sequencing data.

*homepage*: <https://github.com/BoevaLab/FREEC>

version |toolchain
--------|----------------------------------------
``11.5``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``
``11.6``|``GCC/10.2.0``

### cooler

Cooler is a support library for a storage format, also called cooler, used to store  genomic interaction data of any size, such as Hi-C contact matrices.

*homepage*: <https://open2c.github.io/cooler>

version  |toolchain
---------|--------------
``0.9.1``|``foss/2022a``

### CoordgenLibs

Schrodinger-developed 2D Coordinate Generation

*homepage*: <https://github.com/schrodinger/coordgenlibs>

version  |toolchain
---------|------------------------------------------------------------------
``1.3.2``|``gompi/2019a``, ``iimpi/2019a``
``3.0.1``|``gompi/2019b``, ``gompi/2021a``, ``gompi/2022a``, ``iimpi/2020a``

### Coot

Coot is for macromolecular model building, model completion  and validation, particularly suitable for protein modelling using X-ray data.

*homepage*: <http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot>

version  |versionsuffix                              |toolchain
---------|-------------------------------------------|----------
``0.8.1``|``-binary-Linux-x86_64-rhel-6-python-gtk2``|``system``

### CopyKAT

CopyKAT: Inference of genomic copy number and subclonal structure of human tumors from  high-throughput single cell RNAseq data

*homepage*: <https://github.com/navinlabcode/copykat>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.1.0``|``-R-4.2.1`` |``foss/2022a``

### core-counter

Tool to check available cores and OMP threads

*homepage*: <https://github.com/gjbex/core-counter>

version  |toolchain
---------|----------
``1.1.1``|``system``

### Coreutils

The GNU Core Utilities are the basic file, shell and text manipulation utilities of the GNU operating system. These are the core utilities which are expected to exist on every operating system. 

*homepage*: <https://www.gnu.org/software/coreutils/>

version |toolchain
--------|------------------------------------
``8.23``|``GCC/4.9.2``
``8.27``|``GCCcore/5.4.0``
``8.29``|``GCCcore/6.4.0``
``8.32``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0`` |``GCCcore/11.2.0``

### corner

Make some beautiful corner plots.

*homepage*: <https://corner.readthedocs.io/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.0.1``|``-Python-2.7.15``|``foss/2019a``
``2.0.1``|``-Python-3.7.2`` |``foss/2019a``
``2.2.2``|                  |``foss/2021b``

### CoSymLib

Cosymlib is a python library for computing continuous symmetry & shape measures (CSMs & CShMs). Although its main aim is to provide simple and ready-to-use tools for the analysis of the symmetry & shape of molecules, many of the procedures contained in cosymlib can be easily applied to any finite geometrical object defined by a set of vertices or a by mass distribution function.

*homepage*: <https://cosymlib.readthedocs.io>

version   |toolchain
----------|--------------
``0.10.9``|``foss/2022a``

### coverage

Coverage.py is a tool for measuring code coverage of Python programs.  It monitors your program, noting which parts of the code have been executed,  then analyzes the source to identify code that could have been executed but was not. 

*homepage*: <https://coverage.readthedocs.io>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``4.5.1``|``-Python-2.7.14``|``intel/2017b``
``5.5``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``7.2.3``|                  |``GCCcore/12.2.0``

### cowsay

Configurable talking characters in ASCII art

*homepage*: <https://github.com/tnalpgge/rank-amateur-cowsay>

version |toolchain
--------|----------
``3.04``|``system``

### CP2K

CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular  simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different  methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and  classical pair and many-body potentials. 

*homepage*: <https://www.cp2k.org/>

version   |versionsuffix|toolchain
----------|-------------|-------------------------------------------------------------------------------------------------
``3.0``   |             |``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``
``3.0``   |``-psmp``    |``intel/2016b``
``4.1``   |             |``intel/2016b``
``4.1``   |``-psmp``    |``foss/2016b``
``5.1``   |             |``foss/2018a``, ``foss/2020a``, ``foss/2020b``, ``intel/2017b``, ``intel/2018a``, ``intel/2020a``
``6.1``   |             |``foss/2019a``, ``intel/2018a``, ``intel/2020a``
``7.1``   |             |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``7.1``   |``-psmp``    |``foss/2020b``
``8.1``   |             |``foss/2020b``
``8.2``   |             |``foss/2021a``, ``intel/2021a``
``9.1``   |             |``foss/2022a``
``2022.1``|             |``foss/2022a``

### CPB

CPB is a novel two-step Pearson correlation based biclustering approach to mine genes that are co-regulated with   a given reference gene in order to discover genes that function in a common biological process.   In the first step, the algorithm identifies subsets of genes with high correlation, reducing false negatives   with a nonparametric filtering scheme.   In the second step, biclusters from multiple datasets are used to extract and rank gene correlation information.

*homepage*: <http://tda.gatech.edu/software/cpb/index.html>

version      |versionsuffix     |toolchain
-------------|------------------|--------------
``11-4-2011``|``-Python-2.7.13``|``foss/2017a``

### CPC2

a fast and accurate coding potential calculator based on sequence intrinsic features  

*homepage*: <http://cpc2.gao-lab.org/>

version  |toolchain
---------|--------------
``1.0.1``|``foss/2022a``

### CPLEX

IBM ILOG CPLEX Optimizer's mathematical programming technology enables  analytical decision support for improving efficiency,  reducing costs, and increasing profitability.

*homepage*: <https://www.ibm.com/analytics/cplex-optimizer>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``12.9``  |                 |``GCCcore/8.2.0``
``12.10`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``22.1.1``|                 |``GCCcore/11.2.0``

### CPPE

CPPE is an open-source, light-weight C++ and Python library for Polarizable Embedding (PE)1,2 calculations. It provides an simple-to-use API to implement PE for ground-state self-consistent field (SCF) calculations and post-SCF methods. A convenient Python interface is also available.

*homepage*: <https://github.com/maxscheurer/cppe>

version  |toolchain
---------|--------------
``0.3.1``|``GCC/11.3.0``

### CppHeaderParser

CppHeaderParser is a pure python module that will parse C++ header files and generate a data structure representing the class.

*homepage*: <https://senexcanis.com/open-source/cppheaderparser>

version  |toolchain
---------|------------------
``2.7.4``|``GCCcore/11.3.0``

### CppUnit

CppUnit is the C++ port of the famous JUnit framework for unit testing. 

*homepage*: <https://freedesktop.org/wiki/Software/cppunit/>

version   |toolchain
----------|------------------------------------------------------------------------------------------------
``1.12.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``
``1.15.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### cppy

A small C++ header library which makes it easier to write Python extension modules. The primary feature is a PyObject smart pointer which automatically handles reference counting and provides convenience methods for performing common object operations.

*homepage*: <https://github.com/nucleic/cppy>

version  |toolchain
---------|--------------------------------------
``1.1.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### cppzmq

cppzmq is a C++ binding for libzmq.

*homepage*: <https://github.com/zeromq/cppzmq>

version  |toolchain
---------|----------
``4.9.0``|``system``

### cpu_features

A cross-platform C library to retrieve CPU features (such as available instructions) at runtime.

*homepage*: <https://github.com/google/cpu_features>

version  |toolchain
---------|------------------
``0.6.0``|``GCCcore/10.2.0``

### cram

Cram is a functional testing framework for command line applications.

*homepage*: <https://bitheap.org/cram>

version|toolchain
-------|-----------------
``0.7``|``GCCcore/8.2.0``

### cramtools

CRAMTools is a set of Java tools and APIs for efficient compression of sequence  read data. Although this is intended as a stable version the code is released as early access. Parts of the CRAMTools are experimental and may not be supported in the future.

*homepage*: <https://github.com/enasequence/cramtools/>

version|versionsuffix     |toolchain
-------|------------------|----------
``2.0``|``-Java-1.7.0_80``|``system``
``3.0``|``-Java-1.7.0_80``|``system``

### CrayCCE

Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: June 2019). 

*homepage*: <https://pubs.cray.com/discover>

version  |toolchain
---------|----------
``19.06``|``system``

### CrayGNU

Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: June 2019). 

*homepage*: <https://pubs.cray.com/discover>

version  |toolchain
---------|----------
``19.06``|``system``

### CrayIntel

Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: June 2019). 

*homepage*: <https://pubs.cray.com/discover>

version  |toolchain
---------|----------
``19.06``|``system``

### CrayPGI

Toolchain using Cray compiler wrapper, PrgEnv-pgi compiler (PE release: June 2019). 

*homepage*: <https://pubs.cray.com/discover>

version  |toolchain
---------|----------
``19.06``|``system``

### crb-blast

Conditional Reciprocal Best BLAST - high confidence ortholog assignment.  CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.  This is particularly useful in genome and transcriptome annotation.

*homepage*: <https://github.com/cboursnell/crb-blast>

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------------------------
``0.6.9``|               |``gompi/2021a``, ``gompi/2021b``, ``intel/2017a``
``0.6.9``|``-Ruby-2.6.1``|``foss/2018b``

### CREST

CREST is an utility/driver program for the xtb program. Originally it was designed  as conformer sampling program, hence the abbreviation Conformer–Rotamer Ensemble Sampling Tool,  but now offers also some utility functions for calculations with the GFNn–xTB methods. Generally  the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb  program) and tool for the creation and analysation of structure ensembles. 

*homepage*: <https://xtb-docs.readthedocs.io/en/latest/crest.html>

version   |toolchain
----------|---------------
``2.11``  |``intel/2021a``
``2.11.2``|``intel/2021a``

### CRF++

CRF++ is a simple, customizable, and open source implementation of  Conditional Random Fields (CRFs) for segmenting/labeling sequential data. CRF++ is  designed for generic purpose and will be applied to a variety of NLP tasks, such as  Named Entity Recognition, Information Extraction and Text Chunking. 

*homepage*: <https://taku910.github.io/crfpp/>

version |toolchain
--------|---------------------------------------------------------
``0.58``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``

### CRISPR-DAV

CRISPR-DAV is a pipeline to analyze amplicon-based NGS data of CRISPR clones in a high throughput manner.

*homepage*: <https://github.com/pinetree1/crispr-dav/>

version  |toolchain
---------|--------------
``2.3.4``|``foss/2020b``

### CRISPResso2

CRISPResso2 is a software pipeline designed to enable rapid and intuitive interpretation of genome editing experiments.

*homepage*: <https://github.com/pinellolab/CRISPResso2/>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``2.0.44``|``-Python-2.7.16``|``foss/2019b``
``2.1.2`` |``-Python-2.7.18``|``foss/2020b``
``2.2.1`` |                  |``foss/2020b``

### cromwell

Cromwell is a Workflow Management System geared towards scientific workflows.

*homepage*: <http://cromwell.readthedocs.io/>

version|versionsuffix|toolchain
-------|-------------|----------
``56`` |``-Java-11`` |``system``

### crossguid

CrossGuid is a minimal, cross platform, C++ GUID library.  It uses the best native GUID/UUID generator on the given platform and has a  generic class for parsing, stringifying, and comparing IDs.  The guid generation technique is determined by your platform:

*homepage*: <https://github.com/graeme-hill/crossguid>

version     |toolchain
------------|------------------
``20190529``|``GCCcore/11.3.0``

### CrossMap

CrossMap is a program for genome coordinates conversion  between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).  It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,  BigWig, BED, GFF, GTF and VCF.

*homepage*: <http://crossmap.sourceforge.net>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.9``|``-Python-3.7.2``|``foss/2019a``

### CrossTalkZ

CrossTalkZ is a statistical method and software to assess the significance of crosstalk enrichment  between pairs of gene or protein groups in large biological networks. 

*homepage*: <http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/>

version|toolchain
-------|--------------
``1.4``|``foss/2016a``

### CRPropa

CRPropa is a publicly available code to study the propagation of ultra high energy nuclei up to iron  on their voyage through an extra galactic environment.

*homepage*: <https://crpropa.desy.de>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.5``|``-Python-3.7.2``|``foss/2019a``
``3.1.6``|``-Python-3.8.2``|``foss/2020a``

### Crumble

Exploration of controlled loss of quality values for compressing CRAM files

*homepage*: <https://github.com/jkbonfield/crumble>

version  |toolchain
---------|--------------
``0.8.3``|``GCC/11.2.0``

### cryoCARE

This package is a memory efficient implementation of cryoCARE.  This setup trains a denoising U-Net for tomographic reconstruction according to   the Noise2Noise training paradigm. Therefore the user has to provide two        tomograms of the same sample. The simplest way to achieve this is with direct-   detector movie-frames.  You can use Warp to generate two reconstructed tomograms based on the even/odd    frames. Alternatively, the movie-frames can be split in two halves (e.g. with    MotionCor2 -SplitSum 1 or with IMOD alignframes -debug 10000) from which two     identical, up to random noise, tomograms can be reconstructed.  These two (even and odd) tomograms can be used as input to this cryoCARE         implementation.

*homepage*: <https://github.com/juglab/cryoCARE_pip>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.2.1``|``-CUDA-11.3.1``|``foss/2021a``

### cryoDRGN

cryoDRGN: Deep Reconstructing Generative Networks for cryo-EM heterogeneous  reconstruction.  CryoDRGN is a neural network based algorithm for heterogeneous cryo-EM reconstruction. In  particular, the method models a continuous distribution over 3D structures by using a neural  network based representation for the volume. 

*homepage*: <https://cb.csail.mit.edu/cb/cryodrgn/>

version       |versionsuffix   |toolchain
--------------|----------------|------------------
``0.3.2``     |                |``fosscuda/2020b``
``0.3.5``     |``-CUDA-11.3.1``|``foss/2021a``
``1.0.0-beta``|``-CUDA-11.3.1``|``foss/2021a``

### CryptoMiniSat

CryptoMiniSat is an advanced SAT solver

*homepage*: <https://github.com/msoos/cryptominisat>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``5.0.1``|``-Python-2.7.12``|``intel/2016b``

### CrystFEL

CrystFEL is a suite of programs for processing diffraction data acquired "serially" in a "snapshot" manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser source.

*homepage*: <http://www.desy.de/~twhite/crystfel/>

version  |toolchain
---------|-------------------------------
``0.8.0``|``foss/2019a``, ``intel/2019a``

### CSB

Computational Structural Biology Toolbox CSB is a python library and application framework, which can be used to solve    problems in the field of structural bioinformatics. If you are a                 bioinformatician, software engineer or a researcher working in this field,       chances are you may find something useful here. Our package consists of a few    major components: 1. Core class library - object-oriented, granular, with an emphasis on design    and clean interfaces. 2. Application framework - console applications ("protocols"), which consume     objects from the core library in order to build something executable (and        hopefully useful). 3. Test framework - ensures that the library actually works. 

*homepage*: <https://github.com/csb-toolbox>

version  |toolchain
---------|--------------
``1.2.5``|``foss/2020b``

### CSBDeep

CSBDeep is a toolbox for Content-aware Image Restoration (CARE).

*homepage*: <https://csbdeep.bioimagecomputing.com/>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.4.1``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

### CSBLAST

Context-specific extension of BLAST that significantly improves sensitivity and alignment quality.

*homepage*: <https://github.com/soedinglab/csblast/>

version  |toolchain
---------|-----------------
``2.2.3``|``GCCcore/8.3.0``

### cscope

Cscope is a developer's tool for browsing source code.

*homepage*: <http://cscope.sourceforge.net/>

version |toolchain
--------|----------
``15.9``|``system``

### csvkit

csvkit is a suite of command-line tools for converting to and working with CSV,  the king of tabular file formats.

*homepage*: <https://github.com/wireservice/csvkit>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.0.4``|                 |``GCCcore/8.2.0``
``1.0.5``|``-Python-3.7.4``|``GCCcore/8.3.0``
``1.1.0``|                 |``GCCcore/11.3.0``

### ctags

Ctags generates an index (or tag) file of language objects found in source files that allows these  items to be quickly and easily located by a text editor or other utility.

*homepage*: <http://ctags.sourceforge.net/>

version|toolchain
-------|----------
``5.8``|``system``

### ctffind

Program for finding CTFs of electron micrographs.

*homepage*: <https://grigoriefflab.umassmed.edu/ctffind4>

version   |versionsuffix   |toolchain
----------|----------------|--------------------------------------------------------------------------------------
``4.1.13``|                |``foss/2019a``, ``fosscuda/2019a``, ``fosscuda/2019b``
``4.1.14``|                |``foss/2019b``, ``foss/2021b``, ``foss/2022a``, ``fosscuda/2019b``, ``fosscuda/2020b``
``4.1.14``|``-CUDA-11.3.1``|``foss/2021a``

### CTPL

Modern and efficient C++ Thread Pool Library

*homepage*: <https://github.com/vit-vit/CTPL.git>

version  |toolchain
---------|--------------
``0.0.2``|``GCC/11.3.0``

### Cube

Cube, which is used as performance report explorer for Scalasca and  Score-P, is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system  resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree  can be collapsed or expanded to achieve the desired level of granularity.

*homepage*: <http://www.scalasca.org/software/cube-4.x/download.html>

version  |toolchain
---------|--------------
``4.3.4``|``foss/2016a``

### CubeGUI

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube graphical report explorer. 

*homepage*: <https://www.scalasca.org/software/cube-4.x/download.html>

version  |toolchain
---------|--------------------------------------
``4.4.4``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.8``  |``GCCcore/11.3.0``

### CubeLib

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube general purpose C++ library component and  command-line tools. 

*homepage*: <https://www.scalasca.org/software/cube-4.x/download.html>

version  |toolchain
---------|-------------------------------------------------------
``4.4.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.8``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### CubeWriter

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube high-performance C writer library component. 

*homepage*: <https://www.scalasca.org/software/cube-4.x/download.html>

version  |toolchain
---------|---------------------------------------------------------------------------
``4.4.3``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.8``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### CuCLARK

Metagenomic classifier for CUDA-enabled GPUs

*homepage*: <https://github.com/Funatiq/cuclark>

version|toolchain
-------|------------------
``1.1``|``fosscuda/2019b``

### CUDA

CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.

*homepage*: <https://developer.nvidia.com/cuda-toolkit>

version          |toolchain
-----------------|--------------------------------------------------------------------------
``5.5.22``       |``GCC/4.8.2``, ``system``
``6.0.37``       |``system``
``6.5.14``       |``system``
``7.0.28``       |``system``
``7.5.18``       |``GCC/4.9.4-2.25``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``system``
``8.0.44``       |``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``system``
``8.0.61``       |``system``
``8.0.61_375.26``|``GCC/5.4.0-2.26``
``9.0.176``      |``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``system``
``9.1.85``       |``GCC/6.4.0-2.28``, ``system``
``9.2.88``       |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``system``
``9.2.148.1``    |``system``
``10.0.130``     |``system``
``10.1.105``     |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``system``
``10.1.168``     |``system``
``10.1.243``     |``GCC/8.3.0``, ``iccifort/2019.5.281``, ``system``
``10.2.89``      |``GCC/8.3.0``
``11.0.2``       |``GCC/9.3.0``, ``iccifort/2020.1.217``
``11.1.1``       |``GCC/10.2.0``, ``iccifort/2020.4.304``
``11.3.1``       |``system``
``11.4.1``       |``system``
``11.4.2``       |``system``
``11.5.0``       |``system``
``11.5.1``       |``system``
``11.5.2``       |``system``
``11.6.0``       |``system``
``11.7.0``       |``system``
``11.8.0``       |``system``
``12.0.0``       |``system``
``12.1.0``       |``system``
``12.1.1``       |``system``
``12.2.0``       |``system``

### CUDA-Samples

Samples for CUDA Developers which demonstrates features in CUDA Toolkit

*homepage*: <https://github.com/NVIDIA/cuda-samples>

version |versionsuffix   |toolchain
--------|----------------|--------------
``11.3``|``-CUDA-11.3.1``|``GCC/10.3.0``

### CUDAcompat

Using the CUDA Forward Compatibility package,  system administrators can run applications built using a newer toolkit  even when an older driver that does not satisfy the minimum required driver version  is installed on the system.  This forward compatibility allows the CUDA deployments in data centers and enterprises  to benefit from the faster release cadence and the latest features and performance of CUDA Toolkit. 

*homepage*: <https://docs.nvidia.com/deploy/cuda-compatibility/index.html>

version |versionsuffix |toolchain
--------|--------------|----------
``11``  |              |``system``
``11.6``|              |``system``
``11.6``|``-510.85.02``|``system``
``11.7``|              |``system``
``11.7``|``-515.65.01``|``system``

### CUDAcore

CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.

*homepage*: <https://developer.nvidia.com/cuda-toolkit>

version   |toolchain
----------|----------
``11.0.2``|``system``
``11.1.1``|``system``
``11.2.1``|``system``
``11.2.2``|``system``
``11.3.0``|``system``
``11.4.0``|``system``
``11.5.1``|``system``

### cuDNN

The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.

*homepage*: <https://developer.nvidia.com/cudnn>

version      |versionsuffix     |toolchain
-------------|------------------|-----------------------------------------------------------
``4.0``      |                  |``system``
``5.0``      |``-CUDA-7.5.18``  |``system``
``5.0-rc``   |                  |``system``
``5.1``      |``-CUDA-8.0.44``  |``system``
``6.0``      |``-CUDA-8.0.61``  |``system``
``6.0.21``   |``-CUDA-7.5.18``  |``system``
``6.0.21``   |``-CUDA-8.0.44``  |``system``
``7.0.2``    |``-CUDA-9.0.176`` |``system``
``7.0.5``    |``-CUDA-8.0.44``  |``system``
``7.0.5``    |``-CUDA-9.0.176`` |``system``
``7.0.5``    |``-CUDA-9.1.85``  |``system``
``7.0.5.15`` |                  |``fosscuda/2017b``, ``fosscuda/2018a``, ``intelcuda/2017b``
``7.1.4.18`` |                  |``fosscuda/2018b``
``7.4.2.24`` |                  |``gcccuda/2019a``
``7.4.2.24`` |``-CUDA-10.0.130``|``system``
``7.5.0.56`` |``-CUDA-10.0.130``|``system``
``7.6.2.24`` |``-CUDA-10.1.243``|``system``
``7.6.4.38`` |                  |``gcccuda/2019a``, ``gcccuda/2019b``
``7.6.4.38`` |``-CUDA-10.0.130``|``system``
``8.0.4.30`` |``-CUDA-11.0.2``  |``system``
``8.0.4.30`` |``-CUDA-11.1.1``  |``system``
``8.0.5.39`` |``-CUDA-11.1.1``  |``system``
``8.1.0.77`` |``-CUDA-11.2.1``  |``system``
``8.1.1.33`` |``-CUDA-11.2.1``  |``system``
``8.2.1.32`` |``-CUDA-11.3.1``  |``system``
``8.2.2.26`` |``-CUDA-11.4.0``  |``system``
``8.2.2.26`` |``-CUDA-11.4.1``  |``system``
``8.4.0.27`` |``-CUDA-11.6.0``  |``system``
``8.4.1.50`` |``-CUDA-11.5.2``  |``system``
``8.4.1.50`` |``-CUDA-11.6.0``  |``system``
``8.4.1.50`` |``-CUDA-11.7.0``  |``system``
``8.6.0.163``|``-CUDA-11.8.0``  |``system``
``8.7.0.84`` |``-CUDA-11.8.0``  |``system``
``8.8.0.121``|``-CUDA-12.0.0``  |``system``
``8.9.2.26`` |``-CUDA-12.1.1``  |``system``

### Cufflinks

Transcript assembly, differential expression, and differential regulation for RNA-Seq

*homepage*: <http://cole-trapnell-lab.github.io/cufflinks/>

version     |toolchain
------------|-------------------------------------------------------------------------------------------------
``2.2.1``   |``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``gompi/2019b``, ``intel/2017b``, ``intel/2018a``
``20190706``|``GCC/10.2.0``, ``GCC/11.2.0``, ``gompi/2019a``

### CUnit

Automated testing framework for C.

*homepage*: <https://sourceforge.net/projects/cunit/>

version  |toolchain
---------|---------------------------------------------------------
``2.1-3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``

### CuPy

CuPy is an open-source array library accelerated with NVIDIA CUDA.

*homepage*: <https://cupy.dev>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``8.2.0`` |``-Python-3.7.4``|``fosscuda/2019b``
``8.5.0`` |                 |``fosscuda/2020b``
``11.4.0``|``-CUDA-11.4.1`` |``foss/2021b``

### cURL

libcurl is a free and simple-to-use client-side URL transfer library,  supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,  LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.  libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP  form based upload, proxies, cookies, user+password authentication (Basic,  Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling  and more. 

*homepage*: <https://curl.haxx.se>

version   |toolchain
----------|-----------------------------------------------------------------------------------
``7.33.0``|``GCC/4.8.2``
``7.34.0``|``GCC/4.8.2``
``7.40.0``|``GCC/4.9.2``
``7.46.0``|``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``7.47.0``|``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``7.49.1``|``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``7.52.1``|``gimkl/2017a``, ``intel/2016b``
``7.53.1``|``GCCcore/6.3.0``
``7.54.0``|``GCCcore/6.3.0``
``7.55.1``|``GCCcore/6.4.0``
``7.56.0``|``GCCcore/6.4.0``
``7.56.1``|``GCCcore/6.4.0``
``7.58.0``|``GCCcore/6.4.0``
``7.59.0``|``GCCcore/6.4.0``
``7.60.0``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``7.63.0``|``GCCcore/8.2.0``
``7.66.0``|``GCCcore/8.3.0``
``7.69.1``|``GCCcore/9.3.0``
``7.72.0``|``GCCcore/10.2.0``
``7.76.0``|``GCCcore/10.3.0``
``7.78.0``|``GCCcore/11.2.0``
``7.83.0``|``GCCcore/11.3.0``
``7.84.0``|``GCCcore/12.1.0``
``7.86.0``|``GCCcore/12.2.0``
``8.0.1`` |``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### cuSPARSELt

NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix

*homepage*: <https://docs.nvidia.com/cuda/cusparselt/index.html>

version    |versionsuffix   |toolchain
-----------|----------------|----------
``0.3.0.3``|``-CUDA-11.4.1``|``system``

### custodian

A simple JIT job management framework in Python.

*homepage*: <https://pypi.python.org/pypi/custodian>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.0``|``-Python-2.7.13``|``intel/2017a``

### cutadapt

Cutadapt finds and removes adapter sequences, primers, poly-A tails and  other types of unwanted sequence from your high-throughput sequencing reads.

*homepage*: <https://opensource.scilifelab.se/projects/cutadapt/>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``1.9.1``|``-Python-2.7.11``|``foss/2016a``
``1.9.1``|``-Python-2.7.12``|``foss/2016b``
``1.14`` |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.15`` |``-Python-3.5.2`` |``foss/2016b``
``1.16`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.16`` |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.16`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.18`` |                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.18`` |``-Python-2.7.15``|``foss/2018b``
``1.18`` |``-Python-2.7.18``|``GCC/10.2.0``
``1.18`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``2.1``  |``-Python-3.6.6`` |``foss/2018b``
``2.7``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.8``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.10`` |                  |``GCCcore/10.2.0``
``2.10`` |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.10`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``3.4``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.5``  |                  |``GCCcore/11.2.0``
``4.2``  |                  |``GCCcore/11.3.0``
``4.4``  |                  |``GCCcore/12.2.0``

### cuTENSOR

The cuTENSOR Library is a GPU-accelerated tensor linear algebra library providing tensor contraction,  reduction and elementwise operations.

*homepage*: <https://developer.nvidia.com/cutensor>

version    |versionsuffix   |toolchain
-----------|----------------|-----------------
``1.2.2.5``|                |``gcccuda/2019b``
``1.2.2.5``|``-CUDA-11.1.1``|``system``
``1.6.0.3``|``-CUDA-11.3.1``|``system``
``1.6.1.5``|``-CUDA-11.4.1``|``system``
``1.6.1.5``|``-CUDA-11.7.0``|``system``
``1.7.0.1``|``-CUDA-12.0.0``|``system``

### cuteSV

cuteSV uses tailored methods to collect the signatures of various types of SVs  and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection.

*homepage*: <https://github.com/tjiangHIT/cuteSV>

version  |toolchain
---------|--------------
``2.0.3``|``foss/2022a``

### CVS

CVS is a version control system, an important component of  Source Configuration Management (SCM).

*homepage*: <https://savannah.nongnu.org/projects/cvs>

version    |toolchain
-----------|-----------------------------------------------------------------------
``1.11.23``|``GCC/4.8.2``, ``GCCcore/11.2.0``, ``GCCcore/4.9.3``, ``GCCcore/6.4.0``

### CVXOPT

CVXOPT is a free software package for convex optimization based on the Python programming language.  Its main purpose is to make the development of software for convex optimization applications straightforward by  building on Python's extensive standard library and on the strengths of Python as a high-level programming language. 

*homepage*: <https://cvxopt.org>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------
``1.1.9``|``-Python-2.7.13``|``intel/2017a``
``1.2.1``|``-Python-3.6.4`` |``intel/2018a``
``1.2.3``|                  |``foss/2019a``
``1.2.3``|``-Python-3.6.6`` |``intel/2018b``
``1.2.4``|``-Python-3.7.4`` |``intel/2019b``
``1.2.6``|                  |``foss/2020b``, ``foss/2021a``
``1.3.1``|                  |``foss/2022a``

### CVXPY

CVXPY is a Python-embedded modeling language for convex optimization problems.  It allows you to express your problem in a natural way that follows the math,  rather than in the restrictive standard form required by solvers. 

*homepage*: <https://www.cvxpy.org/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``1.0.24``|                 |``foss/2019a``
``1.0.28``|``-Python-3.7.4``|``foss/2019b``

### CWIPI

CWIPI (Coupling With Interpolation Parallel Interface) library helps in chaining and coupling codes. Provides exchanges of interpolated fields through a non-compliant geometric interface and allows control of the coupling algorithm using control parameters. CWIPI takes advantage of the distribution of the definition of the coupling algorithm in the different codes. 

*homepage*: <https://w3.onera.fr/cwipi/>

version   |toolchain
----------|---------------
``0.12.0``|``gompi/2021a``

### cwltool

Common workflow language (CWL) reference implementation.

*homepage*: <https://github.com/common-workflow-language/cwltool>

version               |toolchain
----------------------|--------------
``3.1.20221008225030``|``foss/2021a``
``3.1.20221018083734``|``foss/2021a``

### cxxopts

cxxopts is a lightweight C++ command line option parser

*homepage*: <https://github.com/jarro2783/cxxopts>

version  |toolchain
---------|----------
``3.0.0``|``system``

### cysignals

The cysignals package provides mechanisms to handle  interrupts (and other signals and errors) in Cython code.

*homepage*: <https://pypi.org/project/cysignals/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.10.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.10.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``

### Cython

Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). 

*homepage*: <https://cython.org/>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``0.23.4`` |``-Python-2.7.10``|``gimkl/2.11.5``
``0.24.1`` |``-Python-2.7.11``|``foss/2016a``
``0.25.2`` |``-Python-2.7.12``|``foss/2016b``
``0.25.2`` |``-Python-3.6.4`` |``intel/2018a``
``0.27.3`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``0.29.10``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.29.10``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.29.22``|                  |``GCCcore/10.2.0``
``0.29.33``|                  |``GCCcore/11.3.0``
``3.0a5``  |                  |``GCCcore/10.2.0``

### cython-blis

Fast BLAS-like operations from Python and Cython, without the tears. Provides the Blis linear algebra routines as a self-contained Python C-extension. 

*homepage*: <https://github.com/explosion/cython-blis>

version  |toolchain
---------|--------------
``0.9.1``|``foss/2022a``

### cytoolz

Cython implementation of the toolz package, which provides high performance utility functions  for iterables, functions, and dictionaries.

*homepage*: <https://github.com/pytoolz/cytoolz>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``0.10.1``|``-Python-3.6.6``|``foss/2018b``
``0.10.1``|``-Python-3.7.2``|``GCCcore/8.2.0``

### Cytoscape

Cytoscape is an open source software platform for visualizing  complex networks and integrating these with any type of attribute data.  A lot of Apps are available for various kinds of problem domains,  including bioinformatics, social network analysis, and semantic web.

*homepage*: <https://cytoscape.org/>

version  |versionsuffix|toolchain
---------|-------------|----------
``3.9.1``|``-Java-11`` |``system``

### cytosim

Cytosim is a cytoskeleton simulation engine written in  C++ working on Mac OS, GNU/Linux and Windows (with Cygwin).

*homepage*: <https://github.com/nedelec/cytosim>

version     |versionsuffix|toolchain
------------|-------------|---------------
``20190117``|``-mkl``     |``gomkl/2019a``

### cyvcf2

cython + htslib == fast VCF and BCF processing

*homepage*: <https://github.com/brentp/cyvcf2>

version    |versionsuffix    |toolchain
-----------|-----------------|-------------------------------
``0.10.10``|``-Python-3.6.6``|``foss/2018b``
``0.11.5`` |                 |``foss/2019a``, ``intel/2019a``

## D


[dadi](#dadi) - [dagitty](#dagitty) - [Dakota](#dakota) - [DALI](#dali) - [DaliLite](#dalilite) - [Dalton](#dalton) - [damageproto](#damageproto) - [dammit](#dammit) - [DANPOS2](#danpos2) - [DAS_Tool](#das_tool) - [dask](#dask) - [dask-labextension](#dask-labextension) - [datamash](#datamash) - [davix](#davix) - [DB](#db) - [DB_File](#db_file) - [DBCSR](#dbcsr) - [DBD-mysql](#dbd-mysql) - [DBG2OLC](#dbg2olc) - [DBus](#dbus) - [dbus-glib](#dbus-glib) - [dclone](#dclone) - [dcm2niix](#dcm2niix) - [DCMTK](#dcmtk) - [dd](#dd) - [deal.II](#deal.ii) - [deap](#deap) - [decona](#decona) - [deconf](#deconf) - [DeconICA](#deconica) - [deepdiff](#deepdiff) - [DeepLabCut](#deeplabcut) - [deepmedic](#deepmedic) - [DeepMod2](#deepmod2) - [DeepSurv](#deepsurv) - [deepTools](#deeptools) - [DEICODE](#deicode) - [Delly](#delly) - [DeMixT](#demixt) - [DendroPy](#dendropy) - [DensPart](#denspart) - [Deprecated](#deprecated) - [destiny](#destiny) - [Detectron2](#detectron2) - [detonate](#detonate) - [Devito](#devito) - [DFA](#dfa) - [DFT-D3](#dft-d3) - [DFT-D4](#dft-d4) - [DFTB+](#dftb+) - [dftd3-lib](#dftd3-lib) - [DGL](#dgl) - [DIAL](#dial) - [DIALOGUE](#dialogue) - [DIAMOND](#diamond) - [Dice](#dice) - [DiCE-ML](#dice-ml) - [dicom2nifti](#dicom2nifti) - [DicomBrowser](#dicombrowser) - [Diffutils](#diffutils) - [dijitso](#dijitso) - [dill](#dill) - [DIRAC](#dirac) - [distributed](#distributed) - [DistributedStream](#distributedstream) - [DL_POLY_4](#dl_poly_4) - [DL_POLY_Classic](#dl_poly_classic) - [dlb](#dlb) - [dlib](#dlib) - [DLPack](#dlpack) - [dm-haiku](#dm-haiku) - [dm-reverb](#dm-reverb) - [dm-tree](#dm-tree) - [DMCfun](#dmcfun) - [DMLC-Core](#dmlc-core) - [DMTCP](#dmtcp) - [DOLFIN](#dolfin) - [dominate](#dominate) - [dorado](#dorado) - [Doris](#doris) - [DosageConvertor](#dosageconvertor) - [dotNET-Core-Runtime](#dotnet-core-runtime) - [dotNET-SDK](#dotnet-sdk) - [double-conversion](#double-conversion) - [DoubletFinder](#doubletfinder) - [Doxygen](#doxygen) - [DRAGMAP](#dragmap) - [Drake](#drake) - [dRep](#drep) - [DROP](#drop) - [dropEst](#dropest) - [DSA](#dsa) - [dSFMT](#dsfmt) - [DSRC](#dsrc) - [Dsuite](#dsuite) - [dtcmp](#dtcmp) - [dtcwt](#dtcwt) - [DualSPHysics](#dualsphysics) - [DUBStepR](#dubstepr) - [dune-core](#dune-core) - [dune-fem](#dune-fem) - [duplex-tools](#duplex-tools) - [dx-toolkit](#dx-toolkit) - [dxpy](#dxpy) - [DyMat](#dymat)


### dadi

∂a∂i implements methods for demographic history and selection inference from genetic data,  based on diffusion approximations to the allele frequency spectrum.

*homepage*: <https://bitbucket.org/gutenkunstlab/dadi>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.0``|``-Python-2.7.12``|``intel/2016b``

### dagitty

A port of the web-based software 'DAGitty', available at <http://dagitty.net>, for  analyzing structural causal models (also known as directed acyclic graphs or DAGs). This package  computes covariate adjustment sets for estimating causal effects, enumerates instrumental variables,  derives testable implications (d-separation and vanishing tetrads), generates equivalent models, and  includes a simple facility for data simulation.

*homepage*: <https://cran.r-project.org/web/packages/dagitty/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2-2``|``-R-3.5.1`` |``foss/2018b``

### Dakota

The Dakota project delivers both state-of-the-art research and robust, usable  software for optimization and UQ. Broadly, the Dakota software's advanced  parametric analyses enable design exploration, model calibration, risk  analysis, and quantification of margins and uncertainty with computational  models. 

*homepage*: <https://dakota.sandia.gov/>

version   |toolchain
----------|--------------
``6.16.0``|``foss/2021b``

### DALI

R-package for the analysis of single-cell TCR/BCR data in the Seurat ecosystem

*homepage*: <https://github.com/vibscc/DALI>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.1.2``|``-R-4.2.2`` |``foss/2022b``

### DaliLite

DaliLite is a light version of the software run by the Dali server. The web server has search and data visualization options which are not included in this package. DaliLite supports data import (import.pl) to convert PDB entries to Dali's internal data format and pairwise comparison (dali.pl) to structurally align a list of query structures to a list of target structures.

*homepage*: <http://ekhidna2.biocenter.helsinki.fi/dali/>

version|toolchain
-------|---------------
``4.1``|``gompi/2021a``

### Dalton

The Dalton code is a powerful tool for a wide range of molecular properties  at different levels of theory.  Any published work arising from use of one of the Dalton2016 programs  must acknowledge that by a proper reference,  https://www.daltonprogram.org/www/citation.html.

*homepage*: <https://daltonprogram.org/>

version   |versionsuffix|toolchain
----------|-------------|---------------
``2016``  |``-i8``      |``intel/2017b``
``2020.0``|             |``foss/2021a``
``2020.1``|             |``foss/2022b``

### damageproto

X protocol and ancillary headers for xinerama

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2016a``, ``intel/2016a``

### dammit

dammit is a simple de novo transcriptome annotator. It was born out of  the observations that annotation is mundane and annoying,  all the individual pieces of the process exist already, and the existing  solutions are overly complicated or rely on crappy non-free software.

*homepage*: <http://www.camillescott.org/dammit/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.3.2``|``-Python-2.7.13``|``intel/2017a``

### DANPOS2

A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2

*homepage*: <https://sites.google.com/site/danposdoc>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.2``|``-Python-2.7.12``|``foss/2016b``

### DAS_Tool

DAS Tool is an automated method that integrates the results of a flexible number of binning  algorithms to calculate an optimized, non-redundant set of bins from a single assembly.

*homepage*: <https://github.com/cmks/DAS_Tool>

version  |versionsuffix             |toolchain
---------|--------------------------|--------------
``1.1.1``|``-R-3.5.1-Python-2.7.15``|``foss/2018b``
``1.1.1``|``-R-4.1.2``              |``foss/2021b``
``1.1.3``|``-R-4.1.0``              |``foss/2021a``

### dask

Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance  at scale for the tools you love.

*homepage*: <https://dask.org/>

version      |versionsuffix     |toolchain
-------------|------------------|------------------------------------------------------------------------
``0.8.2``    |``-Python-2.7.11``|``intel/2016a``
``0.8.2``    |``-Python-3.5.1`` |``intel/2016a``
``0.11.0``   |``-Python-2.7.11``|``foss/2016a``
``0.11.0``   |``-Python-2.7.12``|``intel/2016b``
``0.11.0``   |``-Python-3.5.2`` |``intel/2016b``
``0.12.0``   |``-Python-2.7.12``|``intel/2016b``
``0.12.0``   |``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.16.0``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.16.0``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.17.0``   |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``0.17.0``   |``-Python-3.6.1`` |``intel/2017a``
``0.17.2``   |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.19.4``   |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.0.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.1.4``    |``-Python-2.7.15``|``fosscuda/2018b``
``2.3.0``    |``-Python-3.7.2`` |``foss/2019a``
``2.8.0``    |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.18.1``   |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2021.2.0`` |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2021.9.1`` |                  |``foss/2021a``
``2022.1.0`` |                  |``foss/2021b``
``2022.10.0``|                  |``foss/2022a``

### dask-labextension

This package provides a JupyterLab extension to manage Dask clusters, as well as embed Dask's dashboard plots directly into JupyterLab panes.

*homepage*: <https://github.com/dask/dask-labextension>

version  |toolchain
---------|--------------
``6.0.0``|``foss/2022a``

### datamash

GNU datamash performs basic numeric, textual and statistical operations on input data files

*homepage*: <https://www.gnu.org/software/datamash/>

version|toolchain
-------|--------------------------------------------------------
``1.3``|``foss/2018a``
``1.5``|``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``1.8``|``GCCcore/11.3.0``

### davix

The davix project aims to make file management over HTTP-based protocols simple.  The focus is on high-performance remote I/O and data management of large collections of files.  Currently, there is support for the WebDav (link is external), Amazon S3 (link is external),  Microsoft Azure (link is external), and HTTP (link is external) protocols.

*homepage*: <https://dmc.web.cern.ch/projects/davix/home>

version  |toolchain
---------|-----------------
``0.6.6``|``intel/2017a``
``0.7.5``|``GCCcore/8.3.0``

### DB

Berkeley DB enables the development of custom data management solutions,  without the overhead traditionally associated with such custom projects.

*homepage*: <http://www.oracle.com/technetwork/products/berkeleydb>

version    |toolchain
-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.8.30`` |``intel/2016a``
``6.2.23`` |``foss/2016a``
``6.2.32`` |``GCCcore/6.4.0``, ``intel/2017a``
``18.1.25``|``GCCcore/7.3.0``
``18.1.32``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``18.1.40``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

### DB_File

Perl5 access to Berkeley DB version 1.x.

*homepage*: <https://perldoc.perl.org/DB_File.html>

version  |versionsuffix   |toolchain
---------|----------------|------------------
``1.835``|                |``GCCcore/9.3.0``
``1.835``|``-Perl-5.20.3``|``intel/2016a``
``1.835``|``-Perl-5.22.1``|``foss/2016a``
``1.855``|                |``GCCcore/10.2.0``
``1.856``|                |``GCCcore/10.3.0``
``1.857``|                |``GCCcore/11.2.0``
``1.858``|                |``GCCcore/11.3.0``

### DBCSR

DBCSR stands for Distributed Blocked Compressed Sparse Row. DBCSR is a library designed to  efficiently perform sparse matrix-matrix multiplication, among other operations.

*homepage*: <https://cp2k.github.io/dbcsr/develop/>

version  |toolchain
---------|--------------
``2.5.0``|``foss/2021b``

### DBD-mysql

Perl binding for MySQL

*homepage*: <https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm>

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------------------
``4.032``|``-Perl-5.22.2``|``intel/2016a``
``4.033``|``-Perl-5.24.0``|``intel/2016b``
``4.042``|``-Perl-5.24.1``|``intel/2017a``
``4.046``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``4.046``|``-Perl-5.26.1``|``intel/2018a``
``4.048``|``-Perl-5.28.0``|``foss/2018b``
``4.050``|                |``GCC/10.2.0``, ``GCC/11.2.0``, ``GCC/11.3.0``
``4.050``|``-Perl-5.28.1``|``foss/2019a``

### DBG2OLC

DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation   Sequencing Technologies

*homepage*: <https://github.com/yechengxi/DBG2OLC>

version     |toolchain
------------|----------------------------------------------------------
``20170208``|``intel/2016b``, ``intel/2017a``
``20180221``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``20200724``|``GCC/11.3.0``

### DBus

D-Bus is a message bus system, a simple way for applications to talk  to one another.  In addition to interprocess communication, D-Bus helps  coordinate process lifecycle; it makes it simple and reliable to code  a "single instance" application or daemon, and to launch applications  and daemons on demand when their services are needed. 

*homepage*: <https://dbus.freedesktop.org/>

version    |toolchain
-----------|----------------------------------------------------------
``1.10.8`` |``foss/2016a``, ``intel/2016a``
``1.10.12``|``intel/2016b``
``1.10.20``|``GCCcore/6.4.0``
``1.11.20``|``intel/2017a``
``1.13.0`` |``intel/2017b``
``1.13.6`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.13.8`` |``GCCcore/8.2.0``
``1.13.12``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.13.18``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.14.0`` |``GCCcore/11.3.0``
``1.15.2`` |``GCCcore/12.2.0``
``1.15.4`` |``GCCcore/12.3.0``

### dbus-glib

D-Bus is a message bus system, a simple way for applications to talk to one another.

*homepage*: <https://dbus.freedesktop.org/doc/dbus-glib>

version  |toolchain
---------|------------------------------------------------------------------------
``0.106``|``foss/2016a``, ``intel/2016a``
``0.108``|``intel/2016b``, ``intel/2017a``
``0.110``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017b``
``0.112``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### dclone

Low level functions for implementing maximum likelihood estimating procedures for complex models  using data cloning and Bayesian Markov chain Monte Carlo methods

*homepage*: <https://cran.r-project.org/web/packages/dclone>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.3-0``|``-R-4.2.1`` |``foss/2022a``

### dcm2niix

dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.

*homepage*: <https://github.com/rordenlab/dcm2niix>

version         |toolchain
----------------|------------------------------------
``1.0.20180622``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.20190902``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.0.20200331``|``GCCcore/8.3.0``
``1.0.20201102``|``GCCcore/8.3.0``
``1.0.20211006``|``GCCcore/10.3.0``
``1.0.20220720``|``GCCcore/11.3.0``

### DCMTK

DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.

*homepage*: <https://dicom.offis.de/dcmtk>

version  |toolchain
---------|--------------------------------------
``3.6.3``|``GCCcore/7.3.0``
``3.6.5``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.6.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.6.7``|``GCCcore/11.3.0``

### dd

dd is a package for working with binary decision diagrams that includes both a pure Python implementation and Cython bindings to C libraries (CUDD, Sylvan, BuDDy). The Python and Cython modules implement the same API, so the same user code runs with both. All the standard operations on BDDs are available, including dynamic variable reordering using sifting, garbage collection, dump/load from files, plotting, and a parser of quantified Boolean expressions. This module includes bindings for: CUDD v3.0.0, Sylvan v1.0.0

*homepage*: <https://github.com/tulip-control/dd>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.5.6``|``-Python-3.8.2``|``foss/2020a``

### deal.II

deal.II is a C++ program library targeted at the computational solution of  partial differential equations using adaptive finite elements.

*homepage*: <https://www.dealii.org>

version  |toolchain
---------|-------------------------------
``9.1.1``|``foss/2019a``, ``intel/2019a``
``9.3.3``|``foss/2021a``

### deap

DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.  It seeks to make algorithms explicit and data structures transparent.

*homepage*: <http://deap.readthedocs.org/en/master/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.9.2``|``-Python-2.7.12``|``intel/2016b``

### decona

fastq to polished sequenses: pipeline suitable for mixed samples and long (Nanopore) reads

*homepage*: <https://github.com/Saskia-Oosterbroek/decona>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.2``|``-Python-3.7.4``|``foss/2019b``

### deconf

decomposition (deconfounding) of OMICS datasets in heterogeneous tissues

*homepage*: <http://web.cbio.uct.ac.za/~renaud/CRAN/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-3.5.1`` |``foss/2018b``

### DeconICA

Deconvolution of transcriptome through Immune Component Analysis (DeconICA) is an R package for identifying immune-related signals in transcriptome through deconvolution or unsupervised source separation methods.

*homepage*: <https://urszulaczerwinska.github.io/DeconICA>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.1.0``|``-R-3.5.1`` |``foss/2018b``

### deepdiff

DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively.

*homepage*: <https://zepworks.com/deepdiff/current/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.3.0``|``-Python-2.7.15``|``intel/2018b``
``3.3.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.0.6``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``5.0.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``5.7.0``|                  |``GCCcore/11.2.0``
``5.8.1``|                  |``GCCcore/11.3.0``

### DeepLabCut

Markerless tracking of user-defined features with deep learning

*homepage*: <http://www.mousemotorlab.org/deeplabcut>

version    |versionsuffix   |toolchain
-----------|----------------|--------------
``2.2.0.6``|``-CUDA-11.3.1``|``foss/2021a``

### deepmedic

Efficient Multi-Scale 3D Convolutional Neural Network for Segmentation of 3D Medical Scans.

*homepage*: <https://biomedia.doc.ic.ac.uk/software/deepmedic/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.8.2``|                |``foss/2021a``
``0.8.2``|``-CUDA-11.3.1``|``foss/2021a``

### DeepMod2

DeepMod2 is a computational tool for detecting DNA methylation and modifications  from Oxford Nanopore reads. 

*homepage*: <https://github.com/WGLab/DeepMod2>

version  |toolchain
---------|--------------
``0.0.1``|``foss/2021a``

### DeepSurv

DeepSurv is a deep learning approach to survival analysis. 

*homepage*: <https://github.com/jaredleekatzman/DeepSurv>

version           |versionsuffix    |toolchain
------------------|-----------------|------------------
``2.0.0-20180922``|``-Python-3.6.6``|``fosscuda/2018b``

### deepTools

deepTools is a suite of python tools particularly developed for the efficient analysis of  high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.

*homepage*: <https://deeptools.readthedocs.io/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.4``|``-Python-3.6.3``|``intel/2017b``
``3.3.1``|``-Python-3.6.6``|``foss/2018b``
``3.3.1``|``-Python-3.7.4``|``intel/2019b``
``3.3.1``|``-Python-3.8.2``|``foss/2020a``
``3.5.0``|                 |``foss/2021a``
``3.5.1``|                 |``foss/2021b``
``3.5.2``|                 |``foss/2022a``

### DEICODE

DEICODE is a form of Aitchison Distance that is robust to high levels of sparsity. DEICODE utilizes a natural solution to the zero problem formulated in recommendation systems called matrix completion. A simple way to interpret the method is, as a robust compositional PCA (via SVD) where zero values do not influence the resulting ordination. 

*homepage*: <https://library.qiime2.org/plugins/deicode/19/>

version  |toolchain
---------|--------------
``0.2.4``|``foss/2022a``

### Delly

Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome.

*homepage*: <https://github.com/dellytools/delly/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.8``|``-linux_x86_64``|``system``
``0.8.7``|                 |``gompi/2020b``
``1.1.5``|                 |``GCC/11.3.0``

### DeMixT

Cell type-specific deconvolution of heterogeneous tumor samples with two or three components  using expression data from RNAseq or microarray platforms.

*homepage*: <https://bioinformatics.mdanderson.org/main/DeMixT>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.1``|``-R-3.5.1`` |``foss/2018b``

### DendroPy

A Python library for phylogenetics and phylogenetic computing:  reading, writing, simulation, processing and manipulation of phylogenetic trees  (phylogenies) and characters.

*homepage*: <https://dendropy.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------------------------------------------
``4.4.0``|                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2019a``
``4.4.0``|``-Python-2.7.15``|``intel/2018b``
``4.5.2``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``4.5.2``|``-Python-2.7.18``|``GCCcore/10.2.0``

### DensPart

Atoms-in-molecules density partitioning schemes based on stockholder recipe

*homepage*: <https://github.com/theochem/denspart>

version     |toolchain
------------|---------------
``20220603``|``intel/2022a``

### Deprecated

If you need to mark a function or a method as deprecated, you can use the @deprecated decorator.

*homepage*: <https://github.com/tantale/deprecated>

version   |toolchain
----------|------------------------------
``1.2.13``|``foss/2021a``, ``foss/2022a``

### destiny

R packages to create and plot diffusion maps.

*homepage*: <https://bioconductor.org/packages/destiny>

version  |versionsuffix|toolchain
---------|-------------|---------------
``2.5.6``|``-R-3.4.0`` |``intel/2017a``

### Detectron2

Detectron2 is Facebook AI Research's next generation library that provides state-of-the-art detection and segmentation algorithms. It is the successor of Detectron and maskrcnn-benchmark. It supports a number of computer vision research projects and production applications in Facebook.

*homepage*: <https://github.com/facebookresearch/detectron2>

version|versionsuffix   |toolchain
-------|----------------|--------------
``0.6``|                |``foss/2021a``
``0.6``|``-CUDA-11.3.1``|``foss/2021a``

### detonate

DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation)  consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used  to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds  of genomic sequences.

*homepage*: <http://deweylab.biostat.wisc.edu/detonate/>

version |toolchain
--------|---------------
``1.11``|``intel/2017b``

### Devito

Devito is a domain-specific Language (DSL) and code generation framework for  performing optimised Finite Difference (FD) computation from high-level symbolic problem definitions.  Devito performs automated code generation and Just-In-time (JIT) compilation based on symbolic  equations defined in SymPy to create and execute highly optimised Finite Difference stencil kernels on  multiple computer platforms.

*homepage*: <https://www.devitoproject.org>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``4.6.1``|``-Python-3.8.2``|``foss/2020a``

### DFA

Python library for modeling DFAs, Moore Machines, and Transition Systems.

*homepage*: <https://github.com/mvcisback/dfa>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.3.4``|``-Python-3.8.2``|``GCCcore/9.3.0``
``2.1.2``|                 |``GCCcore/10.2.0``

### DFT-D3

DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical  quantum chemical methods.

*homepage*: <https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------
``3.2.0``|``GCC/8.3.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``, ``intel/2019a``

### DFT-D4

Generally Applicable Atomic-Charge Dependent London Dispersion Correction.

*homepage*: <https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.2.0``|``-Python-3.7.4``|``intel/2019b``

### DFTB+

DFTB+ is a fast and efficient versatile quantum mechanical simulation package. It is based on the Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of magnitude in speed.

*homepage*: <https://www.dftb-plus.info>

version  |versionsuffix         |toolchain
---------|----------------------|---------------
``1.3.1``|                      |``intel/2017a``
``17.1`` |``-Python-2.7.14``    |``intel/2017b``
``19.1`` |``-Python-2.7.16``    |``foss/2019b``
``19.1`` |``-Python-2.7.16-mpi``|``foss/2019b``
``21.1`` |                      |``intel/2021a``

### dftd3-lib

This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers. The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been converted to free format and encapsulated into modules.

*homepage*: <https://github.com/dftbplus/dftd3-lib>

version|toolchain
-------|-------------------------------------------
``0.9``|``GCC/8.3.0``, ``intel-compilers/2021.2.0``

### DGL

DGL is an simple-to-use, high performance and scalable Python package for deep learning on graphs. DGL is framework agnostic, meaning if a deep graph model is a component of an end-to-end application, the rest of the logics can be implemented in any major frameworks, such as PyTorch, Apache MXNet or TensorFlow.

*homepage*: <https://www.dgl.ai>

version  |versionsuffix                  |toolchain
---------|-------------------------------|------------------
``0.6.1``|``-Python-3.7.4-PyTorch-1.8.1``|``fosscuda/2019b``
``0.9.1``|``-CUDA-11.3.1``               |``foss/2021a``

### DIAL

DIAL (De novo Identification of Alleles) is a collection of programs  to automate the discovery of alleles for a species where we lack a reference sequence.   The SNPs/alleles are specifically selected for a low error rate in genotyping assays.   

*homepage*: <http://www.bx.psu.edu/miller_lab/>

version       |toolchain
--------------|--------------
``2011.06.06``|``foss/2016a``

### DIALOGUE

DIALOGUE is a dimensionality reduction method that uses cross-cell-type associations to  identify multicellular programs (MCPs) and map the cell transcriptome as a function of its environment.

*homepage*: <https://github.com/livnatje/DIALOGUE>

version         |versionsuffix|toolchain
----------------|-------------|--------------
``1.0-20230228``|``-R-4.2.0`` |``foss/2021b``

### DIAMOND

Accelerated BLAST compatible local sequence aligner

*homepage*: <https://github.com/bbuchfink/diamond>

version   |toolchain
----------|----------------------------------------------------------------
``0.9.22``|``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``0.9.24``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.9.30``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.9.36``|``GCC/9.3.0``
``2.0.4`` |``GCC/9.3.0``
``2.0.6`` |``GCC/7.3.0-2.30``
``2.0.7`` |``GCC/10.2.0``
``2.0.11``|``GCC/10.3.0``
``2.0.13``|``GCC/10.3.0``, ``GCC/11.2.0``
``2.1.0`` |``GCC/11.3.0``
``2.1.8`` |``GCC/10.3.0``

### Dice

Dice contains code for performing SHCI, VMC, GFMC, DMC, FCIQMC, stochastic MRCI and SC-NEVPT2, and AFQMC calculations with a focus on ab initio systems.

*homepage*: <https://github.com/sanshar/Dice>

version     |toolchain
------------|--------------
``20221025``|``foss/2022a``

### DiCE-ML

Diverse Counterfactual Explanations (DiCE) for ML

*homepage*: <https://interpret.ml/DiCE>

version|toolchain
-------|--------------
``0.9``|``foss/2022a``

### dicom2nifti

Python library for converting dicom files to nifti

*homepage*: <https://github.com/icometrix/dicom2nifti>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``2.2.12``|``-Python-3.8.2``|``foss/2020a``
``2.3.0`` |                 |``foss/2020b``, ``fosscuda/2020b``

### DicomBrowser

DicomBrowser is an application for inspecting and modifying DICOM metadata in many files at once.

*homepage*: <http://nrg.wustl.edu/software/dicom-browser/>

version    |versionsuffix     |toolchain
-----------|------------------|----------
``1.7.0b5``|``-Java-1.7.0_80``|``system``

### Diffutils

Diffutils: GNU diff utilities - find the differences between files

*homepage*: <http://www.gnu.org/software/diffutils/diffutils.html>

version|toolchain
-------|-------------
``3.3``|``GCC/4.8.2``

### dijitso

dijitso is a Python module for distributed just-in-time shared library building.

*homepage*: <https://bitbucket.org/fenics-project/dijitso>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

### dill

dill extends python's pickle module for serializing and de-serializing python objects to the majority  of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse  of which is converting a byte stream back to on python object hierarchy.

*homepage*: <https://pypi.org/project/dill/>

version  |toolchain
---------|--------------------------------------
``0.3.0``|``GCCcore/8.2.0``
``0.3.3``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``0.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.3.6``|``GCCcore/11.3.0``

### DIRAC

DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations

*homepage*: <http://www.diracprogram.org>

version |versionsuffix               |toolchain
--------|----------------------------|----------------------------------------------------------------
``19.0``|``-Python-2.7.18-int64``    |``intel/2020a``
``19.0``|``-Python-2.7.18-mpi-int64``|``intel/2020a``
``22.0``|                            |``foss/2021a``, ``intel/2021a``
``22.0``|``-int64``                  |``intel/2021a``
``23.0``|                            |``foss/2022a``, ``foss/2022b``, ``intel/2022a``, ``intel/2022b``
``23.0``|``-int64``                  |``intel/2022b``

### distributed

Dask.distributed is a lightweight library for distributed computing in Python.  It extends both the concurrent.futures and dask APIs to moderate sized clusters.

*homepage*: <https://distributed.readthedocs.io/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.14.3``|``-Python-2.7.12``|``intel/2016b``
``1.14.3``|``-Python-3.5.2`` |``intel/2016b``
``1.21.6``|``-Python-3.6.4`` |``intel/2018a``

### DistributedStream

A MPI distributed stream benchmark, useful to identifying nodes with poor memory performance and characterising memory bandwidth variation over systems.

*homepage*: <https://github.com/adrianjhpc/DistributedStream>

version|toolchain
-------|---------------
``1.0``|``gompi/2021a``

### DL_POLY_4

DL_POLY is a general purpose classical molecular dynamics (MD) simulation software

*homepage*: <https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx>

version  |toolchain
---------|-------------------------------
``5.0.0``|``foss/2020b``, ``intel/2020b``

### DL_POLY_Classic

DL_POLY Classic is a freely available molecular dynamics program developed  from the DL_POLY_2 package. This version does not install the java gui.

*homepage*: <http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/>

version |versionsuffix    |toolchain
--------|-----------------|-------------------------------
``1.9`` |                 |``intel/2016b``
``1.9`` |``-PLUMED-2.2.3``|``intel/2016b``
``1.10``|                 |``foss/2019b``, ``intel/2019b``

### dlb

DLB is a dynamic library designed to speed up HPC hybrid applications (i.e., two levels of parallelism) by improving the load balance of the outer level of parallelism (e.g., MPI) by dynamically redistributing the computational resources at the inner level of parallelism (e.g., OpenMP). at run time. 

*homepage*: <https://pm.bsc.es/dlb/>

version  |toolchain
---------|--------------------------------
``3.2``  |``gompi/2022a``, ``iimpi/2022a``
``3.3.1``|``gompi/2022a``, ``iimpi/2022a``

### dlib

Dlib is a modern C++ toolkit containing machine learning algorithms and tools for creating complex software in C++ to solve real world problems. It is used in both industry and academia in a wide range of domains including robotics, embedded devices, mobile phones, and large high performance computing environments.

*homepage*: <https://github.com/davisking/dlib>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``19.22``|                |``foss/2021a``
``19.22``|``-CUDA-11.3.1``|``foss/2021a``

### DLPack

DLPack is a stable in-memory data structure for an ndarray system to interact with a variety of frameworks.

*homepage*: <https://dmlc.github.io/dlpack/latest/>

version|toolchain
-------|--------------
``0.3``|``GCC/10.3.0``

### dm-haiku

Haiku is a simple neural network library for JAX developed by some of the authors of Sonnet, a neural network library for TensorFlow.

*homepage*: <https://github.com/deepmind/dm-haiku>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.0.9``|``-CUDA-11.3.1``|``foss/2021a``

### dm-reverb

Reverb is an efficient and simple-to-use data storage and transport system designed for machine learning research. Reverb is primarily used as an experience replay system for distributed reinforcement learning algorithms but the system also supports multiple data structure representations such as FIFO, LIFO, and priority queues.

*homepage*: <https://github.com/deepmind/reverb>

version  |toolchain
---------|--------------
``0.2.0``|``foss/2020b``
``0.7.0``|``foss/2021b``

### dm-tree

dm-tree provides tree, a library for working with nested data structures. In a way, tree generalizes the builtin map function which only supports flat sequences, and allows to apply a function to each "leaf" preserving the overall structure.

*homepage*: <https://github.com/deepmind/tree>

version  |toolchain
---------|--------------------------------------
``0.1.1``|``GCCcore/8.3.0``
``0.1.5``|``GCCcore/10.2.0``
``0.1.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.1.8``|``GCCcore/11.3.0``

### DMCfun

Diffusion Model of Conflict (DMC) in Reaction Time Tasks

*homepage*: <https://cran.r-project.org/web/packages/DMCfun>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.3.0``|``-R-3.6.2`` |``foss/2019b``

### DMLC-Core

DMLC-Core is the backbone library to support all DMLC projects, offers the bricks to build efficient and scalable distributed machine learning libraries.

*homepage*: <https://dmlc-core.readthedocs.io/en/latest>

version|toolchain
-------|--------------
``0.5``|``GCC/10.3.0``

### DMTCP

DMTCP is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications.

*homepage*: <http://dmtcp.sourceforge.net/index.html>

version  |toolchain
---------|-------------------------------------------------
``2.4.5``|``system``
``2.5.0``|``foss/2016a``
``2.5.1``|``system``
``2.5.2``|``GCCcore/8.3.0``, ``foss/2016b``, ``foss/2018b``
``2.6.0``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``
``3.0.0``|``GCCcore/11.3.0``

### DOLFIN

DOLFIN is the C++/Python interface of FEniCS, providing a consistent PSE   (Problem Solving Environment) for ordinary and partial differential equations.

*homepage*: <https://bitbucket.org/fenics-project/dolfin>

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``2018.1.0.post1``|``-Python-3.6.4``|``foss/2018a``
``2019.1.0.post0``|``-Python-3.7.4``|``foss/2019b``

### dominate

Dominate is a Python library for creating and manipulating HTML documents using an elegant DOM API. It allows you to write HTML pages in pure Python very concisely, which eliminates the need to learn another template language, and lets you take advantage of the more powerful features of Python. 

*homepage*: <https://github.com/Knio/dominate/>

version  |toolchain
---------|------------------
``2.8.0``|``GCCcore/11.3.0``

### dorado

Dorado is a high-performance, simple-to-use, open source basecaller for Oxford Nanopore reads.

*homepage*: <https://github.com/nanoporetech/dorado>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.1``|``-CUDA-11.7.0``|``foss/2022a``
``0.3.0``|``-CUDA-11.7.0``|``foss/2022a``
``0.3.1``|``-CUDA-11.7.0``|``foss/2022a``

### Doris

Delft object-oriented radar interferometric software

*homepage*: <http://doris.tudelft.nl/>

version      |toolchain
-------------|-------------------------------
``4.02``     |``intel/2017a``
``4.04beta4``|``foss/2018a``, ``intel/2017a``
``4.06beta2``|``intel/2017a``

### DosageConvertor

DosageConvertor is a C++ tool to convert dosage files (in VCF format) from Minimac3/4  to other formats such as MaCH or PLINK. Please note that this tool CANNOT handle missing values in  the input files and may NOT work for non-Minimac3/4 VCF files.

*homepage*: <http://genome.sph.umich.edu/wiki/DosageConvertor>

version  |toolchain
---------|--------------
``1.0.4``|``GCC/10.2.0``

### dotNET-Core-Runtime

.NET is a free, cross-platform, open source developer platform for building many different types  of applications.

*homepage*: <https://www.microsoft.com/net/>

version   |toolchain
----------|------------------
``2.0.7`` |``GCCcore/6.4.0``
``5.0.17``|``GCCcore/10.3.0``
``6.0.1`` |``GCCcore/11.2.0``

### dotNET-SDK

.NET is a free, cross-platform, open source developer platform for building many different types  of applications.

*homepage*: <https://www.microsoft.com/net/>

version    |versionsuffix |toolchain
-----------|--------------|----------
``3.1.300``|``-linux-x64``|``system``
``6.0.101``|``-linux-x64``|``system``

### double-conversion

Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.

*homepage*: <https://github.com/google/double-conversion>

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.0.3``|``foss/2018a``
``3.1.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.1.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``3.2.0``|``GCCcore/11.3.0``
``3.2.1``|``GCCcore/12.2.0``
``3.3.0``|``GCCcore/12.3.0``

### DoubletFinder

R package for detecting doublets in single-cell RNA sequencing data

*homepage*: <https://github.com/chris-mcginnis-ucsf/DoubletFinder>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``2.0.3``         |``-R-4.0.0`` |``foss/2020a``
``2.0.3-20230131``|``-R-4.2.1`` |``foss/2022a``

### Doxygen

Doxygen is a documentation system for C++, C, Java, Objective-C, Python,  IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some  extent D. 

*homepage*: <https://www.doxygen.org>

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.9.1``|``GCC/4.9.2``
``1.8.10`` |``GNU/4.9.3-2.25``, ``intel/2016.02-GCC-4.9``
``1.8.11`` |``GCC/4.9.2``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.8.13`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``
``1.8.14`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``1.8.15`` |``GCCcore/8.2.0``
``1.8.16`` |``GCCcore/8.3.0``
``1.8.17`` |``GCCcore/9.3.0``
``1.8.20`` |``GCCcore/10.2.0``
``1.9.1``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.9.4``  |``GCCcore/11.3.0``
``1.9.5``  |``GCCcore/12.2.0``
``1.9.7``  |``GCCcore/12.3.0``

### DRAGMAP

Dragmap is the Dragen mapper/aligner Open Source Software.

*homepage*: <https://github.com/Illumina/DRAGMAP>

version  |toolchain
---------|--------------
``1.3.0``|``foss/2021b``

### Drake

Drake is a simple-to-use, extensible, text-based data workflow tool that organizes command execution around data and its dependencies.

*homepage*: <https://github.com/Factual/drake>

version  |versionsuffix|toolchain
---------|-------------|----------
``1.0.3``|``-Java-1.8``|``system``

### dRep

dRep is a python program which performs rapid pair-wise comparison of genome sets. One of it’s major purposes is for genome de-replication, but it can do a lot more.

*homepage*: <https://drep.readthedocs.io/en/latest/>

version  |toolchain
---------|--------------
``3.0.0``|``foss/2021a``
``3.4.2``|``foss/2022a``

### DROP

Pipeline to find aberrant events in RNA-Seq data, useful for diagnosis of rare disorders

*homepage*: <https://gagneurlab-drop.readthedocs.io>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.3``|``-R-4.0.3`` |``foss/2020b``
``1.1.0``|``-R-4.0.3`` |``foss/2020b``
``1.1.1``|``-R-4.1.2`` |``foss/2021b``

### dropEst

Pipeline for initial analysis of droplet-based single-cell RNA-seq data

*homepage*: <https://github.com/hms-dbmi/dropEst>

version  |versionsuffix|toolchain
---------|-------------|---------------
``0.7.1``|``-R-3.4.3`` |``intel/2017b``

### DSA

Digital Sorting Algorithm

*homepage*: <https://github.com/zhandong/DSA>

version|versionsuffix|toolchain
-------|-------------|--------------
``1.0``|``-R-3.5.1`` |``foss/2018b``

### dSFMT

Double precision SIMD-oriented Fast Mersenne Twister.

*homepage*: <https://github.com/MersenneTwister-Lab/dSFMT>

version  |toolchain
---------|------------------
``2.2.5``|``GCCcore/10.2.0``

### DSRC

DNA Sequence Reads Compression is an application designed for compression of data files  containing reads from DNA sequencing in FASTQ format. The amount of such files can be huge, e.g., a few  (or tens) of gigabytes, so a need for a robust data compression tool is clear. Usually universal compression  programs like gzip or bzip2 are used for this purpose, but it is obvious that a specialized tool can work better.

*homepage*: <http://sun.aei.polsl.pl/dsrc>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``2.0rc``|``-linux-64-bit``|``system``

### Dsuite

Fast calculation of the ABBA-BABA statistics across many populations/species

*homepage*: <https://github.com/millanek/Dsuite>

version     |toolchain
------------|--------------------------------------------
``20190713``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``20210718``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``

### dtcmp

The Datatype Comparison (DTCMP) Library provides pre-defined and user-defined comparison operations to compare the values of two items which can be arbitrary MPI datatypes. Using these comparison operations, the library provides various routines for manipulating data, which may be distributed over the processes of an MPI communicator.

*homepage*: <https://github.com/LLNL/dtcmp>

version  |toolchain
---------|------------------------------------------------------------------
``1.1.0``|``gompi/2019a``, ``gompi/2020a``, ``iimpi/2019a``, ``iimpi/2020a``
``1.1.2``|``gompi/2020b``
``1.1.4``|``gompi/2022a``

### dtcwt

Dual-Tree Complex Wavelet Transform library for Python

*homepage*: <https://github.com/rjw57/dtcwt>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12.0``|``-Python-2.7.15``|``foss/2018b``
``0.12.0``|``-Python-3.7.4`` |``intel/2019b``

### DualSPHysics

DualSPHysics is based on the Smoothed Particle Hydrodynamics model named SPHysics.   The code is developed to study free-surface flow phenomena where Eulerian methods  can be difficult to apply, such as waves or impact of dam-breaks on off-shore  structures. DualSPHysics is a set of C++, CUDA and Java codes designed to deal  with real-life engineering problems.

*homepage*: <https://dual.sphysics.org/>

version    |versionsuffix        |toolchain
-----------|---------------------|--------------
``5.0.175``|                     |``GCC/11.2.0``
``5.0.175``|``-CUDA-%(cudaver)s``|``GCC/11.2.0``

### DUBStepR

DUBStepR (Determining the Underlying Basis using Step-wise Regression) is a feature selection algorithm for cell type identification in single-cell RNA-sequencing data.

*homepage*: <https://github.com/prabhakarlab/DUBStepR>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.2.0``|``-R-4.1.2`` |``foss/2021b``

### dune-core

The Dune core modules build the stable basis of Dune.  They follow a consistent release cycle and have high requirements regarding stability and backwards compatibility.  These modules build the foundation for higher-level components. 

*homepage*: <https://www.dune-project.org/groups/core/>

version        |toolchain
---------------|--------------
``2.8.0.post1``|``foss/2020b``

### dune-fem

DUNE-FEM is a discretization module based on DUNE containing all the building blocks required to implement efficient solvers for a wide range of (systems of non linear) partial differential equations. DUNE-FEM can also be used through an extensive Python interface which brings all components of DUNE-FEM and the DUNE core modules to Python. 

*homepage*: <https://www.dune-project.org/modules/dune-fem/>

version    |toolchain
-----------|--------------
``2.8.0.6``|``foss/2020b``

### duplex-tools

Duplex Tools contains a set of utilities for dealing with Duplex sequencing data. Tools are provided  to identify and prepare duplex pairs for basecalling by Dorado (recommended) and Guppy, and for  recovering simplex basecalls from incorrectly concatenated pairs. 

*homepage*: <https://github.com/nanoporetech/duplex-tools>

version  |toolchain
---------|--------------
``0.3.1``|``foss/2022a``
``0.3.3``|``foss/2022a``

### dx-toolkit

The DNAnexus Platform SDK - also called dx-toolkit - includes the dx command-line client;  tools for building and debugging apps; utilities for working with DNA data on the DNAnexus Platform;  and Python, Java, C++ and R bindings for working on the DNAnexus Platform.  

*homepage*: <https://documentation.dnanexus.com/downloads#dnanexus-platform-sdk>

version    |toolchain
-----------|------------------
``0.350.1``|``GCCcore/12.2.0``

### dxpy

DNAnexus Platform API bindings for Python

*homepage*: <http://autodoc.dnanexus.com/>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.266.1``|``-Python-2.7.14``|``intel/2018a``

### DyMat

Read and process result files from Dymola and OpenModelica with Python.

*homepage*: <https://www.j-raedler.de/projects/DyMat>

version|versionsuffix  |toolchain
-------|---------------|--------------
``0.7``|``-2020-12-12``|``foss/2021b``

## E


[E-ANTIC](#e-antic) - [ea-utils](#ea-utils) - [earthengine-api](#earthengine-api) - [easel](#easel) - [SimpleBuild](#simplebuild) - [SimpleQC](#simpleqc) - [ecBuild](#ecbuild) - [ecCodes](#eccodes) - [eccodes-python](#eccodes-python) - [ecFlow](#ecflow) - [ED2](#ed2) - [EDirect](#edirect) - [edlib](#edlib) - [EggLib](#egglib) - [eggnog-mapper](#eggnog-mapper) - [EGTtools](#egttools) - [eht-imaging](#eht-imaging) - [Eigen](#eigen) - [EigenExa](#eigenexa) - [EIGENSOFT](#eigensoft) - [einops](#einops) - [elastix](#elastix) - [elbencho](#elbencho) - [ELFIO](#elfio) - [elfutils](#elfutils) - [Elk](#elk) - [ELPA](#elpa) - [ELPH](#elph) - [elprep](#elprep) - [ELSI](#elsi) - [ELSI-RCI](#elsi-rci) - [Emacs](#emacs) - [EMAN2](#eman2) - [EMBOSS](#emboss) - [Embree](#embree) - [emcee](#emcee) - [EMU](#emu) - [enaBrowserTool](#enabrowsertool) - [enchant](#enchant) - [enchant-2](#enchant-2) - [EnsEMBLCoreAPI](#ensemblcoreapi) - [entrypoints](#entrypoints) - [EPD](#epd) - [EPIC](#epic) - [epiScanpy](#episcanpy) - [eQuilibrator](#equilibrator) - [EricScript](#ericscript) - [ESL-Bundle](#esl-bundle) - [ESM-2](#esm-2) - [ESMF](#esmf) - [ESMPy](#esmpy) - [ESMValTool](#esmvaltool) - [eSpeak-NG](#espeak-ng) - [ESPResSo](#espresso) - [Essentia](#essentia) - [ETE](#ete) - [ETSF_IO](#etsf_io) - [eudev](#eudev) - [Evcxr-REPL](#evcxr-repl) - [EvidentialGene](#evidentialgene) - [evmix](#evmix) - [ExaBayes](#exabayes) - [ExaML](#examl) - [Excel-Writer-XLSX](#excel-writer-xlsx) - [ExifTool](#exiftool) - [exiv2](#exiv2) - [Exonerate](#exonerate) - [expat](#expat) - [expect](#expect) - [expecttest](#expecttest) - [eXpress](#express) - [Extrae](#extrae) - [EZC3D](#ezc3d)


### E-ANTIC

E-ANTIC is a C/C++ library to deal with real embedded number fields built on top of ANTIC (https://github.com/wbhart/antic). Its aim is to have as fast as possible exact arithmetic operations and comparisons.

*homepage*: <https://github.com/videlec/e-antic>

version  |toolchain
---------|--------------------
``0.1.2``|``GCC/8.2.0-2.31.1``
``0.1.5``|``GCC/8.3.0``

### ea-utils

Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc.  Primarily written to support an Illumina based pipeline -  but should work with any FASTQs.

*homepage*: <http://expressionanalysis.github.io/ea-utils/>

version     |toolchain
------------|-----------------------------------------------
``1.04.807``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``

### earthengine-api

Python and JavaScript bindings for calling the Earth Engine API

*homepage*: <https://github.com/google/earthengine-api>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.1.143``|``-Python-2.7.14``|``intel/2018a``

### easel

Easel supports computational analysis of biological sequences using probabilistic models.

*homepage*: <https://bioeasel.org/>

version |toolchain
--------|--------------
``0.48``|``GCC/12.2.0``

### SimpleBuild

SimpleBuild is a software build and installation framework  written in Python that allows you to install software in a structured,  repeatable and robust way.

*homepage*: <https://simplebuilders.github.io/simplebuild>

version   |toolchain
----------|----------
``1.0.0`` |``system``
``1.0.1`` |``system``
``1.0.2`` |``system``
``1.1.0`` |``system``
``1.2.0`` |``system``
``1.3.0`` |``system``
``1.4.0`` |``system``
``1.5.0`` |``system``
``1.6.0`` |``system``
``1.7.0`` |``system``
``1.8.0`` |``system``
``1.8.1`` |``system``
``1.8.2`` |``system``
``1.9.0`` |``system``
``1.10.0``|``system``
``1.11.0``|``system``
``1.11.1``|``system``
``1.12.0``|``system``
``1.12.1``|``system``
``1.13.0``|``system``
``1.14.0``|``system``
``1.15.0``|``system``
``1.15.1``|``system``
``1.15.2``|``system``
``1.16.0``|``system``
``1.16.1``|``system``
``1.16.2``|``system``
``2.0.0`` |``system``
``2.1.0`` |``system``
``2.1.1`` |``system``
``2.2.0`` |``system``
``2.3.0`` |``system``
``2.4.0`` |``system``
``2.5.0`` |``system``
``2.6.0`` |``system``
``2.7.0`` |``system``
``2.8.0`` |``system``
``2.8.1`` |``system``
``2.8.2`` |``system``
``2.9.0`` |``system``
``3.0.0`` |``system``
``3.0.1`` |``system``
``3.0.2`` |``system``
``3.1.0`` |``system``
``3.1.1`` |``system``
``3.1.2`` |``system``
``3.2.0`` |``system``
``3.2.1`` |``system``
``3.3.0`` |``system``
``3.3.1`` |``system``
``3.4.0`` |``system``
``3.4.1`` |``system``
``3.5.0`` |``system``
``3.5.1`` |``system``
``3.5.2`` |``system``
``3.5.3`` |``system``
``3.6.0`` |``system``
``3.6.1`` |``system``
``3.6.2`` |``system``
``3.7.0`` |``system``
``3.7.1`` |``system``
``3.8.0`` |``system``
``3.8.1`` |``system``
``3.9.0`` |``system``
``3.9.1`` |``system``
``3.9.2`` |``system``
``3.9.3`` |``system``
``3.9.4`` |``system``
``4.0.0`` |``system``
``4.0.1`` |``system``
``4.1.0`` |``system``
``4.1.1`` |``system``
``4.1.2`` |``system``
``4.2.0`` |``system``
``4.2.1`` |``system``
``4.2.2`` |``system``
``4.3.0`` |``system``
``4.3.1`` |``system``
``4.3.2`` |``system``
``4.3.3`` |``system``
``4.3.4`` |``system``
``4.4.0`` |``system``
``4.4.1`` |``system``
``4.4.2`` |``system``
``4.5.0`` |``system``
``4.5.1`` |``system``
``4.5.2`` |``system``
``4.5.3`` |``system``
``4.5.4`` |``system``
``4.5.5`` |``system``
``4.6.0`` |``system``
``4.6.1`` |``system``
``4.6.2`` |``system``
``4.7.0`` |``system``
``4.7.1`` |``system``
``4.7.2`` |``system``

### SimpleQC

SimpleQC is an R-package that provides advanced functionality to  (1) perform file-level QC of single genome-wide association (GWA) data-sets  (2) conduct quality control across several GWA data-sets (meta-level QC)  (3) simplify data-handling of large-scale GWA data-sets.

*homepage*: <http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/>

version|versionsuffix|toolchain
-------|-------------|---------------
``9.2``|``-R-3.3.1`` |``intel/2016b``

### ecBuild

A CMake-based build system, consisting of a collection of CMake macros and functions that ease the managing of software build systems 

*homepage*: <https://ecbuild.readthedocs.io/>

version  |toolchain
---------|----------
``3.7.0``|``system``

### ecCodes

ecCodes is a package developed by ECMWF which provides an application programming interface and  a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,  WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding).

*homepage*: <https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``2.7.3`` |                  |``intel/2018a``
``2.7.3`` |``-Python-2.7.14``|``intel/2017b``
``2.8.2`` |                  |``intel/2018a``
``2.9.2`` |                  |``intel/2018b``, ``iomkl/2018b``
``2.12.5``|                  |``gompi/2019a``
``2.15.0``|                  |``gompi/2019b``, ``iimpi/2019b``
``2.17.0``|                  |``foss/2018b``, ``gompi/2019b``
``2.18.0``|                  |``gompi/2020a``
``2.20.0``|                  |``gompi/2020b``
``2.22.1``|                  |``gompi/2021a``
``2.24.2``|                  |``gompi/2021b``, ``iimpi/2021b``
``2.27.0``|                  |``gompi/2022a``

### eccodes-python

Python 3 interface to decode and encode GRIB and BUFR files via the ECMWF ecCodes library.

*homepage*: <https://github.com/ecmwf/eccodes-python>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.7.4``|``foss/2019b``
``1.0.0``|``-Python-3.8.2``|``foss/2020a``
``1.1.0``|                 |``foss/2020b``

### ecFlow

ecFlow is a client/server workflow package that enables users  to run a large number of programs (with dependencies on each other and on time)  in a controlled environment. It provides reasonable tolerance for hardware and  software failures, combined with restart capabilities.  It is used at ECMWF to run all our operational suites across a range of platforms.

*homepage*: <https://ecflow.readthedocs.io/en/latest/index.html>

version  |toolchain
---------|--------------
``5.7.0``|``GCC/10.2.0``

### ED2

The Ecosystem Demography Biosphere Model (ED2) is an integrated terrestrial biosphere model  incorporating hydrology, land-surface biophysics, vegetation dynamics, and soil carbon and nitrogen biogeochemistry

*homepage*: <https://github.com/EDmodel/ED2>

version     |versionsuffix|toolchain
------------|-------------|---------------
``20170201``|             |``intel/2017a``
``20170201``|``-serial``  |``intel/2017a``

### EDirect

Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases from a Unix terminal window. Search terms are entered as command-line arguments. Individual operations are connected with Unix pipes to construct multi-step queries. Selected records can then be retrieved in a variety of formats.

*homepage*: <https://www.ncbi.nlm.nih.gov/books/NBK25501/>

version          |toolchain
-----------------|------------------
``19.7.20230531``|``GCCcore/10.3.0``

### edlib

Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.

*homepage*: <https://martinsos.github.io/edlib>

version        |versionsuffix    |toolchain
---------------|-----------------|------------------------------------------------------------------------------
``1.3.8.post1``|``-Python-3.7.4``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``1.3.8.post1``|``-Python-3.8.2``|``GCC/9.3.0``
``1.3.8.post2``|``-Python-3.8.2``|``iccifort/2020.1.217``
``1.3.9``      |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### EggLib

EggLib is a C++/Python library and program package for evolutionary genetics and genomics.

*homepage*: <http://egglib.sourceforge.net/>

version   |toolchain
----------|---------------
``2.1.10``|``intel/2016a``

### eggnog-mapper

EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only. Common uses of eggNOG-mapper include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.

*homepage*: <https://github.com/eggnogdb/eggnog-mapper>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.0.3`` |``-Python-2.7.14``|``intel/2018a``
``2.1.4`` |                  |``foss/2020b``
``2.1.7`` |                  |``foss/2021b``
``2.1.9`` |                  |``foss/2022a``
``2.1.10``|                  |``foss/2020b``

### EGTtools

EGTtools provides a centralized repository with analytical and numerical methods to study/model game theoretical problems under the Evolutionary Game Theory (EGT) framework.

*homepage*: <https://github.com/Socrats/EGTTools>

version        |toolchain
---------------|--------------
``0.1.10.dev2``|``foss/2021b``
``0.1.11``     |``foss/2022a``

### eht-imaging

Python modules for simulating and manipulating VLBI data and producing images with regularized maximum likelihood methods.

*homepage*: <https://github.com/achael/eht-imaging>

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021a``

### Eigen

Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,  and related algorithms.

*homepage*: <https://eigen.tuxfamily.org>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------
``3.2.3`` |``foss/2016a``, ``foss/2016b``
``3.2.5`` |``system``
``3.2.6`` |``system``
``3.2.7`` |``foss/2016a``, ``intel/2016a``
``3.2.8`` |``foss/2016a``, ``intel/2016a``, ``system``
``3.2.9`` |``foss/2016b``, ``intel/2016b``
``3.2.10``|``intel/2016b``
``3.3.2`` |``foss/2016b``, ``intel/2016b``
``3.3.3`` |``GCCcore/6.3.0``, ``intel/2016b``
``3.3.4`` |``system``
``3.3.5`` |``system``
``3.3.7`` |``GCCcore/9.3.0``, ``system``
``3.3.8`` |``GCCcore/10.2.0``
``3.3.9`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.4.0`` |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### EigenExa

EigenExa, a part of KMATHLIB, is a high performance eigen-solver.

*homepage*: <https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/>

version |toolchain
--------|---------------
``2.11``|``intel/2020b``

### EIGENSOFT

The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.  2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal  components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of  variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral  populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT  package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.

*homepage*: <http://www.hsph.harvard.edu/alkes-price/software/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------
``6.0.1``|``foss/2016a``
``6.1.1``|``foss/2016a``
``6.1.4``|``foss/2016b``
``7.2.1``|``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2020b``, ``foss/2021a``, ``intel/2019a``

### einops

Flexible and powerful tensor operations for readable and reliable code. Supports numpy, pytorch, tensorflow, jax, and others.

*homepage*: <https://einops.rocks/>

version  |toolchain
---------|--------------------------------------
``0.3.2``|``GCCcore/10.2.0``
``0.4.1``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### elastix

elastix: a toolbox for rigid and nonrigid registration of images. 

*homepage*: <http://elastix.isi.uu.nl/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``4.9.0``|                 |``foss/2018a``
``5.0.0``|``-Python-3.7.4``|``foss/2019b``

### elbencho

A distributed storage benchmark for files, objects & blocks with support for GPUs

*homepage*: <https://github.com/breuner/elbencho>

version  |toolchain
---------|--------------
``2.0-3``|``GCC/10.3.0``

### ELFIO

ELFIO is a header-only C++ library intended  for reading and generating files in the ELF binary format.

*homepage*: <http://elfio.sourceforge.net>

version|toolchain
-------|----------
``3.9``|``system``

### elfutils

The elfutils project provides libraries and tools for ELF files  and DWARF data. 

*homepage*: <https://elfutils.org/>

version  |toolchain
---------|----------------------------------------------------------------------------
``0.182``|``GCCcore/9.3.0``
``0.183``|``GCCcore/10.2.0``
``0.185``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.187``|``GCCcore/11.3.0``
``0.189``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### Elk

An all-electron full-potential linearised augmented-plane wave (FP-LAPW) code with many advanced features. Written originally at Karl-Franzens-Universität Graz as a milestone of the EXCITING EU Research and Training Network, the code is designed to be as simple as possible so that new developments in the field of density functional theory (DFT) can be added quickly and reliably. 

*homepage*: <http://elk.sourceforge.net/>

version   |toolchain
----------|---------------
``4.0.15``|``intel/2016b``
``4.3.6`` |``intel/2017a``
``6.3.2`` |``intel/2019b``
``7.0.12``|``foss/2020b``
``7.2.42``|``foss/2021a``
``8.5.2`` |``foss/2022a``

### ELPA

Eigenvalue SoLvers for Petaflop-Applications.

*homepage*: <https://elpa.mpcdf.mpg.de/>

version            |versionsuffix   |toolchain
-------------------|----------------|---------------------------------------------------------------------------------
``2015.02.002``    |                |``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``2016.05.004``    |                |``intel/2016b``, ``intel/2017a``
``2016.11.001.pre``|                |``foss/2018b``, ``intel/2018b``
``2017.11.001``    |                |``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2018.05.001``    |                |``foss/2018b``, ``intel/2018b``
``2018.11.001``    |                |``intel/2019a``
``2019.11.001``    |                |``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``, ``iomkl/2019b``
``2020.05.001``    |                |``intel/2020a``
``2020.11.001``    |                |``foss/2020b``, ``intel/2020b``
``2021.05.001``    |                |``foss/2021a``, ``foss/2021b``, ``intel/2021a``, ``intel/2021b``
``2021.05.002``    |                |``intel/2020b``
``2021.11.001``    |                |``foss/2021b``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``, ``intel/2022b``
``2022.05.001``    |                |``foss/2022a``, ``foss/2022b``, ``intel/2022a``, ``intel/2022b``
``2022.05.001``    |``-CUDA-11.7.0``|``foss/2022a``
``2022.05.001``    |``-CUDA-12.0.0``|``foss/2022b``

### ELPH

ELPH is a general-purpose Gibbs sampler for finding motifs in a set   of DNA or protein sequences. The program takes as input a set containing anywhere from   a few dozen to thousands of sequences, and searches through them for the most common motif,   assuming that each sequence contains one copy of the motif. We have used ELPH to find   patterns such as ribosome binding sites (RBSs) and exon splicing enhancers (ESEs). 

*homepage*: <http://ccb.jhu.edu/software/ELPH/index.shtml>

version  |toolchain
---------|--------------
``1.0.1``|``foss/2018b``

### elprep

elPrep is a high-performance tool for analyzing .sam/.bam files (up to and including variant calling)  in sequencing pipelines.

*homepage*: <https://github.com/ExaScience/elprep>

version  |toolchain
---------|----------
``5.1.1``|``system``

### ELSI

ELSI provides and enhances scalable, open-source software library solutions for  electronic structure calculations in  materials science, condensed matter physics, chemistry, and many other fields.  ELSI focuses on methods that solve or circumvent eigenvalue problems in electronic structure theory.  The ELSI infrastructure should also be useful for other challenging eigenvalue problems. 

*homepage*: <https://wordpress.elsi-interchange.org/>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``2.5.0``|             |``foss/2019b``, ``intel/2019b``
``2.5.0``|``-PEXSI``   |``foss/2019b``, ``intel/2019b``
``2.6.4``|``-PEXSI``   |``foss/2020b``, ``intel/2020b``
``2.7.1``|``-PEXSI``   |``foss/2021a``, ``intel/2021a``

### ELSI-RCI

ELSI-RCI provides and enhances open-source software packages which iteratively solve or circumvent eigenvalue problems in self-consistent field calculations based on the Kohn-Sham density-functional theory. 

*homepage*: <https://wordpress.elsi-interchange.org/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------
``0.1.0``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2020b``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``, ``intel/2020b``

### Emacs

GNU Emacs is an extensible, customizable text editor--and more.  At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming  language with extensions to support text editing.

*homepage*: <https://www.gnu.org/software/emacs/>

version |versionsuffix|toolchain
--------|-------------|-------------------------------------------------------
``24.3``|             |``GCC/4.8.3``
``24.3``|``-bare``    |``GCC/4.8.3``
``24.4``|             |``GCC/4.9.2``
``24.5``|             |``GCC/4.9.3-2.25``
``25.1``|             |``foss/2016a``
``25.3``|             |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``26.3``|             |``GCCcore/8.3.0``
``27.1``|             |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``27.2``|             |``GCCcore/11.2.0``
``28.1``|             |``GCCcore/10.2.0``
``28.2``|             |``GCCcore/12.2.0``

### EMAN2

EMAN2 is a broadly based greyscale scientific image processing suite with a  primary focus on processing data from transmission electron microscopes. 

*homepage*: <https://blake.bcm.edu/emanwiki/EMAN2>

version  |versionsuffix                  |toolchain
---------|-------------------------------|----------------------------------
``2.3``  |                               |``system``
``2.3``  |``-Python-2.7.15``             |``foss/2019a``, ``fosscuda/2019a``
``2.21a``|``-Python-2.7.14-Boost-1.63.0``|``foss/2018a``

### EMBOSS

EMBOSS is 'The European Molecular Biology Open Software Suite'.  EMBOSS is a free Open Source software analysis package specially developed for  the needs of the molecular biology (e.g. EMBnet) user community.

*homepage*: <http://emboss.sourceforge.net/>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.6.0``|                 |``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018b``, ``foss/2021a``, ``foss/2021b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017a``, ``intel/2018b``
``6.6.0``|``-Java-11``     |``GCC/8.3.0``
``6.6.0``|``-Java-13``     |``GCC/10.2.0``
``6.6.0``|``-X11-20170314``|``intel/2017a``

### Embree

Intel® Embree is a collection of high-performance ray tracing kernels, developed at Intel.  The target users of Intel® Embree are graphics application engineers who want to improve the performance of  their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels.  The kernels are optimized for the latest Intel® processors with support for SSE, AVX, AVX2, and AVX-512 instructions.  Intel® Embree supports runtime code selection to choose the traversal and build algorithms that best matches  the instruction set of your CPU. We recommend using Intel® Embree through its API to get the highest benefit  from future improvements. Intel® Embree is released as Open Source under the Apache 2.0 license.

*homepage*: <https://embree.github.io/>

version   |toolchain
----------|--------------------------------------
``3.4.0`` |``iccifort/2018.1.163-GCC-6.4.0-2.28``
``3.13.4``|``system``

### emcee

Emcee is an extensible, pure-Python implementation of  Goodman & Weare's Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler.  It's designed for Bayesian parameter estimation and it's really sweet! 

*homepage*: <https://emcee.readthedocs.io/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.2.1``|                  |``foss/2019a``
``2.2.1``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.2.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.1.4``|                  |``foss/2021b``

### EMU

EMU infers population structure in the presence of missingness and works for both haploid, psuedo-haploid and diploid genotype datasets 

*homepage*: <http://www.popgen.dk/software/index.php/EMU>

version |versionsuffix    |toolchain
--------|-----------------|--------------
``0.66``|``-Python-3.7.4``|``foss/2019b``

### enaBrowserTool

enaBrowserTools is a set of scripts that interface with the ENA  web services to download data from ENA easily, without any knowledge of scripting required.

*homepage*: <https://github.com/enasequence/enaBrowserTools/>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.5.4``|``-Python-3.7.2``|``GCCcore/8.2.0``
``1.6``  |                 |``GCCcore/10.3.0``

### enchant

Enchant is a library (and command-line program) that wraps a number of different spelling libraries  and programs with a consistent interface.

*homepage*: <https://abiword.github.io/enchant/>

version  |toolchain
---------|---------------
``1.6.1``|``intel/2017a``

### enchant-2

Enchant aims to provide a simple but comprehensive abstraction for dealing with different spell checking libraries in a consistent way. A client, such as a text editor or word processor, need not know anything about a specific spell-checker, and since all back-ends are plugins, new spell-checkers can be added without needing any change to the program using Enchant.

*homepage*: <https://github.com/AbiWord/enchant>

version  |toolchain
---------|--------------------------------------
``2.3.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### EnsEMBLCoreAPI

The Ensembl Core Perl API and SQL schema

*homepage*: <https://www.ensembl.org/info/docs/api/index.html>

version           |versionsuffix   |toolchain
------------------|----------------|--------------
``96.0-r20190601``|``-Perl-5.28.1``|``foss/2019a``

### entrypoints

Entry points are a way for Python packages to advertise objects with some common interface.

*homepage*: <https://github.com/takluyver/entrypoints>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.2.2``|``-Python-2.7.11``|``foss/2016a``
``0.2.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.2.2``|``-Python-3.5.1`` |``foss/2016a``
``0.2.2``|``-Python-3.5.2`` |``intel/2016b``

### EPD

The Enthought Python Distribution provides scientists with a comprehensive set of tools to perform  rigorous data analysis and visualization. Python, distinguished by its flexibility, coherence, and ease-of-use,  is rapidly becoming the programming language of choice for researchers worldwide.  EPD extends this capacity with a powerful collection of Python libraries to enable interactive technical computing and  cross-platform rapid application development.

*homepage*: <http://www.enthought.com/products/edudownload.php>

version      |toolchain
-------------|----------
``7.3-2-rh5``|``system``

### EPIC

Package implementing EPIC method to estimate the proportion of immune, stromal, endothelial  and cancer or other cells from bulk gene expression data.

*homepage*: <https://gfellerlab.shinyapps.io/EPIC_1-1>

version|versionsuffix|toolchain
-------|-------------|--------------
``1.1``|``-R-3.5.1`` |``foss/2018b``

### epiScanpy

EpiScanpy is a toolkit to analyse single-cell open chromatin (scATAC-seq) and single-cell DNA methylation (for example scBS-seq) data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq analysis tool Scanpy (Genome Biology, 2018) [Wolf18].

*homepage*: <https://github.com/colomemaria/episcanpy>

version  |toolchain
---------|--------------
``0.3.1``|``foss/2021a``
``0.4.0``|``foss/2022a``

### eQuilibrator

Calculation of standard thermodynamic potentials of biochemical reactions.

*homepage*: <https://equilibrator.weizmann.ac.il/>

version  |toolchain
---------|--------------
``0.4.7``|``foss/2021a``

### EricScript

EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data.

*homepage*: <https://sites.google.com/site/bioericscript/home>

version  |versionsuffix|toolchain
---------|-------------|---------------
``0.5.5``|``-R-3.4.0`` |``intel/2017a``

### ESL-Bundle

The ESL Bundle is a collection of libraries and utilities broadly used in  electronic structure calculations, put together to make their use easier by  researchers and scientific software developers. ESL stands for Electronic  Structure Library, an initiative which distributes quality software and  promotes open standards for high-performance computing applications in the  field of electronic structure calculations. 

*homepage*: <https://esl.cecam.org>

version  |toolchain
---------|-------------------------------
``0.6.1``|``foss/2020b``, ``intel/2020b``

### ESM-2

ESM-2 outperforms all tested single-sequence protein language models  across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate  accurate structure predictions end to end directly from the sequence of a protein.

*homepage*: <https://github.com/facebookresearch/esm>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``2.0.0``|                |``foss/2021a``
``2.0.0``|``-CUDA-11.3.1``|``foss/2021a``

### ESMF

The Earth System Modeling Framework (ESMF) is a suite of software tools for developing  high-performance, multi-component Earth science modeling applications.

*homepage*: <https://www.earthsystemcog.org/projects/esmf/>

version     |versionsuffix   |toolchain
------------|----------------|---------------------------------------------------------------------------------
``6.3.0rp1``|                |``intel/2017a``
``6.3.0rp1``|``-HDF5-1.8.18``|``intel/2017a``
``7.0.0``   |                |``foss/2016a``
``7.0.2``   |                |``intel/2017b``
``7.1.0r``  |                |``foss/2018b``, ``foss/2019a``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``8.0.0``   |                |``foss/2019b``, ``intel/2019b``
``8.0.1``   |                |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``8.1.1``   |                |``foss/2021a``, ``intel/2021a``
``8.2.0``   |                |``foss/2021b``, ``intel/2021b``
``8.3.0``   |                |``foss/2022a``, ``intel/2022a``

### ESMPy

Earth System Modeling Framework (ESMF) Python Interface

*homepage*: <https://earthsystemmodeling.org/esmpy>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``8.0.0``|``-Python-3.7.4``|``intel/2019b``
``8.0.1``|                 |``intel/2020b``
``8.0.1``|``-Python-3.8.2``|``foss/2020a``

### ESMValTool

The Earth System Model eValuation Tool (ESMValTool)  is a community diagnostics and performance metrics tool  for the evaluation of Earth System Models (ESMs) that  allows for routine comparison of single or multiple models,  either against predecessor versions or against observations.

*homepage*: <https://www.esmvaltool.org/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.0``|``-Python-2.7.14``|``intel/2017b``

### eSpeak-NG

The eSpeak NG is a compact open source software text-to-speech synthesizer for Linux, Windows, Android and other operating systems. It supports more than 100 languages and accents. It is based on the eSpeak engine created by Jonathan Duddington. 

*homepage*: <https://github.com/espeak-ng/espeak-ng>

version |toolchain
--------|---------------
``1.50``|``gompi/2020a``

### ESPResSo

A software package for performing and analyzing scientific Molecular Dynamics simulations.

*homepage*: <https://espressomd.org/wordpress>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``4.2.1``|``-CUDA-11.3.1``|``foss/2021a``

### Essentia

Open-source library and tools for audio and music analysis, description and synthesis

*homepage*: <https://essentia.upf.edu>

version      |versionsuffix     |toolchain
-------------|------------------|--------------
``2.1_beta5``|``-Python-2.7.15``|``foss/2019a``

### ETE

A Python framework for the analysis and visualization of trees

*homepage*: <http://etetoolkit.org>

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------------------
``3.0.0b36``|``-Python-2.7.12``|``intel/2016b``
``3.1.1``   |``-Python-3.6.6`` |``foss/2018b``
``3.1.2``   |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``
``3.1.2``   |``-Python-3.7.4`` |``foss/2019b``
``3.1.2``   |``-Python-3.8.2`` |``foss/2020a``

### ETSF_IO

A library of F90 routines to read/write the ETSF file  format has been written. It is called ETSF_IO and available under LGPL. 

*homepage*: <http://www.etsf.eu/resources/software/libraries_and_tools>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.0.4``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

### eudev

eudev is a fork of systemd-udev with the goal of obtaining better  compatibility with existing software such as OpenRC and Upstart,  older kernels, various toolchains and anything else required by  users and various distributions. 

*homepage*: <https://wiki.gentoo.org/wiki/Project:Eudev>

version  |toolchain
---------|-------------------------------------------------
``3.1.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``3.2``  |``GCCcore/4.9.3``
``3.2.2``|``GCCcore/6.4.0``

### Evcxr-REPL

A Rust REPL (Read-Eval-Print loop) built using the evcxr evaluation context.

*homepage*: <https://github.com/evcxr>

version   |versionsuffix   |toolchain
----------|----------------|------------------
``0.14.2``|``-Rust-1.65.0``|``GCCcore/12.2.0``

### EvidentialGene

EvidentialGene is a genome informatics project for  "Evidence Directed Gene Construction for Eukaryotes",  for constructing high quality, accurate gene sets for  animals and plants (any eukaryotes), being developed by  Don Gilbert at Indiana University, gilbertd at indiana edu.

*homepage*: <http://arthropods.eugenes.org/EvidentialGene/>

version       |versionsuffix   |toolchain
--------------|----------------|---------------
``2018.01.01``|``-Perl-5.24.1``|``intel/2017a``
``2022.01.14``|                |``gompi/2021b``

### evmix

evmix: Extreme Value Mixture Modelling,  Threshold Estimation and Boundary Corrected Kernel Density Estimation

*homepage*: <https://cran.r-project.org/web/packages/evmix>

version|versionsuffix|toolchain
-------|-------------|---------------
``2.6``|``-R-3.3.1`` |``intel/2016b``

### ExaBayes

ExaBayes is a software package for Bayesian tree inference.  It is particularly suitable for large-scale analyses on computer clusters

*homepage*: <https://cme.h-its.org/exelixis/web/software/exabayes/>

version|toolchain
-------|--------------
``1.5``|``foss/2016b``

### ExaML

Exascale Maximum Likelihood (ExaML) code for phylogenetic inference using MPI

*homepage*: <https://github.com/stamatak/ExaML/>

version   |versionsuffix  |toolchain
----------|---------------|---------------
``3.0.22``|``-hybrid-avx``|``gompi/2021a``

### Excel-Writer-XLSX

The Excel::Writer::XLSX module can be used to create an Excel file in the 2007+ XLSX format. Multiple worksheets can be added to a workbook and formatting can be applied to cells. Text, numbers, and formulas can be written to the cells.

*homepage*: <https://metacpan.org/pod/Excel::Writer::XLSX>

version |toolchain
--------|--------------
``1.09``|``foss/2020b``

### ExifTool

Perl module (Image::ExifTool) and program (exiftool)  to read EXIF information from images

*homepage*: <https://owl.phy.queensu.ca/~phil/exiftool/>

version  |toolchain
---------|-----------------
``12.00``|``GCCcore/9.3.0``

### exiv2

Exiv2 is a Cross-platform C++ library and a command line utility to manage image metadata.

*homepage*: <https://exiv2.org>

version   |toolchain
----------|------------------
``0.27.4``|``GCCcore/10.3.0``
``0.27.5``|``GCCcore/11.2.0``

### Exonerate

Exonerate is a generic tool for pairwise sequence comparison.  It allows you to align sequences using a many alignment models, using either   exhaustive dynamic programming, or a variety of heuristics. 

*homepage*: <https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``intel/2017a``

### expat

Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags).

*homepage*: <https://libexpat.github.io>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------
``2.1.0``|``GCC/4.9.2``, ``foss/2016a``, ``intel/2016a``
``2.1.1``|``foss/2016a``, ``intel/2016a``
``2.2.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.2.4``|``GCCcore/6.4.0``
``2.2.5``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.2.6``|``GCCcore/8.2.0``
``2.2.7``|``GCCcore/8.3.0``
``2.2.9``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``2.4.1``|``GCCcore/11.2.0``
``2.4.8``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``
``2.4.9``|``GCCcore/12.2.0``
``2.5.0``|``GCCcore/12.3.0``

### expect

Expect is a tool for automating interactive applications    such as telnet, ftp, passwd, fsck, rlogin, tip, etc.    Expect really makes this stuff trivial.    Expect is also useful for testing these same applications.

*homepage*: <https://core.tcl.tk/expect/index>

version   |toolchain
----------|------------------------------------
``5.45.4``|``GCCcore/7.3.0``, ``GCCcore/9.3.0``

### expecttest

This library implements expect tests (also known as "golden" tests). Expect tests are a method of  writing tests where instead of hard-coding the expected output of a test, you run the test to get the output, and  the test framework automatically populates the expected output. If the output of the test changes, you can rerun  the test with the environment variable EXPECTTEST_ACCEPT=1 to automatically update the expected output.

*homepage*: <https://github.com/ezyang/expecttest>

version  |toolchain
---------|------------------------------------------------------------------------------
``0.1.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### eXpress

Streaming quantification for high-throughput sequencing

*homepage*: <https://pachterlab.github.io/eXpress>

version  |toolchain
---------|---------------
``1.5.1``|``intel/2017b``

### Extrae

Extrae is the core instrumentation package developed by the Performance Tools group at BSC. Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches. The information gathered by Extrae typically includes timestamped events of runtime calls, performance counters and source code references. Besides, Extrae provides its own API to allow the user to manually instrument his or her application.

*homepage*: <https://www.bsc.es/computer-sciences/performance-tools>

version  |toolchain
---------|---------------
``3.4.1``|``foss/2017a``
``3.7.1``|``intel/2019a``
``3.8.0``|``gompi/2020b``
``3.8.3``|``gompi/2021a``
``4.0.4``|``gompi/2022a``

### EZC3D

EZC3D is an simple to use reader, modifier and writer for C3D format files. It is written en C++ with proper binders for Python and MATLAB/Octave scripting langages.

*homepage*: <https://pyomeca.github.io/Documentation/ezc3d/index.html>

version  |toolchain
---------|--------------
``1.5.2``|``foss/2022a``

## F


[f90cache](#f90cache) - [f90wrap](#f90wrap) - [Faber](#faber) - [FabIO](#fabio) - [faceswap](#faceswap) - [FALCON](#falcon) - [FANN](#fann) - [fast5](#fast5) - [FASTA](#fasta) - [fastahack](#fastahack) - [fastai](#fastai) - [FastaIndex](#fastaindex) - [FastANI](#fastani) - [Fastaq](#fastaq) - [FastFold](#fastfold) - [fastjet](#fastjet) - [fastjet-contrib](#fastjet-contrib) - [FastME](#fastme) - [fastp](#fastp) - [fastparquet](#fastparquet) - [fastPHASE](#fastphase) - [fastq-pair](#fastq-pair) - [fastq-tools](#fastq-tools) - [FastQ_Screen](#fastq_screen) - [FastQC](#fastqc) - [fastqsplitter](#fastqsplitter) - [FastQTL](#fastqtl) - [fastqz](#fastqz) - [FastRFS](#fastrfs) - [fastStructure](#faststructure) - [FastTree](#fasttree) - [FastViromeExplorer](#fastviromeexplorer) - [FASTX-Toolkit](#fastx-toolkit) - [fatslim](#fatslim) - [fbm](#fbm) - [FBPIC](#fbpic) - [FCC](#fcc) - [FCM](#fcm) - [fdict](#fdict) - [FDS](#fds) - [fdstools](#fdstools) - [FDTD_Solutions](#fdtd_solutions) - [feh](#feh) - [FEniCS](#fenics) - [fermi-lite](#fermi-lite) - [Ferret](#ferret) - [festival](#festival) - [fetchMG](#fetchmg) - [FFC](#ffc) - [FFLAS-FFPACK](#fflas-ffpack) - [FFmpeg](#ffmpeg) - [ffmpi](#ffmpi) - [ffnet](#ffnet) - [ffnvcodec](#ffnvcodec) - [fftlib](#fftlib) - [FFTW](#fftw) - [FFTW.MPI](#fftw.mpi) - [fgbio](#fgbio) - [FGSL](#fgsl) - [FHI-aims](#fhi-aims) - [FIAT](#fiat) - [FIGARO](#figaro) - [FigureGen](#figuregen) - [Fiji](#fiji) - [file](#file) - [filevercmp](#filevercmp) - [Filtlong](#filtlong) - [find_circ](#find_circ) - [finder](#finder) - [findhap](#findhap) - [findutils](#findutils) - [fineRADstructure](#fineradstructure) - [fineSTRUCTURE](#finestructure) - [fio](#fio) - [Fiona](#fiona) - [Firefox](#firefox) - [FIRESTARTER](#firestarter) - [FireWorks](#fireworks) - [FIt-SNE](#fit-sne) - [FIX](#fix) - [fixesproto](#fixesproto) - [FLAC](#flac) - [FLAIR](#flair) - [flair-NLP](#flair-nlp) - [FLANN](#flann) - [FLASH](#flash) - [Flask](#flask) - [flatbuffers](#flatbuffers) - [flatbuffers-python](#flatbuffers-python) - [FLEUR](#fleur) - [flex](#flex) - [Flexbar](#flexbar) - [FlexiBLAS](#flexiblas) - [FlexiDot](#flexidot) - [Flink](#flink) - [FLINT](#flint) - [flook](#flook) - [flowFDA](#flowfda) - [FLTK](#fltk) - [FLUENT](#fluent) - [Flye](#flye) - [FMILibrary](#fmilibrary) - [FMM3D](#fmm3d) - [FMPy](#fmpy) - [FMRIprep](#fmriprep) - [FMS](#fms) - [fmt](#fmt) - [FoBiS](#fobis) - [FoldX](#foldx) - [fontconfig](#fontconfig) - [fontsproto](#fontsproto) - [FORD](#ford) - [foss](#foss) - [fosscuda](#fosscuda) - [FoX](#fox) - [FOX-Toolkit](#fox-toolkit) - [FPM](#fpm) - [fpocket](#fpocket) - [fqtrim](#fqtrim) - [fqzcomp](#fqzcomp) - [FragGeneScan](#fraggenescan) - [FragPipe](#fragpipe) - [FRANz](#franz) - [FreeBarcodes](#freebarcodes) - [freebayes](#freebayes) - [FreeFEM](#freefem) - [FreeFem++](#freefem++) - [freeglut](#freeglut) - [FreeImage](#freeimage) - [FreeSASA](#freesasa) - [FreeSurfer](#freesurfer) - [FreeTDS](#freetds) - [freetype](#freetype) - [freetype-py](#freetype-py) - [FreeXL](#freexl) - [freud-analysis](#freud-analysis) - [FriBidi](#fribidi) - [FRUIT](#fruit) - [FRUIT_processor](#fruit_processor) - [FSL](#fsl) - [FSLeyes](#fsleyes) - [fsom](#fsom) - [FSON](#fson) - [FTGL](#ftgl) - [Fujitsu](#fujitsu) - [fullrmc](#fullrmc) - [fumi_tools](#fumi_tools) - [funannotate](#funannotate) - [FunGAP](#fungap) - [FUNWAVE-TVD](#funwave-tvd) - [FUSE](#fuse) - [FuSeq](#fuseq) - [FusionCatcher](#fusioncatcher) - [futhark](#futhark) - [futile](#futile) - [future](#future) - [fxtract](#fxtract)


### f90cache

f90cache is a compiler cache. It acts as a caching pre-processor to Fortran compilers,  using the -E compiler switch and a hash to detect when a compilation can be satisfied from cache.  This often results in a great speedup in common compilations.

*homepage*: <http://people.irisa.fr/Edouard.Canot/f90cache/>

version |toolchain
--------|----------
``0.96``|``system``

### f90wrap

f90wrap is a tool to automatically generate Python extension modules which interface to Fortran code that makes use of derived types. It builds on the capabilities of the popular f2py utility by generating a simpler Fortran 90 interface to the original Fortran code which is then suitable for wrapping with f2py, together with a higher-level Pythonic wrapper that makes the existance of an additional layer transparent to the final user.

*homepage*: <https://github.com/jameskermode/f90wrap>

version  |toolchain
---------|--------------
``0.2.8``|``foss/2021a``

### Faber

Faber started as a clone of Boost.Build, to experiment with a  new Python frontend. Meanwhile it has evolved into a new build system, which  retains most of the features found in Boost.Build, but with (hopefully !) much  simplified logic, in addition of course to using Python as scripting language,  rather than Jam. The original bjam engine is still in use as scheduler, though  at this point that is mostly an implementation detail.

*homepage*: <https://stefanseefeld.github.io/faber>

version|toolchain
-------|-----------------
``0.3``|``GCCcore/8.3.0``

### FabIO

FabIO is an I/O library for images produced by 2D X-ray detectors and written in Python.  FabIO support images detectors from a dozen of companies (including Mar, Dectris, ADSC, Hamamatsu, Oxford, ...),  for a total of 20 different file formats (like CBF, EDF, TIFF, ...) and offers an unified interface to their headers  (as a python dictionary) and datasets (as a numpy ndarray of integers or floats).

*homepage*: <http://www.silx.org>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.10.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.11.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.14.0``|                 |``foss/2021b``

### faceswap

Faceswap is a tool that utilizes deep learning to recognize and swap faces in pictures and videos.

*homepage*: <https://github.com/deepfakes/faceswap>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180212``|``-Python-3.6.3``|``foss/2017b``

### FALCON

Falcon: a set of tools for fast aligning long reads for consensus and assembly

*homepage*: <https://github.com/PacificBiosciences/FALCON>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.8.8``|                  |``intel/2017b``
``1.8.8``|``-Python-2.7.16``|``intel/2019b``

### FANN

Fast Artificial Neural Network Library is a free open source neural network library,  which implements multilayer artificial neural networks in C with support for both fully connected  and sparsely connected networks.

*homepage*: <http://leenissen.dk>

version  |toolchain
---------|----------------------------------
``2.2.0``|``GCCcore/6.4.0``, ``intel/2018a``

### fast5

A lightweight C++ library for accessing Oxford Nanopore  Technologies sequencing data.

*homepage*: <http://simpsonlab.github.io/2017/02/27/packing_fast5/>

version  |toolchain
---------|----------
``0.6.5``|``system``

### FASTA

The FASTA programs find regions of local or global (new) similarity between  protein or DNA sequences, either by searching Protein or DNA databases, or by identifying  local duplications within a sequence.

*homepage*: <https://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml>

version    |toolchain
-----------|------------------------------
``36.3.5e``|``foss/2016b``
``36.3.8i``|``GCC/11.2.0``, ``GCC/12.2.0``

### fastahack

Utilities for indexing and sequence extraction from FASTA files.

*homepage*: <https://github.com/ekg/fastahack>

version  |toolchain
---------|-----------------------------------------------------------------------------
``1.0.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### fastai

The fastai deep learning library.

*homepage*: <https://www.fast.ai/>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``2.7.10``|                |``foss/2022a``
``2.7.10``|``-CUDA-11.7.0``|``foss/2022a``

### FastaIndex

FastA index (.fai) handler compatible with samtools faidx

*homepage*: <https://github.com/lpryszcz/FastaIndex>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.11rc7``|``-Python-2.7.14``|``intel/2017b``

### FastANI

FastANI is developed for fast alignment-free computation of  whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean  nucleotide identity of orthologous gene pairs shared between two microbial  genomes. FastANI supports pairwise comparison of both complete and draft  genome assemblies.

*homepage*: <https://github.com/ParBLiSS/FastANI>

version |toolchain
--------|---------------------------------------------------------------------------------------------------------------------
``1.1`` |``foss/2018b``, ``intel/2018b``
``1.2`` |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``1.3`` |``iccifort/2019.5.281``
``1.31``|``iccifort/2020.1.217``
``1.33``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.4.0``

### Fastaq

Python3 scripts to manipulate FASTA and FASTQ files.

*homepage*: <https://github.com/sanger-pathogens/Fastaq>

version   |toolchain
----------|--------------
``3.17.0``|``GCC/10.3.0``

### FastFold

Optimizing Protein Structure Prediction Model Training and Inference on GPU Clusters

*homepage*: <https://github.com/hpcaitech/FastFold>

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20220729``|``-CUDA-11.3.1``|``foss/2021a``

### fastjet

A software package for jet finding in pp and e+e- collisions

*homepage*: <https://fastjet.fr/>

version  |toolchain
---------|---------------
``3.4.0``|``gompi/2022a``

### fastjet-contrib

3rd party extensions of FastJet

*homepage*: <https://fastjet.hepforge.org/contrib/>

version  |toolchain
---------|---------------
``1.049``|``gompi/2022a``

### FastME

FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.

*homepage*: <http://www.atgc-montpellier.fr/fastme/>

version    |toolchain
-----------|----------------------------------------------------------------------------------------
``2.1.5``  |``foss/2016a``
``2.1.6.1``|``GCC/10.2.0``, ``GCC/8.3.0``, ``iccifort/2019.5.281``, ``intel/2018a``, ``intel/2018b``

### fastp

A tool designed to provide fast all-in-one preprocessing for FastQ files.  This tool is developed in C++ with multithreading supported to afford high performance.

*homepage*: <https://github.com/OpenGene/fastp>

version   |toolchain
----------|------------------------------------------------------------
``0.19.7``|``foss/2018b``
``0.20.0``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.20.1``|``iccifort/2020.1.217``
``0.23.2``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``
``0.23.4``|``GCC/12.2.0``

### fastparquet

fastparquet is a python implementation of the parquet format, aiming to integrate into python-based big data work-flows. It is used implicitly by the projects Dask, Pandas and intake-parquet.

*homepage*: <https://fastparquet.readthedocs.io/>

version  |toolchain
---------|--------------
``0.7.2``|``foss/2021a``
``0.8.0``|``foss/2021b``

### fastPHASE

fastPHASE: software for haplotype reconstruction,  and estimating missing genotypes from population data Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf

*homepage*: <http://stephenslab.uchicago.edu/software.html#fastphase>

version  |toolchain
---------|----------
``1.4.8``|``system``

### fastq-pair

Match up paired end fastq files quickly and efficiently.

*homepage*: <https://github.com/linsalrob/fastq-pair>

version|toolchain
-------|-----------------
``1.0``|``GCCcore/8.2.0``

### fastq-tools

This package provides a number of small and efficient programs to perform  common tasks with high throughput sequencing data in the FASTQ format. All of the programs  work with typical FASTQ files as well as gzipped FASTQ files.

*homepage*: <https://homes.cs.washington.edu/~dcjones/fastq-tools/>

version  |toolchain
---------|------------------------------
``0.8``  |``foss/2016b``, ``foss/2018b``
``0.8.3``|``GCC/10.3.0``, ``GCC/11.2.0``

### FastQ_Screen

FastQ Screen allows you to screen a library of sequences in FastQ  format against a set of sequence databases so you can see if the composition of the  library matches with what you expect.

*homepage*: <http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/>

version   |versionsuffix   |toolchain
----------|----------------|---------------
``0.11.3``|``-Perl-5.24.0``|``foss/2016b``
``0.11.4``|``-Perl-5.24.0``|``foss/2016b``
``0.12.0``|``-Perl-5.26.1``|``intel/2018a``
``0.13.0``|``-Perl-5.28.0``|``foss/2018b``
``0.14.0``|                |``GCC/11.3.0``

### FastQC

FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.

*homepage*: <https://www.bioinformatics.babraham.ac.uk/projects/fastqc/>

version   |versionsuffix      |toolchain
----------|-------------------|----------
``0.10.1``|``-Java-1.7.0_80`` |``system``
``0.11.2``|``-Java-1.7.0_60`` |``system``
``0.11.3``|``-Java-1.7.0_80`` |``system``
``0.11.4``|``-Java-1.8.0_66`` |``system``
``0.11.4``|``-Java-1.8.0_74`` |``system``
``0.11.5``|``-Java-1.7.0_80`` |``system``
``0.11.5``|``-Java-1.8.0_144``|``system``
``0.11.5``|``-Java-1.8.0_74`` |``system``
``0.11.7``|``-Java-1.8.0_162``|``system``
``0.11.8``|``-Java-1.8``      |``system``
``0.11.8``|``-Java-11``       |``system``
``0.11.9``|``-Java-1.8``      |``system``
``0.11.9``|``-Java-11``       |``system``

### fastqsplitter

Splits fastq files evenly.

*homepage*: <https://github.com/LUMC/fastqsplitter>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.2.0``|``-Python-3.7.2``|``GCCcore/8.2.0``

### FastQTL

FastQTL is a QTL mapper

*homepage*: <http://fastqtl.sourceforge.net/>

version  |toolchain
---------|------------------------------
``2.184``|``GCC/11.2.0``, ``foss/2018b``

### fastqz

fastqz is a compressor for FASTQ files. FASTQ is the output of DNA sequencing machines.  It is one of the compressors described in the paper: Bonfield JK, Mahoney MV (2013) Compression of  FASTQ and SAM Format Sequencing Data. (mirror) PLoS ONE 8(3): e59190. doi:10.1371/journal.pone.0059190

*homepage*: <http://mattmahoney.net/dc/fastqz/>

version|toolchain
-------|-------------
``1.5``|``GCC/4.8.2``

### FastRFS

Fast Robinson Foulds Supertrees

*homepage*: <https://github.com/pranjalv123/FastRFS>

version         |toolchain
----------------|---------------
``1.0-20190613``|``gompi/2019a``

### fastStructure

fastStructure is a fast algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x. 

*homepage*: <https://rajanil.github.io/fastStructure/>

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.0``|``-Python-2.7.11``|``foss/2016a``
``1.0``|``-Python-2.7.12``|``foss/2016b``
``1.0``|``-Python-2.7.13``|``foss/2017a``
``1.0``|``-Python-2.7.15``|``foss/2019a``

### FastTree

FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. 

*homepage*: <http://www.microbesonline.org/fasttree/>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------
``2.1.10``|``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``2.1.11``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### FastViromeExplorer

Identify the viruses/phages and their abundance in the viral metagenomics data.

*homepage*: <https://code.vt.edu/saima5/FastViromeExplorer>

version     |toolchain
------------|--------------
``20180422``|``foss/2019b``

### FASTX-Toolkit

The FASTX-Toolkit is a collection of command line tools for   Short-Reads FASTA/FASTQ files preprocessing.

*homepage*: <http://hannonlab.cshl.edu/fastx_toolkit/>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------
``0.0.14``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/9.3.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``

### fatslim

FATSLiM stands for “Fast Analysis Toolbox for Simulations of Lipid Membranes” and its goal is to provide an efficient, yet robust, tool to extract physical parameters from MD trajectories.

*homepage*: <https://github.com/FATSLiM/fatslim>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.1``|``-Python-3.6.4``|``foss/2018a``

### fbm

Exact methods for simulating fractional Brownian motion and fractional Gaussian noise in Python

*homepage*: <https://pypi.org/project/fbm>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.0``|``-Python-3.6.4``|``intel/2018a``

### FBPIC

FBPIC (Fourier-Bessel Particle-In-Cell) is a Particle-In-Cell (PIC) code for relativistic plasma  physics. It is especially well-suited for physical simulations of laser-wakefield acceleration and  plasma-wakefield acceleration.

*homepage*: <https://fbpic.github.io/>

version   |toolchain
----------|------------------
``0.20.3``|``fosscuda/2020b``

### FCC

Fujitsu Compiler based compiler toolchain.

*homepage*: <(none)>

version  |toolchain
---------|----------
``4.5.0``|``system``

### FCM

FCM is a set of tools for managing and building source code.

*homepage*: <http://www.metoffice.gov.uk/research/collaboration/fcm>

version  |toolchain
---------|----------
``2.3.1``|``system``

### fdict

A variable and dictionary in pure fortran for retaining any data-type and a fast hash-table dictionary. 

*homepage*: <https://github.com/zerothi/fdict>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------
``0.8.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

### FDS

Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.

*homepage*: <https://pages.nist.gov/fds-smv>

version  |versionsuffix|toolchain
---------|-------------|--------------------------------
``6.0.1``|``-no-OFED`` |``system``
``6.5.2``|             |``intel/2016b``
``6.5.3``|             |``intel/2017a``
``6.6.0``|             |``intel/2017b``, ``intel/2018a``
``6.7.0``|             |``intel/2018a``
``6.7.4``|             |``intel/2020a``
``6.7.5``|             |``intel/2020a``
``6.7.6``|             |``intel/2020b``
``6.7.7``|             |``intel/2021b``
``6.7.9``|             |``intel/2022a``

### fdstools

Forensic DNA Sequencing Tools Tools for characterisation and filtering of PCR stutter artefacts  and other systemic noise in Next Generation Sequencing data of forensic STR markers.

*homepage*: <https://git.lumc.nl/jerryhoogenboom/fdstools>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160322``|``-Python-2.7.11``|``foss/2016a``

### FDTD_Solutions

High performance FDTD-method Maxwell solver for the design, analysis and optimization of nanophotonic devices, processes and materials.

*homepage*: <http://www.lumerical.com/tcad-products/fdtd/>

version      |toolchain
-------------|----------
``8.6.2``    |``system``
``8.11.337`` |``system``
``8.16.982`` |``system``
``8.20.1731``|``system``

### feh

feh is an X11 image viewer aimed mostly at console users. Unlike most other viewers, it does not have a fancy GUI, but simply displays images. It is controlled via commandline arguments and configurable key/mouse actions. 

*homepage*: <https://feh.finalrewind.org/>

version |toolchain
--------|-----------------
``2.26``|``GCCcore/6.4.0``

### FEniCS

FEniCS is a computing platform for solving partial differential equations (PDEs).

*homepage*: <https://fenicsproject.org/>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

### fermi-lite

Standalone C library for assembling Illumina short reads in small regions.

*homepage*: <https://github.com/lh3/fermi-lite>

version     |toolchain
------------|-----------------------------------------------------------------------------
``20190320``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### Ferret

Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets.

*homepage*: <https://ferret.pmel.noaa.gov/Ferret/>

version  |toolchain
---------|---------------
``7.3``  |``intel/2017b``
``7.5.0``|``foss/2019b``

### festival

University of Edinburgh's Festival Speech Synthesis Systems is a free software multi-lingual speech synthesis workbench that runs on multiple-platforms offering black box text to speech, as well as an open architecture for research in speech synthesis. It designed as a component of large speech technology systems. 

*homepage*: <['http://festvox.org/festival/']>

version  |toolchain
---------|-----------------
``2.5.0``|``GCCcore/9.3.0``

### fetchMG

The program “fetchMG” was written to extract the 40 MGs from genomes and metagenomes  in an simple and accurate manner.

*homepage*: <http://vm-lux.embl.de/~mende/fetchMG/about.html>

version|toolchain
-------|------------------------------------
``1.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### FFC

The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms.

*homepage*: <https://bitbucket.org/fenics-project/ffc>

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``2018.1.0``      |``-Python-3.6.4``|``foss/2018a``
``2019.1.0.post0``|``-Python-3.7.4``|``foss/2019b``

### FFLAS-FFPACK

Finite Field Linear Algebra Subroutines / Package

*homepage*: <https://linbox-team.github.io/fflas-ffpack/>

version  |toolchain
---------|--------------
``2.2.0``|``foss/2016a``

### FFmpeg

A complete, cross-platform solution to record, convert and stream audio and video.

*homepage*: <https://www.ffmpeg.org/>

version    |toolchain
-----------|---------------------------------------------------
``0.10.16``|``gimkl/2.11.5``, ``intel/2016a``
``2.8.6``  |``intel/2016a``
``2.8.7``  |``foss/2016a``, ``intel/2016a``
``3.0.2``  |``foss/2016a``, ``intel/2016a``
``3.1.3``  |``foss/2016b``, ``intel/2016b``
``3.2.4``  |``gimkl/2017a``
``3.3.1``  |``foss/2016b``
``3.3.4``  |``intel/2017a``
``3.4``    |``GCCcore/6.4.0``
``3.4.1``  |``foss/2017b``, ``intel/2017b``
``3.4.2``  |``foss/2018a``, ``intel/2018a``
``3.4.5``  |``foss/2018b``
``4.0``    |``foss/2018a``, ``intel/2018a``
``4.0.1``  |``intel/2018a``
``4.1``    |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``4.1.3``  |``GCCcore/8.2.0``
``4.2.1``  |``GCCcore/8.3.0``
``4.2.2``  |``GCCcore/9.3.0``
``4.3.1``  |``GCCcore/10.2.0``
``4.3.2``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``4.4.2``  |``GCCcore/11.3.0``
``5.0.1``  |``GCCcore/11.3.0``
``5.1.2``  |``GCCcore/12.2.0``

### ffmpi

Fujitsu Compiler based compiler toolchain, including Fujitsu MPI for MPI support.

*homepage*: <(none)>

version  |toolchain
---------|----------
``4.5.0``|``system``

### ffnet

Feed-forward neural network solution for python

*homepage*: <http://ffnet.sourceforge.net/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.3``|``-Python-2.7.11``|``intel/2016a``

### ffnvcodec

FFmpeg nvidia headers. Adds support for nvenc and nvdec. Requires Nvidia GPU and drivers to be present (picked up dynamically).

*homepage*: <https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git>

version     |toolchain
------------|----------
``11.1.5.2``|``system``

### fftlib

A library that intercepts FFTW calls and adds features on top of it.  In particular, it enables FFT plan reuse when there are multiple calls for the same geometry.

*homepage*: <https://github.com/flwende/fftlib>

version     |toolchain
------------|---------------
``20170628``|``gompi/2020b``

### FFTW

FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.

*homepage*: <https://www.fftw.org>

version   |versionsuffix|toolchain
----------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.0`` |``-fujitsu`` |``FCC/4.5.0``
``2.1.5`` |             |``intel/2016b``, ``intel/2017a``, ``intel/2018b``
``3.3.4`` |             |``gmpich/2016a``, ``gmvapich2/1.7.20``, ``gmvapich2/2016a``, ``gompi/2016.04``, ``gompi/2016.06``, ``gompi/2016.07``, ``gompi/2016a``, ``gompi/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.5`` |             |``gompi/2016.07``, ``gompi/2016.09``, ``gompi/2016b``, ``intel/2016b``
``3.3.6`` |             |``gimpi/2017b``, ``gimpic/2017b``, ``gompi/2017a``, ``gompi/2017b``, ``gompic/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``3.3.7`` |             |``gimkl/2017a``, ``gimpi/2018a``, ``gmpich/2017.08``, ``gompi/2018a``, ``gompic/2018a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``iomkl/2018a``
``3.3.7`` |``-serial``  |``GCC/6.4.0-2.28``
``3.3.8`` |             |``gompi/2018.08``, ``gompi/2018b``, ``gompi/2019a``, ``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompic/2018b``, ``gompic/2019a``, ``gompic/2019b``, ``gompic/2020a``, ``gompic/2020b``, ``iimpi/2020b``, ``intel/2018b``, ``intel/2019a``, ``intel/2019b``, ``intel/2020a``, ``intel/2020b``, ``iomkl/2018b``, ``iomkl/2020b``
``3.3.8`` |``-amd``     |``gompi/2020a``
``3.3.8`` |``-serial``  |``GCC/9.3.0``
``3.3.9`` |             |``gompi/2021a``, ``intel/2021a``
``3.3.10``|             |``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/12.3.0``, ``NVHPC/22.7-CUDA-11.7.0``, ``gompi/2021b``, ``iimpi/2021b``, ``iimpi/2022a``, ``iimpi/2022b``

### FFTW.MPI

FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data.

*homepage*: <https://www.fftw.org>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------
``3.3.10``|``gompi/2022.05``, ``gompi/2022.10``, ``gompi/2022a``, ``gompi/2022b``, ``gompi/2023a``, ``nvompi/2022.07``

### fgbio

A set of tools to analyze genomic data with a focus on Next Generation Sequencing.

*homepage*: <https://fulcrumgenomics.github.io/fgbio>

version  |toolchain
---------|----------
``1.3.0``|``system``

### FGSL

FGSL: A Fortran interface to the GNU Scientific Library

*homepage*: <https://www.lrz.de/services/software/mathematik/gsl/fortran/>

version  |toolchain
---------|---------------
``1.1.0``|``intel/2016b``

### FHI-aims

FHI-aims is an efficient, accurate all-electron, full-potential electronic structure code package for computational molecular and materials science (non-periodic and periodic systems). The code supports DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is particularly efficient for molecular systems and nanostructures, while maintaining high numerical accuracy for all production tasks. Production calculations handle up to several thousand atoms and can efficiently use (ten) thousands of cores.  

*homepage*: <https://fhi-aims.org/>

version     |toolchain
------------|---------------
``200112_2``|``intel/2019b``
``221103``  |``intel/2022a``

### FIAT

The FInite element Automatic Tabulator (FIAT) supports generation of arbitrary order instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating arbitrary order instances of Jacobi-type quadrature rules on the same element shapes.

*homepage*: <https://bitbucket.org/fenics-project/fiat>

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``1.6.0``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2018.1.0``|``-Python-3.6.4`` |``foss/2018a``
``2019.1.0``|``-Python-3.7.4`` |``foss/2019b``

### FIGARO

FIGARO: An efficient and objective tool for optimizing microbiome rRNA gene trimming parameters.

*homepage*: <https://github.com/Zymo-Research/figaro>

version  |toolchain
---------|---------------
``1.1.2``|``intel/2020b``

### FigureGen

FigureGen is a Fortran program that creates images for ADCIRC files. It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output  files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors.  Using FigureGen, you can go directly from the ADCIRC input and output files to a  presentation-quality figure, for one or multiple time snaps.  

*homepage*: <https://ccht.ccee.ncsu.edu/category/software/figuregen/>

version        |toolchain
---------------|--------------
``51-20190516``|``foss/2019a``

### Fiji

Fiji is an image processing package—a 'batteries-included' distribution of  ImageJ, bundling a lot of plugins which facilitate scientific image analysis. This release is based on ImageJ-2.1.0 and Fiji-2.1.1

*homepage*: <https://fiji.sc/>

version          |versionsuffix|toolchain
-----------------|-------------|----------
``2.9.0``        |``-Java-1.8``|``system``
``2.9.0``        |``-Java-8``  |``system``
``20170530``     |             |``system``
``20191119-2057``|             |``system``
``20201104-1356``|             |``system``

### file

The file command is 'a file type guesser', that is, a command-line tool  that tells you in words what kind of data a file contains.

*homepage*: <https://www.darwinsys.com/file/>

version |toolchain
--------|------------------------------------
``5.17``|``GCC/4.8.2``
``5.25``|``intel/2016a``
``5.28``|``foss/2016b``
``5.30``|``intel/2017a``
``5.33``|``GCCcore/6.4.0``
``5.35``|``GCCcore/7.3.0``
``5.38``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.39``|``GCCcore/10.2.0``
``5.40``|``GCCcore/10.3.0``
``5.41``|``GCCcore/11.2.0``
``5.43``|``GCCcore/11.3.0``

### filevercmp

filevercmp function as in sort --version-sort.

*homepage*: <https://github.com/ekg/filevercmp>

version     |toolchain
------------|----------------------------------------------------------
``20141119``|``GCCcore/9.3.0``
``20191210``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### Filtlong

Filtlong is a tool for filtering long reads by quality. It can take a set  of long reads and produce a smaller, better subset. It uses both read length (longer is better)  and read identity (higher is better) when choosing which reads pass the filter

*homepage*: <https://github.com/rrwick/Filtlong>

version  |toolchain
---------|------------------------------
``0.2.0``|``GCC/10.2.0``, ``foss/2016b``
``0.2.1``|``GCC/10.3.0``

### find_circ

circRNA detection from RNA-seq reads

*homepage*: <https://github.com/marvin-jens/find_circ>

version         |versionsuffix     |toolchain
----------------|------------------|---------------
``1.2-20170228``|``-Python-2.7.14``|``intel/2017b``

### finder

finder is a gene annotator pipeline which automates the process of downloading short reads, aligning them and using the assembled transcripts to generate gene annotations.

*homepage*: <https://github.com/sagnikbanerjee15/Finder/>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2021b``

### findhap

Find haplotypes and impute genotypes using multiple chip sets and sequence data

*homepage*: <http://aipl.arsusda.gov/software/findhap/>

version|toolchain
-------|----------
``4``  |``system``

### findutils

findutils: The GNU find, locate, updatedb, and xargs utilities

*homepage*: <http://www.gnu.org/software/findutils/findutils.html>

version  |toolchain
---------|-------------
``4.4.2``|``GCC/4.8.2``

### fineRADstructure

A package for population structure inference from RAD-seq data

*homepage*: <http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html>

version     |toolchain
------------|---------------
``20180709``|``intel/2018a``

### fineSTRUCTURE

fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using  dense sequencing data.

*homepage*: <https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html>

version  |toolchain
---------|---------------
``2.1.3``|``intel/2017b``

### fio

Flexible I/O tester

*homepage*: <https://github.com/axboe/fio>

version |toolchain
--------|------------------
``3.30``|``GCCcore/10.3.0``
``3.32``|``GCCcore/11.3.0``
``3.34``|``GCCcore/12.2.0``

### Fiona

Fiona is designed to be simple and dependable. It focuses on reading and writing data in standard Python IO style and relies upon familiar Python types and protocols such as files, dictionaries, mappings, and iterators instead of classes specific to OGR. Fiona can read and write real-world data using multi-layered GIS formats and zipped virtual file systems and integrates readily with other Python GIS packages such as pyproj, Rtree, and Shapely.

*homepage*: <https://github.com/Toblerity/Fiona>

version         |versionsuffix    |toolchain
----------------|-----------------|---------------------------------------------------
``1.8.13``      |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.8.13.post1``|``-Python-3.7.2``|``foss/2019a``
``1.8.16``      |``-Python-3.8.2``|``foss/2020a``
``1.8.20``      |                 |``foss/2020b``, ``foss/2021a``, ``intel/2020b``
``1.8.21``      |                 |``foss/2021b``, ``foss/2022a``
``1.9.2``       |                 |``foss/2022b``

### Firefox

Firefox is a free, open source Web browser for Windows, Linux and Mac OS X. It is based on the  Mozilla code base and offers customization options and features such as its capability to block pop-up windows,  tabbed browsing, privacy and security measures, smart searching, and RSS live bookmarks.

*homepage*: <https://www.mozilla.org/en-US/firefox/new/>

version   |toolchain
----------|----------
``44.0.2``|``system``

### FIRESTARTER

FIRESTARTER: A Processor Stress Test Utility. FIRESTARTER maximizes the energy consumption of 64-Bit x86 processors by generating heavy load on the execution units as well as transferring data between the cores and multiple levels of the memory hierarchy.

*homepage*: <https://github.com/tud-zih-energy/FIRESTARTER/>

version|toolchain
-------|------------------------------------
``2.0``|``gcccuda/2020a``, ``gcccuda/2020b``

### FireWorks

FireWorks helps run calculation workflows, with a centralized workflow server  controlling many worker nodes.

*homepage*: <https://pypi.python.org/pypi/FireWorks>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.2``|``-Python-2.7.13``|``intel/2017a``

### FIt-SNE

t-distributed stochastic neighbor embedding (t-SNE) is widely used for  visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large  datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and  hence allowing visualization of rare cell populations. Furthermore, we implement a  heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously  visualizing the expression patterns of thousands of genes.

*homepage*: <https://github.com/KlugerLab/FIt-SNE>

version  |toolchain
---------|---------------
``1.1.0``|``gompi/2018b``

### FIX

FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad components can be removed from the 4D FMRI data.

*homepage*: <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX>

version    |versionsuffix           |toolchain
-----------|------------------------|--------------
``1.06.12``|``-Octave-Python-3.7.2``|``foss/2019a``

### fixesproto

X.org FixesProto protocol headers.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version|toolchain
-------|-------------------------------------------------
``5.0``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### FLAC

FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning that audio is compressed in FLAC without any loss in quality.

*homepage*: <https://xiph.org/flac/>

version  |toolchain
---------|----------------------------------------------------------
``1.3.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.3.4``|``GCCcore/11.3.0``
``1.4.2``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### FLAIR

FLAIR (Full-Length Alternative Isoform analysis of RNA)  for the correction, isoform definition, and alternative splicing analysis of noisy reads.  FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.

*homepage*: <https://github.com/BrooksLabUCSC/flair>

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``1.5``           |``-Python-3.7.4``|``foss/2019b``
``1.5.1-20200630``|``-Python-3.7.4``|``foss/2019b``

### flair-NLP

A very simple framework for state-of-the-art NLP

*homepage*: <https://github.com/flairNLP/flair>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.11.3``|                |``foss/2021a``
``0.11.3``|``-CUDA-11.3.1``|``foss/2021a``

### FLANN

FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces.

*homepage*: <https://github.com/mariusmuja/flann/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.8.4``|``-Python-2.7.11``|``intel/2016a``
``1.8.4``|``-Python-2.7.14``|``intel/2017b``
``1.8.4``|``-Python-3.8.2`` |``foss/2020a``
``1.9.1``|``-Python-3.8.2`` |``foss/2020a``

### FLASH

FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. 

*homepage*: <https://ccb.jhu.edu/software/FLASH/>

version   |toolchain
----------|------------------------------------------------------------------
``1.2.11``|``GCC/8.3.0``, ``foss/2016a``, ``foss/2018a``, ``foss/2018b``
``2.2.00``|``GCC/11.2.0``, ``GCCcore/12.2.0``, ``foss/2018b``, ``foss/2020b``

### Flask

Flask is a lightweight WSGI web application framework. It is designed to make getting started quick and simple, with the ability to scale up to complex applications. This module includes the Flask extensions: Flask-Cors

*homepage*: <https://www.palletsprojects.com/p/flask/>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.0.2``|``-Python-3.6.6``|``foss/2018b``
``1.1.2``|                 |``GCCcore/10.2.0``
``1.1.2``|``-Python-3.7.4``|``GCCcore/8.3.0``
``1.1.2``|``-Python-3.8.2``|``GCCcore/9.3.0``
``1.1.4``|                 |``GCCcore/10.3.0``
``2.0.2``|                 |``GCCcore/11.2.0``
``2.2.2``|                 |``GCCcore/11.3.0``
``2.2.3``|                 |``GCCcore/12.2.0``

### flatbuffers

FlatBuffers: Memory Efficient Serialization Library

*homepage*: <https://github.com/google/flatbuffers/>

version   |toolchain
----------|----------------------------------------------------------
``1.12.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.0`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.0.7`` |``GCCcore/11.3.0``
``23.1.4``|``GCCcore/12.2.0``

### flatbuffers-python

Python Flatbuffers runtime library.

*homepage*: <https://github.com/google/flatbuffers/>

version |versionsuffix    |toolchain
--------|-----------------|----------------------------------------------------------
``1.12``|                 |``GCCcore/10.2.0``
``1.12``|``-Python-3.7.4``|``GCCcore/8.3.0``
``2.0`` |                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### FLEUR

FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,  based on density-functional theory.

*homepage*: <http://www.flapw.de/>

version  |toolchain
---------|---------------
``0.26e``|``intel/2016a``

### flex

Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text. 

*homepage*: <http://flex.sourceforge.net/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.5.35``|``system``
``2.5.38``|``GCC/4.8.2``
``2.5.39``|``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``system``
``2.6.0`` |``GCC/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.6.2`` |``intel/2016b``
``2.6.3`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/7.1.0``, ``gimkl/2017a``, ``system``
``2.6.4`` |``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/11.4.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/9.4.0``, ``GCCcore/9.5.0``, ``GCCcore/system``, ``system``

### Flexbar

The program Flexbar preprocesses high-throughput sequencing data efficiently

*homepage*: <https://github.com/seqan/flexbar>

version  |toolchain
---------|--------------
``3.5.0``|``foss/2018b``

### FlexiBLAS

FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it.

*homepage*: <https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release>

version  |toolchain
---------|------------------------------------------
``3.0.4``|``GCC/10.3.0``, ``GCC/11.2.0``
``3.1.3``|``GCC/11.2.0``
``3.2.0``|``GCC/11.3.0``, ``NVHPC/22.7-CUDA-11.7.0``
``3.2.1``|``GCC/12.2.0``
``3.3.1``|``GCC/12.3.0``

### FlexiDot

Highly customizable, ambiguity-aware dotplots for visual sequence analyses 

*homepage*: <https://github.com/molbio-dresden/flexidot>

version |versionsuffix     |toolchain
--------|------------------|--------------
``1.06``|``-Python-2.7.15``|``foss/2018b``

### Flink

Apache Flink is a framework and distributed processing engine for stateful computations over unbounded and bounded data streams. Flink has been designed to run in all common cluster environments, perform computations at in-memory speed and at any scale.

*homepage*: <https://flink.apache.org/>

version   |versionsuffix      |toolchain
----------|-------------------|----------
``1.11.2``|``-bin-scala_2.11``|``system``

### FLINT

FLINT (Fast Library for Number Theory) is a C library in support of computations  in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs,  factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides  various low-level routines for fast arithmetic. FLINT is extensively documented and tested.

*homepage*: <https://www.flintlib.org/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------
``2.5.2``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2.7.1``|``GCC/10.3.0``
``2.8.4``|``foss/2021b``
``2.9.0``|``gfbf/2022a``

### flook

The fortran-Lua-hook library.

*homepage*: <https://github.com/ElectronicStructureLibrary/flook>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------
``0.8.1``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

### flowFDA

R package for analysing flow cytometry experiments with model based clustering,  functional principal component analysis

*homepage*: <https://github.com/FMKerckhof/flowFDA>

version          |versionsuffix|toolchain
-----------------|-------------|--------------
``0.99-20220602``|``-R-4.2.1`` |``foss/2022a``

### FLTK

FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,  and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL  and its built-in GLUT emulation.

*homepage*: <https://www.fltk.org>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.3``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.4``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``
``1.3.5``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.3.6``|``GCCcore/10.3.0``
``1.3.7``|``GCCcore/11.2.0``
``1.3.8``|``GCCcore/11.3.0``

### FLUENT

ANSYS FLUENT software contains the broad physical modeling capabilities needed to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms, from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants.

*homepage*: <https://www.ansys.com/products/fluids/ansys-fluent>

version   |toolchain
----------|----------
``14.5``  |``system``
``15.0.7``|``system``
``16.0``  |``system``
``17.1``  |``system``
``18.0``  |``system``
``18.1``  |``system``
``18.2``  |``system``
``2019R3``|``system``
``2021R1``|``system``
``2021R2``|``system``

### Flye

Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio  and Oxford Nanopore Technologies.

*homepage*: <https://github.com/fenderglass/Flye>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------
``2.4``  |``-Python-2.7.15``|``intel/2018b``
``2.6``  |``-Python-3.7.2`` |``foss/2019a``
``2.6``  |``-Python-3.7.4`` |``intel/2019b``
``2.8.1``|``-Python-3.8.2`` |``intel/2020a``
``2.8.2``|``-Python-3.8.2`` |``foss/2020a``
``2.8.3``|                  |``GCC/10.2.0``, ``iccifort/2020.4.304``
``2.9``  |                  |``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``2.9.1``|                  |``GCC/11.2.0``

### FMILibrary

FMI library is intended as a foundation for applications interfacing  FMUs (Functional Mockup Units) that follow FMI Standard. This version of the library supports  FMI 1.0 and FMI2.0. See http://www.fmi-standard.org/

*homepage*: <https://jmodelica.org/>

version  |toolchain
---------|---------------
``2.0.3``|``intel/2018b``

### FMM3D

Flatiron Institute Fast Multipole Libraries: a set of libraries to compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine.

*homepage*: <https://fmm3d.readthedocs.io>

version     |toolchain
------------|--------------
``20211018``|``foss/2020b``

### FMPy

FMPy is a free Python library to simulate Functional Mock-up Units (FMUs).

*homepage*: <https://github.com/CATIA-Systems/FMPy>

version  |toolchain
---------|--------------
``0.3.2``|``foss/2021a``

### FMRIprep

FMRIprep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is  designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan  acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive  error and output reporting.

*homepage*: <https://fmriprep.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.8``|``-Python-3.6.6``|``foss/2018b``
``1.4.1``|``-Python-3.6.6``|``foss/2018b``

### FMS

The Flexible Modeling System (FMS) is a software framework for   supporting the efficient development, construction, execution, and scientific  interpretation of atmospheric, oceanic, and climate system models.

*homepage*: <https://github.com/NOAA-GFDL/FMS>

version    |toolchain
-----------|--------------------------------
``2022.02``|``gompi/2022a``, ``iimpi/2022a``

### fmt

fmt (formerly cppformat) is an open-source formatting library.

*homepage*: <http://fmtlib.net/>

version   |toolchain
----------|--------------------------------------
``3.0.1`` |``foss/2016b``, ``intel/2016b``
``3.0.2`` |``GCCcore/6.4.0``, ``intel/2017a``
``5.3.0`` |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``6.2.1`` |``GCCcore/9.3.0``
``7.0.3`` |``GCCcore/9.3.0``
``7.1.1`` |``GCCcore/11.2.0``
``9.1.0`` |``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``10.0.0``|``GCCcore/12.3.0``

### FoBiS

A Fortran Building System for automatic building modern Fortran projects

*homepage*: <https://github.com/szaghi/FoBiS>

version  |toolchain
---------|------------------
``3.0.5``|``GCCcore/10.3.0``

### FoldX

FoldX is used to provide a fast and quantitative estimation of the importance of the interactions  contributing to the stability of proteins and protein complexes.

*homepage*: <http://foldx.crg.es/>

version        |toolchain
---------------|----------
``2.5.2``      |``system``
``3.0-beta5.1``|``system``
``3.0-beta6``  |``system``
``3.0-beta6.1``|``system``

### fontconfig

Fontconfig is a library designed to provide system-wide font configuration,  customization and application access. 

*homepage*: <https://www.freedesktop.org/wiki/Software/fontconfig/>

version    |versionsuffix     |toolchain
-----------|------------------|--------------------------------------------------------------------------------------
``2.11.94``|                  |``foss/2016a``, ``intel/2016a``
``2.11.95``|                  |``foss/2016a``, ``intel/2016a``
``2.12.1`` |                  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.12.1`` |``-libpng-1.6.29``|``GCCcore/6.3.0``
``2.12.4`` |                  |``GCCcore/6.4.0``
``2.12.6`` |                  |``GCCcore/6.4.0``
``2.13.0`` |                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.13.1`` |                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.13.92``|                  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.13.93``|                  |``GCCcore/10.3.0``
``2.13.94``|                  |``GCCcore/11.2.0``
``2.14.0`` |                  |``GCCcore/11.3.0``
``2.14.1`` |                  |``GCCcore/12.2.0``
``2.14.2`` |                  |``GCCcore/12.3.0``

### fontsproto

X11 font extension wire protocol

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``2.1.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### FORD

FORD is an automatic documentation generator for modern Fortran programs

*homepage*: <https://github.com/Fortran-FOSS-Programmers/ford/wiki>

version   |toolchain
----------|------------------
``6.1.1`` |``GCCcore/10.2.0``
``6.1.6`` |``GCCcore/10.3.0``
``6.1.15``|``GCCcore/11.3.0``

### foss

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <https://simplebuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain>

version    |toolchain
-----------|----------
``2016.04``|``system``
``2016.06``|``system``
``2016.07``|``system``
``2016.09``|``system``
``2016a``  |``system``
``2016b``  |``system``
``2017a``  |``system``
``2017b``  |``system``
``2018.08``|``system``
``2018a``  |``system``
``2018b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``
``2021a``  |``system``
``2021b``  |``system``
``2022.05``|``system``
``2022.10``|``system``
``2022a``  |``system``
``2022b``  |``system``
``2023a``  |``system``

### fosscuda

GCC based compiler toolchain __with CUDA support__, and including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``
``2018b``|``system``
``2019a``|``system``
``2019b``|``system``
``2020a``|``system``
``2020b``|``system``

### FoX

FoX is an XML library written in Fortran 95. It allows software developers to read, write and modify XML documents from Fortran applications without the complications of dealing with multi-language development.

*homepage*: <http://homepages.see.leeds.ac.uk/~earawa/FoX/>

version  |toolchain
---------|---------------------------------------------------------------
``4.1.2``|``GCC/11.2.0``, ``GCC/9.3.0``, ``intel/2017b``, ``intel/2018a``

### FOX-Toolkit

FOX is a C++ based Toolkit for developing Graphical User Interfaces easily and effectively. It offers a wide, and growing, collection of Controls, and provides state of the art facilities such as drag and drop, selection, as well as OpenGL widgets for 3D graphical manipulation. FOX also implements icons, images, and user-convenience features such as status line help, and tooltips.

*homepage*: <https://www.fox-toolkit.org/>

version   |toolchain
----------|-------------------------------------
``1.6.57``|``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### FPM

Effing package management! Build packages for multiple platforms (deb, rpm, etc) with great ease  and sanity.

*homepage*: <https://github.com/jordansissel/fpm>

version  |versionsuffix  |toolchain
---------|---------------|----------
``1.3.3``|``-Ruby-2.1.6``|``system``

### fpocket

The fpocket suite of programs is a very fast open source protein pocket detection algorithm based on Voronoi tessellation. The platform is suited for the scientific community willing to develop new scoring functions and  extract pocket descriptors on a large scale level.

*homepage*: <https://github.com/Discngine/fpocket>

version    |toolchain
-----------|---------------
``3.1.4.2``|``gompi/2020a``

### fqtrim

fqtrim is a versatile stand-alone utility that can be used to trim adapters, poly-A tails,  terminal unknown bases (Ns) and low quality 3' regions in reads from high-throughput next-generation sequencing  machines.

*homepage*: <http://ccb.jhu.edu/software/fqtrim/>

version  |toolchain
---------|---------------
``0.9.4``|``intel/2016b``
``0.9.5``|``intel/2017a``

### fqzcomp

Fqzcomp is a basic fastq compressor, designed primarily for high performance.  Despite that it is comparable to bzip2 for compression levels.

*homepage*: <http://sourceforge.net/projects/fqzcomp/>

version|toolchain
-------|-------------
``4.6``|``GCC/4.8.2``

### FragGeneScan

FragGeneScan is an application for finding (fragmented) genes in short reads.

*homepage*: <http://omics.informatics.indiana.edu/FragGeneScan/>

version |toolchain
--------|---------------------------------------------------------------------------------------------
``1.31``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``foss/2018b``

### FragPipe

FragPipe is a Java Graphical User Interface (GUI) for a suite of computational  tools enabling comprehensive analysis of mass spectrometry-based proteomics data.

*homepage*: <https://fragpipe.nesvilab.org>

version |versionsuffix|toolchain
--------|-------------|----------
``20.0``|``-Java-11`` |``system``

### FRANz

A fast and flexible parentage inference program for natural populations.

*homepage*: <https://www.bioinf.uni-leipzig.de/Software/FRANz>

version  |toolchain
---------|--------------
``2.0.0``|``foss/2018a``

### FreeBarcodes

A package for the generation and decoding of FREE divergence error-correcting DNA barcodes

*homepage*: <https://github.com/hawkjo/freebarcodes>

version   |toolchain
----------|--------------
``3.0.a5``|``foss/2021b``

### freebayes

Bayesian haplotype-based genetic polymorphism discovery and genotyping.

*homepage*: <https://github.com/freebayes/freebayes>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.3.5``|                 |``GCC/10.2.0``
``1.3.5``|``-Python-3.8.2``|``GCC/9.3.0``
``1.3.6``|``-R-4.1.0``     |``foss/2021a``
``1.3.6``|``-R-4.1.2``     |``foss/2021b``

### FreeFEM

FreeFEM offers a fast interpolation algorithm and a language for the manipulation  of data on multiple meshes.

*homepage*: <https://freefem.org>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``4.5``|``-Python-3.7.4``|``foss/2019b``

### FreeFem++

FreeFem++ is a partial differential equation solver. It has its own language. freefem scripts can solve multiphysics non linear systems in 2D and 3D. 

*homepage*: <http://www.freefem.org/>

version   |versionsuffix       |toolchain
----------|--------------------|---------------
``3.58``  |``-downloaded-deps``|``foss/2017b``
``3.60``  |``-downloaded-deps``|``intel/2018a``
``3.61-1``|``-downloaded-deps``|``intel/2018a``

### freeglut

freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.

*homepage*: <http://freeglut.sourceforge.net/>

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0.0``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.0.0``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``3.2.1``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``3.2.2``|                |``GCCcore/11.3.0``
``3.4.0``|                |``GCCcore/12.2.0``

### FreeImage

FreeImage is an Open Source library project for developers who would like to support popular graphics image formats like PNG, BMP, JPEG, TIFF and others as needed by today's multimedia applications. FreeImage is simple to use, fast, multithreading safe.

*homepage*: <http://freeimage.sourceforge.net>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------
``3.18.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### FreeSASA

FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA).

*homepage*: <https://freesasa.github.io>

version  |toolchain
---------|-------------
``2.0.3``|``GCC/8.3.0``

### FreeSurfer

FreeSurfer is a set of tools for analysis and visualization  of structural and functional brain imaging data.  FreeSurfer contains a fully  automatic structural imaging stream for processing cross sectional and  longitudinal data.

*homepage*: <https://surfer.nmr.mgh.harvard.edu/>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``5.3.0``|``-centos4_x86_64``|``system``
``5.3.0``|``-centos6_x86_64``|``system``
``6.0.0``|``-centos6_x86_64``|``system``
``6.0.1``|``-centos6_x86_64``|``system``
``7.1.1``|``-centos6_x86_64``|``system``
``7.1.1``|``-centos7_x86_64``|``system``
``7.1.1``|``-centos8_x86_64``|``system``
``7.2.0``|``-centos7_x86_64``|``system``
``7.2.0``|``-centos8_x86_64``|``system``
``7.2.0``|``-ubuntu18_amd64``|``system``
``7.3.2``|``-centos7_x86_64``|``system``
``7.3.2``|``-centos8_x86_64``|``system``
``7.4.0``|``-centos8_x86_64``|``system``
``7.4.0``|``-ubuntu20_amd64``|``system``
``7.4.0``|``-ubuntu22_amd64``|``system``

### FreeTDS

FreeTDS is a set of libraries for Unix and Linux that allows your programs to  natively talk to Microsoft SQL Server and Sybase databases.

*homepage*: <https://www.freetds.org/>

version  |toolchain
---------|------------------
``1.3.3``|``GCCcore/10.3.0``

### freetype

FreeType 2 is a software font engine that is designed to be small, efficient,  highly customizable, and portable while capable of producing high-quality  output (glyph images). It can be used in graphics libraries, display servers,  font conversion tools, text image generation tools, and many other products  as well. 

*homepage*: <https://freetype.org>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------
``2.6.2`` |                  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.6.3`` |                  |``foss/2016a``, ``intel/2016a``
``2.6.5`` |                  |``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``2.7``   |                  |``foss/2016b``, ``intel/2016b``
``2.7.1`` |                  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``
``2.7.1`` |``-libpng-1.6.29``|``GCCcore/6.3.0``
``2.8``   |                  |``GCCcore/6.4.0``
``2.8.1`` |                  |``GCCcore/6.4.0``
``2.9``   |                  |``GCCcore/6.4.0``
``2.9.1`` |                  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.10.1``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.10.3``|                  |``GCCcore/10.2.0``
``2.10.4``|                  |``GCCcore/10.3.0``
``2.11.0``|                  |``GCCcore/11.2.0``
``2.12.1``|                  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``2.13.0``|                  |``GCCcore/12.3.0``

### freetype-py

Python binding for the freetype library

*homepage*: <https://github.com/rougier/freetype-py>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``2.2.0``|``-Python-3.7.4``|``GCCcore/8.3.0``

### FreeXL

FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. 

*homepage*: <https://www.gaia-gis.it/fossil/freexl/index>

version  |toolchain
---------|-------------------------------------------------------
``1.0.2``|``foss/2016b``, ``intel/2016b``
``1.0.3``|``GCCcore/6.4.0``
``1.0.5``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.0.6``|``GCCcore/11.2.0``

### freud-analysis

The freud Python library provides a simple, flexible, powerful set of tools for analyzing trajectories obtained from molecular dynamics or Monte Carlo simulations. High performance, parallelized C++ is used to compute standard tools such as radial distribution functions, correlation functions, order parameters, and clusters, as well as original analysis methods including potentials of mean force and torque (PMFTs) and local environment matching. The freud library supports many input formats and outputs NumPy arrays, enabling integration with the scientific Python ecosystem for many typical materials science workflows.

*homepage*: <https://github.com/glotzerlab/freud>

version  |toolchain
---------|--------------
``2.6.2``|``foss/2020b``

### FriBidi

The Free Implementation of the Unicode Bidirectional Algorithm. 

*homepage*: <https://github.com/fribidi/fribidi>

version   |toolchain
----------|----------------------------------------------------------
``1.0.1`` |``foss/2018a``, ``intel/2018a``
``1.0.2`` |``GCCcore/6.4.0``
``1.0.5`` |``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.0.9`` |``GCCcore/9.3.0``
``1.0.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.0.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### FRUIT

FORTRAN Unit Test Framework (FRUIT)

*homepage*: <https://fortranxunit.sourceforge.io>

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``3.4.3``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

### FRUIT_processor

FORTRAN Unit Test Framework (FRUIT)

*homepage*: <https://fortranxunit.sourceforge.io>

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``3.4.3``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

### FSL

FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.

*homepage*: <https://www.fmrib.ox.ac.uk/fsl/>

version    |versionsuffix                 |toolchain
-----------|------------------------------|------------------------------------------------
``5.0.9``  |                              |``intel/2016a``
``5.0.9``  |``-Mesa-11.2.1``              |``intel/2016a``
``5.0.9``  |``-centos6_64``               |``system``
``5.0.10`` |                              |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0.11`` |                              |``foss/2018b``
``5.0.11`` |``-Python-3.6.6``             |``foss/2018b``
``6.0.1``  |``-Python-2.7.15``            |``foss/2019a``
``6.0.1``  |``-Python-3.7.2``             |``foss/2019a``
``6.0.2``  |``-Python-2.7.15``            |``foss/2018b``, ``foss/2019a``
``6.0.2``  |``-Python-2.7.15-CUDA-9.2.88``|``foss/2018b``
``6.0.2``  |``-Python-3.7.2``             |``foss/2019a``
``6.0.3``  |``-Python-3.7.4``             |``foss/2019b``
``6.0.4``  |``-Python-3.7.4``             |``foss/2019b``
``6.0.5.1``|                              |``foss/2021a``

### FSLeyes

FSLeyes is the FSL image viewer.

*homepage*: <https://git.fmrib.ox.ac.uk/fsl/fsleyes/fsleyes>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.15.0``|``-Python-2.7.13``|``intel/2017a``

### fsom

A tiny C library for managing SOM (Self-Organizing Maps) neural networks.

*homepage*: <https://github.com/ekg/fsom>

version     |toolchain
------------|-----------------------------------------------------------------------------
``20141119``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### FSON

Fortran 95 JSON Parser

*homepage*: <https://github.com/josephalevin/fson>

version  |toolchain
---------|--------------
``1.0.5``|``GCC/10.2.0``

### FTGL

FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications. 

*homepage*: <https://github.com/frankheckenbach/ftgl>

version      |toolchain
-------------|----------------------------------------------------------------------------------------------------------
``2.1.3-rc5``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``
``2.4.0``    |``GCCcore/11.3.0``

### Fujitsu

Toolchain using Fujitsu compilers and libraries.

*homepage*: <(none)>

version  |toolchain
---------|----------
``21.05``|``system``

### fullrmc

Reverse Monte Carlo (RMC) is probably best known for its applications  in condensed matter physics and solid state chemistry. fullrmc which stands for  FUndamental Library Language for Reverse Monte Carlo is different than traditional  RMC but a stochastic modelling method to solve an inverse problem whereby  an atomic/molecular model is adjusted until its atoms position havei  the greatest consistency with a set of experimental data.

*homepage*: <https://bachiraoun.github.io/fullrmc>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.2.0``|``-Python-2.7.14``|``intel/2018a``

### fumi_tools

This tool is intended to deduplicate UMIs from single-end and paired-end  sequencing data. Reads are considered identical when their UMIs have the same sequence,  they have the same length and map at the same position.

*homepage*: <https://ccb-gitlab.cs.uni-saarland.de/tobias/fumi_tools>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.18.2``|                 |``GCC/10.3.0``, ``GCC/11.2.0``
``0.18.2``|``-Python-3.6.6``|``foss/2018b``

### funannotate

funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work  with higher eukaryotes)

*homepage*: <https://funannotate.readthedocs.io>

version   |toolchain
----------|--------------
``1.8.13``|``foss/2021b``

### FunGAP

Fungal Genome Annotation Pipeline using evidence-based gene model evaluation.

*homepage*: <https://github.com/CompSynBioLab-KoreaUniv/FunGAP>

version  |toolchain
---------|--------------
``1.1.1``|``foss/2022a``

### FUNWAVE-TVD

FUNWAVE–TVD is the TVD version of the fully nonlinear Boussinesq wave model (FUNWAVE)  initially developed by Kirby et al.

*homepage*: <https://fengyanshi.github.io>

version         |versionsuffix|toolchain
----------------|-------------|---------------
``3.1-20170525``|             |``intel/2017a``
``3.1-20170525``|``-no-storm``|``intel/2017a``

### FUSE

The reference implementation of the Linux FUSE (Filesystem in Userspace) interface

*homepage*: <https://github.com/libfuse/libfuse>

version   |toolchain
----------|--------------------------------------
``3.2.6`` |``intel/2018a``
``3.4.1`` |``foss/2018a``
``3.14.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### FuSeq

FuSeq is a novel method to discover fusion genes from paired-end RNA sequencing data.

*homepage*: <https://github.com/nghiavtr/FuSeq>

version  |toolchain
---------|---------------
``1.1.2``|``gompi/2019b``

### FusionCatcher

FusionCatcher searches for novel/known somatic fusion genes, translocations, and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples.

*homepage*: <https://github.com/ndaniel/fusioncatcher>

version |versionsuffix     |toolchain
--------|------------------|--------------
``1.20``|``-Python-2.7.16``|``foss/2019b``
``1.30``|``-Python-2.7.16``|``foss/2019b``

### futhark

Futhark is a small programming language designed to be compiled to efficient parallel code. It     is a statically typed, data-parallel, and purely functional array language in the ML family, and comes     with a heavily optimising ahead-of-time compiler that presently generates GPU code via CUDA and OpenCL,     although the language itself is hardware-agnostic and can also run on multicore CPUs

*homepage*: <https://futhark-lang.org/>

version   |toolchain
----------|----------
``0.19.5``|``system``

### futile

The FUTILE project (Fortran Utilities for the Treatment of Innermost Level of Executables) is a set of modules   and wrapper that encapsulate the most common low-level operations of a Fortran code.  

*homepage*: <https://launchpad.net/futile>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.8.3``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

### future

python-future is the missing compatibility layer between Python 2 and Python 3.

*homepage*: <https://python-future.org/>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------
``0.16.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``0.16.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.16.0``|``-Python-3.6.3`` |``intel/2017b``
``0.16.0``|``-Python-3.6.6`` |``foss/2018b``

### fxtract

Extract sequences from a fastx (fasta or fastq) file given a subsequence.

*homepage*: <https://github.com/ctSkennerton/fxtract>

version|toolchain
-------|-----------------
``2.3``|``GCCcore/8.2.0``

## G


[G-PhoCS](#g-phocs) - [g2clib](#g2clib) - [g2lib](#g2lib) - [g2log](#g2log) - [Gaia](#gaia) - [GAMESS-US](#gamess-us) - [gap](#gap) - [GapCloser](#gapcloser) - [GapFiller](#gapfiller) - [gappa](#gappa) - [GARLI](#garli) - [garnett](#garnett) - [GAT](#gat) - [GATB-Core](#gatb-core) - [GATE](#gate) - [GATK](#gatk) - [Gaussian](#gaussian) - [gawk](#gawk) - [gbasis](#gbasis) - [Gblocks](#gblocks) - [GBprocesS](#gbprocess) - [gbs2ploidy](#gbs2ploidy) - [gc](#gc) - [GC3Pie](#gc3pie) - [GCC](#gcc) - [GCCcore](#gcccore) - [gcccuda](#gcccuda) - [gcloud](#gcloud) - [GConf](#gconf) - [GCTA](#gcta) - [Gctf](#gctf) - [GD](#gd) - [GDAL](#gdal) - [GDB](#gdb) - [gdbgui](#gdbgui) - [gdbm](#gdbm) - [gdc-client](#gdc-client) - [GDCHART](#gdchart) - [GDCM](#gdcm) - [GDGraph](#gdgraph) - [gdist](#gdist) - [Gdk-Pixbuf](#gdk-pixbuf) - [GDRCopy](#gdrcopy) - [Geant4](#geant4) - [Geant4-data](#geant4-data) - [gearshifft](#gearshifft) - [GEGL](#gegl) - [GEM](#gem) - [GEM-library](#gem-library) - [gemelli](#gemelli) - [GEMMA](#gemma) - [gemmi](#gemmi) - [gencore_variant_detection](#gencore_variant_detection) - [GeneMark-ET](#genemark-et) - [gengetopt](#gengetopt) - [GenMap](#genmap) - [Genome_Profiler](#genome_profiler) - [GenomeMapper](#genomemapper) - [genomepy](#genomepy) - [GenomeTester4](#genometester4) - [GenomeThreader](#genomethreader) - [GenomeTools](#genometools) - [GenomeWorks](#genomeworks) - [GenotypeHarmonizer](#genotypeharmonizer) - [genozip](#genozip) - [gensim](#gensim) - [geocube](#geocube) - [geopandas](#geopandas) - [geopy](#geopy) - [georges](#georges) - [GEOS](#geos) - [Gerris](#gerris) - [GETORB](#getorb) - [GetOrganelle](#getorganelle) - [gettext](#gettext) - [gexiv2](#gexiv2) - [gfbf](#gfbf) - [GffCompare](#gffcompare) - [gffread](#gffread) - [gflags](#gflags) - [GFOLD](#gfold) - [gh](#gh) - [GHC](#ghc) - [Ghostscript](#ghostscript) - [Gibbs2](#gibbs2) - [giflib](#giflib) - [gifsicle](#gifsicle) - [GIMIC](#gimic) - [gimkl](#gimkl) - [GimmeMotifs](#gimmemotifs) - [GIMP](#gimp) - [gimpi](#gimpi) - [gimpic](#gimpic) - [GIMPS](#gimps) - [giolf](#giolf) - [giolfc](#giolfc) - [Giotto-Suite](#giotto-suite) - [git](#git) - [git-extras](#git-extras) - [git-lfs](#git-lfs) - [GitPython](#gitpython) - [Givaro](#givaro) - [Giza](#giza) - [GKeyll](#gkeyll) - [GL2PS](#gl2ps) - [Glade](#glade) - [glew](#glew) - [GLFW](#glfw) - [GLI](#gli) - [GLib](#glib) - [glib-networking](#glib-networking) - [glibc](#glibc) - [GLibmm](#glibmm) - [GLIMMER](#glimmer) - [GlimmerHMM](#glimmerhmm) - [GLIMPSE](#glimpse) - [GLM](#glm) - [GLM-AED](#glm-aed) - [GlobalArrays](#globalarrays) - [Globus-CLI](#globus-cli) - [GlobusConnectPersonal](#globusconnectpersonal) - [glog](#glog) - [GLPK](#glpk) - [glproto](#glproto) - [Glucose](#glucose) - [GMAP-GSNAP](#gmap-gsnap) - [GMP](#gmp) - [gmpich](#gmpich) - [gmpolf](#gmpolf) - [gmpy2](#gmpy2) - [gmsh](#gmsh) - [GMT](#gmt) - [gmvapich2](#gmvapich2) - [gmvolf](#gmvolf) - [GNU](#gnu) - [gnuplot](#gnuplot) - [GnuTLS](#gnutls) - [Go](#go) - [goalign](#goalign) - [GOATOOLS](#goatools) - [gobff](#gobff) - [GObject-Introspection](#gobject-introspection) - [goblf](#goblf) - [GOBNILP](#gobnilp) - [Godon](#godon) - [gofasta](#gofasta) - [golf](#golf) - [gomkl](#gomkl) - [gompi](#gompi) - [gompic](#gompic) - [google-java-format](#google-java-format) - [googletest](#googletest) - [gotree](#gotree) - [GP2C](#gp2c) - [GPAW](#gpaw) - [GPAW-setups](#gpaw-setups) - [gperf](#gperf) - [gperftools](#gperftools) - [gpustat](#gpustat) - [GPy](#gpy) - [GPyOpt](#gpyopt) - [GPyTorch](#gpytorch) - [Grace](#grace) - [Gradle](#gradle) - [gradunwarp](#gradunwarp) - [graph-tool](#graph-tool) - [GraphDB](#graphdb) - [Graphene](#graphene) - [GraphicsMagick](#graphicsmagick) - [graphite2](#graphite2) - [GraPhlAn](#graphlan) - [GraphMap](#graphmap) - [GraphMap2](#graphmap2) - [Graphviz](#graphviz) - [graphviz-python](#graphviz-python) - [GRASP](#grasp) - [GRASS](#grass) - [Greenlet](#greenlet) - [Grep](#grep) - [gretl](#gretl) - [grib_api](#grib_api) - [grid](#grid) - [GRIDSS](#gridss) - [GRIT](#grit) - [GRNBoost](#grnboost) - [groff](#groff) - [GroIMP](#groimp) - [GROMACS](#gromacs) - [GromacsWrapper](#gromacswrapper) - [Groovy](#groovy) - [gRPC](#grpc) - [GSEA](#gsea) - [gsettings-desktop-schemas](#gsettings-desktop-schemas) - [GSL](#gsl) - [gSOAP](#gsoap) - [gsport](#gsport) - [GST-plugins-bad](#gst-plugins-bad) - [GST-plugins-base](#gst-plugins-base) - [GStreamer](#gstreamer) - [gsutil](#gsutil) - [gsw](#gsw) - [GTDB-Tk](#gtdb-tk) - [GTK+](#gtk+) - [GTK2](#gtk2) - [GTK3](#gtk3) - [GTK4](#gtk4) - [GtkSourceView](#gtksourceview) - [GTOOL](#gtool) - [GTS](#gts) - [gubbins](#gubbins) - [guenomu](#guenomu) - [Guile](#guile) - [GULP](#gulp) - [Gurobi](#gurobi) - [GUSHR](#gushr) - [gzip](#gzip)


### G-PhoCS

G-PhoCS is a software package for inferring ancestral population sizes, population divergence times, and migration rates from individual genome sequences. G-PhoCS accepts as input a set of multiple sequence alignments from separate neutrally evolving loci along the genome. Parameter inference is done in a Bayesian manner, using a Markov Chain Monte Carlo (MCMC) to jointly sample model parameters and genealogies at the input loci. 

*homepage*: <http://compgen.cshl.edu/GPhoCS/>

version  |toolchain
---------|----------------------------------------
``1.2.3``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

### g2clib

Library contains GRIB2 encoder/decoder ('C' version).

*homepage*: <https://github.com/NOAA-EMC/NCEPLIBS-g2c>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------
``1.6.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``1.6.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.7.0``|``GCCcore/11.3.0``

### g2lib

Library contains GRIB2 encoder/decoder and search/indexing routines.

*homepage*: <https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/>

version  |toolchain
---------|---------------------------------------------------------------------
``1.4.0``|``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.1.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2018b``
``3.2.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

### g2log

g2log, efficient asynchronous logger using C++11

*homepage*: <https://sites.google.com/site/kjellhedstrom2//g2log-efficient-background-io-processign-with-c11>

version|toolchain
-------|---------------------------------
``1.0``|``GCCcore/8.3.0``, ``foss/2016b``

### Gaia

Gaia is a C++ library with python bindings which implements similarity measures and classiﬁcations  on the results of audio analysis, and generates classiﬁcation models that Essentia can use to compute high-level  description of music.

*homepage*: <https://github.com/MTG/gaia>

version  |versionsuffix     |toolchain
---------|------------------|-----------------
``2.4.5``|``-Python-2.7.15``|``GCCcore/8.2.0``

### GAMESS-US

The General Atomic and Molecular Electronic Structure System (GAMESS)  is a general ab initio quantum chemistry package. 

*homepage*: <http://www.msg.chem.iastate.edu/gamess/index.html>

version        |versionsuffix|toolchain
---------------|-------------|---------------
``20141205-R1``|             |``intel/2016a``
``20170420-R1``|             |``intel/2016b``
``20170420-R1``|``-sockets`` |``intel/2016b``
``20180214-R1``|             |``foss/2016b``
``20180214-R1``|``-sockets`` |``foss/2016b``

### gap

GAP is a system for computational discrete algebra, with particular emphasis on Computational Group Theory.

*homepage*: <https://www.gap-system.org>

version   |toolchain
----------|---------------
``4.9.3`` |``intel/2018b``
``4.11.0``|``foss/2019a``

### GapCloser

GapCloser is designed to close the gaps emerging during the scaffolding process  by SOAPdenovo or other assembler, using the abundant pair relationships of short reads.

*homepage*: <https://sourceforge.net/projects/soapdenovo2/files/GapCloser/>

version    |toolchain
-----------|-------------------------------
``1.12-r6``|``foss/2018a``, ``intel/2017b``

### GapFiller

GapFiller is a seed-and-extend local assembler to fill the gap within paired reads.  It can be used for both DNA and RNA and it has been tested on Illumina data.

*homepage*: <https://sourceforge.net/projects/gapfiller>

version  |toolchain
---------|---------------
``2.1.1``|``intel/2017a``
``2.1.2``|``GCC/11.3.0``

### gappa

gappa is a collection of commands for working with phylogenetic data. Its main focus are evolutionary placements of short environmental sequences on a reference phylogenetic tree. Such data is typically produced by tools like EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files. 

*homepage*: <https://github.com/lczech/gappa>

version  |toolchain
---------|--------------
``0.7.1``|``GCC/10.3.0``

### GARLI

GARLI, Genetic Algorithm for Rapid Likelihood Inference is a   program for inferring phylogenetic trees. Using an approach similar to a   classical genetic algorithm, it rapidly searches the space of evolutionary   trees and model parameters to find the solution maximizing the likelihood   score. It implements nucleotide, amino acid and codon-based models of sequence   evolution, and runs on all platforms.

*homepage*: <https://code.google.com/archive/p/garli/>

version |toolchain
--------|---------------
``2.01``|``gompi/2019a``

### garnett

Garnett is a software package that faciliates automated cell type classification from single-cell expression data.

*homepage*: <https://cole-trapnell-lab.github.io/garnett>

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.1.20``|``-R-4.0.3`` |``foss/2020b``

### GAT

The Genomic Association Tester (GAT) is a tool for computing the significance of overlap between   multiple sets of genomic intervals. GAT estimates significance based on simulation.

*homepage*: <https://gat.readthedocs.io/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.2.2``|``-Python-2.7.11``|``foss/2016a``

### GATB-Core

GATB-Core is a high-performance and low memory footprint C++ library that provides a set of highly  efficient algorithms to analyse NGS data sets

*homepage*: <https://gatb.inria.fr/software/gatb-core>

version  |toolchain
---------|---------------
``1.4.2``|``gompi/2022a``

### GATE

GATE is an advanced opensource software developed by the international OpenGATE collaboration and  dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography

*homepage*: <http://www.opengatecollaboration.org/>

version    |versionsuffix                  |toolchain
-----------|-------------------------------|-------------------------------
``6.2``    |``-Python-2.7.11``             |``intel/2016a``
``7.1``    |``-Python-2.7.11``             |``intel/2016a``
``7.2``    |``-Python-2.7.11``             |``intel/2016a``
``8.0``    |``-Python-2.7.14``             |``foss/2017b``, ``intel/2017b``
``8.0``    |``-Python-2.7.14-Geant4-10.04``|``intel/2017b``
``8.1.p01``|``-Python-2.7.15``             |``foss/2018b``
``8.2``    |``-Python-2.7.14``             |``foss/2017b``, ``intel/2017b``
``8.2``    |``-Python-2.7.15``             |``foss/2018b``
``9.0``    |``-Python-3.7.4``              |``foss/2019b``
``9.1``    |                               |``foss/2021b``
``9.2``    |                               |``foss/2021b``, ``foss/2022a``

### GATK

The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute  to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on  data quality assurance. Its robust architecture, powerful processing engine and  high-performance computing features make it capable of taking on projects of any size.

*homepage*: <https://www.broadinstitute.org/gatk/>

version     |versionsuffix      |toolchain
------------|-------------------|------------------
``1.0.5083``|                   |``system``
``2.5-2``   |``-Java-1.7.0_10`` |``system``
``2.6-5``   |``-Java-1.7.0_10`` |``system``
``2.7-4``   |                   |``system``
``2.7-4``   |``-Java-1.7.0_10`` |``system``
``2.8-1``   |``-Java-1.7.0_10`` |``system``
``3.0-0``   |``-Java-1.7.0_10`` |``system``
``3.3-0``   |``-Java-1.7.0_21`` |``system``
``3.3-0``   |``-Java-1.7.0_80`` |``system``
``3.3-0``   |``-Java-1.8.0_66`` |``system``
``3.5``     |``-Java-1.8.0_66`` |``system``
``3.5``     |``-Java-1.8.0_74`` |``system``
``3.6``     |``-Java-1.8.0_92`` |``system``
``3.7``     |``-Java-1.8.0_112``|``system``
``3.8-0``   |``-Java-1.8.0_144``|``system``
``4.0.1.2`` |``-Java-1.8``      |``system``
``4.0.4.0`` |``-Python-2.7.14`` |``intel/2018a``
``4.0.4.0`` |``-Python-3.6.4``  |``intel/2018a``
``4.0.5.1`` |``-Python-3.6.4``  |``foss/2018a``
``4.0.7.0`` |``-Python-2.7.14`` |``intel/2018a``
``4.0.7.0`` |``-Python-3.6.4``  |``intel/2018a``
``4.0.8.1`` |``-Python-2.7.15`` |``foss/2018b``
``4.0.8.1`` |``-Python-3.6.6``  |``foss/2018b``
``4.0.10.0``|``-Python-3.6.6``  |``foss/2018b``
``4.0.12.0``|``-Python-3.6.6``  |``foss/2018b``
``4.1.0.0`` |``-Python-3.6.6``  |``foss/2018b``
``4.1.2.0`` |``-Java-1.8``      |``GCCcore/8.2.0``
``4.1.3.0`` |``-Java-1.8``      |``GCCcore/8.3.0``
``4.1.4.1`` |``-Java-1.8``      |``GCCcore/8.3.0``
``4.1.4.1`` |``-Java-11``       |``GCCcore/8.3.0``
``4.1.5.0`` |``-Java-1.8``      |``GCCcore/9.3.0``
``4.1.5.0`` |``-Java-11``       |``GCCcore/9.3.0``
``4.1.8.1`` |``-Java-1.8``      |``GCCcore/9.3.0``
``4.2.0.0`` |``-Java-1.8``      |``GCCcore/10.2.0``
``4.2.0.0`` |``-Java-11``       |``GCCcore/10.2.0``
``4.2.3.0`` |``-Java-11``       |``GCCcore/11.2.0``
``4.2.5.0`` |``-Java-11``       |``GCCcore/11.2.0``
``4.2.6.1`` |``-Java-11``       |``GCCcore/11.2.0``
``4.3.0.0`` |``-Java-11``       |``GCCcore/11.3.0``
``4.4.0.0`` |``-Java-17``       |``GCCcore/12.2.0``

### Gaussian

Gaussian provides state-of-the-art capabilities for electronic structure modeling. Gaussian 09 is licensed for a wide variety of computer systems. All versions of Gaussian 09 contain every scientific/modeling feature, and none imposes any artificial limitations on calculations other than your computing resources and patience.  This is the official gaussian AVX2 build. 

*homepage*: <https://www.gaussian.com/>

version    |versionsuffix|toolchain
-----------|-------------|----------
``09.e.01``|``-AVX``     |``system``
``16.A.03``|``-AVX2``    |``system``

### gawk

The awk utility interprets a special-purpose programming language that makes it possible to handle simple data-reformatting jobs with just a few lines of code.

*homepage*: <https://www.gnu.org/software/gawk>

version  |toolchain
---------|----------------------------------------------
``5.1.0``|``GCC/10.2.0``
``5.1.1``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

### gbasis

Python library for analytical evaluation and integration of Gaussian-type basis functions and  related quantities.

*homepage*: <https://github.com/theochem/gbasis>

version     |toolchain
------------|---------------
``20210904``|``intel/2022a``

### Gblocks

Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis

*homepage*: <http://molevol.cmima.csic.es/castresana/Gblocks.html>

version  |toolchain
---------|----------
``0.91b``|``system``

### GBprocesS

GBprocesS allows for the extraction of genomic inserts from NGS data for GBS experiments. Preprocessing is performed in different stages that are part of a linear pipeline where the steps are performed in order. GBprocesS provides a flexible way to adjust the functionality to your needs, as the operations required and the execution order vary depending on the GBS protocol used.

*homepage*: <https://gbprocess.readthedocs.io/>

version        |versionsuffix    |toolchain
---------------|-----------------|---------------
``2.3``        |``-Python-3.8.2``|``intel/2020a``
``4.0.0.post1``|                 |``foss/2022a``

### gbs2ploidy

Inference of Ploidy from (Genotyping-by-Sequencing) GBS Data

*homepage*: <https://cran.r-project.org/web/packages/gbs2ploidy>

version|versionsuffix|toolchain
-------|-------------|---------------
``1.0``|``-R-3.4.3`` |``intel/2017b``

### gc

The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting replacement for C malloc or C++ new. 

*homepage*: <https://hboehm.info/gc/>

version   |toolchain
----------|--------------------------------------
``7.4.4`` |``GCC/4.9.3-2.25``, ``foss/2016a``
``7.6.0`` |``GCCcore/6.4.0``
``7.6.4`` |``GCCcore/7.3.0``
``7.6.10``|``GCCcore/8.2.0``
``7.6.12``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``8.0.4`` |``GCCcore/10.3.0``
``8.2.0`` |``GCCcore/11.2.0``
``8.2.2`` |``GCCcore/10.2.0``, ``GCCcore/11.3.0``

### GC3Pie

GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution  environments.

*homepage*: <https://gc3pie.readthedocs.org>

version  |toolchain
---------|----------
``2.4.2``|``system``
``2.5.0``|``system``
``2.5.2``|``system``

### GCC

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: <http://gcc.gnu.org/>

version   |versionsuffix      |toolchain
----------|-------------------|----------
``4.8.1`` |                   |``system``
``4.8.1`` |``-CLooG``         |``system``
``4.8.2`` |                   |``system``
``4.8.2`` |``-CLooG``         |``system``
``4.8.2`` |``-CLooG-multilib``|``system``
``4.8.2`` |``-multilib``      |``system``
``4.8.3`` |                   |``system``
``4.8.3`` |``-CLooG-multilib``|``system``
``4.8.4`` |                   |``system``
``4.8.4`` |``-CLooG``         |``system``
``4.8.4`` |``-CLooG-multilib``|``system``
``4.8.5`` |                   |``system``
``4.9.0`` |                   |``system``
``4.9.0`` |``-CLooG``         |``system``
``4.9.0`` |``-CLooG-multilib``|``system``
``4.9.1`` |                   |``system``
``4.9.1`` |``-CLooG``         |``system``
``4.9.1`` |``-CLooG-multilib``|``system``
``4.9.2`` |                   |``system``
``4.9.2`` |``-CLooG``         |``system``
``4.9.2`` |``-CLooG-multilib``|``system``
``4.9.2`` |``-binutils-2.25`` |``system``
``4.9.3`` |                   |``system``
``4.9.3`` |``-2.25``          |``system``
``4.9.3`` |``-binutils-2.25`` |``system``
``4.9.4`` |``-2.25``          |``system``
``5.1.0`` |                   |``system``
``5.1.0`` |``-binutils-2.25`` |``system``
``5.2.0`` |                   |``system``
``5.3.0`` |                   |``system``
``5.3.0`` |``-2.26``          |``system``
``5.4.0`` |``-2.26``          |``system``
``5.5.0`` |``-2.26``          |``system``
``6.1.0`` |``-2.27``          |``system``
``6.2.0`` |``-2.27``          |``system``
``6.3.0`` |``-2.27``          |``system``
``6.3.0`` |``-2.28``          |``system``
``6.4.0`` |``-2.28``          |``system``
``7.1.0`` |``-2.28``          |``system``
``7.2.0`` |``-2.29``          |``system``
``7.3.0`` |``-2.30``          |``system``
``7.4.0`` |``-2.31.1``        |``system``
``8.1.0`` |``-2.30``          |``system``
``8.2.0`` |``-2.31.1``        |``system``
``8.3.0`` |                   |``system``
``8.3.0`` |``-2.32``          |``system``
``8.4.0`` |                   |``system``
``9.1.0`` |``-2.32``          |``system``
``9.2.0`` |                   |``system``
``9.2.0`` |``-2.32``          |``system``
``9.3.0`` |                   |``system``
``9.4.0`` |                   |``system``
``9.5.0`` |                   |``system``
``10.1.0``|                   |``system``
``10.2.0``|                   |``system``
``10.3.0``|                   |``system``
``11.1.0``|                   |``system``
``11.2.0``|                   |``system``
``11.3.0``|                   |``system``
``11.4.0``|                   |``system``
``12.1.0``|                   |``system``
``12.2.0``|                   |``system``
``12.3.0``|                   |``system``
``13.1.0``|                   |``system``
``system``|                   |``system``
``system``|``-2.29``          |``system``

### GCCcore

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: <http://gcc.gnu.org/>

version   |toolchain
----------|----------
``4.9.2`` |``system``
``4.9.3`` |``system``
``4.9.4`` |``system``
``5.3.0`` |``system``
``5.4.0`` |``system``
``5.5.0`` |``system``
``6.1.0`` |``system``
``6.2.0`` |``system``
``6.3.0`` |``system``
``6.4.0`` |``system``
``6.5.0`` |``system``
``7.1.0`` |``system``
``7.2.0`` |``system``
``7.3.0`` |``system``
``7.4.0`` |``system``
``8.1.0`` |``system``
``8.2.0`` |``system``
``8.3.0`` |``system``
``8.4.0`` |``system``
``9.1.0`` |``system``
``9.2.0`` |``system``
``9.3.0`` |``system``
``9.4.0`` |``system``
``9.5.0`` |``system``
``10.1.0``|``system``
``10.2.0``|``system``
``10.3.0``|``system``
``11.1.0``|``system``
``11.2.0``|``system``
``11.3.0``|``system``
``11.4.0``|``system``
``12.1.0``|``system``
``12.2.0``|``system``
``12.3.0``|``system``
``13.1.0``|``system``
``system``|``system``

### gcccuda

GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2016.08``|``system``
``2017b``  |``system``
``2018a``  |``system``
``2018b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

### gcloud

Libraries and tools for interacting with Google Cloud products and services.

*homepage*: <https://cloud.google.com/sdk>

version    |toolchain
-----------|----------
``382.0.0``|``system``

### GConf

GConf is a system for storing application preferences.  It is intended for user preferences; not configuration  of something like Apache, or arbitrary data storage.

*homepage*: <https://projects.gnome.org/gconf/>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``

### GCTA

GCTA (Genome-wide Complex Trait Analysis) is a software package, which was  initially developed to estimate the proportion of phenotypic variance explained  by all genome-wide SNPs for a complex trait but has been extensively extended  for many other analyses of data from genome-wide association studies (GWASs). 

*homepage*: <https://yanglab.westlake.edu.cn/software/gcta/>

version       |toolchain
--------------|------------------------------
``1.94.0beta``|``foss/2021b``, ``gfbf/2022a``
``1.94.1``    |``gfbf/2022a``

### Gctf

Gctf: real-time CTF determination and correction, Kai Zhang, 2016

*homepage*: <https://www.mrc-lmb.cam.ac.uk/kzhang/>

version |toolchain
--------|----------
``1.06``|``system``

### GD

GD.pm - Interface to Gd Graphics Library

*homepage*: <https://github.com/lstein/Perl-GD>

version |versionsuffix   |toolchain
--------|----------------|------------------
``2.66``|``-Perl-5.24.0``|``foss/2016b``
``2.68``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.69``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.71``|                |``GCCcore/9.3.0``
``2.73``|                |``GCCcore/10.3.0``
``2.75``|                |``GCCcore/11.3.0``

### GDAL

GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for  data translation and processing.

*homepage*: <https://www.gdal.org>

version  |versionsuffix                 |toolchain
---------|------------------------------|----------------------------------------------------------------
``2.0.2``|                              |``foss/2016a``, ``intel/2016a``
``2.1.0``|                              |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``2.1.1``|                              |``foss/2016a``
``2.1.1``|``-Python-2.7.12``            |``intel/2016b``
``2.1.2``|``-Python-2.7.12``            |``intel/2016b``
``2.1.3``|``-Python-2.7.12``            |``foss/2016b``, ``intel/2016b``
``2.1.3``|``-Python-2.7.13``            |``intel/2017a``
``2.1.3``|``-Python-3.6.1``             |``intel/2017a``
``2.2.0``|``-Python-2.7.13-HDF5-1.8.18``|``intel/2017a``
``2.2.0``|``-Python-2.7.13-HDF5-HDF``   |``intel/2017a``
``2.2.0``|``-Python-3.6.1``             |``intel/2017a``
``2.2.2``|``-Python-2.7.14``            |``intel/2017b``
``2.2.2``|``-Python-2.7.14-HDF5-1.8.19``|``intel/2017b``
``2.2.3``|``-Python-2.7.14``            |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.2.3``|``-Python-2.7.15``            |``foss/2018b``
``2.2.3``|``-Python-3.6.3``             |``foss/2017b``
``2.2.3``|``-Python-3.6.4``             |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``2.2.3``|``-Python-3.6.6``             |``foss/2018b``, ``intel/2018b``
``2.4.4``|                              |``foss/2021b``, ``intel/2021b``
``3.0.0``|``-Python-2.7.15``            |``foss/2019a``, ``intel/2019a``
``3.0.0``|``-Python-3.7.2``             |``foss/2019a``, ``intel/2019a``
``3.0.2``|``-Python-3.7.4``             |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.0.4``|``-Python-3.8.2``             |``foss/2020a``, ``intel/2020a``
``3.2.1``|                              |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.3.0``|                              |``foss/2021a``
``3.3.2``|                              |``foss/2021b``
``3.5.0``|                              |``foss/2022a``
``3.6.2``|                              |``foss/2022b``

### GDB

The GNU Project Debugger

*homepage*: <https://www.gnu.org/software/gdb/gdb.html>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``7.8.2`` |                  |``GCC/4.9.2``
``7.9``   |                  |``GCC/4.9.2``
``7.10.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``7.11``  |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``7.11.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``8.0.1`` |``-Python-2.7.14``|``foss/2017b``
``8.0.1`` |``-Python-3.6.3`` |``foss/2017b``
``8.1``   |``-Python-2.7.14``|``foss/2018a``
``8.1.1`` |``-Python-2.7.14``|``intel/2018a``
``8.3``   |``-Python-3.7.2`` |``GCCcore/8.2.0``
``9.1``   |``-Python-3.7.4`` |``GCCcore/8.3.0``
``10.1``  |                  |``GCCcore/10.2.0``
``10.2``  |                  |``GCCcore/10.3.0``
``10.2``  |``-Python-3.8.2`` |``GCCcore/9.3.0``
``11.1``  |                  |``GCCcore/11.2.0``
``12.1``  |                  |``GCCcore/11.3.0``

### gdbgui

Browser-based frontend to gdb (gnu debugger). Add breakpoints, view the stack, visualize data structures, and more in C, C++, Go, Rust, and Fortran. Run gdbgui from the terminal and a new tab will open in your browser.

*homepage*: <https://gdbgui.com>

version     |versionsuffix    |toolchain
------------|-----------------|-----------------
``0.13.1.2``|``-Python-3.7.2``|``GCCcore/8.2.0``

### gdbm

GNU dbm (or GDBM, for short) is a library of database functions that use extensible hashing  and work similar to the standard UNIX dbm.  These routines are provided to a programmer needing to create and manipulate a hashed database.

*homepage*: <https://www.gnu.org.ua/software/gdbm/gdbm.html>

version   |toolchain
----------|------------------
``1.18.1``|``foss/2020a``
``1.21``  |``GCCcore/10.2.0``

### gdc-client

The gdc-client provides several convenience functions over  the GDC API which provides general download/upload via HTTPS.

*homepage*: <https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.0.1``|``-Python-2.7.12``|``intel/2016b``
``1.3.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``1.3.0``|``-Python-2.7.15``|``foss/2018b``
``1.6.0``|                  |``GCCcore/10.2.0``

### GDCHART

Simple to use C API, high performance library to create charts  and graphs in PNG, GIF and WBMP format.

*homepage*: <http://users.fred.net/brv/chart>

version      |toolchain
-------------|-----------------
``0.11.5dev``|``GCCcore/8.2.0``

### GDCM

Grassroots DICOM: Cross-platform DICOM implementation

*homepage*: <https://sourceforge.net/projects/gdcm>

version   |toolchain
----------|-------------------------------------
``2.8.8`` |``GCCcore/6.4.0``
``2.8.9`` |``GCCcore/7.3.0``
``3.0.4`` |``GCCcore/8.2.0``
``3.0.5`` |``GCCcore/8.3.0``
``3.0.8`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.0.20``|``GCCcore/11.3.0``
``3.0.21``|``GCCcore/12.2.0``

### GDGraph

GDGraph is a Perl package to generate charts

*homepage*: <https://metacpan.org/release/GDGraph>

version |versionsuffix   |toolchain
--------|----------------|------------------
``1.54``|``-Perl-5.26.1``|``intel/2018a``
``1.54``|``-Perl-5.28.0``|``foss/2018b``
``1.56``|                |``GCCcore/11.3.0``

### gdist

The gdist module is a Cython interface to a C++ library (http://code.google.com/p/geodesic/) for  computing geodesic distance which is the length of shortest line between two vertices on a triangulated mesh in three  dimensions, such that the line lies on the surface.

*homepage*: <https://pypi.python.org/pypi/gdist>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.3``|``-Python-2.7.11``|``intel/2016a``

### Gdk-Pixbuf

The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.  It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it  was distributed as part of GTK+ 2 but it was split off into a separate package  in preparation for the change to GTK+ 3. 

*homepage*: <https://docs.gtk.org/gdk-pixbuf/>

version    |toolchain
-----------|------------------------------------------------------------------------------------
``2.32.3`` |``intel/2016a``
``2.35.1`` |``foss/2016a``, ``intel/2016a``
``2.36.0`` |``foss/2016b``, ``intel/2016b``
``2.36.8`` |``intel/2017a``
``2.36.10``|``intel/2017a``
``2.36.11``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``2.36.12``|``foss/2018b``, ``fosscuda/2018b``
``2.38.1`` |``GCCcore/8.2.0``
``2.38.2`` |``GCCcore/8.3.0``
``2.40.0`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.42.6`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.42.8`` |``GCCcore/11.3.0``
``2.42.10``|``GCCcore/12.2.0``

### GDRCopy

A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology.

*homepage*: <https://github.com/NVIDIA/gdrcopy>

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------------------------------
``2.1``  |``-CUDA-11.0.2``|``GCCcore/9.3.0``
``2.1``  |``-CUDA-11.1.1``|``GCCcore/10.2.0``
``2.1``  |``-CUDA-11.2.1``|``GCCcore/10.2.0``
``2.2``  |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.3``  |                |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``2.3.1``|                |``GCCcore/12.3.0``

### Geant4

Geant4 is a toolkit for the simulation of the passage of particles through matter.  Its areas of application include high energy, nuclear and accelerator physics,   as well as studies in medical and space science.

*homepage*: <https://geant4.cern.ch>

version      |toolchain
-------------|----------------------------------------------------------------
``9.5.p02``  |``intel/2016a``
``9.6.p04``  |``intel/2016a``
``10.01.p03``|``intel/2016a``
``10.02.p01``|``intel/2016a``
``10.03.p03``|``foss/2017b``, ``intel/2017b``
``10.04``    |``intel/2017b``
``10.5``     |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
``10.6``     |``foss/2019b``
``10.6.2``   |``foss/2020a``
``10.7.1``   |``GCC/10.2.0``, ``GCC/11.2.0``
``11.0.0``   |``GCC/11.2.0``
``11.0.1``   |``GCC/11.2.0``
``11.0.2``   |``GCC/11.2.0``, ``GCC/11.3.0``
``11.1.2``   |``GCC/11.3.0``

### Geant4-data

Datasets for Geant4.

*homepage*: <https://geant4.cern.ch/>

version     |toolchain
------------|----------
``11.1``    |``system``
``20201103``|``system``
``20210510``|``system``

### gearshifft

Benchmark Suite for Heterogenuous FFT Implementations

*homepage*: <https://github.com/mpicbg-scicomp/gearshifft>

version  |toolchain
---------|--------------
``0.4.0``|``foss/2019a``

### GEGL

GEGL (Generic Graphics Library) is a data flow based image processing framework,  providing floating point processing and non-destructive image processing capabilities to  GNU Image Manipulation Program (GIMP) and other projects.

*homepage*: <https://www.gegl.org>

version   |toolchain
----------|------------------
``0.4.30``|``GCCcore/10.3.0``

### GEM

GEM (Gene-Environment interaction analysis for Millions of samples) is a  software program for large-scale gene-environment interaction testing in  samples from unrelated individuals. It enables genome-wide association  studies in up to millions of samples while allowing for multiple exposures,  control for genotype-covariate interactions, and robust inference. 

*homepage*: <https://github.com/large-scale-gxe-methods/GEM>

version  |toolchain
---------|------------------------------
``1.5.1``|``foss/2022a``, ``foss/2022b``

### GEM-library

Next-generation sequencing platforms (Illumina/Solexa, ABI/SOLiD, etc.)   call for powerful and very optimized tools to index/analyze huge genomes. The GEM library   strives to be a true "next-generation" tool for handling any kind of sequence data, offering   state-of-the-art algorithms and data structures specifically tailored to this demanding task.   At the moment, efficient indexing and searching algorithms based on the Burrows-Wheeler   transform (BWT) have been implemented. The library core is written in C for maximum speed,   with concise interfaces to higher-level programming languages like OCaml and Python.   Many high-performance standalone programs (mapper, splice mapper, etc.) are provided along   with the library; in general, new algorithms and tools can be easily implemented on the top of it. 

*homepage*: <http://algorithms.cnag.cat/wiki/The_GEM_library>

version            |versionsuffix                  |toolchain
-------------------|-------------------------------|----------
``20130406-045632``|``_pre-release-3_Linux-x86_64``|``system``

### gemelli

Gemelli is a tool box for running both Robust Aitchison PCA (RPCA) and Compositional Tensor Factorization (CTF) on sparse compositional omics datasets. 

*homepage*: <https://github.com/biocore/gemelli>

version  |toolchain
---------|--------------
``0.0.9``|``foss/2022a``

### GEMMA

Genome-wide Efficient Mixed Model Association

*homepage*: <https://github.com/genetics-statistics/GEMMA>

version   |toolchain
----------|------------------------------
``0.97``  |``foss/2016b``, ``foss/2017a``
``0.98.1``|``foss/2018b``
``0.98.5``|``foss/2021b``, ``foss/2022b``

### gemmi

Gemmi is a library, accompanied by a set of programs, developed primarily for   use in macromolecular crystallography (MX). For working with:  macromolecular models (content of PDB, PDBx/mmCIF and mmJSON files), refinement restraints (CIF files), reflection data (MTZ and mmCIF formats), data on a 3D   grid (electron density maps, masks, MRC/CCP4 format) crystallographic symmetry. Parts of this library can be useful in structural bioinformatics (for symmetry- aware analysis of protein models), and in other molecular-structure sciences    that use CIF files (we have the fastest open-source CIF parser). 

*homepage*: <https://gemmi.readthedocs.io/>

version  |toolchain
---------|------------------
``0.4.5``|``GCCcore/10.2.0``

### gencore_variant_detection

This is a bundled install of many software packages for doing variant detection analysis. 

*homepage*: <https://nyuad-cgsb.github.io/variant_detection/public/index.html>

version|toolchain
-------|----------
``1.0``|``system``

### GeneMark-ET

Eukaryotic gene prediction suite with automatic training

*homepage*: <http://exon.gatech.edu/GeneMark>

version |toolchain
--------|--------------------------------------
``4.38``|``GCCcore/8.2.0``
``4.57``|``GCCcore/8.3.0``
``4.65``|``GCCcore/10.2.0``
``4.71``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### gengetopt

Gengetopt is a tool to write command line option parsing code for C programs.

*homepage*: <https://www.gnu.org/software/gengetopt/gengetopt.html>

version |toolchain
--------|-----------------------------------------------------------------------------
``2.23``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### GenMap

GenMap - Fast and Exact Computation of Genome Mappability

*homepage*: <https://github.com/cpockrandt/genmap>

version  |toolchain
---------|------------------
``1.3.0``|``GCCcore/11.2.0``

### Genome_Profiler

Genome Profiler (GeP) is a program to perform whole-genome multilocus  sequence typing (wgMLST) analysis for bacterial isolates

*homepage*: <https://sourceforge.net/projects/genomeprofiler/>

version|versionsuffix   |toolchain
-------|----------------|--------------
``2.1``|``-Perl-5.24.0``|``foss/2016b``

### GenomeMapper

GenomeMapper is a short read mapping tool designed for accurate read alignments.   It quickly aligns millions of reads either with ungapped or gapped alignments.   This version is used to align against a single reference.   If you are unsure which one is the appropriate GenomeMapper, you might want to use this one.

*homepage*: <https://1001genomes.org/software/genomemapper_singleref.html>

version  |toolchain
---------|--------------
``0.4.4``|``foss/2016a``

### genomepy

genomepy is designed to provide a simple and straightforward way to download and use genomic data

*homepage*: <https://github.com/vanheeringen-lab/genomepy>

version   |toolchain
----------|--------------
``0.15.0``|``foss/2022a``

### GenomeTester4

A toolkit for performing set operations - union, intersection and complement - on k-mer lists.

*homepage*: <https://github.com/bioinfo-ut/GenomeTester4>

version|toolchain
-------|---------------
``4.0``|``intel/2018a``

### GenomeThreader

GenomeThreader is a software tool to compute gene structure predictions.

*homepage*: <https://genomethreader.org>

version  |versionsuffix          |toolchain
---------|-----------------------|----------
``1.7.1``|``-Linux_x86_64-64bit``|``system``
``1.7.3``|``-Linux_x86_64-64bit``|``system``

### GenomeTools

A comprehensive software library for efficient processing of structured genome annotations.

*homepage*: <http://genometools.org>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------
``1.5.10``|                  |``foss/2018b``
``1.5.10``|``-Python-2.7.15``|``foss/2018b``
``1.6.1`` |                  |``GCC/10.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``
``1.6.1`` |``-Python-2.7.16``|``GCC/8.3.0``
``1.6.2`` |                  |``GCC/10.3.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### GenomeWorks

SDK for GPU accelerated genome assembly and analysis

*homepage*: <https://github.com/clara-parabricks/GenomeWorks>

version      |toolchain
-------------|------------------
``2021.02.2``|``fosscuda/2020b``

### GenotypeHarmonizer

The Genotype Harmonizer is an simple to use command-line tool that allows harmonization of genotype data  stored using different file formats with different and potentially unknown strands.

*homepage*: <https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer>

version   |versionsuffix     |toolchain
----------|------------------|----------
``1.4.14``|``-Java-1.7.0_80``|``system``

### genozip

Genozip is a compressor for genomic files - it compresses FASTQ, SAM/BAM/CRAM, VCF, FASTA and others.

*homepage*: <https://genozip.com>

version   |toolchain
----------|------------------
``13.0.5``|``GCCcore/11.2.0``

### gensim

Gensim is a Python library for topic modelling, document indexing and similarity retrieval with  large corpora.

*homepage*: <https://radimrehurek.com/gensim>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.13.2``|``-Python-2.7.11``|``foss/2016a``
``3.8.3`` |                  |``foss/2020b``, ``intel/2020b``
``4.2.0`` |                  |``foss/2021a``

### geocube

Tool to convert geopandas vector data into rasterized xarray data.

*homepage*: <https://corteva.github.io/geocube/latest/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.0.14``|``-Python-3.8.2``|``foss/2020a``

### geopandas

GeoPandas is a project to add support for geographic data to pandas objects. It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and perform geometric operations.

*homepage*: <https://github.com/geopandas/geopandas>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.7.0`` |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.8.0`` |``-Python-3.7.2``|``foss/2019a``
``0.8.1`` |``-Python-3.7.4``|``intel/2019b``
``0.8.1`` |``-Python-3.8.2``|``foss/2020a``
``0.10.2``|                 |``foss/2021a``
``0.11.0``|                 |``foss/2021b``
``0.12.2``|                 |``foss/2022a``, ``foss/2022b``

### geopy

geopy is a Python client for several popular geocoding web services. geopy makes it simple for Python developers to locate the coordinates of addresses, cities, countries, and landmarks across the globe using third-party geocoders and other data sources. 

*homepage*: <https://geopy.readthedocs.io/en/stable/>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.11.0``|``-Python-3.6.1``|``intel/2017a``
``2.1.0`` |                 |``GCCcore/10.2.0``

### georges

Georges the lemur opinionated particle accelerator modeling Python package. Also a thin wrapper over MAD-X/PTC, BDSim and G4Beamline.

*homepage*: <https://github.com/ULB-Metronu/georges>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2019.2``|                 |``foss/2021a``
``2019.2``|``-Python-3.7.4``|``foss/2019b``

### GEOS

GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)

*homepage*: <https://trac.osgeo.org/geos>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------------------------
``3.5.0`` |``-Python-2.7.11``|``intel/2016a``
``3.5.0`` |``-Python-2.7.12``|``intel/2016b``
``3.6.1`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.6.1`` |``-Python-2.7.13``|``intel/2017a``
``3.6.1`` |``-Python-3.6.1`` |``intel/2017a``
``3.6.2`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.6.2`` |``-Python-2.7.15``|``foss/2018b``
``3.6.2`` |``-Python-3.6.2`` |``foss/2017b``
``3.6.2`` |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``, ``intel/2018.01``
``3.6.2`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.6.2`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.7.2`` |``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``
``3.7.2`` |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.8.0`` |                  |``GCC/8.3.0``
``3.8.0`` |``-Python-3.7.4`` |``GCC/8.3.0``, ``iccifort/2019.5.281``
``3.8.1`` |``-Python-3.8.2`` |``GCC/9.3.0``, ``iccifort/2020.1.217``
``3.9.1`` |                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.4.0``
``3.10.3``|                  |``GCC/11.3.0``
``3.11.1``|                  |``GCC/12.2.0``

### Gerris

Gerris is a Free Software program for the solution of the partial                  differential equations describing fluid flow

*homepage*: <http://gfs.sourceforge.net/wiki/index.php/Main_Page>

version     |toolchain
------------|--------------
``20131206``|``foss/2017b``

### GETORB

GETORB software package contains programs to handle the orbital data records (ODRs)

*homepage*: <http://www.deos.tudelft.nl/ers/precorbs/tools/getorb_pack.shtml>

version  |toolchain
---------|---------------
``2.3.2``|``intel/2017a``

### GetOrganelle

This toolkit assemblies organelle genome from genomic skimming data.

*homepage*: <https://github.com/Kinggerm/GetOrganelle>

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``1.7.2``     |``-Python-3.8.2``|``foss/2020a``
``1.7.4-pre2``|                 |``foss/2020b``
``1.7.5.3``   |                 |``foss/2021b``
``1.7.6.1``   |                 |``foss/2021b``
``1.7.7.0``   |                 |``foss/2022a``

### gettext

GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation

*homepage*: <https://www.gnu.org/software/gettext/>

version     |versionsuffix     |toolchain
------------|------------------|---------------------------------------------------------------------------------------------------
``0.18.2``  |                  |``system``
``0.19.4``  |                  |``GCC/4.9.2``, ``system``
``0.19.6``  |                  |``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``system``
``0.19.7``  |                  |``foss/2016a``, ``intel/2016a``, ``system``
``0.19.8``  |                  |``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016b``, ``intel/2016b``, ``system``
``0.19.8.1``|                  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``system``
``0.19.8.1``|``-libxml2-2.9.7``|``GCCcore/6.4.0``
``0.20.1``  |                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``0.21``    |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``system``
``0.21.1``  |                  |``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``system``

### gexiv2

gexiv2 is a GObject wrapper around the Exiv2 photo metadata library.

*homepage*: <https://wiki.gnome.org/Projects/gexiv2>

version   |toolchain
----------|------------------
``0.12.2``|``GCCcore/10.3.0``

### gfbf

GNU Compiler Collection (GCC) based compiler toolchain, including  FlexiBLAS (BLAS and LAPACK support) and (serial) FFTW.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2022a``|``system``
``2022b``|``system``
``2023a``|``system``

### GffCompare

GffCompare provides classification and reference annotation mapping and  matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files.

*homepage*: <https://ccb.jhu.edu/software/stringtie/gffcompare.shtml>

version   |toolchain
----------|------------------------------------
``0.10.1``|``foss/2016b``
``0.10.6``|``GCCcore/7.3.0``
``0.11.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.12.2``|``GCC/10.3.0``
``0.12.6``|``GCC/11.2.0``

### gffread

GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and  more.

*homepage*: <https://github.com/gpertea/gffread>

version   |toolchain
----------|----------------------------------------------------------
``0.9.12``|``foss/2016b``
``0.10.6``|``GCCcore/7.3.0``
``0.11.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.12.7``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/12.2.0``

### gflags

The gflags package contains a C++ library that implements commandline flags processing.  It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. 

*homepage*: <https://github.com/gflags/gflags>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------
``2.1.2``|``foss/2016a``
``2.2.1``|``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``2.2.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### GFOLD

Generalized fold change for ranking differentially expressed genes from RNA-seq data

*homepage*: <http://www.tongji.edu.cn/~zhanglab/GFOLD/index.html>

version  |toolchain
---------|---------------
``1.1.4``|``intel/2016a``

### gh

gh is GitHub on the command line.

*homepage*: <https://github.com/cli/cli>

version   |toolchain
----------|----------
``1.3.1`` |``system``
``2.20.2``|``system``

### GHC

The Glorious/Glasgow Haskell Compiler

*homepage*: <https://haskell.org/ghc/>

version   |versionsuffix|toolchain
----------|-------------|----------
``6.12.3``|             |``system``
``8.6.5`` |``-x86_64``  |``system``

### Ghostscript

Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to  different targets. It used to be part of the cups printing stack, but is no longer used for that.

*homepage*: <https://ghostscript.com>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------
``9.19``  |                  |``intel/2016a``, ``intel/2016b``
``9.20``  |                  |``foss/2016b``, ``intel/2016b``
``9.21``  |                  |``intel/2017a``
``9.22``  |                  |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``9.22``  |``-cairo-1.14.12``|``GCCcore/6.4.0``
``9.23``  |                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``9.23``  |``-cairo-1.14.12``|``GCCcore/6.4.0``
``9.27``  |                  |``GCCcore/8.2.0``
``9.50``  |                  |``GCCcore/8.3.0``
``9.52``  |                  |``GCCcore/9.3.0``
``9.53.3``|                  |``GCCcore/10.2.0``
``9.54.0``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``9.56.1``|                  |``GCCcore/11.3.0``
``10.0.0``|                  |``GCCcore/12.2.0``

### Gibbs2

Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary  conditions of temperature and pressure. Gibbs2 uses the results of periodic solid-state quantum-mechanical  calculations, specifically the energy-volume curve and possibly the harmonic phonon frequencies, to compute the  thermodynamic properties of the solid within the framework of the quasiharmonic approximation.

*homepage*: <https://aoterodelaroza.github.io/gibbs2/>

version|toolchain
-------|--------------
``1.0``|``GCC/10.3.0``

### giflib

giflib is a library for reading and writing gif images. It is API and ABI compatible with libungif which was in wide use while the LZW compression algorithm was patented.

*homepage*: <http://giflib.sourceforge.net/>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------------
``5.1.4``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``5.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### gifsicle

Gifsicle is a command-line tool for creating, editing,  and getting information about GIF images and animations.   Making a GIF animation with gifsicle is simple.

*homepage*: <https://github.com/kohler/gifsicle>

version |toolchain
--------|------------------
``1.92``|``GCCcore/8.2.0``
``1.93``|``GCCcore/11.3.0``

### GIMIC

The GIMIC program calculates magnetically induced currents in molecules.  You need to provide this program with a density matrix in atomic-orbital (AO) basis  and three (effective) magnetically perturbed AO density matrices in the proper format. Currently ACES2, Turbomole, G09, QChem, FERMION++, and LSDalton can produce these matrices.

*homepage*: <http://gimic.readthedocs.io>

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2018.04.20``|``-Python-2.7.14``|``intel/2018a``

### gimkl

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL

*homepage*: <(none)>

version   |toolchain
----------|----------
``2.11.5``|``system``
``2017a`` |``system``
``2018b`` |``system``

### GimmeMotifs

Suite of motif tools, including a motif prediction pipeline for ChIP-seq experiments

*homepage*: <https://github.com/vanheeringen-lab/gimmemotifs>

version   |toolchain
----------|--------------
``0.17.2``|``foss/2022a``

### GIMP

GIMP is a cross-platform image editor available for GNU/Linux, OS X, Windows and more operating systems.

*homepage*: <https://www.gimp.org>

version    |toolchain
-----------|--------------
``2.10.24``|``GCC/10.3.0``

### gimpi

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.

*homepage*: <(none)>

version   |toolchain
----------|----------
``2.11.5``|``system``
``2017a`` |``system``
``2017b`` |``system``
``2018a`` |``system``
``2018b`` |``system``

### gimpic

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including IntelMPI for MPI support with CUDA features enabled.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2017b``|``system``

### GIMPS

GIMPS: Great Internet Mersenne Prime Search;  it can be useful for limited stress testing of nodes, in user space

*homepage*: <http://www.mersenne.org/>

version            |toolchain
-------------------|-------------
``p95v279``        |``GCC/4.8.2``
``p95v279.linux64``|``system``

### giolf

GNU Compiler Collection (GCC) based compiler toolchain, including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``

### giolfc

GCC based compiler toolchain __with CUDA support__, and including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2017b``|``system``

### Giotto-Suite

Giotto Suite is focused on building a modular platform for analyzing spatial-omics technologies  and strives to be interoperable with other popular spatial analysis tools and classes. Using established packages  optimized for large(r) data, Giotto Suite adopts fast and memory efficient methods to create an interactive  analysis.

*homepage*: <https://github.com/drieslab/Giotto>

version  |versionsuffix|toolchain
---------|-------------|--------------
``3.0.1``|``-R-4.2.1`` |``foss/2022a``

### git

Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.

*homepage*: <http://git-scm.com/>

version    |versionsuffix|toolchain
-----------|-------------|------------------------------------
``1.8.5.6``|             |``GCC/4.9.2``
``2.2.2``  |             |``GCC/4.9.2``
``2.4.1``  |             |``GCC/4.9.2``
``2.8.0``  |             |``foss/2016a``
``2.12.2`` |             |``foss/2016b``
``2.13.1`` |             |``foss/2016b``
``2.14.1`` |             |``GCCcore/6.4.0``
``2.16.1`` |             |``foss/2018a``
``2.18.0`` |             |``GCCcore/7.3.0``
``2.19.1`` |             |``GCCcore/7.3.0``
``2.21.0`` |             |``GCCcore/8.2.0``
``2.21.0`` |``-nodocs``  |``GCCcore/8.2.0``
``2.23.0`` |             |``GCCcore/8.3.0``
``2.23.0`` |``-nodocs``  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.28.0`` |``-nodocs``  |``GCCcore/10.2.0``
``2.32.0`` |``-nodocs``  |``GCCcore/10.3.0``
``2.33.1`` |``-nodocs``  |``GCCcore/11.2.0``
``2.36.0`` |``-nodocs``  |``GCCcore/11.3.0``
``2.38.1`` |``-nodocs``  |``GCCcore/12.2.0``
``2.41.0`` |``-nodocs``  |``GCCcore/12.3.0``

### git-extras

Extra useful scripts for git

*homepage*: <https://github.com/tj/git-extras>

version  |toolchain
---------|--------------
``5.1.0``|``foss/2016a``

### git-lfs

Git Large File Storage (LFS) replaces large files such as audio  samples, videos, datasets, and graphics with text pointers inside Git, while  storing the file contents on a remote server like GitHub.com

*homepage*: <https://git-lfs.github.com>

version   |toolchain
----------|----------
``1.1.1`` |``system``
``2.7.1`` |``system``
``2.11.0``|``system``
``3.2.0`` |``system``

### GitPython

GitPython is a python library used to interact with Git repositories 

*homepage*: <https://github.com/gitpython-developers/GitPython>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``2.1.11``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``2.1.15``|                 |``system``
``3.0.3`` |``-Python-3.7.2``|``GCCcore/8.2.0``
``3.1.0`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.1.9`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``3.1.14``|                 |``GCCcore/10.2.0``
``3.1.18``|                 |``GCCcore/10.3.0``
``3.1.24``|                 |``GCCcore/11.2.0``
``3.1.27``|                 |``GCCcore/11.3.0``
``3.1.31``|                 |``GCCcore/12.2.0``

### Givaro

C++ library for arithmetic and algebraic computations

*homepage*: <http://givaro.forge.imag.fr/>

version  |toolchain
---------|--------------
``4.0.1``|``foss/2016a``

### Giza

Giza is an open, lightweight scientific plotting library built on  top of cairo that provides uniform output to multiple devices.

*homepage*: <https://danieljprice.github.io/giza/>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2018b``

### GKeyll

Gkeyll v2.0 (pronounced as in the book “The Strange Case of Dr. Jekyll  and Mr. Hyde”) is a computational plasma physics code mostly written in C/C++ and LuaJIT.  Gkeyll contains solvers for gyrokinetic equations, Vlasov-Maxwell equations, and  multi-fluid equations. Gkeyll contains ab-initio and novel implementations of a number  of algorithms, and perhaps is unique in using a JIT compiled typeless dynamic language  for as its main implementation language.

*homepage*: <https://gkeyll.readthedocs.io>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20220803``|``-Python-3.8.2``|``foss/2020a``

### GL2PS

GL2PS: an OpenGL to PostScript printing library

*homepage*: <https://www.geuz.org/gl2ps/>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------------------
``1.3.9``|                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.9``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``1.4.0``|                |``GCCcore/8.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``1.4.2``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/9.3.0``

### Glade

Glade is a RAD tool to enable quick & simple development of user interfaces for the GTK+ toolkit  and the GNOME desktop environment.

*homepage*: <https://glade.gnome.org/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.8.5``|``-Python-2.7.11``|``intel/2016a``
``3.8.5``|``-Python-2.7.14``|``intel/2017b``
``3.8.6``|``-Python-2.7.14``|``intel/2018a``
``3.8.6``|``-Python-2.7.15``|``foss/2018b``

### glew

The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform.

*homepage*: <https://github.com/nigels-com/glew>

version  |versionsuffix|toolchain
---------|-------------|---------------------------------------------------------------------------------------------------------------
``2.1.0``|             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``2.2.0``|``-egl``     |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.2.0``|``-glx``     |``GCCcore/10.2.0``, ``GCCcore/11.2.0``
``2.2.0``|``-osmesa``  |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### GLFW

GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop

*homepage*: <https://www.glfw.org>

version  |toolchain
---------|--------------------------------------
``3.2.1``|``intel/2018a``
``3.3.2``|``GCCcore/9.3.0``
``3.3.3``|``GCCcore/10.2.0``
``3.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.3.8``|``GCCcore/11.3.0``

### GLI

Graphics Language Interpreter

*homepage*: <https://pgi-jcns.fz-juelich.de/portal/pages/gli.html>

version   |toolchain
----------|--------------------------------------
``4.5.31``|``GCCcore/10.2.0``, ``GCCcore/12.2.0``

### GLib

GLib is one of the base libraries of the GTK+ project

*homepage*: <https://www.gtk.org/>

version   |toolchain
----------|-------------------------------------------------
``2.42.1``|``GCC/4.9.2``
``2.44.0``|``GCC/4.9.2``
``2.47.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.48.0``|``foss/2016a``, ``intel/2016a``
``2.49.5``|``foss/2016b``, ``intel/2016b``
``2.52.0``|``foss/2017a``, ``intel/2017a``
``2.53.5``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``2.54.2``|``GCCcore/6.4.0``
``2.54.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.60.1``|``GCCcore/8.2.0``
``2.62.0``|``GCCcore/8.3.0``
``2.64.1``|``GCCcore/9.3.0``
``2.66.1``|``GCCcore/10.2.0``
``2.68.2``|``GCCcore/10.3.0``
``2.69.1``|``GCCcore/11.2.0``
``2.72.1``|``GCCcore/11.3.0``
``2.75.0``|``GCCcore/12.2.0``

### glib-networking

Network extensions for GLib

*homepage*: <https://gitlab.gnome.org/GNOME/glib-networking>

version   |toolchain
----------|--------------------------------------
``2.68.1``|``GCCcore/10.3.0``
``2.72.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### glibc

The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems,  as well as many other systems that use Linux as the kernel.

*homepage*: <https://www.gnu.org/software/libc/>

version |toolchain
--------|-----------------
``2.17``|``GCCcore/6.4.0``
``2.26``|``GCCcore/6.4.0``
``2.30``|``GCCcore/8.3.0``

### GLibmm

C++ bindings for Glib

*homepage*: <https://www.gtk.org/>

version   |toolchain
----------|-------------------------------------------------------
``2.49.7``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.66.4``|``GCCcore/10.3.0``

### GLIMMER

Glimmer is a system for finding genes in microbial DNA, especially  the genomes of bacteria, archaea, and viruses.

*homepage*: <http://ccb.jhu.edu/software/glimmer/index.shtml>

version  |toolchain
---------|------------------------------
``3.02b``|``foss/2016b``, ``foss/2018b``

### GlimmerHMM

GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model.  Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally  it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted  from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models.

*homepage*: <https://ccb.jhu.edu/software/glimmerhmm>

version   |toolchain
----------|---------------------------------------------
``3.0.4`` |``foss/2016b``, ``foss/2018b``
``3.0.4c``|``GCC/10.2.0``, ``GCC/11.2.0``, ``GCC/8.3.0``

### GLIMPSE

GLIMPSE2 is a set of tools for phasing and imputation for low-coverage sequencing datasets

*homepage*: <https://github.com/odelaneau/GLIMPSE>

version  |toolchain
---------|----------------------------------------------
``2.0.0``|``GCC/10.3.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### GLM

OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on  the OpenGL Shading Language (GLSL) specifications.

*homepage*: <https://github.com/g-truc/glm>

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------
``0.9.7.6``|``intel/2016a``
``0.9.8.3``|``GCCcore/5.4.0``, ``GCCcore/7.3.0``
``0.9.9.0``|``GCCcore/6.4.0``
``0.9.9.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

### GLM-AED

The General Lake Model (GLM) is a water balance and one-dimensional vertical stratification hydrodynamic model, which is dynamically coupled with the AED water quality modelling library. GLM-AED is suitable for simulating conditions in a wide range of natural and engineered lakes, including shallow (well-mixed) and deep (stratified) systems. The model has been successfully applied to systems from the scale of individual ponds and wetlands, to actively operated reservoirs, upto the scale of the Great Lakes.

*homepage*: <https://github.com/AquaticEcoDynamics/glm-aed>

version    |toolchain
-----------|---------------
``3.3.0a5``|``gompi/2021b``

### GlobalArrays

Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model

*homepage*: <https://hpc.pnl.gov/globalarrays>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------
``5.7``  |             |``intel/2018b``
``5.7``  |``-peigs``   |``intel/2019a``
``5.7.2``|             |``intel/2019b``
``5.7.2``|``-peigs``   |``intel/2019b``
``5.8``  |             |``intel/2020a``, ``intel/2021a``, ``iomkl/2021a``
``5.8.1``|             |``intel/2022a``
``5.8.2``|             |``intel/2022a``

### Globus-CLI

A Command Line Wrapper over the Globus SDK for Python, which provides an interface to Globus services  from the shell, and is suited to both interactive and simple scripting use cases.

*homepage*: <https://docs.globus.org/cli/>

version   |toolchain
----------|------------------
``1.11.0``|``GCCcore/8.3.0``
``3.1.1`` |``GCCcore/10.2.0``
``3.2.0`` |``GCCcore/10.3.0``
``3.6.0`` |``GCCcore/11.2.0``

### GlobusConnectPersonal

Globus Connect Personal turns your laptop or other personal computer into a  Globus endpoint with a just a few clicks. With Globus Connect Personal you can  share and transfer files to/from a local machine—campus server, desktop computer or laptop—even if it's behind a firewall and you don't have administrator  privileges. 

*homepage*: <https://www.globus.org/globus-connect-personal>

version  |toolchain
---------|----------
``2.3.6``|``system``

### glog

A C++ implementation of the Google logging module.

*homepage*: <https://github.com/google/glog>

version  |toolchain
---------|-------------------------------------------------------
``0.3.4``|``foss/2016a``
``0.3.5``|``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``0.4.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.5.0``|``GCCcore/10.2.0``

### GLPK

The GLPK (GNU Linear Programming Kit) package is intended for  solving large-scale linear programming (LP),  mixed integer programming (MIP), and other related problems.  It is a set of routines written in ANSI C  and organized in the form of a callable library.

*homepage*: <https://www.gnu.org/software/glpk/>

version |toolchain
--------|-----------------------------------------------------------------------------------------------------------------
``4.58``|``foss/2016a``, ``intel/2016a``
``4.60``|``GCCcore/5.4.0``, ``intel/2016b``
``4.61``|``intel/2017a``
``4.65``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.0`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### glproto

X protocol and ancillary headers

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version   |toolchain
----------|-------------------------------------------------
``1.4.17``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### Glucose

Glucose is based on a new scoring scheme (well, not so new now, it was introduced in 2009) for the clause learning mechanism of so called Modern SAT solvers (it is based on our IJCAI'09 paper). It is designed to be parallel, since v4.0.

*homepage*: <https://www.labri.fr/perso/lsimon/glucose/>

version|toolchain
-------|-------------
``4.1``|``GCC/9.3.0``

### GMAP-GSNAP

GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences  GSNAP: Genomic Short-read Nucleotide Alignment Program

*homepage*: <http://research-pub.gene.com/gmap/>

version       |toolchain
--------------|---------------
``2016-05-01``|``foss/2016a``
``2016-11-07``|``foss/2016b``
``2018-05-11``|``intel/2018a``
``2018-07-04``|``intel/2018a``
``2019-03-15``|``foss/2018b``
``2019-09-12``|``GCC/8.3.0``
``2020-12-17``|``GCC/9.3.0``
``2021-12-17``|``GCC/11.2.0``
``2023-02-17``|``GCC/11.3.0``
``2023-04-20``|``GCC/12.2.0``

### GMP

GMP is a free library for arbitrary precision arithmetic, operating on signed  integers, rational numbers, and floating point numbers. 

*homepage*: <https://gmplib.org/>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------
``4.3.2`` |``system``
``5.1.3`` |``GCC/4.8.2``
``6.0.0`` |``GCC/4.9.2``
``6.0.0a``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``6.1.0`` |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``6.1.1`` |``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``6.1.2`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``
``6.2.0`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``6.2.1`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### gmpich

gcc and GFortran based compiler toolchain,  including MPICH for MPI support.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2016a``  |``system``
``2017.08``|``system``

### gmpolf

gcc and GFortran based compiler toolchain,  MPICH for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2016a``  |``system``
``2017.10``|``system``

### gmpy2

GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x

*homepage*: <https://github.com/aleaxit/gmpy>

version    |versionsuffix     |toolchain
-----------|------------------|----------------------------------------------------------------------
``2.0.8``  |``-Python-2.7.13``|``intel/2017a``
``2.0.8``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.0.8``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0b1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.1.0b1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0b4``|                  |``GCC/8.3.0``
``2.1.0b5``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``, ``iccifort/2020.4.304``
``2.1.2``  |                  |``GCC/11.2.0``, ``GCC/11.3.0``, ``intel-compilers/2022.1.0``

### gmsh

Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor.

*homepage*: <https://gmsh.info/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``3.0.6`` |``-Python-2.7.14``|``foss/2017b``
``3.0.6`` |``-Python-3.6.6`` |``foss/2018b``
``4.2.2`` |``-Python-3.6.6`` |``foss/2018b``
``4.5.6`` |``-Python-2.7.16``|``intel/2019b``
``4.5.6`` |``-Python-3.7.4`` |``foss/2019b``
``4.7.1`` |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``4.8.4`` |``-Python-3.6.4`` |``foss/2018a``
``4.9.0`` |                  |``foss/2021a``
``4.11.1``|                  |``foss/2022a``

### GMT

GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools. 

*homepage*: <https://www.generic-mapping-tools.org/>

version   |toolchain
----------|-------------------------------
``4.5.17``|``foss/2018a``
``5.4.1`` |``intel/2017a``
``5.4.3`` |``foss/2018a``, ``intel/2017b``
``5.4.5`` |``foss/2019a``
``6.2.0`` |``foss/2019b``, ``foss/2020b``

### gmvapich2

GNU Compiler Collection (GCC) based compiler toolchain,  including MVAPICH2 for MPI support.

*homepage*: <(none)>

version   |toolchain
----------|----------
``1.7.20``|``system``
``2016a`` |``system``

### gmvolf

GNU Compiler Collection (GCC) based compiler toolchain, including  MVAPICH2 for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version   |toolchain
----------|----------
``1.7.20``|``system``
``2016a`` |``system``

### GNU

Compiler-only toolchain with GCC and binutils.

*homepage*: <http://www.gnu.org/software/>

version       |toolchain
--------------|----------
``4.9.2-2.25``|``system``
``4.9.3-2.25``|``system``
``5.1.0-2.25``|``system``

### gnuplot

Portable interactive, function plotting utility

*homepage*: <http://gnuplot.sourceforge.net>

version  |toolchain
---------|----------------------------------------------------------------
``5.0.3``|``foss/2016a``, ``intel/2016a``
``5.0.5``|``foss/2016b``, ``intel/2016b``
``5.0.6``|``intel/2017a``
``5.2.2``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``5.2.5``|``foss/2018b``
``5.2.6``|``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``
``5.2.8``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.4.1``|``GCCcore/10.2.0``
``5.4.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.4.4``|``GCCcore/11.3.0``
``5.4.6``|``GCCcore/12.2.0``

### GnuTLS

GnuTLS is a secure communications library implementing the SSL, TLS  and DTLS protocols and technologies around them. It provides a simple  C language application programming interface (API) to access the secure  communications protocols as well as APIs to parse and write X.509, PKCS #12,  OpenPGP and other required structures. It is aimed to be portable  and efficient with focus on security and interoperability.

*homepage*: <https://www.gnutls.org>

version   |toolchain
----------|------------------
``3.3.21``|``intel/2016a``
``3.4.7`` |``GNU/4.9.3-2.25``
``3.4.11``|``foss/2016a``
``3.7.2`` |``GCCcore/10.3.0``
``3.7.3`` |``GCCcore/11.2.0``
``3.7.8`` |``GCCcore/11.3.0``

### Go

Go is an open source programming language that makes it simple to build  simple, reliable, and efficient software.

*homepage*: <http://www.golang.org>

version   |toolchain
----------|-----------------
``1.2.1`` |``GCC/4.8.2``
``1.4.2`` |``GCC/4.8.4``
``1.5``   |``GCC/4.8.4``
``1.8.1`` |``system``
``1.11.5``|``system``
``1.12``  |``system``
``1.12.1``|``GCCcore/7.3.0``
``1.13.1``|``system``
``1.14``  |``system``
``1.14.1``|``system``
``1.16.3``|``system``
``1.16.5``|``system``
``1.16.6``|``system``
``1.17.3``|``system``
``1.17.6``|``system``
``1.18.1``|``system``
``1.18.3``|``system``
``1.20.4``|``system``

### goalign

Goalign is a set of command line tools to manipulate multiple alignments.

*homepage*: <https://github.com/evolbioinfo/goalign>

version  |toolchain
---------|----------
``0.3.2``|``system``

### GOATOOLS

A Python library for Gene Ontology analyses

*homepage*: <https://github.com/tanghaibao/goatools>

version  |toolchain
---------|------------------------------
``1.1.6``|``foss/2020b``
``1.3.1``|``foss/2021b``, ``foss/2022a``

### gobff

GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.

*homepage*: <(none)>

version    |versionsuffix|toolchain
-----------|-------------|----------
``2020.06``|``-amd``     |``system``
``2020.11``|             |``system``
``2020b``  |             |``system``
``2021a``  |             |``system``

### GObject-Introspection

GObject introspection is a middleware layer between C libraries  (using GObject) and language bindings. The C library can be scanned at  compile time and generate a metadata file, in addition to the actual  native C library. Then at runtime, language bindings can read this  metadata and automatically provide bindings to call into the C library.

*homepage*: <https://gi.readthedocs.io/en/latest/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``1.47.1``|                  |``foss/2016a``, ``intel/2016a``
``1.48.0``|                  |``foss/2016a``, ``intel/2016a``
``1.49.1``|                  |``foss/2016b``, ``intel/2016b``
``1.52.0``|                  |``intel/2017a``
``1.53.5``|``-Python-2.7.13``|``intel/2017a``
``1.53.5``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.53.5``|``-Python-3.6.1`` |``intel/2017a``
``1.54.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.54.1``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``
``1.54.1``|``-Python-3.6.6`` |``fosscuda/2018b``
``1.58.3``|``-Python-2.7.16``|``GCCcore/8.3.0``
``1.60.1``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``1.63.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.64.0``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``1.66.1``|                  |``GCCcore/10.2.0``
``1.68.0``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.72.0``|                  |``GCCcore/11.3.0``
``1.74.0``|                  |``GCCcore/12.2.0``

### goblf

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, BLIS (BLAS support), LAPACK, FFTW and ScaLAPACK.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2018b``|``system``
``2020b``|``system``

### GOBNILP

GOBNILP (Globally Optimal Bayesian Network learning using Integer Linear Programming) is a C program which learns Bayesian networks from complete discrete data or from local scores.

*homepage*: <https://www.cs.york.ac.uk/aig/sw/gobnilp/>

version  |toolchain
---------|--------------
``1.6.3``|``GCC/11.3.0``

### Godon

Godon is codon models software written in Go.

*homepage*: <https://bitbucket.org/Davydov/godon/>

version     |versionsuffix|toolchain
------------|-------------|----------
``20210913``|``-x86_64``  |``system``

### gofasta

Some functions for dealing with alignments,  developed to handle SARS-CoV-2 data as part of the COG-UK project.

*homepage*: <https://github.com/cov-ert/gofasta>

version  |toolchain
---------|----------
``0.0.5``|``system``

### golf

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenBLAS (BLAS and LAPACK support) and FFTW.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2018a``|``system``
``2020a``|``system``

### gomkl

GNU Compiler Collection (GCC) based compiler toolchain with OpenMPI and MKL

*homepage*: <(none)>

version  |toolchain
---------|----------
``2018b``|``system``
``2019a``|``system``
``2020a``|``system``
``2020b``|``system``
``2021a``|``system``
``2021b``|``system``
``2022a``|``system``

### gompi

GNU Compiler Collection (GCC) based compiler toolchain,  including OpenMPI for MPI support.

*homepage*: <(none)>

version    |versionsuffix|toolchain
-----------|-------------|----------
``2016.04``|             |``system``
``2016.06``|             |``system``
``2016.07``|             |``system``
``2016.09``|             |``system``
``2016a``  |             |``system``
``2016b``  |             |``system``
``2017a``  |             |``system``
``2017b``  |             |``system``
``2018.08``|             |``system``
``2018a``  |             |``system``
``2018b``  |             |``system``
``2019a``  |             |``system``
``2019b``  |             |``system``
``2020a``  |             |``system``
``2020b``  |             |``system``
``2021a``  |             |``system``
``2021b``  |             |``system``
``2022.05``|             |``system``
``2022.10``|             |``system``
``2022a``  |             |``system``
``2022b``  |             |``system``
``2023a``  |             |``system``
``system`` |``-2.29``    |``system``

### gompic

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including OpenMPI for MPI support with CUDA features enabled.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``
``2018b``|``system``
``2019a``|``system``
``2019b``|``system``
``2020a``|``system``
``2020b``|``system``

### google-java-format

Reformats Java source code to comply with Google Java Style.

*homepage*: <https://github.com/google/google-java-format>

version|versionsuffix|toolchain
-------|-------------|----------
``1.7``|``-Java-1.8``|``system``

### googletest

Google's framework for writing C++ tests on a variety of platforms

*homepage*: <https://github.com/google/googletest>

version   |toolchain
----------|---------------------------------------------------------------------
``1.8.0`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``
``1.8.1`` |``GCCcore/8.2.0``
``1.10.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.11.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.12.1``|``GCCcore/12.2.0``
``1.13.0``|``GCCcore/12.3.0``

### gotree

GoTree is a set of command line tools to manipulate phylogenetic trees.

*homepage*: <https://github.com/evolbioinfo/gotree>

version  |toolchain
---------|----------
``0.4.0``|``system``

### GP2C

The gp2c compiler is a package for translating GP routines into the C programming language, so that  they can  be compiled and used with the PARI system or the GP calculator. 

*homepage*: <http://pari.math.u-bordeaux.fr/pub/pari/manuals/gp2c/gp2c.html>

version     |toolchain
------------|--------------
``0.0.9pl5``|``foss/2016a``

### GPAW

GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)  method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or  atom-centered basis-functions.

*homepage*: <https://wiki.fysik.dtu.dk/gpaw/>

version    |versionsuffix               |toolchain
-----------|----------------------------|----------------------------------------------------------------
``1.4.0``  |``-Python-3.6.6``           |``foss/2018b``, ``intel/2018b``
``19.8.1`` |``-ASE-3.18.0-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``19.8.1`` |``-Python-3.7.2``           |``foss/2019a``, ``intel/2019a``
``20.1.0`` |``-Python-3.7.4``           |``foss/2019b``, ``intel/2019b``
``20.10.0``|                            |``foss/2020b``, ``intel/2020b``
``20.10.0``|``-ASE-3.20.1-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``21.1.0`` |``-ASE-3.21.1``             |``foss/2020b``, ``intel/2020b``
``21.6.0`` |                            |``foss/2021a``
``21.6.0`` |``-ASE-3.22.0``             |``foss/2020b``, ``intel/2020b``
``22.8.0`` |                            |``foss/2021b``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

### GPAW-setups

PAW setup for the GPAW Density Functional Theory package.   Users can install setups manually using 'gpaw install-data' or use setups from this package.   The versions of GPAW and GPAW-setups can be intermixed.

*homepage*: <https://wiki.fysik.dtu.dk/gpaw/>

version      |toolchain
-------------|----------
``0.8.7929`` |``system``
``0.9.9672`` |``system``
``0.9.11271``|``system``
``0.9.20000``|``system``

### gperf

GNU gperf is a perfect hash function generator. For a given list of strings,  it produces a hash function and hash table, in form of C or C++ code, for  looking up a value depending on the input string. The hash function is  perfect, which means that the hash table has no collisions, and the hash  table lookup needs a single string comparison only. 

*homepage*: <https://www.gnu.org/software/gperf/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0.4``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``3.1``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### gperftools

gperftools is a collection of a high-performance multi-threaded malloc() implementation, plus some pretty nifty performance analysis tools. Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler. 

*homepage*: <https://github.com/gperftools/gperftools>

version   |toolchain
----------|----------------------------------------------------------
``2.5``   |``foss/2016a``, ``intel/2016b``
``2.6.3`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.7.90``|``GCCcore/8.3.0``
``2.8``   |``GCCcore/9.3.0``
``2.9.1`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.10``  |``GCCcore/11.3.0``

### gpustat

A simple command-line utility for querying and monitoring GPU status

*homepage*: <https://github.com/wookayin/gpustat>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------------
``0.5.0``  |``-Python-2.7.15``|``fosscuda/2018b``
``0.6.0``  |                  |``GCCcore/10.3.0``, ``gcccuda/2020b``
``0.6.0``  |``-Python-3.7.4`` |``fosscuda/2019b``
``1.0.0b1``|                  |``GCCcore/11.2.0``
``1.1``    |                  |``GCCcore/11.3.0``

### GPy

GPy is a Gaussian Process (GP) framework written in Python

*homepage*: <https://sheffieldml.github.io/GPy>

version   |toolchain
----------|--------------
``1.10.0``|``foss/2021b``

### GPyOpt

GPyOpt is a Python open-source library for Bayesian Optimization

*homepage*: <https://sheffieldml.github.io/GPyOpt>

version  |toolchain
---------|---------------
``1.2.6``|``intel/2020b``

### GPyTorch

GPyTorch is a Gaussian process library implemented using PyTorch.

*homepage*: <https://gpytorch.ai>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.3.0``|                |``foss/2020b``
``1.9.1``|``-CUDA-11.3.1``|``foss/2021a``

### Grace

Grace is a WYSIWYG tool to make two-dimensional plots of numerical data.

*homepage*: <https://plasma-gate.weizmann.ac.il/Grace/>

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------------------------------------------
``5.1.25``|             |``foss/2016a``, ``foss/2021a``, ``foss/2021b``, ``intel/2016a``
``5.1.25``|``-5build1`` |``foss/2017b``, ``foss/2018a``, ``foss/2019a``, ``foss/2019b``, ``intel/2017b``, ``intel/2019a``, ``intel/2019b``

### Gradle

Complete Gradle install. From mobile apps to microservices, from small startups to big enterprises, Gradle helps teams build, automate and deliver better software, faster. 

*homepage*: <https://gradle.org>

version  |toolchain
---------|----------
``4.5.1``|``system``
``6.1.1``|``system``
``6.9.1``|``system``

### gradunwarp

Gradient Unwarping. This is the Human Connectome Project fork of the no longer maintained original.

*homepage*: <https://github.com/Washington-University/gradunwarp>

version  |versionsuffix         |toolchain
---------|----------------------|--------------
``1.1.0``|``-HCP-Python-2.7.15``|``foss/2019a``
``1.2.0``|``-HCP-Python-2.7.15``|``foss/2019a``
``1.2.0``|``-HCP-Python-3.7.2`` |``foss/2019a``

### graph-tool

Graph-tool is an efficient Python module for manipulation and  statistical analysis of graphs (a.k.a. networks). Contrary to  most other python modules with similar functionality, the core  data structures and algorithms are implemented in C++, making  extensive use of template metaprogramming, based heavily on  the Boost Graph Library. This confers it a level of  performance that is comparable (both in memory usage and  computation time) to that of a pure C/C++ library.

*homepage*: <https://graph-tool.skewed.de/>

version |versionsuffix    |toolchain
--------|-----------------|--------------
``2.26``|``-Python-3.6.3``|``foss/2017b``
``2.27``|``-Python-3.6.6``|``foss/2018b``

### GraphDB

GraphDB is an enterprise ready Semantic Graph Database, compliant with W3C Standards. Semantic graph databases (also called RDF triplestores) provide the core infrastructure for solutions where modelling agility, data integration, relationship exploration and cross-enterprise data publishing and consumption are important.

*homepage*: <https://graphdb.ontotext.com/>

version   |toolchain
----------|----------
``10.1.5``|``system``

### Graphene

Graphene is a a thin layer of types for graphic libraries

*homepage*: <https://ebassi.github.io/graphene/>

version   |toolchain
----------|--------------------------------------
``1.6.0`` |``intel/2017a``
``1.10.8``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### GraphicsMagick

GraphicsMagick is the swiss army knife of image processing.

*homepage*: <https://www.graphicsmagick.org/>

version   |toolchain
----------|----------------------------------
``1.3.23``|``foss/2016a``, ``intel/2016a``
``1.3.25``|``intel/2016b``, ``intel/2017a``
``1.3.28``|``foss/2018a``
``1.3.31``|``foss/2018b``
``1.3.34``|``foss/2019a``, ``foss/2019b``
``1.3.36``|``GCCcore/11.2.0``, ``foss/2020b``

### graphite2

Graphite is a "smart font" system developed specifically to  handle the complexities of lesser-known languages of the world.

*homepage*: <https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------
``1.3.14``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/8.3.0``

### GraPhlAn

GraPhlAn is a software tool for producing high-quality circular representations  of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative,  and publication-ready representations of phylogenetically- and taxonomically-driven investigation. 

*homepage*: <https://segatalab.github.io/tools/graphlan>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.3``|``-Python-2.7.16``|``foss/2019b``

### GraphMap

A highly sensitive and accurate mapper for long, error-prone reads

*homepage*: <https://github.com/isovic/graphmap>

version  |toolchain
---------|--------------
``0.5.2``|``foss/2019b``

### GraphMap2

A highly sensitive and accurate mapper for long, error-prone reads

*homepage*: <https://github.com/lbcb-sci/graphmap2>

version  |toolchain
---------|--------------
``0.6.4``|``foss/2019b``

### Graphviz

Graphviz is open source graph visualization software. Graph visualization  is a way of representing structural information as diagrams of  abstract graphs and networks. It has important applications in networking,  bioinformatics,  software engineering, database and web design, machine learning,  and in visual interfaces for other technical domains.

*homepage*: <https://www.graphviz.org/>

version   |versionsuffix            |toolchain
----------|-------------------------|-------------------------------
``2.38.0``|                         |``foss/2016b``, ``intel/2016b``
``2.40.1``|                         |``foss/2018b``, ``intel/2018a``
``2.40.1``|``-Python-2.7.15``       |``foss/2018b``
``2.42.2``|                         |``foss/2019b``
``2.42.2``|``-Java-11``             |``GCCcore/8.3.0``
``2.42.2``|``-Python-3.7.4``        |``foss/2019b``
``2.44.1``|``-Java-11``             |``GCCcore/9.3.0``
``2.44.1``|``-Java-11-Python-3.8.2``|``GCCcore/9.3.0``
``2.47.0``|``-Java-11``             |``GCCcore/10.2.0``
``2.47.2``|                         |``GCCcore/10.3.0``
``2.50.0``|                         |``GCCcore/11.2.0``
``5.0.0`` |                         |``GCCcore/11.3.0``

### graphviz-python

Simple Python interface for Graphviz

*homepage*: <https://pypi.python.org/pypi/graphviz>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.5.1`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.5.1`` |``-Python-3.5.2`` |``intel/2016b``
``0.8.2`` |``-Python-3.6.4`` |``intel/2018a``
``0.20.1``|                  |``GCCcore/11.3.0``

### GRASP

The General Relativistic Atomic Structure Package (GRASP) is a set of  Fortran 90 programs for performing fully-relativistic electron structure  calculations of atoms.

*homepage*: <https://compas.github.io/grasp/>

version |toolchain
--------|--------------
``2018``|``foss/2019b``

### GRASS

The Geographic Resources Analysis Support System - used  for geospatial data management and analysis, image processing,  graphics and maps production, spatial modeling, and visualization

*homepage*: <https://grass.osgeo.org>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``7.6.0``|``-Python-2.7.15``|``foss/2018b``
``7.8.3``|``-Python-3.7.4`` |``fosscuda/2019b``
``8.2.0``|                  |``foss/2021b``

### Greenlet

The greenlet package is a spin-off of Stackless, a version of CPython that supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single or a few OS-level threads) and are synchronized with data exchanges on "channels". A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs. 

*homepage*: <https://github.com/python-greenlet/greenlet>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.4.9`` |``-Python-2.7.11``|``intel/2016a``
``0.4.9`` |``-Python-3.5.1`` |``intel/2016a``
``0.4.11``|``-Python-2.7.12``|``intel/2016b``
``0.4.12``|``-Python-2.7.14``|``intel/2017b``
``2.0.2`` |                  |``foss/2022a``

### Grep

The grep command searches one or more input files for lines containing a match to a specified pattern.  By default, grep prints the matching lines.

*homepage*: <http://www.gnu.org/software/grep/>

version |toolchain
--------|-------------
``2.21``|``GCC/4.9.2``

### gretl

A cross-platform software package for econometric analysis

*homepage*: <http://gretl.sourceforge.net>

version  |toolchain
---------|--------------
``2020a``|``foss/2019a``

### grib_api

The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python  programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of  command line tools is also provided to give quick access to GRIB messages.

*homepage*: <https://software.ecmwf.int/wiki/display/GRIB/Home>

version   |toolchain
----------|------------------------------------------------
``1.16.0``|``intel/2016a``
``1.21.0``|``foss/2017a``
``1.24.0``|``foss/2017b``, ``intel/2017a``, ``intel/2017b``

### grid

Grid is a free and open-source Python library for numerical integration,  interpolation and differentiation of interest for the quantum chemistry community.

*homepage*: <https://grid.qcdevs.org/>

version     |toolchain
------------|---------------
``20220610``|``intel/2022a``

### GRIDSS

GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements. GRIDSS includes a genome-wide break-end assembler, as well as a structural variation caller for Illumina sequencing data. GRIDSS calls variants based on alignment-guided positional de Bruijn graph genome-wide break-end assembly, split read, and read pair evidence.

*homepage*: <https://github.com/PapenfussLab/gridss/wiki/GRIDSS-Documentation>

version   |versionsuffix|toolchain
----------|-------------|--------------
``2.13.2``|``-Java-11`` |``foss/2021b``

### GRIT

GRIT - A tool for the integrative analysis of RNA-seq type assays

*homepage*: <https://github.com/nboley/grit>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.0.5``|``-Python-2.7.12``|``intel/2016b``

### GRNBoost

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: <https://github.com/dmlc/xgboost>

version     |versionsuffix      |toolchain
------------|-------------------|---------------
``20171009``|``-Java-1.8.0_152``|``intel/2017b``

### groff

Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands  and produces formatted output.

*homepage*: <https://www.gnu.org/software/groff>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.22.4``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### GroIMP

GroIMP (Growth Grammar-related Interactive Modelling Platform) is a 3D-modelling platform.

*homepage*: <http://www.grogra.de/software/groimp>

version|toolchain
-------|----------
``1.5``|``system``

### GROMACS

GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.

*homepage*: <http://www.gromacs.org>

version   |versionsuffix                |toolchain
----------|-----------------------------|------------------------------------------------------------------------------------------------------------
``5.1.2`` |``-hybrid``                  |``foss/2016a``, ``intel/2016a``
``5.1.2`` |``-hybrid-dp``               |``intel/2016a``
``5.1.2`` |``-mt``                      |``foss/2016a``
``5.1.4`` |``-hybrid``                  |``foss/2016b``
``5.1.4`` |``-mt``                      |``foss/2016b``
``2016``  |``-hybrid``                  |``foss/2016b``
``2016``  |``-mt``                      |``foss/2016b``
``2016.1``|``-PLUMED``                  |``foss/2017a``
``2016.2``|                             |``foss/2017a``
``2016.3``|                             |``foss/2017a``, ``intel/2017a``
``2016.3``|``-GPU-enabled``             |``foss/2016b``
``2016.4``|                             |``foss/2017b``, ``fosscuda/2017b``, ``giolf/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``2016.5``|                             |``intel/2018a``
``2018``  |                             |``foss/2018a``
``2018.1``|``-PLUMED``                  |``foss/2018b``
``2018.2``|                             |``foss/2017b``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2017b``, ``intelcuda/2017b``
``2018.3``|                             |``foss/2018b``, ``fosscuda/2018b``
``2018.4``|``-PLUMED-2.5.0``            |``foss/2018b``, ``fosscuda/2018b``
``2019``  |                             |``foss/2018b``, ``fosscuda/2018b``
``2019.2``|                             |``fosscuda/2019a``
``2019.3``|                             |``foss/2019a``, ``foss/2019b``, ``fosscuda/2019a``, ``fosscuda/2019b``
``2019.4``|                             |``foss/2019b``, ``fosscuda/2019b``
``2019.4``|``-PLUMED-2.5.4``            |``foss/2019b``, ``fosscuda/2019b``
``2019.6``|                             |``fosscuda/2019b``
``2020``  |                             |``foss/2019b``, ``fosscuda/2019b``
``2020.1``|``-Python-3.8.2``            |``foss/2020a``
``2020.3``|                             |``fosscuda/2019b``
``2020.4``|``-Python-3.8.2``            |``foss/2020a``
``2020.5``|``-Python-3.8.2``            |``fosscuda/2020a``
``2021``  |                             |``foss/2020b``, ``fosscuda/2020b``
``2021.2``|                             |``fosscuda/2020b``
``2021.3``|                             |``foss/2021a``
``2021.3``|``-CUDA-11.3.1``             |``foss/2021a``
``2021.3``|``-CUDA-11.3.1-PLUMED-2.7.2``|``foss/2021a``
``2021.3``|``-PLUMED-2.7.2``            |``foss/2021a``
``2021.5``|                             |``foss/2021b``
``2021.5``|``-CUDA-11.4.1``             |``foss/2021b``
``2021.5``|``-CUDA-11.4.1-PLUMED-2.8.0``|``foss/2021b``
``2021.5``|``-PLUMED-2.8.0``            |``foss/2021b``
``2023.1``|                             |``foss/2022a``
``2023.1``|``-CUDA-11.7.0``             |``foss/2022a``

### GromacsWrapper

GromacsWrapper is a python package that wraps system calls to Gromacs tools into thin classes.  This allows for fairly seamless integration of the gromacs tools into python scripts. 

*homepage*: <https://gromacswrapper.readthedocs.org>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.8.0``|``-Python-3.7.2``|``fosscuda/2019a``

### Groovy

Groovy is a powerful, optionally typed and dynamic language, with static-typing and static compilation capabilities, for the Java platform aimed at improving developer productivity thanks to a concise, familiar and simple to learn syntax.

*homepage*: <https://groovy-lang.org>

version  |versionsuffix|toolchain
---------|-------------|----------
``2.5.9``|``-Java-11`` |``system``
``4.0.3``|``-Java-11`` |``system``

### gRPC

gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. gRPC enables client and server applications to communicate transparently, and simplifies the building of connected systems.

*homepage*: <https://grpc.io/>

version   |toolchain
----------|------------------
``1.44.0``|``GCCcore/11.2.0``

### GSEA

Gene Set Enrichment Analysis (GSEA) is a computational method that  determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes). 

*homepage*: <https://www.gsea-msigdb.org/gsea/index.jsp>

version  |toolchain
---------|----------
``4.0.3``|``system``

### gsettings-desktop-schemas

gsettings-desktop-schemas contains a collection of GSettings schemas for  settings shared by various components of a desktop.

*homepage*: <https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas>

version   |toolchain
----------|-----------------
``3.34.0``|``GCCcore/8.2.0``

### GSL

The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.  The library provides a wide range of mathematical routines such as random number generators, special functions  and least-squares fitting.

*homepage*: <https://www.gnu.org/software/gsl/>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.16`` |``foss/2016a``, ``intel/2016a``
``2.1``  |``GCC/5.4.0-2.26``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.2.1``|``intel/2016a``, ``intel/2016b``
``2.3``  |``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``
``2.4``  |``GCCcore/6.4.0``
``2.5``  |``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.6``  |``GCC/10.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``2.7``  |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

### gSOAP

The gSOAP toolkit is a C and C++ software development toolkit for  SOAP and REST XML Web services and generic C/C++ XML data bindings.  The toolkit analyzes WSDLs and XML schemas (separately or as a combined set) and maps the XML schema types  and the SOAP/REST XML messaging protocols to simple-to-use and efficient C and C++ code.  It also supports exposing (legacy) C and C++ applications as XML Web services  by auto-generating XML serialization code and WSDL specifications.  Or you can simply use it to automatically convert XML to/from C and C++ data.  The toolkit supports options to generate pure ANSI C or C++ with or without STL.

*homepage*: <http://www.cs.fsu.edu/~engelen/soap.html>

version    |toolchain
-----------|-----------------
``2.8.48`` |``GCCcore/6.3.0``
``2.8.100``|``GCCcore/8.3.0``

### gsport

GSPORT command-line tool for accessing GenomeScan Customer Portal

*homepage*: <https://github.com/genomescan/gsport>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.4.2``|``-Python-3.7.4``|``GCCcore/8.3.0``

### GST-plugins-bad

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: <https://gstreamer.freedesktop.org/>

version   |toolchain
----------|--------------
``1.20.2``|``GCC/11.3.0``

### GST-plugins-base

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: <https://gstreamer.freedesktop.org/>

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------------------
``0.10.36``|``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``  |``foss/2016a``
``1.8.3``  |``foss/2016a``
``1.16.0`` |``GCC/8.2.0-2.31.1``
``1.16.2`` |``GCC/8.3.0``
``1.18.4`` |``GCC/10.2.0``, ``GCC/10.3.0``
``1.18.5`` |``GCC/11.2.0``
``1.20.2`` |``GCC/11.3.0``
``1.22.1`` |``GCCcore/12.2.0``

### GStreamer

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: <https://gstreamer.freedesktop.org/>

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------------------
``0.10.36``|``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``  |``foss/2016a``
``1.8.3``  |``foss/2016a``
``1.15.1`` |``fosscuda/2018b``
``1.16.0`` |``GCC/8.2.0-2.31.1``
``1.16.2`` |``GCC/8.3.0``
``1.18.4`` |``GCC/10.2.0``, ``GCC/10.3.0``
``1.18.5`` |``GCC/11.2.0``
``1.20.2`` |``GCC/11.3.0``
``1.22.1`` |``GCCcore/12.2.0``

### gsutil

gsutil is a Python application that lets you access Cloud Storage from the command line.

*homepage*: <https://cloud.google.com/storage/docs/gsutil>

version |toolchain
--------|------------------
``5.10``|``GCCcore/11.2.0``

### gsw

This Python implementation of the Thermodynamic Equation of  Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of the GSW-C  implementation. This library replaces the original python-gsw pure-python  implementation.. The primary reasons for this change are that by building on  the C implementation we reduce code duplication and we gain an immediate update  to the 75-term equation. Additional benefits include a major increase in speed, a reduction in memory  usage, and the inclusion of more functions. The penalty is that a C  (or MSVC C++ for Windows) compiler is required to build the package from source.

*homepage*: <https://github.com/TEOS-10/GSW-python>

version   |toolchain
----------|--------------
``3.6.16``|``foss/2022a``

### GTDB-Tk

A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.

*homepage*: <https://github.com/Ecogenomics/GTDBTk>

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------
``0.2.2``|``-Python-2.7.15``|``intel/2018b``
``0.3.2``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``
``1.0.2``|``-Python-3.7.4`` |``intel/2019b``
``1.3.0``|``-Python-3.8.2`` |``intel/2020a``
``1.5.0``|                  |``intel/2020b``
``1.7.0``|                  |``foss/2020b``, ``foss/2021a``, ``intel/2020b``
``2.0.0``|                  |``foss/2021a``, ``intel/2021b``
``2.1.1``|                  |``foss/2021b``

### GTK+

The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 

*homepage*: <https://developer.gnome.org/gtk+/stable/>

version    |toolchain
-----------|--------------------------------------------------------------------------------
``2.24.28``|``intel/2016a``
``2.24.30``|``foss/2016a``, ``intel/2016a``
``2.24.31``|``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``2.24.32``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``2.24.33``|``GCCcore/10.3.0``
``3.22.30``|``fosscuda/2018b``
``3.24.8`` |``GCCcore/8.2.0``
``3.24.13``|``GCCcore/8.3.0``
``3.24.17``|``GCCcore/9.3.0``
``3.24.23``|``GCCcore/10.2.0``

### GTK2

The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 

*homepage*: <https://www.gtk.org>

version    |toolchain
-----------|--------------------------------------
``2.24.33``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### GTK3

GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. 

*homepage*: <https://developer.gnome.org/gtk3/stable/>

version    |toolchain
-----------|------------------
``3.24.29``|``GCCcore/10.3.0``
``3.24.31``|``GCCcore/11.2.0``
``3.24.33``|``GCCcore/11.3.0``
``3.24.35``|``GCCcore/12.2.0``

### GTK4

GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. 

*homepage*: <https://docs.gtk.org/gtk4/>

version  |toolchain
---------|--------------
``4.7.0``|``GCC/11.3.0``

### GtkSourceView

GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK+  widget for multiline text editing. GtkSourceView adds support for syntax  highlighting, undo/redo, file loading and saving, search and replace, a  completion system, printing, displaying line numbers, and other features  typical of a source code editor. 

*homepage*: <https://wiki.gnome.org/Projects/GtkSourceView>

version    |toolchain
-----------|-------------------------------------
``3.24.11``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``
``4.4.0``  |``GCCcore/8.2.0``

### GTOOL

GTOOL is a program for transforming sets of genotype data for use with  the programs SNPTEST and IMPUTE.  

*homepage*: <http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html>

version  |toolchain
---------|----------
``0.7.5``|``system``

### GTS

GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.

*homepage*: <http://gts.sourceforge.net/>

version     |toolchain
------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.7.6``   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``foss/2019b``, ``foss/2020a``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``
``20121130``|``foss/2017b``

### gubbins

Gubbins (Genealogies Unbiased By recomBinations In Nucleotide Sequences) is an algorithm that iteratively identifies loci containing elevated densities of base substitutions while concurrently constructing a phylogeny based on the putative point mutations outside of these regions. Simulations demonstrate the algorithm generates highly accurate reconstructions under realistic models of short-term bacterial evolution, and can be run in only a few hours on alignments of hundreds of bacterial genome sequences. 

*homepage*: <https://sanger-pathogens.github.io/gubbins>

version  |toolchain
---------|----------
``2.4.0``|``system``

### guenomu

guenomu is a software written in C that estimates the species tree for a given set of gene families.

*homepage*: <https://bitbucket.org/leomrtns/guenomu>

version       |versionsuffix|toolchain
--------------|-------------|---------------
``2019.07.05``|``-mpi``     |``iimpi/2019a``

### Guile

Guile is a programming language, designed to help programmers create flexible  applications that can be extended by users or other programmers with plug-ins,  modules, or scripts. 

*homepage*: <https://www.gnu.org/software/guile/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.8`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``
``2.0.11``|``GCC/4.9.3-2.25``, ``foss/2016a``
``2.2.2`` |``GCCcore/6.4.0``
``2.2.4`` |``GCCcore/7.3.0``, ``GCCcore/9.3.0``
``2.2.7`` |``GCCcore/10.3.0``
``3.0.7`` |``GCCcore/11.2.0``
``3.0.8`` |``GCCcore/11.3.0``
``3.0.9`` |``GCCcore/10.2.0``

### GULP

GULP is a program for performing a variety of types of simulation on materials  using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs  and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations

*homepage*: <https://gulp.curtin.edu.au/gulp/>

version|toolchain
-------|---------------
``5.1``|``intel/2019a``
``6.1``|``foss/2021b``

### Gurobi

The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms.

*homepage*: <https://www.gurobi.com>

version   |versionsuffix    |toolchain
----------|-----------------|--------------------------------------------------
``6.5.1`` |                 |``system``
``6.5.2`` |                 |``system``
``7.0.1`` |                 |``system``
``7.5.2`` |                 |``system``
``7.5.2`` |``-Python-3.6.4``|``intel/2018a``
``8.1.1`` |                 |``system``
``9.0.0`` |                 |``system``
``9.0.0`` |``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``9.0.0`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``9.0.1`` |                 |``system``
``9.0.1`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``9.0.3`` |                 |``GCCcore/10.2.0``, ``system``
``9.1.0`` |                 |``system``
``9.1.1`` |                 |``GCCcore/10.2.0``
``9.1.2`` |                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``system``
``9.5.0`` |                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``9.5.2`` |                 |``GCCcore/11.3.0``
``10.0.1``|                 |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### GUSHR

Assembly-free construction of UTRs from short read RNA-Seq data on the basis of coding sequence annotation. 

*homepage*: <https://github.com/Gaius-Augustus/GUSHR>

version       |toolchain
--------------|--------------
``2020-09-28``|``foss/2021b``

### gzip

gzip (GNU zip) is a popular data compression program as a replacement for compress

*homepage*: <http://www.gnu.org/software/gzip/>

version |toolchain
--------|-------------------------------------------------------------------------------------------------------------------
``1.8`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.9`` |``GCCcore/7.3.0``
``1.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

## H


[h4toh5](#h4toh5) - [H5hut](#h5hut) - [h5netcdf](#h5netcdf) - [h5py](#h5py) - [Hadoop](#hadoop) - [HAL](#hal) - [hanythingondemand](#hanythingondemand) - [HAPGEN2](#hapgen2) - [HarfBuzz](#harfbuzz) - [Harminv](#harminv) - [harmony](#harmony) - [HBase](#hbase) - [HD-BET](#hd-bet) - [HDBSCAN](#hdbscan) - [HDDM](#hddm) - [HDF](#hdf) - [HDF-EOS](#hdf-eos) - [HDF-EOS5](#hdf-eos5) - [HDF5](#hdf5) - [hdf5storage](#hdf5storage) - [HDFView](#hdfview) - [HEALPix](#healpix) - [Health-GPS](#health-gps) - [heaptrack](#heaptrack) - [hector](#hector) - [HeFFTe](#heffte) - [Hello](#hello) - [help2man](#help2man) - [HepMC3](#hepmc3) - [HH-suite](#hh-suite) - [HiC-Pro](#hic-pro) - [HiCExplorer](#hicexplorer) - [HiCMatrix](#hicmatrix) - [hierfstat](#hierfstat) - [hifiasm](#hifiasm) - [HighFive](#highfive) - [Highway](#highway) - [HIP](#hip) - [hipify-clang](#hipify-clang) - [HIPS](#hips) - [hipSYCL](#hipsycl) - [hiredis](#hiredis) - [HISAT2](#hisat2) - [histolab](#histolab) - [hivtrace](#hivtrace) - [hl7apy](#hl7apy) - [HLAminer](#hlaminer) - [HMMER](#hmmer) - [HMMER2](#hmmer2) - [hmmlearn](#hmmlearn) - [HOME](#home) - [HOOMD-blue](#hoomd-blue) - [Horovod](#horovod) - [horton](#horton) - [how_are_we_stranded_here](#how_are_we_stranded_here) - [HPCG](#hpcg) - [HPCX](#hpcx) - [HPDBSCAN](#hpdbscan) - [HPL](#hpl) - [htop](#htop) - [HTSeq](#htseq) - [HTSlib](#htslib) - [HTSplotter](#htsplotter) - [hub](#hub) - [humann](#humann) - [hunspell](#hunspell) - [hwloc](#hwloc) - [Hydra](#hydra) - [Hyperopt](#hyperopt) - [HyperQueue](#hyperqueue) - [hyperspy](#hyperspy) - [HyPhy](#hyphy) - [HyPo](#hypo) - [hypothesis](#hypothesis) - [Hypre](#hypre)


### h4toh5

The h4toh5 software consists of the h4toh5 and h5toh4 command-line utilities,  as well as a conversion library for converting between individual HDF4 and HDF5 objects.

*homepage*: <http://www.hdfgroup.org/h4toh5/>

version  |versionsuffix           |toolchain
---------|------------------------|------------------------------------------------
``2.2.2``|``-linux-x86_64-static``|``system``
``2.2.3``|                        |``foss/2018b``, ``gompi/2019b``, ``gompi/2020b``

### H5hut

HDF5 Utility Toolkit: High-Performance I/O Library for Particle-based Simulations

*homepage*: <https://amas.psi.ch/H5hut/>

version    |toolchain
-----------|---------------
``1.99.13``|``intel/2016b``

### h5netcdf

A Python interface for the netCDF4 file-format that reads and writes local or remote HDF5 files directly via h5py or h5pyd, without relying on the Unidata netCDF library.

*homepage*: <https://h5netcdf.org/>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2021b``

### h5py

HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,  version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous  amounts of data.

*homepage*: <https://www.h5py.org/>

version   |versionsuffix                        |toolchain
----------|-------------------------------------|------------------------------------------------------------------------------------
``2.5.0`` |``-Python-2.7.11-HDF5-1.8.16``       |``foss/2016a``, ``intel/2016a``
``2.5.0`` |``-Python-2.7.11-HDF5-1.8.16-serial``|``foss/2016a``, ``intel/2016a``
``2.5.0`` |``-Python-3.5.1-HDF5-1.8.16``        |``foss/2016a``
``2.6.0`` |``-Python-2.7.11``                   |``intel/2016a``
``2.6.0`` |``-Python-2.7.12-HDF5-1.10.0-patch1``|``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-2.7.12-HDF5-1.8.17``       |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-2.7.12-HDF5-1.8.18``       |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.10.0-patch1`` |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.8.17``        |``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.8.18``        |``foss/2016b``, ``intel/2016b``
``2.7.0`` |``-Python-2.7.12``                   |``intel/2016b``
``2.7.0`` |``-Python-2.7.13``                   |``intel/2017a``
``2.7.0`` |``-Python-2.7.13-HDF5-1.10.1``       |``foss/2017a``
``2.7.0`` |``-Python-2.7.13-HDF5-1.8.19``       |``foss/2017a``
``2.7.0`` |``-Python-3.6.1``                    |``intel/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.10.0-patch1`` |``intel/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.10.1``        |``foss/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.8.19``        |``foss/2017a``
``2.7.1`` |``-Python-2.7.13``                   |``intel/2017a``
``2.7.1`` |``-Python-2.7.14``                   |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``2.7.1`` |``-Python-2.7.14-serial``            |``intel/2018a``
``2.7.1`` |``-Python-3.6.1``                    |``intel/2017a``
``2.7.1`` |``-Python-3.6.3``                    |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``2.7.1`` |``-Python-3.6.4``                    |``foss/2018a``, ``intel/2018a``
``2.8.0`` |``-Python-2.7.15``                   |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.8.0`` |``-Python-2.7.15-serial``            |``intel/2018b``
``2.8.0`` |``-Python-3.6.6``                    |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.9.0`` |                                     |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2.10.0``|``-Python-2.7.18``                   |``intel/2020a``
``2.10.0``|``-Python-3.7.4``                    |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2.10.0``|``-Python-3.8.2``                    |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``2.10.0``|``-serial-Python-3.7.4``             |``foss/2019b``
``3.1.0`` |                                     |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``3.2.1`` |                                     |``foss/2021a``, ``gomkl/2021a``
``3.6.0`` |                                     |``foss/2021b``, ``intel/2021b``
``3.7.0`` |                                     |``foss/2022a``, ``intel/2022a``
``3.8.0`` |                                     |``foss/2022b``

### Hadoop

Hadoop MapReduce by Cloudera

*homepage*: <https://archive.cloudera.com/cdh5/cdh/5/>

version            |versionsuffix             |toolchain
-------------------|--------------------------|------------------
``2.4.0``          |``-seagate-722af1-native``|``system``
``2.5.0-cdh5.3.1`` |``-native``               |``system``
``2.6.0-cdh5.4.5`` |``-native``               |``system``
``2.6.0-cdh5.7.0`` |``-native``               |``system``
``2.6.0-cdh5.8.0`` |``-native``               |``system``
``2.6.0-cdh5.12.0``|``-native``               |``system``
``2.9.2``          |``-native``               |``GCCcore/7.3.0``
``2.10.0``         |``-native``               |``GCCcore/8.3.0``
``2.10.0``         |``-native-Java-1.8``      |``GCCcore/10.2.0``

### HAL

HAL is a structure to efficiently store and index multiple genome alignments and ancestral reconstructions. HAL is a graph-based representation which provides several advantages over matrix/block-based formats such as MAF, such as improved scalability and the ability to perform queries with respect to an arbitrary reference or subtree.  This package includes the HAL API and several analysis and conversion tools which are described below. HAL files are presently stored in either HDF5 or mmap format, but we note that the tools and most of the API are format-independent, so other databases could be implemented in the future. 

*homepage*: <https://github.com/ComparativeGenomicsToolkit/hal>

version|toolchain
-------|--------------
``2.1``|``foss/2020b``

### hanythingondemand

HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster. It uses the Torque resource manager to do node allocation.

*homepage*: <https://github.com/hpcugent/hanythingondemand>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.0``|``-cli``          |``system``
``3.0.1``|``-cli``          |``system``
``3.0.2``|``-cli``          |``system``
``3.0.3``|``-cli``          |``system``
``3.0.4``|``-cli``          |``system``
``3.1.0``|``-Python-2.7.11``|``intel/2016a``
``3.1.0``|``-cli``          |``system``
``3.1.1``|``-Python-2.7.11``|``intel/2016a``
``3.1.1``|``-cli``          |``system``
``3.1.2``|``-Python-2.7.11``|``intel/2016a``
``3.1.2``|``-cli``          |``system``
``3.1.3``|``-Python-2.7.12``|``intel/2016b``
``3.1.3``|``-cli``          |``system``
``3.1.4``|``-Python-2.7.12``|``intel/2016b``
``3.1.4``|``-cli``          |``system``
``3.2.0``|``-Python-2.7.12``|``intel/2016b``
``3.2.0``|``-cli``          |``system``
``3.2.2``|``-Python-2.7.12``|``intel/2016b``
``3.2.2``|``-cli``          |``system``

### HAPGEN2

'HAPGEN2' simulates case control datasets at SNP markers.

*homepage*: <https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html>

version  |toolchain
---------|----------
``2.2.0``|``system``

### HarfBuzz

HarfBuzz is an OpenType text shaping engine.

*homepage*: <https://www.freedesktop.org/wiki/Software/HarfBuzz>

version  |toolchain
---------|------------------------------------------------
``1.1.3``|``foss/2016a``, ``intel/2016a``
``1.2.7``|``foss/2016a``, ``intel/2016a``
``1.3.1``|``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.5.1``|``intel/2017a``
``1.7.1``|``foss/2017b``, ``intel/2017b``
``1.7.5``|``foss/2018a``, ``intel/2018a``
``1.9.0``|``fosscuda/2018b``
``2.2.0``|``foss/2018b``, ``fosscuda/2018b``
``2.4.0``|``GCCcore/8.2.0``
``2.6.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.7``|``GCCcore/10.2.0``
``2.8.1``|``GCCcore/10.3.0``
``2.8.2``|``GCCcore/11.2.0``
``4.2.1``|``GCCcore/11.3.0``
``5.3.1``|``GCCcore/12.2.0``

### Harminv

Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -  given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially  decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those  sinusoids.

*homepage*: <https://github.com/stevengj/harminv>

version  |toolchain
---------|----------------------------------------------------------------
``1.4``  |``foss/2016a``
``1.4.1``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``, ``intel/2020a``
``1.4.2``|``foss/2020b``

### harmony

Harmony is a general-purpose R package with an efficient algorithm for integrating multiple data sets.

*homepage*: <https://portals.broadinstitute.org/harmony>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.0.0-20200224``|``-R-4.0.0`` |``foss/2020a``

### HBase

Apache HBase. is the Hadoop database, a distributed, scalable, big data store. 

*homepage*: <http://hbase.apache.org/>

version  |toolchain
---------|----------
``1.0.2``|``system``

### HD-BET

Tool for brain extraction.

*homepage*: <https://github.com/MIC-DKFZ/HD-BET>

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20220318``|                |``foss/2021a``
``20220318``|``-CUDA-11.3.1``|``foss/2021a``

### HDBSCAN

The hdbscan library is a suite of tools to use unsupervised learning to find clusters, or dense  regions, of a dataset. The primary algorithm is HDBSCAN* as proposed by Campello, Moulavi, and Sander. The library  provides a high performance implementation of this algorithm, along with tools for analysing the resulting  clustering.

*homepage*: <http://hdbscan.readthedocs.io/en/latest/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.8.24``|``-Python-3.7.4``|``foss/2019b``
``0.8.27``|                 |``foss/2021a``
``0.8.29``|                 |``foss/2022a``

### HDDM

HDDM is a Python toolbox for hierarchical Bayesian parameter estimation  of the Drift Diffusion Model (via PyMC).

*homepage*: <http://ski.clps.brown.edu/hddm_docs>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.1``|``-Python-3.6.6`` |``intel/2018b``
``0.7.5``|``-Python-2.7.16``|``intel/2019b``
``0.7.5``|``-Python-3.7.4`` |``intel/2019b``

### HDF

HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines. 

*homepage*: <https://www.hdfgroup.org/products/hdf4/>

version   |versionsuffix |toolchain
----------|--------------|---------------------------------------------------------------------------------------------------------------------
``4.2.11``|              |``intel/2016a``
``4.2.12``|              |``intel/2017a``
``4.2.13``|              |``GCCcore/6.4.0``
``4.2.13``|``-no-netcdf``|``intel/2017a``
``4.2.14``|              |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.2.15``|              |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/9.3.0``

### HDF-EOS

HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.

*homepage*: <https://hdfeos.org/>

version |toolchain
--------|--------------------------------------------------------
``2.20``|``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``

### HDF-EOS5

HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.

*homepage*: <https://hdfeos.org/>

version |toolchain
--------|------------------------------------------------
``1.16``|``foss/2018b``, ``gompi/2019b``, ``gompi/2020b``

### HDF5

HDF5 is a data model, library, and file format for storing and managing data.  It supports an unlimited variety of datatypes, and is designed for flexible  and efficient I/O and for high volume and complex data.

*homepage*: <https://portal.hdfgroup.org/display/support>

version          |versionsuffix|toolchain
-----------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.10``       |``-serial``  |``GCC/4.8.1``
``1.8.11``       |``-serial``  |``GCC/4.8.1``
``1.8.12``       |             |``foss/2018b``, ``intel/2016b``
``1.8.13``       |             |``foss/2018b``
``1.8.16``       |             |``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.8.16``       |``-serial``  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.8.17``       |             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.8.17``       |``-serial``  |``GCC/5.4.0-2.26``, ``foss/2016a``, ``intel/2016b``
``1.8.18``       |             |``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``
``1.8.18``       |``-serial``  |``intel/2017a``
``1.8.19``       |             |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``1.8.20``       |             |``foss/2018a``, ``gmpolf/2017.10``, ``intel/2017b``, ``intel/2018a``
``1.10.0-patch1``|             |``foss/2016b``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``
``1.10.1``       |             |``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2017b``, ``iomkl/2018a``
``1.10.2``       |             |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``1.10.5``       |             |``gompi/2019a``, ``gompi/2019b``, ``gompic/2019a``, ``gompic/2019b``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpic/2019a``, ``iimpic/2019b``, ``iompi/2019b``
``1.10.5``       |``-serial``  |``GCC/8.3.0``
``1.10.6``       |             |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``, ``iimpic/2020a``, ``iompi/2020a``
``1.10.7``       |             |``gompi/2020b``, ``gompi/2021a``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpic/2020b``, ``iompi/2021a``
``1.10.8``       |             |``gompi/2021b``, ``gompi/2022a``
``1.12.0``       |             |``gompi/2020a``, ``iimpi/2020a``
``1.12.1``       |             |``gompi/2021a``, ``gompi/2021b``, ``iimpi/2021b``
``1.12.2``       |             |``gompi/2022a``, ``iimpi/2022a``, ``nvompi/2022.07``
``1.12.2``       |``-serial``  |``GCC/11.3.0``, ``NVHPC/22.7-CUDA-11.7.0``
``1.13.1``       |             |``gompi/2022a``, ``iimpi/2022a``
``1.13.1``       |``-serial``  |``GCC/11.3.0``
``1.14.0``       |             |``gompi/2022b``, ``iimpi/2022b``

### hdf5storage

This Python package provides high level utilities to read/write a variety of Python types to/from  HDF5 (Heirarchal Data Format) formatted files. This package also provides support for MATLAB MAT v7.3 formatted  files, which are just HDF5 files with a different extension and some extra meta-data. All of this is done without  pickling data. Pickling is bad for security because it allows arbitrary code to be executed in the interpreter.  One wants to be able to read possibly HDF5 and MAT files from untrusted sources, so pickling is avoided in this  package.

*homepage*: <https://pythonhosted.org/hdf5storage/>

version   |toolchain
----------|----------------------------------
``0.1.15``|``foss/2019a``, ``fosscuda/2019a``

### HDFView

HDFView is a visual tool for browsing and editing HDF4 and HDF5 files.

*homepage*: <https://support.hdfgroup.org/products/java/hdfview/>

version |versionsuffix              |toolchain
--------|---------------------------|----------
``2.14``|``-Java-1.8.0_152-centos6``|``system``
``2.14``|``-Java-1.8.0_152-centos7``|``system``

### HEALPix

Hierarchical Equal Area isoLatitude Pixelation of a sphere.

*homepage*: <http://healpix.sourceforge.net/>

version |toolchain
--------|------------------------------------
``3.50``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

### Health-GPS

Health-GPS microsimulation is part of the STOP project, and supports researchers  and policy makers in the analysis of the health and economic impacts of alternative  measures to tackle chronic diseases and obesity in children. The model reproduces  the characteristics of a population and simulates key individual event histories  associated with key components of relevant behaviours, such as physical activity,  and diseases such as diabetes or cancer. To run the test-jobs with HealthGPS.Tests the data-directory, found in your installation folder, must be in the current path.  

*homepage*: <https://github.com/imperialCHEPI/healthgps>

version    |toolchain
-----------|------------------
``1.1.3.0``|``GCCcore/11.3.0``
``1.2.2.0``|``GCCcore/11.3.0``

### heaptrack

A heap memory profiler for Linux.

*homepage*: <http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2016b``

### hector

This is the repository for Hector, an open source, object-oriented,  simple global climate carbon-cycle model. It runs essentially instantaneously while  still representing the most critical global scale earth system processes, and is  one of a class of models heavily used for for emulating complex climate models  and uncertainty analyses.

*homepage*: <https://github.com/JGCRI/hector>

version  |toolchain
---------|--------------
``2.5.0``|``GCC/11.2.0``

### HeFFTe

Highly Efficient FFT for Exascale (HeFFTe) library

*homepage*: <https://icl.utk.edu/fft>

version|toolchain
-------|--------------
``1.0``|``foss/2020a``

### Hello

The GNU Hello program produces a familiar, friendly greeting. Yes, this is another implementation of the classic program that prints "Hello, world!" when you run it.  However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior, supports greetings in many languages, and so on. 

*homepage*: <https://www.gnu.org/software/hello/>

version |toolchain
--------|-----------------
``2.10``|``GCCcore/8.2.0``

### help2man

help2man produces simple manual pages from the '--help' and '--version' output of other commands.

*homepage*: <https://www.gnu.org/software/help2man/>

version    |toolchain
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.47.4`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/system``, ``gimkl/2017a``, ``intel/2016b``, ``system``
``1.47.5`` |``GCCcore/5.5.0``
``1.47.6`` |``GCCcore/8.1.0``
``1.47.7`` |``GCCcore/8.2.0``
``1.47.8`` |``GCCcore/7.4.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``
``1.47.10``|``GCCcore/9.1.0``, ``GCCcore/9.2.0``
``1.47.12``|``GCCcore/9.3.0``
``1.47.15``|``GCCcore/10.1.0``
``1.47.16``|``GCCcore/10.2.0``
``1.48.3`` |``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/9.4.0``
``1.49.2`` |``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.5.0``
``1.49.3`` |``GCCcore/11.4.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### HepMC3

HepMC is a standard for storing Monte Carlo event data.

*homepage*: <http://hepmc.web.cern.ch/hepmc/>

version  |toolchain
---------|--------------
``3.2.5``|``GCC/11.3.0``

### HH-suite

The HH-suite is an open-source software package  for sensitive protein sequence searching based on the pairwise  alignment of hidden Markov models (HMMs).

*homepage*: <https://github.com/soedinglab/hh-suite>

version       |versionsuffix    |toolchain
--------------|-----------------|----------------------------------------------------------------------------------------------------
``3.0-beta.3``|                 |``intel/2018a``
``3.2.0``     |                 |``foss/2019b``, ``fosscuda/2019b``
``3.3.0``     |                 |``foss/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``gompic/2020b``
``3.3.0``     |``-Python-3.7.4``|``gompic/2019b``

### HiC-Pro

HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data)   to the normalized contact maps.

*homepage*: <http://nservant.github.io/HiC-Pro>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.9.0``|``-Python-2.7.12``|``foss/2016b``

### HiCExplorer

HiCexplorer addresses the common tasks of Hi-C analysis from processing to visualization.

*homepage*: <http://hicexplorer.readthedocs.org/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.1.1``|``-Python-3.6.4``|``foss/2018a``
``3.7.2``|                 |``foss/2022a``

### HiCMatrix

This library implements the central class of HiCExplorer to manage Hi-C interaction matrices.

*homepage*: <https://github.com/deeptools/HiCMatrix>

version|toolchain
-------|--------------
``17`` |``foss/2022a``

### hierfstat

Estimates hierarchical F-statistics from haploid or diploid genetic data with any numbers of levels in the hierarchy.

*homepage*: <https://cran.r-project.org/package=hierfstat>

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.5-7``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

### hifiasm

Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.

*homepage*: <https://github.com/chhylp123/hifiasm>

version   |toolchain
----------|-------------------------------------
``0.15.2``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.16.1``|``GCCcore/10.3.0``
``0.19.5``|``GCCcore/11.2.0``

### HighFive

HighFive is a modern header-only C++11 friendly interface for libhdf5.

*homepage*: <https://github.com/BlueBrain/HighFive>

version  |toolchain
---------|--------------------------------
``2.6.2``|``gompi/2021a``, ``gompi/2022a``

### Highway

Highway is a C++ library for SIMD (Single Instruction, Multiple Data), i.e. applying the same operation to 'lanes'.

*homepage*: <https://github.com/google/highway>

version   |toolchain
----------|--------------------------------------
``0.12.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.0.3`` |``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.0.4`` |``GCCcore/11.3.0``, ``GCCcore/12.3.0``

### HIP

HIP is a C++ Runtime API and Kernel Language that allows developers to create portable applications for AMD and NVIDIA GPUs from single source code.

*homepage*: <https://github.com/ROCm-Developer-Tools/HIP>

version  |versionsuffix|toolchain
---------|-------------|------------------
``4.5.0``|``-amd``     |``GCCcore/11.2.0``

### hipify-clang

Hipify-clang is a clang-based tool for translating CUDA   sources into HIP sources. It translates CUDA source into an abstract syntax   tree, which is traversed by transformation matchers. After applying all the   matchers, the output HIP source is produced.

*homepage*: <https://github.com/ROCm-Developer-Tools/HIPIFY>

version  |toolchain
---------|-----------------
``4.2.0``|``gcccuda/2020b``

### HIPS

HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides  an efficient parallel iterative solver for very large sparse linear systems. 

*homepage*: <http://hips.gforge.inria.fr/>

version     |toolchain
------------|--------------
``1.2b-rc5``|``foss/2017b``

### hipSYCL

hipSYCL is a modern SYCL implementation targeting CPUs and GPUs, with a focus on leveraging existing toolchains such as CUDA or HIP

*homepage*: <https://github.com/illuhad/hipSYCL>

version  |toolchain
---------|---------------------------------
``0.9.1``|``GCC/10.2.0``, ``gcccuda/2020b``

### hiredis

Hiredis is a minimalistic C client library for the Redis database.  It is minimalistic because it just adds minimal support for the protocol, but at the same time it uses a high level printf-alike API in order to make it much higher level than otherwise suggested by its minimal code base and the lack of explicit bindings for every Redis command.

*homepage*: <https://github.com/redis/hiredis>

version  |toolchain
---------|--------------------------------------
``1.0.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### HISAT2

HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads  (both DNA and RNA) against the general human population (as well as against a single reference genome).

*homepage*: <https://daehwankimlab.github.io/hisat2>

version       |toolchain
--------------|---------------------------------------------------------------------------------
``2.0.3-beta``|``intel/2016a``
``2.0.4``     |``foss/2016b``
``2.0.5``     |``intel/2017a``
``2.1.0``     |``foss/2017b``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``2.2.0``     |``foss/2018b``
``2.2.1``     |``foss/2019b``, ``foss/2020a``, ``gompi/2020b``, ``gompi/2021b``, ``gompi/2022a``

### histolab

Library for Digital Pathology Image Processing

*homepage*: <https://github.com/histolab/histolab>

version  |toolchain
---------|------------------------------
``0.4.1``|``foss/2021a``, ``foss/2021b``

### hivtrace

HIV-TRACE is an application that identifies potential transmission clusters within a  supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.

*homepage*: <https://github.com/veg/hivtrace>

version  |toolchain
---------|--------------
``0.6.2``|``foss/2021a``

### hl7apy

Python library to parse, create and handle HL7 v2 messages.

*homepage*: <https://github.com/crs4/hl7apy>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.3.3``|``-Python-3.7.4``|``GCCcore/8.3.0``

### HLAminer

HLAminer is a software for HLA predictions from next-generation shotgun (NGS) sequence read data and supports direct  read alignment and targeted de novo assembly of sequence reads. 

*homepage*: <http://www.bcgsc.ca/platform/bioinfo/software/hlaminer>

version|versionsuffix   |toolchain
-------|----------------|--------------
``1.4``|``-Perl-5.28.0``|``foss/2018b``

### HMMER

HMMER is used for searching sequence databases for homologs  of protein sequences, and for making protein sequence alignments. It  implements methods using probabilistic models called profile hidden Markov  models (profile HMMs).  Compared to BLAST, FASTA, and other sequence  alignment and database search tools based on older scoring methodology,  HMMER aims to be significantly more accurate and more able to detect remote  homologs because of the strength of its underlying mathematical models. In the  past, this strength came at significant computational expense, but in the new  HMMER3 project, HMMER is now essentially as fast as BLAST.

*homepage*: <http://hmmer.org/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.1b2``|``GCC/6.4.0-2.28``, ``foss/2016a``, ``foss/2016b``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``, ``intel/2018a``
``3.2.1``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``gompi/2019b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iimpi/2019b``, ``intel/2018b``
``3.3.1``|``gompi/2020a``, ``iimpi/2020a``
``3.3.2``|``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``gompi/2022b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021b``

### HMMER2

HMMER is used for searching sequence databases for sequence homologs,  and for making sequence alignments.

*homepage*: <http://hmmer.org>

version  |toolchain
---------|---------------------------------------------
``2.3.2``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.3.0``

### hmmlearn

hmmlearn is a set of algorithms for unsupervised learning and inference of Hidden Markov Models

*homepage*: <https://github.com/hmmlearn/hmmlearn>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.2.0``|``-Python-2.7.14``|``intel/2017b``
``0.2.0``|``-Python-3.6.4`` |``intel/2018a``

### HOME

HOME (histogram of methylation) is a python package for differential methylation region (DMR) identification. The method uses histogram of methylation features and the linear Support Vector Machine (SVM) to identify DMRs  from whole genome bisulfite sequencing (WGBS) data.

*homepage*: <https://github.com/ListerLab/HOME>

version|versionsuffix     |toolchain
-------|------------------|--------------
``0.9``|``-Python-2.7.13``|``foss/2017a``

### HOOMD-blue

HOOMD-blue is a general-purpose particle simulation toolkit, implementing molecular dynamics and hard particle Monte Carlo optimized for fast execution on both GPUs and CPUs.

*homepage*: <https://bitbucket.org/glotzer/hoomd-blue>

version  |toolchain
---------|--------------
``4.0.1``|``foss/2022a``

### Horovod

Horovod is a distributed training framework for TensorFlow.

*homepage*: <https://github.com/uber/horovod>

version   |versionsuffix                      |toolchain
----------|-----------------------------------|----------------------------------
``0.9.10``|``-Python-3.6.3``                  |``intel/2017b``
``0.18.1``|``-Python-3.7.2``                  |``foss/2019a``
``0.18.2``|``-TensorFlow-1.15.0-Python-3.7.4``|``fosscuda/2019b``
``0.18.2``|``-TensorFlow-1.15.2-Python-3.7.4``|``fosscuda/2019b``
``0.18.2``|``-TensorFlow-2.0.0-Python-3.7.4`` |``fosscuda/2019b``
``0.19.1``|``-TensorFlow-2.1.0-Python-3.7.4`` |``fosscuda/2019b``
``0.19.5``|``-TensorFlow-2.2.0-Python-3.7.4`` |``fosscuda/2019b``
``0.20.3``|``-TensorFlow-2.3.1-Python-3.7.4`` |``fosscuda/2019b``
``0.21.1``|``-PyTorch-1.7.1-Python-3.7.4``    |``fosscuda/2019b``
``0.21.1``|``-TensorFlow-2.4.1``              |``fosscuda/2020b``
``0.21.1``|``-TensorFlow-2.4.1-Python-3.7.4`` |``fosscuda/2019b``
``0.21.3``|``-PyTorch-1.7.1``                 |``fosscuda/2020b``
``0.21.3``|``-TensorFlow-2.3.1-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``0.22.0``|``-PyTorch-1.8.1``                 |``fosscuda/2020b``
``0.22.1``|``-CUDA-11.3.1-TensorFlow-2.6.0``  |``foss/2021a``
``0.22.1``|``-TensorFlow-2.5.0``              |``fosscuda/2020b``
``0.22.1``|``-TensorFlow-2.5.0-Python-3.7.4`` |``fosscuda/2019b``
``0.23.0``|``-CUDA-11.3.1-PyTorch-1.10.0``    |``foss/2021a``
``0.23.0``|``-TensorFlow-2.5.0``              |``fosscuda/2020b``
``0.25.0``|``-CUDA-11.3.1-PyTorch-1.10.0``    |``foss/2021a``

### horton

HORTON is a Helpful Open-source Research TOol for N-fermion systems, written  primarily in the Python programming language. (HORTON is named after the helpful  pachyderm, not the Canadian caffeine supply store.) The ultimate goal of HORTON  is to provide a platform for testing new ideas on the quantum many-body problem  at a reasonable computational cost. Although HORTON is primarily designed to be  a quantum-chemistry program, it can perform computations involving model  Hamiltonians, and could be extended for computations in nuclear physics.

*homepage*: <https://theochem.github.io/horton>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.1.1``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.1.1``|``-Python-2.7.18``|``intel/2020a``

### how_are_we_stranded_here

Python package for testing strandedness of RNA-Seq fastq files

*homepage*: <https://github.com/signalbash/how_are_we_stranded_here>

version  |toolchain
---------|--------------
``1.0.1``|``foss/2021b``

### HPCG

The HPCG Benchmark project is an effort to create a more relevant metric for ranking HPC systems than  the High Performance LINPACK (HPL) benchmark, that is currently used by the TOP500 benchmark.

*homepage*: <https://software.sandia.gov/hpcg>

version|toolchain
-------|-------------------------------------------------------------------------------------------------
``3.0``|``foss/2016b``, ``foss/2018b``, ``intel/2018b``
``3.1``|``foss/2018b``, ``foss/2021a``, ``foss/2021b``, ``intel/2018b``, ``intel/2021a``, ``intel/2021b``

### HPCX

The Mellanox HPC-X Toolkit is a comprehensive MPI and SHMEM/PGAS software suite for high performance computing environments

*homepage*: <http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x>

version  |toolchain
---------|--------------------
``2.3.0``|``GCC/8.2.0-2.31.1``

### HPDBSCAN

Highly parallel density based spatial clustering for application with noise

*homepage*: <https://github.com/Markus-Goetz/hpdbscan>

version     |toolchain
------------|--------------
``20171110``|``foss/2017b``
``20210826``|``foss/2020b``

### HPL

HPL is a software package that solves a (random) dense linear system in double precision (64 bits)  arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available  implementation of the High Performance Computing Linpack Benchmark.

*homepage*: <https://www.netlib.org/benchmark/hpl/>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``foss/2016.04``, ``foss/2016.06``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gmpolf/2016a``, ``gmvolf/1.7.20``, ``gmvolf/2016a``, ``intel/2016.00``, ``intel/2016.01``, ``intel/2016.02-GCC-4.9``, ``intel/2016.02-GCC-5.3``, ``intel/2016.03-GCC-4.9``, ``intel/2016.03-GCC-5.3``, ``intel/2016.03-GCC-5.4``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``pomkl/2016.03``, ``pomkl/2016.04``, ``pomkl/2016.09``
``2.2``|``foss/2016.07``, ``foss/2016.09``, ``foss/2017a``, ``foss/2017b``, ``foss/2018.08``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2018b``, ``giolf/2017b``, ``giolf/2018a``, ``giolfc/2017b``, ``gmpolf/2017.10``, ``goblf/2018b``, ``gomkl/2018b``, ``intel/2017.00``, ``intel/2017.01``, ``intel/2017.02``, ``intel/2017.09``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018.02``, ``intel/2018.04``, ``intel/2018a``, ``intel/2018b``, ``intel/2019.00``, ``intel/2019.01``, ``intelcuda/2016.10``, ``intelcuda/2017b``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2016.09-GCC-5.4.0-2.26``, ``iomkl/2017.01``, ``iomkl/2017a``, ``iomkl/2017b``, ``iomkl/2018.02``, ``iomkl/2018a``, ``iomkl/2018b``, ``pomkl/2016.09``
``2.3``|``CrayCCE/19.06``, ``CrayGNU/19.06``, ``CrayIntel/19.06``, ``Fujitsu/21.05``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022.05``, ``foss/2022.10``, ``foss/2022a``, ``foss/2022b``, ``foss/2023a``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``gobff/2020.06-amd``, ``gobff/2020.11``, ``gobff/2020b``, ``gobff/2021a``, ``goblf/2020b``, ``gomkl/2019a``, ``gomkl/2020b``, ``gomkl/2021a``, ``gomkl/2022a``, ``iibff/2020b``, ``intel/2019.02``, ``intel/2019.03``, ``intel/2019a``, ``intel/2019b``, ``intel/2020.00``, ``intel/2020.06-impi-18.5``, ``intel/2020.12``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022.00``, ``intel/2022.09``, ``intel/2022.11``, ``intel/2022.12``, ``intel/2022b``, ``intel/2023.03``, ``intel/2023a``, ``intelcuda/2019b``, ``intelcuda/2020a``, ``iomkl/2019.01``, ``iomkl/2021a``, ``iomkl/2021b``

### htop

An interactive process viewer for Unix

*homepage*: <https://htop.dev>

version  |toolchain
---------|----------
``2.0.0``|``system``
``2.0.1``|``system``
``3.2.1``|``system``

### HTSeq

HTSeq is a Python library to facilitate processing and analysis  of data from high-throughput sequencing (HTS) experiments.

*homepage*: <https://github.com/simon-anders/htseq>

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------
``0.6.1p1``|``-Python-2.7.11``|``foss/2016a``
``0.6.1p1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.9.1``  |``-Python-2.7.12``|``foss/2016b``
``0.9.1``  |``-Python-2.7.13``|``intel/2017a``
``0.9.1``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``0.9.1``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.10.0`` |``-Python-2.7.14``|``foss/2018a``
``0.11.0`` |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.11.2`` |``-Python-3.6.6`` |``foss/2018b``
``0.11.2`` |``-Python-3.7.2`` |``foss/2019a``
``0.11.2`` |``-Python-3.7.4`` |``foss/2019b``
``0.11.3`` |                  |``foss/2020b``, ``foss/2021b``
``2.0.2``  |                  |``foss/2022a``

### HTSlib

PacBio fork of C library for reading/writing high-throughput sequencing data.  This package includes the utilities bgzip and tabix

*homepage*: <https://github.com/PacificBiosciences/htslib>

version     |versionsuffix|toolchain
------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2.1``   |             |``foss/2016b``
``1.3``     |             |``foss/2016a``, ``intel/2016a``
``1.3.1``   |             |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.3.2``   |             |``intel/2016b``
``1.4``     |             |``foss/2016b``, ``intel/2016b``
``1.4.1``   |             |``foss/2016a``, ``intel/2017a``
``1.6``     |             |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``1.7``     |             |``intel/2018a``
``1.8``     |             |``GCC/6.4.0-2.28``, ``foss/2018a``, ``intel/2018a``
``1.9``     |             |``GCC/6.4.0-2.28``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10.2``  |             |``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``    |             |``GCC/10.2.0``, ``iccifort/2020.4.304``
``1.12``    |             |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.14``    |             |``GCC/11.2.0``
``1.15.1``  |             |``GCC/11.3.0``
``1.17``    |             |``GCC/12.2.0``
``20160107``|``-PacBio``  |``intel/2017a``

### HTSplotter

HTSplotter allows an end-to-end data processing and analysis of chemical and genetic in vitro perturbation screens.

*homepage*: <https://github.com/CBIGR/HTSplotter>

version |toolchain
--------|--------------
``0.15``|``foss/2022a``
``2.11``|``foss/2022b``

### hub

hub is a command-line wrapper for git that makes you better at GitHub.

*homepage*: <https://hub.github.com/>

version  |versionsuffix   |toolchain
---------|----------------|----------
``2.2.2``|``-linux-amd64``|``system``

### humann

HUMAnN v3 is a pipeline for efficiently  and accurately determining the coverage and abundance  of microbial pathways in a community from metagenomic data. Sequencing a metagenome typically produces millions  of short DNA/RNA reads. This process, referred  to as functional profiling, aims to describe the metabolic potential  of a microbial community and its members. More generally,  functional profiling answers the question:  What are the microbes in my community-of-interest doing (or capable of doing)?

*homepage*: <http://huttenhower.sph.harvard.edu/humann>

version|toolchain
-------|--------------
``3.6``|``foss/2022a``

### hunspell

Hunspell is a spell checker and morphological analyzer library and program designed for languageswith rich morphology and complex word compounding or character encoding.

*homepage*: <https://hunspell.github.io/>

version  |toolchain
---------|--------------------------------------
``1.6.1``|``intel/2017a``
``1.7.0``|``GCCcore/8.2.0``
``1.7.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### hwloc

The Portable Hardware Locality (hwloc) software package provides a portable  abstraction (across OS, versions, architectures, ...) of the hierarchical  topology of modern architectures, including NUMA memory nodes, sockets, shared  caches, cores and simultaneous multithreading. It also gathers various system  attributes such as cache and memory information as well as the locality of I/O  devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily  aims at helping applications with gathering information about modern computing  hardware so as to exploit it accordingly and efficiently. 

*homepage*: <https://www.open-mpi.org/projects/hwloc/>

version    |toolchain
-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.7.2``  |``GCC/4.8.2``
``1.8.1``  |``GCC/4.8.2``, ``GCC/4.8.3``
``1.9``    |``GCC/4.8.3``
``1.10.0`` |``GCC/4.9.2``
``1.10.1`` |``GCC/4.8.4``, ``GNU/4.9.2-2.25``
``1.11.0`` |``GNU/4.9.3-2.25``
``1.11.1`` |``GCC/4.9.3``
``1.11.2`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``1.11.3`` |``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``1.11.4`` |``GCC/6.2.0-2.27``, ``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``
``1.11.5`` |``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``1.11.6`` |``GCC/6.3.0-2.28``
``1.11.7`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``
``1.11.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``intel/2017a``
``1.11.10``|``GCCcore/7.3.0``
``1.11.11``|``GCCcore/8.2.0``
``1.11.12``|``GCCcore/8.3.0``
``2.0.2``  |``GCCcore/8.2.0``
``2.0.3``  |``GCCcore/8.3.0``
``2.1.0``  |``GCCcore/9.2.0``
``2.2.0``  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.4.1``  |``GCCcore/10.3.0``
``2.5.0``  |``GCCcore/11.2.0``
``2.7.1``  |``GCCcore/11.3.0``
``2.8.0``  |``GCCcore/12.2.0``
``2.9.1``  |``GCCcore/12.3.0``

### Hydra

Hydra is an open-source Python framework that simplifies the development of research and other complex applications. The key feature is the ability to dynamically create a hierarchical configuration by composition and override it through config files and the command line. The name Hydra comes from its ability to run multiple similar jobs - much like a Hydra with multiple heads.

*homepage*: <https://hydra.cc/>

version  |toolchain
---------|------------------
``1.1.1``|``GCCcore/10.3.0``

### Hyperopt

hyperopt is a Python library for optimizing over awkward search spaces with real-valued,  discrete, and conditional dimensions.

*homepage*: <http://hyperopt.github.io/hyperopt/>

version  |versionsuffix             |toolchain
---------|--------------------------|------------------------------
``0.1``  |``-Python-2.7.14``        |``intel/2017b``
``0.1.1``|``-Python-3.6.6``         |``intel/2018b``
``0.2.4``|``-Python-3.7.4-Java-1.8``|``intel/2019b``
``0.2.5``|                          |``fosscuda/2020b``
``0.2.7``|                          |``foss/2021a``, ``foss/2022a``

### HyperQueue

HyperQueue lets you build a computation plan consisting of a large amount of tasks and then execute it transparently over a system like SLURM/PBS. It dynamically groups jobs into SLURM/PBS jobs and distributes them to fully utilize allocated nodes. You thus do not have to manually aggregate your tasks into SLURM/PBS jobs.

*homepage*: <https://it4innovations.github.io/hyperqueue/>

version   |toolchain
----------|----------
``0.13.0``|``system``

### hyperspy

HyperSpy is an open source Python library which provides tools to facilitate the interactive  data analysis of multi-dimensional datasets that can be described as multi-dimensional arrays of a given signal  (e.g. a 2D array of spectra a.k.a spectrum image)

*homepage*: <https://hyperspy.org/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.5.2``|``intel/2016b``

### HyPhy

HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package   for the analysis of genetic sequences (in particular the inference of natural selection)   using techniques in phylogenetics, molecular evolution, and machine learning

*homepage*: <https://veg.github.io/hyphy-site/>

version   |toolchain
----------|---------------
``2.3.13``|``foss/2016b``
``2.5.1`` |``gompi/2019a``
``2.5.33``|``gompi/2021a``

### HyPo

HyPo: Super Fast & Accurate Polisher for Long Read Genome Assemblies

*homepage*: <https://github.com/kensung-lab/hypo>

version  |versionsuffix    |toolchain
---------|-----------------|-------------
``1.0.3``|``-Python-3.7.4``|``GCC/8.3.0``

### hypothesis

Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized  by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets  you find more bugs in your code with less work.

*homepage*: <https://github.com/HypothesisWorks/hypothesis>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------
``4.5.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``4.23.4``|                  |``GCCcore/8.2.0``
``4.39.3``|``-Python-3.6.4`` |``intel/2018a``
``4.44.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``4.53.1``|                  |``GCCcore/10.2.0``
``4.57.1``|``-Python-2.7.18``|``GCCcore/11.2.0``
``5.6.0`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.41.2``|                  |``GCCcore/10.2.0``
``5.41.5``|                  |``GCCcore/10.2.0``
``6.7.0`` |                  |``GCCcore/10.2.0``
``6.13.1``|                  |``GCCcore/10.3.0``
``6.14.6``|                  |``GCCcore/11.2.0``
``6.46.7``|                  |``GCCcore/11.3.0``
``6.68.2``|                  |``GCCcore/12.2.0``

### Hypre

Hypre is a library for solving large, sparse linear systems of equations on massively  parallel computers. The problems of interest arise in the simulation codes being developed at LLNL  and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.

*homepage*: <https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods>

version   |toolchain
----------|----------------------------------------------------------------
``2.11.1``|``foss/2016a``, ``intel/2016a``
``2.14.0``|``foss/2018a``, ``intel/2018a``
``2.15.1``|``foss/2019a``, ``intel/2019a``
``2.18.2``|``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``2.20.0``|``foss/2020b``, ``intel/2020b``
``2.21.0``|``foss/2021a``, ``fosscuda/2020b``, ``intel/2021a``
``2.24.0``|``intel/2021b``
``2.25.0``|``foss/2022a``
``2.27.0``|``foss/2022b``

## I


[i-cisTarget](#i-cistarget) - [i-PI](#i-pi) - [I-TASSER](#i-tasser) - [i7z](#i7z) - [ICA-AROMA](#ica-aroma) - [icc](#icc) - [iccifort](#iccifort) - [iccifortcuda](#iccifortcuda) - [ichorCNA](#ichorcna) - [icmake](#icmake) - [iCount](#icount) - [ICU](#icu) - [IDBA-UD](#idba-ud) - [idemux](#idemux) - [ieeg-cli](#ieeg-cli) - [ifort](#ifort) - [IgBLAST](#igblast) - [IGMPlot](#igmplot) - [igraph](#igraph) - [IGV](#igv) - [igv-reports](#igv-reports) - [IGVTools](#igvtools) - [iibff](#iibff) - [iimkl](#iimkl) - [iimpi](#iimpi) - [iimpic](#iimpic) - [IJulia](#ijulia) - [ILAMB](#ilamb) - [IMa2](#ima2) - [IMa2p](#ima2p) - [imagecodecs](#imagecodecs) - [imageio](#imageio) - [ImageJ](#imagej) - [ImageMagick](#imagemagick) - [imake](#imake) - [Imath](#imath) - [IMB](#imb) - [imbalanced-learn](#imbalanced-learn) - [imgaug](#imgaug) - [imkl](#imkl) - [imkl-FFTW](#imkl-fftw) - [Imlib2](#imlib2) - [immunedeconv](#immunedeconv) - [IMOD](#imod) - [impi](#impi) - [IMPUTE2](#impute2) - [InChI](#inchi) - [indicators](#indicators) - [Inelastica](#inelastica) - [inferCNV](#infercnv) - [infercnvpy](#infercnvpy) - [Inferelator](#inferelator) - [Infernal](#infernal) - [Infomap](#infomap) - [inline](#inline) - [InParanoid](#inparanoid) - [inputproto](#inputproto) - [Inspector](#inspector) - [IntaRNA](#intarna) - [INTEGRATE](#integrate) - [INTEGRATE-Neo](#integrate-neo) - [intel](#intel) - [intel-compilers](#intel-compilers) - [IntelClusterChecker](#intelclusterchecker) - [intelcuda](#intelcuda) - [IntelDAAL](#inteldaal) - [IntelPython](#intelpython) - [InterProScan](#interproscan) - [InterProScan_data](#interproscan_data) - [intervaltree](#intervaltree) - [intervaltree-python](#intervaltree-python) - [intltool](#intltool) - [io_lib](#io_lib) - [ioapi](#ioapi) - [iodata](#iodata) - [iomkl](#iomkl) - [iompi](#iompi) - [IOR](#ior) - [IOzone](#iozone) - [IPM](#ipm) - [Ipopt](#ipopt) - [ipp](#ipp) - [IPy](#ipy) - [ipympl](#ipympl) - [ipyparallel](#ipyparallel) - [ipyrad](#ipyrad) - [IPython](#ipython) - [IQ-TREE](#iq-tree) - [Iris](#iris) - [IRkernel](#irkernel) - [irodsfs](#irodsfs) - [IronPython](#ironpython) - [ISA-L](#isa-l) - [ISL](#isl) - [isoCirc](#isocirc) - [IsoNet](#isonet) - [IsoSeq](#isoseq) - [ispc](#ispc) - [itac](#itac) - [ITK](#itk) - [itpp](#itpp) - [ITSTool](#itstool) - [ITSx](#itsx) - [iVar](#ivar)


### i-cisTarget

An integrative genomics method for the prediction of regulatory features and cis-regulatory modules  in Human, Mouse, and Fly

*homepage*: <https://gbiomed.kuleuven.be/apps/lcb/i-cisTarget>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20160602``|``-Python-2.7.11``|``intel/2016a``

### i-PI

A Python wrapper for (ab initio) (path integrals) molecular dynamics

*homepage*: <https://github.com/i-pi/i-pi>

version         |versionsuffix     |toolchain
----------------|------------------|---------------
``1.0-20160213``|``-Python-2.7.11``|``intel/2016a``

### I-TASSER

I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function modelling and comparison.

*homepage*: <http://zhanglab.ccmb.med.umich.edu/I-TASSER/>

version|toolchain
-------|----------
``4.0``|``system``
``4.2``|``system``
``5.1``|``system``

### i7z

A better i7 (and now i3, i5) reporting tool for Linux 

*homepage*: <https://github.com/ajaiantilal/i7z>

version     |toolchain
------------|------------------
``20131012``|``GCCcore/11.2.0``

### ICA-AROMA

ICA-AROMA (i.e. 'ICA-based Automatic Removal Of Motion Artifacts') concerns a data-driven method  to identify and remove motion-related independent components from fMRI data.

*homepage*: <https://github.com/maartenmennes/ICA-AROMA>

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``0.4.4-beta``|``-Python-3.6.6``|``foss/2018b``

### icc

C and C++ compiler from Intel

*homepage*: <http://software.intel.com/en-us/intel-compilers/>

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2017.6.256``|``-GCC-6.4.0-2.28``  |``system``
``2017.7.259``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``system``    |``-GCC-system-2.29`` |``system``

### iccifort

Intel C, C++ & Fortran compilers

*homepage*: <http://software.intel.com/en-us/intel-cluster-toolkit-compiler/>

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``2019.4.243``|                     |``system``
``2019.5.281``|                     |``system``
``2020.0.166``|                     |``system``
``2020.0.166``|``-GCC-9.2.0``       |``system``
``2020.1.217``|                     |``system``
``2020.4.304``|                     |``system``
``system``    |``-GCC-system-2.29`` |``system``

### iccifortcuda

Intel C, C++ & Fortran compilers with CUDA toolkit

*homepage*: <(none)>

version       |versionsuffix      |toolchain
--------------|-------------------|----------
``2016.10``   |                   |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``|``system``
``2019a``     |                   |``system``
``2019b``     |                   |``system``
``2020a``     |                   |``system``
``2020b``     |                   |``system``

### ichorCNA

ichorCNA is a tool for estimating the fraction of tumor in cell-free DNA from ultra-low-pass whole genome sequencing

*homepage*: <https://github.com/broadinstitute/ichorCNA/wiki>

version           |toolchain
------------------|--------------
``0.2.0``         |``foss/2019b``
``0.3.2-20191219``|``foss/2020a``

### icmake

Icmake is a hybrid between a 'make' utility and a 'shell script' language. Originally, it was written  to provide a useful tool for automatic program maintenance and system administrative tasks on old MS-DOS platforms.

*homepage*: <http://icmake.sourceforge.net/>

version    |toolchain
-----------|--------------
``7.23.02``|``foss/2016a``

### iCount

iCount: protein-RNA interaction analysis  is a Python module and associated command-line interface (CLI),  which provides all the commands needed to process iCLIP data on protein-RNA interactions.

*homepage*: <https://github.com/tomazc/iCount>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180820``|``-Python-3.6.6``|``foss/2018b``

### ICU

ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.

*homepage*: <https://icu.unicode.org>

version |toolchain
--------|--------------------------------------
``61.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``64.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``65.1``|``GCCcore/8.3.0``
``66.1``|``GCCcore/9.3.0``
``67.1``|``GCCcore/10.2.0``
``69.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``71.1``|``GCCcore/11.3.0``
``72.1``|``GCCcore/12.2.0``

### IDBA-UD

IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads  Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.  IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth  contigs are removed iteratively with cutoff threshold from low to high to reduce the errors  in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled  locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD  can iterate k value of de Bruijn graph to a very large value with less gaps and less branches  to form long contigs in both low-depth and high-depth regions.

*homepage*: <http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------
``1.1.3``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018a``, ``foss/2018b``

### idemux

idemux - inline barcode demultiplexing Idemux is a command line tool designed to demultiplex paired-end FASTQ files from QuantSeq-Pool.

*homepage*: <https://github.com/Lexogen-Tools/idemux>

version  |toolchain
---------|------------------
``0.1.6``|``GCCcore/10.2.0``

### ieeg-cli

IEEG.ORG is a collaborative initiative funded by the National Institutes  of Neurological Disorders and Stroke. This initiative seeks to advance research towards  the understanding of epilepsy by providing a platform for sharing data,  tools and expertise between researchers.

*homepage*: <https://www.ieeg.org/>

version    |toolchain
-----------|----------
``1.14.56``|``system``

### ifort

Fortran compiler from Intel

*homepage*: <http://software.intel.com/en-us/intel-compilers/>

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2017.6.256``|``-GCC-6.4.0-2.28``  |``system``
``2017.7.259``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``system``    |``-GCC-system-2.29`` |``system``

### IgBLAST

IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.

*homepage*: <https://ncbi.github.io/igblast>

version   |versionsuffix |toolchain
----------|--------------|----------
``1.15.0``|``-x64-linux``|``system``
``1.18.0``|``-x64-linux``|``system``
``1.21.0``|``-x64-linux``|``system``

### IGMPlot

IGMPlot is a free open-source program developed to identify molecular interactions and  prepare data to build 2D and 3D representations of them in the molecular environment.

*homepage*: <http://igmplot.univ-reims.fr>

version   |toolchain
----------|--------------------------------------
``2.4.2`` |``GCC/8.3.0``, ``iccifort/2019.5.281``
``2.6.9b``|``GCC/11.2.0``

### igraph

igraph is a collection of network analysis tools with the emphasis on efficiency, portability and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++.

*homepage*: <https://igraph.org>

version   |toolchain
----------|------------------------------------------------
``0.7.1`` |``foss/2018b``, ``intel/2016b``, ``intel/2017b``
``0.8.0`` |``foss/2019b``
``0.8.2`` |``foss/2020a``
``0.8.5`` |``foss/2020b``
``0.9.1`` |``foss/2020b``, ``fosscuda/2020b``
``0.9.4`` |``foss/2021a``
``0.9.5`` |``foss/2021b``
``0.10.3``|``foss/2022a``

### IGV

This package contains command line utilities for  preprocessing, computing feature count density (coverage),  sorting, and  indexing data files.

*homepage*: <https://www.broadinstitute.org/software/igv/>

version   |versionsuffix     |toolchain
----------|------------------|----------
``2.3.68``|``-Java-1.7.0_80``|``system``
``2.3.80``|``-Java-1.7.0_80``|``system``
``2.5.0`` |``-Java-11``      |``system``
``2.8.0`` |``-Java-11``      |``system``
``2.9.4`` |``-Java-11``      |``system``
``2.12.3``|``-Java-11``      |``system``
``2.16.0``|``-Java-11``      |``system``

### igv-reports

Python application to generate self-contained igv.js pages that can be opened within a browser with "file" protocol.

*homepage*: <https://github.com/igvteam/igv-reports>

version  |versionsuffix    |toolchain
---------|-----------------|-------------
``0.9.8``|``-Python-3.7.4``|``GCC/8.3.0``

### IGVTools

This package contains command line utilities for preprocessing,   computing feature count density (coverage),  sorting, and indexing data files.  See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 

*homepage*: <https://software.broadinstitute.org/software/igv/igvtools>

version   |versionsuffix     |toolchain
----------|------------------|----------
``2.3.68``|``-Java-1.7.0_80``|``system``
``2.3.72``|``-Java-1.7.0_80``|``system``
``2.3.75``|``-Java-1.7.0_80``|``system``
``2.4.18``|``-Java-1.8``     |``system``

### iibff

GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.

*homepage*: <(none)>

version  |toolchain
---------|----------
``2020b``|``system``

### iimkl

Intel C/C++ and Fortran compilers, alongside Intel Math Kernel Library (MKL).

*homepage*: <https://software.intel.com/en-us/intel-cluster-toolkit-compiler/>

version  |toolchain
---------|----------
``2018a``|``system``
``2022b``|``system``

### iimpi

Intel C/C++ and Fortran compilers, alongside Intel MPI.

*homepage*: <http://software.intel.com/en-us/intel-cluster-toolkit-compiler/>

version              |versionsuffix       |toolchain
---------------------|--------------------|----------
``8.1.5``            |``-GCC-4.9.3-2.25`` |``system``
``2016.00``          |``-GCC-4.9.3-2.25`` |``system``
``2016.01``          |``-GCC-4.9.3-2.25`` |``system``
``2016.02``          |``-GCC-4.9.3-2.25`` |``system``
``2016.02``          |``-GCC-5.3.0-2.26`` |``system``
``2016.03``          |``-GCC-4.9.3-2.25`` |``system``
``2016.03``          |``-GCC-5.3.0-2.26`` |``system``
``2016.03``          |``-GCC-5.4.0-2.26`` |``system``
``2016b``            |                    |``system``
``2017.00``          |``-GCC-5.4.0-2.26`` |``system``
``2017.01``          |``-GCC-5.4.0-2.26`` |``system``
``2017.02``          |``-GCC-6.3.0-2.27`` |``system``
``2017.09``          |                    |``system``
``2017a``            |                    |``system``
``2017b``            |                    |``system``
``2018.00``          |                    |``system``
``2018.01``          |                    |``system``
``2018.02``          |                    |``system``
``2018.04``          |                    |``system``
``2018a``            |                    |``system``
``2018b``            |                    |``system``
``2019.00``          |                    |``system``
``2019.01``          |                    |``system``
``2019.02``          |                    |``system``
``2019.03``          |                    |``system``
``2019a``            |                    |``system``
``2019b``            |                    |``system``
``2020.00``          |                    |``system``
``2020.06-impi-18.5``|                    |``system``
``2020.12``          |                    |``system``
``2020a``            |                    |``system``
``2020b``            |                    |``system``
``2021a``            |                    |``system``
``2021b``            |                    |``system``
``2022.00``          |                    |``system``
``2022.05``          |                    |``system``
``2022.09``          |                    |``system``
``2022.11``          |                    |``system``
``2022.12``          |                    |``system``
``2022a``            |                    |``system``
``2022b``            |                    |``system``
``2023.03``          |                    |``system``
``2023a``            |                    |``system``
``system``           |``-GCC-system-2.29``|``system``

### iimpic

Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2016.10``|``system``
``2017b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

### IJulia

Julia kernel for Jupyter

*homepage*: <https://github.com/JuliaLang/IJulia.jl>

version   |versionsuffix   |toolchain
----------|----------------|----------
``1.23.3``|``-Julia-1.6.7``|``system``
``1.24.0``|``-Julia-1.8.5``|``system``

### ILAMB

The International Land Model Benchmarking (ILAMB) project is a model-data intercomparison and integration project designed to improve the performance of land models and, in parallel, improve the design of new measurement campaigns to reduce uncertainties associated with key land surface processes. 

*homepage*: <https://www.ilamb.org/>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``2.5``|``-Python-3.8.2``|``foss/2020a``

### IMa2

IMa2 is a progam for population genetic analysis that can handle two or more populations.

*homepage*: <https://bio.cst.temple.edu/~hey/software/software.htm#IMa2>

version    |toolchain
-----------|--------------
``8.27.12``|``foss/2016a``

### IMa2p

IMa2p is a parallel implementation of IMa2, using OpenMPI-C++ - a Bayesian MCMC  based method for inferring population demography under the IM (Isolation with  Migration) model. http://dx.doi.org/10.1111/1755-0998.12437 

*homepage*: <https://github.com/arunsethuraman/ima2p>

version     |toolchain
------------|---------------
``20151123``|``foss/2016a``
``20160804``|``intel/2016b``

### imagecodecs

Imagecodecs is a Python library that provides block-oriented, in-memory buffer transformation, compression, and decompression functions for use in the tifffile, czifile, zarr, and other scientific image input/output modules.

*homepage*: <https://github.com/cgohlke/imagecodecs>

version      |toolchain
-------------|------------------------------
``2021.8.26``|``foss/2020b``
``2022.9.26``|``foss/2021a``, ``foss/2022a``

### imageio

Imageio is a Python library that provides an simple interface to read and write a wide range of  image data, including animated images, video, volumetric data, and scientific formats.

*homepage*: <https://imageio.github.io>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``2.3.0`` |``-Python-3.6.4``|``intel/2018a``
``2.5.0`` |                 |``foss/2019a``
``2.9.0`` |                 |``foss/2020b``, ``fosscuda/2020b``
``2.9.0`` |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``2.10.5``|                 |``foss/2021a``
``2.13.5``|                 |``foss/2021b``
``2.22.2``|                 |``foss/2022a``

### ImageJ

Image Processing and Analysis in Java

*homepage*: <https://imagej.nih.gov/ij>

version  |versionsuffix|toolchain
---------|-------------|----------
``1.51a``|             |``system``
``1.51i``|             |``system``
``1.51k``|             |``system``
``1.52q``|``-Java-1.8``|``system``

### ImageMagick

ImageMagick is a software suite to create, edit, compose, or convert bitmap images

*homepage*: <https://www.imagemagick.org/>

version      |versionsuffix                      |toolchain
-------------|-----------------------------------|-------------------------------
``6.9.4-8``  |                                   |``intel/2016a``
``7.0.1-6``  |                                   |``intel/2016a``
``7.0.1-9``  |                                   |``intel/2016a``
``7.0.2-9``  |                                   |``intel/2016a``
``7.0.3-1``  |                                   |``intel/2016b``
``7.0.5-4``  |                                   |``intel/2017a``
``7.0.5-10`` |                                   |``foss/2016b``
``7.0.7-8``  |``-JasPer-1.900.1``                |``intel/2017a``
``7.0.7-14`` |                                   |``foss/2017b``, ``intel/2017b``
``7.0.7-15`` |                                   |``GCCcore/6.4.0``
``7.0.7-26`` |                                   |``foss/2018a``
``7.0.7-30`` |                                   |``GCCcore/6.4.0``
``7.0.7-30`` |``-Ghostscript-9.22-cairo-1.14.12``|``GCCcore/6.4.0``
``7.0.7-39`` |``-Ghostscript-9.23-cairo-1.14.12``|``GCCcore/6.4.0``
``7.0.8-11`` |                                   |``GCCcore/7.3.0``
``7.0.8-46`` |                                   |``GCCcore/8.2.0``
``7.0.9-5``  |                                   |``GCCcore/8.3.0``
``7.0.10-1`` |                                   |``GCCcore/9.3.0``
``7.0.10-35``|                                   |``GCCcore/10.2.0``
``7.0.11-14``|                                   |``GCCcore/10.3.0``
``7.1.0-4``  |                                   |``GCCcore/11.2.0``
``7.1.0-37`` |                                   |``GCCcore/11.3.0``
``7.1.0-53`` |                                   |``GCCcore/12.2.0``

### imake

imake is a Makefile-generator that is intended to make it easier to develop software portably for multiple systems.

*homepage*: <https://www.x.org/>

version  |toolchain
---------|----------------------------------------------------------
``1.0.7``|``intel/2016a``
``1.0.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### Imath

Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics 

*homepage*: <https://imath.readthedocs.io/en/latest/>

version  |toolchain
---------|------------------
``3.1.5``|``GCCcore/11.3.0``
``3.1.6``|``GCCcore/12.2.0``
``3.1.7``|``GCCcore/12.3.0``

### IMB

The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and  global communication operations for a range of message sizes

*homepage*: <https://software.intel.com/en-us/articles/intel-mpi-benchmarks>

version   |toolchain
----------|-----------------------------------------------------------------------------------
``4.1``   |``foss/2016a``, ``foss/2017a``, ``intel/2017.02``, ``intel/2017a``
``2018.1``|``intel/2017a``
``2019.3``|``gompi/2019a``, ``iimpi/2019a``
``2021.3``|``gompi/2021b``, ``gompi/2022a``, ``gompi/2022b``, ``iimpi/2022a``, ``iimpi/2022b``

### imbalanced-learn

imbalanced-learn is a Python package offering a number of re-sampling techniques commonly used in  datasets showing strong between-class imbalance.

*homepage*: <https://github.com/scikit-learn-contrib/imbalanced-learn>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.2.1`` |``-Python-2.7.12``|``intel/2016b``
``0.2.1`` |``-Python-3.5.2`` |``intel/2016b``
``0.3.3`` |``-Python-3.6.4`` |``foss/2018a``
``0.4.3`` |``-Python-3.6.6`` |``foss/2018b``
``0.7.0`` |                  |``foss/2020b``
``0.9.0`` |                  |``foss/2021b``
``0.10.1``|                  |``foss/2022a``

### imgaug

This python library helps you with augmenting images for your machine learning projects.  It converts a set of input images into a new, much larger set of slightly altered images. 

*homepage*: <https://imgaug.readthedocs.io/en/latest/>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.2.8``|``-Python-3.6.6``|``foss/2018b``
``0.4.0``|                 |``foss/2021a``, ``foss/2021b``, ``foss/2022a``
``0.4.0``|``-CUDA-11.3.1`` |``foss/2021a``
``0.4.0``|``-CUDA-11.4.1`` |``foss/2021b``
``0.4.0``|``-CUDA-11.7.0`` |``foss/2022a``
``0.4.0``|``-Python-3.7.4``|``foss/2019b``

### imkl

Intel oneAPI Math Kernel Library

*homepage*: <https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html>

version       |versionsuffix|toolchain
--------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``11.2.3.187``|             |``gimpi/2.11.5``
``11.3.0.109``|             |``iimpi/2016.00-GCC-4.9.3-2.25``
``11.3.1.150``|             |``iimpi/2016.01-GCC-4.9.3-2.25``, ``iimpi/8.1.5-GCC-4.9.3-2.25``
``11.3.2.181``|             |``iimpi/2016.02-GCC-4.9.3-2.25``, ``iimpi/2016.02-GCC-5.3.0-2.26``, ``pompi/2016.03``
``11.3.3.210``|             |``iimpi/2016.03-GCC-4.9.3-2.25``, ``iimpi/2016.03-GCC-5.3.0-2.26``, ``iimpi/2016.03-GCC-5.4.0-2.26``, ``iimpi/2016b``, ``iimpic/2016.10``, ``iompi/2016.07``, ``iompi/2016.09-GCC-4.9.3-2.25``, ``iompi/2016.09-GCC-5.4.0-2.26``, ``pompi/2016.04``, ``pompi/2016.09``
``2017.0.098``|             |``iimpi/2017.00-GCC-5.4.0-2.26``
``2017.1.132``|             |``gimpi/2017a``, ``iimpi/2017.01-GCC-5.4.0-2.26``, ``iimpi/2017a``, ``iompi/2017.01``, ``iompi/2017a``
``2017.2.174``|             |``iimpi/2017.02-GCC-6.3.0-2.27``
``2017.3.196``|             |``gompi/2017b``, ``iimpi/2017b``, ``iimpic/2017b``, ``iompi/2017b``
``2017.4.239``|             |``iimpi/2017.09``
``2018.0.128``|             |``iimpi/2018.00``
``2018.1.163``|             |``iimpi/2018.01``, ``iimpi/2018a``, ``iompi/2018a``
``2018.1.163``|``-serial``  |``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2018.2.199``|             |``iimpi/2018.02``, ``iompi/2018.02``
``2018.3.222``|             |``gimpi/2018b``, ``gompi/2018b``, ``iimpi/2018b``, ``iompi/2018b``
``2018.4.274``|             |``iimpi/2018.04``
``2019.0.117``|             |``iimpi/2019.00``
``2019.1.144``|             |``gompi/2019a``, ``iimpi/2019.01``, ``iimpi/2019a``, ``iimpic/2019a``, ``iompi/2019.01``
``2019.2.187``|             |``iimpi/2019.02``
``2019.3.199``|             |``iimpi/2019.03``
``2019.5.281``|             |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``, ``iompi/2019b``
``2020.0.166``|             |``iimpi/2020.00``
``2020.1.217``|             |``gompi/2020a``, ``iimpi/2020.06-impi-18.5``, ``iimpi/2020a``, ``iimpic/2020a``, ``iompi/2020a``
``2020.4.304``|             |``NVHPC/21.2``, ``gompi/2020b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpic/2020b``, ``iompi/2020b``
``2021.1.1``  |             |``iimpi/2020.12``
``2021.2.0``  |             |``gompi/2021a``, ``iimpi/2021a``, ``iompi/2021a``
``2021.3.0``  |             |``gompi/2021a``
``2021.4.0``  |             |``iompi/2021b``, ``system``
``2022.0.1``  |             |``system``
``2022.1.0``  |             |``gompi/2022a``, ``system``
``2022.2.0``  |             |``system``
``2022.2.1``  |             |``system``
``2023.0.0``  |             |``system``
``2023.1.0``  |             |``system``

### imkl-FFTW

FFTW interfaces using Intel oneAPI Math Kernel Library

*homepage*: <https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html>

version     |toolchain
------------|----------------------------------
``2021.4.0``|``gompi/2021b``, ``iimpi/2021b``
``2022.0.1``|``iimpi/2022.00``
``2022.1.0``|``iimpi/2022.05``, ``iimpi/2022a``
``2022.2.0``|``iimpi/2022.09``
``2022.2.1``|``iimpi/2022.11``, ``iimpi/2022b``
``2023.0.0``|``iimpi/2022.12``
``2023.1.0``|``iimpi/2023.03``, ``iimpi/2023a``

### Imlib2

This is the Imlib 2 library - a library that does image file loading and saving as well as rendering, manipulation, arbitrary polygon support, etc. It does ALL of these operations FAST. Imlib2 also tries to be highly intelligent about doing them, so writing naive programs can be done easily, without sacrificing speed. 

*homepage*: <https://docs.enlightenment.org/api/imlib2/html/>

version  |toolchain
---------|-----------------
``1.5.1``|``GCCcore/6.4.0``

### immunedeconv

immunedeconv is an R package for unified access to computational methods for estimating immune cell fractions from bulk RNA sequencing data.

*homepage*: <https://github.com/icbi-lab/immunedeconv>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.0.2``|``-R-4.0.0`` |``foss/2020a``

### IMOD

IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections. The package contains tools for assembling and aligning data within multiple types and sizes of image stacks, viewing 3-D data from any orientation, and modeling and display of the image files. IMOD was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim Kremer, Quanren Xiong, and John Heumann at the University of Colorado.

*homepage*: <http://bio3d.colorado.edu/imod/>

version   |versionsuffix        |toolchain
----------|---------------------|----------------------------------
``4.7.15``|``_RHEL6-64_CUDA6.0``|``system``
``4.11.5``|                     |``foss/2020b``, ``fosscuda/2020b``

### impi

The Intel(R) MPI Library for Linux* OS is a multi-fabric message  passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for  Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification.

*homepage*: <http://software.intel.com/en-us/intel-mpi-library/>

version        |versionsuffix|toolchain
---------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.2.006``  |             |``system``
``4.0.0.028``  |             |``system``
``4.0.0.028``  |``-32bit``   |``system``
``4.0.2.003``  |             |``system``
``4.1.0.027``  |             |``system``
``4.1.0.030``  |             |``system``
``4.1.1.036``  |             |``system``
``4.1.2.040``  |             |``system``
``4.1.3.045``  |             |``system``
``4.1.3.049``  |             |``GCC/4.8.3``, ``system``
``5.0.3.048``  |             |``GCC/4.9.3``
``5.1.1.109``  |             |``iccifort/2016.0.109-GCC-4.9.3-2.25``
``5.1.2.150``  |             |``iccifort/2016.1.150-GCC-4.9.3-2.25``
``5.1.3.181``  |             |``iccifort/2016.2.181-GCC-4.9.3-2.25``, ``iccifort/2016.2.181-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``iccifortcuda/2016.10``
``2017.0.098`` |             |``iccifort/2017.0.098-GCC-5.4.0-2.26``
``2017.1.132`` |             |``GCC/5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``2017.2.174`` |             |``iccifort/2017.2.174-GCC-6.3.0-2.27``
``2017.3.196`` |             |``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifortcuda/2017.4.196-GCC-6.4.0-2.28``
``2017.4.239`` |             |``iccifort/2017.5.239-GCC-6.4.0-2.28``
``2018.0.128`` |             |``iccifort/2018.0.128-GCC-6.4.0-2.28``
``2018.1.163`` |             |``GCC/6.4.0-2.28``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2018.2.199`` |             |``iccifort/2018.2.199-GCC-6.4.0-2.28``
``2018.3.222`` |             |``GCC/7.3.0-2.30``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2018.4.274`` |             |``iccifort/2018.5.274-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifortcuda/2019a``
``2018.5.288`` |             |``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifortcuda/2019b``
``2019.0.117`` |             |``iccifort/2019.0.117-GCC-8.2.0-2.31.1``
``2019.1.144`` |             |``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2019.2.187`` |             |``iccifort/2019.2.187-GCC-8.2.0-2.31.1``
``2019.3.199`` |             |``iccifort/2019.3.199-GCC-8.3.0-2.32``
``2019.6.166`` |             |``iccifort/2020.0.166-GCC-9.2.0``
``2019.7.217`` |             |``iccifort/2020.1.217``, ``iccifortcuda/2020a``
``2019.9.304`` |             |``iccifort/2020.4.304``, ``iccifortcuda/2020b``
``2019.12.320``|             |``iccifort/2020.4.304``
``2021.1.1``   |             |``intel-compilers/2021.1.2``
``2021.2.0``   |             |``intel-compilers/2021.2.0``
``2021.3.0``   |             |``intel-compilers/2021.3.0``
``2021.4.0``   |             |``intel-compilers/2021.4.0``
``2021.5.0``   |             |``intel-compilers/2022.0.1``
``2021.6.0``   |             |``intel-compilers/2022.1.0``
``2021.7.0``   |             |``intel-compilers/2022.2.0``
``2021.7.1``   |             |``intel-compilers/2022.2.1``
``2021.8.0``   |             |``intel-compilers/2023.0.0``
``2021.9.0``   |             |``intel-compilers/2023.1.0``
``system``     |             |``iccifort/system-GCC-system-2.29``

### IMPUTE2

IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation   and haplotype phasing program based on ideas from Howie et al. 2009 

*homepage*: <http://mathgen.stats.ox.ac.uk/impute/impute_v2.html>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``2.3.0``|``_x86_64_dynamic``|``system``
``2.3.0``|``_x86_64_static`` |``system``
``2.3.2``|``_x86_64_dynamic``|``system``
``2.3.2``|``_x86_64_static`` |``system``

### InChI

The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary identifier for chemical substances that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.

*homepage*: <https://www.inchi-trust.org/>

version |toolchain
--------|--------------
``1.06``|``GCC/10.3.0``

### indicators

- Thread-safe progress bars and spinners - Header-only library. Grab a copy of include/indicators. - Single-header version in single_include/indicators. 

*homepage*: <https://github.com/p-ranav/indicators>

version|toolchain
-------|------------------
``2.2``|``GCCcore/11.3.0``

### Inelastica

Python package for eigenchannels, vibrations and inelastic electron transport based on SIESTA/TranSIESTA DFT.

*homepage*: <https://github.com/tfrederiksen/inelastica>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.3.5``|``-Python-2.7.15``|``intel/2018b``

### inferCNV

InferCNV is used to explore tumor single cell RNA-Seq data to identify evidence  for somatic large-scale chromosomal copy number alterations, such as gains or  deletions of entire chromosomes or large segments of chromosomes.

*homepage*: <https://github.com/broadinstitute/inferCNV/wiki>

version   |versionsuffix            |toolchain
----------|-------------------------|------------------------------
``1.0.4`` |``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``1.2.1`` |``-Python-3.7.4``        |``foss/2019b``
``1.3.3`` |                         |``foss/2020b``, ``foss/2021a``
``1.3.3`` |``-Python-3.8.2``        |``foss/2020a``
``1.10.1``|``-R-4.1.2``             |``foss/2021b``

### infercnvpy

Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.

*homepage*: <https://github.com/icbi-lab/infercnvpy>

version  |toolchain
---------|--------------
``0.4.0``|``foss/2021b``
``0.4.2``|``foss/2022a``

### Inferelator

Inferelator 3.0 is a package for gene regulatory network inference that is based on  regularized regression.

*homepage*: <https://github.com/flatironinstitute/inferelator>

version  |toolchain
---------|--------------
``0.6.1``|``foss/2022a``

### Infernal

Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases  for RNA structure and sequence similarities.

*homepage*: <http://eddylab.org/infernal/>

version  |toolchain
---------|------------------------------------------------------------------------------
``1.1.2``|``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018b``
``1.1.4``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``foss/2022b``

### Infomap

Multi-level network clustering based on the Map equation.

*homepage*: <https://www.mapequation.org/code.html#Linux>

version     |toolchain
------------|--------------------
``20190308``|``GCC/8.2.0-2.31.1``

### inline

Functionality to dynamically define R functions and S4 methods  with 'inlined' C, C++ or Fortran code supporting the .C and .Call calling  conventions.

*homepage*: <https://cran.r-project.org/web/packages/inline>

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.3.19``|``-R-4.0.4`` |``foss/2020b``

### InParanoid

InParanoid: ortholog groups with inparalogs.

*homepage*: <https://inparanoid.sbc.su.se>

version         |toolchain
----------------|--------------
``5.0-20220607``|``GCC/10.3.0``

### inputproto

X.org InputProto protocol headers.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``2.3.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.3.2``|``intel/2016a``

### Inspector

Intel Inspector is a dynamic memory and threading error  checking tool for users developing serial and parallel applications

*homepage*: <https://software.intel.com/en-us/inspector>

version         |toolchain
----------------|----------
``2013_update6``|``system``
``2013_update7``|``system``
``2016_update3``|``system``
``2017_update1``|``system``
``2017_update2``|``system``
``2018_update1``|``system``
``2018_update2``|``system``
``2018_update3``|``system``
``2019_update2``|``system``
``2019_update5``|``system``
``2021.4.0``    |``system``
``2022.0.0``    |``system``
``2022.1.0``    |``system``

### IntaRNA

Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites

*homepage*: <https://github.com/BackofenLab/IntaRNA>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.1``|``-Python-2.7.15``|``foss/2018b``

### INTEGRATE

INTEGRATE is a tool calling gene fusions with exact fusion junctions and genomic breakpoints  by combining RNA-Seq and WGS data. It is highly sensitive and accurate by applying a fast split-read  mapping algorithm based on Burrow-Wheeler transform. 

*homepage*: <https://sourceforge.net/p/integrate-fusion/wiki/Home/>

version  |toolchain
---------|-----------------
``0.2.6``|``GCCcore/8.2.0``

### INTEGRATE-Neo

INTEGRATE-Neo is a gene fusion neoantigen discovering tool using next-generation sequencing data.  It is written in C++ and Python. 

*homepage*: <>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.1``|``-Python-3.6.6``|``foss/2018b``

### intel

Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).

*homepage*: <https://simplebuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain>

version              |versionsuffix|toolchain
---------------------|-------------|----------
``2016.00``          |             |``system``
``2016.01``          |             |``system``
``2016.02``          |``-GCC-4.9`` |``system``
``2016.02``          |``-GCC-5.3`` |``system``
``2016.03``          |``-GCC-4.9`` |``system``
``2016.03``          |``-GCC-5.3`` |``system``
``2016.03``          |``-GCC-5.4`` |``system``
``2016a``            |             |``system``
``2016b``            |             |``system``
``2017.00``          |             |``system``
``2017.01``          |             |``system``
``2017.02``          |             |``system``
``2017.09``          |             |``system``
``2017a``            |             |``system``
``2017b``            |             |``system``
``2018.00``          |             |``system``
``2018.01``          |             |``system``
``2018.02``          |             |``system``
``2018.04``          |             |``system``
``2018a``            |             |``system``
``2018b``            |             |``system``
``2019.00``          |             |``system``
``2019.01``          |             |``system``
``2019.02``          |             |``system``
``2019.03``          |             |``system``
``2019a``            |             |``system``
``2019b``            |             |``system``
``2020.00``          |             |``system``
``2020.06-impi-18.5``|             |``system``
``2020.12``          |             |``system``
``2020a``            |             |``system``
``2020b``            |             |``system``
``2021a``            |             |``system``
``2021b``            |             |``system``
``2022.00``          |             |``system``
``2022.05``          |             |``system``
``2022.09``          |             |``system``
``2022.11``          |             |``system``
``2022.12``          |             |``system``
``2022a``            |             |``system``
``2022b``            |             |``system``
``2023.03``          |             |``system``
``2023a``            |             |``system``

### intel-compilers

Intel C, C++ & Fortran compilers (classic and oneAPI)

*homepage*: <https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html>

version     |toolchain
------------|----------
``2021.1.2``|``system``
``2021.2.0``|``system``
``2021.3.0``|``system``
``2021.4.0``|``system``
``2022.0.1``|``system``
``2022.0.2``|``system``
``2022.1.0``|``system``
``2022.2.0``|``system``
``2022.2.1``|``system``
``2023.0.0``|``system``
``2023.1.0``|``system``

### IntelClusterChecker

Intel Cluster Checker verifies the configuration and  performance of Linux OS-based clusters. Anomalies and performance differences  can be identified and practical resolutions provided. 

*homepage*: <https://software.intel.com/en-us/cluster-checker>

version       |toolchain
--------------|----------
``2017.1.016``|``system``
``2021.5.0``  |``system``

### intelcuda

Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,  Intel MPI & Intel MKL, with CUDA toolkit

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2016.10``|``system``
``2017b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

### IntelDAAL

Intel® Data Analytics Acceleration Library (Intel® DAAL) is the library of  Intel® architecture optimized building blocks covering all stages of data analytics: data acquisition from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.  

*homepage*: <https://software.intel.com/en-us/daal>

version       |toolchain
--------------|----------
``2019.4.007``|``system``

### IntelPython

Intel® Distribution for Python. Powered by Anaconda.  Accelerating Python* performance on modern architectures from Intel. 

*homepage*: <https://software.intel.com/en-us/intel-distribution-for-python>

version   |versionsuffix  |toolchain
----------|---------------|----------
``2.7.15``|``-2019.2.066``|``system``
``3.6.8`` |``-2019.2.066``|``system``

### InterProScan

InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [code only: libraries and external binaries  but no data].

*homepage*: <https://www.ebi.ac.uk/interpro/>

version      |toolchain
-------------|------------------
``5.26-65.0``|``intel/2017b``
``5.27-66.0``|``intel/2017b``
``5.28-67.0``|``intel/2018a``
``5.52-86.0``|``GCCcore/10.3.0``
``5.55-88.0``|``foss/2021a``
``5.62-94.0``|``foss/2022b``

### InterProScan_data

InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [data only].

*homepage*: <https://www.ebi.ac.uk/interpro/>

version      |toolchain
-------------|--------------
``5.55-88.0``|``foss/2021a``

### intervaltree

An interval tree can be used to efficiently find a set of numeric intervals  overlapping or containing another interval. This library provides a basic implementation of an  interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. 

*homepage*: <https://github.com/ekg/intervaltree>

version|toolchain
-------|-----------------------------------------------------------------------------
``0.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### intervaltree-python

A mutable, self-balancing interval tree. Queries may be by  point, by range overlap, or by range containment. 

*homepage*: <https://github.com/chaimleib/intervaltree>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``3.0.2``|``-Python-3.6.6``|``foss/2018b``
``3.1.0``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### intltool

intltool is a set of tools to centralize translation of  many different file formats using GNU gettext-compatible PO files.

*homepage*: <http://freedesktop.org/wiki/Software/intltool/>

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.51.0``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.51.0``|``-Perl-5.20.3``|``intel/2016a``
``0.51.0``|``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``0.51.0``|``-Perl-5.24.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``0.51.0``|``-Perl-5.24.1``|``GCCcore/6.3.0``, ``intel/2017a``
``0.51.0``|``-Perl-5.26.0``|``GCCcore/6.4.0``
``0.51.0``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``0.51.0``|``-Perl-5.28.0``|``GCCcore/7.3.0``

### io_lib

Io_lib is a library of file reading and writing code to provide a general purpose trace file (and  Experiment File) reading interface. The programmer simply calls the (eg) read_reading to create a "Read" C structure  with the data loaded into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS   Windows.

*homepage*: <http://sourceforge.net/projects/staden/files/io_lib/>

version   |toolchain
----------|--------------
``1.14.8``|``foss/2016a``

### ioapi

The Models-3/EDSS Input/Output Applications Programming Interface (I/O API) provides the  environmental model developer with an simple-to-learn, simple-to-use programming library for data storage and  access, available from both Fortran and C. The same routines can be used for both file storage (using netCDF  files) and model coupling (using PVM mailboxes). It is the standard data access library for both the  NCSC/CMAS's EDSS project and EPA's Models-3, CMAQ, and SMOKE, as well as various other atmospheric and  hydrological modeling systems.

*homepage*: <https://www.cmascenter.org/ioapi/>

version        |versionsuffix|toolchain
---------------|-------------|---------------
``3.2-2020111``|``-nocpl``   |``gompi/2019b``

### iodata

Python library for reading, writing, and converting computational chemistry file formats and  generating input files.

*homepage*: <https://github.com/theochem/iodata>

version    |toolchain
-----------|---------------
``1.0.0a2``|``intel/2022a``

### iomkl

Compiler toolchain including Intel compilers, Open MPI and Intel Math Kernel Library (MKL).

*homepage*: <https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html>

version                   |toolchain
--------------------------|----------
``2016.07``               |``system``
``2016.09-GCC-4.9.3-2.25``|``system``
``2016.09-GCC-5.4.0-2.26``|``system``
``2017.01``               |``system``
``2017a``                 |``system``
``2017b``                 |``system``
``2018.02``               |``system``
``2018a``                 |``system``
``2018b``                 |``system``
``2019.01``               |``system``
``2019b``                 |``system``
``2020a``                 |``system``
``2020b``                 |``system``
``2021a``                 |``system``
``2021b``                 |``system``

### iompi

Intel C/C++ and Fortran compilers, alongside Open MPI.

*homepage*: <https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html>

version                   |toolchain
--------------------------|----------
``2016.07``               |``system``
``2016.09-GCC-4.9.3-2.25``|``system``
``2016.09-GCC-5.4.0-2.26``|``system``
``2017.01``               |``system``
``2017a``                 |``system``
``2017b``                 |``system``
``2018.02``               |``system``
``2018a``                 |``system``
``2018b``                 |``system``
``2019.01``               |``system``
``2019b``                 |``system``
``2020a``                 |``system``
``2020b``                 |``system``
``2021a``                 |``system``
``2021b``                 |``system``

### IOR

The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,  or HDF5 interfaces. 

*homepage*: <https://github.com/hpc/ior>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------
``3.0.1``|``-mpiio``   |``foss/2016a``
``3.2.1``|             |``gompi/2019b``
``3.3.0``|             |``gompi/2020b``, ``gompi/2021a``, ``gompi/2022a``

### IOzone

IOzone is a filesystem benchmark tool. The benchmark generates and measures a variety of file  operations. Iozone has been ported to many machines and runs under many operating systems.

*homepage*: <http://www.iozone.org/>

version  |toolchain
---------|--------------
``3.434``|``foss/2016a``

### IPM

IPM is a portable profiling infrastructure for parallel codes. It provides  a low-overhead profile of application performance and resource utilization  in a parallel program. Communication, computation, and IO are the primary  focus. 

*homepage*: <https://github.com/nerscadmin/IPM>

version  |toolchain
---------|-----------------------------------------------------------------------------------
``2.0.6``|``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iompi/2020a``

### Ipopt

IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt)  is an open source software package for large-scale nonlinear optimization. 

*homepage*: <https://projects.coin-or.org/Ipopt>

version    |toolchain
-----------|---------------
``3.12.9`` |``foss/2017b``
``3.12.13``|``intel/2019a``

### ipp

Intel Integrated Performance Primitives (Intel IPP) is an extensive library  of multicore-ready, highly optimized software functions for multimedia, data processing,  and communications applications. Intel IPP offers thousands of optimized functions  covering frequently used fundamental algorithms.

*homepage*: <https://software.intel.com/en-us/articles/intel-ipp/>

version       |toolchain
--------------|----------
``7.0.5.233`` |``system``
``8.1.0.144`` |``system``
``9.0.1.150`` |``system``
``2017.1.132``|``system``

### IPy

Class and tools for handling of IPv4 and IPv6 addresses and networks

*homepage*: <https://pypi.python.org/pypi/IPy>

version |toolchain
--------|----------
``0.83``|``system``

### ipympl

Leveraging the Jupyter interactive widgets framework, ipympl enables the interactive features of matplotlib in the Jupyter notebook and in JupyterLab. Besides, the figure canvas element is a proper Jupyter interactive widget which can be positioned in interactive widget layouts. 

*homepage*: <https://matplotlib.org/ipympl>

version  |toolchain
---------|--------------
``0.9.3``|``foss/2022a``

### ipyparallel

ipyparallel is a Python package and collection  of CLI scripts for controlling clusters for Jupyter

*homepage*: <https://ipyparallel.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``6.2.2``|``-Python-3.6.4``|``foss/2018a``

### ipyrad

ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic  data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.

*homepage*: <https://ipyrad.readthedocs.io>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.6.15``|``-Python-2.7.13``|``intel/2017a``

### IPython

IPython provides a rich architecture for interactive computing with:  Powerful interactive shells (terminal and Qt-based).  A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.  Support for interactive data visualization and use of GUI toolkits.  Flexible, embeddable interpreters to load into your own projects.  Simple to use, high performance tools for parallel computing.

*homepage*: <https://ipython.org/index.html>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------
``3.2.3`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.2.0`` |``-Python-2.7.11``|``intel/2016a``
``5.0.0`` |``-Python-2.7.11``|``foss/2016a``
``5.0.0`` |``-Python-3.5.1`` |``foss/2016a``
``5.1.0`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``5.1.0`` |``-Python-3.5.2`` |``intel/2016b``
``5.2.2`` |``-Python-2.7.12``|``intel/2016b``
``5.3.0`` |``-Python-2.7.13``|``intel/2017a``
``5.7.0`` |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``5.8.0`` |``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.8.0`` |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``fosscuda/2018b``, ``fosscuda/2019a``, ``intel/2018b``
``5.8.0`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.10.0``|``-Python-2.7.18``|``foss/2021b``
``6.2.1`` |``-Python-3.6.4`` |``foss/2017a``
``6.3.1`` |``-Python-3.6.4`` |``intel/2018a``
``6.4.0`` |``-Python-3.6.4`` |``foss/2018a``
``7.2.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``7.7.0`` |``-Python-3.7.2`` |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``7.9.0`` |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``7.13.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``7.15.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``7.18.1``|                  |``GCCcore/10.2.0``
``7.25.0``|                  |``GCCcore/10.3.0``
``7.26.0``|                  |``GCCcore/11.2.0``
``8.5.0`` |                  |``GCCcore/11.3.0``

### IQ-TREE

Efficient phylogenomic software by maximum likelihood

*homepage*: <http://www.iqtree.org/>

version    |versionsuffix|toolchain
-----------|-------------|-----------------------------------------------
``1.5.5``  |``-omp-mpi`` |``foss/2016a``
``1.6.6``  |             |``intel/2018a``
``1.6.12`` |             |``foss/2018b``, ``foss/2020a``, ``intel/2019b``
``2.1.2``  |             |``foss/2020a``, ``gompi/2020b``
``2.1.3``  |             |``gompi/2021a``
``2.2.1``  |             |``gompi/2021b``
``2.2.2.3``|             |``gompi/2022a``
``2.2.2.6``|             |``gompi/2022a``, ``gompi/2022b``

### Iris

A module for improving the insertion sequences of structural variant calls

*homepage*: <https://github.com/mkirsche/Iris>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.5``|``-Java-15`` |``GCC/11.2.0``

### IRkernel

The R kernel for the 'Jupyter' environment executes R code   which the front-end (Jupyter Notebook or other front-ends) submits to the   kernel via the network.

*homepage*: <https://irkernel.github.io>

version   |versionsuffix             |toolchain
----------|--------------------------|----------------------------------
``0.8.15``|``-R-3.4.3-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.1``   |``-R-3.6.2-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``1.1``   |``-R-3.6.3-Python-3.8.2`` |``foss/2020a``
``1.2``   |``-R-4.0.0-Python-3.8.2`` |``foss/2020a``
``1.2``   |``-R-4.1.0``              |``foss/2021a``
``1.3``   |``-R-4.2.0``              |``foss/2021b``
``1.3.2`` |``-R-4.2.1``              |``foss/2022a``

### irodsfs

FUSE implementation of iRODS Client written in Golang.

*homepage*: <https://github.com/cyverse/irodsfs>

version   |versionsuffix   |toolchain
----------|----------------|----------
``0.8.9`` |``-linux-amd64``|``system``
``0.8.11``|``-linux-amd64``|``system``
``0.8.12``|``-linux-amd64``|``system``

### IronPython

IronPython is an open-source implementation of the Python programming language  which is tightly integrated with the .NET Framework. IronPython can use the .NET Framework and Python libraries, and other .NET languages can use Python code just as easily.

*homepage*: <http://ironpython.net/>

version|toolchain
-------|---------------
``2.7``|``intel/2016b``

### ISA-L

Intelligent Storage Acceleration Library

*homepage*: <https://github.com/intel/isa-l>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------
``2.30.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### ISL

isl is a library for manipulating sets and relations of integer points bounded by linear constraints.

*homepage*: <http://isl.gforge.inria.fr/>

version |toolchain
--------|---------------------------------------------------------
``0.14``|``GCC/4.9.2``
``0.15``|``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``
``0.16``|``GCC/4.9.3-2.25``
``0.17``|``foss/2016a``
``0.23``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.24``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### isoCirc

isoCirc: computational pipeline to identify high-confidence BSJs and full-length circRNA isoforms from isoCirc long-read data 

*homepage*: <https://github.com/Xinglab/isoCirc>

version  |toolchain
---------|--------------
``1.0.4``|``foss/2020b``

### IsoNet

IsoNet stands for for ISOtropic reconstructioN of Electron Tomography. It trains deep convolutional neural networks to reconstruct meaningful contents in the mis sing wedge for electron tomography, and to increase signal-to-noise ratio,       using the information learned from the original tomogram. The software requires  tomograms as input. Observing at about 30A resolution, the IsoNet generated      tomograms are largely isotropic.

*homepage*: <https://github.com/Heng-Z/IsoNet>

version                    |toolchain
---------------------------|------------------
``0.1_20210822_04_674f67f``|``fosscuda/2020b``

### IsoSeq

IsoSeq v3 contains the newest tools to identify transcripts                  in PacBio single-molecule sequencing data. Starting in SMRT                   Link v6.0.0, those tools power the IsoSeq GUI-based analysis                   application. A composable workflow of existing tools and                  algorithms, combined with a new clustering technique, allows                  to process the ever-increasing yield of PacBio machines with                  similar performance to IsoSeq versions 1 and 2. Starting with                   version 3.4, support for UMI and cell barcode based                   deduplication has been added.                  

*homepage*: <https://github.com/PacificBiosciences/ioseq3>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``3.8.2``|``-linux-x86_64``|``system``

### ispc

Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for 'single program, multiple data' (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.

*homepage*: <http://ispc.github.io/ , https://github.com/ispc/ispc/>

version   |toolchain
----------|----------
``1.6.0`` |``system``
``1.10.0``|``system``
``1.12.0``|``system``
``1.16.0``|``system``

### itac

The Intel Trace Collector is a low-overhead tracing library that performs  event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application  activities gathered by the Intel Trace Collector through graphical displays. 

*homepage*: <https://software.intel.com/en-us/intel-trace-analyzer/>

version       |toolchain
--------------|----------
``8.0.0.011`` |``system``
``8.1.4.045`` |``system``
``9.0.3.051`` |``system``
``2017.1.024``|``system``
``2018.1.017``|``system``
``2018.3.022``|``system``
``2019.2.026``|``system``
``2019.4.036``|``system``
``2021.2.0``  |``system``
``2021.5.0``  |``system``
``2021.6.0``  |``system``

### ITK

Insight Segmentation and Registration Toolkit (ITK) provides  an extensive suite of software tools for registering and segmenting  multidimensional imaging data.

*homepage*: <https://itk.org>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------------------
``4.12.2``|``-Python-2.7.12``|``foss/2016b``
``4.13.0``|``-Python-2.7.14``|``foss/2018a``
``4.13.0``|``-Python-3.6.4`` |``foss/2018a``
``4.13.1``|``-Python-2.7.14``|``foss/2018a``
``4.13.1``|``-Python-2.7.15``|``foss/2018b``
``4.13.1``|``-Python-3.6.4`` |``foss/2018a``
``4.13.1``|``-Python-3.6.6`` |``foss/2018b``
``4.13.1``|``-Python-3.7.4`` |``foss/2019b``
``5.0.1`` |``-Python-3.7.2`` |``foss/2019a``
``5.0.1`` |``-Python-3.7.4`` |``foss/2019b``
``5.0b01``|``-Python-3.6.6`` |``foss/2018b``
``5.1.2`` |                  |``foss/2020a``, ``fosscuda/2020a``
``5.1.2`` |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``5.2.1`` |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``fosscuda/2020b``

### itpp

IT++ is a C++ library of mathematical, signal processing and communication  classes and functions. Its main use is in simulation of communication systems and for  performing research in the area of communications.

*homepage*: <https://sourceforge.net/projects/itpp/>

version  |toolchain
---------|--------------
``4.3.1``|``foss/2019b``

### ITSTool

ITS Tool allows you to translate your XML documents with PO files

*homepage*: <http://itstool.org/>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``2.0.5``|``-Python-2.7.14``|``intel/2018a``
``2.0.5``|``-Python-2.7.15``|``foss/2018b``
``2.0.6``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``2.0.7``|                  |``GCCcore/11.3.0``

### ITSx

ITSx: Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences of fungi and other eukaryotes for use in environmental sequencing.

*homepage*: <https://microbiology.se/software/itsx/>

version  |toolchain
---------|--------------------------------------
``1.1.2``|``GCCcore/9.3.0``
``1.1.3``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### iVar

iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. 

*homepage*: <https://github.com/andersen-lab/ivar>

version  |toolchain
---------|-----------------------------
``1.0.1``|``foss/2018b``
``1.3.1``|``GCC/10.2.0``, ``GCC/9.3.0``

## J


[JAGS](#jags) - [Jansson](#jansson) - [Jasmine](#jasmine) - [JasPer](#jasper) - [Java](#java) - [JavaFX](#javafx) - [jax](#jax) - [JAXFrontCE](#jaxfrontce) - [jbigkit](#jbigkit) - [Jellyfish](#jellyfish) - [jemalloc](#jemalloc) - [jhbuild](#jhbuild) - [JiTCODE](#jitcode) - [jModelTest](#jmodeltest) - [Jorg](#jorg) - [joypy](#joypy) - [jq](#jq) - [json-c](#json-c) - [json-fortran](#json-fortran) - [JSON-GLib](#json-glib) - [JsonCpp](#jsoncpp) - [JUBE](#jube) - [Judy](#judy) - [Julia](#julia) - [JUnit](#junit) - [jupyter-contrib-nbextensions](#jupyter-contrib-nbextensions) - [jupyter-matlab-proxy](#jupyter-matlab-proxy) - [jupyter-resource-usage](#jupyter-resource-usage) - [jupyter-server](#jupyter-server) - [jupyter-server-proxy](#jupyter-server-proxy) - [JupyterHub](#jupyterhub) - [JupyterLab](#jupyterlab) - [jupyterlab-lmod](#jupyterlab-lmod) - [jupyterlmod](#jupyterlmod) - [JWM](#jwm) - [jxrlib](#jxrlib)


### JAGS

JAGS is Just Another Gibbs Sampler.  It is a program for analysis  of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  

*homepage*: <http://mcmc-jags.sourceforge.net/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.2.0``|``foss/2016a``, ``intel/2016a``, ``intel/2017a``
``4.3.0``|``foss/2017b``, ``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2017b``
``4.3.1``|``foss/2022a``
``4.3.2``|``foss/2022b``

### Jansson

Jansson is a C library for encoding, decoding and manipulating JSON data.  Its main features and design principles are:  * Simple and intuitive API and data model  * Comprehensive documentation  * No dependencies on other libraries  * Full Unicode support (UTF-8)  * Extensive test suite

*homepage*: <http://www.digip.org/jansson/>

version   |toolchain
----------|------------------------------
``2.6``   |``GCC/4.8.3``
``2.13.1``|``GCC/10.2.0``, ``GCC/11.2.0``
``2.14``  |``GCC/11.3.0``

### Jasmine

SV Merging Across Samples

*homepage*: <https://github.com/mkirsche/Jasmine>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.1.4``|``-Java-15`` |``GCC/11.2.0``

### JasPer

The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in  the JPEG-2000 Part-1 standard. 

*homepage*: <https://www.ece.uvic.ca/~frodo/jasper/>

version    |toolchain
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.900.1``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``2.0.10`` |``intel/2016b``
``2.0.12`` |``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``
``2.0.14`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.16`` |``GCCcore/9.3.0``
``2.0.24`` |``GCCcore/10.2.0``
``2.0.28`` |``GCCcore/10.3.0``
``2.0.33`` |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.0.0``  |``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### Java

Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers.

*homepage*: <https://java.com/>

version      |versionsuffix           |toolchain
-------------|------------------------|----------
``1.6.0_24`` |                        |``system``
``1.7.0_10`` |                        |``system``
``1.7.0_15`` |                        |``system``
``1.7.0_21`` |                        |``system``
``1.7.0_40`` |                        |``system``
``1.7.0_45`` |                        |``system``
``1.7.0_60`` |                        |``system``
``1.7.0_75`` |                        |``system``
``1.7.0_76`` |                        |``system``
``1.7.0_79`` |                        |``system``
``1.7.0_80`` |                        |``system``
``1.8``      |                        |``system``
``1.8.0_20`` |                        |``system``
``1.8.0_25`` |                        |``system``
``1.8.0_31`` |                        |``system``
``1.8.0_40`` |                        |``system``
``1.8.0_45`` |                        |``system``
``1.8.0_60`` |                        |``system``
``1.8.0_65`` |                        |``system``
``1.8.0_66`` |                        |``system``
``1.8.0_72`` |                        |``system``
``1.8.0_74`` |                        |``system``
``1.8.0_77`` |                        |``system``
``1.8.0_92`` |                        |``system``
``1.8.0_112``|                        |``system``
``1.8.0_121``|                        |``system``
``1.8.0_131``|                        |``system``
``1.8.0_141``|                        |``system``
``1.8.0_144``|                        |``system``
``1.8.0_152``|                        |``system``
``1.8.0_162``|                        |``system``
``1.8.0_172``|                        |``system``
``1.8.0_181``|                        |``system``
``1.8.0_192``|                        |``system``
``1.8.0_202``|                        |``system``
``1.8.0_212``|                        |``system``
``1.8.0_221``|                        |``system``
``1.8.0_231``|                        |``system``
``1.8.0_241``|                        |``system``
``1.8.0_271``|                        |``system``
``1.8.0_281``|                        |``system``
``1.8.0_292``|``-OpenJDK``            |``system``
``1.8.0_311``|                        |``system``
``1.8_191``  |``-b26-OpenJDK``        |``system``
``1.8_265``  |``-b01-OpenJDK-aarch64``|``system``
``1.9.0.4``  |                        |``system``
``8``        |                        |``system``
``8.345``    |                        |``system``
``8.362``    |                        |``system``
``11``       |                        |``system``
``11.0.2``   |                        |``system``
``11.0.6``   |``-ppc64le``            |``system``
``11.0.8``   |``-aarch64``            |``system``
``11.0.16``  |                        |``system``
``11.0.18``  |                        |``system``
``13``       |                        |``system``
``13.0.2``   |                        |``system``
``15``       |                        |``system``
``15.0.1``   |                        |``system``
``16``       |                        |``system``
``16.0.1``   |                        |``system``
``17``       |                        |``system``
``17.0.1``   |                        |``system``
``17.0.2``   |                        |``system``
``17.0.4``   |                        |``system``
``17.0.6``   |                        |``system``

### JavaFX

OpenJFX is an open source, next generation client application platform for desktop,  mobile and embedded systems built on Java 

*homepage*: <https://openjfx.io/>

version   |versionsuffix         |toolchain
----------|----------------------|----------
``11.0.2``|``_linux-x64_bin-sdk``|``system``

### jax

Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more

*homepage*: <https://pypi.python.org/pypi/jax>

version   |versionsuffix   |toolchain
----------|----------------|----------------------------------
``0.2.19``|                |``foss/2020b``, ``fosscuda/2020b``
``0.2.20``|                |``foss/2021a``
``0.2.24``|                |``foss/2021a``
``0.2.24``|``-CUDA-11.3.1``|``foss/2021a``
``0.3.9`` |                |``foss/2021a``
``0.3.9`` |``-CUDA-11.3.1``|``foss/2021a``
``0.3.14``|``-CUDA-11.7.0``|``foss/2022a``
``0.3.23``|                |``foss/2022a``
``0.3.23``|``-CUDA-11.4.1``|``foss/2021b``
``0.3.25``|                |``foss/2022a``
``0.3.25``|``-CUDA-11.7.0``|``foss/2022a``
``0.4.4`` |                |``foss/2022a``
``0.4.4`` |``-CUDA-11.7.0``|``foss/2022a``

### JAXFrontCE

JAXFront is a technology to generate graphical user interfaces  on multiple channels (Java Swing, HTML, PDF) on the basis of an XML schema.

*homepage*: <http://www.jaxfront.org/pages/free_community_edition.html>

version |toolchain
--------|----------
``2.75``|``system``

### jbigkit

JBIG-KIT is a software implementation of the JBIG1 data  compression standard (ITU-T T.82), which was designed for bi-level image  data, such as scanned documents.

*homepage*: <https://www.cl.cam.ac.uk/~mgk25/jbigkit/>

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### Jellyfish

Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.

*homepage*: <http://www.genome.umd.edu/jellyfish.html>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------
``1.1.11``|``foss/2016a``, ``foss/2016b``
``1.1.12``|``foss/2018b``, ``intel/2018a``
``2.2.6`` |``foss/2016b``, ``intel/2017a``
``2.2.10``|``foss/2018b``, ``intel/2018a``
``2.3.0`` |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``

### jemalloc

jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and  scalable concurrency support.

*homepage*: <http://jemalloc.net>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``4.1.0``|``intel/2016a``
``4.2.0``|``foss/2016a``, ``intel/2016a``
``4.2.1``|``intel/2016b``
``4.5.0``|``intel/2017a``
``5.0.1``|``GCCcore/6.4.0``
``5.1.0``|``GCCcore/7.3.0``
``5.2.0``|``GCCcore/8.2.0``
``5.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.3.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### jhbuild

JHBuild allows you to automatically download and compile “modules” (i.e. source code packages).  Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover  what modules need to be built and in what order.

*homepage*: <https://wiki.gnome.org/action/show/Projects/Jhbuild>

version    |toolchain
-----------|-----------------
``3.15.92``|``GCCcore/4.9.3``

### JiTCODE

Just-in-time compilation for ordinary/delay/stochastic differential equations (DDEs)

*homepage*: <https://jitcde-common.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.3.2``|``-Python-3.6.4``|``intel/2018a``
``1.4.0``|``-Python-3.7.2``|``foss/2019a``

### jModelTest

jModelTest is a tool to carry out statistical selection of best-fit models of nucleotide substitution.

*homepage*: <https://github.com/ddarriba/jmodeltest2>

version            |versionsuffix     |toolchain
-------------------|------------------|----------
``2.1.10r20160303``|``-Java-1.8.0_92``|``system``

### Jorg

A MAG Circularization Method By Lauren Lui, Torben Nielsen, and Adam Arkin

*homepage*: <https://github.com/lmlui/Jorg>

version  |toolchain
---------|--------------
``1.0.1``|``foss/2020b``

### joypy

Joyplots in Python with matplotlib & pandas

*homepage*: <https://github.com/sbebo/joypy>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.2``|``-Python-3.7.4``|``intel/2019b``
``0.2.4``|                 |``intel/2020b``

### jq

jq is a lightweight and flexible command-line JSON processor.

*homepage*: <https://stedolan.github.io/jq/>

version|toolchain
-------|----------------------------------------------------------
``1.5``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``
``1.6``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### json-c

JSON-C implements a reference counting object model that allows you to easily construct JSON objects  in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON objects.

*homepage*: <https://github.com/json-c/json-c>

version |toolchain
--------|----------------------------------------------------------
``0.15``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.16``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### json-fortran

JSON-Fortran: A Modern Fortran JSON API

*homepage*: <https://github.com/jacobwilliams/json-fortran>

version  |toolchain
---------|------------------
``8.3.0``|``GCCcore/11.2.0``

### JSON-GLib

JSON-GLib implements a full JSON parser and generator using GLib and GObject,  and integrates JSON with GLib data types.

*homepage*: <https://wiki.gnome.org/Projects/JsonGlib>

version  |toolchain
---------|------------------
``1.6.2``|``GCCcore/10.3.0``

### JsonCpp

JsonCpp is a C++ library that allows manipulating JSON values,  including serialization and deserialization to and from strings. It can also preserve existing comment in  unserialization/serialization steps, making it a convenient format to store user input files. 

*homepage*: <https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html>

version   |toolchain
----------|-----------------------------------------------------------------------------
``0.10.7``|``GCCcore/8.2.0``
``1.9.3`` |``GCCcore/8.3.0``
``1.9.4`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``1.9.5`` |``GCCcore/11.3.0``

### JUBE

The JUBE benchmarking environment provides a script based framework to easily create benchmark sets, run those sets on different computer systems and evaluate the results. 

*homepage*: <https://www.fz-juelich.de/jsc/jube>

version  |toolchain
---------|----------
``2.0.3``|``system``
``2.0.4``|``system``
``2.0.5``|``system``
``2.4.0``|``system``
``2.4.1``|``system``
``2.4.2``|``system``

### Judy

A C library that implements a dynamic array.

*homepage*: <http://judy.sourceforge.net/>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------------
``1.0.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

### Julia

Julia is a high-level, high-performance dynamic programming language for numerical computing

*homepage*: <https://julialang.org>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``1.1.1``|``-linux-x86_64``|``system``
``1.2.0``|``-linux-x86_64``|``system``
``1.3.1``|``-linux-x86_64``|``system``
``1.4.0``|``-linux-x86_64``|``system``
``1.4.1``|``-linux-x86_64``|``system``
``1.4.2``|``-linux-x86_64``|``system``
``1.5.1``|``-linux-x86_64``|``system``
``1.5.3``|``-linux-x86_64``|``system``
``1.6.1``|``-linux-x86_64``|``system``
``1.6.2``|``-linux-x86_64``|``system``
``1.6.4``|``-linux-x86_64``|``system``
``1.6.5``|``-linux-x86_64``|``system``
``1.6.6``|``-linux-x86_64``|``system``
``1.6.7``|``-linux-x86_64``|``system``
``1.7.0``|``-linux-x86_64``|``system``
``1.7.1``|``-linux-x86_64``|``system``
``1.7.2``|``-linux-x86_64``|``system``
``1.7.3``|``-linux-x86_64``|``system``
``1.8.0``|``-linux-x86_64``|``system``
``1.8.2``|``-linux-x86_64``|``system``
``1.8.5``|``-linux-x86_64``|``system``
``1.9.0``|``-linux-x86_64``|``system``

### JUnit

A programmer-oriented testing framework for Java.

*homepage*: <http://sourceforge.net/projects/junit>

version |versionsuffix      |toolchain
--------|-------------------|----------
``4.10``|``-Java-1.7.0_10`` |``system``
``4.10``|``-Java-1.7.0_21`` |``system``
``4.11``|``-Java-1.7.0_15`` |``system``
``4.11``|``-Java-1.7.0_21`` |``system``
``4.11``|``-Java-1.7.0_60`` |``system``
``4.11``|``-Java-1.7.0_75`` |``system``
``4.11``|``-Java-1.7.0_79`` |``system``
``4.12``|``-Java-1.7.0_80`` |``system``
``4.12``|``-Java-1.8``      |``system``
``4.12``|``-Java-1.8.0_112``|``system``
``4.12``|``-Java-1.8.0_121``|``system``
``4.12``|``-Java-1.8.0_144``|``system``
``4.12``|``-Java-1.8.0_152``|``system``
``4.12``|``-Java-1.8.0_162``|``system``
``4.12``|``-Java-1.8.0_66`` |``system``
``4.12``|``-Java-1.8.0_72`` |``system``
``4.12``|``-Java-1.8.0_77`` |``system``
``4.12``|``-Java-1.8.0_92`` |``system``

### jupyter-contrib-nbextensions

A collection of various notebook extensions for Jupyter

*homepage*: <https://github.com/ipython-contrib/jupyter_contrib_nbextensions>

version  |toolchain
---------|------------------
``0.7.0``|``GCCcore/11.3.0``

### jupyter-matlab-proxy

MATLAB Integration for Jupyter

*homepage*: <https://github.com/mathworks/jupyter-matlab-proxy>

version  |toolchain
---------|------------------
``0.3.4``|``GCCcore/10.3.0``
``0.5.0``|``GCCcore/11.3.0``

### jupyter-resource-usage

Jupyter Notebook Extension for monitoring your own Resource Usage (memory and/or CPU)

*homepage*: <https://github.com/jupyter-server/jupyter-resource-usage>

version  |toolchain
---------|------------------
``0.6.1``|``GCCcore/10.3.0``
``0.6.2``|``GCCcore/10.3.0``
``0.6.3``|``GCCcore/11.3.0``

### jupyter-server

The Jupyter Server provides the backend (i.e. the core services, APIs, and REST endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and Voila.

*homepage*: <https://jupyter.org/>

version   |toolchain
----------|------------------
``1.21.0``|``GCCcore/11.3.0``

### jupyter-server-proxy

Jupyter Server Proxy lets you run arbitrary external processes (such as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc) alongside your notebook server and provide authenticated web access to them     using a path like /rstudio next to others like /lab. Alongside the python package that provides the main functionality, the JupyterLab extension (@jupyterlab/server-proxy) provides buttons in the JupyterLab launcher window to get to RStudio for example.

*homepage*: <https://github.com/jupyterhub/jupyter-server-proxy>

version  |toolchain
---------|------------------
``3.2.1``|``GCCcore/10.3.0``
``3.2.2``|``GCCcore/11.3.0``

### JupyterHub

JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed  for centralized deployments in companies, university classrooms and research labs.

*homepage*: <https://jupyter.org>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.6.1``|``-Python-3.5.1``|``foss/2016a``
``0.8.1``|``-Python-3.6.4``|``foss/2017a``
``1.1.0``|                 |``GCCcore/10.2.0``
``1.4.1``|                 |``GCCcore/10.3.0``
``3.0.0``|                 |``GCCcore/11.3.0``

### JupyterLab

JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar  building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,  etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter  Notebook.

*homepage*: <https://jupyter.org/>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``1.2.5`` |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.2.8`` |                 |``GCCcore/10.2.0``
``3.0.16``|                 |``GCCcore/10.3.0``
``3.1.6`` |                 |``GCCcore/11.2.0``
``3.2.8`` |                 |``GCCcore/10.3.0``
``3.5.0`` |                 |``GCCcore/11.3.0``

### jupyterlab-lmod

JupyterLab extension that allows user to interact with environment modules before launching kernels. The extension use Lmod's Python interface to accomplish module related task like loading, unloading, saving collection, etc.

*homepage*: <https://github.com/cmd-ntrf/jupyter-lmod>

version  |toolchain
---------|------------------
``1.0.2``|``GCCcore/11.3.0``

### jupyterlmod

Jupyter interactive notebook server extension that allows users to interact with environment modules before launching kernels. The extension uses Lmod's Python interface to accomplish module-related tasks like loading, unloading, saving collections, etc.

*homepage*: <https://github.com/cmd-ntrf/jupyter-lmod>

version  |toolchain
---------|------------------
``3.0.0``|``GCCcore/11.3.0``
``4.0.3``|``GCCcore/11.3.0``

### JWM

JWM is a light-weight window manager for the X11 Window System.

*homepage*: <https://joewing.net/projects/jwm/>

version  |toolchain
---------|---------------
``2.3.5``|``intel/2016a``

### jxrlib

Open source implementation of jpegxr

*homepage*: <https://github.com/4creators/jxrlib>

version|toolchain
-------|----------------------------------------------------------
``1.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``

## K


[KaHIP](#kahip) - [Kaiju](#kaiju) - [Kaleido](#kaleido) - [Kalign](#kalign) - [kallisto](#kallisto) - [KAT](#kat) - [kb-python](#kb-python) - [kbproto](#kbproto) - [kedro](#kedro) - [Kent_tools](#kent_tools) - [Keras](#keras) - [KerasTuner](#kerastuner) - [khmer](#khmer) - [kim-api](#kim-api) - [kineto](#kineto) - [king](#king) - [KITE](#kite) - [kma](#kma) - [KMC](#kmc) - [KMCP](#kmcp) - [KmerGenie](#kmergenie) - [KNIME](#knime) - [kpcalg](#kpcalg) - [Kraken](#kraken) - [Kraken2](#kraken2) - [KrakenUniq](#krakenuniq) - [Kratos](#kratos) - [krbalancing](#krbalancing) - [KronaTools](#kronatools) - [kwant](#kwant) - [KWIML](#kwiml) - [kWIP](#kwip) - [KyotoCabinet](#kyotocabinet)


### KaHIP

The graph partitioning framework KaHIP -- Karlsruhe High Quality Partitioning.

*homepage*: <https://kahip.github.io/>

version |toolchain
--------|--------------------------------
``3.14``|``gompi/2022a``, ``gompi/2022b``

### Kaiju

Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments

*homepage*: <http://kaiju.binf.ku.dk/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.5.0``|                 |``intel/2016b``
``1.7.2``|``-Python-3.7.2``|``iimpi/2019a``
``1.7.3``|``-Python-3.7.4``|``gompi/2019b``

### Kaleido

Fast static image export for web-based visualization libraries with zero dependencies

*homepage*: <https://github.com/plotly/Kaleido>

version  |toolchain
---------|--------------------------------------
``0.1.0``|``GCCcore/10.2.0``
``0.2.1``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### Kalign

Kalign is a fast multiple sequence alignment program for biological sequences.

*homepage*: <https://github.com/TimoLassmann/kalign>

version  |toolchain
---------|--------------------------------------
``2.0.4``|``GCCcore/10.2.0``
``3.3.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.3.2``|``GCCcore/11.2.0``
``3.3.5``|``GCCcore/11.3.0``

### kallisto

kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally  of target sequences using high-throughput sequencing reads.

*homepage*: <https://pachterlab.github.io/kallisto/>

version   |toolchain
----------|------------------------------------------------
``0.42.5``|``foss/2016a``
``0.43.0``|``intel/2016b``
``0.43.1``|``foss/2016b``, ``intel/2017a``, ``intel/2017b``
``0.44.0``|``foss/2016b``, ``intel/2018a``
``0.45.0``|``foss/2018b``
``0.45.1``|``foss/2019a``
``0.46.0``|``intel/2019a``
``0.46.1``|``foss/2019b``, ``iimpi/2020a``, ``iimpi/2020b``
``0.46.2``|``foss/2020b``
``0.48.0``|``gompi/2021b``, ``gompi/2022a``

### KAT

The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.

*homepage*: <https://www.earlham.ac.uk/kat-tools>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.4.2``|``-Python-3.6.4``|``foss/2018a``
``2.4.2``|``-Python-3.7.2``|``foss/2019a``

### kb-python

kallisto | bustools is a workflow for pre-processing single-cell RNA-seq data.  Pre-processing single-cell RNA-seq involves: (1) association of reads with their cells of origin,  (2) collapsing of reads according to unique molecular identifiers (UMIs), and (3) generation of  gene or feature counts from the reads to generate a cell x gene matrix.

*homepage*: <https://www.kallistobus.tools/>

version   |toolchain
----------|--------------
``0.27.3``|``foss/2021b``

### kbproto

X.org KBProto protocol headers.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|------------------------------------------------------------------
``1.0.7``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

### kedro

Kedro is an open-source Python framework that applies software engineering best-practice to data and machine-learning pipelines. 

*homepage*: <https://github.com/quantumblacklabs/kedro>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.16.5``|``-Python-3.8.2``|``foss/2020a``

### Kent_tools

Kent utilities: collection of tools used by the UCSC genome browser.

*homepage*: <https://genome.cse.ucsc.edu/>

version     |versionsuffix    |toolchain
------------|-----------------|---------------
``401``     |                 |``gompi/2019b``
``411``     |                 |``GCC/10.2.0``
``418``     |                 |``GCC/10.3.0``
``422``     |                 |``GCC/11.2.0``
``442``     |                 |``GCC/11.3.0``
``20130806``|``-linux.x86_64``|``system``
``20171107``|``-linux.x86_64``|``system``
``20180716``|``-linux.x86_64``|``system``
``20190326``|``-linux.x86_64``|``system``

### Keras

Keras is a deep learning API written in Python, running on top of the machine learning platform TensorFlow. 

*homepage*: <https://keras.io/>

version  |versionsuffix        |toolchain
---------|---------------------|----------------------------------
``1.0.8``|``-Python-3.5.2``    |``intel/2016b``
``1.1.0``|``-Python-3.5.2``    |``intel/2016b``
``2.0.4``|``-Python-2.7.13``   |``intel/2017a``
``2.0.4``|``-Python-3.6.1``    |``intel/2017a``
``2.0.5``|``-Python-3.6.1``    |``intel/2017a``
``2.0.8``|``-Python-3.6.1``    |``intel/2017a``
``2.1.1``|``-Python-2.7.14``   |``intel/2017b``
``2.1.1``|``-Python-3.6.3``    |``intel/2017b``
``2.1.2``|``-Python-2.7.14``   |``intel/2017b``
``2.1.2``|``-Python-3.6.3``    |``foss/2017b``
``2.1.3``|``-Python-3.6.3``    |``foss/2017b``, ``intel/2017b``
``2.2.0``|``-Python-2.7.14``   |``fosscuda/2017b``
``2.2.0``|``-Python-3.6.3``    |``fosscuda/2017b``
``2.2.0``|``-Python-3.6.4``    |``foss/2018a``
``2.2.2``|``-Python-2.7.15``   |``fosscuda/2018b``
``2.2.4``|``-Python-3.6.4``    |``intel/2018a``
``2.2.4``|``-Python-3.6.6``    |``foss/2018b``, ``fosscuda/2018b``
``2.2.4``|``-Python-3.7.2``    |``foss/2019a``, ``fosscuda/2019a``
``2.3.1``|``-Python-3.7.4``    |``foss/2019b``, ``fosscuda/2019b``
``2.3.1``|``-Python-3.8.2``    |``foss/2020a``
``2.4.3``|                     |``foss/2020b``, ``fosscuda/2020b``
``2.4.3``|``-TensorFlow-2.5.0``|``fosscuda/2020b``

### KerasTuner

KerasTuner is an simple-to-use, scalable hyperparameter optimization framework that solves the pain points of hyperparameter search.

*homepage*: <https://keras.io/keras_tuner>

version  |toolchain
---------|--------------
``1.3.5``|``foss/2022a``

### khmer

In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more 

*homepage*: <https://github.com/ged-lab/khmer/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.12``|``foss/2016b``
``2.1.1``|``-Python-2.7.13``|``intel/2017a``

### kim-api

Open Knowledgebase of Interatomic Models.  KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations.  This is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This SimpleBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: <https://openkim.org/>

version  |toolchain
---------|----------------------------------------------------------------
``2.1.2``|``foss/2019a``, ``intel/2019a``
``2.1.3``|``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``2.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.3.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### kineto

A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters

*homepage*: <https://github.com/pytorch/kineto>

version  |toolchain
---------|--------------
``0.4.0``|``GCC/11.3.0``

### king

KING is a toolset that makes use of high-throughput  SNP data typically seen in a genome-wide association study (GWAS) or  a sequencing project. Applications of KING include family relationship  inference and pedigree error checking, quality control, population  substructure identification, forensics, gene mapping, etc.

*homepage*: <https://kingrelatedness.com/>

version  |toolchain
---------|----------
``2.2.4``|``system``
``2.2.7``|``system``

### KITE

KITE is an open-source Python/C++ software suite for efficient real-space tight-binding (TB)  simulations of electronic structure and bulk quantum transport properties of disordered systems scalable to  multi billions of atomic orbitals.

*homepage*: <https://github.com/quantum-kite/kite>

version|toolchain
-------|---------------
``1.1``|``gompi/2022a``

### kma

KMA is a mapping method designed to map raw reads directly against redundant databases,  in an ultra-fast manner using seed and extend.

*homepage*: <https://bitbucket.org/genomicepidemiology/kma>

version   |toolchain
----------|---------------
``1.2.22``|``intel/2019b``

### KMC

KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.

*homepage*: <http://sun.aei.polsl.pl/kmc>

version     |versionsuffix    |toolchain
------------|-----------------|------------------------------
``3.1.0``   |                 |``foss/2018a``, ``foss/2018b``
``3.1.1``   |``-Python-3.7.2``|``GCC/8.2.0-2.31.1``
``3.1.2rc1``|``-Python-3.7.4``|``GCC/8.3.0``
``3.2.1``   |                 |``GCC/11.2.0``

### KMCP

KMCP: accurate metagenomic profiling of both prokaryotic and viral populations by pseudo-mapping 

*homepage*: <https://bioinf.shenwei.me/kmcp>

version  |toolchain
---------|----------
``0.9.1``|``system``

### KmerGenie

KmerGenie estimates the best k-mer length for genome de novo assembly.

*homepage*: <http://kmergenie.bx.psu.edu/>

version   |toolchain
----------|---------------
``1.7044``|``intel/2017a``
``1.7048``|``intel/2018a``

### KNIME

KNIME Analytics Platform is the open source software for creating data science applications and services. KNIME stands for KoNstanz Information MinEr.  

*homepage*: <https://www.knime.com/>

version  |toolchain
---------|----------
``3.6.2``|``system``

### kpcalg

Kernel PC (kPC) algorithm for causal structure learning and  causal inference using graphical models. kPC is a version of PC algorithm that  uses kernel based independence criteria in order to be able to deal with  non-linear relationships and non-Gaussian noise.  Includes pcalg: Functions for causal structure learning and causal inference  using graphical models.

*homepage*: <https://cran.r-project.org/package=kpcalg>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-3.5.1`` |``foss/2018b``

### Kraken

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: <https://ccb.jhu.edu/software/kraken/>

version        |versionsuffix   |toolchain
---------------|----------------|-----------------------------------------------------------------------------
``0.10.5-beta``|``-Perl-5.22.1``|``foss/2016a``
``0.10.5-beta``|``-Perl-5.24.0``|``foss/2016b``
``1.0``        |``-Perl-5.26.1``|``intel/2018a``
``1.1``        |``-Perl-5.28.0``|``foss/2018b``
``1.1.1``      |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``
``1.1.1``      |``-Perl-5.28.1``|``GCCcore/8.2.0``

### Kraken2

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: <https://github.com/DerrickWood/kraken2/wiki>

version       |versionsuffix   |toolchain
--------------|----------------|-------------------------------------------------
``2.0.6-beta``|``-Perl-5.26.1``|``foss/2018a``
``2.0.7-beta``|``-Perl-5.28.0``|``foss/2018b``
``2.0.8-beta``|``-Perl-5.30.0``|``gompi/2019b``
``2.0.9-beta``|``-Perl-5.28.0``|``foss/2018b``
``2.0.9-beta``|``-Perl-5.30.2``|``gompi/2020a``
``2.1.1``     |                |``gompi/2020b``
``2.1.2``     |                |``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``

### KrakenUniq

KrakenUniq: confident and fast metagenomics classification using unique k-mer counts

*homepage*: <https://github.com/fbreitwieser/krakenuniq/>

version  |toolchain
---------|--------------
``1.0.3``|``GCC/11.3.0``

### Kratos

Kratos Multiphysics (A.K.A Kratos) is a framework for building parallel multi-disciplinary  simulation software.

*homepage*: <https://www.cimne.com/kratos>

version|versionsuffix    |toolchain
-------|-----------------|-------------------------------
``6.0``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``

### krbalancing

A C++ extension for Python which computes K.R. balanced matrices.

*homepage*: <https://github.com/deeptools/Knight-Ruiz-Matrix-balancing-algorithm>

version    |toolchain
-----------|------------------
``0.5.0b0``|``GCCcore/11.3.0``

### KronaTools

Krona Tools is a set of scripts to create Krona charts from  several Bioinformatics tools as well as from text and XML files.

*homepage*: <https://github.com/marbl/Krona/wiki/KronaTools>

version  |toolchain
---------|--------------------------------------
``2.7``  |``GCCcore/7.3.0``
``2.7.1``|``GCCcore/8.2.0``
``2.8``  |``GCC/10.3.0``, ``GCCcore/10.2.0``
``2.8.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### kwant

Kwant is a free (open source), powerful, and simple to use Python package for   numerical calculations on tight-binding models with a strong focus on quantum transport.

*homepage*: <https://kwant-project.org/>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.4.1``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``

### KWIML

The Kitware Information Macro Library

*homepage*: <https://gitlab.kitware.com/utils/kwiml>

version     |toolchain
------------|-----------------
``20180201``|``GCCcore/6.4.0``

### kWIP

This software implements a de novo, alignment free measure of sample  genetic dissimilarity which operates upon raw sequencing reads.  It is able to calculate the genetic dissimilarity between samples without any reference genome,  and without assembling one.

*homepage*: <https://github.com/kdmurray91/kWIP>

version  |toolchain
---------|-----------------
``0.2.0``|``GCCcore/6.4.0``

### KyotoCabinet

Kyoto Cabinet is a library of routines for managing a database.

*homepage*: <https://fallabs.com/kyotocabinet>

version   |toolchain
----------|------------------------------------
``1.2.77``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

## L


[L_RNA_scaffolder](#l_rna_scaffolder) - [Lace](#lace) - [LADR](#ladr) - [lagrangian-filtering](#lagrangian-filtering) - [LAME](#lame) - [LAMMPS](#lammps) - [lancet](#lancet) - [LAPACK](#lapack) - [LASSO-Python](#lasso-python) - [LAST](#last) - [LASTZ](#lastz) - [lavaan](#lavaan) - [LayoutParser](#layoutparser) - [LBFGS++](#lbfgs++) - [LCov](#lcov) - [LDC](#ldc) - [lDDT](#lddt) - [LeadIT](#leadit) - [leidenalg](#leidenalg) - [LEMON](#lemon) - [Leptonica](#leptonica) - [LERC](#lerc) - [less](#less) - [LevelDB](#leveldb) - [lftp](#lftp) - [LHAPDF](#lhapdf) - [LIANA](#liana) - [libaec](#libaec) - [libaio](#libaio) - [libarchive](#libarchive) - [libav](#libav) - [libavif](#libavif) - [libBigWig](#libbigwig) - [libbitmask](#libbitmask) - [libcdms](#libcdms) - [libcerf](#libcerf) - [libcint](#libcint) - [libcircle](#libcircle) - [libcmaes](#libcmaes) - [libconfig](#libconfig) - [libcpuset](#libcpuset) - [libcroco](#libcroco) - [libctl](#libctl) - [libdap](#libdap) - [libde265](#libde265) - [libdeflate](#libdeflate) - [libdivsufsort](#libdivsufsort) - [libdrm](#libdrm) - [libdrs](#libdrs) - [libdwarf](#libdwarf) - [libedit](#libedit) - [libelf](#libelf) - [libemf](#libemf) - [libepoxy](#libepoxy) - [libev](#libev) - [libevent](#libevent) - [libexif](#libexif) - [libfabric](#libfabric) - [libfdf](#libfdf) - [libffcall](#libffcall) - [libffi](#libffi) - [libFLAME](#libflame) - [libfontenc](#libfontenc) - [libgcrypt](#libgcrypt) - [libgd](#libgd) - [libgdiplus](#libgdiplus) - [libGDSII](#libgdsii) - [libgeotiff](#libgeotiff) - [libgit2](#libgit2) - [libglade](#libglade) - [libGLU](#libglu) - [libglvnd](#libglvnd) - [libgpg-error](#libgpg-error) - [libgpuarray](#libgpuarray) - [libGridXC](#libgridxc) - [libgtextutils](#libgtextutils) - [libharu](#libharu) - [libheif](#libheif) - [libibmad](#libibmad) - [libibumad](#libibumad) - [libICE](#libice) - [libiconv](#libiconv) - [libidn](#libidn) - [libidn2](#libidn2) - [Libint](#libint) - [LiBis](#libis) - [libjpeg-turbo](#libjpeg-turbo) - [libjxl](#libjxl) - [LibLZF](#liblzf) - [libmad](#libmad) - [libmatheval](#libmatheval) - [libmaus2](#libmaus2) - [libmbd](#libmbd) - [libMemcached](#libmemcached) - [libmicrohttpd](#libmicrohttpd) - [libmo_unpack](#libmo_unpack) - [libmypaint](#libmypaint) - [libnsl](#libnsl) - [libobjcryst](#libobjcryst) - [libogg](#libogg) - [libopus](#libopus) - [libosmium](#libosmium) - [libpci](#libpci) - [libpciaccess](#libpciaccess) - [libplinkio](#libplinkio) - [libpng](#libpng) - [libpsl](#libpsl) - [libPSML](#libpsml) - [libpsortb](#libpsortb) - [libpspio](#libpspio) - [libpthread-stubs](#libpthread-stubs) - [libQGLViewer](#libqglviewer) - [libreadline](#libreadline) - [libRmath](#librmath) - [librosa](#librosa) - [librsb](#librsb) - [librsvg](#librsvg) - [librttopo](#librttopo) - [libsamplerate](#libsamplerate) - [libSBML](#libsbml) - [libsigc++](#libsigc++) - [libsigsegv](#libsigsegv) - [libSM](#libsm) - [libsndfile](#libsndfile) - [libsodium](#libsodium) - [LibSoup](#libsoup) - [libspatialindex](#libspatialindex) - [libspatialite](#libspatialite) - [libssh](#libssh) - [libStatGen](#libstatgen) - [LIBSVM](#libsvm) - [libtar](#libtar) - [libtasn1](#libtasn1) - [libtecla](#libtecla) - [LibTIFF](#libtiff) - [libtirpc](#libtirpc) - [libtool](#libtool) - [libtree](#libtree) - [libunistring](#libunistring) - [libunwind](#libunwind) - [libutempter](#libutempter) - [LibUUID](#libuuid) - [libuv](#libuv) - [libvdwxc](#libvdwxc) - [libvorbis](#libvorbis) - [libWallModelledLES](#libwallmodelledles) - [libwebp](#libwebp) - [libwpe](#libwpe) - [libX11](#libx11) - [libXau](#libxau) - [libxc](#libxc) - [libxcb](#libxcb) - [libXcursor](#libxcursor) - [libXdamage](#libxdamage) - [libXdmcp](#libxdmcp) - [libXext](#libxext) - [libXfixes](#libxfixes) - [libXfont](#libxfont) - [libXft](#libxft) - [libXi](#libxi) - [libXinerama](#libxinerama) - [libxkbcommon](#libxkbcommon) - [libxml++](#libxml++) - [libxml2](#libxml2) - [libxml2-python](#libxml2-python) - [libXmu](#libxmu) - [libXp](#libxp) - [libXpm](#libxpm) - [libXrandr](#libxrandr) - [libXrender](#libxrender) - [libxslt](#libxslt) - [libxsmm](#libxsmm) - [libXt](#libxt) - [libXxf86vm](#libxxf86vm) - [libyaml](#libyaml) - [libzeep](#libzeep) - [libzip](#libzip) - [lie_learn](#lie_learn) - [lifelines](#lifelines) - [Lighter](#lighter) - [liknorm](#liknorm) - [likwid](#likwid) - [limix](#limix) - [LinBox](#linbox) - [line_profiler](#line_profiler) - [Lingeling](#lingeling) - [LISFLOOD-FP](#lisflood-fp) - [LittleCMS](#littlecms) - [LLDB](#lldb) - [LLVM](#llvm) - [LMDB](#lmdb) - [LMfit](#lmfit) - [Lmod](#lmod) - [LncLOOM](#lncloom) - [LocARNA](#locarna) - [LoFreq](#lofreq) - [Log-Log4perl](#log-log4perl) - [logaddexp](#logaddexp) - [LOHHLA](#lohhla) - [Loki](#loki) - [longestrunsubsequence](#longestrunsubsequence) - [longread_umi](#longread_umi) - [Longshot](#longshot) - [loompy](#loompy) - [loomR](#loomr) - [LoRDEC](#lordec) - [LPeg](#lpeg) - [LPJmL](#lpjml) - [lpsolve](#lpsolve) - [lrslib](#lrslib) - [LS-PrePost](#ls-prepost) - [LSD2](#lsd2) - [LSMS](#lsms) - [LTR_retriever](#ltr_retriever) - [LtrDetector](#ltrdetector) - [Lua](#lua) - [LuaJIT](#luajit) - [LuaJIT2-OpenResty](#luajit2-openresty) - [LuaRocks](#luarocks) - [Lucene-Geo-Gazetteer](#lucene-geo-gazetteer) - [LUMPY](#lumpy) - [LUSCUS](#luscus) - [lwgrp](#lwgrp) - [lxml](#lxml) - [lynx](#lynx) - [lz4](#lz4) - [LZO](#lzo)


### L_RNA_scaffolder

L_RNA_scaffolder is a genome scaffolding tool with long trancriptome reads

*homepage*: <https://github.com/CAFS-bioinformatics/L_RNA_scaffolder>

version     |versionsuffix   |toolchain
------------|----------------|---------------
``20141124``|``-Perl-5.24.0``|``intel/2016b``
``20190530``|                |``GCC/11.3.0``

### Lace

Building SuperTranscripts: A linear representation of transcriptome data

*homepage*: <https://github.com/Oshlack/Lace>

version   |toolchain
----------|--------------
``1.14.1``|``foss/2022a``

### LADR

Prover9, Mace4, and several related programs come packaged in  a system called LADR (Library for Automated Deduction Research). 

*homepage*: <https://www.cs.unm.edu/~mccune/mace4/download>

version     |toolchain
------------|------------------
``2009-11A``|``GCCcore/10.2.0``

### lagrangian-filtering

Temporal filtering of data in a Lagrangian frame of reference.

*homepage*: <https://github.com/angus-g/lagrangian-filtering>

version  |toolchain
---------|--------------
``0.8.3``|``foss/2022a``

### LAME

LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.

*homepage*: <http://lame.sourceforge.net/>

version   |toolchain
----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.99.5``|``foss/2016b``, ``intel/2017a``
``3.100`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017b``

### LAMMPS

LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be simple to modify or extend with new functionality. 

*homepage*: <https://lammps.sandia.gov/>

version      |versionsuffix                |toolchain
-------------|-----------------------------|----------------------------------------------
``3Mar2020`` |``-Python-3.7.4-kokkos``     |``foss/2019b``, ``intel/2019b``
``3Mar2020`` |``-Python-3.8.2-kokkos``     |``foss/2020a``, ``intel/2020a``
``7Aug2019`` |``-Python-3.7.4-kokkos``     |``foss/2019b``, ``intel/2019b``
``7Aug2019`` |``-Python-3.7.4-kokkos-OCTP``|``intel/2019b``
``23Jun2022``|``-kokkos``                  |``foss/2021a``, ``foss/2021b``, ``foss/2022a``
``23Jun2022``|``-kokkos-CUDA-11.3.1``      |``foss/2021a``
``23Jun2022``|``-kokkos-CUDA-11.4.1``      |``foss/2021b``

### lancet

Lancet is a somatic variant caller (SNVs and indels) for short read data.

*homepage*: <https://github.com/nygenome/lancet>

version  |toolchain
---------|-----------------------
``1.1.0``|``iccifort/2019.5.281``

### LAPACK

LAPACK is written in Fortran90 and provides routines for solving systems of  simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue  problems, and singular value problems.

*homepage*: <https://www.netlib.org/lapack/>

version   |toolchain
----------|------------------------------
``3.8.0`` |``GCC/7.3.0-2.30``
``3.9.1`` |``GCC/10.2.0``
``3.10.1``|``GCC/11.2.0``, ``GCC/11.3.0``

### LASSO-Python

This python library is designed for general purpose usage in the field of Computer Aided Engineering (CAE). It's name originates from the original initiator and donator of the project Lasso GmbH. The library is now maintained by an open-source community.

*homepage*: <https://open-lasso-python.github.io/lasso-python/>

version  |toolchain
---------|--------------
``2.0.0``|``foss/2022b``

### LAST

LAST finds similar regions between sequences.

*homepage*: <http://last.cbrc.jp/>

version |toolchain
--------|---------------
``869`` |``intel/2017a``
``914`` |``intel/2017b``
``1045``|``intel/2019b``
``1179``|``GCC/10.2.0``

### LASTZ

LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle  sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS   sequencing technologies such as Roche 454. 

*homepage*: <https://www.bx.psu.edu/~rsharris/lastz/>

version    |toolchain
-----------|---------------------------------
``1.02.00``|``GCCcore/8.2.0``, ``foss/2016a``
``1.04.03``|``foss/2019b``

### lavaan

Fit a variety of latent variable models, including confirmatory factor analysis, structural  equation modeling and latent growth curve models.

*homepage*: <https://cran.r-project.org/web/packages/lavaan>

version       |versionsuffix|toolchain
--------------|-------------|---------------
``0.6-2``     |``-R-3.4.4`` |``intel/2018a``
``0.6-4.1433``|``-R-3.6.0`` |``foss/2019a``
``0.6-9``     |``-R-4.1.0`` |``foss/2021a``

### LayoutParser

A Unified Toolkit for Deep Learning Based Document Image Analysis

*homepage*: <https://layout-parser.github.io/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.3.4``|                |``foss/2022a``
``0.3.4``|``-CUDA-11.7.0``|``foss/2022a``

### LBFGS++

A header-only C++ library for L-BFGS and L-BFGS-B algorithms

*homepage*: <https://lbfgspp.statr.me>

version  |toolchain
---------|----------
``0.1.0``|``system``

### LCov

LCOV - the LTP GCOV extension

*homepage*: <http://ltp.sourceforge.net/coverage/lcov.php>

version |toolchain
--------|-----------------
``1.13``|``GCCcore/7.2.0``

### LDC

The LLVM-based D Compiler

*homepage*: <https://wiki.dlang.org/LDC>

version   |versionsuffix|toolchain
----------|-------------|------------------
``0.17.6``|``-x86_64``  |``system``
``1.24.0``|``-x86_64``  |``system``
``1.25.1``|             |``GCCcore/10.2.0``
``1.26.0``|             |``GCCcore/10.3.0``
``1.30.0``|             |``GCCcore/11.3.0``

### lDDT

The local Distance Difference Test (lDDT) is a superposition-free score which evaluates local distance differences in a model compared to a reference structure.

*homepage*: <https://openstructure.org>

version|toolchain
-------|----------
``1.2``|``system``

### LeadIT

Visually Informed LeadOpt

*homepage*: <http://www.biosolveit.de/LeadIT/index.html>

version  |toolchain
---------|----------
``2.1.9``|``system``

### leidenalg

Implementation of the Leiden algorithm for various quality functions to be used with igraph in Python.

*homepage*: <https://github.com/vtraag/leidenalg>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.8.2``|``-Python-3.8.2``|``foss/2020a``
``0.8.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.8.7``|                 |``foss/2021a``
``0.8.8``|                 |``foss/2021b``
``0.9.1``|                 |``foss/2022a``

### LEMON

LEMON stands for Library for Efficient Modeling and Optimization in Networks.  It is a C++ template library providing efficient implementations of common data structures and algorithms  with focus on combinatorial optimization tasks connected mainly with graphs and networks. 

*homepage*: <https://lemon.cs.elte.hu>

version  |toolchain
---------|--------------------
``1.3.1``|``GCC/8.2.0-2.31.1``

### Leptonica

Leptonica is a collection of pedagogically-oriented open source software  that is broadly useful for image processing and image analysis applications.

*homepage*: <http://www.leptonica.org>

version   |toolchain
----------|------------------
``1.77.0``|``GCCcore/7.3.0``
``1.78.0``|``GCCcore/8.2.0``
``1.82.0``|``GCCcore/10.3.0``
``1.83.0``|``GCCcore/11.3.0``

### LERC

LERC is an open-source image or raster format which supports rapid encoding and decoding for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding, so the precision of the original input image is preserved (within user defined error bounds).

*homepage*: <https://github.com/Esri/lerc>

version  |toolchain
---------|--------------------------------------
``3.0``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.0.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### less

Less is a free, open-source file pager. It can be found on most versions of Linux,  Unix and Mac OS, as well as on many other operating systems.

*homepage*: <http://www.greenwoodsoftware.com/less/>

version|toolchain
-------|-------------
``458``|``GCC/4.8.2``

### LevelDB

LevelDB is a fast key-value storage library written at Google that provides an  ordered mapping from string keys to string values.

*homepage*: <https://github.com/google/leveldb>

version |toolchain
--------|------------------------------------------------
``1.18``|``foss/2016a``, ``intel/2017a``, ``intel/2017b``
``1.20``|``GCCcore/7.3.0``
``1.22``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``

### lftp

LFTP is a sophisticated ftp/http client, and a file transfer program supporting a number of network protocols. Like BASH, it has job control and uses the readline library for input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel. It was designed with reliability in mind.

*homepage*: <https://lftp.yar.ru>

version  |toolchain
---------|------------------
``4.6.4``|``GNU/4.9.3-2.25``
``4.8.4``|``GCCcore/6.4.0``
``4.9.2``|``GCCcore/11.2.0``

### LHAPDF

Les Houches Parton Density Function  LHAPDF is the standard tool for evaluating parton distribution functions (PDFs) in high-energy physics. 

*homepage*: <http://lhapdf.hepforge.org/>

version  |toolchain
---------|--------------
``6.5.3``|``GCC/11.3.0``

### LIANA

LIANA: a LIgand-receptor ANalysis frAmework. LIANA enables the use of any   combination of ligand-receptor methods and resources, and their consensus.

*homepage*: <https://saezlab.github.io/liana/>

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.1.11``|``-R-4.2.1`` |``foss/2022a``

### libaec

Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa.

*homepage*: <https://gitlab.dkrz.de/k202009/libaec>

version  |toolchain
---------|------------------------------------------------------------------------------
``1.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### libaio

Asynchronous input/output library that uses the kernels native interface.

*homepage*: <https://pagure.io/libaio>

version    |toolchain
-----------|------------------------------------------------------------------------------
``0.3.111``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.3.112``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``0.3.113``|``GCCcore/12.2.0``

### libarchive

Multi-format archive and compression library 

*homepage*: <https://www.libarchive.org/>

version  |toolchain
---------|---------------------------------------------------------
``3.4.0``|``GCCcore/8.2.0``
``3.4.2``|``GCCcore/9.3.0``
``3.4.3``|``GCCcore/10.2.0``
``3.5.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``
``3.6.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``3.6.2``|``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### libav

Libav is a friendly and community-driven effort to provide its users with  a set of portable, functional and high-performance libraries for dealing  with multimedia formats of all sorts. 

*homepage*: <https://libav.org/>

version  |toolchain
---------|-----------------
``11.10``|``GCCcore/6.4.0``

### libavif

This library aims to be a friendly, portable C implementation of the AV1 Image File Format, as described here: https://aomediacodec.github.io/av1-avif/ 

*homepage*: <https://github.com/AOMediaCodec/libavif>

version   |toolchain
----------|------------------------------
``0.9.0`` |``foss/2020b``
``0.11.1``|``foss/2021a``, ``foss/2022a``

### libBigWig

A C library for handling bigWig files

*homepage*: <https://github.com/dpryan79/libBigWig>

version  |toolchain
---------|------------------
``0.4.4``|``GCCcore/8.3.0``
``0.4.6``|``GCCcore/11.2.0``

### libbitmask

libbitmask provides a convenient, powerful bitmask data type

*homepage*: <http://oss.sgi.com/projects/cpusets/>

version|toolchain
-------|----------
``2.0``|``system``

### libcdms

Climate Data Management System Library.

*homepage*: <https://github.com/CDAT/libcdms/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.2``|``-Python-3.8.2``|``foss/2020a``

### libcerf

libcerf is a self-contained numeric library that provides an efficient and  accurate implementation of complex error functions, along with Dawson,  Faddeeva, and Voigt functions. 

*homepage*: <https://jugit.fz-juelich.de/mlz/libcerf>

version |toolchain
--------|---------------------------------------------------------------------------------------------------------
``1.4`` |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.5`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.7`` |``GCCcore/7.3.0``
``1.11``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.13``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.14``|``GCCcore/10.2.0``
``1.15``|``GCCcore/10.3.0``
``1.17``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.1`` |``GCCcore/11.3.0``
``2.3`` |``GCCcore/12.2.0``

### libcint

libcint is an open source library for analytical Gaussian integrals.

*homepage*: <https://github.com/sunqm/libcint>

version  |toolchain
---------|-----------------------------------------------
``4.4.0``|``foss/2020b``, ``foss/2021a``, ``gomkl/2021a``
``5.1.6``|``foss/2022a``

### libcircle

An API to provide an efficient distributed queue on a cluster. libcircle is an  API for distributing embarrassingly parallel workloads using self-stabilization. 

*homepage*: <https://github.com/hpc/libcircle/>

version       |toolchain
--------------|------------------------------------------------------------------
``0.2.1-rc.1``|``gompi/2019a``, ``iimpi/2019a``
``0.3``       |``gompi/2020a``, ``gompi/2020b``, ``gompi/2022a``, ``iimpi/2020a``

### libcmaes

libcmaes is a multithreaded C++11 library for high performance blackbox stochastic optimization using the CMA-ES algorithm for Covariance Matrix Adaptation Evolution Strategy.

*homepage*: <http://beniz.github.io/libcmaes/>

version  |toolchain
---------|--------------
``0.9.5``|``foss/2016a``

### libconfig

Libconfig is a simple library for processing structured configuration files

*homepage*: <https://hyperrealm.github.io/libconfig>

version  |toolchain
---------|--------------------------------------
``1.5``  |``intel/2016b``
``1.7.1``|``GCCcore/6.4.0``
``1.7.2``|``GCCcore/7.3.0``
``1.7.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### libcpuset

libcpuset provides full access to cpuset capabilities

*homepage*: <http://oss.sgi.com/projects/cpusets/>

version|toolchain
-------|----------
``1.0``|``system``

### libcroco

Libcroco is a standalone css2 parsing and manipulation library.

*homepage*: <https://github.com/GNOME/libcroco>

version   |toolchain
----------|------------------------------
``0.6.11``|``intel/2016a``
``0.6.13``|``GCC/10.2.0``, ``foss/2019a``

### libctl

libctl is a free Guile-based library implementing flexible control files for scientific simulations.

*homepage*: <https://libctl.readthedocs.io/en/latest/>

version  |toolchain
---------|------------------
``3.2.2``|``foss/2016a``
``4.0.0``|``intel/2020a``
``4.1.3``|``GCCcore/6.4.0``
``4.5.1``|``GCCcore/10.2.0``

### libdap

A C++ SDK which contains an implementation of DAP 2.0 and  DAP4.0. This includes both Client- and Server-side support classes.

*homepage*: <https://www.opendap.org/software/libdap>

version    |versionsuffix     |toolchain
-----------|------------------|---------------------------------------------------------
``3.18.1`` |                  |``intel/2017a``
``3.18.1`` |``-Python-2.7.11``|``foss/2016a``
``3.19.1`` |                  |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``3.20.3`` |                  |``GCCcore/7.3.0``
``3.20.4`` |                  |``GCCcore/8.2.0``
``3.20.6`` |                  |``GCCcore/8.3.0``
``3.20.7`` |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``3.20.8`` |                  |``GCCcore/11.2.0``
``3.20.11``|                  |``GCCcore/11.3.0``

### libde265

libde265 is an open source implementation of the h.265 video codec

*homepage*: <https://github.com/strukturag/libde265>

version   |toolchain
----------|------------------------------
``1.0.8`` |``GCC/10.3.0``, ``GCC/11.2.0``
``1.0.11``|``GCC/11.3.0``

### libdeflate

Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.

*homepage*: <https://github.com/ebiggers/libdeflate>

version |toolchain
--------|---------------------------------------------------------
``1.5`` |``GCCcore/7.3.0``
``1.7`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.8`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.10``|``GCCcore/11.3.0``
``1.15``|``GCCcore/12.2.0``
``1.18``|``GCCcore/12.3.0``

### libdivsufsort

libdivsufsort is a software library that implements a lightweight suffix  array construction algorithm.

*homepage*: <https://github.com/y-256/libdivsufsort>

version  |toolchain
---------|--------------
``2.0.1``|``foss/2018b``

### libdrm

Direct Rendering Manager runtime library.

*homepage*: <https://dri.freedesktop.org>

version    |toolchain
-----------|--------------------------------------------------
``2.4.67`` |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.4.68`` |``foss/2016a``, ``intel/2016a``
``2.4.70`` |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``2.4.76`` |``GCCcore/6.3.0``, ``intel/2017a``
``2.4.88`` |``GCCcore/6.4.0``
``2.4.91`` |``GCCcore/6.4.0``
``2.4.92`` |``GCCcore/7.3.0``
``2.4.97`` |``GCCcore/8.2.0``
``2.4.99`` |``GCCcore/8.3.0``
``2.4.100``|``GCCcore/9.3.0``
``2.4.102``|``GCCcore/10.2.0``
``2.4.106``|``GCCcore/10.3.0``
``2.4.107``|``GCCcore/11.2.0``
``2.4.110``|``GCCcore/11.3.0``
``2.4.114``|``GCCcore/12.2.0``

### libdrs

PCMDI's old DRS format implementation

*homepage*: <https://github.com/CDAT/libdrs/>

version  |toolchain
---------|--------------
``3.1.2``|``foss/2020a``

### libdwarf

The DWARF Debugging Information Format is of interest to programmers working on compilers and debuggers (and anyone interested in reading or writing DWARF information))

*homepage*: <https://www.prevanders.net/dwarf.html>

version     |toolchain
------------|---------------------------------------------------
``0.4.1``   |``GCCcore/11.3.0``
``20140805``|``GCC/4.9.2``
``20150310``|``GCC/4.9.2``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``
``20190529``|``GCCcore/8.2.0``
``20201201``|``GCCcore/10.2.0``
``20210305``|``GCCcore/10.3.0``
``20210528``|``GCCcore/11.2.0``

### libedit

This BSD-style licensed command line editor library provides generic line editing, history, and tokenization functions, similar to those found in GNU Readline. 

*homepage*: <https://thrysoee.dk/editline/>

version     |toolchain
------------|------------------
``20150325``|``GNU/4.9.3-2.25``
``20180525``|``GCCcore/6.4.0``
``20191231``|``GCCcore/9.3.0``
``20210910``|``GCCcore/11.2.0``

### libelf

libelf is a free ELF object file access library

*homepage*: <https://directory.fsf.org/wiki/Libelf>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.8.13``|``GCC/4.8.3``, ``GCC/4.9.2``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``

### libemf

Library implementation of ECMA-234 API for the generation of enhanced metafiles.

*homepage*: <https://sourceforge.net/projects/libemf/>

version   |toolchain
----------|------------------
``1.0.13``|``GCCcore/11.3.0``

### libepoxy

Epoxy is a library for handling OpenGL function pointer management for you

*homepage*: <https://github.com/anholt/libepoxy>

version   |toolchain
----------|--------------------------------------------------------
``1.5.2`` |``foss/2018a``
``1.5.3`` |``GCCcore/8.2.0``, ``fosscuda/2018b``
``1.5.4`` |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.5.8`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.5.10``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### libev

A full-featured and high-performance (see benchmark) event loop that is loosely modelled after libevent, but without its limitations and bugs. It is used in GNU Virtual Private Ethernet, rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many other programs.

*homepage*: <http://software.schmorp.de/pkg/libev.html>

version |toolchain
--------|------------------------------
``4.33``|``GCC/11.2.0``, ``GCC/11.3.0``

### libevent

The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.

*homepage*: <http://libevent.org/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------
``2.0.22``|``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GNU/4.9.3-2.25``
``2.1.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``system``
``2.1.11``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.1.12``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``system``

### libexif

A library for parsing, editing, and saving EXIF data.

*homepage*: <https://libexif.github.io/>

version   |toolchain
----------|------------------
``0.6.24``|``GCCcore/11.2.0``

### libfabric

Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. 

*homepage*: <https://ofiwg.github.io/libfabric/>

version   |toolchain
----------|-------------------------------------
``1.9.1`` |``GCCcore/9.3.0``
``1.10.1``|``GCCcore/9.3.0``
``1.11.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.12.1``|``GCCcore/10.3.0``
``1.13.0``|``GCCcore/11.2.0``
``1.13.1``|``GCCcore/11.2.0``
``1.13.2``|``GCCcore/11.2.0``
``1.15.1``|``GCCcore/11.3.0``
``1.16.1``|``GCCcore/12.2.0``
``1.18.0``|``GCCcore/12.3.0``

### libfdf

LibFDF provides a Fortran API to parse files in the  Flexible Data Format (FDF).

*homepage*: <https://launchpad.net/libfdf>

version  |versionsuffix|toolchain
---------|-------------|-----------------------------------------------------------------------------------------------------------------------------------
``0.2.2``|``-serial``  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

### libffcall

GNU Libffcall is a collection of four libraries which can be used to build  foreign function call interfaces in embedded interpreters 

*homepage*: <https://www.gnu.org/software/libffcall/>

version |toolchain
--------|------------------------------------
``1.13``|``GCCcore/6.4.0``
``2.2`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.4`` |``GCCcore/10.2.0``

### libffi

The libffi library provides a portable, high level programming interface to  various calling conventions. This allows a programmer to call any function  specified by a call interface description at run-time.

*homepage*: <https://sourceware.org/libffi/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.1``|``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``system``
``3.3``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``3.4.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.4.4``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### libFLAME

libFLAME is a portable library for dense matrix computations, providing much of the functionality present in LAPACK.

*homepage*: <https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------------------------------
``1.0``  |``-amd``     |``GCC/7.3.0-2.30``
``2.2``  |``-amd``     |``GCCcore/9.3.0``
``5.2.0``|             |``GCC/10.3.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``

### libfontenc

X11 font encoding library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs/>

version  |toolchain
---------|-------------------------------------------------
``1.1.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libgcrypt

Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG

*homepage*: <https://gnupg.org/related_software/libgcrypt/index.html>

version   |toolchain
----------|--------------------------------------
``1.6.5`` |``intel/2016a``
``1.8.4`` |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.8.5`` |``GCCcore/8.3.0``
``1.9.2`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.9.3`` |``GCCcore/11.2.0``
``1.10.1``|``GCCcore/11.3.0``

### libgd

GD is an open source code library for the dynamic creation of images by programmers.

*homepage*: <https://libgd.github.io>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------
``2.1.1``|``foss/2016a``, ``intel/2016a``
``2.2.3``|``foss/2016b``, ``intel/2016b``
``2.2.4``|``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``
``2.2.5``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017b``, ``intel/2018a``
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.3.1``|``GCCcore/10.3.0``
``2.3.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### libgdiplus

Libgdiplus is the Mono library that provides a GDI+-compatible  API on non-Windows operating systems.

*homepage*: <https://www.mono-project.com/docs/gui/libgdiplus>

version|toolchain
-------|------------------
``6.1``|``GCCcore/11.2.0``

### libGDSII

libGDSII is a C++ library for working with GDSII binary data files,  intended primarily for use with the computational electromagnetism codes  scuff-em and meep but sufficiently general-purpose to allow other uses as well.

*homepage*: <https://github.com/HomerReid/libGDSII>

version |toolchain
--------|-------------------------------------
``0.21``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``

### libgeotiff

Library for reading and writing coordinate system information from/to GeoTIFF files

*homepage*: <https://directory.fsf.org/wiki/Libgeotiff>

version  |toolchain
---------|----------------------------------------------------------
``1.4.2``|``foss/2018a``, ``foss/2018b``, ``intel/2018b``
``1.5.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.6.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.7.0``|``GCCcore/11.2.0``
``1.7.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libgit2

libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant linkable library with a solid API, allowing you to write native speed custom Git applications in any language which supports C bindings.

*homepage*: <https://libgit2.org/>

version  |toolchain
---------|---------------------------------------------------------
``1.0.0``|``GCCcore/8.3.0``
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.1.1``|``GCCcore/11.2.0``
``1.4.3``|``GCCcore/11.3.0``
``1.5.0``|``GCCcore/12.2.0``

### libglade

Libglade is a library for constructing user interfaces dynamically from XML descriptions.

*homepage*: <https://developer.gnome.org/libglade/>

version  |toolchain
---------|-----------------------------------------------------------------
``2.6.4``|``foss/2018b``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``

### libGLU

The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 

*homepage*: <https://mesa.freedesktop.org/archive/glu/>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``9.0.0``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2018a``
``9.0.0``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``9.0.1``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0.2``|                |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### libglvnd

libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.

*homepage*: <https://gitlab.freedesktop.org/glvnd/libglvnd>

version  |toolchain
---------|-------------------------------------------------------
``1.2.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.3.2``|``GCCcore/10.2.0``
``1.3.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.4.0``|``GCCcore/11.3.0``
``1.6.0``|``GCCcore/12.2.0``

### libgpg-error

Libgpg-error is a small library that defines common error values for all GnuPG components.

*homepage*: <https://gnupg.org/related_software/libgpg-error/index.html>

version |toolchain
--------|--------------------------------------
``1.21``|``intel/2016a``
``1.35``|``GCCcore/7.3.0``
``1.36``|``GCCcore/8.2.0``
``1.38``|``GCCcore/8.3.0``
``1.41``|``GCCcore/10.2.0``
``1.42``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.46``|``GCCcore/11.3.0``

### libgpuarray

Library to manipulate tensors on the GPU.

*homepage*: <http://deeplearning.net/software/libgpuarray/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------
``0.7.5``|``-Python-2.7.14``|``fosscuda/2017b``, ``intelcuda/2017b``
``0.7.5``|``-Python-3.6.3`` |``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.7.6``|                  |``fosscuda/2019a``, ``fosscuda/2020b``
``0.7.6``|``-Python-2.7.15``|``fosscuda/2018b``
``0.7.6``|``-Python-3.6.6`` |``fosscuda/2018b``
``0.7.6``|``-Python-3.7.4`` |``fosscuda/2019b``

### libGridXC

A library to compute the exchange and correlation energy  and potential in spherical (i.e. atoms) or periodic systems.

*homepage*: <https://gitlab.com/siesta-project/libraries/libgridxc>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------
``0.8.5``|``iimpi/2019b``
``0.9.6``|``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``
``1.1.0``|``gompi/2022a``

### libgtextutils

ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream

*homepage*: <http://hannonlab.cshl.edu/fastx_toolkit/>

version|toolchain
-------|------------------------------------------------------------------
``0.7``|``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``

### libharu

libHaru is a free, cross platform, open source library for generating PDF files.

*homepage*: <http://libharu.org/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2021a``, ``foss/2021b``, ``intel/2017a``, ``intel/2018b``

### libheif

libheif is an HEIF and AVIF file format decoder and encoder

*homepage*: <https://github.com/strukturag/libheif>

version   |toolchain
----------|------------------------------
``1.12.0``|``GCC/10.3.0``, ``GCC/11.2.0``
``1.16.2``|``GCC/11.3.0``

### libibmad

libibmad is a convenience library to encode, decode, and dump IB MAD packets. It  is implemented on top of and in conjunction with libibumad (the umad kernel  interface library.)

*homepage*: <http://www.openfabrics.org>

version   |toolchain
----------|------------------
``1.3.12``|``GCC/4.9.3-2.25``

### libibumad

libibumad is the umad kernel interface library.

*homepage*: <http://www.openfabrics.org>

version     |toolchain
------------|------------------
``1.3.10.2``|``GCC/4.9.3-2.25``

### libICE

X Inter-Client Exchange library for freedesktop.org

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.0.9``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libiconv

Libiconv converts from one character encoding to another through Unicode conversion

*homepage*: <https://www.gnu.org/software/libiconv>

version |toolchain
--------|-------------------------------------------------------------------------------------------------------------------
``1.15``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``1.16``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.17``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libidn

GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications. Libidn's purpose is to encode and decode internationalized domain names.

*homepage*: <http://www.gnu.org/software/libidn>

version |toolchain
--------|--------------------------------------------------------------------------------------
``1.32``|``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.34``|``GCCcore/6.4.0``
``1.35``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.36``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.38``|``GCCcore/11.2.0``
``1.41``|``GCCcore/11.3.0``

### libidn2

Libidn2 implements the revised algorithm for internationalized domain names called IDNA2008/TR46.

*homepage*: <http://www.gnu.org/software/libidn2>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.3.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### Libint

Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.

*homepage*: <https://github.com/evaleev/libint>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.4``|                |``intel/2016a``
``1.1.6``|                |``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018a``, ``foss/2020a``, ``foss/2020b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2020a``
``2.0.3``|                |``foss/2018b``, ``gompi/2019a``, ``intel/2018b``
``2.1.0``|                |``intel/2016b``
``2.4.2``|                |``intel/2018a``
``2.5.0``|                |``gompi/2019a``, ``iimpi/2019a``
``2.6.0``|``-lmax-6-cp2k``|``GCC/10.2.0``, ``GCC/10.3.0``, ``gompi/2020a``, ``iccifort/2020.4.304``, ``iimpi/2020a``, ``iimpi/2021a``
``2.7.2``|``-lmax-6-cp2k``|``GCC/11.3.0``

### LiBis

An ultrasensitive alignment method for low input bisulfite sequencing

*homepage*: <https://github.com/Dangertrip/LiBis>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20200428``|``-Python-3.7.4``|``foss/2019b``

### libjpeg-turbo

libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to  accelerate baseline JPEG compression and decompression. libjpeg is a library  that implements JPEG image encoding, decoding and transcoding. 

*homepage*: <https://sourceforge.net/projects/libjpeg-turbo/>

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------------------------------------------
``1.4.2``  |                 |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.4.2``  |``-NASM-2.12.01``|``foss/2016a``, ``intel/2016a``
``1.5.0``  |                 |``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.5.1``  |                 |``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.5.2``  |                 |``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.5.3``  |                 |``GCCcore/6.4.0``
``2.0.0``  |                 |``GCCcore/7.3.0``
``2.0.2``  |                 |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.0.3``  |                 |``GCCcore/8.3.0``
``2.0.4``  |                 |``GCCcore/9.3.0``
``2.0.5``  |                 |``GCCcore/10.2.0``
``2.0.6``  |                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.1.3``  |                 |``GCCcore/11.3.0``
``2.1.4``  |                 |``GCCcore/12.2.0``
``2.1.5.1``|                 |``GCCcore/12.3.0``

### libjxl

JPEG XL image format reference implementation

*homepage*: <https://github.com/libjxl/libjxl>

version  |toolchain
---------|------------------
``0.5``  |``GCCcore/10.3.0``
``0.6.1``|``GCCcore/10.2.0``
``0.8.1``|``foss/2022a``

### LibLZF

LibLZF is a very small data compression library. It consists of only two .c and two .h files and is very simple to incorporate into your own programs. The compression algorithm is very, very fast, yet still written in portable C.

*homepage*: <http://oldhome.schmorp.de/marc/liblzf.html>

version|toolchain
-------|--------------------------------------
``3.4``|``GCCcore/10.2.0``
``3.6``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### libmad

MAD is a high-quality MPEG audio decoder.

*homepage*: <https://www.underbit.com/products/mad/>

version    |toolchain
-----------|------------------
``0.15.1b``|``GCCcore/11.3.0``

### libmatheval

GNU libmatheval is a library (callable from C and Fortran) to parse  and evaluate symbolic expressions input as text.

*homepage*: <http://www.gnu.org/software/libmatheval/>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.11``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``

### libmaus2

libmaus2 is a collection of data structures and algorithms.

*homepage*: <https://github.com/gt1/libmaus2>

version    |toolchain
-----------|---------------
``2.0.453``|``intel/2018a``
``2.0.499``|``GCC/11.3.0``

### libmbd

Libmbd implements the many-body dispersion (MBD) method in several programming languages and frameworks:   - The Fortran implementation is the reference, most advanced implementation, with support for analytical     gradients and distributed parallelism, and additional functionality beyond the MBD method itself.     It provides a low-level and a high-level Fortran API, as well as a C API. Furthermore, Python bindings     to the C API are provided.  - The Python/Numpy implementation is intended for prototyping, and as a high-level language reference.  - The Python/Tensorflow implementation is an experiment that should enable rapid prototyping of machine     learning applications with MBD.  The Python-based implementations as well as Python bindings to the Libmbd C API are accessible from the  Python package called Pymbd. 

*homepage*: <https://libmbd.github.io/index.html>

version   |toolchain
----------|----------------------------------------------------------------
``0.10.4``|``foss/2020b``, ``foss/2021a``, ``intel/2020b``, ``intel/2021a``

### libMemcached

libMemcached is an open source C/C++ client library and tools for  the memcached server (http://danga.com/memcached). It has been designed to be light  on memory usage, thread safe, and provide full access to server side methods.

*homepage*: <https://memcached.org>

version   |toolchain
----------|------------------------------------
``1.0.18``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

### libmicrohttpd

GNU libmicrohttpd is a small C library that is supposed to make it simple to run  an HTTP server as part of another application. 

*homepage*: <https://www.gnu.org/software/libmicrohttpd/>

version   |toolchain
----------|--------------------------------------------------------
``0.9.71``|``GCCcore/9.3.0``
``0.9.73``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

### libmo_unpack

A library for handling the WGDOS and RLE compression schemes used in UM files.

*homepage*: <https://github.com/SciTools/libmo_unpack>

version  |toolchain
---------|-----------------
``3.1.2``|``GCCcore/6.4.0``

### libmypaint

libmypaint, a.k.a. "brushlib", is a library for making brushstrokes which is used by MyPaint  and other projects

*homepage*: <https://github.com/mypaint/libmypaint>

version  |toolchain
---------|------------------
``1.6.1``|``GCCcore/10.3.0``

### libnsl

The libnsl package contains the public client interface for NIS(YP).

*homepage*: <https://github.com/thkukuk/libnsl>

version  |toolchain
---------|------------------
``1.3.0``|``GCCcore/10.3.0``
``2.0.0``|``GCCcore/11.3.0``

### libobjcryst

ObjCryst++ is Object-Oriented Crystallographic Library for C++

*homepage*: <https://github.com/diffpy/libobjcryst>

version     |toolchain
------------|---------------
``2017.2.3``|``intel/2020a``
``2021.1.2``|``foss/2021b``

### libogg

Ogg is a multimedia container format, and the native file and stream format for the Xiph.org multimedia codecs.

*homepage*: <https://xiph.org/ogg/>

version  |toolchain
---------|------------------------------------------------------------------------------
``1.3.4``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.3.5``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libopus

Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive  speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is  standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skype’s  SILK codec and Xiph.Org’s CELT codec.

*homepage*: <https://www.opus-codec.org/>

version  |toolchain
---------|--------------------------------------
``1.3.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.4``  |``GCCcore/12.3.0``

### libosmium

A fast and flexible C++ library for working with OpenStreetMap data.  The Osmium Library has extensive support for all types of OSM entities:  nodes, ways, relations, and changesets.  It allows reading from and writing to OSM files in XML and PBF formats, including change files and full history files.  Osmium can store OSM data in memory and on disk in various formats and using various indexes.  Its simple to use handler interface allows you to quickly write data filtering and conversion functions.  Osmium can create WKT, WKB, OGR, GEOS and GeoJSON geometries for simple conversion into many GIS formats  and it can assemble multipolygons from ways and relations.

*homepage*: <https://osmcode.org/libosmium/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2.15.4``|``-Python-3.6.6``|``foss/2018b``
``2.15.6``|``-Python-3.7.4``|``foss/2019b``

### libpci

Library for portable access to PCI bus configuration registers from PCI Utils.

*homepage*: <https://github.com/pciutils/pciutils>

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.7.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/6.4.0``

### libpciaccess

Generic PCI access library.

*homepage*: <https://cgit.freedesktop.org/xorg/lib/libpciaccess/>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------
``0.13.4``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``0.14``  |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.16``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``0.17``  |``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libplinkio

A small C and Python library for reading PLINK genotype files.

*homepage*: <https://github.com/mfranberg/libplinkio>

version  |toolchain
---------|-----------------
``0.9.8``|``GCCcore/9.3.0``

### libpng

libpng is the official PNG reference library

*homepage*: <http://www.libpng.org/pub/png/libpng.html>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------
``1.2.58``|``system``
``1.2.59``|``system``
``1.5.30``|``system``
``1.6.21``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.6.23``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.6.24``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``1.6.26``|``foss/2016b``, ``intel/2016b``
``1.6.27``|``intel/2016b``
``1.6.28``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``
``1.6.29``|``GCCcore/6.3.0``
``1.6.32``|``GCCcore/6.4.0``
``1.6.34``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.6.36``|``GCCcore/8.2.0``
``1.6.37``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.6.38``|``GCCcore/12.2.0``
``1.6.39``|``GCCcore/12.3.0``

### libpsl

C library for the Public Suffix List

*homepage*: <https://rockdaboot.github.io/libpsl>

version   |toolchain
----------|------------------------------------------------------------------------------
``0.20.2``|``GCCcore/7.3.0``
``0.21.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.21.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### libPSML

A library to handle PSML, the pseudopotential markup language.

*homepage*: <https://launchpad.net/libpsml>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------
``1.1.7`` |``foss/2016b``, ``foss/2017a``
``1.1.8`` |``iccifort/2019.5.281``
``1.1.10``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``
``1.1.12``|``GCC/11.3.0``

### libpsortb

PSORT family of programs for subcellular localization prediction as well as other datasets and  resources relevant to localization prediction.

*homepage*: <http://psort.org/>

version|toolchain
-------|--------------
``1.0``|``foss/2016a``

### libpspio

libpspio is a library to read and write pseudopotentials in multiple formats.

*homepage*: <https://gitlab.com/ElectronicStructureLibrary/libpspio>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------
``0.2.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

### libpthread-stubs

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring  a small footprint, latency hiding, direct access to the protocol, improved  threading support, and extensibility. 

*homepage*: <http://xcb.freedesktop.org/>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------
``0.3``|``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``0.4``|``GCCcore/6.4.0``

### libQGLViewer

libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers.

*homepage*: <http://libqglviewer.com/>

version  |versionsuffix   |toolchain
---------|----------------|-----------------------------------------------
``2.6.3``|                |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``2.6.3``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``2.6.4``|                |``intel/2016b``
``2.7.1``|                |``intel/2018a``
``2.8.0``|                |``GCCcore/10.3.0``

### libreadline

The GNU Readline library provides a set of functions for use by applications  that allow users to edit command lines as they are typed in. Both Emacs and  vi editing modes are available. The Readline library includes additional  functions to maintain a list of previously-entered command lines, to recall  and perhaps reedit those lines, and perform csh-like history expansion on  previous commands. 

*homepage*: <https://tiswww.case.edu/php/chet/readline/rltop.html>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.3``  |``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``7.0``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``8.0``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``8.1``  |``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``8.1.2``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``
``8.2``  |``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### libRmath

The routines supporting the distribution and special functions in R and a few others are declared  in C header file Rmath.h. These can be compiled into a standalone library for linking to other applications.

*homepage*: <https://cran.r-project.org/doc/manuals/r-release/R-admin.html#The-standalone-Rmath-library>

version  |toolchain
---------|------------------
``3.6.0``|``foss/2018b``
``4.0.0``|``GCCcore/9.3.0``
``4.1.0``|``GCCcore/10.2.0``
``4.1.2``|``GCCcore/11.2.0``
``4.2.0``|``GCCcore/10.3.0``
``4.2.1``|``GCCcore/11.3.0``

### librosa

Python module for audio and music processing

*homepage*: <https://librosa.github.io>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.7.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``

### librsb

A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format

*homepage*: <https://sourceforge.net/projects/librsb/>

version     |toolchain
------------|--------------
``1.2.0.9`` |``GCC/10.2.0``
``1.2.0.10``|``GCC/10.2.0``
``1.2.0.11``|``GCC/10.2.0``
``1.3.0.0`` |``GCC/10.2.0``
``1.3.0.1`` |``GCC/10.2.0``

### librsvg

Librsvg is a library to render SVG files using cairo.

*homepage*: <https://wiki.gnome.org/Projects/LibRsvg>

version    |toolchain
-----------|------------------
``2.40.15``|``intel/2016a``
``2.48.4`` |``foss/2019a``
``2.51.2`` |``GCCcore/10.3.0``
``2.52.8`` |``GCCcore/11.2.0``
``2.55.1`` |``GCCcore/11.3.0``

### librttopo

The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies using user-provided data stores.

*homepage*: <https://git.osgeo.org/gitea/rttopo/librttopo>

version  |toolchain
---------|--------------
``1.1.0``|``GCC/11.2.0``

### libsamplerate

Secret Rabbit Code (aka libsamplerate) is a Sample Rate Converter for audio.

*homepage*: <http://www.mega-nerd.com/libsamplerate>

version  |toolchain
---------|-----------------
``0.1.9``|``GCCcore/8.2.0``

### libSBML

libSBML (Systems Biology Markup Language library) is a free, open-source programming library to help you read, write, manipulate, translate, and validate SBML files and data streams. It's not an application itself (though it does come with example programs), but rather a library you embed in your own applications.

*homepage*: <http://sbml.org/Software/libSBML>

version   |toolchain
----------|------------------------------
``5.19.0``|``GCC/10.2.0``, ``GCC/10.3.0``

### libsigc++

The libsigc++ package implements a typesafe callback system  for standard C++.

*homepage*: <https://libsigcplusplus.github.io/libsigcplusplus/>

version   |toolchain
----------|------------------------------------
``2.10.0``|``GCCcore/6.4.0``
``2.10.1``|``GCCcore/7.3.0``
``2.10.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.10.8``|``GCCcore/10.3.0``
``3.4.0`` |``GCCcore/11.3.0``

### libsigsegv

GNU libsigsegv is a library for handling page faults in user mode.

*homepage*: <https://www.gnu.org/software/libsigsegv/>

version |toolchain
--------|------------------
``2.11``|``GCCcore/6.4.0``
``2.12``|``GCCcore/9.3.0``
``2.13``|``GCCcore/10.2.0``

### libSM

X11 Session Management library, which allows for applications to both manage sessions,  and make use of session managers to save and restore their state for later use.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.2.2``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libsndfile

Libsndfile is a C library for reading and writing files containing sampled sound  (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.

*homepage*: <http://www.mega-nerd.com/libsndfile>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------
``1.0.28``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017a``
``1.0.31``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.1.0`` |``GCCcore/11.3.0``
``1.2.0`` |``GCCcore/12.2.0``

### libsodium

Sodium is a modern, simple-to-use software library for encryption, decryption, signatures,  password hashing and more.

*homepage*: <https://doc.libsodium.org/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------
``1.0.6`` |``intel/2016a``
``1.0.8`` |``foss/2016a``
``1.0.11``|``foss/2016b``, ``intel/2016b``
``1.0.12``|``GCCcore/6.4.0``, ``intel/2017a``
``1.0.13``|``GCCcore/6.4.0``, ``foss/2017a``
``1.0.16``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.17``|``GCCcore/8.2.0``
``1.0.18``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### LibSoup

libsoup is an HTTP client/server library for GNOME. It uses GObjects and the glib main loop, to integrate well with GNOME applications, and also has a synchronous API, for use in threaded applications.

*homepage*: <https://wiki.gnome.org/Projects/libsoup>

version   |toolchain
----------|------------------
``2.66.1``|``GCCcore/8.2.0``
``2.70.0``|``GCCcore/8.3.0``
``2.72.0``|``GCCcore/10.2.0``
``2.74.0``|``GCCcore/10.3.0``
``3.0.7`` |``GCC/11.2.0``
``3.0.8`` |``GCC/11.3.0``

### libspatialindex

C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API

*homepage*: <https://libspatialindex.org>

version  |toolchain
---------|--------------------------------------------------------------------------------------
``1.8.5``|``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2018a``
``1.9.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### libspatialite

SpatiaLite is an open source library intended to extend the SQLite core to support  fully fledged Spatial SQL capabilities.

*homepage*: <https://www.gaia-gis.it/fossil/libspatialite/home>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------
``4.3.0a``|                 |``foss/2016b``, ``foss/2018b``, ``intel/2016b``
``4.3.0a``|``-Python-3.7.2``|``foss/2019a``
``4.3.0a``|``-Python-3.7.4``|``GCC/8.3.0``
``5.0.1`` |                 |``GCC/11.2.0``

### libssh

Multiplatform C library implementing the SSHv2 protocol on client and server side

*homepage*: <https://www.libssh.org>

version  |toolchain
---------|-----------------
``0.9.0``|``GCCcore/6.4.0``

### libStatGen

Useful set of classes for creating statistical genetic programs.

*homepage*: <https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen>

version     |toolchain
------------|------------------
``1.0.15``  |``GCCcore/10.2.0``
``20190330``|``GCCcore/6.4.0``

### LIBSVM

LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression  (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification.

*homepage*: <https://www.csie.ntu.edu.tw/~cjlin/libsvm/>

version |toolchain
--------|----------------------------------------------------------
``3.22``|``intel/2016b``, ``intel/2017b``
``3.23``|``foss/2018b``, ``intel/2018b``
``3.24``|``GCCcore/9.3.0``
``3.25``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.30``|``GCCcore/11.3.0``

### libtar

C library for manipulating POSIX tar files

*homepage*: <https://repo.or.cz/libtar.git>

version   |toolchain
----------|------------------------------------
``1.2.20``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

### libtasn1

Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and some other packages.  It was written by Fabio Fiorina, and has been shipped as part of GnuTLS  for some time but is now a proper GNU package.

*homepage*: <https://www.gnu.org/software/libtasn1/>

version   |toolchain
----------|---------------------------------------------------
``4.7``   |``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``4.12``  |``GCCcore/5.4.0``
``4.13``  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``4.16.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``4.17.0``|``GCCcore/10.3.0``
``4.18.0``|``GCCcore/11.2.0``
``4.19.0``|``GCCcore/11.3.0``

### libtecla

The tecla library provides UNIX and LINUX programs with interactive command line editing facilities,  similar to those of the UNIX tcsh shell. In addition to simple command-line editing, it supports recall of previously  entered command lines, TAB completion of file names or other tokens, and in-line wild-card expansion of filenames.  The internal functions which perform file-name completion and wild-card expansion are also available externally for  optional use by programs. 

*homepage*: <https://sites.astro.caltech.edu/~mcs/tecla/>

version  |toolchain
---------|------------------
``1.6.3``|``GCCcore/10.2.0``

### LibTIFF

tiff: Library and tools for reading and writing TIFF data files

*homepage*: <https://libtiff.gitlab.io/libtiff/>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------
``4.0.6`` |``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.0.7`` |``foss/2016b``, ``intel/2017a``
``4.0.8`` |``intel/2017a``, ``intel/2017b``
``4.0.9`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018b``
``4.0.10``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.1.0`` |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.2.0`` |``GCCcore/10.3.0``
``4.3.0`` |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.4.0`` |``GCCcore/12.2.0``
``4.5.0`` |``GCCcore/12.3.0``

### libtirpc

Libtirpc is a port of Suns Transport-Independent RPC library to Linux.

*homepage*: <https://sourceforge.net/projects/libtirpc/>

version  |toolchain
---------|----------------------------------------------------------
``1.1.4``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.2.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.3.1``|``GCCcore/10.2.0``
``1.3.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.3.3``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libtool

GNU libtool is a generic library support script. Libtool hides the complexity  of using shared libraries behind a consistent, portable interface. 

*homepage*: <https://www.gnu.org/software/libtool>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.2``|``GCC/4.8.2``, ``GCC/4.9.2``
``2.4.5``|``GCC/4.8.4``, ``GCC/4.9.2``
``2.4.6``|``FCC/4.5.0``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.4.7``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### libtree

libtree is a tool that turns ldd into a tree, explains why  shared libraries are found and why not and optionally deploys executables and  dependencies into a single directory

*homepage*: <https://github.com/haampie/libtree>

version  |toolchain
---------|------------------
``2.0.0``|``GCCcore/10.3.0``
``3.0.3``|``system``

### libunistring

This library provides functions for manipulating Unicode strings and for  manipulating C strings according to the Unicode standard.

*homepage*: <https://www.gnu.org/software/libunistring/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------
``0.9.3`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``0.9.6`` |``GCCcore/5.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016b``
``0.9.7`` |``GCCcore/6.4.0``
``0.9.10``|``GCCcore/10.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.0``   |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.1``   |``GCCcore/10.2.0``

### libunwind

The primary goal of libunwind is to define a portable and efficient C programming interface  (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the  preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain  (non-local goto). The API supports both local (same-process) and remote (across-process) operation.  As such, the API is useful in a number of applications

*homepage*: <https://www.nongnu.org/libunwind/>

version  |toolchain
---------|-----------------------------------------------------------------------
``1.1``  |``GCC/4.9.2``, ``foss/2016a``, ``intel/2016b``
``1.2.1``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016b``
``1.3.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.6.2``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libutempter

libutempter is library that provides an interface for terminal  emulators such as screen and xterm to record user sessions to utmp and wtmp files.

*homepage*: <http://git.altlinux.org/people/ldv/packages/?p=libutempter.git>

version    |toolchain
-----------|------------------
``1.1.6.2``|``GCC/6.4.0-2.28``

### LibUUID

Portable uuid C library

*homepage*: <http://sourceforge.net/projects/libuuid/>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------
``1.0.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``intel/2017a``, ``intel/2017b``

### libuv

libuv is a multi-platform support library with a focus on asynchronous I/O.

*homepage*: <https://libuv.org>

version   |toolchain
----------|-----------------
``1.37.0``|``GCCcore/8.3.0``

### libvdwxc

libvdwxc is a general library for evaluating energy and potential for exchange-correlation (XC) functionals from the vdW-DF family that can be used with various of density functional theory (DFT) codes.

*homepage*: <https://libvdwxc.org>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.3.2``|``foss/2018b``
``0.4.0``|``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

### libvorbis

Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed audio format

*homepage*: <https://xiph.org/vorbis/>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------
``1.3.7``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libWallModelledLES

libWallModelledLES is a library based on OpenFOAM® technology, extending the capabilities of OpenFOAM in the area of wall-modelled LES (WMLES). This is a turbulence modelling methodology, which allows to make LES cheaper by not resolving the inner region of turbulent boundary layers.

*homepage*: <https://bitbucket.org/lesituu/libwallmodelledles/>

version  |toolchain
---------|--------------
``0.6.1``|``foss/2021b``

### libwebp

WebP is a modern image format that provides superior lossless and lossy compression for images on the web. Using WebP, webmasters and web developers can create smaller, richer images that make the web faster.

*homepage*: <https://developers.google.com/speed/webp/>

version  |toolchain
---------|--------------------------------------------------------
``1.0.0``|``foss/2018b``
``1.0.2``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.2.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.4``|``GCCcore/11.3.0``

### libwpe

WPE is the reference WebKit port for embedded and low-consumption computer devices. It has been designed from the ground-up with performance, small footprint, accelerated content rendering, and simplicity of deployment in mind, bringing the excellence of the WebKit engine to countless platforms and target devices.

*homepage*: <https://webkit.org/wpe>

version   |toolchain
----------|------------------
``1.13.3``|``GCCcore/11.2.0``

### libX11

X11 client-side library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.6.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libXau

The libXau package contains a library implementing the X11 Authorization Protocol. This is useful for restricting client access to the display.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|--------------------------------------------------------------------
``1.0.8``|``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libxc

Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.

*homepage*: <https://www.tddft.org/programs/libxc>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.2.2``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``intel/2018b``
``2.2.3``|``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``
``3.0.0``|``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``3.0.1``|``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``foss/2020a``, ``foss/2020b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``, ``intel/2020a``
``4.0.1``|``foss/2017b``, ``intel/2017b``
``4.0.3``|``foss/2016b``, ``foss/2017a``
``4.2.3``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``
``4.3.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``
``5.1.2``|``GCC/10.2.0``
``5.1.3``|``GCC/10.2.0``
``5.1.5``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``5.1.6``|``GCC/11.2.0``, ``intel-compilers/2021.4.0``
``5.2.3``|``GCC/11.3.0``, ``intel-compilers/2022.1.0``
``6.1.0``|``GCC/12.2.0``, ``intel-compilers/2022.2.1``

### libxcb

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: <https://xcb.freedesktop.org/>

version   |toolchain
----------|-------------------------------------------------
``1.11.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.13``  |``GCCcore/6.4.0``

### libXcursor

X Cursor management library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version   |toolchain
----------|-------------------------------
``1.1.14``|``foss/2016a``, ``intel/2016a``

### libXdamage

X Damage extension library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.1.4``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libXdmcp

The libXdmcp package contains a library implementing the X Display Manager Control Protocol. This is useful for allowing clients to interact with the X Display Manager. 

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|--------------------------------------------------------------------
``1.1.2``|``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libXext

Common X Extensions library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.3.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libXfixes

X Fixes extension library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``5.0.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``5.0.2``|``intel/2016a``

### libXfont

X font libary

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |versionsuffix      |toolchain
---------|-------------------|-------------------------------------------------
``1.5.1``|                   |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.5.1``|``-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``

### libXft

X11 client-side library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |versionsuffix          |toolchain
---------|-----------------------|-------------------------------
``2.3.2``|                       |``foss/2016a``, ``intel/2016a``
``2.3.2``|``-fontconfig-2.11.95``|``intel/2016a``
``2.3.2``|``-freetype-2.6.3``    |``foss/2016a``

### libXi

LibXi provides an X Window System client interface to the XINPUT extension to the X protocol.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.7.6``|``foss/2016a``, ``intel/2016a``

### libXinerama

Xinerama multiple monitor library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.1.3``|``foss/2016a``, ``intel/2016a``

### libxkbcommon

xkbcommon is a library to handle keyboard descriptions,  including loading them from disk, parsing them and handling their state.  It's mainly meant for client toolkits, window systems,  and other system applications.

*homepage*: <http://xkbcommon.org/>

version  |toolchain
---------|-------------------------------
``0.6.1``|``foss/2016a``, ``intel/2016a``

### libxml++

libxml++ is a C++ wrapper for the libxml XML parser library.

*homepage*: <http://libxmlplusplus.sourceforge.net>

version   |toolchain
----------|-------------------------------------------------------
``2.40.1``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.42.1``|``GCC/10.3.0``

### libxml2

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). 

*homepage*: <http://xmlsoft.org/>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.9.2`` |                  |``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``2.9.3`` |                  |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.9.3`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2.9.4`` |                  |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.9.4`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.9.4`` |``-Python-2.7.13``|``intel/2017a``
``2.9.5`` |                  |``GCCcore/6.3.0``
``2.9.6`` |                  |``GCCcore/6.4.0``
``2.9.7`` |                  |``GCCcore/6.4.0``
``2.9.8`` |                  |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.9.9`` |                  |``GCCcore/8.3.0``
``2.9.10``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``2.9.13``|                  |``GCCcore/11.3.0``
``2.10.3``|                  |``GCCcore/12.2.0``
``2.11.4``|                  |``GCCcore/12.3.0``

### libxml2-python

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). This is the Python binding.

*homepage*: <http://xmlsoft.org/>

version   |versionsuffix     |toolchain
----------|------------------|------------------
``2.9.7`` |``-Python-2.7.14``|``intel/2018a``
``2.9.8`` |``-Python-2.7.15``|``foss/2018b``
``2.9.8`` |``-Python-3.7.2`` |``GCCcore/8.2.0``
``2.9.13``|                  |``GCCcore/11.3.0``

### libXmu

libXmu provides a set of miscellaneous utility convenience functions for X libraries to use.  libXmuu is a lighter-weight version that does not depend on libXt or libXext

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.1.2``|``foss/2016a``, ``intel/2016a``

### libXp

libXp provides the X print library.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|---------------
``1.0.3``|``intel/2016a``

### libXpm

libXp provides the X print library.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version   |toolchain
----------|-------------------------------
``3.5.11``|``foss/2016a``, ``intel/2016a``

### libXrandr

X Resize, Rotate and Reflection extension library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.5.0``|``foss/2016a``, ``intel/2016a``

### libXrender

X11 client-side library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``0.9.9``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libxslt

Libxslt is the XSLT C library developed for the GNOME project  (but usable outside of the Gnome platform).

*homepage*: <http://xmlsoft.org/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------------------------------------------------------
``1.1.28``|                  |``foss/2016a``, ``intel/2016a``
``1.1.28``|``-Python-2.7.11``|``intel/2016a``
``1.1.29``|                  |``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.1.30``|                  |``GCCcore/6.3.0``
``1.1.32``|                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.1.33``|                  |``GCCcore/8.2.0``
``1.1.34``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.1.37``|                  |``GCCcore/12.2.0``
``1.1.38``|                  |``GCCcore/12.3.0``

### libxsmm

LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86).

*homepage*: <https://github.com/hfp/libxsmm>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------
``1.4``   |``intel/2016a``
``1.4.4`` |``foss/2016b``, ``intel/2016b``
``1.6.4`` |``foss/2016b``, ``intel/2016b``
``1.8.2`` |``intel/2017b``
``1.8.3`` |``foss/2018a``, ``intel/2018a``
``1.10``  |``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.16.1``|``GCC/10.2.0``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``1.16.2``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``1.17``  |``GCC/11.2.0``, ``GCC/11.3.0``

### libXt

libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are  based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation).

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------------
``1.1.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### libXxf86vm

X11 XFree86 video mode extension library

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.1.4``|``foss/2016a``, ``intel/2016a``

### libyaml

LibYAML is a YAML parser and emitter written in C.

*homepage*: <https://pyyaml.org/wiki/LibYAML>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------
``0.1.6``|``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.1.7``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``system``
``0.2.1``|``GCCcore/7.3.0``, ``system``
``0.2.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.2.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### libzeep

Libzeep was originally developed to make it simple to create SOAP servers.

*homepage*: <https://github.com/mhekkel/libzeep>

version  |toolchain
---------|---------------
``5.0.1``|``gompi/2019b``

### libzip

libzip is a C library for reading, creating, and modifying zip archives.

*homepage*: <https://libzip.org/>

version  |toolchain
---------|----------------------------------------------------------
``1.5.2``|``GCCcore/8.2.0``
``1.7.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.9.2``|``GCCcore/11.3.0``

### lie_learn

lie_learn is a python package that knows how to do various tricky computations related to Lie groups and manifolds (mainly the sphere S2 and rotation group SO3).

*homepage*: <https://github.com/AMLab-Amsterdam/lie_learn>

version        |versionsuffix    |toolchain
---------------|-----------------|------------------
``0.0.1.post1``|``-Python-3.7.4``|``fosscuda/2019b``

### lifelines

lifelines is a complete survival analysis library, written in pure Python.

*homepage*: <https://lifelines.readthedocs.io/en/latest/>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.22.8``|``-Python-3.7.2``|``fosscuda/2019a``
``0.26.3``|                 |``fosscuda/2020b``
``0.27.4``|                 |``foss/2022a``

### Lighter

Fast and memory-efficient sequencing error corrector

*homepage*: <https://github.com/mourisl/Lighter>

version  |toolchain
---------|------------------------------
``1.1.1``|``foss/2018a``
``1.1.2``|``GCC/11.2.0``, ``foss/2018b``

### liknorm

Moments of the product of an exponential-family likelihood with a Normal distribution.

*homepage*: <https://github.com/limix/liknorm>

version  |toolchain
---------|-----------------
``1.5.2``|``GCCcore/7.3.0``

### likwid

Likwid stands for Like I knew what I am doing. This project contributes simple to use command line tools for Linux to support programmers in developing high performance multi threaded programs. 

*homepage*: <https://github.com/RRZE-HPC/likwid>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------
``4.0.1``|``GNU/4.9.3-2.25``
``4.1.0``|``GCCcore/4.9.3``
``4.2.0``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2017a``, ``intel/2017a``
``4.3.2``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``5.0.1``|``GCCcore/8.3.0``
``5.1.0``|``GCCcore/9.3.0``
``5.2.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``
``5.2.1``|``GCC/11.2.0``
``5.2.2``|``GCC/11.3.0``

### limix

Limix is a flexible and efficient linear mixed model library with interfaces to Python.  Genomic analyses require flexible models that can be adapted to the needs of the user.  Limix is smart about how particular models are fitted to save computational cost. 

*homepage*: <https://github.com/limix/limix>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.4``|``-Python-3.6.6``|``foss/2018b``

### LinBox

C++ library for exact, high-performance linear algebra

*homepage*: <http://linalg.org/>

version  |toolchain
---------|--------------
``1.4.0``|``foss/2016a``

### line_profiler

line_profiler is a module for doing line-by-line profiling of functions. kernprof is a convenient script for running either line_profiler or the Python standard library's cProfile or profile modules, depending on what is available.

*homepage*: <https://github.com/pyutils/line_profiler>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.0``|``-Python-3.8.2``|``foss/2020a``
``3.5.1``|                 |``foss/2021b``
``4.0.0``|                 |``foss/2022a``

### Lingeling

One of the design principles of the state-of-the-art SAT solver Lingeling is to use as compact data structures as possible. These reduce memory usage, increase cache efficiency and thus improve runtime, particularly, when using multiple solver instances on multi-core machines, as in our parallel portfolio solver Plingeling and our cube and conquer solver Treengeling.

*homepage*: <http://fmv.jku.at/lingeling/>

version|toolchain
-------|-------------
``bcp``|``GCC/9.3.0``

### LISFLOOD-FP

The LISFLOOD-FP is a raster-based hydrodynamic model originally developed by the University of Bristol. It has undergone extensive development since conception and includes a collection of numerical schemes implemented to solve a variety of mathematical approximations of the 2D shallow water equations of different complexity. The local inertia solver, known as the ACC solver, is widely used to simulate floods with gradually-varying, subcritical flow over sufficiently rough surfaces with Manning’s coefficient of at least 0.03. It has a version with CPU-specific optimisations and enhanced with a subgrid channel model. LISFLOOD-FP also includes second-order discontinuous Galerkin (DG2) and first-order finite volume (FV1) solvers of the full shallow water equations for modelling a wide range of flows, including rapidly-propagating, supercritical flows, shock waves, or flows over very smooth surfaces. The DG2/FV1 solvers are parallelised for the multi-core CPU architecture, but do not integrate with the subgrid channel model nor with the CPU-specific optimisations. 

*homepage*: <https://www.seamlesswave.com/LISFLOOD8.0.html>

version|versionsuffix   |toolchain
-------|----------------|---------------
``8.1``|                |``gompi/2022a``
``8.1``|``-CUDA-11.7.0``|``gompi/2022a``

### LittleCMS

Little CMS intends to be an OPEN SOURCE small-footprint color management engine,  with special focus on accuracy and performance. 

*homepage*: <http://www.littlecms.com/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------
``2.7``   |``intel/2016a``
``2.8``   |``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``2.9``   |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
``2.11``  |``GCCcore/10.2.0``
``2.12``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.13.1``|``GCCcore/11.3.0``
``2.14``  |``GCCcore/12.2.0``

### LLDB

The debugger component of the LLVM project

*homepage*: <https://lldb.llvm.org/>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``11.0.0``|``-Python-3.8.2``|``GCCcore/9.3.0``

### LLVM

The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  ("LLVM IR"). The LLVM Core libraries are well documented, and it is  particularly simple to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.

*homepage*: <https://llvm.org/>

version   |toolchain
----------|------------------------------------------------------------------------
``3.7.1`` |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``3.8.0`` |``foss/2016a``, ``intel/2016a``
``3.8.1`` |``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``
``3.9.0`` |``foss/2016b``, ``intel/2016b``
``3.9.1`` |``foss/2017a``
``4.0.0`` |``foss/2017a``, ``intel/2017a``
``4.0.1`` |``intel/2017a``, ``intel/2017b``
``5.0.0`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.0.1`` |``GCCcore/6.4.0``
``6.0.0`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``7.0.0`` |``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``7.0.1`` |``GCCcore/8.2.0``
``8.0.1`` |``GCCcore/8.3.0``
``9.0.0`` |``GCCcore/8.3.0``
``9.0.1`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``10.0.0``|``GCCcore/8.3.0``
``10.0.1``|``GCCcore/10.2.0``
``11.0.0``|``GCCcore/10.2.0``
``11.1.0``|``GCCcore/10.3.0``
``12.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``14.0.3``|``GCCcore/11.3.0``
``15.0.5``|``GCCcore/12.2.0``

### LMDB

LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance  of a pure in-memory database while retaining the persistence of standard disk-based databases.

*homepage*: <https://symas.com/lmdb>

version   |toolchain
----------|----------------------------------------------------------
``0.9.18``|``foss/2016a``
``0.9.21``|``GCCcore/6.4.0``, ``intel/2017a``
``0.9.22``|``GCCcore/7.3.0``
``0.9.23``|``GCCcore/8.2.0``
``0.9.24``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.9.28``|``GCCcore/10.3.0``
``0.9.29``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### LMfit

Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python

*homepage*: <https://lmfit.github.io/lmfit-py>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------
``0.9.9`` |``-Python-3.6.4`` |``intel/2018a``
``0.9.14``|``-Python-2.7.15``|``intel/2018b``
``1.0.0`` |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.0.1`` |``-Python-3.8.2`` |``foss/2020a``
``1.0.2`` |                  |``foss/2020b``, ``intel/2020b``
``1.0.3`` |                  |``foss/2021a``, ``foss/2022a``, ``intel/2022a``
``1.2.1`` |                  |``foss/2021b``

### Lmod

Lmod is a Lua based module system. Modules allow for dynamic modification  of a user's environment under Unix systems. See www.tacc.utexas.edu/tacc-projects/lmod  for a complete description. Lmod is a new implementation that easily handles the MODULEPATH  Hierarchical problem. It is drop-in replacement for TCL/C modules and reads TCL modulefiles directly.

*homepage*: <http://sourceforge.net/projects/lmod/>

version  |toolchain
---------|----------------------------
``5.2``  |``GCC/4.8.2``
``5.2.5``|``GCC/4.8.2``
``5.3``  |``GCC/4.8.2``
``5.4``  |``GCC/4.8.2``
``5.4.2``|``GCC/4.8.2``
``5.5``  |``GCC/4.8.2``
``5.5.1``|``GCC/4.8.2``
``5.6``  |``GCC/4.8.2``
``5.7``  |``GCC/4.8.2``
``5.8``  |``GCC/4.8.2``
``5.8.5``|``GCC/4.8.2``
``5.9``  |``GCC/4.8.2``, ``GCC/4.8.4``
``6.4.2``|``system``
``7.3``  |``system``

### LncLOOM

LncLOOM is a graph-based framework that uses integer programming to identify combinations of short motifs that are deeply conserved in rapidly evolving sequences.

*homepage*: <https://github.com/lncLOOM/lncLOOMv2/>

version|toolchain
-------|--------------
``2.0``|``foss/2020b``

### LocARNA

LocARNA is a collection of alignment tools for the structural analysis of RNA.  Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for  your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for  analyzing sets of related RNAs without known common structure.

*homepage*: <http://www.bioinf.uni-freiburg.de/Software/LocARNA/>

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``1.9.2``  |                 |``foss/2016b``
``1.9.2.2``|``-Python-3.6.6``|``foss/2018b``
``1.9.2.3``|                 |``foss/2021b``

### LoFreq

Fast and sensitive variant calling from next-gen sequencing data

*homepage*: <http://csb5.github.io/lofreq>

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------
``2.1.2``  |``-Python-2.7.12``|``intel/2016b``
``2.1.3.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``

### Log-Log4perl

Log4perl

*homepage*: <https://metacpan.org/pod/Log::Log4perl>

version |toolchain
--------|--------------
``1.47``|``foss/2016a``

### logaddexp

C library that implements the logarithm of the sum of exponentiations. Inspired by NumPy's logaddexp function.

*homepage*: <https://github.com/horta/logaddexp>

version  |toolchain
---------|-----------------
``1.0.3``|``GCCcore/7.3.0``

### LOHHLA

LOHHLA, Loss Of Heterozygosity in Human Leukocyte Antigen, a computational tool to evaluate HLA loss using next-generation sequencing data.

*homepage*: <https://bitbucket.org/mcgranahanlab/lohhla>

version       |versionsuffix|toolchain
--------------|-------------|--------------
``2018.11.05``|``-R-3.5.1`` |``foss/2018b``

### Loki

Loki is a C++ library of designs, containing flexible implementations of common design patterns and   idioms. 

*homepage*: <http://loki-lib.sourceforge.net/>

version  |toolchain
---------|---------------------------------------------------------------------------------
``0.1.7``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``

### longestrunsubsequence

Implementation of a solver for the Longest Run Subsequence Problem. Given a  sequence as input, compute a longest subsequence such that there is at most one run for every  character. 

*homepage*: <https://pypi.org/project/longestrunsubsequence/>

version  |toolchain
---------|------------------
``1.0.1``|``GCCcore/10.3.0``

### longread_umi

A collection of scripts for processing longread UMI data.

*homepage*: <https://github.com/SorenKarst/longread_umi>

version  |toolchain
---------|--------------
``0.3.2``|``foss/2020b``

### Longshot

Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific  Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs  a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can  be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).

*homepage*: <https://github.com/pjedge/longshot>

version  |toolchain
---------|-------------------------------------
``0.3.4``|``GCCcore/8.2.0``
``0.4.1``|``GCC/7.3.0-2.30``, ``GCCcore/8.3.0``
``0.4.3``|``GCCcore/10.2.0``
``0.4.5``|``GCCcore/11.3.0``

### loompy

Python implementation of the Loom file format, an efficient file format for large omics datasets

*homepage*: <https://loompy.org/>

version  |toolchain
---------|---------------------------------------------------------------
``3.0.6``|``intel/2020b``
``3.0.7``|``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2021b``

### loomR

An R interface for loom files

*homepage*: <https://github.com/mojaveazure/loomR>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``0.2.0-20180425``|``-R-4.2.1`` |``foss/2022a``

### LoRDEC

Program for correcting sequencing errors in long reads (e.g., PacBio, Oxford Nanopore) using highly accurate short reads (e.g., Illumina).

*homepage*: <https://www.lirmm.fr/~rivals/lordec>

version|toolchain
-------|---------------
``0.9``|``gompi/2022a``

### LPeg

LPeg is a new pattern-matching library for Lua, based on Parsing Expression Grammars (PEGs).

*homepage*: <http://www.inf.puc-rio.br/~roberto/lpeg/>

version  |toolchain
---------|------------------
``1.0.2``|``GCCcore/10.2.0``

### LPJmL

Dynamic global vegetation model with managed land and river routing

*homepage*: <https://github.com/PIK-LPJmL/LPJmL>

version    |toolchain
-----------|---------------
``4.0.003``|``iimpi/2020b``

### lpsolve

Mixed Integer Linear Programming (MILP) solver

*homepage*: <https://sourceforge.net/projects/lpsolve/>

version     |toolchain
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.5.2.5`` |``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``intel/2017a``, ``intel/2018b``
``5.5.2.11``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/9.3.0``

### lrslib

lrslib is a self-contained ANSI C implementation of the reverse search algorithm for vertex enumeration/convex hull problems

*homepage*: <http://cgm.cs.mcgill.ca/~avis/C/lrs.html>

version |toolchain
--------|---------------
``6.2`` |``intel/2018b``
``7.0a``|``gompi/2019a``

### LS-PrePost

LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA.

*homepage*: <http://lstc.com/products/ls-prepost>

version   |versionsuffix|toolchain
----------|-------------|----------
``4.6``   |``-centos6`` |``system``
``4.6``   |``-centos7`` |``system``
``4.6.24``|             |``system``
``4.7.8`` |             |``system``
``4.7.15``|             |``system``

### LSD2

Least-squares methods to estimate rates and dates from phylogenies

*homepage*: <https://github.com/tothuhien/lsd2>

version  |toolchain
---------|----------------------------------------------------------
``1.9.7``|``GCCcore/9.3.0``
``2.2``  |``GCCcore/10.2.0``
``2.3``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.4.1``|``GCCcore/12.2.0``

### LSMS

LSMS benchmark, part of CORAL suite

*homepage*: <https://asc.llnl.gov/CORAL-benchmarks/#lsms>

version     |toolchain
------------|--------------
``3_rev237``|``foss/2016a``

### LTR_retriever

LTR_retriever is a highly accurate and sensitive program for  identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also  included in this package. 

*homepage*: <https://github.com/oushujun/LTR_retriever>

version  |toolchain
---------|------------------------------
``2.9.0``|``foss/2020b``, ``foss/2022a``

### LtrDetector

A modern tool-suite for detectinglong terminal repeat retrotransposons de-novo onthe genomic scale

*homepage*: <https://github.com/TulsaBioinformaticsToolsmith/LtrDetector>

version|versionsuffix    |toolchain
-------|-----------------|---------------
``1.0``|``-Python-3.7.4``|``intel/2019b``

### Lua

Lua is a powerful, fast, lightweight, embeddable scripting language.  Lua combines simple procedural syntax with powerful data description constructs based  on associative arrays and extensible semantics. Lua is dynamically typed,  runs by interpreting bytecode for a register-based virtual machine,  and has automatic memory management with incremental garbage collection,  making it ideal for configuration, scripting, and rapid prototyping.

*homepage*: <https://www.lua.org/>

version    |toolchain
-----------|---------------------------------------------------------------------------------------
``5.1.4-5``|``GCC/4.8.2``
``5.1.4-8``|``GCC/4.8.2``, ``GCC/4.8.4``, ``system``
``5.1.5``  |``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``5.2.4``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``5.3.4``  |``GCCcore/7.2.0``, ``system``
``5.3.5``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``5.4.2``  |``GCCcore/10.2.0``
``5.4.3``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.4.4``  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### LuaJIT

LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. 

*homepage*: <https://luajit.org/>

version                 |toolchain
------------------------|--------------
``2.0.2``               |``GCC/4.9.2``
``2.1.0-beta3_20230602``|``GCC/11.3.0``

### LuaJIT2-OpenResty

openresty/luajit2 - OpenResty's maintained branch of LuaJIT. LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. 

*homepage*: <https://github.com/openresty/luajit2>

version         |toolchain
----------------|-------------
``2.1-20220411``|``GCC/9.3.0``

### LuaRocks

LuaRocks is the package manager for Lua modules.  It allows you to create and install Lua modules as self-contained packages called rocks. 

*homepage*: <https://luarocks.org>

version  |toolchain
---------|------------------
``3.9.2``|``GCCcore/11.3.0``

### Lucene-Geo-Gazetteer

A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library  to create a searchable gazetteer.

*homepage*: <https://github.com/chrismattmann/lucene-geo-gazetteer>

version     |toolchain
------------|----------
``20170718``|``system``

### LUMPY

A probabilistic framework for structural variant discovery. 

*homepage*: <https://github.com/arq5x/lumpy-sv>

version   |toolchain
----------|--------------
``0.2.13``|``foss/2016b``
``0.3.1`` |``foss/2020b``

### LUSCUS

Luscus is the program for graphical display and editing of molecular systems.

*homepage*: <https://sourceforge.net/projects/luscus/>

version  |toolchain
---------|-------------------------------
``0.8.6``|``foss/2018b``, ``intel/2018a``

### lwgrp

The light-weight group library defines data structures and collective operations to group MPI processes as an ordered set.  Such groups are useful as substitutes for MPI communicators when the overhead of communicator creation is too costly.  For example, certain sorting algorithms recursively divide processes into subgroups as the sort algorithm progresses.  These groups may be different with each invocation, so that it is inefficient to create and destroy communicators during the sort routine.

*homepage*: <https://github.com/LLNL/lwgrp>

version  |toolchain
---------|------------------------------------------------------------------
``1.0.2``|``gompi/2019a``, ``gompi/2020a``, ``iimpi/2019a``, ``iimpi/2020a``
``1.0.3``|``gompi/2020b``
``1.0.5``|``gompi/2022a``

### lxml

The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.

*homepage*: <https://lxml.de/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``3.5.0``|``-Python-2.7.11``|``intel/2016a``
``3.6.0``|``-Python-2.7.11``|``intel/2016a``
``3.6.4``|``-Python-2.7.12``|``intel/2016b``
``4.0.0``|``-Python-2.7.13``|``intel/2017a``
``4.1.1``|``-Python-2.7.14``|``intel/2017b``
``4.2.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``4.2.0``|``-Python-3.6.4`` |``intel/2018a``
``4.2.5``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``4.2.5``|``-Python-3.6.6`` |``foss/2018b``
``4.3.3``|                  |``GCCcore/8.2.0``
``4.4.2``|                  |``GCCcore/8.3.0``
``4.5.2``|                  |``GCCcore/9.3.0``
``4.6.2``|                  |``GCCcore/10.2.0``
``4.6.3``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``4.9.1``|                  |``GCCcore/11.3.0``
``4.9.2``|                  |``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### lynx

lynx is an alphanumeric display oriented World-Wide Web Client

*homepage*: <http://lynx.browser.org/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.8.9``|``-develop`` |``foss/2016b``

### lz4

LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core.

*homepage*: <https://lz4.github.io/lz4/>

version  |toolchain
---------|----------------------------------------------------------
``1.8.2``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``
``1.9.0``|``GCCcore/7.3.0``
``1.9.1``|``GCCcore/8.2.0``
``1.9.2``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.9.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.9.4``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### LZO

Portable lossless data compression library

*homepage*: <http://www.oberhumer.com/opensource/lzo/>

version |toolchain
--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.09``|``intel/2016b``, ``intel/2017b``
``2.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017a``

## M


[M1QN3](#m1qn3) - [M3GNet](#m3gnet) - [M4](#m4) - [MACH](#mach) - [MACS2](#macs2) - [MACSE](#macse) - [maeparser](#maeparser) - [MAFFT](#mafft) - [MAGeCK](#mageck) - [magick](#magick) - [Magics](#magics) - [magma](#magma) - [MAGMA-gene-analysis](#magma-gene-analysis) - [MagresPython](#magrespython) - [mahotas](#mahotas) - [MAJIQ](#majiq) - [make](#make) - [makedepend](#makedepend) - [makedepf90](#makedepf90) - [makeinfo](#makeinfo) - [MAKER](#maker) - [Mako](#mako) - [Mamba](#mamba) - [mandrake](#mandrake) - [mannkendall](#mannkendall) - [manta](#manta) - [mapDamage](#mapdamage) - [Maple](#maple) - [MapSplice](#mapsplice) - [Maq](#maq) - [MariaDB](#mariadb) - [MariaDB-connector-c](#mariadb-connector-c) - [MARS](#mars) - [Mash](#mash) - [MaSuRCA](#masurca) - [Mathematica](#mathematica) - [MathGL](#mathgl) - [MATIO](#matio) - [MATLAB](#matlab) - [MATLAB-Engine](#matlab-engine) - [matlab-proxy](#matlab-proxy) - [matplotlib](#matplotlib) - [matplotlib-inline](#matplotlib-inline) - [MATSim](#matsim) - [Maude](#maude) - [mauveAligner](#mauvealigner) - [Maven](#maven) - [mawk](#mawk) - [MaxBin](#maxbin) - [MaxQuant](#maxquant) - [mayavi](#mayavi) - [maze](#maze) - [MbedTLS](#mbedtls) - [MBROLA](#mbrola) - [mbuffer](#mbuffer) - [mc](#mc) - [MCL](#mcl) - [MCR](#mcr) - [mctc-lib](#mctc-lib) - [mcu](#mcu) - [MDAnalysis](#mdanalysis) - [MDBM](#mdbm) - [MDSplus](#mdsplus) - [MDSplus-Java](#mdsplus-java) - [MDSplus-Python](#mdsplus-python) - [mdtest](#mdtest) - [MDTraj](#mdtraj) - [medaka](#medaka) - [medImgProc](#medimgproc) - [MedPy](#medpy) - [Meep](#meep) - [MEGA](#mega) - [MEGACC](#megacc) - [MEGAHIT](#megahit) - [Megalodon](#megalodon) - [MEGAN](#megan) - [Meld](#meld) - [MEM](#mem) - [MEME](#meme) - [memkind](#memkind) - [memory-profiler](#memory-profiler) - [MEMOTE](#memote) - [memtester](#memtester) - [meRanTK](#merantk) - [MERCKX](#merckx) - [Mercurial](#mercurial) - [Mesa](#mesa) - [Mesa-demos](#mesa-demos) - [meshalyzer](#meshalyzer) - [meshio](#meshio) - [meshtool](#meshtool) - [Meson](#meson) - [meson-python](#meson-python) - [Mesquite](#mesquite) - [MESS](#mess) - [MetaBAT](#metabat) - [MetaboAnalystR](#metaboanalystr) - [metaerg](#metaerg) - [MetaEuk](#metaeuk) - [MetaGeneAnnotator](#metageneannotator) - [Metagenome-Atlas](#metagenome-atlas) - [Metal](#metal) - [MetaMorpheus](#metamorpheus) - [MetaPhlAn](#metaphlan) - [MetaPhlAn2](#metaphlan2) - [metaWRAP](#metawrap) - [Metaxa2](#metaxa2) - [methylartist](#methylartist) - [MethylDackel](#methyldackel) - [methylpy](#methylpy) - [METIS](#metis) - [mgltools](#mgltools) - [mhcflurry](#mhcflurry) - [mhcnuggets](#mhcnuggets) - [MicrobeAnnotator](#microbeannotator) - [microctools](#microctools) - [MiGEC](#migec) - [MIGRATE-N](#migrate-n) - [Miller](#miller) - [mimalloc](#mimalloc) - [MINC](#minc) - [MinCED](#minced) - [Mini-XML](#mini-xml) - [miniasm](#miniasm) - [minibar](#minibar) - [MiniCARD](#minicard) - [Miniconda2](#miniconda2) - [Miniconda3](#miniconda3) - [minieigen](#minieigen) - [Minimac4](#minimac4) - [minimap2](#minimap2) - [Minipolish](#minipolish) - [MiniSat](#minisat) - [minizip](#minizip) - [MINPACK](#minpack) - [MinPath](#minpath) - [MIRA](#mira) - [miRDeep2](#mirdeep2) - [Mish-Cuda](#mish-cuda) - [misha](#misha) - [MITgcmutils](#mitgcmutils) - [MITObim](#mitobim) - [MitoZ](#mitoz) - [MiXCR](#mixcr) - [MixMHC2pred](#mixmhc2pred) - [mkl-dnn](#mkl-dnn) - [mkl-service](#mkl-service) - [mkl_fft](#mkl_fft) - [MLC](#mlc) - [MLxtend](#mlxtend) - [mm-common](#mm-common) - [Mmg](#mmg) - [MMSEQ](#mmseq) - [MMseqs2](#mmseqs2) - [mmtf-cpp](#mmtf-cpp) - [MNE-Python](#mne-python) - [MOABB](#moabb) - [MOABS](#moabs) - [MOB-suite](#mob-suite) - [ModelTest-NG](#modeltest-ng) - [modred](#modred) - [Molcas](#molcas) - [mold](#mold) - [Molden](#molden) - [molecularGSM](#moleculargsm) - [Molekel](#molekel) - [molmod](#molmod) - [Molpro](#molpro) - [MONA](#mona) - [MONAI](#monai) - [MONAI-Label](#monai-label) - [mongolite](#mongolite) - [Mono](#mono) - [Monocle3](#monocle3) - [moonjit](#moonjit) - [MOOSE](#moose) - [mordecai](#mordecai) - [MoreRONN](#moreronn) - [mosdepth](#mosdepth) - [Mothur](#mothur) - [motif](#motif) - [MotionCor2](#motioncor2) - [motionSegmentation](#motionsegmentation) - [MoviePy](#moviepy) - [MPB](#mpb) - [MPC](#mpc) - [MPFR](#mpfr) - [mpi4py](#mpi4py) - [MPICH](#mpich) - [MPICH2](#mpich2) - [mpifileutils](#mpifileutils) - [mpiP](#mpip) - [MPJ-Express](#mpj-express) - [mpmath](#mpmath) - [MrBayes](#mrbayes) - [mrcfile](#mrcfile) - [MRChem](#mrchem) - [MRCPP](#mrcpp) - [MRIcron](#mricron) - [MRPRESSO](#mrpresso) - [MRtrix](#mrtrix) - [msgpack-c](#msgpack-c) - [MSM](#msm) - [MSPC](#mspc) - [msprime](#msprime) - [mstore](#mstore) - [MTL4](#mtl4) - [MuJoCo](#mujoco) - [mujoco-py](#mujoco-py) - [multichoose](#multichoose) - [MultilevelEstimators](#multilevelestimators) - [MultiNest](#multinest) - [MultiQC](#multiqc) - [Multiwfn](#multiwfn) - [muMerge](#mumerge) - [MUMmer](#mummer) - [MUMPS](#mumps) - [muParser](#muparser) - [muparserx](#muparserx) - [MuPeXI](#mupexi) - [MUSCLE](#muscle) - [MuSiC](#music) - [MUST](#must) - [MuTect](#mutect) - [mutil](#mutil) - [MVAPICH2](#mvapich2) - [MView](#mview) - [mxml](#mxml) - [mxmlplus](#mxmlplus) - [MXNet](#mxnet) - [MyCC](#mycc) - [mygene](#mygene) - [MyMediaLite](#mymedialite) - [mympingpong](#mympingpong) - [Myokit](#myokit) - [mypy](#mypy) - [MySQL](#mysql) - [MySQL-python](#mysql-python) - [mysqlclient](#mysqlclient)


### M1QN3

A solver of large-scale unconstrained minimization problems

*homepage*: <https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html>

version|toolchain
-------|--------------
``3.3``|``GCC/10.3.0``

### M3GNet

" M3GNet is a new materials graph neural network architecture that incorporates 3-body interactions. A key difference with prior materials graph implementations such as MEGNet is the addition of the coordinates for atoms and the 3×3 lattice matrix in crystals, which are necessary for obtaining tensorial quantities such as forces and stresses via auto-differentiation. 

*homepage*: <https://materialsvirtuallab.github.io/m3gnet/>

version  |toolchain
---------|--------------
``0.2.4``|``foss/2022a``

### M4

GNU M4 is an implementation of the traditional Unix macro processor. It is  mostly SVR4 compatible although it has some extensions (for example, handling  more than 9 positional parameters to macros). GNU M4 also has built-in  functions for including files, running shell commands, doing arithmetic, etc. 

*homepage*: <https://www.gnu.org/software/m4/m4.html>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.4.16``|``GCC/4.8.1``, ``GCC/4.8.2``, ``system``
``1.4.17``|``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.4.18``|``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/system``, ``system``
``1.4.19``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/11.4.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``GCCcore/9.4.0``, ``GCCcore/9.5.0``, ``system``

### MACH

MACH 1.0 is a Markov Chain based haplotyper  that can resolve long haplotypes or infer missing genotypes  in samples of unrelated individuals.

*homepage*: <http://csg.sph.umich.edu/abecasis/MACH/>

version   |toolchain
----------|----------
``1.0.18``|``system``

### MACS2

Model Based Analysis for ChIP-Seq data

*homepage*: <https://github.com/taoliu/MACS>

version           |versionsuffix     |toolchain
------------------|------------------|-------------------------------
``2.1.1.20160309``|``-Python-2.7.14``|``intel/2017b``
``2.1.2.1``       |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.1.2.1``       |``-Python-2.7.15``|``intel/2019a``
``2.2.5``         |``-Python-3.6.6`` |``foss/2018b``
``2.2.7.1``       |                  |``foss/2021b``

### MACSE

MACSE aligns coding NT sequences with respect to their AA translation while allowing NT sequences to contain multiple frameshifts and/or stop codons. MACSE is hence the first automatic solution to align protein-coding gene datasets containing non-functional sequences (pseudogenes) without disrupting the underlying codon structure.

*homepage*: <https://bioweb.supagro.inra.fr/macse/index.php>

version |versionsuffix|toolchain
--------|-------------|----------
``2.06``|``-Java-15`` |``system``

### maeparser

maeparser is a parser for Schrodinger Maestro files.

*homepage*: <https://github.com/schrodinger/maeparser>

version  |toolchain
---------|------------------------------------------------------------------
``1.2.2``|``gompi/2019a``, ``iimpi/2019a``
``1.3.0``|``gompi/2019b``, ``gompi/2021a``, ``gompi/2022a``, ``iimpi/2020a``

### MAFFT

MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.

*homepage*: <https://mafft.cbrc.jp/alignment/software/source.html>

version  |versionsuffix       |toolchain
---------|--------------------|---------------------------------------------------------------------------------------
``7.305``|``-with-extensions``|``foss/2016b``
``7.397``|``-with-extensions``|``intel/2018a``
``7.427``|``-with-extensions``|``foss/2018b``, ``intel/2018b``
``7.429``|``-with-extensions``|``GCC/8.2.0-2.31.1``
``7.453``|``-with-extensions``|``GCC/8.3.0``, ``GCC/9.3.0``, ``gompi/2020a``, ``iccifort/2019.5.281``, ``iimpi/2020a``
``7.470``|``-with-extensions``|``GCC/9.3.0``, ``gompi/2020a``
``7.471``|``-with-extensions``|``iimpi/2020a``
``7.475``|``-with-extensions``|``GCC/10.2.0``, ``gompi/2020b``
``7.487``|``-with-extensions``|``gompi/2021a``
``7.490``|``-with-extensions``|``GCC/10.3.0``, ``GCC/11.2.0``, ``gompi/2021b``
``7.505``|``-with-extensions``|``GCC/11.3.0``, ``GCC/12.2.0``

### MAGeCK

Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify  important genes from the recent genome-scale CRISPR-Cas9 knockout screens (or GeCKO) technology. MAGeCK is developed  by Wei Li and Han Xu from Dr. Xiaole Shirley Liu's lab at Dana-Farber Cancer Institute, and is being actively updated  by Wei Li lab from Children's National Medical Center.

*homepage*: <https://sourceforge.net/p/mageck/wiki/Home/>

version    |toolchain
-----------|------------------------------
``0.5.9.4``|``foss/2021a``, ``foss/2022a``
``0.5.9.5``|``gfbf/2022b``

### magick

R bindings to the open-source image processing library ImageMagick

*homepage*: <https://cran.r-project.org/web/packages/magick/>

version|versionsuffix|toolchain
-------|-------------|--------------
``2.0``|``-R-3.5.1`` |``foss/2018b``

### Magics

Magics is ECMWF's meteorological plotting software and can be either accessed directly through its Python or Fortran interfaces or by using Metview.

*homepage*: <https://confluence.ecmwf.int/display/MAGP/>

version   |toolchain
----------|---------------
``4.13.0``|``gompi/2022a``

### magma

The MAGMA project aims to develop a dense linear algebra library similar to  LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.

*homepage*: <https://icl.cs.utk.edu/magma/>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------
``2.3.0``|                |``fosscuda/2017b``, ``intelcuda/2017b``
``2.3.0``|``-CUDA-9.1.85``|``foss/2018a``
``2.4.0``|                |``fosscuda/2018b``
``2.5.0``|                |``fosscuda/2018b``
``2.5.1``|                |``fosscuda/2019a``, ``fosscuda/2019b``
``2.5.4``|                |``fosscuda/2019a``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``fosscuda/2020b``, ``intelcuda/2019b``, ``intelcuda/2020b``
``2.6.1``|``-CUDA-11.3.1``|``foss/2021a``
``2.6.2``|``-CUDA-11.4.1``|``foss/2021b``
``2.6.2``|``-CUDA-11.5.2``|``foss/2021b``
``2.6.2``|``-CUDA-11.7.0``|``foss/2022a``

### MAGMA-gene-analysis

MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous  GWAS or meta-analysis.

*homepage*: <https://ctg.cncr.nl/software/magma>

version   |toolchain
----------|--------------
``1.07b`` |``foss/2018b``
``1.07bb``|``GCC/8.3.0``
``1.09b`` |``GCC/11.2.0``

### MagresPython

MagresPython is a Python library for parsing the CCP-NC ab-initio magnetic resonance file format.  This is used in the latest version of the CASTEP and Quantum ESPRESSO (PWSCF) codes. 

*homepage*: <http://tfgg.me/magres-python>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160329``|``-Python-2.7.15``|``foss/2018b``

### mahotas

Mahotas is a computer vision and image processing library for Python

*homepage*: <https://mahotas.readthedocs.io/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.3``|``-Python-2.7.12``|``intel/2016b``

### MAJIQ

MAJIQ and Voila are two software packages that together detect, quantify, and visualize  local splicing variations (LSV) from RNA-Seq data.

*homepage*: <https://majiq.biociphers.org>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.6.4``|``intel/2018a``

### make

GNU version of make utility

*homepage*: <https://www.gnu.org/software/make/make.html>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------
``3.82`` |``GCC/4.8.2``
``4.1``  |``GCC/4.9.2``
``4.2.1``|``GCC/7.3.0-2.30``, ``GCCcore/8.3.0``
``4.3``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/9.3.0``
``4.4.1``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### makedepend

The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.

*homepage*: <https://linux.die.net/man/1/makedepend>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------
``1.0.5``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.0.7``|``GCCcore/11.3.0``

### makedepf90

Makedepf90 is a program for automatic creation of  Makefile-style dependency lists for Fortran source code. 

*homepage*: <https://linux.die.net/man/1/makedepf90>

version  |toolchain
---------|--------------
``2.8.8``|``foss/2017a``

### makeinfo

makeinfo is part of the Texinfo project, the official documentation format of the GNU project.

*homepage*: <https://www.gnu.org/software/texinfo/>

version|versionsuffix|toolchain
-------|-------------|-------------------------------------------------------------------------------------------
``6.7``|             |``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.7``|``-minimal`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.8``|             |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### MAKER

MAKER is a portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases. 

*homepage*: <https://yandell-lab.org/software/maker.html>

version    |toolchain
-----------|--------------
``3.01.04``|``foss/2022a``

### Mako

A super-fast templating language that borrows the best ideas from the existing templating languages

*homepage*: <https://www.makotemplates.org>

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.4``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.0.6``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.0.7``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``1.0.7``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.0.7``|``-Python-3.6.4`` |``intel/2018a``
``1.0.7``|``-Python-3.6.6`` |``fosscuda/2018b``
``1.0.8``|                  |``GCCcore/8.2.0``
``1.1.0``|                  |``GCCcore/8.3.0``
``1.1.2``|                  |``GCCcore/9.3.0``
``1.1.3``|                  |``GCCcore/10.2.0``
``1.1.4``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.0``|                  |``GCCcore/11.3.0``
``1.2.4``|                  |``GCCcore/12.2.0``

### Mamba

Mamba is a fast, robust, and cross-platform package manager. It runs on Windows, OS X and Linux  (ARM64 and PPC64LE included) and is fully compatible with conda packages and supports most of conda's commands. 

*homepage*: <https://mamba.readthedocs.io/>

version     |toolchain
------------|----------
``4.14.0-0``|``system``

### mandrake

Fast visualisation of the population structure of pathogens  using Stochastic Cluster Embedding.

*homepage*: <https://mandrake.readthedocs.io>

version  |toolchain
---------|--------------
``1.2.2``|``foss/2022a``

### mannkendall

A python package for non parametric Mann Kendall family of trend tests.

*homepage*: <https://github.com/mmhs013/pymannkendall>

version  |toolchain
---------|--------------
``1.1.1``|``foss/2022a``

### manta

Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. It is optimized for analysis of germline variation in small sets of individuals and  somatic variation in tumor/normal sample pairs. Manta discovers, assembles and  scores large-scale SVs, medium-sized indels and large insertions within a  single efficient workflow.  

*homepage*: <https://github.com/Illumina/manta>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.1``|                  |``system``
``1.6.0``|                  |``system``
``1.6.0``|``-Python-2.7.16``|``gompi/2019b``
``1.6.0``|``-Python-2.7.18``|``GCC/10.2.0``, ``gompi/2020a``

### mapDamage

mapDamage2 is a computational framework written in Python and     R, which tracks and quantifies DNA damage patterns among ancient DNA     sequencing reads generated by Next-Generation Sequencing platforms.

*homepage*: <https://ginolhac.github.io/mapDamage/>

version  |versionsuffix|toolchain
---------|-------------|------------------------------
``2.2.1``|             |``foss/2021b``, ``foss/2022a``
``2.2.1``|``-R-4.0.4`` |``foss/2020b``
``2.2.1``|``-R-4.1.0`` |``foss/2021a``

### Maple

Maple combines the world's most powerful mathematical computation engine with an intuitive,    'clickable' user interface.

*homepage*: <http://www.maplesoft.com/products/maple/>

version   |toolchain
----------|----------
``15``    |``system``
``2017.2``|``system``
``2022.1``|``system``

### MapSplice

MapSplice is a software for mapping RNA-seq data to reference genome for splice  junction discovery that depends only on reference genome, and not on any further annotations.

*homepage*: <http://www.netlab.uky.edu/p/bioinfo/MapSplice2>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.2.1``|``-Python-2.7.12``|``foss/2016b``
``2.2.1``|``-Python-2.7.15``|``intel/2018b``

### Maq

Maq is a software that builds mapping assemblies from short reads generated by the  next-generation sequencing  machines.

*homepage*: <http://maq.sourceforge.net/maq-man.shtml>

version  |toolchain
---------|----------
``0.7.0``|``system``

### MariaDB

MariaDB is an enhanced, drop-in replacement for MySQL. Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.

*homepage*: <https://mariadb.org/>

version    |toolchain
-----------|-------------------------------
``10.0.21``|``GNU/4.9.3-2.25``
``10.1.13``|``intel/2016a``
``10.1.14``|``foss/2016a``, ``intel/2016a``
``10.1.17``|``intel/2016b``
``10.1.24``|``intel/2017a``
``10.2.11``|``foss/2017b``, ``intel/2017b``
``10.3.7`` |``intel/2018a``
``10.3.10``|``foss/2018b``
``10.3.14``|``foss/2019a``
``10.4.13``|``gompi/2019b``
``10.5.8`` |``GCC/10.2.0``
``10.6.4`` |``GCC/10.3.0``, ``GCC/11.2.0``
``10.9.3`` |``GCC/11.3.0``
``10.11.2``|``GCC/12.2.0``

### MariaDB-connector-c

MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.

*homepage*: <https://downloads.mariadb.org/connector-c/>

version   |toolchain
----------|----------------------------------------------------
``2.3.4`` |``foss/2017b``, ``intel/2017b``
``2.3.5`` |``intel/2018a``, ``iomkl/2018a``
``2.3.7`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``3.1.7`` |``GCCcore/9.3.0``
``3.1.11``|``GCCcore/10.2.0``
``3.2.2`` |``GCCcore/10.3.0``

### MARS

improving Multiple circular sequence Alignment using Refined Sequences

*homepage*: <https://github.com/lorrainea/MARS>

version     |toolchain
------------|-----------------
``20191101``|``GCCcore/8.3.0``

### Mash

Fast genome and metagenome distance estimation using MinHash

*homepage*: <http://mash.readthedocs.org>

version|toolchain
-------|----------------------------------------------------------------------------
``2.0``|``foss/2018a``
``2.1``|``foss/2018b``
``2.2``|``GCC/9.3.0``
``2.3``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``intel-compilers/2021.4.0``

### MaSuRCA

MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing  only short reads from Illumina sequencing or a mixture of short reads and long reads  (Sanger, 454, Pacbio and Nanopore).

*homepage*: <https://www.genome.umd.edu/masurca.html>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.2.2``|                |``foss/2016a``
``3.2.2``|``-Perl-5.26.0``|``foss/2017b``
``3.2.4``|                |``foss/2018a``
``3.2.5``|``-Perl-5.26.0``|``foss/2017b``
``3.3.1``|``-Perl-5.28.0``|``foss/2018b``
``4.0.1``|``-Perl-5.30.2``|``foss/2020a``
``4.0.9``|``-Perl-5.32.1``|``foss/2021a``
``4.1.0``|                |``GCC/11.3.0``

### Mathematica

Mathematica is a computational software program used in many scientific, engineering, mathematical and computing fields.

*homepage*: <http://www.wolfram.com/mathematica>

version   |toolchain
----------|----------
``9.0.1`` |``system``
``10.0.2``|``system``
``10.1.0``|``system``
``10.4.1``|``system``
``11.0.1``|``system``
``11.1.1``|``system``
``11.3.0``|``system``
``12.0.0``|``system``
``12.1.1``|``system``
``13.0.0``|``system``
``13.1.0``|``system``

### MathGL

MathGL is ...  a library for making high-quality scientific graphics under Linux and Windows;  a library for the fast data plotting and data processing of large data arrays;  a library for working in window and console modes and for simple embedding into other programs;  a library with large and growing set of graphics. 

*homepage*: <http://mathgl.sourceforge.net/doc_en/Main.html>

version  |toolchain
---------|--------------
``2.4.1``|``foss/2018a``

### MATIO

matio is an C library for reading and writing Matlab MAT files.

*homepage*: <http://sourceforge.net/projects/matio/>

version   |toolchain
----------|------------------
``1.5.9`` |``GCCcore/5.4.0``
``1.5.11``|``foss/2017b``
``1.5.12``|``GCCcore/6.4.0``
``1.5.17``|``GCCcore/8.3.0``
``1.5.19``|``GCCcore/9.3.0``
``1.5.21``|``GCCcore/10.2.0``
``1.5.23``|``GCCcore/11.3.0``

### MATLAB

MATLAB is a high-level language and interactive environment  that enables you to perform computationally intensive tasks faster than with  traditional programming languages such as C, C++, and Fortran.

*homepage*: <https://www.mathworks.com/products/matlab>

version  |versionsuffix|toolchain
---------|-------------|----------
``2012b``|             |``system``
``2013b``|             |``system``
``2015a``|             |``system``
``2016a``|             |``system``
``2017a``|             |``system``
``2018b``|             |``system``
``2019b``|             |``system``
``2020a``|             |``system``
``2020b``|             |``system``
``2021a``|             |``system``
``2021b``|             |``system``
``2022a``|             |``system``
``2022a``|``-r3``      |``system``
``2022b``|             |``system``
``2022b``|``-r5``      |``system``
``2023a``|             |``system``

### MATLAB-Engine

The MATLAB Engine API for Python provides a package for Python  to call MATLAB as a computational engine.

*homepage*: <https://www.mathworks.com/help/matlab/matlab-engine-for-python.html>

version          |versionsuffix     |toolchain
-----------------|------------------|-------------------------------
``2018b``        |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2018b``        |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2019b``        |                  |``GCCcore/8.3.0``
``2021a-9.10.1`` |                  |``GCCcore/10.2.0``
``2021b-9.11.19``|                  |``GCCcore/11.2.0``

### matlab-proxy

A Python package which enables you to launch MATLAB and access it from a web browser.

*homepage*: <https://github.com/mathworks/matlab-proxy>

version  |toolchain
---------|------------------
``0.5.4``|``GCCcore/11.3.0``

### matplotlib

matplotlib is a python 2D plotting library which produces publication quality figures in a variety of  hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python  and ipython shell, web application servers, and six graphical user interface toolkits.

*homepage*: <https://matplotlib.org>

version  |versionsuffix                    |toolchain
---------|---------------------------------|-----------------------------------------------------------------------------------
``1.4.3``|``-Python-2.7.12``               |``intel/2016b``
``1.5.1``|``-Python-2.7.11``               |``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-2.7.11-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-2.7.12``               |``intel/2016b``
``1.5.1``|``-Python-3.5.1``                |``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-3.5.2``                |``intel/2016b``
``1.5.2``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.5.2``|``-Python-3.5.2``                |``intel/2016b``
``1.5.3``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.5.3``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``2.0.0``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``2.0.0``|``-Python-2.7.13``               |``intel/2017a``
``2.0.1``|``-Python-3.6.1``                |``intel/2017a``
``2.0.2``|``-Python-2.7.13``               |``foss/2017a``, ``intel/2017a``
``2.0.2``|``-Python-2.7.13-Qt-4.8.7``      |``intel/2017a``
``2.0.2``|``-Python-2.7.13-libpng-1.6.29`` |``intel/2017a``
``2.0.2``|``-Python-3.6.1``                |``foss/2017a``
``2.0.2``|``-Python-3.6.1-libpng-1.6.29``  |``intel/2017a``
``2.1.0``|``-Python-2.7.14``               |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.0``|``-Python-3.6.2``                |``foss/2017b``
``2.1.0``|``-Python-3.6.3``                |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.1``|``-Python-2.7.14``               |``intel/2017b``
``2.1.1``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``2.1.2``|``-Python-2.7.14``               |``foss/2018a``, ``intel/2018a``
``2.1.2``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``2.2.3``|``-Python-2.7.15``               |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.2.3``|``-Python-3.6.6``                |``intel/2018b``
``2.2.4``|``-Python-2.7.15``               |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2.2.4``|``-Python-2.7.16``               |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.2.5``|``-Python-2.7.16``               |``foss/2019b``
``2.2.5``|``-Python-2.7.18``               |``foss/2020a``, ``foss/2020b``, ``foss/2021b``, ``fosscuda/2020a``, ``intel/2020a``
``3.0.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.0.2``|``-Python-3.6.6``                |``foss/2018b``
``3.0.3``|``-Python-3.7.2``                |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``3.1.1``|``-Python-3.7.4``                |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``3.2.1``|``-Python-3.8.2``                |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``3.3.3``|                                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``3.4.2``|                                 |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``3.4.3``|                                 |``foss/2021b``, ``intel/2021b``
``3.5.1``|                                 |``foss/2020b``, ``intel/2020b``
``3.5.2``|                                 |``foss/2021b``, ``foss/2022a``, ``intel/2022a``
``3.7.0``|                                 |``gfbf/2022b``

### matplotlib-inline

Matplotlib Inline Back-end for IPython and Jupyter.

*homepage*: <https://github.com/ipython/matplotlib-inline>

version  |toolchain
---------|------------------
``0.1.3``|``GCCcore/10.2.0``

### MATSim

MATSim is an open-source framework to implement large-scale agent-based transport simulations.

*homepage*: <https://www.matsim.org/>

version  |versionsuffix      |toolchain
---------|-------------------|------------------------------
``0.8.1``|``-Java-1.8.0_112``|``intel/2016b``
``14.0`` |``-Java-11``       |``GCCcore/11.2.0``, ``system``

### Maude

Maude is a high-performance reflective language and system supporting both equational and rewriting  logic specification and programming for a wide range of applications. Maude has been influenced in important ways by  the OBJ3 language, which can be regarded as an equational logic sublanguage. Besides supporting equational  specification and programming, Maude also supports rewriting logic computation. 

*homepage*: <https://maude.cs.illinois.edu/w/index.php/The_Maude_System>

version|toolchain
-------|------------------
``3.1``|``GCCcore/10.2.0``

### mauveAligner

Mauve is a system for constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. Multiple genome alignments provide a basis for research into comparative genomics and the study of genome-wide evolutionary dynamics. This version was built without Graphical User Interface. 

*homepage*: <https://darlinglab.org/mauve/mauve.html>

version |toolchain
--------|---------------
``4736``|``gompi/2020a``

### Maven

Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. 

*homepage*: <https://maven.apache.org/index.html>

version  |toolchain
---------|----------
``3.2.3``|``system``
``3.3.3``|``system``
``3.3.9``|``system``
``3.5.0``|``system``
``3.5.2``|``system``
``3.6.0``|``system``
``3.6.3``|``system``

### mawk

mawk is an interpreter for the AWK Programming Language.

*homepage*: <https://invisible-island.net/mawk/>

version           |toolchain
------------------|-------------------------------------------------------------------------------------------------
``1.3.4-20141206``|``GCC/4.9.2``
``1.3.4-20171017``|``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2018a``, ``intel/2018b``, ``intel/2019a``

### MaxBin

MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.

*homepage*: <https://sourceforge.net/projects/maxbin/>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------
``2.2.6``|``-Perl-5.28.0``|``foss/2018b``
``2.2.7``|                |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``
``2.2.7``|``-Perl-5.28.1``|``GCC/8.2.0-2.31.1``

### MaxQuant

MaxQuant is a quantitative proteomics software package designed for analyzing large  mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling  techniques as well as label-free quantification are supported.

*homepage*: <https://maxquant.org/maxquant>

version      |toolchain
-------------|------------------
``1.6.10.43``|``foss/2018b``
``2.0.3.0``  |``GCCcore/11.2.0``
``2.2.0.0``  |``GCCcore/11.2.0``
``2.4.2.0``  |``system``

### mayavi

The Mayavi scientific data 3-dimensional visualizer

*homepage*: <http://code.enthought.com/projects/mayavi/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.4.4``|``-Python-2.7.11``|``intel/2016a``
``4.6.2``|``-Python-3.6.6`` |``foss/2018b``
``4.7.1``|``-Python-2.7.15``|``foss/2019a``
``4.7.1``|``-Python-3.7.2`` |``foss/2019a``
``4.7.4``|                  |``foss/2021b``

### maze

In a similar manner to dot plots, maze highlights local sequence similarity between two DNA sequences. In particular, maximal exact substring matches are computed with MUMmer3 and visualised.

*homepage*: <https://github.com/dellytools/maze>

version     |toolchain
------------|--------------
``20170124``|``foss/2020b``

### MbedTLS

An open source, portable, simple to use, readable and flexible SSL library.

*homepage*: <https://tls.mbed.org/>

version   |toolchain
----------|------------------
``2.26.0``|``GCCcore/10.2.0``

### MBROLA

MBROLA is a speech synthesizer based on the concatenation of diphones. It takes a list of phonemes as input, together with prosodic information (duration of phonemes and a piecewise linear description of pitch), and produces speech samples on 16 bits (linear), at the sampling frequency of the diphone database.  MBROLA voices project provides list of MBROLA speech synthesizer voices. It is intended to provide easier collaboration and automatic updates for individual users and packagers. 

*homepage*: <['https://github.com/numediart/MBROLA', 'https://github.com/numediart/MBROLA-voices']>

version|versionsuffix       |toolchain
-------|--------------------|-----------------
``3.3``|``-voices-20200330``|``GCCcore/9.3.0``

### mbuffer

mbuffer is a tool for buffering data streams with a large set of unique features. 

*homepage*: <https://www.maier-komor.de/mbuffer.html>

version     |toolchain
------------|-----------------
``20191016``|``GCCcore/9.3.0``

### mc

mc-4.6.1: User-friendly file manager and visual shell

*homepage*: <https://www.midnight-commander.org/>

version   |toolchain
----------|-------------
``4.8.13``|``GCC/4.9.2``

### MCL

The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for graphs (also known as networks) based on simulation of (stochastic) flow in graphs. 

*homepage*: <https://micans.org/mcl/>

version   |versionsuffix   |toolchain
----------|----------------|---------------------------------------------------------------------------------------------------------------------------------
``02.063``|                |``intel/2016b``
``14.137``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``intel/2016b``
``14.137``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``14.137``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``22.282``|                |``GCCcore/11.3.0``

### MCR

The MATLAB Runtime is a standalone set of shared libraries  that enables the execution of compiled MATLAB applications  or components on computers that do not have MATLAB installed.

*homepage*: <https://www.mathworks.com/products/compiler/mcr/>

version   |versionsuffix|toolchain
----------|-------------|----------
``R2013a``|             |``system``
``R2013b``|             |``system``
``R2014a``|             |``system``
``R2014b``|             |``system``
``R2015a``|             |``system``
``R2015b``|             |``system``
``R2016a``|             |``system``
``R2016b``|             |``system``
``R2018a``|             |``system``
``R2018b``|             |``system``
``R2019a``|             |``system``
``R2019b``|``.8``       |``system``
``R2020a``|``.6``       |``system``
``R2020b``|``.5``       |``system``
``R2021a``|``.0``       |``system``
``R2021a``|``.3``       |``system``
``R2021b``|             |``system``
``R2021b``|``.1``       |``system``
``R2021b``|``.2``       |``system``
``R2022a``|             |``system``
``R2022a``|``.1``       |``system``
``R2022a``|``.5``       |``system``

### mctc-lib

Common tool chain for working with molecular structure data in various applications. This library provides a unified way to perform operations on molecular structure data, like reading and writing to common geometry file formats.

*homepage*: <https://grimme-lab.github.io/mctc-lib>

version  |toolchain
---------|------------------
``0.3.1``|``GCCcore/11.3.0``

### mcu

A package for periodic wavefunction and crystallography analysis.  mcu is designed to support large scale analysis and topological descriptions for periodic wavefunction.

*homepage*: <https://hungqpham.com/mcu/>

version       |toolchain
--------------|---------------
``2021-04-06``|``gomkl/2021a``

### MDAnalysis

MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD) simulations in many popular formats.

*homepage*: <https://www.mdanalysis.org/>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------
``0.20.1``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.1.1`` |                 |``foss/2020b``
``2.0.0`` |                 |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``2.2.0`` |                 |``foss/2022a``
``2.4.2`` |                 |``foss/2021a``, ``foss/2022b``

### MDBM

MDBM is a super-fast memory-mapped key/value store

*homepage*: <https://github.com/yahoo/mdbm>

version   |toolchain
----------|------------------------------------
``4.13.0``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

### MDSplus

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: <https://mdsplus.org/>

version    |versionsuffix                   |toolchain
-----------|--------------------------------|-----------------
``7.0.67`` |``-Java-1.7.0_79-Python-2.7.11``|``foss/2016a``
``7.46.1`` |                                |``foss/2018a``
``7.96.8`` |                                |``GCCcore/9.3.0``
``7.96.12``|                                |``GCCcore/9.3.0``

### MDSplus-Java

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: <https://mdsplus.org>

version    |versionsuffix|toolchain
-----------|-------------|-----------------
``7.96.12``|``-Java-13`` |``GCCcore/9.3.0``

### MDSplus-Python

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: <https://mdsplus.org>

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``7.96.12``|``-Python-3.8.2``|``foss/2020a``

### mdtest

mdtest is an MPI-coordinated metadata benchmark test that performs   open/stat/close operations on files and directories and then reports the performance.

*homepage*: <http://sourceforge.net/projects/mdtest/>

version  |toolchain
---------|---------------
``1.9.3``|``intel/2017a``

### MDTraj

Read, write and analyze MD trajectories with only a few lines of Python code.

*homepage*: <https://mdtraj.org>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------------------------
``1.9.1``|``-Python-3.6.3``|``intel/2017b``
``1.9.2``|``-Python-3.6.6``|``intel/2018b``
``1.9.3``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.9.4``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``1.9.5``|                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.9.7``|                 |``foss/2021a``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

### medaka

medaka is a tool to create a consensus sequence from nanopore sequencing data.

*homepage*: <https://github.com/nanoporetech/medaka>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.4.3`` |``-Python-3.6.6``|``foss/2018b``
``0.11.4``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|``-Python-3.6.6``|``foss/2018b``
``1.1.1`` |``-Python-3.7.4``|``foss/2019b``
``1.1.3`` |``-Python-3.7.4``|``foss/2019b``
``1.2.0`` |``-Python-3.7.4``|``foss/2019b``
``1.4.3`` |                 |``foss/2020b``
``1.4.3`` |``-Python-3.7.4``|``foss/2019b``
``1.5.0`` |                 |``foss/2021a``
``1.6.0`` |                 |``foss/2021b``

### medImgProc

Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.

*homepage*: <https://github.com/WeiXuanChan/motionSegmentation>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.5.7``|``-Python-3.7.4``|``foss/2019b``

### MedPy

MedPy is a library and script collection for medical image processing in Python, providing basic  functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main  contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready  to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package.

*homepage*: <https://pypi.org/project/MedPy/>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.4.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.4.0``|``-Python-3.7.4``|``foss/2019b``

### Meep

Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package  developed at MIT to model electromagnetic systems.

*homepage*: <https://meep.readthedocs.io/en/latest/>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------
``1.3``   |                  |``foss/2016a``
``1.4.3`` |                  |``intel/2020a``
``1.6.0`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``
``1.26.0``|                  |``foss/2020b``

### MEGA

MEGA-CC (Molecular Evolutionary Genetics Analysis Computational Core) is an integrated suite of tools  for statistics-based comparative analysis of molecular sequence data based on evolutionary principles.

*homepage*: <http://www.megasoftware.net/>

version     |toolchain
------------|----------
``7.0.20-1``|``system``
``10.0.5``  |``system``
``11.0.10`` |``system``

### MEGACC

MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and   protein sequence data from species and populations.

*homepage*: <http://www.megasoftware.net>

version     |toolchain
------------|----------
``7.0.18-1``|``system``

### MEGAHIT

An ultra-fast single-node solution for large and complex  metagenomics assembly via succinct de Bruijn graph

*homepage*: <https://github.com/voutcn/megahit>

version  |versionsuffix     |toolchain
---------|------------------|---------------------------------------------------------------------------------------------
``1.1.2``|``-Python-2.7.14``|``foss/2018a``
``1.1.3``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``
``1.1.3``|``-Python-3.6.3`` |``foss/2017b``
``1.1.4``|``-Python-2.7.15``|``foss/2018b``
``1.1.4``|``-Python-3.6.6`` |``foss/2018b``
``1.2.8``|                  |``GCCcore/8.2.0``
``1.2.9``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``1.2.9``|``-Python-2.7.18``|``GCCcore/10.2.0``

### Megalodon

Megalodon is a research command line tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads by anchoring the information rich basecalling neural network output to a reference genome/transriptome.

*homepage*: <https://github.com/nanoporetech/megalodon>

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------
``2.3.5``|                |``foss/2020b``, ``fosscuda/2020b``
``2.5.0``|                |``foss/2021a``
``2.5.0``|``-CUDA-11.3.1``|``foss/2021a``

### MEGAN

MEGAN is a comprehensive toolbox for interactively analyzing microbiome data

*homepage*: <http://megan.informatik.uni-tuebingen.de/>

version   |versionsuffix|toolchain
----------|-------------|----------
``6.22.0``|``-Java-17`` |``system``

### Meld

Meld is a visual diff and merge tool targeted at developers.   Meld helps you compare files, directories, and version controlled projects.   It provides two- and three-way comparison of both files and directories, and has support   for many popular version control systems.   

*homepage*: <https://meldmerge.org/>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``3.20.1``|``-Python-3.7.2``|``GCCcore/8.2.0``

### MEM

Marker Enrichment Modeling (MEM) is a tool designed to calculate enrichment scores.

*homepage*: <https://github.com/cytolab/mem>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20191023``|             |``foss/2019b``
``20191023``|``-R-4.0.0`` |``foss/2020a``

### MEME

The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or  GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using  MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate  motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment  using SpaMo or CentriMo.

*homepage*: <https://meme-suite.org/meme/index.html>

version  |versionsuffix                 |toolchain
---------|------------------------------|-------------------------------
``5.0.4``|``-Perl-5.26.0-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``5.0.4``|``-Perl-5.26.0-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``5.1.1``|``-Python-3.6.6``             |``foss/2018b``
``5.1.1``|``-Python-3.7.4``             |``foss/2019b``, ``intel/2019b``
``5.4.1``|                              |``GCC/10.3.0``, ``gompi/2021b``
``5.4.1``|``-Python-2.7.18``            |``gompi/2021b``

### memkind

User Extensible Heap Manager built on top of jemalloc which enables control of memory characteristics  and a partitioning of the heap between kinds of memory. 

*homepage*: <http://memkind.github.io>

version  |toolchain
---------|-----------------
``1.5.0``|``GCCcore/5.4.0``

### memory-profiler

memory-profiler is a Python module for monitoring memory consumption of a process  as well as line-by-line analysis of memory consumption for python programs.

*homepage*: <https://pypi.org/project/memory-profiler>

version   |toolchain
----------|-------------------------------
``0.55.0``|``foss/2019a``, ``intel/2019a``

### MEMOTE

The genome-scale metabolic model test suite

*homepage*: <https://github.com/opencobra/memote>

version   |toolchain
----------|--------------
``0.13.0``|``foss/2021a``

### memtester

A userspace utility for testing the memory subsystem for faults

*homepage*: <https://pyropus.ca./software/memtester/>

version  |toolchain
---------|------------------
``4.5.1``|``GCCcore/11.3.0``

### meRanTK

meRanTK is a versatile high performance toolkit for complete analysis of methylated RNA data.

*homepage*: <http://www.icbi.at/index.html>

version   |toolchain
----------|----------
``1.1.1b``|``system``

### MERCKX

Multilingual Entity/Resource Combiner & Knowledge eXtractor

*homepage*: <https://github.com/madewild/MERCKX>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170330``|``-Python-2.7.13``|``intel/2017a``

### Mercurial

Mercurial is a free, distributed source control management tool. It efficiently handles projects of any size and offers an simple and intuitive interface. 

*homepage*: <https://www.mercurial-scm.org>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``3.8.3``|``-Python-2.7.11``|``foss/2016a``
``5.7.1``|                  |``GCCcore/10.2.0``
``5.7.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.8``  |                  |``GCCcore/10.3.0``
``6.2``  |                  |``GCCcore/11.3.0``

### Mesa

Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.

*homepage*: <https://www.mesa3d.org/>

version   |toolchain
----------|--------------------------------------------------------------------
``11.1.2``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``11.2.1``|``foss/2016a``, ``intel/2016a``
``12.0.2``|``foss/2016b``, ``intel/2016b``
``17.0.2``|``foss/2017a``, ``intel/2017a``
``17.2.4``|``intel/2017b``, ``intelcuda/2017b``
``17.2.5``|``foss/2017b``, ``fosscuda/2017b``
``17.3.6``|``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``, ``iomkl/2018a``
``18.1.1``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``19.0.1``|``GCCcore/8.2.0``
``19.1.7``|``GCCcore/8.3.0``
``19.2.1``|``GCCcore/8.3.0``
``20.0.2``|``GCCcore/9.3.0``
``20.2.1``|``GCCcore/10.2.0``
``21.1.1``|``GCCcore/10.3.0``
``21.1.7``|``GCCcore/11.2.0``
``22.0.3``|``GCCcore/11.3.0``
``22.2.4``|``GCCcore/12.2.0``

### Mesa-demos

Mesa utility and demo programs, including glxinfo and eglinfo.

*homepage*: <https://www.mesa3d.org/>

version  |toolchain
---------|------------------
``8.4.0``|``GCCcore/10.3.0``

### meshalyzer

Graphical program for display time dependent data on 3D finite element meshes

*homepage*: <https://git.opencarp.org/openCARP/meshalyzer>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2.0``     |``-Python-3.8.2``|``foss/2020a``
``2.2``     |                 |``foss/2020b``
``20200308``|``-Python-3.8.2``|``foss/2020a``

### meshio

meshio is a tool for reading/writing various mesh formats representing unstructured meshes

*homepage*: <https://github.com/nschloe/meshio>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.1``|``-Python-2.7.12``|``intel/2016b``
``2.0.2``|``-Python-2.7.14``|``intel/2018a``
``2.0.2``|``-Python-3.6.4`` |``intel/2018a``
``5.3.4``|                  |``foss/2022b``

### meshtool

Meshtool is a comand-line tool written in C++. It is designed to apply various manipulations to volumetric meshes.

*homepage*: <https://bitbucket.org/aneic/meshtool>

version|toolchain
-------|-----------------------------
``16`` |``GCC/10.2.0``, ``GCC/9.3.0``

### Meson

Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.

*homepage*: <https://mesonbuild.com>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.43.0``|``-Python-3.6.3``|``intel/2017b``
``0.46.1``|``-Python-3.6.4``|``foss/2018a``
``0.48.1``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``
``0.48.1``|``-Python-3.6.6``|``foss/2018b``, ``fosscuda/2018b``
``0.50.0``|``-Python-3.7.2``|``GCCcore/8.2.0``
``0.51.2``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.53.1``|``-Python-3.6.3``|``intel/2017b``
``0.53.2``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.55.1``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.55.3``|                 |``GCCcore/10.2.0``
``0.58.0``|                 |``GCCcore/10.3.0``
``0.58.2``|                 |``GCCcore/11.2.0``
``0.59.1``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.62.1``|                 |``GCCcore/11.3.0``
``0.64.0``|                 |``GCCcore/12.2.0``
``1.1.1`` |                 |``GCCcore/12.3.0``

### meson-python

Python build backend (PEP 517) for Meson projects

*homepage*: <https://github.com/mesonbuild/meson-python>

version   |toolchain
----------|------------------
``0.11.0``|``GCCcore/12.2.0``

### Mesquite

Mesh-Quality Improvement Library

*homepage*: <https://software.sandia.gov/mesquite/>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/8.3.0``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017a``

### MESS

Master Equation System Solver (MESS)

*homepage*: <https://github.com/PACChem/MESS>

version  |toolchain
---------|--------------
``0.1.6``|``foss/2019b``

### MetaBAT

An efficient tool for accurately reconstructing single genomes from complex microbial communities

*homepage*: <https://bitbucket.org/berkeleylab/metabat>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------
``2.12.1``|``-Python-2.7.15``|``foss/2018b``
``2.14``  |                  |``gompi/2019a``
``2.15``  |                  |``GCC/11.2.0``, ``gompi/2021a``, ``gompi/2021b``
``2.15``  |``-Python-2.7.18``|``gompi/2020b``

### MetaboAnalystR

MetaboAnalystR contains the R functions and libraries underlying the popular MetaboAnalyst web  server, including > 500 functions for metabolomic data analysis, visualization, and functional interpretation.

*homepage*: <https://www.metaboanalyst.ca>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``2.0.1-20190827``|``-R-3.6.0`` |``foss/2019a``

### metaerg

MetaErg is a stand-alone and fully automated metagenomic and metaproteomic data annotation pipeline.

*homepage*: <https://github.com/xiaoli-dong/metaerg>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.2.3``|``-Python-2.7.16``|``intel/2019b``

### MetaEuk

MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic  metagenomic contigs.

*homepage*: <https://metaeuk.soedinglab.org>

version|toolchain
-------|----------------------------------------------
``4``  |``GCC/10.2.0``
``5``  |``GCC/10.3.0``
``6``  |``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### MetaGeneAnnotator

MetaGeneAnnotator is a gene-finding program for prokaryote and phage.

*homepage*: <http://metagene.nig.ac.jp/>

version     |versionsuffix|toolchain
------------|-------------|----------
``20080819``|``-x86-64``  |``system``

### Metagenome-Atlas

Metagenome-atlas is a simple-to-use metagenomic pipeline based on snakemake.  It handles all steps from QC, Assembly, Binning, to Annotation.

*homepage*: <https://github.com/metagenome-atlas/atlas>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.4.3``|``-Python-3.8.2``|``intel/2020a``

### Metal

Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large  datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.

*homepage*: <http://csg.sph.umich.edu/abecasis/Metal/>

version       |toolchain
--------------|--------------
``2011-03-25``|``foss/2016a``
``2020-05-05``|``GCC/10.2.0``

### MetaMorpheus

MetaMorpheus is a bottom-up proteomics database search software  with integrated post-translational modification (PTM) discovery capability.  This program combines features of Morpheus and G-PTM-D in a single tool. 

*homepage*: <https://github.com/smith-chem-wisc/MetaMorpheus>

version    |toolchain
-----------|------------------
``0.0.320``|``GCCcore/10.3.0``

### MetaPhlAn

MetaPhlAn is a computational tool for profiling the composition  of microbial communities from metagenomic shotgun sequencing data 

*homepage*: <https://github.com/biobakery/MetaPhlAn>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.9``|``-Python-3.8.2``|``foss/2020a``
``4.0.6``|                 |``foss/2022a``

### MetaPhlAn2

MetaPhlAn is a computational tool for profiling the composition of microbial  communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing  data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now  possible to perform accurate strain-level microbial profiling.

*homepage*: <https://bitbucket.org/biobakery/metaphlan2/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.7.8``|``-Python-2.7.12``|``foss/2016b``
``2.7.8``|``-Python-3.6.6`` |``foss/2018b``
``2.7.8``|``-Python-3.8.2`` |``foss/2020a``

### metaWRAP

MetaWRAP aims to be an simple-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.

*homepage*: <https://github.com/bxlab/metaWRAP>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.2``  |``-Python-2.7.15``|``foss/2018b``
``1.2.2``|``-Python-2.7.15``|``foss/2019a``
``1.3``  |``-Python-2.7.18``|``foss/2020b``

### Metaxa2

Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically

*homepage*: <https://microbiology.se/software/metaxa2/>

version|toolchain
-------|---------------
``2.2``|``gompi/2019a``

### methylartist

Tools for plotting methylation data in various ways 

*homepage*: <https://github.com/adamewing/methylartist>

version  |toolchain
---------|--------------
``1.2.6``|``foss/2021b``

### MethylDackel

A (mostly) universal methylation extractor for BS-seq experiments.

*homepage*: <https://github.com/dpryan79/MethylDackel>

version  |toolchain
---------|----------------------------------------
``0.4.0``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.5.0``|``iccifort/2019.5.281``
``0.6.1``|``GCC/11.2.0``

### methylpy

Bisulfite sequencing data processing and differential methylation analysis.

*homepage*: <https://pypi.python.org/pypi/methylpy>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.9``|``-Python-2.7.13``|``foss/2017a``
``1.2.9``|                  |``foss/2021b``

### METIS

METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes.

*homepage*: <http://glaros.dtc.umn.edu/gkhome/metis/metis/overview>

version  |versionsuffix|toolchain
---------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.0.2``|             |``gimkl/2.11.5``
``5.1.0``|             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``5.1.0``|``-32bitIDX``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### mgltools

The MGLTools software suite can be used for visualization     and analysis of molecular structures and comprises     the Python Molecular Viewer (PMV, a general purpose molecular viewer),     AutoDockTools (ADT, a set of PMV commands specifically developed to support AutoDock users)     and Vision (a visual programming environment). 

*homepage*: <https://ccsb.scripps.edu/mgltools/>

version  |toolchain
---------|----------
``1.5.7``|``system``

### mhcflurry

MHCflurry implements class I peptide/MHC binding affinity  prediction. By default it supports 112 MHC alleles using ensembles of  allele-specific models. Pan-allele predictors supporting virtually any MHC  allele of known sequence are available for testing (see below). MHCflurry  runs on Python 2.7 and 3.4+ using the keras neural network library. It  exposes command-line and Python library interfaces.

*homepage*: <https://github.com/openvax/mhcflurry>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.2.4``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

### mhcnuggets

MHCnuggets: Neoantigen peptide MHC binding prediction for class I and II.

*homepage*: <https://github.com/KarchinLab/mhcnuggets>

version|versionsuffix    |toolchain
-------|-----------------|----------------------------------
``2.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``2.3``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

### MicrobeAnnotator

Simple-to-use pipeline for the comprehensive metabolic annotation of microbial genomes.

*homepage*: <https://github.com/cruizperez/MicrobeAnnotator>

version  |toolchain
---------|--------------
``2.0.5``|``foss/2021a``

### microctools

Various worker functions for microclimc package

*homepage*: <https://github.com/ilyamaclean/microctools>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``0.1.0-20201209``|``-R-4.0.4`` |``foss/2020b``

### MiGEC

MIGEC is a software pipeline that facilitates processing and analysis of   immune repertoire sequencing data generated using molecular barcoding technique

*homepage*: <https://milaboratory.com/software/migec/>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.2.8``|``-Java-1.8.0_162``|``system``
``1.2.9``|``-Java-1.8``      |``system``

### MIGRATE-N

Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data.  It  uses a coalescent theory approach taking into account history of  mutations and uncertainty of the genealogy.  

*homepage*: <https://peterbeerli.com/migrate-html5/index.html>

version   |toolchain
----------|--------------
``4.2.8`` |``foss/2016a``
``4.2.14``|``foss/2018a``
``5.0.4`` |``foss/2021b``

### Miller

Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON

*homepage*: <https://miller.readthedocs.io/>

version  |toolchain
---------|----------
``6.4.0``|``system``

### mimalloc

mimalloc is a general purpose allocator with excellent performance characteristics.

*homepage*: <https://microsoft.github.io/mimalloc/>

version  |toolchain
---------|--------------------------------------
``1.7.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### MINC

Medical Image NetCDF or MINC isn't netCDF.

*homepage*: <https://github.com/BIC-MNI/libminc>

version   |toolchain
----------|-----------------------------------------------
``2.4.03``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``

### MinCED

Mining CRISPRs in Environmental Datasets

*homepage*: <https://github.com/ctSkennerton/minced>

version  |versionsuffix|toolchain
---------|-------------|-----------------
``0.4.2``|``-Java-11`` |``GCCcore/8.3.0``

### Mini-XML

Mini-XML is a small XML parsing library that you can use to read XML data files or strings in your  application without requiring large non-standard libraries

*homepage*: <https://www.msweet.org/mxml>

version  |toolchain
---------|------------------
``2.9``  |``GCCcore/8.2.0``
``2.12`` |``GCCcore/9.3.0``
``3.2``  |``GCCcore/10.3.0``
``3.3.1``|``GCCcore/11.2.0``

### miniasm

Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format. Different from mainstream assemblers, miniasm does not have a consensus step. It simply concatenates pieces of read sequences to generate the final unitig sequences. Thus the per-base error rate is similar to the raw input reads.

*homepage*: <https://github.com/lh3/minimap2>

version         |toolchain
----------------|--------------------------------------
``0.3-20191007``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### minibar

Dual barcode and primer demultiplexing for MinION sequenced reads

*homepage*: <https://github.com/calacademy-research/minibar>

version     |versionsuffix    |toolchain
------------|-----------------|-----------------------
``20200326``|``-Python-3.7.4``|``iccifort/2019.5.281``
``20200326``|``-Python-3.8.2``|``iccifort/2020.1.217``

### MiniCARD

MiniCARD is a *cardinality solver* based on MiniSAT [www.minisat.se].  MiniCARD handles cardinality constraints natively, using the same efficient data structures and techniques MiniSAT uses for clauses, giving it much better performance on cardinality constraints than CNF encodings of those constraints passed to a typical SAT solver.  It can read the standard DIMACS CNF format, the OPB pseudo-boolean format (with linear cardinality constraints only), and CNF+, a format that extends CNF to include cardinality constraints.

*homepage*: <https://github.com/liffiton/minicard>

version|toolchain
-------|-------------
``1.2``|``GCC/9.3.0``

### Miniconda2

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: <https://docs.conda.io/en/latest/miniconda.html>

version   |toolchain
----------|----------
``4.3.21``|``system``
``4.6.14``|``system``
``4.7.10``|``system``

### Miniconda3

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: <https://docs.conda.io/en/latest/miniconda.html>

version      |toolchain
-------------|----------
``4.4.10``   |``system``
``4.5.12``   |``system``
``4.6.14``   |``system``
``4.7.10``   |``system``
``4.8.3``    |``system``
``4.9.2``    |``system``
``4.12.0``   |``system``
``22.11.1-1``|``system``

### minieigen

A small wrapper for core parts of EIgen, c++ library for linear algebra.

*homepage*: <https://launchpad.net/minieigen/>

version  |versionsuffix                  |toolchain
---------|-------------------------------|-------------------------------
``0.5.3``|``-Python-2.7.11``             |``foss/2016a``, ``intel/2016a``
``0.5.3``|``-Python-2.7.12``             |``foss/2016b``, ``intel/2016b``
``0.5.3``|``-Python-2.7.12-Boost-1.63.0``|``intel/2016b``
``0.5.4``|``-Python-2.7.14``             |``intel/2018a``

### Minimac4

Minimac4 is a latest version in the series of genotype imputation software  - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010).  Minimac4 is a lower memory and more computationally efficient implementation  of the original algorithms with comparable imputation quality.

*homepage*: <https://genome.sph.umich.edu/wiki/Minimac4>

version  |toolchain
---------|--------------
``1.0.0``|``foss/2018a``

### minimap2

Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.

*homepage*: <https://github.com/lh3/minimap2>

version   |toolchain
----------|------------------------------------------------------
``2.0rc1``|``foss/2016b``
``2.10``  |``foss/2018a``
``2.11``  |``intel/2018a``
``2.12``  |``foss/2018a``
``2.13``  |``foss/2018b``
``2.17``  |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCCcore/9.3.0``
``2.18``  |``GCCcore/10.2.0``
``2.20``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.22``  |``GCCcore/11.2.0``
``2.24``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.26``  |``GCCcore/12.2.0``

### Minipolish

A tool for Racon polishing of miniasm assemblies

*homepage*: <https://github.com/rrwick/Minipolish>

version  |toolchain
---------|--------------
``0.1.3``|``GCC/11.2.0``

### MiniSat

MiniSat is a minimalistic, open-source SAT solver, developed to help researchers and developers alike to get started on SAT.

*homepage*: <http://minisat.se/>

version     |toolchain
------------|-------------
``2.2.0``   |``GCC/9.3.0``
``20130925``|``GCC/9.3.0``

### minizip

Mini zip and unzip based on zlib

*homepage*: <https://www.winimage.com/zLibDll/minizip.html>

version|toolchain
-------|------------------
``1.1``|``GCCcore/11.2.0``

### MINPACK

Minpack includes software for solving nonlinear equations and  nonlinear least squares problems.  Five algorithmic paths each include  a core subroutine and an simple-to-use driver.  The algorithms proceed  either from an analytic specification of the Jacobian matrix or  directly from the problem functions.  The paths include facilities for  systems of equations with a banded Jacobian matrix, for least squares  problems with a large amount of data, and for checking the consistency  of the Jacobian matrix with the functions. 

*homepage*: <https://www.netlib.org/minpack/>

version     |toolchain
------------|--------------
``19961126``|``GCC/10.3.0``

### MinPath

MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions  using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological  pathways for a query dataset.

*homepage*: <https://omics.informatics.indiana.edu/MinPath>

version|versionsuffix     |toolchain
-------|------------------|-------------------------------
``1.4``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.4``|``-Python-2.7.16``|``intel/2019b``
``1.6``|                  |``GCCcore/11.2.0``

### MIRA

MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),  IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads).

*homepage*: <https://sourceforge.net/p/mira-assembler/wiki/Home/>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------
``4.0.2`` |                  |``foss/2018b``, ``gompi/2019b``, ``intel/2017b``
``4.0.2`` |``-Python-2.7.11``|``foss/2016a``
``4.9.6`` |                  |``intel/2017b``
``5.0rc2``|                  |``foss/2020b``

### miRDeep2

miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs

*homepage*: <https://www.mdc-berlin.de/8551903/en/>

version    |versionsuffix    |toolchain
-----------|-----------------|---------------
``0.1.1``  |``-Python-3.6.6``|``foss/2018b``
``2.0.0.8``|                 |``intel/2016b``

### Mish-Cuda

Mish-Cuda: Self Regularized Non-Monotonic Activation Function

*homepage*: <https://github.com/JunnYu/mish-cuda>

version     |versionsuffix     |toolchain
------------|------------------|------------------
``20210309``|``-PyTorch-1.9.0``|``fosscuda/2020b``

### misha

The misha package is intended to help users to efficiently analyze genomic data achieved  from various experiments.

*homepage*: <https://github.com/tanaylab/misha>

version   |versionsuffix|toolchain
----------|-------------|--------------
``4.0.10``|``-R-4.0.0`` |``foss/2020a``

### MITgcmutils

A numerical model designed for study of the atmosphere, ocean,  and climate, MITgcm’s flexible non-hydrostatic formulation enables it to  efficiently simulate fluid phenomena over a wide range of scales; its adjoint  capabilities enable it to be applied to sensitivity questions and to parameter  and state estimation problems. By employing fluid equation isomorphisms, a  single dynamical kernel can be used to simulate flow of both the atmosphere  and ocean. The model is developed to perform efficiently on a wide variety of  computational platforms.

*homepage*: <https://mitgcm.org/>

version  |toolchain
---------|--------------
``0.1.2``|``foss/2022a``

### MITObim

The MITObim procedure (mitochondrial baiting and iterative mapping) represents  a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms  directly from total genomic DNA derived NGS reads.

*homepage*: <https://github.com/chrishah/MITObim>

version  |toolchain
---------|-----------------------------------------------
``1.9.1``|``foss/2018b``, ``foss/2020b``, ``gompi/2019b``

### MitoZ

MitoZ is a Python3-based toolkit which aims to automatically filter pair-end raw data (fastq files),  assemble genome, search for mitogenome sequences from the genome assembly result, annotate mitogenome  (genbank file as result), and mitogenome visualization.

*homepage*: <https://github.com/linzhi2013/MitoZ>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``2.3``|``-Python-3.6.6``|``foss/2018b``

### MiXCR

MiXCR processes big immunome data from raw sequences  to quantitated clonotypes 

*homepage*: <https://milaboratory.com/software/mixcr/>

version   |versionsuffix      |toolchain
----------|-------------------|----------
``2.1.9`` |``-Java-1.8.0_162``|``system``
``3.0.3`` |``-Java-1.8``      |``system``
``3.0.13``|``-Java-1.8``      |``system``
``3.0.13``|``-Java-11``       |``system``

### MixMHC2pred

MixMHC2pred is a predictor of HLA class II ligands and epitopes.  It is described in publication Racle, J., et al. Robust prediction of HLA class II  epitopes by deep motif deconvolution of immunopeptidomes

*homepage*: <https://github.com/GfellerLab/MixMHC2pred>

version|toolchain
-------|----------
``1.2``|``system``

### mkl-dnn

Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) MKL-DNN)

*homepage*: <https://01.org/mkl-dnn>

version   |toolchain
----------|-------------------------------
``0.11``  |``intel/2017b``
``0.13``  |``intel/2018a``
``0.16``  |``foss/2018b``, ``intel/2018b``
``0.17.2``|``foss/2018a``

### mkl-service

Python hooks for Intel(R) Math Kernel Library runtime control settings.

*homepage*: <https://github.com/IntelPython/mkl-service>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------
``2.0.2``|                 |``intel/2019a``
``2.3.0``|                 |``intel/2020b``, ``intel/2021b``
``2.3.0``|``-Python-3.7.4``|``intel/2019b``

### mkl_fft

NumPy-based Python interface to Intel(R) MKL FFT functionality

*homepage*: <https://github.com/IntelPython/mkl_fft>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.0.14``|``-Python-3.6.6``|``intel/2018b``

### MLC

Intel Memory Latency Checker (Intel MLC) is a tool used to measure memory latencies and b/w,  and how they change with increasing load on the system.

*homepage*: <https://software.intel.com/en-us/articles/intelr-memory-latency-checker>

version|toolchain
-------|----------
``3.0``|``system``

### MLxtend

Mlxtend (machine learning extensions) is a Python library of useful tools for the day-to-day data  science tasks.

*homepage*: <https://rasbt.github.io/mlxtend/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.17.3``|``-Python-3.8.2``|``foss/2020a``

### mm-common

The mm-common module provides the build infrastructure and  utilities shared among the GNOME C++ binding libraries.

*homepage*: <https://gitlab.gnome.org/GNOME/mm-common>

version  |toolchain
---------|------------------
``1.0.4``|``GCCcore/10.3.0``
``1.0.5``|``GCCcore/11.3.0``

### Mmg

Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries: the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation and generation of a triangulation from a set of points or from given boundary edges the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries.

*homepage*: <https://www.mmgtools.org/>

version  |toolchain
---------|--------------------------------
``5.3.9``|``foss/2017b``
``5.6.0``|``gompi/2021a``, ``gompi/2021b``

### MMSEQ

The MMSEQ package contains a collection of statistical tools  for analysing RNA-seq expression data. 

*homepage*: <https://github.com/eturro/mmseq>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.0.8``|``-linux64-static``|``system``

### MMseqs2

MMseqs2: ultra fast and sensitive search and clustering suite

*homepage*: <https://mmseqs.com>

version              |toolchain
---------------------|-------------------------------------------------
``1-c7a89``          |``foss/2016b``
``5-9375b``          |``intel/2018a``
``8-fac81``          |``intel/2018b``
``10-6d92c``         |``gompi/2019b``, ``iimpi/2019b``
``11-e1a1c``         |``iimpi/2019b``
``13-45111``         |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``
``13-45111-20211006``|``gompi/2020b``
``13-45111-20211019``|``gompi/2020b``
``14-7e284``         |``gompi/2022a``

### mmtf-cpp

The Macromolecular Transmission Format (MMTF) is a new compact binary format to  transmit and store biomolecular structures for fast 3D visualization and        analysis. 

*homepage*: <https://mmtf.rcsb.org/>

version  |toolchain
---------|------------------
``1.0.0``|``GCCcore/10.2.0``

### MNE-Python

MNE-Python software is an open-source Python package for exploring, visualizing, and analyzing human neurophysiological data such as MEG, EEG, sEEG, ECoG, and more. It includes modules for data input/output, preprocessing, visualization, source estimation, time-frequency analysis, connectivity analysis, machine learning, and statistics.

*homepage*: <https://mne.tools/stable/index.html>

version   |toolchain
----------|--------------
``0.24.1``|``foss/2021a``

### MOABB

Build a comprehensive benchmark of popular Brain-Computer Interface (BCI) algorithms applied on an extensive list of freely available EEG datasets.

*homepage*: <https://neurotechx.github.io/moabb/>

version  |toolchain
---------|--------------
``0.4.6``|``foss/2021a``

### MOABS

MOABS: MOdel based Analysis of Bisulfite Sequencing data

*homepage*: <https://github.com/sunnyisgalaxy/moabs>

version    |toolchain
-----------|---------------
``1.3.9.6``|``gompi/2019b``

### MOB-suite

Software tools for clustering, reconstruction and typing of plasmids from draft assemblies

*homepage*: <https://github.com/phac-nml/mob-suite>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.0``|``-Python-3.7.4``|``foss/2019b``

### ModelTest-NG

ModelTest-NG is a tool for selecting the best-fit model of evolution for DNA and protein alignments. ModelTest-NG supersedes jModelTest and ProtTest in one single tool, with graphical and command console interfaces. 

*homepage*: <https://github.com/ddarriba/modeltest>

version  |toolchain
---------|---------------
``0.1.7``|``gompi/2021b``

### modred

Compute modal decompositions and reduced-order models, easily, efficiently, and in parallel.

*homepage*: <https://pypi.python.org/pypi/modred/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.2``|``-Python-3.5.2``|``foss/2016b``

### Molcas

Molcas is an ab initio quantum chemistry software package developed by scientists to be used by scientists. The basic philosophy is is to be able to treat general electronic structures for molecules consisting of atoms from most of the periodic table. As such, the primary focus of the package is on multiconfigurational methods with applications typically connected to the treatment of highly degenerate states.

*homepage*: <http://www.molcas.org>

version         |versionsuffix         |toolchain
----------------|----------------------|----------
``8.0-15.06.18``|``_CentOS_6.6_x86_64``|``system``
``8.2``         |``-centos-mkl-par``   |``system``
``8.2``         |``-centos-par``       |``system``

### mold

mold is a high-performance drop-in replacement for existing Unix linkers.

*homepage*: <https://github.com/rui314/mold>

version   |toolchain
----------|------------------
``0.9.6`` |``GCCcore/11.2.0``
``1.0.0`` |``GCCcore/11.2.0``
``1.2.1`` |``GCCcore/11.3.0``
``1.3.0`` |``GCCcore/11.3.0``
``1.7.1`` |``GCCcore/12.2.0``
``1.11.0``|``GCCcore/12.3.0``

### Molden

Molden is a package for displaying Molecular Density from the  Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical  packages Mopac/Ampac

*homepage*: <https://www.theochem.ru.nl/molden/>

version|toolchain
-------|-------------------------------------
``5.6``|``foss/2016a``
``5.7``|``intel/2016b``
``5.8``|``foss/2018a``
``6.1``|``GCCcore/8.2.0``
``6.8``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``7.1``|``GCCcore/11.3.0``

### molecularGSM

Code for single-ended and double-ended molecular GSM. The growing string method is a reaction path and transition state finding method developed in c++.

*homepage*: <https://github.com/ZimmermanGroup/molecularGSM>

version     |toolchain
------------|---------------
``20190826``|``intel/2020b``

### Molekel

Molekel is an open-source multi-platform molecular visualization program. 

*homepage*: <http://ugovaretto.github.io/molekel/wiki/pmwiki.php/Main/HomePage.html>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``5.4.0``|``-Linux_x86_64``|``system``

### molmod

MolMod is a Python library with many compoments that are useful to write molecular modeling programs.

*homepage*: <https://molmod.github.io/molmod/>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------
``1.1``  |``-Python-2.7.11``|``intel/2016a``
``1.1``  |``-Python-2.7.12``|``intel/2016b``
``1.1``  |``-Python-2.7.13``|``intel/2017a``
``1.4.3``|``-Python-2.7.14``|``intel/2017b``
``1.4.3``|``-Python-3.6.3`` |``intel/2017b``
``1.4.4``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.4.4``|``-Python-2.7.15``|``intel/2018b``
``1.4.4``|``-Python-3.7.2`` |``intel/2019a``
``1.4.5``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.4.5``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.4.8``|                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``

### Molpro

Molpro is a complete system of ab initio programs for molecular electronic structure calculations.

*homepage*: <https://www.molpro.net>

version      |versionsuffix           |toolchain
-------------|------------------------|----------
``2010.1.23``|``.Linux_x86_64``       |``system``
``2015.1.0`` |``.linux_x86_64_intel`` |``system``
``2015.1.3`` |``.linux_x86_64_openmp``|``system``
``2015.1.10``|``.linux_x86_64_openmp``|``system``

### MONA

MONA is a tool that translates formulas to finite-state automata. The formulas may express search patterns, temporal properties of reactive systems, parse tree constraints, etc. MONA analyses the automaton resulting from the compilation and prints out "valid" or a counter-example.

*homepage*: <https://www.brics.dk/mona/index.html>

version   |toolchain
----------|--------------
``1.4-18``|``GCC/11.2.0``

### MONAI

MONAI is a PyTorch-based, open-source framework for deep learning in healthcare imaging, part of PyTorch Ecosystem. 

*homepage*: <https://monai.io/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.8.0``|                |``foss/2021a``
``0.8.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.1``|                |``foss/2022a``
``1.0.1``|``-CUDA-11.7.0``|``foss/2022a``

### MONAI-Label

MONAI Label is an intelligent open source image labeling and learning tool  that enables users to create annotated datasets and build AI annotation models for clinical evaluation.  MONAI Label enables application developers to build labeling apps in a serverless way,  where custom labeling apps are exposed as a service through the MONAI Label Server.

*homepage*: <https://docs.monai.io/projects/label/en/latest/>

version  |versionsuffix                  |toolchain
---------|-------------------------------|--------------
``0.5.2``|``-PyTorch-1.12.0``            |``foss/2022a``
``0.5.2``|``-PyTorch-1.12.0-CUDA-11.7.0``|``foss/2022a``

### mongolite

High-performance MongoDB client based on 'mongo-c-driver' and 'jsonlite'.  Includes support for aggregation, indexing, map-reduce, streaming, encryption,  enterprise authentication, and GridFS. The online user manual provides an overview  of the available methods in the package: <https://jeroen.github.io/mongolite/>.

*homepage*: <https://cran.r-project.org/web/packages/mongolite>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.3.0``|``-R-4.0.0`` |``foss/2020a``
``2.3.0``|``-R-4.0.3`` |``foss/2020b``
``2.3.0``|``-R-4.0.4`` |``foss/2020b``

### Mono

An open source, cross-platform, implementation of C# and the CLR that is  binary compatible with Microsoft.NET.

*homepage*: <https://www.mono-project.com/>

version       |toolchain
--------------|-------------------------------
``2.10.6``    |``intel/2016b``
``4.6.2.7``   |``intel/2016b``, ``system``
``4.8.0.495`` |``intel/2017a``
``5.4.1.6``   |``foss/2017b``, ``intel/2017b``
``5.10.0.160``|``foss/2018a``
``5.18.1.0``  |``foss/2018a``
``6.4.0.198`` |``foss/2018b``
``6.8.0.105`` |``GCCcore/8.3.0``
``6.12.0.122``|``GCCcore/11.2.0``

### Monocle3

An analysis toolkit for single-cell RNA-seq. 

*homepage*: <https://cole-trapnell-lab.github.io/monocle3/>

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.2.0``|``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``0.2.3``|``-R-4.0.3``             |``foss/2020b``

### moonjit

Moonjit is a Just-In-Time Compiler (JIT) for the Lua programming  language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language.

*homepage*: <https://github.com/moonjit/moonjit>

version  |toolchain
---------|-----------------
``2.2.0``|``GCCcore/9.3.0``

### MOOSE

The Multiphysics Object-Oriented Simulation Environment (MOOSE) is a finite-element, multiphysics framework primarily developed by Idaho National Laboratory

*homepage*: <https://mooseframework.inl.gov>

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2021-05-18``|``-Python-3.7.4``|``foss/2019b``

### mordecai

mordecai is a full text geoparsing as a Python library. Extract the place names from a piece of text, resolve them to the correct place, and return their coordinates and structured geographic information.

*homepage*: <https://github.com/openeventdata/mordecai>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.1``|``-Python-3.6.4``|``foss/2018a``

### MoreRONN

MoreRONN is the spiritual successor of RONN and is useful for surveying  disorder in proteins as well as designing expressible constructs for X-ray crystallography.

*homepage*: <https://github.com/varun-ramraj/MoreRONN>

version|toolchain
-------|-------------
``4.9``|``GCC/8.3.0``

### mosdepth

Fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing

*homepage*: <https://github.com/brentp/mosdepth>

version  |toolchain
---------|---------------
``0.2.2``|``intel/2018a``
``0.2.3``|``intel/2018a``
``0.2.4``|``foss/2018b``
``0.3.3``|``GCC/11.2.0``

### Mothur

Mothur is a single piece of open-source, expandable software  to fill the bioinformatics needs of the microbial ecology community.

*homepage*: <https://www.mothur.org/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.39.5``|``-Python-2.7.13``|``intel/2017a``
``1.41.0``|``-Python-2.7.15``|``foss/2018b``
``1.43.0``|``-Python-3.7.2`` |``foss/2019a``

### motif

Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building  applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.  It was the standard toolkit for the Common Desktop Environment and thus for Unix.

*homepage*: <https://motif.ics.com/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.3.5``|``foss/2016a``, ``intel/2016a``
``2.3.7``|``intel/2017a``
``2.3.8``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2017b``, ``intel/2018a``, ``intel/2019a``

### MotionCor2

MotionCor2 correct anisotropic image motion at the single pixel level across the whole frame, suitable for both single particle and tomographic images. Iterative, patch-based motion detection is combined with spatial and temporal constraints and dose weighting.  Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache, Yifan Cheng and David A. Agard (2016) Anisotropic Correction of Beam-induced Motion for Improved Single-particle Electron Cryo-microscopy, Nature Methods, submitted. BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960 

*homepage*: <https://msg.ucsf.edu/>

version  |toolchain
---------|--------------------------------------
``1.2.6``|``GCCcore/8.2.0``
``1.3.1``|``GCCcore/8.3.0``
``1.3.2``|``GCCcore/8.3.0``
``1.4.2``|``GCCcore/10.2.0``
``1.4.4``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0``|``GCCcore/11.3.0``

### motionSegmentation

Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.

*homepage*: <https://github.com/WeiXuanChan/motionSegmentation>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.7.9``|``-Python-3.7.4``|``foss/2019b``

### MoviePy

MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,  title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects.

*homepage*: <https://zulko.github.io/moviepy/>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``1.0.1``|``-Python-3.7.2``|``foss/2019a``
``1.0.3``|                 |``foss/2021a``, ``foss/2021b``

### MPB

MPB is a free and open-source software package for computing  the band structures, or dispersion relations, and electromagnetic  modes of periodic dielectric structures, on both serial  and parallel computers. MPB is an acronym for MIT Photonic Bands.

*homepage*: <https://mpb.readthedocs.io/en/latest/>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------
``1.6.2`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``
``1.11.1``|                  |``foss/2020b``

### MPC

Gnu Mpc is a C library for the arithmetic of  complex numbers with arbitrarily high precision and correct  rounding of the result. It extends the principles of the IEEE-754  standard for fixed precision real floating point numbers to  complex numbers, providing well-defined semantics for every  operation. At the same time, speed of operation at high precision  is a major design goal.

*homepage*: <http://www.multiprecision.org/>

version  |versionsuffix  |toolchain
---------|---------------|------------------------------------------------------------------------------
``1.0.3``|               |``intel/2017a``
``1.0.3``|``-MPFR-3.1.6``|``foss/2017b``, ``intel/2017b``
``1.1.0``|               |``GCC/8.3.0``, ``GCC/9.3.0``, ``GCCcore/9.3.0``
``1.2.1``|               |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### MPFR

The MPFR library is a C library for multiple-precision floating-point  computations with correct rounding. 

*homepage*: <https://www.mpfr.org>

version  |toolchain
---------|------------------------------------------------------------------------------
``2.4.2``|``system``
``3.1.4``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.1.5``|``GCCcore/6.4.0``, ``intel/2017a``
``3.1.6``|``GCCcore/6.4.0``
``4.0.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``4.0.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.2.0``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### mpi4py

MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for  the Python programming language, allowing any Python program to exploit multiple processors.

*homepage*: <https://github.com/mpi4py/mpi4py>

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------
``1.3.1``|``-Python-2.7.11-timed-pingpong``|``intel/2016a``
``1.3.1``|``-Python-2.7.12-timed-pingpong``|``intel/2016b``
``2.0.0``|``-Python-2.7.12``               |``intel/2016b``
``2.0.0``|``-Python-2.7.13-timed-pingpong``|``intel/2017a``
``3.0.0``|``-Python-2.7.14-timed-pingpong``|``intel/2018a``
``3.0.0``|``-Python-3.6.3``                |``intel/2017b``
``3.0.1``|``-Python-3.6.6``                |``intel/2018b``
``3.0.2``|``-timed-pingpong``              |``gompi/2019a``, ``iimpi/2019a``
``3.1.4``|                                 |``gompi/2022b``

### MPICH

MPICH is a high-performance and widely portable implementation  of the Message Passing Interface (MPI) standard (MPI-1, MPI-2 and MPI-3).

*homepage*: <https://www.mpich.org/>

version  |toolchain
---------|--------------------------------------
``3.0.4``|``GCC/4.8.1``
``3.2``  |``GCC/4.9.3-2.25``, ``GCC/7.2.0-2.29``
``3.2.1``|``GCC/7.2.0-2.29``
``3.3.2``|``GCC/10.2.0``, ``GCC/9.3.0``
``3.4.2``|``GCC/10.3.0``

### MPICH2

MPICH v3.x is an open source high-performance MPI 3.0 implementation. It does not support InfiniBand (use MVAPICH2 with InfiniBand devices).

*homepage*: <http://www.mpich.org/>

version|toolchain
-------|-------------
``1.1``|``GCC/4.8.1``

### mpifileutils

MPI-Based File Utilities For Distributed Systems 

*homepage*: <https://hpc.github.io/mpifileutils/>

version   |toolchain
----------|--------------------------------
``0.9.1`` |``gompi/2019a``, ``iimpi/2019a``
``0.10``  |``gompi/2020a``, ``iimpi/2020a``
``0.10.1``|``gompi/2020a``
``0.11.1``|``gompi/2022a``

### mpiP

mpiP is a lightweight profiling library for MPI applications.  Because it  only collects statistical information about MPI functions, mpiP generates  considerably less overhead and much less data than tracing tools.  All the  information captured by mpiP is task-local. It only uses communication during  report generation, typically at the end of the experiment, to merge results  from all of the tasks into one output file. 

*homepage*: <http://mpip.sourceforge.net/>

version  |toolchain
---------|---------------------------------------------------
``3.4.1``|``gompi/2019a``, ``iimpi/2019a``, ``iompi/2019.01``

### MPJ-Express

MPJ Express is an open source Java message passing library that allows application   developers to write and execute parallel applications for multicore processors and compute clusters/clouds.

*homepage*: <http://mpj-express.org/>

version |versionsuffix     |toolchain
--------|------------------|--------------
``0.44``|``-Java-1.8.0_92``|``foss/2016a``

### mpmath

mpmath can be used as an arbitrary-precision substitute for Python's float/complex  types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation  can be performed just as well at 10-digit or 1000-digit precision, with either real or complex  numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely  high precision work.

*homepage*: <https://mpmath.org/>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------
``0.19`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.0.0``|``-Python-2.7.14``|``intel/2018a``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.1.0``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``1.2.1``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### MrBayes

MrBayes is a program for Bayesian inference and model choice across  a wide range of phylogenetic and evolutionary models.

*homepage*: <https://nbisweden.github.io/MrBayes/>

version   |toolchain
----------|--------------------------------------------------
``3.2.6`` |``foss/2016a``, ``foss/2017a``, ``gompi/2020b``
``3.2.7`` |``gompi/2020b``, ``gompi/2022a``, ``gompic/2019b``
``3.2.7a``|``foss/2020a``, ``iimpi/2019a``

### mrcfile

mrcfile is a Python implementation of the MRC2014 file format, which is used in  structural biology to store image and volume data. It allows MRC files to be created and opened easily using a very simple API,    which exposes the file’s header and data as numpy arrays. The code runs in      Python 2 and 3 and is fully unit-tested. This library aims to allow users and developers to read and write standard- compliant MRC files in Python as easily as possible, and with no dependencies on any compiled libraries except numpy. You can use it interactively to inspect  files, correct headers and so on, or in scripts and larger software packages to  provide basic MRC file I/O functions. 

*homepage*: <https://github.com/ccpem/mrcfile>

version  |toolchain
---------|--------------------------------------------------
``1.3.0``|``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``

### MRChem

MRChem is a numerical real-space code for molecular electronic structure calculations within the self-consistent field (SCF) approximations of quantum chemistry: Hartree-Fock and Density Functional Theory.

*homepage*: <https://mrchem.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.8.2``|``foss/2020a``
``1.1.1``|                 |``foss/2022a``

### MRCPP

MultiResolution Computation Program Package

*homepage*: <https://mrcpp.readthedocs.io>

version  |toolchain
---------|--------------
``1.3.6``|``foss/2020a``
``1.4.1``|``foss/2022a``

### MRIcron

MRIcron allows viewing of medical images. It  includes tools to complement SPM and FSL. Native format is NIFTI  but includes a conversion program (see dcm2nii) for converting DICOM images.  Features layers, ROIs, and volume rendering.  

*homepage*: <http://www.mccauslandcenter.sc.edu/mricro/mricron/>

version         |toolchain
----------------|----------
``1.0.20180614``|``system``
``20150601``    |``system``

### MRPRESSO

Performs the Mendelian Randomization Pleiotropy RESidual Sum and Outlier (MR-PRESSO) method..

*homepage*: <https://github.com/rondolab/MR-PRESSO>

version         |toolchain
----------------|--------------
``1.0-20230502``|``foss/2022a``

### MRtrix

MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner  robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines.

*homepage*: <http://www.brain.org.au/software/index.html#mrtrix>

version            |versionsuffix     |toolchain
-------------------|------------------|---------------
``0.3.14``         |``-Python-2.7.11``|``intel/2016a``
``0.3.15``         |``-Python-2.7.12``|``intel/2016b``
``3.0-rc-20191217``|``-Python-2.7.16``|``foss/2019b``
``3.0-rc-20191217``|``-Python-3.7.4`` |``foss/2019b``
``3.0.0``          |``-Python-3.8.2`` |``foss/2020a``
``3.0.3``          |                  |``foss/2021a``
``3.0_RC2``        |``-Python-2.7.13``|``foss/2017a``
``3.0_RC3``        |``-Python-2.7.14``|``intel/2018a``

### msgpack-c

MessagePack is an efficient binary serialization format, which lets you exchange data among multiple languages like JSON, except that it's faster and smaller.    Small integers are encoded into a single byte while typical short strings        require only one extra byte in addition to the strings themselves.

*homepage*: <http://msgpack.org/>

version  |toolchain
---------|------------------
``3.3.0``|``GCCcore/10.2.0``

### MSM

Multimodal Surface Matching with Higher order Clique Reduction

*homepage*: <https://github.com/ecr05/MSM_HOCR>

version|toolchain
-------|-------------------------------
``1.0``|``foss/2017b``, ``intel/2017b``

### MSPC

Using combined evidence from replicates to evaluate ChIP-seq peaks

*homepage*: <https://genometric.github.io/MSPC/>

version  |toolchain
---------|----------
``3.3.1``|``system``

### msprime

msprime is a coalescent simulator and library for processing tree-based genetic data.

*homepage*: <https://msprime.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``0.7.0``|``-Python-3.7.2``|``intel/2019a``
``1.2.0``|                 |``foss/2021b``, ``foss/2022a``

### mstore

Molecular structure store for testing

*homepage*: <https://github.com/grimme-lab/mstore>

version  |toolchain
---------|----------------------------------
``0.2.0``|``GCC/11.2.0``, ``GCCcore/11.3.0``

### MTL4

The Matrix Template Library 4 incorporates the most modern programming techniques  to provide an simple and intuitive interface to users while enabling optimal performance. The natural  mathematical notation in MTL4 empowers all engineers and scientists to implement their algorithms and  models in minimal time. All technical aspects are encapsulated in the library.

*homepage*: <http://www.simunova.com/mtl4>

version     |toolchain
------------|----------
``4.0.8878``|``system``
``4.0.9555``|``system``

### MuJoCo

MuJoCo stands for Multi-Joint dynamics with Contact. It is a general purpose physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, machine learning, and other areas which demand fast and accurate simulation of articulated structures interacting with their environment.

*homepage*: <https://mujoco.org/>

version  |toolchain
---------|------------------
``2.1.1``|``GCCcore/11.2.0``
``2.2.2``|``GCCcore/11.3.0``

### mujoco-py

MuJoCo is a physics engine for detailed, efficient rigid body simulations with contacts. mujoco-py allows using MuJoCo from Python 3.

*homepage*: <https://github.com/openai/mujoco-py>

version     |toolchain
------------|--------------
``2.1.2.14``|``foss/2021b``

### multichoose

generate multiset combinations (n multichoose k).

*homepage*: <https://github.com/ekg/multichoose>

version  |toolchain
---------|-----------------------------------------------------------------------------
``1.0.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### MultilevelEstimators

The Julia module for Multilevel Monte Carlo methods

*homepage*: <https://github.com/PieterjanRobbe/MultilevelEstimators.jl>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.0``|``-Julia-1.7.2``|``GCC/11.2.0``

### MultiNest

MultiNest is a Bayesian inference tool which calculates the evidence and explores the parameter space  which may contain multiple posterior modes and pronounced (curving) degeneracies in moderately high dimensions.

*homepage*: <https://ccpforge.cse.rl.ac.uk/gf/project/multinest/>

version |toolchain
--------|---------------
``3.10``|``intel/2016a``

### MultiQC

Aggregate results from bioinformatics analyses across many samples into a single   report.   MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general  use tool, perfect for summarising the output from numerous bioinformatics tools.

*homepage*: <https://multiqc.info>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.9``   |``-Python-2.7.12``|``foss/2016b``
``1.2``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.2``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.6``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.6``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.7``   |``-Python-2.7.15``|``intel/2018b``
``1.7``   |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.8``   |``-Python-2.7.16``|``intel/2019b``
``1.8``   |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.9``   |``-Python-3.7.4`` |``foss/2019b``
``1.9``   |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.10.1``|                  |``foss/2020b``
``1.11``  |                  |``foss/2021a``
``1.12``  |                  |``foss/2021b``
``1.14``  |                  |``foss/2022a``, ``foss/2022b``

### Multiwfn

Multiwfn is an extremely powerful program for realizingi  electronic wavefunction analysis, which is a key ingredient of quantum  chemistry. Multiwfn is free, open-source, high-efficient, very user-friendly  and flexible, it supports almost all of the most important wavefunction  analysis methods.

*homepage*: <http://sobereva.com/multiwfn/>

version  |toolchain
---------|--------------------------------
``3.4.1``|``intel/2017b``
``3.6``  |``intel/2019a``, ``intel/2019b``

### muMerge

muMerge is a tool for combining bed regions from multiple bed files that overlap.

*homepage*: <https://github.com/Dowell-Lab/mumerge>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2022a``

### MUMmer

MUMmer is a system for rapidly aligning entire genomes,  whether in complete or draft form. AMOS makes use of it. 

*homepage*: <https://mummer.sourceforge.net/>

version       |toolchain
--------------|-------------------------------------------------------------------------
``3.23``      |``GCCcore/10.3.0``, ``GCCcore/9.3.0``, ``foss/2016b``
``4.0.0beta2``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``4.0.0rc1``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### MUMPS

A parallel sparse direct solver

*homepage*: <https://graal.ens-lyon.fr/MUMPS/>

version  |versionsuffix |toolchain
---------|--------------|-----------------------------------------------------------------------------------------------------------------
``5.1.2``|``-metis``    |``foss/2017b``
``5.2.1``|``-metis``    |``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``intel/2019a``, ``intel/2019b``, ``intel/2020a``
``5.2.1``|``-metis-seq``|``foss/2019a``, ``intel/2019a``
``5.3.5``|``-metis``    |``foss/2020b``, ``intel/2020b``
``5.4.0``|``-metis``    |``foss/2021a``, ``intel/2021a``
``5.4.1``|``-metis``    |``foss/2021b``, ``intel/2021b``
``5.5.0``|``-metis``    |``foss/2021a``
``5.5.1``|``-metis``    |``foss/2022a``

### muParser

muParser is an extensible high performance math expression parser library  written in C++. It works by transforming a mathematical expression into  bytecode and precalculating constant parts of the expression. 

*homepage*: <https://beltoforion.de/article.php?a=muparser>

version  |toolchain
---------|-------------------------------------
``2.2.5``|``GCCcore/6.4.0``
``2.3.2``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.3.3``|``GCCcore/10.3.0``
``2.3.4``|``GCCcore/11.3.0``

### muparserx

A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more.

*homepage*: <https://beltoforion.de/en/muparserx/>

version  |toolchain
---------|--------------------------------------
``4.0.8``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### MuPeXI

MuPeXI: Mutant Peptide eXtractor and Informer. Given a list of somatic mutations (VCF file) as input,  MuPeXI returns a table containing all mutated peptides (neo-peptides) of user-defined lengths,  along with several pieces of information relevant for identifying  which of these neo-peptides are likely to serve as neo-epitopes.

*homepage*: <https://github.com/ambj/MuPeXI>

version  |versionsuffix                 |toolchain
---------|------------------------------|--------------
``1.2.0``|``-Perl-5.28.0-Python-2.7.15``|``foss/2018b``

### MUSCLE

MUSCLE is one of the best-performing multiple alignment programs  according to published benchmark tests, with accuracy and speed that are consistently  better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users  learn everything they need to know about MUSCLE in a few minutes-only a handful of  command-line options are needed to perform common alignment tasks.

*homepage*: <https://drive5.com/muscle/>

version     |versionsuffix  |toolchain
------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.8.31``  |               |``GCC/7.3.0-2.30``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2017b``, ``foss/2018a``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``3.8.31``  |``-i86linux64``|``system``
``3.8.1551``|               |``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``
``5.0.1428``|               |``GCCcore/10.3.0``
``5.1``     |               |``GCCcore/11.2.0``
``5.1.0``   |               |``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### MuSiC

Multi-subject Single Cell deconvolution (MuSiC) is a deconvolution method that utilizes  cross-subject scRNA-seq to estimate cell type proportions in bulk RNA-seq data.

*homepage*: <https://github.com/xuranw/MuSiC>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.6.2``|``-R-3.5.1`` |``foss/2018b``

### MUST

MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.  As MPI calls are complex and usage errors common, this functionality is extremely helpful for application developers  that want to develop correct MPI applications.  This includes errors that already manifest – segmentation faults or incorrect results – as well as many errors  that are not visible to the application developer or do not manifest on a certain system or MPI implementation.

*homepage*: <https://hpc.rwth-aachen.de/must/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.6``  |``-Python-3.6.6``|``foss/2018b``
``1.6``  |``-Python-3.7.4``|``foss/2019b``
``1.7.1``|                 |``foss/2020b``
``1.7.2``|                 |``foss/2021a``

### MuTect

MuTect is a method developed at the Broad Institute for the reliable  and accurate identification of somatic point mutations in next generation sequencing  data of cancer genomes. 

*homepage*: <http://www.broadinstitute.org/cancer/cga/mutect>

version  |versionsuffix     |toolchain
---------|------------------|----------
``1.1.4``|``-Java-1.7.0_76``|``system``
``1.1.4``|``-Java-1.7.0_80``|``system``
``1.1.7``|``-Java-1.7.0_80``|``system``

### mutil

Mutil is a set of standard utilities that have been parallelized to maximize performance on  modern file systems. These currently include multi-threaded drop-in replacements for cp and md5sum from GNU coreutils, which have achieved 10/30x rates on one/many nodes. 

*homepage*: <http://people.nas.nasa.gov/~kolano/projects/mutil.html>

version    |toolchain
-----------|---------------
``1.822.3``|``intel/2016a``

### MVAPICH2

This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH.

*homepage*: <http://mvapich.cse.ohio-state.edu/overview/mvapich2/>

version  |toolchain
---------|------------------
``2.0.1``|``GCC/4.8.4``
``2.1``  |``GCC/4.9.3-2.25``
``2.2b`` |``GCC/4.9.3-2.25``

### MView

MView reformats the results of a sequence database search or a   multiple alignment, optionally adding HTML markup.

*homepage*: <https://desmid.github.io/mview/>

version |toolchain
--------|--------------
``1.67``|``GCC/11.3.0``

### mxml

Mini-XML is a tiny XML library that you can use to read and write XML and  XML-like data files in your application without requiring large non-standard  libraries. 

*homepage*: <https://github.com/michaelrsweet/mxml>

version|toolchain
-------|-----------------
``3.2``|``GCCcore/8.3.0``

### mxmlplus

Mxml is a pure C library (yet having an object oriented layout) that is meant to help developers  implementing XML file interpretation in their projects.

*homepage*: <http://mxml.sourceforge.net>

version  |toolchain
---------|-------------
``0.9.2``|``GCC/9.3.0``

### MXNet

Flexible and Efficient Library for Deep Learning

*homepage*: <https://mxnet.io/>

version  |versionsuffix             |toolchain
---------|--------------------------|--------------
``0.9.3``|``-Python-2.7.12-R-3.3.3``|``foss/2016b``

### MyCC

MyCC is built and delivered as a tailored solution for metagenomics sequencesclassfication.

*homepage*: <https://sourceforge.net/projects/sb2nhri/files/MyCC/>

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2017-03-01``|``-Python-2.7.16``|``intel/2019b``

### mygene

Python Client for MyGene.Info services.

*homepage*: <https://github.com/biothings/mygene.py>

version  |toolchain
---------|---------------
``3.1.0``|``intel/2019a``
``3.2.2``|``foss/2022a``

### MyMediaLite

MyMediaLite is a lightweight, multi-purpose library of recommender system algorithms.

*homepage*: <http://www.ismll.uni-hildesheim.de/mymedialite/>

version |toolchain
--------|---------------
``3.10``|``intel/2016b``
``3.11``|``intel/2016b``
``3.12``|``intel/2017a``

### mympingpong

A mpi4py based random pair pingpong network stress test.

*homepage*: <https://github.com/hpcugent/mympingpong>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.7.0``|``-Python-2.7.11``|``intel/2016a``
``0.7.1``|``-Python-2.7.12``|``intel/2016b``
``0.8.0``|``-Python-2.7.13``|``intel/2017a``
``0.8.0``|``-Python-2.7.14``|``intel/2018a``
``0.8.0``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``

### Myokit

Myokit is an open-source Python-based toolkit that facilitates modeling and simulation of cardiac cellular electrophysiology.

*homepage*: <http://myokit.org>

version   |toolchain
----------|----------------------------------
``1.32.0``|``foss/2020b``, ``fosscuda/2020b``

### mypy

Optional static typing for Python

*homepage*: <http://www.mypy-lang.org/>

version  |toolchain
---------|---------------
``0.4.5``|``intel/2016b``

### MySQL

MySQL is one of the world's most widely used open-source  relational database management system (RDBMS).

*homepage*: <http://www.mysql.com/>

version   |versionsuffix  |toolchain
----------|---------------|------------------
``5.6.26``|``-clientonly``|``GNU/4.9.3-2.25``
``5.7.21``|``-clientonly``|``GCCcore/6.4.0``

### MySQL-python

MySQL database connector for Python

*homepage*: <https://github.com/farcepest/MySQLdb1>

version  |versionsuffix                     |toolchain
---------|----------------------------------|---------------
``1.2.5``|``-Python-2.7.11``                |``intel/2016a``
``1.2.5``|``-Python-2.7.11-MariaDB-10.1.14``|``intel/2016a``

### mysqlclient

Python interface to MySQL

*homepage*: <https://github.com/PyMySQL/mysqlclient-python>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.3.7``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``

## N


[n2v](#n2v) - [NAG](#nag) - [NAGfor](#nagfor) - [NAMD](#namd) - [namedlist](#namedlist) - [nano](#nano) - [NanoComp](#nanocomp) - [nanocompore](#nanocompore) - [NanoFilt](#nanofilt) - [nanoflann](#nanoflann) - [nanoget](#nanoget) - [NanoLyse](#nanolyse) - [nanomath](#nanomath) - [nanomax-analysis-utils](#nanomax-analysis-utils) - [nanonet](#nanonet) - [NanoPlot](#nanoplot) - [nanopolish](#nanopolish) - [NanopolishComp](#nanopolishcomp) - [NanoStat](#nanostat) - [napari](#napari) - [NASM](#nasm) - [nauty](#nauty) - [NBO](#nbo) - [NCBI-Toolkit](#ncbi-toolkit) - [ncbi-vdb](#ncbi-vdb) - [NCCL](#nccl) - [NCCL-tests](#nccl-tests) - [ncdf4](#ncdf4) - [ncdu](#ncdu) - [NCIPLOT](#nciplot) - [NCL](#ncl) - [NCO](#nco) - [ncompress](#ncompress) - [ncurses](#ncurses) - [ncview](#ncview) - [nd2reader](#nd2reader) - [ne](#ne) - [NECI](#neci) - [NEdit](#nedit) - [Nek5000](#nek5000) - [Nektar++](#nektar++) - [neon](#neon) - [neptune-client](#neptune-client) - [Net-core](#net-core) - [netCDF](#netcdf) - [netCDF-C++](#netcdf-c++) - [netCDF-C++4](#netcdf-c++4) - [netCDF-Fortran](#netcdf-fortran) - [netcdf4-python](#netcdf4-python) - [netloc](#netloc) - [NetLogo](#netlogo) - [netMHC](#netmhc) - [netMHCII](#netmhcii) - [netMHCIIpan](#netmhciipan) - [netMHCpan](#netmhcpan) - [NetPIPE](#netpipe) - [NetPyNE](#netpyne) - [nettle](#nettle) - [networkTools](#networktools) - [networkx](#networkx) - [NeuroKit](#neurokit) - [NEURON](#neuron) - [NewHybrids](#newhybrids) - [Nextflow](#nextflow) - [NextGenMap](#nextgenmap) - [NEXUS-CL](#nexus-cl) - [nf-core-mag](#nf-core-mag) - [NFFT](#nfft) - [nghttp2](#nghttp2) - [nghttp3](#nghttp3) - [NGLess](#ngless) - [nglview](#nglview) - [NGS](#ngs) - [NGS-Python](#ngs-python) - [NGSadmix](#ngsadmix) - [NGSpeciesID](#ngspeciesid) - [ngspice](#ngspice) - [ngtcp2](#ngtcp2) - [NiBabel](#nibabel) - [nichenetr](#nichenetr) - [NIfTI](#nifti) - [nifti2dicom](#nifti2dicom) - [Nilearn](#nilearn) - [Nim](#nim) - [NIMBLE](#nimble) - [Ninja](#ninja) - [Nipype](#nipype) - [NLMpy](#nlmpy) - [nlohmann_json](#nlohmann_json) - [NLopt](#nlopt) - [NLTK](#nltk) - [nnU-Net](#nnu-net) - [Node-RED](#node-red) - [nodejs](#nodejs) - [Normaliz](#normaliz) - [nose-parameterized](#nose-parameterized) - [novoalign](#novoalign) - [NOVOPlasty](#novoplasty) - [npstat](#npstat) - [NRGLjubljana](#nrgljubljana) - [Nsight-Compute](#nsight-compute) - [Nsight-Systems](#nsight-systems) - [NSPR](#nspr) - [NSS](#nss) - [nsync](#nsync) - [ntCard](#ntcard) - [ntEdit](#ntedit) - [ntHits](#nthits) - [NTL](#ntl) - [NTPoly](#ntpoly) - [num2words](#num2words) - [numactl](#numactl) - [numba](#numba) - [numdiff](#numdiff) - [numexpr](#numexpr) - [numpy](#numpy) - [NVHPC](#nvhpc) - [nvofbf](#nvofbf) - [nvompi](#nvompi) - [NVSHMEM](#nvshmem) - [nvtop](#nvtop) - [NWChem](#nwchem) - [NxTrim](#nxtrim)


### n2v

Noise2Void - Learning Denoising from Single Noisy Images The field of image denoising is currently dominated by discriminative deep       learning methods that are trained on pairs of noisy input and clean target       images. Recently it has been shown that such methods can also be trained without clean targets. Instead, independent pairs of noisy images can be used, in an     approach known as NOISE2NOISE (N2N). Here, we introduce NOISE2VOID (N2V), a      training scheme that takes this idea one step further. It does not require noisy image pairs, nor clean target images. Consequently, N2V allows us to train       directly on the body of data to be denoised and can therefore be applied when    other methods cannot. Especially interesting is the application to biomedical    image data, where the acquisition of training targets, clean or noisy, is        frequently not possible. We compare the performance of N2V to approaches that    have either clean target images and/or noisy image pairs available. Intuitively, N2V cannot be expected to outperform methods that have more information          available during training. Still, we observe that the denoising performance of   NOISE2VOID drops in moderation and compares favorably to training-free denoising methods.

*homepage*: <https://github.com/juglab/n2v>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.3.2``|``-CUDA-11.3.1``|``foss/2021a``

### NAG

The worlds largest collection of robust, documented, tested and maintained numerical algorithms.

*homepage*: <http://www.nag.co.uk>

version|toolchain
-------|----------------------------------
``7.1``|``gompi/2022b``
``24`` |``PGI/17.4-GCC-6.4.0-2.28``
``26`` |``GCCcore/6.4.0``, ``intel/2018a``

### NAGfor

The checking compiler for improved code portability and detailed error reporting.

*homepage*: <http://www.nag.co.uk>

version   |toolchain
----------|----------
``6.2.14``|``system``
``7.1``   |``system``

### NAMD

NAMD is a parallel molecular dynamics code designed for high-performance simulation of  large biomolecular systems.

*homepage*: <https://www.ks.uiuc.edu/Research/namd/>

version |versionsuffix   |toolchain
--------|----------------|-------------------------------------------------------------------------------
``2.11``|``-mpi``        |``intel/2016a``
``2.12``|``-CUDA-8.0.61``|``foss/2016b``
``2.12``|``-mpi``        |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.13``|                |``fosscuda/2018b``
``2.13``|``-mpi``        |``foss/2018b``, ``foss/2019b``, ``intel/2018b``
``2.14``|                |``fosscuda/2019b``, ``fosscuda/2020b``
``2.14``|``-CUDA-11.3.1``|``foss/2021a``
``2.14``|``-mpi``        |``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2022a``, ``intel/2020a``

### namedlist

A Python object, similar to namedtuple, but for lists.

*homepage*: <https://gitlab.com/ericvsmith/namedlist>

version|toolchain
-------|------------------
``1.8``|``GCCcore/11.2.0``

### nano

a simple editor, inspired by Pico

*homepage*: <https://www.nano-editor.org/>

version|toolchain
-------|------------------
``6.4``|``GCCcore/11.3.0``
``7.0``|``GCCcore/11.3.0``
``7.1``|``GCCcore/12.2.0``
``7.2``|``GCCcore/12.2.0``

### NanoComp

Comparing runs of Oxford Nanopore sequencing data and alignments

*homepage*: <https://github.com/wdecoster/NanoComp>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.10.1``|``-Python-3.7.4``|``intel/2019b``
``1.13.1``|                 |``intel/2020b``

### nanocompore

Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples

*homepage*: <https://nanocompore.rna.rocks/>

version       |versionsuffix    |toolchain
--------------|-----------------|---------------
``1.0.0rc3-2``|``-Python-3.8.2``|``intel/2020a``

### NanoFilt

Filtering and trimming of long read sequencing data.

*homepage*: <https://github.com/wdecoster/nanofilt>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.5.0``|``-Python-3.6.6``|``foss/2018b``
``2.6.0``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``2.6.0``|``-Python-3.8.2``|``intel/2020a``
``2.8.0``|                 |``foss/2021b``

### nanoflann

nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies.

*homepage*: <https://github.com/jlblancoc/nanoflann>

version  |toolchain
---------|------------------
``1.4.0``|``GCCcore/10.3.0``

### nanoget

Functions to extract information from Oxford Nanopore sequencing data and alignments

*homepage*: <https://github.com/wdecoster/nanoget>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``1.12.1``|``-Python-3.7.4``|``intel/2019b``
``1.15.0``|                 |``intel/2020b``
``1.18.1``|                 |``foss/2021a``, ``foss/2022a``

### NanoLyse

Remove reads mapping to the lambda phage genome from a fastq file.

*homepage*: <https://github.com/wdecoster/nanolyse>

version  |toolchain
---------|--------------
``1.2.1``|``foss/2021a``

### nanomath

A few simple math function for other Oxford Nanopore processing scripts

*homepage*: <https://github.com/wdecoster/nanomath>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.23.1``|``-Python-3.7.4``|``intel/2019b``
``1.2.0`` |                 |``intel/2020b``
``1.2.1`` |                 |``foss/2021a``
``1.3.0`` |                 |``foss/2022a``

### nanomax-analysis-utils

A set of tools for handling and analysing data at the NanoMAX beamline.

*homepage*: <https://github.com/maxiv-science/nanomax-analysis-utils>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.4``  |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.4.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.4.4``|                 |``foss/2021b``

### nanonet

Nanonet provides recurrent neural network basecalling for Oxford Nanopore MinION data.

*homepage*: <https://github.com/nanoporetech/nanonet>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.0.0``|``-Python-2.7.13``|``intel/2017a``

### NanoPlot

Plotting suite for long read sequencing data and alignments

*homepage*: <https://github.com/wdecoster/NanoPlot>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``1.28.4``|``-Python-3.7.4``|``intel/2019b``
``1.33.0``|                 |``foss/2021a``, ``intel/2020b``

### nanopolish

Software package for signal-level analysis of Oxford Nanopore sequencing data.

*homepage*: <https://github.com/jts/nanopolish>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.9.2`` |                  |``intel/2018a``
``0.10.2``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.13.1``|``-Python-3.6.6`` |``foss/2018b``
``0.13.2``|``-Python-3.8.2`` |``foss/2020a``
``0.13.3``|                  |``foss/2020b``

### NanopolishComp

NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files

*homepage*: <https://adrienleger.com/NanopolishComp/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.6.11``|``-Python-3.8.2``|``foss/2020a``

### NanoStat

Calculate various statistics from a long read sequencing dataset in fastq,  bam or albacore sequencing summary format.

*homepage*: <https://github.com/wdecoster/nanostat>

version  |toolchain
---------|------------------------------
``1.6.0``|``foss/2021a``, ``foss/2022a``

### napari

napari is a fast, interactive, multi-dimensional image viewer for Python. It's   designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based           rendering), and the scientific Python stack (numpy, scipy).

*homepage*: <https://napari.org/>

version   |toolchain
----------|--------------
``0.4.15``|``foss/2021b``

### NASM

NASM: General-purpose x86 assembler

*homepage*: <https://www.nasm.us/>

version    |toolchain
-----------|--------------------------------------------------------------------------------------------------
``2.11.08``|``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``
``2.12.01``|``foss/2016a``, ``intel/2016a``
``2.12.02``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``, ``system``
``2.13.01``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``2.13.03``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.14.02``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.15.05``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``2.16.01``|``GCCcore/12.3.0``

### nauty

nauty and Traces are programs for computing automorphism groups of graphs and digraphs. They can also produce a canonical label.

*homepage*: <https://pallini.di.uniroma1.it/>

version   |toolchain
----------|--------------------
``2.6r12``|``GCC/8.2.0-2.31.1``
``2.7rc2``|``GCC/8.2.0-2.31.1``
``2.7rc5``|``GCC/8.3.0``

### NBO

The Natural Bond Orbital (NBO) program is a discovery tool for chemical insights from complex wavefunctions. 

*homepage*: <http://nbo.chem.wisc.edu/>

version|toolchain
-------|---------------
``1.1``|``intel/2016a``
``7.0``|``intel/2017b``

### NCBI-Toolkit

The NCBI Toolkit is a collection of utilities developed for the   production and distribution of GenBank, Entrez, BLAST, and related services   by the National Center for Biotechnology Information.

*homepage*: <http://ncbi.github.io/cxx-toolkit/>

version   |toolchain
----------|---------------
``18.0.0``|``intel/2017a``

### ncbi-vdb

The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for  using data in the INSDC Sequence Read Archives.

*homepage*: <https://github.com/ncbi/ncbi-vdb>

version    |toolchain
-----------|------------------------------------------------
``2.5.8-1``|``foss/2016b``, ``intel/2016a``
``2.7.0``  |``foss/2016b``
``2.8.2``  |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.9.1-1``|``foss/2018a``, ``intel/2018a``
``2.9.3``  |``foss/2018b``
``2.10.4`` |``gompi/2019b``
``2.10.7`` |``gompi/2020a``
``2.10.9`` |``gompi/2020b``, ``gompi/2021a``
``2.11.2`` |``gompi/2021b``
``3.0.0``  |``gompi/2021b``, ``gompi/2022a``
``3.0.2``  |``gompi/2022a``
``3.0.5``  |``gompi/2021a``, ``gompi/2022b``

### NCCL

The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs.

*homepage*: <https://developer.nvidia.com/nccl>

version    |versionsuffix      |toolchain
-----------|-------------------|------------------------------
``2.1.4``  |``-CUDA-9.0.176``  |``system``
``2.2.13`` |``-CUDA-9.2.148.1``|``system``
``2.3.7``  |                   |``fosscuda/2018b``
``2.4.2``  |                   |``gcccuda/2019a``
``2.4.8``  |                   |``gcccuda/2019b``
``2.8.3``  |``-CUDA-11.0.2``   |``GCCcore/9.3.0``, ``system``
``2.8.3``  |``-CUDA-11.1.1``   |``GCCcore/10.2.0``, ``system``
``2.9.9``  |``-CUDA-11.3.1``   |``system``
``2.10.3`` |``-CUDA-11.3.1``   |``GCCcore/10.3.0``
``2.10.3`` |``-CUDA-11.4.1``   |``GCCcore/11.2.0``
``2.10.3`` |``-CUDA-11.5.2``   |``GCCcore/11.2.0``
``2.11.4`` |                   |``gcccuda/2019b``
``2.12.12``|``-CUDA-11.7.0``   |``GCCcore/11.3.0``
``2.16.2`` |``-CUDA-12.0.0``   |``GCCcore/12.2.0``
``2.18.3`` |``-CUDA-12.1.1``   |``GCCcore/12.3.0``

### NCCL-tests

Tests check both the performance and the correctness of NCCL operations.

*homepage*: <https://github.com/NVIDIA/nccl-tests>

version   |versionsuffix   |toolchain
----------|----------------|----------------
``2.0.0`` |                |``gompic/2019b``
``2.13.6``|``-CUDA-11.7.0``|``gompi/2022a``

### ncdf4

ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files

*homepage*: <https://cran.r-project.org/web/packages/ncdf4>

version   |versionsuffix|toolchain
----------|-------------|-------------------------------
``1.16``  |``-R-3.4.0`` |``intel/2017a``
``1.16``  |``-R-3.4.3`` |``intel/2017b``
``1.16``  |``-R-3.4.4`` |``intel/2018a``
``1.16.1``|``-R-3.5.1`` |``foss/2018b``, ``intel/2018b``
``1.16.1``|``-R-3.6.0`` |``foss/2019a``
``1.17``  |             |``foss/2019b``
``1.17``  |``-R-4.0.0`` |``foss/2020a``
``1.17``  |``-R-4.0.3`` |``foss/2020b``
``1.17``  |``-R-4.1.0`` |``foss/2021a``

### ncdu

Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a  remote server where you don't have an entire graphical setup available, but it is a useful tool even on regular  desktop systems. Ncdu aims to be fast, simple and simple to use, and should be able to run in any minimal POSIX-like  environment with ncurses installed.

*homepage*: <https://dev.yorhel.nl/ncdu>

version   |toolchain
----------|------------------------------
``1.13``  |``GCCcore/7.3.0``
``1.14``  |``GCCcore/7.3.0``
``1.15.1``|``GCCcore/9.3.0``
``1.16``  |``GCC/10.3.0``, ``GCC/11.2.0``
``1.17``  |``GCC/11.3.0``

### NCIPLOT

NCIPLOT is a program for revealing non covalent interactions  based on the reduced density gradient. 

*homepage*: <https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html>

version         |toolchain
----------------|-----------------------
``4.0-20190718``|``iccifort/2019.5.281``
``4.0-20200106``|``iccifort/2019.5.281``
``4.0-20200624``|``GCC/11.2.0``

### NCL

NCL is an interpreted language designed specifically for scientific data analysis and visualization.

*homepage*: <https://www.ncl.ucar.edu>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------
``6.4.0``|``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.5.0``|``intel/2018a``
``6.6.2``|``foss/2018b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``intel/2018b``, ``intel/2019b``, ``intel/2020a``

### NCO

The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats,  including DAP, HDF4, and HDF5.

*homepage*: <https://github.com/nco/nco>

version  |toolchain
---------|-------------------------------
``4.6.0``|``foss/2016a``
``4.6.6``|``intel/2017a``
``4.7.0``|``intel/2017b``
``4.7.1``|``intel/2017b``
``4.7.4``|``foss/2017b``
``4.7.6``|``intel/2018a``
``4.7.9``|``foss/2018b``, ``intel/2018b``
``4.8.1``|``foss/2019a``
``4.9.3``|``foss/2019b``
``4.9.7``|``foss/2020a``, ``foss/2020b``
``5.0.1``|``foss/2021a``
``5.0.3``|``foss/2021b``, ``intel/2021b``
``5.0.6``|``intel/2019b``
``5.1.0``|``foss/2022a``
``5.1.3``|``foss/2021a``, ``foss/2022a``

### ncompress

Compress is a fast, simple LZW file compressor. Compress does not have the  highest compression rate, but it is one of the fastest programs to compress  data. Compress is the defacto standard in the UNIX community for compressing  files. 

*homepage*: <http://ncompress.sourceforge.net/>

version    |toolchain
-----------|-----------------
``4.2.4.4``|``GCCcore/6.4.0``

### ncurses

The Ncurses (new curses) library is a free software emulation of curses in  System V Release 4.0, and more. It uses Terminfo format, supports pads and  color and multiple highlights and forms characters and function-key mapping,  and has all the other SYSV-curses enhancements over BSD Curses. 

*homepage*: <https://www.gnu.org/software/ncurses/>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.9``|``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``, ``system``
``6.0``|``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``6.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``system``
``6.2``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``system``
``6.3``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``system``
``6.4``|``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### ncview

Ncview is a visual browser for netCDF format files.  Typically you would use ncview to get a quick and simple, push-button  look at your netCDF files. You can view simple movies of the data,  view along various dimensions, take a look at the actual data values,  change color maps, invert the data, etc.

*homepage*: <http://meteora.ucsd.edu/~pierce/ncview_home_page.html>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1.7``|``foss/2017b``, ``foss/2018b``, ``foss/2019b``, ``gompi/2019a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2019b``, ``iomkl/2018b``
``2.1.8``|``gompi/2020a``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``

### nd2reader

nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+.

*homepage*: <https://www.lighthacking.nl/nd2reader/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.6``|``-Python-2.7.14``|``intel/2017b``

### ne

ne is a free (GPL'd) text editor based on the POSIX standard that runs (we hope) on almost any UN*X machine. ne is simple to use for the beginner, but powerful and fully configurable for the wizard, and most sparing in its resource usage.

*homepage*: <http://ne.di.unimi.it/>

version  |toolchain
---------|---------------
``3.0.1``|``gimkl/2017a``

### NECI

Standalone NECI codebase designed for FCIQMC and other stochastic quantum chemistry methods.

*homepage*: <https://github.com/ghb24/NECI_STABLE>

version     |toolchain
------------|--------------
``20220711``|``foss/2022a``

### NEdit

NEdit is a multi-purpose text editor for the X Window System,  which combines a standard, simple to use, graphical user interface with the thorough  functionality and stability required by users who edit text eight hours a day.

*homepage*: <http://www.nedit.org>

version|versionsuffix |toolchain
-------|--------------|----------
``5.5``|``-Linux-x86``|``system``

### Nek5000

a fast and scalable high-order solver for computational fluid dynamics

*homepage*: <https://nek5000.mcs.anl.gov/>

version |toolchain
--------|---------------
``17.0``|``intel/2018a``

### Nektar++

Nektar++ is a tensor product based finite element package designed to allow one to construct efficient classical low polynomial order h-type solvers (where h is the size of the finite element) as well as higher p-order piecewise polynomial order solvers.

*homepage*: <https://www.nektar.info>

version  |toolchain
---------|--------------
``5.0.1``|``foss/2020b``

### neon

neon is an HTTP/1.1 and WebDAV client library, with a C interface. 

*homepage*: <https://github.com/notroj/neon>

version   |toolchain
----------|-----------------
``0.31.2``|``GCCcore/8.3.0``

### neptune-client

Neptune is an experiment tracking hub that brings organization and collaboration to your data science team. 

*homepage*: <https://docs.neptune.ai>

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------
``0.4.129``|``-Python-3.8.2``|``foss/2020a``
``0.10.5`` |                 |``foss/2020b``
``0.16.2`` |                 |``foss/2021a``, ``foss/2022a``

### Net-core

.NET Core is a free and open-source managed computer software framework for the Windows,  Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic

*homepage*: <https://dotnet.microsoft.com/>

version  |toolchain
---------|----------
``2.1.8``|``system``
``2.2.5``|``system``
``3.0.0``|``system``

### netCDF

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: <https://www.unidata.ucar.edu/software/netcdf/>

version    |versionsuffix   |toolchain
-----------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------
``4.3.3.1``|                |``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``4.4.0``  |                |``foss/2016a``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``4.4.1``  |                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.4.1.1``|                |``foss/2016b``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.4.1.1``|``-HDF5-1.10.1``|``foss/2017a``, ``intel/2017a``
``4.4.1.1``|``-HDF5-1.8.18``|``intel/2017a``
``4.4.1.1``|``-HDF5-1.8.19``|``foss/2017a``, ``foss/2017b``, ``intel/2017b``
``4.5.0``  |                |``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``
``4.6.0``  |                |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``4.6.1``  |                |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``4.6.2``  |                |``gompi/2019a``, ``iimpi/2019a``
``4.7.1``  |                |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``4.7.4``  |                |``gompi/2020a``, ``gompi/2020b``, ``gompic/2020a``, ``gompic/2020b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpic/2020b``, ``iompi/2020a``
``4.8.0``  |                |``gompi/2021a``, ``iimpi/2021a``
``4.8.1``  |                |``gompi/2021b``, ``iimpi/2021b``
``4.9.0``  |                |``gompi/2022a``, ``gompi/2022b``, ``iimpi/2022a``, ``iimpi/2022b``

### netCDF-C++

NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.

*homepage*: <http://www.unidata.ucar.edu/software/netcdf/>

version|toolchain
-------|--------------
``4.2``|``foss/2016a``

### netCDF-C++4

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: <https://www.unidata.ucar.edu/software/netcdf/>

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.3.0``|                |``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``4.3.0``|``-HDF5-1.8.18``|``intel/2017a``
``4.3.0``|``-HDF5-1.8.19``|``intel/2017b``
``4.3.1``|                |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``, ``iimpi/2022a``

### netCDF-Fortran

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: <https://www.unidata.ucar.edu/software/netcdf/>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.4.1``|                |``foss/2016b``, ``intel/2016b``
``4.4.2``|                |``intel/2016.02-GCC-4.9``
``4.4.3``|                |``foss/2016a``, ``intel/2016a``
``4.4.4``|                |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2018b``
``4.4.4``|``-HDF5-1.8.18``|``intel/2017a``
``4.4.4``|``-HDF5-1.8.19``|``foss/2017b``, ``intel/2017b``
``4.4.5``|                |``gompi/2019a``, ``iimpi/2019a``
``4.5.2``|                |``gompi/2019b``, ``gompi/2020a``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpic/2019b``
``4.5.3``|                |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``
``4.6.0``|                |``gompi/2022a``, ``gompi/2022b``, ``iimpi/2022a``

### netcdf4-python

Python/numpy interface to netCDF.

*homepage*: <https://unidata.github.io/netcdf4-python/>

version    |versionsuffix                |toolchain
-----------|-----------------------------|---------------------------------------------------
``1.2.9``  |``-Python-2.7.13``           |``intel/2017a``
``1.3.1``  |``-Python-3.6.3``            |``intel/2017b``
``1.3.1``  |``-Python-3.6.3-HDF5-1.8.19``|``intel/2017b``
``1.4.0``  |``-Python-3.6.2-HDF5-1.8.19``|``foss/2017b``
``1.4.0``  |``-Python-3.6.4``            |``intel/2018a``
``1.4.1``  |``-Python-2.7.15``           |``intel/2018b``
``1.4.1``  |``-Python-3.6.6``            |``foss/2018b``, ``intel/2018b``
``1.4.2``  |``-Python-3.6.4``            |``foss/2018a``
``1.4.3``  |``-Python-3.6.6``            |``foss/2018b``
``1.5.2``  |                             |``intel/2019a``
``1.5.3``  |``-Python-2.7.16``           |``foss/2019b``
``1.5.3``  |``-Python-3.7.4``            |``foss/2019b``, ``intel/2019b``
``1.5.3``  |``-Python-3.8.2``            |``foss/2020a``, ``intel/2020a``
``1.5.5.1``|                             |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.5.7``  |                             |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``1.6.1``  |                             |``foss/2022a``, ``intel/2022a``
``1.6.3``  |                             |``foss/2022b``

### netloc

The Portable Network Locality (netloc) software package provides  network topology discovery tools, and an abstract representation of those networks  topologies for a range of network types and configurations.  It is provided as a companion to the Portable Hardware Locality (hwloc) package.

*homepage*: <http://www.open-mpi.org/software/netloc>

version|toolchain
-------|-------------
``0.5``|``GCC/4.8.3``

### NetLogo

NetLogo is a multi-agent programmable modeling environment. It is used by tens of thousands of students, teachers and researchers worldwide. It also powers HubNet participatory simulations. It is authored by Uri Wilensky and developed at the CCL.

*homepage*: <https://ccl.northwestern.edu/netlogo/>

version  |versionsuffix|toolchain
---------|-------------|----------
``5.3.1``|``-64``      |``system``
``6.0.4``|``-64``      |``system``
``6.2.0``|``-64``      |``system``
``6.2.2``|``-64``      |``system``
``6.3.0``|``-64``      |``system``

### netMHC

NetMHC 4.0 predicts binding of peptides to a number of different  HLA alleles using artificial neural networks (ANN).

*homepage*: <https://www.cbs.dtu.dk/services/NetMHC/>

version |toolchain
--------|----------
``4.0a``|``system``

### netMHCII

NetMHCII 2.3 server predicts binding of peptides to HLA-DR,  HLA-DQ, HLA-DP and mouse MHC class II alleles using articial neuron networks.  Predictions can be obtained for 25 HLA-DR alleles, 20 HLA-DQ, 9 HLA-DP, and  7 mouse H2 class II alleles.

*homepage*: <https://www.cbs.dtu.dk/services/NetMHCII/>

version|toolchain
-------|----------
``2.3``|``system``

### netMHCIIpan

NetMHCIIpan 3.2 server predicts binding of peptides to MHC  class II molecules. The predictions are available for the three human MHC  class II isotypes HLA-DR, HLA-DP and HLA-DQ, as well as mouse molecules (H-2).

*homepage*: <https://www.cbs.dtu.dk/services/NetMHCIIpan/>

version|toolchain
-------|------------------------------------
``3.2``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

### netMHCpan

The NetMHCpan software predicts binding of peptides to any known MHC  molecule using artificial neural networks (ANNs).

*homepage*: <https://www.cbs.dtu.dk/services/NetMHCpan/>

version |toolchain
--------|----------
``4.0a``|``system``

### NetPIPE

NetPIPE is a protocol independent communication performance benchmark that visually represents  the network performance under a variety of conditions.

*homepage*: <https://netpipe.cs.ksu.edu/>

version  |toolchain
---------|--------------------------------
``5.1``  |``intel/2018a``
``5.1.4``|``gompi/2020b``, ``iimpi/2020b``

### NetPyNE

NetPyNE is an open-source Python package to facilitate the development, parallel simulation, analysis, and optimization of biological neuronal networks using the NEURON simulator.

*homepage*: <http://www.netpyne.org>

version    |toolchain
-----------|--------------
``1.0.2.1``|``foss/2021b``

### nettle

Nettle is a cryptographic library that is designed to fit easily  in more or less any context: In crypto toolkits for object-oriented  languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,  or even in kernel space.

*homepage*: <https://www.lysator.liu.se/~nisse/nettle/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------
``3.1.1``|``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``3.2``  |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``3.3``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017a``
``3.4``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``3.4.1``|``GCCcore/8.2.0``
``3.5.1``|``GCCcore/8.3.0``
``3.6``  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.7.2``|``GCCcore/10.3.0``
``3.7.3``|``GCCcore/11.2.0``
``3.8``  |``GCCcore/11.3.0``
``3.8.1``|``GCCcore/12.2.0``

### networkTools

Dynamical Network Analysis is a method of characterizing allosteric signalling through biomolecular complexes.

*homepage*: <http://faculty.scs.illinois.edu/schulten/software/networkTools/>

version|toolchain
-------|-------------
``2``  |``GCC/9.3.0``

### networkx

NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics, and functions of complex networks.

*homepage*: <https://pypi.python.org/pypi/networkx>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------
``1.1``  |``-Python-2.7.12``|``intel/2016b``
``1.11`` |``-Python-2.7.11``|``foss/2016a``
``1.11`` |``-Python-2.7.14``|``intel/2017b``
``1.11`` |``-Python-3.5.1`` |``foss/2016a``
``2.0``  |``-Python-2.7.14``|``intel/2017b``
``2.1``  |``-Python-2.7.14``|``intel/2017b``
``2.2``  |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``intel/2018b``
``2.2``  |``-Python-2.7.16``|``foss/2019b``
``2.2``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``2.3``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``2.4``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``2.4``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2.4``  |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``2.5``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2.5.1``|                  |``foss/2021a``
``2.6.2``|                  |``foss/2020b``
``2.6.3``|                  |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``2.8.4``|                  |``foss/2022a``, ``intel/2022a``
``2.8.8``|                  |``gfbf/2022b``
``3.0``  |                  |``gfbf/2022b``

### NeuroKit

NeuroKit is a Python module that provides high-level integrative functions with good and  flexible defaults, allowing users to focus on what’s important.

*homepage*: <https://neurokit.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.7``|``-Python-3.6.4``|``intel/2018a``

### NEURON

Empirically-based simulations of neurons and networks of neurons.

*homepage*: <https://www.neuron.yale.edu/neuron>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``7.4``  |                  |``intel/2016b``
``7.6.5``|``-Python-2.7.15``|``intel/2018b``
``7.8.2``|                  |``foss/2021b``

### NewHybrids

This implements a Gibbs sampler to estimate the posterior probability that genetically sampled individuals fall into each of a set of user-defined hybrid categories.

*homepage*: <https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs>

version      |toolchain
-------------|--------------
``1.1_Beta3``|``GCC/10.2.0``

### Nextflow

Nextflow is a reactive workflow framework and a programming DSL  that eases writing computational pipelines with complex data

*homepage*: <https://www.nextflow.io/>

version    |toolchain
-----------|----------
``19.04.0``|``system``
``19.07.0``|``system``
``19.12.0``|``system``
``20.01.0``|``system``
``20.04.1``|``system``
``20.10.0``|``system``
``21.03.0``|``system``
``21.08.0``|``system``
``21.10.6``|``system``
``22.04.0``|``system``
``22.10.0``|``system``
``22.10.1``|``system``
``22.10.5``|``system``
``22.10.6``|``system``
``23.04.2``|``system``

### NextGenMap

NextGenMap is a flexible highly sensitive short read mapping tool that  handles much higher mismatch rates than comparable algorithms while still outperforming  them in terms of runtime.

*homepage*: <http://cibiv.github.io/NextGenMap/>

version  |toolchain
---------|------------------------------
``0.5.5``|``GCC/11.2.0``, ``foss/2016b``

### NEXUS-CL

The NEXUS Class Library is a C++ library for parsing NEXUS files.

*homepage*: <https://ncl.sourceforge.net/>

version   |toolchain
----------|--------------------
``2.1.18``|``GCC/8.2.0-2.31.1``

### nf-core-mag

The Nextflow pipeline 'mag' ported to SimpleBuild/EESSI.

*homepage*: <https://nf-co.re/mag>

version     |toolchain
------------|--------------
``20221110``|``foss/2021a``

### NFFT

The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine  library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more  dimensions, of arbitrary input size, and of complex data.

*homepage*: <https://www-user.tu-chemnitz.de/~potts/nfft/>

version  |toolchain
---------|----------------------------------------------
``3.1.3``|``foss/2020b``, ``fosscuda/2020b``
``3.5.1``|``foss/2018b``, ``foss/2019a``
``3.5.2``|``foss/2020a``, ``foss/2021a``, ``foss/2021b``
``3.5.3``|``foss/2022a``

### nghttp2

This is an implementation of the Hypertext Transfer Protocol version 2 in C.  The framing layer of HTTP/2 is implemented as a reusable C library.  On top of that, we have implemented an HTTP/2 client, server and proxy.  We have also developed load test and benchmarking tools for HTTP/2.  An HPACK encoder and decoder are available as a public API.

*homepage*: <https://github.com/nghttp2/nghttp2>

version   |toolchain
----------|------------------------------
``1.48.0``|``GCC/11.2.0``, ``GCC/11.3.0``

### nghttp3

nghttp3 is an implementation of RFC 9114 HTTP/3  mapping over QUIC and RFC 9204 QPACK in C. It does not depend on any particular QUIC transport implementation.

*homepage*: <https://github.com/ngtcp2/nghttp3>

version  |toolchain
---------|--------------------------------------
``0.6.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### NGLess

NGLess is a domain-specific language for NGS (next-generation sequencing data) processing.

*homepage*: <https://ngless.embl.de/>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.1.1``|``-static-Linux64``|``system``

### nglview

IPython widget to interactively view molecular structures and trajectories.

*homepage*: <https://github.com/arose/nglview>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.7.0``|``-Python-3.7.2``|``intel/2019a``
``2.7.7``|``-Python-3.8.2``|``intel/2020a``
``3.0.3``|                 |``foss/2021a``

### NGS

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: <https://github.com/ncbi/ngs>

version   |versionsuffix      |toolchain
----------|-------------------|--------------------------------------
``1.2.3`` |                   |``foss/2016b``, ``intel/2016a``
``1.2.5`` |                   |``foss/2016b``
``1.3.0`` |                   |``GCCcore/6.4.0``, ``intel/2017a``
``2.9.1`` |``-Java-1.8.0_162``|``foss/2018a``, ``intel/2018a``
``2.9.3`` |``-Java-1.8``      |``foss/2018b``
``2.10.0``|``-Java-1.8``      |``GCCcore/8.2.0``
``2.10.0``|``-Java-11``       |``GCCcore/8.2.0``
``2.10.4``|``-Java-11``       |``GCCcore/8.3.0``
``2.10.5``|                   |``GCCcore/9.3.0``
``2.10.9``|                   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.11.2``|                   |``GCCcore/11.2.0``

### NGS-Python

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: <https://github.com/ncbi/ngs>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.2.3`` |``-Python-2.7.11``|``intel/2016a``
``2.9.3`` |``-Python-3.6.6`` |``intel/2018b``
``2.10.4``|``-Python-2.7.16``|``gompi/2019b``

### NGSadmix

NGSadmix is a tool for finding admixture proportions  from NGS data, based on genotype likelihoods.

*homepage*: <http://www.popgen.dk/software/index.php/NgsAdmix>

version|toolchain
-------|------------------
``32`` |``GCC/7.3.0-2.30``

### NGSpeciesID

NGSpeciesID is a tool for clustering and consensus forming of targeted ONT reads.

*homepage*: <https://github.com/ksahlin/NGSpeciesID>

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``0.1.1.1``|``-Python-3.7.4``|``foss/2019b``
``0.1.2.1``|                 |``foss/2021b``

### ngspice

Ngspice is a mixed-level/mixed-signal circuit simulator. Its code is based on three open source software packages: Spice3f5, Cider1b1 and Xspice. 

*homepage*: <https://ngspice.sourceforge.net>

version|toolchain
-------|--------------
``31`` |``foss/2019b``
``39`` |``foss/2022a``

### ngtcp2

'Call it TCP/2. One More Time.'  ngtcp2 project is an effort to implement RFC9000 QUIC protocol.

*homepage*: <https://github.com/ngtcp2/ngtcp2>

version  |toolchain
---------|------------------------------
``0.7.0``|``GCC/11.2.0``, ``GCC/11.3.0``

### NiBabel

NiBabel provides read/write access to some common medical and neuroimaging file formats,  including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT  as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and  annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI.

*homepage*: <https://nipy.github.io/nibabel>

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------------------------
``2.0.2``|``-Python-2.7.11``               |``intel/2016a``
``2.0.2``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``2.1.0``|``-Python-2.7.13``               |``intel/2017a``
``2.2.1``|``-Python-3.6.4``                |``intel/2018a``
``2.3.0``|``-Python-2.7.14``               |``foss/2017b``, ``intel/2017b``
``2.3.0``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``2.3.0``|``-Python-3.6.6``                |``foss/2018b``
``2.4.0``|                                 |``foss/2019a``, ``intel/2019a``
``2.5.1``|``-Python-3.6.6``                |``foss/2018b``
``3.1.0``|``-Python-3.7.4``                |``foss/2019b``, ``intel/2019b``
``3.2.0``|``-Python-3.8.2``                |``foss/2020a``
``3.2.1``|                                 |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``
``3.2.2``|                                 |``foss/2021b``
``4.0.2``|                                 |``foss/2022a``

### nichenetr

R implementation of the NicheNet method,  to predict active ligand-target links between interacting cells

*homepage*: <https://github.com/saeyslab/nichenetr>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.1.1-20230223``|``-R-4.2.1`` |``foss/2022a``

### NIfTI

Niftilib is a set of i/o libraries for reading and writing files in the nifti-1 data format.

*homepage*: <http://niftilib.sourceforge.net/>

version  |toolchain
---------|-----------------
``2.0.0``|``GCCcore/6.4.0``

### nifti2dicom

Nifti2Dicom is a conversion tool that converts 3D NIfTI files (and other formats supported by ITK, including Analyze, MetaImage Nrrd and VTK) to DICOM. Unlike other conversion tools, it can import a DICOM file that is used to import the patient and study DICOM tags, and allows you to edit the accession number and other DICOM tags, in order to create a valid DICOM that can be imported in a PACS.

*homepage*: <https://github.com/biolab-unige/nifti2dicom>

version   |toolchain
----------|--------------
``0.4.11``|``foss/2019b``

### Nilearn

Nilearn is a Python module for fast and simple statistical learning on NeuroImaging data.

*homepage*: <https://nilearn.github.io/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.5.2``|                  |``foss/2019a``, ``intel/2019a``
``0.5.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.5.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.7.0``|``-Python-3.8.2`` |``foss/2020a``
``0.7.1``|                  |``foss/2020b``

### Nim

Nim is a systems and applications programming language.

*homepage*: <https://nim-lang.org/>

version   |toolchain
----------|------------------
``0.18.0``|``GCCcore/6.4.0``
``0.19.2``|``GCCcore/7.3.0``
``1.0.0`` |``GCCcore/8.3.0``
``1.4.6`` |``GCCcore/10.2.0``
``1.4.8`` |``GCCcore/10.3.0``
``1.6.6`` |``GCCcore/11.2.0``

### NIMBLE

NIMBLE is a system for building and sharing analysis methods for statistical models,  especially for hierarchical models and computationally-intensive methods.

*homepage*: <https://r-nimble.org>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.7.0``|``-R-3.5.1`` |``foss/2018b``

### Ninja

Ninja is a small build system with a focus on speed.

*homepage*: <https://ninja-build.org/>

version   |toolchain
----------|------------------------------------------------------------------------------------
``1.8.2`` |``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``1.9.0`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``1.10.0``|``GCCcore/9.3.0``
``1.10.1``|``GCCcore/10.2.0``
``1.10.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.11.1``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### Nipype

Nipype is a Python project that provides a uniform interface to existing neuroimaging software and  facilitates interaction between these packages within a single workflow.

*homepage*: <https://nipype.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.0.2``|``-Python-3.6.4``|``intel/2018a``
``1.1.3``|``-Python-3.6.6``|``foss/2018b``
``1.4.2``|``-Python-3.7.4``|``intel/2019b``
``1.8.5``|                 |``foss/2021a``

### NLMpy

NLMpy is a Python package for the creation of neutral landscape models that are widely used  in the modelling of ecological patterns and processes across landscapes.

*homepage*: <https://pypi.org/project/nlmpy>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.1.5``|``-Python-3.7.4``|``intel/2019b``

### nlohmann_json

JSON for Modern C++

*homepage*: <https://github.com/nlohmann/json>

version   |toolchain
----------|--------------------------------------
``3.10.0``|``GCCcore/10.3.0``
``3.10.4``|``GCCcore/11.2.0``
``3.10.5``|``GCCcore/11.3.0``
``3.11.2``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### NLopt

NLopt is a free/open-source library for nonlinear optimization,  providing a common interface for a number of different free optimization routines  available online as well as original implementations of various other algorithms. 

*homepage*: <http://ab-initio.mit.edu/wiki/index.php/NLopt>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.2``|``GCC/5.4.0-2.26``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``2.6.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.2``|``GCCcore/10.2.0``
``2.7.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### NLTK

NLTK is a leading platform for building Python programs to work with human language data.

*homepage*: <https://www.nltk.org/>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``3.2.2``|``-Python-2.7.13``|``intel/2017a``
``3.2.4``|``-Python-2.7.13``|``intel/2017a``
``3.5``  |``-Python-3.7.4`` |``foss/2019b``
``3.5``  |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``3.6.7``|                  |``foss/2021a``
``3.7``  |                  |``foss/2021b``

### nnU-Net

nnU-Net is the first segmentation method that is designed to deal with the dataset diversity found  in the domain It condenses and automates the keys decisions for designing a successful segmentation pipeline for  any given dataset.

*homepage*: <https://github.com/MIC-DKFZ/nnUNet>

version  |toolchain
---------|----------------------------------
``1.7.0``|``foss/2020b``, ``fosscuda/2020b``

### Node-RED

Node-RED is a programming tool for wiring together hardware devices, APIs and online services  in new and interesting ways.

*homepage*: <http://nodered.org>

version   |toolchain
----------|--------------
``0.16.2``|``foss/2017a``

### nodejs

Node.js is a platform built on Chrome's JavaScript runtime   for easily building fast, scalable network applications. Node.js uses an   event-driven, non-blocking I/O model that makes it lightweight and efficient,   perfect for data-intensive real-time applications that run across distributed devices.

*homepage*: <http://nodejs.org>

version    |toolchain
-----------|-------------------------------------------------------
``4.4.7``  |``foss/2016a``
``6.10.3`` |``foss/2017a``
``8.9.4``  |``foss/2017a``
``10.15.1``|``foss/2018b``
``10.15.3``|``GCCcore/8.2.0``
``12.16.1``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``12.19.0``|``GCCcore/10.2.0``
``14.17.0``|``GCCcore/10.3.0``
``14.17.2``|``GCCcore/10.3.0``
``14.17.6``|``GCCcore/11.2.0``
``16.15.1``|``GCCcore/11.3.0``
``18.12.1``|``GCCcore/12.2.0``

### Normaliz

Normaliz is a open source tool for computations in affine monoids, vector configurations, rational polyhedra and rational cones. Normaliz now computes rational and algebraic polyhedra, i.e., polyhedra defined over real algebraic extensions of QQ.

*homepage*: <https://www.normaliz.uni-osnabrueck.de/>

version  |toolchain
---------|---------------
``3.6.3``|``intel/2018b``
``3.7.4``|``gompi/2019a``
``3.8.4``|``GCC/8.3.0``

### nose-parameterized

Parameterized testing with any Python test framework.

*homepage*: <hmat://github.com/wolever/nose-parameterized>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.5.0``|``-Python-2.7.11``|``foss/2016a``
``0.5.0``|``-Python-3.5.1`` |``foss/2016a``
``0.5.0``|``-Python-3.5.2`` |``intel/2016b``
``0.6.0``|``-Python-3.6.1`` |``intel/2017a``

### novoalign

NovoCraft is a software bundle.  NovoAlign:  Market’s leading aligner with fully packed features designed for mapping  of short reads onto a reference genome from Illumina, Ion Torrent, 454,  and Color Spance NGS platforms.  NovoAlignCS:  Leading aligner packed with features designed to fully support reads from  ABI SOLiD Color Space.  NovoSort:  Custom designed multi-threaded sort/merge tools for BAM files.  NovoMethyl:  It can analyse a set of alignments to identify methylated cytosine’s.  WARNING!  You can only use the sofware without a license (with some features disabled)  if you are a non-profit organisation and use is for your own research or for  use by students as part of their course. A license is required for use where  these programs are part of a service where a third party is billed.

*homepage*: <http://www.novocraft.com/>

version    |versionsuffix|toolchain
-----------|-------------|--------------
``3.09.00``|             |``system``
``3.09.01``|``-R-3.5.1`` |``foss/2018b``

### NOVOPlasty

NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.

*homepage*: <https://github.com/ndierckx/NOVOPlasty>

version|toolchain
-------|-----------------
``3.7``|``GCCcore/8.3.0``

### npstat

npstat implements some population genetics tests and estimators that can be applied  to pooled sequences from Next Generation Sequencing experiments.

*homepage*: <https://code.google.com/archive/p/npstat/>

version |toolchain
--------|-------------------------------
``0.99``|``foss/2016a``, ``intel/2016a``

### NRGLjubljana

NRG Ljubljana is an efficient implementation of the numerical renormalization group (NRG)   technique for solving quantum impurity problems that arise as simplified models of   magnetic impurities and as effective models in the dynamical mean field theory (DMFT)   approach to bulk correlated materials. 

*homepage*: <http://nrgljubljana.ijs.si/>

version     |toolchain
------------|--------------
``2.4.3.23``|``foss/2016b``

### Nsight-Compute

NVIDIA® Nsight™ Compute is an interactive kernel profiler for CUDA applications. It provides detailed performance metrics and API debugging via a user interface and a command line tool. In addition, its baseline feature allows users to compare results within the tool. Nsight Compute provides a customizable and data-driven user interface and fast metric collection, which can be extended with rules-based analysis scripts for post-processing results. The rules-based guided analysis helps isolate and fix memory throughput, compute, and occupancy inefficiencies in the target application.

*homepage*: <https://developer.nvidia.com/nsight-compute>

version     |toolchain
------------|----------
``2020.3.0``|``system``
``2021.2.0``|``system``

### Nsight-Systems

NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an application’s algorithm, help you select the largest opportunities to optimize, and tune to scale efficiently across any quantity of CPUs and GPUs in your computer; from laptops to DGX servers.

*homepage*: <https://developer.nvidia.com/nsight-compute>

version     |toolchain
------------|----------
``2020.5.1``|``system``

### NSPR

Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level  and libc-like functions.

*homepage*: <https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR>

version |toolchain
--------|--------------------------------------
``4.20``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``4.21``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.25``|``GCCcore/9.3.0``
``4.29``|``GCCcore/10.2.0``
``4.30``|``GCCcore/10.3.0``
``4.32``|``GCCcore/11.2.0``
``4.34``|``GCCcore/11.3.0``
``4.35``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### NSS

Network Security Services (NSS) is a set of libraries designed to support cross-platform development  of security-enabled client and server applications.

*homepage*: <https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS>

version   |toolchain
----------|------------------------------------
``3.39``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.42.1``|``GCCcore/8.2.0``
``3.45``  |``GCCcore/8.3.0``
``3.51``  |``GCCcore/9.3.0``
``3.57``  |``GCCcore/10.2.0``
``3.65``  |``GCCcore/10.3.0``
``3.69``  |``GCCcore/11.2.0``
``3.79``  |``GCCcore/11.3.0``
``3.85``  |``GCCcore/12.2.0``
``3.89.1``|``GCCcore/12.3.0``

### nsync

nsync is a C library that exports various synchronization primitives, such as mutexes

*homepage*: <https://github.com/google/nsync>

version   |toolchain
----------|------------------------------------------------------------------------------------------------
``1.24.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.25.0``|``GCCcore/11.3.0``

### ntCard

ntCard is a streaming algorithm for estimating the frequencies of k-mers in genomics datasets.

*homepage*: <https://www.bcgsc.ca/resources/software/ntcard>

version  |toolchain
---------|-----------------------------
``1.2.1``|``GCC/11.2.0``, ``GCC/8.3.0``

### ntEdit

ntEdit is a fast and scalable genomics application for polishing genome assembly drafts.

*homepage*: <https://github.com/bcgsc/ntEdit>

version  |toolchain
---------|--------------------------------------
``1.3.1``|``iccifort/2018.3.222-GCC-7.3.0-2.30``

### ntHits

ntHits is a method for identifying repeats in high-throughput DNA sequencing data.

*homepage*: <https://github.com/bcgsc/ntHits>

version  |toolchain
---------|--------------------------------------
``0.0.1``|``iccifort/2018.3.222-GCC-7.3.0-2.30``

### NTL

NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields.

*homepage*: <https://shoup.net/ntl/>

version   |toolchain
----------|------------------------------
``11.3.4``|``GCC/8.2.0-2.31.1``
``11.5.1``|``GCC/11.2.0``, ``GCC/11.3.0``

### NTPoly

is a massively parallel library for computing the functions of sparse, symmetric matrices based on  polynomial expansions. For sufficiently sparse matrices, most of the matrix functions  in NTPoly can be computed in linear time.

*homepage*: <https://github.com/william-dawson/NTPoly>

version  |toolchain
---------|-------------------------------
``2.5.1``|``foss/2020b``, ``intel/2020b``
``2.7.0``|``foss/2021a``, ``intel/2021a``
``2.7.1``|``intel/2020b``

### num2words

Modules to convert numbers to words. 42 --> forty-two

*homepage*: <https://github.com/savoirfairelinux/num2words>

version   |toolchain
----------|------------------
``0.5.10``|``GCCcore/10.3.0``

### numactl

The numactl program allows you to run your application program on specific cpu's and memory nodes. It does this by supplying a NUMA memory policy to the operating system before running your program. The libnuma library provides convenient ways for you to add NUMA memory policies into your own program.

*homepage*: <http://oss.sgi.com/projects/libnuma/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.0.9`` |``GCC/4.8.3``
``2.0.10``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``
``2.0.11``|``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``GCC/6.2.0-2.27``, ``GCC/6.3.0-2.27``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.0.12``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.0.13``|``GCCcore/10.2.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``2.0.14``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.0.16``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### numba

Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.

*homepage*: <https://numba.pydata.org/>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.24.0``|``-Python-2.7.11``|``intel/2016a``
``0.24.0``|``-Python-3.5.1`` |``intel/2016a``
``0.26.0``|``-Python-2.7.11``|``intel/2016a``
``0.32.0``|``-Python-2.7.13``|``intel/2017a``
``0.37.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``0.37.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.39.0``|``-Python-3.6.6`` |``foss/2018b``
``0.43.1``|                  |``intel/2019a``
``0.46.0``|                  |``foss/2019a``
``0.47.0``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``0.50.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.50.1``|``-Python-3.8.2`` |``foss/2020a``
``0.52.0``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.52.0``|``-Python-3.8.2`` |``foss/2020a``
``0.53.1``|                  |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``
``0.54.1``|                  |``foss/2021b``, ``intel/2021b``
``0.54.1``|``-CUDA-11.4.1``  |``foss/2021b``
``0.56.4``|                  |``foss/2022a``
``0.56.4``|``-CUDA-11.7.0``  |``foss/2022a``

### numdiff

Numdiff (which I will also write numdiff) is a little program that can be used to compare putatively  similar files line by line and field by field, ignoring small numeric differences or/and different numeric formats.  Equivalently, Numdiff is a program with the capability to appropriately compare files containing numerical fields  (and not only).

*homepage*: <https://www.nongnu.org/numdiff/>

version  |toolchain
---------|------------------
``5.9.0``|``GCCcore/10.2.0``

### numexpr

The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.  It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into  code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a  compiler at runtime.

*homepage*: <https://numexpr.readthedocs.io/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|---------------------------------------------------
``2.5.2``|``-Python-2.7.11``|``intel/2016a``
``2.6.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.6.1``|``-Python-3.5.2`` |``intel/2016b``
``2.6.4``|``-Python-2.7.13``|``foss/2017a``
``2.6.4``|``-Python-2.7.14``|``intel/2018a``
``2.6.4``|``-Python-3.5.1`` |``foss/2016a``
``2.6.4``|``-Python-3.6.1`` |``intel/2017a``
``2.6.4``|``-Python-3.6.3`` |``intel/2017b``
``2.6.4``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``2.6.5``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.6.5``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.7.0``|                  |``intel/2019a``
``2.7.1``|``-Python-2.7.16``|``intel/2019b``
``2.7.1``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``2.7.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.8.1``|                  |``foss/2021a``, ``intel/2021a``
``2.8.4``|                  |``foss/2022a``

### numpy

NumPy is the fundamental package for scientific computing with Python. It contains among other things:  a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran  code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,  NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be   defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases.

*homepage*: <http://www.numpy.org>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.8.2`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.9.2`` |``-Python-2.7.12``|``intel/2016b``
``1.10.4``|``-Python-2.7.11``|``intel/2016a``
``1.11.0``|``-Python-2.7.11``|``intel/2016a``
``1.12.1``|``-Python-3.5.2`` |``intel/2016b``
``1.13.0``|``-Python-2.7.13``|``intel/2017a``
``1.13.1``|``-Python-3.6.1`` |``intel/2017a``

### NVHPC

C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)

*homepage*: <https://developer.nvidia.com/hpc-sdk/>

version  |versionsuffix   |toolchain
---------|----------------|----------
``20.7`` |                |``system``
``20.9`` |                |``system``
``20.11``|                |``system``
``21.2`` |                |``system``
``21.3`` |                |``system``
``21.5`` |                |``system``
``21.7`` |                |``system``
``21.9`` |                |``system``
``21.11``|                |``system``
``22.7`` |``-CUDA-11.7.0``|``system``
``22.9`` |``-CUDA-11.7.0``|``system``
``22.11``|``-CUDA-11.7.0``|``system``
``23.1`` |``-CUDA-12.0.0``|``system``

### nvofbf

NVHPC based toolchain, including OpenMPI for MPI support, OpenBLAS (via FlexiBLAS for BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2022.07``|``system``

### nvompi

NVHPC based compiler toolchain, including OpenMPI for MPI support.

*homepage*: <(none)>

version    |toolchain
-----------|----------
``2022.07``|``system``

### NVSHMEM

NVSHMEM is a parallel programming interface based on OpenSHMEM that provides  efficient and scalable communication for NVIDIA GPU clusters. NVSHMEM creates a  global address space for data that spans the memory of multiple GPUs and can be  accessed with fine-grained GPU-initiated operations, CPU-initiated operations,  and operations on CUDA streams. 

*homepage*: <https://developer.nvidia.com/nvshmem>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.4.1``|``-CUDA-11.4.1``|``gompi/2021b``
``2.5.0``|``-CUDA-11.7.0``|``gompi/2022a``
``2.7.0``|``-CUDA-11.7.0``|``gompi/2022a``
``2.8.0``|``-CUDA-11.7.0``|``gompi/2022a``

### nvtop

htop-like GPU usage monitor

*homepage*: <https://github.com/Syllo/nvtop>

version  |toolchain
---------|--------------------------------------
``1.0.0``|``fosscuda/2018b``
``1.1.0``|``fosscuda/2019b``
``1.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.2.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.0.2``|``GCCcore/11.3.0``
``3.0.1``|``GCCcore/12.2.0``

### NWChem

NWChem aims to provide its users with computational chemistry tools that are scalable both in  their ability to treat large scientific computational chemistry problems efficiently, and in their use of available  parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.  NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all  combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties  and relativity.

*homepage*: <https://nwchemgit.github.io/>

version              |versionsuffix                                 |toolchain
---------------------|----------------------------------------------|--------------------------------
``6.6.revision27746``|``-2015-10-20-Python-2.7.12``                 |``iomkl/2017a``
``6.6.revision27746``|``-2015-10-20-patches-20170814-Python-2.7.13``|``intel/2017a``
``6.8.revision47``   |``-2017-12-14-Python-2.7.14``                 |``intel/2017b``, ``intel/2018a``
``7.0.0``            |``-Python-3.7.4``                             |``intel/2019b``
``7.0.2``            |                                              |``intel/2021a``, ``intel/2022a``
``7.0.2``            |``-Python-3.7.4``                             |``intel/2019b``

### NxTrim

NxTrim is a software to remove Nextera Mate Pair junction adapters and categorise reads according to the orientation implied by the adapter location.

*homepage*: <https://github.com/sequencing/NxTrim>

version  |toolchain
---------|--------------
``0.4.3``|``foss/2018a``

## O


[Oases](#oases) - [OBITools](#obitools) - [OBITools3](#obitools3) - [OCaml](#ocaml) - [occt](#occt) - [oceanspy](#oceanspy) - [OCNet](#ocnet) - [Octave](#octave) - [Octopus-vcf](#octopus-vcf) - [OGDF](#ogdf) - [olaFlow](#olaflow) - [olego](#olego) - [OMA](#oma) - [OmegaFold](#omegafold) - [onedrive](#onedrive) - [ONNX](#onnx) - [ONNX-Runtime](#onnx-runtime) - [ont-fast5-api](#ont-fast5-api) - [ont-guppy](#ont-guppy) - [ont-remora](#ont-remora) - [OOMPA](#oompa) - [OPARI2](#opari2) - [Open-Data-Cube-Core](#open-data-cube-core) - [OpenAI-Gym](#openai-gym) - [OpenBabel](#openbabel) - [OpenBLAS](#openblas) - [openCARP](#opencarp) - [OpenCensus-python](#opencensus-python) - [OpenCoarrays](#opencoarrays) - [OpenColorIO](#opencolorio) - [OpenCV](#opencv) - [OpenEXR](#openexr) - [OpenFace](#openface) - [OpenFAST](#openfast) - [OpenFOAM](#openfoam) - [OpenFOAM-Extend](#openfoam-extend) - [OpenFold](#openfold) - [OpenForceField](#openforcefield) - [OpenImageIO](#openimageio) - [OpenJPEG](#openjpeg) - [OpenKIM-API](#openkim-api) - [openkim-models](#openkim-models) - [OpenMM](#openmm) - [OpenMM-PLUMED](#openmm-plumed) - [OpenMMTools](#openmmtools) - [OpenMolcas](#openmolcas) - [OpenMPI](#openmpi) - [OpenMS](#openms) - [OpenNLP](#opennlp) - [OpenPGM](#openpgm) - [OpenPIV](#openpiv) - [openpyxl](#openpyxl) - [OpenRefine](#openrefine) - [OpenSceneGraph](#openscenegraph) - [OpenSees](#opensees) - [OpenSlide](#openslide) - [openslide-python](#openslide-python) - [OpenSSL](#openssl) - [OpenStackClient](#openstackclient) - [OPERA](#opera) - [OPERA-MS](#opera-ms) - [OptaDOS](#optados) - [OptiType](#optitype) - [OptiX](#optix) - [Optuna](#optuna) - [OR-Tools](#or-tools) - [ORCA](#orca) - [ORFfinder](#orffinder) - [OrfM](#orfm) - [OrthoFinder](#orthofinder) - [OrthoMCL](#orthomcl) - [Osi](#osi) - [OSPRay](#ospray) - [OSU-Micro-Benchmarks](#osu-micro-benchmarks) - [OTF2](#otf2) - [OVITO](#ovito) - [ownCloud](#owncloud) - [oxDNA](#oxdna) - [oxford_asl](#oxford_asl)


### Oases

Oases is a de novo transcriptome assembler designed to produce transcripts from  short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly.

*homepage*: <http://www.ebi.ac.uk/~zerbino/oases/>

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.2.08``|             |``foss/2016b``
``0.2.08``|``-kmer_101``|``intel/2017b``

### OBITools

The OBITools programs aims to help you to manipulate various data and sequence files in a  convenient way using the Unix command line interface. They follow the standard Unix interface for command line  program, allowing to chain a set of commands using the pipe mecanism.

*homepage*: <http://www.grenoble.prabi.fr/trac/OBITools>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.2.9`` |``-Python-2.7.11``|``foss/2016a``
``1.2.13``|``-Python-2.7.15``|``foss/2019a``

### OBITools3

A package for the management of analyses and data in DNA metabarcoding.

*homepage*: <https://metabarcoding.org/obitools3>

version    |toolchain
-----------|------------------
``3.0.1b8``|``GCCcore/10.2.0``

### OCaml

OCaml is a general purpose industrial-strength programming language  with an emphasis on expressiveness and safety. Developed for more than 20 years at Inria  it benefits from one of the most advanced type systems and supports functional,  imperative and object-oriented styles of programming.

*homepage*: <http://ocaml.org/>

version   |toolchain
----------|--------------
``4.02.3``|``foss/2016a``
``4.07.1``|``foss/2018b``

### occt

Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications.

*homepage*: <https://www.opencascade.com/>

version    |toolchain
-----------|------------------------------
``7.3.0p4``|``foss/2019b``
``7.5.0p1``|``foss/2021a``, ``foss/2022a``

### oceanspy

OceanSpy - A Python package to facilitate ocean model data analysis  and visualization.

*homepage*: <https://github.com/hainegroup/oceanspy>

version  |toolchain
---------|--------------
``0.2.0``|``foss/2022a``

### OCNet

Generate and analyze Optimal Channel Networks (OCNs): oriented spanning trees reproducing all scaling features characteristic of real, natural river networks. As such, they can be used in a variety of numerical experiments in the fields of hydrology, ecology and epidemiology.

*homepage*: <https://cran.r-project.org/web/packages/OCNet>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.0``|``-R-3.6.0`` |``foss/2019a``

### Octave

GNU Octave is a high-level interpreted language, primarily intended for numerical computations.

*homepage*: <https://www.gnu.org/software/octave/>

version  |versionsuffix|toolchain
---------|-------------|------------------------------------------------
``4.0.0``|             |``foss/2016a``
``4.0.3``|             |``intel/2016b``
``4.2.1``|             |``foss/2018a``, ``intel/2016b``, ``intel/2017a``
``4.2.1``|``-mt``      |``intel/2017a``
``4.2.2``|             |``foss/2018a``
``4.4.1``|             |``foss/2018b``
``5.1.0``|             |``foss/2019a``, ``foss/2019b``
``6.2.0``|             |``foss/2020b``
``7.1.0``|             |``foss/2021b``

### Octopus-vcf

Octopus is a mapping-based variant caller that implements several calling models within  a unified haplotype-aware framework. Octopus takes inspiration from particle filtering by constructing a  tree of haplotypes and dynamically pruning and extending the tree based on haplotype posterior probabilities  in a sequential manner. This allows octopus to implicitly consider all possible haplotypes at a given loci in  reasonable time.

*homepage*: <https://github.com/luntergroup/octopus>

version  |toolchain
---------|--------------
``0.7.1``|``foss/2020b``
``0.7.2``|``foss/2020b``

### OGDF

OGDF is a self-contained C++ library for graph algorithms, in particular for (but not restricted to) automatic graph drawing. It offers sophisticated algorithms and data structures to use within your own applications or scientific projects.

*homepage*: <https://ogdf.uos.de/>

version           |toolchain
------------------|--------------
``dogwood-202202``|``GCC/11.2.0``

### olaFlow

olaFlow CFD Suite is a free and open source project committed to bringing the latest advances for the simulation of wave dynamics to the OpenFOAM® and FOAM-extend communities.

*homepage*: <https://olaflow.github.io/>

version     |toolchain
------------|--------------
``20210820``|``foss/2021b``

### olego

OLego is a program specifically designed for de novo spliced mapping of mRNA-seq reads. OLego adopts a seed-and-extend scheme, and does not rely on a separate external mapper.

*homepage*: <https://github.com/chaolinzhanglab/olego>

version  |toolchain
---------|--------------
``1.1.9``|``GCC/11.2.0``

### OMA

The OMA ('Orthologous MAtrix') project is a method and database for the inference   of orthologs among complete genomes

*homepage*: <https://omabrowser.org/standalone/>

version  |toolchain
---------|----------
``2.1.1``|``system``

### OmegaFold

OmegaFold: High-resolution de novo Structure Prediction from Primary Sequence

*homepage*: <https://github.com/HeliXonProtein/OmegaFold>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.1.0``|``-CUDA-11.7.0``|``foss/2022a``

### onedrive

A free Microsoft OneDrive Client which supports OneDrive Personal, OneDrive for Business, OneDrive for Office365 and SharePoint.

*homepage*: <https://abraunegg.github.io/>

version   |toolchain
----------|------------------
``2.4.11``|``GCCcore/10.2.0``
``2.4.21``|``GCCcore/11.3.0``

### ONNX

Open Neural Network Exchange (ONNX) is an open ecosystem that empowers AI developers to choose the right tools as their project evolves. ONNX provides an open source format for AI models, both deep learning and traditional ML. It defines an extensible computation graph model, as well as definitions of built-in operators and standard data types. Currently we focus on the capabilities needed for inferencing (scoring).

*homepage*: <https://onnx.ai/>

version   |toolchain
----------|--------------
``1.11.0``|``foss/2021a``

### ONNX-Runtime

ONNX Runtime inference can enable faster customer experiences and lower costs, supporting models from deep learning frameworks such as PyTorch and TensorFlow/Keras as well as classical machine learning libraries such as scikit-learn, LightGBM, XGBoost, etc. ONNX Runtime is compatible with different hardware, drivers, and operating systems, and provides optimal performance by leveraging hardware accelerators where applicable alongside graph optimizations and transforms.

*homepage*: <https://onnxruntime.ai/>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``1.10.0``|                |``foss/2021a``
``1.10.0``|``-CUDA-11.3.1``|``foss/2021a``

### ont-fast5-api

ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore .fast5 file format.

*homepage*: <https://github.com/nanoporetech/ont_fast5_api>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``3.3.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``3.3.0``|``-Python-3.7.4``|``foss/2019b``
``4.0.0``|                 |``foss/2021a``
``4.0.2``|                 |``foss/2021a``, ``foss/2021b``
``4.1.1``|                 |``foss/2022b``

### ont-guppy

Guppy is a bioinformatics toolkit that enables real-time basecalling and several  post-processing features that works on Oxford Nanopore Technologies™ sequencing platforms.   For Research Use Only 

*homepage*: <https://community.nanoporetech.com/protocols/Guppy-protocol>

version  |versionsuffix   |toolchain
---------|----------------|----------
``6.4.6``|                |``system``
``6.4.6``|``-CUDA-11.7.0``|``system``
``6.4.8``|``-CUDA-11.7.0``|``system``

### ont-remora

Methylation/modified base calling separated from basecalling. Remora primarily provides an API to call modified bases for basecaller programs such as Bonito. Remora also provides the tools to prepare datasets, train modified base models and run simple inference.

*homepage*: <https://github.com/nanoporetech/remora>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.2``|                |``foss/2021a``
``0.1.2``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.0``|                |``foss/2021a``
``1.0.0``|``-CUDA-11.3.1``|``foss/2021a``

### OOMPA

OOMPA is a suite of R packages for the analysis of gene expression (RNA), proteomics profiling,  and other high throughput molecular biology data.

*homepage*: <https://r-forge.r-project.org/projects/oompa>

version  |versionsuffix|toolchain
---------|-------------|---------------
``3.1.2``|``-R-3.3.1`` |``intel/2016b``

### OPARI2

OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a  source-to-source instrumentation tool for OpenMP and hybrid codes.  It surrounds OpenMP directives and runtime library calls with calls  to the POMP2 measurement interface. 

*homepage*: <https://www.score-p.org>

version  |toolchain
---------|-------------------------------------------------------
``2.0``  |``foss/2016a``
``2.0.5``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.0.7``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### Open-Data-Cube-Core

The Open Data Cube Core provides an integrated gridded data analysis environment for decades of analysis ready earth observation satellite and related data from multiple satellite and other acquisition systems.

*homepage*: <https://github.com/opendatacube/datacube-core>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.8.3``|``-Python-3.8.2``|``foss/2020a``

### OpenAI-Gym

A toolkit for developing and comparing reinforcement learning algorithms.

*homepage*: <https://gym.openai.com>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.17.1``|``-Python-3.7.4``|``foss/2019b``
``0.21.0``|                 |``foss/2021b``

### OpenBabel

Open Babel is a chemical toolbox designed to speak the many  languages of chemical data. It's an open, collaborative project allowing anyone  to search, convert, analyze, or store data from molecular modeling, chemistry,  solid-state materials, biochemistry, or related areas.

*homepage*: <https://openbabel.org>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------
``2.3.2``|``-Python-2.7.11``|``foss/2016a``
``2.4.1``|``-Python-2.7.14``|``intel/2017b``
``2.4.1``|``-Python-2.7.15``|``intel/2018b``
``2.4.1``|``-Python-3.6.6`` |``intel/2018b``
``2.4.1``|``-Python-3.7.2`` |``intel/2019a``
``3.0.0``|``-Python-3.7.4`` |``gompi/2019b``
``3.1.1``|                  |``gompi/2021a``, ``gompi/2022a``
``3.1.1``|``-Python-3.7.4`` |``gompi/2019b``
``3.1.1``|``-Python-3.8.2`` |``iimpi/2020a``

### OpenBLAS

OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.

*homepage*: <https://xianyi.github.com/OpenBLAS/>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------
``0.2.9`` |``-LAPACK-3.5.0``|``GCC/4.8.3``
``0.2.12``|``-LAPACK-3.5.0``|``GCC/4.9.2``
``0.2.13``|``-LAPACK-3.5.0``|``GCC/4.8.4``, ``GCC/4.9.2``
``0.2.14``|``-LAPACK-3.5.0``|``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``
``0.2.15``|``-LAPACK-3.6.0``|``GCC/4.9.3-2.25``
``0.2.18``|``-LAPACK-3.6.0``|``GCC/4.9.4-2.25``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``
``0.2.18``|``-LAPACK-3.6.1``|``GCC/5.4.0-2.26``, ``gompi/2016.07``
``0.2.19``|``-LAPACK-3.6.1``|``gompi/2016.09``
``0.2.19``|``-LAPACK-3.7.0``|``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``
``0.2.20``|                 |``GCC/5.4.0-2.26``, ``GCC/6.4.0-2.28``, ``GCC/7.2.0-2.29``
``0.3.0`` |                 |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``
``0.3.1`` |                 |``GCC/7.3.0-2.30``
``0.3.3`` |                 |``GCC/8.2.0-2.31.1``
``0.3.4`` |                 |``GCC/8.2.0-2.31.1``
``0.3.5`` |                 |``GCC/8.2.0-2.31.1``
``0.3.6`` |                 |``GCC/8.3.0-2.32``
``0.3.7`` |                 |``GCC/8.3.0``
``0.3.8`` |                 |``GCC/9.2.0``
``0.3.9`` |                 |``GCC/9.3.0``
``0.3.12``|                 |``GCC/10.2.0``
``0.3.15``|                 |``GCC/10.3.0``
``0.3.17``|                 |``GCC/10.3.0``, ``GCC/11.2.0``
``0.3.18``|                 |``GCC/11.2.0``
``0.3.20``|                 |``GCC/11.2.0``, ``GCC/11.3.0``, ``NVHPC/22.7-CUDA-11.7.0``
``0.3.21``|                 |``GCC/12.2.0``
``0.3.23``|                 |``GCC/12.3.0``

### openCARP

openCARP is an open cardiac electrophysiology simulator for in-silico experiments.

*homepage*: <https://opencarp.org>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``3.2``|``-Python-3.8.2``|``foss/2020a``
``6.0``|                 |``foss/2020b``
``8.2``|                 |``foss/2021a``

### OpenCensus-python

OpenCensus for Python. OpenCensus provides a framework to measure a server's resource usage and collect performance stats. This repository contains Python related utilities and supporting software needed by OpenCensus.

*homepage*: <https://opencensus.io/>

version  |toolchain
---------|------------------
``0.8.0``|``GCCcore/11.2.0``

### OpenCoarrays

OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran.

*homepage*: <https://github.com/sourceryinstitute/opencoarrays>

version  |toolchain
---------|---------------
``1.9.0``|``gompi/2017a``
``2.2.0``|``gompi/2018b``
``2.8.0``|``gompi/2019b``
``2.9.2``|``gompi/2020a``

### OpenColorIO

OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production  with an emphasis on visual effects and computer animation.

*homepage*: <http://opencolorio.org/>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2018b``

### OpenCV

OpenCV (Open Source Computer Vision Library) is an open source computer vision  and machine learning software library. OpenCV was built to provide  a common infrastructure for computer vision applications and to accelerate  the use of machine perception in the commercial products.  Includes extra modules for OpenCV from the contrib repository.

*homepage*: <https://opencv.org/>

version   |versionsuffix            |toolchain
----------|-------------------------|----------------------------------------------------------------
``2.4.12``|                         |``intel/2016a``
``3.1.0`` |                         |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.0`` |``-Python-2.7.13``       |``intel/2017a``
``3.3.0`` |``-Python-2.7.14``       |``foss/2017b``, ``intel/2017b``
``3.3.0`` |``-Python-3.6.1``        |``intel/2017a``
``3.3.0`` |``-Python-3.6.3``        |``foss/2017b``, ``intel/2017b``
``3.4.1`` |``-Python-2.7.14``       |``foss/2018a``, ``intel/2018a``
``3.4.1`` |``-Python-3.6.4``        |``foss/2018a``, ``intel/2018a``
``3.4.1`` |``-Python-3.6.4-contrib``|``foss/2018a``
``3.4.5`` |``-Python-2.7.15``       |``foss/2018b``
``3.4.7`` |``-Python-2.7.15``       |``foss/2019a``, ``fosscuda/2019a``
``3.4.7`` |``-Python-3.7.2``        |``foss/2019a``, ``fosscuda/2019a``
``4.0.1`` |``-Python-2.7.15``       |``foss/2018b``
``4.0.1`` |``-Python-3.6.6``        |``foss/2018b``
``4.2.0`` |``-Python-3.7.4``        |``foss/2019b``, ``fosscuda/2019b``
``4.2.0`` |``-Python-3.8.2``        |``foss/2020a``
``4.2.0`` |``-Python-3.8.2-contrib``|``foss/2020a``
``4.5.1`` |``-contrib``             |``foss/2020b``, ``fosscuda/2020b``
``4.5.3`` |``-CUDA-11.3.1-contrib`` |``foss/2021a``
``4.5.3`` |``-contrib``             |``foss/2021a``
``4.5.5`` |``-CUDA-11.4.1-contrib`` |``foss/2021b``
``4.5.5`` |``-contrib``             |``foss/2021b``
``4.6.0`` |``-CUDA-11.7.0-contrib`` |``foss/2022a``
``4.6.0`` |``-contrib``             |``foss/2022a``

### OpenEXR

OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic  for use in computer imaging applications

*homepage*: <https://www.openexr.com/>

version  |toolchain
---------|--------------------------------------
``2.2.0``|``intel/2016b``, ``intel/2017a``
``2.3.0``|``foss/2018b``, ``intel/2018b``
``2.4.0``|``GCCcore/8.3.0``
``2.4.1``|``GCCcore/9.3.0``
``2.5.5``|``GCCcore/10.2.0``
``3.0.1``|``GCCcore/10.3.0``
``3.1.1``|``GCCcore/11.2.0``
``3.1.5``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``3.1.7``|``GCCcore/12.3.0``

### OpenFace

OpenFace – a state-of-the art tool intended for facial landmark detection, head pose estimation, facial action unit recognition, and eye-gaze estimation.

*homepage*: <https://github.com/TadasBaltrusaitis/OpenFace>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``2.2.0``|                |``foss/2021a``
``2.2.0``|``-CUDA-11.3.1``|``foss/2021a``

### OpenFAST

OpenFAST is a wind turbine simulation tool which builds on FAST v8. FAST.Farm extends the capability of OpenFAST to simulate multi-turbine wind farms

*homepage*: <http://openfast.readthedocs.io/>

version  |toolchain
---------|--------------
``3.0.0``|``foss/2021b``

### OpenFOAM

OpenFOAM is a free, open source CFD software package.  OpenFOAM has an extensive range of features to solve anything from complex fluid flows  involving chemical reactions, turbulence and heat transfer,  to solid dynamics and electromagnetics.

*homepage*: <https://www.openfoam.com/>

version         |versionsuffix|toolchain
----------------|-------------|-------------------------------------------------
``2.2.2``       |             |``intel/2016a``, ``intel/2017a``, ``intel/2018a``
``2.2.x``       |             |``intel/2019a``
``2.3.1``       |             |``intel/2017a``, ``intel/2019b``
``2.4.0``       |             |``intel/2017a``, ``intel/2019a``
``3.0.0``       |             |``foss/2016a``
``3.0.1``       |             |``intel/2016b``, ``intel/2018a``
``4.0``         |             |``foss/2016b``, ``intel/2016b``
``4.1``         |             |``foss/2016b``, ``intel/2017a``
``4.x-20170904``|             |``intel/2016b``
``5.0``         |             |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0-20180108``|             |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``6``           |             |``foss/2018b``, ``foss/2019b``, ``intel/2018a``
``7``           |             |``foss/2019b``
``7``           |``-20200508``|``foss/2019b``
``8``           |             |``foss/2020a``, ``foss/2020b``
``8``           |``-20210316``|``foss/2020b``
``9``           |             |``foss/2021a``, ``intel/2021a``
``10``          |             |``foss/2022a``
``10``          |``-20230119``|``foss/2022a``
``v1606+``      |             |``foss/2018b``
``v1612+``      |             |``foss/2018b``
``v1712``       |             |``foss/2017b``, ``intel/2017b``
``v1806``       |             |``foss/2018b``
``v1812``       |             |``foss/2018b``
``v1906``       |             |``foss/2019b``
``v1912``       |             |``foss/2019b``, ``intel/2019b``
``v2006``       |             |``foss/2020a``, ``intel/2020a``
``v2012``       |             |``foss/2020a``
``v2106``       |             |``foss/2021a``
``v2112``       |             |``foss/2020b``, ``foss/2021b``
``v2206``       |             |``foss/2022a``

### OpenFOAM-Extend

OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.

*homepage*: <http://www.extend-project.de/>

version         |versionsuffix     |toolchain
----------------|------------------|---------------------------------
``3.1``         |                  |``gimkl/2.11.5``
``3.2``         |                  |``gimkl/2.11.5``, ``intel/2016a``
``4.0``         |                  |``intel/2017a``
``4.0``         |``-Python-2.7.16``|``intel/2019b``
``4.1-20191120``|``-Python-2.7.16``|``intel/2019b``
``4.1-20200408``|``-Python-2.7.16``|``foss/2019b``

### OpenFold

A faithful PyTorch reproduction of DeepMind's AlphaFold 2

*homepage*: <https://github.com/aqlaboratory/openfold>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.0.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.1``|``-CUDA-11.3.1``|``foss/2021a``

### OpenForceField

Simulation and Parameter Estimation in Geophysics  - A python package for simulation and gradient based parameter estimation in the context of geophysical applications.

*homepage*: <https://github.com/openforcefield/openforcefield/releases>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.0``|``-Python-3.8.2``|``intel/2020a``

### OpenImageIO

OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities,  and applications.

*homepage*: <https://openimageio.org/>

version     |toolchain
------------|--------------------------------
``1.6.17``  |``intel/2016b``
``1.7.17``  |``intel/2017a``
``1.8.16``  |``foss/2018b``, ``intel/2018b``
``2.0.12``  |``gompi/2019b``, ``iimpi/2019b``
``2.1.12.0``|``gompi/2020a``, ``iimpi/2020a``
``2.3.17.0``|``GCC/11.3.0``

### OpenJPEG

OpenJPEG is an open-source JPEG 2000 codec written in  C language. It has been developed in order to promote the use of JPEG 2000,  a still-image compression standard from the Joint Photographic Experts Group  (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as  a JPEG 2000 Reference Software.

*homepage*: <https://www.openjpeg.org/>

version  |toolchain
---------|-----------------------------------------------------------------------------
``2.1``  |``GCCcore/6.4.0``
``2.3.0``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.3.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.4.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.5.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### OpenKIM-API

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This SimpleBuild only installs the API, the models have to be installed by the user by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: <https://openkim.org/>

version  |toolchain
---------|------------------------------------------------
``1.9.2``|``foss/2016b``, ``foss/2017b``
``1.9.7``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``

### openkim-models

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This SimpleBuild installs the models.  The API itself is in the kim-api package.  

*homepage*: <https://openkim.org/>

version     |toolchain
------------|------------------------------------------------------------------------------------
``20190725``|``GCCcore/10.2.0``, ``foss/2019a``, ``foss/2019b``, ``intel/2019a``, ``intel/2019b``
``20210128``|``GCCcore/10.2.0``

### OpenMM

OpenMM is a toolkit for molecular simulation.

*homepage*: <https://openmm.org>

version  |versionsuffix                  |toolchain
---------|-------------------------------|---------------------------------------------------
``7.1.1``|``-Python-3.6.3``              |``intel/2017b``
``7.4.1``|``-Python-3.7.4``              |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``7.4.2``|``-Python-3.8.2``              |``intel/2020a``
``7.5.0``|                               |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``7.5.0``|``-Python-3.8.2``              |``fosscuda/2020a``, ``intel/2020a``
``7.5.1``|                               |``foss/2020b``, ``fosscuda/2020b``
``7.5.1``|``-CUDA-11.3.1-DeepMind-patch``|``foss/2021a``
``7.5.1``|``-CUDA-11.4.1-DeepMind-patch``|``foss/2021b``
``7.5.1``|``-DeepMind-patch``            |``foss/2021a``, ``foss/2021b``
``7.7.0``|                               |``foss/2021a``, ``foss/2022a``
``7.7.0``|``-CUDA-11.3.1``               |``foss/2021a``
``8.0.0``|                               |``foss/2022a``
``8.0.0``|``-CUDA-11.7.0``               |``foss/2022a``

### OpenMM-PLUMED

This project provides a connection between OpenMM and PLUMED. It allows you to bias or analyze an OpenMM simulation based on collective variables.

*homepage*: <https://github.com/openmm/openmm-plumed/>

version|versionsuffix    |toolchain
-------|-----------------|---------------
``1.0``|``-Python-3.8.2``|``intel/2020a``

### OpenMMTools

A batteries-included toolkit for the GPU-accelerated OpenMM molecular simulation engine. openmmtools is a Python library layer that sits on top of OpenMM to provide access to a variety of useful tools  for building full-featured molecular simulation packages. 

*homepage*: <https://github.com/choderalab/openmmtools>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.20.0``|``-Python-3.8.2``|``intel/2020a``

### OpenMolcas

OpenMolcas is a quantum chemistry software package.

*homepage*: <https://gitlab.com/Molcas/OpenMolcas>

version  |versionsuffix         |toolchain
---------|----------------------|--------------------------------
``18.09``|``-Python-3.6.6``     |``intel/2018b``
``20.10``|``-Python-3.8.2``     |``intel/2020a``
``20.10``|``-Python-3.8.2-noGA``|``intel/2020a``
``21.06``|                      |``intel/2021a``, ``iomkl/2021a``
``22.06``|                      |``intel/2022a``
``22.10``|                      |``intel/2022a``
``22.10``|``-noGA``             |``intel/2022a``

### OpenMPI

The Open MPI Project is an open source MPI implementation.

*homepage*: <http://www.open-mpi.org/>

version   |versionsuffix   |toolchain
----------|----------------|--------------------------------------------------------------------------------------------------------------------
``1.6.5`` |                |``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``
``1.6.5`` |``-no-OFED``    |``GCC/4.8.1``, ``GCC/4.8.2``
``1.7.3`` |                |``GCC/4.8.2``
``1.8.1`` |                |``GCC/4.8.3``
``1.8.3`` |                |``GCC/4.9.2``
``1.8.4`` |                |``GCC/4.8.4``, ``GCC/4.9.2``
``1.8.5`` |                |``GNU/4.9.2-2.25``
``1.8.6`` |                |``GNU/4.9.3-2.25``
``1.8.8`` |                |``GNU/4.9.3-2.25``
``1.10.1``|                |``GCC/4.9.3-2.25``
``1.10.2``|                |``GCC/4.9.3-2.25``, ``GCC/5.3.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``
``1.10.3``|                |``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``1.10.4``|                |``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``
``2.0.0`` |                |``GCC/5.2.0``
``2.0.1`` |                |``GCC/6.2.0-2.27``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``
``2.0.2`` |                |``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``2.0.2`` |``-opa``        |``GCC/6.3.0-2.27``
``2.1.0`` |                |``GCC/6.3.0-2.28``
``2.1.1`` |                |``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``2.1.2`` |                |``GCC/6.4.0-2.28``, ``gcccuda/2018a``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2.1.3`` |                |``iccifort/2018.2.199-GCC-6.4.0-2.28``
``3.0.0`` |                |``GCC/7.2.0-2.29``
``3.1.0`` |                |``GCC/7.3.0-2.30``
``3.1.1`` |                |``GCC/7.3.0-2.30``, ``gcccuda/2018b``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``3.1.2`` |                |``GCC/8.2.0-2.31.1``
``3.1.3`` |                |``GCC/8.2.0-2.31.1``, ``gcccuda/2019a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``3.1.4`` |                |``GCC/8.3.0``, ``GCC/8.3.0-2.32``, ``gcccuda/2019b``, ``iccifort/2019.5.281``
``4.0.0`` |                |``GCC/8.2.0-2.31.1``
``4.0.0`` |``-hpcx``       |``GCC/8.2.0-2.31.1``
``4.0.1`` |                |``GCC/8.3.0-2.32``
``4.0.2`` |                |``GCC/9.2.0-2.32``
``4.0.3`` |                |``GCC/9.3.0``, ``gcccuda/2020a``, ``iccifort/2020.1.217``, ``iccifortcuda/2020a``
``4.0.5`` |                |``GCC/10.2.0``, ``GCC/9.3.0``, ``gcccuda/2020b``, ``iccifort/2020.4.304``
``4.0.5`` |``-CUDA-11.2.1``|``NVHPC/21.2``
``4.0.6`` |                |``GCC/10.3.0``
``4.0.7`` |                |``GCC/10.3.0``
``4.1.0`` |                |``GCC/10.2.0``
``4.1.1`` |                |``GCC/10.3.0``, ``GCC/11.2.0``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``
``4.1.2`` |                |``GCC/10.2.0``, ``GCC/11.2.0``
``4.1.4`` |                |``GCC/11.3.0``, ``GCC/12.2.0``, ``NVHPC/22.7-CUDA-11.7.0``
``4.1.5`` |                |``GCC/12.2.0``, ``GCC/12.3.0``
``system``|                |``GCC/system-2.29``

### OpenMS

As part of the deNBI Center for integrative Bioinformatics, OpenMS offers an open-source software  C++ library (+ python bindings) for LC/MS data management and analyses. It provides an infrastructure  for the rapid development of mass spectrometry related software as well as a rich toolset built on top of it.

*homepage*: <https://www.openms.de/>

version  |toolchain
---------|--------------
``2.4.0``|``foss/2018b``

### OpenNLP

The Apache OpenNLP library is a machine learning based toolkit for the processing of  natural language text.

*homepage*: <https://opennlp.apache.org/>

version  |toolchain
---------|----------
``1.8.1``|``system``

### OpenPGM

OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM)  specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol  that enables receivers to detect loss, request retransmission of lost data, or notify an application  of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible  for ensuring all data is received, absolving the sender of reception responsibility.

*homepage*: <http://code.google.com/p/openpgm/>

version    |toolchain
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.2.122``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

### OpenPIV

OpenPIV is an open source Particle Image Velocimetry analysis software

*homepage*: <http://www.openpiv.net>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.21.8``|``-Python-3.7.4``|``intel/2019b``

### openpyxl

A Python library to read/write Excel 2010 xlsx/xlsm files

*homepage*: <https://openpyxl.readthedocs.io>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``2.6.2`` |                  |``GCCcore/8.2.0``
``2.6.4`` |``-Python-2.7.16``|``GCCcore/8.3.0``
``3.0.3`` |``-Python-3.7.4`` |``GCCcore/8.3.0``
``3.0.7`` |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.0.9`` |                  |``GCCcore/11.2.0``
``3.0.10``|                  |``GCCcore/11.3.0``
``3.1.2`` |                  |``GCCcore/12.2.0``

### OpenRefine

OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it,  and augment it with data coming from the web.

*homepage*: <https://github.com/OpenRefine/OpenRefine>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``2.7``  |``-Java-1.8.0_144``|``system``
``3.4.1``|``-Java-11``       |``system``

### OpenSceneGraph

The OpenSceneGraph is an open source high performance 3D graphics toolkit,  used by application developers in fields such as visual simulation, games, virtual reality,  scientific visualization and modelling. Written entirely in Standard C++ and OpenGL it runs  on all Windows platforms, OSX, GNU/Linux, IRIX, Solaris, HP-Ux, AIX and FreeBSD operating  systems. The OpenSceneGraph is now well established as the world leading scene graph  technology, used widely in the vis-sim, space, scientific, oil-gas, games and virtual  reality industries.

*homepage*: <http://www.openscenegraph.org/>

version  |toolchain
---------|----------------------------------------------
``3.6.5``|``foss/2021a``, ``foss/2021b``, ``foss/2022a``

### OpenSees

Open System for Earthquake Engineering Simulation

*homepage*: <https://opensees.berkeley.edu/index.php>

version  |versionsuffix             |toolchain
---------|--------------------------|---------------
``3.2.0``|``-Python-3.8.2``         |``intel/2020a``
``3.2.0``|``-Python-3.8.2-parallel``|``intel/2020a``

### OpenSlide

OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).

*homepage*: <https://openslide.org/>

version  |versionsuffix  |toolchain
---------|---------------|---------------------------------------------------------
``3.4.1``|               |``GCCcore/11.2.0``, ``GCCcore/8.2.0``
``3.4.1``|``-largefiles``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``

### openslide-python

OpenSlide Python is a Python interface to the OpenSlide library.

*homepage*: <https://github.com/openslide/openslide-python>

version  |toolchain
---------|--------------------------------------
``1.1.1``|``GCCcore/8.2.0``
``1.1.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.0``|``GCCcore/11.3.0``

### OpenSSL

The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,  and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)  protocols as well as a full-strength general purpose cryptography library. 

*homepage*: <https://www.openssl.org/>

version   |toolchain
----------|---------------------------------------------------------------------
``1.0``   |``system``
``1.0.1f``|``GCC/4.8.2``
``1.0.1k``|``GCC/4.9.2``
``1.0.1s``|``foss/2016a``, ``intel/2016a``
``1.0.2g``|``GCCcore/4.9.3``
``1.0.2h``|``foss/2016.04``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.1``   |``system``
``1.1.0c``|``GCC/5.4.0-2.26``
``1.1.0e``|``intel/2016b``
``1.1.0h``|``GCCcore/7.3.0``
``1.1.1b``|``GCCcore/8.2.0``
``1.1.1d``|``GCCcore/8.3.0``
``1.1.1e``|``GCCcore/9.3.0``
``1.1.1h``|``GCCcore/10.2.0``
``1.1.1k``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.1.1n``|``GCCcore/11.3.0``
``1.1.1q``|``GCCcore/10.3.0``

### OpenStackClient

OpenStackClient (aka OSC) is a command-line client for OpenStack that brings the command set for Compute, Identity, Image, Network, Object Store and Block        Storage APIs together in a single shell with a uniform command structure.

*homepage*: <https://docs.openstack.org/python-openstackclient/latest/>

version  |toolchain
---------|------------------
``5.5.0``|``GCCcore/10.2.0``
``5.8.0``|``GCCcore/11.2.0``
``6.0.0``|``GCCcore/12.2.0``

### OPERA

An optimal genome scaffolding program

*homepage*: <https://sourceforge.net/projects/operasf>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------
``2.0.6``|``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``

### OPERA-MS

OPERA-MS is a hybrid metagenomic assembler which combines the  advantages of short and long-read technologies to provide high quality  assemblies, addressing issues of low contiguity for short-read only assemblies,  and low base-pair quality for long-read only assemblies.

*homepage*: <https://github.com/CSB5/OPERA-MS>

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``0.9.0-20200802``|``-Python-3.8.2``|``foss/2020a``

### OptaDOS

OptaDOS is a program for calculating core-electron and low-loss electron energy loss spectra (EELS) and optical spectra along with total-, projected- and joint-density of electronic states (DOS) from single-particle eigenenergies and dipole transition coefficients.

*homepage*: <http://www.optados.org/>

version    |toolchain
-----------|--------------
``1.2.380``|``GCC/11.3.0``

### OptiType

OptiType is a novel HLA genotyping algorithm based on integer linear programming,  capable of producing accurate 4-digit HLA genotyping predictions from NGS data  by simultaneously selecting all major and minor HLA Class I alleles. 

*homepage*: <https://github.com/FRED-2/OptiType>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.3.2``|``-Python-2.7.15``|``foss/2018b``
``1.3.2``|``-Python-3.6.6`` |``foss/2018b``

### OptiX

OptiX is NVIDIA SDK for simple ray tracing performance.  It provides a simple framework for accessing the GPU’s massive ray tracing  power using state-of-the-art GPU algorithms.

*homepage*: <https://developer.nvidia.com/optix>

version  |toolchain
---------|------------------
``3.8.0``|``GNU/4.9.3-2.25``
``3.9.0``|``GNU/4.9.3-2.25``
``6.5.0``|``system``
``7.2.0``|``system``

### Optuna

Optuna is an automatic hyperparameter optimization software framework, particularly designed for machine learning. It features an imperative, define-by-run style user API. Thanks to our define-by-run API, the code written with Optuna enjoys high modularity, and the user of Optuna can dynamically construct the search spaces for the hyperparameters.

*homepage*: <https://optuna.org/>

version   |toolchain
----------|--------------
``2.9.1`` |``foss/2021a``
``2.10.0``|``foss/2021b``
``3.1.0`` |``foss/2022a``

### OR-Tools

Google Optimization Tools (a.k.a., OR-Tools) is an open-source, fast and portable software suite for solving combinatorial optimization problems.

*homepage*: <https://developers.google.com/optimization/>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``7.1``|``-Python-3.7.2``|``foss/2019a``

### ORCA

ORCA is a flexible, efficient and simple-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects.

*homepage*: <https://orcaforum.kofo.mpg.de>

version               |versionsuffix     |toolchain
----------------------|------------------|--------------------------------
``3_0_2-linux_x86-64``|``-OpenMPI-1.8.1``|``system``
``4.0.0.2``           |``-OpenMPI-2.0.2``|``system``
``4.0.1``             |``-OpenMPI-2.0.2``|``system``
``4.1.0``             |``-OpenMPI-3.1.3``|``system``
``4.2.0``             |                  |``gompi/2019b``
``4.2.1``             |                  |``gompi/2019b``
``5.0.0``             |                  |``gompi/2021a``
``5.0.0``             |``-static``       |``gompi/2021a``
``5.0.1``             |                  |``gompi/2021a``
``5.0.1``             |``-static``       |``gompi/2021a``
``5.0.2``             |                  |``gompi/2021a``, ``gompi/2021b``
``5.0.2``             |``-static``       |``gompi/2021a``, ``gompi/2021b``
``5.0.3``             |                  |``gompi/2021b``
``5.0.4``             |                  |``gompi/2022a``

### ORFfinder

ORF finder searches for open reading frames (ORFs) in the  DNA sequence you enter. The program returns the range of each ORF,  along with its protein translation. Use ORF finder to search newly  sequenced DNA for potential protein encoding segments, verify  predicted protein using newly developed SMART BLAST or regular BLASTP.

*homepage*: <https://www.ncbi.nlm.nih.gov/orffinder/>

version  |toolchain
---------|----------
``0.4.3``|``system``

### OrfM

A simple and not slow open reading frame (ORF) caller.

*homepage*: <https://github.com/wwood/OrfM>

version  |toolchain
---------|----------------------------------------
``0.6.1``|``foss/2016b``
``0.7.1``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

### OrthoFinder

OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics

*homepage*: <https://github.com/davidemms/OrthoFinder>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.7`` |``-Python-2.7.14``|``intel/2018a``
``2.3.3`` |``-Python-2.7.15``|``intel/2018b``
``2.3.8`` |``-Python-2.7.16``|``foss/2019b``
``2.3.11``|``-Python-3.7.4`` |``intel/2019b``
``2.5.2`` |                  |``foss/2020b``
``2.5.4`` |                  |``foss/2020b``

### OrthoMCL

OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences.

*homepage*: <http://orthomcl.org/>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``1.4``  |``-Perl-5.24.0``|``intel/2016b``
``2.0.9``|``-Perl-5.24.0``|``intel/2016b``

### Osi

Osi (Open Solver Interface) provides an abstract base class to a generic linear programming (LP) solver, along with derived classes for specific solvers. Many applications may be able to use the Osi to insulate themselves from a specific LP solver. That is, programs written to the OSI standard may be linked to any solver with an OSI interface and should produce correct results. The OSI has been significantly extended compared to its first incarnation. Currently, the OSI supports linear programming solvers and has rudimentary support for integer programming.

*homepage*: <https://github.com/coin-or/Osi>

version    |toolchain
-----------|----------------------------------
``0.108.5``|``GCCcore/7.3.0``, ``foss/2018b``
``0.108.6``|``GCC/10.3.0``, ``GCCcore/10.2.0``
``0.108.7``|``GCC/11.2.0``

### OSPRay

Open, Scalable, and Portable Ray Tracing Engine

*homepage*: <https://www.ospray.org>

version  |toolchain
---------|----------
``2.5.0``|``system``

### OSU-Micro-Benchmarks

OSU Micro-Benchmarks

*homepage*: <https://mvapich.cse.ohio-state.edu/benchmarks/>

version  |versionsuffix   |toolchain
---------|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.3.2``|                |``foss/2016a``, ``foss/2017a``
``5.6.2``|                |``gompi/2019a``
``5.6.3``|                |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpic/2019b``, ``iimpic/2020a``
``5.7``  |                |``gompi/2020b``, ``gompic/2020b``, ``iimpi/2020b``
``5.7.1``|                |``ffmpi/4.5.0``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2021a``, ``iompi/2021a``
``5.7.1``|``-CUDA-11.3.1``|``gompi/2021a``
``5.8``  |                |``iimpi/2021b``
``5.9``  |                |``gompi/2022.05``, ``gompi/2022a``, ``iimpi/2022a``
``5.9``  |``-CUDA-11.3.1``|``gompi/2021a``
``5.9``  |``-CUDA-11.4.1``|``gompi/2021b``
``5.9``  |``-CUDA-11.7.0``|``gompi/2022a``
``5.9``  |``-ROCm-4.5.0`` |``gompi/2021b``
``6.2``  |                |``gompi/2022.10``, ``gompi/2022b``
``7.1-1``|                |``gompi/2023a``

### OTF2

The Open Trace Format 2 is a highly scalable, memory efficient event trace  data format plus support library. It is the new standard trace format for  Scalasca, Vampir, and TAU and is open for other tools.  

*homepage*: <https://www.score-p.org>

version  |toolchain
---------|-------------------------------------------------------
``2.0``  |``foss/2016a``, ``foss/2017a``
``2.2``  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.3``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.0``  |``GCCcore/11.3.0``
``3.0.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### OVITO

OVITO is a scientific visualization and data analysis solution for atomistic and other particle-based models. It helps scientists gain meaningful and quick insights from numerical simulation results.

*homepage*: <https://www.ovito.org>

version   |versionsuffix|toolchain
----------|-------------|---------------
``3.7.11``|``-basic``   |``gompi/2022a``

### ownCloud

The ownCloud Desktop Client is a tool to synchronize files from ownCloud Server with your computer.

*homepage*: <https://github.com/owncloud/client>

version  |toolchain
---------|-----------------
``2.4.3``|``foss/2018b``
``2.5.4``|``GCCcore/8.2.0``

### oxDNA

oxDNA is a simulation code that was initially conceived as an implementation  of the coarse-grained DNA model introduced by T. E. Ouldridge, J. P. K. Doye and A. A. Louis.  It has been since reworked and it is now an extensible simulation+analysis framework.  It natively supports DNA, RNA, Lennard-Jones and patchy particle simulations of different kinds  on both single CPU cores and NVIDIA GPUs.

*homepage*: <https://github.com/lorenzo-rovigatti/oxDNA>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.5.2``|``-CUDA-11.7.0``|``foss/2022a``

### oxford_asl

A command line tool for quantification of perfusion from ASL data

*homepage*: <https://github.com/ibme-qubic/oxford_asl>

version  |versionsuffix             |toolchain
---------|--------------------------|---------------
``3.9.6``|``-centos7-Python-2.7.13``|``intel/2017a``

## P


[p11-kit](#p11-kit) - [p4-phylogenetics](#p4-phylogenetics) - [p4est](#p4est) - [p4vasp](#p4vasp) - [p7zip](#p7zip) - [packmol](#packmol) - [pagmo](#pagmo) - [pairsnp](#pairsnp) - [PAL2NAL](#pal2nal) - [paladin](#paladin) - [PALEOMIX](#paleomix) - [PAML](#paml) - [panaroo](#panaroo) - [pandapower](#pandapower) - [pandas](#pandas) - [pandas-datareader](#pandas-datareader) - [PANDAseq](#pandaseq) - [Pandoc](#pandoc) - [Panedr](#panedr) - [Pango](#pango) - [pangolin](#pangolin) - [panito](#panito) - [PAPI](#papi) - [parallel](#parallel) - [parallel-fastq-dump](#parallel-fastq-dump) - [Parallel-Hashmap](#parallel-hashmap) - [ParallelIO](#parallelio) - [parameterized](#parameterized) - [parasail](#parasail) - [Paraver](#paraver) - [ParaView](#paraview) - [Parcels](#parcels) - [PARI-GP](#pari-gp) - [ParmEd](#parmed) - [ParMETIS](#parmetis) - [ParMGridGen](#parmgridgen) - [PartitionFinder](#partitionfinder) - [pasta](#pasta) - [PaStiX](#pastix) - [pastml](#pastml) - [patch](#patch) - [patchelf](#patchelf) - [path.py](#path.py) - [PAUP](#paup) - [pauvre](#pauvre) - [pbbam](#pbbam) - [pbcopper](#pbcopper) - [pbdagcon](#pbdagcon) - [pbipa](#pbipa) - [pbmm2](#pbmm2) - [pbs_python](#pbs_python) - [PBSuite](#pbsuite) - [PCAngsd](#pcangsd) - [PCC](#pcc) - [PCL](#pcl) - [PCMSolver](#pcmsolver) - [PCRaster](#pcraster) - [PCRE](#pcre) - [PCRE2](#pcre2) - [pdsh](#pdsh) - [PDT](#pdt) - [PEAR](#pear) - [PennCNV](#penncnv) - [PEPT](#pept) - [Percolator](#percolator) - [Perl](#perl) - [perl-app-cpanminus](#perl-app-cpanminus) - [Perl-bundle-CPAN](#perl-bundle-cpan) - [Perl4-CoreLibs](#perl4-corelibs) - [Perseus](#perseus) - [PEST++](#pest++) - [PETSc](#petsc) - [petsc4py](#petsc4py) - [PfamScan](#pfamscan) - [pfind](#pfind) - [pftoolsV3](#pftoolsv3) - [pFUnit](#pfunit) - [PGDSpider](#pgdspider) - [PGI](#pgi) - [PGPLOT](#pgplot) - [PHANOTATE](#phanotate) - [Phantompeakqualtools](#phantompeakqualtools) - [PHASE](#phase) - [PHAST](#phast) - [Phenoflow](#phenoflow) - [PheWAS](#phewas) - [PheWeb](#pheweb) - [PhiPack](#phipack) - [PHLAT](#phlat) - [phonemizer](#phonemizer) - [phono3py](#phono3py) - [phonopy](#phonopy) - [photontorch](#photontorch) - [phototonic](#phototonic) - [PHYLIP](#phylip) - [PhyloBayes-MPI](#phylobayes-mpi) - [phylokit](#phylokit) - [phylonaut](#phylonaut) - [PhyloPhlAn](#phylophlan) - [PhyML](#phyml) - [phyx](#phyx) - [picard](#picard) - [PICI-LIGGGHTS](#pici-liggghts) - [pigz](#pigz) - [PIL](#pil) - [PileOMeth](#pileometh) - [Pillow](#pillow) - [Pillow-SIMD](#pillow-simd) - [Pilon](#pilon) - [PIMS](#pims) - [Pindel](#pindel) - [Pingouin](#pingouin) - [Pint](#pint) - [pip](#pip) - [PIPITS](#pipits) - [PIRATE](#pirate) - [pIRS](#pirs) - [Pisces](#pisces) - [piSvM](#pisvm) - [piSvM-JSC](#pisvm-jsc) - [pixman](#pixman) - [pizzly](#pizzly) - [pkg-config](#pkg-config) - [pkgconf](#pkgconf) - [pkgconfig](#pkgconfig) - [PLAMS](#plams) - [plantcv](#plantcv) - [PlaScope](#plascope) - [PlasmaPy](#plasmapy) - [PLAST](#plast) - [Platanus](#platanus) - [Platypus](#platypus) - [plc](#plc) - [PLINK](#plink) - [plinkliftover](#plinkliftover) - [plinkQC](#plinkqc) - [PLINKSEQ](#plinkseq) - [plot1cell](#plot1cell) - [Ploticus](#ploticus) - [plotly](#plotly) - [plotly-orca](#plotly-orca) - [plotly.py](#plotly.py) - [PLplot](#plplot) - [PLUMED](#plumed) - [PLY](#ply) - [PMIx](#pmix) - [pmt](#pmt) - [pmx](#pmx) - [PnetCDF](#pnetcdf) - [pocl](#pocl) - [pod5-file-format](#pod5-file-format) - [poetry](#poetry) - [polars](#polars) - [polymake](#polymake) - [pomkl](#pomkl) - [pompi](#pompi) - [poppler](#poppler) - [popscle](#popscle) - [popt](#popt) - [Porechop](#porechop) - [porefoam](#porefoam) - [poretools](#poretools) - [Portcullis](#portcullis) - [Postgres-XL](#postgres-xl) - [PostgreSQL](#postgresql) - [POT](#pot) - [POV-Ray](#pov-ray) - [powerlaw](#powerlaw) - [pp-sketchlib](#pp-sketchlib) - [PPanGGOLiN](#ppanggolin) - [PPfold](#ppfold) - [ppl](#ppl) - [pplacer](#pplacer) - [pplpy](#pplpy) - [PRANK](#prank) - [PRC](#prc) - [preCICE](#precice) - [PREQUAL](#prequal) - [preseq](#preseq) - [presto](#presto) - [pretty-yaml](#pretty-yaml) - [Primer3](#primer3) - [PRINSEQ](#prinseq) - [printproto](#printproto) - [PRISMS-PF](#prisms-pf) - [ProbABEL](#probabel) - [prodigal](#prodigal) - [ProFit](#profit) - [PROJ](#proj) - [ProjectQ](#projectq) - [prokka](#prokka) - [prompt-toolkit](#prompt-toolkit) - [proovread](#proovread) - [propy](#propy) - [Proteinortho](#proteinortho) - [ProtHint](#prothint) - [protobuf](#protobuf) - [protobuf-python](#protobuf-python) - [protozero](#protozero) - [PRSice](#prsice) - [pscom](#pscom) - [PSI](#psi) - [PSI4](#psi4) - [PsiCLASS](#psiclass) - [PSIPRED](#psipred) - [psmc](#psmc) - [psmpi](#psmpi) - [psmpi2](#psmpi2) - [PSolver](#psolver) - [PSORTb](#psortb) - [psrecord](#psrecord) - [pstoedit](#pstoedit) - [psutil](#psutil) - [psycopg2](#psycopg2) - [ptemcee](#ptemcee) - [PTESFinder](#ptesfinder) - [pubtcrs](#pubtcrs) - [pugixml](#pugixml) - [pullseq](#pullseq) - [PuLP](#pulp) - [purge_dups](#purge_dups) - [py](#py) - [py-aiger](#py-aiger) - [py-aiger-bdd](#py-aiger-bdd) - [py-cpuinfo](#py-cpuinfo) - [py3Dmol](#py3dmol) - [pyABC](#pyabc) - [PyAMG](#pyamg) - [PyAPS3](#pyaps3) - [pybedtools](#pybedtools) - [PyBerny](#pyberny) - [pyBigWig](#pybigwig) - [pybind11](#pybind11) - [pybinding](#pybinding) - [PyBioLib](#pybiolib) - [PyCairo](#pycairo) - [PyCalib](#pycalib) - [pyccel](#pyccel) - [PyCharm](#pycharm) - [PyCheMPS2](#pychemps2) - [Pychopper](#pychopper) - [PyCifRW](#pycifrw) - [PyClone](#pyclone) - [pycma](#pycma) - [pycocotools](#pycocotools) - [pycodestyle](#pycodestyle) - [PyCogent](#pycogent) - [pycoQC](#pycoqc) - [PyCUDA](#pycuda) - [PycURL](#pycurl) - [PyDamage](#pydamage) - [pydantic](#pydantic) - [PyDatastream](#pydatastream) - [pydicom](#pydicom) - [pydicom-seg](#pydicom-seg) - [pydlpoly](#pydlpoly) - [pydot](#pydot) - [pyEGA3](#pyega3) - [pyenchant](#pyenchant) - [PyEVTK](#pyevtk) - [pyFAI](#pyfai) - [pyfaidx](#pyfaidx) - [pyfasta](#pyfasta) - [PyFFmpeg](#pyffmpeg) - [pyFFTW](#pyfftw) - [pyfits](#pyfits) - [PyFMI](#pyfmi) - [PyFoam](#pyfoam) - [PyFR](#pyfr) - [PyFrag](#pyfrag) - [pygccxml](#pygccxml) - [pyGenomeTracks](#pygenometracks) - [PyGEOS](#pygeos) - [pyGIMLi](#pygimli) - [Pygments](#pygments) - [pygmo](#pygmo) - [PyGObject](#pygobject) - [pygraphviz](#pygraphviz) - [pygrib](#pygrib) - [PyGTK](#pygtk) - [PyGTS](#pygts) - [PyGWAS](#pygwas) - [pyhdf](#pyhdf) - [PyImageJ](#pyimagej) - [pyiron](#pyiron) - [Pyke3](#pyke3) - [pylift](#pylift) - [Pylint](#pylint) - [pyMannKendall](#pymannkendall) - [pymatgen](#pymatgen) - [pymatgen-db](#pymatgen-db) - [pymbar](#pymbar) - [PyMC3](#pymc3) - [pymca](#pymca) - [pymemcache](#pymemcache) - [PyMOL](#pymol) - [PyNAST](#pynast) - [pyobjcryst](#pyobjcryst) - [PyOD](#pyod) - [Pyomo](#pyomo) - [PyOpenCL](#pyopencl) - [PyOpenGL](#pyopengl) - [pyparsing](#pyparsing) - [pyperf](#pyperf) - [pyplusplus](#pyplusplus) - [pypmt](#pypmt) - [PYPOWER](#pypower) - [pyproj](#pyproj) - [PyPSA](#pypsa) - [pyqstem](#pyqstem) - [PyQt](#pyqt) - [PyQt5](#pyqt5) - [PyQtGraph](#pyqtgraph) - [pyradiomics](#pyradiomics) - [PyRe](#pyre) - [PyRETIS](#pyretis) - [pyringe](#pyringe) - [pyro-api](#pyro-api) - [pyro-ppl](#pyro-ppl) - [Pyro4](#pyro4) - [PyRosetta](#pyrosetta) - [Pysam](#pysam) - [pysamstats](#pysamstats) - [PySAT](#pysat) - [pyScaf](#pyscaf) - [pySCENIC](#pyscenic) - [PySCF](#pyscf) - [pysheds](#pysheds) - [pyshp](#pyshp) - [PySide2](#pyside2) - [pyslim](#pyslim) - [pysndfx](#pysndfx) - [Pysolar](#pysolar) - [pyspoa](#pyspoa) - [pysqlite](#pysqlite) - [PyStan](#pystan) - [pysteps](#pysteps) - [pystran](#pystran) - [PyTables](#pytables) - [pytesseract](#pytesseract) - [pytest](#pytest) - [pytest-benchmark](#pytest-benchmark) - [pytest-cpp](#pytest-cpp) - [pytest-flakefinder](#pytest-flakefinder) - [pytest-rerunfailures](#pytest-rerunfailures) - [pytest-shard](#pytest-shard) - [pytest-xdist](#pytest-xdist) - [PYTHIA](#pythia) - [Python](#python) - [Python-bundle](#python-bundle) - [python-docx](#python-docx) - [python-hl7](#python-hl7) - [python-igraph](#python-igraph) - [python-irodsclient](#python-irodsclient) - [python-isal](#python-isal) - [python-Levenshtein](#python-levenshtein) - [python-libsbml](#python-libsbml) - [python-louvain](#python-louvain) - [python-mujoco](#python-mujoco) - [python-parasail](#python-parasail) - [python-telegram-bot](#python-telegram-bot) - [python-weka-wrapper3](#python-weka-wrapper3) - [python-xxhash](#python-xxhash) - [pythran](#pythran) - [PyTorch](#pytorch) - [PyTorch-bundle](#pytorch-bundle) - [pytorch-CycleGAN-pix2pix](#pytorch-cyclegan-pix2pix) - [PyTorch-Geometric](#pytorch-geometric) - [PyTorch-Ignite](#pytorch-ignite) - [PyTorch-Image-Models](#pytorch-image-models) - [PyTorch-Lightning](#pytorch-lightning) - [PyTorch3D](#pytorch3d) - [PyVCF](#pyvcf) - [PyVCF3](#pyvcf3) - [pyWannier90](#pywannier90) - [PyWavelets](#pywavelets) - [PyWBGT](#pywbgt) - [PyYAML](#pyyaml) - [PyZMQ](#pyzmq)


### p11-kit

Provides a way to load and enumerate PKCS#11 modules.  Provides a standard configuration setup for installing  PKCS#11 modules in such a way that they're discoverable.  Also solves problems with coordinating the use of PKCS#11  by different components or libraries living in the same process.

*homepage*: <https://p11-glue.freedesktop.org/p11-kit.html>

version   |toolchain
----------|----------------------------------------------------------------------
``0.23.2``|``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``0.24.0``|``GCCcore/10.3.0``
``0.24.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### p4-phylogenetics

A Python phyloinformatic toolkit, and an implementation of tree-heterogeneous models of evolution.

*homepage*: <http://p4.nhm.ac.uk/>

version         |versionsuffix    |toolchain
----------------|-----------------|--------------
``1.4-20210322``|``-Python-3.7.4``|``foss/2019b``

### p4est

p4est is a C library to manage a collection (a forest) of multiple connected adaptive quadtrees or octrees in parallel.

*homepage*: <https://www.p4est.org>

version|toolchain
-------|-------------------------------
``2.2``|``foss/2019a``, ``intel/2019a``
``2.8``|``foss/2021a``

### p4vasp

Visualization suite for VASP

*homepage*: <http://www.p4vasp.at/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``0.3.29``|``-Python-2.7.11``|``intel/2016a``
``0.3.30``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``0.3.30``|``-Python-2.7.15``|``foss/2018b``

### p7zip

p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.  7-Zip is a file archiver with highest compression ratio.

*homepage*: <http://p7zip.sourceforge.net/>

version   |toolchain
----------|---------------------------------------------------------------------------
``9.38.1``|``GCC/4.9.2``, ``system``
``16.02`` |``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``
``17.03`` |``GCCcore/10.2.0``
``17.04`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### packmol

Packing Optimization for Molecular Dynamics Simulations

*homepage*: <http://m3g.iqm.unicamp.br/packmol>

version    |toolchain
-----------|-------------------------------
``16.103`` |``intel/2016a``
``18.013`` |``foss/2018a``, ``intel/2018a``
``20.2.2`` |``GCC/10.2.0``
``v20.2.2``|``iccifort/2020.1.217``

### pagmo

pagmo is a C++ scientific library for massively parallel optimization.

*homepage*: <https://esa.github.io/pagmo2>

version   |toolchain
----------|----------------------------------------------
``2.17.0``|``foss/2020b``
``2.18.0``|``foss/2021a``, ``foss/2021b``, ``foss/2022a``

### pairsnp

A set of scripts for very quickly obtaining pairwise SNP distance matrices from multiple sequence alignments  using sparse matrix libraries to improve performance. 

*homepage*: <https://github.com/gtonkinhill/pairsnp>

version  |toolchain
---------|--------------
``0.0.7``|``foss/2021a``

### PAL2NAL

PAL2NAL is a program that converts a multiple sequence alignment of proteins and  the corresponding DNA (or mRNA) sequences into a codon alignment. The program automatically  assigns the corresponding codon sequence even if the input DNA sequence has mismatches with the  input protein sequence, or contains UTRs, polyA tails. It can also deal with frame shifts in the  input alignment, which is suitable for the analysis of pseudogenes. The resulting codon alignment  can further be subjected to the calculation of synonymous (d_S) and non-synonymous (d_N) subs-  titution rates. 

*homepage*: <http://www.bork.embl.de/pal2nal/>

version|toolchain
-------|--------------------------------------
``14`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

### paladin

Protein ALignment And Detection INterface  PALADIN is a protein sequence alignment tool designed for the accurate functional characterization of metagenomes. 

*homepage*: <https://github.com/ToniWestbrook/paladin>

version  |toolchain
---------|--------------------------------------
``1.4.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### PALEOMIX

The PALEOMIX pipelines are a set of pipelines and tools designed to aid the rapid processing  of High-Throughput Sequencing (HTS) data.

*homepage*: <https://github.com/MikkelSchubert/paleomix>

version  |toolchain
---------|--------------
``1.3.7``|``foss/2022a``

### PAML

PAML is a package of programs for phylogenetic   analyses of DNA or protein sequences using maximum likelihood.

*homepage*: <http://abacus.gene.ucl.ac.uk/software/paml.html>

version   |toolchain
----------|-----------------------------------------------------------------------------
``4.9i``  |``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``4.9j``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``4.10.5``|``GCCcore/11.3.0``

### panaroo

A pangenome analysis pipeline.

*homepage*: <https://gtonkinhill.github.io/panaroo/>

version  |toolchain
---------|--------------
``1.2.8``|``foss/2020b``
``1.2.9``|``foss/2021a``
``1.3.2``|``foss/2021b``

### pandapower

An simple to use open source tool for power system modeling, analysis and optimization with a high degree of automation

*homepage*: <http://www.pandapower.org/>

version  |toolchain
---------|--------------
``2.7.0``|``foss/2020b``

### pandas

pandas is an open source, BSD-licensed library providing high-performance, simple-to-use data structures  and data analysis tools for the Python programming language.

*homepage*: <https://pypi.python.org/pypi/pandas/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.18.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.18.0``|``-Python-3.5.1`` |``foss/2016a``, ``intel/2016a``
``0.18.1``|``-Python-2.7.12``|``intel/2016b``
``0.18.1``|``-Python-3.5.2`` |``intel/2016b``
``0.19.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.19.0``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.19.1``|``-Python-2.7.12``|``intel/2016b``
``0.19.1``|``-Python-3.5.2`` |``intel/2016b``
``0.20.1``|``-Python-3.6.1`` |``intel/2017a``
``0.21.0``|``-Python-2.7.13``|``intel/2017a``
``0.21.0``|``-Python-3.6.3`` |``intel/2017b``
``1.1.2`` |``-Python-3.8.2`` |``foss/2020a``

### pandas-datareader

Up to date remote data access for pandas, works for multiple versions of pandas.

*homepage*: <https://pypi.org/project/pandas-datareader>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.0``|``-Python-3.6.4``|``intel/2018a``

### PANDAseq

PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded     in the sequence, and reconstruct an overlapping sequence.

*homepage*: <https://github.com/neufeld/pandaseq>

version |toolchain
--------|----------------------------------------------------------
``2.10``|``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``2.11``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``

### Pandoc

If you need to convert files from one markup format into another, pandoc is your swiss-army knife

*homepage*: <https://pandoc.org>

version  |toolchain
---------|----------
``2.1.3``|``system``
``2.5``  |``system``
``2.10`` |``system``
``2.13`` |``system``
``3.1.2``|``system``

### Panedr

Panedr uses the Pyedr library to read a Gromacs EDR binary energy XDR file and returns its contents as a pandas dataframe.

*homepage*: <https://github.com/MDAnalysis/panedr>

version  |toolchain
---------|--------------
``0.7.0``|``foss/2021a``

### Pango

Pango is a library for laying out and rendering of text, with an emphasis on internationalization. Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.

*homepage*: <https://www.pango.org/>

version    |toolchain
-----------|------------------------------------
``1.39.0`` |``foss/2016a``, ``intel/2016a``
``1.40.1`` |``foss/2016a``, ``intel/2016a``
``1.40.3`` |``foss/2016b``, ``intel/2016b``
``1.40.5`` |``intel/2017a``
``1.40.12``|``intel/2017a``
``1.40.14``|``foss/2017b``, ``intel/2017b``
``1.41.0`` |``foss/2017b``, ``intel/2017b``
``1.41.1`` |``foss/2018a``, ``intel/2018a``
``1.42.4`` |``foss/2018b``, ``fosscuda/2018b``
``1.43.0`` |``GCCcore/8.2.0``
``1.44.7`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.47.0`` |``GCCcore/10.2.0``
``1.48.5`` |``GCCcore/10.3.0``
``1.48.8`` |``GCCcore/11.2.0``
``1.50.7`` |``GCCcore/11.3.0``
``1.50.12``|``GCCcore/12.2.0``

### pangolin

Software package for assigning SARS-CoV-2 genome sequences to global lineages. This module also contains the faToVcf tool

*homepage*: <https://cov-lineages.org/pangolin.html>

version   |versionsuffix             |toolchain
----------|--------------------------|--------------
``3.1.11``|                          |``foss/2020b``
``3.1.16``|``-pangoLEARN-2021-10-18``|``foss/2021b``
``3.1.16``|``-pangoLEARN-2021-11-25``|``foss/2021b``

### panito

Calculate genome wide average nucleotide identity (gwANI) for a multiFASTA alignment.

*homepage*: <https://github.com/sanger-pathogens/panito>

version  |toolchain
---------|--------------
``0.0.1``|``GCC/10.3.0``

### PAPI

PAPI provides the tool designer and application engineer with a consistent  interface and methodology for use of the performance counter hardware found  in most major microprocessors. PAPI enables software engineers to see, in near  real time, the relation between software performance and processor events.  In addition Component PAPI provides access to a collection of components  that expose performance measurement opportunites across the hardware and  software stack. 

*homepage*: <https://icl.cs.utk.edu/projects/papi/>

version    |toolchain
-----------|--------------------------------------------------------
``5.4.3``  |``foss/2016a``
``5.5.1``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``5.6.0``  |``GCCcore/6.4.0``
``5.7.0``  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``6.0.0``  |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.0.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``7.0.0``  |``GCCcore/11.3.0``

### parallel

parallel: Build and execute shell commands in parallel

*homepage*: <https://savannah.gnu.org/projects/parallel/>

version     |toolchain
------------|-------------------------------
``20141122``|``GCC/4.9.2``
``20150322``|``GCC/4.9.2``
``20150822``|``GCC/4.9.2``
``20160622``|``foss/2016a``
``20170822``|``intel/2017a``
``20171022``|``intel/2017b``
``20171122``|``foss/2017b``, ``intel/2017b``
``20180422``|``intel/2018a``
``20180822``|``foss/2018b``
``20181222``|``intel/2018b``
``20190222``|``GCCcore/7.3.0``
``20190622``|``GCCcore/8.2.0``
``20190922``|``GCCcore/8.3.0``
``20200422``|``GCCcore/9.3.0``
``20200522``|``GCCcore/9.3.0``
``20210322``|``GCCcore/10.2.0``
``20210622``|``GCCcore/10.3.0``
``20210722``|``GCCcore/11.2.0``
``20220722``|``GCCcore/11.3.0``

### parallel-fastq-dump

parallel fastq-dump wrapper

*homepage*: <https://github.com/rvalieris/parallel-fastq-dump>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------
``0.6.5``|``-Python-3.7.2``|``GCCcore/8.2.0``
``0.6.6``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.6.7``|                 |``gompi/2020b``, ``gompi/2022a``

### Parallel-Hashmap

Parallel Hashmap is built on a modified version of Abseil's flat_hash_map. Parallel Hashmap has lower space requirements, is nearly as fast as the underlying flat_hash_map, and can be used from multiple threads with high levels of concurrency.

*homepage*: <https://github.com/greg7mdp/parallel-hashmap>

version |toolchain
--------|------------------
``1.33``|``GCCcore/10.3.0``

### ParallelIO

A high-level Parallel I/O Library for structured grid applications

*homepage*: <https://github.com/NCAR/ParallelIO>

version   |toolchain
----------|--------------------------------
``2.2.2a``|``intel/2017a``
``2.5.10``|``gompi/2022a``, ``iimpi/2022a``

### parameterized

Parameterized testing with any Python test framework 

*homepage*: <https://github.com/wolever/parameterized>

version  |toolchain
---------|------------------
``0.8.1``|``GCCcore/10.3.0``

### parasail

parasail is a SIMD C (C99) library containing implementations  of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global  pairwise sequence alignment algorithms. 

*homepage*: <https://github.com/jeffdaily/parasail>

version  |toolchain
---------|--------------------------------------
``2.2``  |``intel/2018a``
``2.4``  |``foss/2018b``
``2.4.1``|``GCC/8.3.0``, ``intel/2019b``
``2.4.2``|``GCC/9.3.0``, ``iccifort/2020.1.217``
``2.4.3``|``GCC/10.2.0``, ``GCC/10.3.0``
``2.5``  |``GCC/11.2.0``
``2.6``  |``GCC/11.3.0``
``2.6.2``|``GCC/12.2.0``

### Paraver

A very powerful performance visualization and analysis tool based on  traces that can be used to analyse any information that is expressed on its input trace format.  Traces for parallel MPI, OpenMP and other programs can be genereated with Extrae.

*homepage*: <https://tools.bsc.es/paraver>

version   |toolchain
----------|--------------
``4.8.1`` |``foss/2019a``
``4.9.2`` |``foss/2021a``
``4.11.1``|``foss/2022a``

### ParaView

ParaView is a scientific parallel visualizer.

*homepage*: <https://www.paraview.org>

version   |versionsuffix         |toolchain
----------|----------------------|-------------------------------------------------------------------------------------------------
``4.4.0`` |                      |``foss/2016a``, ``intel/2016a``
``4.4.0`` |``-mpi``              |``gimkl/2.11.5``
``5.1.2`` |``-mpi``              |``foss/2016b``, ``intel/2016b``
``5.2.0`` |``-mpi``              |``foss/2016b``, ``intel/2017a``
``5.3.0`` |``-mpi``              |``foss/2016b``
``5.4.1`` |``-mpi``              |``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``5.5.2`` |``-Python-2.7.15-mpi``|``foss/2018b``
``5.6.2`` |``-Python-3.7.4-mpi`` |``foss/2019b``, ``intel/2019b``
``5.8.0`` |``-Python-3.8.2-mpi`` |``foss/2020a``, ``intel/2020a``
``5.8.1`` |``-mpi``              |``foss/2020b``
``5.9.1`` |``-mpi``              |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``5.10.1``|``-mpi``              |``foss/2022a``
``5.11.0``|``-mpi``              |``foss/2022b``
``5.11.1``|``-mpi``              |``foss/2022a``

### Parcels

Parcels (Probably A Really Computationally Efficient Lagrangian Simulator) is a set of Python classes and methods to create customisable particle tracking simulations using output from Ocean Circulation models. Parcels can be used to track passive and active particulates such as water, plankton, plastic and fish.

*homepage*: <http://www.oceanparcels.org/>

version  |toolchain
---------|--------------
``2.4.0``|``foss/2022a``

### PARI-GP

PARI/GP is a widely used computer algebra system designed for fast computations in number theory  (factorizations, algebraic  number theory, elliptic curves...), but also contains a large number of other useful  functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc.,  and a lot of transcendental functions. PARI is  also available as a C library to allow for faster computations. 

*homepage*: <http://pari.math.u-bordeaux.fr>

version  |toolchain
---------|--------------
``2.7.6``|``foss/2016a``

### ParmEd

ParmEd is a general tool for aiding in investigations of biomolecular systems using popular  molecular simulation packages, like Amber, CHARMM, and OpenMM written in Python.

*homepage*: <https://parmed.github.io/ParmEd>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.7.3``|``-Python-3.6.3``|``intel/2017b``
``3.2.0``|``-Python-3.7.4``|``intel/2019b``
``3.2.0``|``-Python-3.8.2``|``intel/2020a``

### ParMETIS

ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning  unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the  functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and  large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way  graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.

*homepage*: <http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.0.3``|``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2.11.5``, ``gompi/2019a``, ``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``gompi/2022b``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

### ParMGridGen

ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen,  that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited  for geometric multigrid methods.

*homepage*: <http://www-users.cs.umn.edu/~moulitsa/software.html>

version|toolchain
-------|----------------------------------------------------------------------------------------------------------------------
``1.0``|``gimkl/2.11.5``, ``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``intel/2016a``, ``intel/2017a``

### PartitionFinder

PartitionFinder 2 is a Python program for simultaneously choosing partitioning schemes and models of molecular evolution for phylogenetic analyses of DNA, protein, and morphological data. You can PartitionFinder 2 before running a phylogenetic analysis, in order to decide how to divide up your sequence data into separate blocks before analysis, and to simultaneously perform model selection on each of those blocks.

*homepage*: <https://www.robertlanfear.com/partitionfinder>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.1``|``-Python-2.7.16``|``intel/2019b``

### pasta

PASTA (Practical Alignment using SATe and Transitivity)

*homepage*: <https://github.com/smirarab/pasta>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------
``1.8.5``|``-Python-3.7.2``|``GCC/8.2.0-2.31.1``

### PaStiX

PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance  parallel solver for very large sparse linear systems based on direct methods.

*homepage*: <http://pastix.gforge.inria.fr/>

version  |toolchain
---------|--------------
``5.2.3``|``foss/2017b``

### pastml

Ancestor character reconstruction and visualisation for rooted phylogenetic trees

*homepage*: <https://pastml.pasteur.fr/>

version   |toolchain
----------|--------------
``1.9.34``|``foss/2021a``

### patch

Patch takes a patch file containing a difference listing produced by the diff program  and applies those differences to one or more original files, producing patched versions.

*homepage*: <https://savannah.gnu.org/projects/patch>

version  |toolchain
---------|----------
``2.7.6``|``system``

### patchelf

PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.

*homepage*: <http://nixos.org/patchelf.html>

version   |toolchain
----------|---------------------------------------------------------
``0.8``   |``GNU/4.9.3-2.25``
``0.9``   |``GCCcore/6.4.0``, ``foss/2016a``
``0.10``  |``GCCcore/7.2.0``, ``GCCcore/8.3.0``
``0.12``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.13``  |``GCCcore/11.2.0``
``0.15.0``|``GCCcore/11.3.0``
``0.18.0``|``GCCcore/12.3.0``

### path.py

path.py is a Python library implementing path objects as first-class entities,  allowing common operations on files to be invoked on those path objects directly.

*homepage*: <https://github.com/jaraco/path.py>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``8.2.1``|``-Python-2.7.11``|``foss/2016a``
``8.2.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``8.2.1``|``-Python-3.5.1`` |``foss/2016a``
``8.2.1``|``-Python-3.5.2`` |``intel/2016b``
``10.1`` |``-Python-2.7.12``|``intel/2016b``

### PAUP

PAUP* (Phylogenetic Analysis Using Parsimony *and other methods) is a computational phylogenetics program for inferring evolutionary trees.

*homepage*: <https://paup.phylosolutions.com/>

version    |versionsuffix|toolchain
-----------|-------------|----------
``4.0a166``|``-centos64``|``system``
``4.0a168``|``-centos64``|``system``

### pauvre

Tools for plotting Oxford Nanopore and other long-read data

*homepage*: <https://github.com/conchoecia/pauvre>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.2.3`` |                 |``foss/2021a``
``0.1923``|``-Python-3.7.4``|``intel/2019b``
``0.1924``|                 |``intel/2020b``

### pbbam

The pbbam software package provides components to create, query, & edit PacBio BAM files  and associated indices.

*homepage*: <http://pbbam.readthedocs.io/>

version     |toolchain
------------|---------------
``1.0.6``   |``gompi/2019a``
``20170508``|``intel/2017a``

### pbcopper

The pbcopper library provides a suite of data structures, algorithms,  and utilities for C++ applications.

*homepage*: <https://github.com/PacificBiosciences/pbcopper>

version  |toolchain
---------|---------------
``1.3.0``|``gompi/2019a``

### pbdagcon

pbdagcon is a tool that implements DAGCon (Directed Acyclic Graph Consensus) which is a sequence  consensus algorithm based on using directed acyclic graphs to encode multiple sequence alignment.

*homepage*: <https://github.com/PacificBiosciences/pbdagcon>

version     |toolchain
------------|---------------
``20170330``|``intel/2017a``

### pbipa

Improved Phased Assembler (IPA) is the official PacBio software for HiFi genome assembly. IPA was designed to utilize the accuracy of PacBio HiFi reads to produce high-quality phased genome assemblies. IPA is an end-to-end solution, starting with input reads and resulting in a polished assembly. IPA is fast, providing an simple to use local run mode or a distributed pipeline for a cluster.

*homepage*: <https://github.com/PacificBiosciences/pbipa>

version  |toolchain
---------|--------------
``1.8.0``|``foss/2021b``

### pbmm2

A minimap2 frontend for PacBio native data formats

*homepage*: <https://github.com/PacificBiosciences/pbmm2>

version  |toolchain
---------|---------------
``1.1.0``|``gompi/2019a``

### pbs_python

The pbs_python package is a wrapper class for the Torque C library. With this package you now can  write utilities/extensions in Python instead of C. We developed this package because we want to replace xpbsmon by  an ascii version named pbsmon. PBSQuery is also included in this package. This is a python module build on top of  the pbs python module to simplify querying the batch server, eg: how many jobs, how many nodes, ...

*homepage*: <https://oss.trac.surfsara.nl/pbs_python>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.6.0``|                  |``system``
``4.6.0``|``-Python-2.7.11``|``intel/2016a``
``4.6.0``|``-Python-2.7.12``|``intel/2016b``
``4.6.0``|``-Python-2.7.13``|``intel/2017a``

### PBSuite

PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or  long 454 reads in fasta format) to high-confidence draft assembles.

*homepage*: <https://sourceforge.net/p/pb-jelly/wiki/Home/>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``15.8.24``|``-Python-2.7.12``|``intel/2016b``

### PCAngsd

PCAngsd, which estimates the covariance matrix for low depth  NGS data in an iterative procedure based on genotype likelihoods and is able  to perform multiple population genetic analyses in heterogeneous populations.

*homepage*: <http://www.popgen.dk/software/index.php/PCAngsd>

version |versionsuffix     |toolchain
--------|------------------|--------------
``0.97``|``-Python-2.7.14``|``foss/2018a``

### PCC

The compiler is based on the original Portable C Compiler by S. C. Johnson,  written in the late 70's. About 50% of the frontend code and 80% of the backend code has been modified.

*homepage*: <http://pcc.ludd.ltu.se/>

version     |toolchain
------------|----------
``20131024``|``system``

### PCL

The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and  point cloud processing.

*homepage*: <http://pointclouds.org/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.2``|``-Python-2.7.11``|``intel/2016a``
``1.8.1``|``-Python-2.7.14``|``intel/2017b``

### PCMSolver

An API for the Polarizable Continuum Model.

*homepage*: <https://pcmsolver.readthedocs.org>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.1.4``   |``-Python-2.7.11``|``intel/2016a``
``1.2.3``   |``-Python-3.6.6`` |``foss/2018b``
``1.2.3``   |``-Python-3.7.2`` |``gompi/2019a``
``1.2.3``   |``-Python-3.7.4`` |``iimpi/2019b``
``20160205``|``-Python-2.7.11``|``intel/2016a``

### PCRaster

PCRaster Is a collection of software targeted at the development  and deployment of spatio-temporal environmental models.

*homepage*: <http://pcraster.geo.uu.nl/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.1.0``|``-Python-2.7.14``|``intel/2017b``

### PCRE

The PCRE library is a set of functions that implement regular expression  pattern matching using the same syntax and semantics as Perl 5. 

*homepage*: <https://www.pcre.org/>

version |toolchain
--------|----------------------------------------------------------------------------------
``8.38``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``8.39``|``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``8.40``|``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017a``
``8.41``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``8.42``|``GCCcore/6.4.0``
``8.43``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.44``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``8.45``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### PCRE2

The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5. 

*homepage*: <https://www.pcre.org/>

version  |toolchain
---------|--------------------------------------
``10.21``|``foss/2016a``
``10.31``|``foss/2018b``
``10.33``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``10.34``|``GCCcore/9.3.0``
``10.35``|``GCCcore/10.2.0``
``10.36``|``GCCcore/10.3.0``
``10.37``|``GCCcore/11.2.0``
``10.40``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``10.42``|``GCCcore/12.3.0``

### pdsh

A high performance, parallel remote shell utility

*homepage*: <https://github.com/chaos/pdsh>

version |toolchain
--------|--------------------------------------
``2.34``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### PDT

Program Database Toolkit (PDT) is a framework for analyzing source code  written in several programming languages and for making rich program  knowledge accessible to developers of static and dynamic analysis tools.  PDT implements a standard program representation, the program database  (PDB), that can be accessed in a uniform way through a class library  supporting common PDB operations. 

*homepage*: <https://www.cs.uoregon.edu/research/pdt/>

version   |toolchain
----------|-----------------------------------------------------------------------------
``3.22``  |``foss/2016a``
``3.25``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.25.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### PEAR

PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger.   It is fully parallelized and can run with as low as just a few kilobytes of memory.

*homepage*: <http://sco.h-its.org/exelixis/web/software/pear/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------------
``0.9.8`` |``foss/2016b``, ``intel/2016b``
``0.9.10``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``0.9.11``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``

### PennCNV

A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently  it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format,  it can also handle other types of SNP arrays and oligonucleotide arrays.

*homepage*: <https://penncnv.openbioinformatics.org/>

version  |toolchain
---------|-----------------
``1.0.5``|``GCCcore/8.3.0``

### PEPT

A Python library that integrates all the tools necessary to perform research using Positron Emission  Particle Tracking (PEPT). The library includes algorithms for the location, identification and tracking of particles,  in addition to tools for visualisation and analysis, and utilities allowing the realistic simulation of PEPT data.

*homepage*: <https://pept.readthedocs.io/en/latest/>

version  |toolchain
---------|--------------
``0.4.1``|``foss/2021a``

### Percolator

Semi-supervised learning for peptide identification from shotgun proteomics datasets

*homepage*: <https://github.com/percolator/percolator>

version|toolchain
-------|---------------
``3.4``|``gompi/2019a``

### Perl

Larry Wall's Practical Extraction and Report Language  Includes a small selection of extra CPAN packages for core functionality. 

*homepage*: <https://www.perl.org/>

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------------------
``5.20.1``|``-bare``    |``GCC/4.8.2``, ``GCC/4.9.2``
``5.20.2``|``-bare``    |``GCC/4.9.2``
``5.20.3``|             |``foss/2016a``, ``intel/2016a``
``5.22.0``|``-bare``    |``GCC/4.9.2``
``5.22.1``|             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``5.22.1``|``-bare``    |``foss/2016a``, ``intel/2016a``
``5.22.2``|             |``intel/2016a``
``5.24.0``|             |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``5.24.0``|``-bare``    |``foss/2016b``
``5.24.1``|             |``GCCcore/6.3.0``, ``foss/2017a``, ``intel/2017a``
``5.26.0``|             |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``
``5.26.1``|             |``GCCcore/6.4.0``, ``foss/2018a``
``5.26.1``|``-bare``    |``foss/2018a``
``5.28.0``|             |``GCCcore/7.3.0``
``5.28.1``|             |``GCCcore/8.2.0``
``5.30.0``|             |``GCCcore/8.3.0``
``5.30.0``|``-minimal`` |``GCCcore/8.3.0``
``5.30.2``|             |``GCCcore/9.3.0``
``5.30.2``|``-minimal`` |``GCCcore/9.3.0``
``5.32.0``|             |``GCCcore/10.2.0``
``5.32.0``|``-minimal`` |``GCCcore/10.2.0``
``5.32.1``|             |``FCC/4.5.0``, ``GCCcore/10.3.0``
``5.32.1``|``-minimal`` |``GCCcore/10.3.0``
``5.34.0``|             |``GCCcore/11.2.0``
``5.34.0``|``-minimal`` |``GCCcore/11.2.0``
``5.34.1``|             |``GCCcore/11.3.0``
``5.34.1``|``-minimal`` |``GCCcore/11.3.0``
``5.36.0``|             |``GCCcore/12.1.0``, ``GCCcore/12.2.0``
``5.36.0``|``-minimal`` |``GCCcore/12.2.0``
``5.36.1``|             |``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### perl-app-cpanminus

cpanm - get, unpack build and install modules from CPAN 

*homepage*: <https://github.com/miyagawa/cpanminus>

version   |toolchain
----------|----------
``1.7039``|``system``

### Perl-bundle-CPAN

A set of common packages from CPAN

*homepage*: <https://www.perl.org/>

version   |toolchain
----------|------------------
``5.36.1``|``GCCcore/12.3.0``

### Perl4-CoreLibs

Libraries historically supplied with Perl 4

*homepage*: <https://metacpan.org/pod/Perl4::CoreLibs>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``0.003``|``-Perl-5.24.1``|``intel/2017a``

### Perseus

The Perseus software platform supports biological and biomedical researchers     in interpreting protein quantification, interaction and post-translational modification data.

*homepage*: <https://maxquant.org/perseus>

version    |toolchain
-----------|------------------
``2.0.7.0``|``GCCcore/11.2.0``

### PEST++

PEST++ is a software suite aimed at supporting  complex numerical models in the decision-support context.  Much focus has been devoted to supporting environmental models  (groundwater, surface water, etc) but these tools are readily  applicable to any computer model. 

*homepage*: <https://github.com/usgs/pestpp>

version  |toolchain
---------|--------------
``5.0.5``|``foss/2020a``

### PETSc

PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the  scalable (parallel) solution of scientific applications modeled by partial differential equations.

*homepage*: <http://www.mcs.anl.gov/petsc>

version   |versionsuffix       |toolchain
----------|--------------------|-------------------------------
``3.7.2`` |``-Python-2.7.11``  |``intel/2016a``
``3.7.3`` |``-Python-2.7.11``  |``foss/2016a``
``3.7.5`` |``-downloaded-deps``|``intel/2016b``
``3.8.3`` |``-downloaded-deps``|``foss/2017b``
``3.9.1`` |``-downloaded-deps``|``foss/2018a``
``3.9.3`` |                    |``foss/2018a``, ``intel/2018a``
``3.11.0``|``-downloaded-deps``|``foss/2018b``
``3.11.1``|``-Python-3.7.2``   |``foss/2019a``, ``intel/2019a``
``3.12.4``|``-Python-2.7.16``  |``intel/2019b``
``3.12.4``|``-Python-3.7.4``   |``foss/2019b``, ``intel/2019b``
``3.12.4``|``-Python-3.8.2``   |``foss/2020a``, ``intel/2020a``
``3.14.4``|                    |``foss/2020b``, ``intel/2020b``
``3.15.1``|                    |``foss/2021a``, ``intel/2021a``
``3.17.4``|                    |``foss/2022a``
``3.18.4``|                    |``intel/2021b``

### petsc4py

petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.

*homepage*: <https://bitbucket.org/petsc/petsc4py>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``3.9.1`` |``-Python-3.6.4``|``foss/2018a``
``3.12.0``|``-Python-3.7.4``|``foss/2019b``
``3.15.0``|                 |``foss/2021a``

### PfamScan

PfamScan is used to search a FASTA sequence against a library of Pfam HMM.

*homepage*: <https://www.ebi.ac.uk/seqdb/confluence/display/THD/PfamScan>

version|toolchain
-------|---------------
``1.6``|``gompi/2022a``

### pfind

Drop-in replacement for find, implemented for using parallel access and MPI.

*homepage*: <https://github.com/VI4IO/pfind>

version     |toolchain
------------|---------------
``20220613``|``gompi/2021b``

### pftoolsV3

A suite of tools to build and search generalized profiles (protein and DNA).

*homepage*: <https://github.com/sib-swiss/pftools3>

version     |toolchain
------------|--------------------------------------
``3.2.11``  |``GCCcore/10.3.0``, ``foss/2021a``
``3.2.12``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``20160324``|``foss/2016a``

### pFUnit

pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.

*homepage*: <https://github.com/Goddard-Fortran-Ecosystem/pFUnit>

version  |toolchain
---------|---------------
``3.2.9``|``gompi/2018b``
``4.2.0``|``gompi/2020b``

### PGDSpider

An automated data conversion tool for connecting population genetics and genomics programs

*homepage*: <http://cmpg.unibe.ch/software/PGDSpider/>

version    |versionsuffix     |toolchain
-----------|------------------|----------
``2.1.0.3``|``-Java-1.7.0_80``|``system``

### PGI

C, C++ and Fortran compilers from The Portland Group - PGI

*homepage*: <https://www.pgroup.com/>

version  |versionsuffix          |toolchain
---------|-----------------------|----------
``15.7`` |``-GNU-4.9.2-2.25``    |``system``
``15.7`` |``-GNU-4.9.3-2.25``    |``system``
``15.10``|``-GCC-4.9.3-2.25``    |``system``
``16.1`` |``-CDK-GCC-4.9.2-2.25``|``system``
``16.3`` |``-GCC-4.9.3-2.25``    |``system``
``16.4`` |``-GCC-5.3.0-2.26``    |``system``
``16.7`` |``-GCC-5.4.0-2.26``    |``system``
``16.10``|``-GCC-5.4.0-2.26``    |``system``
``17.1`` |``-GCC-6.3.0-2.27``    |``system``
``17.3`` |``-GCC-6.3.0-2.28``    |``system``
``17.4`` |``-GCC-6.4.0-2.28``    |``system``
``17.10``|``-GCC-6.4.0-2.28``    |``system``
``18.1`` |``-GCC-7.2.0-2.29``    |``system``
``18.4`` |``-GCC-6.4.0-2.28``    |``system``
``18.7`` |``-GCC-7.3.0-2.30``    |``system``
``18.10``|``-GCC-6.4.0-2.28``    |``system``
``19.1`` |``-GCC-8.2.0-2.31.1``  |``system``
``19.4`` |``-GCC-8.2.0-2.31.1``  |``system``
``19.7`` |``-GCC-8.3.0-2.32``    |``system``
``19.10``|``-GCC-8.3.0-2.32``    |``system``

### PGPLOT

The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable,  device-independent graphics package for making simple scientific graphs. It is intended  for making graphical images of publication quality with minimum effort on the part of  the user. For most applications, the program can be device-independent, and the output  can be directed to the appropriate device at run time.

*homepage*: <https://sites.astro.caltech.edu/~tjp/pgplot/>

version  |toolchain
---------|------------------
``5.2.2``|``GCCcore/11.2.0``

### PHANOTATE

PHANOTATE: a tool to annotate phage genomes

*homepage*: <https://github.com/deprekate/PHANOTATE>

version     |toolchain
------------|--------------
``20190724``|``foss/2018b``

### Phantompeakqualtools

It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data.

*homepage*: <https://github.com/kundajelab/phantompeakqualtools>

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021b``

### PHASE

The program PHASE implements a Bayesian statistical  method for reconstructing haplotypes from population genotype data. Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf

*homepage*: <http://stephenslab.uchicago.edu/phase/download.html>

version  |toolchain
---------|----------
``2.1.1``|``system``

### PHAST

PHAST is a freely available software package for comparative and evolutionary genomics.

*homepage*: <http://compgen.cshl.edu/phast/>

version|toolchain
-------|----------------------------------------------------------
``1.4``|``intel/2017a``
``1.5``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

### Phenoflow

R package offering functionality for the advanced analysis of microbial flow cytometry data

*homepage*: <https://github.com/rprops/Phenoflow_package>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.1.2-20200917``|``-R-4.2.1`` |``foss/2022a``

### PheWAS

Provides an accessible R interface to the phenome wide association study.

*homepage*: <https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package>

version     |versionsuffix|toolchain
------------|-------------|-------------------------------
``0.12``    |``-R-3.3.3`` |``foss/2016b``, ``intel/2016b``
``0.99.5-2``|``-R-3.6.0`` |``foss/2019a``, ``intel/2019a``

### PheWeb

A tool for building PheWAS websites from association files

*homepage*: <https://github.com/statgen/pheweb>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``1.1.20``|``-Python-3.7.4``|``foss/2019b``

### PhiPack

The PhiPack software package implements (in C) a few tests for recombination and can produce refined incompatibility matrices as well.

*homepage*: <http://www.maths.otago.ac.nz/~dbryant/software.html>

version       |toolchain
--------------|----------------------------------------
``2016.06.14``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

### PHLAT

PHLAT is a bioinformatics algorithm that offers HLA typing at four-digit resolution (or higher)  using genome-wide transcriptome and exome sequencing data over a wide range of read lengths and sequencing depths.

*homepage*: <https://sites.google.com/site/phlatfortype/>

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.1``|``-Python-2.7.15``|``foss/2018b``

### phonemizer

The phonemizer allows simple phonemization of words and texts in many languages. Provides both the phonemize command-line tool and the Python function phonemizer.phonemize. It is using four backends: espeak, espeak-mbrola, festival and segments. 

*homepage*: <https://github.com/bootphon/phonemizer>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.2.1``|``-Python-3.8.2``|``gompi/2020a``

### phono3py

phono3py calculates phonon-phonon interaction and related properties using the supercell approach.

*homepage*: <https://atztogo.github.io/phono3py/>

version      |versionsuffix     |toolchain
-------------|------------------|-------------------------------
``1.12.5.35``|``-Python-2.7.14``|``intel/2017b``
``1.12.7.55``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``

### phonopy

Phonopy is an open source package of phonon calculations based on the supercell approach.

*homepage*: <https://atztogo.github.io/phonopy/>

version      |versionsuffix     |toolchain
-------------|------------------|-------------------------------
``1.10.1``   |``-Python-2.7.11``|``intel/2016a``
``1.12.2.20``|``-Python-2.7.14``|``intel/2017b``
``1.12.6.66``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.13.0.64``|``-Python-2.7.14``|``intel/2018a``
``1.14.2``   |``-Python-2.7.15``|``intel/2018b``
``2.0.0``    |``-Python-2.7.14``|``intel/2018a``
``2.2.0``    |``-Python-3.7.2`` |``intel/2019a``
``2.7.1``    |``-Python-3.7.4`` |``intel/2019b``
``2.7.1``    |``-Python-3.8.2`` |``intel/2020a``
``2.12.0``   |                  |``foss/2020b``
``2.16.3``   |                  |``foss/2022a``

### photontorch

Photontorch is a photonic simulator for highly parallel simulation and optimization of photonic circuits in time and frequency domain. Photontorch features CUDA enabled simulation and optimization of photonic circuits. It leverages the deep learning framework PyTorch to view the photonic circuit as essentially a recurrent neural network. This enables the use of native PyTorch optimizers to optimize the (physical) parameters of the circuit. 

*homepage*: <https://docs.photontorch.com/>

version  |toolchain
---------|----------------------------------
``0.4.1``|``foss/2020b``, ``fosscuda/2020b``

### phototonic

Phototonic is an image viewer and organizer

*homepage*: <https://github.com/oferkv/phototonic>

version|toolchain
-------|------------------
``2.1``|``GCCcore/10.3.0``

### PHYLIP

PHYLIP is a free package of programs for inferring phylogenies.

*homepage*: <http://evolution.genetics.washington.edu/phylip>

version  |toolchain
---------|-------------------------------------------------------------------------
``3.696``|``foss/2016a``, ``intel/2016a``
``3.697``|``GCC/6.4.0-2.28``, ``GCC/9.3.0``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

### PhyloBayes-MPI

A Bayesian software for phylogenetic reconstruction using mixture models

*homepage*: <https://github.com/bayesiancook/pbmpi>

version     |toolchain
------------|---------------
``20161021``|``intel/2016b``

### phylokit

C++ library for high performance phylogenetics

*homepage*: <https://github.com/pranjalv123/phylokit>

version|toolchain
-------|--------------------
``1.0``|``GCC/8.2.0-2.31.1``

### phylonaut

Dynamic programming for phylogenetics applications

*homepage*: <https://github.com/pranjalv123/phylonaut>

version     |toolchain
------------|---------------
``20190626``|``gompi/2019a``

### PhyloPhlAn

PhyloPhlAn is an integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes. PhyloPhlAn is an accurate, rapid, and simple-to-use method for large-scale microbial genome characterization and phylogenetic analysis at multiple levels of resolution. 

*homepage*: <https://github.com/biobakery/phylophlan>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0``  |``-Python-3.8.2``|``foss/2020a``
``3.0.2``|                 |``foss/2021a``
``3.0.3``|                 |``foss/2022a``

### PhyML

Phylogenetic estimation using (Maximum) Likelihood

*homepage*: <https://github.com/stephaneguindon/phyml>

version         |toolchain
----------------|--------------
``3.3.20190321``|``foss/2018b``
``3.3.20200621``|``foss/2020b``

### phyx

phyx performs phylogenetics analyses on trees and sequences.

*homepage*: <https://github.com/FePhyFoFum/phyx>

version |toolchain
--------|--------------
``1.01``|``foss/2019a``
``1.3`` |``foss/2022a``

### picard

A set of tools (in Java) for working with next generation sequencing data in the BAM format.

*homepage*: <https://broadinstitute.github.io/picard/>

version    |versionsuffix      |toolchain
-----------|-------------------|----------
``1.39``   |                   |``system``
``1.100``  |                   |``system``
``1.109``  |                   |``system``
``1.119``  |                   |``system``
``1.119``  |``-Java-1.7.0_80`` |``system``
``1.120``  |``-Java-1.8.0_66`` |``system``
``1.141``  |``-Java-1.8.0_74`` |``system``
``2.0.1``  |``-Java-1.8.0_66`` |``system``
``2.1.0``  |                   |``system``
``2.1.0``  |``-Java-1.8.0_74`` |``system``
``2.1.1``  |``-Java-1.8.0_112``|``system``
``2.1.1``  |``-Java-1.8.0_74`` |``system``
``2.2.4``  |``-Java-1.8.0_92`` |``system``
``2.6.0``  |``-Java-1.8.0_131``|``system``
``2.10.1`` |``-Java-1.8.0_131``|``system``
``2.18.5`` |``-Java-1.8.0_162``|``system``
``2.18.11``|``-Java-1.8.0_162``|``system``
``2.18.14``|``-Java-1.8``      |``system``
``2.18.17``|``-Java-1.8``      |``system``
``2.18.27``|``-Java-1.8``      |``system``
``2.20.6`` |``-Java-1.8``      |``system``
``2.21.1`` |``-Java-11``       |``system``
``2.21.6`` |``-Java-11``       |``system``
``2.22.1`` |``-Java-11``       |``system``
``2.25.0`` |``-Java-11``       |``system``
``2.25.1`` |``-Java-11``       |``system``
``2.25.5`` |``-Java-13``       |``system``
``2.26.10``|``-Java-15``       |``system``
``3.0.0``  |``-Java-17``       |``system``

### PICI-LIGGGHTS

UoB Positron Imaging Centre's Improved LIGGGHTS distribution with  an emphasis on the Python interface.

*homepage*: <https://github.com/uob-positron-imaging-centre/PICI-LIGGGHTS>

version  |toolchain
---------|--------------
``3.8.1``|``foss/2022a``

### pigz

pigz, which stands for parallel implementation of gzip, is a fully  functional replacement for gzip that exploits multiple processors and multiple  cores to the hilt when compressing data. pigz was written by Mark Adler, and  uses the zlib and pthread libraries. 

*homepage*: <https://zlib.net/pigz/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------
``2.3.3``|``foss/2016b``
``2.3.4``|``GCCcore/6.4.0``
``2.4``  |``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``
``2.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7``  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### PIL

The Python Imaging Library (PIL) adds image processing capabilities to your Python interpreter.  This library supports many file formats, and provides powerful image processing and graphics capabilities.

*homepage*: <http://www.pythonware.com/products/pil>

version  |versionsuffix                    |toolchain
---------|---------------------------------|-------------------------------
``1.1.7``|``-Python-2.7.11``               |``intel/2016a``
``1.1.7``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``1.1.7``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.1.7``|``-Python-2.7.13``               |``intel/2017a``
``1.1.7``|``-Python-2.7.15``               |``foss/2019a``

### PileOMeth

PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments.  PileOMeth extracts per-base methylation metrics from them.  PileOMeth requires an indexed fasta file containing the reference genome as well. 

*homepage*: <https://github.com/dpryan79/PileOMeth>

version   |toolchain
----------|--------------
``0.1.11``|``foss/2016b``

### Pillow

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: <https://pillow.readthedocs.org/>

version  |versionsuffix                    |toolchain
---------|---------------------------------|---------------------------------------------------
``3.2.0``|``-Python-2.7.11``               |``intel/2016a``
``3.2.0``|``-Python-2.7.11-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``
``3.4.2``|``-Python-2.7.12``               |``intel/2016b``
``3.4.2``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``3.4.2``|``-Python-3.5.2-freetype-2.6.5`` |``intel/2016b``
``4.1.0``|``-Python-2.7.13``               |``intel/2017a``
``4.1.1``|``-Python-3.6.1``                |``intel/2017a``
``4.2.1``|``-Python-3.6.1``                |``intel/2017a``
``4.3.0``|``-Python-2.7.13``               |``intel/2017a``
``4.3.0``|``-Python-2.7.14``               |``intel/2017b``
``4.3.0``|``-Python-3.6.3``                |``foss/2017b``
``5.0.0``|``-Python-2.7.14``               |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``5.0.0``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``5.0.0``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``
``5.3.0``|``-Python-2.7.15``               |``foss/2018b``
``5.3.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.0``|                                 |``GCCcore/8.2.0``
``6.2.1``|                                 |``GCCcore/8.3.0``
``6.2.2``|``-Python-2.7.18``               |``foss/2020b``
``7.0.0``|``-Python-3.8.2``                |``GCCcore/9.3.0``
``8.0.1``|                                 |``GCCcore/10.2.0``
``8.2.0``|                                 |``GCCcore/10.3.0``
``8.3.1``|                                 |``GCCcore/11.2.0``
``8.3.2``|                                 |``GCCcore/11.2.0``
``9.1.0``|                                 |``GCCcore/10.3.0``
``9.1.1``|                                 |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``9.2.0``|                                 |``GCCcore/10.2.0``
``9.4.0``|                                 |``GCCcore/12.2.0``

### Pillow-SIMD

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: <https://github.com/uploadcare/pillow-simd>

version        |versionsuffix    |toolchain
---------------|-----------------|---------------------------------------------------
``5.0.0``      |``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``
``5.3.0.post0``|``-Python-3.6.6``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.x.post0``|                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``7.1.2``      |                 |``GCCcore/10.2.0``
``7.1.2``      |``-Python-3.8.2``|``GCCcore/9.3.0``
``8.2.0``      |                 |``GCCcore/10.3.0``
``8.3.1``      |                 |``GCCcore/11.2.0``
``8.3.2``      |                 |``GCCcore/11.2.0``
``9.2.0``      |                 |``GCCcore/11.3.0``

### Pilon

Pilon is an automated genome assembly improvement and variant detection tool

*homepage*: <https://github.com/broadinstitute/pilon>

version |versionsuffix      |toolchain
--------|-------------------|----------
``1.22``|``-Java-1.8``      |``system``
``1.22``|``-Java-1.8.0_162``|``system``
``1.23``|``-Java-1.8``      |``system``
``1.23``|``-Java-11``       |``system``

### PIMS

PIMS is a lazy-loading interface to sequential data with numpy-like slicing.

*homepage*: <https://soft-matter.github.io/pims>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.4.1``|``-Python-2.7.14``|``intel/2017b``

### Pindel

Pindel can detect breakpoints of large deletions, medium sized  insertions, inversions, tandem duplications and other structural variants at single-based  resolution from next-gen sequence data. It uses a pattern growth approach to identify the  breakpoints of these variants from paired-end short reads. 

*homepage*: <http://gmt.genome.wustl.edu/packages/pindel/>

version             |toolchain
--------------------|--------------------------------------------------
``0.2.5b8``         |``foss/2016b``
``0.2.5b9-20170508``|``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/6.4.0-2.28``

### Pingouin

Pingouin is an open-source statistical package written in Python 3 and based mostly on Pandas and  NumPy.

*homepage*: <https://pingouin-stats.org/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.8``|``-Python-3.7.4``|``foss/2019b``

### Pint

Pint is a Python package to define, operate and manipulate physical quantities: the product of a numerical value and a unit of measurement. It allows arithmetic operations between them and conversions from and to different units.

*homepage*: <https://github.com/hgrecco/pint>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.14``  |``-Python-3.7.4``|``GCCcore/8.3.0``
``0.19.2``|                 |``GCCcore/11.2.0``
``0.20.1``|                 |``GCCcore/10.3.0``

### pip

The PyPA recommended tool for installing Python packages.

*homepage*: <https://pip.pypa.io>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``8.0.2``|``-Python-2.7.11``|``intel/2016a``
``8.1.2``|``-Python-2.7.11``|``foss/2016a``
``8.1.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``

### PIPITS

An automated pipeline for analyses of fungal internal transcribed spacer (ITS) sequences  from the Illumina sequencing platform.

*homepage*: <https://github.com/hsgweon/pipits>

version|versionsuffix    |toolchain
-------|-----------------|------------------------------
``2.7``|``-Python-3.8.2``|``foss/2020a``
``2.8``|                 |``foss/2021a``
``3.0``|                 |``foss/2021a``, ``foss/2022a``

### PIRATE

A toolbox for pangenome analysis and threshold evaluation.

*homepage*: <https://github.com/SionBayliss/PIRATE>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.5``|``-R-4.2.1`` |``foss/2022a``

### pIRS

pIRS (profile based Illumina pair-end Reads Simulator) is a program for simulating paired-end reads from a reference genome. It is optimized for simulating reads similar to those generated from the Illumina platform.

*homepage*: <https://github.com/galaxy001/pirs>

version  |toolchain
---------|---------------
``2.0.2``|``gompi/2019b``

### Pisces

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.

*homepage*: <https://github.com/Illumina/Pisces>

version     |toolchain
------------|-----------------
``5.2.7.47``|``GCCcore/6.4.0``

### piSvM

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system.

*homepage*: <http://pisvm.sourceforge.net/>

version|toolchain
-------|---------------
``1.3``|``intel/2017b``

### piSvM-JSC

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system. This version is a fork of the original PiSvM to increase scalability.

*homepage*: <https://github.com/mricherzhagen/pisvm>

version         |toolchain
----------------|---------------
``1.2-20150622``|``intel/2017b``

### pixman

Pixman is a low-level software library for pixel manipulation, providing  features such as image compositing and trapezoid rasterization. Important  users of pixman are the cairo graphics library and the X server. 

*homepage*: <http://www.pixman.org/>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------------------------------
``0.34.0``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.38.0``|``GCCcore/8.2.0``
``0.38.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.40.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``0.42.2``|``GCCcore/12.2.0``

### pizzly

Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.

*homepage*: <https://github.com/pmelsted/pizzly>

version   |toolchain
----------|--------------
``0.37.3``|``foss/2018b``

### pkg-config

pkg-config is a helper tool used when compiling applications and libraries.  It helps you insert the correct compiler options on the command line so an  application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`  for instance, rather than hard-coding values on where to find glib (or other  libraries). 

*homepage*: <http://www.freedesktop.org/wiki/Software/pkg-config/>

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.28``  |``GCC/4.8.2``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``0.29``  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``0.29.1``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``0.29.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017a``, ``system``

### pkgconf

pkgconf is a program which helps to configure compiler and linker flags for development libraries.  It is similar to pkg-config from freedesktop.org.

*homepage*: <https://github.com/pkgconf/pkgconf>

version  |toolchain
---------|--------------------------------------------------
``1.8.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``system``
``1.9.3``|``GCCcore/12.2.0``
``1.9.4``|``GCCcore/13.1.0``
``1.9.5``|``GCCcore/12.3.0``

### pkgconfig

pkgconfig is a Python module to interface with the pkg-config command line tool

*homepage*: <https://github.com/matze/pkgconfig>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------------------
``1.1.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.1.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.1.0``|``-Python-3.5.1`` |``foss/2016a``
``1.1.0``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``1.2.2``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.2.2``|``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.2.2``|``-Python-3.6.1`` |``foss/2017a``, ``intel/2017a``
``1.2.2``|``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.3.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.3.1``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.3.1``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.3.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.5.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.5.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``1.5.1``|``-python``       |``GCCcore/10.2.0``, ``GCCcore/8.2.0``
``1.5.4``|``-python``       |``GCCcore/10.3.0``
``1.5.5``|``-python``       |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### PLAMS

The Python Library for Automating Molecular Simulation (PLAMS) is powerful and flexible Python tool interfaced to the Amsterdam Modeling Suite engines ADF, BAND, DFTB, MOPAC, ReaxFF, and UFF.

*homepage*: <https://www.scm.com/product/plams>

version  |toolchain
---------|---------------
``1.5.1``|``intel/2022a``

### plantcv

PlantCV: Plant phenotyping using computer vision.

*homepage*: <https://pypi.org/project/plantcv/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.8.0``|``-Python-3.8.2``|``foss/2020a``

### PlaScope

Plasmid exploration of bacterial genomes

*homepage*: <https://github.com/GuilhemRoyer/PlaScope>

version  |toolchain
---------|--------------
``1.3.1``|``foss/2018b``

### PlasmaPy

Open source Python ecosystem for plasma research and education

*homepage*: <https://www.plasmapy.org>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.1``|``-Python-3.8.2``|``foss/2020a``

### PLAST

PLAST is a parallel alignment search tool for comparing large protein banks

*homepage*: <http://www.irisa.fr/symbiose/projects/plast/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.1``|``-Java-1.8.0_92``|``foss/2016a``

### Platanus

PLATform for Assembling NUcleotide Sequences

*homepage*: <http://platanus.bio.titech.ac.jp/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.1``|``-linux-x86_64``|``system``
``1.2.4``|                 |``foss/2017a``

### Platypus

Platypus is a tool designed for efficient and accurate variant-detection  in high-throughput sequencing data.

*homepage*: <http://www.well.ox.ac.uk/platypus>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.1``|``-Python-2.7.11``|``intel/2016a``

### plc

plc is the public Planck Likelihood Code.  It provides C and Fortran libraries that allow   users to compute the log likelihoods of the temperature,   polarization, and lensing maps.  Optionally, it also provides a python version of this library,   as well as tools to modify the predetermined options for some likelihoods   (e.g. changing the high-ell and low-ell lmin and lmax values of the temperature). 

*homepage*: <http://pla.esac.esa.int/pla/#home>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.15``|``foss/2019a``, ``intel/2018b``
``3.0.1``|``-Python-3.7.4`` |``foss/2019b``

### PLINK

PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.

*homepage*: <https://www.cog-genomics.org/plink/2.0/>

version                    |toolchain
---------------------------|------------------------------
``1.07``                   |``foss/2016a``, ``foss/2016b``
``1.07-x86_64``            |``system``
``1.9b5``                  |``golf/2020a``
``1.9b_4.1-x86_64``        |``system``
``1.9b_6.17-x86_64``       |``system``
``1.9b_6.21-x86_64``       |``system``
``2.00-alpha1-x86_64``     |``system``
``2.00-alpha2-x86_64``     |``system``
``2.00-alpha2-x86_64_avx2``|``system``
``2.00a2.3``               |``GCC/10.3.0``, ``GCC/11.2.0``
``2.00a2.3_x86_64``        |``system``
``2.00a3.1``               |``GCC/11.2.0``
``2.00a3.6``               |``GCC/11.3.0``
``2.00a3.7``               |``foss/2022a``

### plinkliftover

PLINKLiftOver is a utility enabling liftOver to work on genomics files from PLINK,  allowing one to update the coordinates from one genome reference version to another.

*homepage*: <https://github.com/milescsmith/plinkliftover>

version  |toolchain
---------|--------------
``0.3.0``|``foss/2022b``

### plinkQC

plinkQC is a R/CRAN package for genotype quality control in genetic association studies.  It makes PLINK basic statistics (e.g.missing genotyping rates per individual, allele frequencies per genetic  marker) and relationship functions easily accessible from within R and allows for automatic evaluation of the  results.

*homepage*: <https://meyer-lab-cshl.github.io/plinkQC/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.3.3``|``-R-4.0.0`` |``foss/2020a``

### PLINKSEQ

PLINK/SEQ is an open-source C/C++ library for working with human   genetic variation data. The specific focus is to provide a platform for analytic tool   development for variation data from large-scale resequencing and genotyping projects,   particularly whole-exome and whole-genome studies. It is independent of (but designed   to be complementary to) the existing PLINK package. 

*homepage*: <https://atgu.mgh.harvard.edu/plinkseq/>

version |toolchain
--------|----------------------------------------------------------
``0.10``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

### plot1cell

plot1cell: a package for advanced single cell data visualization

*homepage*: <https://github.com/HaojiaWu/plot1cell>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.0.1``|``-R-4.2.1`` |``foss/2022a``

### Ploticus

Ploticus is a free GPL software utility that can produce various types of plots and graphs

*homepage*: <http://ploticus.sourceforge.net/doc/welcome.html>

version |toolchain
--------|-----------------
``2.42``|``GCCcore/7.3.0``

### plotly

Easily translate 'ggplot2' graphs to an interactive web-based version and/or  create custom web-based visualizations directly from R.

*homepage*: <https://cran.r-project.org/web/packages/plotly>

version  |versionsuffix|toolchain
---------|-------------|---------------
``4.7.1``|``-R-3.4.0`` |``intel/2017a``
``4.8.0``|``-R-3.4.4`` |``intel/2018a``

### plotly-orca

Orca is an Electron app that generates images and reports of Plotly things like  plotly.js graphs, dash apps, dashboards from the command line.

*homepage*: <https://github.com/plotly/orca>

version  |toolchain
---------|-------------------------------------
``1.3.0``|``GCCcore/8.3.0``
``1.3.1``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``

### plotly.py

An open-source, interactive graphing library for Python

*homepage*: <https://plot.ly/python>

version   |toolchain
----------|--------------------------------------
``4.1.0`` |``intel/2019a``
``4.4.1`` |``intel/2019b``
``4.8.1`` |``GCCcore/9.3.0``
``4.14.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``5.1.0`` |``GCCcore/10.3.0``
``5.4.0`` |``GCCcore/11.2.0``
``5.12.0``|``GCCcore/11.3.0``
``5.13.1``|``GCCcore/12.2.0``

### PLplot

PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols  and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer.

*homepage*: <http://plplot.sourceforge.net>

version   |versionsuffix                   |toolchain
----------|--------------------------------|---------------
``5.11.1``|``-Java-1.7.0_80-Python-2.7.11``|``foss/2016a``
``5.11.1``|``-Java-1.7.0_80-Python-2.7.12``|``intel/2016b``

### PLUMED

PLUMED is an open source library for free energy calculations in molecular systems which  works together with some of the most popular molecular dynamics engines. Free energy calculations can be  performed as a function of many order parameters with a particular  focus on biological problems, using  state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.  The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. 

*homepage*: <https://www.plumed.org>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``2.2.2``|                  |``intel/2016a``
``2.2.3``|                  |``foss/2016b``, ``intel/2016b``
``2.3.0``|                  |``foss/2016b``, ``foss/2017a``, ``intel/2016b``
``2.3.4``|                  |``intel/2017b``
``2.4.0``|                  |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.4.0``|``-PathCV``       |``intel/2018a``
``2.4.1``|                  |``iomkl/2018a``
``2.4.2``|                  |``foss/2018b``, ``intel/2018b``
``2.5.0``|                  |``intel/2018b``
``2.5.0``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``
``2.5.1``|                  |``foss/2019a``, ``intel/2018b``
``2.5.1``|``-PathCV``       |``intel/2018b``
``2.5.2``|``-Python-3.7.2`` |``intel/2019a``
``2.5.3``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``2.5.4``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``2.5b`` |``-Python-2.7.14``|``intel/2018a``
``2.6.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.6.2``|                  |``foss/2020b``, ``intel/2020b``
``2.7.0``|                  |``foss/2020b``
``2.7.2``|                  |``foss/2021a``, ``intel/2021a``
``2.7.3``|                  |``foss/2021b``
``2.8.0``|                  |``foss/2021b``
``2.8.1``|                  |``foss/2022a``

### PLY

PLY is yet another implementation of lex and yacc for Python.

*homepage*: <http://www.dabeaz.com/ply/>

version |versionsuffix    |toolchain
--------|-----------------|-----------------
``3.11``|``-Python-3.6.4``|``foss/2018a``
``3.11``|``-Python-3.7.4``|``GCCcore/8.3.0``

### PMIx

Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. 

*homepage*: <https://pmix.org/>

version  |toolchain
---------|-------------------------------------------------------
``1.2.5``|``GCCcore/6.4.0``
``2.1.3``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``2.2.1``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.2.3``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``3.0.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.0.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.1.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``3.1.4``|``GCCcore/8.3.0``
``3.1.5``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.2.3``|``GCCcore/10.3.0``
``4.1.0``|``GCCcore/11.2.0``
``4.1.2``|``GCCcore/11.3.0``
``4.2.2``|``GCCcore/12.2.0``
``4.2.4``|``GCCcore/12.3.0``

### pmt

PMT is a high-level software library capable of      collecting power consumption measurements on various hardware.

*homepage*: <https://git.astron.nl/RD/pmt>

version  |toolchain
---------|------------------
``1.1.0``|``GCCcore/11.3.0``

### pmx

pmx (formerly pymacs) is a small bunch of classes to read structure files such as pdb or gro and trajectory data in gromacs xtc format. Over the years it has been extended towards a versatile (bio-) molecular structure manipulation package with some additional functionalities, e.g. gromacs file parsers and scripts for setup and analysis of free energy calculations.

*homepage*: <https://github.com/deGrootLab/pmx>

version|versionsuffix     |toolchain
-------|------------------|--------------
``2.0``|``-Python-2.7.18``|``foss/2020b``

### PnetCDF

Parallel netCDF: A Parallel I/O Library for NetCDF File Access

*homepage*: <https://trac.mcs.anl.gov/projects/parallel-netcdf>

version   |toolchain
----------|-------------------------------------------------------------------------------------
``1.8.1`` |``intel/2017a``
``1.9.0`` |``intel/2018a``
``1.10.0``|``foss/2018b``, ``intel/2018b``
``1.12.1``|``gompi/2019b``, ``gompi/2020a``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2020a``
``1.12.2``|``gompi/2020b``, ``gompi/2021a``, ``gompic/2020b``, ``iimpi/2021a``
``1.12.3``|``gompi/2021b``, ``gompi/2022a``, ``iimpi/2022a``

### pocl

Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard

*homepage*: <http://portablecl.org>

version|toolchain
-------|----------------------------------------------------------
``1.2``|``GCC/7.3.0-2.30``
``1.3``|``GCC/8.2.0-2.31.1``, ``gcccuda/2019a``
``1.4``|``GCC/8.3.0``, ``gcccuda/2019b``
``1.5``|``GCC/9.3.0``
``1.6``|``GCC/10.2.0``, ``gcccuda/2020b``, ``iccifort/2020.4.304``
``1.8``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

### pod5-file-format

POD5 is a file format for storing nanopore dna data in an easily accessible way.  The format is able to be written in a streaming manner which allows a sequencing  instrument to directly write the format.

*homepage*: <https://github.com/nanoporetech/pod5-file-format>

version  |toolchain
---------|--------------
``0.1.8``|``foss/2022a``

### poetry

Python packaging and dependency management made simple. Poetry helps you declare, manage and install  dependencies of Python projects, ensuring you have the right stack everywhere.

*homepage*: <https://python-poetry.org>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.0.9``|``-Python-3.8.2``|``GCCcore/9.3.0``
``1.2.2``|                 |``GCCcore/11.3.0``

### polars

Lightning-fast DataFrame library for Rust and Python.

*homepage*: <https://pola.rs/>

version   |toolchain
----------|--------------
``0.15.6``|``foss/2022a``

### polymake

polymake is open source software for research in polyhedral geometry. It deals with polytopes, polyhedra and fans as well as simplicial complexes, matroids, graphs, tropical hypersurfaces, and other objects.

*homepage*: <https://polymake.org>

version  |toolchain
---------|------------------------------
``4.0r1``|``foss/2019b``
``4.8``  |``foss/2021b``, ``gfbf/2022a``

### pomkl

Toolchain with PGI C, C++ and Fortran compilers, alongside Intel MKL & OpenMPI.

*homepage*: <http://www.pgroup.com/index.htm>

version    |toolchain
-----------|----------
``2016.03``|``system``
``2016.04``|``system``
``2016.09``|``system``

### pompi

Toolchain with PGI C, C++ and Fortran compilers, alongside OpenMPI.

*homepage*: <http://www.pgroup.com/index.htm>

version    |toolchain
-----------|----------
``2016.03``|``system``
``2016.04``|``system``
``2016.09``|``system``

### poppler

Poppler is a PDF rendering library

*homepage*: <https://poppler.freedesktop.org>

version    |toolchain
-----------|------------------------------
``0.70.1`` |``foss/2018b``
``0.90.1`` |``GCCcore/8.3.0``
``21.06.1``|``GCC/10.2.0``, ``GCC/10.3.0``
``22.01.0``|``GCC/11.2.0``
``22.12.0``|``GCC/11.3.0``

### popscle

A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools 

*homepage*: <https://github.com/statgen/popscle>

version              |toolchain
---------------------|--------------
``0.1-beta``         |``foss/2019b``
``0.1-beta-20210505``|``GCC/11.3.0``

### popt

Popt is a C library for parsing command line parameters.

*homepage*: <http://freecode.com/projects/popt>

version |toolchain
--------|-----------------------------------------
``1.14``|``GCC/4.8.2``
``1.16``|``GCC/10.2.0``, ``GCC/4.9.2``, ``system``

### Porechop

Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.  Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,  it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments  to effectively find adapters, even at low sequence identity

*homepage*: <https://github.com/rrwick/Porechop>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``0.2.3``|``-Python-3.5.2``|``foss/2016b``
``0.2.4``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.2.4``|``-Python-3.6.6``|``foss/2018b``
``0.2.4``|``-Python-3.7.4``|``intel/2019b``

### porefoam

Direct pore-scale simulation of single- and two-phase flow through confined media

*homepage*: <https://github.com/ImperialCollegeLondon/porefoam>

version       |toolchain
--------------|--------------
``2021-09-21``|``foss/2020a``

### poretools

A toolkit for working with nanopore sequencing data from Oxford Nanopore.

*homepage*: <https://poretools.readthedocs.io/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.0``|``-Python-2.7.14``|``intel/2018a``

### Portcullis

Portcullis stands for PORTable CULLing of Invalid Splice junctions from pre-aligned RNA-seq data. It is known that RNAseq mapping tools generate many invalid junction predictions, particularly in deep datasets with high coverage over splice sites. In order to address this, instead for creating a new RNAseq mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file generated by an RNAseq mapper of the user's own choice (e.g. Tophat2, Gsnap, STAR2 or HISAT2) as input (i.e. it's portable). It then, analyses and quantifies all splice junctions in the file before, filtering (culling) those which are unlikely to be genuine. Portcullis output's junctions in a variety of formats making it suitable for downstream analysis (such as differential splicing analysis and gene modelling) without additional work. Portcullis can also filter the original BAM file removing alignments associated with bad junctions.

*homepage*: <https://github.com/maplesond/portcullis>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.2``|``-Python-3.7.4``|``foss/2019b``

### Postgres-XL

Postgres-XL is a horizontally scalable open source SQL database cluster, flexible enough to handle varying database workloads:

*homepage*: <http://www.postgres-xl.org>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``9.5r1``|``-Python-2.7.11``|``intel/2016a``

### PostgreSQL

PostgreSQL is a powerful, open source object-relational database system.  It is fully ACID compliant, has full support for foreign keys,  joins, views, triggers, and stored procedures (in multiple languages).  It includes most SQL:2008 data types, including INTEGER,  NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.  It also supports storage of binary large objects, including pictures,  sounds, or video. It has native programming interfaces for C/C++, Java,  .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.

*homepage*: <https://www.postgresql.org/>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------------------------
``9.4.7``|``-Python-2.7.11``|``intel/2016a``
``9.5.2``|``-Python-2.7.11``|``intel/2016a``
``9.6.0``|``-Python-2.7.12``|``intel/2016b``
``9.6.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``10.2`` |``-Python-2.7.14``|``intel/2018a``
``10.3`` |                  |``foss/2018b``
``10.3`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``11.3`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``11.3`` |``-Python-3.7.2`` |``GCCcore/8.2.0``
``12.4`` |                  |``GCCcore/9.3.0``
``13.2`` |                  |``GCCcore/10.2.0``
``13.3`` |                  |``GCCcore/10.3.0``
``13.4`` |                  |``GCCcore/11.2.0``
``14.4`` |                  |``GCCcore/11.3.0``
``15.2`` |                  |``GCCcore/12.2.0``

### POT

POT (Python Optimal Transport) is a Python library provide several solvers for optimization problems  related to Optimal Transport for signal, image processing and machine learning.

*homepage*: <https://github.com/rflamary/POT>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.1``|``-Python-3.6.6``|``intel/2018b``
``0.9.0``|                 |``foss/2022a``

### POV-Ray

The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program  which generates images from a text-based scene description, and is available for a variety  of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning  three-dimensional graphics. The source code is available for those wanting to do their own ports.

*homepage*: <https://www.povray.org/>

version     |toolchain
------------|---------------------------------------------------------------------------------
``3.7.0.0`` |``intel/2016b``
``3.7.0.7`` |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``3.7.0.8`` |``GCC/10.2.0``, ``iccifort/2020.4.304``
``3.7.0.10``|``GCC/11.3.0``

### powerlaw

powerlaw: A Python Package for Analysis of Heavy-Tailed Distributions

*homepage*: <http://www.github.com/jeffalstott/powerlaw>

version|toolchain
-------|--------------
``1.5``|``foss/2022a``

### pp-sketchlib

Library of sketching functions used by PopPUNK

*homepage*: <https://github.com/bacpop/pp-sketchlib>

version  |toolchain
---------|--------------
``2.1.1``|``foss/2022a``

### PPanGGOLiN

PPanGGOLiN is a software suite used to create and manipulate prokaryotic pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage of large scale environmental studies and letting users study the pangenome of uncultivable species. 

*homepage*: <https://github.com/labgem/PPanGGOLiN>

version    |toolchain
-----------|--------------
``1.1.136``|``foss/2021b``

### PPfold

PPfold is a new implementation of pfold, written in Java 6.0. It can predict the consensus secondary  structure of RNA alignments through a stochastic context-free grammar coupled to an evolutionary model.  It can also use data from chemical probing experiments to predict RNA secondary structure. PPfold is multithreaded,  and can solve the structure of much longer alignments than pfold.

*homepage*: <ihttp://www.daimi.au.dk/~compbio/pfold/downloads.html>

version  |versionsuffix     |toolchain
---------|------------------|----------
``3.1.1``|``-Java-1.8.0_66``|``system``

### ppl

The Parma Polyhedra Library (PPL) provides numerical abstractions especially  targeted at applications in the field of analysis and verification of complex systems.

*homepage*: <https://www.bugseng.com/parma-polyhedra-library>

version|toolchain
-------|-----------------
``1.2``|``GCCcore/6.4.0``

### pplacer

Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic  likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give  useful information about uncertainty, and to offer advanced visualization and downstream analysis.

*homepage*: <http://matsen.fhcrc.org/pplacer/>

version        |toolchain
---------------|----------
``1.1.alpha19``|``system``

### pplpy

This Python package provides a wrapper to the C++ Parma Polyhedra Library (PPL).

*homepage*: <https://pypi.org/project/pplpy/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.8.4``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.8.4``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``

### PRANK

PRANK is a probabilistic multiple alignment program for DNA,   codon and amino-acid sequences. PRANK is based on a novel algorithm that treats   insertions correctly and avoids over-estimation of the number of deletion events.

*homepage*: <http://wasabiapp.org/software/prank/>

version   |toolchain
----------|----------------------------------------------------------------------------------------------
``170427``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/9.3.0``, ``intel/2018a``

### PRC

PRC is a stand-alone program for aligning and scoring two profile hidden Markov models.  This can be used to detect remote relationships between profiles more effectively than by doing simple  profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both  hidden Markov models. 

*homepage*: <http//supfam.org/PRC/>

version  |toolchain
---------|---------------
``1.5.6``|``intel/2018a``

### preCICE

preCICE (Precise Code Interaction Coupling Environment) is a coupling library for partitioned multi-physics simulations, including, but not restricted to fluid-structure interaction and conjugate heat transfer simulations. Partitioned means that preCICE couples existing programs (solvers) capable of simulating a subpart of the complete physics involved in a simulation. This allows for the high flexibility that is needed to keep a decent time-to-solution for complex multi-physics scenarios.

*homepage*: <https://www.precice.org/>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.1``|``-Python-3.8.2``|``foss/2020a``
``2.2.0``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``

### PREQUAL

A program to identify and mask regions with non-homologous adjacent characters in FASTA files.

*homepage*: <https://github.com/simonwhelan/prequal>

version |toolchain
--------|------------------
``1.02``|``GCCcore/11.2.0``

### preseq

Software for predicting library complexity and genome coverage in high-throughput sequencing.

*homepage*: <https://smithlabresearch.org/software/preseq>

version  |toolchain
---------|-------------------------------
``2.0.2``|``foss/2016b``
``2.0.3``|``foss/2018b``, ``intel/2018a``
``3.1.2``|``GCC/10.3.0``, ``GCC/11.2.0``
``3.2.0``|``GCC/11.3.0``

### presto

Presto performs a fast Wilcoxon rank sum test and auROC analysis.

*homepage*: <https://github.com/immunogenomics/presto>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.0.0-20200718``|``-R-4.1.2`` |``foss/2021b``
``1.0.0-20230113``|``-R-4.2.1`` |``foss/2022a``

### pretty-yaml

PyYAML-based python module to produce pretty and readable YAML-serialized data. This module is for serialization only, see ruamel.yaml module for literate YAML parsing (keeping track of comments, spacing, line/column numbers of values, etc).

*homepage*: <https://github.com/mk-fg/pretty-yaml>

version    |toolchain
-----------|------------------
``19.12.0``|``GCCcore/8.3.0``
``20.4.0`` |``GCCcore/9.3.0``
``21.10.1``|``GCCcore/10.3.0``

### Primer3

Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').  PCR is an essential and ubiquitous tool in genetics and molecular biology.  Primer3 can also design hybridization probes and sequencing primers.

*homepage*: <https://primer3.org/>

version  |toolchain
---------|---------------------------------------------------------
``2.3.7``|``intel/2017b``
``2.4.0``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``2.5.0``|``GCC/9.3.0``

### PRINSEQ

A bioinformatics tool to PRe-process and show INformation of SEQuence data.

*homepage*: <http://prinseq.sourceforge.net>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.20.4``|``-Perl-5.28.0``|``foss/2018b``
``0.20.4``|``-Perl-5.32.0``|``foss/2020b``
``0.20.4``|``-Perl-5.34.0``|``foss/2021b``

### printproto

X.org PrintProto protocol headers.

*homepage*: <http://xorg.freedesktop.org/>

version  |toolchain
---------|---------------
``1.0.5``|``intel/2016a``

### PRISMS-PF

PRISMS-PF is a powerful, massively parallel finite element code for conducting phase field and  other related simulations of microstructural evolution.

*homepage*: <https://prisms-center.github.io/phaseField>

version  |toolchain
---------|-------------------------------
``2.1.1``|``foss/2019a``, ``intel/2019a``
``2.2``  |``foss/2021a``

### ProbABEL

Tool for genome-wide association  analysis of imputed genetic data.

*homepage*: <http://www.genabel.org/packages/ProbABEL>

version  |toolchain
---------|-----------------------------
``0.5.0``|``GCCcore/9.3.0``, ``system``

### prodigal

Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee.

*homepage*: <https://github.com/hyattpd/Prodigal/>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.6.2``|``GCC/4.9.3-binutils-2.25``
``2.6.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### ProFit

ProFit (pronounced Pro-Fit, not profit!) is designed to be the ultimate program for  performing least squares fits of two or more protein structures. It performs a very simple and  basic function, but allows as much flexibility as possible in performing this procedure. Thus one  can specify subsets of atoms to be considered, specify zones to be fitted by number, sequence, or  by sequence alignment. 

*homepage*: <http://www.bioinf.org.uk/software/profit>

version|toolchain
-------|--------------
``3.3``|``GCC/10.3.0``

### PROJ

Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates

*homepage*: <https://proj.org>

version  |toolchain
---------|---------------------------------------------------------------------------------
``4.9.2``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.9.3``|``foss/2016b``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.0.0``|``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``6.0.0``|``GCCcore/8.2.0``
``6.2.1``|``GCCcore/8.3.0``
``7.0.0``|``GCCcore/9.3.0``
``7.2.1``|``GCCcore/10.2.0``
``8.0.1``|``GCCcore/10.3.0``
``8.1.0``|``GCCcore/11.2.0``
``9.0.0``|``GCCcore/11.3.0``
``9.1.1``|``GCCcore/12.2.0``
``9.2.0``|``GCCcore/12.3.0``

### ProjectQ

An open source software framework for quantum computing

*homepage*: <https://projectq.ch>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.4.2``|``-Python-3.6.6``|``intel/2018b``

### prokka

Prokka is a software tool for the rapid annotation of prokaryotic genomes.

*homepage*: <https://www.vicbioinformatics.com/software.prokka.shtml>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------------------------------------
``1.11``  |``-BioPerl-1.7.0``|``foss/2016b``
``1.13``  |``-BioPerl-1.7.2``|``intel/2018a``
``1.13.4``|                  |``foss/2018b``
``1.13.7``|                  |``gompi/2019a``
``1.14.5``|                  |``gompi/2019a``, ``gompi/2019b``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``

### prompt-toolkit

prompt_toolkit is a Python library for building powerful interactive command lines and  terminal applications.

*homepage*: <https://github.com/jonathanslenders/python-prompt-toolkit>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0.3`` |``-Python-2.7.11``|``foss/2016a``
``1.0.3`` |``-Python-3.5.1`` |``foss/2016a``
``1.0.6`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.0.6`` |``-Python-3.5.2`` |``intel/2016b``
``1.0.13``|``-Python-2.7.12``|``intel/2016b``

### proovread

PacBio hybrid error correction through iterative short read consensus

*homepage*: <https://github.com/BioInf-Wuerzburg/proovread>

version   |toolchain
----------|---------------
``2.14.1``|``intel/2017b``

### propy

Propy is a protein description software. It allows analyzing sequence-derived structural and physicochemical features, which are very useful in representing and distinguishing proteins or peptides of different structural, functional and interaction properties. These have been widely used in developing methods and software for predicting protein structural and functional classes, protein-protein interactions, drug-target interactions, protein substrates, among others.

*homepage*: <https://code.google.com/archive/p/protpy>

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.0``|``-Python-2.7.13``|``foss/2017a``

### Proteinortho

Proteinortho is a tool to detect orthologous genes within different species.

*homepage*: <https://www.bioinf.uni-leipzig.de/Software/proteinortho>

version  |versionsuffix                |toolchain
---------|-----------------------------|---------------
``5.16b``|``-Python-3.6.4-Perl-5.26.1``|``foss/2018a``
``6.2.3``|                             |``gompi/2021b``

### ProtHint

ProtHint is a pipeline for predicting and scoring hints (in the form of introns, start and  stop codons) in the genome of interest by mapping and spliced aligning predicted genes to a database of  reference protein sequences.

*homepage*: <https://github.com/gatech-genemark/ProtHint>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------
``2.4.0``|``-Python-3.7.2``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.4.0``|``-Python-3.7.4``|``iccifort/2019.5.281``

### protobuf

Google Protocol Buffers

*homepage*: <https://github.com/google/protobuf/>

version    |toolchain
-----------|--------------------------------------------------------------------
``2.5.0``  |``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``system``
``2.6.1``  |``system``
``3.0.2``  |``foss/2016a``
``3.2.0``  |``foss/2016b``, ``intel/2016b``
``3.3.0``  |``foss/2016b``, ``intel/2017a``
``3.4.0``  |``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``3.5.1``  |``intel/2017b``
``3.6.0``  |``GCCcore/7.3.0``
``3.6.1``  |``GCCcore/7.3.0``
``3.6.1.2``|``GCCcore/8.2.0``
``3.7.1``  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.10.0`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``3.13.0`` |``GCCcore/9.3.0``
``3.14.0`` |``GCCcore/10.2.0``
``3.17.3`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.19.4`` |``GCCcore/11.3.0``
``3.21.9`` |``GCCcore/10.3.0``
``23.0``   |``GCCcore/12.2.0``

### protobuf-python

Python Protocol Buffers runtime library.

*homepage*: <https://github.com/google/protobuf/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``3.0.2`` |``-Python-2.7.11``|``foss/2016a``
``3.0.2`` |``-Python-3.5.1`` |``foss/2016a``
``3.2.0`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.2.0`` |``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``3.3.0`` |``-Python-2.7.13``|``intel/2017a``
``3.3.0`` |``-Python-3.5.2`` |``foss/2016b``
``3.3.0`` |``-Python-3.6.1`` |``intel/2017a``
``3.4.0`` |``-Python-2.7.13``|``intel/2017a``
``3.4.0`` |``-Python-2.7.14``|``intel/2017b``
``3.4.0`` |``-Python-3.6.1`` |``intel/2017a``
``3.4.0`` |``-Python-3.6.3`` |``intel/2017b``
``3.6.0`` |``-Python-2.7.15``|``fosscuda/2018b``
``3.6.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``3.10.0``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``3.13.0``|``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``3.14.0``|                  |``GCCcore/10.2.0``
``3.17.3``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.19.4``|                  |``GCCcore/11.3.0``
``4.23.0``|                  |``GCCcore/12.2.0``

### protozero

Minimalistic protocol buffer decoder and encoder in C++.

*homepage*: <https://github.com/mapbox/protozero>

version  |toolchain
---------|-----------------
``1.6.8``|``GCCcore/7.3.0``
``1.7.0``|``GCCcore/8.3.0``

### PRSice

PRSice (pronounced 'precise') is a Polygenic Risk  Score software for calculating, applying, evaluating and  plotting the results of polygenic risk scores (PRS) analyses.

*homepage*: <https://www.prsice.info/>

version   |toolchain
----------|-------------------------------------
``2.2.12``|``GCCcore/8.3.0``
``2.3.1`` |``GCCcore/9.3.0``
``2.3.3`` |``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``2.3.5`` |``GCCcore/11.3.0``

### pscom

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: <http://www.par-tec.com>

version     |toolchain
------------|-------------
``5.0.43``  |``GCC/4.8.2``
``5.0.44-1``|``GCC/4.9.2``
``5.0.48-1``|``system``

### PSI

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.

*homepage*: <http://www.psicode.org/>

version           |versionsuffix        |toolchain
------------------|---------------------|---------------
``4.0b6-20160201``|``-mt-Python-2.7.11``|``intel/2016a``

### PSI4

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform computations with more than 2500 basis functions running serially or in parallel. 

*homepage*: <https://www.psicode.org/>

version  |versionsuffix            |toolchain
---------|-------------------------|---------------
``1.0``  |``-Python-2.7.11``       |``intel/2016a``
``1.0``  |``-mt-Python-2.7.11``    |``intel/2016a``
``1.2.1``|``-Python-2.7.15``       |``intel/2018b``
``1.2.1``|``-Python-2.7.15-maxam8``|``intel/2018b``
``1.3.1``|``-Python-3.7.2``        |``foss/2019a``
``1.3.2``|``-Python-3.7.4``        |``intel/2019b``
``1.7``  |                         |``foss/2021b``

### PsiCLASS

PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.

*homepage*: <https://github.com/splicebox/PsiCLASS>

version  |toolchain
---------|--------------
``1.0.3``|``GCC/11.2.0``

### PSIPRED

Accurate protein secondary structure prediction

*homepage*: <http://bioinf.cs.ucl.ac.uk>

version |toolchain
--------|-------------
``4.02``|``GCC/8.3.0``

### psmc

This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model.

*homepage*: <https://github.com/lh3/psmc>

version  |toolchain
---------|------------------------------
``0.6.5``|``foss/2016a``, ``foss/2018a``

### psmpi

ParaStation MPI is an open source high-performance MPI 3.0 implementation, based on MPICH v3. It provides extra low level communication libraries and integration with  various batch systems for tighter process control. 

*homepage*: <https://github.com/ParaStation/psmpi2>

version    |versionsuffix|toolchain
-----------|-------------|-------------
``5.1.0-1``|             |``GCC/4.9.2``
``5.1.5-1``|             |``GCC/4.9.3``
``5.1.5-1``|``-mt``      |``GCC/4.9.3``

### psmpi2

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: <http://www.par-tec.com>

version   |versionsuffix|toolchain
----------|-------------|-------------
``5.0.29``|             |``GCC/4.8.2``
``5.0.29``|``-mt``      |``GCC/4.8.2``

### PSolver

Interpolating scaling function Poisson Solver Library  

*homepage*: <http://bigdft.org/devel-doc/d1/d81/group__PSOLVER.html>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.7.6``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``1.8.3``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

### PSORTb

PSORTb v3.0.4 is the most precise bacterial localization prediction tool available.

*homepage*: <http://psort.org/psortb/index.html>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.0.4``|``-Perl-5.22.1``|``foss/2016a``

### psrecord

psrecord is a small utility that uses the psutil library to record the CPU and  memory activity of a process.

*homepage*: <https://github.com/astrofrog/psrecord>

version|versionsuffix     |toolchain
-------|------------------|---------------
``1.1``|``-Python-2.7.14``|``intel/2018a``
``1.1``|``-Python-2.7.15``|``intel/2018b``
``1.1``|``-Python-3.6.4`` |``intel/2018a``
``1.1``|``-Python-3.6.6`` |``intel/2018b``

### pstoedit

pstoedit translates PostScript and PDF graphics into other vector formats

*homepage*: <http://pstoedit.net/>

version |toolchain
--------|------------------------------------
``3.70``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``

### psutil

A cross-platform process and system utilities module for Python

*homepage*: <https://github.com/giampaolo/psutil>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``4.2.0``|``-Python-2.7.11``|``intel/2016a``
``4.3.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``5.4.3``|``-Python-2.7.14``|``intel/2017b``
``5.4.7``|``-Python-2.7.15``|``foss/2018b``
``5.4.7``|``-Python-3.6.6`` |``foss/2018b``
``5.6.1``|``-Python-2.7.15``|``fosscuda/2018b``
``5.6.3``|                  |``GCCcore/8.2.0``
``5.9.3``|                  |``GCCcore/10.2.0``, ``GCCcore/11.3.0``
``5.9.4``|                  |``GCCcore/11.2.0``

### psycopg2

Psycopg is the most popular PostgreSQL adapter for the Python programming language.

*homepage*: <https://psycopg.org/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.7``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.8.3``|``-Python-3.7.2`` |``foss/2019a``
``2.8.6``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.9.5``|                  |``GCCcore/11.2.0``
``2.9.6``|                  |``GCCcore/11.3.0``

### ptemcee

ptemcee, pronounced "tem-cee", is fork of Daniel Foreman-Mackey's wonderful emcee to implement  parallel tempering more robustly. If you're trying to characterise awkward, multi-model probability  distributions, then ptemcee is your friend.

*homepage*: <https://github.com/willvousden/ptemcee>

version  |toolchain
---------|--------------
``1.0.0``|``foss/2019a``

### PTESFinder

Post-Transcriptional Exon Shuffling (PTES) Identification Pipeline

*homepage*: <https://sourceforge.net/projects/ptesfinder-v1/>

version|toolchain
-------|---------------
``1``  |``intel/2017b``

### pubtcrs

This repository contains C++ source code for the TCR clustering  and correlation analyses described in the manuscript "Human T cell receptor  occurrence patterns encode immune history, genetic background, and receptor  specificity" by William S DeWitt III, Anajane Smith, Gary Schoch, John A Hansen,  Frederick A Matsen IV and Philip Bradley, available on bioRxiv.

*homepage*: <https://github.com/phbradley/pubtcrs>

version     |toolchain
------------|---------------
``20180622``|``intel/2019a``

### pugixml

pugixml is a light-weight C++ XML processing library

*homepage*: <https://pugixml.org/>

version   |toolchain
----------|--------------------------------------
``1.11.4``|``GCCcore/10.3.0``
``1.12.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### pullseq

Utility program for extracting sequences from a fasta/fastq file

*homepage*: <https://github.com/bcthomas/pullseq>

version  |toolchain
---------|---------------------------------------------------------
``1.0.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/7.3.0``

### PuLP

PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to solve linear problems. 

*homepage*: <https://github.com/coin-or/pulp>

version  |toolchain
---------|--------------
``2.5.1``|``foss/2021a``

### purge_dups

purge haplotigs and overlaps in an assembly based on read depth

*homepage*: <https://github.com/dfguan/purge_dups>

version  |toolchain
---------|--------------
``1.2.5``|``foss/2021b``

### py

library with cross-python path, ini-parsing, io, code, log facilities

*homepage*: < https://pylib.readthedocs.org/>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.4.31``|``-Python-2.7.11``|``foss/2016a``
``1.4.31``|``-Python-3.5.1`` |``foss/2016a``

### py-aiger

A python library for manipulating sequential and combinatorial circuits. This module provides the py-aiger extensions: aiger_bv, aiger_cnf, aiger_ptltl, aiger_coins, aiger_gridworld, aiger_dfa

*homepage*: <https://github.com/mvcisback/py-aiger>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``6.1.1`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``6.1.14``|                 |``GCCcore/10.2.0``

### py-aiger-bdd

Aiger to BDD bridge.

*homepage*: <https://github.com/mvcisback/py-aiger-bdd>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.0``|``-Python-3.8.2``|``foss/2020a``

### py-cpuinfo

py-cpuinfo gets CPU info with pure Python.

*homepage*: <https://github.com/workhorsy/py-cpuinfo>

version  |toolchain
---------|----------------------------------------------------------
``5.0.0``|``system``
``8.0.0``|``GCCcore/11.2.0``
``9.0.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### py3Dmol

A simple IPython/Jupyter widget to embed an interactive 3Dmol.js viewer in a notebook.

*homepage*: <https://github.com/3dmol/3Dmol.js/tree/master/py3Dmol>

version        |toolchain
---------------|------------------
``2.0.1.post1``|``GCCcore/11.3.0``

### pyABC

Massively parallel, distributed and scalable ABC-SMC (Approximate Bayesian Computation - Sequential Monte Carlo) for parameter estimation of complex stochastic models.  Implemented in Python with support of the R language. 

*homepage*: <https://github.com/icb-dcm/pyabc>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.10.4``|``-Python-3.7.4``|``foss/2019b``

### PyAMG

PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface.

*homepage*: <https://pyamg.github.io>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.11``|``intel/2016a``
``4.0.0``|                  |``foss/2020b``, ``intel/2020b``
``4.2.3``|                  |``foss/2021a``

### PyAPS3

Python 3 Atmospheric Phase Screen

*homepage*: <https://github.com/AngeliqueBenoit/pyaps3>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20190407``|``-Python-3.7.2``|``foss/2019a``

### pybedtools

pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.

*homepage*: <https://daler.github.io/pybedtools>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------------------------------
``0.7.10``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``0.7.10``|``-Python-3.6.6`` |``foss/2018b``
``0.8.0`` |                  |``foss/2019a``, ``intel/2019a``
``0.8.1`` |                  |``foss/2019b``
``0.8.2`` |                  |``GCC/10.2.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``
``0.8.2`` |``-Python-2.7.18``|``GCC/10.2.0``
``0.9.0`` |                  |``GCC/11.3.0``

### PyBerny

PyBerny is an optimizer of molecular geometries with respect to the total energy, using nuclear gradient information.

*homepage*: <https://github.com/jhrmnn/pyberny>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.6.2``|``-Python-3.8.2``|``intel/2020a``
``0.6.3``|                 |``foss/2022a``

### pyBigWig

A python extension, written in C, for quick access to bigBed  files and access to and creation of bigWig files.

*homepage*: <https://github.com/deeptools/pyBigWig>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------------------------------------------
``0.3.13``|``-Python-3.6.6``|``foss/2018b``
``0.3.17``|                 |``GCCcore/8.2.0``, ``GCCcore/9.3.0``
``0.3.18``|                 |``GCCcore/10.2.0``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``

### pybind11

pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,  mainly to create Python bindings of existing C++ code.

*homepage*: <https://pybind11.readthedocs.io>

version   |versionsuffix     |toolchain
----------|------------------|------------------
``2.2.4`` |``-Python-3.6.4`` |``foss/2018a``
``2.2.4`` |``-Python-3.6.6`` |``intel/2018b``
``2.4.3`` |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.4.3`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.6.0`` |                  |``GCCcore/10.2.0``
``2.6.2`` |                  |``GCCcore/10.3.0``
``2.7.1`` |                  |``GCCcore/11.2.0``
``2.7.1`` |``-Python-2.7.18``|``GCCcore/11.2.0``
``2.9.2`` |                  |``GCCcore/11.3.0``
``2.10.3``|                  |``GCCcore/12.2.0``

### pybinding

Pybinding is a Python package for numerical tight-binding  calculations in solid state physics.

*homepage*: <https://github.com/dean0x7d/pybinding>

version  |toolchain
---------|--------------
``0.9.5``|``foss/2022a``

### PyBioLib

PyBioLib is a Python package for running BioLib applications from Python scripts and the command line. BioLib is a library of biological data science applications. Applications on BioLib range from small bioinformatics utilities to state-of-the-art machine learning algorithms for predicting characteristics of biological molecules.

*homepage*: <https://biolib.com/>

version    |toolchain
-----------|------------------
``1.1.988``|``GCCcore/11.3.0``

### PyCairo

Python bindings for the cairo library

*homepage*: <https://pycairo.readthedocs.io/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``1.10.0``|``-Python-2.7.11``|``intel/2016a``
``1.16.1``|``-Python-3.6.3`` |``foss/2017b``
``1.16.2``|``-Python-2.7.14``|``intel/2017b``
``1.18.0``|``-Python-2.7.14``|``intel/2018a``
``1.18.0``|``-Python-2.7.15``|``foss/2018b``
``1.18.0``|``-Python-3.6.6`` |``foss/2018b``
``1.18.0``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``1.18.2``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.20.0``|                  |``GCCcore/10.2.0``
``1.20.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.21.0``|                  |``GCCcore/11.3.0``

### PyCalib

Python library for classifier calibration

*homepage*: <https://github.com/classifier-calibration/PyCalib>

version       |toolchain
--------------|--------------
``0.1.0.dev0``|``foss/2021b``
``20230531``  |``gfbf/2022b``

### pyccel

Python extension language using accelerators

*homepage*: <https://github.com/pyccel/pyccel>

version  |toolchain
---------|--------------
``1.7.0``|``foss/2022a``

### PyCharm

PyCharm Community Edition: Python IDE for Professional Developers

*homepage*: <https://www.jetbrains.com/pycharm/>

version     |toolchain
------------|----------
``2017.2.3``|``system``
``2019.3.1``|``system``
``2021.1.1``|``system``
``2022.2.2``|``system``
``2022.3.2``|``system``

### PyCheMPS2

PyCheMPS2 is a python interface to CheMPS2, for compilation without MPI. CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.

*homepage*: <https://sebwouters.github.io/CheMPS2>

version   |toolchain
----------|--------------
``1.8.12``|``foss/2022a``

### Pychopper

A tool to identify, orient, trim and rescue full length cDNA reads.

*homepage*: <https://github.com/nanoporetech/pychopper>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.3.1``|``-Python-3.7.4``|``intel/2019b``

### PyCifRW

PyCIFRW provides support for reading and writing CIF (Crystallographic Information Format) files using Python.

*homepage*: <https://bitbucket.org/jamesrhester/pycifrw/src/development>

version  |toolchain
---------|-----------------
``4.4.2``|``GCCcore/8.3.0``

### PyClone

PyClone is a Python package that wraps rclone and provides a threaded interface for  an installation at the host or container level.

*homepage*: <https://gitlab.com/ltgiv/pyclone>

version     |toolchain
------------|------------------
``2020.9b2``|``GCCcore/10.2.0``

### pycma

A stochastic numerical optimization algorithm for difficult (non-convex, ill-conditioned,  multi-modal, rugged, noisy) optimization problems in continuous search spaces, implemented in Python.

*homepage*: <https://github.com/CMA-ES/pycma>

version  |toolchain
---------|---------------
``2.7.0``|``intel/2019a``

### pycocotools

Official APIs for the MS-COCO dataset

*homepage*: <https://pypi.org/project/pycocotools>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``2.0.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.0.1``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.0.2``|``-Python-3.8.2``|``foss/2020a``, ``fosscuda/2020a``
``2.0.4``|                 |``foss/2021a``
``2.0.6``|                 |``foss/2022a``

### pycodestyle

pycodestyle is a tool to check your Python code against some of the style conventions in PEP 8.

*homepage*: <https://pycodestyle.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.0``|``-Python-3.6.4``|``intel/2018a``

### PyCogent

PyCogent is a software library for genomic biology. It is a fully  integrated and thoroughly tested framework for: controlling third-party applications;  devising workflows; querying databases; conducting novel probabilistic analyses of  biological sequence evolution; and generating publication quality graphics.

*homepage*: <http://pycogent.org>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.5.3``|``-Python-2.7.12``|``intel/2016b``
``1.9``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.9``  |``-Python-2.7.15``|``foss/2018b``

### pycoQC

PycoQC computes metrics and generates interactive QC plots for  Oxford Nanopore technologies sequencing data.

*homepage*: <https://tleonardi.github.io/pycoQC>

version  |toolchain
---------|--------------
``2.5.2``|``foss/2021a``

### PyCUDA

PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python.

*homepage*: <https://mathema.tician.de/software/pycuda>

version     |versionsuffix                  |toolchain
------------|-------------------------------|---------------------------------------
``2016.1.2``|``-Python-2.7.12``             |``intel/2016b``
``2017.1.1``|``-CUDA-9.0.176-Python-2.7.14``|``foss/2017b``
``2017.1.1``|``-Python-2.7.14``             |``intel/2018a``
``2018.1``  |``-Python-3.6.4-CUDA-9.1.85``  |``intel/2018a``
``2019.1.2``|``-Python-3.7.4``              |``fosscuda/2019b``, ``intelcuda/2019b``
``2020.1``  |                               |``fosscuda/2020b``

### PycURL

PycURL is a Python interface to libcurl. PycURL can be used to fetch objects identified by a URL  from a Python program, similar to the urllib Python module. PycURL is mature, very fast, and supports a lot of  features.

*homepage*: <http://pycurl.io/>

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``7.43.0.5``|``-Python-3.7.4``|``GCCcore/8.3.0``
``7.45.2``  |                 |``GCCcore/11.3.0``

### PyDamage

Pydamage, is a Python software to automate the process of contig damage identification and estimation. After modelling the ancient DNA damage using the C to T transitions, Pydamage uses a likelihood ratio test to discriminate between truly ancient, and modern contigs originating from sample contamination.

*homepage*: <https://github.com/maxibor/pydamage>

version |toolchain
--------|--------------
``0.70``|``foss/2021a``

### pydantic

Data validation and settings management using Python type hinting.

*homepage*: <https://github.com/samuelcolvin/pydantic>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.6.1`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``1.10.2``|                 |``GCCcore/11.2.0``
``1.10.4``|                 |``GCCcore/11.3.0``

### PyDatastream

Lightweight SOAP client

*homepage*: <https://pypi.python.org/pypi/suds-py3>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.1``|``-Python-3.6.4``|``intel/2018a``

### pydicom

Pure python package for DICOM medical file reading and writing.

*homepage*: <https://pydicom.github.io/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``0.9.9``|``-Python-2.7.11``|``intel/2016a``
``1.2.2``|                  |``GCCcore/8.2.0``
``1.4.2``|                  |``GCCcore/8.3.0``
``2.1.2``|                  |``GCCcore/10.2.0``
``2.1.2``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.2.2``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.3.0``|                  |``GCCcore/11.3.0``

### pydicom-seg

Reading and writing of DICOM-SEG medical image segmentation storage files  using pydicom as DICOM serialization/deserialization library.

*homepage*: <https://github.com/razorx89/pydicom-seg>

version  |toolchain
---------|--------------
``0.4.1``|``foss/2022a``

### pydlpoly

Pydlpoly is a molecular dynamics simulation package which is a modified version of DL-POLY with  a Python language interface.

*homepage*: <http://cmc.aci.ruhr-uni-bochum.de/cmc/>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20150225``|``-Python-2.7.12``|``intel/2016b``
``20150225``|``-Python-2.7.13``|``intel/2017a``

### pydot

Python interface to Graphviz's Dot language.

*homepage*: <https://github.com/pydot/pydot>

version  |toolchain
---------|------------------------------------------------------------------------------
``1.4.1``|``GCCcore/9.3.0``, ``foss/2019b``
``1.4.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### pyEGA3

A basic Python-based EGA download client 

*homepage*: <https://github.com/EGA-archive/ega-download-client>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``3.0.33``|``-Python-3.7.2``|``GCCcore/8.2.0``
``3.4.0`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.4.0`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``4.0.0`` |                 |``GCCcore/11.2.0``

### pyenchant

PyEnchant is a spellchecking library for Python, based on the excellent Enchant library.

*homepage*: <https://pythonhosted.org/pyenchant/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.6.8``|``-Python-2.7.13``|``intel/2017a``

### PyEVTK

EVTK (Export VTK) package allows exporting data to binary VTK files for visualization and data analysis with any of the visualization packages that support VTK files

*homepage*: <https://github.com/paulo-herrera/PyEVTK>

version  |toolchain
---------|--------------
``1.4.1``|``foss/2021b``
``2.0.0``|``foss/2021b``

### pyFAI

Python implementation of fast azimuthal integration.

*homepage*: <https://github.com/silx-kit/pyFAI>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.19.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.20.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.21.3``|                 |``foss/2021b``

### pyfaidx

pyfaidx: efficient pythonic random access to fasta subsequences

*homepage*: <https://pypi.python.org/pypi/pyfaidx>

version    |versionsuffix    |toolchain
-----------|-----------------|------------------
``0.5.9.5``|                 |``GCCcore/10.2.0``
``0.5.9.5``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.6.3.1``|                 |``GCCcore/10.3.0``
``0.7.0``  |                 |``GCCcore/11.2.0``
``0.7.1``  |                 |``GCCcore/11.3.0``
``0.7.2.1``|                 |``GCCcore/12.2.0``

### pyfasta

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files

*homepage*: <https://pypi.org/project/pyfasta/>

version  |toolchain
---------|--------------
``0.5.2``|``foss/2020b``

### PyFFmpeg

Python FFmpeg wrapper

*homepage*: <https://github.com/mhaller/pyffmpeg>

version    |versionsuffix     |toolchain
-----------|------------------|----------------
``2.1beta``|``-Python-2.7.10``|``gimkl/2.11.5``
``2.1beta``|``-Python-2.7.11``|``intel/2016a``

### pyFFTW

A pythonic wrapper around FFTW, the FFT library, presenting a unified interface  for all the supported transforms.

*homepage*: <https://github.com/pyFFTW/pyFFTW>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.11.1``|                 |``intel/2019a``
``0.11.1``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.13.1``|                 |``foss/2022a``

### pyfits

The PyFITS module is a Python library providing access to FITS (Flexible Image Transport System)

*homepage*: <https://pythonhosted.org/pyfits/>

version|versionsuffix     |toolchain
-------|------------------|---------------
``3.5``|``-Python-2.7.15``|``intel/2018b``

### PyFMI

PyFMI is a package for loading and interacting with Functional Mock-Up Units (FMUs),  which are compiled dynamic models compliant with the Functional Mock-Up Interface (FMI)

*homepage*: <https://pypi.org/project/PyFMI/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.4.0``|``-Python-2.7.15``|``intel/2018b``

### PyFoam

A Python library to control OpenFOAM-runs and manipulate OpenFOAM-data.

*homepage*: <http://openfoamwiki.net/index.php/Contrib/PyFoam>

version   |toolchain
----------|--------------
``2020.5``|``foss/2020b``

### PyFR

PyFR is an open-source Python based framework for solving advection-diffusion type problems on streaming architectures using the Flux Reconstruction approach of Huynh. The framework is designed to solve a range of governing systems on mixed unstructured grids containing various element types. It is also designed to target a range of hardware platforms via use of an in-built domain specific language derived from the Mako templating engine.

*homepage*: <http://www.pyfr.org>

version  |versionsuffix                |toolchain
---------|-----------------------------|-------------------
``1.7.6``|``-Python-3.6.4-CUDA-9.1.85``|``intel/2018a``
``1.9.0``|``-Python-3.7.4``            |``intelcuda/2019b``

### PyFrag

PyFrag 2019 resolves three main challenges associated with the automatized computational exploration of reaction mechanisms: 1) the management of multiple parallel calculations to automatically find a reaction path; 2) the monitoring of the entire computational process along with the extraction and plotting of relevant information from large amounts of data; and 3) the analysis and presentation of these data in a clear and informative way.  This module provides the Activation Strain Analysis (ASA) Module of PyFrag 2019 

*homepage*: <https://pyfragdocument.readthedocs.io/en/latest/includeme.html>

version          |versionsuffix|toolchain
-----------------|-------------|---------------
``2019-20220216``|``-ASA``     |``intel/2020b``

### pygccxml

Python package for simple C++ declarations navigation.

*homepage*: <https://pypi.python.org/pypi/pygccxml>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160706``|``-Python-2.7.11``|``foss/2016a``
``20160706``|``-Python-3.5.1`` |``foss/2016a``

### pyGenomeTracks

pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable.

*homepage*: <https://pygenometracks.readthedocs.io>

version|toolchain
-------|--------------
``3.7``|``foss/2021b``
``3.8``|``foss/2022a``

### PyGEOS

PyGEOS is a C/Python library with vectorized geometry functions. The geometry operations are done in  the open-source geometry library GEOS. PyGEOS wraps these operations in NumPy ufuncs providing a performance  improvement when operating on arrays of geometries.

*homepage*: <https://pygeos.readthedocs.io>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.7.1`` |``-Python-3.7.4``|``foss/2019b``
``0.8``   |``-Python-3.8.2``|``foss/2020a``
``0.10.2``|                 |``intel/2020b``
``0.14``  |                 |``gfbf/2022b``

### pyGIMLi

pyGIMLi is an open-source multi-method library for solving inverse  and forward tasks related to geophysical problems. Written in C++ and Python,  it offers both efficiency and flexibility allowing you to quickly build  your own robust inversion applications for the geophysical problem at hand.

*homepage*: <http://www.pygimli.org/>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160803``|``-Python-2.7.11``|``foss/2016a``
``20160803``|``-Python-3.5.1`` |``foss/2016a``

### Pygments

Generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications  that need to prettify source code.

*homepage*: <https://pygments.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.1.3``|``-Python-2.7.11``|``foss/2016a``
``2.1.3``|``-Python-3.5.1`` |``foss/2016a``

### pygmo

pygmo is a scientific Python library for massively parallel optimization.

*homepage*: <https://esa.github.io/pygmo2>

version   |toolchain
----------|----------------------------------------------
``2.16.1``|``foss/2020b``
``2.18.0``|``foss/2021a``, ``foss/2021b``, ``foss/2022a``

### PyGObject

PyGObject is a Python package which provides bindings for GObject based libraries such as GTK, GStreamer, WebKitGTK, GLib, GIO and many more.

*homepage*: <https://pygobject.readthedocs.io/>

version   |versionsuffix     |toolchain
----------|------------------|------------------
``2.28.6``|``-Python-2.7.11``|``intel/2016a``
``2.28.6``|``-Python-2.7.14``|``intel/2017b``
``2.28.7``|``-Python-2.7.14``|``intel/2018a``
``2.28.7``|``-Python-2.7.15``|``foss/2018b``
``3.34.0``|``-Python-2.7.16``|``GCCcore/8.3.0``
``3.34.0``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``3.34.0``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``3.42.1``|                  |``GCCcore/11.3.0``

### pygraphviz

PyGraphviz is a Python interface to the Graphviz graph layout and visualization package.  With PyGraphviz you can create, edit, read, write, and draw graphs using Python to access the Graphviz  graph data structure and layout algorithms.

*homepage*: <https://pygraphviz.github.io/>

version |toolchain
--------|------------------------------
``1.5`` |``foss/2019b``
``1.7`` |``foss/2020b``, ``foss/2021a``
``1.10``|``GCCcore/11.3.0``

### pygrib

Python interface for reading and writing GRIB data

*homepage*: <https://jswhit.github.io/pygrib>

version  |toolchain
---------|--------------
``2.0.4``|``foss/2019a``

### PyGTK

PyGTK lets you to easily create programs with a graphical user interface  using the Python programming language.

*homepage*: <http://www.pygtk.org/>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``2.24.0``|``-Python-2.7.11``|``intel/2016a``
``2.24.0``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``2.24.0``|``-Python-2.7.15``|``foss/2018b``

### PyGTS

PyGTS is a python package used to construct, manipulate,   and perform computations on triangulated surfaces.   It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. 

*homepage*: <https://sourceforge.net/projects/pygts/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.3.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.3.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.3.1``|``-Python-2.7.14``|``intel/2018a``

### PyGWAS

PyGWAS is a library for running Genome Wide Association studies.

*homepage*: <https://github.com/timeu/pygwas>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.2.0``|``-Python-2.7.11``|``foss/2016a``
``1.3.1``|``-Python-2.7.11``|``foss/2016a``
``1.4.0``|``-Python-2.7.11``|``foss/2016a``
``1.5.0``|``-Python-2.7.11``|``foss/2016a``
``1.6.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.7.1``|``-Python-2.7.13``|``foss/2017a``

### pyhdf

Python wrapper around the NCSA HDF version 4 library

*homepage*: <https://github.com/fhs/pyhdf>

version   |toolchain
----------|--------------
``0.10.1``|``foss/2019a``

### PyImageJ

PyImageJ provides a set of wrapper functions for integration between ImageJ2 and Python. It also  supports the original ImageJ API and data structures. A major advantage of this approach is the ability to combine  ImageJ and ImageJ2 with other tools available from the Python software ecosystem, including NumPy, SciPy,  scikit-image, CellProfiler, OpenCV, ITK and many more.

*homepage*: <https://pyimagej.readthedocs.io/>

version  |toolchain
---------|--------------
``1.3.1``|``foss/2021a``

### pyiron

An integrated development environment (IDE) for computational materials science.

*homepage*: <https://github.com/pyiron/pyiron>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.5``|``-Python-3.7.2``|``intel/2019a``
``0.3.0``|``-Python-3.8.2``|``intel/2020a``

### Pyke3

Pyke introduces a form of Logic Programming (inspired by Prolog)  to the Python community by providing a knowledge-based inference engine  (expert system) written in 100% Python.

*homepage*: <http://sourceforge.net/projects/pyke/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.6.6``|``intel/2018b``

### pylift

pylift is an uplift library that provides, primarily:  (1) Fast uplift modeling implementations and (2) Evaluation tools (UpliftEval class).

*homepage*: <https://github.com/df-foundation/pylift>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.5``|``-Python-3.7.4``|``foss/2019b``
``0.1.5``|``-Python-3.8.2``|``foss/2020a``

### Pylint

Pylint is a tool that checks for errors in Python code, tries to enforce  a coding standard and looks for code smells. It can also look for certain type errors,  it can recommend suggestions about how particular blocks can be refactored and  can offer you details about the code's complexity.

*homepage*: <https://www.pylint.org/>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``1.9.3``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``1.9.5``|``-Python-2.7.15``|``GCCcore/8.2.0``
``1.9.5``|``-Python-2.7.16``|``GCCcore/8.3.0``
``2.7.4``|                  |``GCCcore/10.2.0``

### pyMannKendall

A python package for non parametric Mann Kendall family of trend tests.

*homepage*: <https://github.com/mmhs013/pymannkendall>

version  |toolchain
---------|--------------
``1.4.3``|``foss/2022a``

### pymatgen

Python Materials Genomics is a robust materials analysis code that defines core object  representations for structures and molecules with support for many electronic structure codes.

*homepage*: <https://pypi.python.org/pypi/pymatgen>

version       |versionsuffix     |toolchain
--------------|------------------|-------------------------
``3.5.0``     |``-Python-2.7.11``|``intel/2016.02-GCC-4.9``
``4.1.1``     |``-Python-2.7.12``|``intel/2016b``
``4.3.2``     |``-Python-2.7.12``|``intel/2016b``
``4.7.3``     |``-Python-2.7.13``|``intel/2017a``
``2017.10.16``|``-Python-2.7.14``|``intel/2017b``
``2017.10.16``|``-Python-3.6.3`` |``intel/2017b``
``2022.0.4``  |                  |``foss/2020b``
``2023.3.10`` |                  |``foss/2022a``

### pymatgen-db

Pymatgen-db is a database add-on for the Python Materials Genomics (pymatgen)  materials analysis library.

*homepage*: <https://pypi.python.org/pypi/pymatgen-db>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.5``|``-Python-2.7.13``|``intel/2017a``

### pymbar

The pymbar package contains the pymbar suite of tools for the analysis of simulated and experimental data with the multistate Bennett acceptance ratio (MBAR) estimator.

*homepage*: <http://pymbar.readthedocs.io/en/master/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.0.3``|``-Python-3.6.3``|``intel/2017b``
``3.0.3``|``-Python-3.8.2``|``intel/2020a``

### PyMC3

Probabilistic Programming in Python:  Bayesian Modeling and Probabilistic Machine Learning with Theano

*homepage*: <https://docs.pymc.io/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------------------------------------------------------------
``3.8``   |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``3.11.1``|                 |``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021b``

### pymca

The PyMca X-Ray Fluorescence Toolkit, including PyMca5 and fisx.

*homepage*: <https://github.com/vasole/pymca>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``5.6.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``5.6.3``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``5.7.6``|                 |``foss/2021b``

### pymemcache

A comprehensive, fast, pure-Python memcached client.

*homepage*: <https://github.com/pinterest/pymemcache>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.1``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``

### PyMOL

PyMOL is a Python-enhanced molecular graphics tool. It excels at 3D              visualization of proteins, small molecules, density, surfaces, and trajectories. It also includes molecular editing, ray tracing, and movies. Open Source PyMOL   is free to everyone! 

*homepage*: <https://github.com/schrodinger/pymol-open-source>

version  |toolchain
---------|--------------
``2.5.0``|``foss/2020b``

### PyNAST

PyNAST is a reimplementation of the NAST sequence aligner, which has  become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.  This reimplementation is more flexible, faster, and easier to install and maintain than  the original NAST implementation.

*homepage*: <https://biocore.github.io/pynast>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.2.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.2.2``|``-Python-2.7.15``|``foss/2018b``

### pyobjcryst

Python bindings to ObjCryst++, the Object-Oriented Crystallographic Library.

*homepage*: <https://github.com/diffpy/pyobjcryst>

version        |versionsuffix    |toolchain
---------------|-----------------|---------------
``2.1.0.post2``|``-Python-3.8.2``|``intel/2020a``
``2.2.1``      |                 |``foss/2021b``

### PyOD

PyOD is a comprehensive and scalable Python toolkit for detecting outlying objects in multivariate data.

*homepage*: <https://pyod.readthedocs.io>

version  |toolchain
---------|-------------------------------
``0.8.7``|``foss/2020b``, ``intel/2020b``

### Pyomo

Pyomo is a Python-based open-source software package that supports a diverse set of optimization  capabilities for formulating and analyzing optimization models. 

*homepage*: <https://www.pyomo.org/>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------
``5.5.0``|``-Python-2.7.15``|``foss/2018b``
``5.5.0``|``-Python-3.6.6`` |``foss/2018b``
``6.0.1``|                  |``foss/2020b``, ``foss/2021a``
``6.4.2``|                  |``foss/2022a``
``6.5.0``|                  |``foss/2022b``

### PyOpenCL

PyOpenCL lets you access GPUs and other massively parallel compute devices from Python.

*homepage*: <https://mathema.tician.de/software/pyopencl/>

version      |versionsuffix    |toolchain
-------------|-----------------|---------------------------------------------------
``2020.2.2`` |``-Python-3.7.4``|``fosscuda/2019b``
``2021.1.2`` |                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``2021.2.13``|                 |``foss/2021b``
``2021.2.13``|``-CUDA-11.4.1`` |``foss/2021b``

### PyOpenGL

PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.

*homepage*: <http://pyopengl.sourceforge.net>

version    |versionsuffix     |toolchain
-----------|------------------|---------------------------------------------------------
``3.1.1a1``|                  |``GCCcore/8.2.0``
``3.1.1a1``|``-Python-2.7.11``|``intel/2016a``
``3.1.1a1``|``-Python-2.7.12``|``foss/2016b``
``3.1.1a1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``3.1.3b2``|``-Python-2.7.14``|``intel/2018a``
``3.1.3b2``|``-Python-2.7.15``|``foss/2018b``
``3.1.5``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``
``3.1.6``  |                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### pyparsing

The pyparsing module is an alternative approach to creating and executing simple grammars, vs. the traditional lex/yacc approach, or the use of regular expressions. The pyparsing module provides a library of classes that client code uses to construct the grammar directly in Python code.

*homepage*: <https://github.com/pyparsing/pyparsing>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``2.4.6``|``-Python-2.7.16``|``GCCcore/8.3.0``
``3.0.9``|                  |``GCCcore/11.3.0``

### pyperf

The Python pyperf module is a toolkit to write, run and analyze benchmarks

*homepage*: <https://github.com/psf/pyperf>

version  |toolchain
---------|------------------
``2.5.0``|``GCCcore/11.3.0``
``2.6.0``|``GCCcore/12.2.0``

### pyplusplus

Py++ is a code generator for Boost.Python that simplifies  writing Python bindings of a C/C++ library  The tool is implemented as a Python module which is controlled by a user script.

*homepage*: <https://bitbucket.org/ompl/pyplusplus>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160707``|``-Python-2.7.11``|``foss/2016a``
``20160707``|``-Python-3.5.1`` |``foss/2016a``

### pypmt

PMT is a high-level software library capable of      collecting power consumption measurements on various hardware.

*homepage*: <https://git.astron.nl/RD/pmt>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2022a``

### PYPOWER

PYPOWER is a power flow and Optimal Power Flow (OPF) solver. It is a port of MATPOWER to the Python programming language.

*homepage*: <https://github.com/rwl/PYPOWER>

version   |toolchain
----------|--------------
``5.1.15``|``foss/2020b``

### pyproj

Python interface to PROJ4 library for cartographic transformations

*homepage*: <https://pyproj4.github.io/pyproj>

version        |versionsuffix    |toolchain
---------------|-----------------|--------------------------------------
``2.1.3``      |                 |``GCCcore/8.2.0``
``2.4.2``      |``-Python-3.7.4``|``GCCcore/8.3.0``
``2.6.1.post1``|``-Python-3.8.2``|``GCCcore/9.3.0``
``3.0.1``      |                 |``GCCcore/10.2.0``
``3.1.0``      |                 |``GCCcore/10.3.0``
``3.3.1``      |                 |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.4.0``      |                 |``GCCcore/11.3.0``
``3.5.0``      |                 |``GCCcore/12.2.0``

### PyPSA

PyPSA is an open source toolbox for simulating and optimising modern power systems that include features such as conventional generators with unit commitment, variable wind and solar generation, storage units, coupling to other energy sectors, and mixed alternating and direct current networks. PyPSA is designed to scale well with large networks and long time series.

*homepage*: <https://pypsa.readthedocs.io/>

version   |toolchain
----------|--------------
``0.17.1``|``foss/2020b``

### pyqstem

QSTEM is a program for quantitative image simulation in electron microscopy, including TEM, STEM and CBED image simulation.  This project interfaces the QSTEM code with Python and the Atomic Simulation Environment (ASE) to provide a single environment for building models, simulating and analysing images.

*homepage*: <https://github.com/jacobjma/PyQSTEM>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.0.3``|``-ASE-3.22.0``  |``foss/2020b``, ``fosscuda/2020b``
``1.0.3``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``

### PyQt

PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework.

*homepage*: <http://www.riverbankcomputing.co.uk/software/pyqt>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``4.11.4``|``-Python-2.7.11``|``intel/2016a``
``4.11.4``|``-Python-2.7.12``|``intel/2016b``
``4.12``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.12``  |``-Python-2.7.13``|``intel/2017a``
``4.12.1``|``-Python-2.7.14``|``foss/2018a``
``4.12.3``|``-Python-2.7.15``|``fosscuda/2018b``

### PyQt5

PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.

*homepage*: <http://www.riverbankcomputing.co.uk/software/pyqt>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------
``5.7``   |``-Python-2.7.11``|``foss/2016a``
``5.7.1`` |``-Python-2.7.12``|``intel/2016b``
``5.8.2`` |``-Python-2.7.13``|``intel/2017a``
``5.9.2`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``5.9.2`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``5.11.3``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``5.12.1``|``-Python-2.7.15``|``GCCcore/8.2.0``
``5.12.1``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``5.13.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``5.15.1``|                  |``GCCcore/10.2.0``
``5.15.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.15.4``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.15.5``|                  |``GCCcore/11.3.0``

### PyQtGraph

PyQtGraph is a pure-python graphics and GUI library built on PyQt5/PySide2 and numpy.

*homepage*: <http://www.pyqtgraph.org/>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.10.0``|``-Python-3.6.4``|``intel/2018a``
``0.10.0``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``0.10.0``|``-Python-3.7.2``|``intel/2019a``
``0.11.0``|``-Python-3.7.4``|``foss/2019b``
``0.11.1``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.12.3``|                 |``foss/2021a``
``0.13.3``|                 |``foss/2022a``

### pyradiomics

Open-source python package for the extraction of Radiomics features from 2D and 3D   images and binary masks.

*homepage*: <https://pyradiomics.readthedocs.io/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.1``|                 |``foss/2021a``
``3.0.1``|``-Python-3.7.4``|``foss/2019b``

### PyRe

PyRe (Python Reliability) is a Python module for structural reliability analysis.

*homepage*: <https://hackl.science/pyre>

version           |versionsuffix    |toolchain
------------------|-----------------|-------------------------------
``5.0.3-20190221``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``

### PyRETIS

PyRETIS is a Python library for rare event molecular simulations with emphasis on methods based  on transition interface sampling and replica exchange transition interface sampling.

*homepage*: <http://www.pyretis.org>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.0``|``-Python-3.6.6``|``intel/2018b``
``2.5.0``|                 |``foss/2020b``, ``intel/2020b``
``2.5.0``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``

### pyringe

Debugger capable of attaching to and injecting code into python processes.

*homepage*: <https://github.com/google/pyringe>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.2``|``-Python-2.7.11``|``intel/2016a``

### pyro-api

Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.

*homepage*: <https://github.com/pyro-ppl/pyro-api>

version  |toolchain
---------|------------------
``0.1.2``|``fosscuda/2020b``

### pyro-ppl

Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.

*homepage*: <https://github.com/pyro-ppl/pyro>

version  |toolchain
---------|------------------
``1.5.2``|``fosscuda/2020b``

### Pyro4

Pyro means PYthon Remote Objects. It is a library that enables you to build applications in which  objects can talk to eachother over the network, with minimal programming effort.

*homepage*: <https://pypi.python.org/pypi/Pyro4>

version |versionsuffix     |toolchain
--------|------------------|--------------
``4.47``|``-Python-2.7.11``|``foss/2016a``

### PyRosetta

PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling suite. It enables users to design their own custom molecular modeling algorithms using Rosetta sampling methods and energy functions.

*homepage*: <https://www.pyrosetta.org/>

version          |versionsuffix    |toolchain
-----------------|-----------------|-----------------
``4.release-292``|``-Python-3.7.4``|``GCCcore/8.3.0``

### Pysam

Pysam is a python module for reading and manipulating Samfiles.   It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.

*homepage*: <https://github.com/pysam-developers/pysam>

version     |versionsuffix     |toolchain
------------|------------------|--------------------------------------------------------------------------------------------------------------
``0.8.4``   |``-Python-2.7.12``|``intel/2016b``
``0.9.1.4`` |``-Python-2.7.12``|``foss/2016b``
``0.10.0``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.12.0.1``|``-Python-2.7.13``|``intel/2017a``
``0.12.0.1``|``-Python-2.7.14``|``intel/2017b``
``0.12.0.1``|``-Python-3.6.3`` |``intel/2017b``
``0.13``    |``-Python-2.7.14``|``intel/2017b``
``0.13.0``  |``-Python-3.6.3`` |``intel/2017b``
``0.14``    |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.14``    |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.14``    |``-Python-3.6.4`` |``intel/2018a``
``0.14.1``  |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``0.14.1``  |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.15.1``  |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.15.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.15.2``  |                  |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.15.3``  |                  |``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.16.0.1``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``0.16.0.1``|``-Python-2.7.18``|``GCC/10.2.0``
``0.17.0``  |                  |``GCC/11.2.0``
``0.18.0``  |                  |``GCC/11.2.0``
``0.19.1``  |                  |``GCC/11.3.0``
``0.20.0``  |                  |``GCC/11.3.0``
``0.21.0``  |                  |``GCC/12.2.0``

### pysamstats

A Python utility for calculating statistics against genome positions based on sequence alignments from a SAM or BAM file.

*homepage*: <https://github.com/alimanfoo/pysamstats>

version  |toolchain
---------|--------------
``1.1.2``|``foss/2020b``

### PySAT

PySAT is a Python toolkit, which aims at providing a simple and unified interface to a number of state-of-art Boolean satisfiability (SAT) solvers as well as to a variety of cardinality and pseudo-Boolean encodings.

*homepage*: <https://pysathq.github.io/>

version        |versionsuffix    |toolchain
---------------|-----------------|--------------
``0.1.6.dev11``|``-Python-3.8.2``|``GCC/9.3.0``
``0.1.7.dev1`` |                 |``GCC/10.2.0``

### pyScaf

pyScaf orders contigs from genome assemblies utilising several types of information

*homepage*: <https://github.com/lpryszcz/pyScaf>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12a4``|``-Python-2.7.14``|``intel/2017b``

### pySCENIC

pySCENIC is a lightning-fast python implementation of the SCENIC pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.

*homepage*: <https://github.com/aertslab/pySCENIC>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.10.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``0.12.1``|                 |``foss/2022a``

### PySCF

PySCF is an open-source collection of electronic structure modules powered by Python.

*homepage*: <http://www.pyscf.org>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------------------------------------
``1.6.3``|``-Python-3.7.2``|``foss/2019a``
``1.7.6``|                 |``foss/2020b``, ``foss/2021a``, ``gomkl/2021a``
``2.1.1``|                 |``foss/2022a``

### pysheds

Simple and fast watershed delineation in python.

*homepage*: <https://mattbartos.com/pysheds/>

version    |toolchain
-----------|--------------
``0.2.7.1``|``foss/2020b``

### pyshp

Pure Python read/write support for ESRI Shapefile format

*homepage*: <https://github.com/GeospatialPython/pyshp>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.2.12``|``-Python-3.6.2``|``foss/2017b``
``2.1.3`` |                 |``GCCcore/10.2.0``

### PySide2

PySide2 is the official Python module from the Qt for Python project,     which provides access to the complete Qt 5.12+ framework.

*homepage*: <https://pypi.python.org/pypi/PySide2>

version     |toolchain
------------|------------------
``5.14.2.3``|``GCCcore/10.2.0``

### pyslim

A Python API for reading and modifying tskit tree sequence files produced by SLiM, or modifying files produced by other programs (e.g., msprime, fwdpy11, and tsinfer) for use in SLiM.

*homepage*: <https://tskit.dev/pyslim/docs/stable/introduction.html>

version  |toolchain
---------|------------------------------
``1.0.1``|``foss/2021b``, ``foss/2022a``

### pysndfx

A lightweight Python wrapper for SoX - Sound eXchange.  Supported effects range from EQ and compression to phasers, reverb and pitch shifters.

*homepage*: <https://github.com/carlthome/python-audio-effects>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.6``|``-Python-3.7.4``|``foss/2019b``

### Pysolar

Pysolar is a collection of Python libraries for simulating the irradiation  of any point on earth by the sun.

*homepage*: <https://pysolar.org/>

version|versionsuffix    |toolchain
-------|-----------------|-----------------
``0.7``|``-Python-3.6.3``|``intel/2017b``
``0.7``|``-Python-3.6.4``|``intel/2018a``
``0.8``|``-Python-3.6.6``|``intel/2018b``
``0.8``|``-Python-3.7.2``|``GCCcore/8.2.0``

### pyspoa

Python bindings to spoa.

*homepage*: <https://github.com/nanoporetech/pyspoa>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.0.4``|``-Python-3.7.4``|``GCC/8.3.0``
``0.0.8``|                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``
``0.0.9``|                 |``GCC/11.3.0``, ``GCC/12.2.0``

### pysqlite

pysqlite is an interface to the SQLite 3.x embedded relational database engine.  It is almost fully compliant with the Python database API version 2.0 also exposes the unique features of SQLite.

*homepage*: <https://pypi.python.org/pypi/pysqlite>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.8.2``|``-Python-2.7.11``|``foss/2016a``

### PyStan

Python interface to Stan, a package for Bayesian inference  using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo.

*homepage*: <https://github.com/stan-dev/pystan>

version     |versionsuffix    |toolchain
------------|-----------------|-------------------------------
``2.19.0.0``|``-Python-3.6.4``|``intel/2018a``
``2.19.1.1``|                 |``foss/2020b``, ``intel/2020b``
``3.5.0``   |                 |``foss/2021b``

### pysteps

Pysteps is an open-source and community-driven Python library for probabilistic precipitation nowcasting, i.e. short-term ensemble prediction systems.

*homepage*: <https://pysteps.github.io/>

version  |toolchain
---------|--------------
``1.7.1``|``foss/2022a``

### pystran

Toolset of dynamical model STRucture ANalysis algorithms

*homepage*: <https://stijnvanhoey.github.io/pystran/>

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2017.04.20``|``-Python-2.7.14``|``intel/2017b``

### PyTables

PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope  with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the  NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical  parts of the code (generated using Cython), makes it a fast, yet extremely simple to use tool for interactively browsing,  processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and   disk resources so that data takes much less space (specially if on-flight compression is used) than other solutions   such as relational or object oriented databases.

*homepage*: <https://www.pytables.org>

version    |versionsuffix     |toolchain
-----------|------------------|-----------------------------------------------------------------------------------
``3.2.2``  |``-Python-2.7.12``|``foss/2016b``
``3.2.3.1``|``-Python-2.7.12``|``intel/2016b``
``3.3.0``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.3.0``  |``-Python-3.5.2`` |``intel/2016b``
``3.4.2``  |``-Python-2.7.13``|``foss/2017a``
``3.4.2``  |``-Python-3.6.1`` |``intel/2017a``
``3.4.2``  |``-Python-3.6.3`` |``intel/2017b``
``3.4.2``  |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.4.4``  |``-Python-2.7.15``|``foss/2018b``
``3.4.4``  |``-Python-3.5.1`` |``foss/2016a``
``3.4.4``  |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``3.5.2``  |                  |``intel/2019a``
``3.5.2``  |``-Python-2.7.14``|``intel/2018a``
``3.5.2``  |``-Python-2.7.16``|``intel/2019b``
``3.5.2``  |``-Python-2.7.18``|``foss/2020b``
``3.6.1``  |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``
``3.6.1``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``3.6.1``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``3.8.0``  |                  |``foss/2022a``, ``foss/2022b``

### pytesseract

Python-tesseract is an optical character recognition (OCR) tool for python. That is, it will recognize and "read" the text embedded in images.  Python-tesseract is a wrapper for Google's Tesseract-OCR Engine. It is also useful as a stand-alone invocation script to tesseract, as it can read all image types supported by the Pillow and Leptonica imaging libraries, including jpeg, png, gif, bmp, tiff, and others. Additionally, if used as a script, Python-tesseract will print the recognized text instead of writing it to a file.

*homepage*: <https://github.com/madmaze/pytesseract>

version   |toolchain
----------|------------------
``0.3.10``|``GCCcore/11.3.0``

### pytest

The pytest framework makes it simple to write small, readable tests, and can scale to support complex functional testing for applications and libraries.

*homepage*: <https://docs.pytest.org/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.11``|``foss/2016a``
``3.0.1``|``-Python-3.5.1`` |``foss/2016a``
``3.8.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.8.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``3.8.2``|``-Python-2.7.15``|``intel/2018b``
``3.8.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``3.8.2``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.3.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.4.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``4.4.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``6.0.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``7.1.3``|                  |``GCCcore/11.2.0``
``7.2.2``|                  |``GCCcore/11.2.0``

### pytest-benchmark

A pytest fixture for benchmarking code.

*homepage*: <https://github.com/ionelmc/pytest-benchmark>

version  |toolchain
---------|------------------
``3.4.1``|``GCCcore/10.2.0``

### pytest-cpp

Use pytest runner to discover and execute C++ tests.

*homepage*: <http://github.com/pytest-dev/pytest-cpp>

version  |toolchain
---------|------------------
``2.3.0``|``GCCcore/11.3.0``

### pytest-flakefinder

Runs tests multiple times to expose flakiness.

*homepage*: <https://github.com/dropbox/pytest-flakefinder>

version  |toolchain
---------|------------------
``1.1.0``|``GCCcore/11.3.0``

### pytest-rerunfailures

pytest plugin to re-run tests to eliminate flaky failures.

*homepage*: <https://github.com/pytest-dev/pytest-rerunfailures>

version |toolchain
--------|------------------
``11.1``|``GCCcore/11.3.0``

### pytest-shard

pytest plugin to support parallelism across multiple machines.  Shards tests based on a hash of their test name enabling simple parallelism across machines, suitable for a wide variety of continuous integration services. Tests are split at the finest level of granularity, individual test cases, enabling parallelism even if all of your tests are in a single file (or even single parameterized test method). 

*homepage*: <https://github.com/AdamGleave/pytest-shard>

version  |toolchain
---------|------------------
``0.1.2``|``GCCcore/11.3.0``

### pytest-xdist

xdist: pytest distributed testing plugin  The pytest-xdist plugin extends pytest with some unique test execution modes:      * test run parallelization: if you have multiple CPUs or hosts you       can use those for a combined test run. This allows to speed up       development or to use special resources of remote machines.      * --looponfail: run your tests repeatedly in a subprocess. After         each run pytest waits until a file in your project changes and         then re-runs the previously failing tests. This is repeated         until all tests pass after which again a full run is         performed.      * Multi-Platform coverage: you can specify different Python       interpreters or different platforms and run tests in parallel on       all of them.  Before running tests remotely, pytest efficiently “rsyncs” your program source code to the remote place. All test results are reported back and displayed to your local terminal. You may specify different Python versions and interpreters.

*homepage*: <https://github.com/pytest-dev/pytest-xdist>

version  |toolchain
---------|--------------------------------------
``2.1.0``|``GCCcore/10.2.0``
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.5.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### PYTHIA

PYTHIA is a program for the generation of high-energy physics collision events, i.e. for the description of collisions at high energies between electrons, protons, photons and heavy nuclei. It contains theory and models for a number of physics aspects, including hard and soft interactions, parton distributions, initial- and final-state parton showers, multiparton interactions, fragmentation and decay. It is largely based on original research, but also borrows many formulae and other knowledge from the literature. As such it is categorized as a general purpose Monte Carlo event generator.

*homepage*: <https://www.pythia.org>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``8.226``|``-Python-2.7.13``|``intel/2017a``
``8.309``|                  |``foss/2022b``

### Python

Python is a programming language that lets you work more quickly and integrate your systems  more effectively.

*homepage*: <https://python.org/>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.7.9`` |``-bare``        |``GCC/4.8.4``, ``GCC/4.9.2``
``2.7.10``|                 |``gimkl/2.11.5``
``2.7.10``|``-bare``        |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``2.7.11``|                 |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``2.7.11``|``-bare``        |``GCC/4.9.3-2.25``
``2.7.11``|``-libX11-1.6.3``|``intel/2016a``
``2.7.12``|                 |``foss/2016b``, ``intel/2016b``, ``iomkl/2017a``
``2.7.12``|``-bare``        |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``2.7.13``|                 |``foss/2017a``, ``intel/2017a``
``2.7.13``|``-bare``        |``GCCcore/6.3.0``
``2.7.14``|                 |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``2.7.14``|``-bare``        |``GCCcore/6.4.0``
``2.7.15``|                 |``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``2.7.15``|``-bare``        |``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.7.16``|                 |``GCCcore/8.3.0``
``2.7.16``|``-bare``        |``GCCcore/8.3.0``
``2.7.18``|                 |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.7.18``|``-bare``        |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/9.3.0``
``3.5.1`` |                 |``foss/2016a``, ``intel/2016a``
``3.5.2`` |                 |``foss/2016.04``, ``foss/2016b``, ``intel/2016b``
``3.5.2`` |``-bare``        |``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``3.6.1`` |                 |``foss/2017a``, ``intel/2017a``
``3.6.2`` |                 |``foss/2017b``, ``intel/2017b``, ``intel/2018.00``
``3.6.3`` |                 |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intelcuda/2017b``
``3.6.4`` |                 |``foss/2017a``, ``foss/2018a``, ``fosscuda/2018a``, ``golf/2018a``, ``iimkl/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``3.6.6`` |                 |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.0`` |                 |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.2`` |                 |``GCCcore/8.2.0``
``3.7.4`` |                 |``GCCcore/8.3.0``
``3.8.2`` |                 |``GCCcore/9.3.0``
``3.8.6`` |                 |``GCCcore/10.2.0``
``3.9.5`` |                 |``GCCcore/10.3.0``
``3.9.5`` |``-bare``        |``GCCcore/10.3.0``
``3.9.6`` |                 |``GCCcore/11.2.0``
``3.9.6`` |``-bare``        |``GCCcore/11.2.0``
``3.10.4``|                 |``GCCcore/11.3.0``
``3.10.4``|``-bare``        |``GCCcore/11.3.0``
``3.10.8``|                 |``GCCcore/12.2.0``
``3.10.8``|``-bare``        |``GCCcore/12.2.0``
``3.11.2``|``-bare``        |``GCCcore/12.2.0``
``3.11.3``|                 |``GCCcore/12.3.0``

### Python-bundle

Python distribution with a number of widely used  extensions incl. NumPy, SciPy, Matplotlib, JupyterLab, MPI4PY, ... 

*homepage*: <https://simplebuild.io/>

version   |toolchain
----------|--------------
``3.10.4``|``foss/2022a``

### python-docx

python-docx is a Python library for creating and updating Microsoft Word (.docx) files

*homepage*: <https://python-docx.readthedocs.io/en/latest/>

version   |toolchain
----------|--------------------------------------
``0.8.11``|``GCCcore/10.2.0``, ``GCCcore/12.2.0``

### python-hl7

A simple library for parsing messages of Health Level 7 (HL7) version 2.x into Python objects.

*homepage*: <https://github.com/johnpaulett/python-hl7>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``0.3.4``|``-Python-3.7.4``|``GCCcore/8.3.0``

### python-igraph

Python interface to the igraph high performance graph library, primarily aimed at complex network  research and analysis.

*homepage*: <https://igraph.org/python>

version        |versionsuffix     |toolchain
---------------|------------------|----------------------------------
``0.7.1.post6``|``-Python-2.7.14``|``intel/2017b``
``0.7.1.post6``|``-Python-3.6.6`` |``foss/2018b``
``0.8.0``      |                  |``foss/2019b``, ``foss/2020a``
``0.9.0``      |                  |``foss/2020b``, ``fosscuda/2020b``
``0.9.6``      |                  |``foss/2021a``
``0.9.8``      |                  |``foss/2021b``
``0.10.3``     |                  |``foss/2022a``

### python-irodsclient

A python API for iRODS

*homepage*: <https://github.com/irods/python-irodsclient>

version  |toolchain
---------|--------------------------------------
``1.1.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### python-isal

Faster zlib and gzip compatible compression and decompression  by providing python bindings for the isa-l library. 

*homepage*: <https://github.com/pycompression/python-isal>

version   |toolchain
----------|--------------------------------------
``0.11.0``|``GCCcore/10.3.0``
``0.11.1``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``
``1.1.0`` |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### python-Levenshtein

Python extension for computing string edit distances and similarities.

*homepage*: <https://pypi.org/project/python-Levenshtein/>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.12.0``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|``-Python-3.7.4``|``foss/2019b``
``0.12.1``|                 |``foss/2020b``

### python-libsbml

LibSBML Python API.

*homepage*: <https://sbml.org/>

version   |toolchain
----------|--------------
``5.19.7``|``foss/2021a``

### python-louvain

Louvain algorithm for community detection

*homepage*: <https://pypi.org/project/python-louvain>

version |toolchain
--------|--------------
``0.16``|``foss/2022a``

### python-mujoco

This package is the canonical Python bindings for the MuJoCo physics engine. The mujoco package provides direct access to raw MuJoCo C API functions, structs, constants, and enumerations. Structs are provided as Python classes, with Pythonic initialization and deletion semantics. 

*homepage*: <https://www.mujoco.org>

version  |toolchain
---------|--------------
``2.2.2``|``foss/2022a``

### python-parasail

Python Bindings for the Parasail C Library

*homepage*: <https://github.com/jeffdaily/parasail-python>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------
``1.1.12``|``-Python-2.7.14``|``intel/2018a``
``1.1.16``|``-Python-3.6.6`` |``foss/2018b``
``1.2``   |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.2.2`` |``-Python-3.8.2`` |``intel/2020a``
``1.2.3`` |``-Python-3.8.2`` |``foss/2020a``
``1.2.4`` |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``
``1.3.3`` |                  |``foss/2022a``
``1.3.4`` |                  |``foss/2022b``

### python-telegram-bot

This library provides a pure Python, asynchronous interface for the Telegram Bot API.  It's compatible with Python versions 3.7+.

*homepage*: <https://python-telegram-bot.org/>

version   |toolchain
----------|------------------
``20.0a0``|``GCCcore/10.2.0``

### python-weka-wrapper3

Python3 wrapper for the Weka Machine Learning Workbench

*homepage*: <https://github.com/fracpete/python-weka-wrapper3>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.1.11``|``-Python-3.7.4``|``foss/2019b``

### python-xxhash

xxhash is a Python binding for the xxHash library by Yann Collet.

*homepage*: <https://github.com/ifduyue/python-xxhash>

version  |toolchain
---------|------------------
``3.2.0``|``GCCcore/12.2.0``

### pythran

Pythran is an ahead of time compiler for a subset of the Python language, with a focus on scientific computing. It takes a Python module annotated with a few interface description and turns it into a native Python module with the same interface, but (hopefully) faster. 

*homepage*: <https://pythran.readthedocs.io>

version        |versionsuffix    |toolchain
---------------|-----------------|--------------
``0.9.4.post1``|``-Python-3.7.4``|``foss/2019b``

### PyTorch

Tensors and Dynamic neural networks in Python with strong GPU acceleration. PyTorch is a deep learning framework that puts Python first.

*homepage*: <https://pytorch.org/>

version   |versionsuffix                |toolchain
----------|-----------------------------|----------------------------------------------
``0.3.1`` |``-Python-2.7.14``           |``fosscuda/2017b``
``0.3.1`` |``-Python-3.6.3``            |``fosscuda/2017b``
``0.3.1`` |``-Python-3.6.4``            |``intel/2018a``
``0.3.1`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``0.4.1`` |``-Python-3.6.4``            |``intel/2018a``
``1.0.1`` |``-Python-3.6.6``            |``foss/2018b``, ``fosscuda/2018b``
``1.1.0`` |``-Python-3.7.2``            |``foss/2019a``
``1.2.0`` |``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.3.1`` |                             |``fosscuda/2020b``
``1.3.1`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.4.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.6.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.6.0`` |``-Python-3.7.4-imkl``       |``fosscuda/2019b``
``1.7.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.7.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``1.7.1`` |``-Python-3.8.2``            |``fosscuda/2020a``
``1.8.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.8.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``1.9.0`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.9.0`` |``-imkl``                    |``fosscuda/2020b``
``1.10.0``|                             |``foss/2021a``, ``fosscuda/2020b``
``1.10.0``|``-CUDA-11.3.1``             |``foss/2021a``
``1.11.0``|``-CUDA-11.3.1``             |``foss/2021a``
``1.12.0``|                             |``foss/2022a``
``1.12.0``|``-CUDA-11.7.0``             |``foss/2022a``
``1.12.1``|                             |``foss/2021a``, ``foss/2021b``, ``foss/2022a``
``1.12.1``|``-CUDA-11.3.1``             |``foss/2021a``
``1.12.1``|``-CUDA-11.5.2``             |``foss/2021b``
``1.12.1``|``-CUDA-11.7.0``             |``foss/2022a``
``1.13.1``|                             |``foss/2022a``

### PyTorch-bundle

PyTorch with compatible versions of official Torch extensions.

*homepage*: <https://pytorch.org/>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``1.12.1``|``-CUDA-11.7.0``|``foss/2022a``

### pytorch-CycleGAN-pix2pix

PyTorch implementations for both unpaired and paired image-to-image translation.

*homepage*: <https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix>

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20230314``|``-CUDA-11.7.0``|``foss/2022a``

### PyTorch-Geometric

PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.

*homepage*: <https://github.com/rusty1s/pytorch_geometric>

version  |versionsuffix                  |toolchain
---------|-------------------------------|----------------------------------
``1.3.2``|``-Python-3.7.4``              |``foss/2019b``
``1.4.2``|``-Python-3.7.4-PyTorch-1.4.0``|``foss/2019b``
``1.6.3``|                               |``foss/2020b``, ``fosscuda/2020b``
``1.6.3``|``-Python-3.7.4-PyTorch-1.8.1``|``fosscuda/2019b``
``2.0.1``|``-PyTorch-1.9.0``             |``fosscuda/2020b``
``2.1.0``|``-PyTorch-1.12.0-CUDA-11.7.0``|``foss/2022a``
``2.1.0``|``-PyTorch-1.12.1-CUDA-11.3.1``|``foss/2021a``

### PyTorch-Ignite

Ignite is a high-level library to help with training and evaluating neural networks in PyTorch flexibly and transparently.

*homepage*: <https://pytorch-ignite.ai/>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.4.9``|``-CUDA-11.3.1``|``foss/2021a``

### PyTorch-Image-Models

PyTorch Image Models (timm) is a collection of image models, layers, utilities, optimizers, schedulers, data-loaders / augmentations, and reference training / validation scripts that aim to pull together a wide variety of SOTA models with ability to reproduce ImageNet training results.

*homepage*: <https://huggingface.co/docs/timm>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.9.2``|                |``foss/2022a``
``0.9.2``|``-CUDA-11.7.0``|``foss/2022a``

### PyTorch-Lightning

PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers.

*homepage*: <https://pytorchlightning.ai>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.9``|                |``foss/2021a``
``1.5.9``|``-CUDA-11.3.1``|``foss/2021a``
``1.7.7``|                |``foss/2022a``
``1.7.7``|``-CUDA-11.7.0``|``foss/2022a``
``1.8.4``|                |``foss/2022a``
``1.8.4``|``-CUDA-11.7.0``|``foss/2022a``

### PyTorch3D

PyTorch3D is FAIR's library of reusable components for deep learning with 3D data.

*homepage*: <https://pytorch3d.org/>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``0.4.0``|``-PyTorch-1.7.1``|``fosscuda/2020b``

### PyVCF

A Variant Call Format reader for Python.

*homepage*: <https://github.com/jamescasbon/PyVCF>

version  |versionsuffix     |toolchain
---------|------------------|-------------
``0.6.8``|``-Python-2.7.16``|``GCC/8.3.0``

### PyVCF3

A VCFv4.0 and 4.1 parser for Python. The intent of this module is to mimic the csv module in the Python stdlib, as opposed to more flexible serialization formats like JSON or YAML. vcf will attempt to parse the content of each record based on the data types specified in the meta-information lines -- specifically the ##INFO and ##FORMAT lines. If these lines are missing or incomplete, it will check against the reserved types mentioned in the spec. Failing that, it will just return strings. PyVCF3 has been created because the Official PyVCF repository is no longer maintained and do not accept any pull requests. This fork is for python 3 only and has been published on pyPI as PyVCF3.

*homepage*: <https://github.com/dridk/PyVCF3>

version  |toolchain
---------|------------------
``1.0.3``|``GCCcore/11.3.0``

### pyWannier90

A Wannier90 Python interface for VASP and PySCF

*homepage*: <https://github.com/hungpham2017/pyWannier90>

version       |toolchain
--------------|-------------------------------
``2021-12-07``|``foss/2021a``, ``gomkl/2021a``

### PyWavelets

PyWavelets is open source wavelet transform software for Python.

*homepage*: <https://pywavelets.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------
``1.1.1``|                 |``intelcuda/2020b``
``1.1.1``|``-Python-3.7.4``|``intel/2019b``

### PyWBGT

Cython source code for estimating wet bulb globe temperature (WBGT) from datasets of standard meterological measurements using models developed by Liljegren et al (2008)

*homepage*: <https://github.com/QINQINKONG/PyWBGT>

version  |toolchain
---------|------------------------------
``1.0.0``|``foss/2021b``, ``foss/2022a``

### PyYAML

PyYAML is a YAML parser and emitter for the Python programming language.

*homepage*: <https://github.com/yaml/pyyaml>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------------
``3.11`` |``-Python-2.7.11``|``intel/2016a``
``3.12`` |                  |``system``
``3.12`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.12`` |``-Python-2.7.13``|``intel/2017a``
``3.12`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``3.12`` |``-Python-3.5.2`` |``intel/2016b``
``3.12`` |``-Python-3.6.1`` |``intel/2017a``
``3.12`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``3.12`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.13`` |                  |``system``
``3.13`` |``-Python-2.7.15``|``fosscuda/2018b``, ``intel/2018b``
``3.13`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``5.1``  |                  |``GCCcore/8.2.0``
``5.1.2``|                  |``GCCcore/8.3.0``
``5.3``  |                  |``GCCcore/9.3.0``
``5.3.1``|                  |``GCCcore/10.2.0``
``5.4.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``6.0``  |                  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### PyZMQ

Python bindings for ZeroMQ

*homepage*: <https://www.zeromq.org/bindings:python>

version   |versionsuffix          |toolchain
----------|-----------------------|-------------------------------
``15.2.0``|``-Python-2.7.11-zmq4``|``foss/2016a``, ``intel/2016a``
``15.2.0``|``-Python-3.5.1-zmq4`` |``intel/2016a``
``15.3.0``|``-Python-2.7.11-zmq4``|``foss/2016a``
``15.3.0``|``-Python-3.5.1-zmq4`` |``foss/2016a``
``15.4.0``|``-Python-2.7.12-zmq4``|``intel/2016b``
``15.4.0``|``-Python-3.5.2-zmq4`` |``intel/2016b``
``16.0.2``|``-Python-2.7.12-zmq4``|``foss/2016b``, ``intel/2016b``
``16.0.2``|``-Python-2.7.13-zmq4``|``foss/2017a``, ``intel/2017a``
``16.0.2``|``-Python-3.5.2-zmq4`` |``foss/2016b``, ``intel/2016b``
``16.0.3``|``-Python-2.7.14-zmq4``|``intel/2017b``
``17.0.0``|``-Python-2.7.14-zmq4``|``foss/2018a``
``17.0.0``|``-Python-3.6.4-zmq4`` |``foss/2018a``
``18.1.1``|``-Python-3.7.4``      |``GCCcore/8.3.0``
``22.3.0``|                       |``GCCcore/10.3.0``
``24.0.1``|                       |``GCCcore/11.3.0``

## Q


[Q6](#q6) - [QCA](#qca) - [qcat](#qcat) - [QCG-PilotJob](#qcg-pilotjob) - [qcint](#qcint) - [QCxMS](#qcxms) - [QD](#qd) - [QDD](#qdd) - [QEMU](#qemu) - [QGIS](#qgis) - [Qhull](#qhull) - [QIIME](#qiime) - [QIIME2](#qiime2) - [Qiskit](#qiskit) - [QJson](#qjson) - [QML](#qml) - [qnorm](#qnorm) - [qpth](#qpth) - [qrupdate](#qrupdate) - [QScintilla](#qscintilla) - [Qt](#qt) - [Qt5](#qt5) - [Qt5Webkit](#qt5webkit) - [Qtconsole](#qtconsole) - [QtKeychain](#qtkeychain) - [QTLtools](#qtltools) - [qtop](#qtop) - [QtPy](#qtpy) - [Qualimap](#qualimap) - [Quandl](#quandl) - [QuantumESPRESSO](#quantumespresso) - [QUAST](#quast) - [QuaZIP](#quazip) - [QuickFF](#quickff) - [QuickPIC](#quickpic) - [Quip](#quip) - [Quorum](#quorum) - [QuTiP](#qutip) - [Qwt](#qwt) - [QwtPolar](#qwtpolar)


### Q6

EVB, FEP and LIE simulator.

*homepage*: <https://github.com/qusers/Q6>

version     |toolchain
------------|---------------
``20180205``|``gompi/2019a``

### QCA

Taking a hint from the similarly-named Java Cryptography Architecture,  QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes  and conventions. QCA separates the API from the implementation, using plugins known as  Providers. The advantage of this model is to allow applications to avoid linking to or  explicitly depending on any particular cryptographic library. This allows one to easily  change or upgrade crypto implementations without even needing to recompile the  application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX.

*homepage*: <https://userbase.kde.org/QCA>

version  |toolchain
---------|--------------------------------------------------
``2.1.0``|``foss/2016a``, ``intel/2016b``
``2.1.3``|``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``
``2.3.5``|``GCCcore/11.2.0``

### qcat

qcat is a Python command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files

*homepage*: <https://github.com/nanoporetech/qcat/releases>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.1.0``|                 |``foss/2022b``
``1.1.0``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.1.0``|``-Python-3.8.2``|``intel/2020a``

### QCG-PilotJob

A python service for simple execution of many tasks inside a single allocation.

*homepage*: <https://qcg-pilotjob.readthedocs.org>

version   |toolchain
----------|--------------
``0.12.3``|``foss/2021a``
``0.13.1``|``foss/2022a``

### qcint

libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform.

*homepage*: <http://wiki.sunqm.net/libcint>

version   |toolchain
----------|--------------
``3.0.18``|``foss/2019a``

### QCxMS

QCxMS is a quantum chemical based program to calculate electron ionization (EI)  and collision induced dissociation (CID) mass spectra using Born-Oppenheimer Molecular Dynamics  (BO-MD). It is the successor of the QCEIMS program, in which the EI part is exchanged to x to  account for the greater general applicibility of the program. 

*homepage*: <https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html>

version  |toolchain
---------|----------
``5.0.3``|``system``

### QD

Quad Double computation package

*homepage*: <https://github.com/scibuilder/QD>

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.8.9`` |             |``foss/2021a``
``2.3.17``|``-20160110``|``NVHPC/21.2``

### QDD

A user-friendly program to select microsatellite markers and design primers  from large sequencing projects.

*homepage*: <http://net.imbe.fr/~emeglecz/qdd.html>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.1.2``|``-Perl-5.28.0``|``intel/2018b``

### QEMU

QEMU is a generic and open source machine emulator and virtualizer.

*homepage*: <https://www.qemu.org/>

version   |toolchain
----------|---------------
``2.10.1``|``intel/2017b``

### QGIS

QGIS is a user friendly Open Source Geographic Information System (GIS)

*homepage*: <http://www.qgis.org/>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``2.14.12``|``-Python-2.7.12``|``intel/2016b``
``2.18.4`` |``-Python-2.7.12``|``foss/2016b``
``3.4.12`` |``-Python-3.7.2`` |``foss/2019a``
``3.28.1`` |                  |``foss/2021b``

### Qhull

Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,  halfspace intersection about a point, furthest-site Delaunay triangulation,  and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and  higher dimensions. Qhull implements the Quickhull algorithm for computing the  convex hull. 

*homepage*: <http://www.qhull.org>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2015.2``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``2019.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2020.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### QIIME

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: <http://qiime.org/>

version  |toolchain
---------|----------
``1.9.1``|``system``

### QIIME2

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: <http://qiime2.org/>

version    |toolchain
-----------|----------
``2017.10``|``system``
``2018.2`` |``system``
``2019.4`` |``system``
``2019.7`` |``system``
``2020.8`` |``system``
``2020.11``|``system``
``2021.8`` |``system``
``2022.8`` |``system``
``2022.11``|``system``

### Qiskit

Qiskit is an open-source framework for working with noisy quantum computers  at the level of pulses, circuits, and algorithms.

*homepage*: <https://qiskit.org>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.11.1``|``-Python-3.7.2``|``foss/2019a``
``0.12.0``|``-Python-3.7.2``|``foss/2019a``
``0.31.0``|                 |``foss/2021a``

### QJson

QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa.

*homepage*: <http://qjson.sourceforge.net/>

version  |toolchain
---------|----------------------------------------------------------------------
``0.9.0``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``

### QML

QML is a Python2/3-compatible toolkit for representation learning of properties  of molecules and solids.

*homepage*: <https://www.qmlcode.org>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.2.10``|``-Python-2.7.13``|``intel/2017a``

### qnorm

Fast-ish (and correct!) quantile normalization in Python

*homepage*: <https://github.com/Maarten-vd-Sande/qnorm>

version  |toolchain
---------|--------------
``0.8.1``|``foss/2022a``

### qpth

A fast and differentiable QP solver for PyTorch. 

*homepage*: <https://locuslab.github.io/qpth/>

version            |versionsuffix    |toolchain
-------------------|-----------------|--------------
``0.0.13-20190626``|``-Python-3.7.2``|``foss/2019a``

### qrupdate

qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions.

*homepage*: <https://sourceforge.net/projects/qrupdate/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.2``|``GCC/5.4.0-2.26``, ``GCC/8.2.0-2.31.1``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

### QScintilla

QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control

*homepage*: <https://www.riverbankcomputing.com/software/qscintilla>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.9.4`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.10``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.11.2``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``2.11.6``|                  |``GCCcore/11.2.0``

### Qt

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: <http://qt.io/>

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.3.8``|                |``intel/2016a``
``4.8.6``|                |``system``
``4.8.7``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``4.8.7``|``-GLib-2.48.0``|``foss/2016a``, ``intel/2016a``

### Qt5

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: <https://qt.io/>

version   |toolchain
----------|------------------------------------------------------------------------------------
``5.6.0`` |``foss/2016a``, ``intel/2016a``
``5.7.0`` |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``5.7.1`` |``intel/2016b``
``5.8.0`` |``foss/2017a``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.9.3`` |``foss/2017b``
``5.9.8`` |``fosscuda/2018b``
``5.10.1``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``5.11.2``|``foss/2018b``
``5.12.3``|``GCCcore/8.2.0``
``5.13.1``|``GCCcore/8.3.0``
``5.14.1``|``GCCcore/9.3.0``
``5.14.2``|``GCCcore/10.2.0``
``5.15.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.15.5``|``GCCcore/11.3.0``
``5.15.7``|``GCCcore/12.2.0``

### Qt5Webkit

Qt Port of WebKit. WebKit is an open source web browser engine.

*homepage*: <https://github.com/qt/qtwebkit>

version           |toolchain
------------------|----------------------------------------------------------
``5.212.0-alpha3``|``GCCcore/8.2.0``
``5.212.0-alpha4``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### Qtconsole

A rich Qt-based console for working with Jupyter kernels, supporting rich media  output, session export, and more. The Qtconsole is a very lightweight application that largely feels like a        terminal, but provides a number of enhancements only possible in a GUI, such as  inline figures, proper multiline editing with syntax highlighting, graphical     calltips, and more.

*homepage*: <https://jupyter.org/>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``4.7.7``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``5.0.2``|                 |``GCCcore/10.2.0``
``5.3.2``|                 |``GCCcore/11.2.0``
``5.4.0``|                 |``GCCcore/11.3.0``

### QtKeychain

Platform-independent Qt API for storing passwords securely.

*homepage*: <https://github.com/frankosterfeld/qtkeychain>

version   |toolchain
----------|---------------------------------
``0.9.1`` |``GCCcore/8.2.0``, ``foss/2018b``
``0.13.2``|``GCCcore/11.2.0``

### QTLtools

QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs)  in few simple-to-perform steps.

*homepage*: <https://qtltools.github.io/qtltools/>

version  |toolchain
---------|---------------
``1.1``  |``intel/2016b``
``1.3.1``|``foss/2020b``

### qtop

qtop is a nifty command-line tool for monitoring queueing systems, esp. PBS/torque.  It tries to fit as much information as possible in your screen's real estate,  by stitching together the output of commands like pbsnodes -a, qstat & qstat -q.  It is possible to write wrappers for other platforms -people have done so for SGE, OAR etc-  or, even examine traces offline and present the sampled information.

*homepage*: <http://cern.ch/fotis/QTOP/>

version|versionsuffix|toolchain
-------|-------------|----------
``53`` |``-1``       |``system``

### QtPy

QtPy is a small abstraction layer that lets you write applications using a single API call to  either PyQt or PySide. It provides support for PyQt5, PyQt4, PySide2 and PySide.

*homepage*: <https://github.com/spyder-ide/qtpy>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.9.0``|                 |``GCCcore/10.2.0``
``1.9.0``|``-Python-3.7.4``|``GCCcore/8.3.0``
``2.2.1``|                 |``GCCcore/11.2.0``
``2.3.0``|                 |``GCCcore/11.3.0``

### Qualimap

Qualimap 2 is a platform-independent application written in Java and R that provides both  a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of  alignment sequencing data and its derivatives like feature counts.

*homepage*: <http://qualimap.bioinfo.cipf.es/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.2.1``|``-R-3.6.0`` |``foss/2019a``
``2.2.1``|``-R-4.0.3`` |``foss/2020b``
``2.2.1``|``-R-4.1.2`` |``foss/2021b``

### Quandl

A Python library for Quandl’s RESTful API.

*homepage*: <https://pypi.python.org/pypi/Quandl>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``3.4.2``|``-Python-3.6.4``|``intel/2018a``
``3.4.8``|                 |``foss/2019a``
``3.6.1``|                 |``foss/2020b``, ``foss/2021a``

### QuantumESPRESSO

Quantum ESPRESSO  is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). 

*homepage*: <https://www.quantum-espresso.org>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------------------------------------------------------
``5.3.0``|             |``intel/2016.02-GCC-4.9``
``5.4.0``|             |``intel/2016.02-GCC-4.9``
``5.4.0``|``-hybrid``  |``foss/2016b``
``6.0``  |             |``intel/2016b``
``6.1``  |             |``intel/2017a``
``6.2``  |             |``intel/2017b``
``6.2.1``|             |``iomkl/2017b``
``6.3``  |             |``foss/2018b``, ``intel/2018b``
``6.4.1``|             |``intel/2019a``
``6.5``  |             |``intel/2019a``, ``intel/2019b``
``6.6``  |             |``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``intel/2019b``, ``intel/2020a``
``6.7``  |             |``foss/2019b``, ``foss/2020b``, ``foss/2021a``, ``intel/2019b``, ``intel/2021a``, ``iomkl/2019b``
``6.8``  |             |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``7.0``  |             |``foss/2021b``, ``intel/2021b``
``7.1``  |             |``foss/2022a``, ``intel/2022a``

### QUAST

QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison.

*homepage*: <https://github.com/ablab/quast>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------
``4.6.0``|``-Python-3.5.2`` |``foss/2016b``
``4.6.3``|``-Python-3.6.4`` |``foss/2018a``
``5.0.2``|                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``
``5.0.2``|``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``
``5.0.2``|``-Python-2.7.18``|``foss/2020b``
``5.0.2``|``-Python-3.7.2`` |``foss/2019a``
``5.0.2``|``-Python-3.8.2`` |``foss/2020a``
``5.2.0``|                  |``foss/2022a``

### QuaZIP

QuaZIP is the C++ wrapper for Gilles Vollant's ZIP/UNZIP   package (AKA Minizip) using Trolltech's Qt library.

*homepage*: <https://stachenov.github.io/quazip>

version  |toolchain
---------|-----------------
``0.8.1``|``GCCcore/8.2.0``

### QuickFF

QuickFF is a Python package developed at the Center for Molecular Modeling (CMM) to quickly derive accurate force fields from ab initio calculations.

*homepage*: <https://molmod.github.io/QuickFF/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.4``|``-Python-2.7.12``|``intel/2016b``
``2.2.0``|``-Python-2.7.14``|``intel/2017b``
``2.2.4``|``-Python-3.7.2`` |``intel/2019a``
``2.2.4``|``-Python-3.8.2`` |``intel/2020a``
``2.2.7``|``-Python-3.8.2`` |``intel/2020a``

### QuickPIC

QuickPIC is a 3D parallel (MPI & OpenMP Hybrid) Quasi-Static PIC code,  which is developed based on the framework UPIC. QuickPIC can efficiently simulate plasma  based accelerator problems. This is the UCLA Plasma Simulation Group's official open-source  repository for QuickPIC.

*homepage*: <https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource>

version     |toolchain
------------|---------------
``20210224``|``gompi/2021b``

### Quip

Quip compresses next-generation sequencing data with extreme prejudice.  It supports input and output in the FASTQ and SAM/BAM formats,  compressing large datasets to as little as 15% of their original size.

*homepage*: <http://homes.cs.washington.edu/~dcjones/quip>

version  |toolchain
---------|-------------
``1.1.8``|``GCC/4.8.2``

### Quorum

QuorUM is an error corrector for Illumina reads

*homepage*: <http://www.genome.umd.edu/quorum.html>

version  |toolchain
---------|---------------
``1.1.1``|``intel/2017a``

### QuTiP

QuTiP is open-source software for simulating the dynamics of open quantum systems.

*homepage*: <http://qutip.org>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``4.1.0``|``-Python-2.7.12``|``intel/2016b``
``4.3.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``

### Qwt

The Qwt library contains GUI Components and utility classes which are primarily useful for programs  with a technical background.

*homepage*: <https://qwt.sourceforge.io/>

version  |toolchain
---------|----------------------------------------------------
``6.1.2``|``intel/2016a``
``6.1.3``|``foss/2016b``, ``intel/2016b``
``6.1.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``6.1.5``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``6.2.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### QwtPolar

The QwtPolar library contains classes for displaying values on a polar coordinate system.

*homepage*: <http://qwtpolar.sourceforge.net/>

version  |toolchain
---------|----------------------------------------------------------------------
``1.1.1``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``

## R


[R](#r) - [R-bundle-Bioconductor](#r-bundle-bioconductor) - [R-INLA](#r-inla) - [R-keras](#r-keras) - [R-MXM](#r-mxm) - [R-opencv](#r-opencv) - [R-tesseract](#r-tesseract) - [R-transport](#r-transport) - [R2jags](#r2jags) - [Racon](#racon) - [radeontop](#radeontop) - [RaGOO](#ragoo) - [Ragout](#ragout) - [RagTag](#ragtag) - [rampart](#rampart) - [randfold](#randfold) - [randrproto](#randrproto) - [rapidcsv](#rapidcsv) - [RapidJSON](#rapidjson) - [rapidNJ](#rapidnj) - [rapidtide](#rapidtide) - [RAPSearch2](#rapsearch2) - [Rascaf](#rascaf) - [RASPA2](#raspa2) - [rasterio](#rasterio) - [rasterstats](#rasterstats) - [Ratatosk](#ratatosk) - [Raven](#raven) - [RAxML](#raxml) - [RAxML-NG](#raxml-ng) - [Ray-assembler](#ray-assembler) - [Ray-project](#ray-project) - [Raysect](#raysect) - [RBFOpt](#rbfopt) - [rclone](#rclone) - [Rcorrector](#rcorrector) - [RcppGSL](#rcppgsl) - [rCUDA](#rcuda) - [RDFlib](#rdflib) - [RDKit](#rdkit) - [RDP-Classifier](#rdp-classifier) - [RE2](#re2) - [re2c](#re2c) - [Reads2snp](#reads2snp) - [Reapr](#reapr) - [ReaxFF](#reaxff) - [RECON](#recon) - [Red](#red) - [Redis](#redis) - [redis-py](#redis-py) - [Redundans](#redundans) - [ReFrame](#reframe) - [regionmask](#regionmask) - [RegTools](#regtools) - [Relate](#relate) - [RELION](#relion) - [ReMatCh](#rematch) - [REMORA](#remora) - [renderproto](#renderproto) - [RepastHPC](#repasthpc) - [RepeatMasker](#repeatmasker) - [RepeatModeler](#repeatmodeler) - [RepeatScout](#repeatscout) - [request](#request) - [requests](#requests) - [RERconverge](#rerconverge) - [resolos](#resolos) - [retworkx](#retworkx) - [RevBayes](#revbayes) - [rgdal](#rgdal) - [rgeos](#rgeos) - [Rgurobi](#rgurobi) - [rhdf5](#rhdf5) - [RheoTool](#rheotool) - [rickflow](#rickflow) - [RInChI](#rinchi) - [rioxarray](#rioxarray) - [ripunzip](#ripunzip) - [rising](#rising) - [Rivet](#rivet) - [rjags](#rjags) - [RLCard](#rlcard) - [rmarkdown](#rmarkdown) - [Rmath](#rmath) - [rMATS-turbo](#rmats-turbo) - [RMBlast](#rmblast) - [RNA-Bloom](#rna-bloom) - [RNA-SeQC](#rna-seqc) - [RNAclust](#rnaclust) - [RNAcode](#rnacode) - [RNAIndel](#rnaindel) - [RNAmmer](#rnammer) - [rnaQUAST](#rnaquast) - [RNAz](#rnaz) - [RnBeads](#rnbeads) - [Roary](#roary) - [ROCm](#rocm) - [rocm-cmake](#rocm-cmake) - [ROCm-CompilerSupport](#rocm-compilersupport) - [rocm-smi](#rocm-smi) - [rocminfo](#rocminfo) - [ROCR-Runtime](#rocr-runtime) - [ROCT-Thunk-Interface](#roct-thunk-interface) - [ROI_PAC](#roi_pac) - [ROME](#rome) - [ROOT](#root) - [root_numpy](#root_numpy) - [rootpy](#rootpy) - [Rosetta](#rosetta) - [rpmrebuild](#rpmrebuild) - [RPostgreSQL](#rpostgresql) - [rpy2](#rpy2) - [RQGIS3](#rqgis3) - [RSEM](#rsem) - [RSeQC](#rseqc) - [RStan](#rstan) - [rstanarm](#rstanarm) - [RStudio-Server](#rstudio-server) - [RTG-Tools](#rtg-tools) - [Rtree](#rtree) - [ruamel.yaml](#ruamel.yaml) - [Ruby](#ruby) - [Ruby-Tk](#ruby-tk) - [ruffus](#ruffus) - [Rust](#rust) - [rustworkx](#rustworkx)


### R

R is a free software environment for statistical computing  and graphics.

*homepage*: <https://www.r-project.org/>

version  |versionsuffix                |toolchain
---------|-----------------------------|---------------------------------------------------
``3.2.3``|                             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``3.2.3``|``-bare``                    |``foss/2016a``, ``intel/2016a``
``3.2.3``|``-libX11-1.6.3``            |``intel/2016a``
``3.3.1``|                             |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``3.3.3``|``-X11-20160819``            |``foss/2016b``, ``intel/2016b``
``3.3.3``|``-X11-20170314``            |``intel/2017a``
``3.4.0``|``-X11-20170314``            |``intel/2017a``
``3.4.1``|``-X11-20160819``            |``foss/2016b``
``3.4.3``|``-X11-20171023``            |``foss/2017b``, ``intel/2017b``
``3.4.3``|``-X11-20171023-HDF5-1.8.19``|``intel/2017b``
``3.4.4``|``-X11-20180131``            |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.5.0``|``-X11-20180131``            |``iomkl/2018a``
``3.5.1``|                             |``foss/2018b``, ``intel/2018b``
``3.5.1``|``-Python-2.7.15``           |``foss/2018b``
``3.5.1``|``-bare``                    |``foss/2018b``
``3.6.0``|                             |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``3.6.2``|                             |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.6.3``|                             |``foss/2020a``
``4.0.0``|                             |``foss/2020a``, ``fosscuda/2020a``
``4.0.3``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.0.4``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.0.5``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.1.0``|                             |``foss/2021a``
``4.1.2``|                             |``foss/2021b``
``4.2.0``|                             |``foss/2021b``
``4.2.1``|                             |``foss/2022a``
``4.2.2``|                             |``foss/2022b``

### R-bundle-Bioconductor

Bioconductor provides tools for the analysis and coprehension  of high-throughput genomic data.

*homepage*: <https://bioconductor.org>

version |versionsuffix|toolchain
--------|-------------|-------------------------------
``3.2`` |``-R-3.2.3`` |``foss/2016a``, ``intel/2016a``
``3.3`` |``-R-3.3.1`` |``intel/2016b``
``3.5`` |``-R-3.4.0`` |``intel/2017a``
``3.6`` |``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``
``3.6`` |``-R-3.4.4`` |``intel/2018a``
``3.7`` |``-R-3.5.0`` |``iomkl/2018a``
``3.7`` |``-R-3.5.1`` |``foss/2018b``
``3.8`` |``-R-3.5.1`` |``foss/2018b``
``3.9`` |``-R-3.6.0`` |``foss/2019a``
``3.10``|             |``foss/2019b``
``3.11``|``-R-4.0.0`` |``foss/2020a``
``3.12``|``-R-4.0.3`` |``foss/2020b``
``3.13``|``-R-4.1.0`` |``foss/2021a``
``3.14``|``-R-4.1.2`` |``foss/2021b``
``3.15``|``-R-4.2.0`` |``foss/2021b``
``3.15``|``-R-4.2.1`` |``foss/2022a``
``3.16``|``-R-4.2.2`` |``foss/2022b``

### R-INLA

R-INLA is a package in R that do approximate Bayesian inference for Latent Gaussian Models.

*homepage*: <https://www.r-inla.org>

version     |versionsuffix|toolchain
------------|-------------|--------------
``21.05.02``|``-R-4.0.4`` |``foss/2020b``

### R-keras

Interface to 'Keras' <https://keras.io>, a high-level neural networks 'API'. 

*homepage*: <https://cran.r-project.org/web/packages/keras>

version    |versionsuffix            |toolchain
-----------|-------------------------|----------------------------------
``2.1.6``  |``-R-3.4.4``             |``foss/2018a``
``2.2.5.0``|``-Python-3.7.2-R-3.6.0``|``foss/2019a``, ``fosscuda/2019a``
``2.2.5.0``|``-Python-3.7.4-R-3.6.2``|``foss/2019b``, ``fosscuda/2019b``
``2.4.0``  |``-R-4.0.4``             |``foss/2020b``, ``fosscuda/2020b``

### R-MXM

MXM: Feature Selection (Including Multiple Solutions) and Bayesian Networks

*homepage*: <https://cran.r-project.org/package=MXM>

version  |toolchain
---------|--------------
``1.5.5``|``foss/2021b``

### R-opencv

Experimenting with computer vision and machine learning in R. This package exposes some of the available OpenCV algorithms, such as edge, body or face detection. These can either be applied to analyze static images, or to filter live video footage from a camera device.

*homepage*: <https://cran.r-project.org/web/packages/opencv/>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.0``|``-R-4.0.0`` |``foss/2020a``

### R-tesseract

The R extension for using tesseract

*homepage*: <https://cran.r-project.org/package=tesseract>

version  |versionsuffix|toolchain
---------|-------------|--------------
``4.0``  |``-R-3.5.1`` |``foss/2018b``
``5.1.0``|``-R-4.2.1`` |``foss/2022a``

### R-transport

transport: Computation of Optimal Transport Plans and Wasserstein Distances

*homepage*: <https://cran.r-project.org/package=transport>

version   |toolchain
----------|--------------
``0.13-0``|``foss/2021b``

### R2jags

Providing wrapper functions to implement Bayesian analysis in JAGS.

*homepage*: <https://cran.r-project.org/web/packages/R2jags>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.7-1``|``-R-4.2.1`` |``foss/2022a``

### Racon

Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.

*homepage*: <https://github.com/lbcb-sci/racon>

version   |toolchain
----------|--------------------------------------
``1.3.2`` |``GCCcore/8.2.0``
``1.4.7`` |``GCCcore/8.2.0``, ``gcccuda/2019b``
``1.4.10``|``GCC/7.3.0-2.30``
``1.4.13``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.21``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0`` |``GCCcore/11.2.0``

### radeontop

View your GPU utilization, both for the total activity percent and individual blocks.

*homepage*: <https://github.com/clbr/radeontop>

version|toolchain
-------|------------------
``1.3``|``GCCcore/10.3.0``

### RaGOO

A tool to order and orient genome assembly contigs via Minimap2 alignments to a reference genome

*homepage*: <https://github.com/malonge/RaGOO>

version |versionsuffix    |toolchain
--------|-----------------|--------------
``1.11``|``-Python-3.6.6``|``foss/2018b``

### Ragout

Ragout (Reference-Assisted Genome Ordering UTility) is a tool for chromosome-level scaffolding using multiple references. Given initial assembly fragments (contigs/scaffolds) and one or multiple related references (complete or draft), it produces a chromosome-scale assembly (as a set of scaffolds).

*homepage*: <https://github.com/fenderglass/Ragout>

version|versionsuffix     |toolchain
-------|------------------|--------------
``2.0``|``-Python-2.7.12``|``foss/2016b``
``2.3``|                  |``foss/2020b``

### RagTag

RagTag is a collection of software tools for scaffolding and improving modern genome assemblies. Tasks include: homology-based misassembly correction, homology-based assembly scaffolding and patching, and scaffold merging. RagTag also provides command line utilities for working with common genome assembly file formats.

*homepage*: <https://github.com/malonge/RagTag>

version  |toolchain
---------|--------------
``2.0.1``|``foss/2020b``

### rampart

Read Assignment, Mapping, and Phylogenetic Analysis in Real Time.

*homepage*: <https://github.com/artic-network/rampart/>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``1.2.0``   |                 |``foss/2020b``
``1.2.0rc3``|``-Python-3.6.6``|``foss/2018b``

### randfold

Minimum free energy of folding randomization test software

*homepage*: <http://bioinformatics.psb.ugent.be/software/details/Randfold>

version  |toolchain
---------|-------------------------------
``2.0.1``|``foss/2018b``, ``intel/2016b``

### randrproto

Xrandr protocol and ancillary headers

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.5.0``|``foss/2016a``, ``intel/2016a``

### rapidcsv

Rapidcsv is a C++ header-only library for CSV parsing.  While the name admittedly was inspired by the rapidjson project,  the objectives are not the same. The goal of rapidcsv is to be  an simple-to-use CSV library enabling rapid development.  For optimal performance (be it CPU or memory usage) a CSV parser  implemented for the specific use-case is likely to be more performant. 

*homepage*: <https://github.com/d99kris/rapidcsv>

version |toolchain
--------|------------------
``8.64``|``GCCcore/11.3.0``

### RapidJSON

A fast JSON parser/generator for C++ with both SAX/DOM style API

*homepage*: <https://rapidjson.org>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### rapidNJ

RapidNJ is an algorithmic engineered implementation of canonical  neighbour-joining. It uses an efficient search heuristic to speed-up the core  computations of the neighbour-joining method that enables RapidNJ to  outperform other state-of-the-art neighbour-joining implementations.

*homepage*: <https://github.com/somme89/rapidNJ>

version  |toolchain
---------|------------------
``2.3.3``|``GCCcore/11.3.0``

### rapidtide

Rapidtide is a suite of python programs used to perform time delay analysis on functional imaging  data to find time lagged correlations between the voxelwise time series and other time series.

*homepage*: <https://rapidtide.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.8.0``|``-Python-3.7.2``|``intel/2019a``

### RAPSearch2

RAPSearch stands for Reduced Alphabet based Protein similarity Search 

*homepage*: <https://omics.informatics.indiana.edu/mg/RAPSearch2>

version |toolchain
--------|-------------
``2.24``|``GCC/9.3.0``

### Rascaf

Rascaf (RnA-seq SCAFfolder) uses continuity and order information from paired-end RNA-seq reads  to improve a draft assembly, particularly in the gene regions.

*homepage*: <https://github.com/mourisl/Rascaf>

version  |toolchain
---------|---------------
``1.0.2``|``intel/2017a``

### RASPA2

RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials.

*homepage*: <https://iraspa.org/raspa/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.0.3`` |``-Python-2.7.12``|``intel/2016b``
``2.0.41``|                  |``foss/2020b``

### rasterio

Rasterio reads and writes geospatial raster data.

*homepage*: <https://github.com/mapbox/rasterio>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``1.1.7`` |``-Python-3.8.2``|``foss/2020a``
``1.2.3`` |                 |``foss/2020b``
``1.2.10``|                 |``foss/2021b``
``1.3.4`` |                 |``foss/2022a``
``1.3.8`` |                 |``foss/2022b``

### rasterstats

rasterstats is a Python module for summarizing geospatial raster datasets based on vector geometries.

*homepage*: <https://github.com/perrygeo/python-rasterstats>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.15.0``|``-Python-3.8.2``|``foss/2020a``

### Ratatosk

Phased hybrid error correction of long reads using colored de Bruijn graphs

*homepage*: <https://github.com/DecodeGenetics/Ratatosk>

version|toolchain
-------|--------------
``0.4``|``GCC/10.2.0``

### Raven

Raven is a de novo genome assembler for long uncorrected reads.

*homepage*: <https://github.com/lbcb-sci/raven>

version  |toolchain
---------|--------------
``1.8.1``|``GCC/11.2.0``

### RAxML

RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.

*homepage*: <https://github.com/stamatak/standard-RAxML>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------------------------------------
``8.2.4`` |``-hybrid-avx2``  |``foss/2016a``
``8.2.9`` |``-hybrid-avx2``  |``foss/2016a``
``8.2.10``|``-hybrid-avx2``  |``intel/2017a``
``8.2.11``|``-hybrid-avx``   |``foss/2017b``, ``intel/2017b``
``8.2.11``|``-hybrid-avx2``  |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``8.2.11``|``-hybrid-sse3``  |``foss/2017b``, ``intel/2017b``
``8.2.12``|``-hybrid-avx2``  |``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2019b``, ``intel/2018b``, ``intel/2019a``
``8.2.12``|``-pthreads-avx2``|``GCC/10.2.0``

### RAxML-NG

RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)  optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree  Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree.

*homepage*: <https://github.com/amkozlov/raxml-ng>

version  |toolchain
---------|------------------------------
``0.9.0``|``GCC/8.3.0``, ``gompi/2019b``
``1.0.1``|``gompi/2019b``
``1.0.2``|``gompi/2020b``
``1.0.3``|``GCC/10.2.0``
``1.1.0``|``GCC/11.2.0``

### Ray-assembler

Parallel genome assemblies for parallel DNA sequencing

*homepage*: <http://denovoassembler.sourceforge.net/>

version  |toolchain
---------|---------------
``2.3.1``|``iimpi/2019a``

### Ray-project

Ray is a fast and simple framework for building and running distributed applications.

*homepage*: <https://docs.ray.io/en/latest/>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.8.4`` |``-Python-3.7.4``|``foss/2019b``
``1.0.1`` |``-Python-3.7.4``|``fosscuda/2019b``
``1.9.2`` |                 |``foss/2021b``
``1.13.0``|                 |``foss/2021a``, ``foss/2021b``
``2.2.0`` |                 |``foss/2022a``

### Raysect

Raysect is an OOP ray-tracing framework for Python

*homepage*: <https://raysect.org>

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``0.6.0``|``-Python-3.6.6``|``intel/2018b``
``0.7.1``|                 |``foss/2020b``, ``intel/2020b``

### RBFOpt

RBFOpt is a Python library for black-box optimization (also known as derivative-free optimization).

*homepage*: <https://github.com/coin-or/rbfopt>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``4.1.1``|                 |``intel/2019a``
``4.1.1``|``-Python-3.6.6``|``intel/2018b``

### rclone

Rclone is a command line program to sync files and directories to and from  a variety of online storage services 

*homepage*: <https://rclone.org>

version   |versionsuffix|toolchain
----------|-------------|----------
``1.42``  |``-amd64``   |``system``
``1.54.1``|``-amd64``   |``system``
``1.56.0``|``-amd64``   |``system``
``1.57.0``|             |``system``

### Rcorrector

Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.

*homepage*: <https://github.com/mourisl/Rcorrector>

version  |toolchain
---------|---------------
``1.0.2``|``intel/2017a``

### RcppGSL

The 'RcppGSL' package provides an simple-to-use interface  between 'GSL' data structures and R using concepts from 'Rcpp' which is itself  a package that eases the interfaces between R and C++.

*homepage*: <https://cran.r-project.org/web/packages/RcppGSL>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.3.8``|``-R-4.0.4`` |``foss/2020b``

### rCUDA

The rCUDA Framework enables the concurrent usage of CUDA-compatible devices remotely.

*homepage*: <http://www.rcuda.net/>

version  |versionsuffix                 |toolchain
---------|------------------------------|----------
``4.0.1``|``_linux_64_Ubuntu10.04``     |``system``
``5.0``  |``_linux_64_scientificLinux6``|``system``

### RDFlib

RDFLib is a Python library for working with RDF, a simple yet powerful language  for representing information.

*homepage*: <https://github.com/RDFLib/rdflib>

version  |toolchain
---------|--------------------------------------
``4.2.2``|``GCCcore/8.3.0``, ``foss/2019a``
``5.0.0``|``GCCcore/10.2.0``
``6.2.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### RDKit

RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.

*homepage*: <https://www.rdkit.org>

version      |versionsuffix    |toolchain
-------------|-----------------|-------------------------------
``2018.09.3``|``-Python-3.6.6``|``intel/2018b``
``2019.09.3``|``-Python-3.7.4``|``foss/2019b``
``2020.03.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``2020.09.3``|``-Python-3.7.4``|``foss/2019b``
``2021.03.4``|                 |``foss/2021a``
``2022.03.5``|                 |``foss/2021b``
``2022.09.4``|                 |``foss/2022a``

### RDP-Classifier

The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic  assignments from domain to genus, with confidence estimates for each assignment.

*homepage*: <http://sourceforge.net/projects/rdp-classifier>

version |versionsuffix     |toolchain
--------|------------------|----------
``2.7`` |``-Java-1.7.0_60``|``system``
``2.12``|``-Java-1.8``     |``system``
``2.13``|``-Java-11``      |``system``
``2.13``|``-Java-17``      |``system``

### RE2

RE2 is a fast, safe, thread-friendly alternative to backtracking regular expression engines like those used in PCRE, Perl, and Python. It is a C++ library. 

*homepage*: <https://github.com/google/re2>

version       |toolchain
--------------|--------------------------------------
``2020-07-01``|``GCCcore/8.3.0``
``2021-06-01``|``GCCcore/10.2.0``
``2022-02-01``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2022-06-01``|``GCCcore/11.3.0``
``2023-03-01``|``GCCcore/12.2.0``

### re2c

re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons.

*homepage*: <https://re2c.org>

version  |toolchain
---------|--------------------------------------
``1.1.1``|``GCCcore/8.2.0``
``1.2.1``|``GCCcore/8.3.0``
``1.3``  |``GCCcore/9.3.0``
``2.0.3``|``GCCcore/10.2.0``
``2.1.1``|``GCCcore/10.3.0``
``2.2``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.0``  |``GCCcore/12.2.0``

### Reads2snp

reads2snp is a SNP and genotype caller: it predicts the genotype of distinct individuals   at distinct positions of a set of sequences based on read mapping / read counts. Its typical input is a bam file.   Its typical output is a vcf file. It is written in C++, based on the bio++ libraries, multi-threaded with openMP,   available under Linux and MacOS

*homepage*: <http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools>

version|toolchain
-------|----------
``2.0``|``system``

### Reapr

A tool that evaluates the accuracy of a genome assembly using mapped paired end reads,  without the use of a reference genome for comparison.

*homepage*: <https://www.sanger.ac.uk/science/tools/reapr>

version   |toolchain
----------|--------------
``1.0.18``|``foss/2019a``

### ReaxFF

REAXFF Reactive force field program 

*homepage*: <https://www.engr.psu.edu/adri/ReaxffManual.aspx>

version|toolchain
-------|--------------
``2.0``|``GCC/11.3.0``

### RECON

Patched version of RECON to be used with RepeatModeler.

*homepage*: <https://www.repeatmasker.org/RepeatModeler/>

version |toolchain
--------|------------------------------
``1.08``|``GCC/10.2.0``, ``GCC/11.3.0``

### Red

Red (REpeat Detector)

*homepage*: <http://toolsmith.ens.utulsa.edu/>

version       |toolchain
--------------|----------------------------------------
``2015-05-22``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

### Redis

Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes, and streams. Redis has built-in replication, Lua scripting, LRU eviction, transactions, and different levels of on-disk persistence, and provides high availability via Redis Sentinel and automatic partitioning with Redis Cluster.

*homepage*: <https://redis.io>

version  |toolchain
---------|------------------------------
``6.2.6``|``GCC/10.3.0``, ``GCC/11.2.0``
``7.0.8``|``GCC/11.3.0``

### redis-py

The Python interface to the Redis key-value store.

*homepage*: <https://github.com/redis/redis-py>

version  |toolchain
---------|--------------
``4.3.1``|``foss/2021a``
``4.3.3``|``foss/2021b``
``4.5.1``|``foss/2022a``

### Redundans

Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes.

*homepage*: <https://github.com/lpryszcz/redundans>

version  |toolchain
---------|---------------
``0.13c``|``intel/2017b``

### ReFrame

ReFrame is a framework for writing regression tests for HPC systems.

*homepage*: <https://github.com/reframe-hpc/reframe>

version   |toolchain
----------|----------
``2.18``  |``system``
``2.19``  |``system``
``2.20``  |``system``
``2.21``  |``system``
``3.0``   |``system``
``3.2``   |``system``
``3.3``   |``system``
``3.4.1`` |``system``
``3.5.0`` |``system``
``3.5.1`` |``system``
``3.5.2`` |``system``
``3.6.2`` |``system``
``3.6.3`` |``system``
``3.7.3`` |``system``
``3.8.0`` |``system``
``3.9.0`` |``system``
``3.9.1`` |``system``
``3.10.1``|``system``
``3.11.0``|``system``
``3.11.1``|``system``
``3.11.2``|``system``
``3.12.0``|``system``
``4.0.1`` |``system``
``4.0.5`` |``system``
``4.2.0`` |``system``

### regionmask

regionmask creates masks of geographical regions. It determines to which geographic region each grid point belongs.

*homepage*: <https://regionmask.readthedocs.io>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.9.0`` |                 |``foss/2021b``, ``foss/2022a``
``0.9.0`` |``-Python-3.8.2``|``foss/2020a``
``0.10.0``|                 |``foss/2022b``

### RegTools

RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context.

*homepage*: <https://regtools.readthedocs.org>

version  |toolchain
---------|------------------------------
``0.4.2``|``foss/2020b``
``0.5.2``|``foss/2020b``, ``foss/2021b``

### Relate

Software for estimating genome-wide genealogies for thousands of samples

*homepage*: <https://myersgroup.github.io/relate/>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20211123``|``-R-4.0.3`` |``foss/2020b``

### RELION

RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer  program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class  averages in electron cryo-microscopy (cryo-EM).

*homepage*: <http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page>

version                |versionsuffix   |toolchain
-----------------------|----------------|------------------------------------------------------------------------------------------------------------
``1.4``                |                |``intel/2016b``
``1.4``                |``-single``     |``intel/2016b``
``2.0.1``              |                |``intel/2016b``
``2.1``                |                |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intelcuda/2017b``
``2.1``                |``-CUDA-9.1.85``|``foss/2018a``
``3.0.4``              |                |``foss/2017b``, ``intel/2017b``
``3.0_beta.2018.08.02``|                |``fosscuda/2018a``, ``intel/2018a``

### ReMatCh

Reads mapping against target sequences, checking mapping and consensus sequences production

*homepage*: <https://github.com/B-UMMI/ReMatCh>

version|versionsuffix     |toolchain
-------|------------------|--------------
``3.2``|``-Python-2.7.12``|``foss/2016b``

### REMORA

REsource MOnitoring for Remote Applications

*homepage*: <https://github.com/TACC/remora>

version  |toolchain
---------|----------------------------------------------------------------
``1.8.2``|``foss/2017a``, ``foss/2018a``, ``intel/2017a``, ``intel/2018a``
``1.8.3``|``gompi/2019a``

### renderproto

Xrender protocol and ancillary headers

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version |toolchain
--------|------------------------------------------------------------------
``0.11``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

### RepastHPC

The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years:  Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based  modeling system that is designed for use on large computing clusters and supercomputers.

*homepage*: <https://repast.github.io/>

version  |toolchain
---------|--------------
``2.2.0``|``foss/2016a``

### RepeatMasker

RepeatMasker is a program that screens DNA sequences for interspersed repeats  and low complexity DNA sequences.

*homepage*: <https://www.repeatmasker.org/>

version     |versionsuffix         |toolchain
------------|----------------------|----------------------------------------------------------
``4.0.8``   |``-Perl-5.26.0-HMMER``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``4.0.8``   |``-Perl-5.28.0-HMMER``|``intel/2018b``
``4.0.9-p2``|``-HMMER``            |``gompi/2019b``
``4.1.2-p1``|                      |``foss/2020b``
``4.1.4``   |                      |``foss/2022a``
``4.1.5``   |                      |``foss/2022a``

### RepeatModeler

RepeatModeler is a de novo transposable element (TE) family identification and modeling package. 

*homepage*: <https://www.repeatmasker.org/>

version   |toolchain
----------|--------------
``2.0.2a``|``foss/2020b``
``2.0.4`` |``foss/2022a``

### RepeatScout

De Novo Repeat Finder, Price A.L., Jones N.C. and Pevzner P.A. Developed and  tested with our multiple sequence version of RepeatScout ( 1.0.6 )

*homepage*: <https://www.repeatmasker.org/>

version  |toolchain
---------|------------------------------
``1.0.6``|``GCC/10.2.0``, ``GCC/11.3.0``

### request

Request is designed to be the simplest way possible to make http calls.      It supports HTTPS and follows redirects by default.

*homepage*: <https://github.com/request/request/>

version   |versionsuffix      |toolchain
----------|-------------------|------------------
``2.88.1``|``-nodejs-12.19.0``|``fosscuda/2020b``

### requests

Python http for humans

*homepage*: <https://pypi.python.org/pypi/requests>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.10.0``|``-Python-2.7.11``|``foss/2016a``
``2.10.0``|``-Python-3.5.1`` |``foss/2016a``
``2.11.1``|``-Python-2.7.12``|``intel/2016b``
``2.11.1``|``-Python-3.5.2`` |``intel/2016b``
``2.13.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``

### RERconverge

RERconverge is a set of software written in R that  estimates the correlation between relative evolutionary rates of  genes and the evolution of a convergent binary or continuous trait  across a phylogeny. 

*homepage*: <https://github.com/nclark-lab/RERconverge>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``0.1.0``|``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``

### resolos

Resolos is a toolkit written in Python for maintaining reproducible environments  for scientific computations. It's main goal is to enable researchers to easily replicate environments through space (running code on HPC environment) and time (environment preservation for long term archival). For installation and detailed usage, check out the documentation.

*homepage*: <https://gitlab.unige.ch/resolos/resolos>

version  |toolchain
---------|------------------
``0.3.5``|``GCCcore/11.3.0``

### retworkx

retworkx is a general purpose graph library for python3 written in Rust to take advantage of the performance and safety that Rust provides. It was built as a replacement for qiskit's previous (and current) networkx usage (hence the name) but is designed to provide a high performance general purpose graph library for any python application. The project was originally started to build a faster directed graph to use as the underlying data structure for the DAG at the center of qiskit-terra's transpiler, but it has since grown to cover all the graph usage in Qiskit and other applications.

*homepage*: <https://github.com/Qiskit/retworkx>

version  |toolchain
---------|--------------
``0.9.0``|``foss/2021a``

### RevBayes

RevBayes provides an interactive environment for statistical computation in  phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in  evolutionary biology, particularly phylogenetics.

*homepage*: <https://revbayes.github.io>

version  |toolchain
---------|------------------------------
``1.1.1``|``GCC/10.2.0``, ``GCC/11.2.0``
``1.2.1``|``gompi/2022a``

### rgdal

Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (>= 1.11.4 and <= 2.5.0) and  access to projection/transformation operations from the 'PROJ.4' library.

*homepage*: <https://rgdal.r-forge.r-project.org/>

version   |versionsuffix|toolchain
----------|-------------|--------------
``1.4-4`` |``-R-3.6.0`` |``foss/2019a``
``1.4-8`` |``-R-3.6.2`` |``foss/2019b``
``1.4-8`` |``-R-4.0.0`` |``foss/2020a``
``1.5-16``|``-R-4.0.0`` |``foss/2020a``
``1.5-23``|``-R-4.0.4`` |``foss/2020b``
``1.5-23``|``-R-4.1.0`` |``foss/2021a``
``1.6-6`` |             |``foss/2022a``

### rgeos

R interface to Geometry Engine - Open Source (GEOS) using the C API for topology operations  on geometries

*homepage*: <https://cran.r-project.org/web/packages/rgeos/>

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.3-17``|``-R-3.2.3`` |``intel/2016a``
``0.5-1`` |``-R-3.6.0`` |``foss/2019a``
``0.5-2`` |``-R-3.6.2`` |``foss/2019b``
``0.5-5`` |``-R-4.0.0`` |``foss/2020a``
``0.5-5`` |``-R-4.1.0`` |``foss/2021a``

### Rgurobi

Gurobi Optimizer 9.5 interface

*homepage*: <https://www.gurobi.com>

version  |versionsuffix|toolchain
---------|-------------|--------------
``9.1.2``|``-R-4.1.0`` |``foss/2021a``
``9.5.0``|``-R-4.1.0`` |``foss/2021a``

### rhdf5

This R/Bioconductor package provides an interface between HDF5 and R.

*homepage*: <https://bioconductor.org/packages/release/bioc/html/rhdf5.html>

version   |versionsuffix|toolchain
----------|-------------|---------------
``2.16.0``|``-R-3.2.3`` |``intel/2016a``
``2.18.0``|``-R-3.3.1`` |``intel/2016b``

### RheoTool

RheoTool is an open-source toolbox based on OpenFOAM to simulate Generalized Newtonian Fluids (GNF) and viscoelastic fluids under pressure-driven and/or electrically-driven flows.

*homepage*: <https://github.com/fppimenta/rheoTool>

version|toolchain
-------|--------------
``5.0``|``foss/2019b``

### rickflow

Running and Analyzing OpenMM Jobs

*homepage*: <https://gitlab.com/Olllom/rickflow>

version           |versionsuffix    |toolchain
------------------|-----------------|---------------
``0.7.0``         |``-Python-3.7.4``|``intel/2019b``
``0.7.0-20200529``|``-Python-3.7.4``|``intel/2019b``

### RInChI

The reaction IUPAC International Chemical Identifier (RInChI TM) is a non-proprietary identifier for chemical reactions that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.

*homepage*: <https://www.inchi-trust.org/>

version |versionsuffix|toolchain
--------|-------------|----------
``1.00``|``-x86_64``  |``system``

### rioxarray

geospatial xarray extension powered by rasterio

*homepage*: <https://github.com/corteva/rioxarray>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.0.12``|``-Python-3.7.2``|``intel/2019a``
``0.0.24``|``-Python-3.7.4``|``foss/2019b``
``0.1.1`` |``-Python-3.8.2``|``foss/2020a``
``0.11.1``|                 |``foss/2021b``
``0.14.0``|                 |``foss/2022a``

### ripunzip

A tool to unzip files in parallel.

*homepage*: <https://github.com/google/ripunzip>

version  |toolchain
---------|----------
``0.4.0``|``system``

### rising

Provides everything needed for high performance data loading and augmentation in PyTorch.

*homepage*: <https://rising.rtfd.io/>

version  |versionsuffix   |toolchain
---------|----------------|------------------------------
``0.2.2``|                |``foss/2021a``, ``foss/2022a``
``0.2.2``|``-CUDA-11.3.1``|``foss/2021a``
``0.2.2``|``-CUDA-11.7.0``|``foss/2022a``

### Rivet

Rivet toolkit (Robust Independent Validation of Experiment and Theory)  To use your own analysis you must append the path to `RIVET_ANALYSIS_PATH`. 

*homepage*: <https://gitlab.com/hepcedar/rivet>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.1.7``|``-HepMC3-3.2.5``|``gompi/2022a``

### rjags

The rjags package is an interface to the JAGS library.

*homepage*: <https://cran.r-project.org/web/packages/rjags>

version |versionsuffix|toolchain
--------|-------------|----------------------------------
``4-6`` |``-R-3.4.0`` |``intel/2017a``
``4-6`` |``-R-3.4.3`` |``intel/2017b``
``4-8`` |``-R-3.5.1`` |``foss/2018b``
``4-9`` |``-R-3.6.0`` |``foss/2019a``
``4-10``|             |``foss/2019b``
``4-10``|``-R-4.0.0`` |``foss/2020a``
``4-10``|``-R-4.0.3`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.0.4`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.0.5`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.1.0`` |``foss/2021a``
``4-12``|``-R-4.1.2`` |``foss/2021b``
``4-13``|``-R-4.2.1`` |``foss/2022a``

### RLCard

RLCard is a toolkit for Reinforcement Learning (RL) in card games.

*homepage*: <https://www.rlcard.org>

version  |toolchain
---------|--------------
``1.0.9``|``foss/2022a``

### rmarkdown

Convert R Markdown documents into a variety of formats.

*homepage*: <https://cran.r-project.org/web/packages/rmarkdown>

version |versionsuffix|toolchain
--------|-------------|--------------
``2.20``|``-R-4.1.0`` |``foss/2021a``

### Rmath

Rmath is the standalone version of the R math library.  Rmath can be used in your own C/C++ routines.

*homepage*: <https://www.r-project.org/>

version  |toolchain
---------|---------------
``3.3.1``|``intel/2016b``
``4.0.4``|``foss/2020b``

### rMATS-turbo

rMATS turbo is the C/Cython version of rMATS (refer to http://rnaseq-mats.sourceforge.net).

*homepage*: <https://github.com/Xinglab/rmats-turbo>

version  |toolchain
---------|--------------
``4.1.1``|``foss/2020b``

### RMBlast

RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary  difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn'  for use with RepeatMasker and RepeatModeler.

*homepage*: <http://www.repeatmasker.org/RMBlast.html>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.28``|``-Python-2.7.11``|``foss/2016a``
``2.9.0`` |                  |``gompi/2019b``
``2.10.0``|                  |``gompi/2019b``
``2.11.0``|                  |``gompi/2020b``
``2.13.0``|                  |``gompi/2022a``

### RNA-Bloom

RNA-Bloom is a fast and memory-efficient de novo transcript sequence assembler.

*homepage*: <https://github.com/bcgsc/RNA-Bloom>

version  |toolchain
---------|--------------
``1.2.3``|``GCC/8.3.0``
``1.4.3``|``GCC/11.2.0``

### RNA-SeQC

Fast, efficient RNA-Seq metrics for quality control and process optimization

*homepage*: <https://github.com/getzlab/rnaseqc>

version  |versionsuffix      |toolchain
---------|-------------------|------------------------------
``1.1.8``|``-Java-1.8``      |``foss/2018b``
``1.1.8``|``-Java-1.8.0_121``|``foss/2016b``
``1.1.8``|``-Java-11``       |``GCCcore/11.2.0``
``2.4.2``|                   |``foss/2021a``, ``foss/2021b``

### RNAclust

RNAclust is a perl script summarizing all the single steps required for clustering  of structured RNA motifs, i.e. identifying groups of RNA sequences sharing a secondary structure  motif. It requires as input a multiple FASTA file.

*homepage*: <http://www.bioinf.uni-leipzig.de/~kristin/Software/RNAclust/>

version|versionsuffix    |toolchain
-------|-----------------|--------------
``1.3``|``-Perl-5.24.0`` |``foss/2016b``
``1.3``|``-Python-3.6.6``|``foss/2018b``

### RNAcode

RNAcode - Analyze the protein coding potential in multiple sequence alignments

*homepage*: <https://wash.github.io/rnacode/>

version|toolchain
-------|--------------
``0.3``|``foss/2017a``

### RNAIndel

RNAIndel calls coding indels and classifies them into somatic, germline,  and artifact from tumor RNA-Seq data.

*homepage*: <https://github.com/stjude/RNAIndel>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.0``|``-Python-3.6.6``|``intel/2018b``
``1.0.0``|``-Python-3.6.6``|``intel/2018b``

### RNAmmer

This is an example description.

*homepage*: <https://www.cbs.dtu.dk/services/RNAmmer>

version|versionsuffix   |toolchain
-------|----------------|---------------
``1.2``|``-Perl-5.28.0``|``intel/2018b``

### rnaQUAST

rnaQUAST is a tool for evaluating RNA-Seq assemblies using reference genome and gene database. In addition, rnaQUAST is also capable of estimating gene database coverage by raw reads and de novo quality assessment using third-party software.

*homepage*: <https://cab.spbu.ru/software/rnaquast/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.5.2``|``-Python-2.7.15``|``foss/2018b``
``2.2.0``|``-Python-3.8.2`` |``foss/2020a``
``2.2.2``|                  |``foss/2021b``

### RNAz

RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.

*homepage*: <http://www.tbi.univie.ac.at/~wash/RNAz/>

version|toolchain
-------|------------------------------
``2.1``|``foss/2016b``, ``foss/2018b``

### RnBeads

RnBeads is an R package for comprehensive analysis of DNA methylation data obtained with any experimental protocol that provides single-CpG resolution.

*homepage*: <https://rnbeads.org>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.6.0``|``-R-4.0.0`` |``foss/2020a``

### Roary

Rapid large-scale prokaryote pan genome analysis

*homepage*: <https://github.com/sanger-pathogens/Roary>

version   |versionsuffix   |toolchain
----------|----------------|----------------------------------------------
``3.12.0``|                |``system``
``3.12.0``|``-Perl-5.26.1``|``intel/2018a``
``3.13.0``|                |``foss/2020a``, ``foss/2021a``, ``foss/2022a``

### ROCm

AMD ROCm is the first open-source software development platform for HPC/Hyperscale-class GPU computing. AMD ROCm brings the UNIX philosophy of choice, minimalism and modular software development to GPU computing.

*homepage*: <https://rocmdocs.amd.com/en/latest/>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### rocm-cmake

ROCM cmake modules provides cmake modules for common build tasks needed for the ROCM software stack

*homepage*: <https://github.com/RadeonOpenCompute/rocm-cmake>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### ROCm-CompilerSupport

The compiler support repository provides various Lightning Compiler related services

*homepage*: <https://github.com/RadeonOpenCompute/ROCm-CompilerSupport>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### rocm-smi

The ROCm System Management Interface Library, or ROCm SMI library, is part of the Radeon Open Compute ROCm software stack. It is a C library for Linux that provides a user space interface for applications to monitor and control GPU applications.

*homepage*: <https://github.com/RadeonOpenCompute/rocm_smi_lib>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``
``5.4.4``|``GCCcore/11.3.0``

### rocminfo

ROCm Application for Reporting System Info

*homepage*: <https://github.com/RadeonOpenCompute/rocminfo>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### ROCR-Runtime

The user-mode API interfaces and libraries necessary for host applications to launch compute kernels to available HSA ROCm kernel agents

*homepage*: <https://github.com/RadeonOpenCompute/ROCR-Runtime>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### ROCT-Thunk-Interface

The user-mode API interfaces used to interact with the ROCk driver

*homepage*: <https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface>

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

### ROI_PAC

Repeat Orbit Interferometry PACkage (ROI_PAC), software for processing synthetic aperture radar data  to produce differential interferograms

*homepage*: <http://roipac.org/>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.0.1``|``-Perl-5.24.1``|``intel/2017a``

### ROME

The ROME (Refinement and Optimization via Machine Learning for cryo-EM)  Software package is one of the major research products at the Intel® PCCSB.

*homepage*: <https://ipccsb.dfci.harvard.edu/rome>

version  |toolchain
---------|-----------------
``1.1.2``|``intel/2019.02``

### ROOT

The ROOT system provides a set of OO frameworks with all the functionality     needed to handle and analyze large amounts of data in a very efficient way.

*homepage*: <http://root.cern.ch/drupal/>

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``6.10.02`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``6.10.04`` |``-Python-2.7.13``|``intel/2017a``
``6.10.08`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``6.14.06`` |``-Python-2.7.15``|``foss/2018b``
``6.14.06`` |``-Python-3.6.6`` |``foss/2018b``
``6.20.04`` |``-Python-3.7.4`` |``foss/2019b``
``6.22.08`` |                  |``foss/2020b``
``6.24.06`` |                  |``foss/2021b``
``6.26.06`` |                  |``foss/2022a``
``6.26.10`` |                  |``foss/2022b``
``v5.34.34``|``-Python-2.7.11``|``intel/2016a``
``v5.34.36``|``-Python-2.7.11``|``intel/2016a``
``v6.06.02``|``-Python-2.7.12``|``intel/2016b``
``v6.08.02``|``-Python-2.7.11``|``foss/2016a``

### root_numpy

root_numpy is a Python extension module that provides an efficient interface between ROOT and  NumPy. root_numpy’s internals are compiled C++ and can therefore handle large amounts of data much faster than  equivalent pure Python implementations.

*homepage*: <https://scikit-hep.org/root_numpy/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``4.8.0``|``-Python-3.6.6``|``foss/2018b``

### rootpy

The rootpy project is a community-driven initiative aiming to provide a more pythonic interface with ROOT on top of the existing PyROOT bindings. Given Python’s reflective and dynamic nature, rootpy also aims to improve ROOT design flaws and supplement existing ROOT functionality. The scientific Python community also offers a multitude of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn, and PyTables, but a suitable interface between them and ROOT has been lacking. rootpy provides the interfaces and conversion mechanisms required to liberate your data and to take advantage of these alternatives if needed.

*homepage*: <http://www.rootpy.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.8.0``|``-Python-2.7.11``|``foss/2016a``
``1.0.1``|``-Python-3.6.6`` |``foss/2018b``

### Rosetta

Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.

*homepage*: <https://www.rosettacommons.org>

version          |toolchain
-----------------|--------------
``3.7``          |``foss/2016b``
``2016.13.58602``|``foss/2016a``
``2016.46.59086``|``foss/2016b``

### rpmrebuild

rpmrebuild is a tool to build an RPM file from a package that has already been     installed in a basic use

*homepage*: <http://rpmrebuild.sourceforge.net/>

version |toolchain
--------|----------
``2.11``|``system``

### RPostgreSQL

Database interface and 'PostgreSQL' driver for 'R'. This package provides a Database Interface 'DBI' compliant driver for 'R' to access 'PostgreSQL' database systems.

*homepage*: <https://cran.r-project.org/package=tesseract>

version  |toolchain
---------|--------------
``0.7-5``|``foss/2022a``

### rpy2

rpy2 is an interface to R running embedded in a Python process.

*homepage*: <https://rpy2.bitbucket.io/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.7.9``|``-Python-2.7.12``|``foss/2016b``
``2.8.2``|``-Python-2.7.13``|``intel/2017a``
``3.2.6``|``-Python-3.7.4`` |``foss/2019b``
``3.4.5``|                  |``foss/2021b``

### RQGIS3

RQGIS3 establishes an interface between R and QGIS3, i.e.,  it allows the user to access QGIS3 functionalities from within R.

*homepage*: <https://github.com/r-spatial/RQGIS3>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20190903``|``-R-3.6.0`` |``foss/2019a``

### RSEM

RNA-Seq by Expectation-Maximization

*homepage*: <https://deweylab.github.io/RSEM/>

version   |toolchain
----------|------------------------------------------------
``1.2.26``|``GNU/4.9.3-2.25``
``1.2.30``|``foss/2016a``, ``intel/2016b``
``1.3.0`` |``foss/2016b``, ``intel/2017a``
``1.3.1`` |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.3.2`` |``foss/2018b``
``1.3.3`` |``foss/2019b``, ``foss/2021b``, ``foss/2022a``

### RSeQC

RSeQC provides a number of useful modules that can  comprehensively evaluate high throughput sequence data especially RNA-seq  data. Some basic modules quickly inspect sequence quality, nucleotide  composition bias, PCR bias and GC bias, while RNA-seq specific modules  evaluate sequencing saturation, mapped reads distribution, coverage  uniformity, strand specificity, transcript level RNA integrity etc.

*homepage*: <http://rseqc.sourceforge.net/>

version  |versionsuffix             |toolchain
---------|--------------------------|------------------------------
``2.6.4``|``-Python-2.7.12-R-3.3.1``|``foss/2016b``
``2.6.4``|``-Python-2.7.14``        |``intel/2018a``
``3.0.0``|``-Python-3.6.6``         |``foss/2018b``
``4.0.0``|                          |``foss/2021a``, ``foss/2021b``
``4.0.0``|``-Python-3.8.2``         |``foss/2020a``

### RStan

RStan is the R interface to Stan. Stan is a state-of-the-art   platform for statistical modeling and high-performance statistical computation.

*homepage*: <https://github.com/stan-dev/rstan>

version   |versionsuffix|toolchain
----------|-------------|-------------------------------
``2.18.2``|``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``
``2.18.2``|``-R-3.5.1`` |``foss/2018b``

### rstanarm

Estimates previously compiled regression models using the 'rstan' package, which provides  the R interface to the Stan C++ library for Bayesian estimation.

*homepage*: <https://r-forge.r-project.org/projects/rstanarm>

version   |versionsuffix|toolchain
----------|-------------|--------------
``2.19.3``|``-R-3.6.2`` |``foss/2019b``

### RStudio-Server

This is the RStudio Server version. RStudio is a set of integrated tools designed to help you be more productive with R.  The server can be started with:   rserver --server-daemonize=0 --www-port=8787  If you need a database config one can be created with:   MYTMP=`mktemp -d` && echo -e "provider=sqlite\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf" and then used with:   rserver ... --database-config-file="${MYTMP}/db.conf" 

*homepage*: <https://www.rstudio.com/>

version          |versionsuffix       |toolchain
-----------------|--------------------|------------------
``1.2.5033``     |``-Java-11``        |``fosscuda/2019b``
``1.2.5042``     |``-Java-11``        |``foss/2019b``
``1.3.959``      |``-Java-11-R-4.0.0``|``foss/2020a``
``1.3.1093``     |``-Java-11-R-4.0.0``|``foss/2020a``
``1.4.1717``     |``-Java-11-R-4.1.0``|``foss/2021a``
``2022.07.2+576``|``-Java-11-R-4.2.1``|``foss/2022a``

### RTG-Tools

RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,  as well as utilities for processing other common NGS data formats. 

*homepage*: <https://www.realtimegenomics.com/products/rtg-tools>

version  |versionsuffix|toolchain
---------|-------------|----------
``3.9.1``|``-Java-1.8``|``system``

### Rtree

Rtree is a ctypes Python wrapper of libspatialindex that provides a number of advanced spatial  indexing features for the spatially curious Python user.

*homepage*: <https://toblerity.org/rtree/>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``0.8.3``|``-Python-2.7.14``|``intel/2018a``
``1.0.1``|                  |``GCCcore/12.2.0``

### ruamel.yaml

ruamel.yaml is a YAML 1.2 loader/dumper package for Python.

*homepage*: <https://sourceforge.net/projects/ruamel-yaml>

version    |toolchain
-----------|------------------------------------------------------------------------------
``0.17.21``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### Ruby

Ruby is a dynamic, open source programming language with  a focus on simplicity and productivity. It has an elegant syntax that is  natural to read and simple to write.

*homepage*: <https://www.ruby-lang.org>

version  |toolchain
---------|------------------------------------------------
``2.1.6``|``system``
``2.3.1``|``intel/2016b``, ``system``
``2.3.3``|``system``
``2.3.4``|``system``
``2.4.2``|``foss/2017b``
``2.5.0``|``foss/2018a``, ``intel/2017a``, ``intel/2017b``
``2.5.1``|``foss/2018a``, ``intel/2018a``
``2.6.1``|``GCCcore/7.3.0``
``2.6.3``|``GCCcore/8.2.0``
``2.7.1``|``GCCcore/8.3.0``
``2.7.2``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.0.3``|``GCCcore/11.3.0``
``3.0.5``|``GCCcore/11.3.0``
``3.2.2``|``GCCcore/12.2.0``

### Ruby-Tk

Ruby Tk interface module using tcltklib

*homepage*: <https://github.com/ruby/tk>

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``0.2.0``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

### ruffus

Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and user-friendly,  and widely used in science and bioinformatics.

*homepage*: <http://www.ruffus.org.uk/>

version  |toolchain
---------|----------------------------------
``2.8.4``|``GCCcore/11.3.0``, ``foss/2021b``

### Rust

Rust is a systems programming language that runs blazingly fast, prevents segfaults,  and guarantees thread safety.

*homepage*: <https://www.rust-lang.org>

version   |toolchain
----------|--------------------------------------------------------------------------
``1.8.0`` |``foss/2016a``
``1.12.0``|``foss/2016b``
``1.12.1``|``foss/2016b``
``1.18.0``|``foss/2017a``
``1.21.0``|``foss/2017b``
``1.22.1``|``GCCcore/6.4.0``
``1.29.2``|``GCCcore/7.3.0``
``1.30.1``|``GCCcore/6.4.0``
``1.35.0``|``GCCcore/8.2.0``
``1.37.0``|``GCCcore/8.3.0``
``1.42.0``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.52.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.54.0``|``GCCcore/11.2.0``
``1.56.0``|``GCCcore/11.2.0``
``1.60.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``
``1.65.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.70.0``|``GCCcore/12.3.0``

### rustworkx

rustworkx (previously retworkx) is a general purpose graph library for Python written in Rust to take advantage of the performance and safety that Rust provides. It is designed to provide a high performance general purpose graph library for any Python application.

*homepage*: <https://github.com/Qiskit/rustworkx>

version   |toolchain
----------|--------------
``0.12.1``|``foss/2022a``

## S


[S-Lang](#s-lang) - [S4](#s4) - [Safetensors](#safetensors) - [SAGE](#sage) - [Sailfish](#sailfish) - [SALib](#salib) - [Salmon](#salmon) - [SALMON-TDDFT](#salmon-tddft) - [Sambamba](#sambamba) - [samblaster](#samblaster) - [Samcef](#samcef) - [samclip](#samclip) - [samplot](#samplot) - [SAMtools](#samtools) - [sansa](#sansa) - [SAP](#sap) - [SAS](#sas) - [Satsuma2](#satsuma2) - [savvy](#savvy) - [Saxon-HE](#saxon-he) - [SBCL](#sbcl) - [sbt](#sbt) - [ScaFaCoS](#scafacos) - [ScaLAPACK](#scalapack) - [Scalasca](#scalasca) - [SCALCE](#scalce) - [Scalene](#scalene) - [scanpy](#scanpy) - [scArches](#scarches) - [scsimple](#scsimple) - [SCENIC](#scenic) - [scGeneFit](#scgenefit) - [SCGid](#scgid) - [scHiCExplorer](#schicexplorer) - [Schrodinger](#schrodinger) - [scib](#scib) - [sciClone](#sciclone) - [ScientificPython](#scientificpython) - [scikit-allel](#scikit-allel) - [scikit-bio](#scikit-bio) - [scikit-build](#scikit-build) - [scikit-cuda](#scikit-cuda) - [scikit-image](#scikit-image) - [scikit-learn](#scikit-learn) - [scikit-misc](#scikit-misc) - [scikit-multilearn](#scikit-multilearn) - [scikit-optimize](#scikit-optimize) - [scikit-uplift](#scikit-uplift) - [SCIP](#scip) - [SCIPhI](#sciphi) - [scipy](#scipy) - [SciPy-bundle](#scipy-bundle) - [SCnorm](#scnorm) - [Scoary](#scoary) - [SCons](#scons) - [SCOOP](#scoop) - [SCopeLoomR](#scopeloomr) - [Score-P](#score-p) - [SCOTCH](#scotch) - [scp](#scp) - [scPred](#scpred) - [Scrappie](#scrappie) - [scVelo](#scvelo) - [scvi-tools](#scvi-tools) - [Scythe](#scythe) - [SDCC](#sdcc) - [SDL](#sdl) - [SDL2](#sdl2) - [SDL2_image](#sdl2_image) - [SDL_image](#sdl_image) - [SDSL](#sdsl) - [Seaborn](#seaborn) - [SearchGUI](#searchgui) - [SeaView](#seaview) - [SECAPR](#secapr) - [Seeder](#seeder) - [segemehl](#segemehl) - [segmentation-models](#segmentation-models) - [SeisSol](#seissol) - [SelEstim](#selestim) - [SELFIES](#selfies) - [SentencePiece](#sentencepiece) - [sep](#sep) - [SEPP](#sepp) - [Seq-Gen](#seq-gen) - [seq2HLA](#seq2hla) - [SeqAn](#seqan) - [SeqAn3](#seqan3) - [SeqKit](#seqkit) - [SeqLib](#seqlib) - [Seqmagick](#seqmagick) - [SeqPrep](#seqprep) - [seqtk](#seqtk) - [Serf](#serf) - [setuptools](#setuptools) - [Seurat](#seurat) - [SeuratData](#seuratdata) - [SeuratDisk](#seuratdisk) - [SeuratWrappers](#seuratwrappers) - [sf](#sf) - [sfftk](#sfftk) - [Shannon](#shannon) - [SHAP](#shap) - [shapAAR](#shapaar) - [SHAPEIT](#shapeit) - [SHAPEIT4](#shapeit4) - [Shapely](#shapely) - [sharutils](#sharutils) - [Shasta](#shasta) - [ShengBTE](#shengbte) - [shift](#shift) - [SHORE](#shore) - [Short-Pair](#short-pair) - [shovill](#shovill) - [shrinkwrap](#shrinkwrap) - [SHTns](#shtns) - [Sibelia](#sibelia) - [SICER2](#sicer2) - [sickle](#sickle) - [Siesta](#siesta) - [SignalP](#signalp) - [silhouetteRank](#silhouetterank) - [silx](#silx) - [simint](#simint) - [SimNIBS](#simnibs) - [SimPEG](#simpeg) - [SIMPLE](#simple) - [Simple-DFTD3](#simple-dftd3) - [SimpleElastix](#simpleelastix) - [SimpleITK](#simpleitk) - [simpy](#simpy) - [SimVascular](#simvascular) - [SingleM](#singlem) - [Singular](#singular) - [SiNVICT](#sinvict) - [SIONlib](#sionlib) - [SIP](#sip) - [siscone](#siscone) - [SISSO](#sisso) - [SISSO++](#sisso++) - [SKESA](#skesa) - [sketchmap](#sketchmap) - [skewer](#skewer) - [sklearn-pandas](#sklearn-pandas) - [sklearn-som](#sklearn-som) - [skorch](#skorch) - [SlamDunk](#slamdunk) - [SLATEC](#slatec) - [SLEPc](#slepc) - [slepc4py](#slepc4py) - [sleuth](#sleuth) - [slidingwindow](#slidingwindow) - [SLiM](#slim) - [slow5tools](#slow5tools) - [smafa](#smafa) - [smallgenomeutilities](#smallgenomeutilities) - [SMAP](#smap) - [SMARTdenovo](#smartdenovo) - [SMC++](#smc++) - [smfishHmrf](#smfishhmrf) - [smithwaterman](#smithwaterman) - [Smoldyn](#smoldyn) - [smooth-topk](#smooth-topk) - [SMRT-Link](#smrt-link) - [SMV](#smv) - [snakemake](#snakemake) - [SNAP](#snap) - [SNAP-HMM](#snap-hmm) - [SNAPE-pooled](#snape-pooled) - [snaphu](#snaphu) - [snappy](#snappy) - [Sniffles](#sniffles) - [snippy](#snippy) - [snp-sites](#snp-sites) - [snpEff](#snpeff) - [SNPhylo](#snphylo) - [SNPomatic](#snpomatic) - [SOAPaligner](#soapaligner) - [SOAPdenovo-Trans](#soapdenovo-trans) - [SOAPdenovo2](#soapdenovo2) - [SOAPfuse](#soapfuse) - [socat](#socat) - [SOCI](#soci) - [SolexaQA++](#solexaqa++) - [sonic](#sonic) - [SoPlex](#soplex) - [SoQt](#soqt) - [SortMeRNA](#sortmerna) - [SoupX](#soupx) - [SoX](#sox) - [SoXt](#soxt) - [SpaceRanger](#spaceranger) - [Spack](#spack) - [spaCy](#spacy) - [SPAdes](#spades) - [spaln](#spaln) - [Spark](#spark) - [sparse-neighbors-search](#sparse-neighbors-search) - [sparsehash](#sparsehash) - [SpatialDE](#spatialde) - [spatialreg](#spatialreg) - [spdlog](#spdlog) - [SpectrA](#spectra) - [spectral.methods](#spectral.methods) - [speech_tools](#speech_tools) - [spektral](#spektral) - [spglib](#spglib) - [spglib-python](#spglib-python) - [Sphinx](#sphinx) - [Sphinx-RTD-Theme](#sphinx-rtd-theme) - [SpiceyPy](#spiceypy) - [SpiecEasi](#spieceasi) - [SplAdder](#spladder) - [SPLASH](#splash) - [SpliceMap](#splicemap) - [split-seq](#split-seq) - [splitRef](#splitref) - [SPM](#spm) - [spoa](#spoa) - [SPOOLES](#spooles) - [SPOTPY](#spotpy) - [SPRNG](#sprng) - [Spyder](#spyder) - [SQLite](#sqlite) - [SqueezeMeta](#squeezemeta) - [Squidpy](#squidpy) - [SRA-Toolkit](#sra-toolkit) - [sradownloader](#sradownloader) - [SRPRISM](#srprism) - [SRST2](#srst2) - [SSAHA2](#ssaha2) - [SSN](#ssn) - [SSPACE_Basic](#sspace_basic) - [SSW](#ssw) - [STACEY](#stacey) - [Stack](#stack) - [Stacks](#stacks) - [STAMP](#stamp) - [StaMPS](#stamps) - [Stampy](#stampy) - [STAR](#star) - [STAR-CCM+](#star-ccm+) - [STAR-Fusion](#star-fusion) - [stardist](#stardist) - [stars](#stars) - [Stata](#stata) - [Statistics-R](#statistics-r) - [statsmodels](#statsmodels) - [STEAK](#steak) - [STIR](#stir) - [stpipeline](#stpipeline) - [strace](#strace) - [Strainberry](#strainberry) - [STREAM](#stream) - [strelka](#strelka) - [StringTie](#stringtie) - [stripy](#stripy) - [STRique](#strique) - [Structure](#structure) - [Structure_threader](#structure_threader) - [STRUMPACK](#strumpack) - [suave](#suave) - [Subread](#subread) - [subset-bam](#subset-bam) - [Subversion](#subversion) - [suds](#suds) - [SuiteSparse](#suitesparse) - [SUMACLUST](#sumaclust) - [SUMATRA](#sumatra) - [SUMO](#sumo) - [SUNDIALS](#sundials) - [SunPy](#sunpy) - [SuperLU](#superlu) - [SuperLU_DIST](#superlu_dist) - [supermagic](#supermagic) - [supernova](#supernova) - [SURVIVOR](#survivor) - [SVclone](#svclone) - [SVDetect](#svdetect) - [SVDquest](#svdquest) - [SVG](#svg) - [SVIM](#svim) - [svist4get](#svist4get) - [swarm](#swarm) - [SWASH](#swash) - [SWAT+](#swat+) - [swifter](#swifter) - [SWIG](#swig) - [SWIPE](#swipe) - [swissknife](#swissknife) - [SymEngine](#symengine) - [SymEngine-python](#symengine-python) - [SYMPHONY](#symphony) - [sympy](#sympy) - [synthcity](#synthcity) - [SyRI](#syri) - [sysbench](#sysbench) - [Szip](#szip)


### S-Lang

S-Lang is a multi-platform programmer's library designed to allow  a developer to create robust multi-platform software. It provides facilities  required by interactive applications such as display/screen management, keyboard  input, keymaps, and so on.

*homepage*: <http://www.jedsoft.org/slang/>

version  |toolchain
---------|-------------
``2.3.0``|``GCC/4.9.2``

### S4

S4 stands for Stanford Stratified Structure Solver, a frequency domain code  to solve the linear Maxwell’s equations in layered periodic structures. Internally, it uses  Rigorous Coupled Wave Analysis (RCWA, also called the Fourier Modal Method (FMM)) and the  S-matrix algorithm.

*homepage*: <https://web.stanford.edu/group/fan/S4>

version           |toolchain
------------------|--------------
``1.1.1-20180610``|``foss/2017b``

### Safetensors

Safetensors is a new simple format for storing tensors safely (as opposed to pickle) and that is still fast (zero-copy). Safetensors is really fast.

*homepage*: <https://huggingface.co/docs/safetensors>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.3.1``|                |``foss/2022a``
``0.3.1``|``-CUDA-11.7.0``|``foss/2022a``

### SAGE

S.A.G.E. (Statistical Analysis for Genetic  Epidemiology) is free software package containing programs for  use in the genetic analysis of family, pedigree and individual data.

*homepage*: <http://darwin.cwru.edu/sage/>

version|toolchain
-------|----------
``6.3``|``system``
``6.4``|``system``

### Sailfish

Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of  isoform abundances directly from a set of reference sequences and RNA-seq reads. 

*homepage*: <http://www.cs.cmu.edu/~ckingsf/software/sailfish/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.10.1``|                  |``gompi/2019b``
``0.10.1``|``-Python-2.7.12``|``foss/2016b``

### SALib

Sensitivity Analysis Library in Python (Numpy). Contains Sobol, Morris, Fractional Factorial and FAST methods.

*homepage*: <https://SALib.github.io/SALib/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.3``|``-Python-2.7.14``|``intel/2017b``

### Salmon

Salmon is a wicked-fast program to produce a highly-accurate,  transcript-level quantification estimates from RNA-seq data.

*homepage*: <https://github.com/COMBINE-lab/salmon>

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``0.8.2`` |                  |``system``
``0.8.2`` |``-Python-2.7.12``|``foss/2016b``
``0.11.2``|                  |``intel/2018a``
``0.12.0``|                  |``foss/2018b``
``0.14.1``|                  |``foss/2018b``
``0.14.2``|                  |``gompi/2019a``
``1.0.0`` |                  |``gompi/2019a``, ``gompi/2019b``
``1.1.0`` |                  |``gompi/2019b``
``1.2.0`` |                  |``gompi/2019b``
``1.3.0`` |                  |``gompi/2020a``
``1.4.0`` |                  |``GCC/11.2.0``, ``gompi/2020b``
``1.9.0`` |                  |``GCC/11.3.0``

### SALMON-TDDFT

SALMON is an open-source computer program for ab-initio quantum-mechanical calculations of  electron dynamics at the nanoscale that takes place in various situations of light-matter interactions.  It is based on time-dependent density functional theory, solving time-dependent Kohn-Sham equation in  real time and real space with norm-conserving pseudopotentials.

*homepage*: <https://salmon-tddft.jp>

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2018b``, ``intel/2018b``

### Sambamba

Sambamba is a high performance modern robust and fast tool  (and library), written in the D programming language, for working with SAM  and BAM files. Current functionality is an important subset of samtools  functionality, including view, index, sort, markdup, and depth.

*homepage*: <https://lomereiter.github.io/sambamba/>

version  |toolchain
---------|--------------
``0.6.6``|``system``
``0.7.1``|``system``
``0.8.0``|``GCC/10.2.0``
``0.8.2``|``GCC/10.3.0``

### samblaster

samblaster is a fast and flexible program for marking  duplicates in read-id grouped1 paired-end SAM files.

*homepage*: <https://github.com/GregoryFaust/samblaster>

version   |toolchain
----------|----------------------------------------------
``0.1.24``|``foss/2018b``
``0.1.26``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.3.0``

### Samcef

FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and  thermal applications.

*homepage*: <http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/>

version    |toolchain
-----------|----------
``17.0-03``|``system``

### samclip

Filter SAM file for soft and hard clipped alignments.

*homepage*: <https://github.com/tseemann/samclip>

version  |versionsuffix   |toolchain
---------|----------------|---------------------------------------------------------
``0.2``  |``-Perl-5.28.0``|``GCCcore/7.3.0``
``0.4.0``|                |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### samplot

Plot structural variant signals from many BAMs and CRAMs. 

*homepage*: <https://github.com/ryanlayer/samplot>

version  |toolchain
---------|--------------
``1.3.0``|``foss/2021b``

### SAMtools

SAM Tools provide various utilities for manipulating alignments in the SAM format,   including sorting, merging, indexing and generating alignments in a per-position format.

*homepage*: <http://www.htslib.org/>

version   |versionsuffix    |toolchain
----------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------
``0.1.17``|                 |``intel/2017a``
``0.1.19``|                 |``GCC/10.3.0``, ``foss/2016a``, ``foss/2016b``
``0.1.20``|                 |``GCC/8.3.0``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2019b``
``1.2``   |                 |``foss/2016b``
``1.3``   |                 |``foss/2016a``, ``intel/2016a``
``1.3.1`` |                 |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.1`` |``-HTSlib-1.3.2``|``intel/2016b``
``1.4``   |                 |``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.4.1`` |                 |``intel/2017a``
``1.5``   |                 |``foss/2016b``, ``intel/2017a``
``1.6``   |                 |``GCC/6.4.0-2.28``, ``foss/2017a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017b``
``1.7``   |                 |``foss/2018a``, ``intel/2018a``
``1.8``   |                 |``intel/2018a``
``1.9``   |                 |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10``  |                 |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``  |                 |``GCC/10.2.0``, ``GCC/9.3.0``, ``iccifort/2020.4.304``
``1.12``  |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.13``  |                 |``GCC/10.3.0``, ``GCC/11.3.0``
``1.14``  |                 |``GCC/11.2.0``
``1.15``  |                 |``GCC/11.2.0``
``1.15.1``|                 |``GCC/11.2.0``, ``GCC/11.3.0``
``1.16.1``|                 |``GCC/11.2.0``, ``GCC/11.3.0``
``1.17``  |                 |``GCC/12.2.0``

### sansa

Structural variant (SV) annotation, a companion to the 'dolly' tool.

*homepage*: <https://github.com/dellytools/sansa/>

version  |toolchain
---------|---------------
``0.0.7``|``gompi/2020b``

### SAP

SAP is a pairwise structure alignment via double dynamic programming

*homepage*: <https://github.com/mathbio-nimr-mrc-ac-uk/SAP>

version  |toolchain
---------|--------------
``1.1.3``|``GCC/11.3.0``

### SAS

SAS is a software suite for advanced analytics, multivariate analyses, business intelligence,  data management, and predictive analytics.

*homepage*: <https://www.sas.com>

version|toolchain
-------|----------
``9.4``|``system``

### Satsuma2

Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to accommodate the billions of base pairs in vertebrate genomes).

*homepage*: <https://github.com/bioinfologics/satsuma2>

version     |toolchain
------------|--------------
``20220304``|``GCC/11.3.0``

### savvy

Interface to various variant calling formats.

*homepage*: <https://github.com/statgen/savvy>

version  |toolchain
---------|--------------------
``1.3.0``|``GCC/8.2.0-2.31.1``

### Saxon-HE

Open Source SAXON XSLT processor developed by Saxonica Limited.

*homepage*: <http://saxon.sourceforge.net>

version     |versionsuffix      |toolchain
------------|-------------------|----------
``9.7.0.4`` |``-Java-1.7.0_79`` |``system``
``9.7.0.21``|``-Java-1.8.0_162``|``system``
``9.9.1.7`` |``-Java-13``       |``system``

### SBCL

Steel Bank Common Lisp (SBCL) is a high performance Common Lisp compiler. It is open source / free software, with a permissive license. In addition to the compiler and runtime system for ANSI Common Lisp, it provides an interactive environment including a debugger, a statistical profiler, a code coverage tool, and many other extensions.

*homepage*: <http://sbcl.sourceforge.net/>

version  |toolchain
---------|------------------
``2.0.9``|``GCCcore/9.3.0``
``2.2.1``|``GCCcore/10.3.0``

### sbt

sbt is a build tool for Scala, Java, and more.

*homepage*: <http://www.scala-sbt.org/>

version   |versionsuffix      |toolchain
----------|-------------------|----------
``1.0.2`` |``-Java-1.8.0_152``|``system``
``1.3.13``|``-Java-1.8``      |``system``

### ScaFaCoS

ScaFaCoS is a library of scalable fast coulomb solvers.

*homepage*: <http://www.scafacos.de/>

version  |toolchain
---------|-------------------------------------------------------------------------------
``1.0.1``|``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020a``
``1.0.4``|``foss/2022a``

### ScaLAPACK

The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines  redesigned for distributed memory MIMD parallel computers.

*homepage*: <https://www.netlib.org/scalapack/>

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------
``2.0.2``|                                 |``gompi/2019b``, ``gompic/2019b``
``2.0.2``|``-BLIS-0.3.2``                  |``gompi/2018b``
``2.0.2``|``-OpenBLAS-0.2.13-LAPACK-3.5.0``|``gmvapich2/1.7.20``
``2.0.2``|``-OpenBLAS-0.2.15-LAPACK-3.6.0``|``gmpich/2016a``, ``gmvapich2/2016a``, ``gompi/2016a``
``2.0.2``|``-OpenBLAS-0.2.18-LAPACK-3.6.0``|``gompi/2016.04``, ``gompi/2016.06``
``2.0.2``|``-OpenBLAS-0.2.18-LAPACK-3.6.1``|``gompi/2016.07``, ``gompi/2016b``
``2.0.2``|``-OpenBLAS-0.2.19-LAPACK-3.6.1``|``gompi/2016.09``
``2.0.2``|``-OpenBLAS-0.2.19-LAPACK-3.7.0``|``gompi/2017a``
``2.0.2``|``-OpenBLAS-0.2.20``             |``gimpi/2017b``, ``gimpi/2018a``, ``gimpic/2017b``, ``gmpich/2017.08``, ``gompi/2017b``, ``gompi/2018a``, ``gompic/2017b``, ``gompic/2018a``
``2.0.2``|``-OpenBLAS-0.3.1``              |``gompi/2018b``, ``gompic/2018b``
``2.0.2``|``-OpenBLAS-0.3.3``              |``gompi/2018.08``
``2.0.2``|``-OpenBLAS-0.3.5``              |``gompi/2019a``, ``gompic/2019a``
``2.1.0``|                                 |``gompi/2020a``, ``gompi/2020b``, ``gompic/2020a``, ``gompic/2020b``
``2.1.0``|``-bf``                          |``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``iimpi/2020b``
``2.1.0``|``-bl``                          |``gompi/2020b``
``2.1.0``|``-fb``                          |``gompi/2021a``, ``gompi/2021b``
``2.2``  |``-amd``                         |``gompi/2020a``
``2.2.0``|``-fb``                          |``gompi/2022.05``, ``gompi/2022.10``, ``gompi/2022a``, ``gompi/2022b``, ``gompi/2023a``, ``nvompi/2022.07``

### Scalasca

Scalasca is a software tool that supports the performance optimization of  parallel programs by measuring and analyzing their runtime behavior. The  analysis identifies potential performance bottlenecks -- in particular  those concerning communication and synchronization -- and offers guidance  in exploring their causes. 

*homepage*: <https://www.scalasca.org/>

version  |toolchain
---------|---------------------------------
``2.3``  |``foss/2016a``
``2.5``  |``gompi/2019a``, ``gompi/2020a``
``2.6``  |``gompi/2021a``, ``gompic/2020b``
``2.6.1``|``gompi/2022a``

### SCALCE

SCALCE [skeɪlz] is a FASTQ compression tool that uses locally consistet parsing  to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can  handle other technologies (that generate base pair reads) if the read length is the same throughout the file.

*homepage*: <http://scalce.sourceforge.net/Home>

version|toolchain
-------|-------------
``2.7``|``GCC/4.8.2``

### Scalene

Scalene is a high-performance CPU, GPU and memory profiler for Python that does a number of things that other Python profilers do not and cannot do. It runs orders of magnitude faster than other profilers while delivering far more detailed information.

*homepage*: <https://github.com/plasma-umass/scalene>

version   |toolchain
----------|------------------
``1.5.13``|``GCCcore/11.2.0``

### scanpy

Scanpy is a scalable toolkit for analyzing single-cell gene expression data built  jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference  and differential expression testing. The Python-based implementation efficiently deals with  datasets of more than one million cells. 

*homepage*: <https://scanpy.readthedocs.io/en/stable/>

version  |toolchain
---------|------------------------------
``1.7.2``|``foss/2020b``
``1.8.1``|``foss/2020b``, ``foss/2021a``
``1.8.2``|``foss/2021b``
``1.9.1``|``foss/2021b``, ``foss/2022a``

### scArches

Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell  data.

*homepage*: <https://github.com/theislab/scarches>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.5.6``|                |``foss/2021a``
``0.5.6``|``-CUDA-11.3.1``|``foss/2021a``

### scsimple

scsimple is a package that helps simple conversion of different single-cell data formats to each other

*homepage*: <https://github.com/cellgeni/scsimple>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.0.7``|``-R-4.2.1`` |``foss/2022a``

### SCENIC

SCENIC Suite is a set of tools to study and decipher gene regulation.

*homepage*: <https://scenic.aertslab.org>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.2.4``|``-R-4.1.0`` |``foss/2021a``

### scGeneFit

Python code for genetic marker selection using linear programming.

*homepage*: <https://github.com/solevillar/scGeneFit-python>

version  |toolchain
---------|--------------
``1.0.2``|``foss/2021a``

### SCGid

A consensus approach to contig filtering and genome prediction from single-cell sequencing libraries

*homepage*: <https://github.com/amsesk/SCGid>

version  |toolchain
---------|--------------
``0.9b0``|``foss/2021b``

### scHiCExplorer

The scHiCExplorer is a software to demultiplex, process, correct, normalize, manipulate, analyse and visualize single-cell Hi-C data.

*homepage*: <https://schicexplorer.readthedocs.io>

version|toolchain
-------|--------------
``7``  |``foss/2022a``

### Schrodinger

Schrodinger aims to provide integrated software solutions and services   that truly meet its customers needs. We want to empower researchers around the world to   achieve their goals of improving human health and quality of life through advanced computational   techniques that transform the way chemists design compounds and materials. 

*homepage*: <https://www.schrodinger.com/>

version   |toolchain
----------|----------
``2020-4``|``system``
``2021-4``|``system``
``2022-1``|``system``
``2022-2``|``system``
``2022-3``|``system``

### scib

Benchmarking atlas-level data integration in single-cell genomics.

*homepage*: <https://github.com/theislab/scib>

version  |toolchain
---------|--------------
``1.1.1``|``foss/2022a``
``1.1.3``|``foss/2021a``

### sciClone

An R package for inferring the subclonal architecture of tumors 

*homepage*: <https://github.com/genome/sciclone>

version|versionsuffix|toolchain
-------|-------------|--------------
``1.1``|``-R-3.5.1`` |``foss/2018b``

### ScientificPython

ScientificPython is a collection of Python modules for scientific computing.  It contains support for geometry, mathematical functions, statistics, physical units, IO, visualization,  and parallelization.

*homepage*: <http://dirac.cnrs-orleans.fr/ScientificPython>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.9.4``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``

### scikit-allel

This package provides utilities for exploratory analysis of large scale genetic variation data.  It is based on numpy, scipy and other general-purpose Python scientific libraries.

*homepage*: <https://scikit-allel.readthedocs.io/en/latest/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.8``|``-Python-2.7.13``|``foss/2017a``
``1.2.1``|``-Python-3.8.2`` |``foss/2020a``
``1.3.2``|                  |``foss/2020b``
``1.3.3``|                  |``foss/2021a``

### scikit-bio

scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms and educational resources for bioinformatics.

*homepage*: <http://scikit-bio.org>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.5.6``|``-Python-3.8.2``|``foss/2020a``
``0.5.7``|                 |``foss/2020b``, ``foss/2021b``, ``foss/2022a``

### scikit-build

Scikit-Build, or skbuild, is an improved build system generator for CPython C/C++/Fortran/Cython extensions.

*homepage*: <https://scikit-build.readthedocs.io/en/latest>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------------------------------------------------------------
``0.10.0``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``0.10.0``|``-Python-3.8.2``|``foss/2020a``, ``fosscuda/2020a``
``0.11.1``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.15.0``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``0.17.2``|                 |``GCCcore/12.2.0``

### scikit-cuda

SciKit-cuda, a.k.a. skcuda, provides Python interfaces to many of the functions in the CUDA  device/runtime, CUBLAS, CUFFT, and CUSOLVER libraries distributed as part of NVIDIA's CUDA Programming Toolkit.

*homepage*: <https://github.com/lebedov/scikit-cuda>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.5.3``|``-Python-3.7.4``|``fosscuda/2019b``

### scikit-image

scikit-image is a collection of algorithms for image processing.

*homepage*: <https://scikit-image.org/>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------------
``0.12.3``|``-Python-2.7.11``|``foss/2016a``
``0.12.3``|``-Python-2.7.12``|``intel/2016b``
``0.12.3``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.13.0``|``-Python-2.7.13``|``intel/2017a``
``0.13.0``|``-Python-3.6.1`` |``intel/2017a``
``0.13.0``|``-Python-3.6.3`` |``foss/2017b``
``0.13.1``|``-Python-2.7.14``|``intel/2017b``
``0.13.1``|``-Python-3.6.3`` |``foss/2017b``
``0.13.1``|``-Python-3.6.4`` |``foss/2018a``
``0.14.0``|``-Python-3.6.4`` |``intel/2018a``
``0.14.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``0.15.0``|``-Python-3.7.2`` |``foss/2019a``
``0.16.2``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``0.17.1``|``-Python-3.8.2`` |``foss/2020a``
``0.18.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.18.3``|                  |``foss/2021a``
``0.19.1``|                  |``foss/2021b``
``0.19.3``|                  |``foss/2022a``

### scikit-learn

Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.

*homepage*: <https://scikit-learn.org/stable/index.html>

version   |versionsuffix                    |toolchain
----------|---------------------------------|------------------------------------------------------------------------
``0.16.1``|``-Python-2.7.13``               |``foss/2017a``
``0.17.1``|``-Python-2.7.11``               |``foss/2016a``, ``intel/2016a``
``0.17.1``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``0.17.1``|``-Python-2.7.12``               |``intel/2016b``
``0.17.1``|``-Python-3.5.1``                |``foss/2016a``, ``intel/2016a``
``0.18``  |``-Python-2.7.12``               |``intel/2016b``
``0.18``  |``-Python-3.5.2``                |``intel/2016b``
``0.18.1``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``0.18.1``|``-Python-2.7.13``               |``intel/2017a``
``0.18.1``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``0.18.1``|``-Python-3.6.1``                |``intel/2017a``
``0.18.2``|``-Python-3.6.1``                |``intel/2017a``
``0.19.0``|``-Python-3.6.1``                |``intel/2017a``
``0.19.1``|``-Python-2.7.13``               |``foss/2017a``
``0.19.1``|``-Python-2.7.14``               |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``|``-Python-3.6.3``                |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``
``0.20.0``|``-Python-2.7.15``               |``foss/2018b``, ``intel/2018b``
``0.20.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``0.20.2``|``-Python-2.7.15``               |``foss/2018b``
``0.20.2``|``-Python-3.6.6``                |``foss/2018b``
``0.20.3``|                                 |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``0.20.4``|``-Python-2.7.16``               |``intel/2019b``
``0.20.4``|``-Python-2.7.18``               |``foss/2020b``
``0.21.3``|``-Python-3.7.4``                |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``0.23.1``|``-Python-3.8.2``                |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``0.23.2``|                                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.24.2``|                                 |``foss/2021a``, ``intel/2021a``
``1.0.1`` |                                 |``foss/2021b``, ``intel/2021b``
``1.0.2`` |                                 |``foss/2021b``
``1.1.2`` |                                 |``foss/2022a``, ``intel/2022a``
``1.2.1`` |                                 |``gfbf/2022b``

### scikit-misc

Miscellaneous tools for data analysis and scientific computing

*homepage*: <https://github.com/has2k1/scikit-misc>

version  |toolchain
---------|------------------------------
``0.1.4``|``foss/2021a``, ``foss/2022a``

### scikit-multilearn

Scikit-multilearn is a BSD-licensed library for multi-label classification that is built on top of the well-known scikit-learn ecosystem.

*homepage*: <http://scikit.ml>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.6.6``|``foss/2018b``

### scikit-optimize

Scikit-Optimize, or skopt, is a simple and efficient library to minimize (very) expensive  and noisy black-box functions.

*homepage*: <https://scikit-optimize.github.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.2``|``-Python-3.6.6``|``intel/2018b``
``0.7.4``|``-Python-3.7.4``|``foss/2019b``
``0.8.1``|``-Python-3.8.2``|``foss/2020a``
``0.9.0``|                 |``foss/2021a``

### scikit-uplift

scikit-uplift is a Python module for classic approaches for uplift modeling built on top of scikit-learn. Uplift prediction aims to estimate the causal impact of a treatment at the individual level. 

*homepage*: <https://github.com/maks-sh/scikit-uplift>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.8.2``|``foss/2020a``

### SCIP

SCIP is currently one of the fastest non-commercial solvers for mixed integer programming (MIP) and mixed integer nonlinear programming (MINLP). It is also a framework for constraint integer programming and branch-cut-and-price. It allows for total control of the solution process and the access of detailed information down to the guts of the solver.

*homepage*: <https://www.scipopt.org/>

version  |toolchain
---------|--------------
``3.2.1``|``GCC/11.3.0``

### SCIPhI

Single-cell mutation identification via phylogenetic inference (SCIPhI) is a new approach  to mutation detection in individual tumor cells by leveraging the evolutionary relationship among cells.

*homepage*: <https://github.com/cbg-ethz/SCIPhI>

version  |toolchain
---------|---------------
``0.1.3``|``intel/2018b``

### scipy

SciPy is a collection of mathematical algorithms and convenience  functions built on the Numpy extension for Python.

*homepage*: <https://www.scipy.org>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------
``0.16.0``|``-Python-2.7.12``|``intel/2016b``
``0.17.0``|``-Python-2.7.11``|``intel/2016a``
``0.19.0``|``-Python-3.5.2`` |``intel/2016b``
``1.4.1`` |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``

### SciPy-bundle

Bundle of Python packages for scientific software

*homepage*: <https://python.org/>

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------------------------------
``2019.03``|                  |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2019.10``|``-Python-2.7.16``|``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2019.10``|``-Python-3.7.2`` |``intel/2019a``
``2019.10``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2020.03``|``-Python-2.7.18``|``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``2020.03``|``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``2020.11``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2020.11``|``-Python-2.7.18``|``foss/2020b``
``2021.05``|                  |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``2021.10``|                  |``foss/2021b``, ``intel/2021b``
``2021.10``|``-Python-2.7.18``|``foss/2021b``
``2022.05``|                  |``foss/2022.05``, ``foss/2022a``, ``intel/2022.05``, ``intel/2022a``
``2023.02``|                  |``gfbf/2022b``

### SCnorm

This package implements SCnorm — a method to normalize single-cell RNA-seq data.

*homepage*: <https://bioconductor.org/packages/SCnorm>

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.99.7``|``-R-3.4.0`` |``intel/2017a``

### Scoary

Microbial pan-GWAS using the output from Roary

*homepage*: <https://github.com/AdmiralenOla/Scoary>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.6.16``|                  |``foss/2021a``
``1.6.16``|``-Python-2.7.14``|``intel/2018a``

### SCons

SCons is a software construction tool.

*homepage*: <https://www.scons.org/>

version        |versionsuffix          |toolchain
---------------|-----------------------|---------------------------------------------------------------------------------
``2.4.1``      |``-Python-2.7.11``     |``foss/2016a``
``2.5.0``      |``-Python-2.7.12``     |``foss/2016b``, ``intel/2016b``
``2.5.1``      |                       |``intel/2017a``
``2.5.1``      |``-Python-2.7.12``     |``foss/2016b``, ``intel/2016b``
``2.5.1``      |``-Python-2.7.13``     |``intel/2017a``
``3.0.1``      |``-Python-2.7.14``     |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``3.0.1``      |``-Python-2.7.15``     |``foss/2018b``, ``fosscuda/2018b``
``3.0.1``      |``-Python-2.7.15-bare``|``GCCcore/7.3.0``
``3.0.1``      |``-Python-3.6.3``      |``foss/2017b``, ``intel/2017b``
``3.0.1``      |``-Python-3.6.4``      |``foss/2018a``, ``intel/2018a``
``3.0.4``      |``-Python-2.7.15``     |``foss/2018b``
``3.0.5``      |                       |``GCCcore/8.2.0``
``3.0.5``      |``-Python-2.7.15``     |``GCCcore/8.2.0``
``3.0.5``      |``-Python-3.7.2``      |``GCCcore/8.2.0``
``3.1.1``      |                       |``GCCcore/8.3.0``
``3.1.2``      |                       |``GCCcore/9.3.0``
``4.0.1``      |                       |``GCCcore/10.2.0``
``4.1.0.post1``|                       |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.2.0``      |                       |``GCCcore/11.2.0``
``4.4.0``      |                       |``GCCcore/11.3.0``

### SCOOP

SCOOP (Scalable COncurrent Operations in Python) is a distributed task module  allowing concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.

*homepage*: <http://code.google.com/p/scoop/>

version    |versionsuffix     |toolchain
-----------|------------------|-----------------
``0.6.2``  |``-Python-2.7.12``|``intel/2016b``
``0.7.1.1``|                  |``GCCcore/8.2.0``
``0.7.1.1``|``-Python-2.7.11``|``intel/2016a``
``0.7.1.1``|``-Python-2.7.14``|``intel/2017b``
``0.7.1.1``|``-Python-3.5.1`` |``intel/2016a``

### SCopeLoomR

An R package (compatible with SCope) to create generic .loom files and extend them with other data e.g.: SCENIC regulons, Seurat clusters and markers, ...

*homepage*: <https://github.com/aertslab/SCopeLoomR>

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.13.0``|``-R-4.1.2`` |``foss/2021b``

### Score-P

The Score-P measurement infrastructure is a highly scalable and simple-to-use  tool suite for profiling, event tracing, and online analysis of HPC  applications. 

*homepage*: <https://www.score-p.org>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------------------------------------------------------------
``2.0.1``|                |``foss/2016a``
``6.0``  |                |``gompi/2019a``, ``gompi/2019b``, ``gompi/2020a``, ``gompic/2019a``, ``gompic/2019b``, ``gompic/2020a``
``7.0``  |                |``gompi/2020b``, ``gompi/2021a``, ``gompic/2020b``
``7.1``  |``-CUDA-11.3.1``|``gompi/2021a``
``8.0``  |                |``gompi/2021b``, ``gompi/2022a``
``8.0``  |``-CUDA-11.4.1``|``gompi/2021b``
``8.0``  |``-CUDA-11.7.0``|``gompi/2022a``

### SCOTCH

Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.

*homepage*: <https://www.labri.fr/perso/pelegrin/scotch/>

version            |versionsuffix|toolchain
-------------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------
``5.1.12b_esmumps``|             |``foss/2017b``
``6.0.4``          |             |``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.0.4``          |``-64bitint``|``foss/2017b``
``6.0.6``          |             |``foss/2018a``, ``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2018a``
``6.0.9``          |             |``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpic/2019b``
``6.1.0``          |             |``gompi/2020b``, ``gompi/2021a``, ``iimpi/2020b``, ``iimpi/2021a``
``6.1.2``          |             |``gompi/2021b``, ``iimpi/2021b``
``7.0.1``          |             |``gompi/2022a``, ``iimpi/2022a``

### scp

The scp.py module uses a paramiko transport to send and recieve files via the scp1 protocol.

*homepage*: <https://github.com/jbardin/scp.py>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.10.2``|``-Python-2.7.12``|``intel/2016b``
``0.13.1``|``-Python-2.7.15``|``intel/2018b``
``0.13.2``|``-Python-2.7.15``|``intel/2018b``

### scPred

scPred package for cell type prediction from scRNA-seq data 

*homepage*: <https://github.com/powellgenomicslab/scPred>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.9.2``|``-R-4.1.2`` |``foss/2021b``

### Scrappie

Scrappie is a technology demonstrator for the Oxford Nanopore Research Algorithms group.

*homepage*: <https://github.com/nanoporetech/scrappie>

version  |toolchain
---------|---------------
``1.3.2``|``intel/2018a``

### scVelo

scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using  dynamical modeling.

*homepage*: <https://scvelo.org>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.1.24``|``-Python-3.7.4``|``foss/2019b``
``0.2.3`` |                 |``foss/2021a``

### scvi-tools

scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and analysis of single-cell omics data, built on top of PyTorch and AnnData.

*homepage*: <https://github.com/scverse/scvi-tools>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.16.4``|                |``foss/2021a``
``0.16.4``|``-CUDA-11.3.1``|``foss/2021a``

### Scythe

Scythe uses a Naive Bayesian approach to classify contaminant substrings in  sequence reads. It considers quality information, which can make it robust in  picking out 3'-end adapters, which often include poor quality bases. 

*homepage*: <https://github.com/ucdavis-bioinformatics/scythe>

version  |toolchain
---------|-----------------
``0.994``|``GCCcore/9.3.0``

### SDCC

SDCC is a retargettable, optimizing ANSI - C compiler suite that targets  the Intel MCS51 based microprocessors (8031, 8032, 8051, 8052, etc.), Maxim (formerly Dallas)  DS80C390 variants, Freescale (formerly Motorola) HC08 based (hc08, s08) and Zilog Z80 based  MCUs (z80, z180, gbz80, Rabbit 2000/3000, Rabbit 3000A). Work is in progress on supporting the  Microchip PIC16 and PIC18 targets. It can be retargeted for other microprocessors.

*homepage*: <http://sdcc.sourceforge.net/>

version  |toolchain
---------|----------
``3.3.0``|``system``

### SDL

SDL: Simple DirectMedia Layer, a cross-platform multimedia library

*homepage*: <https://www.libsdl.org/>

version   |toolchain
----------|-----------------
``1.2.15``|``GCCcore/6.4.0``

### SDL2

SDL: Simple DirectMedia Layer, a cross-platform multimedia library

*homepage*: <https://www.libsdl.org/>

version   |toolchain
----------|----------------------------------------------------------------------
``2.0.4`` |``intel/2016b``
``2.0.8`` |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.0.9`` |``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.0.10``|``GCCcore/8.3.0``
``2.0.14``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.0.20``|``GCCcore/11.2.0``
``2.0.22``|``GCCcore/11.3.0``
``2.26.3``|``GCCcore/12.2.0``

### SDL2_image

SDL_image is an image file loading library.

*homepage*: <https://www.libsdl.org/projects/SDL_image/>

version  |toolchain
---------|-----------------
``2.0.3``|``GCCcore/6.4.0``

### SDL_image

SDL_image is an image file loading library.

*homepage*: <https://www.libsdl.org/projects/SDL_image/release-1.2.html>

version   |toolchain
----------|-----------------
``1.2.12``|``GCCcore/6.4.0``

### SDSL

The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing  succinct data structures.

*homepage*: <https://github.com/simongog/sdsl-lite>

version           |toolchain
------------------|-----------------
``2.1.1-20191211``|``GCCcore/8.3.0``

### Seaborn

Seaborn is a Python visualization library based on matplotlib.  It provides a high-level interface for drawing attractive statistical graphics. 

*homepage*: <https://seaborn.pydata.org/>

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------------
``0.7.1`` |``-Python-2.7.12``|``intel/2016b``
``0.8.1`` |``-Python-2.7.13``|``foss/2017a``
``0.8.1`` |``-Python-2.7.14``|``intel/2018a``
``0.9.0`` |``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``0.9.0`` |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``
``0.9.0`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.9.0`` |``-Python-3.6.4`` |``intel/2018a``
``0.9.0`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.9.0`` |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``0.9.1`` |``-Python-2.7.16``|``foss/2019b``
``0.9.1`` |``-Python-2.7.18``|``foss/2020b``
``0.10.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.10.1``|                  |``intel/2020b``
``0.10.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.11.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.11.2``|                  |``foss/2021a``, ``foss/2021b``
``0.12.1``|                  |``foss/2022a``
``0.12.2``|                  |``foss/2022b``

### SearchGUI

SearchGUI is a user-friendly open-source graphical user interface for configuring and  running proteomics identification search engines and de novo sequencing algorithms,  currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,  Novor and DirecTag.

*homepage*: <https://github.com/compomics/searchgui>

version  |versionsuffix      |toolchain
---------|-------------------|----------
``3.3.3``|``-Java-1.8.0_152``|``system``

### SeaView

SeaView is a multiplatform, graphical user interface for multiple sequence alignment  and molecular phylogeny.

*homepage*: <https://doua.prabi.fr/software/seaview>

version  |toolchain
---------|------------------
``5.0.5``|``GCCcore/11.2.0``

### SECAPR

SECAPR is a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments

*homepage*: <https://github.com/AntonelliLab/seqcap_processor>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.1.15``|``-Python-2.7.16``|``foss/2019b``

### Seeder

Seeder is a framework for DNA motif discovery.

*homepage*: <https://metacpan.org/pod/Seeder>

version |versionsuffix   |toolchain
--------|----------------|-----------------
``0.01``|``-Perl-5.28.1``|``GCCcore/8.2.0``

### segemehl

segemehl is a software to map short sequencer reads to reference genomes.   Unlike other methods, segemehl is able to detect not only mismatches but also insertions   and deletions. Furthermore, segemehl is not limited to a specific read length and is able   to map primer- or polyadenylation contaminated reads correctly. segemehl implements a matching   strategy based on enhanced suffix arrays (ESA). Segemehl now supports the SAM format, reads   gziped queries to save both disk and memory space and allows bisulfite sequencing mapping   and split read mapping.

*homepage*: <https://www.bioinf.uni-leipzig.de/Software/segemehl/>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------
``0.2.0``|``foss/2016b``, ``intel/2017b``, ``intel/2018a``
``0.3.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.3.0``, ``foss/2018b``, ``iccifort/2020.4.304``

### segmentation-models

Python library with Neural Networks for Image Segmentation based on Keras and TensorFlow.

*homepage*: <https://github.com/qubvel/segmentation_models>

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.0.1``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``

### SeisSol

SeisSol is a software package for simulating wave propagation and dynamic rupture based on  the arbitrary high-order accurate derivative discontinuous Galerkin method (ADER-DG).

*homepage*: <http://www.seissol.org>

version   |versionsuffix     |toolchain
----------|------------------|--------------
``201703``|``-Python-2.7.15``|``foss/2018b``

### SelEstim

SelEstim is aimed at distinguishing neutral from selected polymorphisms and estimate   the intensity of selection at the latter. The SelEstim model accounts explicitly for positive selection,   and it is assumed that all marker loci in the dataset are responding to selection, to some extent

*homepage*: <http://www1.montpellier.inra.fr/CBGP/software/selestim/index.html>

version  |versionsuffix    |toolchain
---------|-----------------|----------
``1.1.4``|``-Linux-64bits``|``system``

### SELFIES

Robust representation of semantically constrained graphs, in particular for molecules in chemistry

*homepage*: <https://github.com/aspuru-guzik-group/selfies>

version  |toolchain
---------|--------------
``2.1.1``|``GCC/11.2.0``

### SentencePiece

Unsupervised text tokenizer for Neural Network-based text generation.

*homepage*: <https://github.com/google/sentencepiece>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.1.85``|``-Python-3.7.4``|``GCC/8.3.0``
``0.1.94``|``-Python-3.8.2``|``GCC/9.3.0``
``0.1.96``|                 |``GCC/10.2.0``, ``GCC/10.3.0``
``0.1.97``|                 |``GCC/11.3.0``

### sep

Python and C library for Source Extraction and Photometry.  (this simpleconfig provides python library only)

*homepage*: <https://sep.readthedocs.io>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.0.3``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``

### SEPP

SATe-enabled Phylogenetic Placement - addresses the problem of phylogenetic placement of short reads into reference alignments and trees.

*homepage*: <https://github.com/smirarab/sepp>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------
``4.3.10``|``-Python-3.7.4``|``foss/2019b``
``4.3.10``|``-Python-3.8.2``|``foss/2020a``
``4.4.0`` |                 |``foss/2020b``
``4.5.0`` |                 |``foss/2021a``
``4.5.1`` |                 |``foss/2021b``, ``foss/2022a``, ``foss/2022b``

### Seq-Gen

Seq-Gen is a program that will simulate the evolution of nucleotide or amino acid sequences  along a phylogeny, using common models of the substitution process.

*homepage*: <http://tree.bio.ed.ac.uk/software/Seq-Gen>

version  |toolchain
---------|---------------
``1.3.4``|``intel/2017b``

### seq2HLA

In-silico method written in Python and R to determine HLA genotypes of a sample.  seq2HLA takes standard RNA-Seq sequence reads in fastq format as input,  uses a bowtie index comprising all HLA alleles and outputs the most likely  HLA class I and class II genotypes (in 4 digit resolution),  a p-value for each call, and the expression of each class. 

*homepage*: <https://github.com/TRON-Bioinformatics/seq2HLA>

version|versionsuffix     |toolchain
-------|------------------|---------------
``2.3``|``-Python-2.7.14``|``intel/2018a``
``2.3``|``-Python-2.7.15``|``foss/2018b``

### SeqAn

SeqAn is an open source C++ library of efficient algorithms and data structures  for the analysis of sequences with the focus on biological data

*homepage*: <https://www.seqan.de/>

version  |versionsuffix|toolchain
---------|-------------|-----------------------------------------------------------------------------------------------------------------------------------
``1.4.2``|``-library`` |``system``
``2.3.2``|             |``foss/2016b``
``2.4.0``|             |``GCC/8.2.0-2.31.1``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2018b``

### SeqAn3

SeqAn is an open source C++ library   of efficient algorithms and data structures for the   analysis of sequences with the focus on biological   data. Our library applies a unique generic design   that guarantees high performance, generality,   extensibility, and integration with other libraries.   SeqAn is simple to use and simplifies the development   of new software tools with a minimal loss of   performance. 

*homepage*: <https://github.com/seqan/seqan3>

version  |toolchain
---------|----------
``3.0.0``|``system``

### SeqKit

SeqKit - a cross-platform and ultrafast toolkit for FASTA/Q file manipulation

*homepage*: <https://bioinf.shenwei.me/seqkit/>

version   |toolchain
----------|----------
``0.8.1`` |``system``
``0.13.2``|``system``
``2.1.0`` |``system``
``2.2.0`` |``system``
``2.3.1`` |``system``

### SeqLib

C++ interface to HTSlib, BWA-MEM and Fermi.

*homepage*: <https://github.com/walaj/SeqLib>

version  |toolchain
---------|-------------------------------------------------------------
``1.2.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

### Seqmagick

We often have to convert between sequence formats and do  little tasks on them, and it's not worth writing scripts for that. Seqmagick  is a kickass little utility built in the spirit of imagemagick to expose the  file format conversion in Biopython in a convenient way. Instead of having a  big mess of scripts, there is one that takes arguments.

*homepage*: <https://fhcrc.github.io/seqmagick/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.6.1``|``-Python-2.7.11``|``foss/2016a``
``0.6.2``|``-Python-2.7.15``|``foss/2018b``

### SeqPrep

Tool for stripping adaptors and/or merging paired reads with overlap into single reads.

*homepage*: <https://github.com/jstjohn/SeqPrep>

version  |toolchain
---------|-----------------
``1.3.2``|``GCCcore/7.3.0``

### seqtk

Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.  It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.

*homepage*: <https://github.com/lh3/seqtk/>

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------------------------------------
``1.2``|``foss/2016b``, ``intel/2017a``
``1.3``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018a``, ``foss/2018b``
``1.4``|``GCC/12.2.0``

### Serf

The serf library is a high performance C-based HTTP client library  built upon the Apache Portable Runtime (APR) library

*homepage*: <http://serf.apache.org/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``intel/2017b``, ``iomkl/2018a``

### setuptools

Easily download, build, install, upgrade, and uninstall Python packages

*homepage*: <https://pypi.org/project/setuptools>

version   |versionsuffix|toolchain
----------|-------------|----------
``1.4.2`` |             |``system``
``41.0.1``|``-py3``     |``system``

### Seurat

Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.

*homepage*: <https://satijalab.org/seurat>

version     |versionsuffix|toolchain
------------|-------------|---------------
``1.4.0.16``|``-R-3.4.0`` |``intel/2017a``
``2.3.4``   |``-R-3.4.4`` |``intel/2018a``
``2.3.4``   |``-R-3.5.1`` |``foss/2018b``
``3.1.2``   |``-R-3.6.0`` |``foss/2019a``
``3.1.5``   |``-R-4.0.0`` |``foss/2020a``
``4.0.1``   |``-R-4.0.3`` |``foss/2020b``
``4.0.3``   |``-R-4.0.3`` |``foss/2020b``
``4.2.0``   |``-R-4.2.1`` |``foss/2022a``
``4.3.0``   |``-R-4.1.2`` |``foss/2021b``
``4.3.0``   |``-R-4.2.1`` |``foss/2022a``

### SeuratData

SeuratData is a mechanism for distributing datasets in the form of Seurat objects using R's internal package and data management systems.

*homepage*: <https://github.com/satijalab/seurat-data>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20210514``|``-R-4.0.3`` |``foss/2020b``

### SeuratDisk

Interfaces for HDF5-based Single Cell File Formats

*homepage*: <https://github.com/mojaveazure/seurat-disk>

version       |versionsuffix|toolchain
--------------|-------------|--------------
``0.0.0.9020``|``-R-4.2.1`` |``foss/2022a``

### SeuratWrappers

SeuratWrappers is a collection of community-provided methods and extensions for Seurat

*homepage*: <https://github.com/satijalab/seurat-wrappers>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20210528``|``-R-4.0.3`` |``foss/2020b``

### sf

Support for simple features, a standardized way to encode spatial vector data. Binds to GDAL for reading and writing data, to GEOS for geometrical operations, and to PROJ for projection conversions and datum transformations.

*homepage*: <https://cran.r-project.org/package=sf>

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.9-5``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

### sfftk

sfftk is a set of utilities that facilitate creation, conversion and             modification of Electron Microscopy Data Bank - Segmentation File Format         (EMDB-SFF) files. EMDB-SFF is an open, community-driven file format to handle    annotated segmentations and subtomogram averages that facilitates segmentation   file interchange. It is written in Python and provides both a command-line suite of commands and a Python API.

*homepage*: <https://emdb-empiar.github.io/EMDB-SFF/>

version  |toolchain
---------|--------------
``0.7.4``|``foss/2021a``

### Shannon

Shannon is a program for assembling transcripts from RNA-Seq data

*homepage*: <https://sreeramkannan.github.io/Shannon/>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170511``|``-Python-2.7.13``|``intel/2017a``

### SHAP

SHAP (SHapley Additive exPlanations) is a game theoretic approach to explain the output of any  machine learning model. It connects optimal credit allocation with local explanations using the classic Shapley  values from game theory and their related extensions.

*homepage*: <https://github.com/slundberg/shap>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.35.0``|``-Python-3.7.4``|``foss/2019b``
``0.41.0``|                 |``foss/2022a``

### shapAAR

An R package for the extraction, analysis and classification of (not only) archaeological objects  from scanned images.

*homepage*: <https://github.com/ISAAKiel/shapAAR>

version           |versionsuffix|toolchain
------------------|-------------|--------------
``0.1.0-20180425``|``-R-3.6.0`` |``foss/2019a``

### SHAPEIT

SHAPEIT is a fast and accurate method for estimation of  haplotypes (aka phasing) from genotype or sequencing data.

*homepage*: <https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html>

version   |versionsuffix  |toolchain
----------|---------------|----------
``2.r837``|``.GLIBCv2.12``|``system``
``2.r904``|``.glibcv2.17``|``system``

### SHAPEIT4

SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. 

*homepage*: <https://odelaneau.github.io/shapeit4/>

version  |toolchain
---------|----------------------------------------------
``4.1.3``|``foss/2019b``
``4.2.0``|``foss/2019b``, ``foss/2020a``, ``foss/2020b``
``4.2.2``|``foss/2020b``, ``foss/2021a``

### Shapely

Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects. It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.

*homepage*: <https://github.com/Toblerity/Shapely>

version        |versionsuffix    |toolchain
---------------|-----------------|-------------------------------------------------------
``1.7.0``      |``-Python-3.7.2``|``foss/2019a``
``1.7.0``      |``-Python-3.7.4``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``1.7.1``      |``-Python-3.8.2``|``GCC/9.3.0``
``1.8.1.post1``|                 |``GCC/11.2.0``
``1.8.2``      |                 |``foss/2021b``, ``foss/2022a``
``1.8a1``      |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``iccifort/2020.4.304``
``2.0.1``      |                 |``foss/2022b``

### sharutils

GNU shar makes so-called shell archives out of many files, preparing them for transmission by  electronic mail services, while unshar helps unpacking shell archives after reception.

*homepage*: <https://www.gnu.org/software/sharutils/>

version |toolchain
--------|-----------------
``4.15``|``GCCcore/6.3.0``

### Shasta

The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include: Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k ≈ 10). 

*homepage*: <https://github.com/chanzuckerberg/shasta>

version  |toolchain
---------|--------------
``0.8.0``|``foss/2020b``

### ShengBTE

A solver for the Boltzmann transport equation for phonons.

*homepage*: <https://bitbucket.org/sousaw/shengbte>

version  |toolchain
---------|--------------
``1.1.1``|``foss/2021a``
``1.5.0``|``foss/2022a``

### shift

Shift is a framework for Self-Healing Independent File Transfer that provides high performance and resilience for local and remote transfers through a variety of techniques. 

*homepage*: <http://people.nas.nasa.gov/~kolano/projects/shift.html>

version|toolchain
-------|---------------
``4.0``|``intel/2016a``

### SHORE

SHORE, for Short Read, is a mapping and analysis pipeline for short read data produced on the Illumina platform. 

*homepage*: <http://1001genomes.org/software/shore.html>

version  |toolchain
---------|--------------
``0.9.3``|``foss/2016a``

### Short-Pair

Sensitive Short Read Homology Search for Paired-End Reads

*homepage*: <https://sourceforge.net/projects/short-pair>

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20170125``|                  |``foss/2021b``
``20170125``|``-Python-2.7.15``|``foss/2018b``

### shovill

Faster SPAdes assembly of Illumina reads

*homepage*: <https://github.com/tseemann/shovill>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.9.0``|``-Python-2.7.14``|``foss/2018a``
``1.0.4``|``-Python-2.7.15``|``foss/2018b``
``1.1.0``|                  |``gompi/2021b``

### shrinkwrap

A std::streambuf wrapper for compression formats.

*homepage*: <https://github.com/jonathonl/shrinkwrap>

version       |toolchain
--------------|------------------
``1.0.0-beta``|``GCCcore/8.2.0``
``1.1.0``     |``GCCcore/10.2.0``

### SHTns

Spherical Harmonic Transform library aimed at high performance numerical simulations in spherical geometries.

*homepage*: <https://bitbucket.org/nschaeff/shtns>

version  |toolchain
---------|--------------
``2.7``  |``foss/2021b``
``3.5.1``|``foss/2021b``

### Sibelia

Sibelia: A comparative genomics tool: It assists biologists in analysing   the genomic variations that correlate with pathogens, or the genomic changes that help   microorganisms adapt in different environments. Sibelia will also be helpful for the   evolutionary and genome rearrangement studies for multiple strains of microorganisms.

*homepage*: <http://bioinf.spbau.ru/en/sibelia>

version  |toolchain
---------|------------------------------
``3.0.6``|``foss/2016b``
``3.0.7``|``foss/2018b``, ``foss/2020b``

### SICER2

Redesigned and improved ChIP-seq broad peak calling tool SICER

*homepage*: <https://zanglab.github.io/SICER2/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.8.2``|``foss/2020a``

### sickle

Windowed Adaptive Trimming for fastq files using quality 

*homepage*: <https://github.com/najoshi/sickle>

version |toolchain
--------|-------------------------------
``1.33``|``foss/2017a``, ``intel/2018a``

### Siesta

SIESTA is both a method and its computer program implementation, to perform efficient electronic  structure calculations and ab initio molecular dynamics simulations of molecules and solids.

*homepage*: <http://departments.icmab.es/leem/siesta>

version        |versionsuffix|toolchain
---------------|-------------|------------------------------------------------------------------------------------------------
``4.0``        |             |``foss/2017b``, ``intel/2017a``
``4.0.1``      |             |``intel/2017a``
``4.1-b2``     |             |``intel/2017a``
``4.1-b3``     |             |``intel/2017a``
``4.1-b4``     |             |``foss/2018b``, ``intel/2018b``
``4.1-MaX-1.0``|             |``intel/2019b``
``4.1-MaX-1.0``|``-PEXSI``   |``intel/2019b``
``4.1.5``      |             |``foss/2020a``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2020a``, ``intel/2022a``

### SignalP

SignalP predicts the presence  and  location of signal peptide  cleavage sites in  amino acid  sequences  from  different organisms

*homepage*: <https://services.healthtech.dtu.dk/software.php>

version |versionsuffix        |toolchain
--------|---------------------|------------------------------
``5.0b``|``-Linux``           |``system``
``6.0g``|``-fast``            |``foss/2021b``, ``foss/2022a``
``6.0g``|``-fast-CUDA-11.7.0``|``foss/2022a``

### silhouetteRank

silhouetteRank is a tool for finding spatially variable genes based on computing silhouette  coefficient from binarized spatial gene expression data

*homepage*: <https://pypi.org/project/silhouetteRank>

version     |toolchain
------------|--------------
``1.0.5.13``|``foss/2022a``

### silx

The silx project provides a collection of Python packages to support the  development of data assessment, reduction and analysis applications at synchrotron radiation facilities.

*homepage*: <http://www.silx.org/>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.13.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.14.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``1.0.0`` |                 |``foss/2021b``

### simint

Simint is a vectorized implementation of the Obara-Saika (OS)  method of calculating electron repulsion integrals. Speedup is gained by  vectorizing the primitive loop of the OS algorithm, with additional vectorization  and optimizations left to the compiler.

*homepage*: <https://www.bennyp.org/research/simint/>

version|versionsuffix           |toolchain
-------|------------------------|--------------
``0.7``|``-lmax-5-vec-avx-psi4``|``GCC/11.2.0``

### SimNIBS

SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation.

*homepage*: <https://simnibs.github.io/simnibs>

version  |toolchain
---------|--------------
``3.2.4``|``foss/2020b``

### SimPEG

Simulation and Parameter Estimation in Geophysics: a python package for simulation and gradient based parameter estimation in the context of geophysical applications.

*homepage*: <https://simpeg.xyz>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.3.1`` |``-Python-2.7.12``|``intel/2016b``
``0.14.1``|``-Python-3.8.2`` |``intel/2020a``
``0.18.1``|                  |``foss/2021b``, ``intel/2021b``

### SIMPLE

Single-particle IMage Processing Linux Engine SIMPLE is an open-source software package for analysis of cryogenic transmission electron microscopy (cryo-EM)  movies of single-particles (Single-Particle Analysis, SPA).

*homepage*: <https://simplecryoem.com/>

version  |toolchain
---------|--------------
``2.5``  |``foss/2018a``
``3.0.0``|``foss/2020b``

### Simple-DFTD3

Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to provide a user-friendly and uniform interface to the D3 dispersion model and for the calculation of DFT-D3 dispersion corrections.

*homepage*: <https://dftd3.readthedocs.io>

version  |toolchain
---------|--------------
``0.7.0``|``foss/2022a``

### SimpleElastix

Multi-lingual medical image registration library.

*homepage*: <https://simpleelastix.github.io>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.10.0``|``-Python-3.6.3``|``foss/2017b``
``0.10.0``|``-Python-3.6.4``|``foss/2018a``
``1.1.0`` |``-Python-3.7.4``|``foss/2019b``

### SimpleITK

SimpleITK is a simplified programming interface to the algorithms and data structures of  the Insight Toolkit (ITK).

*homepage*: <https://www.simpleitk.org>

version    |versionsuffix     |toolchain
-----------|------------------|----------------------------------
``1.1.0``  |``-Python-2.7.15``|``foss/2018b``
``1.1.0``  |``-Python-3.6.4`` |``foss/2018a``
``1.1.0``  |``-Python-3.6.6`` |``foss/2018b``
``1.2.4``  |``-Python-3.7.4`` |``foss/2019b``
``2.1.0``  |                  |``foss/2020b``, ``fosscuda/2020b``
``2.1.1``  |                  |``foss/2021a``
``2.1.1.2``|                  |``foss/2021b``, ``foss/2022a``

### simpy

SimPy is a process-based discrete-event simulation framework based on standard Python.

*homepage*: <https://simpy.readthedocs.io>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``3.0.11``|                 |``intel/2019a``
``3.0.11``|``-Python-3.6.6``|``intel/2018b``

### SimVascular

SimVascular is an open source software suite for cardiovascular simulation, providing a complete  pipeline from medical image data to 3D model construction, meshing, and blood flow simulation.

*homepage*: <http://simvascular.github.io/>

version      |toolchain
-------------|---------------
``2.16.0406``|``intel/2016b``

### SingleM

SingleM is a tool to find the abundances of discrete operational taxonomic units (OTUs) directly  from shotgun metagenome data, without heavy reliance on reference sequence databases.

*homepage*: <https://github.com/wwood/singlem>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12.1``|``-Python-2.7.15``|``intel/2019a``

### Singular

Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory.

*homepage*: <https://www.singular.uni-kl.de/>

version  |toolchain
---------|--------------------------------
``4.1.2``|``GCC/8.2.0-2.31.1``, ``system``

### SiNVICT

SiNVICT is a tool for the detection of SNVs and indels from cfDNA/ctDNA samples obtained  by ultra-deep sequencing.

*homepage*: <https://github.com/sfu-compbio/sinvict>

version         |toolchain
----------------|-------------
``1.0-20180817``|``GCC/9.3.0``

### SIONlib

SIONlib is a scalable I/O library for parallel access to task-local files.  The library not only supports writing and reading binary data to or from  several thousands of processors into a single or a small number of physical  files, but also provides global open and close functions to access SIONlib  files in parallel. This package provides a stripped-down installation of  SIONlib for use with performance tools (e.g., Score-P), with renamed symbols  to avoid conflicts when an application using SIONlib itself is linked against  a tool requiring a different SIONlib version. 

*homepage*: <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html>

version  |versionsuffix|toolchain
---------|-------------|----------------------------------------------------------------------------
``1.6.1``|             |``foss/2016a``
``1.6.1``|``-tools``   |``foss/2016a``
``1.7.1``|             |``foss/2017a``
``1.7.1``|``-tools``   |``foss/2017a``
``1.7.4``|``-tools``   |``GCCcore/8.2.0``
``1.7.6``|``-tools``   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.7.7``|``-tools``   |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### SIP

SIP is a tool that makes it very simple to create Python bindings for C and C++ libraries.

*homepage*: <http://www.riverbankcomputing.com/software/sip/>

version    |versionsuffix     |toolchain
-----------|------------------|----------------------------------------------------------------
``4.18``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.18.1`` |``-Python-2.7.11``|``foss/2016a``
``4.18.1`` |``-Python-2.7.12``|``intel/2016b``
``4.19``   |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.19.2`` |``-Python-2.7.13``|``intel/2017a``
``4.19.8`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``4.19.8`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``4.19.13``|``-Python-2.7.15``|``fosscuda/2018b``

### siscone

Hadron Seedless Infrared-Safe Cone jet algorithm

*homepage*: <https://siscone.hepforge.org/>

version  |toolchain
---------|------------------
``3.0.5``|``GCCcore/11.3.0``

### SISSO

A data-driven method combining symbolic regression and compressed sensing toward accurate &  interpretable models.

*homepage*: <https://github.com/rouyang2017/SISSO>

version         |toolchain
----------------|---------------
``3.0.2``       |``iimpi/2021b``
``3.1-20220324``|``iimpi/2021b``

### SISSO++

C++ implementation of SISSO with built in Python bindings for an efficient python interface

*homepage*: <https://gitlab.com/sissopp_developers/sissopp>

version|toolchain
-------|--------------
``1.1``|``foss/2021b``

### SKESA

SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.

*homepage*: <https://github.com/ncbi/SKESA>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.2``  |                  |``foss/2018a``
``2.3.0``|                  |``foss/2018b``
``2.4.0``|``_saute.1.3.0_1``|``gompi/2021b``

### sketchmap

Sketch-map is a dimensionality reduction algorithm that is particularly well suited to examining  the high-dimensionality data that is routinely produced in atomistic simulations.

*homepage*: <http://sketchmap.org/>

version     |toolchain
------------|---------------
``20170130``|``intel/2016b``

### skewer

skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter  trimming and it is specially designed for processing next-generation sequencing (NGS) paired-end sequences.

*homepage*: <https://github.com/relipmoc/skewer>

version  |toolchain
---------|---------------
``0.2.2``|``intel/2016b``

### sklearn-pandas

This module provides a bridge between Scikit-Learn's machine learning methods and pandas-style Data Frames. In particular, it provides a way to map DataFrame columns to transformations, which are later recombined into features.

*homepage*: <https://github.com/scikit-learn-contrib/sklearn-pandas>

version  |toolchain
---------|--------------
``2.2.0``|``foss/2021b``

### sklearn-som

A simple, planar self-organizing map with methods similar to clustering methods in Scikit Learn.

*homepage*: <https://github.com/rileypsmith/sklearn-som>

version  |toolchain
---------|--------------
``1.1.0``|``foss/2020b``

### skorch

A scikit-learn compatible neural network library that wraps PyTorch.

*homepage*: <https://skorch.readthedocs.io/>

version   |versionsuffix                  |toolchain
----------|-------------------------------|--------------
``0.11.0``|``-PyTorch-1.10.0``            |``foss/2021a``
``0.11.0``|``-PyTorch-1.10.0-CUDA-11.3.1``|``foss/2021a``

### SlamDunk

SlamDunk is a novel, fully automated software tool for automated, robust, scalable and reproducible SLAMseq data analysis.

*homepage*: <https://t-neumann.github.io/slamdunk>

version  |toolchain
---------|--------------
``0.4.3``|``foss/2021b``

### SLATEC

SLATEC Common Mathematical Library, a comprehensive software library containing     over 1400 general purpose mathematical and statistical routines written in Fortran 77.

*homepage*: <https://www.netlib.org/slatec/>

version|toolchain
-------|-------------------------------------------------------------------------
``4.1``|``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``

### SLEPc

SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution  of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for  either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a  partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.

*homepage*: <https://www.grycap.upv.es/slepc/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``3.8.3`` |                  |``foss/2017b``
``3.9.2`` |                  |``foss/2018a``
``3.11.0``|                  |``foss/2018b``
``3.12.2``|``-Python-2.7.16``|``intel/2019b``
``3.12.2``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``3.12.2``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``3.14.2``|                  |``foss/2020b``
``3.15.1``|                  |``foss/2021a``
``3.17.2``|                  |``foss/2022a``
``3.18.2``|                  |``intel/2021b``

### slepc4py

Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations.

*homepage*: <https://bitbucket.org/slepc/slepc4py>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``3.9.0`` |``-Python-3.6.4``|``foss/2018a``
``3.12.0``|``-Python-3.7.4``|``foss/2019b``
``3.15.1``|                 |``foss/2021a``

### sleuth

Investigate RNA-Seq transcript abundance from kallisto and perform differential expression analysis.

*homepage*: <http://pachterlab.github.io/sleuth>

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.29.0``|``-R-3.4.0`` |``intel/2017a``
``0.30.0``|``-R-3.5.1`` |``foss/2018b``

### slidingwindow

slidingwindow is a simple little Python library for computing a set of windows into a larger dataset,  designed for use with image-processing algorithms that utilise a sliding window to break the processing up  into a series of smaller chunks.

*homepage*: <https://github.com/adamrehn/slidingwindow>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.0.13``|``-Python-2.7.15``|``intel/2018b``
``0.0.13``|``-Python-3.6.6`` |``intel/2018b``

### SLiM

SLiM is an evolutionary simulation framework that combines a powerful engine for population  genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios.

*homepage*: <https://messerlab.org/slim>

version  |toolchain
---------|--------------
``3.4``  |``GCC/9.3.0``
``4.0``  |``GCC/11.2.0``
``4.0.1``|``GCC/11.3.0``

### slow5tools

slow5tools is a toolkit for converting (FAST5 <-> SLOW5), compressing, viewing, indexing  and manipulating data in SLOW5 format.

*homepage*: <https://hasindu2008.github.io/slow5tools>

version  |toolchain
---------|---------------
``0.4.0``|``gompi/2021b``

### smafa

Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences  which are all the same length.

*homepage*: <https://github.com/wwood/smafa>

version  |toolchain
---------|-----------------
``0.4.0``|``GCCcore/8.2.0``

### smallgenomeutilities

The smallgenomeutilities are a collection of scripts that is  useful for dealing and manipulating NGS data of small viral genomes. They  are written in Python 3 with a small number of dependencies.

*homepage*: <https://github.com/cbg-ethz/smallgenomeutilities>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.1``|``-Python-3.6.6``|``foss/2018b``

### SMAP

SMAP is an analysis tool for stack-based NGS read mapping

*homepage*: <https://ngs-smap.readthedocs.io>

version  |toolchain
---------|--------------
``4.6.5``|``foss/2022a``

### SMARTdenovo

SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy.

*homepage*: <https://github.com/ruanjue/smartdenovo>

version     |toolchain
------------|--------------
``20180219``|``foss/2018b``

### SMC++

SMC++ is a program for estimating the size history of populations from whole genome sequence data.

*homepage*: <https://github.com/popgenmethods/smcpp>

version   |toolchain
----------|--------------
``1.15.4``|``foss/2022a``

### smfishHmrf

smFish spatial pattern mining and cell type prediction

*homepage*: <https://pypi.org/project/smfishHmrf>

version  |toolchain
---------|--------------
``1.3.3``|``foss/2022a``

### smithwaterman

smith-waterman-gotoh alignment algorithm.

*homepage*: <https://github.com/ekg/smithwaterman>

version     |toolchain
------------|-----------------------------------------------------------------------------
``20160702``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### Smoldyn

Smoldyn is a computer program for cell-scale biochemical simulations. It simulates each molecule of  interest individually to capture natural stochasticity and to yield nanometer-scale spatial resolution. It treats  other molecules implicitly, enabling it to simulate hundreds of thousands of molecules over several minutes of real  time. Simulated molecules diffuse, react, are confined by surfaces, and bind to membranes much as they would in a real  biological system. 

*homepage*: <http://www.smoldyn.org/>

version |toolchain
--------|--------------
``2.48``|``foss/2016a``

### smooth-topk

Smooth Loss Functions for Deep Top-k Classification

*homepage*: <https://github.com/oval-group/smooth-topk>

version         |versionsuffix   |toolchain
----------------|----------------|--------------
``1.0-20210817``|                |``foss/2021a``
``1.0-20210817``|``-CUDA-11.3.1``|``foss/2021a``

### SMRT-Link

PacBio’s open-source SMRT Analysis software suite is designed for use with Single Molecule,   Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI   or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through   the extensive set of APIs provided

*homepage*: <https://www.pacb.com/support/software-downloads/>

version        |versionsuffix      |toolchain
---------------|-------------------|----------
``5.1.0.26412``|``-cli-tools-only``|``system``
``6.0.0.47841``|``-cli-tools-only``|``system``

### SMV

Smokeview is a visualization program that displays output of FDS and CFAST simulations.

*homepage*: <https://github.com/firemodels/smv>

version   |toolchain
----------|-----------------------
``6.7.17``|``iccifort/2020.4.304``

### snakemake

The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.

*homepage*: <https://snakemake.readthedocs.io>

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``5.2.2`` |``-Python-3.6.4``|``intel/2018a``
``5.2.4`` |``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``5.7.1`` |``-Python-3.7.2``|``foss/2019a``
``5.26.1``|``-Python-3.8.2``|``intel/2020a``
``6.1.0`` |                 |``foss/2020b``
``6.6.1`` |                 |``foss/2021a``
``6.10.0``|                 |``foss/2021b``
``7.18.2``|                 |``foss/2021b``
``7.22.0``|                 |``foss/2022a``

### SNAP

Scalable Nucleotide Alignment Program -- a fast and accurate read aligner for  high-throughput sequencing data

*homepage*: <https://www.microsoft.com/en-us/research/project/snap>

version       |toolchain
--------------|----------------------------------------------
``1.0beta.23``|``intel/2017b``
``2.0.1``     |``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### SNAP-HMM

SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser. 

*homepage*: <https://korflab.github.io/>

version       |toolchain
--------------|-------------------------------------------------------------------------
``2013-11-29``|``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``20190603``  |``GCC/10.2.0``
``20221022``  |``GCC/11.3.0``

### SNAPE-pooled

SNAPE-pooled computes the probability distribution for the frequency of the minor allele  in a certain population, at a certain position in the genome.

*homepage*: <https://code.google.com/archive/p/snape-pooled/>

version|toolchain
-------|--------------
``r32``|``foss/2016a``

### snaphu

SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping  proposed by Chen and Zebker

*homepage*: <https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/>

version  |toolchain
---------|---------------------------------------------------
``1.4.2``|``GCCcore/6.3.0``, ``intel/2016b``, ``intel/2017a``

### snappy

Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.

*homepage*: <https://github.com/google/snappy>

version   |toolchain
----------|------------------------------------------------------------------------------------------------------------
``1.1.2`` |``GCC/4.9.2``
``1.1.3`` |``GCC/4.9.3``, ``GCC/4.9.3-2.25``
``1.1.6`` |``system``
``1.1.7`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017a``, ``intel/2017b``
``1.1.8`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.1.9`` |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.1.10``|``GCCcore/12.3.0``

### Sniffles

A fast structural variant caller for long-read sequencing,  Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data.

*homepage*: <https://github.com/fritzsedlazeck/Sniffles>

version  |toolchain
---------|--------------
``2.0.7``|``GCC/11.3.0``

### snippy

Rapid haploid variant calling and core genome alignment

*homepage*: <https://github.com/tseemann/snippy>

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``4.4.1``|``-Perl-5.28.0``         |``foss/2018b``
``4.6.0``|                         |``GCC/10.2.0``
``4.6.0``|``-Java-13-Python-3.8.2``|``GCC/9.3.0``
``4.6.0``|``-R-4.1.2``             |``foss/2021b``

### snp-sites

Finds SNP sites from a multi-FASTA alignment file.

*homepage*: <https://sanger-pathogens.github.io/snp-sites/>

version  |toolchain
---------|-----------------------------------------------------------------------------
``2.5.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

### snpEff

SnpEff is a variant annotation and effect prediction tool.  It annotates and predicts the effects of genetic variants (such as amino acid changes).

*homepage*: <https://pcingola.github.io/SnpEff/>

version |versionsuffix            |toolchain
--------|-------------------------|------------------
``3.6`` |``-Java-1.7.0_80``       |``system``
``4.1d``|``-Java-1.7.0_80``       |``system``
``4.3t``|``-Java-1.8``            |``system``
``5.0`` |``-Java-13``             |``system``
``5.0`` |``-Java-13-Python-3.8.2``|``GCCcore/9.3.0``
``5.0e``|``-Java-11``             |``GCCcore/11.2.0``
``5.0e``|``-Java-13``             |``GCCcore/10.2.0``

### SNPhylo

SNPhylo: a pipeline to generate a phylogenetic tree from huge SNP data

*homepage*: <http://chibba.pgml.uga.edu/snphylo/>

version     |versionsuffix             |toolchain
------------|--------------------------|-------------------------------
``20140701``|                          |``foss/2016a``, ``intel/2016a``
``20160204``|``-Python-2.7.14-R-3.4.3``|``foss/2017b``, ``intel/2017b``

### SNPomatic

High throughput sequencing technologies generate large amounts of short reads.  Mapping these to a reference sequence consumes large amounts of processing  time and memory, and read mapping errors can lead to noisy or incorrect  alignments. SNP-o-matic is a fast, memory-efficient, and stringent read  mapping tool offering a variety of analytical output functions, with an  emphasis on genotyping. 

*homepage*: <https://github.com/magnusmanske/snpomatic>

version|toolchain
-------|-----------------
``1.0``|``GCCcore/9.3.0``

### SOAPaligner

SOAPaligner/soap2 is a member of the SOAP (Short Oligonucleotide   Analysis Package). It is an updated version of SOAP software for short oligonucleotide   alignment. 

*homepage*: <http://soap.genomics.org.cn/soapaligner.html>

version |versionsuffix    |toolchain
--------|-----------------|----------
``2.21``|``_Linux-x86_64``|``system``

### SOAPdenovo-Trans

SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. 

*homepage*: <http://soap.genomics.org.cn/SOAPdenovo-Trans.html>

version  |toolchain
---------|---------------
``1.0.4``|``intel/2017a``

### SOAPdenovo2

SOAPdenovo is a novel short-read assembly method that can build a   de novo draft assembly for human-sized genomes. The program is specially designed to   assemble Illumina short reads. It creates new opportunities for building reference   sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.   SOAPdenovo2 is the successor of SOAPdenovo.

*homepage*: <http://soap.genomics.org.cn/index.html>

version |toolchain
--------|-------------------------------------------------------------------------------------------
``r240``|``GCC/5.4.0-2.26``
``r241``|``GCC/6.4.0-2.28``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``

### SOAPfuse

SOAPfuse is an open source tool developed for genome-wide detection of fusion  transcripts from paired-end RNA-Seq data.

*homepage*: <https://sourceforge.net/p/soapfuse/wiki/Home/>

version |versionsuffix   |toolchain
--------|----------------|--------------
``1.27``|``-Perl-5.24.0``|``foss/2016b``
``1.27``|``-Perl-5.28.0``|``foss/2018b``

### socat

socat is a relay for bidirectional data transfer between two independent data channels.

*homepage*: <http://www.dest-unreach.org/socat>

version    |toolchain
-----------|-----------------
``1.7.3.3``|``GCCcore/8.2.0``

### SOCI

SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the  regular C++ code, staying entirely within the Standard C++.

*homepage*: <http://soci.sourceforge.net/>

version  |toolchain
---------|----------------------------------------------
``4.0.1``|``GCC/10.2.0``, ``GCCcore/9.3.0``
``4.0.2``|``GCC/10.3.0``
``4.0.3``|``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### SolexaQA++

SolexaQA calculates sequence quality statistics and creates visual representations  of data quality for second-generation sequencing data. Originally developed for the Illumina system  (historically known as “Solexa”), SolexaQA now also supports Ion Torrent and 454 data.

*homepage*: <http://solexaqa.sourceforge.net>

version  |toolchain
---------|--------------
``3.1.5``|``foss/2016b``

### sonic

Sonic is a simple algorithm for speeding up or slowing down speech.  However, it's optimized for speed ups of over 2X, unlike previous algorithms for changing speech rate.  The Sonic library is a very simple ANSI C library that is designed to easily be integrated into streaming voice applications, like TTS back ends. 

*homepage*: <https://github.com/espeak-ng/sonic>

version     |toolchain
------------|---------------
``20180202``|``gompi/2020a``

### SoPlex

SoPlex is an optimization package for solving linear programming problems (LPs) based on an advanced implementation of the primal and dual revised simplex algorithm. It provides special support for the exact solution of LPs with rational input data. It can be used as a standalone solver reading MPS or LP format files via a command line interface as well as embedded into other programs via a C++ class library.

*homepage*: <https://soplex.zib.de/>

version  |toolchain
---------|--------------
``2.2.1``|``GCC/11.3.0``

### SoQt

SoQt is a library which provides the glue between Systems in  Motion's Coin high-level 3D visualization library and the Qt 2D user interface  library. 

*homepage*: <https://coin3d.github.io/SoQt/html/>

version  |toolchain
---------|------------------------------
``1.6.0``|``GCC/10.3.0``, ``GCC/11.2.0``

### SortMeRNA

SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.

*homepage*: <http://bioinfo.lifl.fr/RNA/sortmerna/>

version|toolchain
-------|-----------------------------
``2.1``|``GCC/9.3.0``, ``foss/2016a``

### SoupX

" Quantify, profile and remove ambient mRNA contamination (the "soup") from droplet based single cell RNA-seq experiments. Implements the method described in Young et al. (2018) <doi:10.1101/303727>.

*homepage*: <https://github.com/constantAmateur/SoupX>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.6.2``|``-R-4.2.1`` |``foss/2022a``

### SoX

Sound eXchange, the Swiss Army knife of audio manipulation

*homepage*: <http://sox.sourceforge.net/>

version   |toolchain
----------|---------------------------------
``14.4.2``|``GCC/8.3.0``, ``GCCcore/11.3.0``

### SoXt

SoXt is an Xt/Motif glue library for Coin. It can also be used  on top of the SGI or TGS implementation of Open Inventor, and is designed to  be source code compatible with SGI's InventorXt library. 

*homepage*: <https://coin3d.github.io/SoXt/html/>

version  |toolchain
---------|--------------
``1.4.0``|``GCC/11.2.0``

### SpaceRanger

Space Ranger is a set of analysis pipelines that process Visium spatial RNA-seq output and brightfield microscope images in order to detect tissue, align reads, generate feature-spot matrices, perform clustering and gene expression analysis, and place spots in spatial context on the slide image. 

*homepage*: <https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger>

version  |toolchain
---------|--------------
``1.1.0``|``GCC/9.3.0``
``1.2.2``|``GCC/9.3.0``
``1.3.0``|``GCC/10.3.0``
``1.3.1``|``GCC/11.2.0``
``2.0.0``|``GCC/11.2.0``
``2.0.1``|``GCC/11.3.0``

### Spack

Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific  software simple. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,  and all of these builds can coexist on the same machine.

*homepage*: <https://spack.io/>

version   |toolchain
----------|----------
``0.10.0``|``system``
``0.11.2``|``system``
``0.12.1``|``system``
``0.16.2``|``system``
``0.17.0``|``system``
``0.17.2``|``system``

### spaCy

Industrial-strength Natural Language Processing (NLP) in Python.

*homepage*: <https://spacy.io/>

version  |toolchain
---------|--------------
``3.4.4``|``foss/2022a``

### SPAdes

Genome assembler for single-cell and isolates data sets

*homepage*: <http://bioinf.spbau.ru/en/spades>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------
``3.9.0`` |                  |``foss/2016a``, ``foss/2016b``
``3.10.1``|                  |``foss/2016b``, ``foss/2017a``
``3.11.1``|                  |``foss/2017b``, ``foss/2018a``
``3.12.0``|                  |``foss/2016b``, ``foss/2018a``, ``foss/2018b``
``3.13.0``|                  |``GCC/10.3.0``, ``foss/2018b``
``3.13.1``|                  |``GCC/8.2.0-2.31.1``
``3.14.0``|``-Python-3.7.2`` |``GCC/8.2.0-2.31.1``
``3.14.0``|``-Python-3.7.4`` |``GCC/8.3.0``
``3.14.1``|``-Python-3.8.2`` |``GCC/9.3.0``
``3.15.2``|                  |``GCC/10.2.0``
``3.15.2``|``-Python-2.7.18``|``GCC/10.2.0``
``3.15.3``|                  |``GCC/10.3.0``, ``GCC/11.2.0``
``3.15.4``|                  |``GCC/12.2.0``
``3.15.5``|                  |``GCC/11.3.0``

### spaln

Spaln (space-efficient spliced alignment) is a stand-alone program that maps  and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.

*homepage*: <https://github.com/ogotoh/spaln>

version    |toolchain
-----------|----------------------------------------
``2.3.3c`` |``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.4.03`` |``iccifort/2019.5.281``
``2.4.12`` |``GCC/10.2.0``, ``GCC/11.2.0``
``2.4.13f``|``GCC/11.3.0``

### Spark

Spark is Hadoop MapReduce done in memory

*homepage*: <https://spark.apache.org>

version  |versionsuffix                              |toolchain
---------|-------------------------------------------|----------------------------------
``1.3.0``|                                           |``system``
``1.4.1``|                                           |``system``
``1.5.0``|                                           |``system``
``1.6.0``|                                           |``system``
``1.6.1``|                                           |``system``
``2.0.0``|                                           |``system``
``2.0.2``|                                           |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_144``             |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_152``             |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_152-Python-3.6.3``|``intel/2017b``
``2.3.0``|``-Hadoop-2.7-Java-1.8.0_162``             |``system``
``2.4.0``|``-Hadoop-2.7-Java-1.8``                   |``system``
``2.4.0``|``-Hadoop-2.7-Java-1.8-Python-3.6.6``      |``intel/2018b``
``2.4.0``|``-Python-2.7.15``                         |``foss/2018b``, ``intel/2018b``
``2.4.0``|``-Python-3.6.6``                          |``intel/2018b``
``2.4.5``|``-Python-3.7.4-Java-1.8``                 |``intel/2019b``
``3.0.0``|``-Python-2.7.15``                         |``foss/2018b``, ``intel/2018b``
``3.1.1``|                                           |``foss/2020b``, ``fosscuda/2020b``
``3.1.1``|``-Python-3.8.2``                          |``foss/2020a``
``3.2.1``|                                           |``foss/2021b``
``3.3.1``|                                           |``foss/2022a``

### sparse-neighbors-search

A Python/C++ implementation of an approximate nearest neighbor search for sparse data structures  based on the idea of local sensitive hash functions.

*homepage*: <https://github.com/joachimwolff/sparse-neighbors-search>

version|toolchain
-------|--------------
``0.7``|``foss/2022a``

### sparsehash

An extremely memory-efficient hash_map implementation. 2 bits/entry overhead!  The SparseHash library contains several hash-map implementations, including  implementations that optimize for space or speed. 

*homepage*: <https://github.com/sparsehash/sparsehash>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------
``2.0.2``|``foss/2016a``
``2.0.3``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``, ``intel/2017a``
``2.0.4``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### SpatialDE

SpatialDE is a method to identify genes which significantly depend on spatial coordinates in  non-linear and non-parametric ways. The intended applications are spatially resolved RNA-sequencing from e.g.  Spatial Transcriptomics, or in situ gene expression measurements from e.g. SeqFISH or MERFISH.

*homepage*: <https://pypi.org/project/SpatialDE>

version  |toolchain
---------|--------------
``1.1.3``|``foss/2022a``

### spatialreg

A collection of all the estimation functions for spatial cross-sectional models (on lattice/areal  data using spatial weights matrices) contained up to now in 'spdep', 'sphet' and 'spse'.

*homepage*: <https://cran.r-project.org/package=spatialreg>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.1-5``|``-R-3.6.2`` |``foss/2019b``
``1.1-8``|``-R-4.1.0`` |``foss/2021a``

### spdlog

Very fast, header-only/compiled, C++ logging library.

*homepage*: <https://github.com/gabime/spdlog>

version   |toolchain
----------|--------------------------------------
``1.9.2`` |``GCCcore/10.3.0``
``1.11.0``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### SpectrA

Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++  library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library.

*homepage*: <https://spectralib.org/>

version  |toolchain
---------|--------------------------------------
``1.0.0``|``GCC/10.2.0``
``1.0.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### spectral.methods

Contains some implementations of Singular Spectrum Analysis (SSA) for the gapfilling and  spectral decomposition of time series.

*homepage*: <https://cran.r-project.org/web/packages/spectral.methods>

version      |versionsuffix|toolchain
-------------|-------------|---------------
``0.7.2.133``|``-R-3.4.3`` |``intel/2017b``
``0.7.2.133``|``-R-3.4.4`` |``intel/2018a``

### speech_tools

The Edinburgh Speech Tools Library is a collection of C++ class, functions and related programs for manipulating the sorts of objects used in speech processing. It includes support for reading and writing waveforms, parameter files (LPC, Ceptra, F0) in various formats and converting between them. It also includes support for linguistic type objects and support for various label files and ngrams (with smoothing). 

*homepage*: <['http://festvox.org/festival/']>

version  |toolchain
---------|-----------------
``2.5.0``|``GCCcore/9.3.0``

### spektral

Spektral is a Python library for graph deep learning. The main goal of this  project is to provide a simple but flexible framework for creating graph neural networks (GNNs).

*homepage*: <https://github.com/danielegrattarola/spektral>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.1.0``|``-CUDA-11.4.1``|``foss/2021b``

### spglib

Spglib is a C library for finding and handling crystal symmetries.

*homepage*: <https://spglib.github.io/spglib/>

version   |toolchain
----------|-------------------------
``1.9.2`` |``intel/2016.02-GCC-4.9``
``1.9.9`` |``intel/2017b``
``1.16.1``|``GCCcore/10.2.0``
``1.16.2``|``GCCcore/10.3.0``
``2.0.2`` |``GCCcore/11.3.0``

### spglib-python

Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.

*homepage*: <https://pypi.python.org/pypi/spglib>

version         |versionsuffix     |toolchain
----------------|------------------|--------------------------------------------------------
``1.9.4.2``     |``-Python-2.7.12``|``intel/2016b``
``1.9.5``       |``-Python-2.7.12``|``intel/2016b``
``1.9.9.38``    |``-Python-2.7.13``|``intel/2017a``
``1.10.0.2``    |``-Python-2.7.14``|``intel/2017b``
``1.10.0.2``    |``-Python-3.6.3`` |``intel/2017b``
``1.14.1.post0``|``-Python-3.7.2`` |``intel/2019a``
``1.16.0``      |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.16.0``      |``-Python-3.7.4`` |``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``1.16.0``      |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.16.1``      |                  |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``1.16.3``      |                  |``foss/2021b``, ``intel/2021b``
``2.0.0``       |                  |``foss/2022a``, ``intel/2022a``
``2.0.2``       |                  |``gfbf/2022b``

### Sphinx

Sphinx is a tool that makes it simple to create intelligent and beautiful documentation.  It was originally created for the new Python documentation, and it has excellent facilities for the   documentation of Python projects, but C/C++ is already supported as well, and it is planned to add   special support for other languages as well.

*homepage*: <http://sphinx.pocoo.org/>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.4.8``|``-Python-2.7.11``|``foss/2016a``
``1.4.8``|``-Python-3.5.1`` |``foss/2016a``
``1.8.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.8.1``|``-Python-2.7.15``|``foss/2018b``
``1.8.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.8.1``|``-Python-3.6.4`` |``foss/2018a``
``1.8.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.8.3``|``-Python-3.6.4`` |``intel/2018a``

### Sphinx-RTD-Theme

Sphinx theme designed to provide a great reader experience  for documentation users on both desktop and mobile devices.

*homepage*: <https://sphinx-rtd-theme.readthedocs.io>

version  |toolchain
---------|------------------
``1.1.1``|``GCCcore/10.2.0``

### SpiceyPy

SpiceyPy is a Python wrapper for the NAIF C SPICE Toolkit (N65)

*homepage*: <https://github.com/AndrewAnnex/SpiceyPy>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.0``|``-Python-2.7.12``|``intel/2016b``
``1.1.1``|``-Python-3.6.1`` |``intel/2017a``
``2.1.0``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0``|``-Python-3.6.4`` |``foss/2018a``

### SpiecEasi

Sparse InversE Covariance estimation for Ecological Association and Statistical Inference

*homepage*: <https://github.com/zdk123/SpiecEasi>

version     |versionsuffix|toolchain
------------|-------------|---------------
``1.0.0``   |``-R-3.4.4`` |``intel/2018a``
``20160830``|``-R-3.3.1`` |``intel/2016b``

### SplAdder

Splicing Adder is a toolbox for alternative splicing analysis based on RNA-Seq alignment data. Briefly, the software takes a given annotation and RNA-Seq read alignments in standardized formats, transforms the annotation into a splicing graph representation, augments the splicing graph with additional information extracted from the read data, extracts alternative splicing events from the graph and quantifies the events based on the alignment data.

*homepage*: <https://github.com/ratschlab/spladder>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.4.2``|``-Python-3.8.2``|``foss/2020a``

### SPLASH

SPLASH is a free and open source visualisation tool  for Smoothed Particle Hydrodynamics (SPH) simulations.

*homepage*: <http://users.monash.edu.au/~dprice/splash/index.html>

version  |toolchain
---------|--------------
``2.8.0``|``foss/2018b``

### SpliceMap

SpliceMap is a de novo splice junction discovery and alignment tool.  It offers high sensitivity and support for arbitrary RNA-seq read lengths.

*homepage*: <https://web.stanford.edu/group/wonglab/SpliceMap/>

version    |toolchain
-----------|------------------
``3.3.5.2``|``GCC/7.3.0-2.30``

### split-seq

Analysis tools for split-seq.

*homepage*: <https://github.com/yjzhang/split-seq-pipeline>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20190717``|``-Python-3.6.6``|``foss/2018b``

### splitRef

splitRef splits a reference haplotype file into smaller files   with subsets of markers. The current version is a pre-release. 

*homepage*: <http://www.sph.umich.edu/csg/yli/splitRef/index.html>

version  |toolchain
---------|----------
``0.0.2``|``system``

### SPM

SPM (Statistical Parametric Mapping) refers to the construction  and assessment of spatially extended statistical processes used to test hypo-  theses about functional imaging data. These ideas have been instantiated in  software that is called SPM.  The SPM software package has been designed for the analysis of brain imaging  data sequences. The sequences can be a series of images from different cohorts,  or time-series from the same subject. The current release is designed for the  analysis of fMRI, PET, SPECT, EEG and MEG. 

*homepage*: <https://www.fil.ion.ucl.ac.uk/spm>

version       |versionsuffix    |toolchain
--------------|-----------------|----------
``12.5_r7771``|``-MATLAB-2021a``|``system``
``12.5_r7771``|``-MATLAB-2021b``|``system``

### spoa

Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm  which is used to generate consensus sequences

*homepage*: <https://github.com/rvaser/spoa>

version  |toolchain
---------|------------------------------------------------------------------------------
``3.0.1``|``GCC/7.3.0-2.30``
``3.4.0``|``GCC/10.2.0``
``4.0.0``|``GCC/8.3.0``
``4.0.7``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``

### SPOOLES

SPOOLES is a library for solving sparse real and complex linear  systems of equations, written in the C language using object oriented design. 

*homepage*: <https://netlib.org/linalg/spooles/spooles.2.2.html>

version|toolchain
-------|---------------
``2.2``|``gompi/2021a``

### SPOTPY

SPOTPY is a Python framework that enables the use of Computational optimization techniques for calibration, uncertainty and sensitivity analysis techniques of almost every (environmental-) model.

*homepage*: <https://spotpy.readthedocs.io>

version   |toolchain
----------|---------------
``1.5.14``|``intel/2021b``

### SPRNG

Scalable Parallel Pseudo Random Number Generators Library

*homepage*: <http://www.sprng.org/>

version |toolchain
--------|--------------
``2.0b``|``foss/2016a``

### Spyder

Spyder is an interactive Python development environment providing MATLAB-like features in a simple  and light-weighted software.

*homepage*: <https://github.com/spyder-ide/spyder>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.1.4``|``-Python-2.7.13``|``intel/2017a``
``3.3.1``|``-Python-3.6.4`` |``foss/2018a``
``3.3.2``|``-Python-3.6.6`` |``foss/2018b``
``4.1.5``|``-Python-3.7.2`` |``foss/2019a``
``4.1.5``|``-Python-3.8.2`` |``foss/2020a``

### SQLite

SQLite: SQL Database Engine in a C Library

*homepage*: <http://www.sqlite.org/>

version     |toolchain
------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.8.8.1`` |``GCC/4.8.4``, ``GCC/4.9.2``
``3.8.10.2``|``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``
``3.9.2``   |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``3.13.0``  |``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``3.14.1``  |``GCCcore/4.9.3``
``3.17.0``  |``GCCcore/6.3.0``
``3.20.1``  |``GCCcore/6.4.0``
``3.21.0``  |``GCCcore/6.4.0``
``3.23.0``  |``GCCcore/6.4.0``
``3.24.0``  |``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``3.26.0``  |``GCCcore/8.2.0``
``3.27.2``  |``GCCcore/8.2.0``
``3.29.0``  |``GCCcore/8.3.0``
``3.31.1``  |``GCCcore/9.3.0``
``3.33.0``  |``GCCcore/10.2.0``
``3.35.4``  |``GCCcore/10.3.0``
``3.36``    |``GCCcore/11.2.0``
``3.38.3``  |``GCCcore/11.3.0``
``3.39.4``  |``GCCcore/12.2.0``
``3.41.2``  |``GCCcore/13.1.0``
``3.42.0``  |``GCCcore/12.3.0``

### SqueezeMeta

SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,  covering all steps of the analysis.

*homepage*: <https://github.com/jtamames/SqueezeMeta>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.4.3``|``-Python-2.7.15``|``foss/2018b``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.5.0``|                  |``foss/2021b``

### Squidpy

Squidpy is a tool for the analysis and visualization of spatial molecular data.

*homepage*: <https://squidpy.readthedocs.io>

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021b``

### SRA-Toolkit

The SRA Toolkit, and the source-code SRA System Development  Kit (SDK), will allow you to programmatically access data housed within SRA  and convert it from the SRA format

*homepage*: <https://github.com/ncbi/sra-tools>

version    |versionsuffix      |toolchain
-----------|-------------------|---------------
``2.3.5``  |``-centos_linux64``|``system``
``2.5.4-1``|``-centos_linux64``|``system``
``2.5.7``  |``-centos_linux64``|``system``
``2.8.2-1``|``-centos_linux64``|``system``
``2.9.0``  |``-centos_linux64``|``system``
``2.9.2``  |``-ubuntu64``      |``system``
``2.9.4``  |``-centos_linux64``|``system``
``2.9.6-1``|``-centos_linux64``|``system``
``2.10.4`` |                   |``gompi/2019b``
``2.10.5`` |``-centos_linux64``|``system``
``2.10.8`` |                   |``gompi/2020a``
``2.10.9`` |                   |``gompi/2020b``
``3.0.0``  |                   |``gompi/2021b``
``3.0.0``  |``-centos_linux64``|``system``
``3.0.3``  |                   |``gompi/2022a``
``3.0.5``  |                   |``gompi/2021a``

### sradownloader

SRAdownloader takes the annotation table from the SRA run selector tool  and retrieves the raw fastq files for the selected samples

*homepage*: <https://github.com/s-andrews/sradownloader>

version|toolchain
-------|------------------
``3.9``|``GCCcore/11.3.0``

### SRPRISM

Single Read Paired Read Indel Substitution Minimizer

*homepage*: <https://github.com/ncbi/SRPRISM>

version  |versionsuffix|toolchain
---------|-------------|------------------
``3.0.0``|             |``foss/2018b``
``3.1.1``|``-Java-11`` |``GCCcore/8.2.0``
``3.1.2``|             |``GCCcore/10.2.0``

### SRST2

Short Read Sequence Typing for Bacterial Pathogens

*homepage*: <https://katholt.github.io/srst2>

version           |versionsuffix     |toolchain
------------------|------------------|--------------
``0.2.0-20210620``|``-Python-2.7.18``|``foss/2020b``

### SSAHA2

SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a   pairwise sequence alignment program designed for the efficient mapping of sequencing   reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms   (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.)   are supported. A pile-up pipeline for analysis and genotype calling is available as   a separate package. 

*homepage*: <http://www.sanger.ac.uk/resources/software/ssaha2/>

version  |versionsuffix|toolchain
---------|-------------|----------
``2.5.5``|``-i686``    |``system``
``2.5.5``|``-x86_64``  |``system``

### SSN

Spatial statistical modeling and prediction for data on stream networks, including models based on in-stream distance. Models are created using moving average constructions. Spatial linear models, including explanatory variables, can be fit with (restricted) maximum likelihood. Mapping and other graphical functions are included.

*homepage*: <https://cran.r-project.org/web/packages/SSN>

version   |versionsuffix|toolchain
----------|-------------|--------------
``1.1.14``|``-R-3.6.0`` |``foss/2019a``

### SSPACE_Basic

SSPACE Basic, SSAKE-based Scaffolding of Pre-Assembled Contigs after Extension

*homepage*: <https://github.com/nsoranzo/sspace_basic>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.1``|``-Perl-5.24.1``  |``intel/2017a``
``2.1.1``|``-Perl-5.26.0``  |``intel/2017b``
``2.1.1``|``-Python-2.7.18``|``GCC/10.2.0``

### SSW

SSW is a fast implementation of the Smith-Waterman algorithm, which uses the  Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the  instruction level. SSW library provides an API that can be flexibly used by programs written in  C, C++ and other languages. We also provide a software that can do protein and genome alignment  directly. Current version of our implementation is ~50 times faster than an ordinary  Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path  (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and  location heuristically. 

*homepage*: <https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library>

version  |toolchain
---------|-----------------------------------------------------------------------------
``1.1``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``1.2.4``|``GCCcore/10.3.0``

### STACEY

The BEAST2 package STACEY can be used for species delimitation and species tree estimation, based on the multispecies coalescent model.

*homepage*: <http://www.indriid.com/software.html>

version  |toolchain
---------|--------------
``1.2.5``|``GCC/10.2.0``

### Stack

Stack is a cross-platform program for developing Haskell projects. It is intended for Haskellers both new and experienced.

*homepage*: <https://docs.haskellstack.org>

version  |versionsuffix|toolchain
---------|-------------|----------
``2.3.3``|``-x86_64``  |``system``

### Stacks

Stacks is a software pipeline for building loci from short-read sequences, such as those generated on  the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,  for the purpose of building genetic maps and conducting population genomics and phylogeography. 

*homepage*: <https://catchenlab.life.illinois.edu/stacks/>

version       |toolchain
--------------|------------------------------------------------------------------------------
``1.40``      |``foss/2016a``
``1.42``      |``foss/2016a``
``1.44``      |``foss/2016a``
``1.45``      |``foss/2016a``
``1.46``      |``intel/2017a``
``1.47``      |``foss/2016a``
``1.48``      |``intel/2017b``, ``intel/2018b``
``2.0``       |``foss/2018a``, ``intel/2018a``
``2.0Beta7c`` |``intel/2017b``
``2.0Beta8c`` |``intel/2017b``
``2.0Beta9``  |``intel/2018a``
``2.0Beta10a``|``foss/2018a``
``2.2``       |``foss/2018a``
``2.3b``      |``foss/2018a``
``2.3e``      |``foss/2018b``
``2.5``       |``iccifort/2019.5.281``
``2.41``      |``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.53``      |``foss/2019b``, ``iccifort/2019.5.281``
``2.54``      |``foss/2020a``
``2.62``      |``foss/2022a``

### STAMP

STAMP is a tool for characterizing similarities between transcription factor binding motifs

*homepage*: <http://www.benoslab.pitt.edu/stamp/>

version|toolchain
-------|--------------------------------
``1.2``|``intel/2016a``
``1.3``|``intel/2016a``, ``intel/2016b``

### StaMPS

A software package to extract ground displacements from time series of synthetic aperture radar  (SAR) acquisitions.

*homepage*: <http://homepages.see.leeds.ac.uk/~earahoo/stamps/>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.3b1``|``-Perl-5.24.1``|``intel/2017a``

### Stampy

Stampy is a package for the mapping of short reads from illumina sequencing machines onto  a reference genome.

*homepage*: <http://www.well.ox.ac.uk/stampy>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.0.31``|``-Python-2.7.12``|``intel/2016b``
``1.0.32``|``-Python-2.7.14``|``intel/2018a``

### STAR

STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.

*homepage*: <https://github.com/alexdobin/STAR>

version                 |toolchain
------------------------|--------------------------------------------------------------
``2.4.2a``              |``foss/2018b``
``2.5.0a``              |``GNU/4.9.3-2.25``
``2.5.2a``              |``foss/2016a``
``2.5.2b``              |``intel/2016b``
``2.5.3a``              |``GCC/8.3.0``, ``GCC/9.3.0``, ``intel/2017a``, ``intel/2017b``
``2.5.4b``              |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``2.6.0c``              |``foss/2018a``, ``intel/2018a``
``2.6.1c``              |``foss/2018b``
``2.7.0d``              |``foss/2018b``
``2.7.0f``              |``GCC/8.2.0-2.31.1``, ``foss/2018b``
``2.7.1a``              |``GCC/8.2.0-2.31.1``, ``foss/2018b``
``2.7.2b``              |``GCC/8.3.0``
``2.7.3a``              |``GCC/8.3.0``, ``GCC/9.3.0``
``2.7.4a``              |``GCC/9.3.0``
``2.7.5b``              |``GCC/9.3.0``
``2.7.6a``              |``GCC/10.2.0``, ``GCC/9.3.0``
``2.7.7a``              |``GCC/10.2.0``, ``GCC/9.3.0``
``2.7.8a``              |``GCC/10.2.0``
``2.7.9a``              |``GCC/10.3.0``, ``GCC/11.2.0``
``2.7.10a_alpha_220601``|``GCC/10.3.0``
``2.7.10b``             |``GCC/11.3.0``, ``GCC/12.2.0``

### STAR-CCM+

STAR-CCM+ is a multiphysics computational fluid dynamics (CFD) software for the simulation of products operating under real-world conditions.

*homepage*: <https://mdx.plm.automation.siemens.com/star-ccm-plus>

version      |versionsuffix|toolchain
-------------|-------------|----------
``13.04.011``|             |``system``
``17.02.008``|             |``system``
``17.02.008``|``-r8``      |``system``
``17.04.008``|             |``system``
``17.04.008``|``-r8``      |``system``
``17.06.007``|             |``system``
``17.06.007``|``-r8``      |``system``
``18.02.008``|             |``system``
``18.02.008``|``-r8``      |``system``

### STAR-Fusion

STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts  supported by Illumina reads. STAR-Fusion further processes the output generated  by the STAR aligner to map junction reads and spanning reads to a reference  annotation set.

*homepage*: <https://github.com/STAR-Fusion/STAR-Fusion>

version  |versionsuffix                |toolchain
---------|-----------------------------|--------------------
``1.5.0``|``-Perl-5.28.0``             |``foss/2018b``
``1.6.0``|``-Perl-5.28.1-Python-3.7.2``|``GCC/8.2.0-2.31.1``
``1.8.1``|``-Python-3.7.4``            |``foss/2019b``

### stardist

Object Detection with Star-convex Shapes.

*homepage*: <https://github.com/stardist/stardist>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.8.3``|                |``foss/2021b``
``0.8.3``|``-CUDA-11.4.1``|``foss/2021b``

### stars

Reading, manipulating, writing and plotting spatiotemporal arrays (raster and vector data cubes)  in R, using GDAL bindings provided by sf, and NetCDF bindings by ncmeta and RNetCDF.

*homepage*: <https://cran.r-project.org/package=stars>

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.4-3``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

### Stata

Stata is a complete, integrated statistical software package that provides everything you need  for data analysis, data management, and graphics.

*homepage*: <https://www.stata.com/>

version|versionsuffix|toolchain
-------|-------------|----------
``15`` |             |``system``
``16`` |``-legacy``  |``system``
``17`` |             |``system``

### Statistics-R

Perl interface with the R statistical program

*homepage*: <https://metacpan.org/pod/Statistics::R>

version |toolchain
--------|--------------
``0.34``|``foss/2020a``

### statsmodels

Statsmodels is a Python module that provides classes and functions  for the estimation of many different statistical models, as well as for conducting   statistical tests, and statistical data exploration.

*homepage*: <https://www.statsmodels.org>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.6.1`` |``-Python-2.7.13``|``foss/2017a``
``0.6.1`` |``-Python-3.5.1`` |``intel/2016a``
``0.6.1`` |``-Python-3.5.2`` |``intel/2016b``
``0.8.0`` |``-Python-2.7.13``|``intel/2017a``
``0.9.0`` |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.9.0`` |``-Python-2.7.16``|``intel/2019b``
``0.9.0`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.10.1``|                  |``foss/2019a``
``0.11.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.11.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.12.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.12.2``|                  |``foss/2021a``
``0.13.1``|                  |``foss/2021b``, ``foss/2022a``, ``intel/2021b``
``0.14.0``|                  |``gfbf/2022b``

### STEAK

Detects integrations of any sort in high-throughput  sequencing (HTS) data. STEAK was built for validating and discovering  transposable element (TE) and retroviral integrations in a variety of  HTS data. The software performs on both single-end (SE) and paired-end ( PE) libraries and on a variety of HTS sequencing strategies. It can  be applied to a broad range of research interests and clinical uses  such as population genetic studies and detecting polymorphic integrations.

*homepage*: <https://omictools.com/steak-tool>

version       |versionsuffix     |toolchain
--------------|------------------|--------------
``2019.09.12``|                  |``foss/2021b``
``2019.09.12``|``-Python-2.7.16``|``foss/2019b``

### STIR

Software for Tomographic Image Reconstruction

*homepage*: <http://stir.sourceforge.net/>

version|toolchain
-------|---------------
``3.0``|``intel/2018a``

### stpipeline

The ST Pipeline contains the tools and scripts needed to process  and analyze the raw files generated with the Spatial Transcriptomics method  in FASTQ format to generated datasets for down-stream analysis. The ST  pipeline can also be used to process single cell data as long as a file  with barcodes identifying each cell is provided. The ST Pipeline can also  process RNA-Seq datasets generated with or without UMIs.

*homepage*: <https://github.com/SpatialTranscriptomicsResearch/st_pipeline>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.7.3``|``-Python-2.7.15``|``foss/2018b``
``1.7.6``|``-Python-3.7.2`` |``foss/2019a``
``1.7.6``|``-Python-3.7.4`` |``foss/2019b``

### strace

strace is a diagnostic, debugging and instructional userspace utility for Linux. It is used to monitor and tamper with interactions between processes and the Linux kernel, which include system calls, signal deliveries, and changes of process state. 

*homepage*: <https://strace.io/>

version |toolchain
--------|------------------
``5.14``|``GCCcore/11.2.0``

### Strainberry

Strainberry is a method that performs strain separation in low-complexity metagenomes using error-prone long-read technologies. It exploits state-of-the-art tools for variant calling, haplotype phasing, and genome assembly, in order to achieve single-sample assembly of strains with higher quality than other state-of-the-art long-read assemblers.

*homepage*: <https://github.com/rvicedomini/strainberry>

version|toolchain
-------|--------------
``1.1``|``foss/2022a``

### STREAM

The STREAM benchmark is a simple synthetic benchmark program that measures sustainable  memory bandwidth (in MB/s) and the corresponding computation rate for simple vector kernels.

*homepage*: <https://www.cs.virginia.edu/stream/>

version |toolchain
--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.10``|``GCC/11.3.0``, ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``intel-compilers/2022.2.1``, ``intel/2016b``, ``intel/2018b``

### strelka

Strelka2 is a fast and accurate small variant caller  optimized for analysis of germline variation in small cohorts and  somatic variation in tumor/normal sample pairs.

*homepage*: <https://github.com/Illumina/strelka>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.9.7`` |                  |``intel/2018a``
``2.9.9`` |                  |``foss/2018b``
``2.9.10``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``

### StringTie

StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts

*homepage*: <https://ccb.jhu.edu/software/stringtie/>

version   |toolchain
----------|----------------------------------
``1.3.0`` |``intel/2016b``
``1.3.3`` |``GCCcore/6.4.0``, ``intel/2017a``
``1.3.3b``|``foss/2016b``
``1.3.5`` |``GCCcore/8.2.0``, ``foss/2018b``
``2.0.3`` |``GCCcore/7.3.0``
``2.1.0`` |``foss/2018b``
``2.1.1`` |``GCC/8.3.0``
``2.1.3`` |``GCC/8.3.0``, ``GCC/9.3.0``
``2.1.4`` |``GCC/8.3.0``, ``GCC/9.3.0``
``2.1.7`` |``GCC/10.3.0``
``2.2.1`` |``GCC/11.2.0``

### stripy

A Python interface to TRIPACK and STRIPACK Fortran code for (constrained) triangulation in Cartesian  coordinates and on a sphere. Stripy is an object-oriented package and includes routines from SRFPACK and SSRFPACK for  interpolation (nearest neighbor, linear and hermite cubic) and to evaluate derivatives (Renka 1996a,b and 1997a,b).

*homepage*: <https://github.com/underworldcode/stripy>

version  |toolchain
---------|--------------
``2.1.0``|``foss/2021a``

### STRique

STRique is a python package to analyze repeat expansion and methylation states of short tandem repeats (STR) in Oxford Nanopore Technology (ONT) long read sequencing data.

*homepage*: <https://github.com/giesselmann/STRique>

version  |toolchain
---------|--------------
``0.4.2``|``foss/2021b``

### Structure

The program structure is a free software package for using multi-locus genotype data to investigate  population structure.

*homepage*: <https://web.stanford.edu/group/pritchardlab/structure.html>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------
``2.3.4``|``GCC/12.2.0``, ``GCC/8.2.0-2.31.1``, ``iccifort/2019.3.199-GCC-8.3.0-2.32``, ``iccifort/2019.5.281``

### Structure_threader

A program to parallelize the runs of Structure,  fastStructure, MavericK and ALStructure software.

*homepage*: <https://github.com/StuntsPT/Structure_threader>

version   |toolchain
----------|--------------
``1.3.10``|``foss/2022b``

### STRUMPACK

STRUMPACK - STRUctured Matrix PACKage - Fast linear solvers and preconditioner  for both dense and sparse systems using low-rank structured factorization with randomized sampling.

*homepage*: <https://fastmath-scidac.llnl.gov/software/strumpack.html>

version  |toolchain
---------|-------------------------------
``6.1.0``|``foss/2020b``, ``intel/2020b``

### suave

suave is an interactive web application to visualize read depth ratios between two samples and the structural variants of one of the samples (typically the "case" sample in a case/control setup such as tumor/normal comparison).

*homepage*: <https://github.com/dellytools/suave>

version     |toolchain
------------|--------------
``20160529``|``foss/2020b``

### Subread

High performance read alignment, quantification and mutation discovery

*homepage*: <https://subread.sourceforge.net/>

version     |toolchain
------------|---------------------------------------------
``1.5.0-p1``|``foss/2016a``, ``foss/2016b``
``1.6.3``   |``foss/2018b``
``1.6.4``   |``foss/2018b``
``2.0.0``   |``GCC/7.3.0-2.30``, ``GCC/8.3.0``
``2.0.2``   |``GCC/10.2.0``
``2.0.3``   |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``
``2.0.4``   |``GCC/11.3.0``

### subset-bam

subset-bam is a tool to subset a 10x Genomics BAM file based on a tag,  most commonly the cell barcode tag.

*homepage*: <https://github.com/10XGenomics/subset-bam>

version  |toolchain
---------|------------------
``1.1.0``|``GCCcore/10.3.0``

### Subversion

Subversion is an open source version control system.

*homepage*: <http://subversion.apache.org/>

version   |toolchain
----------|--------------------------------------
``1.9.7`` |``iomkl/2018a``
``1.9.9`` |``GCCcore/7.3.0``
``1.10.0``|``foss/2017b``, ``intel/2017b``
``1.12.0``|``GCCcore/8.2.0``
``1.14.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.14.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.14.2``|``GCCcore/11.3.0``

### suds

Lightweight SOAP client

*homepage*: <https://pypi.python.org/pypi/suds-py3>

version    |versionsuffix    |toolchain
-----------|-----------------|---------------
``1.3.3.0``|``-Python-3.6.4``|``intel/2018a``

### SuiteSparse

SuiteSparse is a collection of libraries manipulate sparse matrices.

*homepage*: <https://faculty.cse.tamu.edu/davis/suitesparse.html>

version   |versionsuffix               |toolchain
----------|----------------------------|-------------------------------------------------------------------------------------------------
``4.5.1`` |``-METIS-5.1.0``            |``foss/2016a``, ``intel/2016a``
``4.5.2`` |``-METIS-5.1.0``            |``foss/2016a``, ``intel/2016a``
``4.5.3`` |``-METIS-5.1.0``            |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``4.5.3`` |``-ParMETIS-4.0.3``         |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.5.5`` |``-METIS-5.1.0``            |``foss/2017a``, ``intel/2017a``
``4.5.5`` |``-ParMETIS-4.0.3``         |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``4.5.6`` |``-METIS-5.1.0``            |``foss/2017b``
``5.1.2`` |``-METIS-5.1.0``            |``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``5.1.2`` |``-ParMETIS-4.0.3``         |``foss/2017b``, ``intel/2017b``
``5.4.0`` |``-METIS-5.1.0``            |``foss/2019a``, ``intel/2018b``, ``intel/2019a``
``5.6.0`` |``-METIS-5.1.0``            |``foss/2019b``, ``intel/2019b``
``5.7.1`` |``-METIS-5.1.0``            |``foss/2020a``, ``intel/2020a``
``5.8.1`` |``-METIS-5.1.0``            |``foss/2020b``, ``intel/2020b``
``5.10.1``|``-METIS-5.1.0``            |``foss/2021a``, ``foss/2021b``, ``intel/2021a``, ``intel/2021b``
``5.10.1``|``-METIS-5.1.0-CUDA-11.3.1``|``foss/2021a``
``5.13.0``|``-METIS-5.1.0``            |``foss/2022a``, ``foss/2022b``

### SUMACLUST

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: <http://metabarcoding.org/sumatra>

version   |toolchain
----------|--------------
``1.0.20``|``foss/2016a``

### SUMATRA

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: <http://metabarcoding.org/sumatra>

version   |toolchain
----------|--------------
``1.0.20``|``foss/2016a``

### SUMO

Simulation of Urban MObility" (SUMO) is an open source, highly portable, microscopic and continuous traffic simulation package designed to handle large networks. It allows for intermodal simulation including pedestrians and comes with a large set of tools for scenario creation.

*homepage*: <https://www.eclipse.org/sumo/>

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``1.3.1`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``1.7.0`` |``-Python-3.8.2``|``foss/2020a``
``1.12.0``|                 |``foss/2021b``
``1.14.1``|                 |``foss/2021b``

### SUNDIALS

SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers

*homepage*: <https://computing.llnl.gov/projects/sundials>

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------
``2.6.2``|``intel/2016b``, ``intel/2018b``
``2.7.0``|``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``5.1.0``|``foss/2019b``, ``intel/2019b``
``5.7.0``|``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``6.2.0``|``foss/2020b``, ``intel/2020b``
``6.3.0``|``foss/2021b``, ``foss/2022a``
``6.5.1``|``foss/2022a``

### SunPy

The community-developed, free and open-source solar data analysis environment for Python.

*homepage*: <https://sunpy.org/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.3``|``-Python-3.7.4``|``foss/2019b``

### SuperLU

SuperLU is a general purpose library for the  direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.

*homepage*: <https://crd-legacy.lbl.gov/~xiaoye/SuperLU/>

version  |toolchain
---------|--------------------------------------------------------------------------------
``5.1.1``|``foss/2016a``, ``intel/2016a``
``5.2.1``|``foss/2017b``, ``intel/2017b``
``5.2.2``|``foss/2020a``, ``intel/2020a``
``5.3.0``|``foss/2020b``, ``foss/2021a``, ``foss/2022a``, ``intel/2020b``, ``intel/2022a``

### SuperLU_DIST

SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.

*homepage*: <https://crd-legacy.lbl.gov/~xiaoye/SuperLU/>

version  |versionsuffix      |toolchain
---------|-------------------|-------------------------------
``5.4.0``|``-trisolve-merge``|``intel/2020a``
``6.4.0``|                   |``foss/2020a``, ``intel/2020a``
``8.1.0``|                   |``foss/2022a``

### supermagic

Very simple MPI sanity code. Nothing more, nothing less.

*homepage*: <https://github.com/hpc/supermagic>

version     |toolchain
------------|-------------------------------
``20170824``|``foss/2017a``, ``gompi/2019b``

### supernova

Supernova is a software package for de novo assembly from Chromium Linked-Reads  that are made from a single whole-genome library from an individual DNA source

*homepage*: <https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome>

version  |toolchain
---------|----------
``2.0.1``|``system``
``2.1.1``|``system``

### SURVIVOR

Toolset for SV simulation, comparison and filtering

*homepage*: <https://github.com/fritzsedlazeck/SURVIVOR>

version              |toolchain
---------------------|--------------
``1.0.7-19-ged1ca51``|``GCC/11.2.0``

### SVclone

Cluster structural variants of similar cancer cell fraction (CCF).

*homepage*: <https://github.com/mcmero/SVclone>

version  |toolchain
---------|--------------
``1.1.2``|``foss/2022b``

### SVDetect

SVDetect is a application for the isolation and the type prediction of intra- and inter-chromosomal  rearrangements from paired-end/mate-pair sequencing data provided by the high-throughput sequencing technologies.  This tool aims to identifying structural variations with both clustering and sliding-window strategies, and  helping in their visualization at the genome scale.

*homepage*: <http://svdetect.sourceforge.net/Site/Home.html>

version |versionsuffix   |toolchain
--------|----------------|----------------------------------------------------------
``0.8b``|``-Perl-5.26.0``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

### SVDquest

SVDquartets-based species trees

*homepage*: <https://github.com/pranjalv123/SVDquest>

version     |toolchain
------------|---------------
``20190627``|``gompi/2019a``

### SVG

Perl binding for SVG

*homepage*: <https://metacpan.org/pod/SVG>

version |versionsuffix   |toolchain
--------|----------------|------------------------------
``2.84``|``-Perl-5.30.0``|``foss/2019b``
``2.87``|                |``GCC/11.2.0``, ``GCC/11.3.0``

### SVIM

SVIM (pronounced swim) is a structural variant caller for third-generation sequencing reads.  It is able to detect and classify the following six classes of structural variation: deletions, insertions,  inversions, tandem duplications, interspersed duplications and translocations.

*homepage*: <https://github.com/eldariont/svim>

version  |toolchain
---------|--------------
``2.0.0``|``foss/2022a``

### svist4get

Svist4get is a simple bioinformatics tool for visualization of  genomic signal tracks in user-defined genomic windows, either arbitrary selected  by genomic coordinates or anchored to particular transcripts or genes.

*homepage*: <https://github.com/art-egorov/svist4get>

version  |toolchain
---------|--------------
``1.3.1``|``foss/2020b``

### swarm

A robust and fast clustering method for amplicon-based studies 

*homepage*: <https://github.com/torognes/swarm>

version  |toolchain
---------|--------------
``2.2.2``|``foss/2018b``

### SWASH

SWASH is a general-purpose numerical tool for simulating unsteady, non-hydrostatic, free-surface,  rotational flow and transport phenomena in coastal waters as driven by waves, tides, buoyancy and wind forces.

*homepage*: <http://swash.sourceforge.net/>

version |versionsuffix|toolchain
--------|-------------|--------------------------------
``3.14``|``-mpi``     |``intel/2016b``, ``intel/2017a``
``4.01``|``-mpi``     |``intel/2017a``

### SWAT+

The Soil & Water Assessment Tool (SWAT) is a small watershed to river basin-scale model used to simulate the quality and quantity of surface and ground water and predict the environmental impact of land use, land management practices, and climate change. In order to face present and future challenges in water resources modeling SWAT code has undergone major modifications over the past few years, resulting in SWAT+, a completely revised version of the model. SWAT+ provides a more flexible spatial representation of interactions and processes within a watershed.

*homepage*: <https://swatplus.gitbook.io>

version   |toolchain
----------|-----------------------
``60.4.1``|``GCC/9.3.0``
``60.5.1``|``iccifort/2020.4.304``

### swifter

A package which efficiently applies any function to a pandas dataframe or series in the fastest available manner.

*homepage*: <https://github.com/jmcarpenter2/swifter>

version  |toolchain
---------|----------------------------------
``1.0.9``|``foss/2020b``, ``fosscuda/2020b``

### SWIG

SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.

*homepage*: <http://www.swig.org/>

version   |versionsuffix               |toolchain
----------|----------------------------|------------------------------------------------------------------------------
``3.0.8`` |``-Python-2.7.11``          |``foss/2016a``, ``intel/2016a``
``3.0.8`` |``-Python-3.5.1``           |``foss/2016a``
``3.0.10``|``-Python-2.7.12``          |``foss/2016b``, ``intel/2016b``
``3.0.10``|``-Python-2.7.12-PCRE-8.39``|``intel/2016b``
``3.0.11``|``-Python-2.7.12``          |``foss/2016b``, ``intel/2016b``
``3.0.12``|                            |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.0.12``|``-Python-2.7.13``          |``intel/2017a``
``3.0.12``|``-Python-2.7.14``          |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.0.12``|``-Python-2.7.14-bare``     |``GCCcore/6.4.0``
``3.0.12``|``-Python-2.7.15``          |``GCCcore/8.2.0``, ``foss/2018b``, ``intel/2018b``
``3.0.12``|``-Python-3.6.1``           |``intel/2017a``
``3.0.12``|``-Python-3.6.2``           |``foss/2017b``, ``intel/2018.00``
``3.0.12``|``-Python-3.6.3``           |``foss/2017b``, ``intel/2017b``, ``intel/2018.01``
``3.0.12``|``-Python-3.6.4``           |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.0.12``|``-Python-3.6.6``           |``foss/2018b``, ``intel/2018b``
``3.0.12``|``-Python-3.7.2``           |``GCCcore/8.2.0``
``4.0.1`` |                            |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.0.2`` |                            |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### SWIPE

Smith-Waterman database searches with inter-sequence SIMD parallelisation

*homepage*: <https://github.com/torognes/swipe>

version  |toolchain
---------|--------------
``2.1.1``|``GCC/10.3.0``

### swissknife

Perl module for reading and writing UniProtKB data in plain text format.

*homepage*: <https://sourceforge.net/projects/swissknife/>

version |toolchain
--------|-----------------
``1.80``|``GCCcore/8.3.0``

### SymEngine

SymEngine is a standalone fast C++ symbolic manipulation library

*homepage*: <https://github.com/symengine/symengine>

version  |versionsuffix|toolchain
---------|-------------|--------------------
``0.3.0``|``-20181006``|``intel/2018a``
``0.4.0``|             |``GCC/8.2.0-2.31.1``
``0.7.0``|             |``GCC/10.3.0``

### SymEngine-python

Python wrappers to the C++ library SymEngine, a fast C++ symbolic manipulation library.

*homepage*: <https://github.com/symengine/symengine.py>

version  |toolchain
---------|--------------
``0.7.2``|``GCC/10.3.0``

### SYMPHONY

SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.

*homepage*: <https://projects.coin-or.org/SYMPHONY>

version   |toolchain
----------|--------------
``5.6.16``|``foss/2018b``

### sympy

SymPy is a Python library for symbolic mathematics. It aims to  become a full-featured computer algebra system (CAS) while keeping the code as  simple as possible in order to be comprehensible and easily extensible. SymPy  is written entirely in Python and does not require any external libraries.

*homepage*: <https://sympy.org/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.3``   |``-Python-2.7.14``|``intel/2018a``
``1.3``   |``-Python-2.7.15``|``foss/2018b``
``1.3``   |``-Python-3.6.4`` |``intel/2018a``
``1.3``   |``-Python-3.6.6`` |``foss/2018b``
``1.4``   |                  |``foss/2019a``, ``intel/2019a``
``1.5.1`` |``-Python-3.7.4`` |``foss/2019b``
``1.6.2`` |``-Python-3.8.2`` |``foss/2020a``
``1.7.1`` |                  |``foss/2020b``, ``intel/2020b``
``1.8``   |                  |``foss/2021a``
``1.9``   |                  |``foss/2021b``
``1.10.1``|                  |``foss/2022a``, ``intel/2022a``
``1.11.1``|                  |``foss/2022a``, ``intel/2022a``

### synthcity

A library for generating and evaluating synthetic tabular data.

*homepage*: <https://github.com/vanderschaarlab/synthcity>

version  |toolchain
---------|--------------
``0.2.4``|``foss/2022a``

### SyRI

Synteny and Rearrangement Identifier (SyRI).

*homepage*: <https://schneebergerlab.github.io/syri/>

version|toolchain
-------|--------------
``1.4``|``foss/2021a``

### sysbench

sysbench is a scriptable multi-threaded benchmark tool based on LuaJIT.  It is most frequently used for database benchmarks, but can also be used to create arbitrarily complex  workloads that do not involve a database server.

*homepage*: <https://github.com/akopytov/sysbench>

version   |toolchain
----------|--------------
``1.0.20``|``GCC/12.2.0``

### Szip

Szip compression software, providing lossless compression of scientific data 

*homepage*: <https://www.hdfgroup.org/doc_resource/SZIP/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``  |``GCC/4.8.1``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``gmpolf/2017.10``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``2.1.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

## T


[tabix](#tabix) - [tabixpp](#tabixpp) - [taco](#taco) - [TagDust](#tagdust) - [TagLib](#taglib) - [Taiyaki](#taiyaki) - [TALON](#talon) - [TALYS](#talys) - [TAMkin](#tamkin) - [tantan](#tantan) - [Tapenade](#tapenade) - [task-spooler](#task-spooler) - [taxator-tk](#taxator-tk) - [TBA](#tba) - [tbb](#tbb) - [tbl2asn](#tbl2asn) - [TCC](#tcc) - [Tcl](#tcl) - [TCLAP](#tclap) - [tcsh](#tcsh) - [tecplot360ex](#tecplot360ex) - [TELEMAC-MASCARET](#telemac-mascaret) - [Telescope](#telescope) - [Teneto](#teneto) - [tensorboard](#tensorboard) - [tensorboardX](#tensorboardx) - [TensorFlow](#tensorflow) - [TensorFlow-Datasets](#tensorflow-datasets) - [TensorFlow-Graphics](#tensorflow-graphics) - [tensorflow-probability](#tensorflow-probability) - [TensorRT](#tensorrt) - [terastructure](#terastructure) - [termcolor](#termcolor) - [Tesla-Deployment-Kit](#tesla-deployment-kit) - [tesseract](#tesseract) - [testpath](#testpath) - [TetGen](#tetgen) - [TEToolkit](#tetoolkit) - [TEtranscripts](#tetranscripts) - [texinfo](#texinfo) - [texlive](#texlive) - [Text-CSV](#text-csv) - [TFEA](#tfea) - [Theano](#theano) - [ThemisPy](#themispy) - [THetA](#theta) - [thirdorder](#thirdorder) - [thurstonianIRT](#thurstonianirt) - [TiCCutils](#ticcutils) - [tidybayes](#tidybayes) - [tidymodels](#tidymodels) - [Tika](#tika) - [TiMBL](#timbl) - [time](#time) - [timm](#timm) - [TINKER](#tinker) - [TinyDB](#tinydb) - [TinyXML](#tinyxml) - [Tk](#tk) - [Tkinter](#tkinter) - [TM-align](#tm-align) - [tMAE](#tmae) - [tmap](#tmap) - [tmux](#tmux) - [TN93](#tn93) - [TOBIAS](#tobias) - [ToFu](#tofu) - [Togl](#togl) - [toil](#toil) - [tokenizers](#tokenizers) - [Tombo](#tombo) - [TOML-Fortran](#toml-fortran) - [TOPAS](#topas) - [topaz](#topaz) - [TopHat](#tophat) - [torchaudio](#torchaudio) - [torchdata](#torchdata) - [torchinfo](#torchinfo) - [torchsampler](#torchsampler) - [torchtext](#torchtext) - [torchvision](#torchvision) - [TotalView](#totalview) - [tqdm](#tqdm) - [Tracer](#tracer) - [TranscriptClean](#transcriptclean) - [TransDecoder](#transdecoder) - [Transformers](#transformers) - [Transrate](#transrate) - [travis](#travis) - [TRAVIS-Analyzer](#travis-analyzer) - [treatSens](#treatsens) - [TreeMix](#treemix) - [TreeShrink](#treeshrink) - [TRF](#trf) - [Triangle](#triangle) - [Trilinos](#trilinos) - [Trim_Galore](#trim_galore) - [trimAl](#trimal) - [trimesh](#trimesh) - [Trimmomatic](#trimmomatic) - [Trinity](#trinity) - [Trinotate](#trinotate) - [Triplexator](#triplexator) - [TRIQS](#triqs) - [TRIQS-cthyb](#triqs-cthyb) - [TRIQS-dft_tools](#triqs-dft_tools) - [TRIQS-tprf](#triqs-tprf) - [tRNAscan-SE](#trnascan-se) - [TRUST](#trust) - [TRUST4](#trust4) - [Trycycler](#trycycler) - [tsne](#tsne) - [turbinesFoam](#turbinesfoam) - [TurboVNC](#turbovnc) - [TVB](#tvb) - [tvb-data](#tvb-data) - [TVB-deps](#tvb-deps) - [tvb-framework](#tvb-framework) - [tvb-library](#tvb-library) - [TWL-NINJA](#twl-ninja) - [typing-extensions](#typing-extensions)


### tabix

Generic indexer for TAB-delimited genome position files 

*homepage*: <http://samtools.sourceforge.net>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.2.6``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``intel/2016a``, ``intel/2016b``

### tabixpp

C++ wrapper to tabix indexer

*homepage*: <https://github.com/ekg/tabixpp>

version  |toolchain
---------|-------------------------------------------------------------
``1.1.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

### taco

Multi-sample transcriptome assembly from RNA-Seq

*homepage*: <http://tacorna.github.io>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.5.1``|``-Python-2.7.12``|``intel/2016b``

### TagDust

Raw sequences produced by next generation sequencing (NGS) machines may contain adapter,  linker, barcode and fingerprint sequences. TagDust2 is a program to extract and correctly label the  sequences to be mapped in downstream pipelines.

*homepage*: <http://tagdust.sourceforge.net/>

version |toolchain
--------|-------------------------------------
``2.33``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``

### TagLib

TagLib is a library for reading and editing the meta-data of several popular audio formats.

*homepage*: <https://taglib.org/>

version   |toolchain
----------|-----------------
``1.11.1``|``GCCcore/8.2.0``

### Taiyaki

Taiyaki is research software for training models for basecalling Oxford Nanopore reads.

*homepage*: <https://github.com/nanoporetech/taiyaki>

version           |versionsuffix                  |toolchain
------------------|-------------------------------|----------------------------------
``5.1.0-20200617``|``-Python-3.7.2-PyTorch-1.2.0``|``foss/2019a``, ``fosscuda/2019a``

### TALON

TALON is a Python package for identifying and quantifying known and novel genes/isoforms in long-read transcriptome data sets. TALON is technology-agnostic in that it works from mapped SAM files, allowing data from different sequencing platforms (i.e. PacBio and Oxford Nanopore) to be analyzed side by side.

*homepage*: <https://github.com/mortazavilab/TALON>

version|toolchain
-------|--------------
``5.0``|``foss/2020b``

### TALYS

TALYS is a nuclear reaction program.

*homepage*: <https://tendl.web.psi.ch/tendl_2019/talys.html>

version |toolchain
--------|-------------------------------------
``1.95``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``

### TAMkin

TAMkin is a post-processing toolkit for normal mode analysis,  thermochemistry and reaction kinetics. It uses a Hessian computation from a  standard computational chemistry program as its input.

*homepage*: <https://molmod.github.io/tamkin/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.9``|``-Python-2.7.11``|``intel/2016a``
``1.2.4``|``-Python-2.7.14``|``intel/2017b``
``1.2.6``|``-Python-3.7.2`` |``intel/2019a``
``1.2.6``|``-Python-3.8.2`` |``intel/2020a``

### tantan

tantan identifies simple regions / low complexity / tandem repeats in DNA or protein sequences

*homepage*: <https://gitlab.com/mcfrith/tantan>

version|toolchain
-------|--------------
``40`` |``GCC/11.2.0``

### Tapenade

Tool for Algorithmic Differentiation of programs.

*homepage*: <https://tapenade.gitlabpages.inria.fr/tapenade/docs/html/index.html>

version |versionsuffix|toolchain
--------|-------------|----------
``3.16``|``-Java-17`` |``system``

### task-spooler

task spooler is a Unix batch system where the tasks spooled run one after the other.

*homepage*: <http://viric.name/soft/ts/>

version  |toolchain
---------|------------------
``1.0.2``|``GCCcore/11.3.0``

### taxator-tk

A set of programs for the taxonomic analysis of nucleotide sequence data

*homepage*: <https://github.com/fungs/taxator-tk>

version  |toolchain
---------|-----------------------------------------------
``1.3.3``|``GCC/10.2.0``, ``foss/2018b``, ``gompi/2019a``

### TBA

TBA (a Transcription factor Binding Analysis): TBA is a multi-functional machine learning tool for identifying  transcription factors associated with genomic features

*homepage*: <https://github.com/jenhantao/tba>

version|toolchain
-------|--------------
``1.0``|``foss/2020b``

### tbb

Intel Threading Building Blocks (Intel TBB)  is a widely used, award-winning C++ template library for creating reliable,  portable, and scalable parallel applications.  Use Intel TBB for a simple and rapid way of developing robust task-based  parallel applications that scale to available processor cores, are compatible  with multiple environments, and are easier to maintain.  Intel TBB is the most proficient way to implement future-proof parallel applications  that tap into the power and performance of multicore and manycore hardware platforms.

*homepage*: <https://software.intel.com/en-us/articles/intel-tbb/>

version       |toolchain
--------------|----------------------------------------------------------
``4.0.0.233`` |``system``
``4.0.5.339`` |``system``
``4.3.6.211`` |``system``
``4.4.2.152`` |``system``
``2017.2.132``|``system``
``2017.4.174``|``system``
``2017.6.196``|``system``
``2017_U5``   |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2017a``
``2017_U6``   |``GCCcore/6.3.0``, ``intel/2017a``
``2018_U1``   |``GCCcore/6.4.0``
``2018_U2``   |``GCCcore/6.4.0``
``2018_U3``   |``GCCcore/6.4.0``
``2018_U5``   |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2019_U4``   |``GCCcore/8.2.0``
``2019_U9``   |``GCCcore/8.3.0``
``2020.1``    |``GCCcore/9.3.0``
``2020.2``    |``GCCcore/8.3.0``
``2020.3``    |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2021.4.0``  |``GCCcore/11.2.0``
``2021.5.0``  |``GCCcore/11.3.0``

### tbl2asn

Tbl2asn is a command-line program that automates the creation of  sequence records for submission to GenBank

*homepage*: <https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/>

version     |versionsuffix|toolchain
------------|-------------|----------
``25.8``    |``-linux64`` |``system``
``20180227``|``-linux64`` |``system``
``20200302``|``-linux64`` |``system``
``20220427``|``-linux64`` |``system``

### TCC

The Tiny C Compiler (aka TCC, tCc, or TinyCC) is an x86 and x86-64 C compiler  created by Fabrice Bellard. It is designed to work for slow computers with little disk space  and can run shebang style !/usr/bin/tcc . TCC is distributed under the LGPL. TCC claims to  implement all of ANSI C (C89/C90),[1] much of the new ISO C99 standard,  and many GNU C extensions including inline assembly.

*homepage*: <http://bellard.org/tcc/>

version   |toolchain
----------|----------
``0.9.26``|``system``

### Tcl

Tcl (Tool Command Language) is a very powerful but simple to learn dynamic  programming language, suitable for a very wide range of uses, including web  and desktop applications, networking, administration, testing and many more. 

*homepage*: <https://www.tcl.tk/>

version   |toolchain
----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``8.5.19``|``foss/2017a``
``8.6.3`` |``GCC/4.8.4``, ``GCC/4.9.2``
``8.6.4`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``8.6.5`` |``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``8.6.6`` |``GCCcore/4.9.3``, ``GCCcore/6.3.0``
``8.6.7`` |``GCCcore/6.4.0``
``8.6.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``8.6.9`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.6.10``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``8.6.11``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``8.6.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``8.6.13``|``GCCcore/12.3.0``, ``GCCcore/13.1.0``

### TCLAP

TCLAP is a small, flexible library that provides a simple interface for defining and accessing command line arguments. It was intially inspired by the user friendly CLAP libary.

*homepage*: <http://tclap.sourceforge.net/>

version  |toolchain
---------|--------------------------------------
``1.2.2``|``GCCcore/8.3.0``
``1.2.4``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.2.5``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### tcsh

Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh).  It is a command language interpreter usable both as an interactive login shell and a shell script command  processor. It includes a command-line editor, programmable word completion, spelling correction, a history  mechanism, job control and a C-like syntax.

*homepage*: <https://www.tcsh.org>

version    |toolchain
-----------|-------------------------------------------------------
``6.19.00``|``intel/2016a``
``6.20.00``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``6.22.02``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.22.03``|``GCCcore/10.2.0``
``6.22.04``|``GCCcore/10.3.0``
``6.24.01``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``6.24.07``|``GCCcore/12.2.0``

### tecplot360ex

Quickly plot and animate your CFD results exactly the way you want. Analyze complex solutions, arrange multiple layouts, and communicate your results with professional images and animations.

*homepage*: <http://www.tecplot.com/products/tecplot-360/>

version    |toolchain
-----------|----------
``linux64``|``system``

### TELEMAC-MASCARET

TELEMAC-MASCARET is an integrated suite of solvers for use in the field of free-surface flow. Having been used in the context of many studies throughout the world, it has become one of the major standards in its field.

*homepage*: <http://www.opentelemac.org>

version  |toolchain
---------|--------------
``8p3r1``|``foss/2021b``

### Telescope

Single locus resolution of Transposable ELEment expression using next-generation sequencing.

*homepage*: <https://github.com/mlbendall/telescope>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.7.4``|``foss/2019b``

### Teneto

Teneto is package for deriving, analysing and plotting temporal network representations. Additional tools for temporal network analysis with neuroimaging contexts.

*homepage*: <https://teneto.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.5.1``|``-Python-3.8.2``|``foss/2020a``

### tensorboard

TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs.

*homepage*: <https://github.com/tensorflow/tensorboard>

version   |toolchain
----------|--------------
``2.8.0`` |``foss/2021a``
``2.10.0``|``foss/2022a``

### tensorboardX

Tensorboard for PyTorch.

*homepage*: <https://github.com/lanpa/tensorboardX>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``2.0``  |``-Python-3.7.4`` |``foss/2019b``
``2.1``  |``-PyTorch-1.7.1``|``fosscuda/2020b``
``2.2``  |``-PyTorch-1.7.1``|``foss/2020b``, ``fosscuda/2020b``
``2.5.1``|                  |``foss/2022a``

### TensorFlow

An open-source software library for Machine Intelligence

*homepage*: <https://www.tensorflow.org/>

version   |versionsuffix                |toolchain
----------|-----------------------------|---------------------------------------------------
``0.12.1``|``-Python-2.7.12``           |``foss/2016b``, ``intel/2016b``
``0.12.1``|``-Python-3.5.2``            |``foss/2016b``, ``intel/2016b``
``1.0.1`` |``-Python-2.7.12``           |``intel/2016b``
``1.0.1`` |``-Python-3.5.2``            |``intel/2016b``
``1.1.0`` |``-Python-2.7.13``           |``intel/2017a``
``1.1.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.2.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.2.1`` |``-GPU-Python-3.5.2``        |``foss/2016b``
``1.2.1`` |``-Python-3.5.2``            |``foss/2016b``
``1.3.0`` |``-Python-2.7.13``           |``intel/2017a``
``1.3.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.3.0`` |``-Python-3.6.3``            |``intel/2017b``
``1.4.0`` |``-Python-3.6.3``            |``foss/2017b``, ``intel/2017b``
``1.4.1`` |``-Python-3.6.3``            |``foss/2017b``
``1.5.0`` |``-Python-3.6.3``            |``foss/2017b``, ``intel/2017b``
``1.6.0`` |``-Python-3.6.4``            |``foss/2018a``, ``intel/2018a``
``1.6.0`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``1.7.0`` |``-Python-3.6.4``            |``foss/2018a``
``1.7.0`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``1.8.0`` |``-Python-2.7.14``           |``foss/2017b``, ``fosscuda/2017b``
``1.8.0`` |``-Python-3.6.3``            |``foss/2017b``, ``fosscuda/2017b``
``1.8.0`` |``-Python-3.6.4``            |``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``
``1.10.0``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.10.1``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.10.1``|``-Python-3.6.6``            |``foss/2018b``
``1.11.0``|``-Python-3.6.6``            |``foss/2018b``
``1.12.0``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.12.0``|``-Python-3.6.6``            |``foss/2018b``, ``fosscuda/2018b``
``1.13.1``|``-Python-3.6.6``            |``foss/2018b``
``1.13.1``|``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.14.0``|``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.15.0``|``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.15.2``|``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.15.5``|``-Python-3.7.4``            |``fosscuda/2019b``
``1.15.5``|``-Python-3.7.4-nompi``      |``fosscuda/2019b``
``2.0.0`` |``-Python-3.7.2``            |``foss/2019a``
``2.0.0`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.0.1`` |``-Python-3.7.2``            |``foss/2019a``
``2.0.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.1.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.2.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.2.2`` |``-Python-3.7.4``            |``foss/2019b``
``2.2.3`` |                             |``foss/2020b``
``2.3.1`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.3.1`` |``-Python-3.8.2``            |``foss/2020a``, ``fosscuda/2020a``
``2.4.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``2.4.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.4.4`` |                             |``foss/2021a``
``2.5.0`` |                             |``foss/2020b``, ``fosscuda/2020b``
``2.5.0`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.5.3`` |                             |``foss/2021a``
``2.5.3`` |``-CUDA-11.3.1``             |``foss/2021a``
``2.6.0`` |                             |``foss/2021a``
``2.6.0`` |``-CUDA-11.3.1``             |``foss/2021a``
``2.7.1`` |                             |``foss/2021b``
``2.7.1`` |``-CUDA-11.4.1``             |``foss/2021b``
``2.8.4`` |                             |``foss/2021b``
``2.8.4`` |``-CUDA-11.4.1``             |``foss/2021b``
``2.9.1`` |                             |``foss/2022a``
``2.9.1`` |``-CUDA-11.7.0``             |``foss/2022a``
``2.11.0``|                             |``foss/2022a``
``2.11.0``|``-CUDA-11.7.0``             |``foss/2022a``

### TensorFlow-Datasets

TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as Jax. All datasets are exposed as tf.data.Datasets , enabling simple-to-use and high-performance input pipelines. 

*homepage*: <https://www.tensorflow.org/datasets>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``4.7.0``|``-CUDA-11.3.1``|``foss/2021a``
``4.8.3``|``-CUDA-11.4.1``|``foss/2021b``

### TensorFlow-Graphics

Tensorflow Graphics provides a set of differentiable graphics and geometry layers (e.g. cameras, reflectance models, spatial transformations, mesh convolutions) and 3D viewer functionalities (e.g. 3D TensorBoard) that can be used to train and debug your machine learning models of choice.

*homepage*: <https://github.com/tensorflow/graphics>

version      |versionsuffix   |toolchain
-------------|----------------|--------------
``2021.12.3``|``-CUDA-11.4.1``|``foss/2021b``

### tensorflow-probability

TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis.

*homepage*: <https://www.tensorflow.org/probability>

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.9.0`` |``-Python-3.7.4``|``foss/2019b``
``0.12.1``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.16.0``|                 |``foss/2021b``

### TensorRT

NVIDIA TensorRT is a platform for high-performance deep learning inference

*homepage*: <https://developer.nvidia.com/tensorrt>

version    |versionsuffix     |toolchain
-----------|------------------|------------------
``4.0.1.6``|``-Python-2.7.15``|``fosscuda/2018b``

### terastructure

TeraStructure is a new algorithm to fit Bayesian models of genetic variation in human  populations on tera-sample-sized data sets (10^12 observed genotypes, i.e., 1M individuals at 1M SNPs).  This package provides a scalable, multi-threaded C++ implementation that can be run on a single computer. 

*homepage*: <https://github.com/StoreyLab/terastructure>

version|toolchain
-------|-------------
``1.0``|``GCC/8.3.0``

### termcolor

Termcolor is a header-only C++ library for printing colored  messages to the terminal.

*homepage*: <https://termcolor.readthedocs.io/>

version  |toolchain
---------|----------
``2.0.0``|``system``

### Tesla-Deployment-Kit

The Tesla Deployment Kit is a set of tools provided primarily for the NVIDIA Tesla  range of GPUs. They aim to empower users to better manage their NVIDIA GPUs by providing a broad  range of functionalities. The kit contains:  * NVIDIA Management Library (NVML),  * Tesla Deployment Kit - Linux Edition (Aug 1st, 2013)

*homepage*: <https://developer.nvidia.com/tesla-deployment-kit>

version     |toolchain
------------|----------
``5.319.43``|``system``

### tesseract

Tesseract is an optical character recognition engine

*homepage*: <https://github.com/tesseract-ocr/tesseract>

version  |toolchain
---------|-------------------------------------
``4.0.0``|``GCCcore/7.3.0``
``4.1.0``|``GCCcore/10.3.0``, ``GCCcore/8.2.0``
``5.3.0``|``GCCcore/11.3.0``

### testpath

Test utilities for code working with files and commands

*homepage*: <https://github.com/jupyter/testpath>

version|versionsuffix     |toolchain
-------|------------------|-------------------------------
``0.3``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.3``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.3``|``-Python-3.5.1`` |``foss/2016a``
``0.3``|``-Python-3.5.2`` |``intel/2016b``

### TetGen

A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator 

*homepage*: <http://wias-berlin.de/software/index.jsp?id=TetGen>

version  |toolchain
---------|------------------
``1.5.0``|``GCCcore/6.4.0``
``1.6.0``|``GCCcore/10.2.0``

### TEToolkit

Tools for estimating differential enrichment of Transposable  Elements and other highly repetitive regions

*homepage*: <http://hammelllab.labsites.cshl.edu/software/#TEToolkit>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.5.1``|``-Python-2.7.11``|``foss/2016a``

### TEtranscripts

TEtranscripts and TEcount takes RNA-seq (and similar data) and annotates reads to both genes & transposable elements. TEtranscripts then performs differential analysis using DESeq2.

*homepage*: <https://github.com/mhammell-laboratory/TEtranscripts>

version  |toolchain
---------|--------------
``2.2.0``|``foss/2020a``

### texinfo

Texinfo is the official documentation format of the GNU project.

*homepage*: <https://www.gnu.org/software/texinfo/>

version  |toolchain
---------|--------------------------------------------------------
``4.13a``|``system``
``5.2``  |``GCC/4.8.2``
``6.4``  |``GCCcore/5.4.0``
``6.5``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``6.6``  |``GCCcore/8.2.0``
``6.7``  |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.8``  |``GCCcore/11.2.0``
``7.0.2``|``GCCcore/11.3.0``

### texlive

TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript  text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a  PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to  be a representation of the final typeset output, and so can be easily edited and manipulated.

*homepage*: <https://tug.org>

version     |toolchain
------------|------------------------------
``20200406``|``GCCcore/8.3.0``
``20210324``|``GCC/10.3.0``, ``GCC/11.2.0``
``20220321``|``GCC/11.2.0``

### Text-CSV

Text-CSV parser

*homepage*: <https://metacpan.org/pod/Text::CSV>

version |versionsuffix   |toolchain
--------|----------------|--------------
``1.33``|``-Perl-5.22.1``|``foss/2016a``

### TFEA

Transcription Factor Enrichment Analysis 

*homepage*: <https://github.com/Dowell-Lab/TFEA>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.4``|``-muMerge-1.1.0``|``foss/2020b``

### Theano

Theano is a Python library that allows you to define, optimize, and evaluate mathematical expressions involving multi-dimensional arrays efficiently.

*homepage*: <https://deeplearning.net/software/theano>

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------------
``0.8.2``|``-Python-2.7.11``|``foss/2016a``
``0.8.2``|``-Python-3.5.1`` |``foss/2016a``
``0.8.2``|``-Python-3.5.2`` |``intel/2016b``
``0.9.0``|``-Python-2.7.13``|``intel/2017a``
``0.9.0``|``-Python-3.6.1`` |``intel/2017a``
``1.0.0``|``-Python-2.7.14``|``intel/2017b``
``1.0.0``|``-Python-3.6.3`` |``intel/2017b``
``1.0.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.0.2``|``-Python-2.7.14``|``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``|``-Python-2.7.15``|``fosscuda/2018b``
``1.0.2``|``-Python-3.6.3`` |``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.0.3``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``1.0.4``|                  |``foss/2019a``, ``fosscuda/2019a``
``1.0.4``|``-Python-3.6.4`` |``foss/2018a``
``1.0.4``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``1.0.4``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.0.4``|``-Python-3.8.2`` |``foss/2020a``
``1.1.2``|``-PyMC``         |``foss/2020b``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021b``

### ThemisPy

A header-only C++ library for L-BFGS and L-BFGS-B algorithms

*homepage*: <https://github.com/aeb/ThemisPy>

version  |toolchain
---------|--------------
``0.3.0``|``foss/2021a``

### THetA

Tumor Heterogeneity Analysis (THetA) and THetA2 are algorithms  that estimate the tumor purity and clonal/subclonal copy number aberrations  directly from high-throughput DNA sequencing data.

*homepage*: <https://github.com/raphael-group/THetA>

version|versionsuffix     |toolchain
-------|------------------|--------------
``0.7``|``-Python-2.7.15``|``foss/2018b``

### thirdorder

A Python script to help create input files for computing anhamonic interatomic force constants, harnessing the symmetries of the system to minimize the number of required DFT calculations. A second mode of operation allows the user to build the third-order IFC matrix from the results of those runs.

*homepage*: <https://bitbucket.org/sousaw/thirdorder/>

version  |toolchain
---------|--------------
``1.1.1``|``foss/2022a``

### thurstonianIRT

Fit Thurstonian IRT models in R using Stan, lavaan, or Mplus

*homepage*: <https://github.com/paul-buerkner/thurstonianIRT>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.9.0``|``-R-3.6.0`` |``foss/2019a``

### TiCCutils

TiCC utils is a collection of generic C++ software which is used in a lot of programs produced at  Tilburg centre for Cognition and Communication (TiCC) at Tilburg University and  Centre for Dutch Language and Speech at University of Antwerp.

*homepage*: <https://github.com/LanguageMachines/ticcutils>

version |toolchain
--------|--------------------------------
``0.11``|``foss/2016a``
``0.21``|``iimpi/2019a``, ``intel/2018b``

### tidybayes

Compose data for and extract, manipulate, and visualize posterior draws from Bayesian models ('JAGS', 'Stan', 'rstanarm', 'brms', 'MCMCglmm', 'coda', ...) in a tidy data format. 

*homepage*: <https://cran.r-project.org/package=tidybayes>

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.1.1``|``-R-4.0.0`` |``foss/2020a``

### tidymodels

tidymodels is a 'meta-package' for modeling and statistical analysis that shares the underlying  design philosophy, grammar, and data structures of the tidyverse.

*homepage*: <https://tidymodels.tidymodels.org>

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.1.0``|``-R-4.0.0`` |``foss/2020a``
``1.1.0``|             |``foss/2022b``

### Tika

The Apache Tika toolkit detects and extracts metadata and text from over a thousand different  file types (such as PPT, XLS, and PDF).

*homepage*: <https://tika.apache.org/>

version |toolchain
--------|----------
``1.16``|``system``

### TiMBL

TiMBL (Tilburg Memory Based Learner)  is an open source software package implementing several memory-based learning algorithms,  among which IB1-IG, an implementation of k-nearest neighbor classification with feature weighting suitable for  symbolic feature spaces, and IGTree, a decision-tree approximation of IB1-IG. All implemented algorithms have in  common that they store some representation of the training set explicitly in memory. During testing, new cases are  classified by extrapolation from the most similar stored cases.

*homepage*: <http://ilk.uvt.nl/timbl/>

version   |toolchain
----------|--------------------------------
``6.4.7`` |``foss/2016a``
``6.4.13``|``iimpi/2019a``, ``intel/2018b``

### time

The `time' command runs another program, then displays information about the resources used by that  program, collected by the system while the program was running.

*homepage*: <https://www.gnu.org/software/time/>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.7``|``system``
``1.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### timm

timm is a library containing SOTA computer vision models, layers, utilities, optimizers, schedulers, data-loaders, augmentations, and training/evaluation scripts.  It comes packaged with >700 pretrained models, and is designed to be flexible and simple to use. 

*homepage*: <https://huggingface.co/docs/timm>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.6.13``|``-CUDA-11.7.0``|``foss/2022a``

### TINKER

The TINKER molecular modeling software is a complete and general package for molecular mechanics  and dynamics, with some special features for biopolymers.

*homepage*: <https://dasher.wustl.edu/tinker>

version  |toolchain
---------|--------------
``8.6.1``|``foss/2018b``
``8.7.2``|``foss/2019b``
``8.8.1``|``foss/2020a``

### TinyDB

TinyDB is a lightweight document oriented database optimized for your happiness :) It's written in  pure Python and has no external dependencies. The target are small apps that would be blown away by a SQL-DB or an  external database server.

*homepage*: <https://tinydb.readthedocs.io/>

version   |toolchain
----------|-----------------
``3.15.2``|``GCCcore/8.3.0``

### TinyXML

TinyXML is a simple, small, minimal, C++ XML parser that can be easily integrating  into other programs. It reads XML and creates C++ objects representing the XML document. The  objects can be manipulated, changed, and saved again as XML. 

*homepage*: <https://sourceforge.net/projects/tinyxml>

version  |toolchain
---------|--------------------------------------
``2.6.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### Tk

Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for  building a graphical user interface (GUI) in many different programming languages.

*homepage*: <https://www.tcl.tk/>

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------
``8.6.3`` |``-no-X11``      |``GCC/4.8.4``, ``GCC/4.9.2``
``8.6.4`` |                 |``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``8.6.4`` |``-libX11-1.6.3``|``intel/2016a``
``8.6.4`` |``-no-X11``      |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``8.6.5`` |                 |``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``8.6.6`` |                 |``foss/2017a``, ``intel/2017a``
``8.6.7`` |                 |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``8.6.8`` |                 |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``iomkl/2018a``
``8.6.9`` |                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.6.10``|                 |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``8.6.11``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``8.6.12``|                 |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### Tkinter

Tkinter module, built with the Python buildsystem

*homepage*: <https://python.org/>

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------------------------
``2.7.13``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``2.7.14``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``2.7.15``|                  |``GCCcore/8.2.0``
``2.7.15``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.7.16``|                  |``GCCcore/8.3.0``
``2.7.18``|                  |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``3.6.1`` |``-Python-3.6.1`` |``foss/2017a``, ``intel/2017a``
``3.6.2`` |``-Python-3.6.2`` |``foss/2017b``
``3.6.3`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``3.6.4`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``3.6.6`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.2`` |                  |``GCCcore/8.2.0``
``3.7.4`` |                  |``GCCcore/8.3.0``
``3.8.2`` |                  |``GCCcore/9.3.0``
``3.8.6`` |                  |``GCCcore/10.2.0``
``3.9.5`` |                  |``GCCcore/10.3.0``
``3.9.6`` |                  |``GCCcore/11.2.0``
``3.10.4``|                  |``GCCcore/11.3.0``
``3.10.8``|                  |``GCCcore/12.2.0``

### TM-align

This package unifies protein structure alignment and RNA structure alignment  into the standard TM-align program for single chain structure alignment, MM-align program for  multi-chain structure alignment, and TM-score program for sequence dependent structure superposition.

*homepage*: <https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/>

version     |toolchain
------------|-------------------------------
``20180426``|``foss/2018b``, ``intel/2019a``
``20190822``|``GCC/11.3.0``

### tMAE

Package containing functions to:      perform a negative binomial test on allele-specific counts     add gnomAD minor allele frequencies     MAplot (FC vs total counts) of allele-specific counts and results     allelic counts (ALT vs REF)

*homepage*: <https://github.com/mumichae/tMAE>

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.0``|``-R-4.0.3`` |``foss/2020b``
``1.0.1``|``-R-4.1.2`` |``foss/2021b``

### tmap

tmap is a very fast visualization library for large, high-dimensional data sets. Currently, tmap is available for Python. tmap's graph layouts are based on the OGDF library.

*homepage*: <https://tmap.gdb.tools>

version     |toolchain
------------|--------------
``20220502``|``GCC/11.2.0``

### tmux

tmux is a terminal multiplexer: it enables a number of terminals to be created, accessed, and controlled from a single screen. tmux may be detached from a screen and continue running in the background, then later reattached.

*homepage*: <https://github.com/tmux/tmux/>

version |toolchain
--------|----------------------------------------------------------------------
``1.9a``|``GCC/4.9.2``
``2.2`` |``GCCcore/4.9.3``
``2.3`` |``GCC/5.4.0-2.26``, ``system``
``3.1c``|``GCCcore/8.3.0``
``3.2`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.2a``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``system``
``3.3a``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``system``

### TN93

This is a simple program meant to compute pairwise distances between  aligned nucleotide sequences in sequential FASTA format using the Tamura Nei 93 distance.

*homepage*: <https://github.com/veg/tn93>

version  |toolchain
---------|------------------
``1.0.7``|``GCCcore/10.3.0``

### TOBIAS

TOBIAS is a collection of command-line bioinformatics tools  for performing footprinting analysis on ATAC-seq data.

*homepage*: <https://github.com/loosolab/TOBIAS>

version    |toolchain
-----------|--------------
``0.12.12``|``foss/2021b``
``0.14.0`` |``foss/2020b``

### ToFu

Tomography for Fusion.

*homepage*: <https://github.com/ToFuProject/tofu>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.3.17``|``-Python-2.7.14``|``foss/2018a``
``1.3.17``|``-Python-3.6.4`` |``foss/2018a``
``1.4.0`` |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.4.0`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.4.1`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.5.0`` |                  |``foss/2020b``, ``intel/2020b``

### Togl

A Tcl/Tk widget for OpenGL rendering.

*homepage*: <https://sourceforge.net/projects/togl/>

version|toolchain
-------|---------------------------------------------------------
``2.0``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

### toil

A scalable, efficient, cross-platform (Linux/macOS) and simple-to-use workflow engine in pure Python.

*homepage*: <https://github.com/DataBiosphere/toil>

version  |toolchain
---------|--------------
``5.8.0``|``foss/2021a``

### tokenizers

Fast State-of-the-Art Tokenizers optimized for Research and Production

*homepage*: <https://github.com/huggingface/tokenizers>

version   |toolchain
----------|------------------
``0.12.1``|``GCCcore/10.3.0``
``0.13.3``|``GCCcore/12.2.0``

### Tombo

Tombo is a suite of tools primarily for the identification of modified nucleotides  from raw nanopore sequencing data.

*homepage*: <https://github.com/nanoporetech/tombo>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.5.1``|``-Python-3.7.4``|``foss/2019b``

### TOML-Fortran

TOML parser for Fortran projects

*homepage*: <https://github.com/toml-f/toml-f>

version  |toolchain
---------|---------------------------------------
``0.2.2``|``GCC/10.2.0``, ``iccifort/2020.4.304``
``0.3.1``|``GCC/11.2.0``, ``GCC/11.3.0``

### TOPAS

TOPAS wraps and extends the Geant4 Simulation Toolkit to make advanced Monte Carlo simulation of all forms of radiotherapy easier to use for medical physicists.

*homepage*: <https://www.topasmc.org>

version|toolchain
-------|--------------
``3.9``|``foss/2022b``

### topaz

A pipeline for particle detection in cryo-electron microscopy  images using convolutional neural networks trained from positive and unlabeled  examples. Topaz also includes methods for micrograph denoising using deep de-  noising models. 

*homepage*: <https://github.com/tbepler/topaz>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.2.5``|                |``foss/2021a``
``0.2.5``|``-CUDA-11.3.1``|``foss/2021a``

### TopHat

TopHat is a fast splice junction mapper for RNA-Seq reads.

*homepage*: <http://ccb.jhu.edu/software/tophat/>

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------------------------
``2.1.0``|``-Python-2.7.14``|``intel/2018a``
``2.1.1``|                  |``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.1.2``|                  |``foss/2018b``, ``gompi/2019b``, ``iimpi/2019b``
``2.1.2``|``-Python-2.7.18``|``GCC/10.2.0``, ``GCC/11.3.0``, ``gompi/2020a``, ``iimpi/2020a``

### torchaudio

Data manipulation and transformation for audio signal processing, powered by PyTorch 

*homepage*: <https://github.com/pytorch/audio>

version   |versionsuffix                  |toolchain
----------|-------------------------------|----------------------------------
``0.5.0`` |``-Python-3.7.4-PyTorch-1.4.0``|``foss/2019b``, ``fosscuda/2019b``
``0.12.0``|``-PyTorch-1.12.0``            |``foss/2022a``
``0.12.0``|``-PyTorch-1.12.0-CUDA-11.7.0``|``foss/2022a``

### torchdata

TorchData is a prototype library of common modular data loading primitives for easily constructing flexible and performant data pipelines." 

*homepage*: <https://github.com/pytorch/data>

version  |versionsuffix                  |toolchain
---------|-------------------------------|--------------
``0.3.0``|``-PyTorch-1.11.0-CUDA-11.3.1``|``foss/2021a``

### torchinfo

" Torchinfo provides information complementary to what is provided by print(your_model) in PyTorch, similar to Tensorflow's model.summary() API to view the visualization of the model, which is helpful while debugging your network. 

*homepage*: <https://github.com/TylerYep/torchinfo>

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``1.5.2``|``-PyTorch-1.7.1``|``foss/2020b``, ``fosscuda/2020b``

### torchsampler

A (PyTorch) imbalanced dataset sampler for oversampling low classes and undersampling high frequent ones.

*homepage*: <https://github.com/ufoym/imbalanced-dataset-sampler>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.2``|                |``foss/2022a``
``0.1.2``|``-CUDA-11.7.0``|``foss/2022a``

### torchtext

Data loaders and abstractions for text and NLP

*homepage*: <https://github.com/pytorch/text>

version   |versionsuffix                  |toolchain
----------|-------------------------------|----------------------------------
``0.5.0`` |``-PyTorch-1.4.0-Python-3.7.4``|``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0``|``foss/2019b``, ``fosscuda/2019b``
``0.8.1`` |``-PyTorch-1.7.1``             |``fosscuda/2020b``
``0.9.1`` |``-PyTorch-1.8.1``             |``fosscuda/2020b``
``0.10.0``|``-PyTorch-1.9.0``             |``fosscuda/2020b``
``0.14.1``|``-PyTorch-1.12.0``            |``foss/2022a``

### torchvision

Datasets, Transforms and Models specific to Computer Vision

*homepage*: <https://github.com/pytorch/vision>

version   |versionsuffix                       |toolchain
----------|------------------------------------|----------------------------------
``0.2.0`` |``-Python-3.6.4``                   |``intel/2018a``
``0.2.0`` |``-Python-3.6.4-CUDA-9.1.85``       |``foss/2018a``
``0.2.2`` |``-Python-3.6.6``                   |``foss/2018b``, ``fosscuda/2018b``
``0.3.0`` |``-Python-3.7.2``                   |``foss/2019a``
``0.4.2`` |``-PyTorch-1.3.1``                  |``fosscuda/2020b``
``0.4.2`` |``-Python-3.7.4``                   |``foss/2019b``, ``fosscuda/2019b``
``0.5.0`` |``-Python-3.7.4-PyTorch-1.4.0``     |``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0``     |``foss/2019b``, ``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0-imkl``|``fosscuda/2019b``
``0.8.2`` |``-PyTorch-1.7.1``                  |``foss/2020b``, ``fosscuda/2020b``
``0.8.2`` |``-Python-3.7.4-PyTorch-1.7.1``     |``fosscuda/2019b``
``0.9.1`` |``-PyTorch-1.8.1``                  |``fosscuda/2020b``
``0.9.1`` |``-Python-3.7.4-PyTorch-1.8.1``     |``fosscuda/2019b``
``0.10.0``|``-PyTorch-1.9.0``                  |``fosscuda/2020b``
``0.11.1``|                                    |``foss/2021a``
``0.11.1``|``-CUDA-11.3.1``                    |``foss/2021a``
``0.11.3``|                                    |``foss/2021a``
``0.11.3``|``-CUDA-11.3.1``                    |``foss/2021a``
``0.12.0``|``-PyTorch-1.11.0-CUDA-11.3.1``     |``foss/2021a``
``0.13.1``|                                    |``foss/2022a``
``0.13.1``|``-CUDA-11.7.0``                    |``foss/2022a``

### TotalView

TotalView is a GUI-based source code defect analysis tool that gives you unprecedented  control over processes and thread execution and visibility into program state and variables. It allows  you to debug one or many processes and/or threads in a single window with complete control over program  execution. This allows you to set breakpoints, stepping line by line through the code on a single thread,  or with coordinated groups of processes or threads, and run or halt arbitrary sets of processes or threads.  You can reproduce and troubleshoot difficult problems that can occur in concurrent programs that take  advantage of threads, OpenMP, MPI, GPUs or coprocessors.

*homepage*: <http://www.roguewave.com/products/totalview.aspx>

version     |versionsuffix    |toolchain
------------|-----------------|----------
``8.11.0-0``|``-linux-x86-64``|``system``
``8.11.0-2``|``-linux-x86-64``|``system``
``8.12.0-0``|``-linux-x86-64``|``system``

### tqdm

A fast, extensible progress bar for Python and CLI

*homepage*: <https://github.com/tqdm/tqdm>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``4.24.0``|``-Python-2.7.15``|``foss/2018b``
``4.24.0``|``-Python-3.5.1`` |``foss/2016a``
``4.29.0``|``-Python-3.6.4`` |``intel/2018a``
``4.32.1``|                  |``GCCcore/8.2.0``
``4.41.1``|                  |``GCCcore/8.3.0``
``4.41.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.47.0``|                  |``GCCcore/9.3.0``
``4.51.0``|``-Python-3.8.2`` |``intel/2020a``
``4.56.2``|                  |``GCCcore/10.2.0``
``4.60.0``|                  |``GCCcore/10.2.0``
``4.61.1``|                  |``GCCcore/10.3.0``
``4.61.2``|                  |``GCCcore/10.3.0``
``4.62.3``|                  |``GCCcore/11.2.0``
``4.64.0``|                  |``GCCcore/11.3.0``
``4.64.1``|                  |``GCCcore/12.2.0``

### Tracer

Tracer is a graphical tool for visualization and diagnostics of MCMC output.

*homepage*: <https://github.com/beast-dev/tracer>

version  |toolchain
---------|-----------------
``1.7.1``|``GCCcore/8.2.0``

### TranscriptClean

TranscriptClean is a Python program that corrects mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome.

*homepage*: <https://github.com/mortazavilab/TranscriptClean>

version  |toolchain
---------|--------------
``2.0.2``|``foss/2020b``

### TransDecoder

TransDecoder identifies candidate coding regions within transcript sequences,  such as those generated by de novo RNA-Seq transcript assembly using Trinity,  or constructed based on RNA-Seq alignments to the genome using  Tophat and Cufflinks.

*homepage*: <https://github.com/TransDecoder/TransDecoder/wiki>

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.1.0``|``-Perl-5.24.1``|``intel/2017a``
``5.5.0``|                |``GCC/11.3.0``

### Transformers

State-of-the-art Natural Language Processing for PyTorch and TensorFlow 2.0

*homepage*: <https://github.com/huggingface/transformers>

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``4.2.1`` |``-Python-3.8.2``|``foss/2020a``
``4.20.1``|                 |``foss/2021a``
``4.21.1``|                 |``foss/2021b``
``4.24.0``|                 |``foss/2022a``
``4.29.2``|                 |``foss/2022a``
``4.30.2``|                 |``foss/2022b``

### Transrate

Transrate is software for de-novo transcriptome assembly quality analysis.  It examines your assembly in detail and compares it to experimental evidence  such as the sequencing reads, reporting quality scores for contigs and  assemblies. This allows you to choose between assemblers and parameters,  filter out the bad contigs from an assembly, and help decide when  to stop trying to improve the assembly.

*homepage*: <http://hibberdlab.com/transrate/>

version  |toolchain
---------|----------
``1.0.3``|``system``

### travis

Travis CI Client (CLI and Ruby library)

*homepage*: <https://github.com/travis-ci/travis.rb>

version  |versionsuffix  |toolchain
---------|---------------|----------
``1.8.2``|``-Ruby-2.3.1``|``system``
``1.8.4``|``-Ruby-2.3.3``|``system``

### TRAVIS-Analyzer

TRAVIS is a free tool for analyzing and visualizing trajectories from all kinds of Molecular Dynamics or Monte Carlo simulations.

*homepage*: <http://www.travis-analyzer.de/>

version   |toolchain
----------|--------------
``210521``|``GCC/10.3.0``

### treatSens

Utilities to investigate sensitivity to unmeasured confounding in parametric models  with either binary or continuous treatment.

*homepage*: <https://github.com/vdorie/treatSens>

version         |versionsuffix|toolchain
----------------|-------------|--------------
``3.0-20201002``|``-R-4.0.0`` |``foss/2020a``

### TreeMix

TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a  set of populations.

*homepage*: <http://bitbucket.org/nygcresearch/treemix>

version |toolchain
--------|-----------------------------------------------
``1.13``|``GCC/10.3.0``, ``GCC/11.2.0``, ``intel/2018a``

### TreeShrink

TreeShrink is an algorithm for detecting abnormally long branches in one or more phylogenetic trees.

*homepage*: <https://github.com/uym2/TreeShrink>

version  |versionsuffix    |toolchain
---------|-----------------|--------------------
``1.3.2``|``-Python-3.7.2``|``GCC/8.2.0-2.31.1``

### TRF

Tandem Repeats Finder: a program to analyze DNA sequences.

*homepage*: <https://tandem.bu.edu/trf/trf.html>

version   |versionsuffix|toolchain
----------|-------------|---------------------------------------------------------------------------------------------
``4.09``  |``-linux64`` |``system``
``4.09.1``|             |``GCC/10.3.0``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### Triangle

Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,  conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.  The latter can be generated with no small or large angles,  and are thus suitable for finite element analysis.

*homepage*: <http://www.cs.cmu.edu/~quake/triangle.html>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------
``1.6``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2018a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``

### Trilinos

The Trilinos Project is an effort to develop algorithms and enabling technologies  within an object-oriented software framework for the solution of large-scale, complex multi-physics  engineering and scientific problems. A unique design feature of Trilinos is its focus on packages.

*homepage*: <https://trilinos.org>

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``12.12.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``12.12.1``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``12.12.1``|``-Python-3.7.4`` |``foss/2019b``
``13.4.1`` |                  |``foss/2022a``

### Trim_Galore

Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control, with some added functionality to remove biased methylation positions for RRBS sequence files (for directional, non-directional  (or paired-end) sequencing).

*homepage*: <https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/>

version   |versionsuffix            |toolchain
----------|-------------------------|--------------------------------------
``0.4.2`` |                         |``foss/2016b``
``0.4.4`` |                         |``foss/2016b``, ``intel/2017a``
``0.5.0`` |                         |``foss/2018b``
``0.5.0`` |``-Python-3.6.6``        |``intel/2018b``
``0.6.0`` |``-Python-2.7.15``       |``foss/2018b``
``0.6.0`` |``-Python-3.6.6``        |``foss/2018b``
``0.6.1`` |``-Python-2.7.15``       |``foss/2018b``
``0.6.1`` |``-Python-3.6.6``        |``foss/2018b``
``0.6.2`` |``-Java-11``             |``GCCcore/8.2.0``
``0.6.2`` |``-Python-2.7.15``       |``foss/2018b``
``0.6.2`` |``-Python-3.6.6``        |``foss/2018b``
``0.6.5`` |``-Java-11-Python-3.7.4``|``GCCcore/8.3.0``
``0.6.6`` |``-Python-2.7.18``       |``GCC/10.2.0``
``0.6.6`` |``-Python-3.8.2``        |``GCCcore/9.3.0``
``0.6.7`` |                         |``GCCcore/10.3.0``
``0.6.10``|                         |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### trimAl

A tool for automated alignment trimming in large-scale phylogenetic analyses

*homepage*: <http://trimal.cgenomics.org/>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``1.4.1``|``GCC/8.2.0-2.31.1``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``, ``intel/2018a``

### trimesh

Trimesh is a Python (2.7- 3.3+) library for loading and using triangular meshes with an emphasis on watertight meshes. The goal of the library is to provide a fully featured Trimesh object which allows for simple manipulation and analysis, in the style of the excellent Polygon object in the Shapely library.

*homepage*: <https://trimsh.org/>

version   |toolchain
----------|--------------
``3.17.1``|``foss/2022a``
``3.21.5``|``gfbf/2022b``

### Trimmomatic

Trimmomatic performs a variety of useful trimming tasks for illumina   paired-end and single ended data.The selection of trimming steps and their associated   parameters are supplied on the command line. 

*homepage*: <http://www.usadellab.org/cms/?page=trimmomatic>

version |versionsuffix      |toolchain
--------|-------------------|----------
``0.32``|``-Java-1.7.0_80`` |``system``
``0.36``|``-Java-1.8.0_92`` |``system``
``0.38``|``-Java-1.8``      |``system``
``0.38``|``-Java-1.8.0_162``|``system``
``0.39``|``-Java-1.8``      |``system``
``0.39``|``-Java-11``       |``system``
``0.39``|``-Java-17``       |``system``

### Trinity

Trinity represents a novel method for the efficient and robust de novo reconstruction  of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,  Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.

*homepage*: <https://trinityrnaseq.github.io>

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------
``2.2.0`` |                  |``foss/2016a``
``2.4.0`` |                  |``foss/2017a``
``2.5.1`` |                  |``intel/2017a``, ``intel/2017b``
``2.6.6`` |                  |``intel/2018a``
``2.8.4`` |                  |``foss/2018b``
``2.8.5`` |                  |``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.8.5`` |``-Java-11``      |``GCC/8.3.0``
``2.9.1`` |                  |``foss/2020b``
``2.9.1`` |``-Python-2.7.16``|``foss/2019b``
``2.9.1`` |``-Python-3.7.4`` |``foss/2019b``
``2.10.0``|``-Python-3.7.4`` |``foss/2019b``
``2.15.1``|                  |``foss/2021b``, ``foss/2022a``

### Trinotate

Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms. Trinotate makes use of a number of different well referenced methods for functional annotation including homology search to known sequence data (BLAST+/SwissProt), protein domain identification (HMMER/PFAM), protein signal peptide and transmembrane domain prediction (signalP/tmHMM), and leveraging various annotation databases (eggNOG/GO/Kegg databases). All functional annotation data derived from the analysis of transcripts is integrated into a SQLite database which allows fast efficient searching for terms with specific qualities related to a desired scientific hypothesis or a means to create a whole annotation report for a transcriptome.

*homepage*: <https://github.com/Trinotate/Trinotate/wiki>

version  |toolchain
---------|--------------
``4.0.1``|``foss/2022a``

### Triplexator

Triplexator is a tool for detecting nucleic acid triple helices and triplex features in nucleotide  sequences using the canonical triplex-formation rules.

*homepage*: <https://github.com/Gurado/triplexator>

version  |toolchain
---------|--------------
``1.3.3``|``GCC/11.2.0``

### TRIQS

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems. 

*homepage*: <https://triqs.github.io/triqs>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``2.2.1``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.1``|                  |``foss/2022a``

### TRIQS-cthyb

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   cthyb = continuous-time hybridisation-expansion quantum Monte Carlo   The TRIQS-based hybridization-expansion solver allows to solve the  generic problem of a quantum impurity embedded in a conduction bath  for an arbitrary local interaction vertex. The “impurity” can be any  set of orbitals, on one or several atoms. 

*homepage*: <https://triqs.github.io/cthyb/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.0``|                  |``foss/2022a``

### TRIQS-dft_tools

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   This TRIQS-based-based application is aimed at ab-initio calculations  for correlated materials, combining realistic DFT band-structure calculation   with the dynamical mean-field theory. Together with the necessary tools to  perform the DMFT self-consistency loop for realistic multi-band problems,  the package provides a full-fledged charge self-consistent interface to the  Wien2K package. In addition, if Wien2k is not available, it provides a generic  interface for one-shot DFT+DMFT calculations, where only the single-particle  Hamiltonian in orbital space has to be provided. 

*homepage*: <https://triqs.github.io/dft_tools/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.0``|                  |``foss/2022a``

### TRIQS-tprf

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   TPRF is a TRIQS-based two-particle response function tool box that  implements basic operations for higher order response functions such  as inversion, products, the random phase approximation, the bethe  salpeter equation (in the local vertex approximation), etc..   The aim is to provide efficient (C++/OpenMP/MPI) implementations of  the basic operations needed to compute the two-particle response in  the different two-particle channels (particle-hole, particle-particle). 

*homepage*: <https://triqs.github.io/tprf>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.1``|                  |``foss/2022a``

### tRNAscan-SE

tRNAscan-SE is the most widely employed tool for identifying  and annotating tRNA genes in genomes.

*homepage*: <http://trna.ucsc.edu/tRNAscan-SE/>

version   |toolchain
----------|------------------------------
``2.0.12``|``GCC/11.2.0``, ``foss/2022b``

### TRUST

Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze  TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. 

*homepage*: <https://bitbucket.org/liulab/trust>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.2``|``-Python-2.7.14``|``intel/2018a``

### TRUST4

Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. TRUST4 performs de novo assembly on V, J, C genes including the hypervariable complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to report the corresponding information. TRUST4 supports both single-end and paired-end sequencing data with any read length.

*homepage*: <https://github.com/liulab-dfci/TRUST4>

version    |toolchain
-----------|--------------
``1.0.5.1``|``system``
``1.0.6``  |``GCC/11.2.0``
``1.0.7``  |``GCC/11.3.0``

### Trycycler

Trycycler is a tool for generating consensus long-read assemblies for bacterial genomes.

*homepage*: <https://github.com/rrwick/Trycycler>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.3``|``-Python-3.8.2``|``foss/2020a``
``0.5.2``|                 |``foss/2021a``
``0.5.3``|                 |``foss/2021b``

### tsne

Python library containing T-SNE algorithms.

*homepage*: <https://github.com/danielfrg/tsne>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.1.8``|``-Python-2.7.16``|``intel/2019b``

### turbinesFoam

turbinesFoam is a library for simulating wind and marine hydrokinetic turbines in OpenFOAM using the actuator line method. 

*homepage*: <https://github.com/turbinesFoam/turbinesFoam>

version     |versionsuffix  |toolchain
------------|---------------|--------------
``20220516``|``-OpenFOAM-8``|``foss/2020a``

### TurboVNC

TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide  peak performance for 3D and video workloads.

*homepage*: <https://www.turbovnc.org>

version  |toolchain
---------|------------------
``2.2.3``|``GCCcore/8.2.0``
``3.0.1``|``GCCcore/11.3.0``

### TVB

The Virtual Brain will deliver the first open simulation of the human brain  based on individual large-scale connectivity.

*homepage*: <http://thevirtualbrain.org>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.11``|``intel/2016a``
``1.5``  |``-Python-2.7.11``|``intel/2016a``

### tvb-data

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  Various demonstration data for use with TVB.

*homepage*: <http://www.thevirtualbrain.org/>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150915``|``-Python-2.7.11``|``intel/2016a``

### TVB-deps

Bundle of dependency Python packages for TVB (The Virtual Brain)

*homepage*: <https://thevirtualbrain.org>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20160618``|``-Python-2.7.11``|``intel/2016a``

### tvb-framework

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: <http://www.thevirtualbrain.org/>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150921``|``-Python-2.7.11``|``intel/2016a``

### tvb-library

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: <http://www.thevirtualbrain.org/>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150922``|``-Python-2.7.11``|``intel/2016a``

### TWL-NINJA

Nearly Infinite Neighbor Joining Application.

*homepage*: <https://github.com/TravisWheelerLab/NINJA>

version              |toolchain
---------------------|--------------
``0.97-cluster_only``|``GCC/10.2.0``
``0.98-cluster_only``|``GCC/11.3.0``

### typing-extensions

Typing Extensions - Backported and Experimental Type Hints for Python

*homepage*: <https://github.com/python/typing/blob/master/typing_extensions/README.rst>

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``3.7.4.3`` |                 |``GCCcore/10.2.0``
``3.7.4.3`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.7.4.3`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``3.10.0.0``|                 |``GCCcore/10.3.0``
``3.10.0.2``|                 |``GCCcore/11.2.0``
``4.3.0``   |                 |``GCCcore/11.3.0``
``4.4.0``   |                 |``GCCcore/10.3.0``

## U


[UCC](#ucc) - [UCC-CUDA](#ucc-cuda) - [UCLUST](#uclust) - [UCX](#ucx) - [UCX-CUDA](#ucx-cuda) - [ucx-py](#ucx-py) - [UCX-ROCm](#ucx-rocm) - [udocker](#udocker) - [UDUNITS](#udunits) - [UFL](#ufl) - [Ultralytics](#ultralytics) - [umap-learn](#umap-learn) - [UMI-tools](#umi-tools) - [umi4cPackage](#umi4cpackage) - [umis](#umis) - [UNAFold](#unafold) - [uncertainty-calibration](#uncertainty-calibration) - [unicore-uftp](#unicore-uftp) - [Unicycler](#unicycler) - [Unidecode](#unidecode) - [unimap](#unimap) - [units](#units) - [unixODBC](#unixodbc) - [unrar](#unrar) - [UnZip](#unzip) - [UQTk](#uqtk) - [USEARCH](#usearch) - [UShER](#usher) - [USPEX](#uspex) - [utf8proc](#utf8proc) - [util-linux](#util-linux)


### UCC

UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes. 

*homepage*: <https://www.openucx.org/>

version  |toolchain
---------|------------------
``1.0.0``|``GCCcore/11.3.0``
``1.1.0``|``GCCcore/12.2.0``
``1.2.0``|``GCCcore/12.3.0``

### UCC-CUDA

UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes.  This module adds the UCC CUDA support. 

*homepage*: <https://www.openucx.org/>

version  |versionsuffix   |toolchain
---------|----------------|------------------
``1.0.0``|``-CUDA-11.7.0``|``GCCcore/11.3.0``
``1.1.0``|``-CUDA-12.0.0``|``GCCcore/12.2.0``
``1.2.0``|``-CUDA-12.1.1``|``GCCcore/12.3.0``

### UCLUST

UCLUST: Extreme high-speed sequence clustering, alignment and database search.

*homepage*: <https://www.drive5.com/usearch/index.html>

version    |versionsuffix  |toolchain
-----------|---------------|----------
``1.2.22q``|``-i86linux64``|``system``

### UCX

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications 

*homepage*: <http://www.openucx.org/>

version     |versionsuffix   |toolchain
------------|----------------|-------------------------------------------------------
``1.3.1``   |                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.5.0``   |                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.5.0rc1``|``-hpcx``       |``GCCcore/8.2.0``
``1.5.1``   |                |``GCCcore/8.2.0``
``1.6.1``   |                |``GCCcore/8.3.0``
``1.8.0``   |                |``GCCcore/9.3.0``
``1.8.0``   |``-CUDA-11.0.2``|``GCCcore/9.3.0``
``1.9.0``   |                |``GCCcore/10.2.0``
``1.9.0``   |``-CUDA-11.1.1``|``GCCcore/10.2.0``
``1.9.0``   |``-CUDA-11.2.1``|``GCCcore/10.2.0``
``1.10.0``  |                |``GCCcore/10.3.0``
``1.11.0``  |                |``GCCcore/11.2.0``
``1.11.2``  |                |``GCCcore/11.2.0``
``1.12.1``  |                |``GCCcore/11.3.0``
``1.13.1``  |                |``GCCcore/12.2.0``
``1.14.0``  |                |``GCCcore/12.2.0``
``1.14.1``  |                |``GCCcore/12.3.0``

### UCX-CUDA

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications  This module adds the UCX CUDA support. 

*homepage*: <http://www.openucx.org/>

version   |versionsuffix   |toolchain
----------|----------------|------------------
``1.10.0``|``-CUDA-11.3.1``|``GCCcore/10.3.0``
``1.11.0``|``-CUDA-11.4.1``|``GCCcore/11.2.0``
``1.11.2``|``-CUDA-11.4.1``|``GCCcore/11.2.0``
``1.11.2``|``-CUDA-11.5.2``|``GCCcore/11.2.0``
``1.12.1``|``-CUDA-11.7.0``|``GCCcore/11.3.0``
``1.13.1``|``-CUDA-12.0.0``|``GCCcore/12.2.0``
``1.14.1``|``-CUDA-12.1.1``|``GCCcore/12.3.0``

### ucx-py

UCX-Py is the Python interface for UCX, a low-level high-performance networking library. UCX and UCX-Py supports several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.

*homepage*: <https://ucx-py.readthedocs.io>

version   |toolchain
----------|--------------
``0.21.0``|``foss/2020b``

### UCX-ROCm

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications  This module adds the UCX ROCm support. 

*homepage*: <http://www.openucx.org/>

version   |versionsuffix  |toolchain
----------|---------------|------------------
``1.11.2``|``-ROCm-4.5.0``|``GCCcore/11.2.0``

### udocker

A basic user tool to execute simple docker containers in batch  or interactive systems without root privileges.

*homepage*: <https://github.com/indigo-dc/udocker>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.1``|``-Python-2.7.14``|``foss/2017b``
``1.1.3``|``-Python-2.7.14``|``intel/2018a``

### UDUNITS

UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.

*homepage*: <https://www.unidata.ucar.edu/software/udunits/>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.2.20``|``foss/2016a``, ``intel/2016b``
``2.2.24``|``intel/2017a``
``2.2.25``|``foss/2017b``, ``intel/2017b``
``2.2.26``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``, ``foss/2018b``, ``foss/2020a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``2.2.28``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### UFL

The Unified Form Language (UFL) is a domain specific language   for declaration of finite element discretizations of variational forms.   More precisely, it defines a flexible interface for choosing finite element   spaces and defining expressions for weak forms in a notation close to   mathematical notation.

*homepage*: <https://bitbucket.org/fenics-project/ufl>

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2018.1.0``|``-Python-3.6.4``|``foss/2018a``
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

### Ultralytics

Ultralytics YOLOv8 is a cutting-edge, state-of-the-art (SOTA) model that builds upon the success of previous YOLO versions and introduces new features and improvements to further boost performance and flexibility. YOLOv8 is designed to be fast, accurate, and simple to use, making it an excellent choice for a wide range of object detection and tracking, instance segmentation, image classification and pose estimation tasks.

*homepage*: <https://github.com/ultralytics/ultralytics>

version   |versionsuffix   |toolchain
----------|----------------|--------------
``8.0.92``|``-CUDA-11.7.0``|``foss/2022a``

### umap-learn

Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique  that can be used for visualisation similarly to t-SNE, but also for general non-linear  dimension reduction. 

*homepage*: <https://umap-learn.readthedocs.io/en/latest/>

version  |toolchain
---------|--------------------------------------------------------------
``0.4.6``|``foss/2020b``, ``fosscuda/2020b``
``0.5.3``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``

### UMI-tools

Tools for handling Unique Molecular Identifiers in NGS data sets

*homepage*: <https://umi-tools.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.6.6``|``foss/2018b``
``1.0.1``|``-Python-3.7.4``|``foss/2019b``
``1.0.1``|``-Python-3.8.2``|``foss/2020a``

### umi4cPackage

umi4cPackage is a processing and analysis pipeline for UMI-4C experiment.

*homepage*: <https://github.com/tanaylab/umi4cpackage>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20200116``|``-R-4.0.0`` |``foss/2020a``

### umis

Package for estimating UMI counts in Transcript Tag Counting data.

*homepage*: <https://github.com/vals/umis>

version  |toolchain
---------|---------------
``1.0.3``|``intel/2019a``

### UNAFold

The UNAFold package contains several programs for performing energy minimization and partition  function calculations on nucleic acid sequences.

*homepage*: <http://mfold.rna.albany.edu/?q=DINAMelt/software>

version|versionsuffix   |toolchain
-------|----------------|---------------
``3.8``|``-Perl-5.24.1``|``intel/2017a``

### uncertainty-calibration

Python library to measure the calibration error of models, including confidence intervals computed by Bootstrap resampling, and code to recalibrate models.

*homepage*: <https://github.com/p-lambda/verified_calibration>

version  |toolchain
---------|--------------
``0.0.9``|``foss/2021b``

### unicore-uftp

UNICORE Java-based client for UFTP

*homepage*: <https://sourceforge.net/projects/unicore/>

version  |versionsuffix|toolchain
---------|-------------|----------
``1.4.2``|``-Java-11`` |``system``

### Unicycler

Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets  where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore)  where it runs a miniasm+Racon pipeline. 

*homepage*: <https://github.com/rrwick/Unicycler>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.4.7``|``-Python-3.7.2``|``foss/2019a``
``0.4.8``|``-Python-3.8.2``|``gompi/2020a``
``0.4.9``|                 |``gompi/2021a``
``0.5.0``|                 |``gompi/2021b``

### Unidecode

Python library for lossy ASCII transliterations of Unicode text (port of Text::Unidecode Perl module)

*homepage*: <https://github.com/avian2/unidecode>

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.1.1``|``-Python-3.7.4``|``GCCcore/8.3.0``
``1.3.6``|                 |``GCCcore/11.3.0``

### unimap

Unimap is a fork of minimap2 optimized for assembly-to-reference alignment. It integrates the minigraph chaining algorithm and can align through long INDELs (up to 100kb by default) much faster than minimap2. Unimap is a better fit for resolving segmental duplications and is recommended over minimap2 for alignment between high-quality assemblies.  Unimap does not replace minimap2 for other types of alignment. It drops the support of multi-part index and short-read mapping. Its long-read alignment is different from minimap2 but is not necessarily better. Unimap is more of a specialized minimap2 at the moment.

*homepage*: <https://github.com/lh3/unimap>

version|toolchain
-------|------------------
``0.1``|``GCCcore/10.2.0``

### units

GNU Units converts quantities expressed in various systems of measurement to their equivalents in other systems of measurement. Like many similar programs, it can handle multiplicative scale changes. It can also handle nonlinear conversions such as Fahrenheit to Celsius or wire gauge, and it can convert from and to sums of units, such as converting between meters and feet plus inches.

*homepage*: <https://www.gnu.org/software/units/>

version |toolchain
--------|-----------------
``2.19``|``GCCcore/8.2.0``

### unixODBC

unixODBC provides a uniform interface between application and database driver

*homepage*: <https://www.unixodbc.org>

version  |toolchain
---------|--------------
``2.3.7``|``foss/2018b``

### unrar

RAR is a powerful archive manager.

*homepage*: <https://www.rarlab.com/>

version  |toolchain
---------|------------------
``5.6.1``|``GCCcore/7.3.0``
``5.7.3``|``GCCcore/8.2.0``
``6.0.2``|``GCCcore/10.2.0``

### UnZip

UnZip is an extraction utility for archives compressed in .zip format (also called "zipfiles"). Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own Zip program, our primary objectives have been portability and non-MSDOS functionality.

*homepage*: <http://www.info-zip.org/UnZip.html>

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``

### UQTk

The UQ Toolkit (UQTk) is a collection of libraries and tools  for the quantification of uncertainty in numerical model predictions.

*homepage*: <https://www.sandia.gov/UQToolkit/>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.1.0``|``-Python-3.7.4``|``intel/2019b``

### USEARCH

USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are  often orders of magnitude faster than BLAST.

*homepage*: <http://www.drive5.com/usearch/index.html>

version            |versionsuffix  |toolchain
-------------------|---------------|----------
``5.2.236``        |``-i86linux32``|``system``
``5.2.236-6.1.544``|``-i86linux32``|``system``
``6.1.544``        |``-i86linux32``|``system``
``11.0.667``       |``-i86linux32``|``system``

### UShER

UShER is now a package consisting of a family of programs for  rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes.

*homepage*: <https://github.com/yatisht/usher>

version  |toolchain
---------|---------------
``0.4.1``|``gompi/2020b``
``0.5.0``|``gompi/2021b``

### USPEX

USPEX (Universal Structure Predictor: Evolutionary Xtallography...  and in Russian "uspekh" means "success" - owing to the high success rate and many  useful results produced by this method) is a method developed by the Oganov  laboratory since 2004. The problem of crystal structure prediction is very old and  does, in fact, constitute the central problem of theoretical crystal chemistry.  USPEX can also be used for finding low-energy metastable phases, as well as stable  structures of nanoparticles, surface reconstructions, molecular packings in organic  crystals, and for searching for materials with desired physical (mechanical,  electronic) properties. The USPEX code is based on an efficient evolutionary  algorithm developed by A.R. Oganov's group, but also has options for using  alternative methods (random sampling, metadynamics, corrected particle swarm  optimization algorithms). USPEX is interfaced with many ab initio codes, such as  VASP, SIESTA, GULP, Quantum Espresso, CP2K, CASTEP, LAMMPS, and so on. 

*homepage*: <http://uspex-team.org/uspex/overview>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``9.4.4``|``-Python-2.7.12``|``intel/2016b``

### utf8proc

utf8proc is a small, clean C library that provides Unicode normalization, case-folding,  and other operations for data in the UTF-8 encoding.

*homepage*: <https://github.com/JuliaStrings/utf8proc>

version  |toolchain
---------|--------------------------------------------------------
``2.2.0``|``GCCcore/6.4.0``
``2.3.0``|``GCCcore/8.2.0``
``2.5.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7.0``|``GCCcore/11.3.0``
``2.8.0``|``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### util-linux

Set of Linux utilities

*homepage*: <https://www.kernel.org/pub/linux/utils/util-linux>

version   |toolchain
----------|--------------------------------------
``2.27.1``|``foss/2016a``, ``intel/2016a``
``2.28``  |``foss/2016a``, ``intel/2016a``
``2.28.1``|``intel/2016b``
``2.29``  |``foss/2016b``, ``intel/2016b``
``2.29.2``|``GCCcore/6.3.0``, ``intel/2017a``
``2.30``  |``GCCcore/6.4.0``
``2.30.1``|``GCCcore/6.3.0``, ``foss/2017a``
``2.31``  |``GCCcore/6.4.0``
``2.31.1``|``GCCcore/6.4.0``
``2.32``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.32.1``|``GCCcore/7.3.0``
``2.33``  |``GCCcore/8.2.0``
``2.34``  |``GCCcore/8.3.0``
``2.35``  |``GCCcore/9.3.0``
``2.36``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.37``  |``GCCcore/11.2.0``
``2.38``  |``GCCcore/11.3.0``
``2.38.1``|``GCCcore/12.2.0``
``2.39``  |``GCCcore/12.3.0``

## V


[V8](#v8) - [Vala](#vala) - [Valgrind](#valgrind) - [Vampir](#vampir) - [Vampire](#vampire) - [VarDict](#vardict) - [variant_tools](#variant_tools) - [VariantMetaCaller](#variantmetacaller) - [VarScan](#varscan) - [vartools](#vartools) - [VASP](#vasp) - [VAtools](#vatools) - [vawk](#vawk) - [VBZ-Compression](#vbz-compression) - [VCF-kit](#vcf-kit) - [vcflib](#vcflib) - [vcfnp](#vcfnp) - [VCFtools](#vcftools) - [vConTACT2](#vcontact2) - [VEGAS](#vegas) - [velocyto](#velocyto) - [Velvet](#velvet) - [VEP](#vep) - [verifyBamID](#verifybamid) - [VERSE](#verse) - [VESTA](#vesta) - [ViennaRNA](#viennarna) - [Vim](#vim) - [VirSorter](#virsorter) - [VirSorter2](#virsorter2) - [VirtualGL](#virtualgl) - [Virtuoso-opensource](#virtuoso-opensource) - [visdom](#visdom) - [vispr](#vispr) - [VisPy](#vispy) - [vitessce-python](#vitessce-python) - [vitessceR](#vitesscer) - [VMD](#vmd) - [VMTK](#vmtk) - [voltools](#voltools) - [vorbis-tools](#vorbis-tools) - [Voro++](#voro++) - [vsc-base](#vsc-base) - [vsc-install](#vsc-install) - [vsc-mympirun](#vsc-mympirun) - [vsc-mympirun-scoop](#vsc-mympirun-scoop) - [vsc-processcontrol](#vsc-processcontrol) - [VSEARCH](#vsearch) - [vt](#vt) - [VTK](#vtk) - [VTune](#vtune) - [VV](#vv) - [VXL](#vxl)


### V8

R interface to Google's open source JavaScript engine

*homepage*: <https://cran.r-project.org/web/packages/V8/>

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``2.2``  |``-R-3.5.1`` |``foss/2018b``
``2.3``  |``-R-3.6.0`` |``foss/2019a``, ``intel/2019a``
``3.2.0``|``-R-3.6.2`` |``foss/2019b``
``3.4.0``|``-R-4.0.0`` |``foss/2020a``

### Vala

Vala is a programming language using modern high level abstractions without imposing additional runtime requirements and without using a different ABI compared to applications and libraries written in C.

*homepage*: <https://wiki.gnome.org/Projects/Vala>

version   |toolchain
----------|------------------
``0.52.4``|``GCCcore/10.3.0``

### Valgrind

Valgrind: Debugging and profiling tools

*homepage*: <https://valgrind.org>

version   |toolchain
----------|---------------------------------------------------------------------------------
``3.11.0``|``foss/2016a``, ``intel/2016a``
``3.13.0``|``foss/2017b``, ``foss/2018a``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.14.0``|``foss/2018b``
``3.16.1``|``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``iimpi/2020a``
``3.17.0``|``gompi/2021a``
``3.18.1``|``gompi/2021b``, ``iimpi/2021b``
``3.19.0``|``gompi/2022a``
``3.20.0``|``gompi/2022a``

### Vampir

The Vampir software tool provides an simple-to-use framework that enables  developers to quickly display and analyze arbitrary program behavior at any level of detail.  The tool suite implements optimized event analysis algorithms and customizable displays that  enable fast and interactive rendering of very complex performance monitoring data.

*homepage*: <http://www.vampir.eu>

version  |versionsuffix|toolchain
---------|-------------|----------
``8.4.1``|             |``system``
``8.4.1``|``-demo``    |``system``

### Vampire

The Vampire Theorem Prover.

*homepage*: <https://vprover.github.io/>

version  |toolchain
---------|------------------
``4.5.1``|``GCCcore/10.2.0``

### VarDict

VarDict is an ultra sensitive variant caller for both single  and paired sample variant calling from BAM files.

*homepage*: <https://github.com/AstraZeneca-NGS/VarDict>

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.7``|``-Perl-5.28.0``|``foss/2018b``

### variant_tools

Variant tools is a software tool for the manipulation, annotation, selection, simulation, and  analysis of variants in the context of next-gen sequencing analysis. Unlike some other tools used for Next-Gen  sequencing analysis, variant tools is project based and provides a whole set of tools to manipulate and analyze  genetic variants.

*homepage*: <https://vatlab.github.io/vat-docs/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.3``|``-Python-3.7.4``|``foss/2019b``

### VariantMetaCaller

VariantMetaCaller automatically integrates variant calling pipelines into a better performing  overall model that also predicts accurate variant probabilities.

*homepage*: <http://bioinformatics.mit.bme.hu/VariantMetaCaller/>

version|toolchain
-------|---------------
``1.0``|``intel/2017a``

### VarScan

Variant calling and somatic mutation/CNV detection for next-generation sequencing data

*homepage*: <https://github.com/dkoboldt/varscan>

version  |versionsuffix     |toolchain
---------|------------------|----------
``2.3.6``|``-Java-1.7.0_80``|``system``
``2.4.1``|``-Java-1.7.0_80``|``system``
``2.4.4``|``-Java-1.8``     |``system``
``2.4.4``|``-Java-11``      |``system``

### vartools

Command line utility that provides tools for processing   and analyzing astronomical time series data.

*homepage*: <https://www.astro.princeton.edu/~jhartman/vartools.html>

version |toolchain
--------|-------------------------------
``1.35``|``foss/2016b``, ``intel/2016b``

### VASP

The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.

*homepage*: <https://www.vasp.at>

version  |toolchain
---------|-------------------------
``5.4.1``|``intel/2016.02-GCC-4.9``
``6.3.2``|``nvofbf/2022.07``

### VAtools

VAtools is a python package that includes several tools to annotate VCF files  with data from other tools. 

*homepage*: <https://vatools.readthedocs.io/en/latest/>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.1``|``-Python-3.6.6``|``foss/2018b``

### vawk

An awk-like VCF parser  

*homepage*: <https://github.com/cc2qe/vawk>

version  |versionsuffix     |toolchain
---------|------------------|------------------
``0.0.1``|``-Python-2.7.18``|``GCCcore/10.2.0``

### VBZ-Compression

VBZ compression HDF5 plugin for nanopolish

*homepage*: <https://github.com/nanoporetech/vbz_compression>

version  |toolchain
---------|---------------
``1.0.1``|``gompi/2020b``

### VCF-kit

VCF-kit is a command-line based collection of utilities for performing analysis on  Variant Call Format (VCF) files.

*homepage*: <https://github.com/AndersenLab/VCF-kit>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.1.6``|``-Python-2.7.15``|``intel/2018b``

### vcflib

vcflib provides methods to manipulate and interpret sequence variation as it can be  described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended  to concisely describe reference-indexed genetic variations between individuals.

*homepage*: <https://github.com/vcflib/vcflib>

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``1.0.2``|                 |``GCC/10.2.0``, ``GCC/10.3.0``
``1.0.2``|``-Python-3.8.2``|``GCC/9.3.0``
``1.0.3``|``-R-4.1.0``     |``foss/2021a``
``1.0.3``|``-R-4.1.2``     |``foss/2021b``

### vcfnp

Load data from a VCF (variant call format) file into numpy arrays,   and (optionally) from there into an HDF5 file.

*homepage*: <https://github.com/alimanfoo/vcfnp>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.0``|``-Python-2.7.11``|``foss/2016a``

### VCFtools

The aim of VCFtools is to provide   easily accessible methods for working with complex   genetic variation data in the form of VCF files.

*homepage*: <https://vcftools.github.io>

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------------------------------------------------------------------------------------------------------
``0.1.14``|``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``0.1.15``|``-Perl-5.24.0``|``foss/2016b``
``0.1.15``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``0.1.15``|``-Perl-5.26.1``|``foss/2018a``
``0.1.16``|                |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/12.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``0.1.16``|``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``

### vConTACT2

vConTACT2 is a tool to perform guilt-by-contig-association classification of  viral genomic sequence data.

*homepage*: <https://bitbucket.org/MAVERICLab/vcontact2>

version   |toolchain
----------|--------------
``0.11.3``|``foss/2022a``

### VEGAS

VEGAS (Versatile Gene-based Association Study) is  a free program for performing gene-based tests for association using  the results from genetic association studies 

*homepage*: <http://gump.qimr.edu.au/VEGAS/>

version   |toolchain
----------|----------
``0.8.27``|``system``

### velocyto

Velocyto is a library for the analysis of RNA velocity.

*homepage*: <https://velocyto.org/velocyto.py/>

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------
``0.17.17``|                 |``foss/2021a``, ``foss/2022a``
``0.17.17``|``-Python-3.8.2``|``intel/2020a``

### Velvet

Sequence assembler for very short reads

*homepage*: <https://github.com/dzerbino/velvet/>

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------------------------------
``1.2.10``|``-mt-kmer_191``|``GCC/11.2.0``, ``GCC/8.3.0``, ``foss/2018a``, ``foss/2018b``
``1.2.10``|``-mt-kmer_37`` |``intel/2017a``

### VEP

Variant Effect Predictor (VEP) determines the effect of your  variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,  transcripts, and protein sequence, as well as regulatory regions.  Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently  used routines in VEP.

*homepage*: <https://www.ensembl.org/info/docs/tools/vep>

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------
``93.4`` |``-Perl-5.26.1``|``intel/2018a``
``94.0`` |``-Perl-5.28.0``|``foss/2018b``
``94.5`` |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``95.0`` |``-Perl-5.28.0``|``foss/2018b``
``96.0`` |``-Perl-5.28.1``|``foss/2019a``
``103.1``|                |``GCC/10.2.0``
``105``  |                |``GCC/11.2.0``
``107``  |                |``GCC/11.3.0``

### verifyBamID

verifyBamID is a software that verifies whether the reads in particular file match previously known   genotypes for an individual (or group of individuals), and checks whether the reads are contaminated as a mixture of   two samples. verifyBamID can detect sample contamination and swaps when external genotypes are available. When   external genotypes are not available, verifyBamID still robustly detects sample swaps. 

*homepage*: <http://csg.sph.umich.edu/kang/verifyBamID/index.html>

version  |toolchain
---------|--------------
``1.1.3``|``foss/2016a``

### VERSE

A versatile and efficient RNA-Seq read counting tool

*homepage*: <https://github.com/qinzhu/VERSE>

version  |toolchain
---------|--------------
``0.1.5``|``foss/2016b``

### VESTA

VESTA is a 3D visualization program for structured models,    volumetric data such as electron/nuclear densities,    and crystal morphologies. 

*homepage*: <http://www.jp-minerals.org/vesta>

version  |versionsuffix|toolchain
---------|-------------|----------
``3.5.8``|``-gtk3``    |``system``

### ViennaRNA

The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.

*homepage*: <https://www.tbi.univie.ac.at/RNA/>

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.2.3`` |                  |``intel/2016b``
``2.3.4`` |                  |``foss/2016b``
``2.3.5`` |                  |``intel/2017a``
``2.4.10``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.4.11``|``-Python-3.6.6`` |``foss/2018b``
``2.4.14``|``-Python-3.6.6`` |``foss/2018b``
``2.5.0`` |                  |``foss/2021b``
``2.5.1`` |                  |``foss/2021b``

### Vim

Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set. 

*homepage*: <http://www.vim.org>

version     |versionsuffix     |toolchain
------------|------------------|------------------
``8.0``     |``-Python-2.7.11``|``foss/2016a``
``8.1.0483``|``-Python-2.7.15``|``foss/2018b``
``8.1.1209``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``9.0.0950``|                  |``GCCcore/11.3.0``
``9.0.1434``|                  |``GCCcore/12.2.0``

### VirSorter

VirSorter: mining viral signal from microbial genomic data.

*homepage*: <https://github.com/simroux/VirSorter>

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20160601``|``-Perl-5.22.1``|``foss/2016a``

### VirSorter2

VirSorter2 applies a multi-classifier, expert-guided approach to detect diverse DNA and  RNA virus genomes.

*homepage*: <https://github.com/jiarong/VirSorter2>

version  |toolchain
---------|--------------
``2.2.4``|``foss/2022a``

### VirtualGL

VirtualGL is an open source toolkit that gives any Linux or  Unix remote display software the ability to run OpenGL applications with full  hardware acceleration.

*homepage*: <https://virtualgl.org/>

version  |toolchain
---------|-----------------
``2.6.1``|``foss/2018b``
``2.6.2``|``GCCcore/9.3.0``
``3.0``  |``GCC/11.2.0``

### Virtuoso-opensource

Virtuoso is a high-performance and scalable Multi-Model RDBMS, Data Integration Middleware, Linked Data Deployment, and HTTP Application Server Platform.

*homepage*: <https://github.com/openlink/virtuoso-opensource>

version    |toolchain
-----------|--------------
``7.2.6.1``|``GCC/10.3.0``

### visdom

A flexible tool for creating, organizing, and sharing visualizations of live, rich data. Supports Torch and Numpy.

*homepage*: <https://github.com/fossasia/visdom>

version  |toolchain
---------|--------------
``0.2.4``|``foss/2022a``

### vispr

VISPR - A visualization framework for CRISPR data. 

*homepage*: <https://pypi.org/project/vispr/>

version   |toolchain
----------|--------------
``0.4.14``|``foss/2022a``

### VisPy

VisPy is a high-performance interactive 2D/3D data visualization library  leveraging the computational power of modern Graphics Processing Units (GPUs) through the  OpenGL library to display very large datasets.

*homepage*: <https://vispy.org>

version  |toolchain
---------|-------------------------------
``0.6.6``|``foss/2020b``, ``intel/2020b``

### vitessce-python

Python API and Jupyter widget facilitating interactive visualization of spatial single-cell data with Vitessce.

*homepage*: <https://github.com/vitessce/vitessce-python>

version     |toolchain
------------|--------------
``20230222``|``foss/2022a``

### vitessceR

Vitessce is a visual integration tool for exploration of spatial single-cell experiments.

*homepage*: <https://github.com/vitessce/vitessceR>

version            |versionsuffix|toolchain
-------------------|-------------|--------------
``0.99.0-20230110``|``-R-4.2.1`` |``foss/2022a``

### VMD

VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular  systems using 3-D graphics and built-in scripting.

*homepage*: <https://www.ks.uiuc.edu/Research/vmd>

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------
``1.9.3``   |``-Python-2.7.12``|``intel/2016b``
``1.9.3``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2018a``
``1.9.3``   |``-Python-2.7.15``|``intel/2018b``
``1.9.4a43``|``-Python-3.7.4`` |``fosscuda/2019b``
``1.9.4a51``|                  |``foss/2020b``, ``fosscuda/2020b``
``1.9.4a57``|                  |``foss/2022a``

### VMTK

vmtk is a collection of libraries and tools for 3D reconstruction, geometric analysis, mesh generation and surface data analysis for image-based modeling of blood vessels. 

*homepage*: <https://vmtk.org>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.4.0``|``-Python-3.7.4``|``foss/2019b``

### voltools

CUDA-accelerated numpy 3D affine transformations

*homepage*: <https://github.com/the-lay/voltools>

version  |toolchain
---------|--------------
``0.4.2``|``foss/2020b``

### vorbis-tools

Command-line tools for creating and playing Ogg Vorbis files.

*homepage*: <https://xiph.org/vorbis/>

version  |toolchain
---------|----------------------------------------------------------
``1.4.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

### Voro++

Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles.

*homepage*: <http://math.lbl.gov/voro++/>

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.4.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2019b``, ``intel/2016a``, ``intel/2019b``

### vsc-base

Basic Python libraries used by UGent's HPC group

*homepage*: <http://hpcugent.github.com/vsc-base/>

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.6.9`` |                  |``system``
``1.7.3`` |                  |``system``
``2.0.4`` |                  |``system``
``2.1.2`` |                  |``system``
``2.4.2`` |                  |``system``
``2.4.17``|``-Python-2.7.11``|``intel/2016a``
``2.5.1`` |                  |``system``
``2.5.1`` |``-Python-2.7.11``|``intel/2016a``
``2.5.1`` |``-Python-2.7.12``|``intel/2016b``
``2.5.8`` |                  |``system``
``2.8.0`` |                  |``system``
``2.8.1`` |                  |``system``
``2.8.3`` |                  |``system``

### vsc-install

vsc-install provides shared setuptools functions and classes for Python libraries  developed by UGent's HPC group

*homepage*: <https://pypi.python.org/pypi/vsc-install>

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.9.18`` |``-Python-2.7.11``|``intel/2016a``
``0.10.6`` |                  |``system``
``0.10.6`` |``-Python-2.7.11``|``intel/2016a``
``0.10.11``|``-Python-2.7.11``|``intel/2016a``
``0.10.11``|``-Python-2.7.12``|``intel/2016b``
``0.10.25``|                  |``system``
``0.10.26``|                  |``system``
``0.11.1`` |                  |``system``
``0.11.2`` |                  |``system``

### vsc-mympirun

mympirun is a tool to make it easier for users of HPC clusters to run MPI programs with good performance.

*homepage*: <https://github.com/hpcugent/vsc-mympirun>

version    |versionsuffix                     |toolchain
-----------|----------------------------------|---------------
``3.2.1``  |                                  |``system``
``3.3.0``  |                                  |``system``
``3.4.2``  |                                  |``system``
``3.4.2``  |``-Python-2.7.11-vsc-base-2.4.17``|``intel/2016a``
``3.4.2``  |``-Python-2.7.11-vsc-base-2.5.1`` |``intel/2016a``
``3.4.2``  |``-vsc-base-2.4.2``               |``system``
``3.4.3``  |``-Python-2.7.12``                |``intel/2016b``
``4.0.0``  |                                  |``system``
``4.0.0b0``|                                  |``system``
``4.0.1``  |                                  |``system``
``4.0.2``  |                                  |``system``
``4.1.4``  |                                  |``system``
``4.1.5``  |                                  |``system``
``4.1.6``  |                                  |``system``
``4.1.8``  |                                  |``system``
``4.1.9``  |                                  |``system``
``5.1.0``  |                                  |``system``
``5.2.0``  |                                  |``system``
``5.2.2``  |                                  |``system``
``5.2.3``  |                                  |``system``
``5.2.4``  |                                  |``system``
``5.2.5``  |                                  |``system``
``5.2.6``  |                                  |``system``
``5.2.7``  |                                  |``system``
``5.2.9``  |                                  |``system``
``5.2.10`` |                                  |``system``
``5.2.11`` |                                  |``system``
``5.3.0``  |                                  |``system``
``5.3.1``  |                                  |``system``

### vsc-mympirun-scoop

VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent.

*homepage*: <http://hpcugent.github.com/vsc-mympirun/>

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.4.1``|``-Python-2.7.12``|``intel/2016b``

### vsc-processcontrol

vsc-processcontrol is a module to abstract process control like scheduler settings and affinity  from actual implementations like vsc.affinity and psutil.

*homepage*: <http://hpcugent.github.com/vsc-processcontrol/>

version|versionsuffix      |toolchain
-------|-------------------|----------
``1.0``|                   |``system``
``1.0``|``-vsc-base-2.1.2``|``system``

### VSEARCH

VSEARCH supports de novo and reference based chimera detection,  clustering, full-length and prefix dereplication, rereplication,  reverse complementation, masking, all-vs-all pairwise global alignment,  exact and global alignment searching, shuffling, subsampling and sorting.  It also supports FASTQ file analysis, filtering,  conversion and merging of paired-end reads.

*homepage*: <https://github.com/torognes/vsearch>

version   |toolchain
----------|----------------------------------------
``2.9.1`` |``foss/2018b``
``2.13.4``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.15.0``|``GCC/9.3.0``
``2.18.0``|``GCC/10.2.0``
``2.21.1``|``GCC/10.3.0``
``2.22.1``|``GCC/11.3.0``

### vt

A tool set for short variant discovery in genetic sequence data.

*homepage*: <http://genome.sph.umich.edu/wiki/Vt>

version    |toolchain
-----------|-----------------------------------------------------------------------------
``0.57721``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``, ``foss/2018b``

### VTK

The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.

*homepage*: <https://www.vtk.org>

version      |versionsuffix     |toolchain
-------------|------------------|---------------------------------------------------
``6.3.0``    |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``6.3.0``    |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``7.0.0``    |``-Python-2.7.12``|``intel/2016b``
``7.1.0``    |``-Python-2.7.12``|``intel/2016b``
``7.1.1``    |``-Python-2.7.13``|``intel/2017a``
``8.0.1``    |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``8.1.0``    |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``8.1.0``    |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``8.1.1``    |``-Python-2.7.14``|``intel/2018a``
``8.1.1``    |``-Python-2.7.15``|``foss/2018b``
``8.1.1``    |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``8.2.0``    |                  |``foss/2021a``
``8.2.0``    |``-Python-2.7.15``|``foss/2019a``
``8.2.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``8.2.0``    |``-Python-3.7.4`` |``foss/2019b``
``8.2.0``    |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``9.0.1``    |                  |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``
``9.1.0``    |                  |``foss/2021b``
``9.2.0.rc2``|                  |``foss/2022a``
``9.2.2``    |                  |``foss/2022a``
``9.2.6``    |                  |``foss/2022b``

### VTune

Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.

*homepage*: <https://software.intel.com/en-us/vtune>

version          |toolchain
-----------------|----------
``2013_update6`` |``system``
``2013_update8`` |``system``
``2013_update9`` |``system``
``2013_update10``|``system``
``2013_update11``|``system``
``2013_update12``|``system``
``2016_update3`` |``system``
``2017``         |``system``
``2017_update1`` |``system``
``2017_update2`` |``system``
``2017_update3`` |``system``
``2018_update1`` |``system``
``2018_update2`` |``system``
``2018_update3`` |``system``
``2019_update2`` |``system``
``2019_update3`` |``system``
``2019_update5`` |``system``
``2020_update3`` |``system``
``2021.6.0``     |``system``
``2021.9.0``     |``system``
``2022.0.0``     |``system``
``2022.2.0``     |``system``
``2022.3.0``     |``system``

### VV

VV is an open-source and cross platform image viewer, designed for fast and  simple visualization of spatio-temporal images: 2D, 2D+t, 3D and 3D+t (or 4D)  images. Only the command-line (clitk) tools are build.

*homepage*: <https://www.creatis.insa-lyon.fr/rio/vv>

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2018.09.19``|``-Python-3.6.6``|``foss/2018b``

### VXL

A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.

*homepage*: <https://sf.net/projects/vxl>

version   |toolchain
----------|--------------
``1.18.0``|``foss/2018a``

## W


[wandb](#wandb) - [Wannier90](#wannier90) - [WannierTools](#wanniertools) - [Wayland](#wayland) - [Waylandpp](#waylandpp) - [WCSLIB](#wcslib) - [WCT](#wct) - [wcwidth](#wcwidth) - [webin-cli](#webin-cli) - [WebKitGTK+](#webkitgtk+) - [WebSocket++](#websocket++) - [WEKA](#weka) - [wget](#wget) - [wgsim](#wgsim) - [WHAM](#wham) - [WhatsHap](#whatshap) - [wheel](#wheel) - [WIEN2k](#wien2k) - [WildMagic](#wildmagic) - [Winnowmap](#winnowmap) - [WisecondorX](#wisecondorx) - [WISExome](#wisexome) - [wkhtmltopdf](#wkhtmltopdf) - [worker](#worker) - [wpebackend-fdo](#wpebackend-fdo) - [WPS](#wps) - [wrapt](#wrapt) - [WRF](#wrf) - [WRF-Fire](#wrf-fire) - [wrf-python](#wrf-python) - [wtdbg2](#wtdbg2) - [wxPropertyGrid](#wxpropertygrid) - [wxPython](#wxpython) - [wxWidgets](#wxwidgets)


### wandb

CLI and Python API for Weights and Biases (wandb), a tool for visualizing and tracking your machine learning experiments.

*homepage*: <https://www.wandb.com/>

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.8.30``|``-Python-3.7.4``|``gcccuda/2019b``
``0.13.4``|                 |``GCCcore/11.3.0``
``0.13.6``|                 |``GCC/11.3.0``

### Wannier90

A tool for obtaining maximally-localised Wannier functions

*homepage*: <http://www.wannier.org>

version    |versionsuffix|toolchain
-----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2``    |             |``intel/2016.02-GCC-4.9``
``2.0.1``  |             |``intel/2016.02-GCC-4.9``, ``intel/2016a``
``2.0.1.1``|``-abinit``  |``intel/2018b``
``2.1.0``  |             |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``3.0.0``  |             |``intel/2018b``
``3.1.0``  |             |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``gomkl/2021a``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022a``

### WannierTools

WannierTools is an open source software that studies the physical properties of given tight-binding  model.

*homepage*: <http://www.wanniertools.com>

version  |toolchain
---------|---------------
``2.3.0``|``intel/2018a``
``2.5.1``|``intel/2020b``

### Wayland

Wayland is a project to define a protocol for a compositor to talk to  its clients as well as a library implementation of the protocol.  The  compositor can be a standalone display server running on Linux kernel  modesetting and evdev input devices, an X application, or a wayland  client itself.  The clients can be traditional applications, X servers  (rootless or fullscreen) or other display servers. 

*homepage*: <https://wayland.freedesktop.org/>

version   |toolchain
----------|--------------------------------------
``1.20.0``|``GCCcore/11.3.0``
``1.21.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### Waylandpp

Wayland is an object oriented display protocol, which features request and events. Requests can be seen as method calls on certain objects, whereas events can be seen as signals of an object. This makes the Wayland protocol a perfect candidate for a C++ binding.  The goal of this library is to create such a C++ binding for Wayland using the most modern C++ technology currently available, providing an simple to use C++ API to Wayland.

*homepage*: <https://github.com/NilsBrause/waylandpp>

version  |toolchain
---------|--------------------------------------
``1.0.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### WCSLIB

The FITS "World Coordinate System" (WCS) standard defines keywords  and usage that provide for the description of astronomical coordinate systems in a  FITS image header.

*homepage*: <https://www.atnf.csiro.au/people/mcalabre/WCS/>

version |toolchain
--------|--------------
``7.11``|``GCC/11.2.0``

### WCT

NOAA's Weather and Climate Toolkit (WCT) is free, platform independent software  distributed from NOAA's National Centers for Environmental Information (NCEI).  The WCT allows the visualization and data export of weather and climate data,  including Radar, Satellite and Model data. The WCT also provides access to  weather/climate web services provided from NCEI and other organizations. 

*homepage*: <https://www.ncdc.noaa.gov/wct/index.php>

version  |versionsuffix|toolchain
---------|-------------|----------
``4.6.0``|``-Java-11`` |``system``

### wcwidth

wcwidth is a low-level Python library to simplify Terminal emulation.

*homepage*: <https://github.com/jquast/wcwidth>

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.1.7``|``-Python-2.7.11``|``foss/2016a``
``0.1.7``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.1.7``|``-Python-3.5.1`` |``foss/2016a``
``0.1.7``|``-Python-3.5.2`` |``intel/2016b``

### webin-cli

The Webin command line submission interface can be used to validate,  upload and submit files to the European Nucleotide Archive (ENA)

*homepage*: <https://github.com/enasequence/webin-cli>

version  |toolchain
---------|----------
``1.8.9``|``system``

### WebKitGTK+

WebKitGTK+ is a full-featured port of the WebKit rendering engine, suitable for projects requiring any kind of web integration, from hybrid HTML/CSS applications to full-fledged web browsers. It offers WebKit's full functionality and is useful in a wide range of systems from desktop computers to embedded systems like phones, tablets, and televisions.

*homepage*: <https://webkitgtk.org/>

version   |toolchain
----------|-----------------------------
``2.24.1``|``GCC/8.2.0-2.31.1``
``2.27.4``|``GCC/10.3.0``, ``GCC/8.3.0``
``2.37.1``|``GCC/11.2.0``

### WebSocket++

WebSocket++ is an open source (BSD license) header only C++ library  that implements RFC6455 The WebSocket Protocol. 

*homepage*: <https://www.zaphoyd.com/websocketpp/>

version  |toolchain
---------|---------------
``0.8.1``|``gompi/2019a``

### WEKA

Weka is a collection of machine learning algorithms for data mining tasks.   The algorithms can either be applied directly to a dataset or called from your own Java code.   Weka contains tools for data pre-processing, classification, regression, clustering,   association rules, and visualization. It is also well-suited for developing new machine   learning schemes.

*homepage*: <http://www.cs.waikato.ac.nz/ml/weka/index.html>

version   |versionsuffix     |toolchain
----------|------------------|----------
``3.6.12``|``-Java-1.7.0_80``|``system``
``3.7.0`` |``-Java-1.7.0_80``|``system``

### wget

GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,  the most widely-used Internet protocols. It is a non-interactive commandline tool,  so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.

*homepage*: <https://www.gnu.org/software/wget>

version   |toolchain
----------|-------------------------------------
``1.17.1``|``foss/2016a``, ``foss/2016b``
``1.19.4``|``GCCcore/6.4.0``
``1.20.1``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``1.20.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.21.1``|``GCCcore/10.3.0``
``1.21.2``|``GCCcore/11.2.0``
``1.21.3``|``GCCcore/11.3.0``

### wgsim

Wgsim is a small tool for simulating sequence reads from a reference genome.

*homepage*: <https://github.com/lh3/wgsim/>

version     |toolchain
------------|------------------------------
``20111017``|``GCC/10.2.0``, ``GCC/11.2.0``

### WHAM

An implementation of WHAM:  the Weighted Histogram Analysis Method

*homepage*: <http://membrane.urmc.rochester.edu/?page_id=126>

version     |versionsuffix|toolchain
------------|-------------|---------------
``2.0.9.1`` |             |``intel/2019a``
``2.0.9.1`` |``-kj_mol``  |``intel/2019a``
``2.0.10.2``|             |``intel/2020a``
``2.0.10.2``|``-kj_mol``  |``intel/2020a``

### WhatsHap

WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly. It is especially suitable for long reads, but works also well with short reads.

*homepage*: <https://whatshap.readthedocs.io>

version|toolchain
-------|------------------------------
``1.1``|``foss/2020b``, ``foss/2021a``
``1.4``|``foss/2021b``
``1.7``|``foss/2022a``

### wheel

A built-package format for Python.

*homepage*: <https://pypi.python.org/pypi/wheel>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------------
``0.29.0``|``-Python-2.7.11``|``foss/2016a``
``0.29.0``|``-Python-3.5.1`` |``foss/2016a``
``0.30.0``|``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``|``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.31.0``|``-Python-2.7.14``|``fosscuda/2018a``, ``intel/2018a``
``0.31.0``|``-Python-3.6.4`` |``foss/2018a``, ``fosscuda/2018a``
``0.31.1``|``-Python-2.7.15``|``fosscuda/2018b``
``0.31.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``

### WIEN2k

The program package WIEN2k allows to perform electronic structure calculations of solids using density functional theory (DFT). It is based on the full-potential (linearized) augmented plane-wave ((L)APW) + local orbitals (lo) method, one among the most accurate schemes for band structure calculations. WIEN2k is an all-electron scheme including relativistic effects and has many features.

*homepage*: <http://www.wien2k.at/>

version |toolchain
--------|------------------------------------------------
``17.1``|``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``18.1``|``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``19.1``|``intel/2019a``
``19.2``|``intel/2020b``
``21.1``|``intel/2021a``, ``intel/2021b``

### WildMagic

Wild Magic 5.17

*homepage*: <https://www.geometrictools.com/index.html>

version |toolchain
--------|----------------------------------
``5.17``|``GCCcore/10.3.0``, ``foss/2018b``

### Winnowmap

Winnowmap is a long-read mapping algorithm, and a result of our exploration  into superior minimizer sampling techniques.

*homepage*: <https://github.com/marbl/Winnowmap>

version|toolchain
-------|-------------
``1.0``|``GCC/8.3.0``

### WisecondorX

WisecondorX -- an evolved WISECONDOR

*homepage*: <https://github.com/CenterForMedicalGeneticsGhent/WisecondorX>

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.6``|``-Python-3.8.2``|``foss/2020a``

### WISExome

A within-sample comparison approach to detect copy number variations in whole exome sequencing data

*homepage*: <https://github.com/VUmcCGP/wisexome>

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20180517``|``-Python-2.7.14``|``intel/2018a``

### wkhtmltopdf

wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render   HTML into PDF and various image formats using the Qt WebKit rendering engine. These run entirely headless   and do not require a display or display service.

*homepage*: <http://wkhtmltopdf.org/>

version   |versionsuffix    |toolchain
----------|-----------------|----------
``0.12.3``|``-Linux-x86_64``|``system``

### worker

The Worker framework has been developed to help deal with parameter exploration experiments  that would otherwise result in many jobs, forcing the user resort to scripting to retain her sanity;  see also https://vscentrum.be/neutral/documentation/cluster-doc/running-jobs/worker-framework.

*homepage*: <https://github.com/gjbex/worker>

version   |versionsuffix   |toolchain
----------|----------------|------------------------------
``1.6.4`` |                |``intel/2016a``
``1.6.5`` |                |``intel/2016a``
``1.6.6`` |                |``intel/2016b``
``1.6.7`` |``-intel-2016b``|``system``
``1.6.7`` |``-intel-2017a``|``system``
``1.6.7`` |``-intel-2017b``|``system``
``1.6.8`` |``-intel-2018a``|``system``
``1.6.8`` |``-intel-2018b``|``system``
``1.6.11``|                |``intel/2019b``
``1.6.12``|                |``foss/2019a``, ``foss/2021b``
``1.6.13``|                |``iimpi/2021b``

### wpebackend-fdo

WPE WebKit allows embedders to create simple and performant  systems based on Web platform technologies. It is a WebKit port designed with  flexibility and hardware acceleration in mind, leveraging common 3D graphics  APIs for best performance.

*homepage*: <https://wpewebkit.org/>

version   |toolchain
----------|------------------
``1.13.1``|``GCCcore/11.2.0``

### WPS

WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is  a next-generation mesoscale numerical weather prediction system designed to serve both operational  forecasting and atmospheric research needs.

*homepage*: <http://www.wrf-model.org>

version  |versionsuffix|toolchain
---------|-------------|---------------
``3.8.0``|``-dmpar``   |``intel/2016a``
``3.9.1``|``-dmpar``   |``foss/2020b``
``4.0.1``|``-dmpar``   |``intel/2018b``
``4.0.2``|``-dmpar``   |``foss/2018b``
``4.1``  |``-dmpar``   |``intel/2019b``
``4.2``  |``-dmpar``   |``foss/2020b``
``4.3.1``|``-dmpar``   |``foss/2021a``
``4.4``  |``-dmpar``   |``foss/2022a``

### wrapt

The aim of the wrapt module is to provide a transparent object proxy for Python, which can be used as the basis for the construction of function wrappers and decorator functions.

*homepage*: <https://pypi.org/project/wrapt/>

version   |toolchain
----------|-------------------------------
``1.15.0``|``foss/2022a``, ``intel/2022a``

### WRF

The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale  numerical weather prediction system designed to serve both operational forecasting and atmospheric  research needs.

*homepage*: <https://www.wrf-model.org>

version    |versionsuffix|toolchain
-----------|-------------|----------------------------------------------------------------
``3.8.0``  |``-dmpar``   |``intel/2016a``, ``intel/2016b``
``3.9.1.1``|``-dmpar``   |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``4.0.1``  |``-dmpar``   |``intel/2018b``
``4.0.2``  |``-dmpar``   |``foss/2018b``
``4.1.3``  |``-dm+sm``   |``intel/2019b``
``4.1.3``  |``-dmpar``   |``foss/2019b``, ``foss/2020a``, ``intel/2019b``
``4.2.2``  |``-dmpar``   |``foss/2020b``
``4.3``    |``-dmpar``   |``foss/2021a``
``4.4``    |``-dmpar``   |``foss/2022a``
``4.4.1``  |``-dmpar``   |``foss/2022b``

### WRF-Fire

WRF-Fire combines the Weather Research and Forecasting model (WRF) with a fire code implementing a  surface fire behavior model, called SFIRE, based on semi-empirical formulas calculate the rate of spread of the fire  line (the interface between burning and unignited fuel) based on fuel properties, wind velocities from WRF, and  terrain slope. The fire spread is implemented by the level set method.

*homepage*: <http://www.openwfm.org/wiki/WRF-SFIRE>

version     |versionsuffix|toolchain
------------|-------------|---------------
``20170221``|``-dmpar``   |``intel/2016b``

### wrf-python

A collection of diagnostic and interpolation routines for use with output from  the Weather Research and Forecasting (WRF-ARW) Model.

*homepage*: <https://wrf-python.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.2.0``|``-Python-3.6.4``|``intel/2018a``
``1.3.1``|``-Python-3.6.6``|``intel/2018b``

### wtdbg2

Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio  or Oxford Nanopore Technologies (ONT). It assembles raw reads without error  correction and then builds the consensus from intermediate assembly output. 

*homepage*: <https://github.com/ruanjue/wtdbg2>

version|toolchain
-------|---------------------------------------------------------
``2.3``|``GCC/7.3.0-2.30``
``2.5``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### wxPropertyGrid

wxPropertyGrid is a property sheet control for wxWidgets. In other words, it is  a specialized two-column grid for editing properties such as strings, numbers, flagsets, string arrays, and colours.

*homepage*: <http://wxpropgrid.sourceforge.net/>

version   |toolchain
----------|-------------
``1.4.15``|``GCC/4.9.2``

### wxPython

Wraps the wxWidgets C++ toolkit and provides access to the user interface portions of the wxWidgets  API, enabling Python applications to have a native GUI on Windows, Macs or Unix systems, with a native look and feel  and requiring very little (if any) platform specific code.

*homepage*: <https://www.wxpython.org/>

version        |versionsuffix     |toolchain
---------------|------------------|-------------------------------
``3.0.2.0``    |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``3.0.2.0``    |``-Python-2.7.12``|``intel/2016b``
``3.0.2.0``    |``-Python-2.7.13``|``intel/2017a``
``3.0.2.0``    |``-Python-2.7.14``|``intel/2017b``
``3.0.2.0``    |``-Python-2.7.15``|``foss/2018b``
``4.0.4``      |``-Python-2.7.15``|``GCC/8.2.0-2.31.1``
``4.0.4``      |``-Python-3.7.2`` |``GCC/8.2.0-2.31.1``
``4.0.7.post2``|``-Python-3.7.4`` |``GCC/8.3.0``
``4.1.1``      |                  |``foss/2021a``
``4.2.0``      |                  |``foss/2021b``

### wxWidgets

wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI.

*homepage*: <https://www.wxwidgets.org>

version    |toolchain
-----------|------------------------------
``3.0.3``  |``foss/2018a``
``3.0.4``  |``GCC/8.2.0-2.31.1``
``3.1.3``  |``GCC/8.3.0``
``3.1.4``  |``GCC/10.2.0``
``3.1.5``  |``GCC/10.3.0``, ``GCC/11.2.0``
``3.2.0``  |``GCC/11.2.0``
``3.2.1``  |``GCC/11.3.0``
``3.2.2.1``|``GCCcore/12.2.0``

## X


[X11](#x11) - [x13as](#x13as) - [x264](#x264) - [x265](#x265) - [XALT](#xalt) - [xarray](#xarray) - [xbitmaps](#xbitmaps) - [xcb-proto](#xcb-proto) - [xcb-util](#xcb-util) - [xcb-util-image](#xcb-util-image) - [xcb-util-keysyms](#xcb-util-keysyms) - [xcb-util-renderutil](#xcb-util-renderutil) - [xcb-util-wm](#xcb-util-wm) - [xCell](#xcell) - [XCFun](#xcfun) - [xclip](#xclip) - [XCrySDen](#xcrysden) - [xdotool](#xdotool) - [Xerces-C++](#xerces-c++) - [xESMF](#xesmf) - [xextproto](#xextproto) - [xf86vidmodeproto](#xf86vidmodeproto) - [XGBoost](#xgboost) - [XGrafix](#xgrafix) - [xineramaproto](#xineramaproto) - [XKeyboardConfig](#xkeyboardconfig) - [XlsxWriter](#xlsxwriter) - [XMDS2](#xmds2) - [Xmipp](#xmipp) - [xmitgcm](#xmitgcm) - [XML-Compile](#xml-compile) - [XML-LibXML](#xml-libxml) - [XML-Parser](#xml-parser) - [xmlf90](#xmlf90) - [XMLSec](#xmlsec) - [XMLStarlet](#xmlstarlet) - [xonsh](#xonsh) - [xorg-macros](#xorg-macros) - [XPLOR-NIH](#xplor-nih) - [xprop](#xprop) - [xproto](#xproto) - [XSD](#xsd) - [XTandem](#xtandem) - [xtb](#xtb) - [xtensor](#xtensor) - [xtrans](#xtrans) - [Xvfb](#xvfb) - [xxd](#xxd) - [xxHash](#xxhash) - [XZ](#xz)


### X11

The X Window System (X11) is a windowing system for bitmap displays

*homepage*: <https://www.x.org>

version     |toolchain
------------|--------------------------------------------------
``20160819``|``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``20170129``|``GCCcore/6.3.0``, ``gimkl/2017a``
``20170314``|``GCCcore/6.3.0``
``20171023``|``GCCcore/6.4.0``
``20180131``|``GCCcore/6.4.0``
``20180604``|``GCCcore/7.3.0``
``20190311``|``GCCcore/8.2.0``
``20190717``|``GCCcore/8.3.0``
``20200222``|``GCCcore/9.3.0``
``20201008``|``GCCcore/10.2.0``
``20210518``|``GCCcore/10.3.0``
``20210802``|``GCCcore/11.2.0``
``20220504``|``GCCcore/11.3.0``
``20221110``|``GCCcore/12.2.0``
``20230603``|``GCCcore/12.3.0``

### x13as

X-13ARIMA-SEATS is seasonal adjustment software produced, distributed,  and maintained by the Census Bureau.  Features of X-13ARIMA-SEATS include:     - Extensive time series modeling and model selection capabilities for        linear regression models with ARIMA errors (regARIMA models);     - The capability to generate ARIMA model-based seasonal adjustment        using a version of the SEATS software originally developed by Victor        Gómez and Agustín Maravall at the Bank of Spain, as well as nonparametric        adjustments from the X-11 procedure;     - Diagnostics of the quality and stability of the adjustments        achieved under the options selected;     - The ability to efficiently process many series at once. 

*homepage*: <https://www.census.gov/data/software/x13as.About_X-13.html>

version    |toolchain
-----------|--------------------------------------
``1-1-b59``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``

### x264

x264 is a free software library and application for encoding video streams  into the H.264/MPEG-4 AVC compression format, and is released under the  terms of the GNU GPL. 

*homepage*: <https://www.videolan.org/developers/x264.html>

version     |toolchain
------------|---------------------------------
``20160114``|``gimkl/2.11.5``, ``intel/2016a``
``20160430``|``foss/2016a``, ``intel/2016a``
``20160614``|``foss/2016b``, ``intel/2016b``
``20170406``|``gimkl/2017a``
``20170721``|``GCCcore/6.4.0``
``20170913``|``intel/2017a``
``20171217``|``foss/2017b``, ``intel/2017b``
``20180128``|``GCCcore/6.4.0``
``20180325``|``GCCcore/6.4.0``
``20181203``|``GCCcore/7.3.0``
``20190413``|``GCCcore/8.2.0``
``20190925``|``GCCcore/8.3.0``
``20191217``|``GCCcore/9.3.0``
``20201026``|``GCCcore/10.2.0``
``20210414``|``GCCcore/10.3.0``
``20210613``|``GCCcore/11.2.0``
``20220620``|``GCCcore/11.3.0``
``20230226``|``GCCcore/12.2.0``

### x265

x265 is a free software library and application for encoding video streams  into the H.265 AVC compression format, and is released under the terms of  the GNU GPL. 

*homepage*: <https://x265.org/>

version|toolchain
-------|------------------------------------------------------------------------------
``2.4``|``foss/2016b``
``2.5``|``intel/2017a``
``2.6``|``GCCcore/6.4.0``, ``intel/2017b``
``2.7``|``GCCcore/6.4.0``
``2.9``|``GCCcore/7.3.0``
``3.0``|``GCCcore/8.2.0``
``3.2``|``GCCcore/8.3.0``
``3.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.5``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### XALT

XALT 2 is a tool to allow a site to track user executables   and library usage on a cluster. When installed it can tell a site what are   the top executables by Node-Hours or by the number of users or the number   of times it is run. XALT 2 also tracks library usage as well. XALT 2 can   also track package use by R, MATLAB or Python. It tracks both MPI and   non-MPI programs.

*homepage*: <https://github.com/xalt/xalt>

version  |toolchain
---------|----------
``2.8.4``|``system``

### xarray

xarray (formerly xray) is an open source project and Python package that aims to bring  the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the  core pandas data structures.

*homepage*: <https://github.com/pydata/xarray>

version     |versionsuffix     |toolchain
------------|------------------|---------------------------------------------------
``0.9.5``   |``-Python-2.7.13``|``intel/2017a``
``0.9.5``   |``-Python-3.6.1`` |``intel/2017a``
``0.9.6``   |``-Python-2.7.13``|``intel/2017a``
``0.10.0``  |``-Python-2.7.14``|``intel/2017b``
``0.10.0``  |``-Python-3.6.3`` |``intel/2017b``
``0.10.3``  |``-Python-3.6.4`` |``intel/2018a``
``0.10.4``  |``-Python-3.6.4`` |``intel/2018a``
``0.10.8``  |``-Python-3.6.4`` |``intel/2018a``
``0.12.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.13.0``  |``-Python-3.7.2`` |``intel/2019a``
``0.15.1``  |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.16.1``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.16.2``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.19.0``  |                  |``foss/2021a``
``0.20.1``  |                  |``foss/2021b``, ``intel/2021b``
``2022.6.0``|                  |``foss/2022a``, ``intel/2022a``
``2022.9.0``|                  |``foss/2022a``
``2023.4.2``|                  |``gfbf/2022b``

### xbitmaps

provides bitmaps for x

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------------------
``1.1.1``|``foss/2016a``, ``intel/2016a``, ``system``
``1.1.2``|``system``

### xcb-proto

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: <https://xcb.freedesktop.org/>

version |toolchain
--------|----------
``1.11``|``system``
``1.13``|``system``

### xcb-util

The xcb-util package provides additional extensions to the XCB library,  many that were previously found in Xlib, but are not part of core X protocol

*homepage*: <http://xcb.freedesktop.org/>

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

### xcb-util-image

The xcb-util-image package provides additional extensions to the XCB library.

*homepage*: <http://xcb.freedesktop.org/>

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

### xcb-util-keysyms

The xcb-util-keysyms package contains a library for  handling standard X key constants and conversion to/from keycodes.

*homepage*: <http://xcb.freedesktop.org/>

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

### xcb-util-renderutil

The xcb-util-renderutil package provides additional extensions to the XCB library.

*homepage*: <http://xcb.freedesktop.org/>

version  |toolchain
---------|-------------------------------
``0.3.9``|``foss/2016a``, ``intel/2016a``

### xcb-util-wm

The xcb-util-wm package contains libraries which provide client and window-manager helpers  for EWMH and ICCCM.

*homepage*: <http://xcb.freedesktop.org/>

version  |toolchain
---------|-------------------------------
``0.4.1``|``foss/2016a``, ``intel/2016a``

### xCell

xCell is a gene signatures-based method learned from thousands of pure cell types from various sources.

*homepage*: <https://github.com/dviraran/xCell>

version |versionsuffix|toolchain
--------|-------------|--------------
``1.12``|``-R-3.5.1`` |``foss/2018b``

### XCFun

XCFun is a library of DFT exchange-correlation (XC) functionals.  It is based on automatic differentiation and can therefore generate arbitrary order  derivatives of these functionals. 

*homepage*: <http://dftlibs.org/xcfun/>

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------------------------------
``2.1.0``   |                  |``GCCcore/9.3.0``
``2.1.1``   |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``
``20180122``|``-Python-2.7.14``|``intel/2017b``
``20190127``|``-Python-3.7.2`` |``foss/2019a``

### xclip

xclip is a command line utility that is designed to run on any system with an X11 implementation.

*homepage*: <https://github.com/astrand/xclip>

version |toolchain
--------|------------------
``0.13``|``GCCcore/11.3.0``

### XCrySDen

XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. It also possesses some tools for analysis of properties in reciprocal space such as interactive selection of k-paths in the Brillouin zone for the band-structure plots, and visualisation of Fermi surfaces. 

*homepage*: <http://www.xcrysden.org/>

version   |toolchain
----------|--------------------------------------------------------------------------------
``1.5.60``|``intel/2019a``
``1.6.2`` |``foss/2019b``, ``foss/2020b``, ``foss/2022a``, ``intel/2019b``, ``intel/2022a``

### xdotool

xdotool lets you simulate keyboard input and mouse activity, move and resize windows, etc.  It does this using X11’s XTEST extension and other Xlib functions. 

*homepage*: <https://github.com/jordansissel/xdotool>

version         |toolchain
----------------|------------------
``3.20211022.1``|``GCCcore/11.3.0``

### Xerces-C++

Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it simple to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs.

*homepage*: <https://xerces.apache.org/xerces-c/>

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.1.4``|``GCCcore/6.4.0``
``3.2.0``|``GCCcore/7.3.0``
``3.2.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.2.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``3.2.4``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### xESMF

xESMF: Universal Regridder for Geospatial Data

*homepage*: <https://xesmf.readthedocs.io>

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.0``|                 |``intel/2020b``
``0.3.0``|``-Python-3.8.2``|``foss/2020a``

### xextproto

XExtProto protocol headers.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|------------------------------------------------------------------
``7.3.0``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

### xf86vidmodeproto

X11 XFree86 video mode extension protocol headers.

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``2.3.1``|``foss/2016a``, ``intel/2016a``

### XGBoost

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: <https://github.com/dmlc/xgboost>

version     |versionsuffix      |toolchain
------------|-------------------|---------------
``0.6a2``   |``-Python-2.7.12`` |``intel/2016b``
``0.6a2``   |``-Python-3.5.2``  |``intel/2016b``
``0.6a2``   |``-Python-3.6.1``  |``intel/2017a``
``0.72.1``  |``-Python-2.7.14`` |``intel/2017b``
``0.90``    |``-Python-3.7.4``  |``foss/2019b``
``1.2.0``   |``-Python-3.8.2``  |``foss/2020a``
``1.5.0``   |                   |``foss/2021a``
``1.7.1``   |                   |``foss/2022a``
``1.7.2``   |                   |``foss/2022a``
``1.7.2``   |``-CUDA-11.7.0``   |``foss/2022a``
``20171120``|``-Java-1.8.0_152``|``intel/2017b``

### XGrafix

A graphics library and controller for physics simulation programs. 3-d surface plots, scatter plots, 2-d line plots.

*homepage*: <https://ptsg.egr.msu.edu/>

version |toolchain
--------|-----------------
``2.41``|``GCCcore/9.3.0``

### xineramaproto

X protocol and ancillary headers for xinerama

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2016a``, ``intel/2016a``

### XKeyboardConfig

The non-arch keyboard configuration database for X Window.  The goal is to provide the consistent, well-structured,  frequently released open source of X keyboard configuration data  for X Window System implementations (free, open source and commercial).  The project is targeted to XKB-based systems.

*homepage*: <http://www.freedesktop.org/wiki/Software/XKeyboardConfig/>

version |toolchain
--------|-------------------------------
``2.17``|``foss/2016a``, ``intel/2016a``

### XlsxWriter

A Python module for creating Excel XLSX files

*homepage*: <https://xlsxwriter.readthedocs.io/>

version  |toolchain
---------|------------------
``1.4.0``|``GCCcore/10.2.0``
``1.4.4``|``GCCcore/10.3.0``
``3.0.2``|``GCCcore/11.2.0``
``3.0.8``|``GCCcore/11.3.0``
``3.1.2``|``GCCcore/12.2.0``

### XMDS2

The purpose of XMDS2 is to simplify the process of creating simulations that  solve systems of initial-value first-order partial and ordinary differential equations.

*homepage*: <http://www.xmds.org/>

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.3``|``-Python-2.7.15``|``foss/2018b``

### Xmipp

Scipion is an image processing framework to obtain 3D models of macromolecular complexes using Electron Microscopy (3DEM). It integrates several software packages and presents an unified interface for both biologists and developers. Scipion allows to execute workflows combining different software tools, while taking care of formats and conversions. Additionally, all steps are tracked and can be reproduced later on.  

*homepage*: <https://github.com/I2PC/scipion/wiki>

version           |versionsuffix     |toolchain
------------------|------------------|----------------------------------
``3.19.04-Apollo``|``-Python-2.7.15``|``foss/2019a``, ``fosscuda/2019a``
``3.22.07-Helios``|                  |``foss/2022a``
``3.22.07-Helios``|``-CUDA-11.7.0``  |``foss/2022a``

### xmitgcm

xmitgcm is a python package for reading MITgcm binary  MDS files into xarray data structures. By storing data in dask arrays,  xmitgcm enables parallel, out-of-core analysis of MITgcm output data.

*homepage*: <https://github.com/MITgcm/xmitgcm>

version  |toolchain
---------|--------------
``0.5.2``|``foss/2022a``

### XML-Compile

Perl module for compilation based XML processing

*homepage*: <https://metacpan.org/pod/XML::Compile>

version |toolchain
--------|------------------
``1.63``|``GCCcore/11.2.0``

### XML-LibXML

Perl binding for libxml2

*homepage*: <https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod>

version   |versionsuffix   |toolchain
----------|----------------|----------------------------------------------------------
``2.0132``|``-Perl-5.24.0``|``foss/2016b``, ``intel/2016b``
``2.0132``|``-Perl-5.24.1``|``intel/2017a``
``2.0132``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``2.0132``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.0132``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.0200``|``-Perl-5.28.1``|``GCCcore/8.2.0``
``2.0201``|                |``GCCcore/8.3.0``
``2.0205``|                |``GCCcore/9.3.0``
``2.0206``|                |``GCCcore/10.2.0``
``2.0207``|                |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.0208``|                |``GCCcore/12.2.0``

### XML-Parser

This is a Perl extension interface to James Clark's XML parser, expat.

*homepage*: <https://search.cpan.org/~toddr/XML-Parser-2.46/>

version    |versionsuffix   |toolchain
-----------|----------------|--------------------------------------------------------------------------------------
``2.41``   |``-Perl-5.20.3``|``intel/2016a``
``2.44``   |``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``2.44_01``|``-Perl-5.24.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.44_01``|``-Perl-5.24.1``|``GCCcore/6.3.0``, ``intel/2017a``
``2.44_01``|``-Perl-5.26.0``|``GCCcore/6.4.0``
``2.44_01``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.44_01``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.46``   |``-Perl-5.32.1``|``GCCcore/10.3.0``
``2.46``   |``-Perl-5.34.1``|``GCCcore/11.3.0``

### xmlf90

A fast XML parser and generator in Fortran

*homepage*: <https://gitlab.com/siesta-project/libraries/xmlf90>

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.5.3``|``foss/2016b``, ``foss/2017a``
``1.5.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``
``1.5.6``|``GCC/11.3.0``

### XMLSec

XML Security Library is a C library based on LibXML2,  supporting major XML security standards.

*homepage*: <https://www.aleksey.com/xmlsec/index.html>

version   |toolchain
----------|-----------------
``1.2.26``|``GCCcore/6.4.0``

### XMLStarlet

Command line XML tool

*homepage*: <http://xmlstar.sourceforge.net>

version  |toolchain
---------|---------------------------------
``1.6.1``|``GCCcore/6.4.0``, ``foss/2016a``

### xonsh

Xonsh is a Python-ish, BASHwards-looking shell language and command prompt.

*homepage*: <https://xon.sh/>

version  |toolchain
---------|---------------
``0.3.2``|``intel/2016a``

### xorg-macros

X.org macros utilities.

*homepage*: <https://gitlab.freedesktop.org/xorg/util/macros>

version   |toolchain
----------|------------------------------------------------------------------------------------------------------------------------------------
``1.19.0``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``1.19.1``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.19.2``|``GCCcore/10.2.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``1.19.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.20.0``|``GCCcore/12.3.0``

### XPLOR-NIH

A System for X-ray Crystallography and NMR

*homepage*: <https://nmr.cit.nih.gov/xplor-nih/>

version|versionsuffix    |toolchain
-------|-----------------|----------
``3.4``|``-Linux_x86_64``|``system``

### xprop

The xprop utility is for displaying window and font properties in an X server.  One window or font is selected using the command line arguments or possibly  in the case of a window, by clicking on the desired window. A list of  properties is then given, possibly with formatting information.

*homepage*: <https://www.x.org/wiki/>

version  |versionsuffix    |toolchain
---------|-----------------|-----------------------------------------------------------------------------------------------------------------------
``1.2.2``|                 |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.2.2``|``-X11-20180131``|``GCCcore/6.4.0``
``1.2.3``|                 |``GCCcore/7.3.0``
``1.2.4``|                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.2.5``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### xproto

X protocol and ancillary headers

*homepage*: <https://www.freedesktop.org/wiki/Software/xlibs>

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``7.0.28``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``7.0.29``|``intel/2016a``
``7.0.31``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### XSD

CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.

*homepage*: <https://www.codesynthesis.com/products/xsd/>

version  |toolchain
---------|-----------------
``4.0.0``|``GCCcore/8.2.0``

### XTandem

X!Tandem open source is software that can match tandem mass  spectra with peptide sequences, in a process that has come to be known as  protein identification.

*homepage*: <https://www.thegpm.org/tandem>

version       |toolchain
--------------|----------------------------------------------------------
``17.02.01.4``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

### xtb

xtb - An extended tight-binding semi-empirical program package. 

*homepage*: <https://xtb-docs.readthedocs.io>

version         |versionsuffix    |toolchain
----------------|-----------------|-------------------------------
``6.2.2-hotfix``|``-Python-3.6.3``|``intel/2017b``
``6.2.3``       |                 |``foss/2019b``
``6.4.1``       |                 |``foss/2021b``, ``intel/2021a``
``6.5.0``       |                 |``foss/2021b``
``6.5.1``       |                 |``foss/2022a``

### xtensor

xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions.

*homepage*: <https://github.com/xtensor-stack/xtensor>

version   |toolchain
----------|--------------
``0.24.0``|``foss/2021b``

### xtrans

xtrans includes a number of routines to make X implementations transport-independent;  at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet. 

*homepage*: <http://www.freedesktop.org/wiki/Software/xlibs>

version  |toolchain
---------|---------------------------------------------------------------------
``1.3.5``|``GCCcore/11.3.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

### Xvfb

Xvfb is an X server that can run on machines with no display hardware and no physical input devices.  It emulates a dumb framebuffer using virtual memory.

*homepage*: <https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml>

version    |toolchain
-----------|-------------------------------------
``1.20.8`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.20.9`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.20.11``|``GCCcore/10.3.0``
``1.20.13``|``GCCcore/11.2.0``
``21.1.3`` |``GCCcore/11.3.0``
``21.1.6`` |``GCCcore/12.2.0``

### xxd

xxd is part of the VIM package and this will only install xxd, not vim! xxd converts to/from hexdumps of binary files.

*homepage*: <https://www.vim.org>

version     |toolchain
------------|------------------------------------------------------------------------------
``8.2.4220``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

### xxHash

xxHash is an extremely fast non-cryptographic hash algorithm, working at RAM speed limit.

*homepage*: <https://cyan4973.github.io/xxHash>

version  |toolchain
---------|------------------
``0.8.0``|``GCCcore/11.2.0``
``0.8.1``|``GCCcore/12.2.0``

### XZ

xz: XZ utilities

*homepage*: <https://tukaani.org/xz/>

version  |versionsuffix      |toolchain
---------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``5.0.5``|                   |``GCC/4.8.2``
``5.2.0``|                   |``GCC/4.9.2``
``5.2.2``|                   |``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``5.2.2``|``-gettext-0.19.7``|``foss/2016a``, ``intel/2016a``
``5.2.3``|                   |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``
``5.2.4``|                   |``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``
``5.2.5``|                   |``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``
``5.2.7``|                   |``GCCcore/12.2.0``
``5.4.2``|                   |``GCCcore/12.3.0``, ``GCCcore/13.1.0``

## Y


[YACS](#yacs) - [Yade](#yade) - [yaff](#yaff) - [Yambo](#yambo) - [yaml-cpp](#yaml-cpp) - [YANK](#yank) - [YAPS](#yaps) - [Yasm](#yasm) - [YAXT](#yaxt) - [Yices](#yices) - [YODA](#yoda)


### YACS

YACS was created as a lightweight library to define and manage system configurations, such as those commonly found in software designed for scientific experimentation. These "configurations" typically cover concepts like hyperparameters used in training a machine learning model or configurable model hyperparameters, such as the depth of a convolutional neural network.

*homepage*: <https://github.com/rbgirshick/yacs>

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``0.1.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

### Yade

Yade is an extensible open-source framework for discrete numerical models,   focused on Discrete Element Method. The computation parts are written in c++ using flexible object model,   allowing independent implementation of new alogrithms and interfaces.   Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. 

*homepage*: <https://yade-dem.org/doc/>

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``1.20.0``  |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2016.06a``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2017.01a``|``-Python-2.7.12``|``intel/2016b``
``2018.02b``|``-Python-2.7.14``|``intel/2018a``

### yaff

Yaff stands for 'Yet another force field'. It is a pythonic force-field code.

*homepage*: <https://molmod.github.io/yaff/>

version             |versionsuffix                 |toolchain
--------------------|------------------------------|------------------------------------------------
``1.0.develop.2.15``|``-Python-2.7.12-HDF5-1.8.18``|``intel/2016b``
``1.1.2``           |``-Python-2.7.13``            |``intel/2017a``
``1.1.3``           |``-Python-2.7.13``            |``intel/2017a``
``1.4.2``           |``-Python-2.7.14``            |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``1.4.2``           |``-Python-3.6.3``             |``intel/2017b``
``1.4.5``           |``-Python-2.7.15``            |``intel/2018b``
``1.5.0``           |``-Python-2.7.15``            |``intel/2018b``
``1.5.0``           |``-Python-3.7.2``             |``intel/2019a``
``1.6.0``           |                              |``foss/2020b``
``1.6.0``           |``-Python-3.7.2``             |``intel/2019a``
``1.6.0``           |``-Python-3.7.4``             |``foss/2019b``, ``intel/2019b``
``1.6.0``           |``-Python-3.8.2``             |``foss/2020a``, ``intel/2020a``

### Yambo

Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.  Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf.

*homepage*: <http://www.yambo-code.org>

version  |toolchain
---------|-------------------------
``3.4.2``|``intel/2016.02-GCC-4.9``
``5.0.4``|``intel/2021a``

### yaml-cpp

yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec

*homepage*: <https://github.com/jbeder/yaml-cpp>

version  |toolchain
---------|----------------------------------------------------------
``0.6.3``|``GCCcore/8.3.0``
``0.7.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``

### YANK

A GPU-accelerated Python framework for exploring algorithms for alchemical free energy calculations

*homepage*: <http://getyank.org/>

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.25.2``|``-Python-3.8.2``|``intel/2020a``

### YAPS

YAPS - Yet Another Positioning Solver

*homepage*: <https://github.com/baktoft/yaps>

version  |versionsuffix|toolchain
---------|-------------|---------------
``1.1.0``|``-R-3.5.1`` |``foss/2018b``
``1.1.0``|``-R-3.6.0`` |``intel/2019a``

### Yasm

Yasm: Complete rewrite of the NASM assembler with BSD license

*homepage*: <http://www.tortall.net/projects/yasm/>

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

### YAXT

Yet Another eXchange Tool

*homepage*: <https://www.dkrz.de/redmine/projects/yaxt>

version    |toolchain
-----------|-------------------------------------------------
``0.5.1``  |``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``0.6.0``  |``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``0.6.2``  |``foss/2018b``, ``gompi/2019b``, ``iimpi/2019b``
``0.9.0``  |``gompi/2020b``
``0.9.1``  |``gompi/2021a``
``0.9.2``  |``iimpi/2021b``
``0.9.2.1``|``gompi/2021b``

### Yices

Yices 2 is an SMT solver that decides the satisfiability of formulas containing uninterpreted  function symbols with equality, real and integer arithmetic, bitvectors, scalar types, and tuples. Yices 2 supports  both linear and nonlinear arithmetic.

*homepage*: <https://yices.csl.sri.com/index.html>

version  |toolchain
---------|------------------
``2.6.2``|``GCCcore/10.2.0``

### YODA

Yet more Objects for (High Energy Physics) Data Analysis 

*homepage*: <https://yoda.hepforge.org/>

version  |toolchain
---------|--------------
``1.9.7``|``GCC/11.3.0``

## Z


[Z3](#z3) - [zarr](#zarr) - [ZeroMQ](#zeromq) - [zfp](#zfp) - [Zgoubi](#zgoubi) - [ZIMPL](#zimpl) - [zingeR](#zinger) - [Zip](#zip) - [zlib](#zlib) - [zlib-ng](#zlib-ng) - [zlibbioc](#zlibbioc) - [Zopfli](#zopfli) - [ZPAQ](#zpaq) - [zsh](#zsh) - [zstd](#zstd)


### Z3

Z3 is a theorem prover from Microsoft Research. 

*homepage*: <https://github.com/Z3Prover/z3>

version   |toolchain
----------|------------------------------------
``4.8.9`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.8.10``|``GCCcore/10.2.0``
``4.8.11``|``GCCcore/10.3.0``
``4.8.12``|``GCCcore/11.2.0``
``4.8.16``|``GCCcore/11.3.0``
``4.10.2``|``GCCcore/11.3.0``

### zarr

Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays,  designed for use in parallel computing.

*homepage*: <https://zarr.readthedocs.io/en/stable/>

version   |versionsuffix     |toolchain
----------|------------------|------------------------------
``2.1.4`` |``-Python-2.7.13``|``foss/2017a``
``2.4.0`` |``-Python-3.8.2`` |``foss/2020a``
``2.8.1`` |                  |``foss/2020b``
``2.10.1``|                  |``foss/2021a``
``2.13.3``|                  |``foss/2021b``, ``foss/2022a``

### ZeroMQ

ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.  It gives you sockets that carry atomic messages across various transports like in-process,  inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,  pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered  products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous  message-processing tasks. It has a score of language APIs and runs on most operating systems.

*homepage*: <https://www.zeromq.org/>

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------
``4.1.4``|``foss/2016a``, ``intel/2016a``
``4.1.5``|``intel/2016b``
``4.2.0``|``foss/2016b``, ``intel/2016b``
``4.2.2``|``foss/2017a``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.2.5``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``4.3.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.3.3``|``GCCcore/10.2.0``
``4.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

### zfp

zfp is a compressed format for representing multidimensional floating-point and integer arrays. zfp provides compressed-array classes that support high throughput read and write random access to individual array elements. zfp also supports serial and parallel (OpenMP and CUDA) compression of whole arrays, e.g., for applications that read and write large data sets to and from disk.

*homepage*: <https://github.com/LLNL/zfp>

version  |toolchain
---------|---------------------------------------------------------
``0.5.5``|``GCCcore/10.2.0``
``1.0.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

### Zgoubi

Zgoubi is a ray-tracing code in use for charged particle beam dynamics simulations.  It can simulate beam dynamics in a large variety of machines and optical systems.

*homepage*: <https://zgoubi.sourceforge.io>

version  |toolchain
---------|------------------
``6.0.2``|``GCCcore/10.3.0``

### ZIMPL

ZIMPL is a little language to translate the mathematical model of a problem into a linear or nonlinear (mixed-) integer mathematical program expressed in .lp or .mps file format which can be read and (hopefully) solved by a LP or MIP solver.

*homepage*: <https://zimpl.zib.de/>

version  |toolchain
---------|------------------
``3.3.4``|``GCCcore/11.3.0``

### zingeR

Zero-Inflated Negative binomial Gene Expression in R

*homepage*: <https://github.com/statOmics/zingeR>

version     |versionsuffix|toolchain
------------|-------------|--------------
``20180131``|``-R-3.5.1`` |``foss/2018b``

### Zip

Zip is a compression and file packaging/archive utility. Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives have been portability and other-than-MSDOS functionality

*homepage*: <http://www.info-zip.org/Zip.html>

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/12.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

### zlib

zlib is designed to be a free, general-purpose, legally unencumbered -- that is,  not covered by any patents -- lossless data-compression library for use on virtually any  computer hardware and operating system.

*homepage*: <http://www.zlib.net/>

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2.7`` |``GCC/4.8.1``, ``GCC/4.8.2``
``1.2.8`` |``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.2.10``|``system``
``1.2.11``|``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/9.4.0``, ``GCCcore/system``, ``gimkl/2017a``, ``system``
``1.2.12``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.5.0``, ``system``
``1.2.13``|``GCCcore/11.4.0``, ``GCCcore/12.3.0``, ``GCCcore/13.1.0``, ``system``

### zlib-ng

zlib data compression library for the next generation systems

*homepage*: <https://github.com/zlib-ng/zlib-ng>

version  |toolchain
---------|------------------
``2.0.5``|``GCCcore/10.2.0``
``2.0.6``|``GCCcore/10.3.0``
``2.0.7``|``GCCcore/11.3.0``

### zlibbioc

This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have  these available via other means.

*homepage*: <https://bioconductor.org/packages/release/bioc/html/zlibbioc.html>

version   |versionsuffix|toolchain
----------|-------------|---------------
``1.18.0``|``-R-3.2.3`` |``intel/2016a``
``1.20.0``|``-R-3.3.1`` |``intel/2016b``

### Zopfli

Zopfli Compression Algorithm is a compression library programmed in C to perform very good, but slow, deflate or zlib compression.

*homepage*: <https://github.com/google/zopfli>

version  |toolchain
---------|----------------------------------------------------------
``1.0.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``

### ZPAQ

zpaq is a free and open source (GPL v3) incremental,  journaling command-line archiver for Windows, Linux and Mac OS/X

*homepage*: <http://mattmahoney.net/dc/zpaq.html>

version |toolchain
--------|-------------
``7.00``|``GCC/4.8.2``

### zsh

Zsh is a shell designed for interactive use, although it is also a powerful scripting language.

*homepage*: <https://www.zsh.org/>

version  |toolchain
---------|-------------------------
``5.1.1``|``GNU/4.9.3-2.25``
``5.2``  |``foss/2016b``
``5.8``  |``GCC/8.3.0``, ``system``
``5.9``  |``system``

### zstd

Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.

*homepage*: <https://facebook.github.io/zstd>

version  |toolchain
---------|----------------------------------------------------
``1.3.4``|``foss/2016b``
``1.4.0``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2018b``
``1.4.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.5``|``GCCcore/10.2.0``
``1.4.9``|``GCCcore/10.3.0``
``1.5.0``|``GCCcore/11.2.0``
``1.5.2``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
``1.5.5``|``GCCcore/12.3.0``
