
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://docs.easybuild.io/version-specific/supported-software/">
      
      
        <link rel="prev" href="../../api/easybuild/tools/version/">
      
      
        <link rel="next" href="../../getting-help/">
      
      <link rel="icon" href="../../favicon.ico">
      <meta name="generator" content="mkdocs-1.4.3, mkdocs-material-9.1.14">
    
    
      
        <title>List of supported software - EasyBuild - building software with ease</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.85bb2934.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.a6bdf11c.min.css">
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
      <link rel="stylesheet" href="../../stylesheets/asciinema-player-2.6.1.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  


  <script id="__analytics">function __md_analytics(){function n(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],n("js",new Date),n("config","G-BLVCH6991S"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&n("event","search",{search_term:this.value})}),document$.subscribe(function(){var a=document.forms.feedback;if(void 0!==a)for(var e of a.querySelectorAll("[type=submit]"))e.addEventListener("click",function(e){e.preventDefault();var t=document.location.pathname,e=this.getAttribute("data-md-value");n("event","feedback",{page:t,data:e}),a.firstElementChild.disabled=!0;e=a.querySelector(".md-feedback__note [data-md-value='"+e+"']");e&&(e.hidden=!1)}),a.hidden=!1}),location$.subscribe(function(e){n("config","G-BLVCH6991S",{page_path:e.pathname})})});var e=document.createElement("script");e.async=!0,e.src="https://www.googletagmanager.com/gtag/js?id=G-BLVCH6991S",document.getElementById("__analytics").insertAdjacentElement("afterEnd",e)}</script>

  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="custom">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#list-of-supported-software" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="EasyBuild - building software with ease" class="md-header__button md-logo" aria-label="EasyBuild - building software with ease" data-md-component="logo">
      
  <img src="../../img/easybuild_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            EasyBuild - building software with ease
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              List of supported software
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
          
            
            
            
            <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="custom"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
            
              <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
              </label>
            
          
            
            
            
            <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="custom" data-md-color-accent="custom"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_2">
            
              <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12c0-2.42-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
              </label>
            
          
        </form>
      
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/easybuilders/easybuild-easyconfigs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    EasyBuild @ GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="EasyBuild - building software with ease" class="md-nav__button md-logo" aria-label="EasyBuild - building software with ease" data-md-component="logo">
      
  <img src="../../img/easybuild_logo.png" alt="logo">

    </a>
    EasyBuild - building software with ease
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/easybuilders/easybuild-easyconfigs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    EasyBuild @ GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
      
      
      
        <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
          Basic info
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Basic info
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../what-is-easybuild/" class="md-nav__link">
        What is EasyBuild?
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../terminology/" class="md-nav__link">
        Terminology
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
      
      
      
        <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
          Getting started with EasyBuild
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Getting started with EasyBuild
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../installation/" class="md-nav__link">
        Installation
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../configuration/" class="md-nav__link">
        Configuration
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../using-easybuild/" class="md-nav__link">
        Basic usage
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../typical-workflow-example/" class="md-nav__link">
        Typical workflow example
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
      
      
      
        <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
          Using EasyBuild
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_4">
          <span class="md-nav__icon md-icon"></span>
          Using EasyBuild
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../backup-modules/" class="md-nav__link">
        Backing up existing modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../common-toolchains/" class="md-nav__link">
        Common toolchains
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../controlling-compiler-optimization-flags/" class="md-nav__link">
        Controlling optimization flags
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../detecting-loaded-modules/" class="md-nav__link">
        Detecting loaded modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../log-files/" class="md-nav__link">
        EasyBuild log files
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../extended-dry-run/" class="md-nav__link">
        Extended dry run
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../locks/" class="md-nav__link">
        Locks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../manipulating-dependencies/" class="md-nav__link">
        Manipulating dependencies
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../partial-installations/" class="md-nav__link">
        Partial installations
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../python-2-3-compatibility/" class="md-nav__link">
        Compatibility with Python 2 + 3
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../progress-bars/" class="md-nav__link">
        Progress bars
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../easyconfigs-search-index/" class="md-nav__link">
        Search index for easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../system-toolchain/" class="md-nav__link">
        System toolchain
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../submitting-jobs/" class="md-nav__link">
        Submitting installations as jobs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../tracing-progress/" class="md-nav__link">
        Tracing installation progress
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../writing-easyconfig-files/" class="md-nav__link">
        Writing easyconfig files
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
      
      
      
        <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
          Advanced topics
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_5">
          <span class="md-nav__icon md-icon"></span>
          Advanced topics
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../cray-support/" class="md-nav__link">
        Cray support
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../hooks/" class="md-nav__link">
        Customizing EasyBuild via hooks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../including-additional-python-modules/" class="md-nav__link">
        Including Python modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../packaging-support/" class="md-nav__link">
        Packaging support
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../rpath-support/" class="md-nav__link">
        RPATH support
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../using-external-modules/" class="md-nav__link">
        Using external modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../wrapping-dependencies/" class="md-nav__link">
        Wrapping dependencies
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
      
      
      
        <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
          For developers/contributors
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_6">
          <span class="md-nav__icon md-icon"></span>
          For developers/contributors
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../archived-easyconfigs/" class="md-nav__link">
        Archived easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../code-style/" class="md-nav__link">
        Code style
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../contributing/" class="md-nav__link">
        Contributing to EasyBuild
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../integration-with-github/" class="md-nav__link">
        GitHub integration
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../implementing-easyblocks/" class="md-nav__link">
        Implementing easyblocks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../easyconfig-files-local-variables/" class="md-nav__link">
        Local variables in easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../unit-tests/" class="md-nav__link">
        Unit tests
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../framework-overview/" class="md-nav__link">
        Framework Overview
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
      
      
      
        <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
          Experimental features
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_7">
          <span class="md-nav__icon md-icon"></span>
          Experimental features
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../experimental-features/" class="md-nav__link">
        (overview)
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../containers/" class="md-nav__link">
        Creating container images/recipes
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../easystack-files/" class="md-nav__link">
        Easystack files
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../writing-yeb-easyconfig-files/" class="md-nav__link">
        Easyconfig files in YAML syntax
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../installing-extensions-in-parallel/" class="md-nav__link">
        Installing extensions in parallel
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
      
      
      
        <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
          Version-specific info
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_8">
          <span class="md-nav__icon md-icon"></span>
          Version-specific info
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../" class="md-nav__link">
        (overview)
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../config-file-constants/" class="md-nav__link">
        Constants for config files
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../easyconfig-constants/" class="md-nav__link">
        Constants for easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../easyblocks/" class="md-nav__link">
        Easyblocks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../eb-help/" class="md-nav__link">
        EasyBuild configuration options
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../easyconfig-parameters/" class="md-nav__link">
        Easyconfig parameters
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../eb-a-wrf/" class="md-nav__link">
        Easyconfig parameters for WRF
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../generic-easyblocks/" class="md-nav__link">
        Generic easyblocks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../easyconfig-license-constants/" class="md-nav__link">
        License constants for easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../easyconfig-templates/" class="md-nav__link">
        Templates for easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../toolchain-opts/" class="md-nav__link">
        Toolchain options
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../toolchains/" class="md-nav__link">
        Toolchains
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../release-notes/" class="md-nav__link">
        EasyBuild release notes
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../maintainers/" class="md-nav__link">
        EasyBuild maintainers
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11" >
      
      
      
        <label class="md-nav__link" for="__nav_11" id="__nav_11_label" tabindex="0">
          API
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_11_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11">
          <span class="md-nav__icon md-icon"></span>
          API
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/">easybuild</a>
          
            <label for="__nav_11_1">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_11_1_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1">
          <span class="md-nav__icon md-icon"></span>
          easybuild
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_1" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/base/">base</a>
          
            <label for="__nav_11_1_1">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_11_1_1_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_1">
          <span class="md-nav__icon md-icon"></span>
          base
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/exceptions/" class="md-nav__link">
        exceptions
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/fancylogger/" class="md-nav__link">
        fancylogger
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/frozendict/" class="md-nav__link">
        frozendict
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/generaloption/" class="md-nav__link">
        generaloption
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/optcomplete/" class="md-nav__link">
        optcomplete
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/rest/" class="md-nav__link">
        rest
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/testing/" class="md-nav__link">
        testing
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/base/wrapper/" class="md-nav__link">
        wrapper
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_2" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/framework/">framework</a>
          
            <label for="__nav_11_1_2">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_11_1_2_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_2">
          <span class="md-nav__icon md-icon"></span>
          framework
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyblock/" class="md-nav__link">
        easyblock
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_2_2" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/framework/easyconfig/">easyconfig</a>
          
            <label for="__nav_11_1_2_2">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_2_2_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_2_2">
          <span class="md-nav__icon md-icon"></span>
          easyconfig
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/constants/" class="md-nav__link">
        constants
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/default/" class="md-nav__link">
        default
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/easyconfig/" class="md-nav__link">
        easyconfig
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_2_2_4" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/framework/easyconfig/format/">format</a>
          
            <label for="__nav_11_1_2_2_4">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="5" aria-labelledby="__nav_11_1_2_2_4_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_2_2_4">
          <span class="md-nav__icon md-icon"></span>
          format
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/convert/" class="md-nav__link">
        convert
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/format/" class="md-nav__link">
        format
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/one/" class="md-nav__link">
        one
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/pyheaderconfigobj/" class="md-nav__link">
        pyheaderconfigobj
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/two/" class="md-nav__link">
        two
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/version/" class="md-nav__link">
        version
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/format/yeb/" class="md-nav__link">
        yeb
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/licenses/" class="md-nav__link">
        licenses
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/parser/" class="md-nav__link">
        parser
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/style/" class="md-nav__link">
        style
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/templates/" class="md-nav__link">
        templates
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/tools/" class="md-nav__link">
        tools
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/tweak/" class="md-nav__link">
        tweak
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easyconfig/types/" class="md-nav__link">
        types
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/easystack/" class="md-nav__link">
        easystack
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/extension/" class="md-nav__link">
        extension
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/framework/extensioneasyblock/" class="md-nav__link">
        extensioneasyblock
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/main/" class="md-nav__link">
        main
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_4" >
      
      
      
        <label class="md-nav__link" for="__nav_11_1_4" id="__nav_11_1_4_label" tabindex="0">
          scripts
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_11_1_4_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_4">
          <span class="md-nav__icon md-icon"></span>
          scripts
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/bootstrap_eb/" class="md-nav__link">
        bootstrap_eb
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/clean_gists/" class="md-nav__link">
        clean_gists
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/findPythonDeps/" class="md-nav__link">
        findPythonDeps
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/fix_docs/" class="md-nav__link">
        fix_docs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/mk_tmpl_easyblock_for/" class="md-nav__link">
        mk_tmpl_easyblock_for
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/scripts/rpath_args/" class="md-nav__link">
        rpath_args
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_5" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/toolchains/">toolchains</a>
          
            <label for="__nav_11_1_5">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_11_1_5_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_5">
          <span class="md-nav__icon md-icon"></span>
          toolchains
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgmpich/" class="md-nav__link">
        cgmpich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgmpolf/" class="md-nav__link">
        cgmpolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgmvapich2/" class="md-nav__link">
        cgmvapich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgmvolf/" class="md-nav__link">
        cgmvolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgompi/" class="md-nav__link">
        cgompi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/cgoolf/" class="md-nav__link">
        cgoolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/clanggcc/" class="md-nav__link">
        clanggcc
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_5_8" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/toolchains/compiler/">compiler</a>
          
            <label for="__nav_11_1_5_8">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_5_8_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_5_8">
          <span class="md-nav__icon md-icon"></span>
          compiler
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/clang/" class="md-nav__link">
        clang
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/craype/" class="md-nav__link">
        craype
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/cuda/" class="md-nav__link">
        cuda
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/dummycompiler/" class="md-nav__link">
        dummycompiler
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/fujitsu/" class="md-nav__link">
        fujitsu
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/gcc/" class="md-nav__link">
        gcc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/ibmxl/" class="md-nav__link">
        ibmxl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/intel_compilers/" class="md-nav__link">
        intel_compilers
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/inteliccifort/" class="md-nav__link">
        inteliccifort
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/nvhpc/" class="md-nav__link">
        nvhpc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/pgi/" class="md-nav__link">
        pgi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/compiler/systemcompiler/" class="md-nav__link">
        systemcompiler
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/craycce/" class="md-nav__link">
        craycce
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/craygnu/" class="md-nav__link">
        craygnu
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/crayintel/" class="md-nav__link">
        crayintel
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/craypgi/" class="md-nav__link">
        craypgi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/dummy/" class="md-nav__link">
        dummy
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fcc/" class="md-nav__link">
        fcc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/ffmpi/" class="md-nav__link">
        ffmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_5_16" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/toolchains/fft/">fft</a>
          
            <label for="__nav_11_1_5_16">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_5_16_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_5_16">
          <span class="md-nav__icon md-icon"></span>
          fft
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fft/fftw/" class="md-nav__link">
        fftw
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fft/fujitsufftw/" class="md-nav__link">
        fujitsufftw
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fft/intelfftw/" class="md-nav__link">
        intelfftw
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/foss/" class="md-nav__link">
        foss
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fosscuda/" class="md-nav__link">
        fosscuda
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/fujitsu/" class="md-nav__link">
        fujitsu
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gcc/" class="md-nav__link">
        gcc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gcccore/" class="md-nav__link">
        gcccore
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gcccuda/" class="md-nav__link">
        gcccuda
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gfbf/" class="md-nav__link">
        gfbf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gimkl/" class="md-nav__link">
        gimkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gimpi/" class="md-nav__link">
        gimpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gimpic/" class="md-nav__link">
        gimpic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/giolf/" class="md-nav__link">
        giolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/giolfc/" class="md-nav__link">
        giolfc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmacml/" class="md-nav__link">
        gmacml
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmkl/" class="md-nav__link">
        gmkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmklc/" class="md-nav__link">
        gmklc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmpich/" class="md-nav__link">
        gmpich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmpich2/" class="md-nav__link">
        gmpich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmpit/" class="md-nav__link">
        gmpit
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmpolf/" class="md-nav__link">
        gmpolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmvapich2/" class="md-nav__link">
        gmvapich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gmvolf/" class="md-nav__link">
        gmvolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gnu/" class="md-nav__link">
        gnu
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/goalf/" class="md-nav__link">
        goalf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gobff/" class="md-nav__link">
        gobff
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/goblf/" class="md-nav__link">
        goblf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gofbf/" class="md-nav__link">
        gofbf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/golf/" class="md-nav__link">
        golf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/golfc/" class="md-nav__link">
        golfc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gomkl/" class="md-nav__link">
        gomkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gomklc/" class="md-nav__link">
        gomklc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gompi/" class="md-nav__link">
        gompi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gompic/" class="md-nav__link">
        gompic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/goolf/" class="md-nav__link">
        goolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/goolfc/" class="md-nav__link">
        goolfc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gpsmpi/" class="md-nav__link">
        gpsmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gpsolf/" class="md-nav__link">
        gpsolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gqacml/" class="md-nav__link">
        gqacml
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gsmpi/" class="md-nav__link">
        gsmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/gsolf/" class="md-nav__link">
        gsolf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iccifort/" class="md-nav__link">
        iccifort
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iccifortcuda/" class="md-nav__link">
        iccifortcuda
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/ictce/" class="md-nav__link">
        ictce
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iibff/" class="md-nav__link">
        iibff
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iimkl/" class="md-nav__link">
        iimkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iimklc/" class="md-nav__link">
        iimklc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iimpi/" class="md-nav__link">
        iimpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iimpic/" class="md-nav__link">
        iimpic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iiqmpi/" class="md-nav__link">
        iiqmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/impich/" class="md-nav__link">
        impich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/impmkl/" class="md-nav__link">
        impmkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/intel-para/" class="md-nav__link">
        intel-para
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/intel/" class="md-nav__link">
        intel
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/intel_compilers/" class="md-nav__link">
        intel_compilers
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/intelcuda/" class="md-nav__link">
        intelcuda
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iomkl/" class="md-nav__link">
        iomkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iomklc/" class="md-nav__link">
        iomklc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iompi/" class="md-nav__link">
        iompi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iompic/" class="md-nav__link">
        iompic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/ipsmpi/" class="md-nav__link">
        ipsmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/iqacml/" class="md-nav__link">
        iqacml
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/ismkl/" class="md-nav__link">
        ismkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_5_78" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/toolchains/linalg/">linalg</a>
          
            <label for="__nav_11_1_5_78">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_5_78_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_5_78">
          <span class="md-nav__icon md-icon"></span>
          linalg
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/acml/" class="md-nav__link">
        acml
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/atlas/" class="md-nav__link">
        atlas
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/blacs/" class="md-nav__link">
        blacs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/blis/" class="md-nav__link">
        blis
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/flame/" class="md-nav__link">
        flame
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/flexiblas/" class="md-nav__link">
        flexiblas
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/fujitsussl/" class="md-nav__link">
        fujitsussl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/gotoblas/" class="md-nav__link">
        gotoblas
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/intelmkl/" class="md-nav__link">
        intelmkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/lapack/" class="md-nav__link">
        lapack
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/libsci/" class="md-nav__link">
        libsci
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/openblas/" class="md-nav__link">
        openblas
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/linalg/scalapack/" class="md-nav__link">
        scalapack
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_5_79" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/toolchains/mpi/">mpi</a>
          
            <label for="__nav_11_1_5_79">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_5_79_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_5_79">
          <span class="md-nav__icon md-icon"></span>
          mpi
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/craympich/" class="md-nav__link">
        craympich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/fujitsumpi/" class="md-nav__link">
        fujitsumpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/intelmpi/" class="md-nav__link">
        intelmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/mpich/" class="md-nav__link">
        mpich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/mpich2/" class="md-nav__link">
        mpich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/mpitrampoline/" class="md-nav__link">
        mpitrampoline
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/mvapich2/" class="md-nav__link">
        mvapich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/openmpi/" class="md-nav__link">
        openmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/psmpi/" class="md-nav__link">
        psmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/qlogicmpi/" class="md-nav__link">
        qlogicmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/mpi/spectrummpi/" class="md-nav__link">
        spectrummpi
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvhpc/" class="md-nav__link">
        nvhpc
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvofbf/" class="md-nav__link">
        nvofbf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvompi/" class="md-nav__link">
        nvompi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvompic/" class="md-nav__link">
        nvompic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvpsmpi/" class="md-nav__link">
        nvpsmpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/nvpsmpic/" class="md-nav__link">
        nvpsmpic
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/pgi/" class="md-nav__link">
        pgi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/pmkl/" class="md-nav__link">
        pmkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/pomkl/" class="md-nav__link">
        pomkl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/pompi/" class="md-nav__link">
        pompi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/system/" class="md-nav__link">
        system
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/xlcxlf/" class="md-nav__link">
        xlcxlf
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/xlmpich/" class="md-nav__link">
        xlmpich
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/xlmpich2/" class="md-nav__link">
        xlmpich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/xlmvapich2/" class="md-nav__link">
        xlmvapich2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/toolchains/xlompi/" class="md-nav__link">
        xlompi
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/">tools</a>
          
            <label for="__nav_11_1_6">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_11_1_6_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6">
          <span class="md-nav__icon md-icon"></span>
          tools
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/asyncprocess/" class="md-nav__link">
        asyncprocess
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/build_details/" class="md-nav__link">
        build_details
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/build_log/" class="md-nav__link">
        build_log
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/config/" class="md-nav__link">
        config
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/configobj/" class="md-nav__link">
        configobj
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_6" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/containers/">containers</a>
          
            <label for="__nav_11_1_6_6">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_6_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_6">
          <span class="md-nav__icon md-icon"></span>
          containers
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/containers/base/" class="md-nav__link">
        base
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/containers/common/" class="md-nav__link">
        common
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/containers/docker/" class="md-nav__link">
        docker
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/containers/singularity/" class="md-nav__link">
        singularity
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/containers/utils/" class="md-nav__link">
        utils
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/convert/" class="md-nav__link">
        convert
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/deprecated/" class="md-nav__link">
        deprecated
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/docs/" class="md-nav__link">
        docs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/environment/" class="md-nav__link">
        environment
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/filetools/" class="md-nav__link">
        filetools
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/github/" class="md-nav__link">
        github
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/hooks/" class="md-nav__link">
        hooks
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/include/" class="md-nav__link">
        include
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/jenkins/" class="md-nav__link">
        jenkins
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_16" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/job/">job</a>
          
            <label for="__nav_11_1_6_16">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_16_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_16">
          <span class="md-nav__icon md-icon"></span>
          job
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/job/backend/" class="md-nav__link">
        backend
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/job/gc3pie/" class="md-nav__link">
        gc3pie
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/job/pbs_python/" class="md-nav__link">
        pbs_python
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/job/slurm/" class="md-nav__link">
        slurm
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/loose_version/" class="md-nav__link">
        loose_version
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_generator/" class="md-nav__link">
        module_generator
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_19" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/module_naming_scheme/">module_naming_scheme</a>
          
            <label for="__nav_11_1_6_19">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_19_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_19">
          <span class="md-nav__icon md-icon"></span>
          module_naming_scheme
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/categorized_hmns/" class="md-nav__link">
        categorized_hmns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/categorized_mns/" class="md-nav__link">
        categorized_mns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/easybuild_mns/" class="md-nav__link">
        easybuild_mns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/hierarchical_mns/" class="md-nav__link">
        hierarchical_mns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/migrate_from_eb_to_hmns/" class="md-nav__link">
        migrate_from_eb_to_hmns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/mns/" class="md-nav__link">
        mns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/toolchain/" class="md-nav__link">
        toolchain
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/module_naming_scheme/utilities/" class="md-nav__link">
        utilities
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/modules/" class="md-nav__link">
        modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/multidiff/" class="md-nav__link">
        multidiff
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/options/" class="md-nav__link">
        options
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/output/" class="md-nav__link">
        output
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_24" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/package/">package</a>
          
            <label for="__nav_11_1_6_24">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_24_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_24">
          <span class="md-nav__icon md-icon"></span>
          package
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_24_1" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/package/package_naming_scheme/">package_naming_scheme</a>
          
            <label for="__nav_11_1_6_24_1">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="5" aria-labelledby="__nav_11_1_6_24_1_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_24_1">
          <span class="md-nav__icon md-icon"></span>
          package_naming_scheme
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/package/package_naming_scheme/easybuild_deb_friendly_pns/" class="md-nav__link">
        easybuild_deb_friendly_pns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/package/package_naming_scheme/easybuild_pns/" class="md-nav__link">
        easybuild_pns
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/package/package_naming_scheme/pns/" class="md-nav__link">
        pns
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/package/utilities/" class="md-nav__link">
        utilities
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/parallelbuild/" class="md-nav__link">
        parallelbuild
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_26" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/py2vs3/">py2vs3</a>
          
            <label for="__nav_11_1_6_26">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_26_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_26">
          <span class="md-nav__icon md-icon"></span>
          py2vs3
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/py2vs3/py2/" class="md-nav__link">
        py2
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/py2vs3/py3/" class="md-nav__link">
        py3
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_27" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/repository/">repository</a>
          
            <label for="__nav_11_1_6_27">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_27_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_27">
          <span class="md-nav__icon md-icon"></span>
          repository
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/repository/filerepo/" class="md-nav__link">
        filerepo
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/repository/gitrepo/" class="md-nav__link">
        gitrepo
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/repository/hgrepo/" class="md-nav__link">
        hgrepo
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/repository/repository/" class="md-nav__link">
        repository
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/repository/svnrepo/" class="md-nav__link">
        svnrepo
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/robot/" class="md-nav__link">
        robot
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/run/" class="md-nav__link">
        run
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/systemtools/" class="md-nav__link">
        systemtools
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/testing/" class="md-nav__link">
        testing
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11_1_6_32" >
      
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../../api/easybuild/tools/toolchain/">toolchain</a>
          
            <label for="__nav_11_1_6_32">
              <span class="md-nav__icon md-icon"></span>
            </label>
          
        </div>
      
      <nav class="md-nav" data-md-level="4" aria-labelledby="__nav_11_1_6_32_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_11_1_6_32">
          <span class="md-nav__icon md-icon"></span>
          toolchain
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/compiler/" class="md-nav__link">
        compiler
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/constants/" class="md-nav__link">
        constants
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/fft/" class="md-nav__link">
        fft
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/linalg/" class="md-nav__link">
        linalg
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/mpi/" class="md-nav__link">
        mpi
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/options/" class="md-nav__link">
        options
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/toolchain/" class="md-nav__link">
        toolchain
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/toolchainvariables/" class="md-nav__link">
        toolchainvariables
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/utilities/" class="md-nav__link">
        utilities
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/toolchain/variables/" class="md-nav__link">
        variables
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/utilities/" class="md-nav__link">
        utilities
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/variables/" class="md-nav__link">
        variables
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../api/easybuild/tools/version/" class="md-nav__link">
        version
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          List of supported software
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        List of supported software
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#3" class="md-nav__link">
    3
  </a>
  
    <nav class="md-nav" aria-label="3">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#3to2" class="md-nav__link">
    3to2
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#4" class="md-nav__link">
    4
  </a>
  
    <nav class="md-nav" aria-label="4">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#4ti2" class="md-nav__link">
    4ti2
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#a" class="md-nav__link">
    A
  </a>
  
    <nav class="md-nav" aria-label="A">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#abaqus" class="md-nav__link">
    ABAQUS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abinit" class="md-nav__link">
    ABINIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abra2" class="md-nav__link">
    ABRA2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abricate" class="md-nav__link">
    ABRicate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abseil" class="md-nav__link">
    Abseil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abtem" class="md-nav__link">
    abTEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#abyss" class="md-nav__link">
    ABySS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ack" class="md-nav__link">
    ack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#act" class="md-nav__link">
    ACT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#actc" class="md-nav__link">
    ACTC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ada" class="md-nav__link">
    ada
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adapterremoval" class="md-nav__link">
    AdapterRemoval
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adda" class="md-nav__link">
    ADDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adf" class="md-nav__link">
    ADF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adios" class="md-nav__link">
    ADIOS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adjusttext" class="md-nav__link">
    adjustText
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#admixture" class="md-nav__link">
    ADMIXTURE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#adol-c" class="md-nav__link">
    ADOL-C
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#advisor" class="md-nav__link">
    Advisor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#afni" class="md-nav__link">
    AFNI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#agat" class="md-nav__link">
    AGAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#agfusion" class="md-nav__link">
    AGFusion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aiohttp" class="md-nav__link">
    aiohttp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aladin" class="md-nav__link">
    ALADIN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#albacore" class="md-nav__link">
    Albacore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#albumentations" class="md-nav__link">
    Albumentations
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alevin-fry" class="md-nav__link">
    alevin-fry
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alfa" class="md-nav__link">
    ALFA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alfred" class="md-nav__link">
    Alfred
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#allelecount" class="md-nav__link">
    alleleCount
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#allinea" class="md-nav__link">
    Allinea
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#allpaths-lg" class="md-nav__link">
    ALLPATHS-LG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#almosthere" class="md-nav__link">
    almosthere
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alpha" class="md-nav__link">
    Alpha
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alphafold" class="md-nav__link">
    AlphaFold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alps" class="md-nav__link">
    ALPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alsa-lib" class="md-nav__link">
    alsa-lib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amapvox" class="md-nav__link">
    AMAPVox
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amara" class="md-nav__link">
    Amara
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amask" class="md-nav__link">
    amask
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amber" class="md-nav__link">
    Amber
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ambermini" class="md-nav__link">
    AmberMini
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ambertools" class="md-nav__link">
    AmberTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amd-libm" class="md-nav__link">
    AMD-LibM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amd-rng" class="md-nav__link">
    AMD-RNG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amd-securerng" class="md-nav__link">
    AMD-SecureRNG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amd-uprof" class="md-nav__link">
    AMD-uProf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amos" class="md-nav__link">
    AMOS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amphora2" class="md-nav__link">
    AMPHORA2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ampl-mp" class="md-nav__link">
    AMPL-MP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amplimap" class="md-nav__link">
    amplimap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amptk" class="md-nav__link">
    AMPtk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amrplusplus" class="md-nav__link">
    AmrPlusPlus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ams" class="md-nav__link">
    AMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#anaconda2" class="md-nav__link">
    Anaconda2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#anaconda3" class="md-nav__link">
    Anaconda3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#anci" class="md-nav__link">
    aNCI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#andi" class="md-nav__link">
    andi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#angel" class="md-nav__link">
    ANGEL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#angsd" class="md-nav__link">
    angsd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#anicalculator" class="md-nav__link">
    ANIcalculator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#annif" class="md-nav__link">
    Annif
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#annovar" class="md-nav__link">
    annovar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ansys" class="md-nav__link">
    ANSYS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ansys_cfd" class="md-nav__link">
    ANSYS_CFD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ant" class="md-nav__link">
    ant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#antismash" class="md-nav__link">
    antiSMASH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#antlr" class="md-nav__link">
    ANTLR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ants" class="md-nav__link">
    ANTs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#anvio" class="md-nav__link">
    anvio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#any2fasta" class="md-nav__link">
    any2fasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aocc" class="md-nav__link">
    AOCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aomp" class="md-nav__link">
    AOMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#apbs" class="md-nav__link">
    APBS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#apex" class="md-nav__link">
    apex
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#apr" class="md-nav__link">
    APR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#apr-util" class="md-nav__link">
    APR-util
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aptasuite" class="md-nav__link">
    AptaSUITE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aragorn" class="md-nav__link">
    ARAGORN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arb" class="md-nav__link">
    Arb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arcade-learning-environment" class="md-nav__link">
    Arcade-Learning-Environment
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arcashla" class="md-nav__link">
    arcasHLA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arch" class="md-nav__link">
    ARCH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#archive-zip" class="md-nav__link">
    Archive-Zip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#archr" class="md-nav__link">
    ArchR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#archspec" class="md-nav__link">
    archspec
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#argos" class="md-nav__link">
    ARGoS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#argtable" class="md-nav__link">
    argtable
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aria2" class="md-nav__link">
    aria2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arlequin" class="md-nav__link">
    Arlequin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#armadillo" class="md-nav__link">
    Armadillo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arosics" class="md-nav__link">
    arosics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arpack" class="md-nav__link">
    ARPACK++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arpack-ng" class="md-nav__link">
    arpack-ng
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arrayfire" class="md-nav__link">
    ArrayFire
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arriba" class="md-nav__link">
    Arriba
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arrow" class="md-nav__link">
    Arrow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arrow-r" class="md-nav__link">
    arrow-R
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#art" class="md-nav__link">
    ART
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#artic-ncov2019" class="md-nav__link">
    artic-ncov2019
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arts" class="md-nav__link">
    ARTS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arviz" class="md-nav__link">
    ArviZ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#arwen" class="md-nav__link">
    ARWEN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#asap" class="md-nav__link">
    ASAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#asap3" class="md-nav__link">
    ASAP3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ase" class="md-nav__link">
    ASE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ashs" class="md-nav__link">
    ASHS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aspera-cli" class="md-nav__link">
    Aspera-CLI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#aspera-connect" class="md-nav__link">
    Aspera-Connect
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assimp" class="md-nav__link">
    assimp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assimulo" class="md-nav__link">
    Assimulo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#astrid" class="md-nav__link">
    ASTRID
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#astro-tulips" class="md-nav__link">
    astro-tulips
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#astropy" class="md-nav__link">
    astropy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#at-spi2-atk" class="md-nav__link">
    at-spi2-atk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#at-spi2-core" class="md-nav__link">
    at-spi2-core
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atk" class="md-nav__link">
    ATK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atlas" class="md-nav__link">
    ATLAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atomate" class="md-nav__link">
    atomate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atompaw" class="md-nav__link">
    AtomPAW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atools" class="md-nav__link">
    atools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atropos" class="md-nav__link">
    atropos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#atsas" class="md-nav__link">
    ATSAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#attr" class="md-nav__link">
    attr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#attrdict" class="md-nav__link">
    attrdict
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#attrdict3" class="md-nav__link">
    attrdict3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#augur" class="md-nav__link">
    augur
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#augustus" class="md-nav__link">
    AUGUSTUS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#austin" class="md-nav__link">
    Austin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#auto-07p" class="md-nav__link">
    AUTO-07p
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autoconf" class="md-nav__link">
    Autoconf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autoconf-archive" class="md-nav__link">
    Autoconf-archive
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autodock" class="md-nav__link">
    AutoDock
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autodock-gpu" class="md-nav__link">
    AutoDock-GPU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autodock-vina" class="md-nav__link">
    AutoDock-Vina
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autogenes" class="md-nav__link">
    AutoGeneS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autogrid" class="md-nav__link">
    AutoGrid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#automake" class="md-nav__link">
    Automake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#automap" class="md-nav__link">
    AutoMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autopep8" class="md-nav__link">
    autopep8
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#autotools" class="md-nav__link">
    Autotools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#avogadro2" class="md-nav__link">
    Avogadro2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#awscli" class="md-nav__link">
    awscli
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#b" class="md-nav__link">
    B
  </a>
  
    <nav class="md-nav" aria-label="B">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ba3-snps-autotune" class="md-nav__link">
    BA3-SNPS-autotune
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#babelstream" class="md-nav__link">
    BabelStream
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#babl" class="md-nav__link">
    babl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bader" class="md-nav__link">
    Bader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bagel" class="md-nav__link">
    BAGEL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bali-phy" class="md-nav__link">
    BAli-Phy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bam-readcount" class="md-nav__link">
    bam-readcount
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bambi" class="md-nav__link">
    Bambi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bamfilters" class="md-nav__link">
    bamFilters
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bamsurgeon" class="md-nav__link">
    BAMSurgeon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bamtofastq" class="md-nav__link">
    bamtofastq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bamtools" class="md-nav__link">
    BamTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bamutil" class="md-nav__link">
    BamUtil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bandage" class="md-nav__link">
    Bandage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#barrnap" class="md-nav__link">
    barrnap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#basemap" class="md-nav__link">
    basemap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bash" class="md-nav__link">
    Bash
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bashplotlib" class="md-nav__link">
    bashplotlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bat" class="md-nav__link">
    bat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#batmeth2" class="md-nav__link">
    BatMeth2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayesass" class="md-nav__link">
    BayesAss
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayesass3-snps" class="md-nav__link">
    BayesAss3-SNPs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayescan" class="md-nav__link">
    BayeScan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayescenv" class="md-nav__link">
    BayeScEnv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayestraits" class="md-nav__link">
    BayesTraits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bazel" class="md-nav__link">
    Bazel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bbcp" class="md-nav__link">
    bbcp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bbftp" class="md-nav__link">
    bbFTP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bbmap" class="md-nav__link">
    BBMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bc" class="md-nav__link">
    bc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcalm" class="md-nav__link">
    BCALM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcbio-gff" class="md-nav__link">
    bcbio-gff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcel" class="md-nav__link">
    BCEL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcftools" class="md-nav__link">
    BCFtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcgtree" class="md-nav__link">
    bcgTree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcl-convert" class="md-nav__link">
    bcl-convert
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcl2fastq2" class="md-nav__link">
    bcl2fastq2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bcolz" class="md-nav__link">
    bcolz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bdbag" class="md-nav__link">
    BDBag
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    Beagle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle-lib" class="md-nav__link">
    beagle-lib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beast" class="md-nav__link">
    Beast
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beautifulsoup" class="md-nav__link">
    BeautifulSoup
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bedops" class="md-nav__link">
    BEDOPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bedtools" class="md-nav__link">
    BEDTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beef" class="md-nav__link">
    BEEF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#behave" class="md-nav__link">
    behave
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#berkeleygw" class="md-nav__link">
    BerkeleyGW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bfast" class="md-nav__link">
    BFAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bfc" class="md-nav__link">
    BFC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bgen" class="md-nav__link">
    bgen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bgen-reader" class="md-nav__link">
    bgen-reader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bibtexparser" class="md-nav__link">
    bibtexparser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#big-scape" class="md-nav__link">
    BiG-SCAPE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bigdft" class="md-nav__link">
    BigDFT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#binsanity" class="md-nav__link">
    BinSanity
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#binutils" class="md-nav__link">
    binutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bio-db-hts" class="md-nav__link">
    Bio-DB-HTS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bio-eutilities" class="md-nav__link">
    Bio-EUtilities
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bio-samtools" class="md-nav__link">
    Bio-SamTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bio-searchio-hmmer" class="md-nav__link">
    Bio-SearchIO-hmmer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bioawk" class="md-nav__link">
    bioawk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biobambam2" class="md-nav__link">
    biobambam2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biogeme" class="md-nav__link">
    biogeme
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biom-format" class="md-nav__link">
    biom-format
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biomart-perl" class="md-nav__link">
    biomart-perl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bioperl" class="md-nav__link">
    BioPerl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biopp" class="md-nav__link">
    BioPP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biopython" class="md-nav__link">
    Biopython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bioservices" class="md-nav__link">
    BioServices
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#birdnet" class="md-nav__link">
    BirdNET
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biscuit" class="md-nav__link">
    biscuit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bismark" class="md-nav__link">
    Bismark
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bison" class="md-nav__link">
    Bison
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bitarray" class="md-nav__link">
    bitarray
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blacs" class="md-nav__link">
    BLACS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blasr" class="md-nav__link">
    BLASR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blasr_libcpp" class="md-nav__link">
    blasr_libcpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blast" class="md-nav__link">
    BLAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blast_1" class="md-nav__link">
    BLAST+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blat" class="md-nav__link">
    BLAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blender" class="md-nav__link">
    Blender
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blis" class="md-nav__link">
    BLIS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blitz" class="md-nav__link">
    Blitz++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blobtools" class="md-nav__link">
    BlobTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#block" class="md-nav__link">
    Block
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blosc" class="md-nav__link">
    Blosc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blosc2" class="md-nav__link">
    Blosc2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#blt" class="md-nav__link">
    BLT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bmtagger" class="md-nav__link">
    bmtagger
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bnpy" class="md-nav__link">
    bnpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boinc" class="md-nav__link">
    BOINC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bokeh" class="md-nav__link">
    bokeh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boltztrap" class="md-nav__link">
    BoltzTraP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bonito" class="md-nav__link">
    Bonito
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bonmin" class="md-nav__link">
    Bonmin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bonnie" class="md-nav__link">
    Bonnie++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boost" class="md-nav__link">
    Boost
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boostmpi" class="md-nav__link">
    Boost.MPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boostpython" class="md-nav__link">
    Boost.Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boost_histogram" class="md-nav__link">
    boost_histogram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#boto3" class="md-nav__link">
    boto3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bottleneck" class="md-nav__link">
    Bottleneck
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bowtie" class="md-nav__link">
    Bowtie
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bowtie2" class="md-nav__link">
    Bowtie2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bpipe" class="md-nav__link">
    Bpipe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bpp" class="md-nav__link">
    bpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bpytop" class="md-nav__link">
    bpytop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bracken" class="md-nav__link">
    Bracken
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#braker" class="md-nav__link">
    BRAKER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#breakdancer" class="md-nav__link">
    BreakDancer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#breseq" class="md-nav__link">
    breseq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#brotli" class="md-nav__link">
    Brotli
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#brotli-python" class="md-nav__link">
    Brotli-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#brunsli" class="md-nav__link">
    Brunsli
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bsddb3" class="md-nav__link">
    bsddb3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bsmapz" class="md-nav__link">
    BSMAPz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bsoft" class="md-nav__link">
    Bsoft
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bsseeker2" class="md-nav__link">
    BSseeker2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#buddy" class="md-nav__link">
    BuDDy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bufrlib" class="md-nav__link">
    BUFRLIB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#buildenv" class="md-nav__link">
    buildenv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#buildingspy" class="md-nav__link">
    buildingspy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bullet" class="md-nav__link">
    Bullet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#busco" class="md-nav__link">
    BUSCO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bustools" class="md-nav__link">
    BUStools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bwa" class="md-nav__link">
    BWA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bwa-meth" class="md-nav__link">
    bwa-meth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bwakit" class="md-nav__link">
    bwakit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bwidget" class="md-nav__link">
    bwidget
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bwise" class="md-nav__link">
    BWISE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bx-python" class="md-nav__link">
    bx-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bxh_xcede_tools" class="md-nav__link">
    BXH_XCEDE_TOOLS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#byacc" class="md-nav__link">
    byacc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#byobu" class="md-nav__link">
    byobu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bzip2" class="md-nav__link">
    bzip2
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#c" class="md-nav__link">
    C
  </a>
  
    <nav class="md-nav" aria-label="C">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#c-ares" class="md-nav__link">
    c-ares
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#c3d" class="md-nav__link">
    C3D
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cadaver" class="md-nav__link">
    cadaver
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cadical" class="md-nav__link">
    CaDiCaL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cafe5" class="md-nav__link">
    CAFE5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caffe" class="md-nav__link">
    Caffe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cairo" class="md-nav__link">
    cairo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cairomm" class="md-nav__link">
    cairomm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#calcam" class="md-nav__link">
    Calcam
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#calendrical" class="md-nav__link">
    Calendrical
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#calib" class="md-nav__link">
    Calib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#campari" class="md-nav__link">
    CAMPARI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cantera" class="md-nav__link">
    Cantera
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#canu" class="md-nav__link">
    canu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#canvas" class="md-nav__link">
    Canvas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cap3" class="md-nav__link">
    CAP3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#capnproto" class="md-nav__link">
    CapnProto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#captum" class="md-nav__link">
    captum
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cargo" class="md-nav__link">
    Cargo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carma" class="md-nav__link">
    Carma
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carputils" class="md-nav__link">
    carputils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cartopy" class="md-nav__link">
    Cartopy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#casper" class="md-nav__link">
    CaSpER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#castep" class="md-nav__link">
    CASTEP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#castxml" class="md-nav__link">
    CastXML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cat-bat" class="md-nav__link">
    CAT-BAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#catch2" class="md-nav__link">
    Catch2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#category_encoders" class="md-nav__link">
    category_encoders
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#catlearn" class="md-nav__link">
    CatLearn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#catmap" class="md-nav__link">
    CatMAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causallift" class="md-nav__link">
    causallift
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causalml" class="md-nav__link">
    causalml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caveman" class="md-nav__link">
    CaVEMan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caviar" class="md-nav__link">
    CAVIAR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cbc" class="md-nav__link">
    Cbc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cblas" class="md-nav__link">
    CBLAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ccache" class="md-nav__link">
    ccache
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ccfits" class="md-nav__link">
    CCfits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ccl" class="md-nav__link">
    CCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cclib" class="md-nav__link">
    cclib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cctools" class="md-nav__link">
    cctools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cd-hit" class="md-nav__link">
    CD-HIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdat" class="md-nav__link">
    CDAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdbfasta" class="md-nav__link">
    cdbfasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdbtools" class="md-nav__link">
    CDBtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cddlib" class="md-nav__link">
    cddlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdflib" class="md-nav__link">
    CDFlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdna_cupcake" class="md-nav__link">
    cDNA_Cupcake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdo" class="md-nav__link">
    CDO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdo-bindings" class="md-nav__link">
    cdo-bindings
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cdsapi" class="md-nav__link">
    cdsapi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cell2location" class="md-nav__link">
    cell2location
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellmix" class="md-nav__link">
    CellMix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellpose" class="md-nav__link">
    Cellpose
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellranger" class="md-nav__link">
    CellRanger
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellranger-arc" class="md-nav__link">
    CellRanger-ARC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellranger-atac" class="md-nav__link">
    CellRanger-ATAC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cellrank" class="md-nav__link">
    CellRank
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#celltypist" class="md-nav__link">
    CellTypist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#centrifuge" class="md-nav__link">
    Centrifuge
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cereal" class="md-nav__link">
    Cereal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cesm-deps" class="md-nav__link">
    CESM-deps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cfdemcoupling" class="md-nav__link">
    CFDEMcoupling
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cfitsio" class="md-nav__link">
    CFITSIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cftime" class="md-nav__link">
    cftime
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cgal" class="md-nav__link">
    CGAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cget" class="md-nav__link">
    cget
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cgl" class="md-nav__link">
    Cgl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cgmaptools" class="md-nav__link">
    CGmapTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cgns" class="md-nav__link">
    CGNS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#charls" class="md-nav__link">
    CharLS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chase" class="md-nav__link">
    CHASE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#check" class="md-nav__link">
    Check
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#checkm" class="md-nav__link">
    CheckM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#checkm-database" class="md-nav__link">
    CheckM-Database
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cheetah" class="md-nav__link">
    Cheetah
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chemaxon-marvin" class="md-nav__link">
    Chemaxon-Marvin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chemprop" class="md-nav__link">
    chemprop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chemps2" class="md-nav__link">
    CheMPS2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cherab" class="md-nav__link">
    CHERAB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chewbbaca" class="md-nav__link">
    chewBBACA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chi2comb" class="md-nav__link">
    chi2comb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chimera" class="md-nav__link">
    Chimera
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chimpipe" class="md-nav__link">
    ChimPipe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chromaprint" class="md-nav__link">
    Chromaprint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cicero" class="md-nav__link">
    cicero
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cif2cell" class="md-nav__link">
    CIF2Cell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cimfomfa" class="md-nav__link">
    cimfomfa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#circexplorer" class="md-nav__link">
    CIRCexplorer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#circexplorer2" class="md-nav__link">
    CIRCexplorer2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#circos" class="md-nav__link">
    Circos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ciri" class="md-nav__link">
    CIRI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ciri-long" class="md-nav__link">
    CIRI-long
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cistem" class="md-nav__link">
    cisTEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cite-seq-count" class="md-nav__link">
    CITE-seq-Count
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clang" class="md-nav__link">
    Clang
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clang-aomp" class="md-nav__link">
    Clang-AOMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clang-python-bindings" class="md-nav__link">
    Clang-Python-bindings
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clapack" class="md-nav__link">
    CLAPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clease" class="md-nav__link">
    CLEASE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clhep" class="md-nav__link">
    CLHEP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#climetlab" class="md-nav__link">
    CliMetLab
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clisp" class="md-nav__link">
    CLISP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clonalframeml" class="md-nav__link">
    ClonalFrameML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cloog" class="md-nav__link">
    CLooG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cloudcompare" class="md-nav__link">
    CloudCompare
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clp" class="md-nav__link">
    Clp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clustal-omega" class="md-nav__link">
    Clustal-Omega
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clustalw2" class="md-nav__link">
    ClustalW2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cluster-buster" class="md-nav__link">
    Cluster-Buster
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#clustershell" class="md-nav__link">
    ClusterShell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cmake" class="md-nav__link">
    CMake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cmaverse" class="md-nav__link">
    CMAverse
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cmdstanr" class="md-nav__link">
    CmdStanR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cmocean" class="md-nav__link">
    cmocean
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cmseq" class="md-nav__link">
    CMSeq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cnt-ilp" class="md-nav__link">
    CNT-ILP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cnvkit" class="md-nav__link">
    CNVkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cnvnator" class="md-nav__link">
    CNVnator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#co-phylog" class="md-nav__link">
    Co-phylog
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cobrapy" class="md-nav__link">
    COBRApy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cocoalib" class="md-nav__link">
    CoCoALib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#codan" class="md-nav__link">
    CodAn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#code-server" class="md-nav__link">
    code-server
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#codex2" class="md-nav__link">
    CODEX2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cogent" class="md-nav__link">
    Cogent
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coin" class="md-nav__link">
    Coin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coinutils" class="md-nav__link">
    CoinUtils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#colossalai" class="md-nav__link">
    colossalai
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#commet" class="md-nav__link">
    Commet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#comparem" class="md-nav__link">
    CompareM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#compress-raw-zlib" class="md-nav__link">
    Compress-Raw-Zlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#comsol" class="md-nav__link">
    COMSOL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#con3f" class="md-nav__link">
    Con3F
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#conan" class="md-nav__link">
    conan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#concoct" class="md-nav__link">
    CONCOCT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#configparser" class="md-nav__link">
    configparser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#configurable-http-proxy" class="md-nav__link">
    configurable-http-proxy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#conn" class="md-nav__link">
    CONN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#connectomeworkbench" class="md-nav__link">
    ConnectomeWorkbench
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#control-freec" class="md-nav__link">
    Control-FREEC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cooler" class="md-nav__link">
    cooler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coordgenlibs" class="md-nav__link">
    CoordgenLibs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coot" class="md-nav__link">
    Coot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#core-counter" class="md-nav__link">
    core-counter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coreutils" class="md-nav__link">
    Coreutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#corner" class="md-nav__link">
    corner
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cosymlib" class="md-nav__link">
    CoSymLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#coverage" class="md-nav__link">
    coverage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cowsay" class="md-nav__link">
    cowsay
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cp2k" class="md-nav__link">
    CP2K
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cpb" class="md-nav__link">
    CPB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cplex" class="md-nav__link">
    CPLEX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cppe" class="md-nav__link">
    CPPE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cppunit" class="md-nav__link">
    CppUnit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cppy" class="md-nav__link">
    cppy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cpu_features" class="md-nav__link">
    cpu_features
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cram" class="md-nav__link">
    cram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cramtools" class="md-nav__link">
    cramtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#craycce" class="md-nav__link">
    CrayCCE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#craygnu" class="md-nav__link">
    CrayGNU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crayintel" class="md-nav__link">
    CrayIntel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#craypgi" class="md-nav__link">
    CrayPGI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crb-blast" class="md-nav__link">
    crb-blast
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crest" class="md-nav__link">
    CREST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crf" class="md-nav__link">
    CRF++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crispr-dav" class="md-nav__link">
    CRISPR-DAV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crispresso2" class="md-nav__link">
    CRISPResso2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cromwell" class="md-nav__link">
    cromwell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crossguid" class="md-nav__link">
    crossguid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crossmap" class="md-nav__link">
    CrossMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crosstalkz" class="md-nav__link">
    CrossTalkZ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crpropa" class="md-nav__link">
    CRPropa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crumble" class="md-nav__link">
    Crumble
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cryocare" class="md-nav__link">
    cryoCARE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cryodrgn" class="md-nav__link">
    cryoDRGN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cryptominisat" class="md-nav__link">
    CryptoMiniSat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#crystfel" class="md-nav__link">
    CrystFEL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#csb" class="md-nav__link">
    CSB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#csbdeep" class="md-nav__link">
    CSBDeep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#csblast" class="md-nav__link">
    CSBLAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cscope" class="md-nav__link">
    cscope
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#csvkit" class="md-nav__link">
    csvkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ctags" class="md-nav__link">
    ctags
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ctffind" class="md-nav__link">
    ctffind
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ctpl" class="md-nav__link">
    CTPL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cube" class="md-nav__link">
    Cube
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cubegui" class="md-nav__link">
    CubeGUI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cubelib" class="md-nav__link">
    CubeLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cubewriter" class="md-nav__link">
    CubeWriter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cuda" class="md-nav__link">
    CUDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cuda-samples" class="md-nav__link">
    CUDA-Samples
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cudacompat" class="md-nav__link">
    CUDAcompat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cudacore" class="md-nav__link">
    CUDAcore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cudnn" class="md-nav__link">
    cuDNN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cufflinks" class="md-nav__link">
    Cufflinks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cunit" class="md-nav__link">
    CUnit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cupy" class="md-nav__link">
    CuPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#curl" class="md-nav__link">
    cURL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cusparselt" class="md-nav__link">
    cuSPARSELt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#custodian" class="md-nav__link">
    custodian
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cutadapt" class="md-nav__link">
    cutadapt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cutensor" class="md-nav__link">
    cuTENSOR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cvs" class="md-nav__link">
    CVS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cvxopt" class="md-nav__link">
    CVXOPT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cvxpy" class="md-nav__link">
    CVXPY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cwipi" class="md-nav__link">
    CWIPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cwltool" class="md-nav__link">
    cwltool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cxxopts" class="md-nav__link">
    cxxopts
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cysignals" class="md-nav__link">
    cysignals
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cython" class="md-nav__link">
    Cython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cython-blis" class="md-nav__link">
    cython-blis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cytoolz" class="md-nav__link">
    cytoolz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cytoscape" class="md-nav__link">
    Cytoscape
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cytosim" class="md-nav__link">
    cytosim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cyvcf2" class="md-nav__link">
    cyvcf2
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#d" class="md-nav__link">
    D
  </a>
  
    <nav class="md-nav" aria-label="D">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#dadi" class="md-nav__link">
    dadi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dagitty" class="md-nav__link">
    dagitty
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dakota" class="md-nav__link">
    Dakota
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dalton" class="md-nav__link">
    Dalton
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#damageproto" class="md-nav__link">
    damageproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dammit" class="md-nav__link">
    dammit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#danpos2" class="md-nav__link">
    DANPOS2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#das_tool" class="md-nav__link">
    DAS_Tool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dask" class="md-nav__link">
    dask
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#datamash" class="md-nav__link">
    datamash
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#davix" class="md-nav__link">
    davix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#db" class="md-nav__link">
    DB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#db_file" class="md-nav__link">
    DB_File
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbcsr" class="md-nav__link">
    DBCSR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbd-mysql" class="md-nav__link">
    DBD-mysql
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbg2olc" class="md-nav__link">
    DBG2OLC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbus" class="md-nav__link">
    DBus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbus-glib" class="md-nav__link">
    dbus-glib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dclone" class="md-nav__link">
    dclone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dcm2niix" class="md-nav__link">
    dcm2niix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dcmtk" class="md-nav__link">
    DCMTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dd" class="md-nav__link">
    dd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dealii" class="md-nav__link">
    deal.II
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deap" class="md-nav__link">
    deap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#decona" class="md-nav__link">
    decona
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deconf" class="md-nav__link">
    deconf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deconica" class="md-nav__link">
    DeconICA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deepdiff" class="md-nav__link">
    deepdiff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deeplabcut" class="md-nav__link">
    DeepLabCut
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deepmedic" class="md-nav__link">
    deepmedic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deepmod2" class="md-nav__link">
    DeepMod2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deepsurv" class="md-nav__link">
    DeepSurv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deeptools" class="md-nav__link">
    deepTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#delly" class="md-nav__link">
    Delly
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#demixt" class="md-nav__link">
    DeMixT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dendropy" class="md-nav__link">
    DendroPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#denspart" class="md-nav__link">
    DensPart
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deprecated" class="md-nav__link">
    Deprecated
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#destiny" class="md-nav__link">
    destiny
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#detectron2" class="md-nav__link">
    Detectron2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#detonate" class="md-nav__link">
    detonate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#devito" class="md-nav__link">
    Devito
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dfa" class="md-nav__link">
    DFA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dft-d3" class="md-nav__link">
    DFT-D3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dft-d4" class="md-nav__link">
    DFT-D4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dftb" class="md-nav__link">
    DFTB+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dftd3-lib" class="md-nav__link">
    dftd3-lib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dgl" class="md-nav__link">
    DGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dial" class="md-nav__link">
    DIAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#diamond" class="md-nav__link">
    DIAMOND
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dice" class="md-nav__link">
    Dice
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dicom2nifti" class="md-nav__link">
    dicom2nifti
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dicombrowser" class="md-nav__link">
    DicomBrowser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#diffutils" class="md-nav__link">
    Diffutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dijitso" class="md-nav__link">
    dijitso
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dill" class="md-nav__link">
    dill
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dirac" class="md-nav__link">
    DIRAC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#distributed" class="md-nav__link">
    distributed
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#distributedstream" class="md-nav__link">
    DistributedStream
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dl_poly_4" class="md-nav__link">
    DL_POLY_4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dl_poly_classic" class="md-nav__link">
    DL_POLY_Classic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dlb" class="md-nav__link">
    dlb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dlib" class="md-nav__link">
    dlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dlpack" class="md-nav__link">
    DLPack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dm-reverb" class="md-nav__link">
    dm-reverb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dm-tree" class="md-nav__link">
    dm-tree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dmcfun" class="md-nav__link">
    DMCfun
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dmlc-core" class="md-nav__link">
    DMLC-Core
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dmtcp" class="md-nav__link">
    DMTCP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dolfin" class="md-nav__link">
    DOLFIN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dorado" class="md-nav__link">
    dorado
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#doris" class="md-nav__link">
    Doris
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dosageconvertor" class="md-nav__link">
    DosageConvertor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dotnet-core-runtime" class="md-nav__link">
    dotNET-Core-Runtime
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dotnet-sdk" class="md-nav__link">
    dotNET-SDK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#double-conversion" class="md-nav__link">
    double-conversion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#doubletfinder" class="md-nav__link">
    DoubletFinder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#doxygen" class="md-nav__link">
    Doxygen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dragmap" class="md-nav__link">
    DRAGMAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#drake" class="md-nav__link">
    Drake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#drep" class="md-nav__link">
    dRep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#drop" class="md-nav__link">
    DROP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dropest" class="md-nav__link">
    dropEst
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dsa" class="md-nav__link">
    DSA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dsfmt" class="md-nav__link">
    dSFMT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dsrc" class="md-nav__link">
    DSRC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dsuite" class="md-nav__link">
    Dsuite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dtcmp" class="md-nav__link">
    dtcmp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dtcwt" class="md-nav__link">
    dtcwt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dualsphysics" class="md-nav__link">
    DualSPHysics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dubstepr" class="md-nav__link">
    DUBStepR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dune-core" class="md-nav__link">
    dune-core
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dune-fem" class="md-nav__link">
    dune-fem
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#duplex-tools" class="md-nav__link">
    duplex-tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dxpy" class="md-nav__link">
    dxpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dymat" class="md-nav__link">
    DyMat
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#e" class="md-nav__link">
    E
  </a>
  
    <nav class="md-nav" aria-label="E">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#e-antic" class="md-nav__link">
    E-ANTIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ea-utils" class="md-nav__link">
    ea-utils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#earthengine-api" class="md-nav__link">
    earthengine-api
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#easybuild" class="md-nav__link">
    EasyBuild
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#easyqc" class="md-nav__link">
    EasyQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ecbuild" class="md-nav__link">
    ecBuild
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eccodes" class="md-nav__link">
    ecCodes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eccodes-python" class="md-nav__link">
    eccodes-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ed2" class="md-nav__link">
    ED2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#edlib" class="md-nav__link">
    edlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#egglib" class="md-nav__link">
    EggLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eggnog-mapper" class="md-nav__link">
    eggnog-mapper
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#egttools" class="md-nav__link">
    EGTtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eht-imaging" class="md-nav__link">
    eht-imaging
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eigen" class="md-nav__link">
    Eigen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eigenexa" class="md-nav__link">
    EigenExa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eigensoft" class="md-nav__link">
    EIGENSOFT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#einops" class="md-nav__link">
    einops
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elastix" class="md-nav__link">
    elastix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elbencho" class="md-nav__link">
    elbencho
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elfio" class="md-nav__link">
    ELFIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elfutils" class="md-nav__link">
    elfutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elk" class="md-nav__link">
    Elk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elpa" class="md-nav__link">
    ELPA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elph" class="md-nav__link">
    ELPH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elprep" class="md-nav__link">
    elprep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elsi" class="md-nav__link">
    ELSI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elsi-rci" class="md-nav__link">
    ELSI-RCI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emacs" class="md-nav__link">
    Emacs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eman2" class="md-nav__link">
    EMAN2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emboss" class="md-nav__link">
    EMBOSS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#embree" class="md-nav__link">
    Embree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emcee" class="md-nav__link">
    emcee
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emu" class="md-nav__link">
    EMU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enabrowsertool" class="md-nav__link">
    enaBrowserTool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enchant" class="md-nav__link">
    enchant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enchant-2" class="md-nav__link">
    enchant-2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ensemblcoreapi" class="md-nav__link">
    EnsEMBLCoreAPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#entrypoints" class="md-nav__link">
    entrypoints
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#epd" class="md-nav__link">
    EPD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#epic" class="md-nav__link">
    EPIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#episcanpy" class="md-nav__link">
    epiScanpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#equilibrator" class="md-nav__link">
    eQuilibrator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ericscript" class="md-nav__link">
    EricScript
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#esl-bundle" class="md-nav__link">
    ESL-Bundle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#esm-2" class="md-nav__link">
    ESM-2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#esmf" class="md-nav__link">
    ESMF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#esmpy" class="md-nav__link">
    ESMPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#esmvaltool" class="md-nav__link">
    ESMValTool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#espeak-ng" class="md-nav__link">
    eSpeak-NG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#essentia" class="md-nav__link">
    Essentia
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ete" class="md-nav__link">
    ETE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#etsf_io" class="md-nav__link">
    ETSF_IO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eudev" class="md-nav__link">
    eudev
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#evidentialgene" class="md-nav__link">
    EvidentialGene
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#evmix" class="md-nav__link">
    evmix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exabayes" class="md-nav__link">
    ExaBayes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#examl" class="md-nav__link">
    ExaML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#excel-writer-xlsx" class="md-nav__link">
    Excel-Writer-XLSX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exiftool" class="md-nav__link">
    ExifTool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exiv2" class="md-nav__link">
    exiv2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exonerate" class="md-nav__link">
    Exonerate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#expat" class="md-nav__link">
    expat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#expect" class="md-nav__link">
    expect
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#expecttest" class="md-nav__link">
    expecttest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#express" class="md-nav__link">
    eXpress
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#extrae" class="md-nav__link">
    Extrae
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#f" class="md-nav__link">
    F
  </a>
  
    <nav class="md-nav" aria-label="F">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#f90cache" class="md-nav__link">
    f90cache
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#f90wrap" class="md-nav__link">
    f90wrap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#faber" class="md-nav__link">
    Faber
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fabio" class="md-nav__link">
    FabIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#faceswap" class="md-nav__link">
    faceswap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#falcon" class="md-nav__link">
    FALCON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fann" class="md-nav__link">
    FANN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fast5" class="md-nav__link">
    fast5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fasta" class="md-nav__link">
    FASTA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastahack" class="md-nav__link">
    fastahack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastai" class="md-nav__link">
    fastai
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastaindex" class="md-nav__link">
    FastaIndex
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastani" class="md-nav__link">
    FastANI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastaq" class="md-nav__link">
    Fastaq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastfold" class="md-nav__link">
    FastFold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastjet" class="md-nav__link">
    fastjet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastjet-contrib" class="md-nav__link">
    fastjet-contrib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastme" class="md-nav__link">
    FastME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastp" class="md-nav__link">
    fastp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastparquet" class="md-nav__link">
    fastparquet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastphase" class="md-nav__link">
    fastPHASE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastq-pair" class="md-nav__link">
    fastq-pair
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastq-tools" class="md-nav__link">
    fastq-tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastq_screen" class="md-nav__link">
    FastQ_Screen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastqc" class="md-nav__link">
    FastQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastqsplitter" class="md-nav__link">
    fastqsplitter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastqtl" class="md-nav__link">
    FastQTL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastqz" class="md-nav__link">
    fastqz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastrfs" class="md-nav__link">
    FastRFS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#faststructure" class="md-nav__link">
    fastStructure
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fasttree" class="md-nav__link">
    FastTree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastviromeexplorer" class="md-nav__link">
    FastViromeExplorer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastx-toolkit" class="md-nav__link">
    FASTX-Toolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fatslim" class="md-nav__link">
    fatslim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fbm" class="md-nav__link">
    fbm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fbpic" class="md-nav__link">
    FBPIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fcc" class="md-nav__link">
    FCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fcm" class="md-nav__link">
    FCM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fdict" class="md-nav__link">
    fdict
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fds" class="md-nav__link">
    FDS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fdstools" class="md-nav__link">
    fdstools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fdtd_solutions" class="md-nav__link">
    FDTD_Solutions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feh" class="md-nav__link">
    feh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fenics" class="md-nav__link">
    FEniCS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fermi-lite" class="md-nav__link">
    fermi-lite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ferret" class="md-nav__link">
    Ferret
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#festival" class="md-nav__link">
    festival
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fetchmg" class="md-nav__link">
    fetchMG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ffc" class="md-nav__link">
    FFC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fflas-ffpack" class="md-nav__link">
    FFLAS-FFPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ffmpeg" class="md-nav__link">
    FFmpeg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ffmpi" class="md-nav__link">
    ffmpi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ffnet" class="md-nav__link">
    ffnet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ffnvcodec" class="md-nav__link">
    ffnvcodec
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fftlib" class="md-nav__link">
    fftlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fftw" class="md-nav__link">
    FFTW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fftwmpi" class="md-nav__link">
    FFTW.MPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fgbio" class="md-nav__link">
    fgbio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fgsl" class="md-nav__link">
    FGSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fhi-aims" class="md-nav__link">
    FHI-aims
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fiat" class="md-nav__link">
    FIAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#figaro" class="md-nav__link">
    FIGARO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#figuregen" class="md-nav__link">
    FigureGen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fiji" class="md-nav__link">
    Fiji
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#file" class="md-nav__link">
    file
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#filevercmp" class="md-nav__link">
    filevercmp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#filtlong" class="md-nav__link">
    Filtlong
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#find_circ" class="md-nav__link">
    find_circ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finder" class="md-nav__link">
    finder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#findhap" class="md-nav__link">
    findhap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#findutils" class="md-nav__link">
    findutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fineradstructure" class="md-nav__link">
    fineRADstructure
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finestructure" class="md-nav__link">
    fineSTRUCTURE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fio" class="md-nav__link">
    fio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fiona" class="md-nav__link">
    Fiona
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#firefox" class="md-nav__link">
    Firefox
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#firestarter" class="md-nav__link">
    FIRESTARTER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fireworks" class="md-nav__link">
    FireWorks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fit-sne" class="md-nav__link">
    FIt-SNE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fix" class="md-nav__link">
    FIX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fixesproto" class="md-nav__link">
    fixesproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flac" class="md-nav__link">
    FLAC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flair" class="md-nav__link">
    FLAIR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flair-nlp" class="md-nav__link">
    flair-NLP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flann" class="md-nav__link">
    FLANN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flash" class="md-nav__link">
    FLASH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flask" class="md-nav__link">
    Flask
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flatbuffers" class="md-nav__link">
    flatbuffers
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flatbuffers-python" class="md-nav__link">
    flatbuffers-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fleur" class="md-nav__link">
    FLEUR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flex" class="md-nav__link">
    flex
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flexbar" class="md-nav__link">
    Flexbar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flexiblas" class="md-nav__link">
    FlexiBLAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flexidot" class="md-nav__link">
    FlexiDot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flink" class="md-nav__link">
    Flink
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flint" class="md-nav__link">
    FLINT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flook" class="md-nav__link">
    flook
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flowfda" class="md-nav__link">
    flowFDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fltk" class="md-nav__link">
    FLTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fluent" class="md-nav__link">
    FLUENT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#flye" class="md-nav__link">
    Flye
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fmilibrary" class="md-nav__link">
    FMILibrary
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fmm3d" class="md-nav__link">
    FMM3D
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fmpy" class="md-nav__link">
    FMPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fmriprep" class="md-nav__link">
    FMRIprep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fms" class="md-nav__link">
    FMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fmt" class="md-nav__link">
    fmt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fobis" class="md-nav__link">
    FoBiS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#foldx" class="md-nav__link">
    FoldX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fontconfig" class="md-nav__link">
    fontconfig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fontsproto" class="md-nav__link">
    fontsproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ford" class="md-nav__link">
    FORD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#foss" class="md-nav__link">
    foss
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fosscuda" class="md-nav__link">
    fosscuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fox" class="md-nav__link">
    FoX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fox-toolkit" class="md-nav__link">
    FOX-Toolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fpm" class="md-nav__link">
    FPM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fpocket" class="md-nav__link">
    fpocket
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fqtrim" class="md-nav__link">
    fqtrim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fqzcomp" class="md-nav__link">
    fqzcomp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fraggenescan" class="md-nav__link">
    FragGeneScan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#franz" class="md-nav__link">
    FRANz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freebarcodes" class="md-nav__link">
    FreeBarcodes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freebayes" class="md-nav__link">
    freebayes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freefem" class="md-nav__link">
    FreeFEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freefem_1" class="md-nav__link">
    FreeFem++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freeglut" class="md-nav__link">
    freeglut
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freeimage" class="md-nav__link">
    FreeImage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freesasa" class="md-nav__link">
    FreeSASA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freesurfer" class="md-nav__link">
    FreeSurfer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freetds" class="md-nav__link">
    FreeTDS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freetype" class="md-nav__link">
    freetype
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freetype-py" class="md-nav__link">
    freetype-py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freexl" class="md-nav__link">
    FreeXL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#freud-analysis" class="md-nav__link">
    freud-analysis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fribidi" class="md-nav__link">
    FriBidi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fruit" class="md-nav__link">
    FRUIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fruit_processor" class="md-nav__link">
    FRUIT_processor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fsl" class="md-nav__link">
    FSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fsleyes" class="md-nav__link">
    FSLeyes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fsom" class="md-nav__link">
    fsom
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fson" class="md-nav__link">
    FSON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ftgl" class="md-nav__link">
    FTGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fujitsu" class="md-nav__link">
    Fujitsu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fullrmc" class="md-nav__link">
    fullrmc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fumi_tools" class="md-nav__link">
    fumi_tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#funwave-tvd" class="md-nav__link">
    FUNWAVE-TVD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fuse" class="md-nav__link">
    FUSE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fuseq" class="md-nav__link">
    FuSeq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fusioncatcher" class="md-nav__link">
    FusionCatcher
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#futhark" class="md-nav__link">
    futhark
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#futile" class="md-nav__link">
    futile
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#future" class="md-nav__link">
    future
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fxtract" class="md-nav__link">
    fxtract
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#g" class="md-nav__link">
    G
  </a>
  
    <nav class="md-nav" aria-label="G">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#g-phocs" class="md-nav__link">
    G-PhoCS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#g2clib" class="md-nav__link">
    g2clib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#g2lib" class="md-nav__link">
    g2lib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#g2log" class="md-nav__link">
    g2log
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gaia" class="md-nav__link">
    Gaia
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gamess-us" class="md-nav__link">
    GAMESS-US
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gap" class="md-nav__link">
    gap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gapcloser" class="md-nav__link">
    GapCloser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gapfiller" class="md-nav__link">
    GapFiller
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gappa" class="md-nav__link">
    gappa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#garli" class="md-nav__link">
    GARLI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#garnett" class="md-nav__link">
    garnett
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gat" class="md-nav__link">
    GAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gate" class="md-nav__link">
    GATE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gatk" class="md-nav__link">
    GATK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gaussian" class="md-nav__link">
    Gaussian
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gawk" class="md-nav__link">
    gawk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gbasis" class="md-nav__link">
    gbasis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gblocks" class="md-nav__link">
    Gblocks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gbprocess" class="md-nav__link">
    GBprocesS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gbs2ploidy" class="md-nav__link">
    gbs2ploidy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gc" class="md-nav__link">
    gc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gc3pie" class="md-nav__link">
    GC3Pie
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcc" class="md-nav__link">
    GCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcccore" class="md-nav__link">
    GCCcore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcccuda" class="md-nav__link">
    gcccuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcloud" class="md-nav__link">
    gcloud
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gconf" class="md-nav__link">
    GConf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta" class="md-nav__link">
    GCTA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gctf" class="md-nav__link">
    Gctf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gd" class="md-nav__link">
    GD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdal" class="md-nav__link">
    GDAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdb" class="md-nav__link">
    GDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdbgui" class="md-nav__link">
    gdbgui
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdbm" class="md-nav__link">
    gdbm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdc-client" class="md-nav__link">
    gdc-client
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdchart" class="md-nav__link">
    GDCHART
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdcm" class="md-nav__link">
    GDCM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdgraph" class="md-nav__link">
    GDGraph
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdist" class="md-nav__link">
    gdist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdk-pixbuf" class="md-nav__link">
    Gdk-Pixbuf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gdrcopy" class="md-nav__link">
    GDRCopy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geant4" class="md-nav__link">
    Geant4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geant4-data" class="md-nav__link">
    Geant4-data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gearshifft" class="md-nav__link">
    gearshifft
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gegl" class="md-nav__link">
    GEGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gem-library" class="md-nav__link">
    GEM-library
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gemma" class="md-nav__link">
    GEMMA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gemmi" class="md-nav__link">
    gemmi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gencore_variant_detection" class="md-nav__link">
    gencore_variant_detection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genemark-et" class="md-nav__link">
    GeneMark-ET
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gengetopt" class="md-nav__link">
    gengetopt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genome_profiler" class="md-nav__link">
    Genome_Profiler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomemapper" class="md-nav__link">
    GenomeMapper
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomepy" class="md-nav__link">
    genomepy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genometester4" class="md-nav__link">
    GenomeTester4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomethreader" class="md-nav__link">
    GenomeThreader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genometools" class="md-nav__link">
    GenomeTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomeworks" class="md-nav__link">
    GenomeWorks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genotypeharmonizer" class="md-nav__link">
    GenotypeHarmonizer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genozip" class="md-nav__link">
    genozip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gensim" class="md-nav__link">
    gensim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geocube" class="md-nav__link">
    geocube
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geopandas" class="md-nav__link">
    geopandas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geopy" class="md-nav__link">
    geopy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#georges" class="md-nav__link">
    georges
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#geos" class="md-nav__link">
    GEOS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gerris" class="md-nav__link">
    Gerris
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#getorb" class="md-nav__link">
    GETORB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#getorganelle" class="md-nav__link">
    GetOrganelle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gettext" class="md-nav__link">
    gettext
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gexiv2" class="md-nav__link">
    gexiv2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gfbf" class="md-nav__link">
    gfbf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gffcompare" class="md-nav__link">
    GffCompare
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gffread" class="md-nav__link">
    gffread
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gflags" class="md-nav__link">
    gflags
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gfold" class="md-nav__link">
    GFOLD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gh" class="md-nav__link">
    gh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ghc" class="md-nav__link">
    GHC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ghostscript" class="md-nav__link">
    Ghostscript
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gibbs2" class="md-nav__link">
    Gibbs2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giflib" class="md-nav__link">
    giflib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gifsicle" class="md-nav__link">
    gifsicle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimic" class="md-nav__link">
    GIMIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimkl" class="md-nav__link">
    gimkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimp" class="md-nav__link">
    GIMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimpi" class="md-nav__link">
    gimpi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimpic" class="md-nav__link">
    gimpic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gimps" class="md-nav__link">
    GIMPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giolf" class="md-nav__link">
    giolf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giolfc" class="md-nav__link">
    giolfc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giotto-suite" class="md-nav__link">
    Giotto-Suite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#git" class="md-nav__link">
    git
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#git-extras" class="md-nav__link">
    git-extras
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#git-lfs" class="md-nav__link">
    git-lfs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gitpython" class="md-nav__link">
    GitPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#givaro" class="md-nav__link">
    Givaro
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giza" class="md-nav__link">
    Giza
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gkeyll" class="md-nav__link">
    GKeyll
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gl2ps" class="md-nav__link">
    GL2PS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glade" class="md-nav__link">
    Glade
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glew" class="md-nav__link">
    glew
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glfw" class="md-nav__link">
    GLFW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glib" class="md-nav__link">
    GLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glib-networking" class="md-nav__link">
    glib-networking
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glibc" class="md-nav__link">
    glibc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glibmm" class="md-nav__link">
    GLibmm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimmer" class="md-nav__link">
    GLIMMER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimmerhmm" class="md-nav__link">
    GlimmerHMM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glm" class="md-nav__link">
    GLM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glm-aed" class="md-nav__link">
    GLM-AED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#globalarrays" class="md-nav__link">
    GlobalArrays
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#globus-cli" class="md-nav__link">
    Globus-CLI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#globusconnectpersonal" class="md-nav__link">
    GlobusConnectPersonal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glog" class="md-nav__link">
    glog
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glpk" class="md-nav__link">
    GLPK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glproto" class="md-nav__link">
    glproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glucose" class="md-nav__link">
    Glucose
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmap-gsnap" class="md-nav__link">
    GMAP-GSNAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmp" class="md-nav__link">
    GMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmpich" class="md-nav__link">
    gmpich
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmpolf" class="md-nav__link">
    gmpolf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmpy2" class="md-nav__link">
    gmpy2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmsh" class="md-nav__link">
    gmsh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmt" class="md-nav__link">
    GMT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmvapich2" class="md-nav__link">
    gmvapich2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmvolf" class="md-nav__link">
    gmvolf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gnu" class="md-nav__link">
    GNU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gnuplot" class="md-nav__link">
    gnuplot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gnutls" class="md-nav__link">
    GnuTLS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#go" class="md-nav__link">
    Go
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#goalign" class="md-nav__link">
    goalign
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#goatools" class="md-nav__link">
    GOATOOLS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gobff" class="md-nav__link">
    gobff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gobject-introspection" class="md-nav__link">
    GObject-Introspection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#goblf" class="md-nav__link">
    goblf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#godon" class="md-nav__link">
    Godon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gofasta" class="md-nav__link">
    gofasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#golf" class="md-nav__link">
    golf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gomkl" class="md-nav__link">
    gomkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gompi" class="md-nav__link">
    gompi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gompic" class="md-nav__link">
    gompic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#google-java-format" class="md-nav__link">
    google-java-format
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#googletest" class="md-nav__link">
    googletest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gotree" class="md-nav__link">
    gotree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gp2c" class="md-nav__link">
    GP2C
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpaw" class="md-nav__link">
    GPAW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpaw-setups" class="md-nav__link">
    GPAW-setups
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gperf" class="md-nav__link">
    gperf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gperftools" class="md-nav__link">
    gperftools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpustat" class="md-nav__link">
    gpustat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpy" class="md-nav__link">
    GPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpyopt" class="md-nav__link">
    GPyOpt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gpytorch" class="md-nav__link">
    GPyTorch
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grace" class="md-nav__link">
    Grace
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gradle" class="md-nav__link">
    Gradle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gradunwarp" class="md-nav__link">
    gradunwarp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graph-tool" class="md-nav__link">
    graph-tool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphdb" class="md-nav__link">
    GraphDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphene" class="md-nav__link">
    Graphene
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphicsmagick" class="md-nav__link">
    GraphicsMagick
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphite2" class="md-nav__link">
    graphite2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphlan" class="md-nav__link">
    GraPhlAn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphmap" class="md-nav__link">
    GraphMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphmap2" class="md-nav__link">
    GraphMap2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphviz" class="md-nav__link">
    Graphviz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#graphviz-python" class="md-nav__link">
    graphviz-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grasp" class="md-nav__link">
    GRASP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grass" class="md-nav__link">
    GRASS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#greenlet" class="md-nav__link">
    Greenlet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grep" class="md-nav__link">
    Grep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gretl" class="md-nav__link">
    gretl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grib_api" class="md-nav__link">
    grib_api
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grid" class="md-nav__link">
    grid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grit" class="md-nav__link">
    GRIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grnboost" class="md-nav__link">
    GRNBoost
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#groff" class="md-nav__link">
    groff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#groimp" class="md-nav__link">
    GroIMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gromacs" class="md-nav__link">
    GROMACS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gromacswrapper" class="md-nav__link">
    GromacsWrapper
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#groovy" class="md-nav__link">
    Groovy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grpc" class="md-nav__link">
    gRPC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsea" class="md-nav__link">
    GSEA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsettings-desktop-schemas" class="md-nav__link">
    gsettings-desktop-schemas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsl" class="md-nav__link">
    GSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsoap" class="md-nav__link">
    gSOAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsport" class="md-nav__link">
    gsport
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gst-plugins-bad" class="md-nav__link">
    GST-plugins-bad
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gst-plugins-base" class="md-nav__link">
    GST-plugins-base
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gstreamer" class="md-nav__link">
    GStreamer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsutil" class="md-nav__link">
    gsutil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gsw" class="md-nav__link">
    gsw
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtdb-tk" class="md-nav__link">
    GTDB-Tk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtk" class="md-nav__link">
    GTK+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtk2" class="md-nav__link">
    GTK2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtk3" class="md-nav__link">
    GTK3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtk4" class="md-nav__link">
    GTK4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtksourceview" class="md-nav__link">
    GtkSourceView
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gtool" class="md-nav__link">
    GTOOL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gts" class="md-nav__link">
    GTS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gubbins" class="md-nav__link">
    gubbins
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#guenomu" class="md-nav__link">
    guenomu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#guile" class="md-nav__link">
    Guile
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gulp" class="md-nav__link">
    GULP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gurobi" class="md-nav__link">
    Gurobi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gushr" class="md-nav__link">
    GUSHR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gzip" class="md-nav__link">
    gzip
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#h" class="md-nav__link">
    H
  </a>
  
    <nav class="md-nav" aria-label="H">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#h4toh5" class="md-nav__link">
    h4toh5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#h5hut" class="md-nav__link">
    H5hut
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#h5py" class="md-nav__link">
    h5py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hadoop" class="md-nav__link">
    Hadoop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hal" class="md-nav__link">
    HAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hanythingondemand" class="md-nav__link">
    hanythingondemand
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hapgen2" class="md-nav__link">
    HAPGEN2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#harfbuzz" class="md-nav__link">
    HarfBuzz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#harminv" class="md-nav__link">
    Harminv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#harmony" class="md-nav__link">
    harmony
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hbase" class="md-nav__link">
    HBase
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdbscan" class="md-nav__link">
    HDBSCAN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hddm" class="md-nav__link">
    HDDM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdf" class="md-nav__link">
    HDF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdf-eos" class="md-nav__link">
    HDF-EOS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdf-eos5" class="md-nav__link">
    HDF-EOS5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdf5" class="md-nav__link">
    HDF5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdf5storage" class="md-nav__link">
    hdf5storage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdfview" class="md-nav__link">
    HDFView
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#healpix" class="md-nav__link">
    HEALPix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#health-gps" class="md-nav__link">
    Health-GPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heaptrack" class="md-nav__link">
    heaptrack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hector" class="md-nav__link">
    hector
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heffte" class="md-nav__link">
    HeFFTe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hello" class="md-nav__link">
    Hello
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#help2man" class="md-nav__link">
    help2man
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hepmc3" class="md-nav__link">
    HepMC3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hh-suite" class="md-nav__link">
    HH-suite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hic-pro" class="md-nav__link">
    HiC-Pro
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hicexplorer" class="md-nav__link">
    HiCExplorer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hicmatrix" class="md-nav__link">
    HiCMatrix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hierfstat" class="md-nav__link">
    hierfstat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hifiasm" class="md-nav__link">
    hifiasm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#highfive" class="md-nav__link">
    HighFive
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#highway" class="md-nav__link">
    Highway
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hip" class="md-nav__link">
    HIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipify-clang" class="md-nav__link">
    hipify-clang
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hips" class="md-nav__link">
    HIPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipsycl" class="md-nav__link">
    hipSYCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hiredis" class="md-nav__link">
    hiredis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hisat2" class="md-nav__link">
    HISAT2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#histolab" class="md-nav__link">
    histolab
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hivtrace" class="md-nav__link">
    hivtrace
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hl7apy" class="md-nav__link">
    hl7apy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hlaminer" class="md-nav__link">
    HLAminer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hmmer" class="md-nav__link">
    HMMER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hmmer2" class="md-nav__link">
    HMMER2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hmmlearn" class="md-nav__link">
    hmmlearn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#home" class="md-nav__link">
    HOME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#horovod" class="md-nav__link">
    Horovod
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#horton" class="md-nav__link">
    horton
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how_are_we_stranded_here" class="md-nav__link">
    how_are_we_stranded_here
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hpcg" class="md-nav__link">
    HPCG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hpcx" class="md-nav__link">
    HPCX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hpdbscan" class="md-nav__link">
    HPDBSCAN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hpl" class="md-nav__link">
    HPL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#htop" class="md-nav__link">
    htop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#htseq" class="md-nav__link">
    HTSeq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#htslib" class="md-nav__link">
    HTSlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hub" class="md-nav__link">
    hub
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#humann" class="md-nav__link">
    humann
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hunspell" class="md-nav__link">
    hunspell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hwloc" class="md-nav__link">
    hwloc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hydra" class="md-nav__link">
    Hydra
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hyperopt" class="md-nav__link">
    Hyperopt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hyperqueue" class="md-nav__link">
    HyperQueue
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hyperspy" class="md-nav__link">
    hyperspy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hyphy" class="md-nav__link">
    HyPhy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hypo" class="md-nav__link">
    HyPo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hypothesis" class="md-nav__link">
    hypothesis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hypre" class="md-nav__link">
    Hypre
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#i" class="md-nav__link">
    I
  </a>
  
    <nav class="md-nav" aria-label="I">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#i-cistarget" class="md-nav__link">
    i-cisTarget
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#i-pi" class="md-nav__link">
    i-PI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#i-tasser" class="md-nav__link">
    I-TASSER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#i7z" class="md-nav__link">
    i7z
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ica-aroma" class="md-nav__link">
    ICA-AROMA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#icc" class="md-nav__link">
    icc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iccifort" class="md-nav__link">
    iccifort
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iccifortcuda" class="md-nav__link">
    iccifortcuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ichorcna" class="md-nav__link">
    ichorCNA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#icmake" class="md-nav__link">
    icmake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#icount" class="md-nav__link">
    iCount
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#icu" class="md-nav__link">
    ICU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#idba-ud" class="md-nav__link">
    IDBA-UD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#idemux" class="md-nav__link">
    idemux
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ieeg-cli" class="md-nav__link">
    ieeg-cli
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ifort" class="md-nav__link">
    ifort
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igblast" class="md-nav__link">
    IgBLAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igmplot" class="md-nav__link">
    IGMPlot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igraph" class="md-nav__link">
    igraph
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igv" class="md-nav__link">
    IGV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igv-reports" class="md-nav__link">
    igv-reports
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#igvtools" class="md-nav__link">
    IGVTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iibff" class="md-nav__link">
    iibff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iimkl" class="md-nav__link">
    iimkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iimpi" class="md-nav__link">
    iimpi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iimpic" class="md-nav__link">
    iimpic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ijulia" class="md-nav__link">
    IJulia
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ilamb" class="md-nav__link">
    ILAMB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ima2" class="md-nav__link">
    IMa2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ima2p" class="md-nav__link">
    IMa2p
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imagecodecs" class="md-nav__link">
    imagecodecs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imageio" class="md-nav__link">
    imageio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imagej" class="md-nav__link">
    ImageJ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imagemagick" class="md-nav__link">
    ImageMagick
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imake" class="md-nav__link">
    imake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imath" class="md-nav__link">
    Imath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imb" class="md-nav__link">
    IMB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imbalanced-learn" class="md-nav__link">
    imbalanced-learn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imgaug" class="md-nav__link">
    imgaug
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imkl" class="md-nav__link">
    imkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imkl-fftw" class="md-nav__link">
    imkl-FFTW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imlib2" class="md-nav__link">
    Imlib2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#immunedeconv" class="md-nav__link">
    immunedeconv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#imod" class="md-nav__link">
    IMOD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impi" class="md-nav__link">
    impi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute2" class="md-nav__link">
    IMPUTE2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inchi" class="md-nav__link">
    InChI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#indicators" class="md-nav__link">
    indicators
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inelastica" class="md-nav__link">
    Inelastica
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#infercnv" class="md-nav__link">
    inferCNV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#infercnvpy" class="md-nav__link">
    infercnvpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inferelator" class="md-nav__link">
    Inferelator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#infernal" class="md-nav__link">
    Infernal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#infomap" class="md-nav__link">
    Infomap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inline" class="md-nav__link">
    inline
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inparanoid" class="md-nav__link">
    InParanoid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inputproto" class="md-nav__link">
    inputproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inspector" class="md-nav__link">
    Inspector
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intarna" class="md-nav__link">
    IntaRNA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#integrate" class="md-nav__link">
    INTEGRATE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#integrate-neo" class="md-nav__link">
    INTEGRATE-Neo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intel" class="md-nav__link">
    intel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intel-compilers" class="md-nav__link">
    intel-compilers
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intelclusterchecker" class="md-nav__link">
    IntelClusterChecker
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intelcuda" class="md-nav__link">
    intelcuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#inteldaal" class="md-nav__link">
    IntelDAAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intelpython" class="md-nav__link">
    IntelPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#interproscan" class="md-nav__link">
    InterProScan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#interproscan_data" class="md-nav__link">
    InterProScan_data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intervaltree" class="md-nav__link">
    intervaltree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intervaltree-python" class="md-nav__link">
    intervaltree-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#intltool" class="md-nav__link">
    intltool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#io_lib" class="md-nav__link">
    io_lib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ioapi" class="md-nav__link">
    ioapi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iodata" class="md-nav__link">
    iodata
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iomkl" class="md-nav__link">
    iomkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iompi" class="md-nav__link">
    iompi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ior" class="md-nav__link">
    IOR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iozone" class="md-nav__link">
    IOzone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipm" class="md-nav__link">
    IPM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipopt" class="md-nav__link">
    Ipopt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipp" class="md-nav__link">
    ipp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipy" class="md-nav__link">
    IPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipyparallel" class="md-nav__link">
    ipyparallel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipyrad" class="md-nav__link">
    ipyrad
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ipython" class="md-nav__link">
    IPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iq-tree" class="md-nav__link">
    IQ-TREE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iris" class="md-nav__link">
    Iris
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#irkernel" class="md-nav__link">
    IRkernel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#irodsfs" class="md-nav__link">
    irodsfs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ironpython" class="md-nav__link">
    IronPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#isa-l" class="md-nav__link">
    ISA-L
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#isl" class="md-nav__link">
    ISL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#isocirc" class="md-nav__link">
    isoCirc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#isonet" class="md-nav__link">
    IsoNet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ispc" class="md-nav__link">
    ispc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#itac" class="md-nav__link">
    itac
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#itk" class="md-nav__link">
    ITK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#itpp" class="md-nav__link">
    itpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#itstool" class="md-nav__link">
    ITSTool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#itsx" class="md-nav__link">
    ITSx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ivar" class="md-nav__link">
    iVar
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#j" class="md-nav__link">
    J
  </a>
  
    <nav class="md-nav" aria-label="J">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#jags" class="md-nav__link">
    JAGS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jansson" class="md-nav__link">
    Jansson
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jasmine" class="md-nav__link">
    Jasmine
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jasper" class="md-nav__link">
    JasPer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#java" class="md-nav__link">
    Java
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#javafx" class="md-nav__link">
    JavaFX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jax" class="md-nav__link">
    jax
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jaxfrontce" class="md-nav__link">
    JAXFrontCE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jbigkit" class="md-nav__link">
    jbigkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jellyfish" class="md-nav__link">
    Jellyfish
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jemalloc" class="md-nav__link">
    jemalloc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jhbuild" class="md-nav__link">
    jhbuild
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jitcode" class="md-nav__link">
    JiTCODE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jmodeltest" class="md-nav__link">
    jModelTest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jorg" class="md-nav__link">
    Jorg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#joypy" class="md-nav__link">
    joypy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jq" class="md-nav__link">
    jq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#json-c" class="md-nav__link">
    json-c
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#json-fortran" class="md-nav__link">
    json-fortran
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#json-glib" class="md-nav__link">
    JSON-GLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jsoncpp" class="md-nav__link">
    JsonCpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jube" class="md-nav__link">
    JUBE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#judy" class="md-nav__link">
    Judy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#julia" class="md-nav__link">
    Julia
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#junit" class="md-nav__link">
    JUnit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyter-contrib-nbextensions" class="md-nav__link">
    jupyter-contrib-nbextensions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyter-matlab-proxy" class="md-nav__link">
    jupyter-matlab-proxy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyter-resource-usage" class="md-nav__link">
    jupyter-resource-usage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyter-server" class="md-nav__link">
    jupyter-server
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyter-server-proxy" class="md-nav__link">
    jupyter-server-proxy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyterhub" class="md-nav__link">
    JupyterHub
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyterlab" class="md-nav__link">
    JupyterLab
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyterlab-lmod" class="md-nav__link">
    jupyterlab-lmod
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jupyterlmod" class="md-nav__link">
    jupyterlmod
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jwm" class="md-nav__link">
    JWM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jxrlib" class="md-nav__link">
    jxrlib
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#k" class="md-nav__link">
    K
  </a>
  
    <nav class="md-nav" aria-label="K">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#kahip" class="md-nav__link">
    KaHIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kaiju" class="md-nav__link">
    Kaiju
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kaleido" class="md-nav__link">
    Kaleido
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kalign" class="md-nav__link">
    Kalign
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kallisto" class="md-nav__link">
    kallisto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kat" class="md-nav__link">
    KAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kb-python" class="md-nav__link">
    kb-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kbproto" class="md-nav__link">
    kbproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kedro" class="md-nav__link">
    kedro
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kent_tools" class="md-nav__link">
    Kent_tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#keras" class="md-nav__link">
    Keras
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#khmer" class="md-nav__link">
    khmer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kim-api" class="md-nav__link">
    kim-api
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kineto" class="md-nav__link">
    kineto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#king" class="md-nav__link">
    king
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kite" class="md-nav__link">
    KITE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kma" class="md-nav__link">
    kma
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kmc" class="md-nav__link">
    KMC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kmcp" class="md-nav__link">
    KMCP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kmergenie" class="md-nav__link">
    KmerGenie
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#knime" class="md-nav__link">
    KNIME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kpcalg" class="md-nav__link">
    kpcalg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kraken" class="md-nav__link">
    Kraken
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kraken2" class="md-nav__link">
    Kraken2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kratos" class="md-nav__link">
    Kratos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#krbalancing" class="md-nav__link">
    krbalancing
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kronatools" class="md-nav__link">
    KronaTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kwant" class="md-nav__link">
    kwant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kwiml" class="md-nav__link">
    KWIML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kwip" class="md-nav__link">
    kWIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kyotocabinet" class="md-nav__link">
    KyotoCabinet
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#l" class="md-nav__link">
    L
  </a>
  
    <nav class="md-nav" aria-label="L">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#l_rna_scaffolder" class="md-nav__link">
    L_RNA_scaffolder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lace" class="md-nav__link">
    Lace
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ladr" class="md-nav__link">
    LADR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lagrangian-filtering" class="md-nav__link">
    lagrangian-filtering
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lame" class="md-nav__link">
    LAME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lammps" class="md-nav__link">
    LAMMPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lancet" class="md-nav__link">
    lancet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lapack" class="md-nav__link">
    LAPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lasso-python" class="md-nav__link">
    LASSO-Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#last" class="md-nav__link">
    LAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lastz" class="md-nav__link">
    LASTZ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lavaan" class="md-nav__link">
    lavaan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lbfgs" class="md-nav__link">
    LBFGS++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lcov" class="md-nav__link">
    LCov
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldc" class="md-nav__link">
    LDC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lddt" class="md-nav__link">
    lDDT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#leadit" class="md-nav__link">
    LeadIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#leidenalg" class="md-nav__link">
    leidenalg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lemon" class="md-nav__link">
    LEMON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#leptonica" class="md-nav__link">
    Leptonica
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lerc" class="md-nav__link">
    LERC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#less" class="md-nav__link">
    less
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#leveldb" class="md-nav__link">
    LevelDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lftp" class="md-nav__link">
    lftp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lhapdf" class="md-nav__link">
    LHAPDF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libaec" class="md-nav__link">
    libaec
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libaio" class="md-nav__link">
    libaio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libarchive" class="md-nav__link">
    libarchive
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libav" class="md-nav__link">
    libav
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libavif" class="md-nav__link">
    libavif
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libbigwig" class="md-nav__link">
    libBigWig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libbitmask" class="md-nav__link">
    libbitmask
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcdms" class="md-nav__link">
    libcdms
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcerf" class="md-nav__link">
    libcerf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcint" class="md-nav__link">
    libcint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcircle" class="md-nav__link">
    libcircle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcmaes" class="md-nav__link">
    libcmaes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libconfig" class="md-nav__link">
    libconfig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcpuset" class="md-nav__link">
    libcpuset
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libcroco" class="md-nav__link">
    libcroco
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libctl" class="md-nav__link">
    libctl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdap" class="md-nav__link">
    libdap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libde265" class="md-nav__link">
    libde265
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdeflate" class="md-nav__link">
    libdeflate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdivsufsort" class="md-nav__link">
    libdivsufsort
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdrm" class="md-nav__link">
    libdrm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdrs" class="md-nav__link">
    libdrs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libdwarf" class="md-nav__link">
    libdwarf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libedit" class="md-nav__link">
    libedit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libelf" class="md-nav__link">
    libelf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libemf" class="md-nav__link">
    libemf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libepoxy" class="md-nav__link">
    libepoxy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libev" class="md-nav__link">
    libev
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libevent" class="md-nav__link">
    libevent
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libexif" class="md-nav__link">
    libexif
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libfabric" class="md-nav__link">
    libfabric
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libfdf" class="md-nav__link">
    libfdf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libffcall" class="md-nav__link">
    libffcall
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libffi" class="md-nav__link">
    libffi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libflame" class="md-nav__link">
    libFLAME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libfontenc" class="md-nav__link">
    libfontenc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgcrypt" class="md-nav__link">
    libgcrypt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgd" class="md-nav__link">
    libgd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgdiplus" class="md-nav__link">
    libgdiplus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgdsii" class="md-nav__link">
    libGDSII
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgeotiff" class="md-nav__link">
    libgeotiff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgit2" class="md-nav__link">
    libgit2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libglade" class="md-nav__link">
    libglade
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libglu" class="md-nav__link">
    libGLU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libglvnd" class="md-nav__link">
    libglvnd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgpg-error" class="md-nav__link">
    libgpg-error
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgpuarray" class="md-nav__link">
    libgpuarray
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgridxc" class="md-nav__link">
    libGridXC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libgtextutils" class="md-nav__link">
    libgtextutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libharu" class="md-nav__link">
    libharu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libheif" class="md-nav__link">
    libheif
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libibmad" class="md-nav__link">
    libibmad
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libibumad" class="md-nav__link">
    libibumad
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libice" class="md-nav__link">
    libICE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libiconv" class="md-nav__link">
    libiconv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libidn" class="md-nav__link">
    libidn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libidn2" class="md-nav__link">
    libidn2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libint" class="md-nav__link">
    Libint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libis" class="md-nav__link">
    LiBis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libjpeg-turbo" class="md-nav__link">
    libjpeg-turbo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libjxl" class="md-nav__link">
    libjxl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liblzf" class="md-nav__link">
    LibLZF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmad" class="md-nav__link">
    libmad
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmatheval" class="md-nav__link">
    libmatheval
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmaus2" class="md-nav__link">
    libmaus2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmbd" class="md-nav__link">
    libmbd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmemcached" class="md-nav__link">
    libMemcached
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmicrohttpd" class="md-nav__link">
    libmicrohttpd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmo_unpack" class="md-nav__link">
    libmo_unpack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libmypaint" class="md-nav__link">
    libmypaint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libnsl" class="md-nav__link">
    libnsl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libobjcryst" class="md-nav__link">
    libobjcryst
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libogg" class="md-nav__link">
    libogg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libopus" class="md-nav__link">
    libopus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libosmium" class="md-nav__link">
    libosmium
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpci" class="md-nav__link">
    libpci
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpciaccess" class="md-nav__link">
    libpciaccess
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libplinkio" class="md-nav__link">
    libplinkio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpng" class="md-nav__link">
    libpng
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpsl" class="md-nav__link">
    libpsl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpsml" class="md-nav__link">
    libPSML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpsortb" class="md-nav__link">
    libpsortb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpspio" class="md-nav__link">
    libpspio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libpthread-stubs" class="md-nav__link">
    libpthread-stubs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libqglviewer" class="md-nav__link">
    libQGLViewer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libreadline" class="md-nav__link">
    libreadline
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#librmath" class="md-nav__link">
    libRmath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#librosa" class="md-nav__link">
    librosa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#librsb" class="md-nav__link">
    librsb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#librsvg" class="md-nav__link">
    librsvg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#librttopo" class="md-nav__link">
    librttopo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsamplerate" class="md-nav__link">
    libsamplerate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsbml" class="md-nav__link">
    libSBML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsigc" class="md-nav__link">
    libsigc++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsigsegv" class="md-nav__link">
    libsigsegv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsm" class="md-nav__link">
    libSM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsndfile" class="md-nav__link">
    libsndfile
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsodium" class="md-nav__link">
    libsodium
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsoup" class="md-nav__link">
    LibSoup
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libspatialindex" class="md-nav__link">
    libspatialindex
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libspatialite" class="md-nav__link">
    libspatialite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libssh" class="md-nav__link">
    libssh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libstatgen" class="md-nav__link">
    libStatGen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libsvm" class="md-nav__link">
    LIBSVM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtar" class="md-nav__link">
    libtar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtasn1" class="md-nav__link">
    libtasn1
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtecla" class="md-nav__link">
    libtecla
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtiff" class="md-nav__link">
    LibTIFF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtirpc" class="md-nav__link">
    libtirpc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtool" class="md-nav__link">
    libtool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libtree" class="md-nav__link">
    libtree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libunistring" class="md-nav__link">
    libunistring
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libunwind" class="md-nav__link">
    libunwind
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libutempter" class="md-nav__link">
    libutempter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libuuid" class="md-nav__link">
    LibUUID
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libuv" class="md-nav__link">
    libuv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libvdwxc" class="md-nav__link">
    libvdwxc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libvorbis" class="md-nav__link">
    libvorbis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libwebp" class="md-nav__link">
    libwebp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libwpe" class="md-nav__link">
    libwpe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libx11" class="md-nav__link">
    libX11
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxau" class="md-nav__link">
    libXau
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxc" class="md-nav__link">
    libxc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxcb" class="md-nav__link">
    libxcb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxcursor" class="md-nav__link">
    libXcursor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxdamage" class="md-nav__link">
    libXdamage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxdmcp" class="md-nav__link">
    libXdmcp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxext" class="md-nav__link">
    libXext
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxfixes" class="md-nav__link">
    libXfixes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxfont" class="md-nav__link">
    libXfont
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxft" class="md-nav__link">
    libXft
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxi" class="md-nav__link">
    libXi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxinerama" class="md-nav__link">
    libXinerama
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxkbcommon" class="md-nav__link">
    libxkbcommon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxml" class="md-nav__link">
    libxml++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxml2" class="md-nav__link">
    libxml2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxml2-python" class="md-nav__link">
    libxml2-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxmu" class="md-nav__link">
    libXmu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxp" class="md-nav__link">
    libXp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxpm" class="md-nav__link">
    libXpm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxrandr" class="md-nav__link">
    libXrandr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxrender" class="md-nav__link">
    libXrender
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxslt" class="md-nav__link">
    libxslt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxsmm" class="md-nav__link">
    libxsmm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxt" class="md-nav__link">
    libXt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libxxf86vm" class="md-nav__link">
    libXxf86vm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libyaml" class="md-nav__link">
    libyaml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libzeep" class="md-nav__link">
    libzeep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#libzip" class="md-nav__link">
    libzip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lie_learn" class="md-nav__link">
    lie_learn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lifelines" class="md-nav__link">
    lifelines
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lighter" class="md-nav__link">
    Lighter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liknorm" class="md-nav__link">
    liknorm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#likwid" class="md-nav__link">
    likwid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#limix" class="md-nav__link">
    limix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#linbox" class="md-nav__link">
    LinBox
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#line_profiler" class="md-nav__link">
    line_profiler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lingeling" class="md-nav__link">
    Lingeling
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#littlecms" class="md-nav__link">
    LittleCMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lldb" class="md-nav__link">
    LLDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#llvm" class="md-nav__link">
    LLVM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lmdb" class="md-nav__link">
    LMDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lmfit" class="md-nav__link">
    LMfit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lmod" class="md-nav__link">
    Lmod
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lncloom" class="md-nav__link">
    LncLOOM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#locarna" class="md-nav__link">
    LocARNA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lofreq" class="md-nav__link">
    LoFreq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#log-log4perl" class="md-nav__link">
    Log-Log4perl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#logaddexp" class="md-nav__link">
    logaddexp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lohhla" class="md-nav__link">
    LOHHLA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#loki" class="md-nav__link">
    Loki
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#longestrunsubsequence" class="md-nav__link">
    longestrunsubsequence
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#longread_umi" class="md-nav__link">
    longread_umi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#longshot" class="md-nav__link">
    Longshot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#loompy" class="md-nav__link">
    loompy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#loomr" class="md-nav__link">
    loomR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lpeg" class="md-nav__link">
    LPeg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lpjml" class="md-nav__link">
    LPJmL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lpsolve" class="md-nav__link">
    lpsolve
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lrslib" class="md-nav__link">
    lrslib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ls-prepost" class="md-nav__link">
    LS-PrePost
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lsd2" class="md-nav__link">
    LSD2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lsms" class="md-nav__link">
    LSMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ltr_retriever" class="md-nav__link">
    LTR_retriever
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ltrdetector" class="md-nav__link">
    LtrDetector
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lua" class="md-nav__link">
    Lua
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#luajit" class="md-nav__link">
    LuaJIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#luajit2-openresty" class="md-nav__link">
    LuaJIT2-OpenResty
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lucene-geo-gazetteer" class="md-nav__link">
    Lucene-Geo-Gazetteer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lumpy" class="md-nav__link">
    LUMPY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#luscus" class="md-nav__link">
    LUSCUS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lwgrp" class="md-nav__link">
    lwgrp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lxml" class="md-nav__link">
    lxml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lynx" class="md-nav__link">
    lynx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lz4" class="md-nav__link">
    lz4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lzo" class="md-nav__link">
    LZO
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#m" class="md-nav__link">
    M
  </a>
  
    <nav class="md-nav" aria-label="M">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#m1qn3" class="md-nav__link">
    M1QN3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#m4" class="md-nav__link">
    M4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach" class="md-nav__link">
    MACH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macs2" class="md-nav__link">
    MACS2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macse" class="md-nav__link">
    MACSE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maeparser" class="md-nav__link">
    maeparser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mafft" class="md-nav__link">
    MAFFT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mageck" class="md-nav__link">
    MAGeCK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#magick" class="md-nav__link">
    magick
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#magics" class="md-nav__link">
    Magics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#magma" class="md-nav__link">
    magma
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#magma-gene-analysis" class="md-nav__link">
    MAGMA-gene-analysis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#magrespython" class="md-nav__link">
    MagresPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mahotas" class="md-nav__link">
    mahotas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#majiq" class="md-nav__link">
    MAJIQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#make" class="md-nav__link">
    make
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#makedepend" class="md-nav__link">
    makedepend
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#makedepf90" class="md-nav__link">
    makedepf90
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#makeinfo" class="md-nav__link">
    makeinfo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maker" class="md-nav__link">
    MAKER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mako" class="md-nav__link">
    Mako
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mamba" class="md-nav__link">
    Mamba
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manta" class="md-nav__link">
    manta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mapdamage" class="md-nav__link">
    mapDamage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maple" class="md-nav__link">
    Maple
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mapsplice" class="md-nav__link">
    MapSplice
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maq" class="md-nav__link">
    Maq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mariadb" class="md-nav__link">
    MariaDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mariadb-connector-c" class="md-nav__link">
    MariaDB-connector-c
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mars" class="md-nav__link">
    MARS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mash" class="md-nav__link">
    Mash
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#masurca" class="md-nav__link">
    MaSuRCA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mathematica" class="md-nav__link">
    Mathematica
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mathgl" class="md-nav__link">
    MathGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matio" class="md-nav__link">
    MATIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matlab" class="md-nav__link">
    MATLAB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matlab-engine" class="md-nav__link">
    MATLAB-Engine
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matlab-proxy" class="md-nav__link">
    matlab-proxy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matplotlib" class="md-nav__link">
    matplotlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matplotlib-inline" class="md-nav__link">
    matplotlib-inline
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#matsim" class="md-nav__link">
    MATSim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maude" class="md-nav__link">
    Maude
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mauvealigner" class="md-nav__link">
    mauveAligner
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maven" class="md-nav__link">
    Maven
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mawk" class="md-nav__link">
    mawk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maxbin" class="md-nav__link">
    MaxBin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maxquant" class="md-nav__link">
    MaxQuant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mayavi" class="md-nav__link">
    mayavi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#maze" class="md-nav__link">
    maze
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mbedtls" class="md-nav__link">
    MbedTLS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mbrola" class="md-nav__link">
    MBROLA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mbuffer" class="md-nav__link">
    mbuffer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mc" class="md-nav__link">
    mc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mcl" class="md-nav__link">
    MCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mcr" class="md-nav__link">
    MCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mcu" class="md-nav__link">
    mcu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdanalysis" class="md-nav__link">
    MDAnalysis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdbm" class="md-nav__link">
    MDBM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdsplus" class="md-nav__link">
    MDSplus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdsplus-java" class="md-nav__link">
    MDSplus-Java
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdsplus-python" class="md-nav__link">
    MDSplus-Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdtest" class="md-nav__link">
    mdtest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mdtraj" class="md-nav__link">
    MDTraj
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#medaka" class="md-nav__link">
    medaka
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#medimgproc" class="md-nav__link">
    medImgProc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#medpy" class="md-nav__link">
    MedPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meep" class="md-nav__link">
    Meep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mega" class="md-nav__link">
    MEGA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megacc" class="md-nav__link">
    MEGACC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megahit" class="md-nav__link">
    MEGAHIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megalodon" class="md-nav__link">
    Megalodon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megan" class="md-nav__link">
    MEGAN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meld" class="md-nav__link">
    Meld
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mem" class="md-nav__link">
    MEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meme" class="md-nav__link">
    MEME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#memkind" class="md-nav__link">
    memkind
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#memory-profiler" class="md-nav__link">
    memory-profiler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#memote" class="md-nav__link">
    MEMOTE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#memtester" class="md-nav__link">
    memtester
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#merantk" class="md-nav__link">
    meRanTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#merckx" class="md-nav__link">
    MERCKX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mercurial" class="md-nav__link">
    Mercurial
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mesa" class="md-nav__link">
    Mesa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mesa-demos" class="md-nav__link">
    Mesa-demos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meshalyzer" class="md-nav__link">
    meshalyzer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meshio" class="md-nav__link">
    meshio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meshtool" class="md-nav__link">
    meshtool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meson" class="md-nav__link">
    Meson
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meson-python" class="md-nav__link">
    meson-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mesquite" class="md-nav__link">
    Mesquite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mess" class="md-nav__link">
    MESS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metabat" class="md-nav__link">
    MetaBAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaboanalystr" class="md-nav__link">
    MetaboAnalystR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaerg" class="md-nav__link">
    metaerg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaeuk" class="md-nav__link">
    MetaEuk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metageneannotator" class="md-nav__link">
    MetaGeneAnnotator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metagenome-atlas" class="md-nav__link">
    Metagenome-Atlas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    Metal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metamorpheus" class="md-nav__link">
    MetaMorpheus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaphlan" class="md-nav__link">
    MetaPhlAn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaphlan2" class="md-nav__link">
    MetaPhlAn2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metawrap" class="md-nav__link">
    metaWRAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metaxa2" class="md-nav__link">
    Metaxa2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#methylartist" class="md-nav__link">
    methylartist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#methyldackel" class="md-nav__link">
    MethylDackel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#methylpy" class="md-nav__link">
    methylpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metis" class="md-nav__link">
    METIS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgltools" class="md-nav__link">
    mgltools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mhcflurry" class="md-nav__link">
    mhcflurry
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mhcnuggets" class="md-nav__link">
    mhcnuggets
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#microbeannotator" class="md-nav__link">
    MicrobeAnnotator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#microctools" class="md-nav__link">
    microctools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#migec" class="md-nav__link">
    MiGEC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#migrate-n" class="md-nav__link">
    MIGRATE-N
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#miller" class="md-nav__link">
    Miller
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mimalloc" class="md-nav__link">
    mimalloc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minc" class="md-nav__link">
    MINC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minced" class="md-nav__link">
    MinCED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mini-xml" class="md-nav__link">
    Mini-XML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#miniasm" class="md-nav__link">
    miniasm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minibar" class="md-nav__link">
    minibar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minicard" class="md-nav__link">
    MiniCARD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#miniconda2" class="md-nav__link">
    Miniconda2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#miniconda3" class="md-nav__link">
    Miniconda3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minieigen" class="md-nav__link">
    minieigen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minimac4" class="md-nav__link">
    Minimac4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minimap2" class="md-nav__link">
    minimap2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minipolish" class="md-nav__link">
    Minipolish
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minisat" class="md-nav__link">
    MiniSat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minizip" class="md-nav__link">
    minizip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minpack" class="md-nav__link">
    MINPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#minpath" class="md-nav__link">
    MinPath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mira" class="md-nav__link">
    MIRA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mirdeep2" class="md-nav__link">
    miRDeep2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mish-cuda" class="md-nav__link">
    Mish-Cuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#misha" class="md-nav__link">
    misha
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mitgcmutils" class="md-nav__link">
    MITgcmutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mitobim" class="md-nav__link">
    MITObim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mitoz" class="md-nav__link">
    MitoZ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixcr" class="md-nav__link">
    MiXCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixmhc2pred" class="md-nav__link">
    MixMHC2pred
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mkl-dnn" class="md-nav__link">
    mkl-dnn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mkl-service" class="md-nav__link">
    mkl-service
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mkl_fft" class="md-nav__link">
    mkl_fft
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mlc" class="md-nav__link">
    MLC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mlxtend" class="md-nav__link">
    MLxtend
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mm-common" class="md-nav__link">
    mm-common
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mmg" class="md-nav__link">
    Mmg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mmseq" class="md-nav__link">
    MMSEQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mmseqs2" class="md-nav__link">
    MMseqs2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mmtf-cpp" class="md-nav__link">
    mmtf-cpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mne-python" class="md-nav__link">
    MNE-Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#moabs" class="md-nav__link">
    MOABS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#modeltest-ng" class="md-nav__link">
    ModelTest-NG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#modred" class="md-nav__link">
    modred
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molcas" class="md-nav__link">
    Molcas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mold" class="md-nav__link">
    mold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molden" class="md-nav__link">
    Molden
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molekel" class="md-nav__link">
    Molekel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molmod" class="md-nav__link">
    molmod
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molpro" class="md-nav__link">
    Molpro
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mona" class="md-nav__link">
    MONA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#monai" class="md-nav__link">
    MONAI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mongolite" class="md-nav__link">
    mongolite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mono" class="md-nav__link">
    Mono
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#monocle3" class="md-nav__link">
    Monocle3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#moonjit" class="md-nav__link">
    moonjit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#moose" class="md-nav__link">
    MOOSE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mordecai" class="md-nav__link">
    mordecai
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#moreronn" class="md-nav__link">
    MoreRONN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mosdepth" class="md-nav__link">
    mosdepth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mothur" class="md-nav__link">
    Mothur
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#motif" class="md-nav__link">
    motif
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#motioncor2" class="md-nav__link">
    MotionCor2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#motionsegmentation" class="md-nav__link">
    motionSegmentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#moviepy" class="md-nav__link">
    MoviePy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpb" class="md-nav__link">
    MPB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpc" class="md-nav__link">
    MPC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpfr" class="md-nav__link">
    MPFR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpi4py" class="md-nav__link">
    mpi4py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpich" class="md-nav__link">
    MPICH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpich2" class="md-nav__link">
    MPICH2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpifileutils" class="md-nav__link">
    mpifileutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpip" class="md-nav__link">
    mpiP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpj-express" class="md-nav__link">
    MPJ-Express
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mpmath" class="md-nav__link">
    mpmath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mrbayes" class="md-nav__link">
    MrBayes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mrcfile" class="md-nav__link">
    mrcfile
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mrchem" class="md-nav__link">
    MRChem
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mrcpp" class="md-nav__link">
    MRCPP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mricron" class="md-nav__link">
    MRIcron
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mrtrix" class="md-nav__link">
    MRtrix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#msgpack-c" class="md-nav__link">
    msgpack-c
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#msm" class="md-nav__link">
    MSM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mspc" class="md-nav__link">
    MSPC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#msprime" class="md-nav__link">
    msprime
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mstore" class="md-nav__link">
    mstore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mtl4" class="md-nav__link">
    MTL4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mujoco" class="md-nav__link">
    MuJoCo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mujoco-py" class="md-nav__link">
    mujoco-py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multichoose" class="md-nav__link">
    multichoose
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multilevelestimators" class="md-nav__link">
    MultilevelEstimators
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multinest" class="md-nav__link">
    MultiNest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiqc" class="md-nav__link">
    MultiQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiwfn" class="md-nav__link">
    Multiwfn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mumerge" class="md-nav__link">
    muMerge
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mummer" class="md-nav__link">
    MUMmer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mumps" class="md-nav__link">
    MUMPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#muparser" class="md-nav__link">
    muParser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#muparserx" class="md-nav__link">
    muparserx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mupexi" class="md-nav__link">
    MuPeXI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#muscle" class="md-nav__link">
    MUSCLE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#music" class="md-nav__link">
    MuSiC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#must" class="md-nav__link">
    MUST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mutect" class="md-nav__link">
    MuTect
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mutil" class="md-nav__link">
    mutil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mvapich2" class="md-nav__link">
    MVAPICH2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mxml" class="md-nav__link">
    mxml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mxmlplus" class="md-nav__link">
    mxmlplus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mxnet" class="md-nav__link">
    MXNet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mycc" class="md-nav__link">
    MyCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mygene" class="md-nav__link">
    mygene
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mymedialite" class="md-nav__link">
    MyMediaLite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mympingpong" class="md-nav__link">
    mympingpong
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#myokit" class="md-nav__link">
    Myokit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mypy" class="md-nav__link">
    mypy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mysql" class="md-nav__link">
    MySQL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mysql-python" class="md-nav__link">
    MySQL-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mysqlclient" class="md-nav__link">
    mysqlclient
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#n" class="md-nav__link">
    N
  </a>
  
    <nav class="md-nav" aria-label="N">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#n2v" class="md-nav__link">
    n2v
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nag" class="md-nav__link">
    NAG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nagfor" class="md-nav__link">
    NAGfor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#namd" class="md-nav__link">
    NAMD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#namedlist" class="md-nav__link">
    namedlist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nano" class="md-nav__link">
    nano
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanocomp" class="md-nav__link">
    NanoComp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanocompore" class="md-nav__link">
    nanocompore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanofilt" class="md-nav__link">
    NanoFilt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanoflann" class="md-nav__link">
    nanoflann
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanoget" class="md-nav__link">
    nanoget
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanolyse" class="md-nav__link">
    NanoLyse
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanomath" class="md-nav__link">
    nanomath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanomax-analysis-utils" class="md-nav__link">
    nanomax-analysis-utils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanonet" class="md-nav__link">
    nanonet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanoplot" class="md-nav__link">
    NanoPlot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanopolish" class="md-nav__link">
    nanopolish
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nanopolishcomp" class="md-nav__link">
    NanopolishComp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#napari" class="md-nav__link">
    napari
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nasm" class="md-nav__link">
    NASM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nauty" class="md-nav__link">
    nauty
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nbo" class="md-nav__link">
    NBO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncbi-toolkit" class="md-nav__link">
    NCBI-Toolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncbi-vdb" class="md-nav__link">
    ncbi-vdb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nccl" class="md-nav__link">
    NCCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nccl-tests" class="md-nav__link">
    NCCL-tests
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncdf4" class="md-nav__link">
    ncdf4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncdu" class="md-nav__link">
    ncdu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nciplot" class="md-nav__link">
    NCIPLOT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncl" class="md-nav__link">
    NCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nco" class="md-nav__link">
    NCO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncompress" class="md-nav__link">
    ncompress
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncurses" class="md-nav__link">
    ncurses
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ncview" class="md-nav__link">
    ncview
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nd2reader" class="md-nav__link">
    nd2reader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ne" class="md-nav__link">
    ne
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#neci" class="md-nav__link">
    NECI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nedit" class="md-nav__link">
    NEdit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nek5000" class="md-nav__link">
    Nek5000
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nektar" class="md-nav__link">
    Nektar++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#neon" class="md-nav__link">
    neon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#neptune-client" class="md-nav__link">
    neptune-client
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#net-core" class="md-nav__link">
    Net-core
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netcdf" class="md-nav__link">
    netCDF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netcdf-c" class="md-nav__link">
    netCDF-C++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netcdf-c4" class="md-nav__link">
    netCDF-C++4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netcdf-fortran" class="md-nav__link">
    netCDF-Fortran
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netcdf4-python" class="md-nav__link">
    netcdf4-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netloc" class="md-nav__link">
    netloc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netlogo" class="md-nav__link">
    NetLogo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netmhc" class="md-nav__link">
    netMHC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netmhcii" class="md-nav__link">
    netMHCII
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netmhciipan" class="md-nav__link">
    netMHCIIpan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netmhcpan" class="md-nav__link">
    netMHCpan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netpipe" class="md-nav__link">
    NetPIPE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#netpyne" class="md-nav__link">
    NetPyNE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nettle" class="md-nav__link">
    nettle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#networktools" class="md-nav__link">
    networkTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#networkx" class="md-nav__link">
    networkx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#neurokit" class="md-nav__link">
    NeuroKit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#neuron" class="md-nav__link">
    NEURON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#newhybrids" class="md-nav__link">
    NewHybrids
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nextflow" class="md-nav__link">
    Nextflow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nextgenmap" class="md-nav__link">
    NextGenMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nexus-cl" class="md-nav__link">
    NEXUS-CL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nf-core-mag" class="md-nav__link">
    nf-core-mag
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nfft" class="md-nav__link">
    NFFT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nghttp2" class="md-nav__link">
    nghttp2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nghttp3" class="md-nav__link">
    nghttp3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngless" class="md-nav__link">
    NGLess
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nglview" class="md-nav__link">
    nglview
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngs" class="md-nav__link">
    NGS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngs-python" class="md-nav__link">
    NGS-Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngsadmix" class="md-nav__link">
    NGSadmix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngspeciesid" class="md-nav__link">
    NGSpeciesID
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngspice" class="md-nav__link">
    ngspice
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ngtcp2" class="md-nav__link">
    ngtcp2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nibabel" class="md-nav__link">
    NiBabel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nichenetr" class="md-nav__link">
    nichenetr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nifti" class="md-nav__link">
    NIfTI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nifti2dicom" class="md-nav__link">
    nifti2dicom
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nilearn" class="md-nav__link">
    Nilearn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nim" class="md-nav__link">
    Nim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nimble" class="md-nav__link">
    NIMBLE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ninja" class="md-nav__link">
    Ninja
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nipype" class="md-nav__link">
    Nipype
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nlmpy" class="md-nav__link">
    NLMpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nlohmann_json" class="md-nav__link">
    nlohmann_json
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nlopt" class="md-nav__link">
    NLopt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nltk" class="md-nav__link">
    NLTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nnu-net" class="md-nav__link">
    nnU-Net
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#node-red" class="md-nav__link">
    Node-RED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nodejs" class="md-nav__link">
    nodejs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#normaliz" class="md-nav__link">
    Normaliz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nose-parameterized" class="md-nav__link">
    nose-parameterized
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#novoalign" class="md-nav__link">
    novoalign
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#novoplasty" class="md-nav__link">
    NOVOPlasty
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#npstat" class="md-nav__link">
    npstat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nrgljubljana" class="md-nav__link">
    NRGLjubljana
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nsight-compute" class="md-nav__link">
    Nsight-Compute
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nsight-systems" class="md-nav__link">
    Nsight-Systems
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nspr" class="md-nav__link">
    NSPR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nss" class="md-nav__link">
    NSS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nsync" class="md-nav__link">
    nsync
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ntcard" class="md-nav__link">
    ntCard
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ntedit" class="md-nav__link">
    ntEdit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nthits" class="md-nav__link">
    ntHits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ntl" class="md-nav__link">
    NTL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ntpoly" class="md-nav__link">
    NTPoly
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#num2words" class="md-nav__link">
    num2words
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#numactl" class="md-nav__link">
    numactl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#numba" class="md-nav__link">
    numba
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#numdiff" class="md-nav__link">
    numdiff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#numexpr" class="md-nav__link">
    numexpr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#numpy" class="md-nav__link">
    numpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nvhpc" class="md-nav__link">
    NVHPC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nvofbf" class="md-nav__link">
    nvofbf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nvompi" class="md-nav__link">
    nvompi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nvshmem" class="md-nav__link">
    NVSHMEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nvtop" class="md-nav__link">
    nvtop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nwchem" class="md-nav__link">
    NWChem
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nxtrim" class="md-nav__link">
    NxTrim
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#o" class="md-nav__link">
    O
  </a>
  
    <nav class="md-nav" aria-label="O">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#oases" class="md-nav__link">
    Oases
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#obitools" class="md-nav__link">
    OBITools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#obitools3" class="md-nav__link">
    OBITools3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ocaml" class="md-nav__link">
    OCaml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#occt" class="md-nav__link">
    occt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#oceanspy" class="md-nav__link">
    oceanspy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ocnet" class="md-nav__link">
    OCNet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#octave" class="md-nav__link">
    Octave
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#octopus-vcf" class="md-nav__link">
    Octopus-vcf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ogdf" class="md-nav__link">
    OGDF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#olaflow" class="md-nav__link">
    olaFlow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#olego" class="md-nav__link">
    olego
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#oma" class="md-nav__link">
    OMA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#omegafold" class="md-nav__link">
    OmegaFold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#onedrive" class="md-nav__link">
    onedrive
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#onnx" class="md-nav__link">
    ONNX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#onnx-runtime" class="md-nav__link">
    ONNX-Runtime
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ont-fast5-api" class="md-nav__link">
    ont-fast5-api
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ont-remora" class="md-nav__link">
    ont-remora
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#oompa" class="md-nav__link">
    OOMPA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opari2" class="md-nav__link">
    OPARI2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#open-data-cube-core" class="md-nav__link">
    Open-Data-Cube-Core
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openai-gym" class="md-nav__link">
    OpenAI-Gym
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openbabel" class="md-nav__link">
    OpenBabel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openblas" class="md-nav__link">
    OpenBLAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opencarp" class="md-nav__link">
    openCARP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opencensus-python" class="md-nav__link">
    OpenCensus-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opencoarrays" class="md-nav__link">
    OpenCoarrays
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opencolorio" class="md-nav__link">
    OpenColorIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opencv" class="md-nav__link">
    OpenCV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openexr" class="md-nav__link">
    OpenEXR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openface" class="md-nav__link">
    OpenFace
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openfast" class="md-nav__link">
    OpenFAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openfoam" class="md-nav__link">
    OpenFOAM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openfoam-extend" class="md-nav__link">
    OpenFOAM-Extend
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openfold" class="md-nav__link">
    OpenFold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openforcefield" class="md-nav__link">
    OpenForceField
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openimageio" class="md-nav__link">
    OpenImageIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openjpeg" class="md-nav__link">
    OpenJPEG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openkim-api" class="md-nav__link">
    OpenKIM-API
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openkim-models" class="md-nav__link">
    openkim-models
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openmm" class="md-nav__link">
    OpenMM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openmm-plumed" class="md-nav__link">
    OpenMM-PLUMED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openmmtools" class="md-nav__link">
    OpenMMTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openmolcas" class="md-nav__link">
    OpenMolcas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openmpi" class="md-nav__link">
    OpenMPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openms" class="md-nav__link">
    OpenMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opennlp" class="md-nav__link">
    OpenNLP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openpgm" class="md-nav__link">
    OpenPGM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openpiv" class="md-nav__link">
    OpenPIV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openpyxl" class="md-nav__link">
    openpyxl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openrefine" class="md-nav__link">
    OpenRefine
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openscenegraph" class="md-nav__link">
    OpenSceneGraph
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opensees" class="md-nav__link">
    OpenSees
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openslide" class="md-nav__link">
    OpenSlide
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openslide-python" class="md-nav__link">
    openslide-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openssl" class="md-nav__link">
    OpenSSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#openstackclient" class="md-nav__link">
    OpenStackClient
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opera" class="md-nav__link">
    OPERA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opera-ms" class="md-nav__link">
    OPERA-MS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#optitype" class="md-nav__link">
    OptiType
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#optix" class="md-nav__link">
    OptiX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#optuna" class="md-nav__link">
    Optuna
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#or-tools" class="md-nav__link">
    OR-Tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#orca" class="md-nav__link">
    ORCA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#orffinder" class="md-nav__link">
    ORFfinder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#orfm" class="md-nav__link">
    OrfM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#orthofinder" class="md-nav__link">
    OrthoFinder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#orthomcl" class="md-nav__link">
    OrthoMCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#osi" class="md-nav__link">
    Osi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ospray" class="md-nav__link">
    OSPRay
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#osu-micro-benchmarks" class="md-nav__link">
    OSU-Micro-Benchmarks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#otf2" class="md-nav__link">
    OTF2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ovito" class="md-nav__link">
    OVITO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#owncloud" class="md-nav__link">
    ownCloud
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#oxford_asl" class="md-nav__link">
    oxford_asl
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#p" class="md-nav__link">
    P
  </a>
  
    <nav class="md-nav" aria-label="P">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#p11-kit" class="md-nav__link">
    p11-kit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#p4-phylogenetics" class="md-nav__link">
    p4-phylogenetics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#p4est" class="md-nav__link">
    p4est
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#p4vasp" class="md-nav__link">
    p4vasp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#p7zip" class="md-nav__link">
    p7zip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#packmol" class="md-nav__link">
    packmol
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pagmo" class="md-nav__link">
    pagmo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pairsnp" class="md-nav__link">
    pairsnp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pal2nal" class="md-nav__link">
    PAL2NAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#paladin" class="md-nav__link">
    paladin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#paml" class="md-nav__link">
    PAML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#panaroo" class="md-nav__link">
    panaroo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pandapower" class="md-nav__link">
    pandapower
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pandas" class="md-nav__link">
    pandas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pandas-datareader" class="md-nav__link">
    pandas-datareader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pandaseq" class="md-nav__link">
    PANDAseq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pandoc" class="md-nav__link">
    Pandoc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#panedr" class="md-nav__link">
    Panedr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pango" class="md-nav__link">
    Pango
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pangolin" class="md-nav__link">
    pangolin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#panito" class="md-nav__link">
    panito
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#papi" class="md-nav__link">
    PAPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parallel" class="md-nav__link">
    parallel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parallel-fastq-dump" class="md-nav__link">
    parallel-fastq-dump
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parallel-hashmap" class="md-nav__link">
    Parallel-Hashmap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parallelio" class="md-nav__link">
    ParallelIO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parameterized" class="md-nav__link">
    parameterized
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parasail" class="md-nav__link">
    parasail
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#paraver" class="md-nav__link">
    Paraver
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#paraview" class="md-nav__link">
    ParaView
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parcels" class="md-nav__link">
    Parcels
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pari-gp" class="md-nav__link">
    PARI-GP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parmed" class="md-nav__link">
    ParmEd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parmetis" class="md-nav__link">
    ParMETIS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parmgridgen" class="md-nav__link">
    ParMGridGen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#partitionfinder" class="md-nav__link">
    PartitionFinder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pasta" class="md-nav__link">
    pasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pastix" class="md-nav__link">
    PaStiX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#patch" class="md-nav__link">
    patch
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#patchelf" class="md-nav__link">
    patchelf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pathpy" class="md-nav__link">
    path.py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#paup" class="md-nav__link">
    PAUP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pauvre" class="md-nav__link">
    pauvre
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbbam" class="md-nav__link">
    pbbam
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbcopper" class="md-nav__link">
    pbcopper
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbdagcon" class="md-nav__link">
    pbdagcon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbmm2" class="md-nav__link">
    pbmm2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbs_python" class="md-nav__link">
    pbs_python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pbsuite" class="md-nav__link">
    PBSuite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcangsd" class="md-nav__link">
    PCAngsd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcc" class="md-nav__link">
    PCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcl" class="md-nav__link">
    PCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcmsolver" class="md-nav__link">
    PCMSolver
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcraster" class="md-nav__link">
    PCRaster
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcre" class="md-nav__link">
    PCRE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pcre2" class="md-nav__link">
    PCRE2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pdsh" class="md-nav__link">
    pdsh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pdt" class="md-nav__link">
    PDT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pear" class="md-nav__link">
    PEAR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#penncnv" class="md-nav__link">
    PennCNV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pept" class="md-nav__link">
    PEPT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#percolator" class="md-nav__link">
    Percolator
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#perl" class="md-nav__link">
    Perl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#perl-app-cpanminus" class="md-nav__link">
    perl-app-cpanminus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#perl4-corelibs" class="md-nav__link">
    Perl4-CoreLibs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#perseus" class="md-nav__link">
    Perseus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pest" class="md-nav__link">
    PEST++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#petsc" class="md-nav__link">
    PETSc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#petsc4py" class="md-nav__link">
    petsc4py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pfamscan" class="md-nav__link">
    PfamScan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pfind" class="md-nav__link">
    pfind
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pftoolsv3" class="md-nav__link">
    pftoolsV3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pfunit" class="md-nav__link">
    pFUnit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgdspider" class="md-nav__link">
    PGDSpider
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgi" class="md-nav__link">
    PGI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phanotate" class="md-nav__link">
    PHANOTATE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phantompeakqualtools" class="md-nav__link">
    Phantompeakqualtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phase" class="md-nav__link">
    PHASE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phast" class="md-nav__link">
    PHAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phenoflow" class="md-nav__link">
    Phenoflow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phewas" class="md-nav__link">
    PheWAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phipack" class="md-nav__link">
    PhiPack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phlat" class="md-nav__link">
    PHLAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phonemizer" class="md-nav__link">
    phonemizer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phono3py" class="md-nav__link">
    phono3py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phonopy" class="md-nav__link">
    phonopy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#photontorch" class="md-nav__link">
    photontorch
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phototonic" class="md-nav__link">
    phototonic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phylip" class="md-nav__link">
    PHYLIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phylobayes-mpi" class="md-nav__link">
    PhyloBayes-MPI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phylokit" class="md-nav__link">
    phylokit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phylonaut" class="md-nav__link">
    phylonaut
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phylophlan" class="md-nav__link">
    PhyloPhlAn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phyml" class="md-nav__link">
    PhyML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phyx" class="md-nav__link">
    phyx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#picard" class="md-nav__link">
    picard
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pigz" class="md-nav__link">
    pigz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pil" class="md-nav__link">
    PIL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pileometh" class="md-nav__link">
    PileOMeth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pillow" class="md-nav__link">
    Pillow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pillow-simd" class="md-nav__link">
    Pillow-SIMD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pilon" class="md-nav__link">
    Pilon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pims" class="md-nav__link">
    PIMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pindel" class="md-nav__link">
    Pindel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pingouin" class="md-nav__link">
    Pingouin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pint" class="md-nav__link">
    Pint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pip" class="md-nav__link">
    pip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pipits" class="md-nav__link">
    PIPITS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pirate" class="md-nav__link">
    PIRATE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pirs" class="md-nav__link">
    pIRS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pisces" class="md-nav__link">
    Pisces
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pisvm" class="md-nav__link">
    piSvM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pisvm-jsc" class="md-nav__link">
    piSvM-JSC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pixman" class="md-nav__link">
    pixman
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pizzly" class="md-nav__link">
    pizzly
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pkg-config" class="md-nav__link">
    pkg-config
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pkgconf" class="md-nav__link">
    pkgconf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pkgconfig" class="md-nav__link">
    pkgconfig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plams" class="md-nav__link">
    PLAMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plantcv" class="md-nav__link">
    plantcv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plascope" class="md-nav__link">
    PlaScope
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plasmapy" class="md-nav__link">
    PlasmaPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plast" class="md-nav__link">
    PLAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#platanus" class="md-nav__link">
    Platanus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#platypus" class="md-nav__link">
    Platypus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plc" class="md-nav__link">
    plc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    PLINK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plinkqc" class="md-nav__link">
    plinkQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plinkseq" class="md-nav__link">
    PLINKSEQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plot1cell" class="md-nav__link">
    plot1cell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ploticus" class="md-nav__link">
    Ploticus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plotly" class="md-nav__link">
    plotly
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plotly-orca" class="md-nav__link">
    plotly-orca
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plotlypy" class="md-nav__link">
    plotly.py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plplot" class="md-nav__link">
    PLplot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plumed" class="md-nav__link">
    PLUMED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ply" class="md-nav__link">
    PLY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pmix" class="md-nav__link">
    PMIx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pmx" class="md-nav__link">
    pmx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pnetcdf" class="md-nav__link">
    PnetCDF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pocl" class="md-nav__link">
    pocl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#poetry" class="md-nav__link">
    poetry
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#polars" class="md-nav__link">
    polars
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#polymake" class="md-nav__link">
    polymake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pomkl" class="md-nav__link">
    pomkl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pompi" class="md-nav__link">
    pompi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#poppler" class="md-nav__link">
    poppler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popscle" class="md-nav__link">
    popscle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popt" class="md-nav__link">
    popt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porechop" class="md-nav__link">
    Porechop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porefoam" class="md-nav__link">
    porefoam
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#poretools" class="md-nav__link">
    poretools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#portcullis" class="md-nav__link">
    Portcullis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#postgres-xl" class="md-nav__link">
    Postgres-XL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#postgresql" class="md-nav__link">
    PostgreSQL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pot" class="md-nav__link">
    POT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pov-ray" class="md-nav__link">
    POV-Ray
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ppanggolin" class="md-nav__link">
    PPanGGOLiN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ppfold" class="md-nav__link">
    PPfold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ppl" class="md-nav__link">
    ppl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pplacer" class="md-nav__link">
    pplacer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pplpy" class="md-nav__link">
    pplpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prank" class="md-nav__link">
    PRANK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prc" class="md-nav__link">
    PRC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#precice" class="md-nav__link">
    preCICE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prequal" class="md-nav__link">
    PREQUAL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#preseq" class="md-nav__link">
    preseq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#presto" class="md-nav__link">
    presto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pretty-yaml" class="md-nav__link">
    pretty-yaml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#primer3" class="md-nav__link">
    Primer3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prinseq" class="md-nav__link">
    PRINSEQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#printproto" class="md-nav__link">
    printproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prisms-pf" class="md-nav__link">
    PRISMS-PF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#probabel" class="md-nav__link">
    ProbABEL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prodigal" class="md-nav__link">
    prodigal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#profit" class="md-nav__link">
    ProFit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proj" class="md-nav__link">
    PROJ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#projectq" class="md-nav__link">
    ProjectQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prokka" class="md-nav__link">
    prokka
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prompt-toolkit" class="md-nav__link">
    prompt-toolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proovread" class="md-nav__link">
    proovread
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#propy" class="md-nav__link">
    propy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proteinortho" class="md-nav__link">
    Proteinortho
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prothint" class="md-nav__link">
    ProtHint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#protobuf" class="md-nav__link">
    protobuf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#protobuf-python" class="md-nav__link">
    protobuf-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#protozero" class="md-nav__link">
    protozero
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prsice" class="md-nav__link">
    PRSice
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pscom" class="md-nav__link">
    pscom
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psi" class="md-nav__link">
    PSI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psi4" class="md-nav__link">
    PSI4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psiclass" class="md-nav__link">
    PsiCLASS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psipred" class="md-nav__link">
    PSIPRED
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psmc" class="md-nav__link">
    psmc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psmpi" class="md-nav__link">
    psmpi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psmpi2" class="md-nav__link">
    psmpi2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psolver" class="md-nav__link">
    PSolver
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psortb" class="md-nav__link">
    PSORTb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psrecord" class="md-nav__link">
    psrecord
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pstoedit" class="md-nav__link">
    pstoedit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psutil" class="md-nav__link">
    psutil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#psycopg2" class="md-nav__link">
    psycopg2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ptemcee" class="md-nav__link">
    ptemcee
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ptesfinder" class="md-nav__link">
    PTESFinder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pubtcrs" class="md-nav__link">
    pubtcrs
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pugixml" class="md-nav__link">
    pugixml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pullseq" class="md-nav__link">
    pullseq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pulp" class="md-nav__link">
    PuLP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#purge_dups" class="md-nav__link">
    purge_dups
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#py" class="md-nav__link">
    py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#py-aiger" class="md-nav__link">
    py-aiger
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#py-aiger-bdd" class="md-nav__link">
    py-aiger-bdd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#py-cpuinfo" class="md-nav__link">
    py-cpuinfo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyabc" class="md-nav__link">
    pyABC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyamg" class="md-nav__link">
    PyAMG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyaps3" class="md-nav__link">
    PyAPS3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pybedtools" class="md-nav__link">
    pybedtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyberny" class="md-nav__link">
    PyBerny
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pybigwig" class="md-nav__link">
    pyBigWig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pybind11" class="md-nav__link">
    pybind11
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pybinding" class="md-nav__link">
    pybinding
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycairo" class="md-nav__link">
    PyCairo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycalib" class="md-nav__link">
    PyCalib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyccel" class="md-nav__link">
    pyccel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycharm" class="md-nav__link">
    PyCharm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pychemps2" class="md-nav__link">
    PyCheMPS2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycifrw" class="md-nav__link">
    PyCifRW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyclone" class="md-nav__link">
    PyClone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycma" class="md-nav__link">
    pycma
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycocotools" class="md-nav__link">
    pycocotools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycodestyle" class="md-nav__link">
    pycodestyle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycogent" class="md-nav__link">
    PyCogent
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycoqc" class="md-nav__link">
    pycoQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycuda" class="md-nav__link">
    PyCUDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pycurl" class="md-nav__link">
    PycURL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydamage" class="md-nav__link">
    PyDamage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydantic" class="md-nav__link">
    pydantic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydatastream" class="md-nav__link">
    PyDatastream
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydicom" class="md-nav__link">
    pydicom
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydlpoly" class="md-nav__link">
    pydlpoly
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pydot" class="md-nav__link">
    pydot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyega3" class="md-nav__link">
    pyEGA3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyenchant" class="md-nav__link">
    pyenchant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyevtk" class="md-nav__link">
    PyEVTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfai" class="md-nav__link">
    pyFAI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfaidx" class="md-nav__link">
    pyfaidx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfasta" class="md-nav__link">
    pyfasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyffmpeg" class="md-nav__link">
    PyFFmpeg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfftw" class="md-nav__link">
    pyFFTW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfits" class="md-nav__link">
    pyfits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfmi" class="md-nav__link">
    PyFMI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfoam" class="md-nav__link">
    PyFoam
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfr" class="md-nav__link">
    PyFR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyfrag" class="md-nav__link">
    PyFrag
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygccxml" class="md-nav__link">
    pygccxml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygenometracks" class="md-nav__link">
    pyGenomeTracks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygeos" class="md-nav__link">
    PyGEOS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygimli" class="md-nav__link">
    pyGIMLi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygments" class="md-nav__link">
    Pygments
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygmo" class="md-nav__link">
    pygmo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygobject" class="md-nav__link">
    PyGObject
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygraphviz" class="md-nav__link">
    pygraphviz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygrib" class="md-nav__link">
    pygrib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygtk" class="md-nav__link">
    PyGTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygts" class="md-nav__link">
    PyGTS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pygwas" class="md-nav__link">
    PyGWAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyhdf" class="md-nav__link">
    pyhdf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyimagej" class="md-nav__link">
    PyImageJ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyiron" class="md-nav__link">
    pyiron
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyke3" class="md-nav__link">
    Pyke3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pylift" class="md-nav__link">
    pylift
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pylint" class="md-nav__link">
    Pylint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymatgen" class="md-nav__link">
    pymatgen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymatgen-db" class="md-nav__link">
    pymatgen-db
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymbar" class="md-nav__link">
    pymbar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymc3" class="md-nav__link">
    PyMC3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymca" class="md-nav__link">
    pymca
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymemcache" class="md-nav__link">
    pymemcache
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pymol" class="md-nav__link">
    PyMOL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pynast" class="md-nav__link">
    PyNAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyobjcryst" class="md-nav__link">
    pyobjcryst
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyod" class="md-nav__link">
    PyOD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyomo" class="md-nav__link">
    Pyomo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyopencl" class="md-nav__link">
    PyOpenCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyopengl" class="md-nav__link">
    PyOpenGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyparsing" class="md-nav__link">
    pyparsing
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyperf" class="md-nav__link">
    pyperf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyplusplus" class="md-nav__link">
    pyplusplus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pypower" class="md-nav__link">
    PYPOWER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyproj" class="md-nav__link">
    pyproj
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pypsa" class="md-nav__link">
    PyPSA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyqstem" class="md-nav__link">
    pyqstem
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyqt" class="md-nav__link">
    PyQt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyqt5" class="md-nav__link">
    PyQt5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyqtgraph" class="md-nav__link">
    PyQtGraph
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyradiomics" class="md-nav__link">
    pyradiomics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyre" class="md-nav__link">
    PyRe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyretis" class="md-nav__link">
    PyRETIS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyringe" class="md-nav__link">
    pyringe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyro-api" class="md-nav__link">
    pyro-api
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyro-ppl" class="md-nav__link">
    pyro-ppl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyro4" class="md-nav__link">
    Pyro4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyrosetta" class="md-nav__link">
    PyRosetta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysam" class="md-nav__link">
    Pysam
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysamstats" class="md-nav__link">
    pysamstats
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysat" class="md-nav__link">
    PySAT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyscaf" class="md-nav__link">
    pyScaf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyscenic" class="md-nav__link">
    pySCENIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyscf" class="md-nav__link">
    PySCF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysheds" class="md-nav__link">
    pysheds
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyshp" class="md-nav__link">
    pyshp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyside2" class="md-nav__link">
    PySide2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyslim" class="md-nav__link">
    pyslim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysndfx" class="md-nav__link">
    pysndfx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysolar" class="md-nav__link">
    Pysolar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyspoa" class="md-nav__link">
    pyspoa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysqlite" class="md-nav__link">
    pysqlite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pystan" class="md-nav__link">
    PyStan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pysteps" class="md-nav__link">
    pysteps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pystran" class="md-nav__link">
    pystran
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytables" class="md-nav__link">
    PyTables
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytest" class="md-nav__link">
    pytest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytest-benchmark" class="md-nav__link">
    pytest-benchmark
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytest-rerunfailures" class="md-nav__link">
    pytest-rerunfailures
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytest-shard" class="md-nav__link">
    pytest-shard
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytest-xdist" class="md-nav__link">
    pytest-xdist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pythia" class="md-nav__link">
    PYTHIA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python" class="md-nav__link">
    Python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-docx" class="md-nav__link">
    python-docx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-hl7" class="md-nav__link">
    python-hl7
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-igraph" class="md-nav__link">
    python-igraph
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-irodsclient" class="md-nav__link">
    python-irodsclient
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-isal" class="md-nav__link">
    python-isal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-levenshtein" class="md-nav__link">
    python-Levenshtein
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-libsbml" class="md-nav__link">
    python-libsbml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-louvain" class="md-nav__link">
    python-louvain
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-parasail" class="md-nav__link">
    python-parasail
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-telegram-bot" class="md-nav__link">
    python-telegram-bot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#python-weka-wrapper3" class="md-nav__link">
    python-weka-wrapper3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pythran" class="md-nav__link">
    pythran
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytorch" class="md-nav__link">
    PyTorch
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytorch-geometric" class="md-nav__link">
    PyTorch-Geometric
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytorch-ignite" class="md-nav__link">
    PyTorch-Ignite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytorch-lightning" class="md-nav__link">
    PyTorch-Lightning
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pytorch3d" class="md-nav__link">
    PyTorch3D
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyvcf" class="md-nav__link">
    PyVCF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyvcf3" class="md-nav__link">
    PyVCF3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pywannier90" class="md-nav__link">
    pyWannier90
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pywavelets" class="md-nav__link">
    PyWavelets
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyyaml" class="md-nav__link">
    PyYAML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pyzmq" class="md-nav__link">
    PyZMQ
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#q" class="md-nav__link">
    Q
  </a>
  
    <nav class="md-nav" aria-label="Q">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#q6" class="md-nav__link">
    Q6
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qca" class="md-nav__link">
    QCA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qcat" class="md-nav__link">
    qcat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qcg-pilotjob" class="md-nav__link">
    QCG-PilotJob
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qcint" class="md-nav__link">
    qcint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qcxms" class="md-nav__link">
    QCxMS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qd" class="md-nav__link">
    QD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qdd" class="md-nav__link">
    QDD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qemu" class="md-nav__link">
    QEMU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qgis" class="md-nav__link">
    QGIS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qhull" class="md-nav__link">
    Qhull
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qiime" class="md-nav__link">
    QIIME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qiime2" class="md-nav__link">
    QIIME2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qiskit" class="md-nav__link">
    Qiskit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qjson" class="md-nav__link">
    QJson
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qml" class="md-nav__link">
    QML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qpth" class="md-nav__link">
    qpth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qrupdate" class="md-nav__link">
    qrupdate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qscintilla" class="md-nav__link">
    QScintilla
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qt" class="md-nav__link">
    Qt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qt5" class="md-nav__link">
    Qt5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qt5webkit" class="md-nav__link">
    Qt5Webkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qtconsole" class="md-nav__link">
    Qtconsole
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qtkeychain" class="md-nav__link">
    QtKeychain
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qtltools" class="md-nav__link">
    QTLtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qtop" class="md-nav__link">
    qtop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qtpy" class="md-nav__link">
    QtPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qualimap" class="md-nav__link">
    Qualimap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quandl" class="md-nav__link">
    Quandl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quantumespresso" class="md-nav__link">
    QuantumESPRESSO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quast" class="md-nav__link">
    QUAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quazip" class="md-nav__link">
    QuaZIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quickff" class="md-nav__link">
    QuickFF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quickpic" class="md-nav__link">
    QuickPIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quip" class="md-nav__link">
    Quip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quorum" class="md-nav__link">
    Quorum
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qutip" class="md-nav__link">
    QuTiP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qwt" class="md-nav__link">
    Qwt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qwtpolar" class="md-nav__link">
    QwtPolar
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r" class="md-nav__link">
    R
  </a>
  
    <nav class="md-nav" aria-label="R">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#r_1" class="md-nav__link">
    R
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r-bundle-bioconductor" class="md-nav__link">
    R-bundle-Bioconductor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r-inla" class="md-nav__link">
    R-INLA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r-keras" class="md-nav__link">
    R-keras
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r-opencv" class="md-nav__link">
    R-opencv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r-tesseract" class="md-nav__link">
    R-tesseract
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#r2jags" class="md-nav__link">
    R2jags
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#racon" class="md-nav__link">
    Racon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#radeontop" class="md-nav__link">
    radeontop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ragoo" class="md-nav__link">
    RaGOO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ragout" class="md-nav__link">
    Ragout
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ragtag" class="md-nav__link">
    RagTag
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rampart" class="md-nav__link">
    rampart
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#randfold" class="md-nav__link">
    randfold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#randrproto" class="md-nav__link">
    randrproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rapidcsv" class="md-nav__link">
    rapidcsv
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rapidjson" class="md-nav__link">
    RapidJSON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rapidnj" class="md-nav__link">
    rapidNJ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rapidtide" class="md-nav__link">
    rapidtide
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rapsearch2" class="md-nav__link">
    RAPSearch2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rascaf" class="md-nav__link">
    Rascaf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raspa2" class="md-nav__link">
    RASPA2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rasterio" class="md-nav__link">
    rasterio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rasterstats" class="md-nav__link">
    rasterstats
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ratatosk" class="md-nav__link">
    Ratatosk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raxml" class="md-nav__link">
    RAxML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raxml-ng" class="md-nav__link">
    RAxML-NG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ray-assembler" class="md-nav__link">
    Ray-assembler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ray-project" class="md-nav__link">
    Ray-project
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raysect" class="md-nav__link">
    Raysect
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rbfopt" class="md-nav__link">
    RBFOpt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rclone" class="md-nav__link">
    rclone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rcorrector" class="md-nav__link">
    Rcorrector
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rcppgsl" class="md-nav__link">
    RcppGSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rcuda" class="md-nav__link">
    rCUDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rdflib" class="md-nav__link">
    RDFlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rdkit" class="md-nav__link">
    RDKit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rdp-classifier" class="md-nav__link">
    RDP-Classifier
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#re2" class="md-nav__link">
    RE2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#re2c" class="md-nav__link">
    re2c
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#reads2snp" class="md-nav__link">
    Reads2snp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#reapr" class="md-nav__link">
    Reapr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#recon" class="md-nav__link">
    RECON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#red" class="md-nav__link">
    Red
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#redis" class="md-nav__link">
    Redis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#redis-py" class="md-nav__link">
    redis-py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#redundans" class="md-nav__link">
    Redundans
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#reframe" class="md-nav__link">
    ReFrame
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#regionmask" class="md-nav__link">
    regionmask
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#regtools" class="md-nav__link">
    RegTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#relate" class="md-nav__link">
    Relate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#relion" class="md-nav__link">
    RELION
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rematch" class="md-nav__link">
    ReMatCh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#remora" class="md-nav__link">
    REMORA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#renderproto" class="md-nav__link">
    renderproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#repasthpc" class="md-nav__link">
    RepastHPC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#repeatmasker" class="md-nav__link">
    RepeatMasker
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#repeatmodeler" class="md-nav__link">
    RepeatModeler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#repeatscout" class="md-nav__link">
    RepeatScout
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#request" class="md-nav__link">
    request
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#requests" class="md-nav__link">
    requests
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rerconverge" class="md-nav__link">
    RERconverge
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#resolos" class="md-nav__link">
    resolos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#retworkx" class="md-nav__link">
    retworkx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#revbayes" class="md-nav__link">
    RevBayes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rgdal" class="md-nav__link">
    rgdal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rgeos" class="md-nav__link">
    rgeos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rgurobi" class="md-nav__link">
    Rgurobi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rhdf5" class="md-nav__link">
    rhdf5
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rheotool" class="md-nav__link">
    RheoTool
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rickflow" class="md-nav__link">
    rickflow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rinchi" class="md-nav__link">
    RInChI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rioxarray" class="md-nav__link">
    rioxarray
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rising" class="md-nav__link">
    rising
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rivet" class="md-nav__link">
    Rivet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rjags" class="md-nav__link">
    rjags
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rlcard" class="md-nav__link">
    RLCard
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rmarkdown" class="md-nav__link">
    rmarkdown
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rmath" class="md-nav__link">
    Rmath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rmats-turbo" class="md-nav__link">
    rMATS-turbo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rmblast" class="md-nav__link">
    RMBlast
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-bloom" class="md-nav__link">
    RNA-Bloom
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-seqc" class="md-nav__link">
    RNA-SeQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnaclust" class="md-nav__link">
    RNAclust
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnacode" class="md-nav__link">
    RNAcode
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnaindel" class="md-nav__link">
    RNAIndel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnammer" class="md-nav__link">
    RNAmmer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnaquast" class="md-nav__link">
    rnaQUAST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnaz" class="md-nav__link">
    RNAz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rnbeads" class="md-nav__link">
    RnBeads
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#roary" class="md-nav__link">
    Roary
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocm" class="md-nav__link">
    ROCm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocm-cmake" class="md-nav__link">
    rocm-cmake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocm-compilersupport" class="md-nav__link">
    ROCm-CompilerSupport
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocm-smi" class="md-nav__link">
    rocm-smi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocminfo" class="md-nav__link">
    rocminfo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rocr-runtime" class="md-nav__link">
    ROCR-Runtime
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#roct-thunk-interface" class="md-nav__link">
    ROCT-Thunk-Interface
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#roi_pac" class="md-nav__link">
    ROI_PAC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rome" class="md-nav__link">
    ROME
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#root" class="md-nav__link">
    ROOT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#root_numpy" class="md-nav__link">
    root_numpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rootpy" class="md-nav__link">
    rootpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rosetta" class="md-nav__link">
    Rosetta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rpmrebuild" class="md-nav__link">
    rpmrebuild
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rpy2" class="md-nav__link">
    rpy2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rqgis3" class="md-nav__link">
    RQGIS3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rsem" class="md-nav__link">
    RSEM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rseqc" class="md-nav__link">
    RSeQC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rstan" class="md-nav__link">
    RStan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rstanarm" class="md-nav__link">
    rstanarm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rstudio-server" class="md-nav__link">
    RStudio-Server
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rtg-tools" class="md-nav__link">
    RTG-Tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rtree" class="md-nav__link">
    Rtree
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruamelyaml" class="md-nav__link">
    ruamel.yaml
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruby" class="md-nav__link">
    Ruby
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruby-tk" class="md-nav__link">
    Ruby-Tk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruffus" class="md-nav__link">
    ruffus
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rust" class="md-nav__link">
    Rust
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#s" class="md-nav__link">
    S
  </a>
  
    <nav class="md-nav" aria-label="S">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#s-lang" class="md-nav__link">
    S-Lang
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#s4" class="md-nav__link">
    S4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sage" class="md-nav__link">
    SAGE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sailfish" class="md-nav__link">
    Sailfish
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salib" class="md-nav__link">
    SALib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salmon" class="md-nav__link">
    Salmon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salmon-tddft" class="md-nav__link">
    SALMON-TDDFT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sambamba" class="md-nav__link">
    Sambamba
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#samblaster" class="md-nav__link">
    samblaster
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#samcef" class="md-nav__link">
    Samcef
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#samclip" class="md-nav__link">
    samclip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#samplot" class="md-nav__link">
    samplot
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#samtools" class="md-nav__link">
    SAMtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sansa" class="md-nav__link">
    sansa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sap" class="md-nav__link">
    SAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sas" class="md-nav__link">
    SAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#satsuma2" class="md-nav__link">
    Satsuma2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#savvy" class="md-nav__link">
    savvy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saxon-he" class="md-nav__link">
    Saxon-HE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbcl" class="md-nav__link">
    SBCL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbt" class="md-nav__link">
    sbt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scafacos" class="md-nav__link">
    ScaFaCoS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scalapack" class="md-nav__link">
    ScaLAPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scalasca" class="md-nav__link">
    Scalasca
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scalce" class="md-nav__link">
    SCALCE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scanpy" class="md-nav__link">
    scanpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scarches" class="md-nav__link">
    scArches
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scenic" class="md-nav__link">
    SCENIC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scgenefit" class="md-nav__link">
    scGeneFit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scgid" class="md-nav__link">
    SCGid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#schicexplorer" class="md-nav__link">
    scHiCExplorer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#schrodinger" class="md-nav__link">
    Schrodinger
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scib" class="md-nav__link">
    scib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sciclone" class="md-nav__link">
    sciClone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scientificpython" class="md-nav__link">
    ScientificPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-allel" class="md-nav__link">
    scikit-allel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-bio" class="md-nav__link">
    scikit-bio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-build" class="md-nav__link">
    scikit-build
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-cuda" class="md-nav__link">
    scikit-cuda
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-image" class="md-nav__link">
    scikit-image
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-learn" class="md-nav__link">
    scikit-learn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-misc" class="md-nav__link">
    scikit-misc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-multilearn" class="md-nav__link">
    scikit-multilearn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-optimize" class="md-nav__link">
    scikit-optimize
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scikit-uplift" class="md-nav__link">
    scikit-uplift
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sciphi" class="md-nav__link">
    SCIPhI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scipy" class="md-nav__link">
    scipy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scipy-bundle" class="md-nav__link">
    SciPy-bundle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scnorm" class="md-nav__link">
    SCnorm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scoary" class="md-nav__link">
    Scoary
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scons" class="md-nav__link">
    SCons
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scoop" class="md-nav__link">
    SCOOP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scopeloomr" class="md-nav__link">
    SCopeLoomR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#score-p" class="md-nav__link">
    Score-P
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scotch" class="md-nav__link">
    SCOTCH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scp" class="md-nav__link">
    scp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scpred" class="md-nav__link">
    scPred
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scrappie" class="md-nav__link">
    Scrappie
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scvelo" class="md-nav__link">
    scVelo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scvi-tools" class="md-nav__link">
    scvi-tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#scythe" class="md-nav__link">
    Scythe
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdcc" class="md-nav__link">
    SDCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdl" class="md-nav__link">
    SDL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdl2" class="md-nav__link">
    SDL2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdl2_image" class="md-nav__link">
    SDL2_image
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdl_image" class="md-nav__link">
    SDL_image
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdsl" class="md-nav__link">
    SDSL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seaborn" class="md-nav__link">
    Seaborn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#searchgui" class="md-nav__link">
    SearchGUI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seaview" class="md-nav__link">
    SeaView
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#secapr" class="md-nav__link">
    SECAPR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seeder" class="md-nav__link">
    Seeder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#segemehl" class="md-nav__link">
    segemehl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#segmentation-models" class="md-nav__link">
    segmentation-models
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seissol" class="md-nav__link">
    SeisSol
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#selestim" class="md-nav__link">
    SelEstim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#selfies" class="md-nav__link">
    SELFIES
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sentencepiece" class="md-nav__link">
    SentencePiece
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sep" class="md-nav__link">
    sep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sepp" class="md-nav__link">
    SEPP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seq-gen" class="md-nav__link">
    Seq-Gen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seq2hla" class="md-nav__link">
    seq2HLA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqan" class="md-nav__link">
    SeqAn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqan3" class="md-nav__link">
    SeqAn3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqkit" class="md-nav__link">
    SeqKit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqlib" class="md-nav__link">
    SeqLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqmagick" class="md-nav__link">
    Seqmagick
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqprep" class="md-nav__link">
    SeqPrep
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seqtk" class="md-nav__link">
    seqtk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#serf" class="md-nav__link">
    Serf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#setuptools" class="md-nav__link">
    setuptools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seurat" class="md-nav__link">
    Seurat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seuratdata" class="md-nav__link">
    SeuratData
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seuratdisk" class="md-nav__link">
    SeuratDisk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#seuratwrappers" class="md-nav__link">
    SeuratWrappers
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sf" class="md-nav__link">
    sf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sfftk" class="md-nav__link">
    sfftk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shannon" class="md-nav__link">
    Shannon
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shap" class="md-nav__link">
    SHAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapaar" class="md-nav__link">
    shapAAR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit" class="md-nav__link">
    SHAPEIT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    SHAPEIT4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapely" class="md-nav__link">
    Shapely
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sharutils" class="md-nav__link">
    sharutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shasta" class="md-nav__link">
    Shasta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shengbte" class="md-nav__link">
    ShengBTE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shift" class="md-nav__link">
    shift
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shore" class="md-nav__link">
    SHORE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-pair" class="md-nav__link">
    Short-Pair
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shovill" class="md-nav__link">
    shovill
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shrinkwrap" class="md-nav__link">
    shrinkwrap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shtns" class="md-nav__link">
    SHTns
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sibelia" class="md-nav__link">
    Sibelia
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sicer2" class="md-nav__link">
    SICER2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sickle" class="md-nav__link">
    sickle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#siesta" class="md-nav__link">
    Siesta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#signalp" class="md-nav__link">
    SignalP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#silhouetterank" class="md-nav__link">
    silhouetteRank
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#silx" class="md-nav__link">
    silx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simint" class="md-nav__link">
    simint
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simnibs" class="md-nav__link">
    SimNIBS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simpeg" class="md-nav__link">
    SimPEG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simple" class="md-nav__link">
    SIMPLE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simpleelastix" class="md-nav__link">
    SimpleElastix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simpleitk" class="md-nav__link">
    SimpleITK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simpy" class="md-nav__link">
    simpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#simvascular" class="md-nav__link">
    SimVascular
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#singlem" class="md-nav__link">
    SingleM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#singular" class="md-nav__link">
    Singular
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sinvict" class="md-nav__link">
    SiNVICT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sionlib" class="md-nav__link">
    SIONlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sip" class="md-nav__link">
    SIP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#siscone" class="md-nav__link">
    siscone
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sisso" class="md-nav__link">
    SISSO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sisso_1" class="md-nav__link">
    SISSO++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skesa" class="md-nav__link">
    SKESA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sketchmap" class="md-nav__link">
    sketchmap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skewer" class="md-nav__link">
    skewer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sklearn-pandas" class="md-nav__link">
    sklearn-pandas
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sklearn-som" class="md-nav__link">
    sklearn-som
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skorch" class="md-nav__link">
    skorch
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slamdunk" class="md-nav__link">
    SlamDunk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slatec" class="md-nav__link">
    SLATEC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slepc" class="md-nav__link">
    SLEPc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slepc4py" class="md-nav__link">
    slepc4py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sleuth" class="md-nav__link">
    sleuth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slidingwindow" class="md-nav__link">
    slidingwindow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slim" class="md-nav__link">
    SLiM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#slow5tools" class="md-nav__link">
    slow5tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smafa" class="md-nav__link">
    smafa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smallgenomeutilities" class="md-nav__link">
    smallgenomeutilities
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smartdenovo" class="md-nav__link">
    SMARTdenovo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smc" class="md-nav__link">
    SMC++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smfishhmrf" class="md-nav__link">
    smfishHmrf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smithwaterman" class="md-nav__link">
    smithwaterman
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smoldyn" class="md-nav__link">
    Smoldyn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smooth-topk" class="md-nav__link">
    smooth-topk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smrt-link" class="md-nav__link">
    SMRT-Link
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smv" class="md-nav__link">
    SMV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snakemake" class="md-nav__link">
    snakemake
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snap" class="md-nav__link">
    SNAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snap-hmm" class="md-nav__link">
    SNAP-HMM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snape-pooled" class="md-nav__link">
    SNAPE-pooled
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snaphu" class="md-nav__link">
    snaphu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snappy" class="md-nav__link">
    snappy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snippy" class="md-nav__link">
    snippy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snp-sites" class="md-nav__link">
    snp-sites
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snpeff" class="md-nav__link">
    snpEff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snphylo" class="md-nav__link">
    SNPhylo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snpomatic" class="md-nav__link">
    SNPomatic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soapaligner" class="md-nav__link">
    SOAPaligner
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soapdenovo-trans" class="md-nav__link">
    SOAPdenovo-Trans
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soapdenovo2" class="md-nav__link">
    SOAPdenovo2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soapfuse" class="md-nav__link">
    SOAPfuse
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#socat" class="md-nav__link">
    socat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soci" class="md-nav__link">
    SOCI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#solexaqa" class="md-nav__link">
    SolexaQA++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sonic" class="md-nav__link">
    sonic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soqt" class="md-nav__link">
    SoQt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sortmerna" class="md-nav__link">
    SortMeRNA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sox" class="md-nav__link">
    SoX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#soxt" class="md-nav__link">
    SoXt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spaceranger" class="md-nav__link">
    SpaceRanger
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spack" class="md-nav__link">
    Spack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spacy" class="md-nav__link">
    spaCy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spades" class="md-nav__link">
    SPAdes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spaln" class="md-nav__link">
    spaln
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spark" class="md-nav__link">
    Spark
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sparse-neighbors-search" class="md-nav__link">
    sparse-neighbors-search
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sparsehash" class="md-nav__link">
    sparsehash
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spatialde" class="md-nav__link">
    SpatialDE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spatialreg" class="md-nav__link">
    spatialreg
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spdlog" class="md-nav__link">
    spdlog
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spectra" class="md-nav__link">
    SpectrA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spectralmethods" class="md-nav__link">
    spectral.methods
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#speech_tools" class="md-nav__link">
    speech_tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spglib" class="md-nav__link">
    spglib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spglib-python" class="md-nav__link">
    spglib-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sphinx" class="md-nav__link">
    Sphinx
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sphinx-rtd-theme" class="md-nav__link">
    Sphinx-RTD-Theme
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spiceypy" class="md-nav__link">
    SpiceyPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spieceasi" class="md-nav__link">
    SpiecEasi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spladder" class="md-nav__link">
    SplAdder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#splash" class="md-nav__link">
    SPLASH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#splicemap" class="md-nav__link">
    SpliceMap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#split-seq" class="md-nav__link">
    split-seq
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#splitref" class="md-nav__link">
    splitRef
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spm" class="md-nav__link">
    SPM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spoa" class="md-nav__link">
    spoa
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spooles" class="md-nav__link">
    SPOOLES
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spotpy" class="md-nav__link">
    SPOTPY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sprng" class="md-nav__link">
    SPRNG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spyder" class="md-nav__link">
    Spyder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sqlite" class="md-nav__link">
    SQLite
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#squeezemeta" class="md-nav__link">
    SqueezeMeta
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#squidpy" class="md-nav__link">
    Squidpy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sra-toolkit" class="md-nav__link">
    SRA-Toolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sradownloader" class="md-nav__link">
    sradownloader
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#srprism" class="md-nav__link">
    SRPRISM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#srst2" class="md-nav__link">
    SRST2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ssaha2" class="md-nav__link">
    SSAHA2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ssn" class="md-nav__link">
    SSN
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sspace_basic" class="md-nav__link">
    SSPACE_Basic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ssw" class="md-nav__link">
    SSW
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stacey" class="md-nav__link">
    STACEY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stack" class="md-nav__link">
    Stack
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stacks" class="md-nav__link">
    Stacks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stamp" class="md-nav__link">
    STAMP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stamps" class="md-nav__link">
    StaMPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stampy" class="md-nav__link">
    Stampy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#star" class="md-nav__link">
    STAR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#star-ccm" class="md-nav__link">
    STAR-CCM+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#star-fusion" class="md-nav__link">
    STAR-Fusion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stardist" class="md-nav__link">
    stardist
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stars" class="md-nav__link">
    stars
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stata" class="md-nav__link">
    Stata
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statistics-r" class="md-nav__link">
    Statistics-R
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statsmodels" class="md-nav__link">
    statsmodels
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#steak" class="md-nav__link">
    STEAK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stir" class="md-nav__link">
    STIR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stpipeline" class="md-nav__link">
    stpipeline
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#strainberry" class="md-nav__link">
    Strainberry
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stream" class="md-nav__link">
    STREAM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#strelka" class="md-nav__link">
    strelka
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stringtie" class="md-nav__link">
    StringTie
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stripy" class="md-nav__link">
    stripy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#strique" class="md-nav__link">
    STRique
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#structure" class="md-nav__link">
    Structure
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#strumpack" class="md-nav__link">
    STRUMPACK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suave" class="md-nav__link">
    suave
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#subread" class="md-nav__link">
    Subread
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#subversion" class="md-nav__link">
    Subversion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suds" class="md-nav__link">
    suds
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suitesparse" class="md-nav__link">
    SuiteSparse
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumaclust" class="md-nav__link">
    SUMACLUST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumatra" class="md-nav__link">
    SUMATRA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumo" class="md-nav__link">
    SUMO
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sundials" class="md-nav__link">
    SUNDIALS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sunpy" class="md-nav__link">
    SunPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#superlu" class="md-nav__link">
    SuperLU
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#superlu_dist" class="md-nav__link">
    SuperLU_DIST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supermagic" class="md-nav__link">
    supermagic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supernova" class="md-nav__link">
    supernova
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#survivor" class="md-nav__link">
    SURVIVOR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#svdetect" class="md-nav__link">
    SVDetect
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#svdquest" class="md-nav__link">
    SVDquest
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#svg" class="md-nav__link">
    SVG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#svist4get" class="md-nav__link">
    svist4get
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swarm" class="md-nav__link">
    swarm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swash" class="md-nav__link">
    SWASH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swat" class="md-nav__link">
    SWAT+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swifter" class="md-nav__link">
    swifter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swig" class="md-nav__link">
    SWIG
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swipe" class="md-nav__link">
    SWIPE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#swissknife" class="md-nav__link">
    swissknife
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#symengine" class="md-nav__link">
    SymEngine
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#symengine-python" class="md-nav__link">
    SymEngine-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#symphony" class="md-nav__link">
    SYMPHONY
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sympy" class="md-nav__link">
    sympy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#syri" class="md-nav__link">
    SyRI
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#szip" class="md-nav__link">
    Szip
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#t" class="md-nav__link">
    T
  </a>
  
    <nav class="md-nav" aria-label="T">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#tabix" class="md-nav__link">
    tabix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tabixpp" class="md-nav__link">
    tabixpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taco" class="md-nav__link">
    taco
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tagdust" class="md-nav__link">
    TagDust
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taglib" class="md-nav__link">
    TagLib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taiyaki" class="md-nav__link">
    Taiyaki
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#talon" class="md-nav__link">
    TALON
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#talys" class="md-nav__link">
    TALYS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tamkin" class="md-nav__link">
    TAMkin
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#task-spooler" class="md-nav__link">
    task-spooler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taxator-tk" class="md-nav__link">
    taxator-tk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tba" class="md-nav__link">
    TBA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tbb" class="md-nav__link">
    tbb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tbl2asn" class="md-nav__link">
    tbl2asn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tcc" class="md-nav__link">
    TCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tcl" class="md-nav__link">
    Tcl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tclap" class="md-nav__link">
    TCLAP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tcsh" class="md-nav__link">
    tcsh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tecplot360ex" class="md-nav__link">
    tecplot360ex
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#telemac-mascaret" class="md-nav__link">
    TELEMAC-MASCARET
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#telescope" class="md-nav__link">
    Telescope
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#teneto" class="md-nav__link">
    Teneto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorboard" class="md-nav__link">
    tensorboard
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorboardx" class="md-nav__link">
    tensorboardX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorflow" class="md-nav__link">
    TensorFlow
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorflow-datasets" class="md-nav__link">
    TensorFlow-Datasets
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorflow-probability" class="md-nav__link">
    tensorflow-probability
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tensorrt" class="md-nav__link">
    TensorRT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#terastructure" class="md-nav__link">
    terastructure
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#termcolor" class="md-nav__link">
    termcolor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tesla-deployment-kit" class="md-nav__link">
    Tesla-Deployment-Kit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tesseract" class="md-nav__link">
    tesseract
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#testpath" class="md-nav__link">
    testpath
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetgen" class="md-nav__link">
    TetGen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetoolkit" class="md-nav__link">
    TEToolkit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetranscripts" class="md-nav__link">
    TEtranscripts
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#texinfo" class="md-nav__link">
    texinfo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#texlive" class="md-nav__link">
    texlive
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#text-csv" class="md-nav__link">
    Text-CSV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tfea" class="md-nav__link">
    TFEA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#theano" class="md-nav__link">
    Theano
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#themispy" class="md-nav__link">
    ThemisPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#theta" class="md-nav__link">
    THetA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#thurstonianirt" class="md-nav__link">
    thurstonianIRT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ticcutils" class="md-nav__link">
    TiCCutils
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tidybayes" class="md-nav__link">
    tidybayes
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tidymodels" class="md-nav__link">
    tidymodels
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tika" class="md-nav__link">
    Tika
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#timbl" class="md-nav__link">
    TiMBL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#time" class="md-nav__link">
    time
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tinker" class="md-nav__link">
    TINKER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tinydb" class="md-nav__link">
    TinyDB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tinyxml" class="md-nav__link">
    TinyXML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tk" class="md-nav__link">
    Tk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tkinter" class="md-nav__link">
    Tkinter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tm-align" class="md-nav__link">
    TM-align
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tmae" class="md-nav__link">
    tMAE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tmap" class="md-nav__link">
    tmap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tmux" class="md-nav__link">
    tmux
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tn93" class="md-nav__link">
    TN93
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tobias" class="md-nav__link">
    TOBIAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tofu" class="md-nav__link">
    ToFu
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#togl" class="md-nav__link">
    Togl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#toil" class="md-nav__link">
    toil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tokenizers" class="md-nav__link">
    tokenizers
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tombo" class="md-nav__link">
    Tombo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#toml-fortran" class="md-nav__link">
    TOML-Fortran
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topaz" class="md-nav__link">
    topaz
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tophat" class="md-nav__link">
    TopHat
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchaudio" class="md-nav__link">
    torchaudio
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchdata" class="md-nav__link">
    torchdata
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchinfo" class="md-nav__link">
    torchinfo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchsampler" class="md-nav__link">
    torchsampler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchtext" class="md-nav__link">
    torchtext
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#torchvision" class="md-nav__link">
    torchvision
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#totalview" class="md-nav__link">
    TotalView
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tqdm" class="md-nav__link">
    tqdm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tracer" class="md-nav__link">
    Tracer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#transcriptclean" class="md-nav__link">
    TranscriptClean
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#transdecoder" class="md-nav__link">
    TransDecoder
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#transformers" class="md-nav__link">
    Transformers
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#transrate" class="md-nav__link">
    Transrate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#travis" class="md-nav__link">
    travis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#travis-analyzer" class="md-nav__link">
    TRAVIS-Analyzer
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#treatsens" class="md-nav__link">
    treatSens
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#treemix" class="md-nav__link">
    TreeMix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#treeshrink" class="md-nav__link">
    TreeShrink
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trf" class="md-nav__link">
    TRF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#triangle" class="md-nav__link">
    Triangle
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trilinos" class="md-nav__link">
    Trilinos
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trim_galore" class="md-nav__link">
    Trim_Galore
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trimal" class="md-nav__link">
    trimAl
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trimesh" class="md-nav__link">
    trimesh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trimmomatic" class="md-nav__link">
    Trimmomatic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trinity" class="md-nav__link">
    Trinity
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#triqs" class="md-nav__link">
    TRIQS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#triqs-cthyb" class="md-nav__link">
    TRIQS-cthyb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#triqs-dft_tools" class="md-nav__link">
    TRIQS-dft_tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#triqs-tprf" class="md-nav__link">
    TRIQS-tprf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trust" class="md-nav__link">
    TRUST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trust4" class="md-nav__link">
    TRUST4
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#trycycler" class="md-nav__link">
    Trycycler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tsne" class="md-nav__link">
    tsne
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#turbovnc" class="md-nav__link">
    TurboVNC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tvb" class="md-nav__link">
    TVB
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tvb-data" class="md-nav__link">
    tvb-data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tvb-deps" class="md-nav__link">
    TVB-deps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tvb-framework" class="md-nav__link">
    tvb-framework
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tvb-library" class="md-nav__link">
    tvb-library
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#twl-ninja" class="md-nav__link">
    TWL-NINJA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#typing-extensions" class="md-nav__link">
    typing-extensions
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#u" class="md-nav__link">
    U
  </a>
  
    <nav class="md-nav" aria-label="U">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ucc" class="md-nav__link">
    UCC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ucc-cuda" class="md-nav__link">
    UCC-CUDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#uclust" class="md-nav__link">
    UCLUST
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ucx" class="md-nav__link">
    UCX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ucx-cuda" class="md-nav__link">
    UCX-CUDA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ucx-py" class="md-nav__link">
    ucx-py
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ucx-rocm" class="md-nav__link">
    UCX-ROCm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#udocker" class="md-nav__link">
    udocker
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#udunits" class="md-nav__link">
    UDUNITS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ufl" class="md-nav__link">
    UFL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#umap-learn" class="md-nav__link">
    umap-learn
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#umi-tools" class="md-nav__link">
    UMI-tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#umi4cpackage" class="md-nav__link">
    umi4cPackage
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#umis" class="md-nav__link">
    umis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unafold" class="md-nav__link">
    UNAFold
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#uncertainty-calibration" class="md-nav__link">
    uncertainty-calibration
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unicycler" class="md-nav__link">
    Unicycler
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unidecode" class="md-nav__link">
    Unidecode
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unimap" class="md-nav__link">
    unimap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#units" class="md-nav__link">
    units
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unixodbc" class="md-nav__link">
    unixODBC
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unrar" class="md-nav__link">
    unrar
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#unzip" class="md-nav__link">
    UnZip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#uqtk" class="md-nav__link">
    UQTk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#usearch" class="md-nav__link">
    USEARCH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#usher" class="md-nav__link">
    UShER
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#uspex" class="md-nav__link">
    USPEX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#utf8proc" class="md-nav__link">
    utf8proc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#util-linux" class="md-nav__link">
    util-linux
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#v" class="md-nav__link">
    V
  </a>
  
    <nav class="md-nav" aria-label="V">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#v8" class="md-nav__link">
    V8
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vala" class="md-nav__link">
    Vala
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#valgrind" class="md-nav__link">
    Valgrind
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vampir" class="md-nav__link">
    Vampir
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vampire" class="md-nav__link">
    Vampire
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vardict" class="md-nav__link">
    VarDict
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#variant_tools" class="md-nav__link">
    variant_tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#variantmetacaller" class="md-nav__link">
    VariantMetaCaller
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#varscan" class="md-nav__link">
    VarScan
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vartools" class="md-nav__link">
    vartools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vasp" class="md-nav__link">
    VASP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vatools" class="md-nav__link">
    VAtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vawk" class="md-nav__link">
    vawk
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vbz-compression" class="md-nav__link">
    VBZ-Compression
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vcf-kit" class="md-nav__link">
    VCF-kit
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vcflib" class="md-nav__link">
    vcflib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vcfnp" class="md-nav__link">
    vcfnp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vcftools" class="md-nav__link">
    VCFtools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vcontact2" class="md-nav__link">
    vConTACT2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vegas" class="md-nav__link">
    VEGAS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#velocyto" class="md-nav__link">
    velocyto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#velvet" class="md-nav__link">
    Velvet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vep" class="md-nav__link">
    VEP
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#verifybamid" class="md-nav__link">
    verifyBamID
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#verse" class="md-nav__link">
    VERSE
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vesta" class="md-nav__link">
    VESTA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#viennarna" class="md-nav__link">
    ViennaRNA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vim" class="md-nav__link">
    Vim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#virsorter" class="md-nav__link">
    VirSorter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#virsorter2" class="md-nav__link">
    VirSorter2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#virtualgl" class="md-nav__link">
    VirtualGL
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#virtuoso-opensource" class="md-nav__link">
    Virtuoso-opensource
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vispy" class="md-nav__link">
    VisPy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vitessce-python" class="md-nav__link">
    vitessce-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vitesscer" class="md-nav__link">
    vitessceR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vmd" class="md-nav__link">
    VMD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vmtk" class="md-nav__link">
    VMTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#voltools" class="md-nav__link">
    voltools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vorbis-tools" class="md-nav__link">
    vorbis-tools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#voro" class="md-nav__link">
    Voro++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsc-base" class="md-nav__link">
    vsc-base
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsc-install" class="md-nav__link">
    vsc-install
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsc-mympirun" class="md-nav__link">
    vsc-mympirun
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsc-mympirun-scoop" class="md-nav__link">
    vsc-mympirun-scoop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsc-processcontrol" class="md-nav__link">
    vsc-processcontrol
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vsearch" class="md-nav__link">
    VSEARCH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vt" class="md-nav__link">
    vt
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vtk" class="md-nav__link">
    VTK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vtune" class="md-nav__link">
    VTune
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vv" class="md-nav__link">
    VV
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#vxl" class="md-nav__link">
    VXL
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#w" class="md-nav__link">
    W
  </a>
  
    <nav class="md-nav" aria-label="W">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#wandb" class="md-nav__link">
    wandb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wannier90" class="md-nav__link">
    Wannier90
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wanniertools" class="md-nav__link">
    WannierTools
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wayland" class="md-nav__link">
    Wayland
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#waylandpp" class="md-nav__link">
    Waylandpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wct" class="md-nav__link">
    WCT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wcwidth" class="md-nav__link">
    wcwidth
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#webin-cli" class="md-nav__link">
    webin-cli
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#webkitgtk" class="md-nav__link">
    WebKitGTK+
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#websocket" class="md-nav__link">
    WebSocket++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#weka" class="md-nav__link">
    WEKA
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wget" class="md-nav__link">
    wget
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wgsim" class="md-nav__link">
    wgsim
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wham" class="md-nav__link">
    WHAM
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#whatshap" class="md-nav__link">
    WhatsHap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wheel" class="md-nav__link">
    wheel
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wien2k" class="md-nav__link">
    WIEN2k
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wildmagic" class="md-nav__link">
    WildMagic
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#winnowmap" class="md-nav__link">
    Winnowmap
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wisecondorx" class="md-nav__link">
    WisecondorX
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wisexome" class="md-nav__link">
    WISExome
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wkhtmltopdf" class="md-nav__link">
    wkhtmltopdf
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#worker" class="md-nav__link">
    worker
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wpebackend-fdo" class="md-nav__link">
    wpebackend-fdo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wps" class="md-nav__link">
    WPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wrf" class="md-nav__link">
    WRF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wrf-fire" class="md-nav__link">
    WRF-Fire
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wrf-python" class="md-nav__link">
    wrf-python
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wtdbg2" class="md-nav__link">
    wtdbg2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wxpropertygrid" class="md-nav__link">
    wxPropertyGrid
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wxpython" class="md-nav__link">
    wxPython
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wxwidgets" class="md-nav__link">
    wxWidgets
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#x" class="md-nav__link">
    X
  </a>
  
    <nav class="md-nav" aria-label="X">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#x11" class="md-nav__link">
    X11
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#x13as" class="md-nav__link">
    x13as
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#x264" class="md-nav__link">
    x264
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#x265" class="md-nav__link">
    x265
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xalt" class="md-nav__link">
    XALT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xarray" class="md-nav__link">
    xarray
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xbitmaps" class="md-nav__link">
    xbitmaps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-proto" class="md-nav__link">
    xcb-proto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-util" class="md-nav__link">
    xcb-util
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-util-image" class="md-nav__link">
    xcb-util-image
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-util-keysyms" class="md-nav__link">
    xcb-util-keysyms
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-util-renderutil" class="md-nav__link">
    xcb-util-renderutil
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcb-util-wm" class="md-nav__link">
    xcb-util-wm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcell" class="md-nav__link">
    xCell
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcfun" class="md-nav__link">
    XCFun
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xcrysden" class="md-nav__link">
    XCrySDen
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xerces-c" class="md-nav__link">
    Xerces-C++
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xesmf" class="md-nav__link">
    xESMF
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xextproto" class="md-nav__link">
    xextproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xf86vidmodeproto" class="md-nav__link">
    xf86vidmodeproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xgboost" class="md-nav__link">
    XGBoost
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xgrafix" class="md-nav__link">
    XGrafix
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xineramaproto" class="md-nav__link">
    xineramaproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xkeyboardconfig" class="md-nav__link">
    XKeyboardConfig
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xlsxwriter" class="md-nav__link">
    XlsxWriter
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmds2" class="md-nav__link">
    XMDS2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmipp" class="md-nav__link">
    Xmipp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmitgcm" class="md-nav__link">
    xmitgcm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xml-compile" class="md-nav__link">
    XML-Compile
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xml-libxml" class="md-nav__link">
    XML-LibXML
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xml-parser" class="md-nav__link">
    XML-Parser
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmlf90" class="md-nav__link">
    xmlf90
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmlsec" class="md-nav__link">
    XMLSec
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xmlstarlet" class="md-nav__link">
    XMLStarlet
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xonsh" class="md-nav__link">
    xonsh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xorg-macros" class="md-nav__link">
    xorg-macros
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xplor-nih" class="md-nav__link">
    XPLOR-NIH
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xprop" class="md-nav__link">
    xprop
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xproto" class="md-nav__link">
    xproto
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xsd" class="md-nav__link">
    XSD
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xtandem" class="md-nav__link">
    XTandem
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xtb" class="md-nav__link">
    xtb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xtensor" class="md-nav__link">
    xtensor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xtrans" class="md-nav__link">
    xtrans
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xvfb" class="md-nav__link">
    Xvfb
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xxd" class="md-nav__link">
    xxd
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xxhash" class="md-nav__link">
    xxHash
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xz" class="md-nav__link">
    XZ
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#y" class="md-nav__link">
    Y
  </a>
  
    <nav class="md-nav" aria-label="Y">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#yacs" class="md-nav__link">
    YACS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yade" class="md-nav__link">
    Yade
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yaff" class="md-nav__link">
    yaff
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yambo" class="md-nav__link">
    Yambo
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yaml-cpp" class="md-nav__link">
    yaml-cpp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yank" class="md-nav__link">
    YANK
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yaps" class="md-nav__link">
    YAPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yasm" class="md-nav__link">
    Yasm
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yaxt" class="md-nav__link">
    YAXT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yices" class="md-nav__link">
    Yices
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yoda" class="md-nav__link">
    YODA
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#z" class="md-nav__link">
    Z
  </a>
  
    <nav class="md-nav" aria-label="Z">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#z3" class="md-nav__link">
    Z3
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zarr" class="md-nav__link">
    zarr
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zeromq" class="md-nav__link">
    ZeroMQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zfp" class="md-nav__link">
    zfp
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zgoubi" class="md-nav__link">
    Zgoubi
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zinger" class="md-nav__link">
    zingeR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zip" class="md-nav__link">
    Zip
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zlib" class="md-nav__link">
    zlib
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zlib-ng" class="md-nav__link">
    zlib-ng
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zlibbioc" class="md-nav__link">
    zlibbioc
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zopfli" class="md-nav__link">
    Zopfli
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zpaq" class="md-nav__link">
    ZPAQ
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zsh" class="md-nav__link">
    zsh
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zstd" class="md-nav__link">
    zstd
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../getting-help/" class="md-nav__link">
        Getting help
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../user-survey/" class="md-nav__link">
        User survey
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_15" >
      
      
      
        <label class="md-nav__link" for="__nav_15" id="__nav_15_label" tabindex="0">
          Other topics
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_15_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_15">
          <span class="md-nav__icon md-icon"></span>
          Other topics
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../installation-alternative/" class="md-nav__link">
        Alternative installation methods
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../legacy/configuration/" class="md-nav__link">
        Configuration (legacy)
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_15_3" >
      
      
      
        <label class="md-nav__link" for="__nav_15_3" id="__nav_15_3_label" tabindex="0">
          Demos
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_15_3_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_15_3">
          <span class="md-nav__icon md-icon"></span>
          Demos
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../demos/" class="md-nav__link">
        (overview)
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../demos/configuring/" class="md-nav__link">
        Configuring EasyBuild
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../demos/review-pr/" class="md-nav__link">
        eb --review-pr
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../deprecated-easyconfigs/" class="md-nav__link">
        Deprecated easyconfigs
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../deprecated-functionality/" class="md-nav__link">
        Deprecated functionality
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../changelog-docs/" class="md-nav__link">
        Documentation changelog
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_15_7" >
      
      
      
        <label class="md-nav__link" for="__nav_15_7" id="__nav_15_7_label" tabindex="0">
          EasyBuild v4
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_15_7_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_15_7">
          <span class="md-nav__icon md-icon"></span>
          EasyBuild v4
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../easybuild-v4/overview-of-changes/" class="md-nav__link">
        Overview of changes
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../easybuild-v4/overview-relocated-functions-constants/" class="md-nav__link">
        Overview of relocated functions/constants
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../installing-environment-modules-without-root-permissions/" class="md-nav__link">
        Installing Environment Modules
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../installing-lmod-without-root-permissions/" class="md-nav__link">
        Installing Lmod
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../removed-functionality/" class="md-nav__link">
        Removed functionality
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../useful-scripts/" class="md-nav__link">
        Useful scripts
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="list-of-supported-software">List of supported software<a class="headerlink" href="#list-of-supported-software" title="Permanent link">&para;</a></h1>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>This page contains a lot of information, it may take a while to load.</p>
</div>
<p>EasyBuild supports 2995 different software packages (incl. toolchains, bundles):</p>
<p><a href="#a">a</a> - <a href="#b">b</a> - <a href="#c">c</a> - <a href="#d">d</a> - <a href="#e">e</a> - <a href="#f">f</a> - <a href="#g">g</a> - <a href="#h">h</a> - <a href="#i">i</a> - <a href="#j">j</a> - <a href="#k">k</a> - <a href="#l">l</a> - <a href="#m">m</a> - <a href="#n">n</a> - <a href="#o">o</a> - <a href="#p">p</a> - <a href="#q">q</a> - <a href="#r">r</a> - <a href="#s">s</a> - <a href="#t">t</a> - <a href="#u">u</a> - <a href="#v">v</a> - <a href="#w">w</a> - <a href="#x">x</a> - <a href="#y">y</a> - <a href="#z">z</a></p>
<h2 id="3">3<a class="headerlink" href="#3" title="Permanent link">&para;</a></h2>
<p><a href="#3to2">3to2</a></p>
<h3 id="3to2">3to2<a class="headerlink" href="#3to2" title="Permanent link">&para;</a></h3>
<p>lib3to2 is a set of fixers that are intended to backport code written for Python version 3.x  into Python version 2.x.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/3to2">https://pypi.python.org/pypi/3to2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h2 id="4">4<a class="headerlink" href="#4" title="Permanent link">&para;</a></h2>
<p><a href="#4ti2">4ti2</a></p>
<h3 id="4ti2">4ti2<a class="headerlink" href="#4ti2" title="Permanent link">&para;</a></h3>
<p>A software package for algebraic, geometric and combinatorial problems on linear spaces</p>
<p><em>homepage</em>: <a href="https://4ti2.github.io/">https://4ti2.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.9</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h2 id="a">A<a class="headerlink" href="#a" title="Permanent link">&para;</a></h2>
<p><a href="#abaqus">ABAQUS</a> - <a href="#abinit">ABINIT</a> - <a href="#abra2">ABRA2</a> - <a href="#abricate">ABRicate</a> - <a href="#abseil">Abseil</a> - <a href="#abtem">abTEM</a> - <a href="#abyss">ABySS</a> - <a href="#ack">ack</a> - <a href="#act">ACT</a> - <a href="#actc">ACTC</a> - <a href="#ada">ada</a> - <a href="#adapterremoval">AdapterRemoval</a> - <a href="#adda">ADDA</a> - <a href="#adf">ADF</a> - <a href="#adios">ADIOS</a> - <a href="#adjusttext">adjustText</a> - <a href="#admixture">ADMIXTURE</a> - <a href="#adol-c">ADOL-C</a> - <a href="#advisor">Advisor</a> - <a href="#afni">AFNI</a> - <a href="#agat">AGAT</a> - <a href="#agfusion">AGFusion</a> - <a href="#aiohttp">aiohttp</a> - <a href="#aladin">ALADIN</a> - <a href="#albacore">Albacore</a> - <a href="#albumentations">Albumentations</a> - <a href="#alevin-fry">alevin-fry</a> - <a href="#alfa">ALFA</a> - <a href="#alfred">Alfred</a> - <a href="#allelecount">alleleCount</a> - <a href="#allinea">Allinea</a> - <a href="#allpaths-lg">ALLPATHS-LG</a> - <a href="#almosthere">almosthere</a> - <a href="#alpha">Alpha</a> - <a href="#alphafold">AlphaFold</a> - <a href="#alps">ALPS</a> - <a href="#alsa-lib">alsa-lib</a> - <a href="#amapvox">AMAPVox</a> - <a href="#amara">Amara</a> - <a href="#amask">amask</a> - <a href="#amber">Amber</a> - <a href="#ambermini">AmberMini</a> - <a href="#ambertools">AmberTools</a> - <a href="#amd-libm">AMD-LibM</a> - <a href="#amd-rng">AMD-RNG</a> - <a href="#amd-securerng">AMD-SecureRNG</a> - <a href="#amd-uprof">AMD-uProf</a> - <a href="#amos">AMOS</a> - <a href="#amphora2">AMPHORA2</a> - <a href="#ampl-mp">AMPL-MP</a> - <a href="#amplimap">amplimap</a> - <a href="#amptk">AMPtk</a> - <a href="#amrplusplus">AmrPlusPlus</a> - <a href="#ams">AMS</a> - <a href="#anaconda2">Anaconda2</a> - <a href="#anaconda3">Anaconda3</a> - <a href="#anci">aNCI</a> - <a href="#andi">andi</a> - <a href="#angel">ANGEL</a> - <a href="#angsd">angsd</a> - <a href="#anicalculator">ANIcalculator</a> - <a href="#annif">Annif</a> - <a href="#annovar">annovar</a> - <a href="#ansys">ANSYS</a> - <a href="#ansys_cfd">ANSYS_CFD</a> - <a href="#ant">ant</a> - <a href="#antismash">antiSMASH</a> - <a href="#antlr">ANTLR</a> - <a href="#ants">ANTs</a> - <a href="#anvio">anvio</a> - <a href="#any2fasta">any2fasta</a> - <a href="#aocc">AOCC</a> - <a href="#aomp">AOMP</a> - <a href="#apbs">APBS</a> - <a href="#apex">apex</a> - <a href="#apr">APR</a> - <a href="#apr-util">APR-util</a> - <a href="#aptasuite">AptaSUITE</a> - <a href="#aragorn">ARAGORN</a> - <a href="#arb">Arb</a> - <a href="#arcade-learning-environment">Arcade-Learning-Environment</a> - <a href="#arcashla">arcasHLA</a> - <a href="#arch">ARCH</a> - <a href="#archive-zip">Archive-Zip</a> - <a href="#archr">ArchR</a> - <a href="#archspec">archspec</a> - <a href="#argos">ARGoS</a> - <a href="#argtable">argtable</a> - <a href="#aria2">aria2</a> - <a href="#arlequin">Arlequin</a> - <a href="#armadillo">Armadillo</a> - <a href="#arosics">arosics</a> - <a href="#arpack++">ARPACK++</a> - <a href="#arpack-ng">arpack-ng</a> - <a href="#arrayfire">ArrayFire</a> - <a href="#arriba">Arriba</a> - <a href="#arrow">Arrow</a> - <a href="#arrow-r">arrow-R</a> - <a href="#art">ART</a> - <a href="#artic-ncov2019">artic-ncov2019</a> - <a href="#arts">ARTS</a> - <a href="#arviz">ArviZ</a> - <a href="#arwen">ARWEN</a> - <a href="#asap">ASAP</a> - <a href="#asap3">ASAP3</a> - <a href="#ase">ASE</a> - <a href="#ashs">ASHS</a> - <a href="#aspera-cli">Aspera-CLI</a> - <a href="#aspera-connect">Aspera-Connect</a> - <a href="#assimp">assimp</a> - <a href="#assimulo">Assimulo</a> - <a href="#astrid">ASTRID</a> - <a href="#astro-tulips">astro-tulips</a> - <a href="#astropy">astropy</a> - <a href="#at-spi2-atk">at-spi2-atk</a> - <a href="#at-spi2-core">at-spi2-core</a> - <a href="#atk">ATK</a> - <a href="#atlas">ATLAS</a> - <a href="#atomate">atomate</a> - <a href="#atompaw">AtomPAW</a> - <a href="#atools">atools</a> - <a href="#atropos">atropos</a> - <a href="#atsas">ATSAS</a> - <a href="#attr">attr</a> - <a href="#attrdict">attrdict</a> - <a href="#attrdict3">attrdict3</a> - <a href="#augur">augur</a> - <a href="#augustus">AUGUSTUS</a> - <a href="#austin">Austin</a> - <a href="#auto-07p">AUTO-07p</a> - <a href="#autoconf">Autoconf</a> - <a href="#autoconf-archive">Autoconf-archive</a> - <a href="#autodock">AutoDock</a> - <a href="#autodock-gpu">AutoDock-GPU</a> - <a href="#autodock-vina">AutoDock-Vina</a> - <a href="#autogenes">AutoGeneS</a> - <a href="#autogrid">AutoGrid</a> - <a href="#automake">Automake</a> - <a href="#automap">AutoMap</a> - <a href="#autopep8">autopep8</a> - <a href="#autotools">Autotools</a> - <a href="#avogadro2">Avogadro2</a> - <a href="#awscli">awscli</a></p>
<h3 id="abaqus">ABAQUS<a class="headerlink" href="#abaqus" title="Permanent link">&para;</a></h3>
<p>Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit  dynamics FEA.</p>
<p><em>homepage</em>: <a href="http://www.simulia.com/products/abaqus_fea.html">http://www.simulia.com/products/abaqus_fea.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.12.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.13.5</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.14.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017</code></td>
<td><code>-hotfix-1721</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018</code></td>
<td><code>-hotfix-1806</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021</code></td>
<td><code>-hotfix-2132</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022</code></td>
<td><code>-hotfix-2214</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022</code></td>
<td><code>-hotfix-2223</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="abinit">ABINIT<a class="headerlink" href="#abinit" title="Permanent link">&para;</a></h3>
<p>ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. </p>
<p><em>homepage</em>: <a href="https://www.abinit.org/">https://www.abinit.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.3</code></td>
<td><code>-x86_64_linux_gnu4.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.5</code></td>
<td><code>-x86_64_linux_gnu4.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.1</code></td>
<td><code>-x86_64_linux_gnu4.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.10.5</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>7.10.5</code></td>
<td><code>-libxc</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>8.0.8</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>8.0.8b</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.2.2</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.4.4</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.6.3</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.10.2</code></td>
<td></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>8.10.3</code></td>
<td></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>9.2.1</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>9.4.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>9.4.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>9.6.2</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2021a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="abra2">ABRA2<a class="headerlink" href="#abra2" title="Permanent link">&para;</a></h3>
<p>Assembly Based ReAligner</p>
<p><em>homepage</em>: <a href="https://github.com/mozack/abra2">https://github.com/mozack/abra2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.22</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.23</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="abricate">ABRicate<a class="headerlink" href="#abricate" title="Permanent link">&para;</a></h3>
<p>Mass screening of contigs for antimicrobial and virulence genes</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/abricate">https://github.com/tseemann/abricate</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.9</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>0.9.9</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="abseil">Abseil<a class="headerlink" href="#abseil" title="Permanent link">&para;</a></h3>
<p>Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives.</p>
<p><em>homepage</em>: <a href="https://abseil.io/">https://abseil.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210324.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="abtem">abTEM<a class="headerlink" href="#abtem" title="Permanent link">&para;</a></h3>
<p>abTEM provides a Python API for running simulations of Transmission Electron Microscopy images.</p>
<p><em>homepage</em>: <a href="https://github.com/jacobjma/abTEM">https://github.com/jacobjma/abTEM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0b24</code></td>
<td><code>-ASE-3.22.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.0.0b26</code></td>
<td><code>-ASE-3.22.0</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="abyss">ABySS<a class="headerlink" href="#abyss" title="Permanent link">&para;</a></h3>
<p>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</p>
<p><em>homepage</em>: <a href="https://www.bcgsc.ca/platform/bioinfo/software/abyss">https://www.bcgsc.ca/platform/bioinfo/software/abyss</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>foss/2016b</code>, <code>foss/2018a</code>, <code>gompi/2019a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.5</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ack">ack<a class="headerlink" href="#ack" title="Permanent link">&para;</a></h3>
<p>ack is a tool like grep, optimized for programmers</p>
<p><em>homepage</em>: <a href="https://beyondgrep.com">https://beyondgrep.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.14</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="act">ACT<a class="headerlink" href="#act" title="Permanent link">&para;</a></h3>
<p>ACT is a Java application for displaying pairwise comparisons between two or more DNA sequences.  It can be used to identify and analyse regions of similarity and difference between genomes  and to explore conservation of synteny, in the context of the entire sequences and their annotation.  It can read complete EMBL, GENBANK and GFF entries or sequences in FASTA or raw format. </p>
<p><em>homepage</em>: <a href="https://sanger-pathogens.github.io/Artemis/ACT/">https://sanger-pathogens.github.io/Artemis/ACT/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>18.0.2</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.0.3</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="actc">ACTC<a class="headerlink" href="#actc" title="Permanent link">&para;</a></h3>
<p>ACTC converts independent triangles into triangle strips or fans.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/actc">https://sourceforge.net/projects/actc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="ada">ada<a class="headerlink" href="#ada" title="Permanent link">&para;</a></h3>
<p>Performs discrete, real, and gentle boost under both exponential and logistic loss on a given data set.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/ada/index.html">https://cran.r-project.org/web/packages/ada/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0-5</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="adapterremoval">AdapterRemoval<a class="headerlink" href="#adapterremoval" title="Permanent link">&para;</a></h3>
<p>AdapterRemoval searches for and removes remnant adapter sequences  from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases  from the 3' end of reads following adapter removal.</p>
<p><em>homepage</em>: <a href="https://github.com/MikkelSchubert/adapterremoval">https://github.com/MikkelSchubert/adapterremoval</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="adda">ADDA<a class="headerlink" href="#adda" title="Permanent link">&para;</a></h3>
<p>ADDA is an open-source parallel implementation of the discrete dipole approximation, capable to simulate light scattering by particles of arbitrary shape and composition in a wide range of particle sizes.</p>
<p><em>homepage</em>: <a href="https://github.com/adda-team/adda/wiki">https://github.com/adda-team/adda/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3b4</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="adf">ADF<a class="headerlink" href="#adf" title="Permanent link">&para;</a></h3>
<p>ADF is an accurate, parallelized, powerful computational chemistry program to understand and  predict chemical structure and reactivity with density functional theory (DFT).</p>
<p><em>homepage</em>: <a href="https://www.scm.com/ADF/">https://www.scm.com/ADF/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2009.01a.pc64_linux.intelmpi</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2014.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2014.11.r48287</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2016.101</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.303</code></td>
<td><code>-intelmpi</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="adios">ADIOS<a class="headerlink" href="#adios" title="Permanent link">&para;</a></h3>
<p>The Adaptable IO System (ADIOS) provides a simple, flexible way for scientists to describe the data in their code that may need to be written, read, or processed outside of the running simulation.</p>
<p><em>homepage</em>: <a href="https://www.olcf.ornl.gov/center-projects/adios/">https://www.olcf.ornl.gov/center-projects/adios/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>20210804</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="adjusttext">adjustText<a class="headerlink" href="#adjusttext" title="Permanent link">&para;</a></h3>
<p>A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps.</p>
<p><em>homepage</em>: <a href="https://github.com/Phlya/adjustText">https://github.com/Phlya/adjustText</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.7.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.7.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="admixture">ADMIXTURE<a class="headerlink" href="#admixture" title="Permanent link">&para;</a></h3>
<p>ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from  multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much  more rapidly using a fast numerical optimization algorithm.</p>
<p><em>homepage</em>: <a href="https://dalexander.github.io/admixture/">https://dalexander.github.io/admixture/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="adol-c">ADOL-C<a class="headerlink" href="#adol-c" title="Permanent link">&para;</a></h3>
<p>The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C.  </p>
<p><em>homepage</em>: <a href="https://projects.coin-or.org/ADOL-C">https://projects.coin-or.org/ADOL-C</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.0</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>2.7.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="advisor">Advisor<a class="headerlink" href="#advisor" title="Permanent link">&para;</a></h3>
<p>Vectorization Optimization and Thread Prototyping  - Vectorize &amp; thread code or performance dies  - Easy workflow + data + tips = faster code faster  - Prioritize, Prototype &amp; Predict performance gain  </p>
<p><em>homepage</em>: <a href="https://software.intel.com/intel-advisor-xe">https://software.intel.com/intel-advisor-xe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="afni">AFNI<a class="headerlink" href="#afni" title="Permanent link">&para;</a></h3>
<p>AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data -  a technique for mapping human brain activity.</p>
<p><em>homepage</em>: <a href="http://afni.nimh.nih.gov/">http://afni.nimh.nih.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>18.1.09</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>18.3.00</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>19.0.01</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20160329</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="agat">AGAT<a class="headerlink" href="#agat" title="Permanent link">&para;</a></h3>
<p>AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations  in any GTF/GFF format.</p>
<p><em>homepage</em>: <a href="https://agat.readthedocs.io/en/latest/">https://agat.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.2</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="agfusion">AGFusion<a class="headerlink" href="#agfusion" title="Permanent link">&para;</a></h3>
<p>AGFusion is a python package for annotating gene fusions  from the human or mouse genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/murphycj/AGFusion">https://github.com/murphycj/AGFusion</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="aiohttp">aiohttp<a class="headerlink" href="#aiohttp" title="Permanent link">&para;</a></h3>
<p>Asynchronous HTTP client/server framework for asyncio and Python.</p>
<p><em>homepage</em>: <a href="https://github.com/aio-libs/aiohttp">https://github.com/aio-libs/aiohttp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.8.3</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="aladin">ALADIN<a class="headerlink" href="#aladin" title="Permanent link">&para;</a></h3>
<p>ALADIN was entirely built on the notion of compatibility with its mother system, IFS/ARPEG.  The latter, a joint development between the European Centre for Medium-Range Weather Forecasts (ECMWF) and  Meteo-France, was only meant to consider global Numerical Weather Prediction applications; hence the idea,  for ALADIN, to complement the IFS/ARPEGE project with a limited area model (LAM) version, while keeping the  differences between the two softwares as small as possible.</p>
<p><em>homepage</em>: <a href="http://www.cnrm.meteo.fr/aladin/">http://www.cnrm.meteo.fr/aladin/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>36t1_op2bf1</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="albacore">Albacore<a class="headerlink" href="#albacore" title="Permanent link">&para;</a></h3>
<p>Albacore is a software project that provides an entry point to the Oxford Nanopore basecalling  algorithms.</p>
<p><em>homepage</em>: <a href="https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016">https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="albumentations">Albumentations<a class="headerlink" href="#albumentations" title="Permanent link">&para;</a></h3>
<p>Albumentations is a Python library for fast and flexible image augmentations</p>
<p><em>homepage</em>: <a href="https://albumentations.ai">https://albumentations.ai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="alevin-fry">alevin-fry<a class="headerlink" href="#alevin-fry" title="Permanent link">&para;</a></h3>
<p>alevin-fry is an efficient and flexible tool for processing single-cell sequencing data,  currently focused on single-cell transcriptomics and feature barcoding.</p>
<p><em>homepage</em>: <a href="https://github.com/COMBINE-lab/alevin-fry">https://github.com/COMBINE-lab/alevin-fry</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="alfa">ALFA<a class="headerlink" href="#alfa" title="Permanent link">&para;</a></h3>
<p>ALFA provides a global overview of features distribution composing NGS dataset(s). Given a set of  aligned reads (BAM files) and an annotation file (GTF format), the tool produces plots of the raw and normalized  distributions of those reads among genomic categories (stop codon, 5'-UTR, CDS, intergenic, etc.) and biotypes  (protein coding genes, miRNA, tRNA, etc.). Whatever the sequencing technique, whatever the organism.</p>
<p><em>homepage</em>: <a href="https://github.com/biocompibens/ALFA">https://github.com/biocompibens/ALFA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="alfred">Alfred<a class="headerlink" href="#alfred" title="Permanent link">&para;</a></h3>
<p>Alfred is an efficient and versatile command-line application  that computes multi-sample quality control metrics in a read-group aware manner.  Alfred supports read counting, feature annotation and haplotype-resolved consensus  computation using multiple sequence alignments.</p>
<p><em>homepage</em>: <a href="https://www.gear-genomics.com/alfred/">https://www.gear-genomics.com/alfred/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.6</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="allelecount">alleleCount<a class="headerlink" href="#allelecount" title="Permanent link">&para;</a></h3>
<p>The alleleCount package primarily exists to prevent code duplication between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl  code wraps the C implementation of allele counting code for BAM/CRAM processing. </p>
<p><em>homepage</em>: <a href="http://cancerit.github.io/alleleCount/">http://cancerit.github.io/alleleCount/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="allinea">Allinea<a class="headerlink" href="#allinea" title="Permanent link">&para;</a></h3>
<p>The Allinea environment is an essential toolkit for developers and computational scientists looking to get results faster.</p>
<p><em>homepage</em>: <a href="http://www.allinea.com">http://www.allinea.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1-32834-Redhat-5.7-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1-32834-Redhat-6.0-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.1-Ubuntu-14.04-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="allpaths-lg">ALLPATHS-LG<a class="headerlink" href="#allpaths-lg" title="Permanent link">&para;</a></h3>
<p>ALLPATHS-LG, the new short read genome assembler.</p>
<p><em>homepage</em>: <a href="http://www.broadinstitute.org/software/allpaths-lg/blog/">http://www.broadinstitute.org/software/allpaths-lg/blog/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>52488</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="almosthere">almosthere<a class="headerlink" href="#almosthere" title="Permanent link">&para;</a></h3>
<p>Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows,  Linux, and macOS. It is non-blocking as the progress update is done via a  dedicated, lightweight thread, as to not impair the performance of the calling program.</p>
<p><em>homepage</em>: <a href="https://github.com/horta/almosthere">https://github.com/horta/almosthere</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.0.10</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="alpha">Alpha<a class="headerlink" href="#alpha" title="Permanent link">&para;</a></h3>
<p>Alpha is a tool designed for detailed comparative study of bacteriophage genomes.</p>
<p><em>homepage</em>: <a href="https://www.lirmm.fr/~swenson/alpha/alpha.htm">https://www.lirmm.fr/~swenson/alpha/alpha.htm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200430</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="alphafold">AlphaFold<a class="headerlink" href="#alphafold" title="Permanent link">&para;</a></h3>
<p>AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known</p>
<p><em>homepage</em>: <a href="https://deepmind.com/research/case-studies/alphafold">https://deepmind.com/research/case-studies/alphafold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-TensorFlow-2.5.0</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="alps">ALPS<a class="headerlink" href="#alps" title="Permanent link">&para;</a></h3>
<p>The ALPS project (Algorithms and Libraries for Physics Simulations) is an open source effort  aiming at providing high-end simulation codes for strongly correlated quantum mechanical systems as well as   C++ libraries for simplifying the development of such code. </p>
<p><em>homepage</em>: <a href="http://alps.comp-phys.org/">http://alps.comp-phys.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.b4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="alsa-lib">alsa-lib<a class="headerlink" href="#alsa-lib" title="Permanent link">&para;</a></h3>
<p>The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality  to the Linux operating system.</p>
<p><em>homepage</em>: <a href="https://www.alsa-project.org">https://www.alsa-project.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.4</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="amapvox">AMAPVox<a class="headerlink" href="#amapvox" title="Permanent link">&para;</a></h3>
<p>LiDAR data voxelisation software</p>
<p><em>homepage</em>: <a href="https://amap-dev.cirad.fr/projects/amapvox">https://amap-dev.cirad.fr/projects/amapvox</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.4</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="amara">Amara<a class="headerlink" href="#amara" title="Permanent link">&para;</a></h3>
<p>Library for XML processing in Python, designed to balance the native idioms of Python  with the native character of XML.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/Amara">https://pypi.org/project/Amara</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="amask">amask<a class="headerlink" href="#amask" title="Permanent link">&para;</a></h3>
<p>amask is a set of tools to to determine the affinity of MPI processes and OpenMP threads  in a parallel environment.</p>
<p><em>homepage</em>: <a href="https://github.com/TACC/amask">https://github.com/TACC/amask</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20171106</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>20190404</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="amber">Amber<a class="headerlink" href="#amber" title="Permanent link">&para;</a></h3>
<p>Amber (originally Assisted Model Building with Energy Refinement) is software for performing  molecular dynamics and structure prediction.</p>
<p><em>homepage</em>: <a href="https://ambermd.org/amber.html">https://ambermd.org/amber.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>14</code></td>
<td><code>-AmberTools-15-patchlevel-13-13</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-16-patchlevel-5-14</code></td>
<td><code>iomkl/2016.07</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-16-patchlevel-5-14-CUDA</code></td>
<td><code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-16-patchlevel-5-14-serial</code></td>
<td><code>iomkl/2016.07</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-17-patchlevel-10-15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-17-patchlevel-10-15-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-AmberTools-17-patchlevel-8-12</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>18</code></td>
<td><code>-AmberTools-18-patchlevel-10-8</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>18</code></td>
<td><code>-AmberTools-19-patchlevel-12-17-Python-2.7.16</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>20.11</code></td>
<td><code>-AmberTools-20.15-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>20.11</code></td>
<td><code>-AmberTools-21.3</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>22.0</code></td>
<td><code>-AmberTools-22.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>22.0</code></td>
<td><code>-AmberTools-22.3-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="ambermini">AmberMini<a class="headerlink" href="#ambermini" title="Permanent link">&para;</a></h3>
<p>A stripped-down set of just antechamber, sqm, and tleap.</p>
<p><em>homepage</em>: <a href="https://github.com/choderalab/ambermini">https://github.com/choderalab/ambermini</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>16.16.0</code></td>
<td><code>intel/2017b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="ambertools">AmberTools<a class="headerlink" href="#ambertools" title="Permanent link">&para;</a></h3>
<p>AmberTools consists of several independently developed packages that work well by themselves,  and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,  with either explicit water or generalized Born solvent models.</p>
<p><em>homepage</em>: <a href="https://ambermd.org/">https://ambermd.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17</code></td>
<td></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>17</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>20</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>21</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>21.12</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>22.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="amd-libm">AMD-LibM<a class="headerlink" href="#amd-libm" title="Permanent link">&para;</a></h3>
<p>AMD LibM is a software library containing a collection of basic math functions optimized for x86-64 processor based machines.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-cpu-libraries/amd-math-library-libm/">https://developer.amd.com/amd-cpu-libraries/amd-math-library-libm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>3.6.0-4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="amd-rng">AMD-RNG<a class="headerlink" href="#amd-rng" title="Permanent link">&para;</a></h3>
<p>AMD Random Number Generator Library is a pseudorandom number generator library.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-cpu-libraries/rng-library/">https://developer.amd.com/amd-cpu-libraries/rng-library/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.2-4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="amd-securerng">AMD-SecureRNG<a class="headerlink" href="#amd-securerng" title="Permanent link">&para;</a></h3>
<p>The AMD Secure Random Number Generator (RNG) is a library that provides APIs to access the cryptographically secure random numbers generated by AMDs hardware-based random number generator implementation.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-cpu-libraries/rng-library/">https://developer.amd.com/amd-cpu-libraries/rng-library/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.2-4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="amd-uprof">AMD-uProf<a class="headerlink" href="#amd-uprof" title="Permanent link">&para;</a></h3>
<p>AMD uProf is a performance analysis tool for applications running on Windows, Linux &amp; FreeBSD  operating systems. It allows developers to better understand the runtime performance of their application and  to identify ways to improve its performance.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-uprof/">https://developer.amd.com/amd-uprof/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.502</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.671</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="amos">AMOS<a class="headerlink" href="#amos" title="Permanent link">&para;</a></h3>
<p>The AMOS consortium is committed to the development of open-source whole genome assembly software</p>
<p><em>homepage</em>: <a href="http://amos.sourceforge.net">http://amos.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td><code>foss/2018b</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="amphora2">AMPHORA2<a class="headerlink" href="#amphora2" title="Permanent link">&para;</a></h3>
<p>An Automated Phylogenomic Inference Pipeline for Bacterial and Archaeal Sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/wu-lab-uva/AMPHORA2">https://github.com/wu-lab-uva/AMPHORA2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190730</code></td>
<td><code>-Java-13-pthreads-avx2</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ampl-mp">AMPL-MP<a class="headerlink" href="#ampl-mp" title="Permanent link">&para;</a></h3>
<p>An open-source library for mathematical programming. </p>
<p><em>homepage</em>: <a href="https://github.com/ampl/mp">https://github.com/ampl/mp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="amplimap">amplimap<a class="headerlink" href="#amplimap" title="Permanent link">&para;</a></h3>
<p>amplimap is a command-line tool to automate the processing and analysis of data from targeted  next-generation sequencing (NGS) experiments with PCR-based amplicons or capture-based enrichment systems.</p>
<p><em>homepage</em>: <a href="https://amplimap.readthedocs.io/en/latest/">https://amplimap.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.16</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="amptk">AMPtk<a class="headerlink" href="#amptk" title="Permanent link">&para;</a></h3>
<p>AMPtk is a series of scripts to process NGS amplicon data using USEARCH and VSEARCH, it can also be used to process any NGS amplicon data and includes databases setup for analysis of fungal ITS, fungal LSU, bacterial 16S, and insect COI amplicons.</p>
<p><em>homepage</em>: <a href="https://amptk.readthedocs.io">https://amptk.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.4</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="amrplusplus">AmrPlusPlus<a class="headerlink" href="#amrplusplus" title="Permanent link">&para;</a></h3>
<p>AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. </p>
<p><em>homepage</em>: <a href="https://megares.meglab.org/amrplusplus/latest/html/v2">https://megares.meglab.org/amrplusplus/latest/html/v2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0-20200114</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ams">AMS<a class="headerlink" href="#ams" title="Permanent link">&para;</a></h3>
<p>The Amsterdam Modeling Suite (AMS) provides a comprehensive set of modules for computational chemistry and materials science, from quantum mechanics to fluid thermodynamics. </p>
<p><em>homepage</em>: <a href="https://www.scm.com/amsterdam-modeling-suite/">https://www.scm.com/amsterdam-modeling-suite/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.102</code></td>
<td><code>-intelmpi</code></td>
<td><code>iimpi/2020b</code></td>
</tr>
<tr>
<td><code>2022.102</code></td>
<td><code>-intelmpi</code></td>
<td><code>iimpi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="anaconda2">Anaconda2<a class="headerlink" href="#anaconda2" title="Permanent link">&para;</a></h3>
<p>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. </p>
<p><em>homepage</em>: <a href="https://www.anaconda.com">https://www.anaconda.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.03</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.10</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="anaconda3">Anaconda3<a class="headerlink" href="#anaconda3" title="Permanent link">&para;</a></h3>
<p>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. </p>
<p><em>homepage</em>: <a href="https://www.anaconda.com">https://www.anaconda.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.03</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.02</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.05</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.10</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="anci">aNCI<a class="headerlink" href="#anci" title="Permanent link">&para;</a></h3>
<p>Non-covalent interaction (NCI) for MD trajectories</p>
<p><em>homepage</em>: <a href="https://www.lct.jussieu.fr/pagesperso/contrera/nci-MD.html">https://www.lct.jussieu.fr/pagesperso/contrera/nci-MD.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="andi">andi<a class="headerlink" href="#andi" title="Permanent link">&para;</a></h3>
<p>This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/evolbioinf/andi/">https://github.com/evolbioinf/andi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="angel">ANGEL<a class="headerlink" href="#angel" title="Permanent link">&para;</a></h3>
<p>ANGEL: Robust Open Reading Frame prediction</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/ANGEL">https://github.com/PacificBiosciences/ANGEL</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="angsd">angsd<a class="headerlink" href="#angsd" title="Permanent link">&para;</a></h3>
<p>Program for analysing NGS data.</p>
<p><em>homepage</em>: <a href="http://www.popgen.dk/angsd">http://www.popgen.dk/angsd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.910</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.921</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.925</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.933</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.935</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.940</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="anicalculator">ANIcalculator<a class="headerlink" href="#anicalculator" title="Permanent link">&para;</a></h3>
<p>This tool will calculate the bidirectional average nucleotide identity (gANI) and  Alignment Fraction (AF) between two genomes. Required input is the full path to the fna file  (nucleotide sequence of genes in fasta format) of each query genome. Either the rRNA and tRNA genes  can be excluded, or provided in a list with the -ignoreList option. This is necessary as the presence of tRNA and/or rRNA genes in the fna will artificially inflate the ANI.</p>
<p><em>homepage</em>: <a href="https://ani.jgi.doe.gov/html/home.php">https://ani.jgi.doe.gov/html/home.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="annif">Annif<a class="headerlink" href="#annif" title="Permanent link">&para;</a></h3>
<p>Annif is a multi-algorithm automated subject indexing tool for libraries, archives and museums.</p>
<p><em>homepage</em>: <a href="https://github.com/NatLibFi/Annif">https://github.com/NatLibFi/Annif</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.40.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="annovar">annovar<a class="headerlink" href="#annovar" title="Permanent link">&para;</a></h3>
<p>ANNOVAR is an efficient software tool to utilize update-to-date information  to functionally annotate genetic variants detected from diverse genomes (including human  genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others).</p>
<p><em>homepage</em>: <a href="http://annovar.openbioinformatics.org/en/latest/">http://annovar.openbioinformatics.org/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016Feb01</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20191024</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>20200607</code></td>
<td><code>-Perl-5.34.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ansys">ANSYS<a class="headerlink" href="#ansys" title="Permanent link">&para;</a></h3>
<p>ANSYS simulation software enables organizations to confidently predict      how their products will operate in the real world. We believe that every product is      a promise of something greater. </p>
<p><em>homepage</em>: <a href="https://www.ansys.com">https://www.ansys.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022R2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ansys_cfd">ANSYS_CFD<a class="headerlink" href="#ansys_cfd" title="Permanent link">&para;</a></h3>
<p>ANSYS computational fluid dynamics (CFD) simulation software allows you to predict, with confidence,  the impact of fluid flows on your product throughout design and manufacturing as well as during end use.  ANSYS renowned CFD analysis tools include the widely used and well-validated ANSYS Fluent and ANSYS CFX.</p>
<p><em>homepage</em>: <a href="http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics">http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>16.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ant">ant<a class="headerlink" href="#ant" title="Permanent link">&para;</a></h3>
<p>Apache Ant is a Java library and command-line tool whose mission is to drive processes described in  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of  Java applications.</p>
<p><em>homepage</em>: <a href="http://ant.apache.org/">http://ant.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Java-1.7.0_21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-Java-1.7.0_15</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-Java-1.7.0_21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Java-1.7.0_60</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Java-1.7.0_79</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.7.0_75</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.7.0_79</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.8.0_72</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.6</code></td>
<td><code>-Java-1.8.0_77</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.7</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Java-1.8.0_121</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.5</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.6</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.7</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.8</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.9</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.11</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.11</code></td>
<td><code>-Java-13</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.12</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.12</code></td>
<td><code>-Java-17</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="antismash">antiSMASH<a class="headerlink" href="#antismash" title="Permanent link">&para;</a></h3>
<p>antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary  metabolite biosynthesis gene clusters in bacterial and fungal genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/antismash/antismash">https://github.com/antismash/antismash</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="antlr">ANTLR<a class="headerlink" href="#antlr" title="Permanent link">&para;</a></h3>
<p>ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.</p>
<p><em>homepage</em>: <a href="http://www.antlr2.org/">http://www.antlr2.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.7</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2017b</code>, <code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="ants">ANTs<a class="headerlink" href="#ants" title="Permanent link">&para;</a></h3>
<p>ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,  interpreting and visualizing multidimensional data.</p>
<p><em>homepage</em>: <a href="https://stnava.github.io/ANTs/">https://stnava.github.io/ANTs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.3.5</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="anvio">anvio<a class="headerlink" href="#anvio" title="Permanent link">&para;</a></h3>
<p>An analysis and visualization platform for 'omics data.</p>
<p><em>homepage</em>: <a href="https://merenlab.org/software/anvio/">https://merenlab.org/software/anvio/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="any2fasta">any2fasta<a class="headerlink" href="#any2fasta" title="Permanent link">&para;</a></h3>
<p>Convert various sequence formats to FASTA</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/any2fasta">https://github.com/tseemann/any2fasta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.4.2</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="aocc">AOCC<a class="headerlink" href="#aocc" title="Permanent link">&para;</a></h3>
<p>AMD Optimized C/C++ &amp; Fortran compilers (AOCC) based on LLVM 13.0</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-aocc/">https://developer.amd.com/amd-aocc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="aomp">AOMP<a class="headerlink" href="#aomp" title="Permanent link">&para;</a></h3>
<p>AMD fork of LLVM, setup for OpenMP offloading to Accelerators</p>
<p><em>homepage</em>: <a href="https://github.com/ROCm-Developer-Tools/aomp">https://github.com/ROCm-Developer-Tools/aomp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>13.0-2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>gcccuda/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="apbs">APBS<a class="headerlink" href="#apbs" title="Permanent link">&para;</a></h3>
<p>APBS is a software package for modeling biomolecular solvation   through solution of the Poisson-Boltzmann equation (PBE), one of the most popular   continuum models for describing electrostatic interactions between molecular solutes   in salty, aqueous media. </p>
<p><em>homepage</em>: <a href="http://www.poissonboltzmann.org/apbs">http://www.poissonboltzmann.org/apbs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>-linux-static-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="apex">apex<a class="headerlink" href="#apex" title="Permanent link">&para;</a></h3>
<p>A PyTorch Extension: Tools for easy mixed precision and distributed training in Pytorch</p>
<p><em>homepage</em>: <a href="https://github.com/nvidia/apex">https://github.com/nvidia/apex</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200325</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>20210420</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="apr">APR<a class="headerlink" href="#apr" title="Permanent link">&para;</a></h3>
<p>Apache Portable Runtime (APR) libraries.</p>
<p><em>homepage</em>: <a href="https://apr.apache.org/">https://apr.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="apr-util">APR-util<a class="headerlink" href="#apr-util" title="Permanent link">&para;</a></h3>
<p>Apache Portable Runtime (APR) util libraries.</p>
<p><em>homepage</em>: <a href="http://apr.apache.org/">http://apr.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code>, <code>iomkl/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="aptasuite">AptaSUITE<a class="headerlink" href="#aptasuite" title="Permanent link">&para;</a></h3>
<p>A full-featured bioinformatics software collection for the  comprehensive analysis of aptamers in HT-SELEX experiments </p>
<p><em>homepage</em>: <a href="https://drivenbyentropy.github.io/">https://drivenbyentropy.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.4</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="aragorn">ARAGORN<a class="headerlink" href="#aragorn" title="Permanent link">&para;</a></h3>
<p>a program to detect tRNA genes and tmRNA genes in nucleotide sequences</p>
<p><em>homepage</em>: <a href="http://www.ansikte.se/ARAGORN/">http://www.ansikte.se/ARAGORN/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.38</code></td>
<td><code>foss/2016b</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.2.41</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="arb">Arb<a class="headerlink" href="#arb" title="Permanent link">&para;</a></h3>
<p>Arb is a C library for arbitrary-precision interval arithmetic.  It has full support for both real and complex numbers. The library is thread-safe,  portable, and extensively tested.</p>
<p><em>homepage</em>: <a href="https://arblib.org/">https://arblib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.16.0</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.17.0</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.19.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.22.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="arcade-learning-environment">Arcade-Learning-Environment<a class="headerlink" href="#arcade-learning-environment" title="Permanent link">&para;</a></h3>
<p>The Arcade Learning Environment (ALE) is a simple framework that allows researchers and hobbyists to develop AI agents for Atari 2600 games. It is built on top of the Atari 2600 emulator Stella and separates the details of emulation from agent design. This video depicts over 50 games currently supported in the ALE.</p>
<p><em>homepage</em>: <a href="https://github.com/mgbellemare/Arcade-Learning-Environment">https://github.com/mgbellemare/Arcade-Learning-Environment</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="arcashla">arcasHLA<a class="headerlink" href="#arcashla" title="Permanent link">&para;</a></h3>
<p>arcasHLA performs high resolution genotyping for HLA class I and class II genes from RNA sequencing, supporting both paired and single-end samples.</p>
<p><em>homepage</em>: <a href="https://github.com/RabadanLab/arcasHLA">https://github.com/RabadanLab/arcasHLA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="arch">ARCH<a class="headerlink" href="#arch" title="Permanent link">&para;</a></h3>
<p>Autoregressive Conditional Heteroskedasticity (ARCH) and other tools for financial econometrics,  written in Python (with Cython and/or Numba used to improve performance).</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/arch">https://pypi.org/project/arch</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="archive-zip">Archive-Zip<a class="headerlink" href="#archive-zip" title="Permanent link">&para;</a></h3>
<p>Provide an interface to ZIP archive files.</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Archive::Zip">https://metacpan.org/pod/Archive::Zip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.68</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="archr">ArchR<a class="headerlink" href="#archr" title="Permanent link">&para;</a></h3>
<p>ArchR is a full-featured R package for processing and analyzing single-cell ATAC-seq data.</p>
<p><em>homepage</em>: <a href="https://www.archrproject.com">https://www.archrproject.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="archspec">archspec<a class="headerlink" href="#archspec" title="Permanent link">&para;</a></h3>
<p>A library for detecting, labeling, and reasoning about microarchitectures</p>
<p><em>homepage</em>: <a href="https://github.com/archspec/archspec">https://github.com/archspec/archspec</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.1.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.1.3</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.1.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>0.2.0</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="argos">ARGoS<a class="headerlink" href="#argos" title="Permanent link">&para;</a></h3>
<p>A parallel, multi-engine simulator for heterogeneous swarm robotics</p>
<p><em>homepage</em>: <a href="http://www.argos-sim.info">http://www.argos-sim.info</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0-beta53</code></td>
<td><code>-Lua-5.2.4</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.0.0-beta59</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="argtable">argtable<a class="headerlink" href="#argtable" title="Permanent link">&para;</a></h3>
<p>Argtable is an ANSI C library for parsing GNU style  command line options with a minimum of fuss. </p>
<p><em>homepage</em>: <a href="http://argtable.sourceforge.net/">http://argtable.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.13</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="aria2">aria2<a class="headerlink" href="#aria2" title="Permanent link">&para;</a></h3>
<p>aria2 is a lightweight multi-protocol &amp; multi-source command-line download utility.</p>
<p><em>homepage</em>: <a href="https://aria2.github.io">https://aria2.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.35.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.36.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="arlequin">Arlequin<a class="headerlink" href="#arlequin" title="Permanent link">&para;</a></h3>
<p>Arlequin: An Integrated Software for Population Genetics Data Analysis</p>
<p><em>homepage</em>: <a href="http://cmpg.unibe.ch/software/arlequin35/Arlequin35.html">http://cmpg.unibe.ch/software/arlequin35/Arlequin35.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.2.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="armadillo">Armadillo<a class="headerlink" href="#armadillo" title="Permanent link">&para;</a></h3>
<p>Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards  a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,  as well as a subset of trigonometric and statistics functions.</p>
<p><em>homepage</em>: <a href="https://arma.sourceforge.net/">https://arma.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.600.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>7.800.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.950.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.300.1</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.400.0</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>9.600.5</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>9.700.2</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>9.880.1</code></td>
<td></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>9.900.1</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code></td>
</tr>
<tr>
<td><code>10.5.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>10.7.5</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>11.4.3</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="arosics">arosics<a class="headerlink" href="#arosics" title="Permanent link">&para;</a></h3>
<p>AROSICS is a python package to perform automatic subpixel co-registration of two satellite image datasets based on an image matching approach working in the frequency domain, combined with a multistage workflow for effective detection of false-positives.</p>
<p><em>homepage</em>: <a href="https://github.com/GFZ/arosics">https://github.com/GFZ/arosics</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.6</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="arpack">ARPACK++<a class="headerlink" href="#arpack" title="Permanent link">&para;</a></h3>
<p>Arpackpp is a C++ interface to the ARPACK Fortran package,  which implements the implicit restarted Arnoldi method for  iteratively solving large-scale sparse eigenvalue problems.</p>
<p><em>homepage</em>: <a href="https://github.com/m-reuter/arpackpp">https://github.com/m-reuter/arpackpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.03.26</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="arpack-ng">arpack-ng<a class="headerlink" href="#arpack-ng" title="Permanent link">&para;</a></h3>
<p>ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.</p>
<p><em>homepage</em>: <a href="https://github.com/opencollab/arpack-ng">https://github.com/opencollab/arpack-ng</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td><code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="arrayfire">ArrayFire<a class="headerlink" href="#arrayfire" title="Permanent link">&para;</a></h3>
<p>ArrayFire is a general-purpose library that simplifies the process of  developing software that targets parallel and massively-parallel architectures  including CPUs, GPUs, and other hardware acceleration devices. </p>
<p><em>homepage</em>: <a href="https://arrayfire.com/">https://arrayfire.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.4</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.6.4</code></td>
<td><code>-CUDA-9.2.88</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="arriba">Arriba<a class="headerlink" href="#arriba" title="Permanent link">&para;</a></h3>
<p>Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.  It was developed for the use in a clinical research setting. Therefore, short runtimes and high  sensitivity were important design criteria.</p>
<p><em>homepage</em>: <a href="https://github.com/suhrig/arriba">https://github.com/suhrig/arriba</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="arrow">Arrow<a class="headerlink" href="#arrow" title="Permanent link">&para;</a></h3>
<p>Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform  for in-memory data.</p>
<p><em>homepage</em>: <a href="https://arrow.apache.org">https://arrow.apache.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td></td>
<td><code>foss/2022.05</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="arrow-r">arrow-R<a class="headerlink" href="#arrow-r" title="Permanent link">&para;</a></h3>
<p>R interface to the Apache Arrow C++ library</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/arrow">https://cran.r-project.org/web/packages/arrow</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.1</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>6.0.0.2</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>6.0.0.2</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>6.0.0.2</code></td>
<td><code>-R-4.2.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="art">ART<a class="headerlink" href="#art" title="Permanent link">&para;</a></h3>
<p>ART is a set of simulation tools to generate synthetic next-generation sequencing reads</p>
<p><em>homepage</em>: <a href="http://www.niehs.nih.gov/research/resources/software/biostatistics/art/">http://www.niehs.nih.gov/research/resources/software/biostatistics/art/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.06.05</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="artic-ncov2019">artic-ncov2019<a class="headerlink" href="#artic-ncov2019" title="Permanent link">&para;</a></h3>
<p>Initial implementation of an ARTIC bioinformatics platform  for nanopore sequencing of nCoV2019 novel coronavirus.</p>
<p><em>homepage</em>: <a href="https://github.com/artic-network/artic-ncov2019">https://github.com/artic-network/artic-ncov2019</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.04.13</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2021.06.24</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="arts">ARTS<a class="headerlink" href="#arts" title="Permanent link">&para;</a></h3>
<p>ARTS is a radiative transfer model for the millimeter and sub-millimeter  spectral range. There are a number of models mostly developed explicitly  for the different sensors.</p>
<p><em>homepage</em>: <a href="http://www.radiativetransfer.org/">http://www.radiativetransfer.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.64</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="arviz">ArviZ<a class="headerlink" href="#arviz" title="Permanent link">&para;</a></h3>
<p>Exploratory analysis of Bayesian models with Python</p>
<p><em>homepage</em>: <a href="https://github.com/arviz-devs/arviz">https://github.com/arviz-devs/arviz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.11.4</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="arwen">ARWEN<a class="headerlink" href="#arwen" title="Permanent link">&para;</a></h3>
<p>ARWEN, tRNA detection in metazoan mitochondrial sequences</p>
<p><em>homepage</em>: <a href="http://mbio-serv2.mbioekol.lu.se/ARWEN">http://mbio-serv2.mbioekol.lu.se/ARWEN</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="asap">ASAP<a class="headerlink" href="#asap" title="Permanent link">&para;</a></h3>
<p>ASAP focuses on fast and fluid image viewing with an easy-to-use interface  for making annotations. It consists of two main components: an IO library for reading and writing  multi-resolution images and a viewer component for visualizing such images.</p>
<p><em>homepage</em>: <a href="https://computationalpathologygroup.github.io/ASAP/">https://computationalpathologygroup.github.io/ASAP/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="asap3">ASAP3<a class="headerlink" href="#asap3" title="Permanent link">&para;</a></h3>
<p>ASAP is a calculator for doing large-scale classical molecular dynamics within the Campos Atomic Simulation Environment (ASE).</p>
<p><em>homepage</em>: <a href="https://wiki.fysik.dtu.dk/asap/">https://wiki.fysik.dtu.dk/asap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.10.7</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.10.7</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.10.10</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.11.10</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.12.2</code></td>
<td><code>-ASE-3.21.1</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.12.7</code></td>
<td><code>-ASE-3.21.1</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ase">ASE<a class="headerlink" href="#ase" title="Permanent link">&para;</a></h3>
<p>ASE is a python package providing an open source Atomic Simulation Environment  in the Python scripting language.</p>
<p><em>homepage</em>: <a href="https://wiki.fysik.dtu.dk/ase/">https://wiki.fysik.dtu.dk/ase/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.1.4567</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.10.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>3.11.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.16.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>iomkl/2018.02</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.16.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.17.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.17.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.18.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.18.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.19.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.19.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.19.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.20.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.20.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.20.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.21.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.21.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.22.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>3.22.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ashs">ASHS<a class="headerlink" href="#ashs" title="Permanent link">&para;</a></h3>
<p>Automatic Segmentation of Hippocampal Subfields (ASHS) </p>
<p><em>homepage</em>: <a href="https://sites.google.com/site/hipposubfields/home">https://sites.google.com/site/hipposubfields/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>rev103_20140612</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="aspera-cli">Aspera-CLI<a class="headerlink" href="#aspera-cli" title="Permanent link">&para;</a></h3>
<p>IBM Aspera Command-Line Interface (the Aspera CLI) is a collection of Aspera tools for performing high-speed, secure data transfers from the command line. The Aspera CLI is for users and organizations who want to automate their transfer workflows.</p>
<p><em>homepage</em>: <a href="https://asperasoft.com">https://asperasoft.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.2</code></td>
<td><code>.354.010c3b8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.0</code></td>
<td><code>.1326.6985b21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td><code>.1467.159c5b1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="aspera-connect">Aspera-Connect<a class="headerlink" href="#aspera-connect" title="Permanent link">&para;</a></h3>
<p>Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and  downloads with an Aspera transfer server.</p>
<p><em>homepage</em>: <a href="http://downloads.asperasoft.com/connect2/">http://downloads.asperasoft.com/connect2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="assimp">assimp<a class="headerlink" href="#assimp" title="Permanent link">&para;</a></h3>
<p>Open Asset Import Library (assimp) is a library to import and export various  3d-model-formats including scene-post-processing to generate missing render data. </p>
<p><em>homepage</em>: <a href="https://github.com/assimp/assimp">https://github.com/assimp/assimp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="assimulo">Assimulo<a class="headerlink" href="#assimulo" title="Permanent link">&para;</a></h3>
<p>Assimulo is a simulation package for solving ordinary differential equations.</p>
<p><em>homepage</em>: <a href="https://jmodelica.org/assimulo/">https://jmodelica.org/assimulo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="astrid">ASTRID<a class="headerlink" href="#astrid" title="Permanent link">&para;</a></h3>
<p>ASTRID-2 is a method for estimating species trees from gene trees.</p>
<p><em>homepage</em>: <a href="https://github.com/pranjalv123/ASTRID">https://github.com/pranjalv123/ASTRID</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="astro-tulips">astro-tulips<a class="headerlink" href="#astro-tulips" title="Permanent link">&para;</a></h3>
<p>tulips creates diagrams of the structure and evolution of stars. It creates plots and movies based on output from the MESA stellar evolution code</p>
<p><em>homepage</em>: <a href="https://astro-tulips.readthedocs.io/en/latest/installation.html">https://astro-tulips.readthedocs.io/en/latest/installation.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="astropy">astropy<a class="headerlink" href="#astropy" title="Permanent link">&para;</a></h3>
<p>The Astropy Project is a community effort to develop a common  core package for Astronomy in Python and foster an ecosystem of interoperable  astronomy packages.  The Astropy community is committed to supporting diversity and inclusion.</p>
<p><em>homepage</em>: <a href="https://www.astropy.org/">https://www.astropy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.12</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.0.14</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>5.0.4</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.1.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="at-spi2-atk">at-spi2-atk<a class="headerlink" href="#at-spi2-atk" title="Permanent link">&para;</a></h3>
<p>AT-SPI 2 toolkit bridge</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Accessibility">https://wiki.gnome.org/Accessibility</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.26.3</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.32.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.34.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.34.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.38.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="at-spi2-core">at-spi2-core<a class="headerlink" href="#at-spi2-core" title="Permanent link">&para;</a></h3>
<p>Assistive Technology Service Provider Interface. </p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Accessibility">https://wiki.gnome.org/Accessibility</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.26.3</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.32.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.34.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.36.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.38.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.40.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.40.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.44.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.46.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="atk">ATK<a class="headerlink" href="#atk" title="Permanent link">&para;</a></h3>
<p>ATK provides the set of accessibility interfaces that are implemented by other  toolkits and applications. Using the ATK interfaces, accessibility tools have  full access to view and control running applications. </p>
<p><em>homepage</em>: <a href="https://developer.gnome.org/atk/">https://developer.gnome.org/atk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.18.0</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.22.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.26.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.26.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.27.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.28.1</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.32.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.34.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.36.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.38.0</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="atlas">ATLAS<a class="headerlink" href="#atlas" title="Permanent link">&para;</a></h3>
<p>ATLAS (Automatically Tuned Linear Algebra Software) is the application of  the AEOS (Automated Empirical Optimization of Software) paradigm, with the present emphasis   on the Basic Linear Algebra Subprograms (BLAS), a widely used, performance-critical, linear   algebra kernel library.</p>
<p><em>homepage</em>: <a href="http://math-atlas.sourceforge.net">http://math-atlas.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.10.2</code></td>
<td><code>-LAPACK-3.6.1</code></td>
<td><code>GCC/5.4.0-2.26</code></td>
</tr>
</tbody>
</table>
<h3 id="atomate">atomate<a class="headerlink" href="#atomate" title="Permanent link">&para;</a></h3>
<p>atomate has implementations of FireWorks workflows for Materials Science.</p>
<p><em>homepage</em>: <a href="https://pythonhosted.org/atomate/">https://pythonhosted.org/atomate/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="atompaw">AtomPAW<a class="headerlink" href="#atompaw" title="Permanent link">&para;</a></h3>
<p>AtomPAW is a Projector-Augmented Wave Dataset Generator that  can be used both as a standalone program and a library.</p>
<p><em>homepage</em>: <a href="http://users.wfu.edu/natalie/papers/pwpaw/man.html">http://users.wfu.edu/natalie/papers/pwpaw/man.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.0.5</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.1.0.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="atools">atools<a class="headerlink" href="#atools" title="Permanent link">&para;</a></h3>
<p>Tools to make using job arrays a lot more convenient.</p>
<p><em>homepage</em>: <a href="https://github.com/gjbex/atools">https://github.com/gjbex/atools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4.6</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.4.8</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="atropos">atropos<a class="headerlink" href="#atropos" title="Permanent link">&para;</a></h3>
<p>Atropos is tool for specific, sensitive, and speedy trimming of NGS reads. </p>
<p><em>homepage</em>: <a href="https://atropos.readthedocs.io">https://atropos.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.21</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="atsas">ATSAS<a class="headerlink" href="#atsas" title="Permanent link">&para;</a></h3>
<p>ATSAS is a program suite for small-angle scattering data analysis from biological macromolecules.</p>
<p><em>homepage</em>: <a href="http://www.embl-hamburg.de/ExternalInfo/Research/Sax/software.html">http://www.embl-hamburg.de/ExternalInfo/Research/Sax/software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.1-1</code></td>
<td><code>.el6.x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.1-1</code></td>
<td><code>.sl5.x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.7.1-1</code></td>
<td><code>.el7.x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="attr">attr<a class="headerlink" href="#attr" title="Permanent link">&para;</a></h3>
<p>Commands for Manipulating Filesystem Extended Attributes</p>
<p><em>homepage</em>: <a href="https://savannah.nongnu.org/projects/attr">https://savannah.nongnu.org/projects/attr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.47</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.4.48</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="attrdict">attrdict<a class="headerlink" href="#attrdict" title="Permanent link">&para;</a></h3>
<p>AttrDict is a Python library that provides mapping objects that allow their elements  to be accessed both as keys and as attributes.</p>
<p><em>homepage</em>: <a href="https://github.com/bcj/AttrDict">https://github.com/bcj/AttrDict</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="attrdict3">attrdict3<a class="headerlink" href="#attrdict3" title="Permanent link">&para;</a></h3>
<p>AttrDict is a Python library that provides mapping objects that allow their elements  to be accessed both as keys and as attributes.</p>
<p><em>homepage</em>: <a href="https://github.com/pirofti/AttrDict3">https://github.com/pirofti/AttrDict3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="augur">augur<a class="headerlink" href="#augur" title="Permanent link">&para;</a></h3>
<p>Pipeline components for real-time phylodynamic analysis</p>
<p><em>homepage</em>: <a href="https://github.com/nextstrain/augur">https://github.com/nextstrain/augur</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="augustus">AUGUSTUS<a class="headerlink" href="#augustus" title="Permanent link">&para;</a></h3>
<p>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences</p>
<p><em>homepage</em>: <a href="https://bioinf.uni-greifswald.de/augustus/">https://bioinf.uni-greifswald.de/augustus/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.3</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="austin">Austin<a class="headerlink" href="#austin" title="Permanent link">&para;</a></h3>
<p>Austin is a Python frame stack sampler for CPython written in pure C.</p>
<p><em>homepage</em>: <a href="https://github.com/P403n1x87/austin">https://github.com/P403n1x87/austin</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="auto-07p">AUTO-07p<a class="headerlink" href="#auto-07p" title="Permanent link">&para;</a></h3>
<p>AUTO is a publicly available software for continuation and bifurcation problems in ordinary differential equations originally written in 1980 and widely used in the dynamical systems community.</p>
<p><em>homepage</em>: <a href="https://github.com/auto-07p">https://github.com/auto-07p</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.3</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="autoconf">Autoconf<a class="headerlink" href="#autoconf" title="Permanent link">&para;</a></h3>
<p>Autoconf is an extensible package of M4 macros that produce shell scripts  to automatically configure software source code packages. These scripts can adapt the  packages to many kinds of UNIX-like systems without manual user intervention. Autoconf  creates a configuration script for a package from a template file that lists the  operating system features that the package can use, in the form of M4 macro calls.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/autoconf/">https://www.gnu.org/software/autoconf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.68</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.69</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>GNU/5.1.0-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.71</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="autoconf-archive">Autoconf-archive<a class="headerlink" href="#autoconf-archive" title="Permanent link">&para;</a></h3>
<p>The GNU Autoconf Archive is a collection of more than 500 macros for GNU Autoconf that have been contributed as free software by friendly supporters of the cause from all over the Internet. Every single one of those macros can be re-used without imposing any restrictions whatsoever on the licensing of the generated configure script. In particular, it is possible to use all those macros in configure scripts that are meant for non-free software. This policy is unusual for a Free Software Foundation project. The FSF firmly believes that software ought to be free, and software licenses like the GPL are specifically designed to ensure that derivative work based on free software must be free as well. In case of Autoconf, however, an exception has been made, because Autoconf is at such a pivotal position in the software development tool chain that the benefits from having this tool available as widely as possible outweigh the disadvantage that some authors may choose to use it, too, for proprietary software. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/autoconf-archive">https://www.gnu.org/software/autoconf-archive</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.01.06</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2021.02.19</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="autodock">AutoDock<a class="headerlink" href="#autodock" title="Permanent link">&para;</a></h3>
<p>AutoDock is a suite of automated docking tools. It is designed to  predict how small molecules, such as substrates or drug candidates, bind to  a receptor of known 3D structure.</p>
<p><em>homepage</em>: <a href="http://autodock.scripps.edu/">http://autodock.scripps.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.5.1</code></td>
<td><code>GCC/5.2.0</code></td>
</tr>
<tr>
<td><code>4.2.6</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="autodock-gpu">AutoDock-GPU<a class="headerlink" href="#autodock-gpu" title="Permanent link">&para;</a></h3>
<p>OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure.</p>
<p><em>homepage</em>: <a href="https://github.com/ccsb-scripps/AutoDock-GPU">https://github.com/ccsb-scripps/AutoDock-GPU</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="autodock-vina">AutoDock-Vina<a class="headerlink" href="#autodock-vina" title="Permanent link">&para;</a></h3>
<p>AutoDock Vina is an open-source program for doing molecular docking.</p>
<p><em>homepage</em>: <a href="https://vina.scripps.edu/">https://vina.scripps.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>-linux_x86</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="autogenes">AutoGeneS<a class="headerlink" href="#autogenes" title="Permanent link">&para;</a></h3>
<p>AutoGeneS automatically extracts informative genes and reveals the cellular heterogeneity of bulk RNA samples.</p>
<p><em>homepage</em>: <a href="https://github.com/theislab/AutoGeneS">https://github.com/theislab/AutoGeneS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="autogrid">AutoGrid<a class="headerlink" href="#autogrid" title="Permanent link">&para;</a></h3>
<p>AutoDock is a suite of automated docking tools. It is designed to   predict how small molecules, such as substrates or drug candidates, bind to   a receptor of known 3D structure.</p>
<p><em>homepage</em>: <a href="http://autodock.scripps.edu/">http://autodock.scripps.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.5.1</code></td>
<td><code>GCC/5.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="automake">Automake<a class="headerlink" href="#automake" title="Permanent link">&para;</a></h3>
<p>Automake: GNU Standards-compliant Makefile generator</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/automake/automake.html">https://www.gnu.org/software/automake/automake.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.3</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>GCC/4.8.2</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.14.1</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.15</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>GNU/5.1.0-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.16.3</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.16.4</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.16.5</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="automap">AutoMap<a class="headerlink" href="#automap" title="Permanent link">&para;</a></h3>
<p>Tool to find regions of homozygosity (ROHs) from sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/mquinodo/AutoMap">https://github.com/mquinodo/AutoMap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-20200324</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="autopep8">autopep8<a class="headerlink" href="#autopep8" title="Permanent link">&para;</a></h3>
<p>A tool that automatically formats Python code to conform to the PEP 8 style guide.</p>
<p><em>homepage</em>: <a href="https://github.com/hhatto/autopep8">https://github.com/hhatto/autopep8</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="autotools">Autotools<a class="headerlink" href="#autotools" title="Permanent link">&para;</a></h3>
<p>This bundle collect the standard GNU build tools: Autoconf, Automake  and libtool </p>
<p><em>homepage</em>: <a href="https://autotools.io">https://autotools.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20150119</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>20150215</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>GNU/5.1.0-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>20170619</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code></td>
</tr>
<tr>
<td><code>20180311</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20200321</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20210128</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>20210726</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>20220317</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="avogadro2">Avogadro2<a class="headerlink" href="#avogadro2" title="Permanent link">&para;</a></h3>
<p>Avogadro is an advanced molecule editor and visualizer designed for cross-platform      use in computational chemistry, molecular modeling, bioinformatics, materials science,      and related areas. </p>
<p><em>homepage</em>: <a href="https://two.avogadro.cc/index.html">https://two.avogadro.cc/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.97.0</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="awscli">awscli<a class="headerlink" href="#awscli" title="Permanent link">&para;</a></h3>
<p>Universal Command Line Environment for AWS</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/awscli">https://pypi.python.org/pypi/awscli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.11.56</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.16.290</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.17.7</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.18.89</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.55</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="b">B<a class="headerlink" href="#b" title="Permanent link">&para;</a></h2>
<p><a href="#ba3-snps-autotune">BA3-SNPS-autotune</a> - <a href="#babelstream">BabelStream</a> - <a href="#babl">babl</a> - <a href="#bader">Bader</a> - <a href="#bagel">BAGEL</a> - <a href="#bali-phy">BAli-Phy</a> - <a href="#bam-readcount">bam-readcount</a> - <a href="#bambi">Bambi</a> - <a href="#bamfilters">bamFilters</a> - <a href="#bamsurgeon">BAMSurgeon</a> - <a href="#bamtofastq">bamtofastq</a> - <a href="#bamtools">BamTools</a> - <a href="#bamutil">BamUtil</a> - <a href="#bandage">Bandage</a> - <a href="#barrnap">barrnap</a> - <a href="#basemap">basemap</a> - <a href="#bash">Bash</a> - <a href="#bashplotlib">bashplotlib</a> - <a href="#bat">bat</a> - <a href="#batmeth2">BatMeth2</a> - <a href="#bayesass">BayesAss</a> - <a href="#bayesass3-snps">BayesAss3-SNPs</a> - <a href="#bayescan">BayeScan</a> - <a href="#bayescenv">BayeScEnv</a> - <a href="#bayestraits">BayesTraits</a> - <a href="#bazel">Bazel</a> - <a href="#bbcp">bbcp</a> - <a href="#bbftp">bbFTP</a> - <a href="#bbmap">BBMap</a> - <a href="#bc">bc</a> - <a href="#bcalm">BCALM</a> - <a href="#bcbio-gff">bcbio-gff</a> - <a href="#bcel">BCEL</a> - <a href="#bcftools">BCFtools</a> - <a href="#bcgtree">bcgTree</a> - <a href="#bcl-convert">bcl-convert</a> - <a href="#bcl2fastq2">bcl2fastq2</a> - <a href="#bcolz">bcolz</a> - <a href="#bdbag">BDBag</a> - <a href="#beagle">Beagle</a> - <a href="#beagle-lib">beagle-lib</a> - <a href="#beast">Beast</a> - <a href="#beautifulsoup">BeautifulSoup</a> - <a href="#bedops">BEDOPS</a> - <a href="#bedtools">BEDTools</a> - <a href="#beef">BEEF</a> - <a href="#behave">behave</a> - <a href="#berkeleygw">BerkeleyGW</a> - <a href="#bfast">BFAST</a> - <a href="#bfc">BFC</a> - <a href="#bgen">bgen</a> - <a href="#bgen-reader">bgen-reader</a> - <a href="#bibtexparser">bibtexparser</a> - <a href="#big-scape">BiG-SCAPE</a> - <a href="#bigdft">BigDFT</a> - <a href="#binsanity">BinSanity</a> - <a href="#binutils">binutils</a> - <a href="#bio-db-hts">Bio-DB-HTS</a> - <a href="#bio-eutilities">Bio-EUtilities</a> - <a href="#bio-samtools">Bio-SamTools</a> - <a href="#bio-searchio-hmmer">Bio-SearchIO-hmmer</a> - <a href="#bioawk">bioawk</a> - <a href="#biobambam2">biobambam2</a> - <a href="#biogeme">biogeme</a> - <a href="#biom-format">biom-format</a> - <a href="#biomart-perl">biomart-perl</a> - <a href="#bioperl">BioPerl</a> - <a href="#biopp">BioPP</a> - <a href="#biopython">Biopython</a> - <a href="#bioservices">BioServices</a> - <a href="#birdnet">BirdNET</a> - <a href="#biscuit">biscuit</a> - <a href="#bismark">Bismark</a> - <a href="#bison">Bison</a> - <a href="#bitarray">bitarray</a> - <a href="#blacs">BLACS</a> - <a href="#blasr">BLASR</a> - <a href="#blasr_libcpp">blasr_libcpp</a> - <a href="#blast">BLAST</a> - <a href="#blast+">BLAST+</a> - <a href="#blat">BLAT</a> - <a href="#blender">Blender</a> - <a href="#blis">BLIS</a> - <a href="#blitz++">Blitz++</a> - <a href="#blobtools">BlobTools</a> - <a href="#block">Block</a> - <a href="#blosc">Blosc</a> - <a href="#blosc2">Blosc2</a> - <a href="#blt">BLT</a> - <a href="#bmtagger">bmtagger</a> - <a href="#bnpy">bnpy</a> - <a href="#boinc">BOINC</a> - <a href="#bokeh">bokeh</a> - <a href="#boltztrap">BoltzTraP</a> - <a href="#bonito">Bonito</a> - <a href="#bonmin">Bonmin</a> - <a href="#bonnie++">Bonnie++</a> - <a href="#boost">Boost</a> - <a href="#boost.mpi">Boost.MPI</a> - <a href="#boost.python">Boost.Python</a> - <a href="#boost_histogram">boost_histogram</a> - <a href="#boto3">boto3</a> - <a href="#bottleneck">Bottleneck</a> - <a href="#bowtie">Bowtie</a> - <a href="#bowtie2">Bowtie2</a> - <a href="#bpipe">Bpipe</a> - <a href="#bpp">bpp</a> - <a href="#bpytop">bpytop</a> - <a href="#bracken">Bracken</a> - <a href="#braker">BRAKER</a> - <a href="#breakdancer">BreakDancer</a> - <a href="#breseq">breseq</a> - <a href="#brotli">Brotli</a> - <a href="#brotli-python">Brotli-python</a> - <a href="#brunsli">Brunsli</a> - <a href="#bsddb3">bsddb3</a> - <a href="#bsmapz">BSMAPz</a> - <a href="#bsoft">Bsoft</a> - <a href="#bsseeker2">BSseeker2</a> - <a href="#buddy">BuDDy</a> - <a href="#bufrlib">BUFRLIB</a> - <a href="#buildenv">buildenv</a> - <a href="#buildingspy">buildingspy</a> - <a href="#bullet">Bullet</a> - <a href="#busco">BUSCO</a> - <a href="#bustools">BUStools</a> - <a href="#bwa">BWA</a> - <a href="#bwa-meth">bwa-meth</a> - <a href="#bwakit">bwakit</a> - <a href="#bwidget">bwidget</a> - <a href="#bwise">BWISE</a> - <a href="#bx-python">bx-python</a> - <a href="#bxh_xcede_tools">BXH_XCEDE_TOOLS</a> - <a href="#byacc">byacc</a> - <a href="#byobu">byobu</a> - <a href="#bzip2">bzip2</a></p>
<h3 id="ba3-snps-autotune">BA3-SNPS-autotune<a class="headerlink" href="#ba3-snps-autotune" title="Permanent link">&para;</a></h3>
<p>This program will automatically tune mixing parameters for BA3-SNPs by implementing  a binary search algorithm and conducting short exploratory runs of BA3-SNPS. </p>
<p><em>homepage</em>: <a href="https://github.com/stevemussmann/BA3-SNPS-autotune">https://github.com/stevemussmann/BA3-SNPS-autotune</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="babelstream">BabelStream<a class="headerlink" href="#babelstream" title="Permanent link">&para;</a></h3>
<p>STREAM, for lots of devices written in many programming models</p>
<p><em>homepage</em>: <a href="https://uob-hpc.github.io/BabelStream">https://uob-hpc.github.io/BabelStream</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4</code></td>
<td><code>-omp</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="babl">babl<a class="headerlink" href="#babl" title="Permanent link">&para;</a></h3>
<p>babl is pixel encoding and color space conversion engine in C</p>
<p><em>homepage</em>: <a href="https://www.gegl.org/babl">https://www.gegl.org/babl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.86</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bader">Bader<a class="headerlink" href="#bader" title="Permanent link">&para;</a></h3>
<p>A fast algorithm for doing Bader's analysis on a charge density grid.</p>
<p><em>homepage</em>: <a href="http://theory.cm.utexas.edu/henkelman/code/bader/">http://theory.cm.utexas.edu/henkelman/code/bader/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.02</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.03</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.04</code></td>
<td><code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="bagel">BAGEL<a class="headerlink" href="#bagel" title="Permanent link">&para;</a></h3>
<p>BAGEL (Brilliantly Advanced General Electronic-structure Library) is a parallel electronic-structure program.</p>
<p><em>homepage</em>: <a href="http://www.nubakery.org">http://www.nubakery.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="bali-phy">BAli-Phy<a class="headerlink" href="#bali-phy" title="Permanent link">&para;</a></h3>
<p>BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid,  or codon sequences.</p>
<p><em>homepage</em>: <a href="http://www.bali-phy.org/">http://www.bali-phy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bam-readcount">bam-readcount<a class="headerlink" href="#bam-readcount" title="Permanent link">&para;</a></h3>
<p>Count DNA sequence reads in BAM files</p>
<p><em>homepage</em>: <a href="https://github.com/genome/bam-readcount">https://github.com/genome/bam-readcount</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="bambi">Bambi<a class="headerlink" href="#bambi" title="Permanent link">&para;</a></h3>
<p>Bambi is a high-level Bayesian model-building interface written in Python. It works with the probabilistic programming frameworks PyMC3 and is designed to make it extremely easy to fit Bayesian mixed-effects models common in biology, social sciences and other disciplines.</p>
<p><em>homepage</em>: <a href="https://bambinos.github.io/bambi">https://bambinos.github.io/bambi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="bamfilters">bamFilters<a class="headerlink" href="#bamfilters" title="Permanent link">&para;</a></h3>
<p>A utility tool to filter alignments from bam files,  using identity percent, low complexity and read coverage.</p>
<p><em>homepage</em>: <a href="https://github.com/institut-de-genomique/bamFilters">https://github.com/institut-de-genomique/bamFilters</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2022-06-30</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bamsurgeon">BAMSurgeon<a class="headerlink" href="#bamsurgeon" title="Permanent link">&para;</a></h3>
<p>Tools for adding mutations to existing .bam files, used for testing mutation callers</p>
<p><em>homepage</em>: <a href="https://github.com/adamewing/bamsurgeon">https://github.com/adamewing/bamsurgeon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bamtofastq">bamtofastq<a class="headerlink" href="#bamtofastq" title="Permanent link">&para;</a></h3>
<p>Convert 10x BAM files to the original FASTQs compatible with 10x pipelines.</p>
<p><em>homepage</em>: <a href="https://github.com/10XGenomics/bamtofastq">https://github.com/10XGenomics/bamtofastq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bamtools">BamTools<a class="headerlink" href="#bamtools" title="Permanent link">&para;</a></h3>
<p>BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.</p>
<p><em>homepage</em>: <a href="https://github.com/pezmaster31/bamtools">https://github.com/pezmaster31/bamtools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code>, <code>iccifort/2020.4.304</code>, <code>intel/2017b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bamutil">BamUtil<a class="headerlink" href="#bamutil" title="Permanent link">&para;</a></h3>
<p>BamUtil is a repository that contains several programs   that perform operations on SAM/BAM files. All of these programs   are built into a single executable, bam.</p>
<p><em>homepage</em>: <a href="http://genome.sph.umich.edu/wiki/BamUtil">http://genome.sph.umich.edu/wiki/BamUtil</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.13</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="bandage">Bandage<a class="headerlink" href="#bandage" title="Permanent link">&para;</a></h3>
<p>Bandage is a program for visualising de novo assembly graphs</p>
<p><em>homepage</em>: <a href="http://rrwick.github.io/Bandage">http://rrwick.github.io/Bandage</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>_Centos</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>_Ubuntu</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="barrnap">barrnap<a class="headerlink" href="#barrnap" title="Permanent link">&para;</a></h3>
<p>Barrnap (BAsic Rapid Ribosomal RNA Predictor) predicts the location of ribosomal RNA genes in genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/barrnap">https://github.com/tseemann/barrnap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>gompi/2020b</code>, <code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="basemap">basemap<a class="headerlink" href="#basemap" title="Permanent link">&para;</a></h3>
<p>The matplotlib basemap toolkit is a library for plotting  2D data on maps in Python</p>
<p><em>homepage</em>: <a href="https://matplotlib.org/basemap/">https://matplotlib.org/basemap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bash">Bash<a class="headerlink" href="#bash" title="Permanent link">&para;</a></h3>
<p>Bash is an sh-compatible command language interpreter that executes commands    read from the standard input or from a file.  Bash also incorporates useful features from the    Korn and C shells (ksh and csh).</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/bash">http://www.gnu.org/software/bash</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="bashplotlib">bashplotlib<a class="headerlink" href="#bashplotlib" title="Permanent link">&para;</a></h3>
<p>bashplotlib is a python package and command line tool for making basic plots in the terminal.  It's a quick way to visualize data when you don't have a GUI.</p>
<p><em>homepage</em>: <a href="https://github.com/glamp/bashplotlib">https://github.com/glamp/bashplotlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bat">bat<a class="headerlink" href="#bat" title="Permanent link">&para;</a></h3>
<p>The BAT Python package supports the processing and analysis of Bro data with Pandas, scikit-learn, and Spark</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/bat">https://pypi.python.org/pypi/bat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="batmeth2">BatMeth2<a class="headerlink" href="#batmeth2" title="Permanent link">&para;</a></h3>
<p>An Integrated Package for Bisulfite DNA Methylation Data Analysis with Indel-sensitive Mapping.</p>
<p><em>homepage</em>: <a href="https://github.com/GuoliangLi-HZAU/BatMeth2">https://github.com/GuoliangLi-HZAU/BatMeth2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="bayesass">BayesAss<a class="headerlink" href="#bayesass" title="Permanent link">&para;</a></h3>
<p>BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes</p>
<p><em>homepage</em>: <a href="http://www.rannala.org/?page_id=245">http://www.rannala.org/?page_id=245</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="bayesass3-snps">BayesAss3-SNPs<a class="headerlink" href="#bayesass3-snps" title="Permanent link">&para;</a></h3>
<p>Modification of BayesAss 3.0.4 to allow handling of large SNP datasets generated via methods such as RADseq protocols. </p>
<p><em>homepage</em>: <a href="https://github.com/stevemussmann/BayesAss3-SNPs">https://github.com/stevemussmann/BayesAss3-SNPs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bayescan">BayeScan<a class="headerlink" href="#bayescan" title="Permanent link">&para;</a></h3>
<p>BayeScan aims at identifying candidate loci under natural selection from genetic data,  using differences in allele frequencies between populations.</p>
<p><em>homepage</em>: <a href="http://cmpg.unibe.ch/software/BayeScan/">http://cmpg.unibe.ch/software/BayeScan/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2016a</code>, <code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="bayescenv">BayeScEnv<a class="headerlink" href="#bayescenv" title="Permanent link">&para;</a></h3>
<p>BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detect local adaptation.</p>
<p><em>homepage</em>: <a href="https://github.com/devillemereuil/bayescenv">https://github.com/devillemereuil/bayescenv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCC/8.3.0</code>, <code>foss/2016a</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="bayestraits">BayesTraits<a class="headerlink" href="#bayestraits" title="Permanent link">&para;</a></h3>
<p>BayesTraits is a computer package for performing analyses of trait   evolution among groups of species for which a phylogeny or sample of phylogenies is   available. This new package incoporates our earlier and separate programes Multistate,   Discrete and Continuous. BayesTraits can be applied to the analysis of traits that adopt   a finite number of discrete states, or to the analysis of continuously varying traits.   Hypotheses can be tested about models of evolution, about ancestral states and about   correlations among pairs of traits. </p>
<p><em>homepage</em>: <a href="http://www.evolution.reading.ac.uk/BayesTraitsV1.html">http://www.evolution.reading.ac.uk/BayesTraitsV1.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-linux32</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0</code></td>
<td><code>-Beta-Linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bazel">Bazel<a class="headerlink" href="#bazel" title="Permanent link">&para;</a></h3>
<p>Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software.</p>
<p><em>homepage</em>: <a href="https://bazel.io/">https://bazel.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.18.0</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.25.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.26.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.29.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.29.1</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.1.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bbcp">bbcp<a class="headerlink" href="#bbcp" title="Permanent link">&para;</a></h3>
<p>BBCP is an alternative to Gridftp when transferring large amounts of data,  capable of breaking up your transfer into multiple simultaneous transferring streams,  thereby transferring data much faster than single-streaming utilities such as SCP and SFTP.  See details at http://pcbunn.cithep.caltech.edu/bbcp/using_bbcp.htm  or http://www.nics.tennessee.edu/computing-resources/data-transfer/bbcp</p>
<p><em>homepage</em>: <a href="http://www.slac.stanford.edu/~abh/bbcp/">http://www.slac.stanford.edu/~abh/bbcp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>12.01.30.00.0</code></td>
<td><code>-amd64_linux26</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bbftp">bbFTP<a class="headerlink" href="#bbftp" title="Permanent link">&para;</a></h3>
<p>bbFTP is a file transfer software. It implements its own transfer protocol,   which is optimized for large files (larger than 2GB) and secure as it does not read the   password in a file and encrypts the connection information. bbFTP main features are:   * Encoded username and password at connection * SSH and Certificate authentication modules   * Multi-stream transfer * Big windows as defined in RFC1323 * On-the-fly data compression   * Automatic retry * Customizable time-outs * Transfer simulation   * AFS authentication integration * RFIO interface</p>
<p><em>homepage</em>: <a href="https://software.in2p3.fr/bbftp/">https://software.in2p3.fr/bbftp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCCcore/9.3.0</code>, <code>intel/2016a</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bbmap">BBMap<a class="headerlink" href="#bbmap" title="Permanent link">&para;</a></h3>
<p>BBMap short read aligner, and other bioinformatic tools.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/bbmap/">https://sourceforge.net/projects/bbmap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>36.62</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>37.93</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>38.26</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>38.50b</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>38.76</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>38.79</code></td>
<td></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>38.87</code></td>
<td></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>38.90</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>38.96</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>38.98</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>39.01</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bc">bc<a class="headerlink" href="#bc" title="Permanent link">&para;</a></h3>
<p>bc is an arbitrary precision numeric processing language.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/bc/">https://www.gnu.org/software/bc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.06.95</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="bcalm">BCALM<a class="headerlink" href="#bcalm" title="Permanent link">&para;</a></h3>
<p>de Bruijn graph compaction in low memory</p>
<p><em>homepage</em>: <a href="https://github.com/GATB/bcalm">https://github.com/GATB/bcalm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="bcbio-gff">bcbio-gff<a class="headerlink" href="#bcbio-gff" title="Permanent link">&para;</a></h3>
<p>Read and write Generic Feature Format (GFF) with Biopython integration. </p>
<p><em>homepage</em>: <a href="https://github.com/chapmanb/bcbb/tree/master/gff">https://github.com/chapmanb/bcbb/tree/master/gff</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.6.7</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bcel">BCEL<a class="headerlink" href="#bcel" title="Permanent link">&para;</a></h3>
<p>The Byte Code Engineering Library (Apache Commons BCEL) is intended to give  users a convenient way to analyze, create, and manipulate (binary) Java class  files (those ending with .class). </p>
<p><em>homepage</em>: <a href="https://commons.apache.org/proper/commons-bcel/">https://commons.apache.org/proper/commons-bcel/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.5.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bcftools">BCFtools<a class="headerlink" href="#bcftools" title="Permanent link">&para;</a></h3>
<p>Samtools is a suite of programs for interacting with high-throughput sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence  variants</p>
<p><em>homepage</em>: <a href="http://www.htslib.org/">http://www.htslib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bcgtree">bcgTree<a class="headerlink" href="#bcgtree" title="Permanent link">&para;</a></h3>
<p>Automatized phylogenetic tree building from bacterial core genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/molbiodiv/bcgTree">https://github.com/molbiodiv/bcgTree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.10</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="bcl-convert">bcl-convert<a class="headerlink" href="#bcl-convert" title="Permanent link">&para;</a></h3>
<p>The Illumina BCL Convert v4.0 is a standalone local software app that converts the  Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files.</p>
<p><em>homepage</em>: <a href="https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html">https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.3-2</code></td>
<td><code>el7.x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bcl2fastq2">bcl2fastq2<a class="headerlink" href="#bcl2fastq2" title="Permanent link">&para;</a></h3>
<p>bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by  Illumina sequencing systems to standard FASTQ file formats for downstream analysis.</p>
<p><em>homepage</em>: <a href="https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html">https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.19.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018b</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="bcolz">bcolz<a class="headerlink" href="#bcolz" title="Permanent link">&para;</a></h3>
<p>bcolz provides columnar, chunked data containers that can be compressed either in-memory and on-disk.  Column storage allows for efficiently querying tables, as well as for cheap column addition and removal.  It is based on NumPy, and uses it as the standard data container to communicate with bcolz objects,  but it also comes with support for import/export facilities to/from HDF5/PyTables tables and pandas dataframes.</p>
<p><em>homepage</em>: <a href="https://bcolz.blosc.org/en/latest/">https://bcolz.blosc.org/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="bdbag">BDBag<a class="headerlink" href="#bdbag" title="Permanent link">&para;</a></h3>
<p>The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the Bagit and Bagit/RO profiles.</p>
<p><em>homepage</em>: <a href="https://github.com/fair-research/bdbag">https://github.com/fair-research/bdbag</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td></td>
<td><code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="beagle">Beagle<a class="headerlink" href="#beagle" title="Permanent link">&para;</a></h3>
<p>Beagle is a software package for phasing genotypes and for imputing  ungenotyped markers.</p>
<p><em>homepage</em>: <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.22Jul22.46e</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="beagle-lib">beagle-lib<a class="headerlink" href="#beagle-lib" title="Permanent link">&para;</a></h3>
<p>beagle-lib is a high-performance library that can perform the core calculations at the heart of most  Bayesian and Maximum Likelihood phylogenetics packages.</p>
<p><em>homepage</em>: <a href="https://github.com/beagle-dev/beagle-lib">https://github.com/beagle-dev/beagle-lib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-CUDA-9.2.88</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/9.3.0</code>, <code>gcccuda/2019b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="beast">Beast<a class="headerlink" href="#beast" title="Permanent link">&para;</a></h3>
<p>BEAST is a cross-platform program for Bayesian MCMC analysis of molecular   sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using   strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies   but is also a framework for testing evolutionary hypotheses without conditioning on a single   tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted   proportional to its posterior probability. </p>
<p><em>homepage</em>: <a href="https://beast2.org">https://beast2.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.10.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.4.7</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>2.6.7</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.7.3</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="beautifulsoup">BeautifulSoup<a class="headerlink" href="#beautifulsoup" title="Permanent link">&para;</a></h3>
<p>Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.</p>
<p><em>homepage</em>: <a href="https://www.crummy.com/software/BeautifulSoup">https://www.crummy.com/software/BeautifulSoup</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.6.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.8.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.10.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bedops">BEDOPS<a class="headerlink" href="#bedops" title="Permanent link">&para;</a></h3>
<p>BEDOPS is an open-source command-line toolkit that performs highly efficient and  scalable Boolean and other set operations, statistical calculations, archiving, conversion and  other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for  distributing whole-genome analyses across a computational cluster.</p>
<p><em>homepage</em>: <a href="http://bedops.readthedocs.io/en/latest/index.html">http://bedops.readthedocs.io/en/latest/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCC/4.8.4</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>2.4.20</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.30</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.4.32</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.4.35</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.4.41</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="bedtools">BEDTools<a class="headerlink" href="#bedtools" title="Permanent link">&para;</a></h3>
<p>BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.</p>
<p><em>homepage</em>: <a href="https://bedtools.readthedocs.io/">https://bedtools.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.25.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.26.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.27.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.28.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.29.2</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.30.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="beef">BEEF<a class="headerlink" href="#beef" title="Permanent link">&para;</a></h3>
<p>BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149.</p>
<p><em>homepage</em>: <a href="https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software">https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.1</code></td>
<td><code>iccifort/2019.5.281</code>, <code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="behave">behave<a class="headerlink" href="#behave" title="Permanent link">&para;</a></h3>
<p>behave: Behavior-driven development (or BDD) is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.</p>
<p><em>homepage</em>: <a href="http://pythonhosted.org/behave">http://pythonhosted.org/behave</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.2.6</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="berkeleygw">BerkeleyGW<a class="headerlink" href="#berkeleygw" title="Permanent link">&para;</a></h3>
<p>The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle  properties and the optical responses of a large variety of materials from bulk periodic crystals to  nanostructures such as slabs, wires and molecules.</p>
<p><em>homepage</em>: <a href="https://www.berkeleygw.org">https://www.berkeleygw.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>1.1-beta2</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="bfast">BFAST<a class="headerlink" href="#bfast" title="Permanent link">&para;</a></h3>
<p>BFAST facilitates the fast and accurate mapping of short reads to reference sequences.  Some advantages of BFAST include:  1) Speed: enables billions of short reads to be mapped quickly.   2) Accuracy: A priori probabilities for mapping reads with defined set of variants.   3) An easy way to measurably tune accuracy at the expense of speed.</p>
<p><em>homepage</em>: <a href="http://bfast.sourceforge.net/">http://bfast.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0a</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="bfc">BFC<a class="headerlink" href="#bfc" title="Permanent link">&para;</a></h3>
<p>BFC is a standalone high-performance tool for correcting sequencing errors from Illumina sequencing data. It is specifically designed for high-coverage whole-genome human data, though also performs well for small genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/bfc">https://github.com/lh3/bfc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="bgen">bgen<a class="headerlink" href="#bgen" title="Permanent link">&para;</a></h3>
<p>A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3.</p>
<p><em>homepage</em>: <a href="https://github.com/limix/bgen">https://github.com/limix/bgen</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.1.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bgen-reader">bgen-reader<a class="headerlink" href="#bgen-reader" title="Permanent link">&para;</a></h3>
<p>A bgen file format reader. This python package is a wrapper around the bgen library, a low-memory footprint reader that efficiently reads bgen files. It fully supports the bgen format specifications: 1.2 and 1.3; as well as their optional compressed formats.</p>
<p><em>homepage</em>: <a href="https://github.com/limix/bgen-reader-py">https://github.com/limix/bgen-reader-py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="bibtexparser">bibtexparser<a class="headerlink" href="#bibtexparser" title="Permanent link">&para;</a></h3>
<p>Bibtex parser in Python 2.7 and 3.x</p>
<p><em>homepage</em>: <a href="https://github.com/sciunto-org/python-bibtexparser">https://github.com/sciunto-org/python-bibtexparser</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="big-scape">BiG-SCAPE<a class="headerlink" href="#big-scape" title="Permanent link">&para;</a></h3>
<p>BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters  (BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene  cluster families, and exploring gene cluster diversity linked to enzyme phylogenies.</p>
<p><em>homepage</em>: <a href="https://bigscape-corason.secondarymetabolites.org/index.html">https://bigscape-corason.secondarymetabolites.org/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="bigdft">BigDFT<a class="headerlink" href="#bigdft" title="Permanent link">&para;</a></h3>
<p>BigDFT: electronic structure calculation based on Daubechies wavelets. bigdft-suite is a set of different packages to run bigdft.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/l_sim/bigdft-suite">https://gitlab.com/l_sim/bigdft-suite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="binsanity">BinSanity<a class="headerlink" href="#binsanity" title="Permanent link">&para;</a></h3>
<p>BinSanity contains a suite a scripts designed to cluster contigs generated from metagenomic assembly into putative genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/edgraham/BinSanity/wiki">https://github.com/edgraham/BinSanity/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="binutils">binutils<a class="headerlink" href="#binutils" title="Permanent link">&para;</a></h3>
<p>binutils: GNU binary utilities</p>
<p><em>homepage</em>: <a href="https://directory.fsf.org/project/binutils/">https://directory.fsf.org/project/binutils/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.25</code></td>
<td><code>GCC/4.9.2</code>, <code>GCC/4.9.2-binutils-2.25</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-binutils-2.25</code>, <code>GCC/5.1.0-binutils-2.25</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/4.9.4</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.25.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.26</code></td>
<td><code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/5.5.0</code>, <code>GCCcore/6.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.27</code></td>
<td><code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.28</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.1.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.29</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/system</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.30</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.1.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.31.1</code></td>
<td><code>GCCcore/7.4.0</code>, <code>GCCcore/8.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.32</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.34</code></td>
<td><code>GCCcore/10.1.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.35</code></td>
<td><code>GCCcore/10.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.36.1</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/8.4.0</code>, <code>GCCcore/9.4.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.37</code></td>
<td><code>GCCcore/11.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.38</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/9.5.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.39</code></td>
<td><code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bio-db-hts">Bio-DB-HTS<a class="headerlink" href="#bio-db-hts" title="Permanent link">&para;</a></h3>
<p>Read files using HTSlib including BAM/CRAM, Tabix and BCF database files</p>
<p><em>homepage</em>: <a href="https://metacpan.org/release/Bio-DB-HTS">https://metacpan.org/release/Bio-DB-HTS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.11</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.11</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.01</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>3.01</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="bio-eutilities">Bio-EUtilities<a class="headerlink" href="#bio-eutilities" title="Permanent link">&para;</a></h3>
<p>BioPerl low-level API for retrieving and storing data from NCBI eUtils</p>
<p><em>homepage</em>: <a href="https://github.com/bioperl/bio-eutilities">https://github.com/bioperl/bio-eutilities</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.76</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bio-samtools">Bio-SamTools<a class="headerlink" href="#bio-samtools" title="Permanent link">&para;</a></h3>
<p>This is a Perl interface to the SAMtools sequence alignment interface.</p>
<p><em>homepage</em>: <a href="http://search.cpan.org/~lds/Bio-SamTools/">http://search.cpan.org/~lds/Bio-SamTools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.43</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="bio-searchio-hmmer">Bio-SearchIO-hmmer<a class="headerlink" href="#bio-searchio-hmmer" title="Permanent link">&para;</a></h3>
<p>Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org.</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Bio::SearchIO::hmmer3">https://metacpan.org/pod/Bio::SearchIO::hmmer3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.3</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bioawk">bioawk<a class="headerlink" href="#bioawk" title="Permanent link">&para;</a></h3>
<p>Bioawk is an extension to Brian Kernighan's awk,  adding the support of several common biological data formats,  including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. </p>
<p><em>homepage</em>: <a href="https://github.com/lh3/bioawk">https://github.com/lh3/bioawk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="biobambam2">biobambam2<a class="headerlink" href="#biobambam2" title="Permanent link">&para;</a></h3>
<p>Tools for processing BAM files</p>
<p><em>homepage</em>: <a href="https://github.com/gt1/biobambam2">https://github.com/gt1/biobambam2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.87</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="biogeme">biogeme<a class="headerlink" href="#biogeme" title="Permanent link">&para;</a></h3>
<p>Biogeme is a open source Python package designed for the maximum likelihood estimation of parametric models in general, with a special emphasis on discrete choice models.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/biogeme">https://pypi.python.org/pypi/biogeme</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.2.6</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>3.2.8</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.2.10</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="biom-format">biom-format<a class="headerlink" href="#biom-format" title="Permanent link">&para;</a></h3>
<p>The BIOM file format (canonically pronounced biome) is designed to be  a general-use format for representing biological sample by observation  contingency tables. BIOM is a recognized standard for the Earth Microbiome  Project and is a Genomics Standards Consortium supported project. </p>
<p><em>homepage</em>: <a href="https://biom-format.org">https://biom-format.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.10</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.1.12</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="biomart-perl">biomart-perl<a class="headerlink" href="#biomart-perl" title="Permanent link">&para;</a></h3>
<p>The BioMart Perl API allows you to go a step further with BioMart   and integrate BioMart Perl Code into custom Perl scripts.</p>
<p><em>homepage</em>: <a href="https://useast.ensembl.org/info/data/biomart/biomart_perl_api.html">https://useast.ensembl.org/info/data/biomart/biomart_perl_api.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7_e6db561</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bioperl">BioPerl<a class="headerlink" href="#bioperl" title="Permanent link">&para;</a></h3>
<p>Bioperl is the product of a community effort to produce Perl code which is useful in biology.  Examples include Sequence objects, Alignment objects and database searching objects.</p>
<p><em>homepage</em>: <a href="https://bioperl.org/">https://bioperl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.924</code></td>
<td><code>-Perl-5.20.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.6.924</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.7.7</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.7.8</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="biopp">BioPP<a class="headerlink" href="#biopp" title="Permanent link">&para;</a></h3>
<p>Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics. Bio++ is Object Oriented and is designed to be both easy to use and computer efficient. Bio++ intends to help programmers to write computer expensive programs, by providing them a set of re-usable tools. </p>
<p><em>homepage</em>: <a href="https://github.com/BioPP/bpp-core">https://github.com/BioPP/bpp-core</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="biopython">Biopython<a class="headerlink" href="#biopython" title="Permanent link">&para;</a></h3>
<p>Biopython is a set of freely available tools for biological  computation written in Python by an international team of developers. It is  a distributed collaborative effort to develop Python libraries and  applications which address the needs of current and future work in  bioinformatics. </p>
<p><em>homepage</em>: <a href="https://www.biopython.org">https://www.biopython.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.65</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.68</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.68</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.70</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.70</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.70</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.70</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.71</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.71</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.72</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.72</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.73</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.73</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.74</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.75</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.75</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.76</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.78</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.78</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.79</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>1.81</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="bioservices">BioServices<a class="headerlink" href="#bioservices" title="Permanent link">&para;</a></h3>
<p>Bioservices is a Python package that provides access to many Bioinformatices Web Services (e.g.,  UniProt) and a framework to easily implement Web Services wrappers (based on WSDL/SOAP or REST protocols).</p>
<p><em>homepage</em>: <a href="https://bioservices.readthedocs.io/">https://bioservices.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="birdnet">BirdNET<a class="headerlink" href="#birdnet" title="Permanent link">&para;</a></h3>
<p>BirdNET is a research platform that aims at recognizing birds by sound at scale. We support various hardware and operating systems such as Arduino microcontrollers, the Raspberry Pi,  smartphones, web browsers, workstation PCs, and even cloud services. BirdNET is a citizen science  platform as well as an analysis software for extremely large collections of audio. BirdNET aims to provide innovative tools for conservationists, biologists, and birders alike.</p>
<p><em>homepage</em>: <a href="https://birdnet.cornell.edu/">https://birdnet.cornell.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20201214</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="biscuit">biscuit<a class="headerlink" href="#biscuit" title="Permanent link">&para;</a></h3>
<p>Utilities to help analyze bisulfite-treated sequence data  </p>
<p><em>homepage</em>: <a href="https://github.com/zwdzwd/biscuit">https://github.com/zwdzwd/biscuit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="bismark">Bismark<a class="headerlink" href="#bismark" title="Permanent link">&para;</a></h3>
<p>A tool to map bisulfite converted sequence reads and determine cytosine methylation states</p>
<p><em>homepage</em>: <a href="https://www.bioinformatics.babraham.ac.uk/projects/bismark/">https://www.bioinformatics.babraham.ac.uk/projects/bismark/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.19.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.20.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.23.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="bison">Bison<a class="headerlink" href="#bison" title="Permanent link">&para;</a></h3>
<p>Bison is a general-purpose parser generator that converts an annotated  context-free grammar into a deterministic LR or generalized LR (GLR) parser  employing LALR(1) parser tables. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/bison">https://www.gnu.org/software/bison</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.4</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCC/4.9.2</code>, <code>GCC/4.9.2-binutils-2.25</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/4.9.3-binutils-2.25</code>, <code>GCC/5.1.0-binutils-2.25</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/4.9.4</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/5.5.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.1.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.1.0</code>, <code>GCCcore/system</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td><code>GCCcore/5.5.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.1.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/system</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCCcore/7.4.0</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/8.4.0</code>, <code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>GCCcore/10.1.0</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.7.6</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.4.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.5.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bitarray">bitarray<a class="headerlink" href="#bitarray" title="Permanent link">&para;</a></h3>
<p>bitarray provides an object type which efficiently represents an array of booleans</p>
<p><em>homepage</em>: <a href="https://github.com/ilanschnell/bitarray">https://github.com/ilanschnell/bitarray</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.8.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blacs">BLACS<a class="headerlink" href="#blacs" title="Permanent link">&para;</a></h3>
<p>The BLACS (Basic Linear Algebra Communication Subprograms) project is  an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface  that may be implemented efficiently and uniformly across a large range of distributed memory platforms.</p>
<p><em>homepage</em>: <a href="http://www.netlib.org/blacs/">http://www.netlib.org/blacs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>gmvapich2/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="blasr">BLASR<a class="headerlink" href="#blasr" title="Permanent link">&para;</a></h3>
<p>The PacBio long read aligner</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/blasr">https://github.com/PacificBiosciences/blasr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.3.3</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>20170330</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="blasr_libcpp">blasr_libcpp<a class="headerlink" href="#blasr_libcpp" title="Permanent link">&para;</a></h3>
<p>Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for  analyzing PacBio sequences</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/blasr_libcpp">https://github.com/PacificBiosciences/blasr_libcpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170426</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="blast">BLAST<a class="headerlink" href="#blast" title="Permanent link">&para;</a></h3>
<p>Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological  sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.</p>
<p><em>homepage</em>: <a href="http://blast.ncbi.nlm.nih.gov/">http://blast.ncbi.nlm.nih.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.26</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="blast_1">BLAST+<a class="headerlink" href="#blast_1" title="Permanent link">&para;</a></h3>
<p>Basic Local Alignment Search Tool, or BLAST, is an algorithm  for comparing primary biological sequence information, such as the amino-acid  sequences of different proteins or the nucleotides of DNA sequences.</p>
<p><em>homepage</em>: <a href="https://blast.ncbi.nlm.nih.gov/">https://blast.ncbi.nlm.nih.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.31</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompi/2021b</code>, <code>iimpi/2019a</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>2.12.0</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.13.0</code></td>
<td></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="blat">BLAT<a class="headerlink" href="#blat" title="Permanent link">&para;</a></h3>
<p>BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more.</p>
<p><em>homepage</em>: <a href="https://genome.ucsc.edu/goldenPath/help/blatSpec.html">https://genome.ucsc.edu/goldenPath/help/blatSpec.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.7</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blender">Blender<a class="headerlink" href="#blender" title="Permanent link">&para;</a></h3>
<p>Blender is the free and open source 3D creation suite. It supports  the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,  compositing and motion tracking, even video editing and game creation.</p>
<p><em>homepage</em>: <a href="https://www.blender.org/">https://www.blender.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.77a</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.79</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.79b</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.79b</code></td>
<td><code>-Python-3.6.6-CUDA-9.2.88</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.81</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-linux-x64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>-linux-x64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>-linux-x64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-linux-x86_64-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-linux-x86_64-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="blis">BLIS<a class="headerlink" href="#blis" title="Permanent link">&para;</a></h3>
<p>AMD's fork of BLIS. BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-cpu-libraries/blas-library/">https://developer.amd.com/amd-cpu-libraries/blas-library/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td></td>
<td><code>GCC/8.3.0-2.32</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code>, <code>GCC/12.2.0</code>, <code>intel-compilers/2022.1.0</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-amd</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-amd</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>-amd</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-amd</code></td>
<td><code>GCC/11.2.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.1</code></td>
<td><code>-amd</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blitz">Blitz++<a class="headerlink" href="#blitz" title="Permanent link">&para;</a></h3>
<p>Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation  in C++ with a speed comparable to Fortran implementations, while preserving an  object-oriented interface </p>
<p><em>homepage</em>: <a href="https://github.com/blitzpp/blitz">https://github.com/blitzpp/blitz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blobtools">BlobTools<a class="headerlink" href="#blobtools" title="Permanent link">&para;</a></h3>
<p>A modular command-line solution for visualisation,  quality control and taxonomic partitioning of genome datasets. </p>
<p><em>homepage</em>: <a href="https://blobtools.readme.io/docs">https://blobtools.readme.io/docs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180528</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="block">Block<a class="headerlink" href="#block" title="Permanent link">&para;</a></h3>
<p>Block implements the density matrix renormalization group (DMRG) algorithm for quantum chemistry.</p>
<p><em>homepage</em>: <a href="https://sanshar.github.io/Block/">https://sanshar.github.io/Block/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.3-20200525</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="blosc">Blosc<a class="headerlink" href="#blosc" title="Permanent link">&para;</a></h3>
<p>Blosc, an extremely fast, multi-threaded, meta-compressor library</p>
<p><em>homepage</em>: <a href="https://www.blosc.org/">https://www.blosc.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.14.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.14.4</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.17.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.17.1</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.21.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.21.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.21.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blosc2">Blosc2<a class="headerlink" href="#blosc2" title="Permanent link">&para;</a></h3>
<p>Blosc, an extremely fast, multi-threaded, meta-compressor library</p>
<p><em>homepage</em>: <a href="https://www.blosc.org/">https://www.blosc.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="blt">BLT<a class="headerlink" href="#blt" title="Permanent link">&para;</a></h3>
<p>BLT is an extension to the Tk toolkit, adding new widgets,  geometry managers, and miscellaneous commands.</p>
<p><em>homepage</em>: <a href="http://blt.sourceforge.net/">http://blt.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20181223</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bmtagger">bmtagger<a class="headerlink" href="#bmtagger" title="Permanent link">&para;</a></h3>
<p>Best Match Tagger for removing human reads from metagenomics datasets</p>
<p><em>homepage</em>: <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/">ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.101</code></td>
<td><code>foss/2018b</code>, <code>gompi/2019a</code>, <code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="bnpy">bnpy<a class="headerlink" href="#bnpy" title="Permanent link">&para;</a></h3>
<p>Bayesian nonparametric machine learning for python provides  code for training popular clustering models on large datasets. The focus is  on Bayesian nonparametric models based on the Dirichlet process, but it also  provides parametric counterparts.</p>
<p><em>homepage</em>: <a href="https://github.com/bnpy/bnpy">https://github.com/bnpy/bnpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="boinc">BOINC<a class="headerlink" href="#boinc" title="Permanent link">&para;</a></h3>
<p>BOINC is a program that lets you donate your idle computer time to science projects  like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others.</p>
<p><em>homepage</em>: <a href="https://boinc.berkeley.edu">https://boinc.berkeley.edu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.2.42</code></td>
<td><code>-client</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="bokeh">bokeh<a class="headerlink" href="#bokeh" title="Permanent link">&para;</a></h3>
<p>Statistical and novel interactive HTML plots for Python</p>
<p><em>homepage</em>: <a href="https://github.com/bokeh/bokeh">https://github.com/bokeh/bokeh</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.15</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.3.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="boltztrap">BoltzTraP<a class="headerlink" href="#boltztrap" title="Permanent link">&para;</a></h3>
<p>Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport  coefficients.</p>
<p><em>homepage</em>: <a href="http://www.icams.de/content/departments/cmat/boltztrap/">http://www.icams.de/content/departments/cmat/boltztrap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="bonito">Bonito<a class="headerlink" href="#bonito" title="Permanent link">&para;</a></h3>
<p>Convolution Basecaller for Oxford Nanopore Reads</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/bonito">https://github.com/nanoporetech/bonito</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.1.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.3.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.3.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.3.8</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="bonmin">Bonmin<a class="headerlink" href="#bonmin" title="Permanent link">&para;</a></h3>
<p>Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for  large-scale nonlinear optimization.</p>
<p><em>homepage</em>: <a href="https://coin-or.github.io/Ipopt">https://coin-or.github.io/Ipopt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.7</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="bonnie">Bonnie++<a class="headerlink" href="#bonnie" title="Permanent link">&para;</a></h3>
<p>Enhanced performance Test of Filesystem I/O</p>
<p><em>homepage</em>: <a href="https://www.coker.com.au/bonnie++">https://www.coker.com.au/bonnie++</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.97</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.00a</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="boost">Boost<a class="headerlink" href="#boost" title="Permanent link">&para;</a></h3>
<p>Boost provides free peer-reviewed portable C++ source libraries.</p>
<p><em>homepage</em>: <a href="https://www.boost.org/">https://www.boost.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.54.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.55.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.55.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.57.0</code></td>
<td><code>-Python-2.7.10</code></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>1.58.0</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.58.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.58.0</code></td>
<td><code>-serial</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.59.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.60.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.60.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.60.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.61.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.61.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.61.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.61.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.62.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.63.0</code></td>
<td></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.63.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.63.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.63.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.63.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.64.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompic/2019a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.64.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.65.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.65.1</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.65.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.65.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.65.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018.01</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2018.01</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-no_mpi</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.67.0</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.67.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.68.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.68.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.69.0</code></td>
<td></td>
<td><code>intel/2019.01</code></td>
</tr>
<tr>
<td><code>1.70.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompic/2019a</code>, <code>iimpi/2019a</code>, <code>iimpic/2019a</code></td>
</tr>
<tr>
<td><code>1.71.0</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompic/2019b</code>, <code>iimpi/2019b</code>, <code>iimpic/2019b</code></td>
</tr>
<tr>
<td><code>1.72.0</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>gompic/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.72.0</code></td>
<td><code>-no_mpi</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.74.0</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>1.75.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.76.0</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>1.77.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>1.79.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.81.0</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="boostmpi">Boost.MPI<a class="headerlink" href="#boostmpi" title="Permanent link">&para;</a></h3>
<p>Boost provides free peer-reviewed portable C++ source libraries.</p>
<p><em>homepage</em>: <a href="https://www.boost.org/">https://www.boost.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.77.0</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>1.79.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="boostpython">Boost.Python<a class="headerlink" href="#boostpython" title="Permanent link">&para;</a></h3>
<p>Boost.Python is a C++ library which enables seamless interoperability between C++  and the Python programming language.</p>
<p><em>homepage</em>: <a href="https://boostorg.github.io/python">https://boostorg.github.io/python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.64.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompic/2019a</code></td>
</tr>
<tr>
<td><code>1.65.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.66.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.67.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.67.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.70.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompic/2019a</code>, <code>iimpi/2019a</code>, <code>iimpic/2019a</code></td>
</tr>
<tr>
<td><code>1.71.0</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompic/2019b</code>, <code>iimpi/2019b</code>, <code>iimpic/2019b</code></td>
</tr>
<tr>
<td><code>1.72.0</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>gompic/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.74.0</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.76.0</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>1.77.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.79.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="boost_histogram">boost_histogram<a class="headerlink" href="#boost_histogram" title="Permanent link">&para;</a></h3>
<p>Boost-histogram is a Python package providing Python bindings for Boost.Histogram.</p>
<p><em>homepage</em>: <a href="https://boost-histogram.readthedocs.io">https://boost-histogram.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="boto3">boto3<a class="headerlink" href="#boto3" title="Permanent link">&para;</a></h3>
<p>Boto3 is the Amazon Web Services (AWS) Software Development Kit (SDK) for Python, which allows Python developers to write software that makes use of services like Amazon S3 and Amazon EC2.</p>
<p><em>homepage</em>: <a href="https://github.com/boto/boto3">https://github.com/boto/boto3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20.13</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.26.37</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bottleneck">Bottleneck<a class="headerlink" href="#bottleneck" title="Permanent link">&para;</a></h3>
<p>Fast NumPy array functions written in C</p>
<p><em>homepage</em>: <a href="https://kwgoodman.github.io/bottleneck-doc">https://kwgoodman.github.io/bottleneck-doc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bowtie">Bowtie<a class="headerlink" href="#bowtie" title="Permanent link">&para;</a></h3>
<p>Bowtie is an ultrafast, memory-efficient short read aligner.  It aligns short DNA sequences (reads) to the human genome.</p>
<p><em>homepage</em>: <a href="http://bowtie-bio.sourceforge.net/index.shtml">http://bowtie-bio.sourceforge.net/index.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.2.1.1</code></td>
<td><code>foss/2016b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bowtie2">Bowtie2<a class="headerlink" href="#bowtie2" title="Permanent link">&para;</a></h3>
<p>Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads  to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s  of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.  Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,  its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.</p>
<p><em>homepage</em>: <a href="https://bowtie-bio.sourceforge.net/bowtie2/index.shtml">https://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.8</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.2.9</code></td>
<td><code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.3.3.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.4.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.4.2</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.3.4.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.3.5.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.4.5</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bpipe">Bpipe<a class="headerlink" href="#bpipe" title="Permanent link">&para;</a></h3>
<p>Bpipe - a tool for running and managing bioinformatics pipelines</p>
<p><em>homepage</em>: <a href="http://docs.bpipe.org/">http://docs.bpipe.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.9.2</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="bpp">bpp<a class="headerlink" href="#bpp" title="Permanent link">&para;</a></h3>
<p>The aim of this project is to implement a versatile high-performance version of the BPP software. </p>
<p><em>homepage</em>: <a href="https://github.com/bpp/bpp">https://github.com/bpp/bpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.8</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bpytop">bpytop<a class="headerlink" href="#bpytop" title="Permanent link">&para;</a></h3>
<p>Resource monitor that shows usage and stats for processor, memory, disks, network and processes.</p>
<p><em>homepage</em>: <a href="https://github.com/aristocratos/bpytop/">https://github.com/aristocratos/bpytop/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.60</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.0.67</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bracken">Bracken<a class="headerlink" href="#bracken" title="Permanent link">&para;</a></h3>
<p>Bracken (Bayesian Reestimation of Abundance with KrakEN)  is a highly accurate statistical method that computes the abundance of  species in DNA sequences from a metagenomics sample. Braken uses the  taxonomy labels assigned by Kraken, a highly accurate metagenomics  classification algorithm, to estimate the number of reads originating  from each species present in a sample. Kraken classifies reads to the  best matching location in the taxonomic tree, but does not estimate  abundances of species. We use the Kraken database itself to derive  probabilities that describe how much sequence from each genome is  identical to other genomes in the database, and combine this information  with the assignments for a particular sample to estimate abundance at  the species level, the genus level, or above. Combined with the Kraken  classifier, Bracken produces accurate species- and genus-level abundance  estimates even when a sample contains two or more near-identical species.  NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the  default kmer length is different depending on the version of Kraken used.  If you use Kraken 1 defaults, specify 31 as the kmer length. If you use  Kraken 2 defaults, specify 35 as the kmer length.</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/bracken/">https://ccb.jhu.edu/software/bracken/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="braker">BRAKER<a class="headerlink" href="#braker" title="Permanent link">&para;</a></h3>
<p>BRAKER is a pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET  and AUGUSTUS in novel eukaryotic genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/Gaius-Augustus/BRAKER">https://github.com/Gaius-Augustus/BRAKER</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.1.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.1.6</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="breakdancer">BreakDancer<a class="headerlink" href="#breakdancer" title="Permanent link">&para;</a></h3>
<p>BreakDancer is a Perl/C++ package that provides genome-wide detection of structural variants from  next generation paired-end sequencing reads</p>
<p><em>homepage</em>: <a href="http://gmt.genome.wustl.edu/packages/breakdancer">http://gmt.genome.wustl.edu/packages/breakdancer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.5</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="breseq">breseq<a class="headerlink" href="#breseq" title="Permanent link">&para;</a></h3>
<p>breseq is a computational pipeline for the analysis of short-read re-sequencing data</p>
<p><em>homepage</em>: <a href="https://barricklab.org/breseq">https://barricklab.org/breseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.35.0</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.35.4</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.36.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="brotli">Brotli<a class="headerlink" href="#brotli" title="Permanent link">&para;</a></h3>
<p>Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.</p>
<p><em>homepage</em>: <a href="https://github.com/google/brotli">https://github.com/google/brotli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="brotli-python">Brotli-python<a class="headerlink" href="#brotli-python" title="Permanent link">&para;</a></h3>
<p>Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.</p>
<p><em>homepage</em>: <a href="https://github.com/google/brotli">https://github.com/google/brotli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="brunsli">Brunsli<a class="headerlink" href="#brunsli" title="Permanent link">&para;</a></h3>
<p>Brunsli is a lossless JPEG repacking library.</p>
<p><em>homepage</em>: <a href="https://github.com/google/brunsli/">https://github.com/google/brunsli/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bsddb3">bsddb3<a class="headerlink" href="#bsddb3" title="Permanent link">&para;</a></h3>
<p>bsddb3 is a nearly complete Python binding of the Oracle/Sleepycat C API for the Database Environment, Database, Cursor, Log Cursor, Sequence and Transaction objects.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/bsddb3/">https://pypi.org/project/bsddb3/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2.6</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.2.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>6.2.9</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bsmapz">BSMAPz<a class="headerlink" href="#bsmapz" title="Permanent link">&para;</a></h3>
<p>Updated and optimized fork of BSMAP. BSMAPz is a short reads mapping program for bisulfite sequencing in DNA methylation study.</p>
<p><em>homepage</em>: <a href="https://github.com/zyndagj/BSMAPz">https://github.com/zyndagj/BSMAPz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="bsoft">Bsoft<a class="headerlink" href="#bsoft" title="Permanent link">&para;</a></h3>
<p>Bsoft is a collection of programs and a platform for development of software  for image and molecular processing in structural biology. Problems in  structural biology are approached with a highly modular design, allowing fast  development of new algorithms without the burden of issues such as file I/O.  It provides an easily accessible interface, a resource that can be and has  been used in other packages. </p>
<p><em>homepage</em>: <a href="https://lsbr.niams.nih.gov/bsoft/">https://lsbr.niams.nih.gov/bsoft/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.0.7</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bsseeker2">BSseeker2<a class="headerlink" href="#bsseeker2" title="Permanent link">&para;</a></h3>
<p>BS-Seeker2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated reads.</p>
<p><em>homepage</em>: <a href="http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2">http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="buddy">BuDDy<a class="headerlink" href="#buddy" title="Permanent link">&para;</a></h3>
<p>A Binary Decision Diagram library, with many highly efficient vectorized BDD operations, dynamic  variable reordering, automated garbage collection, a C++ interface with automatic reference counting, and much  more.</p>
<p><em>homepage</em>: <a href="http://buddy.sourceforge.net/">http://buddy.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bufrlib">BUFRLIB<a class="headerlink" href="#bufrlib" title="Permanent link">&para;</a></h3>
<p>NCEP BUFRLIB software to encode or decode BUFR messages. It is not intended to be a primer on the BUFR code form itself.</p>
<p><em>homepage</em>: <a href="https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB">https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.3.0.2</code></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
</tbody>
</table>
<h3 id="buildenv">buildenv<a class="headerlink" href="#buildenv" title="Permanent link">&para;</a></h3>
<p>This module sets a group of environment variables for compilers, linkers, maths libraries, etc., that  you can use to easily transition between toolchains when building your software. To query the variables being set  please use: module show <this module name></p>
<p><em>homepage</em>: <None></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>default</code></td>
<td></td>
<td><code>FCC/4.5.0</code>, <code>Fujitsu/21.05</code>, <code>foss/2017b</code>, <code>foss/2018b</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>fosscuda/2019b</code>, <code>fosscuda/2020a</code>, <code>fosscuda/2020b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2019b</code>, <code>intel/2020a</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code>, <code>intel/2022a</code>, <code>intelcuda/2019b</code>, <code>intelcuda/2020a</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>default</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>default</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>default</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="buildingspy">buildingspy<a class="headerlink" href="#buildingspy" title="Permanent link">&para;</a></h3>
<p>Python modules for automating Modelica simulations and for running unit test for the Buildings library</p>
<p><em>homepage</em>: <a href="https://simulationresearch.lbl.gov/modelica/buildingspy">https://simulationresearch.lbl.gov/modelica/buildingspy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bullet">Bullet<a class="headerlink" href="#bullet" title="Permanent link">&para;</a></h3>
<p>Bullet professional 3D Game Multiphysics Library provides state  of the art collision detection, soft body and rigid body dynamics.</p>
<p><em>homepage</em>: <a href="http://bulletphysics.org/">http://bulletphysics.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.83.7</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="busco">BUSCO<a class="headerlink" href="#busco" title="Permanent link">&para;</a></h3>
<p>BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</p>
<p><em>homepage</em>: <a href="https://busco.ezlab.org/">https://busco.ezlab.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.22</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.0.6</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.4.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.4.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>5.4.5</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bustools">BUStools<a class="headerlink" href="#bustools" title="Permanent link">&para;</a></h3>
<p>bustools is a program for manipulating BUS files for single cell RNA-Seq datasets.  It can be used to error correct barcodes, collapse  UMIs, produce gene count or transcript compatibility  count matrices, and is useful for many other tasks. See the kallisto | bustools  website for examples  and instructions on how to use bustools as part of a single-cell RNA-seq workflow.</p>
<p><em>homepage</em>: <a href="https://github.com/BUStools/bustools">https://github.com/BUStools/bustools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.40.0</code></td>
<td><code>GCCcore/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="bwa">BWA<a class="headerlink" href="#bwa" title="Permanent link">&para;</a></h3>
<p>Burrows-Wheeler Aligner (BWA) is an efficient program that aligns  relatively short nucleotide sequences against a long reference sequence such as the human genome.</p>
<p><em>homepage</em>: <a href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.7.13</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.7.15</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.7.16a</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.7.17</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code>, <code>iccifort/2020.4.304</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="bwa-meth">bwa-meth<a class="headerlink" href="#bwa-meth" title="Permanent link">&para;</a></h3>
<p>Fast and accurante alignment of BS-Seq reads.</p>
<p><em>homepage</em>: <a href="https://github.com/brentp/bwa-meth">https://github.com/brentp/bwa-meth</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="bwakit">bwakit<a class="headerlink" href="#bwakit" title="Permanent link">&para;</a></h3>
<p>Bwakit is a self-consistent installation-free package of scripts and precompiled binaries, providing an end-to-end solution to read mapping.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/bwa/tree/master/bwakit">https://github.com/lh3/bwa/tree/master/bwakit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.15</code></td>
<td><code>_x64-linux</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="bwidget">bwidget<a class="headerlink" href="#bwidget" title="Permanent link">&para;</a></h3>
<p>The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.</p>
<p><em>homepage</em>: <a href="https://core.tcl-lang.org/bwidget/home">https://core.tcl-lang.org/bwidget/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.13</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.9.14</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.9.15</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bwise">BWISE<a class="headerlink" href="#bwise" title="Permanent link">&para;</a></h3>
<p>de Bruijn Workflow using Integral information of Short pair End reads</p>
<p><em>homepage</em>: <a href="https://github.com/Malfoy/BWISE">https://github.com/Malfoy/BWISE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180820</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="bx-python">bx-python<a class="headerlink" href="#bx-python" title="Permanent link">&para;</a></h3>
<p>The bx-python project is a Python library and associated set of scripts to allow for rapid  implementation of genome scale analyses.</p>
<p><em>homepage</em>: <a href="https://github.com/bxlab/bx-python">https://github.com/bxlab/bx-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.7.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.8.4</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.8.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.8.11</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.13</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="bxh_xcede_tools">BXH_XCEDE_TOOLS<a class="headerlink" href="#bxh_xcede_tools" title="Permanent link">&para;</a></h3>
<p>A collection of data processing and image analysis  tools for data in BXH or XCEDE format. This includes data format  encapsulation/conversion, event-related analysis, QA tools, and more.  These tools form the basis of the fBIRN QA procedures and are also  distributed as part of the fBIRN Data Upload Scripts.</p>
<p><em>homepage</em>: <a href="http://www.nitrc.org/projects/bxh_xcede_tools/">http://www.nitrc.org/projects/bxh_xcede_tools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="byacc">byacc<a class="headerlink" href="#byacc" title="Permanent link">&para;</a></h3>
<p>Berkeley Yacc (byacc) is generally conceded to be the best yacc variant  available.  In contrast to bison, it is written to avoid dependencies  upon a particular compiler. </p>
<p><em>homepage</em>: <a href="http://invisible-island.net/byacc/byacc.html">http://invisible-island.net/byacc/byacc.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160324</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20160606</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20170709</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="byobu">byobu<a class="headerlink" href="#byobu" title="Permanent link">&para;</a></h3>
<p>Byobu is an elegant enhancement of the otherwise functional, plain, practical GNU Screen. Byobu includes an enhanced profile, configuration utilities, and system status notifications for the GNU screen window manager as well as the Tmux terminal multiplexer</p>
<p><em>homepage</em>: <a href="https://byobu.org">https://byobu.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.133</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="bzip2">bzip2<a class="headerlink" href="#bzip2" title="Permanent link">&para;</a></h3>
<p>bzip2 is a freely available, patent free, high-quality data compressor. It  typically compresses files to within 10% to 15% of the best available  techniques (the PPM family of statistical compressors), whilst being around  twice as fast at compression and six times faster at decompression. </p>
<p><em>homepage</em>: <a href="https://sourceware.org/bzip2">https://sourceware.org/bzip2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.0.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h2 id="c">C<a class="headerlink" href="#c" title="Permanent link">&para;</a></h2>
<p><a href="#c-ares">c-ares</a> - <a href="#c3d">C3D</a> - <a href="#cadaver">cadaver</a> - <a href="#cadical">CaDiCaL</a> - <a href="#cafe5">CAFE5</a> - <a href="#caffe">Caffe</a> - <a href="#cairo">cairo</a> - <a href="#cairomm">cairomm</a> - <a href="#calcam">Calcam</a> - <a href="#calendrical">Calendrical</a> - <a href="#calib">Calib</a> - <a href="#campari">CAMPARI</a> - <a href="#cantera">Cantera</a> - <a href="#canu">canu</a> - <a href="#canvas">Canvas</a> - <a href="#cap3">CAP3</a> - <a href="#capnproto">CapnProto</a> - <a href="#captum">captum</a> - <a href="#cargo">Cargo</a> - <a href="#carma">Carma</a> - <a href="#carputils">carputils</a> - <a href="#cartopy">Cartopy</a> - <a href="#casper">CaSpER</a> - <a href="#castep">CASTEP</a> - <a href="#castxml">CastXML</a> - <a href="#cat-bat">CAT-BAT</a> - <a href="#catch2">Catch2</a> - <a href="#category_encoders">category_encoders</a> - <a href="#catlearn">CatLearn</a> - <a href="#catmap">CatMAP</a> - <a href="#causallift">causallift</a> - <a href="#causalml">causalml</a> - <a href="#caveman">CaVEMan</a> - <a href="#caviar">CAVIAR</a> - <a href="#cbc">Cbc</a> - <a href="#cblas">CBLAS</a> - <a href="#ccache">ccache</a> - <a href="#ccfits">CCfits</a> - <a href="#ccl">CCL</a> - <a href="#cclib">cclib</a> - <a href="#cctools">cctools</a> - <a href="#cd-hit">CD-HIT</a> - <a href="#cdat">CDAT</a> - <a href="#cdbfasta">cdbfasta</a> - <a href="#cdbtools">CDBtools</a> - <a href="#cddlib">cddlib</a> - <a href="#cdflib">CDFlib</a> - <a href="#cdna_cupcake">cDNA_Cupcake</a> - <a href="#cdo">CDO</a> - <a href="#cdo-bindings">cdo-bindings</a> - <a href="#cdsapi">cdsapi</a> - <a href="#cell2location">cell2location</a> - <a href="#cellmix">CellMix</a> - <a href="#cellpose">Cellpose</a> - <a href="#cellranger">CellRanger</a> - <a href="#cellranger-arc">CellRanger-ARC</a> - <a href="#cellranger-atac">CellRanger-ATAC</a> - <a href="#cellrank">CellRank</a> - <a href="#celltypist">CellTypist</a> - <a href="#centrifuge">Centrifuge</a> - <a href="#cereal">Cereal</a> - <a href="#cesm-deps">CESM-deps</a> - <a href="#cfdemcoupling">CFDEMcoupling</a> - <a href="#cfitsio">CFITSIO</a> - <a href="#cftime">cftime</a> - <a href="#cgal">CGAL</a> - <a href="#cget">cget</a> - <a href="#cgl">Cgl</a> - <a href="#cgmaptools">CGmapTools</a> - <a href="#cgns">CGNS</a> - <a href="#charls">CharLS</a> - <a href="#chase">CHASE</a> - <a href="#check">Check</a> - <a href="#checkm">CheckM</a> - <a href="#checkm-database">CheckM-Database</a> - <a href="#cheetah">Cheetah</a> - <a href="#chemaxon-marvin">Chemaxon-Marvin</a> - <a href="#chemprop">chemprop</a> - <a href="#chemps2">CheMPS2</a> - <a href="#cherab">CHERAB</a> - <a href="#chewbbaca">chewBBACA</a> - <a href="#chi2comb">chi2comb</a> - <a href="#chimera">Chimera</a> - <a href="#chimpipe">ChimPipe</a> - <a href="#chromaprint">Chromaprint</a> - <a href="#cicero">cicero</a> - <a href="#cif2cell">CIF2Cell</a> - <a href="#cimfomfa">cimfomfa</a> - <a href="#circexplorer">CIRCexplorer</a> - <a href="#circexplorer2">CIRCexplorer2</a> - <a href="#circos">Circos</a> - <a href="#ciri">CIRI</a> - <a href="#ciri-long">CIRI-long</a> - <a href="#cistem">cisTEM</a> - <a href="#cite-seq-count">CITE-seq-Count</a> - <a href="#clang">Clang</a> - <a href="#clang-aomp">Clang-AOMP</a> - <a href="#clang-python-bindings">Clang-Python-bindings</a> - <a href="#clapack">CLAPACK</a> - <a href="#clease">CLEASE</a> - <a href="#clhep">CLHEP</a> - <a href="#climetlab">CliMetLab</a> - <a href="#clisp">CLISP</a> - <a href="#clonalframeml">ClonalFrameML</a> - <a href="#cloog">CLooG</a> - <a href="#cloudcompare">CloudCompare</a> - <a href="#clp">Clp</a> - <a href="#clustal-omega">Clustal-Omega</a> - <a href="#clustalw2">ClustalW2</a> - <a href="#cluster-buster">Cluster-Buster</a> - <a href="#clustershell">ClusterShell</a> - <a href="#cmake">CMake</a> - <a href="#cmaverse">CMAverse</a> - <a href="#cmdstanr">CmdStanR</a> - <a href="#cmocean">cmocean</a> - <a href="#cmseq">CMSeq</a> - <a href="#cnt-ilp">CNT-ILP</a> - <a href="#cnvkit">CNVkit</a> - <a href="#cnvnator">CNVnator</a> - <a href="#co-phylog">Co-phylog</a> - <a href="#cobrapy">COBRApy</a> - <a href="#cocoalib">CoCoALib</a> - <a href="#codan">CodAn</a> - <a href="#code-server">code-server</a> - <a href="#codex2">CODEX2</a> - <a href="#cogent">Cogent</a> - <a href="#coin">Coin</a> - <a href="#coinutils">CoinUtils</a> - <a href="#colossalai">colossalai</a> - <a href="#commet">Commet</a> - <a href="#comparem">CompareM</a> - <a href="#compress-raw-zlib">Compress-Raw-Zlib</a> - <a href="#comsol">COMSOL</a> - <a href="#con3f">Con3F</a> - <a href="#conan">conan</a> - <a href="#concoct">CONCOCT</a> - <a href="#configparser">configparser</a> - <a href="#configurable-http-proxy">configurable-http-proxy</a> - <a href="#conn">CONN</a> - <a href="#connectomeworkbench">ConnectomeWorkbench</a> - <a href="#control-freec">Control-FREEC</a> - <a href="#cooler">cooler</a> - <a href="#coordgenlibs">CoordgenLibs</a> - <a href="#coot">Coot</a> - <a href="#core-counter">core-counter</a> - <a href="#coreutils">Coreutils</a> - <a href="#corner">corner</a> - <a href="#cosymlib">CoSymLib</a> - <a href="#coverage">coverage</a> - <a href="#cowsay">cowsay</a> - <a href="#cp2k">CP2K</a> - <a href="#cpb">CPB</a> - <a href="#cplex">CPLEX</a> - <a href="#cppe">CPPE</a> - <a href="#cppunit">CppUnit</a> - <a href="#cppy">cppy</a> - <a href="#cpu_features">cpu_features</a> - <a href="#cram">cram</a> - <a href="#cramtools">cramtools</a> - <a href="#craycce">CrayCCE</a> - <a href="#craygnu">CrayGNU</a> - <a href="#crayintel">CrayIntel</a> - <a href="#craypgi">CrayPGI</a> - <a href="#crb-blast">crb-blast</a> - <a href="#crest">CREST</a> - <a href="#crf++">CRF++</a> - <a href="#crispr-dav">CRISPR-DAV</a> - <a href="#crispresso2">CRISPResso2</a> - <a href="#cromwell">cromwell</a> - <a href="#crossguid">crossguid</a> - <a href="#crossmap">CrossMap</a> - <a href="#crosstalkz">CrossTalkZ</a> - <a href="#crpropa">CRPropa</a> - <a href="#crumble">Crumble</a> - <a href="#cryocare">cryoCARE</a> - <a href="#cryodrgn">cryoDRGN</a> - <a href="#cryptominisat">CryptoMiniSat</a> - <a href="#crystfel">CrystFEL</a> - <a href="#csb">CSB</a> - <a href="#csbdeep">CSBDeep</a> - <a href="#csblast">CSBLAST</a> - <a href="#cscope">cscope</a> - <a href="#csvkit">csvkit</a> - <a href="#ctags">ctags</a> - <a href="#ctffind">ctffind</a> - <a href="#ctpl">CTPL</a> - <a href="#cube">Cube</a> - <a href="#cubegui">CubeGUI</a> - <a href="#cubelib">CubeLib</a> - <a href="#cubewriter">CubeWriter</a> - <a href="#cuda">CUDA</a> - <a href="#cuda-samples">CUDA-Samples</a> - <a href="#cudacompat">CUDAcompat</a> - <a href="#cudacore">CUDAcore</a> - <a href="#cudnn">cuDNN</a> - <a href="#cufflinks">Cufflinks</a> - <a href="#cunit">CUnit</a> - <a href="#cupy">CuPy</a> - <a href="#curl">cURL</a> - <a href="#cusparselt">cuSPARSELt</a> - <a href="#custodian">custodian</a> - <a href="#cutadapt">cutadapt</a> - <a href="#cutensor">cuTENSOR</a> - <a href="#cvs">CVS</a> - <a href="#cvxopt">CVXOPT</a> - <a href="#cvxpy">CVXPY</a> - <a href="#cwipi">CWIPI</a> - <a href="#cwltool">cwltool</a> - <a href="#cxxopts">cxxopts</a> - <a href="#cysignals">cysignals</a> - <a href="#cython">Cython</a> - <a href="#cython-blis">cython-blis</a> - <a href="#cytoolz">cytoolz</a> - <a href="#cytoscape">Cytoscape</a> - <a href="#cytosim">cytosim</a> - <a href="#cyvcf2">cyvcf2</a></p>
<h3 id="c-ares">c-ares<a class="headerlink" href="#c-ares" title="Permanent link">&para;</a></h3>
<p>c-ares is a C library for asynchronous DNS requests (including name resolves)</p>
<p><em>homepage</em>: <a href="https://c-ares.org/">https://c-ares.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.17.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.18.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="c3d">C3D<a class="headerlink" href="#c3d" title="Permanent link">&para;</a></h3>
<p>Convert3D Medical Image Processing Tool</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/c3d/">https://sourceforge.net/projects/c3d/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cadaver">cadaver<a class="headerlink" href="#cadaver" title="Permanent link">&para;</a></h3>
<p>cadaver is a command-line WebDAV client for Unix.</p>
<p><em>homepage</em>: <a href="http://www.webdav.org/cadaver/">http://www.webdav.org/cadaver/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.23.3</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="cadical">CaDiCaL<a class="headerlink" href="#cadical" title="Permanent link">&para;</a></h3>
<p>CaDiCaL is a simplified satisfiability solver. The original goal of the development of CaDiCaL was to obtain a CDCL solver, which is easy to understand and change, while at the same time not being much slower than other state-of-the-art CDCL solvers.</p>
<p><em>homepage</em>: <a href="https://github.com/arminbiere/cadical">https://github.com/arminbiere/cadical</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cafe5">CAFE5<a class="headerlink" href="#cafe5" title="Permanent link">&para;</a></h3>
<p>Software for Computational Analysis of gene Family Evolution  The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa.</p>
<p><em>homepage</em>: <a href="https://github.com/hahnlab/CAFE5">https://github.com/hahnlab/CAFE5</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="caffe">Caffe<a class="headerlink" href="#caffe" title="Permanent link">&para;</a></h3>
<p>Caffe is a deep learning framework made with expression, speed,  and modularity in mind. It is developed by the Berkeley Vision  and Learning Center (BVLC) and community contributors. </p>
<p><em>homepage</em>: <a href="https://github.com/BVLC/caffe">https://github.com/BVLC/caffe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-CUDA-9.1.85-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>rc3</code></td>
<td><code>-CUDA-7.5.18-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="cairo">cairo<a class="headerlink" href="#cairo" title="Permanent link">&para;</a></h3>
<p>Cairo is a 2D graphics library with support for multiple output devices.  Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,  PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB</p>
<p><em>homepage</em>: <a href="https://cairographics.org">https://cairographics.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14.6</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.14.6</code></td>
<td><code>-GLib-2.48.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.14.8</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.14.10</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.14.12</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.17.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cairomm">cairomm<a class="headerlink" href="#cairomm" title="Permanent link">&para;</a></h3>
<p>The Cairomm package provides a C++ interface to Cairo.</p>
<p><em>homepage</em>: <a href="http://cairographics.org">http://cairographics.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="calcam">Calcam<a class="headerlink" href="#calcam" title="Permanent link">&para;</a></h3>
<p>Calcam is a Python package providing tools for spatial calibration  of cameras, i.e. determining the mapping between pixel coordinates in an image and  real-world 3D sight lines &amp; coordinates.</p>
<p><em>homepage</em>: <a href="https://euratom-software.github.io/calcam">https://euratom-software.github.io/calcam</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="calendrical">Calendrical<a class="headerlink" href="#calendrical" title="Permanent link">&para;</a></h3>
<p>Calendrical module is for calendrical calculations.</p>
<p><em>homepage</em>: <a href="https://www.funaba.org/code#calendrical">https://www.funaba.org/code#calendrical</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0.2a</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0.2a</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="calib">Calib<a class="headerlink" href="#calib" title="Permanent link">&para;</a></h3>
<p>Calib clusters paired-end reads using their barcodes and sequences. Calib is suitable for amplicon sequencing where a molecule is tagged, then PCR amplified with high depth, also known as Unique Molecule Identifier (UMI) sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/vpc-ccg/calib">https://github.com/vpc-ccg/calib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="campari">CAMPARI<a class="headerlink" href="#campari" title="Permanent link">&para;</a></h3>
<p>CAMPARI is a joint package for performing and analyzing molecular simulations, in particular of systems of biological relevance. It focuses on a wide availability of algorithms for (advanced) sampling and is capable of combining Monte Carlo and molecular dynamics in seamless fashion.</p>
<p><em>homepage</em>: <a href="http://campari.sourceforge.net/V4/index.html">http://campari.sourceforge.net/V4/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cantera">Cantera<a class="headerlink" href="#cantera" title="Permanent link">&para;</a></h3>
<p>Chemical kinetics, thermodynamics, and transport tool suite</p>
<p><em>homepage</em>: <a href="https://github.com/Cantera/cantera">https://github.com/Cantera/cantera</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="canu">canu<a class="headerlink" href="#canu" title="Permanent link">&para;</a></h3>
<p>Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing</p>
<p><em>homepage</em>: <a href="https://canu.readthedocs.io">https://canu.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="canvas">Canvas<a class="headerlink" href="#canvas" title="Permanent link">&para;</a></h3>
<p>Copy number variant (CNV) calling from DNA sequencing data</p>
<p><em>homepage</em>: <a href="https://github.com/Illumina/canvas">https://github.com/Illumina/canvas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.39.0.1598</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cap3">CAP3<a class="headerlink" href="#cap3" title="Permanent link">&para;</a></h3>
<p>CAP3 assembly program </p>
<p><em>homepage</em>: <a href="http://seq.cs.iastate.edu/">http://seq.cs.iastate.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20071221-intel-x86</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20071221-intel-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20071221-opteron</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="capnproto">CapnProto<a class="headerlink" href="#capnproto" title="Permanent link">&para;</a></h3>
<p>Capn Proto is an insanely fast data interchange format and capability-based RPC system.</p>
<p><em>homepage</em>: <a href="https://capnproto.org">https://capnproto.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.10.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="captum">captum<a class="headerlink" href="#captum" title="Permanent link">&para;</a></h3>
<p>Captum is a model interpretability and understanding library for PyTorch. Captum means comprehension in Latin and contains general purpose implementations of integrated gradients, saliency maps, smoothgrad, vargrad and others for PyTorch models. It has quick integration for models built with domain-specific libraries such as torchvision, torchtext, and others.</p>
<p><em>homepage</em>: <a href="https://captum.ai/">https://captum.ai/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="cargo">Cargo<a class="headerlink" href="#cargo" title="Permanent link">&para;</a></h3>
<p>The Rust package manager</p>
<p><em>homepage</em>: <a href="https://crates.io/">https://crates.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="carma">Carma<a class="headerlink" href="#carma" title="Permanent link">&para;</a></h3>
<p>Carma - A molecular dynamics analysis program</p>
<p><em>homepage</em>: <a href="http://utopia.duth.gr/~glykos/Carma.html">http://utopia.duth.gr/~glykos/Carma.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.01</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="carputils">carputils<a class="headerlink" href="#carputils" title="Permanent link">&para;</a></h3>
<p>carputils is a Python framework for generating and running openCARP examples.</p>
<p><em>homepage</em>: <a href="https://git.opencarp.org/openCARP/carputils">https://git.opencarp.org/openCARP/carputils</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200915</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>20210513</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cartopy">Cartopy<a class="headerlink" href="#cartopy" title="Permanent link">&para;</a></h3>
<p>Cartopy is a Python package designed to make drawing maps for data analysis and visualisation easy.</p>
<p><em>homepage</em>: <a href="https://scitools.org.uk/cartopy/docs/latest/">https://scitools.org.uk/cartopy/docs/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.18.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.19.0.post1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.20.3</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="casper">CaSpER<a class="headerlink" href="#casper" title="Permanent link">&para;</a></h3>
<p>CaSpER is a signal processing approach for identification, visualization, and integrative analysis  of focal and large-scale CNV events in multiscale resolution using either bulk or single-cell RNA sequencing data. </p>
<p><em>homepage</em>: <a href="https://github.com/akdess/CaSpER">https://github.com/akdess/CaSpER</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="castep">CASTEP<a class="headerlink" href="#castep" title="Permanent link">&para;</a></h3>
<p>CASTEP is an electronic structure materials modelling code based on density functional theory (DFT),  with functionality including geometry optimization molecular dynamics, phonons, NMR chemical shifts and much more.</p>
<p><em>homepage</em>: <a href="http://www.castep.org">http://www.castep.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>21.1.1</code></td>
<td><code>CrayCCE/19.06</code>, <code>CrayGNU/19.06</code>, <code>foss/2019b</code>, <code>intel/2019b</code>, <code>iomkl/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="castxml">CastXML<a class="headerlink" href="#castxml" title="Permanent link">&para;</a></h3>
<p>CastXML is a C-family abstract syntax tree XML output tool.</p>
<p><em>homepage</em>: <a href="https://github.com/CastXML/CastXML">https://github.com/CastXML/CastXML</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20160617</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20180806</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="cat-bat">CAT-BAT<a class="headerlink" href="#cat-bat" title="Permanent link">&para;</a></h3>
<p>Tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs).</p>
<p><em>homepage</em>: <a href="https://github.com/dutilh/CAT">https://github.com/dutilh/CAT</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.3</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="catch2">Catch2<a class="headerlink" href="#catch2" title="Permanent link">&para;</a></h3>
<p>A modern, C++-native, header-only,   test framework for unit-tests, TDD and BDD   - using C++11, C++14, C++17 and later     (or C++03 on the Catch1.x branch) </p>
<p><em>homepage</em>: <a href="https://github.com/catchorg/Catch2">https://github.com/catchorg/Catch2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.13.4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="category_encoders">category_encoders<a class="headerlink" href="#category_encoders" title="Permanent link">&para;</a></h3>
<p>A set of scikit-learn-style transformers for encoding categorical variables into numeric by means of different techniques.</p>
<p><em>homepage</em>: <a href="https://github.com/scikit-learn-contrib/category_encoders">https://github.com/scikit-learn-contrib/category_encoders</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="catlearn">CatLearn<a class="headerlink" href="#catlearn" title="Permanent link">&para;</a></h3>
<p>An environment for atomistic machine learning in Python for applications in catalysis</p>
<p><em>homepage</em>: <a href="https://catlearn.readthedocs.io">https://catlearn.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.2</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="catmap">CatMAP<a class="headerlink" href="#catmap" title="Permanent link">&para;</a></h3>
<p>Catalyst Micro-kinetic Analysis Package for automated creation of micro-kinetic models used  in catalyst screening.</p>
<p><em>homepage</em>: <a href="https://catmap.readthedocs.io">https://catmap.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170927</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20220519</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="causallift">causallift<a class="headerlink" href="#causallift" title="Permanent link">&para;</a></h3>
<p>CausalLift: Python package for Uplift Modeling in real-world business; applicable for both A/B testing and observational data </p>
<p><em>homepage</em>: <a href="https://github.com/Minyus/causallift">https://github.com/Minyus/causallift</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="causalml">causalml<a class="headerlink" href="#causalml" title="Permanent link">&para;</a></h3>
<p>Causal ML: A Python Package for Uplift Modeling and Causal Inference with ML </p>
<p><em>homepage</em>: <a href="https://github.com/uber/causalml">https://github.com/uber/causalml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0-20180610</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.8.0-20200909</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="caveman">CaVEMan<a class="headerlink" href="#caveman" title="Permanent link">&para;</a></h3>
<p>SNV expectation maximisation based mutation calling algorithm   aimed at detecting somatic mutations in paired (tumour/normal) cancer samples.  Supports both bam and cram format via htslib</p>
<p><em>homepage</em>: <a href="http://cancerit.github.io/CaVEMan/">http://cancerit.github.io/CaVEMan/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13.2</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="caviar">CAVIAR<a class="headerlink" href="#caviar" title="Permanent link">&para;</a></h3>
<p>CAusal Variants Identication in Associated Regions. A statistical framework  that quantifies the probability of each variant to be causal while allowing an arbitrary  number of causal variants.</p>
<p><em>homepage</em>: <a href="https://github.com/fhormoz/caviar">https://github.com/fhormoz/caviar</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2-20190419</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="cbc">Cbc<a class="headerlink" href="#cbc" title="Permanent link">&para;</a></h3>
<p>Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming solver written in C++. It can be used as a callable library or using a stand-alone executable.</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/Cbc">https://github.com/coin-or/Cbc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.10.5</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cblas">CBLAS<a class="headerlink" href="#cblas" title="Permanent link">&para;</a></h3>
<p>C interface to the BLAS</p>
<p><em>homepage</em>: <a href="https://www.netlib.org/blas/">https://www.netlib.org/blas/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20110120</code></td>
<td><code>foss/2016b</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="ccache">ccache<a class="headerlink" href="#ccache" title="Permanent link">&para;</a></h3>
<p>Ccache (or ccache) is a compiler cache. It speeds up recompilation by caching previous compilations and detecting when the same compilation is being done again</p>
<p><em>homepage</em>: <a href="https://ccache.dev/">https://ccache.dev/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.4</code></td>
<td><code>-f90</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.6.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ccfits">CCfits<a class="headerlink" href="#ccfits" title="Permanent link">&para;</a></h3>
<p>CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++.</p>
<p><em>homepage</em>: <a href="https://heasarc.gsfc.nasa.gov/fitsio/CCfits/">https://heasarc.gsfc.nasa.gov/fitsio/CCfits/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ccl">CCL<a class="headerlink" href="#ccl" title="Permanent link">&para;</a></h3>
<p>Clozure CL (often called CCL for short) is a free Common Lisp  implementation with a long history. Some distinguishing features of the implementation  include fast compilation speed, native threads, a precise, generational, compacting  garbage collector, and a convenient foreign-function interface.</p>
<p><em>homepage</em>: <a href="https://ccl.clozure.com/">https://ccl.clozure.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cclib">cclib<a class="headerlink" href="#cclib" title="Permanent link">&para;</a></h3>
<p>cclib is a Python library that provides parsers for computational chemistry log files.  It also provides a platform to implement algorithms in a package-independent manner. </p>
<p><em>homepage</em>: <a href="https://cclib.github.io/">https://cclib.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="cctools">cctools<a class="headerlink" href="#cctools" title="Permanent link">&para;</a></h3>
<p>The Cooperating Computing Tools (CCTools) help you to design and deploy  scalable applications that run on hundreds or thousands of machines at once. </p>
<p><em>homepage</em>: <a href="https://ccl.cse.nd.edu/">https://ccl.cse.nd.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.22</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cd-hit">CD-HIT<a class="headerlink" href="#cd-hit" title="Permanent link">&para;</a></h3>
<p>CD-HIT is a very widely used program for clustering and  comparing protein or nucleotide sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/weizhongli/cdhit">https://github.com/weizhongli/cdhit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.4</code></td>
<td><code>-2015-0603</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>4.6.6</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>4.6.8</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018b</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="cdat">CDAT<a class="headerlink" href="#cdat" title="Permanent link">&para;</a></h3>
<p>CDAT is a powerful and complete front-end to a rich set of  visual-data exploration and analysis capabilities well suited for data analysis problems.</p>
<p><em>homepage</em>: <a href="https://github.com/CDAT/cdat">https://github.com/CDAT/cdat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="cdbfasta">cdbfasta<a class="headerlink" href="#cdbfasta" title="Permanent link">&para;</a></h3>
<p>Fasta file indexing and retrival tool</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/cdbfasta">https://sourceforge.net/projects/cdbfasta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="cdbtools">CDBtools<a class="headerlink" href="#cdbtools" title="Permanent link">&para;</a></h3>
<p>CDB (Constant DataBase) indexing and retrieval tools for FASTA files</p>
<p><em>homepage</em>: <a href="http://compbio.dfci.harvard.edu/tgi">http://compbio.dfci.harvard.edu/tgi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cddlib">cddlib<a class="headerlink" href="#cddlib" title="Permanent link">&para;</a></h3>
<p>An efficient implementation of the Double Description Method</p>
<p><em>homepage</em>: <a href="https://github.com/cddlib/cddlib">https://github.com/cddlib/cddlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.94i</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.94j</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cdflib">CDFlib<a class="headerlink" href="#cdflib" title="Permanent link">&para;</a></h3>
<p>cdflib is a python module to read/write CDF (Common Data Format .cdf) files without needing to install the CDF NASA library. </p>
<p><em>homepage</em>: <a href="https://github.com/MAVENSDC/cdflib">https://github.com/MAVENSDC/cdflib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.9</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="cdna_cupcake">cDNA_Cupcake<a class="headerlink" href="#cdna_cupcake" title="Permanent link">&para;</a></h3>
<p>cDNA_Cupcake is a miscellaneous collection of Python and R scripts used for analyzing sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/Magdoll/cDNA_Cupcake">https://github.com/Magdoll/cDNA_Cupcake</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.8</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>24.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>26.0.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cdo">CDO<a class="headerlink" href="#cdo" title="Permanent link">&para;</a></h3>
<p>CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.</p>
<p><em>homepage</em>: <a href="https://code.zmaw.de/projects/cdo">https://code.zmaw.de/projects/cdo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>1.9.8</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.9.10</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td><code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cdo-bindings">cdo-bindings<a class="headerlink" href="#cdo-bindings" title="Permanent link">&para;</a></h3>
<p>Python interface to CDO.</p>
<p><em>homepage</em>: <a href="https://github.com/Try2Code/cdo-bindings">https://github.com/Try2Code/cdo-bindings</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.7</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="cdsapi">cdsapi<a class="headerlink" href="#cdsapi" title="Permanent link">&para;</a></h3>
<p>Climate Data Store API</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/cdsapi">https://pypi.org/project/cdsapi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.4</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.1.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.3.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.5.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cell2location">cell2location<a class="headerlink" href="#cell2location" title="Permanent link">&para;</a></h3>
<p>Comprehensive mapping of tissue cell architecture via integrated  single cell and spatial transcriptomics (cell2location model)</p>
<p><em>homepage</em>: <a href="https://github.com/BayraktarLab/cell2location/">https://github.com/BayraktarLab/cell2location/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.05-alpha</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cellmix">CellMix<a class="headerlink" href="#cellmix" title="Permanent link">&para;</a></h3>
<p>A Comprehensive Toolbox for Gene Expression Deconvolution</p>
<p><em>homepage</em>: <a href="http://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix">http://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.2</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="cellpose">Cellpose<a class="headerlink" href="#cellpose" title="Permanent link">&para;</a></h3>
<p>a generalist algorithm for cellular segmentation</p>
<p><em>homepage</em>: <a href="https://www.cellpose.org/">https://www.cellpose.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cellranger">CellRanger<a class="headerlink" href="#cellranger" title="Permanent link">&para;</a></h3>
<p>Cell Ranger is a set of analysis pipelines that process Chromium  single-cell RNA-seq output to align reads, generate gene-cell matrices and perform  clustering and gene expression analysis.</p>
<p><em>homepage</em>: <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cellranger-arc">CellRanger-ARC<a class="headerlink" href="#cellranger-arc" title="Permanent link">&para;</a></h3>
<p>Cell Ranger ARC is a set of analysis pipelines that process  Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a  variety of analyses pertaining to gene expression, chromatin accessibility and  their linkage. Furthermore, since the ATAC and gene expression measurements are on  the very same cell, we are able to perform analyses that link chromatin  accessibility and gene expression.</p>
<p><em>homepage</em>: <a href="https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc">https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cellranger-atac">CellRanger-ATAC<a class="headerlink" href="#cellranger-atac" title="Permanent link">&para;</a></h3>
<p>Cell Ranger ATAC is a set of analysis pipelines that process  Chromium Single Cell ATAC data.</p>
<p><em>homepage</em>: <a href="https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac">https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cellrank">CellRank<a class="headerlink" href="#cellrank" title="Permanent link">&para;</a></h3>
<p>CellRank is a toolkit to uncover cellular dynamics based on  Markov state modeling of single-cell data.  It contains two main modules: kernels compute cell-cell transition probabilities and estimators generate hypothesis based on these. </p>
<p><em>homepage</em>: <a href="https://cellrank.readthedocs.io/en/stable/">https://cellrank.readthedocs.io/en/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="celltypist">CellTypist<a class="headerlink" href="#celltypist" title="Permanent link">&para;</a></h3>
<p>A tool for semi-automatic cell type annotation</p>
<p><em>homepage</em>: <a href="https://www.celltypist.org/">https://www.celltypist.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="centrifuge">Centrifuge<a class="headerlink" href="#centrifuge" title="Permanent link">&para;</a></h3>
<p>Classifier for metagenomic sequences</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/centrifuge">https://ccb.jhu.edu/software/centrifuge</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>1.0.4-beta</code></td>
<td><code>foss/2018b</code>, <code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="cereal">Cereal<a class="headerlink" href="#cereal" title="Permanent link">&para;</a></h3>
<p>cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly  turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or  used standalone.</p>
<p><em>homepage</em>: <a href="https://uscilab.github.io/cereal/">https://uscilab.github.io/cereal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cesm-deps">CESM-deps<a class="headerlink" href="#cesm-deps" title="Permanent link">&para;</a></h3>
<p>CESM is a fully-coupled, community, global climate model that provides state-of-the-art  computer simulations of the Earth's past, present, and future climate states.</p>
<p><em>homepage</em>: <a href="https://www.cesm.ucar.edu/models/cesm2/">https://www.cesm.ucar.edu/models/cesm2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2</code></td>
<td><code>foss/2018b</code>, <code>foss/2021b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="cfdemcoupling">CFDEMcoupling<a class="headerlink" href="#cfdemcoupling" title="Permanent link">&para;</a></h3>
<p>CFDEMcoupling is an open source CFD-DEM engine. It provides the possibility to couple  the DEM engine LIGGGHTS to a CFD framework.</p>
<p><em>homepage</em>: <a href="https://www.cfdem.com/cfdemrcoupling-open-source-cfd-dem-framework">https://www.cfdem.com/cfdemrcoupling-open-source-cfd-dem-framework</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="cfitsio">CFITSIO<a class="headerlink" href="#cfitsio" title="Permanent link">&para;</a></h3>
<p>CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.</p>
<p><em>homepage</em>: <a href="https://heasarc.gsfc.nasa.gov/fitsio/">https://heasarc.gsfc.nasa.gov/fitsio/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.38</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.41</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.42</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.45</code></td>
<td><code>GCCcore/7.3.0</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.47</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.48</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.49</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cftime">cftime<a class="headerlink" href="#cftime" title="Permanent link">&para;</a></h3>
<p>Time-handling functionality from netcdf4-python</p>
<p><em>homepage</em>: <a href="https://github.com/Unidata/cftime">https://github.com/Unidata/cftime</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.0b1</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="cgal">CGAL<a class="headerlink" href="#cgal" title="Permanent link">&para;</a></h3>
<p>The goal of the CGAL Open Source Project is to provide easy access to efficient  and reliable geometric algorithms in the form of a C++ library.</p>
<p><em>homepage</em>: <a href="https://www.cgal.org/">https://www.cgal.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.9</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.9</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.14</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>4.14.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.14.3</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>iimpi/2021a</code></td>
</tr>
<tr>
<td><code>4.14.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>5.2</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cget">cget<a class="headerlink" href="#cget" title="Permanent link">&para;</a></h3>
<p>Cmake package retrieval. This can be used to download and install cmake packages</p>
<p><em>homepage</em>: <a href="https://cget.readthedocs.io/en/latest/index.html">https://cget.readthedocs.io/en/latest/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="cgl">Cgl<a class="headerlink" href="#cgl" title="Permanent link">&para;</a></h3>
<p>The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that can be used with other COIN-OR packages that make use of cuts, such as, among others, the linear solver Clp or the mixed integer linear programming solvers Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or communicate with a solver. It does not directly call a solver.</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/Cgl">https://github.com/coin-or/Cgl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.60.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.60.3</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cgmaptools">CGmapTools<a class="headerlink" href="#cgmaptools" title="Permanent link">&para;</a></h3>
<p>Command-line Toolset for Bisulfite Sequencing Data Analysis</p>
<p><em>homepage</em>: <a href="https://cgmaptools.github.io/">https://cgmaptools.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="cgns">CGNS<a class="headerlink" href="#cgns" title="Permanent link">&para;</a></h3>
<p>The CGNS system is designed to facilitate the exchange of data between sites and applications,  and to help stabilize the archiving of aerodynamic data.</p>
<p><em>homepage</em>: <a href="https://cgns.github.io/">https://cgns.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>intelcuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="charls">CharLS<a class="headerlink" href="#charls" title="Permanent link">&para;</a></h3>
<p>CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000 compression ratios.</p>
<p><em>homepage</em>: <a href="https://github.com/team-charls/charls">https://github.com/team-charls/charls</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="chase">CHASE<a class="headerlink" href="#chase" title="Permanent link">&para;</a></h3>
<p>Case-control HAplotype Sharing analyses.   Haplotype sharing analyses for genome-wide association studies.</p>
<p><em>homepage</em>: <a href="http://people.duke.edu/~asallen/Software.html">http://people.duke.edu/~asallen/Software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20130626</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="check">Check<a class="headerlink" href="#check" title="Permanent link">&para;</a></h3>
<p>Check is a unit testing framework for C. It features a simple interface for defining unit tests, putting little in the way of the developer. Tests are run in a separate address space, so both assertion failures and code errors that cause segmentation faults or other signals can be caught. Test results are reportable in the following: Subunit, TAP, XML, and a generic logging format.</p>
<p><em>homepage</em>: <a href="https://libcheck.github.io/check/">https://libcheck.github.io/check/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.15.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="checkm">CheckM<a class="headerlink" href="#checkm" title="Permanent link">&para;</a></h3>
<p>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,  single cells, or metagenomes.</p>
<p><em>homepage</em>: <a href="https://github.com/Ecogenomics/CheckM">https://github.com/Ecogenomics/CheckM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.13</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.13</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.13</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.18</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.0.18</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="checkm-database">CheckM-Database<a class="headerlink" href="#checkm-database" title="Permanent link">&para;</a></h3>
<p>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.  This is the corresponding database. </p>
<p><em>homepage</em>: <a href="https://github.com/Ecogenomics/CheckM">https://github.com/Ecogenomics/CheckM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2015_01_16</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cheetah">Cheetah<a class="headerlink" href="#cheetah" title="Permanent link">&para;</a></h3>
<p>Cheetah is an open source template engine and code generation tool.</p>
<p><em>homepage</em>: <a href="https://cheetahtemplate.org">https://cheetahtemplate.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="chemaxon-marvin">Chemaxon-Marvin<a class="headerlink" href="#chemaxon-marvin" title="Permanent link">&para;</a></h3>
<p>Marvin suite is a chemically intelligent desktop toolkit built to help you draw, edit, publish, render, import and export your chemical structures and as well as allowing you to convert between various chemical and graphical file formats. It is free for individual, academic and non-commercial use.</p>
<p><em>homepage</em>: <a href="https://chemaxon.com/products/marvin">https://chemaxon.com/products/marvin</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>21.14</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="chemprop">chemprop<a class="headerlink" href="#chemprop" title="Permanent link">&para;</a></h3>
<p>Message Passing Neural Networks for Molecule Property Prediction</p>
<p><em>homepage</em>: <a href="https://github.com/chemprop/chemprop">https://github.com/chemprop/chemprop</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="chemps2">CheMPS2<a class="headerlink" href="#chemps2" title="Permanent link">&para;</a></h3>
<p>CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.</p>
<p><em>homepage</em>: <a href="https://github.com/SebWouters/CheMPS2">https://github.com/SebWouters/CheMPS2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.7-rc2</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.8</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.8.9</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2018b</code>, <code>intel/2019a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.8.11</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.8.12</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="cherab">CHERAB<a class="headerlink" href="#cherab" title="Permanent link">&para;</a></h3>
<p>CHERAB is a python library for forward modelling diagnostics  based on spectroscopic plasma emission.</p>
<p><em>homepage</em>: <a href="https://cherab.github.io/documentation/index.html">https://cherab.github.io/documentation/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="chewbbaca">chewBBACA<a class="headerlink" href="#chewbbaca" title="Permanent link">&para;</a></h3>
<p>chewBBACA stands for "BSR-Based Allele Calling Algorithm". chewBBACA is a comprehensive pipeline including a set of functions for the creation and validation of whole genome and core genome MultiLocus Sequence Typing (wg/cgMLST) schemas, providing an allele calling algorithm based on Blast Score Ratio that can be run in multiprocessor settings and a set of functions to visualize and validate allele variation in the loci.</p>
<p><em>homepage</em>: <a href="https://github.com/B-UMMI/chewBBACA">https://github.com/B-UMMI/chewBBACA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="chi2comb">chi2comb<a class="headerlink" href="#chi2comb" title="Permanent link">&para;</a></h3>
<p>Cumulative density function of linear combinations of independent chi-square random variables and a standard Normal distribution. As of now, this is basically a repackaging of the davies function implemented in the CompQuadForm library for R.</p>
<p><em>homepage</em>: <a href="https://github.com/limix/chi2comb">https://github.com/limix/chi2comb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.3</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="chimera">Chimera<a class="headerlink" href="#chimera" title="Permanent link">&para;</a></h3>
<p>UCSF Chimera is a highly extensible program for interactive visualization   and analysis of molecular structures and related data, including density maps, supramolecular   assemblies, sequence alignments, docking results, trajectories, and conformational ensembles. </p>
<p><em>homepage</em>: <a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10</code></td>
<td><code>-linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="chimpipe">ChimPipe<a class="headerlink" href="#chimpipe" title="Permanent link">&para;</a></h3>
<p>ChimPipe is a computational method for the detection of novel transcription-induced  chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction  spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair  resolution.</p>
<p><em>homepage</em>: <a href="https://chimpipe.readthedocs.org/">https://chimpipe.readthedocs.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.5</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.9.5</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="chromaprint">Chromaprint<a class="headerlink" href="#chromaprint" title="Permanent link">&para;</a></h3>
<p>Chromaprint is the core component of the AcoustID project. It's a client-side library  that implements a custom algorithm for extracting fingerprints from any audio source.</p>
<p><em>homepage</em>: <a href="https://acoustid.org/chromaprint">https://acoustid.org/chromaprint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cicero">cicero<a class="headerlink" href="#cicero" title="Permanent link">&para;</a></h3>
<p>Cicero is an R package that provides tools for analyzing single-cell chromatin accessibility  experiments.</p>
<p><em>homepage</em>: <a href="https://cole-trapnell-lab.github.io/cicero-release">https://cole-trapnell-lab.github.io/cicero-release</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.4.11</code></td>
<td><code>-R-4.0.3-Monocle3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cif2cell">CIF2Cell<a class="headerlink" href="#cif2cell" title="Permanent link">&para;</a></h3>
<p>CIF2Cell is a tool to generate the geometrical setup for various electronic structure codes from a CIF (Crystallographic Information Framework) file. The program currently supports output for a number of popular electronic structure programs, including ABINIT, ASE, CASTEP, CP2K, CPMD, CRYSTAL09, Elk, EMTO, Exciting, Fleur, FHI-aims, Hutsepot, MOPAC, Quantum Espresso, RSPt, Siesta, SPR-KKR, VASP. Also exports some related formats like .coo, .cfg and .xyz-files.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/cif2cell">https://sourceforge.net/projects/cif2cell</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.10</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cimfomfa">cimfomfa<a class="headerlink" href="#cimfomfa" title="Permanent link">&para;</a></h3>
<p>This library supports both MCL, a cluster algorithm for graphs, and zoem, a macro/DSL language. It supplies abstractions for memory management, I/O, associative arrays, strings, heaps, and a few other things. The string library has had heavy testing as part of zoem. Both understandably and regrettably I chose long ago to make it C-string-compatible, hence nul bytes may not be part of a string. At some point I hope to rectify this, perhaps unrealistically.</p>
<p><em>homepage</em>: <a href="https://github.com/micans/cimfomfa">https://github.com/micans/cimfomfa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>22.273</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="circexplorer">CIRCexplorer<a class="headerlink" href="#circexplorer" title="Permanent link">&para;</a></h3>
<p>CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.</p>
<p><em>homepage</em>: <a href="https://circexplorer2.readthedocs.io/">https://circexplorer2.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.10</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="circexplorer2">CIRCexplorer2<a class="headerlink" href="#circexplorer2" title="Permanent link">&para;</a></h3>
<p>CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.</p>
<p><em>homepage</em>: <a href="http://circexplorer2.readthedocs.io/">http://circexplorer2.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.8</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="circos">Circos<a class="headerlink" href="#circos" title="Permanent link">&para;</a></h3>
<p>Circos is a software package for visualizing data and information.  It visualizes data in a circular layout - this makes Circos ideal for exploring  relationships between objects or positions.</p>
<p><em>homepage</em>: <a href="http://www.circos.ca/">http://www.circos.ca/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.69-5</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.69-6</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.69-6</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.69-9</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ciri">CIRI<a class="headerlink" href="#ciri" title="Permanent link">&para;</a></h3>
<p>CircRNA Identifier. A de novo circular RNA identification tool</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/ciri/">https://sourceforge.net/projects/ciri/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.6</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="ciri-long">CIRI-long<a class="headerlink" href="#ciri-long" title="Permanent link">&para;</a></h3>
<p>Circular RNA Identification for Long-Reads Nanopore Sequencing Data</p>
<p><em>homepage</em>: <a href="https://github.com/bioinfo-biols/CIRI-long/">https://github.com/bioinfo-biols/CIRI-long/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cistem">cisTEM<a class="headerlink" href="#cistem" title="Permanent link">&para;</a></h3>
<p>cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes  and obtain high-resolution 3D reconstructions from them. </p>
<p><em>homepage</em>: <a href="https://cistem.org/">https://cistem.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0-beta</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="cite-seq-count">CITE-seq-Count<a class="headerlink" href="#cite-seq-count" title="Permanent link">&para;</a></h3>
<p>A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.</p>
<p><em>homepage</em>: <a href="https://github.com/Hoohm/CITE-seq-Count">https://github.com/Hoohm/CITE-seq-Count</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="clang">Clang<a class="headerlink" href="#clang" title="Permanent link">&para;</a></h3>
<p>C, C++, Objective-C compiler, based on LLVM.  Does not  include C++ standard library -- use libstdc++ from GCC.</p>
<p><em>homepage</em>: <a href="https://clang.llvm.org/">https://clang.llvm.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3</code></td>
<td></td>
<td><code>GCC/4.8.1</code></td>
</tr>
<tr>
<td><code>3.4</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>7.0.1</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td><code>-CUDA-10.1.105</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-CUDA-10.1.105</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-CUDA-10.1.243</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td><code>-CUDA-10.1.243</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>10.0.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>10.0.1</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>11.0.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>11.0.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>gcccuda/2020b</code></td>
</tr>
<tr>
<td><code>12.0.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>12.0.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>13.0.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>13.0.1</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="clang-aomp">Clang-AOMP<a class="headerlink" href="#clang-aomp" title="Permanent link">&para;</a></h3>
<p>AOMP is an open source Clang/LLVM based compiler with added support for the OpenMP API on Radeon GPUs.</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/llvm-project">https://github.com/RadeonOpenCompute/llvm-project</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="clang-python-bindings">Clang-Python-bindings<a class="headerlink" href="#clang-python-bindings" title="Permanent link">&para;</a></h3>
<p>Python bindings for libclang</p>
<p><em>homepage</em>: <a href="https://clang.llvm.org">https://clang.llvm.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>10.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>13.0.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="clapack">CLAPACK<a class="headerlink" href="#clapack" title="Permanent link">&para;</a></h3>
<p>C version of LAPACK</p>
<p><em>homepage</em>: <a href="http://www.netlib.org/clapack">http://www.netlib.org/clapack</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="clease">CLEASE<a class="headerlink" href="#clease" title="Permanent link">&para;</a></h3>
<p>CLuster Expansion in Atomic Simulation Environment (CLEASE) is a package that automates the cumbersome setup and construction procedure of cluster expansion (CE). It provides a comprehensive list of tools for specifying parameters for CE, generating training structures, fitting effective cluster interaction (ECI) values and running Monte Carlo simulations.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/computationalmaterials/clease">https://gitlab.com/computationalmaterials/clease</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.6</code></td>
<td><code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="clhep">CLHEP<a class="headerlink" href="#clhep" title="Permanent link">&para;</a></h3>
<p>The CLHEP project is intended to be a set of HEP-specific foundation and  utility classes such as random generators, physics vectors, geometry and linear algebra.  CLHEP is structured in a set of packages independent of any external package.</p>
<p><em>homepage</em>: <a href="https://proj-clhep.web.cern.ch/proj-clhep/">https://proj-clhep.web.cern.ch/proj-clhep/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1.0</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.1.3.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.2.0.8</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.1.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.4.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.0.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.1.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.1.3</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.4.4.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.4.5.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="climetlab">CliMetLab<a class="headerlink" href="#climetlab" title="Permanent link">&para;</a></h3>
<p>CliMetLab is a Python package aiming at simplifying access to climate and meteorological datasets, allowing users to focus on science instead of technical issues such as data access and data formats. It is mostly intended to be used in Jupyter notebooks, and be interoperable with all popular data analytic packages, such as Numpy, Pandas, Xarray, SciPy, Matplotlib, etc. as well as machine learning frameworks, such as Tensorflow, Keras or PyTorch.</p>
<p><em>homepage</em>: <a href="https://climetlab.readthedocs.io/">https://climetlab.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.6</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="clisp">CLISP<a class="headerlink" href="#clisp" title="Permanent link">&para;</a></h3>
<p>Common Lisp is a high-level, general-purpose, object-oriented, dynamic,  functional programming language. </p>
<p><em>homepage</em>: <a href="https://clisp.sourceforge.io/">https://clisp.sourceforge.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.49</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="clonalframeml">ClonalFrameML<a class="headerlink" href="#clonalframeml" title="Permanent link">&para;</a></h3>
<p>Efficient Inference of Recombination in Whole Bacterial Genomes</p>
<p><em>homepage</em>: <a href="https://github.com/xavierdidelot/ClonalFrameML">https://github.com/xavierdidelot/ClonalFrameML</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="cloog">CLooG<a class="headerlink" href="#cloog" title="Permanent link">&para;</a></h3>
<p>CLooG is a free software and library to generate code for scanning Z-polyhedra. That is, it finds a  code (e.g. in C, FORTRAN...) that reaches each integral point of one or more parameterized polyhedra. CLooG has been  originally written to solve the code generation problem for optimizing compilers based on the polytope model.  Nevertheless it is used now in various area e.g. to build control automata for high-level synthesis or to find the  best polynomial approximation of a function. CLooG may help in any situation where scanning polyhedra matters. While  the user has full control on generated code quality, CLooG is designed to avoid control overhead and to produce a  very effective code.</p>
<p><em>homepage</em>: <a href="http://www.bastoul.net/cloog/index.php">http://www.bastoul.net/cloog/index.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.1</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="cloudcompare">CloudCompare<a class="headerlink" href="#cloudcompare" title="Permanent link">&para;</a></h3>
<p>3D point cloud and mesh processing software</p>
<p><em>homepage</em>: <a href="https://www.cloudcompare.org/">https://www.cloudcompare.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.12.4</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="clp">Clp<a class="headerlink" href="#clp" title="Permanent link">&para;</a></h3>
<p>Clp (Coin-or linear programming) is an open-source linear programming solver. It is primarily meant to be used as a callable library, but a basic, stand-alone executable version is also available.</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/Clp">https://github.com/coin-or/Clp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.17.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.17.6</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.17.7</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="clustal-omega">Clustal-Omega<a class="headerlink" href="#clustal-omega" title="Permanent link">&para;</a></h3>
<p>Clustal Omega is a multiple sequence alignment  program for proteins. It produces biologically meaningful multiple  sequence alignments of divergent sequences. Evolutionary relationships  can be seen via viewing Cladograms or Phylograms </p>
<p><em>homepage</em>: <a href="http://www.clustal.org/omega/">http://www.clustal.org/omega/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/8.3.0</code>, <code>foss/2018b</code>, <code>intel-compilers/2021.2.0</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="clustalw2">ClustalW2<a class="headerlink" href="#clustalw2" title="Permanent link">&para;</a></h3>
<p>ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.</p>
<p><em>homepage</em>: <a href="https://www.ebi.ac.uk/Tools/msa/clustalw2/">https://www.ebi.ac.uk/Tools/msa/clustalw2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>foss/2021a</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2017b</code>, <code>intel/2018b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="cluster-buster">Cluster-Buster<a class="headerlink" href="#cluster-buster" title="Permanent link">&para;</a></h3>
<p>Cluster-Buster: Finding dense clusters of motifs in DNA sequences</p>
<p><em>homepage</em>: <a href="https://github.com/weng-lab/cluster-buster/">https://github.com/weng-lab/cluster-buster/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160106</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="clustershell">ClusterShell<a class="headerlink" href="#clustershell" title="Permanent link">&para;</a></h3>
<p>ClusterShell is an event-driven open source Python library, designed to run local or distant commands  in parallel on server farms or on large Linux clusters.</p>
<p><em>homepage</em>: <a href="https://cea-hpc.github.io/clustershell/">https://cea-hpc.github.io/clustershell/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cmake">CMake<a class="headerlink" href="#cmake" title="Permanent link">&para;</a></h3>
<p>CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.</p>
<p><em>homepage</em>: <a href="https://www.cmake.org">https://www.cmake.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.11</code></td>
<td><code>GCC/4.8.1</code></td>
</tr>
<tr>
<td><code>2.8.12</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>GCC/4.9.2</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/4.9.2</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>GNU/4.9.3-2.25</code>, <code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>GCC/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>foss/2016a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.2.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td><code>GCCcore/6.3.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.9.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.9.5</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.10.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.10.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.10.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code></td>
</tr>
<tr>
<td><code>3.10.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.11.4</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.12.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.13.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.15.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.15.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.16.4</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.18.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.20.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.21.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.22.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.23.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.24.3</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cmaverse">CMAverse<a class="headerlink" href="#cmaverse" title="Permanent link">&para;</a></h3>
<p>The R package CMAverse provides a suite of functions for reproducible causal mediation analysis including cmdag for DAG visualization, cmest for statistical modeling and cmsens for sensitivity analysis.</p>
<p><em>homepage</em>: <a href="https://bs1125.github.io/CMAverse/">https://bs1125.github.io/CMAverse/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220112</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="cmdstanr">CmdStanR<a class="headerlink" href="#cmdstanr" title="Permanent link">&para;</a></h3>
<p>CmdStanR is a lightweight interface to Stan for R users</p>
<p><em>homepage</em>: <a href="https://mc-stan.org/cmdstanr">https://mc-stan.org/cmdstanr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="cmocean">cmocean<a class="headerlink" href="#cmocean" title="Permanent link">&para;</a></h3>
<p>This package contains colormaps for commonly-used  oceanographic variables. Most of the colormaps started from  matplotlib colormaps, but have now been adjusted using the viscm  tool to be perceptually uniform.</p>
<p><em>homepage</em>: <a href="https://github.com/matplotlib/cmocean">https://github.com/matplotlib/cmocean</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="cmseq">CMSeq<a class="headerlink" href="#cmseq" title="Permanent link">&para;</a></h3>
<p>CMSeq is a set of commands to provide an interface to .bam files for coverage and sequence consensus.</p>
<p><em>homepage</em>: <a href="https://github.com/SegataLab/cmseq/">https://github.com/SegataLab/cmseq/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="cnt-ilp">CNT-ILP<a class="headerlink" href="#cnt-ilp" title="Permanent link">&para;</a></h3>
<p>Integer Linear Program for the Copy-Number Tree Problem </p>
<p><em>homepage</em>: <a href="https://compbio.cs.brown.edu/projects/cnt-ilp/">https://compbio.cs.brown.edu/projects/cnt-ilp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20171031</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="cnvkit">CNVkit<a class="headerlink" href="#cnvkit" title="Permanent link">&para;</a></h3>
<p>A command-line toolkit and Python library for detecting copy  number variants and alterations genome-wide from high-throughput sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/etal/cnvkit">https://github.com/etal/cnvkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.9.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.9.6</code></td>
<td><code>-Python-3.7.2-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.9.8</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cnvnator">CNVnator<a class="headerlink" href="#cnvnator" title="Permanent link">&para;</a></h3>
<p>a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads </p>
<p><em>homepage</em>: <a href="https://github.com/abyzovlab/CNVnator">https://github.com/abyzovlab/CNVnator</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.3</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="co-phylog">Co-phylog<a class="headerlink" href="#co-phylog" title="Permanent link">&para;</a></h3>
<p>Co-phylog: an assembly-free phylogenomic approach for closely related organisms H Yi, L Jin Nucleic acids research 41 (7), e75-e75</p>
<p><em>homepage</em>: <a href="https://github.com/yhg926/co-phylog">https://github.com/yhg926/co-phylog</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20201012</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
</tbody>
</table>
<h3 id="cobrapy">COBRApy<a class="headerlink" href="#cobrapy" title="Permanent link">&para;</a></h3>
<p>COBRApy is a package for constraint-based modeling of metabolic networks.</p>
<p><em>homepage</em>: <a href="https://opencobra.github.io/cobrapy/">https://opencobra.github.io/cobrapy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.26.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cocoalib">CoCoALib<a class="headerlink" href="#cocoalib" title="Permanent link">&para;</a></h3>
<p>CoCoALib is a free GPL3 C++ library for doing Computations in Commutative Algebra.</p>
<p><em>homepage</em>: <a href="http://cocoa.dima.unige.it/cocoalib ">http://cocoa.dima.unige.it/cocoalib </a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99601</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.99700</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="codan">CodAn<a class="headerlink" href="#codan" title="Permanent link">&para;</a></h3>
<p>CodAn (Coding sequence Annotator) is a computational tool designed to characterize the CDS and UTR regions on transcripts from any Eukaryote species.</p>
<p><em>homepage</em>: <a href="https://github.com/pedronachtigall/CodAn">https://github.com/pedronachtigall/CodAn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="code-server">code-server<a class="headerlink" href="#code-server" title="Permanent link">&para;</a></h3>
<p>Run VS Code on any machine anywhere and access it in the browser.</p>
<p><em>homepage</em>: <a href="https://github.com/cdr/code-server">https://github.com/cdr/code-server</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="codex2">CODEX2<a class="headerlink" href="#codex2" title="Permanent link">&para;</a></h3>
<p>Full-spectrum copy number variation detection by high-throughput DNA sequencing</p>
<p><em>homepage</em>: <a href="https://github.com/yuchaojiang/CODEX2">https://github.com/yuchaojiang/CODEX2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180227</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="cogent">Cogent<a class="headerlink" href="#cogent" title="Permanent link">&para;</a></h3>
<p>Cogent is a tool for reconstructing the coding genome using high-quality full-length transcriptome sequences. It is designed to be used on Iso-Seq data and in cases where there is no reference genome or the ref genome is highly incomplete. </p>
<p><em>homepage</em>: <a href="https://github.com/Magdoll/Cogent">https://github.com/Magdoll/Cogent</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="coin">Coin<a class="headerlink" href="#coin" title="Permanent link">&para;</a></h3>
<p>Coin is an OpenGL-based, 3D graphics library that has its roots in the Open Inventor 2.1 API, which Coin still is compatible with.</p>
<p><em>homepage</em>: <a href="https://coin3d.github.io">https://coin3d.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="coinutils">CoinUtils<a class="headerlink" href="#coinutils" title="Permanent link">&para;</a></h3>
<p>CoinUtils (Coin-OR Utilities) is an open-source collection of classes and functions that are generally useful to more than one COIN-OR project.</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/CoinUtils">https://github.com/coin-or/CoinUtils</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.3</code></td>
<td><code>GCCcore/7.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.11.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.11.6</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="colossalai">colossalai<a class="headerlink" href="#colossalai" title="Permanent link">&para;</a></h3>
<p>Colossal-AI: A Unified Deep Learning System for Big Model Era</p>
<p><em>homepage</em>: <a href="https://colossalai.org/">https://colossalai.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.8</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="commet">Commet<a class="headerlink" href="#commet" title="Permanent link">&para;</a></h3>
<p>COMMET ("COmpare Multiple METagenomes") provides a global similarity overview between all datasets of  a large metagenomic project. </p>
<p><em>homepage</em>: <a href="https://colibread.inria.fr/software/commet/">https://colibread.inria.fr/software/commet/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20150415</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="comparem">CompareM<a class="headerlink" href="#comparem" title="Permanent link">&para;</a></h3>
<p>A toolbox for comparative genomics.</p>
<p><em>homepage</em>: <a href="https://github.com/dparks1134/CompareM">https://github.com/dparks1134/CompareM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.23</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.1.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="compress-raw-zlib">Compress-Raw-Zlib<a class="headerlink" href="#compress-raw-zlib" title="Permanent link">&para;</a></h3>
<p>Low-Level Interface to zlib or zlib-ng compression library</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Compress::Raw::Zlib">https://metacpan.org/pod/Compress::Raw::Zlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.202</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="comsol">COMSOL<a class="headerlink" href="#comsol" title="Permanent link">&para;</a></h3>
<p>COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems.  </p>
<p><em>homepage</em>: <a href="https://www.comsol.com">https://www.comsol.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.0.225</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="con3f">Con3F<a class="headerlink" href="#con3f" title="Permanent link">&para;</a></h3>
<p>Con3F is a Python package to read, manipulate and convert force field files</p>
<p><em>homepage</em>: <a href="https://github.ugent.be/CMM/con3f">https://github.ugent.be/CMM/con3f</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-20190329</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="conan">conan<a class="headerlink" href="#conan" title="Permanent link">&para;</a></h3>
<p>Decentralized, open-source (MIT), C/C++ package manager.</p>
<p><em>homepage</em>: <a href="https://conan.io">https://conan.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.58.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="concoct">CONCOCT<a class="headerlink" href="#concoct" title="Permanent link">&para;</a></h3>
<p>Clustering cONtigs with COverage and ComposiTion (CONCOCT) is a program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</p>
<p><em>homepage</em>: <a href="https://concoct.readthedocs.io">https://concoct.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="configparser">configparser<a class="headerlink" href="#configparser" title="Permanent link">&para;</a></h3>
<p>configparser is a Python library that brings the updated configparser from Python 3.5 to Python 2.6-3.5</p>
<p><em>homepage</em>: <a href="https://docs.python.org/3/library/configparser.html">https://docs.python.org/3/library/configparser.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="configurable-http-proxy">configurable-http-proxy<a class="headerlink" href="#configurable-http-proxy" title="Permanent link">&para;</a></h3>
<p>HTTP proxy for node.js including a REST API for updating the routing table.  Developed as a part of the Jupyter Hub multi-user server.</p>
<p><em>homepage</em>: <a href="https://github.com/jupyterhub/configurable-http-proxy">https://github.com/jupyterhub/configurable-http-proxy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>-nodejs-4.4.7</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-nodejs-8.9.4</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="conn">CONN<a class="headerlink" href="#conn" title="Permanent link">&para;</a></h3>
<p>CONN is an open-source Matlab/SPM-based cross-platform software  for the computation, display, and analysis of functional connectivity Magnetic  Resonance Imaging (fcMRI). CONN is used to analyze resting state data (rsfMRI)  as well as task-related designs. </p>
<p><em>homepage</em>: <a href="https://web.conn-toolbox.org/home">https://web.conn-toolbox.org/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>21a</code></td>
<td><code>-MATLAB-2021a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="connectomeworkbench">ConnectomeWorkbench<a class="headerlink" href="#connectomeworkbench" title="Permanent link">&para;</a></h3>
<p>Connectome Workbench is an open source, freely available visualization  and discovery tool used to map neuroimaging data, especially data generated by the  Human Connectome Project.</p>
<p><em>homepage</em>: <a href="https://www.humanconnectome.org/software/connectome-workbench">https://www.humanconnectome.org/software/connectome-workbench</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-rh_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="control-freec">Control-FREEC<a class="headerlink" href="#control-freec" title="Permanent link">&para;</a></h3>
<p>Copy number and genotype annotation from whole  genome and whole exome sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/BoevaLab/FREEC">https://github.com/BoevaLab/FREEC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.5</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>11.6</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cooler">cooler<a class="headerlink" href="#cooler" title="Permanent link">&para;</a></h3>
<p>Cooler is a support library for a storage format, also called cooler, used to store  genomic interaction data of any size, such as Hi-C contact matrices.</p>
<p><em>homepage</em>: <a href="https://open2c.github.io/cooler">https://open2c.github.io/cooler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="coordgenlibs">CoordgenLibs<a class="headerlink" href="#coordgenlibs" title="Permanent link">&para;</a></h3>
<p>Schrodinger-developed 2D Coordinate Generation</p>
<p><em>homepage</em>: <a href="https://github.com/schrodinger/coordgenlibs">https://github.com/schrodinger/coordgenlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2021a</code>, <code>gompi/2022a</code>, <code>iimpi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="coot">Coot<a class="headerlink" href="#coot" title="Permanent link">&para;</a></h3>
<p>Coot is for macromolecular model building, model completion  and validation, particularly suitable for protein modelling using X-ray data.</p>
<p><em>homepage</em>: <a href="http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot">http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>-binary-Linux-x86_64-rhel-6-python-gtk2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="core-counter">core-counter<a class="headerlink" href="#core-counter" title="Permanent link">&para;</a></h3>
<p>Tool to check available cores and OMP threads</p>
<p><em>homepage</em>: <a href="https://github.com/gjbex/core-counter">https://github.com/gjbex/core-counter</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="coreutils">Coreutils<a class="headerlink" href="#coreutils" title="Permanent link">&para;</a></h3>
<p>The GNU Core Utilities are the basic file, shell and text manipulation utilities of the GNU operating system. These are the core utilities which are expected to exist on every operating system. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/coreutils/">https://www.gnu.org/software/coreutils/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.23</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>8.27</code></td>
<td><code>GCCcore/5.4.0</code></td>
</tr>
<tr>
<td><code>8.29</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>8.32</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>9.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="corner">corner<a class="headerlink" href="#corner" title="Permanent link">&para;</a></h3>
<p>Make some beautiful corner plots.</p>
<p><em>homepage</em>: <a href="https://corner.readthedocs.io/en/latest/">https://corner.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="cosymlib">CoSymLib<a class="headerlink" href="#cosymlib" title="Permanent link">&para;</a></h3>
<p>Cosymlib is a python library for computing continuous symmetry &amp; shape measures (CSMs &amp; CShMs). Although its main aim is to provide simple and ready-to-use tools for the analysis of the symmetry &amp; shape of molecules, many of the procedures contained in cosymlib can be easily applied to any finite geometrical object defined by a set of vertices or a by mass distribution function.</p>
<p><em>homepage</em>: <a href="https://cosymlib.readthedocs.io">https://cosymlib.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.9</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="coverage">coverage<a class="headerlink" href="#coverage" title="Permanent link">&para;</a></h3>
<p>Coverage.py is a tool for measuring code coverage of Python programs.  It monitors your program, noting which parts of the code have been executed,  then analyzes the source to identify code that could have been executed but was not. </p>
<p><em>homepage</em>: <a href="https://coverage.readthedocs.io">https://coverage.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cowsay">cowsay<a class="headerlink" href="#cowsay" title="Permanent link">&para;</a></h3>
<p>Configurable talking characters in ASCII art</p>
<p><em>homepage</em>: <a href="https://github.com/tnalpgge/rank-amateur-cowsay">https://github.com/tnalpgge/rank-amateur-cowsay</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.04</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cp2k">CP2K<a class="headerlink" href="#cp2k" title="Permanent link">&para;</a></h3>
<p>CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular  simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different  methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and  classical pair and many-body potentials. </p>
<p><em>homepage</em>: <a href="https://www.cp2k.org/">https://www.cp2k.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td></td>
<td><code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>-psmp</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td><code>-psmp</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.1</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.1</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2018a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2020a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td><code>-psmp</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>8.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cpb">CPB<a class="headerlink" href="#cpb" title="Permanent link">&para;</a></h3>
<p>CPB is a novel two-step Pearson correlation based biclustering approach to mine genes that are co-regulated with   a given reference gene in order to discover genes that function in a common biological process.   In the first step, the algorithm identifies subsets of genes with high correlation, reducing false negatives   with a nonparametric filtering scheme.   In the second step, biclusters from multiple datasets are used to extract and rank gene correlation information.</p>
<p><em>homepage</em>: <a href="http://tda.gatech.edu/software/cpb/index.html">http://tda.gatech.edu/software/cpb/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11-4-2011</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="cplex">CPLEX<a class="headerlink" href="#cplex" title="Permanent link">&para;</a></h3>
<p>IBM ILOG CPLEX Optimizer's mathematical programming technology enables  analytical decision support for improving efficiency,  reducing costs, and increasing profitability.</p>
<p><em>homepage</em>: <a href="https://www.ibm.com/analytics/cplex-optimizer">https://www.ibm.com/analytics/cplex-optimizer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>12.9</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>12.10</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cppe">CPPE<a class="headerlink" href="#cppe" title="Permanent link">&para;</a></h3>
<p>CPPE is an open-source, light-weight C++ and Python library for Polarizable Embedding (PE)1,2 calculations. It provides an easy-to-use API to implement PE for ground-state self-consistent field (SCF) calculations and post-SCF methods. A convenient Python interface is also available.</p>
<p><em>homepage</em>: <a href="https://github.com/maxscheurer/cppe">https://github.com/maxscheurer/cppe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cppunit">CppUnit<a class="headerlink" href="#cppunit" title="Permanent link">&para;</a></h3>
<p>CppUnit is the C++ port of the famous JUnit framework for unit testing. </p>
<p><em>homepage</em>: <a href="https://freedesktop.org/wiki/Software/cppunit/">https://freedesktop.org/wiki/Software/cppunit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cppy">cppy<a class="headerlink" href="#cppy" title="Permanent link">&para;</a></h3>
<p>A small C++ header library which makes it easier to write Python extension modules. The primary feature is a PyObject smart pointer which automatically handles reference counting and provides convenience methods for performing common object operations.</p>
<p><em>homepage</em>: <a href="https://github.com/nucleic/cppy">https://github.com/nucleic/cppy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cpu_features">cpu_features<a class="headerlink" href="#cpu_features" title="Permanent link">&para;</a></h3>
<p>A cross-platform C library to retrieve CPU features (such as available instructions) at runtime.</p>
<p><em>homepage</em>: <a href="https://github.com/google/cpu_features">https://github.com/google/cpu_features</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cram">cram<a class="headerlink" href="#cram" title="Permanent link">&para;</a></h3>
<p>Cram is a functional testing framework for command line applications.</p>
<p><em>homepage</em>: <a href="https://bitheap.org/cram">https://bitheap.org/cram</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cramtools">cramtools<a class="headerlink" href="#cramtools" title="Permanent link">&para;</a></h3>
<p>CRAMTools is a set of Java tools and APIs for efficient compression of sequence  read data. Although this is intended as a stable version the code is released as early access. Parts of the CRAMTools are experimental and may not be supported in the future.</p>
<p><em>homepage</em>: <a href="https://github.com/enasequence/cramtools/">https://github.com/enasequence/cramtools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="craycce">CrayCCE<a class="headerlink" href="#craycce" title="Permanent link">&para;</a></h3>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: June 2019). </p>
<p><em>homepage</em>: <a href="https://pubs.cray.com/discover">https://pubs.cray.com/discover</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.06</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="craygnu">CrayGNU<a class="headerlink" href="#craygnu" title="Permanent link">&para;</a></h3>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: June 2019). </p>
<p><em>homepage</em>: <a href="https://pubs.cray.com/discover">https://pubs.cray.com/discover</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.06</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="crayintel">CrayIntel<a class="headerlink" href="#crayintel" title="Permanent link">&para;</a></h3>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: June 2019). </p>
<p><em>homepage</em>: <a href="https://pubs.cray.com/discover">https://pubs.cray.com/discover</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.06</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="craypgi">CrayPGI<a class="headerlink" href="#craypgi" title="Permanent link">&para;</a></h3>
<p>Toolchain using Cray compiler wrapper, PrgEnv-pgi compiler (PE release: June 2019). </p>
<p><em>homepage</em>: <a href="https://pubs.cray.com/discover">https://pubs.cray.com/discover</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.06</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="crb-blast">crb-blast<a class="headerlink" href="#crb-blast" title="Permanent link">&para;</a></h3>
<p>Conditional Reciprocal Best BLAST - high confidence ortholog assignment.  CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.  This is particularly useful in genome and transcriptome annotation.</p>
<p><em>homepage</em>: <a href="https://github.com/cboursnell/crb-blast">https://github.com/cboursnell/crb-blast</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.9</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.6.9</code></td>
<td><code>-Ruby-2.6.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="crest">CREST<a class="headerlink" href="#crest" title="Permanent link">&para;</a></h3>
<p>CREST is an utility/driver program for the xtb program. Originally it was designed  as conformer sampling program, hence the abbreviation ConformerRotamer Ensemble Sampling Tool,  but now offers also some utility functions for calculations with the GFNnxTB methods. Generally  the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb  program) and tool for the creation and analysation of structure ensembles. </p>
<p><em>homepage</em>: <a href="https://xtb-docs.readthedocs.io/en/latest/crest.html">https://xtb-docs.readthedocs.io/en/latest/crest.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="crf">CRF++<a class="headerlink" href="#crf" title="Permanent link">&para;</a></h3>
<p>CRF++ is a simple, customizable, and open source implementation of  Conditional Random Fields (CRFs) for segmenting/labeling sequential data. CRF++ is  designed for generic purpose and will be applied to a variety of NLP tasks, such as  Named Entity Recognition, Information Extraction and Text Chunking. </p>
<p><em>homepage</em>: <a href="https://taku910.github.io/crfpp/">https://taku910.github.io/crfpp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.58</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="crispr-dav">CRISPR-DAV<a class="headerlink" href="#crispr-dav" title="Permanent link">&para;</a></h3>
<p>CRISPR-DAV is a pipeline to analyze amplicon-based NGS data of CRISPR clones in a high throughput manner.</p>
<p><em>homepage</em>: <a href="https://github.com/pinetree1/crispr-dav/">https://github.com/pinetree1/crispr-dav/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="crispresso2">CRISPResso2<a class="headerlink" href="#crispresso2" title="Permanent link">&para;</a></h3>
<p>CRISPResso2 is a software pipeline designed to enable rapid and intuitive interpretation of genome editing experiments.</p>
<p><em>homepage</em>: <a href="https://github.com/pinellolab/CRISPResso2/">https://github.com/pinellolab/CRISPResso2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.44</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="cromwell">cromwell<a class="headerlink" href="#cromwell" title="Permanent link">&para;</a></h3>
<p>Cromwell is a Workflow Management System geared towards scientific workflows.</p>
<p><em>homepage</em>: <a href="http://cromwell.readthedocs.io/">http://cromwell.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>56</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="crossguid">crossguid<a class="headerlink" href="#crossguid" title="Permanent link">&para;</a></h3>
<p>CrossGuid is a minimal, cross platform, C++ GUID library.  It uses the best native GUID/UUID generator on the given platform and has a  generic class for parsing, stringifying, and comparing IDs.  The guid generation technique is determined by your platform:</p>
<p><em>homepage</em>: <a href="https://github.com/graeme-hill/crossguid">https://github.com/graeme-hill/crossguid</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190529</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="crossmap">CrossMap<a class="headerlink" href="#crossmap" title="Permanent link">&para;</a></h3>
<p>CrossMap is a program for genome coordinates conversion  between different assemblies (such as hg18 (NCBI36) &lt;=&gt; hg19 (GRCh37)).  It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,  BigWig, BED, GFF, GTF and VCF.</p>
<p><em>homepage</em>: <a href="http://crossmap.sourceforge.net">http://crossmap.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.9</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="crosstalkz">CrossTalkZ<a class="headerlink" href="#crosstalkz" title="Permanent link">&para;</a></h3>
<p>CrossTalkZ is a statistical method and software to assess the significance of crosstalk enrichment  between pairs of gene or protein groups in large biological networks. </p>
<p><em>homepage</em>: <a href="http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/">http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="crpropa">CRPropa<a class="headerlink" href="#crpropa" title="Permanent link">&para;</a></h3>
<p>CRPropa is a publicly available code to study the propagation of ultra high energy nuclei up to iron  on their voyage through an extra galactic environment.</p>
<p><em>homepage</em>: <a href="https://crpropa.desy.de">https://crpropa.desy.de</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.1.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="crumble">Crumble<a class="headerlink" href="#crumble" title="Permanent link">&para;</a></h3>
<p>Exploration of controlled loss of quality values for compressing CRAM files</p>
<p><em>homepage</em>: <a href="https://github.com/jkbonfield/crumble">https://github.com/jkbonfield/crumble</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cryocare">cryoCARE<a class="headerlink" href="#cryocare" title="Permanent link">&para;</a></h3>
<p>This package is a memory efficient implementation of cryoCARE.  This setup trains a denoising U-Net for tomographic reconstruction according to   the Noise2Noise training paradigm. Therefore the user has to provide two        tomograms of the same sample. The simplest way to achieve this is with direct-   detector movie-frames.  You can use Warp to generate two reconstructed tomograms based on the even/odd    frames. Alternatively, the movie-frames can be split in two halves (e.g. with    MotionCor2 -SplitSum 1 or with IMOD alignframes -debug 10000) from which two     identical, up to random noise, tomograms can be reconstructed.  These two (even and odd) tomograms can be used as input to this cryoCARE         implementation.</p>
<p><em>homepage</em>: <a href="https://github.com/juglab/cryoCARE_pip">https://github.com/juglab/cryoCARE_pip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cryodrgn">cryoDRGN<a class="headerlink" href="#cryodrgn" title="Permanent link">&para;</a></h3>
<p>cryoDRGN: Deep Reconstructing Generative Networks for cryo-EM heterogeneous  reconstruction.  CryoDRGN is a neural network based algorithm for heterogeneous cryo-EM reconstruction. In  particular, the method models a continuous distribution over 3D structures by using a neural  network based representation for the volume. </p>
<p><em>homepage</em>: <a href="https://cb.csail.mit.edu/cb/cryodrgn/">https://cb.csail.mit.edu/cb/cryodrgn/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.3.5</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.0-beta</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cryptominisat">CryptoMiniSat<a class="headerlink" href="#cryptominisat" title="Permanent link">&para;</a></h3>
<p>CryptoMiniSat is an advanced SAT solver</p>
<p><em>homepage</em>: <a href="https://github.com/msoos/cryptominisat">https://github.com/msoos/cryptominisat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="crystfel">CrystFEL<a class="headerlink" href="#crystfel" title="Permanent link">&para;</a></h3>
<p>CrystFEL is a suite of programs for processing diffraction data acquired "serially" in a "snapshot" manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser source.</p>
<p><em>homepage</em>: <a href="http://www.desy.de/~twhite/crystfel/">http://www.desy.de/~twhite/crystfel/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="csb">CSB<a class="headerlink" href="#csb" title="Permanent link">&para;</a></h3>
<p>Computational Structural Biology Toolbox CSB is a python library and application framework, which can be used to solve    problems in the field of structural bioinformatics. If you are a                 bioinformatician, software engineer or a researcher working in this field,       chances are you may find something useful here. Our package consists of a few    major components: 1. Core class library - object-oriented, granular, with an emphasis on design    and clean interfaces. 2. Application framework - console applications ("protocols"), which consume     objects from the core library in order to build something executable (and        hopefully useful). 3. Test framework - ensures that the library actually works. </p>
<p><em>homepage</em>: <a href="https://github.com/csb-toolbox">https://github.com/csb-toolbox</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="csbdeep">CSBDeep<a class="headerlink" href="#csbdeep" title="Permanent link">&para;</a></h3>
<p>CSBDeep is a toolbox for Content-aware Image Restoration (CARE).</p>
<p><em>homepage</em>: <a href="https://csbdeep.bioimagecomputing.com/">https://csbdeep.bioimagecomputing.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="csblast">CSBLAST<a class="headerlink" href="#csblast" title="Permanent link">&para;</a></h3>
<p>Context-specific extension of BLAST that significantly improves sensitivity and alignment quality.</p>
<p><em>homepage</em>: <a href="https://github.com/soedinglab/csblast/">https://github.com/soedinglab/csblast/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cscope">cscope<a class="headerlink" href="#cscope" title="Permanent link">&para;</a></h3>
<p>Cscope is a developer's tool for browsing source code.</p>
<p><em>homepage</em>: <a href="http://cscope.sourceforge.net/">http://cscope.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15.9</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="csvkit">csvkit<a class="headerlink" href="#csvkit" title="Permanent link">&para;</a></h3>
<p>csvkit is a suite of command-line tools for converting to and working with CSV,  the king of tabular file formats.</p>
<p><em>homepage</em>: <a href="https://github.com/wireservice/csvkit">https://github.com/wireservice/csvkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.0.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ctags">ctags<a class="headerlink" href="#ctags" title="Permanent link">&para;</a></h3>
<p>Ctags generates an index (or tag) file of language objects found in source files that allows these  items to be quickly and easily located by a text editor or other utility.</p>
<p><em>homepage</em>: <a href="http://ctags.sourceforge.net/">http://ctags.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ctffind">ctffind<a class="headerlink" href="#ctffind" title="Permanent link">&para;</a></h3>
<p>Program for finding CTFs of electron micrographs.</p>
<p><em>homepage</em>: <a href="https://grigoriefflab.umassmed.edu/ctffind4">https://grigoriefflab.umassmed.edu/ctffind4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.13</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>4.1.14</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>fosscuda/2019b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4.1.14</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="ctpl">CTPL<a class="headerlink" href="#ctpl" title="Permanent link">&para;</a></h3>
<p>Modern and efficient C++ Thread Pool Library</p>
<p><em>homepage</em>: <a href="https://github.com/vit-vit/CTPL.git">https://github.com/vit-vit/CTPL.git</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cube">Cube<a class="headerlink" href="#cube" title="Permanent link">&para;</a></h3>
<p>Cube, which is used as performance report explorer for Scalasca and  Score-P, is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system  resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree  can be collapsed or expanded to achieve the desired level of granularity.</p>
<p><em>homepage</em>: <a href="http://www.scalasca.org/software/cube-4.x/download.html">http://www.scalasca.org/software/cube-4.x/download.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="cubegui">CubeGUI<a class="headerlink" href="#cubegui" title="Permanent link">&para;</a></h3>
<p>Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube graphical report explorer. </p>
<p><em>homepage</em>: <a href="https://www.scalasca.org/software/cube-4.x/download.html">https://www.scalasca.org/software/cube-4.x/download.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.4</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cubelib">CubeLib<a class="headerlink" href="#cubelib" title="Permanent link">&para;</a></h3>
<p>Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube general purpose C++ library component and  command-line tools. </p>
<p><em>homepage</em>: <a href="https://www.scalasca.org/software/cube-4.x/download.html">https://www.scalasca.org/software/cube-4.x/download.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.4</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.8</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cubewriter">CubeWriter<a class="headerlink" href="#cubewriter" title="Permanent link">&para;</a></h3>
<p>Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube high-performance C writer library component. </p>
<p><em>homepage</em>: <a href="https://www.scalasca.org/software/cube-4.x/download.html">https://www.scalasca.org/software/cube-4.x/download.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.8</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cuda">CUDA<a class="headerlink" href="#cuda" title="Permanent link">&para;</a></h3>
<p>CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/cuda-toolkit">https://developer.nvidia.com/cuda-toolkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.5.22</code></td>
<td><code>GCC/4.8.2</code>, <code>system</code></td>
</tr>
<tr>
<td><code>6.0.37</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.5.14</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.5.18</code></td>
<td><code>GCC/4.9.4-2.25</code>, <code>iccifort/2016.3.210-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>8.0.44</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>system</code></td>
</tr>
<tr>
<td><code>8.0.61</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.0.61_375.26</code></td>
<td><code>GCC/5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>9.0.176</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>system</code></td>
</tr>
<tr>
<td><code>9.1.85</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>system</code></td>
</tr>
<tr>
<td><code>9.2.88</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/7.3.0-2.30</code>, <code>system</code></td>
</tr>
<tr>
<td><code>9.2.148.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.0.130</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.1.105</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>system</code></td>
</tr>
<tr>
<td><code>10.1.168</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.1.243</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code>, <code>system</code></td>
</tr>
<tr>
<td><code>10.2.89</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>11.0.2</code></td>
<td><code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>11.1.1</code></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>11.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cuda-samples">CUDA-Samples<a class="headerlink" href="#cuda-samples" title="Permanent link">&para;</a></h3>
<p>Samples for CUDA Developers which demonstrates features in CUDA Toolkit</p>
<p><em>homepage</em>: <a href="https://github.com/NVIDIA/cuda-samples">https://github.com/NVIDIA/cuda-samples</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cudacompat">CUDAcompat<a class="headerlink" href="#cudacompat" title="Permanent link">&para;</a></h3>
<p>Using the CUDA Forward Compatibility package,  system administrators can run applications built using a newer toolkit  even when an older driver that does not satisfy the minimum required driver version  is installed on the system.  This forward compatibility allows the CUDA deployments in data centers and enterprises  to benefit from the faster release cadence and the latest features and performance of CUDA Toolkit. </p>
<p><em>homepage</em>: <a href="https://docs.nvidia.com/deploy/cuda-compatibility/index.html">https://docs.nvidia.com/deploy/cuda-compatibility/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.6</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.6</code></td>
<td><code>-510.85.02</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.7</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.7</code></td>
<td><code>-515.65.01</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cudacore">CUDAcore<a class="headerlink" href="#cudacore" title="Permanent link">&para;</a></h3>
<p>CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/cuda-toolkit">https://developer.nvidia.com/cuda-toolkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.2.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.5.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cudnn">cuDNN<a class="headerlink" href="#cudnn" title="Permanent link">&para;</a></h3>
<p>The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/cudnn">https://developer.nvidia.com/cudnn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td><code>-CUDA-7.5.18</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0-rc</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1</code></td>
<td><code>-CUDA-8.0.44</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td><code>-CUDA-8.0.61</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.21</code></td>
<td><code>-CUDA-7.5.18</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.21</code></td>
<td><code>-CUDA-8.0.44</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.2</code></td>
<td><code>-CUDA-9.0.176</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.5</code></td>
<td><code>-CUDA-8.0.44</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.5</code></td>
<td><code>-CUDA-9.0.176</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.5</code></td>
<td><code>-CUDA-9.1.85</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.5.15</code></td>
<td></td>
<td><code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>7.1.4.18</code></td>
<td></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>7.4.2.24</code></td>
<td></td>
<td><code>gcccuda/2019a</code></td>
</tr>
<tr>
<td><code>7.4.2.24</code></td>
<td><code>-CUDA-10.0.130</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.5.0.56</code></td>
<td><code>-CUDA-10.0.130</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.6.2.24</code></td>
<td><code>-CUDA-10.1.243</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.6.4.38</code></td>
<td></td>
<td><code>gcccuda/2019a</code>, <code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>7.6.4.38</code></td>
<td><code>-CUDA-10.0.130</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.0.4.30</code></td>
<td><code>-CUDA-11.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.0.4.30</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.0.5.39</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.0.77</code></td>
<td><code>-CUDA-11.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.1.33</code></td>
<td><code>-CUDA-11.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2.1.32</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2.2.26</code></td>
<td><code>-CUDA-11.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2.2.26</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.0.27</code></td>
<td><code>-CUDA-11.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.1.50</code></td>
<td><code>-CUDA-11.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.1.50</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.6.0.163</code></td>
<td><code>-CUDA-11.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.7.0.84</code></td>
<td><code>-CUDA-11.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.8.0.121</code></td>
<td><code>-CUDA-12.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cufflinks">Cufflinks<a class="headerlink" href="#cufflinks" title="Permanent link">&para;</a></h3>
<p>Transcript assembly, differential expression, and differential regulation for RNA-Seq</p>
<p><em>homepage</em>: <a href="http://cole-trapnell-lab.github.io/cufflinks/">http://cole-trapnell-lab.github.io/cufflinks/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>gompi/2019b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>20190706</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code>, <code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="cunit">CUnit<a class="headerlink" href="#cunit" title="Permanent link">&para;</a></h3>
<p>Automated testing framework for C.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/cunit/">https://sourceforge.net/projects/cunit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1-3</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cupy">CuPy<a class="headerlink" href="#cupy" title="Permanent link">&para;</a></h3>
<p>CuPy is an open-source array library accelerated with NVIDIA CUDA.</p>
<p><em>homepage</em>: <a href="https://cupy.dev">https://cupy.dev</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>8.5.0</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="curl">cURL<a class="headerlink" href="#curl" title="Permanent link">&para;</a></h3>
<p>libcurl is a free and easy-to-use client-side URL transfer library,  supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,  LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.  libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP  form based upload, proxies, cookies, user+password authentication (Basic,  Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling  and more. </p>
<p><em>homepage</em>: <a href="https://curl.haxx.se">https://curl.haxx.se</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.33.0</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>7.34.0</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>7.40.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>7.46.0</code></td>
<td><code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>7.47.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.49.1</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.52.1</code></td>
<td><code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.53.1</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>7.54.0</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>7.55.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.56.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.56.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.58.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.59.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.60.0</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>7.63.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>7.66.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>7.69.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.72.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>7.76.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>7.78.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>7.83.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>7.84.0</code></td>
<td><code>GCCcore/12.1.0</code></td>
</tr>
<tr>
<td><code>7.86.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cusparselt">cuSPARSELt<a class="headerlink" href="#cusparselt" title="Permanent link">&para;</a></h3>
<p>NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix</p>
<p><em>homepage</em>: <a href="https://docs.nvidia.com/cuda/cusparselt/index.html">https://docs.nvidia.com/cuda/cusparselt/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0.3</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="custodian">custodian<a class="headerlink" href="#custodian" title="Permanent link">&para;</a></h3>
<p>A simple JIT job management framework in Python.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/custodian">https://pypi.python.org/pypi/custodian</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="cutadapt">cutadapt<a class="headerlink" href="#cutadapt" title="Permanent link">&para;</a></h3>
<p>Cutadapt finds and removes adapter sequences, primers, poly-A tails and  other types of unwanted sequence from your high-throughput sequencing reads.</p>
<p><em>homepage</em>: <a href="https://opensource.scilifelab.se/projects/cutadapt/">https://opensource.scilifelab.se/projects/cutadapt/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.15</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.18</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.18</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.18</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.18</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.4</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cutensor">cuTENSOR<a class="headerlink" href="#cutensor" title="Permanent link">&para;</a></h3>
<p>The cuTENSOR Library is a GPU-accelerated tensor linear algebra library providing tensor contraction,  reduction and elementwise operations.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/cutensor">https://developer.nvidia.com/cutensor</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2.5</code></td>
<td></td>
<td><code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>1.2.2.5</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.0.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.1.5</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.1.5</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cvs">CVS<a class="headerlink" href="#cvs" title="Permanent link">&para;</a></h3>
<p>CVS is a version control system, an important component of  Source Configuration Management (SCM).</p>
<p><em>homepage</em>: <a href="https://savannah.nongnu.org/projects/cvs">https://savannah.nongnu.org/projects/cvs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.23</code></td>
<td><code>GCC/4.8.2</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cvxopt">CVXOPT<a class="headerlink" href="#cvxopt" title="Permanent link">&para;</a></h3>
<p>CVXOPT is a free software package for convex optimization based on the Python programming language.  Its main purpose is to make the development of software for convex optimization applications straightforward by  building on Python's extensive standard library and on the strengths of Python as a high-level programming language. </p>
<p><em>homepage</em>: <a href="https://cvxopt.org">https://cvxopt.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.9</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.2.6</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cvxpy">CVXPY<a class="headerlink" href="#cvxpy" title="Permanent link">&para;</a></h3>
<p>CVXPY is a Python-embedded modeling language for convex optimization problems.  It allows you to express your problem in a natural way that follows the math,  rather than in the restrictive standard form required by solvers. </p>
<p><em>homepage</em>: <a href="https://www.cvxpy.org/">https://www.cvxpy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.24</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.0.28</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="cwipi">CWIPI<a class="headerlink" href="#cwipi" title="Permanent link">&para;</a></h3>
<p>CWIPI (Coupling With Interpolation Parallel Interface) library helps in chaining and coupling codes. Provides exchanges of interpolated fields through a non-compliant geometric interface and allows control of the coupling algorithm using control parameters. CWIPI takes advantage of the distribution of the definition of the coupling algorithm in the different codes. </p>
<p><em>homepage</em>: <a href="https://w3.onera.fr/cwipi/">https://w3.onera.fr/cwipi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.0</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cwltool">cwltool<a class="headerlink" href="#cwltool" title="Permanent link">&para;</a></h3>
<p>Common workflow language (CWL) reference implementation.</p>
<p><em>homepage</em>: <a href="https://github.com/common-workflow-language/cwltool">https://github.com/common-workflow-language/cwltool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.20221008225030</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.1.20221018083734</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="cxxopts">cxxopts<a class="headerlink" href="#cxxopts" title="Permanent link">&para;</a></h3>
<p>cxxopts is a lightweight C++ command line option parser</p>
<p><em>homepage</em>: <a href="https://github.com/jarro2783/cxxopts">https://github.com/jarro2783/cxxopts</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cysignals">cysignals<a class="headerlink" href="#cysignals" title="Permanent link">&para;</a></h3>
<p>The cysignals package provides mechanisms to handle  interrupts (and other signals and errors) in Cython code.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/cysignals/">https://pypi.org/project/cysignals/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="cython">Cython<a class="headerlink" href="#cython" title="Permanent link">&para;</a></h3>
<p>Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). </p>
<p><em>homepage</em>: <a href="https://cython.org/">https://cython.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.23.4</code></td>
<td><code>-Python-2.7.10</code></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>0.24.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.25.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.25.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.27.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.29.10</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.29.10</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.29.22</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.29.33</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.0a5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cython-blis">cython-blis<a class="headerlink" href="#cython-blis" title="Permanent link">&para;</a></h3>
<p>Fast BLAS-like operations from Python and Cython, without the tears. Provides the Blis linear algebra routines as a self-contained Python C-extension. </p>
<p><em>homepage</em>: <a href="https://github.com/explosion/cython-blis">https://github.com/explosion/cython-blis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="cytoolz">cytoolz<a class="headerlink" href="#cytoolz" title="Permanent link">&para;</a></h3>
<p>Cython implementation of the toolz package, which provides high performance utility functions  for iterables, functions, and dictionaries.</p>
<p><em>homepage</em>: <a href="https://github.com/pytoolz/cytoolz">https://github.com/pytoolz/cytoolz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="cytoscape">Cytoscape<a class="headerlink" href="#cytoscape" title="Permanent link">&para;</a></h3>
<p>Cytoscape is an open source software platform for visualizing  complex networks and integrating these with any type of attribute data.  A lot of Apps are available for various kinds of problem domains,  including bioinformatics, social network analysis, and semantic web.</p>
<p><em>homepage</em>: <a href="https://cytoscape.org/">https://cytoscape.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.1</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="cytosim">cytosim<a class="headerlink" href="#cytosim" title="Permanent link">&para;</a></h3>
<p>Cytosim is a cytoskeleton simulation engine written in  C++ working on Mac OS, GNU/Linux and Windows (with Cygwin).</p>
<p><em>homepage</em>: <a href="https://github.com/nedelec/cytosim">https://github.com/nedelec/cytosim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190117</code></td>
<td><code>-mkl</code></td>
<td><code>gomkl/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="cyvcf2">cyvcf2<a class="headerlink" href="#cyvcf2" title="Permanent link">&para;</a></h3>
<p>cython + htslib == fast VCF and BCF processing</p>
<p><em>homepage</em>: <a href="https://github.com/brentp/cyvcf2">https://github.com/brentp/cyvcf2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.10</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.11.5</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h2 id="d">D<a class="headerlink" href="#d" title="Permanent link">&para;</a></h2>
<p><a href="#dadi">dadi</a> - <a href="#dagitty">dagitty</a> - <a href="#dakota">Dakota</a> - <a href="#dalton">Dalton</a> - <a href="#damageproto">damageproto</a> - <a href="#dammit">dammit</a> - <a href="#danpos2">DANPOS2</a> - <a href="#das_tool">DAS_Tool</a> - <a href="#dask">dask</a> - <a href="#datamash">datamash</a> - <a href="#davix">davix</a> - <a href="#db">DB</a> - <a href="#db_file">DB_File</a> - <a href="#dbcsr">DBCSR</a> - <a href="#dbd-mysql">DBD-mysql</a> - <a href="#dbg2olc">DBG2OLC</a> - <a href="#dbus">DBus</a> - <a href="#dbus-glib">dbus-glib</a> - <a href="#dclone">dclone</a> - <a href="#dcm2niix">dcm2niix</a> - <a href="#dcmtk">DCMTK</a> - <a href="#dd">dd</a> - <a href="#deal.ii">deal.II</a> - <a href="#deap">deap</a> - <a href="#decona">decona</a> - <a href="#deconf">deconf</a> - <a href="#deconica">DeconICA</a> - <a href="#deepdiff">deepdiff</a> - <a href="#deeplabcut">DeepLabCut</a> - <a href="#deepmedic">deepmedic</a> - <a href="#deepmod2">DeepMod2</a> - <a href="#deepsurv">DeepSurv</a> - <a href="#deeptools">deepTools</a> - <a href="#delly">Delly</a> - <a href="#demixt">DeMixT</a> - <a href="#dendropy">DendroPy</a> - <a href="#denspart">DensPart</a> - <a href="#deprecated">Deprecated</a> - <a href="#destiny">destiny</a> - <a href="#detectron2">Detectron2</a> - <a href="#detonate">detonate</a> - <a href="#devito">Devito</a> - <a href="#dfa">DFA</a> - <a href="#dft-d3">DFT-D3</a> - <a href="#dft-d4">DFT-D4</a> - <a href="#dftb+">DFTB+</a> - <a href="#dftd3-lib">dftd3-lib</a> - <a href="#dgl">DGL</a> - <a href="#dial">DIAL</a> - <a href="#diamond">DIAMOND</a> - <a href="#dice">Dice</a> - <a href="#dicom2nifti">dicom2nifti</a> - <a href="#dicombrowser">DicomBrowser</a> - <a href="#diffutils">Diffutils</a> - <a href="#dijitso">dijitso</a> - <a href="#dill">dill</a> - <a href="#dirac">DIRAC</a> - <a href="#distributed">distributed</a> - <a href="#distributedstream">DistributedStream</a> - <a href="#dl_poly_4">DL_POLY_4</a> - <a href="#dl_poly_classic">DL_POLY_Classic</a> - <a href="#dlb">dlb</a> - <a href="#dlib">dlib</a> - <a href="#dlpack">DLPack</a> - <a href="#dm-reverb">dm-reverb</a> - <a href="#dm-tree">dm-tree</a> - <a href="#dmcfun">DMCfun</a> - <a href="#dmlc-core">DMLC-Core</a> - <a href="#dmtcp">DMTCP</a> - <a href="#dolfin">DOLFIN</a> - <a href="#dorado">dorado</a> - <a href="#doris">Doris</a> - <a href="#dosageconvertor">DosageConvertor</a> - <a href="#dotnet-core-runtime">dotNET-Core-Runtime</a> - <a href="#dotnet-sdk">dotNET-SDK</a> - <a href="#double-conversion">double-conversion</a> - <a href="#doubletfinder">DoubletFinder</a> - <a href="#doxygen">Doxygen</a> - <a href="#dragmap">DRAGMAP</a> - <a href="#drake">Drake</a> - <a href="#drep">dRep</a> - <a href="#drop">DROP</a> - <a href="#dropest">dropEst</a> - <a href="#dsa">DSA</a> - <a href="#dsfmt">dSFMT</a> - <a href="#dsrc">DSRC</a> - <a href="#dsuite">Dsuite</a> - <a href="#dtcmp">dtcmp</a> - <a href="#dtcwt">dtcwt</a> - <a href="#dualsphysics">DualSPHysics</a> - <a href="#dubstepr">DUBStepR</a> - <a href="#dune-core">dune-core</a> - <a href="#dune-fem">dune-fem</a> - <a href="#duplex-tools">duplex-tools</a> - <a href="#dxpy">dxpy</a> - <a href="#dymat">DyMat</a></p>
<h3 id="dadi">dadi<a class="headerlink" href="#dadi" title="Permanent link">&para;</a></h3>
<p>ai implements methods for demographic history and selection inference from genetic data,  based on diffusion approximations to the allele frequency spectrum.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/gutenkunstlab/dadi">https://bitbucket.org/gutenkunstlab/dadi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="dagitty">dagitty<a class="headerlink" href="#dagitty" title="Permanent link">&para;</a></h3>
<p>A port of the web-based software 'DAGitty', available at <a href="http://dagitty.net">http://dagitty.net</a>, for  analyzing structural causal models (also known as directed acyclic graphs or DAGs). This package  computes covariate adjustment sets for estimating causal effects, enumerates instrumental variables,  derives testable implications (d-separation and vanishing tetrads), generates equivalent models, and  includes a simple facility for data simulation.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/dagitty/">https://cran.r-project.org/web/packages/dagitty/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2-2</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="dakota">Dakota<a class="headerlink" href="#dakota" title="Permanent link">&para;</a></h3>
<p>The Dakota project delivers both state-of-the-art research and robust, usable  software for optimization and UQ. Broadly, the Dakota software's advanced  parametric analyses enable design exploration, model calibration, risk  analysis, and quantification of margins and uncertainty with computational  models. </p>
<p><em>homepage</em>: <a href="https://dakota.sandia.gov/">https://dakota.sandia.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.16.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="dalton">Dalton<a class="headerlink" href="#dalton" title="Permanent link">&para;</a></h3>
<p>The Dalton code is a powerful tool for a wide range of molecular properties  at different levels of theory.  Any published work arising from use of one of the Dalton2016 programs  must acknowledge that by a proper reference,  https://www.daltonprogram.org/www/citation.html.</p>
<p><em>homepage</em>: <a href="https://daltonprogram.org/">https://daltonprogram.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016</code></td>
<td><code>-i8</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2020.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="damageproto">damageproto<a class="headerlink" href="#damageproto" title="Permanent link">&para;</a></h3>
<p>X protocol and ancillary headers for xinerama</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="dammit">dammit<a class="headerlink" href="#dammit" title="Permanent link">&para;</a></h3>
<p>dammit is a simple de novo transcriptome annotator. It was born out of  the observations that annotation is mundane and annoying,  all the individual pieces of the process exist already, and the existing  solutions are overly complicated or rely on crappy non-free software.</p>
<p><em>homepage</em>: <a href="http://www.camillescott.org/dammit/">http://www.camillescott.org/dammit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="danpos2">DANPOS2<a class="headerlink" href="#danpos2" title="Permanent link">&para;</a></h3>
<p>A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2</p>
<p><em>homepage</em>: <a href="https://sites.google.com/site/danposdoc">https://sites.google.com/site/danposdoc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="das_tool">DAS_Tool<a class="headerlink" href="#das_tool" title="Permanent link">&para;</a></h3>
<p>DAS Tool is an automated method that integrates the results of a flexible number of binning  algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</p>
<p><em>homepage</em>: <a href="https://github.com/cmks/DAS_Tool">https://github.com/cmks/DAS_Tool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-R-3.5.1-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="dask">dask<a class="headerlink" href="#dask" title="Permanent link">&para;</a></h3>
<p>Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance  at scale for the tools you love.</p>
<p><em>homepage</em>: <a href="https://dask.org/">https://dask.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.17.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.17.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.17.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.19.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.18.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>2021.9.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2022.10.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="datamash">datamash<a class="headerlink" href="#datamash" title="Permanent link">&para;</a></h3>
<p>GNU datamash performs basic numeric, textual and statistical operations on input data files</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/datamash/">https://www.gnu.org/software/datamash/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="davix">davix<a class="headerlink" href="#davix" title="Permanent link">&para;</a></h3>
<p>The davix project aims to make file management over HTTP-based protocols simple.  The focus is on high-performance remote I/O and data management of large collections of files.  Currently, there is support for the WebDav (link is external), Amazon S3 (link is external),  Microsoft Azure (link is external), and HTTP (link is external) protocols.</p>
<p><em>homepage</em>: <a href="https://dmc.web.cern.ch/projects/davix/home">https://dmc.web.cern.ch/projects/davix/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.6</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.7.5</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="db">DB<a class="headerlink" href="#db" title="Permanent link">&para;</a></h3>
<p>Berkeley DB enables the development of custom data management solutions,  without the overhead traditionally associated with such custom projects.</p>
<p><em>homepage</em>: <a href="http://www.oracle.com/technetwork/products/berkeleydb">http://www.oracle.com/technetwork/products/berkeleydb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.30</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>6.2.23</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.2.32</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>18.1.25</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>18.1.32</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>18.1.40</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="db_file">DB_File<a class="headerlink" href="#db_file" title="Permanent link">&para;</a></h3>
<p>Perl5 access to Berkeley DB version 1.x.</p>
<p><em>homepage</em>: <a href="https://perldoc.perl.org/DB_File.html">https://perldoc.perl.org/DB_File.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.835</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.835</code></td>
<td><code>-Perl-5.20.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.835</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.855</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.856</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.857</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.858</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dbcsr">DBCSR<a class="headerlink" href="#dbcsr" title="Permanent link">&para;</a></h3>
<p>DBCSR stands for Distributed Blocked Compressed Sparse Row. DBCSR is a library designed to  efficiently perform sparse matrix-matrix multiplication, among other operations.</p>
<p><em>homepage</em>: <a href="https://cp2k.github.io/dbcsr/develop/">https://cp2k.github.io/dbcsr/develop/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="dbd-mysql">DBD-mysql<a class="headerlink" href="#dbd-mysql" title="Permanent link">&para;</a></h3>
<p>Perl binding for MySQL</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm">https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.032</code></td>
<td><code>-Perl-5.22.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.033</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.042</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.046</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.046</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.048</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.050</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>4.050</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="dbg2olc">DBG2OLC<a class="headerlink" href="#dbg2olc" title="Permanent link">&para;</a></h3>
<p>DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation   Sequencing Technologies</p>
<p><em>homepage</em>: <a href="https://github.com/yechengxi/DBG2OLC">https://github.com/yechengxi/DBG2OLC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170208</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>20180221</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="dbus">DBus<a class="headerlink" href="#dbus" title="Permanent link">&para;</a></h3>
<p>D-Bus is a message bus system, a simple way for applications to talk  to one another.  In addition to interprocess communication, D-Bus helps  coordinate process lifecycle; it makes it simple and reliable to code  a "single instance" application or daemon, and to launch applications  and daemons on demand when their services are needed. </p>
<p><em>homepage</em>: <a href="https://dbus.freedesktop.org/">https://dbus.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.8</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.10.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.10.20</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.11.20</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.13.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.13.6</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.13.8</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.13.12</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.13.18</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.14.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.15.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dbus-glib">dbus-glib<a class="headerlink" href="#dbus-glib" title="Permanent link">&para;</a></h3>
<p>D-Bus is a message bus system, a simple way for applications to talk to one another.</p>
<p><em>homepage</em>: <a href="https://dbus.freedesktop.org/doc/dbus-glib">https://dbus.freedesktop.org/doc/dbus-glib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.106</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.108</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.110</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.112</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dclone">dclone<a class="headerlink" href="#dclone" title="Permanent link">&para;</a></h3>
<p>Low level functions for implementing maximum likelihood estimating procedures for complex models  using data cloning and Bayesian Markov chain Monte Carlo methods</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/dclone">https://cran.r-project.org/web/packages/dclone</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3-0</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="dcm2niix">dcm2niix<a class="headerlink" href="#dcm2niix" title="Permanent link">&para;</a></h3>
<p>dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.</p>
<p><em>homepage</em>: <a href="https://github.com/rordenlab/dcm2niix">https://github.com/rordenlab/dcm2niix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.20180622</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.0.20190902</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.0.20200331</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.0.20201102</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.0.20211006</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.0.20220720</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dcmtk">DCMTK<a class="headerlink" href="#dcmtk" title="Permanent link">&para;</a></h3>
<p>DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.</p>
<p><em>homepage</em>: <a href="https://dicom.offis.de/dcmtk">https://dicom.offis.de/dcmtk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.3</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.6.5</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.6.6</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.6.7</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dd">dd<a class="headerlink" href="#dd" title="Permanent link">&para;</a></h3>
<p>dd is a package for working with binary decision diagrams that includes both a pure Python implementation and Cython bindings to C libraries (CUDD, Sylvan, BuDDy). The Python and Cython modules implement the same API, so the same user code runs with both. All the standard operations on BDDs are available, including dynamic variable reordering using sifting, garbage collection, dump/load from files, plotting, and a parser of quantified Boolean expressions. This module includes bindings for: CUDD v3.0.0, Sylvan v1.0.0</p>
<p><em>homepage</em>: <a href="https://github.com/tulip-control/dd">https://github.com/tulip-control/dd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="dealii">deal.II<a class="headerlink" href="#dealii" title="Permanent link">&para;</a></h3>
<p>deal.II is a C++ program library targeted at the computational solution of  partial differential equations using adaptive finite elements.</p>
<p><em>homepage</em>: <a href="https://www.dealii.org">https://www.dealii.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.1.1</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>9.3.3</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="deap">deap<a class="headerlink" href="#deap" title="Permanent link">&para;</a></h3>
<p>DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.  It seeks to make algorithms explicit and data structures transparent.</p>
<p><em>homepage</em>: <a href="http://deap.readthedocs.org/en/master/">http://deap.readthedocs.org/en/master/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="decona">decona<a class="headerlink" href="#decona" title="Permanent link">&para;</a></h3>
<p>fastq to polished sequenses: pipeline suitable for mixed samples and long (Nanopore) reads</p>
<p><em>homepage</em>: <a href="https://github.com/Saskia-Oosterbroek/decona">https://github.com/Saskia-Oosterbroek/decona</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="deconf">deconf<a class="headerlink" href="#deconf" title="Permanent link">&para;</a></h3>
<p>decomposition (deconfounding) of OMICS datasets in heterogeneous tissues</p>
<p><em>homepage</em>: <a href="http://web.cbio.uct.ac.za/~renaud/CRAN/">http://web.cbio.uct.ac.za/~renaud/CRAN/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="deconica">DeconICA<a class="headerlink" href="#deconica" title="Permanent link">&para;</a></h3>
<p>Deconvolution of transcriptome through Immune Component Analysis (DeconICA) is an R package for identifying immune-related signals in transcriptome through deconvolution or unsupervised source separation methods.</p>
<p><em>homepage</em>: <a href="https://urszulaczerwinska.github.io/DeconICA">https://urszulaczerwinska.github.io/DeconICA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="deepdiff">deepdiff<a class="headerlink" href="#deepdiff" title="Permanent link">&para;</a></h3>
<p>DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively.</p>
<p><em>homepage</em>: <a href="https://zepworks.com/deepdiff/current/">https://zepworks.com/deepdiff/current/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.6</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.7.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.8.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="deeplabcut">DeepLabCut<a class="headerlink" href="#deeplabcut" title="Permanent link">&para;</a></h3>
<p>Markerless tracking of user-defined features with deep learning</p>
<p><em>homepage</em>: <a href="http://www.mousemotorlab.org/deeplabcut">http://www.mousemotorlab.org/deeplabcut</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0.6</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="deepmedic">deepmedic<a class="headerlink" href="#deepmedic" title="Permanent link">&para;</a></h3>
<p>Efficient Multi-Scale 3D Convolutional Neural Network for Segmentation of 3D Medical Scans.</p>
<p><em>homepage</em>: <a href="https://biomedia.doc.ic.ac.uk/software/deepmedic/">https://biomedia.doc.ic.ac.uk/software/deepmedic/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="deepmod2">DeepMod2<a class="headerlink" href="#deepmod2" title="Permanent link">&para;</a></h3>
<p>DeepMod2 is a computational tool for detecting DNA methylation and modifications  from Oxford Nanopore reads. </p>
<p><em>homepage</em>: <a href="https://github.com/WGLab/DeepMod2">https://github.com/WGLab/DeepMod2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="deepsurv">DeepSurv<a class="headerlink" href="#deepsurv" title="Permanent link">&para;</a></h3>
<p>DeepSurv is a deep learning approach to survival analysis. </p>
<p><em>homepage</em>: <a href="https://github.com/jaredleekatzman/DeepSurv">https://github.com/jaredleekatzman/DeepSurv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0-20180922</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="deeptools">deepTools<a class="headerlink" href="#deeptools" title="Permanent link">&para;</a></h3>
<p>deepTools is a suite of python tools particularly developed for the efficient analysis of  high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.</p>
<p><em>homepage</em>: <a href="https://deeptools.readthedocs.io/">https://deeptools.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.4</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="delly">Delly<a class="headerlink" href="#delly" title="Permanent link">&para;</a></h3>
<p>Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome.</p>
<p><em>homepage</em>: <a href="https://github.com/dellytools/delly/">https://github.com/dellytools/delly/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.8</code></td>
<td><code>-linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.7</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>1.1.5</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="demixt">DeMixT<a class="headerlink" href="#demixt" title="Permanent link">&para;</a></h3>
<p>Cell type-specific deconvolution of heterogeneous tumor samples with two or three components  using expression data from RNAseq or microarray platforms.</p>
<p><em>homepage</em>: <a href="https://bioinformatics.mdanderson.org/main/DeMixT">https://bioinformatics.mdanderson.org/main/DeMixT</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="dendropy">DendroPy<a class="headerlink" href="#dendropy" title="Permanent link">&para;</a></h3>
<p>A Python library for phylogenetics and phylogenetic computing:  reading, writing, simulation, processing and manipulation of phylogenetic trees  (phylogenies) and characters.</p>
<p><em>homepage</em>: <a href="https://dendropy.org/">https://dendropy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="denspart">DensPart<a class="headerlink" href="#denspart" title="Permanent link">&para;</a></h3>
<p>Atoms-in-molecules density partitioning schemes based on stockholder recipe</p>
<p><em>homepage</em>: <a href="https://github.com/theochem/denspart">https://github.com/theochem/denspart</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220603</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="deprecated">Deprecated<a class="headerlink" href="#deprecated" title="Permanent link">&para;</a></h3>
<p>If you need to mark a function or a method as deprecated, you can use the @deprecated decorator.</p>
<p><em>homepage</em>: <a href="https://github.com/tantale/deprecated">https://github.com/tantale/deprecated</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.13</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="destiny">destiny<a class="headerlink" href="#destiny" title="Permanent link">&para;</a></h3>
<p>R packages to create and plot diffusion maps.</p>
<p><em>homepage</em>: <a href="https://bioconductor.org/packages/destiny">https://bioconductor.org/packages/destiny</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.6</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="detectron2">Detectron2<a class="headerlink" href="#detectron2" title="Permanent link">&para;</a></h3>
<p>Detectron2 is Facebook AI Research's next generation library that provides state-of-the-art detection and segmentation algorithms. It is the successor of Detectron and maskrcnn-benchmark. It supports a number of computer vision research projects and production applications in Facebook.</p>
<p><em>homepage</em>: <a href="https://github.com/facebookresearch/detectron2">https://github.com/facebookresearch/detectron2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.6</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="detonate">detonate<a class="headerlink" href="#detonate" title="Permanent link">&para;</a></h3>
<p>DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation)  consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used  to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds  of genomic sequences.</p>
<p><em>homepage</em>: <a href="http://deweylab.biostat.wisc.edu/detonate/">http://deweylab.biostat.wisc.edu/detonate/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="devito">Devito<a class="headerlink" href="#devito" title="Permanent link">&para;</a></h3>
<p>Devito is a domain-specific Language (DSL) and code generation framework for  performing optimised Finite Difference (FD) computation from high-level symbolic problem definitions.  Devito performs automated code generation and Just-In-time (JIT) compilation based on symbolic  equations defined in SymPy to create and execute highly optimised Finite Difference stencil kernels on  multiple computer platforms.</p>
<p><em>homepage</em>: <a href="https://www.devitoproject.org">https://www.devitoproject.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="dfa">DFA<a class="headerlink" href="#dfa" title="Permanent link">&para;</a></h3>
<p>Python library for modeling DFAs, Moore Machines, and Transition Systems.</p>
<p><em>homepage</em>: <a href="https://github.com/mvcisback/dfa">https://github.com/mvcisback/dfa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dft-d3">DFT-D3<a class="headerlink" href="#dft-d3" title="Permanent link">&para;</a></h3>
<p>DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical  quantum chemical methods.</p>
<p><em>homepage</em>: <a href="https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3">https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="dft-d4">DFT-D4<a class="headerlink" href="#dft-d4" title="Permanent link">&para;</a></h3>
<p>Generally Applicable Atomic-Charge Dependent London Dispersion Correction.</p>
<p><em>homepage</em>: <a href="https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4">https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dftb">DFTB+<a class="headerlink" href="#dftb" title="Permanent link">&para;</a></h3>
<p>DFTB+ is a fast and efficient versatile quantum mechanical simulation package. It is based on the Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of magnitude in speed.</p>
<p><em>homepage</em>: <a href="https://www.dftb-plus.info">https://www.dftb-plus.info</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>17.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>19.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>19.1</code></td>
<td><code>-Python-2.7.16-mpi</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>21.1</code></td>
<td></td>
<td><code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="dftd3-lib">dftd3-lib<a class="headerlink" href="#dftd3-lib" title="Permanent link">&para;</a></h3>
<p>This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers. The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been converted to free format and encapsulated into modules.</p>
<p><em>homepage</em>: <a href="https://github.com/dftbplus/dftd3-lib">https://github.com/dftbplus/dftd3-lib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9</code></td>
<td><code>GCC/8.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dgl">DGL<a class="headerlink" href="#dgl" title="Permanent link">&para;</a></h3>
<p>DGL is an easy-to-use, high performance and scalable Python package for deep learning on graphs. DGL is framework agnostic, meaning if a deep graph model is a component of an end-to-end application, the rest of the logics can be implemented in any major frameworks, such as PyTorch, Apache MXNet or TensorFlow.</p>
<p><em>homepage</em>: <a href="https://www.dgl.ai">https://www.dgl.ai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.7.4-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="dial">DIAL<a class="headerlink" href="#dial" title="Permanent link">&para;</a></h3>
<p>DIAL (De novo Identification of Alleles) is a collection of programs  to automate the discovery of alleles for a species where we lack a reference sequence.   The SNPs/alleles are specifically selected for a low error rate in genotyping assays.   </p>
<p><em>homepage</em>: <a href="http://www.bx.psu.edu/miller_lab/">http://www.bx.psu.edu/miller_lab/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2011.06.06</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="diamond">DIAMOND<a class="headerlink" href="#diamond" title="Permanent link">&para;</a></h3>
<p>Accelerated BLAST compatible local sequence aligner</p>
<p><em>homepage</em>: <a href="https://github.com/bbuchfink/diamond">https://github.com/bbuchfink/diamond</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.22</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.9.24</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.9.30</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.9.36</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.0.7</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.11</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.13</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dice">Dice<a class="headerlink" href="#dice" title="Permanent link">&para;</a></h3>
<p>Dice contains code for performing SHCI, VMC, GFMC, DMC, FCIQMC, stochastic MRCI and SC-NEVPT2, and AFQMC calculations with a focus on ab initio systems.</p>
<p><em>homepage</em>: <a href="https://github.com/sanshar/Dice">https://github.com/sanshar/Dice</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20221025</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="dicom2nifti">dicom2nifti<a class="headerlink" href="#dicom2nifti" title="Permanent link">&para;</a></h3>
<p>Python library for converting dicom files to nifti</p>
<p><em>homepage</em>: <a href="https://github.com/icometrix/dicom2nifti">https://github.com/icometrix/dicom2nifti</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.12</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="dicombrowser">DicomBrowser<a class="headerlink" href="#dicombrowser" title="Permanent link">&para;</a></h3>
<p>DicomBrowser is an application for inspecting and modifying DICOM metadata in many files at once.</p>
<p><em>homepage</em>: <a href="http://nrg.wustl.edu/software/dicom-browser/">http://nrg.wustl.edu/software/dicom-browser/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.0b5</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="diffutils">Diffutils<a class="headerlink" href="#diffutils" title="Permanent link">&para;</a></h3>
<p>Diffutils: GNU diff utilities - find the differences between files</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/diffutils/diffutils.html">http://www.gnu.org/software/diffutils/diffutils.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="dijitso">dijitso<a class="headerlink" href="#dijitso" title="Permanent link">&para;</a></h3>
<p>dijitso is a Python module for distributed just-in-time shared library building.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/fenics-project/dijitso">https://bitbucket.org/fenics-project/dijitso</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dill">dill<a class="headerlink" href="#dill" title="Permanent link">&para;</a></h3>
<p>dill extends python's pickle module for serializing and de-serializing python objects to the majority  of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse  of which is converting a byte stream back to on python object hierarchy.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/dill/">https://pypi.org/project/dill/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.3.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.3.4</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.3.6</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dirac">DIRAC<a class="headerlink" href="#dirac" title="Permanent link">&para;</a></h3>
<p>DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations</p>
<p><em>homepage</em>: <a href="http://www.diracprogram.org">http://www.diracprogram.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.0</code></td>
<td><code>-Python-2.7.18-int64</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>19.0</code></td>
<td><code>-Python-2.7.18-mpi-int64</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>22.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>22.0</code></td>
<td><code>-int64</code></td>
<td><code>intel/2021a</code></td>
</tr>
<tr>
<td><code>23.0</code></td>
<td></td>
<td><code>foss/2022b</code>, <code>intel/2022b</code></td>
</tr>
<tr>
<td><code>23.0</code></td>
<td><code>-int64</code></td>
<td><code>intel/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="distributed">distributed<a class="headerlink" href="#distributed" title="Permanent link">&para;</a></h3>
<p>Dask.distributed is a lightweight library for distributed computing in Python.  It extends both the concurrent.futures and dask APIs to moderate sized clusters.</p>
<p><em>homepage</em>: <a href="https://distributed.readthedocs.io/">https://distributed.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.14.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.21.6</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="distributedstream">DistributedStream<a class="headerlink" href="#distributedstream" title="Permanent link">&para;</a></h3>
<p>A MPI distributed stream benchmark, useful to identifying nodes with poor memory performance and characterising memory bandwidth variation over systems.</p>
<p><em>homepage</em>: <a href="https://github.com/adrianjhpc/DistributedStream">https://github.com/adrianjhpc/DistributedStream</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="dl_poly_4">DL_POLY_4<a class="headerlink" href="#dl_poly_4" title="Permanent link">&para;</a></h3>
<p>DL_POLY is a general purpose classical molecular dynamics (MD) simulation software</p>
<p><em>homepage</em>: <a href="https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx">https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.0</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="dl_poly_classic">DL_POLY_Classic<a class="headerlink" href="#dl_poly_classic" title="Permanent link">&para;</a></h3>
<p>DL_POLY Classic is a freely available molecular dynamics program developed  from the DL_POLY_2 package. This version does not install the java gui.</p>
<p><em>homepage</em>: <a href="http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/">http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-PLUMED-2.2.3</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dlb">dlb<a class="headerlink" href="#dlb" title="Permanent link">&para;</a></h3>
<p>DLB is a dynamic library designed to speed up HPC hybrid applications (i.e., two levels of parallelism) by improving the load balance of the outer level of parallelism (e.g., MPI) by dynamically redistributing the computational resources at the inner level of parallelism (e.g., OpenMP). at run time. </p>
<p><em>homepage</em>: <a href="https://pm.bsc.es/dlb/">https://pm.bsc.es/dlb/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="dlib">dlib<a class="headerlink" href="#dlib" title="Permanent link">&para;</a></h3>
<p>Dlib is a modern C++ toolkit containing machine learning algorithms and tools for creating complex software in C++ to solve real world problems. It is used in both industry and academia in a wide range of domains including robotics, embedded devices, mobile phones, and large high performance computing environments.</p>
<p><em>homepage</em>: <a href="https://github.com/davisking/dlib">https://github.com/davisking/dlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.22</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>19.22</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="dlpack">DLPack<a class="headerlink" href="#dlpack" title="Permanent link">&para;</a></h3>
<p>DLPack is a stable in-memory data structure for an ndarray system to interact with a variety of frameworks.</p>
<p><em>homepage</em>: <a href="https://dmlc.github.io/dlpack/latest/">https://dmlc.github.io/dlpack/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dm-reverb">dm-reverb<a class="headerlink" href="#dm-reverb" title="Permanent link">&para;</a></h3>
<p>Reverb is an efficient and easy-to-use data storage and transport system designed for machine learning research. Reverb is primarily used as an experience replay system for distributed reinforcement learning algorithms but the system also supports multiple data structure representations such as FIFO, LIFO, and priority queues.</p>
<p><em>homepage</em>: <a href="https://github.com/deepmind/reverb">https://github.com/deepmind/reverb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="dm-tree">dm-tree<a class="headerlink" href="#dm-tree" title="Permanent link">&para;</a></h3>
<p>dm-tree provides tree, a library for working with nested data structures. In a way, tree generalizes the builtin map function which only supports flat sequences, and allows to apply a function to each "leaf" preserving the overall structure.</p>
<p><em>homepage</em>: <a href="https://github.com/deepmind/tree">https://github.com/deepmind/tree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.1.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.1.6</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.1.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dmcfun">DMCfun<a class="headerlink" href="#dmcfun" title="Permanent link">&para;</a></h3>
<p>Diffusion Model of Conflict (DMC) in Reaction Time Tasks</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/DMCfun">https://cran.r-project.org/web/packages/DMCfun</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dmlc-core">DMLC-Core<a class="headerlink" href="#dmlc-core" title="Permanent link">&para;</a></h3>
<p>DMLC-Core is the backbone library to support all DMLC projects, offers the bricks to build efficient and scalable distributed machine learning libraries.</p>
<p><em>homepage</em>: <a href="https://dmlc-core.readthedocs.io/en/latest">https://dmlc-core.readthedocs.io/en/latest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dmtcp">DMTCP<a class="headerlink" href="#dmtcp" title="Permanent link">&para;</a></h3>
<p>DMTCP is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications.</p>
<p><em>homepage</em>: <a href="http://dmtcp.sourceforge.net/index.html">http://dmtcp.sourceforge.net/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td><code>GCCcore/8.3.0</code>, <code>foss/2016b</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dolfin">DOLFIN<a class="headerlink" href="#dolfin" title="Permanent link">&para;</a></h3>
<p>DOLFIN is the C++/Python interface of FEniCS, providing a consistent PSE   (Problem Solving Environment) for ordinary and partial differential equations.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/fenics-project/dolfin">https://bitbucket.org/fenics-project/dolfin</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.1.0.post1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2019.1.0.post0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dorado">dorado<a class="headerlink" href="#dorado" title="Permanent link">&para;</a></h3>
<p>Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/dorado">https://github.com/nanoporetech/dorado</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.1</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="doris">Doris<a class="headerlink" href="#doris" title="Permanent link">&para;</a></h3>
<p>Delft object-oriented radar interferometric software</p>
<p><em>homepage</em>: <a href="http://doris.tudelft.nl/">http://doris.tudelft.nl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.02</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.04beta4</code></td>
<td><code>foss/2018a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.06beta2</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="dosageconvertor">DosageConvertor<a class="headerlink" href="#dosageconvertor" title="Permanent link">&para;</a></h3>
<p>DosageConvertor is a C++ tool to convert dosage files (in VCF format) from Minimac3/4  to other formats such as MaCH or PLINK. Please note that this tool CANNOT handle missing values in  the input files and may NOT work for non-Minimac3/4 VCF files.</p>
<p><em>homepage</em>: <a href="http://genome.sph.umich.edu/wiki/DosageConvertor">http://genome.sph.umich.edu/wiki/DosageConvertor</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dotnet-core-runtime">dotNET-Core-Runtime<a class="headerlink" href="#dotnet-core-runtime" title="Permanent link">&para;</a></h3>
<p>.NET is a free, cross-platform, open source developer platform for building many different types  of applications.</p>
<p><em>homepage</em>: <a href="https://www.microsoft.com/net/">https://www.microsoft.com/net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.7</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>5.0.17</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dotnet-sdk">dotNET-SDK<a class="headerlink" href="#dotnet-sdk" title="Permanent link">&para;</a></h3>
<p>.NET is a free, cross-platform, open source developer platform for building many different types  of applications.</p>
<p><em>homepage</em>: <a href="https://www.microsoft.com/net/">https://www.microsoft.com/net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.300</code></td>
<td><code>-linux-x64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.101</code></td>
<td><code>-linux-x64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="double-conversion">double-conversion<a class="headerlink" href="#double-conversion" title="Permanent link">&para;</a></h3>
<p>Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.</p>
<p><em>homepage</em>: <a href="https://github.com/google/double-conversion">https://github.com/google/double-conversion</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.3</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="doubletfinder">DoubletFinder<a class="headerlink" href="#doubletfinder" title="Permanent link">&para;</a></h3>
<p>R package for detecting doublets in single-cell RNA sequencing data</p>
<p><em>homepage</em>: <a href="https://github.com/chris-mcginnis-ucsf/DoubletFinder">https://github.com/chris-mcginnis-ucsf/DoubletFinder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.0.3-20230131</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="doxygen">Doxygen<a class="headerlink" href="#doxygen" title="Permanent link">&para;</a></h3>
<p>Doxygen is a documentation system for C++, C, Java, Objective-C, Python,  IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some  extent D. </p>
<p><em>homepage</em>: <a href="https://www.doxygen.org">https://www.doxygen.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.9.1</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.8.10</code></td>
<td><code>GNU/4.9.3-2.25</code>, <code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>1.8.11</code></td>
<td><code>GCC/4.9.2</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.8.13</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>1.8.14</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.8.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.8.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.8.17</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.8.20</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.9.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dragmap">DRAGMAP<a class="headerlink" href="#dragmap" title="Permanent link">&para;</a></h3>
<p>Dragmap is the Dragen mapper/aligner Open Source Software.</p>
<p><em>homepage</em>: <a href="https://github.com/Illumina/DRAGMAP">https://github.com/Illumina/DRAGMAP</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="drake">Drake<a class="headerlink" href="#drake" title="Permanent link">&para;</a></h3>
<p>Drake is a simple-to-use, extensible, text-based data workflow tool that organizes command execution around data and its dependencies.</p>
<p><em>homepage</em>: <a href="https://github.com/Factual/drake">https://github.com/Factual/drake</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="drep">dRep<a class="headerlink" href="#drep" title="Permanent link">&para;</a></h3>
<p>dRep is a python program which performs rapid pair-wise comparison of genome sets. One of its major purposes is for genome de-replication, but it can do a lot more.</p>
<p><em>homepage</em>: <a href="https://drep.readthedocs.io/en/latest/">https://drep.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="drop">DROP<a class="headerlink" href="#drop" title="Permanent link">&para;</a></h3>
<p>Pipeline to find aberrant events in RNA-Seq data, useful for diagnosis of rare disorders</p>
<p><em>homepage</em>: <a href="https://gagneurlab-drop.readthedocs.io">https://gagneurlab-drop.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="dropest">dropEst<a class="headerlink" href="#dropest" title="Permanent link">&para;</a></h3>
<p>Pipeline for initial analysis of droplet-based single-cell RNA-seq data</p>
<p><em>homepage</em>: <a href="https://github.com/hms-dbmi/dropEst">https://github.com/hms-dbmi/dropEst</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="dsa">DSA<a class="headerlink" href="#dsa" title="Permanent link">&para;</a></h3>
<p>Digital Sorting Algorithm</p>
<p><em>homepage</em>: <a href="https://github.com/zhandong/DSA">https://github.com/zhandong/DSA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="dsfmt">dSFMT<a class="headerlink" href="#dsfmt" title="Permanent link">&para;</a></h3>
<p>Double precision SIMD-oriented Fast Mersenne Twister.</p>
<p><em>homepage</em>: <a href="https://github.com/MersenneTwister-Lab/dSFMT">https://github.com/MersenneTwister-Lab/dSFMT</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dsrc">DSRC<a class="headerlink" href="#dsrc" title="Permanent link">&para;</a></h3>
<p>DNA Sequence Reads Compression is an application designed for compression of data files  containing reads from DNA sequencing in FASTQ format. The amount of such files can be huge, e.g., a few  (or tens) of gigabytes, so a need for a robust data compression tool is clear. Usually universal compression  programs like gzip or bzip2 are used for this purpose, but it is obvious that a specialized tool can work better.</p>
<p><em>homepage</em>: <a href="http://sun.aei.polsl.pl/dsrc">http://sun.aei.polsl.pl/dsrc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0rc</code></td>
<td><code>-linux-64-bit</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="dsuite">Dsuite<a class="headerlink" href="#dsuite" title="Permanent link">&para;</a></h3>
<p>Fast calculation of the ABBA-BABA statistics across many populations/species</p>
<p><em>homepage</em>: <a href="https://github.com/millanek/Dsuite">https://github.com/millanek/Dsuite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190713</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>20210718</code></td>
<td><code>GCC/10.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dtcmp">dtcmp<a class="headerlink" href="#dtcmp" title="Permanent link">&para;</a></h3>
<p>The Datatype Comparison (DTCMP) Library provides pre-defined and user-defined comparison operations to compare the values of two items which can be arbitrary MPI datatypes. Using these comparison operations, the library provides various routines for manipulating data, which may be distributed over the processes of an MPI communicator.</p>
<p><em>homepage</em>: <a href="https://github.com/LLNL/dtcmp">https://github.com/LLNL/dtcmp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>gompi/2019a</code>, <code>gompi/2020a</code>, <code>iimpi/2019a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="dtcwt">dtcwt<a class="headerlink" href="#dtcwt" title="Permanent link">&para;</a></h3>
<p>Dual-Tree Complex Wavelet Transform library for Python</p>
<p><em>homepage</em>: <a href="https://github.com/rjw57/dtcwt">https://github.com/rjw57/dtcwt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="dualsphysics">DualSPHysics<a class="headerlink" href="#dualsphysics" title="Permanent link">&para;</a></h3>
<p>DualSPHysics is based on the Smoothed Particle Hydrodynamics model named SPHysics.   The code is developed to study free-surface flow phenomena where Eulerian methods  can be difficult to apply, such as waves or impact of dam-breaks on off-shore  structures. DualSPHysics is a set of C++, CUDA and Java codes designed to deal  with real-life engineering problems.</p>
<p><em>homepage</em>: <a href="https://dual.sphysics.org/">https://dual.sphysics.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.175</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>5.0.175</code></td>
<td><code>-CUDA-%(cudaver)s</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="dubstepr">DUBStepR<a class="headerlink" href="#dubstepr" title="Permanent link">&para;</a></h3>
<p>DUBStepR (Determining the Underlying Basis using Step-wise Regression) is a feature selection algorithm for cell type identification in single-cell RNA-sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/prabhakarlab/DUBStepR">https://github.com/prabhakarlab/DUBStepR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="dune-core">dune-core<a class="headerlink" href="#dune-core" title="Permanent link">&para;</a></h3>
<p>The Dune core modules build the stable basis of Dune.  They follow a consistent release cycle and have high requirements regarding stability and backwards compatibility.  These modules build the foundation for higher-level components. </p>
<p><em>homepage</em>: <a href="https://www.dune-project.org/groups/core/">https://www.dune-project.org/groups/core/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.0.post1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="dune-fem">dune-fem<a class="headerlink" href="#dune-fem" title="Permanent link">&para;</a></h3>
<p>DUNE-FEM is a discretization module based on DUNE containing all the building blocks required to implement efficient solvers for a wide range of (systems of non linear) partial differential equations. DUNE-FEM can also be used through an extensive Python interface which brings all components of DUNE-FEM and the DUNE core modules to Python. </p>
<p><em>homepage</em>: <a href="https://www.dune-project.org/modules/dune-fem/">https://www.dune-project.org/modules/dune-fem/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.0.6</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="duplex-tools">duplex-tools<a class="headerlink" href="#duplex-tools" title="Permanent link">&para;</a></h3>
<p>Duplex Tools contains a set of utilities for dealing with Duplex sequencing data. Tools are provided  to identify and prepare duplex pairs for basecalling by Dorado (recommended) and Guppy, and for  recovering simplex basecalls from incorrectly concatenated pairs. </p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/duplex-tools">https://github.com/nanoporetech/duplex-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="dxpy">dxpy<a class="headerlink" href="#dxpy" title="Permanent link">&para;</a></h3>
<p>DNAnexus Platform API bindings for Python</p>
<p><em>homepage</em>: <a href="http://autodoc.dnanexus.com/">http://autodoc.dnanexus.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.266.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="dymat">DyMat<a class="headerlink" href="#dymat" title="Permanent link">&para;</a></h3>
<p>Read and process result files from Dymola and OpenModelica with Python.</p>
<p><em>homepage</em>: <a href="https://www.j-raedler.de/projects/DyMat">https://www.j-raedler.de/projects/DyMat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>-2020-12-12</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h2 id="e">E<a class="headerlink" href="#e" title="Permanent link">&para;</a></h2>
<p><a href="#e-antic">E-ANTIC</a> - <a href="#ea-utils">ea-utils</a> - <a href="#earthengine-api">earthengine-api</a> - <a href="#easybuild">EasyBuild</a> - <a href="#easyqc">EasyQC</a> - <a href="#ecbuild">ecBuild</a> - <a href="#eccodes">ecCodes</a> - <a href="#eccodes-python">eccodes-python</a> - <a href="#ed2">ED2</a> - <a href="#edlib">edlib</a> - <a href="#egglib">EggLib</a> - <a href="#eggnog-mapper">eggnog-mapper</a> - <a href="#egttools">EGTtools</a> - <a href="#eht-imaging">eht-imaging</a> - <a href="#eigen">Eigen</a> - <a href="#eigenexa">EigenExa</a> - <a href="#eigensoft">EIGENSOFT</a> - <a href="#einops">einops</a> - <a href="#elastix">elastix</a> - <a href="#elbencho">elbencho</a> - <a href="#elfio">ELFIO</a> - <a href="#elfutils">elfutils</a> - <a href="#elk">Elk</a> - <a href="#elpa">ELPA</a> - <a href="#elph">ELPH</a> - <a href="#elprep">elprep</a> - <a href="#elsi">ELSI</a> - <a href="#elsi-rci">ELSI-RCI</a> - <a href="#emacs">Emacs</a> - <a href="#eman2">EMAN2</a> - <a href="#emboss">EMBOSS</a> - <a href="#embree">Embree</a> - <a href="#emcee">emcee</a> - <a href="#emu">EMU</a> - <a href="#enabrowsertool">enaBrowserTool</a> - <a href="#enchant">enchant</a> - <a href="#enchant-2">enchant-2</a> - <a href="#ensemblcoreapi">EnsEMBLCoreAPI</a> - <a href="#entrypoints">entrypoints</a> - <a href="#epd">EPD</a> - <a href="#epic">EPIC</a> - <a href="#episcanpy">epiScanpy</a> - <a href="#equilibrator">eQuilibrator</a> - <a href="#ericscript">EricScript</a> - <a href="#esl-bundle">ESL-Bundle</a> - <a href="#esm-2">ESM-2</a> - <a href="#esmf">ESMF</a> - <a href="#esmpy">ESMPy</a> - <a href="#esmvaltool">ESMValTool</a> - <a href="#espeak-ng">eSpeak-NG</a> - <a href="#essentia">Essentia</a> - <a href="#ete">ETE</a> - <a href="#etsf_io">ETSF_IO</a> - <a href="#eudev">eudev</a> - <a href="#evidentialgene">EvidentialGene</a> - <a href="#evmix">evmix</a> - <a href="#exabayes">ExaBayes</a> - <a href="#examl">ExaML</a> - <a href="#excel-writer-xlsx">Excel-Writer-XLSX</a> - <a href="#exiftool">ExifTool</a> - <a href="#exiv2">exiv2</a> - <a href="#exonerate">Exonerate</a> - <a href="#expat">expat</a> - <a href="#expect">expect</a> - <a href="#expecttest">expecttest</a> - <a href="#express">eXpress</a> - <a href="#extrae">Extrae</a></p>
<h3 id="e-antic">E-ANTIC<a class="headerlink" href="#e-antic" title="Permanent link">&para;</a></h3>
<p>E-ANTIC is a C/C++ library to deal with real embedded number fields built on top of ANTIC (https://github.com/wbhart/antic). Its aim is to have as fast as possible exact arithmetic operations and comparisons.</p>
<p><em>homepage</em>: <a href="https://github.com/videlec/e-antic">https://github.com/videlec/e-antic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.1.5</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ea-utils">ea-utils<a class="headerlink" href="#ea-utils" title="Permanent link">&para;</a></h3>
<p>Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc.  Primarily written to support an Illumina based pipeline -  but should work with any FASTQs.</p>
<p><em>homepage</em>: <a href="http://expressionanalysis.github.io/ea-utils/">http://expressionanalysis.github.io/ea-utils/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.04.807</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="earthengine-api">earthengine-api<a class="headerlink" href="#earthengine-api" title="Permanent link">&para;</a></h3>
<p>Python and JavaScript bindings for calling the Earth Engine API</p>
<p><em>homepage</em>: <a href="https://github.com/google/earthengine-api">https://github.com/google/earthengine-api</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.143</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="easybuild">EasyBuild<a class="headerlink" href="#easybuild" title="Permanent link">&para;</a></h3>
<p>EasyBuild is a software build and installation framework  written in Python that allows you to install software in a structured,  repeatable and robust way.</p>
<p><em>homepage</em>: <a href="https://easybuilders.github.io/easybuild">https://easybuilders.github.io/easybuild</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.11.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.13.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.14.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.15.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.15.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.7.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="easyqc">EasyQC<a class="headerlink" href="#easyqc" title="Permanent link">&para;</a></h3>
<p>EasyQC is an R-package that provides advanced functionality to  (1) perform file-level QC of single genome-wide association (GWA) data-sets  (2) conduct quality control across several GWA data-sets (meta-level QC)  (3) simplify data-handling of large-scale GWA data-sets.</p>
<p><em>homepage</em>: <a href="http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/">http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.2</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="ecbuild">ecBuild<a class="headerlink" href="#ecbuild" title="Permanent link">&para;</a></h3>
<p>A CMake-based build system, consisting of a collection of CMake macros and functions that ease the managing of software build systems </p>
<p><em>homepage</em>: <a href="https://ecbuild.readthedocs.io/">https://ecbuild.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="eccodes">ecCodes<a class="headerlink" href="#eccodes" title="Permanent link">&para;</a></h3>
<p>ecCodes is a package developed by ECMWF which provides an application programming interface and  a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,  WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding).</p>
<p><em>homepage</em>: <a href="https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home">https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.3</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.7.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.9.2</code></td>
<td></td>
<td><code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>2.12.5</code></td>
<td></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>2.15.0</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>2.17.0</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>2.22.1</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>2.24.2</code></td>
<td></td>
<td><code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>2.27.0</code></td>
<td></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="eccodes-python">eccodes-python<a class="headerlink" href="#eccodes-python" title="Permanent link">&para;</a></h3>
<p>Python 3 interface to decode and encode GRIB and BUFR files via the ECMWF ecCodes library.</p>
<p><em>homepage</em>: <a href="https://github.com/ecmwf/eccodes-python">https://github.com/ecmwf/eccodes-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ed2">ED2<a class="headerlink" href="#ed2" title="Permanent link">&para;</a></h3>
<p>The Ecosystem Demography Biosphere Model (ED2) is an integrated terrestrial biosphere model  incorporating hydrology, land-surface biophysics, vegetation dynamics, and soil carbon and nitrogen biogeochemistry</p>
<p><em>homepage</em>: <a href="https://github.com/EDmodel/ED2">https://github.com/EDmodel/ED2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170201</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>20170201</code></td>
<td><code>-serial</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="edlib">edlib<a class="headerlink" href="#edlib" title="Permanent link">&para;</a></h3>
<p>Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.</p>
<p><em>homepage</em>: <a href="https://martinsos.github.io/edlib">https://martinsos.github.io/edlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.8.post1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.3.8.post1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.8.post2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>1.3.9</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="egglib">EggLib<a class="headerlink" href="#egglib" title="Permanent link">&para;</a></h3>
<p>EggLib is a C++/Python library and program package for evolutionary genetics and genomics.</p>
<p><em>homepage</em>: <a href="http://egglib.sourceforge.net/">http://egglib.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.10</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="eggnog-mapper">eggnog-mapper<a class="headerlink" href="#eggnog-mapper" title="Permanent link">&para;</a></h3>
<p>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only. Common uses of eggNOG-mapper include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.</p>
<p><em>homepage</em>: <a href="https://github.com/eggnogdb/eggnog-mapper">https://github.com/eggnogdb/eggnog-mapper</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.1.4</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.1.7</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.1.10</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="egttools">EGTtools<a class="headerlink" href="#egttools" title="Permanent link">&para;</a></h3>
<p>EGTtools provides a centralized repository with analytical and numerical methods to study/model game theoretical problems under the Evolutionary Game Theory (EGT) framework.</p>
<p><em>homepage</em>: <a href="https://github.com/Socrats/EGTTools">https://github.com/Socrats/EGTTools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.10.dev2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.1.11</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="eht-imaging">eht-imaging<a class="headerlink" href="#eht-imaging" title="Permanent link">&para;</a></h3>
<p>Python modules for simulating and manipulating VLBI data and producing images with regularized maximum likelihood methods.</p>
<p><em>homepage</em>: <a href="https://github.com/achael/eht-imaging">https://github.com/achael/eht-imaging</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="eigen">Eigen<a class="headerlink" href="#eigen" title="Permanent link">&para;</a></h3>
<p>Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,  and related algorithms.</p>
<p><em>homepage</em>: <a href="https://eigen.tuxfamily.org">https://eigen.tuxfamily.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.3</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.2.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.7</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2.8</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.2.9</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.2.10</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.7</code></td>
<td><code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.3.8</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.3.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="eigenexa">EigenExa<a class="headerlink" href="#eigenexa" title="Permanent link">&para;</a></h3>
<p>EigenExa, a part of KMATHLIB, is a high performance eigen-solver.</p>
<p><em>homepage</em>: <a href="https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/">https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="eigensoft">EIGENSOFT<a class="headerlink" href="#eigensoft" title="Permanent link">&para;</a></h3>
<p>The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.  2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal  components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of  variation;  the resulting correction is specific to a candidate markers variation in frequency across ancestral  populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT  package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.</p>
<p><em>homepage</em>: <a href="http://www.hsph.harvard.edu/alkes-price/software/">http://www.hsph.harvard.edu/alkes-price/software/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.0.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.1.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.1.4</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>7.2.1</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2021a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="einops">einops<a class="headerlink" href="#einops" title="Permanent link">&para;</a></h3>
<p>Flexible and powerful tensor operations for readable and reliable code. Supports numpy, pytorch, tensorflow, jax, and others.</p>
<p><em>homepage</em>: <a href="https://einops.rocks/">https://einops.rocks/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.4.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="elastix">elastix<a class="headerlink" href="#elastix" title="Permanent link">&para;</a></h3>
<p>elastix: a toolbox for rigid and nonrigid registration of images. </p>
<p><em>homepage</em>: <a href="http://elastix.isi.uu.nl/">http://elastix.isi.uu.nl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="elbencho">elbencho<a class="headerlink" href="#elbencho" title="Permanent link">&para;</a></h3>
<p>A distributed storage benchmark for files, objects &amp; blocks with support for GPUs</p>
<p><em>homepage</em>: <a href="https://github.com/breuner/elbencho">https://github.com/breuner/elbencho</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0-3</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="elfio">ELFIO<a class="headerlink" href="#elfio" title="Permanent link">&para;</a></h3>
<p>ELFIO is a header-only C++ library intended  for reading and generating files in the ELF binary format.</p>
<p><em>homepage</em>: <a href="http://elfio.sourceforge.net">http://elfio.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="elfutils">elfutils<a class="headerlink" href="#elfutils" title="Permanent link">&para;</a></h3>
<p>The elfutils project provides libraries and tools for ELF files  and DWARF data. </p>
<p><em>homepage</em>: <a href="https://elfutils.org/">https://elfutils.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.182</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.183</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.185</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.187</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>0.189</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="elk">Elk<a class="headerlink" href="#elk" title="Permanent link">&para;</a></h3>
<p>An all-electron full-potential linearised augmented-plane wave (FP-LAPW) code with many advanced features. Written originally at Karl-Franzens-Universitt Graz as a milestone of the EXCITING EU Research and Training Network, the code is designed to be as simple as possible so that new developments in the field of density functional theory (DFT) can be added quickly and reliably. </p>
<p><em>homepage</em>: <a href="http://elk.sourceforge.net/">http://elk.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.15</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.3.6</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.3.2</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>7.0.12</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>7.2.42</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>8.5.2</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="elpa">ELPA<a class="headerlink" href="#elpa" title="Permanent link">&para;</a></h3>
<p>Eigenvalue SoLvers for Petaflop-Applications.</p>
<p><em>homepage</em>: <a href="https://elpa.rzg.mpg.de">https://elpa.rzg.mpg.de</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2015.02.002</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>gimkl/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2016.05.004</code></td>
<td></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2016.11.001.pre</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2017.11.001</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2018.05.001</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2018.11.001</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2019.11.001</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019b</code>, <code>intel/2020a</code>, <code>iomkl/2019b</code></td>
</tr>
<tr>
<td><code>2020.05.001</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2020.11.001</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2021.05.001</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021a</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2021.05.002</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2021.11.001</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>2022.05.001</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>2022.05.001</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="elph">ELPH<a class="headerlink" href="#elph" title="Permanent link">&para;</a></h3>
<p>ELPH is a general-purpose Gibbs sampler for finding motifs in a set   of DNA or protein sequences. The program takes as input a set containing anywhere from   a few dozen to thousands of sequences, and searches through them for the most common motif,   assuming that each sequence contains one copy of the motif. We have used ELPH to find   patterns such as ribosome binding sites (RBSs) and exon splicing enhancers (ESEs). </p>
<p><em>homepage</em>: <a href="http://ccb.jhu.edu/software/ELPH/index.shtml">http://ccb.jhu.edu/software/ELPH/index.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="elprep">elprep<a class="headerlink" href="#elprep" title="Permanent link">&para;</a></h3>
<p>elPrep is a high-performance tool for analyzing .sam/.bam files (up to and including variant calling)  in sequencing pipelines.</p>
<p><em>homepage</em>: <a href="https://github.com/ExaScience/elprep">https://github.com/ExaScience/elprep</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="elsi">ELSI<a class="headerlink" href="#elsi" title="Permanent link">&para;</a></h3>
<p>ELSI provides and enhances scalable, open-source software library solutions for  electronic structure calculations in  materials science, condensed matter physics, chemistry, and many other fields.  ELSI focuses on methods that solve or circumvent eigenvalue problems in electronic structure theory.  The ELSI infrastructure should also be useful for other challenging eigenvalue problems. </p>
<p><em>homepage</em>: <a href="https://wordpress.elsi-interchange.org/">https://wordpress.elsi-interchange.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-PEXSI</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-PEXSI</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-PEXSI</code></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="elsi-rci">ELSI-RCI<a class="headerlink" href="#elsi-rci" title="Permanent link">&para;</a></h3>
<p>ELSI-RCI provides and enhances open-source software packages which iteratively solve or circumvent eigenvalue problems in self-consistent field calculations based on the Kohn-Sham density-functional theory. </p>
<p><em>homepage</em>: <a href="https://wordpress.elsi-interchange.org/">https://wordpress.elsi-interchange.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>foss/2020b</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="emacs">Emacs<a class="headerlink" href="#emacs" title="Permanent link">&para;</a></h3>
<p>GNU Emacs is an extensible, customizable text editor--and more.  At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming  language with extensions to support text editing.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/emacs/">https://www.gnu.org/software/emacs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>24.3</code></td>
<td></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>24.3</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>24.4</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>24.5</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>25.1</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>25.3</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>26.3</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>27.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>27.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>28.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="eman2">EMAN2<a class="headerlink" href="#eman2" title="Permanent link">&para;</a></h3>
<p>EMAN2 is a broadly based greyscale scientific image processing suite with a  primary focus on processing data from transmission electron microscopes. </p>
<p><em>homepage</em>: <a href="https://blake.bcm.edu/emanwiki/EMAN2">https://blake.bcm.edu/emanwiki/EMAN2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>2.21a</code></td>
<td><code>-Python-2.7.14-Boost-1.63.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="emboss">EMBOSS<a class="headerlink" href="#emboss" title="Permanent link">&para;</a></h3>
<p>EMBOSS is 'The European Molecular Biology Open Software Suite'.  EMBOSS is a free Open Source software analysis package specially developed for  the needs of the molecular biology (e.g. EMBnet) user community.</p>
<p><em>homepage</em>: <a href="http://emboss.sourceforge.net/">http://emboss.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.6.0</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2017a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>6.6.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>6.6.0</code></td>
<td><code>-Java-13</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>6.6.0</code></td>
<td><code>-X11-20170314</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="embree">Embree<a class="headerlink" href="#embree" title="Permanent link">&para;</a></h3>
<p>Intel Embree is a collection of high-performance ray tracing kernels, developed at Intel.  The target users of Intel Embree are graphics application engineers who want to improve the performance of  their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels.  The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions.  Intel Embree supports runtime code selection to choose the traversal and build algorithms that best matches  the instruction set of your CPU. We recommend using Intel Embree through its API to get the highest benefit  from future improvements. Intel Embree is released as Open Source under the Apache 2.0 license.</p>
<p><em>homepage</em>: <a href="https://embree.github.io/">https://embree.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.0</code></td>
<td><code>iccifort/2018.1.163-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>3.13.4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="emcee">emcee<a class="headerlink" href="#emcee" title="Permanent link">&para;</a></h3>
<p>Emcee is an extensible, pure-Python implementation of  Goodman &amp; Weare's Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler.  It's designed for Bayesian parameter estimation and it's really sweet! </p>
<p><em>homepage</em>: <a href="https://dfm.io/emcee">https://dfm.io/emcee</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="emu">EMU<a class="headerlink" href="#emu" title="Permanent link">&para;</a></h3>
<p>EMU infers population structure in the presence of missingness and works for both haploid, psuedo-haploid and diploid genotype datasets </p>
<p><em>homepage</em>: <a href="http://www.popgen.dk/software/index.php/EMU">http://www.popgen.dk/software/index.php/EMU</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.66</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="enabrowsertool">enaBrowserTool<a class="headerlink" href="#enabrowsertool" title="Permanent link">&para;</a></h3>
<p>enaBrowserTools is a set of scripts that interface with the ENA  web services to download data from ENA easily, without any knowledge of scripting required.</p>
<p><em>homepage</em>: <a href="https://github.com/enasequence/enaBrowserTools/">https://github.com/enasequence/enaBrowserTools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="enchant">enchant<a class="headerlink" href="#enchant" title="Permanent link">&para;</a></h3>
<p>Enchant is a library (and command-line program) that wraps a number of different spelling libraries  and programs with a consistent interface.</p>
<p><em>homepage</em>: <a href="https://abiword.github.io/enchant/">https://abiword.github.io/enchant/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="enchant-2">enchant-2<a class="headerlink" href="#enchant-2" title="Permanent link">&para;</a></h3>
<p>Enchant aims to provide a simple but comprehensive abstraction for dealing with different spell checking libraries in a consistent way. A client, such as a text editor or word processor, need not know anything about a specific spell-checker, and since all back-ends are plugins, new spell-checkers can be added without needing any change to the program using Enchant.</p>
<p><em>homepage</em>: <a href="https://github.com/AbiWord/enchant">https://github.com/AbiWord/enchant</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ensemblcoreapi">EnsEMBLCoreAPI<a class="headerlink" href="#ensemblcoreapi" title="Permanent link">&para;</a></h3>
<p>The Ensembl Core Perl API and SQL schema</p>
<p><em>homepage</em>: <a href="https://www.ensembl.org/info/docs/api/index.html">https://www.ensembl.org/info/docs/api/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>96.0-r20190601</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="entrypoints">entrypoints<a class="headerlink" href="#entrypoints" title="Permanent link">&para;</a></h3>
<p>Entry points are a way for Python packages to advertise objects with some common interface.</p>
<p><em>homepage</em>: <a href="https://github.com/takluyver/entrypoints">https://github.com/takluyver/entrypoints</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="epd">EPD<a class="headerlink" href="#epd" title="Permanent link">&para;</a></h3>
<p>The Enthought Python Distribution provides scientists with a comprehensive set of tools to perform  rigorous data analysis and visualization. Python, distinguished by its flexibility, coherence, and ease-of-use,  is rapidly becoming the programming language of choice for researchers worldwide.  EPD extends this capacity with a powerful collection of Python libraries to enable interactive technical computing and  cross-platform rapid application development.</p>
<p><em>homepage</em>: <a href="http://www.enthought.com/products/edudownload.php">http://www.enthought.com/products/edudownload.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.3-2-rh5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="epic">EPIC<a class="headerlink" href="#epic" title="Permanent link">&para;</a></h3>
<p>Package implementing EPIC method to estimate the proportion of immune, stromal, endothelial  and cancer or other cells from bulk gene expression data.</p>
<p><em>homepage</em>: <a href="https://gfellerlab.shinyapps.io/EPIC_1-1">https://gfellerlab.shinyapps.io/EPIC_1-1</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="episcanpy">epiScanpy<a class="headerlink" href="#episcanpy" title="Permanent link">&para;</a></h3>
<p>EpiScanpy is a toolkit to analyse single-cell open chromatin (scATAC-seq) and single-cell DNA methylation (for example scBS-seq) data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq analysis tool Scanpy (Genome Biology, 2018) [Wolf18].</p>
<p><em>homepage</em>: <a href="https://github.com/colomemaria/episcanpy">https://github.com/colomemaria/episcanpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="equilibrator">eQuilibrator<a class="headerlink" href="#equilibrator" title="Permanent link">&para;</a></h3>
<p>Calculation of standard thermodynamic potentials of biochemical reactions.</p>
<p><em>homepage</em>: <a href="https://equilibrator.weizmann.ac.il/">https://equilibrator.weizmann.ac.il/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.7</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="ericscript">EricScript<a class="headerlink" href="#ericscript" title="Permanent link">&para;</a></h3>
<p>EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://sites.google.com/site/bioericscript/home">https://sites.google.com/site/bioericscript/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.5</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="esl-bundle">ESL-Bundle<a class="headerlink" href="#esl-bundle" title="Permanent link">&para;</a></h3>
<p>The ESL Bundle is a collection of libraries and utilities broadly used in  electronic structure calculations, put together to make their use easier by  researchers and scientific software developers. ESL stands for Electronic  Structure Library, an initiative which distributes quality software and  promotes open standards for high-performance computing applications in the  field of electronic structure calculations. </p>
<p><em>homepage</em>: <a href="https://esl.cecam.org">https://esl.cecam.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="esm-2">ESM-2<a class="headerlink" href="#esm-2" title="Permanent link">&para;</a></h3>
<p>ESM-2 outperforms all tested single-sequence protein language models  across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate  accurate structure predictions end to end directly from the sequence of a protein.</p>
<p><em>homepage</em>: <a href="https://github.com/facebookresearch/esm">https://github.com/facebookresearch/esm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="esmf">ESMF<a class="headerlink" href="#esmf" title="Permanent link">&para;</a></h3>
<p>The Earth System Modeling Framework (ESMF) is a suite of software tools for developing  high-performance, multi-component Earth science modeling applications.</p>
<p><em>homepage</em>: <a href="https://www.earthsystemcog.org/projects/esmf/">https://www.earthsystemcog.org/projects/esmf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.3.0rp1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.3.0rp1</code></td>
<td><code>-HDF5-1.8.18</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>7.0.2</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>7.1.0r</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2020a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>8.3.0</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="esmpy">ESMPy<a class="headerlink" href="#esmpy" title="Permanent link">&para;</a></h3>
<p>Earth System Modeling Framework (ESMF) Python Interface</p>
<p><em>homepage</em>: <a href="https://earthsystemmodeling.org/esmpy">https://earthsystemmodeling.org/esmpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="esmvaltool">ESMValTool<a class="headerlink" href="#esmvaltool" title="Permanent link">&para;</a></h3>
<p>The Earth System Model eValuation Tool (ESMValTool)  is a community diagnostics and performance metrics tool  for the evaluation of Earth System Models (ESMs) that  allows for routine comparison of single or multiple models,  either against predecessor versions or against observations.</p>
<p><em>homepage</em>: <a href="https://www.esmvaltool.org/">https://www.esmvaltool.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="espeak-ng">eSpeak-NG<a class="headerlink" href="#espeak-ng" title="Permanent link">&para;</a></h3>
<p>The eSpeak NG is a compact open source software text-to-speech synthesizer for Linux, Windows, Android and other operating systems. It supports more than 100 languages and accents. It is based on the eSpeak engine created by Jonathan Duddington. </p>
<p><em>homepage</em>: <a href="https://github.com/espeak-ng/espeak-ng">https://github.com/espeak-ng/espeak-ng</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.50</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="essentia">Essentia<a class="headerlink" href="#essentia" title="Permanent link">&para;</a></h3>
<p>Open-source library and tools for audio and music analysis, description and synthesis</p>
<p><em>homepage</em>: <a href="https://essentia.upf.edu">https://essentia.upf.edu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1_beta5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ete">ETE<a class="headerlink" href="#ete" title="Permanent link">&para;</a></h3>
<p>A Python framework for the analysis and visualization of trees</p>
<p><em>homepage</em>: <a href="http://etetoolkit.org">http://etetoolkit.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0b36</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="etsf_io">ETSF_IO<a class="headerlink" href="#etsf_io" title="Permanent link">&para;</a></h3>
<p>A library of F90 routines to read/write the ETSF file  format has been written. It is called ETSF_IO and available under LGPL. </p>
<p><em>homepage</em>: <a href="http://www.etsf.eu/resources/software/libraries_and_tools">http://www.etsf.eu/resources/software/libraries_and_tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="eudev">eudev<a class="headerlink" href="#eudev" title="Permanent link">&para;</a></h3>
<p>eudev is a fork of systemd-udev with the goal of obtaining better  compatibility with existing software such as OpenRC and Upstart,  older kernels, various toolchains and anything else required by  users and various distributions. </p>
<p><em>homepage</em>: <a href="https://wiki.gentoo.org/wiki/Project:Eudev">https://wiki.gentoo.org/wiki/Project:Eudev</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.5</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="evidentialgene">EvidentialGene<a class="headerlink" href="#evidentialgene" title="Permanent link">&para;</a></h3>
<p>EvidentialGene is a genome informatics project for  "Evidence Directed Gene Construction for Eukaryotes",  for constructing high quality, accurate gene sets for  animals and plants (any eukaryotes), being developed by  Don Gilbert at Indiana University, gilbertd at indiana edu.</p>
<p><em>homepage</em>: <a href="http://arthropods.eugenes.org/EvidentialGene/">http://arthropods.eugenes.org/EvidentialGene/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.01.01</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2022.01.14</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="evmix">evmix<a class="headerlink" href="#evmix" title="Permanent link">&para;</a></h3>
<p>evmix: Extreme Value Mixture Modelling,  Threshold Estimation and Boundary Corrected Kernel Density Estimation</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/evmix">https://cran.r-project.org/web/packages/evmix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="exabayes">ExaBayes<a class="headerlink" href="#exabayes" title="Permanent link">&para;</a></h3>
<p>ExaBayes is a software package for Bayesian tree inference.  It is particularly suitable for large-scale analyses on computer clusters</p>
<p><em>homepage</em>: <a href="https://cme.h-its.org/exelixis/web/software/exabayes/">https://cme.h-its.org/exelixis/web/software/exabayes/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="examl">ExaML<a class="headerlink" href="#examl" title="Permanent link">&para;</a></h3>
<p>Exascale Maximum Likelihood (ExaML) code for phylogenetic inference using MPI</p>
<p><em>homepage</em>: <a href="https://github.com/stamatak/ExaML/">https://github.com/stamatak/ExaML/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.22</code></td>
<td><code>-hybrid-avx</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="excel-writer-xlsx">Excel-Writer-XLSX<a class="headerlink" href="#excel-writer-xlsx" title="Permanent link">&para;</a></h3>
<p>The Excel::Writer::XLSX module can be used to create an Excel file in the 2007+ XLSX format. Multiple worksheets can be added to a workbook and formatting can be applied to cells. Text, numbers, and formulas can be written to the cells.</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Excel::Writer::XLSX">https://metacpan.org/pod/Excel::Writer::XLSX</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.09</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="exiftool">ExifTool<a class="headerlink" href="#exiftool" title="Permanent link">&para;</a></h3>
<p>Perl module (Image::ExifTool) and program (exiftool)  to read EXIF information from images</p>
<p><em>homepage</em>: <a href="https://owl.phy.queensu.ca/~phil/exiftool/">https://owl.phy.queensu.ca/~phil/exiftool/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>12.00</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="exiv2">exiv2<a class="headerlink" href="#exiv2" title="Permanent link">&para;</a></h3>
<p>Exiv2 is a Cross-platform C++ library and a command line utility to manage image metadata.</p>
<p><em>homepage</em>: <a href="https://exiv2.org">https://exiv2.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.27.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.27.5</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="exonerate">Exonerate<a class="headerlink" href="#exonerate" title="Permanent link">&para;</a></h3>
<p>Exonerate is a generic tool for pairwise sequence comparison.  It allows you to align sequences using a many alignment models, using either   exhaustive dynamic programming, or a variety of heuristics. </p>
<p><em>homepage</em>: <a href="https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate">https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/6.4.0-2.28</code>, <code>GCC/8.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="expat">expat<a class="headerlink" href="#expat" title="Permanent link">&para;</a></h3>
<p>Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags).</p>
<p><em>homepage</em>: <a href="https://libexpat.github.io">https://libexpat.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCC/4.9.2</code>, <code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.2.6</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.2.9</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.4.8</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code></td>
</tr>
<tr>
<td><code>2.4.9</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="expect">expect<a class="headerlink" href="#expect" title="Permanent link">&para;</a></h3>
<p>Expect is a tool for automating interactive applications    such as telnet, ftp, passwd, fsck, rlogin, tip, etc.    Expect really makes this stuff trivial.    Expect is also useful for testing these same applications.</p>
<p><em>homepage</em>: <a href="https://core.tcl.tk/expect/index">https://core.tcl.tk/expect/index</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.45.4</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="expecttest">expecttest<a class="headerlink" href="#expecttest" title="Permanent link">&para;</a></h3>
<p>This library implements expect tests (also known as "golden" tests). Expect tests are a method of  writing tests where instead of hard-coding the expected output of a test, you run the test to get the output, and  the test framework automatically populates the expected output. If the output of the test changes, you can rerun  the test with the environment variable EXPECTTEST_ACCEPT=1 to automatically update the expected output.</p>
<p><em>homepage</em>: <a href="https://github.com/ezyang/expecttest">https://github.com/ezyang/expecttest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="express">eXpress<a class="headerlink" href="#express" title="Permanent link">&para;</a></h3>
<p>Streaming quantification for high-throughput sequencing</p>
<p><em>homepage</em>: <a href="https://pachterlab.github.io/eXpress">https://pachterlab.github.io/eXpress</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="extrae">Extrae<a class="headerlink" href="#extrae" title="Permanent link">&para;</a></h3>
<p>Extrae is the core instrumentation package developed by the Performance Tools group at BSC. Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches. The information gathered by Extrae typically includes timestamped events of runtime calls, performance counters and source code references. Besides, Extrae provides its own API to allow the user to manually instrument his or her application.</p>
<p><em>homepage</em>: <a href="https://www.bsc.es/computer-sciences/performance-tools">https://www.bsc.es/computer-sciences/performance-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>3.8.3</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h2 id="f">F<a class="headerlink" href="#f" title="Permanent link">&para;</a></h2>
<p><a href="#f90cache">f90cache</a> - <a href="#f90wrap">f90wrap</a> - <a href="#faber">Faber</a> - <a href="#fabio">FabIO</a> - <a href="#faceswap">faceswap</a> - <a href="#falcon">FALCON</a> - <a href="#fann">FANN</a> - <a href="#fast5">fast5</a> - <a href="#fasta">FASTA</a> - <a href="#fastahack">fastahack</a> - <a href="#fastai">fastai</a> - <a href="#fastaindex">FastaIndex</a> - <a href="#fastani">FastANI</a> - <a href="#fastaq">Fastaq</a> - <a href="#fastfold">FastFold</a> - <a href="#fastjet">fastjet</a> - <a href="#fastjet-contrib">fastjet-contrib</a> - <a href="#fastme">FastME</a> - <a href="#fastp">fastp</a> - <a href="#fastparquet">fastparquet</a> - <a href="#fastphase">fastPHASE</a> - <a href="#fastq-pair">fastq-pair</a> - <a href="#fastq-tools">fastq-tools</a> - <a href="#fastq_screen">FastQ_Screen</a> - <a href="#fastqc">FastQC</a> - <a href="#fastqsplitter">fastqsplitter</a> - <a href="#fastqtl">FastQTL</a> - <a href="#fastqz">fastqz</a> - <a href="#fastrfs">FastRFS</a> - <a href="#faststructure">fastStructure</a> - <a href="#fasttree">FastTree</a> - <a href="#fastviromeexplorer">FastViromeExplorer</a> - <a href="#fastx-toolkit">FASTX-Toolkit</a> - <a href="#fatslim">fatslim</a> - <a href="#fbm">fbm</a> - <a href="#fbpic">FBPIC</a> - <a href="#fcc">FCC</a> - <a href="#fcm">FCM</a> - <a href="#fdict">fdict</a> - <a href="#fds">FDS</a> - <a href="#fdstools">fdstools</a> - <a href="#fdtd_solutions">FDTD_Solutions</a> - <a href="#feh">feh</a> - <a href="#fenics">FEniCS</a> - <a href="#fermi-lite">fermi-lite</a> - <a href="#ferret">Ferret</a> - <a href="#festival">festival</a> - <a href="#fetchmg">fetchMG</a> - <a href="#ffc">FFC</a> - <a href="#fflas-ffpack">FFLAS-FFPACK</a> - <a href="#ffmpeg">FFmpeg</a> - <a href="#ffmpi">ffmpi</a> - <a href="#ffnet">ffnet</a> - <a href="#ffnvcodec">ffnvcodec</a> - <a href="#fftlib">fftlib</a> - <a href="#fftw">FFTW</a> - <a href="#fftw.mpi">FFTW.MPI</a> - <a href="#fgbio">fgbio</a> - <a href="#fgsl">FGSL</a> - <a href="#fhi-aims">FHI-aims</a> - <a href="#fiat">FIAT</a> - <a href="#figaro">FIGARO</a> - <a href="#figuregen">FigureGen</a> - <a href="#fiji">Fiji</a> - <a href="#file">file</a> - <a href="#filevercmp">filevercmp</a> - <a href="#filtlong">Filtlong</a> - <a href="#find_circ">find_circ</a> - <a href="#finder">finder</a> - <a href="#findhap">findhap</a> - <a href="#findutils">findutils</a> - <a href="#fineradstructure">fineRADstructure</a> - <a href="#finestructure">fineSTRUCTURE</a> - <a href="#fio">fio</a> - <a href="#fiona">Fiona</a> - <a href="#firefox">Firefox</a> - <a href="#firestarter">FIRESTARTER</a> - <a href="#fireworks">FireWorks</a> - <a href="#fit-sne">FIt-SNE</a> - <a href="#fix">FIX</a> - <a href="#fixesproto">fixesproto</a> - <a href="#flac">FLAC</a> - <a href="#flair">FLAIR</a> - <a href="#flair-nlp">flair-NLP</a> - <a href="#flann">FLANN</a> - <a href="#flash">FLASH</a> - <a href="#flask">Flask</a> - <a href="#flatbuffers">flatbuffers</a> - <a href="#flatbuffers-python">flatbuffers-python</a> - <a href="#fleur">FLEUR</a> - <a href="#flex">flex</a> - <a href="#flexbar">Flexbar</a> - <a href="#flexiblas">FlexiBLAS</a> - <a href="#flexidot">FlexiDot</a> - <a href="#flink">Flink</a> - <a href="#flint">FLINT</a> - <a href="#flook">flook</a> - <a href="#flowfda">flowFDA</a> - <a href="#fltk">FLTK</a> - <a href="#fluent">FLUENT</a> - <a href="#flye">Flye</a> - <a href="#fmilibrary">FMILibrary</a> - <a href="#fmm3d">FMM3D</a> - <a href="#fmpy">FMPy</a> - <a href="#fmriprep">FMRIprep</a> - <a href="#fms">FMS</a> - <a href="#fmt">fmt</a> - <a href="#fobis">FoBiS</a> - <a href="#foldx">FoldX</a> - <a href="#fontconfig">fontconfig</a> - <a href="#fontsproto">fontsproto</a> - <a href="#ford">FORD</a> - <a href="#foss">foss</a> - <a href="#fosscuda">fosscuda</a> - <a href="#fox">FoX</a> - <a href="#fox-toolkit">FOX-Toolkit</a> - <a href="#fpm">FPM</a> - <a href="#fpocket">fpocket</a> - <a href="#fqtrim">fqtrim</a> - <a href="#fqzcomp">fqzcomp</a> - <a href="#fraggenescan">FragGeneScan</a> - <a href="#franz">FRANz</a> - <a href="#freebarcodes">FreeBarcodes</a> - <a href="#freebayes">freebayes</a> - <a href="#freefem">FreeFEM</a> - <a href="#freefem++">FreeFem++</a> - <a href="#freeglut">freeglut</a> - <a href="#freeimage">FreeImage</a> - <a href="#freesasa">FreeSASA</a> - <a href="#freesurfer">FreeSurfer</a> - <a href="#freetds">FreeTDS</a> - <a href="#freetype">freetype</a> - <a href="#freetype-py">freetype-py</a> - <a href="#freexl">FreeXL</a> - <a href="#freud-analysis">freud-analysis</a> - <a href="#fribidi">FriBidi</a> - <a href="#fruit">FRUIT</a> - <a href="#fruit_processor">FRUIT_processor</a> - <a href="#fsl">FSL</a> - <a href="#fsleyes">FSLeyes</a> - <a href="#fsom">fsom</a> - <a href="#fson">FSON</a> - <a href="#ftgl">FTGL</a> - <a href="#fujitsu">Fujitsu</a> - <a href="#fullrmc">fullrmc</a> - <a href="#fumi_tools">fumi_tools</a> - <a href="#funwave-tvd">FUNWAVE-TVD</a> - <a href="#fuse">FUSE</a> - <a href="#fuseq">FuSeq</a> - <a href="#fusioncatcher">FusionCatcher</a> - <a href="#futhark">futhark</a> - <a href="#futile">futile</a> - <a href="#future">future</a> - <a href="#fxtract">fxtract</a></p>
<h3 id="f90cache">f90cache<a class="headerlink" href="#f90cache" title="Permanent link">&para;</a></h3>
<p>f90cache is a compiler cache. It acts as a caching pre-processor to Fortran compilers,  using the -E compiler switch and a hash to detect when a compilation can be satisfied from cache.  This often results in a great speedup in common compilations.</p>
<p><em>homepage</em>: <a href="http://people.irisa.fr/Edouard.Canot/f90cache/">http://people.irisa.fr/Edouard.Canot/f90cache/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.96</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="f90wrap">f90wrap<a class="headerlink" href="#f90wrap" title="Permanent link">&para;</a></h3>
<p>f90wrap is a tool to automatically generate Python extension modules which interface to Fortran code that makes use of derived types. It builds on the capabilities of the popular f2py utility by generating a simpler Fortran 90 interface to the original Fortran code which is then suitable for wrapping with f2py, together with a higher-level Pythonic wrapper that makes the existance of an additional layer transparent to the final user.</p>
<p><em>homepage</em>: <a href="https://github.com/jameskermode/f90wrap">https://github.com/jameskermode/f90wrap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.8</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="faber">Faber<a class="headerlink" href="#faber" title="Permanent link">&para;</a></h3>
<p>Faber started as a clone of Boost.Build, to experiment with a  new Python frontend. Meanwhile it has evolved into a new build system, which  retains most of the features found in Boost.Build, but with (hopefully !) much  simplified logic, in addition of course to using Python as scripting language,  rather than Jam. The original bjam engine is still in use as scheduler, though  at this point that is mostly an implementation detail.</p>
<p><em>homepage</em>: <a href="https://stefanseefeld.github.io/faber">https://stefanseefeld.github.io/faber</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fabio">FabIO<a class="headerlink" href="#fabio" title="Permanent link">&para;</a></h3>
<p>FabIO is an I/O library for images produced by 2D X-ray detectors and written in Python.  FabIO support images detectors from a dozen of companies (including Mar, Dectris, ADSC, Hamamatsu, Oxford, ...),  for a total of 20 different file formats (like CBF, EDF, TIFF, ...) and offers an unified interface to their headers  (as a python dictionary) and datasets (as a numpy ndarray of integers or floats).</p>
<p><em>homepage</em>: <a href="http://www.silx.org">http://www.silx.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.14.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="faceswap">faceswap<a class="headerlink" href="#faceswap" title="Permanent link">&para;</a></h3>
<p>Faceswap is a tool that utilizes deep learning to recognize and swap faces in pictures and videos.</p>
<p><em>homepage</em>: <a href="https://github.com/deepfakes/faceswap">https://github.com/deepfakes/faceswap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180212</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="falcon">FALCON<a class="headerlink" href="#falcon" title="Permanent link">&para;</a></h3>
<p>Falcon: a set of tools for fast aligning long reads for consensus and assembly</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/FALCON">https://github.com/PacificBiosciences/FALCON</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.8</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fann">FANN<a class="headerlink" href="#fann" title="Permanent link">&para;</a></h3>
<p>Fast Artificial Neural Network Library is a free open source neural network library,  which implements multilayer artificial neural networks in C with support for both fully connected  and sparsely connected networks.</p>
<p><em>homepage</em>: <a href="http://leenissen.dk">http://leenissen.dk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fast5">fast5<a class="headerlink" href="#fast5" title="Permanent link">&para;</a></h3>
<p>A lightweight C++ library for accessing Oxford Nanopore  Technologies sequencing data.</p>
<p><em>homepage</em>: <a href="http://simpsonlab.github.io/2017/02/27/packing_fast5/">http://simpsonlab.github.io/2017/02/27/packing_fast5/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fasta">FASTA<a class="headerlink" href="#fasta" title="Permanent link">&para;</a></h3>
<p>The FASTA programs find regions of local or global (new) similarity between  protein or DNA sequences, either by searching Protein or DNA databases, or by identifying  local duplications within a sequence.</p>
<p><em>homepage</em>: <a href="http://fasta.bioch.virginia.edu">http://fasta.bioch.virginia.edu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>36.3.5e</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastahack">fastahack<a class="headerlink" href="#fastahack" title="Permanent link">&para;</a></h3>
<p>Utilities for indexing and sequence extraction from FASTA files.</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/fastahack">https://github.com/ekg/fastahack</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastai">fastai<a class="headerlink" href="#fastai" title="Permanent link">&para;</a></h3>
<p>The fastai deep learning library.</p>
<p><em>homepage</em>: <a href="https://www.fast.ai/">https://www.fast.ai/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.10</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>2.7.10</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fastaindex">FastaIndex<a class="headerlink" href="#fastaindex" title="Permanent link">&para;</a></h3>
<p>FastA index (.fai) handler compatible with samtools faidx</p>
<p><em>homepage</em>: <a href="https://github.com/lpryszcz/FastaIndex">https://github.com/lpryszcz/FastaIndex</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11rc7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastani">FastANI<a class="headerlink" href="#fastani" title="Permanent link">&para;</a></h3>
<p>FastANI is developed for fast alignment-free computation of  whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean  nucleotide identity of orthologous gene pairs shared between two microbial  genomes. FastANI supports pairwise comparison of both complete and draft  genome assemblies.</p>
<p><em>homepage</em>: <a href="https://github.com/ParBLiSS/FastANI">https://github.com/ParBLiSS/FastANI</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.31</code></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>1.33</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastaq">Fastaq<a class="headerlink" href="#fastaq" title="Permanent link">&para;</a></h3>
<p>Python3 scripts to manipulate FASTA and FASTQ files.</p>
<p><em>homepage</em>: <a href="https://github.com/sanger-pathogens/Fastaq">https://github.com/sanger-pathogens/Fastaq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.17.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastfold">FastFold<a class="headerlink" href="#fastfold" title="Permanent link">&para;</a></h3>
<p>Optimizing Protein Structure Prediction Model Training and Inference on GPU Clusters</p>
<p><em>homepage</em>: <a href="https://github.com/hpcaitech/FastFold">https://github.com/hpcaitech/FastFold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220729</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="fastjet">fastjet<a class="headerlink" href="#fastjet" title="Permanent link">&para;</a></h3>
<p>A software package for jet finding in pp and e+e- collisions</p>
<p><em>homepage</em>: <a href="https://fastjet.fr/">https://fastjet.fr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fastjet-contrib">fastjet-contrib<a class="headerlink" href="#fastjet-contrib" title="Permanent link">&para;</a></h3>
<p>3rd party extensions of FastJet</p>
<p><em>homepage</em>: <a href="https://fastjet.hepforge.org/contrib/">https://fastjet.hepforge.org/contrib/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.049</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fastme">FastME<a class="headerlink" href="#fastme" title="Permanent link">&para;</a></h3>
<p>FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.</p>
<p><em>homepage</em>: <a href="http://www.atgc-montpellier.fr/fastme/">http://www.atgc-montpellier.fr/fastme/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.5</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.1.6.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastp">fastp<a class="headerlink" href="#fastp" title="Permanent link">&para;</a></h3>
<p>A tool designed to provide fast all-in-one preprocessing for FastQ files.  This tool is developed in C++ with multithreading supported to afford high performance.</p>
<p><em>homepage</em>: <a href="https://github.com/OpenGene/fastp">https://github.com/OpenGene/fastp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.19.7</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.23.2</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastparquet">fastparquet<a class="headerlink" href="#fastparquet" title="Permanent link">&para;</a></h3>
<p>fastparquet is a python implementation of the parquet format, aiming to integrate into python-based big data work-flows. It is used implicitly by the projects Dask, Pandas and intake-parquet.</p>
<p><em>homepage</em>: <a href="https://fastparquet.readthedocs.io/">https://fastparquet.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastphase">fastPHASE<a class="headerlink" href="#fastphase" title="Permanent link">&para;</a></h3>
<p>fastPHASE: software for haplotype reconstruction,  and estimating missing genotypes from population data Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf</p>
<p><em>homepage</em>: <a href="http://stephenslab.uchicago.edu/software.html#fastphase">http://stephenslab.uchicago.edu/software.html#fastphase</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fastq-pair">fastq-pair<a class="headerlink" href="#fastq-pair" title="Permanent link">&para;</a></h3>
<p>Match up paired end fastq files quickly and efficiently.</p>
<p><em>homepage</em>: <a href="https://github.com/linsalrob/fastq-pair">https://github.com/linsalrob/fastq-pair</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastq-tools">fastq-tools<a class="headerlink" href="#fastq-tools" title="Permanent link">&para;</a></h3>
<p>This package provides a number of small and efficient programs to perform  common tasks with high throughput sequencing data in the FASTQ format. All of the programs  work with typical FASTQ files as well as gzipped FASTQ files.</p>
<p><em>homepage</em>: <a href="https://homes.cs.washington.edu/~dcjones/fastq-tools/">https://homes.cs.washington.edu/~dcjones/fastq-tools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.8.3</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastq_screen">FastQ_Screen<a class="headerlink" href="#fastq_screen" title="Permanent link">&para;</a></h3>
<p>FastQ Screen allows you to screen a library of sequences in FastQ  format against a set of sequence databases so you can see if the composition of the  library matches with what you expect.</p>
<p><em>homepage</em>: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/">http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.3</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.11.4</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.14.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastqc">FastQC<a class="headerlink" href="#fastqc" title="Permanent link">&para;</a></h3>
<p>FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.</p>
<p><em>homepage</em>: <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.1</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td><code>-Java-1.7.0_60</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.3</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.4</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.4</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.5</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.5</code></td>
<td><code>-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.5</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.7</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.8</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.8</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.9</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.9</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fastqsplitter">fastqsplitter<a class="headerlink" href="#fastqsplitter" title="Permanent link">&para;</a></h3>
<p>Splits fastq files evenly.</p>
<p><em>homepage</em>: <a href="https://github.com/LUMC/fastqsplitter">https://github.com/LUMC/fastqsplitter</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastqtl">FastQTL<a class="headerlink" href="#fastqtl" title="Permanent link">&para;</a></h3>
<p>FastQTL is a QTL mapper</p>
<p><em>homepage</em>: <a href="http://fastqtl.sourceforge.net/">http://fastqtl.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.184</code></td>
<td><code>GCC/11.2.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastqz">fastqz<a class="headerlink" href="#fastqz" title="Permanent link">&para;</a></h3>
<p>fastqz is a compressor for FASTQ files. FASTQ is the output of DNA sequencing machines.  It is one of the compressors described in the paper: Bonfield JK, Mahoney MV (2013) Compression of  FASTQ and SAM Format Sequencing Data. (mirror) PLoS ONE 8(3): e59190. doi:10.1371/journal.pone.0059190</p>
<p><em>homepage</em>: <a href="http://mattmahoney.net/dc/fastqz/">http://mattmahoney.net/dc/fastqz/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="fastrfs">FastRFS<a class="headerlink" href="#fastrfs" title="Permanent link">&para;</a></h3>
<p>Fast Robinson Foulds Supertrees</p>
<p><em>homepage</em>: <a href="https://github.com/pranjalv123/FastRFS">https://github.com/pranjalv123/FastRFS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-20190613</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="faststructure">fastStructure<a class="headerlink" href="#faststructure" title="Permanent link">&para;</a></h3>
<p>fastStructure is a fast algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x. </p>
<p><em>homepage</em>: <a href="https://rajanil.github.io/fastStructure/">https://rajanil.github.io/fastStructure/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="fasttree">FastTree<a class="headerlink" href="#fasttree" title="Permanent link">&para;</a></h3>
<p>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. </p>
<p><em>homepage</em>: <a href="http://www.microbesonline.org/fasttree/">http://www.microbesonline.org/fasttree/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.10</code></td>
<td><code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1.11</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fastviromeexplorer">FastViromeExplorer<a class="headerlink" href="#fastviromeexplorer" title="Permanent link">&para;</a></h3>
<p>Identify the viruses/phages and their abundance in the viral metagenomics data.</p>
<p><em>homepage</em>: <a href="https://code.vt.edu/saima5/FastViromeExplorer">https://code.vt.edu/saima5/FastViromeExplorer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180422</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fastx-toolkit">FASTX-Toolkit<a class="headerlink" href="#fastx-toolkit" title="Permanent link">&para;</a></h3>
<p>The FASTX-Toolkit is a collection of command line tools for   Short-Reads FASTA/FASTQ files preprocessing.</p>
<p><em>homepage</em>: <a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.14</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/9.3.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fatslim">fatslim<a class="headerlink" href="#fatslim" title="Permanent link">&para;</a></h3>
<p>FATSLiM stands for Fast Analysis Toolbox for Simulations of Lipid Membranes and its goal is to provide an efficient, yet robust, tool to extract physical parameters from MD trajectories.</p>
<p><em>homepage</em>: <a href="https://github.com/FATSLiM/fatslim">https://github.com/FATSLiM/fatslim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fbm">fbm<a class="headerlink" href="#fbm" title="Permanent link">&para;</a></h3>
<p>Exact methods for simulating fractional Brownian motion and fractional Gaussian noise in Python</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/fbm">https://pypi.org/project/fbm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fbpic">FBPIC<a class="headerlink" href="#fbpic" title="Permanent link">&para;</a></h3>
<p>FBPIC (Fourier-Bessel Particle-In-Cell) is a Particle-In-Cell (PIC) code for relativistic plasma  physics. It is especially well-suited for physical simulations of laser-wakefield acceleration and  plasma-wakefield acceleration.</p>
<p><em>homepage</em>: <a href="https://fbpic.github.io/">https://fbpic.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.20.3</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="fcc">FCC<a class="headerlink" href="#fcc" title="Permanent link">&para;</a></h3>
<p>Fujitsu Compiler based compiler toolchain.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fcm">FCM<a class="headerlink" href="#fcm" title="Permanent link">&para;</a></h3>
<p>FCM is a set of tools for managing and building source code.</p>
<p><em>homepage</em>: <a href="http://www.metoffice.gov.uk/research/collaboration/fcm">http://www.metoffice.gov.uk/research/collaboration/fcm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fdict">fdict<a class="headerlink" href="#fdict" title="Permanent link">&para;</a></h3>
<p>A variable and dictionary in pure fortran for retaining any data-type and a fast hash-table dictionary. </p>
<p><em>homepage</em>: <a href="https://github.com/zerothi/fdict">https://github.com/zerothi/fdict</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fds">FDS<a class="headerlink" href="#fds" title="Permanent link">&para;</a></h3>
<p>Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.</p>
<p><em>homepage</em>: <a href="https://pages.nist.gov/fds-smv">https://pages.nist.gov/fds-smv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.0.1</code></td>
<td><code>-no-OFED</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.5.2</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>6.5.3</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.6.0</code></td>
<td></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.7.0</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.7.4</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.7.5</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.7.6</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>6.7.7</code></td>
<td></td>
<td><code>intel/2021b</code></td>
</tr>
<tr>
<td><code>6.7.9</code></td>
<td></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fdstools">fdstools<a class="headerlink" href="#fdstools" title="Permanent link">&para;</a></h3>
<p>Forensic DNA Sequencing Tools Tools for characterisation and filtering of PCR stutter artefacts  and other systemic noise in Next Generation Sequencing data of forensic STR markers.</p>
<p><em>homepage</em>: <a href="https://git.lumc.nl/jerryhoogenboom/fdstools">https://git.lumc.nl/jerryhoogenboom/fdstools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160322</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="fdtd_solutions">FDTD_Solutions<a class="headerlink" href="#fdtd_solutions" title="Permanent link">&para;</a></h3>
<p>High performance FDTD-method Maxwell solver for the design, analysis and optimization of nanophotonic devices, processes and materials.</p>
<p><em>homepage</em>: <a href="http://www.lumerical.com/tcad-products/fdtd/">http://www.lumerical.com/tcad-products/fdtd/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.6.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.11.337</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.16.982</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.20.1731</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="feh">feh<a class="headerlink" href="#feh" title="Permanent link">&para;</a></h3>
<p>feh is an X11 image viewer aimed mostly at console users. Unlike most other viewers, it does not have a fancy GUI, but simply displays images. It is controlled via commandline arguments and configurable key/mouse actions. </p>
<p><em>homepage</em>: <a href="https://feh.finalrewind.org/">https://feh.finalrewind.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.26</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fenics">FEniCS<a class="headerlink" href="#fenics" title="Permanent link">&para;</a></h3>
<p>FEniCS is a computing platform for solving partial differential equations (PDEs).</p>
<p><em>homepage</em>: <a href="https://fenicsproject.org/">https://fenicsproject.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fermi-lite">fermi-lite<a class="headerlink" href="#fermi-lite" title="Permanent link">&para;</a></h3>
<p>Standalone C library for assembling Illumina short reads in small regions.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/fermi-lite">https://github.com/lh3/fermi-lite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190320</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ferret">Ferret<a class="headerlink" href="#ferret" title="Permanent link">&para;</a></h3>
<p>Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets.</p>
<p><em>homepage</em>: <a href="https://ferret.pmel.noaa.gov/Ferret/">https://ferret.pmel.noaa.gov/Ferret/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>7.5.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="festival">festival<a class="headerlink" href="#festival" title="Permanent link">&para;</a></h3>
<p>University of Edinburgh's Festival Speech Synthesis Systems is a free software multi-lingual speech synthesis workbench that runs on multiple-platforms offering black box text to speech, as well as an open architecture for research in speech synthesis. It designed as a component of large speech technology systems. </p>
<p><em>homepage</em>: &lt;['http://festvox.org/festival/']&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fetchmg">fetchMG<a class="headerlink" href="#fetchmg" title="Permanent link">&para;</a></h3>
<p>The program fetchMG was written to extract the 40 MGs from genomes and metagenomes  in an easy and accurate manner.</p>
<p><em>homepage</em>: <a href="http://vm-lux.embl.de/~mende/fetchMG/about.html">http://vm-lux.embl.de/~mende/fetchMG/about.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ffc">FFC<a class="headerlink" href="#ffc" title="Permanent link">&para;</a></h3>
<p>The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/fenics-project/ffc">https://bitbucket.org/fenics-project/ffc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2019.1.0.post0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fflas-ffpack">FFLAS-FFPACK<a class="headerlink" href="#fflas-ffpack" title="Permanent link">&para;</a></h3>
<p>Finite Field Linear Algebra Subroutines / Package</p>
<p><em>homepage</em>: <a href="https://linbox-team.github.io/fflas-ffpack/">https://linbox-team.github.io/fflas-ffpack/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ffmpeg">FFmpeg<a class="headerlink" href="#ffmpeg" title="Permanent link">&para;</a></h3>
<p>A complete, cross-platform solution to record, convert and stream audio and video.</p>
<p><em>homepage</em>: <a href="https://www.ffmpeg.org/">https://www.ffmpeg.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.16</code></td>
<td><code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.8.6</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.8.7</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.2.4</code></td>
<td><code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.3.4</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.5</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.1.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.3.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.4.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ffmpi">ffmpi<a class="headerlink" href="#ffmpi" title="Permanent link">&para;</a></h3>
<p>Fujitsu Compiler based compiler toolchain, including Fujitsu MPI for MPI support.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ffnet">ffnet<a class="headerlink" href="#ffnet" title="Permanent link">&para;</a></h3>
<p>Feed-forward neural network solution for python</p>
<p><em>homepage</em>: <a href="http://ffnet.sourceforge.net/">http://ffnet.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ffnvcodec">ffnvcodec<a class="headerlink" href="#ffnvcodec" title="Permanent link">&para;</a></h3>
<p>FFmpeg nvidia headers. Adds support for nvenc and nvdec. Requires Nvidia GPU and drivers to be present (picked up dynamically).</p>
<p><em>homepage</em>: <a href="https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git">https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.1.5.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fftlib">fftlib<a class="headerlink" href="#fftlib" title="Permanent link">&para;</a></h3>
<p>A library that intercepts FFTW calls and adds features on top of it.  In particular, it enables FFT plan reuse when there are multiple calls for the same geometry.</p>
<p><em>homepage</em>: <a href="https://github.com/flwende/fftlib">https://github.com/flwende/fftlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170628</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="fftw">FFTW<a class="headerlink" href="#fftw" title="Permanent link">&para;</a></h3>
<p>FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.</p>
<p><em>homepage</em>: <a href="https://www.fftw.org">https://www.fftw.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-fujitsu</code></td>
<td><code>FCC/4.5.0</code></td>
</tr>
<tr>
<td><code>2.1.5</code></td>
<td></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.3.4</code></td>
<td></td>
<td><code>gmpich/2016a</code>, <code>gmvapich2/1.7.20</code>, <code>gmvapich2/2016a</code>, <code>gompi/2016.04</code>, <code>gompi/2016.06</code>, <code>gompi/2016.07</code>, <code>gompi/2016a</code>, <code>gompi/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.5</code></td>
<td></td>
<td><code>gompi/2016.07</code>, <code>gompi/2016.09</code>, <code>gompi/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.6</code></td>
<td></td>
<td><code>gimpi/2017b</code>, <code>gimpic/2017b</code>, <code>gompi/2017a</code>, <code>gompi/2017b</code>, <code>gompic/2017b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>3.3.7</code></td>
<td></td>
<td><code>gimkl/2017a</code>, <code>gimpi/2018a</code>, <code>gmpich/2017.08</code>, <code>gompi/2018a</code>, <code>gompic/2018a</code>, <code>intel/2017b</code>, <code>intel/2018.00</code>, <code>intel/2018.01</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.3.7</code></td>
<td><code>-serial</code></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>3.3.8</code></td>
<td></td>
<td><code>gompi/2018.08</code>, <code>gompi/2018b</code>, <code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompic/2018b</code>, <code>gompic/2019a</code>, <code>gompic/2019b</code>, <code>gompic/2020a</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code>, <code>intel/2018b</code>, <code>intel/2019a</code>, <code>intel/2019b</code>, <code>intel/2020a</code>, <code>intel/2020b</code>, <code>iomkl/2018b</code>, <code>iomkl/2020b</code></td>
</tr>
<tr>
<td><code>3.3.8</code></td>
<td><code>-amd</code></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>3.3.8</code></td>
<td><code>-serial</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>3.3.9</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>3.3.10</code></td>
<td></td>
<td><code>GCC/11.3.0</code>, <code>GCC/12.2.0</code>, <code>NVHPC/22.7-CUDA-11.7.0</code>, <code>gompi/2021b</code>, <code>iimpi/2021b</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fftwmpi">FFTW.MPI<a class="headerlink" href="#fftwmpi" title="Permanent link">&para;</a></h3>
<p>FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data.</p>
<p><em>homepage</em>: <a href="https://www.fftw.org">https://www.fftw.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.10</code></td>
<td><code>gompi/2022.05</code>, <code>gompi/2022.10</code>, <code>gompi/2022a</code>, <code>gompi/2022b</code>, <code>nvompi/2022.07</code></td>
</tr>
</tbody>
</table>
<h3 id="fgbio">fgbio<a class="headerlink" href="#fgbio" title="Permanent link">&para;</a></h3>
<p>A set of tools to analyze genomic data with a focus on Next Generation Sequencing.</p>
<p><em>homepage</em>: <a href="https://fulcrumgenomics.github.io/fgbio">https://fulcrumgenomics.github.io/fgbio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fgsl">FGSL<a class="headerlink" href="#fgsl" title="Permanent link">&para;</a></h3>
<p>FGSL: A Fortran interface to the GNU Scientific Library</p>
<p><em>homepage</em>: <a href="https://www.lrz.de/services/software/mathematik/gsl/fortran/">https://www.lrz.de/services/software/mathematik/gsl/fortran/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="fhi-aims">FHI-aims<a class="headerlink" href="#fhi-aims" title="Permanent link">&para;</a></h3>
<p>FHI-aims is an efficient, accurate all-electron, full-potential electronic structure code package for computational molecular and materials science (non-periodic and periodic systems). The code supports DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is particularly efficient for molecular systems and nanostructures, while maintaining high numerical accuracy for all production tasks. Production calculations handle up to several thousand atoms and can efficiently use (ten) thousands of cores.  </p>
<p><em>homepage</em>: <a href="https://aimsclub.fhi-berlin.mpg.de/">https://aimsclub.fhi-berlin.mpg.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>200112_2</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fiat">FIAT<a class="headerlink" href="#fiat" title="Permanent link">&para;</a></h3>
<p>The FInite element Automatic Tabulator (FIAT) supports generation of arbitrary order instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating arbitrary order instances of Jacobi-type quadrature rules on the same element shapes.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/fenics-project/fiat">https://bitbucket.org/fenics-project/fiat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2018.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2019.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="figaro">FIGARO<a class="headerlink" href="#figaro" title="Permanent link">&para;</a></h3>
<p>FIGARO: An efficient and objective tool for optimizing microbiome rRNA gene trimming parameters.</p>
<p><em>homepage</em>: <a href="https://github.com/Zymo-Research/figaro">https://github.com/Zymo-Research/figaro</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="figuregen">FigureGen<a class="headerlink" href="#figuregen" title="Permanent link">&para;</a></h3>
<p>FigureGen is a Fortran program that creates images for ADCIRC files. It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output  files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors.  Using FigureGen, you can go directly from the ADCIRC input and output files to a  presentation-quality figure, for one or multiple time snaps.  </p>
<p><em>homepage</em>: <a href="https://ccht.ccee.ncsu.edu/category/software/figuregen/">https://ccht.ccee.ncsu.edu/category/software/figuregen/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>51-20190516</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="fiji">Fiji<a class="headerlink" href="#fiji" title="Permanent link">&para;</a></h3>
<p>Fiji is an image processing packagea 'batteries-included' distribution of  ImageJ, bundling a lot of plugins which facilitate scientific image analysis. This release is based on ImageJ-2.1.0 and Fiji-2.1.1</p>
<p><em>homepage</em>: <a href="https://fiji.sc/">https://fiji.sc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td><code>-Java-8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20170530</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20191119-2057</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20201104-1356</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="file">file<a class="headerlink" href="#file" title="Permanent link">&para;</a></h3>
<p>The file command is 'a file type guesser', that is, a command-line tool  that tells you in words what kind of data a file contains.</p>
<p><em>homepage</em>: <a href="https://www.darwinsys.com/file/">https://www.darwinsys.com/file/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.17</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.25</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.28</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.30</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.33</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>5.35</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.38</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.39</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.40</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.41</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.43</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="filevercmp">filevercmp<a class="headerlink" href="#filevercmp" title="Permanent link">&para;</a></h3>
<p>filevercmp function as in sort --version-sort.</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/filevercmp">https://github.com/ekg/filevercmp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20141119</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20191210</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="filtlong">Filtlong<a class="headerlink" href="#filtlong" title="Permanent link">&para;</a></h3>
<p>Filtlong is a tool for filtering long reads by quality. It can take a set  of long reads and produce a smaller, better subset. It uses both read length (longer is better)  and read identity (higher is better) when choosing which reads pass the filter</p>
<p><em>homepage</em>: <a href="https://github.com/rrwick/Filtlong">https://github.com/rrwick/Filtlong</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>GCC/10.2.0</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.2.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="find_circ">find_circ<a class="headerlink" href="#find_circ" title="Permanent link">&para;</a></h3>
<p>circRNA detection from RNA-seq reads</p>
<p><em>homepage</em>: <a href="https://github.com/marvin-jens/find_circ">https://github.com/marvin-jens/find_circ</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2-20170228</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="finder">finder<a class="headerlink" href="#finder" title="Permanent link">&para;</a></h3>
<p>finder is a gene annotator pipeline which automates the process of downloading short reads, aligning them and using the assembled transcripts to generate gene annotations.</p>
<p><em>homepage</em>: <a href="https://github.com/sagnikbanerjee15/Finder/">https://github.com/sagnikbanerjee15/Finder/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="findhap">findhap<a class="headerlink" href="#findhap" title="Permanent link">&para;</a></h3>
<p>Find haplotypes and impute genotypes using multiple chip sets and sequence data</p>
<p><em>homepage</em>: <a href="http://aipl.arsusda.gov/software/findhap/">http://aipl.arsusda.gov/software/findhap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="findutils">findutils<a class="headerlink" href="#findutils" title="Permanent link">&para;</a></h3>
<p>findutils: The GNU find, locate, updatedb, and xargs utilities</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/findutils/findutils.html">http://www.gnu.org/software/findutils/findutils.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="fineradstructure">fineRADstructure<a class="headerlink" href="#fineradstructure" title="Permanent link">&para;</a></h3>
<p>A package for population structure inference from RAD-seq data</p>
<p><em>homepage</em>: <a href="http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html">http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180709</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="finestructure">fineSTRUCTURE<a class="headerlink" href="#finestructure" title="Permanent link">&para;</a></h3>
<p>fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using  dense sequencing data.</p>
<p><em>homepage</em>: <a href="https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html">https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="fio">fio<a class="headerlink" href="#fio" title="Permanent link">&para;</a></h3>
<p>Flexible I/O tester</p>
<p><em>homepage</em>: <a href="https://github.com/axboe/fio">https://github.com/axboe/fio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.30</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.32</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fiona">Fiona<a class="headerlink" href="#fiona" title="Permanent link">&para;</a></h3>
<p>Fiona is designed to be simple and dependable. It focuses on reading and writing data in standard Python IO style and relies upon familiar Python types and protocols such as files, dictionaries, mappings, and iterators instead of classes specific to OGR. Fiona can read and write real-world data using multi-layered GIS formats and zipped virtual file systems and integrates readily with other Python GIS packages such as pyproj, Rtree, and Shapely.</p>
<p><em>homepage</em>: <a href="https://github.com/Toblerity/Fiona">https://github.com/Toblerity/Fiona</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.13</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.8.13.post1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.8.16</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.8.20</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.8.21</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="firefox">Firefox<a class="headerlink" href="#firefox" title="Permanent link">&para;</a></h3>
<p>Firefox is a free, open source Web browser for Windows, Linux and Mac OS X. It is based on the  Mozilla code base and offers customization options and features such as its capability to block pop-up windows,  tabbed browsing, privacy and security measures, smart searching, and RSS live bookmarks.</p>
<p><em>homepage</em>: <a href="https://www.mozilla.org/en-US/firefox/new/">https://www.mozilla.org/en-US/firefox/new/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>44.0.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="firestarter">FIRESTARTER<a class="headerlink" href="#firestarter" title="Permanent link">&para;</a></h3>
<p>FIRESTARTER: A Processor Stress Test Utility. FIRESTARTER maximizes the energy consumption of 64-Bit x86 processors by generating heavy load on the execution units as well as transferring data between the cores and multiple levels of the memory hierarchy.</p>
<p><em>homepage</em>: <a href="https://github.com/tud-zih-energy/FIRESTARTER/">https://github.com/tud-zih-energy/FIRESTARTER/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>gcccuda/2020a</code>, <code>gcccuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="fireworks">FireWorks<a class="headerlink" href="#fireworks" title="Permanent link">&para;</a></h3>
<p>FireWorks helps run calculation workflows, with a centralized workflow server  controlling many worker nodes.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/FireWorks">https://pypi.python.org/pypi/FireWorks</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="fit-sne">FIt-SNE<a class="headerlink" href="#fit-sne" title="Permanent link">&para;</a></h3>
<p>t-distributed stochastic neighbor embedding (t-SNE) is widely used for  visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large  datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and  hence allowing visualization of rare cell populations. Furthermore, we implement a  heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously  visualizing the expression patterns of thousands of genes.</p>
<p><em>homepage</em>: <a href="https://github.com/KlugerLab/FIt-SNE">https://github.com/KlugerLab/FIt-SNE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>gompi/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fix">FIX<a class="headerlink" href="#fix" title="Permanent link">&para;</a></h3>
<p>FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad components can be removed from the 4D FMRI data.</p>
<p><em>homepage</em>: <a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX">https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.06.12</code></td>
<td><code>-Octave-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="fixesproto">fixesproto<a class="headerlink" href="#fixesproto" title="Permanent link">&para;</a></h3>
<p>X.org FixesProto protocol headers.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="flac">FLAC<a class="headerlink" href="#flac" title="Permanent link">&para;</a></h3>
<p>FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning that audio is compressed in FLAC without any loss in quality.</p>
<p><em>homepage</em>: <a href="https://xiph.org/flac/">https://xiph.org/flac/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.3.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flair">FLAIR<a class="headerlink" href="#flair" title="Permanent link">&para;</a></h3>
<p>FLAIR (Full-Length Alternative Isoform analysis of RNA)  for the correction, isoform definition, and alternative splicing analysis of noisy reads.  FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.</p>
<p><em>homepage</em>: <a href="https://github.com/BrooksLabUCSC/flair">https://github.com/BrooksLabUCSC/flair</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.5.1-20200630</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="flair-nlp">flair-NLP<a class="headerlink" href="#flair-nlp" title="Permanent link">&para;</a></h3>
<p>A very simple framework for state-of-the-art NLP</p>
<p><em>homepage</em>: <a href="https://github.com/flairNLP/flair">https://github.com/flairNLP/flair</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.11.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="flann">FLANN<a class="headerlink" href="#flann" title="Permanent link">&para;</a></h3>
<p>FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces.</p>
<p><em>homepage</em>: <a href="https://github.com/mariusmuja/flann/">https://github.com/mariusmuja/flann/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="flash">FLASH<a class="headerlink" href="#flash" title="Permanent link">&para;</a></h3>
<p>FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. </p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/FLASH/">https://ccb.jhu.edu/software/FLASH/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.11</code></td>
<td><code>GCC/8.3.0</code>, <code>foss/2016a</code>, <code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.00</code></td>
<td><code>GCC/11.2.0</code>, <code>foss/2018b</code>, <code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="flask">Flask<a class="headerlink" href="#flask" title="Permanent link">&para;</a></h3>
<p>Flask is a lightweight WSGI web application framework. It is designed to make getting started quick and easy, with the ability to scale up to complex applications. This module includes the Flask extensions: Flask-Cors</p>
<p><em>homepage</em>: <a href="https://www.palletsprojects.com/p/flask/">https://www.palletsprojects.com/p/flask/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flatbuffers">flatbuffers<a class="headerlink" href="#flatbuffers" title="Permanent link">&para;</a></h3>
<p>FlatBuffers: Memory Efficient Serialization Library</p>
<p><em>homepage</em>: <a href="https://github.com/google/flatbuffers/">https://github.com/google/flatbuffers/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.0.7</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>23.1.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flatbuffers-python">flatbuffers-python<a class="headerlink" href="#flatbuffers-python" title="Permanent link">&para;</a></h3>
<p>Python Flatbuffers runtime library.</p>
<p><em>homepage</em>: <a href="https://github.com/google/flatbuffers/">https://github.com/google/flatbuffers/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fleur">FLEUR<a class="headerlink" href="#fleur" title="Permanent link">&para;</a></h3>
<p>FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,  based on density-functional theory.</p>
<p><em>homepage</em>: <a href="http://www.flapw.de/">http://www.flapw.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.26e</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="flex">flex<a class="headerlink" href="#flex" title="Permanent link">&para;</a></h3>
<p>Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text. </p>
<p><em>homepage</em>: <a href="http://flex.sourceforge.net/">http://flex.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.35</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.38</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>2.5.39</code></td>
<td><code>GCC/4.9.2</code>, <code>GCC/4.9.2-binutils-2.25</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-binutils-2.25</code>, <code>GCC/5.1.0-binutils-2.25</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>GCC/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/4.9.4</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/7.1.0</code>, <code>gimkl/2017a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.1.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/5.5.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/7.4.0</code>, <code>GCCcore/8.1.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/8.4.0</code>, <code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code>, <code>GCCcore/9.4.0</code>, <code>GCCcore/9.5.0</code>, <code>GCCcore/system</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="flexbar">Flexbar<a class="headerlink" href="#flexbar" title="Permanent link">&para;</a></h3>
<p>The program Flexbar preprocesses high-throughput sequencing data efficiently</p>
<p><em>homepage</em>: <a href="https://github.com/seqan/flexbar">https://github.com/seqan/flexbar</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="flexiblas">FlexiBLAS<a class="headerlink" href="#flexiblas" title="Permanent link">&para;</a></h3>
<p>FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it.</p>
<p><em>homepage</em>: <a href="https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release">https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCC/11.3.0</code>, <code>NVHPC/22.7-CUDA-11.7.0</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flexidot">FlexiDot<a class="headerlink" href="#flexidot" title="Permanent link">&para;</a></h3>
<p>Highly customizable, ambiguity-aware dotplots for visual sequence analyses </p>
<p><em>homepage</em>: <a href="https://github.com/molbio-dresden/flexidot">https://github.com/molbio-dresden/flexidot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.06</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="flink">Flink<a class="headerlink" href="#flink" title="Permanent link">&para;</a></h3>
<p>Apache Flink is a framework and distributed processing engine for stateful computations over unbounded and bounded data streams. Flink has been designed to run in all common cluster environments, perform computations at in-memory speed and at any scale.</p>
<p><em>homepage</em>: <a href="https://flink.apache.org/">https://flink.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.2</code></td>
<td><code>-bin-scala_2.11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="flint">FLINT<a class="headerlink" href="#flint" title="Permanent link">&para;</a></h3>
<p>FLINT (Fast Library for Number Theory) is a C library in support of computations  in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs,  factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides  various low-level routines for fast arithmetic. FLINT is extensively documented and tested.</p>
<p><em>homepage</em>: <a href="https://www.flintlib.org/">https://www.flintlib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.2</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.8.4</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flook">flook<a class="headerlink" href="#flook" title="Permanent link">&para;</a></h3>
<p>The fortran-Lua-hook library.</p>
<p><em>homepage</em>: <a href="https://github.com/ElectronicStructureLibrary/flook">https://github.com/ElectronicStructureLibrary/flook</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="flowfda">flowFDA<a class="headerlink" href="#flowfda" title="Permanent link">&para;</a></h3>
<p>R package for analysing flow cytometry experiments with model based clustering,  functional principal component analysis</p>
<p><em>homepage</em>: <a href="https://github.com/FMKerckhof/flowFDA">https://github.com/FMKerckhof/flowFDA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99-20220602</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fltk">FLTK<a class="headerlink" href="#fltk" title="Permanent link">&para;</a></h3>
<p>FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,  and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL  and its built-in GLUT emulation.</p>
<p><em>homepage</em>: <a href="https://www.fltk.org">https://www.fltk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.4</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>fosscuda/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>1.3.5</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.3.7</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.3.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fluent">FLUENT<a class="headerlink" href="#fluent" title="Permanent link">&para;</a></h3>
<p>ANSYS FLUENT software contains the broad physical modeling capabilities needed to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms, from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants.</p>
<p><em>homepage</em>: <a href="https://www.ansys.com/products/fluids/ansys-fluent">https://www.ansys.com/products/fluids/ansys-fluent</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>14.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>15.0.7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021R1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021R2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="flye">Flye<a class="headerlink" href="#flye" title="Permanent link">&para;</a></h3>
<p>Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio  and Oxford Nanopore Technologies.</p>
<p><em>homepage</em>: <a href="https://github.com/fenderglass/Flye">https://github.com/fenderglass/Flye</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.8.3</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>2.9</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fmilibrary">FMILibrary<a class="headerlink" href="#fmilibrary" title="Permanent link">&para;</a></h3>
<p>FMI library is intended as a foundation for applications interfacing  FMUs (Functional Mockup Units) that follow FMI Standard. This version of the library supports  FMI 1.0 and FMI2.0. See http://www.fmi-standard.org/</p>
<p><em>homepage</em>: <a href="https://jmodelica.org/">https://jmodelica.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fmm3d">FMM3D<a class="headerlink" href="#fmm3d" title="Permanent link">&para;</a></h3>
<p>Flatiron Institute Fast Multipole Libraries: a set of libraries to compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine.</p>
<p><em>homepage</em>: <a href="https://fmm3d.readthedocs.io">https://fmm3d.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20211018</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="fmpy">FMPy<a class="headerlink" href="#fmpy" title="Permanent link">&para;</a></h3>
<p>FMPy is a free Python library to simulate Functional Mock-up Units (FMUs).</p>
<p><em>homepage</em>: <a href="https://github.com/CATIA-Systems/FMPy">https://github.com/CATIA-Systems/FMPy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="fmriprep">FMRIprep<a class="headerlink" href="#fmriprep" title="Permanent link">&para;</a></h3>
<p>FMRIprep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is  designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan  acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive  error and output reporting.</p>
<p><em>homepage</em>: <a href="https://fmriprep.readthedocs.io">https://fmriprep.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.8</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fms">FMS<a class="headerlink" href="#fms" title="Permanent link">&para;</a></h3>
<p>The Flexible Modeling System (FMS) is a software framework for   supporting the efficient development, construction, execution, and scientific  interpretation of atmospheric, oceanic, and climate system models.</p>
<p><em>homepage</em>: <a href="https://github.com/NOAA-GFDL/FMS">https://github.com/NOAA-GFDL/FMS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2022.02</code></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="fmt">fmt<a class="headerlink" href="#fmt" title="Permanent link">&para;</a></h3>
<p>fmt (formerly cppformat) is an open-source formatting library.</p>
<p><em>homepage</em>: <a href="http://fmtlib.net/">http://fmtlib.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.0.3</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.1.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fobis">FoBiS<a class="headerlink" href="#fobis" title="Permanent link">&para;</a></h3>
<p>A Fortran Building System for automatic building modern Fortran projects</p>
<p><em>homepage</em>: <a href="https://github.com/szaghi/FoBiS">https://github.com/szaghi/FoBiS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.5</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="foldx">FoldX<a class="headerlink" href="#foldx" title="Permanent link">&para;</a></h3>
<p>FoldX is used to provide a fast and quantitative estimation of the importance of the interactions  contributing to the stability of proteins and protein complexes.</p>
<p><em>homepage</em>: <a href="http://foldx.crg.es/">http://foldx.crg.es/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0-beta5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0-beta6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0-beta6.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fontconfig">fontconfig<a class="headerlink" href="#fontconfig" title="Permanent link">&para;</a></h3>
<p>Fontconfig is a library designed to provide system-wide font configuration,  customization and application access. </p>
<p><em>homepage</em>: <a href="https://www.freedesktop.org/wiki/Software/fontconfig/">https://www.freedesktop.org/wiki/Software/fontconfig/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.94</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.11.95</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.12.1</code></td>
<td></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.12.1</code></td>
<td><code>-libpng-1.6.29</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>2.12.4</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.12.6</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.13.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.13.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.13.92</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.13.93</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.13.94</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.14.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.14.1</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fontsproto">fontsproto<a class="headerlink" href="#fontsproto" title="Permanent link">&para;</a></h3>
<p>X11 font extension wire protocol</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ford">FORD<a class="headerlink" href="#ford" title="Permanent link">&para;</a></h3>
<p>FORD is an automatic documentation generator for modern Fortran programs</p>
<p><em>homepage</em>: <a href="https://github.com/Fortran-FOSS-Programmers/ford/wiki">https://github.com/Fortran-FOSS-Programmers/ford/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>6.1.6</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>6.1.15</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="foss">foss<a class="headerlink" href="#foss" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: <a href="https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain">https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.04</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.06</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.08</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fosscuda">fosscuda<a class="headerlink" href="#fosscuda" title="Permanent link">&para;</a></h3>
<p>GCC based compiler toolchain <strong>with CUDA support</strong>, and including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fox">FoX<a class="headerlink" href="#fox" title="Permanent link">&para;</a></h3>
<p>FoX is an XML library written in Fortran 95. It allows software developers to read, write and modify XML documents from Fortran applications without the complications of dealing with multi-language development.</p>
<p><em>homepage</em>: <a href="http://homepages.see.leeds.ac.uk/~earawa/FoX/">http://homepages.see.leeds.ac.uk/~earawa/FoX/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.2</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/9.3.0</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fox-toolkit">FOX-Toolkit<a class="headerlink" href="#fox-toolkit" title="Permanent link">&para;</a></h3>
<p>FOX is a C++ based Toolkit for developing Graphical User Interfaces easily and effectively. It offers a wide, and growing, collection of Controls, and provides state of the art facilities such as drag and drop, selection, as well as OpenGL widgets for 3D graphical manipulation. FOX also implements icons, images, and user-convenience features such as status line help, and tooltips.</p>
<p><em>homepage</em>: <a href="https://www.fox-toolkit.org/">https://www.fox-toolkit.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.57</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fpm">FPM<a class="headerlink" href="#fpm" title="Permanent link">&para;</a></h3>
<p>Effing package management! Build packages for multiple platforms (deb, rpm, etc) with great ease  and sanity.</p>
<p><em>homepage</em>: <a href="https://github.com/jordansissel/fpm">https://github.com/jordansissel/fpm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>-Ruby-2.1.6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fpocket">fpocket<a class="headerlink" href="#fpocket" title="Permanent link">&para;</a></h3>
<p>The fpocket suite of programs is a very fast open source protein pocket detection algorithm based on Voronoi tessellation. The platform is suited for the scientific community willing to develop new scoring functions and  extract pocket descriptors on a large scale level.</p>
<p><em>homepage</em>: <a href="https://github.com/Discngine/fpocket">https://github.com/Discngine/fpocket</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.4.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="fqtrim">fqtrim<a class="headerlink" href="#fqtrim" title="Permanent link">&para;</a></h3>
<p>fqtrim is a versatile stand-alone utility that can be used to trim adapters, poly-A tails,  terminal unknown bases (Ns) and low quality 3' regions in reads from high-throughput next-generation sequencing  machines.</p>
<p><em>homepage</em>: <a href="http://ccb.jhu.edu/software/fqtrim/">http://ccb.jhu.edu/software/fqtrim/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.4</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.5</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="fqzcomp">fqzcomp<a class="headerlink" href="#fqzcomp" title="Permanent link">&para;</a></h3>
<p>Fqzcomp is a basic fastq compressor, designed primarily for high performance.  Despite that it is comparable to bzip2 for compression levels.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/fqzcomp/">http://sourceforge.net/projects/fqzcomp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="fraggenescan">FragGeneScan<a class="headerlink" href="#fraggenescan" title="Permanent link">&para;</a></h3>
<p>FragGeneScan is an application for finding (fragmented) genes in short reads.</p>
<p><em>homepage</em>: <a href="http://omics.informatics.indiana.edu/FragGeneScan/">http://omics.informatics.indiana.edu/FragGeneScan/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.31</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="franz">FRANz<a class="headerlink" href="#franz" title="Permanent link">&para;</a></h3>
<p>A fast and flexible parentage inference program for natural populations.</p>
<p><em>homepage</em>: <a href="https://www.bioinf.uni-leipzig.de/Software/FRANz">https://www.bioinf.uni-leipzig.de/Software/FRANz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="freebarcodes">FreeBarcodes<a class="headerlink" href="#freebarcodes" title="Permanent link">&para;</a></h3>
<p>A package for the generation and decoding of FREE divergence error-correcting DNA barcodes</p>
<p><em>homepage</em>: <a href="https://github.com/hawkjo/freebarcodes">https://github.com/hawkjo/freebarcodes</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.a5</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="freebayes">freebayes<a class="headerlink" href="#freebayes" title="Permanent link">&para;</a></h3>
<p>Bayesian haplotype-based genetic polymorphism discovery and genotyping.</p>
<p><em>homepage</em>: <a href="https://github.com/freebayes/freebayes">https://github.com/freebayes/freebayes</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.5</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.3.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="freefem">FreeFEM<a class="headerlink" href="#freefem" title="Permanent link">&para;</a></h3>
<p>FreeFEM offers a fast interpolation algorithm and a language for the manipulation  of data on multiple meshes.</p>
<p><em>homepage</em>: <a href="https://freefem.org">https://freefem.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="freefem_1">FreeFem++<a class="headerlink" href="#freefem_1" title="Permanent link">&para;</a></h3>
<p>FreeFem++ is a partial differential equation solver. It has its own language. freefem scripts can solve multiphysics non linear systems in 2D and 3D. </p>
<p><em>homepage</em>: <a href="http://www.freefem.org/">http://www.freefem.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.58</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.60</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.61-1</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="freeglut">freeglut<a class="headerlink" href="#freeglut" title="Permanent link">&para;</a></h3>
<p>freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.</p>
<p><em>homepage</em>: <a href="http://freeglut.sourceforge.net/">http://freeglut.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Mesa-11.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freeimage">FreeImage<a class="headerlink" href="#freeimage" title="Permanent link">&para;</a></h3>
<p>FreeImage is an Open Source library project for developers who would like to support popular graphics image formats like PNG, BMP, JPEG, TIFF and others as needed by today's multimedia applications. FreeImage is easy to use, fast, multithreading safe.</p>
<p><em>homepage</em>: <a href="http://freeimage.sourceforge.net">http://freeimage.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.18.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freesasa">FreeSASA<a class="headerlink" href="#freesasa" title="Permanent link">&para;</a></h3>
<p>FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA).</p>
<p><em>homepage</em>: <a href="https://freesasa.github.io">https://freesasa.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freesurfer">FreeSurfer<a class="headerlink" href="#freesurfer" title="Permanent link">&para;</a></h3>
<p>FreeSurfer is a set of tools for analysis and visualization  of structural and functional brain imaging data.  FreeSurfer contains a fully  automatic structural imaging stream for processing cross sectional and  longitudinal data.</p>
<p><em>homepage</em>: <a href="https://surfer.nmr.mgh.harvard.edu/">https://surfer.nmr.mgh.harvard.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.3.0</code></td>
<td><code>-centos4_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-centos6_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>-centos6_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>-centos6_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.1</code></td>
<td><code>-centos6_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.1</code></td>
<td><code>-centos7_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.1</code></td>
<td><code>-centos8_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>-centos7_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>-centos8_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>-ubuntu18_amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.3.2</code></td>
<td><code>-centos7_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.3.2</code></td>
<td><code>-centos8_x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="freetds">FreeTDS<a class="headerlink" href="#freetds" title="Permanent link">&para;</a></h3>
<p>FreeTDS is a set of libraries for Unix and Linux that allows your programs to  natively talk to Microsoft SQL Server and Sybase databases.</p>
<p><em>homepage</em>: <a href="https://www.freetds.org/">https://www.freetds.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freetype">freetype<a class="headerlink" href="#freetype" title="Permanent link">&para;</a></h3>
<p>FreeType 2 is a software font engine that is designed to be small, efficient,  highly customizable, and portable while capable of producing high-quality  output (glyph images). It can be used in graphics libraries, display servers,  font conversion tools, text image generation tools, and many other products  as well. </p>
<p><em>homepage</em>: <a href="https://freetype.org">https://freetype.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.6.5</code></td>
<td></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-libpng-1.6.29</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.9</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.10.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.10.4</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.12.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freetype-py">freetype-py<a class="headerlink" href="#freetype-py" title="Permanent link">&para;</a></h3>
<p>Python binding for the freetype library</p>
<p><em>homepage</em>: <a href="https://github.com/rougier/freetype-py">https://github.com/rougier/freetype-py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freexl">FreeXL<a class="headerlink" href="#freexl" title="Permanent link">&para;</a></h3>
<p>FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. </p>
<p><em>homepage</em>: <a href="https://www.gaia-gis.it/fossil/freexl/index">https://www.gaia-gis.it/fossil/freexl/index</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="freud-analysis">freud-analysis<a class="headerlink" href="#freud-analysis" title="Permanent link">&para;</a></h3>
<p>The freud Python library provides a simple, flexible, powerful set of tools for analyzing trajectories obtained from molecular dynamics or Monte Carlo simulations. High performance, parallelized C++ is used to compute standard tools such as radial distribution functions, correlation functions, order parameters, and clusters, as well as original analysis methods including potentials of mean force and torque (PMFTs) and local environment matching. The freud library supports many input formats and outputs NumPy arrays, enabling integration with the scientific Python ecosystem for many typical materials science workflows.</p>
<p><em>homepage</em>: <a href="https://github.com/glotzerlab/freud">https://github.com/glotzerlab/freud</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="fribidi">FriBidi<a class="headerlink" href="#fribidi" title="Permanent link">&para;</a></h3>
<p>The Free Implementation of the Unicode Bidirectional Algorithm. </p>
<p><em>homepage</em>: <a href="https://github.com/fribidi/fribidi">https://github.com/fribidi/fribidi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.0.9</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.0.10</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.0.12</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fruit">FRUIT<a class="headerlink" href="#fruit" title="Permanent link">&para;</a></h3>
<p>FORTRAN Unit Test Framework (FRUIT)</p>
<p><em>homepage</em>: <a href="https://fortranxunit.sourceforge.io">https://fortranxunit.sourceforge.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.3</code></td>
<td><code>-Ruby-2.5.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fruit_processor">FRUIT_processor<a class="headerlink" href="#fruit_processor" title="Permanent link">&para;</a></h3>
<p>FORTRAN Unit Test Framework (FRUIT)</p>
<p><em>homepage</em>: <a href="https://fortranxunit.sourceforge.io">https://fortranxunit.sourceforge.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.3</code></td>
<td><code>-Ruby-2.5.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fsl">FSL<a class="headerlink" href="#fsl" title="Permanent link">&para;</a></h3>
<p>FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.</p>
<p><em>homepage</em>: <a href="https://www.fmrib.ox.ac.uk/fsl/">https://www.fmrib.ox.ac.uk/fsl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.9</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.0.9</code></td>
<td><code>-Mesa-11.2.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.0.9</code></td>
<td><code>-centos6_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0.10</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0.11</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.0.11</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>6.0.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code></td>
</tr>
<tr>
<td><code>6.0.2</code></td>
<td><code>-Python-2.7.15-CUDA-9.2.88</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.0.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>6.0.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>6.0.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>6.0.5.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="fsleyes">FSLeyes<a class="headerlink" href="#fsleyes" title="Permanent link">&para;</a></h3>
<p>FSLeyes is the FSL image viewer.</p>
<p><em>homepage</em>: <a href="https://git.fmrib.ox.ac.uk/fsl/fsleyes/fsleyes">https://git.fmrib.ox.ac.uk/fsl/fsleyes/fsleyes</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="fsom">fsom<a class="headerlink" href="#fsom" title="Permanent link">&para;</a></h3>
<p>A tiny C library for managing SOM (Self-Organizing Maps) neural networks.</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/fsom">https://github.com/ekg/fsom</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20141119</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fson">FSON<a class="headerlink" href="#fson" title="Permanent link">&para;</a></h3>
<p>Fortran 95 JSON Parser</p>
<p><em>homepage</em>: <a href="https://github.com/josephalevin/fson">https://github.com/josephalevin/fson</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ftgl">FTGL<a class="headerlink" href="#ftgl" title="Permanent link">&para;</a></h3>
<p>FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications. </p>
<p><em>homepage</em>: <a href="https://github.com/frankheckenbach/ftgl">https://github.com/frankheckenbach/ftgl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3-rc5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2018b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="fujitsu">Fujitsu<a class="headerlink" href="#fujitsu" title="Permanent link">&para;</a></h3>
<p>Toolchain using Fujitsu compilers and libraries.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>21.05</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="fullrmc">fullrmc<a class="headerlink" href="#fullrmc" title="Permanent link">&para;</a></h3>
<p>Reverse Monte Carlo (RMC) is probably best known for its applications  in condensed matter physics and solid state chemistry. fullrmc which stands for  FUndamental Library Language for Reverse Monte Carlo is different than traditional  RMC but a stochastic modelling method to solve an inverse problem whereby  an atomic/molecular model is adjusted until its atoms position havei  the greatest consistency with a set of experimental data.</p>
<p><em>homepage</em>: <a href="https://bachiraoun.github.io/fullrmc">https://bachiraoun.github.io/fullrmc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fumi_tools">fumi_tools<a class="headerlink" href="#fumi_tools" title="Permanent link">&para;</a></h3>
<p>This tool is intended to deduplicate UMIs from single-end and paired-end  sequencing data. Reads are considered identical when their UMIs have the same sequence,  they have the same length and map at the same position.</p>
<p><em>homepage</em>: <a href="https://ccb-gitlab.cs.uni-saarland.de/tobias/fumi_tools">https://ccb-gitlab.cs.uni-saarland.de/tobias/fumi_tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.2</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>0.18.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="funwave-tvd">FUNWAVE-TVD<a class="headerlink" href="#funwave-tvd" title="Permanent link">&para;</a></h3>
<p>FUNWAVETVD is the TVD version of the fully nonlinear Boussinesq wave model (FUNWAVE)  initially developed by Kirby et al.</p>
<p><em>homepage</em>: <a href="https://fengyanshi.github.io">https://fengyanshi.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1-20170525</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.1-20170525</code></td>
<td><code>-no-storm</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="fuse">FUSE<a class="headerlink" href="#fuse" title="Permanent link">&para;</a></h3>
<p>The reference implementation of the Linux FUSE (Filesystem in Userspace) interface</p>
<p><em>homepage</em>: <a href="https://github.com/libfuse/libfuse">https://github.com/libfuse/libfuse</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.6</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="fuseq">FuSeq<a class="headerlink" href="#fuseq" title="Permanent link">&para;</a></h3>
<p>FuSeq is a novel method to discover fusion genes from paired-end RNA sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/nghiavtr/FuSeq">https://github.com/nghiavtr/FuSeq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="fusioncatcher">FusionCatcher<a class="headerlink" href="#fusioncatcher" title="Permanent link">&para;</a></h3>
<p>FusionCatcher searches for novel/known somatic fusion genes, translocations, and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples.</p>
<p><em>homepage</em>: <a href="https://github.com/ndaniel/fusioncatcher">https://github.com/ndaniel/fusioncatcher</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.30</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="futhark">futhark<a class="headerlink" href="#futhark" title="Permanent link">&para;</a></h3>
<p>Futhark is a small programming language designed to be compiled to efficient parallel code. It     is a statically typed, data-parallel, and purely functional array language in the ML family, and comes     with a heavily optimising ahead-of-time compiler that presently generates GPU code via CUDA and OpenCL,     although the language itself is hardware-agnostic and can also run on multicore CPUs</p>
<p><em>homepage</em>: <a href="https://futhark-lang.org/">https://futhark-lang.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.19.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="futile">futile<a class="headerlink" href="#futile" title="Permanent link">&para;</a></h3>
<p>The FUTILE project (Fortran Utilities for the Treatment of Innermost Level of Executables) is a set of modules   and wrapper that encapsulate the most common low-level operations of a Fortran code.  </p>
<p><em>homepage</em>: <a href="https://launchpad.net/futile">https://launchpad.net/futile</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.3</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="future">future<a class="headerlink" href="#future" title="Permanent link">&para;</a></h3>
<p>python-future is the missing compatibility layer between Python 2 and Python 3.</p>
<p><em>homepage</em>: <a href="https://python-future.org/">https://python-future.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="fxtract">fxtract<a class="headerlink" href="#fxtract" title="Permanent link">&para;</a></h3>
<p>Extract sequences from a fastx (fasta or fastq) file given a subsequence.</p>
<p><em>homepage</em>: <a href="https://github.com/ctSkennerton/fxtract">https://github.com/ctSkennerton/fxtract</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="g">G<a class="headerlink" href="#g" title="Permanent link">&para;</a></h2>
<p><a href="#g-phocs">G-PhoCS</a> - <a href="#g2clib">g2clib</a> - <a href="#g2lib">g2lib</a> - <a href="#g2log">g2log</a> - <a href="#gaia">Gaia</a> - <a href="#gamess-us">GAMESS-US</a> - <a href="#gap">gap</a> - <a href="#gapcloser">GapCloser</a> - <a href="#gapfiller">GapFiller</a> - <a href="#gappa">gappa</a> - <a href="#garli">GARLI</a> - <a href="#garnett">garnett</a> - <a href="#gat">GAT</a> - <a href="#gate">GATE</a> - <a href="#gatk">GATK</a> - <a href="#gaussian">Gaussian</a> - <a href="#gawk">gawk</a> - <a href="#gbasis">gbasis</a> - <a href="#gblocks">Gblocks</a> - <a href="#gbprocess">GBprocesS</a> - <a href="#gbs2ploidy">gbs2ploidy</a> - <a href="#gc">gc</a> - <a href="#gc3pie">GC3Pie</a> - <a href="#gcc">GCC</a> - <a href="#gcccore">GCCcore</a> - <a href="#gcccuda">gcccuda</a> - <a href="#gcloud">gcloud</a> - <a href="#gconf">GConf</a> - <a href="#gcta">GCTA</a> - <a href="#gctf">Gctf</a> - <a href="#gd">GD</a> - <a href="#gdal">GDAL</a> - <a href="#gdb">GDB</a> - <a href="#gdbgui">gdbgui</a> - <a href="#gdbm">gdbm</a> - <a href="#gdc-client">gdc-client</a> - <a href="#gdchart">GDCHART</a> - <a href="#gdcm">GDCM</a> - <a href="#gdgraph">GDGraph</a> - <a href="#gdist">gdist</a> - <a href="#gdk-pixbuf">Gdk-Pixbuf</a> - <a href="#gdrcopy">GDRCopy</a> - <a href="#geant4">Geant4</a> - <a href="#geant4-data">Geant4-data</a> - <a href="#gearshifft">gearshifft</a> - <a href="#gegl">GEGL</a> - <a href="#gem-library">GEM-library</a> - <a href="#gemma">GEMMA</a> - <a href="#gemmi">gemmi</a> - <a href="#gencore_variant_detection">gencore_variant_detection</a> - <a href="#genemark-et">GeneMark-ET</a> - <a href="#gengetopt">gengetopt</a> - <a href="#genome_profiler">Genome_Profiler</a> - <a href="#genomemapper">GenomeMapper</a> - <a href="#genomepy">genomepy</a> - <a href="#genometester4">GenomeTester4</a> - <a href="#genomethreader">GenomeThreader</a> - <a href="#genometools">GenomeTools</a> - <a href="#genomeworks">GenomeWorks</a> - <a href="#genotypeharmonizer">GenotypeHarmonizer</a> - <a href="#genozip">genozip</a> - <a href="#gensim">gensim</a> - <a href="#geocube">geocube</a> - <a href="#geopandas">geopandas</a> - <a href="#geopy">geopy</a> - <a href="#georges">georges</a> - <a href="#geos">GEOS</a> - <a href="#gerris">Gerris</a> - <a href="#getorb">GETORB</a> - <a href="#getorganelle">GetOrganelle</a> - <a href="#gettext">gettext</a> - <a href="#gexiv2">gexiv2</a> - <a href="#gfbf">gfbf</a> - <a href="#gffcompare">GffCompare</a> - <a href="#gffread">gffread</a> - <a href="#gflags">gflags</a> - <a href="#gfold">GFOLD</a> - <a href="#gh">gh</a> - <a href="#ghc">GHC</a> - <a href="#ghostscript">Ghostscript</a> - <a href="#gibbs2">Gibbs2</a> - <a href="#giflib">giflib</a> - <a href="#gifsicle">gifsicle</a> - <a href="#gimic">GIMIC</a> - <a href="#gimkl">gimkl</a> - <a href="#gimp">GIMP</a> - <a href="#gimpi">gimpi</a> - <a href="#gimpic">gimpic</a> - <a href="#gimps">GIMPS</a> - <a href="#giolf">giolf</a> - <a href="#giolfc">giolfc</a> - <a href="#giotto-suite">Giotto-Suite</a> - <a href="#git">git</a> - <a href="#git-extras">git-extras</a> - <a href="#git-lfs">git-lfs</a> - <a href="#gitpython">GitPython</a> - <a href="#givaro">Givaro</a> - <a href="#giza">Giza</a> - <a href="#gkeyll">GKeyll</a> - <a href="#gl2ps">GL2PS</a> - <a href="#glade">Glade</a> - <a href="#glew">glew</a> - <a href="#glfw">GLFW</a> - <a href="#glib">GLib</a> - <a href="#glib-networking">glib-networking</a> - <a href="#glibc">glibc</a> - <a href="#glibmm">GLibmm</a> - <a href="#glimmer">GLIMMER</a> - <a href="#glimmerhmm">GlimmerHMM</a> - <a href="#glm">GLM</a> - <a href="#glm-aed">GLM-AED</a> - <a href="#globalarrays">GlobalArrays</a> - <a href="#globus-cli">Globus-CLI</a> - <a href="#globusconnectpersonal">GlobusConnectPersonal</a> - <a href="#glog">glog</a> - <a href="#glpk">GLPK</a> - <a href="#glproto">glproto</a> - <a href="#glucose">Glucose</a> - <a href="#gmap-gsnap">GMAP-GSNAP</a> - <a href="#gmp">GMP</a> - <a href="#gmpich">gmpich</a> - <a href="#gmpolf">gmpolf</a> - <a href="#gmpy2">gmpy2</a> - <a href="#gmsh">gmsh</a> - <a href="#gmt">GMT</a> - <a href="#gmvapich2">gmvapich2</a> - <a href="#gmvolf">gmvolf</a> - <a href="#gnu">GNU</a> - <a href="#gnuplot">gnuplot</a> - <a href="#gnutls">GnuTLS</a> - <a href="#go">Go</a> - <a href="#goalign">goalign</a> - <a href="#goatools">GOATOOLS</a> - <a href="#gobff">gobff</a> - <a href="#gobject-introspection">GObject-Introspection</a> - <a href="#goblf">goblf</a> - <a href="#godon">Godon</a> - <a href="#gofasta">gofasta</a> - <a href="#golf">golf</a> - <a href="#gomkl">gomkl</a> - <a href="#gompi">gompi</a> - <a href="#gompic">gompic</a> - <a href="#google-java-format">google-java-format</a> - <a href="#googletest">googletest</a> - <a href="#gotree">gotree</a> - <a href="#gp2c">GP2C</a> - <a href="#gpaw">GPAW</a> - <a href="#gpaw-setups">GPAW-setups</a> - <a href="#gperf">gperf</a> - <a href="#gperftools">gperftools</a> - <a href="#gpustat">gpustat</a> - <a href="#gpy">GPy</a> - <a href="#gpyopt">GPyOpt</a> - <a href="#gpytorch">GPyTorch</a> - <a href="#grace">Grace</a> - <a href="#gradle">Gradle</a> - <a href="#gradunwarp">gradunwarp</a> - <a href="#graph-tool">graph-tool</a> - <a href="#graphdb">GraphDB</a> - <a href="#graphene">Graphene</a> - <a href="#graphicsmagick">GraphicsMagick</a> - <a href="#graphite2">graphite2</a> - <a href="#graphlan">GraPhlAn</a> - <a href="#graphmap">GraphMap</a> - <a href="#graphmap2">GraphMap2</a> - <a href="#graphviz">Graphviz</a> - <a href="#graphviz-python">graphviz-python</a> - <a href="#grasp">GRASP</a> - <a href="#grass">GRASS</a> - <a href="#greenlet">Greenlet</a> - <a href="#grep">Grep</a> - <a href="#gretl">gretl</a> - <a href="#grib_api">grib_api</a> - <a href="#grid">grid</a> - <a href="#grit">GRIT</a> - <a href="#grnboost">GRNBoost</a> - <a href="#groff">groff</a> - <a href="#groimp">GroIMP</a> - <a href="#gromacs">GROMACS</a> - <a href="#gromacswrapper">GromacsWrapper</a> - <a href="#groovy">Groovy</a> - <a href="#grpc">gRPC</a> - <a href="#gsea">GSEA</a> - <a href="#gsettings-desktop-schemas">gsettings-desktop-schemas</a> - <a href="#gsl">GSL</a> - <a href="#gsoap">gSOAP</a> - <a href="#gsport">gsport</a> - <a href="#gst-plugins-bad">GST-plugins-bad</a> - <a href="#gst-plugins-base">GST-plugins-base</a> - <a href="#gstreamer">GStreamer</a> - <a href="#gsutil">gsutil</a> - <a href="#gsw">gsw</a> - <a href="#gtdb-tk">GTDB-Tk</a> - <a href="#gtk+">GTK+</a> - <a href="#gtk2">GTK2</a> - <a href="#gtk3">GTK3</a> - <a href="#gtk4">GTK4</a> - <a href="#gtksourceview">GtkSourceView</a> - <a href="#gtool">GTOOL</a> - <a href="#gts">GTS</a> - <a href="#gubbins">gubbins</a> - <a href="#guenomu">guenomu</a> - <a href="#guile">Guile</a> - <a href="#gulp">GULP</a> - <a href="#gurobi">Gurobi</a> - <a href="#gushr">GUSHR</a> - <a href="#gzip">gzip</a></p>
<h3 id="g-phocs">G-PhoCS<a class="headerlink" href="#g-phocs" title="Permanent link">&para;</a></h3>
<p>G-PhoCS is a software package for inferring ancestral population sizes, population divergence times, and migration rates from individual genome sequences. G-PhoCS accepts as input a set of multiple sequence alignments from separate neutrally evolving loci along the genome. Parameter inference is done in a Bayesian manner, using a Markov Chain Monte Carlo (MCMC) to jointly sample model parameters and genealogies at the input loci. </p>
<p><em>homepage</em>: <a href="http://compgen.cshl.edu/GPhoCS/">http://compgen.cshl.edu/GPhoCS/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="g2clib">g2clib<a class="headerlink" href="#g2clib" title="Permanent link">&para;</a></h3>
<p>Library contains GRIB2 encoder/decoder ('C' version).</p>
<p><em>homepage</em>: <a href="https://github.com/NOAA-EMC/NCEPLIBS-g2c">https://github.com/NOAA-EMC/NCEPLIBS-g2c</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="g2lib">g2lib<a class="headerlink" href="#g2lib" title="Permanent link">&para;</a></h3>
<p>Library contains GRIB2 encoder/decoder and search/indexing routines.</p>
<p><em>homepage</em>: <a href="https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/">https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="g2log">g2log<a class="headerlink" href="#g2log" title="Permanent link">&para;</a></h3>
<p>g2log, efficient asynchronous logger using C++11</p>
<p><em>homepage</em>: <a href="https://sites.google.com/site/kjellhedstrom2//g2log-efficient-background-io-processign-with-c11">https://sites.google.com/site/kjellhedstrom2//g2log-efficient-background-io-processign-with-c11</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/8.3.0</code>, <code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="gaia">Gaia<a class="headerlink" href="#gaia" title="Permanent link">&para;</a></h3>
<p>Gaia is a C++ library with python bindings which implements similarity measures and classications  on the results of audio analysis, and generates classication models that Essentia can use to compute high-level  description of music.</p>
<p><em>homepage</em>: <a href="https://github.com/MTG/gaia">https://github.com/MTG/gaia</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gamess-us">GAMESS-US<a class="headerlink" href="#gamess-us" title="Permanent link">&para;</a></h3>
<p>The General Atomic and Molecular Electronic Structure System (GAMESS)  is a general ab initio quantum chemistry package. </p>
<p><em>homepage</em>: <a href="http://www.msg.chem.iastate.edu/gamess/index.html">http://www.msg.chem.iastate.edu/gamess/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20141205-R1</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20170420-R1</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20170420-R1</code></td>
<td><code>-sockets</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20180214-R1</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>20180214-R1</code></td>
<td><code>-sockets</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="gap">gap<a class="headerlink" href="#gap" title="Permanent link">&para;</a></h3>
<p>GAP is a system for computational discrete algebra, with particular emphasis on Computational Group Theory.</p>
<p><em>homepage</em>: <a href="https://www.gap-system.org">https://www.gap-system.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9.3</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.11.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="gapcloser">GapCloser<a class="headerlink" href="#gapcloser" title="Permanent link">&para;</a></h3>
<p>GapCloser is designed to close the gaps emerging during the scaffolding process  by SOAPdenovo or other assembler, using the abundant pair relationships of short reads.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/soapdenovo2/files/GapCloser/">https://sourceforge.net/projects/soapdenovo2/files/GapCloser/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12-r6</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="gapfiller">GapFiller<a class="headerlink" href="#gapfiller" title="Permanent link">&para;</a></h3>
<p>GapFiller is a seed-and-extend local assembler to fill the gap within paired reads.  It can be used for both DNA and RNA and it has been tested on Illumina data.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/gapfiller">https://sourceforge.net/projects/gapfiller</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="gappa">gappa<a class="headerlink" href="#gappa" title="Permanent link">&para;</a></h3>
<p>gappa is a collection of commands for working with phylogenetic data. Its main focus are evolutionary placements of short environmental sequences on a reference phylogenetic tree. Such data is typically produced by tools like EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files. </p>
<p><em>homepage</em>: <a href="https://github.com/lczech/gappa">https://github.com/lczech/gappa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="garli">GARLI<a class="headerlink" href="#garli" title="Permanent link">&para;</a></h3>
<p>GARLI, Genetic Algorithm for Rapid Likelihood Inference is a   program for inferring phylogenetic trees. Using an approach similar to a   classical genetic algorithm, it rapidly searches the space of evolutionary   trees and model parameters to find the solution maximizing the likelihood   score. It implements nucleotide, amino acid and codon-based models of sequence   evolution, and runs on all platforms.</p>
<p><em>homepage</em>: <a href="https://code.google.com/archive/p/garli/">https://code.google.com/archive/p/garli/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.01</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="garnett">garnett<a class="headerlink" href="#garnett" title="Permanent link">&para;</a></h3>
<p>Garnett is a software package that faciliates automated cell type classification from single-cell expression data.</p>
<p><em>homepage</em>: <a href="https://cole-trapnell-lab.github.io/garnett">https://cole-trapnell-lab.github.io/garnett</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.20</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="gat">GAT<a class="headerlink" href="#gat" title="Permanent link">&para;</a></h3>
<p>The Genomic Association Tester (GAT) is a tool for computing the significance of overlap between   multiple sets of genomic intervals. GAT estimates significance based on simulation.</p>
<p><em>homepage</em>: <a href="https://gat.readthedocs.io/">https://gat.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="gate">GATE<a class="headerlink" href="#gate" title="Permanent link">&para;</a></h3>
<p>GATE is an advanced opensource software developed by the international OpenGATE collaboration and  dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography</p>
<p><em>homepage</em>: <a href="http://www.opengatecollaboration.org/">http://www.opengatecollaboration.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td><code>-Python-2.7.14-Geant4-10.04</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.1.p01</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>9.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>9.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gatk">GATK<a class="headerlink" href="#gatk" title="Permanent link">&para;</a></h3>
<p>The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute  to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on  data quality assurance. Its robust architecture, powerful processing engine and  high-performance computing features make it capable of taking on projects of any size.</p>
<p><em>homepage</em>: <a href="https://www.broadinstitute.org/gatk/">https://www.broadinstitute.org/gatk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5083</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5-2</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6-5</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.7-4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.7-4</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8-1</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0-0</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3-0</code></td>
<td><code>-Java-1.7.0_21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3-0</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3-0</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.8-0</code></td>
<td><code>-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.1.2</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.4.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0.4.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0.5.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.0.7.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0.7.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0.8.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.8.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.10.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.12.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.1.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.1.2.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.1.3.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.1.4.1</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.1.4.1</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.1.5.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.1.5.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.1.8.1</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.2.0.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.2.0.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.2.3.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2.5.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2.6.1</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.3.0.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gaussian">Gaussian<a class="headerlink" href="#gaussian" title="Permanent link">&para;</a></h3>
<p>Gaussian provides state-of-the-art capabilities for electronic structure modeling. Gaussian 09 is licensed for a wide variety of computer systems. All versions of Gaussian 09 contain every scientific/modeling feature, and none imposes any artificial limitations on calculations other than your computing resources and patience.  This is the official gaussian AVX2 build. </p>
<p><em>homepage</em>: <a href="https://www.gaussian.com/">https://www.gaussian.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>09.e.01</code></td>
<td><code>-AVX</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.A.03</code></td>
<td><code>-AVX2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gawk">gawk<a class="headerlink" href="#gawk" title="Permanent link">&para;</a></h3>
<p>The awk utility interprets a special-purpose programming language that makes it possible to handle simple data-reformatting jobs with just a few lines of code.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gawk">https://www.gnu.org/software/gawk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>5.1.1</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gbasis">gbasis<a class="headerlink" href="#gbasis" title="Permanent link">&para;</a></h3>
<p>Python library for analytical evaluation and integration of Gaussian-type basis functions and  related quantities.</p>
<p><em>homepage</em>: <a href="https://github.com/theochem/gbasis">https://github.com/theochem/gbasis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210904</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gblocks">Gblocks<a class="headerlink" href="#gblocks" title="Permanent link">&para;</a></h3>
<p>Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis</p>
<p><em>homepage</em>: <a href="http://molevol.cmima.csic.es/castresana/Gblocks.html">http://molevol.cmima.csic.es/castresana/Gblocks.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.91b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gbprocess">GBprocesS<a class="headerlink" href="#gbprocess" title="Permanent link">&para;</a></h3>
<p>GBprocesS allows for the extraction of genomic inserts from NGS data for GBS experiments. Preprocessing is performed in different stages that are part of a linear pipeline where the steps are performed in order. GBprocesS provides a flexible way to adjust the functionality to your needs, as the operations required and the execution order vary depending on the GBS protocol used.</p>
<p><em>homepage</em>: <a href="https://gbprocess.readthedocs.io/">https://gbprocess.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="gbs2ploidy">gbs2ploidy<a class="headerlink" href="#gbs2ploidy" title="Permanent link">&para;</a></h3>
<p>Inference of Ploidy from (Genotyping-by-Sequencing) GBS Data</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/gbs2ploidy">https://cran.r-project.org/web/packages/gbs2ploidy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="gc">gc<a class="headerlink" href="#gc" title="Permanent link">&para;</a></h3>
<p>The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting replacement for C malloc or C++ new. </p>
<p><em>homepage</em>: <a href="https://hboehm.info/gc/">https://hboehm.info/gc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.4.4</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>7.6.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.6.4</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>7.6.10</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>7.6.12</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.0.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.2.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gc3pie">GC3Pie<a class="headerlink" href="#gc3pie" title="Permanent link">&para;</a></h3>
<p>GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution  environments.</p>
<p><em>homepage</em>: <a href="https://gc3pie.readthedocs.org">https://gc3pie.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gcc">GCC<a class="headerlink" href="#gcc" title="Permanent link">&para;</a></h3>
<p>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).</p>
<p><em>homepage</em>: <a href="http://gcc.gnu.org/">http://gcc.gnu.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.2</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.2</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.2</code></td>
<td><code>-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.3</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.4</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.4</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.0</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.0</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>-CLooG</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>-CLooG-multilib</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>-binutils-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td><code>-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td><code>-binutils-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.4</code></td>
<td><code>-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>-binutils-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.5.0</code></td>
<td><code>-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.0</code></td>
<td><code>-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.4.0</code></td>
<td><code>-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.0</code></td>
<td><code>-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>-2.29</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.3.0</code></td>
<td><code>-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.4.0</code></td>
<td><code>-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td><code>-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.3.0</code></td>
<td><code>-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td><code>-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.2.0</code></td>
<td><code>-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.4.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.5.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gcccore">GCCcore<a class="headerlink" href="#gcccore" title="Permanent link">&para;</a></h3>
<p>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).</p>
<p><em>homepage</em>: <a href="http://gcc.gnu.org/">http://gcc.gnu.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gcccuda">gcccuda<a class="headerlink" href="#gcccuda" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.08</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gcloud">gcloud<a class="headerlink" href="#gcloud" title="Permanent link">&para;</a></h3>
<p>Libraries and tools for interacting with Google Cloud products and services.</p>
<p><em>homepage</em>: <a href="https://cloud.google.com/sdk">https://cloud.google.com/sdk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>382.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gconf">GConf<a class="headerlink" href="#gconf" title="Permanent link">&para;</a></h3>
<p>GConf is a system for storing application preferences.  It is intended for user preferences; not configuration  of something like Apache, or arbitrary data storage.</p>
<p><em>homepage</em>: <a href="https://projects.gnome.org/gconf/">https://projects.gnome.org/gconf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.6</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016a</code>, <code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="gcta">GCTA<a class="headerlink" href="#gcta" title="Permanent link">&para;</a></h3>
<p>GCTA (Genome-wide Complex Trait Analysis) is a software package, which was  initially developed to estimate the proportion of phenotypic variance explained  by all genome-wide SNPs for a complex trait but has been extensively extended  for many other analyses of data from genome-wide association studies (GWASs). </p>
<p><em>homepage</em>: <a href="https://yanglab.westlake.edu.cn/software/gcta/">https://yanglab.westlake.edu.cn/software/gcta/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.94.0beta</code></td>
<td><code>foss/2021b</code>, <code>gfbf/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gctf">Gctf<a class="headerlink" href="#gctf" title="Permanent link">&para;</a></h3>
<p>Gctf: real-time CTF determination and correction, Kai Zhang, 2016</p>
<p><em>homepage</em>: <a href="https://www.mrc-lmb.cam.ac.uk/kzhang/">https://www.mrc-lmb.cam.ac.uk/kzhang/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.06</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gd">GD<a class="headerlink" href="#gd" title="Permanent link">&para;</a></h3>
<p>GD.pm - Interface to Gd Graphics Library</p>
<p><em>homepage</em>: <a href="https://github.com/lstein/Perl-GD">https://github.com/lstein/Perl-GD</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.66</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.68</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.69</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.71</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.73</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.75</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdal">GDAL<a class="headerlink" href="#gdal" title="Permanent link">&para;</a></h3>
<p>GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for  data translation and processing.</p>
<p><em>homepage</em>: <a href="https://www.gdal.org">https://www.gdal.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.13-HDF5-1.8.18</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.13-HDF5-HDF</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>-Python-2.7.14-HDF5-1.8.19</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.4</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="gdb">GDB<a class="headerlink" href="#gdb" title="Permanent link">&para;</a></h3>
<p>The GNU Project Debugger</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gdb/gdb.html">https://www.gnu.org/software/gdb/gdb.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.8.2</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>7.9</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>7.10.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.11</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.11.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>8.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>9.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>10.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>10.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>10.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>11.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>12.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdbgui">gdbgui<a class="headerlink" href="#gdbgui" title="Permanent link">&para;</a></h3>
<p>Browser-based frontend to gdb (gnu debugger). Add breakpoints, view the stack, visualize data structures, and more in C, C++, Go, Rust, and Fortran. Run gdbgui from the terminal and a new tab will open in your browser.</p>
<p><em>homepage</em>: <a href="https://gdbgui.com">https://gdbgui.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.1.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdbm">gdbm<a class="headerlink" href="#gdbm" title="Permanent link">&para;</a></h3>
<p>GNU dbm (or GDBM, for short) is a library of database functions that use extensible hashing  and work similar to the standard UNIX dbm.  These routines are provided to a programmer needing to create and manipulate a hashed database.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org.ua/software/gdbm/gdbm.html">https://www.gnu.org.ua/software/gdbm/gdbm.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.18.1</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.21</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdc-client">gdc-client<a class="headerlink" href="#gdc-client" title="Permanent link">&para;</a></h3>
<p>The gdc-client provides several convenience functions over  the GDC API which provides general download/upload via HTTPS.</p>
<p><em>homepage</em>: <a href="https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool">https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdchart">GDCHART<a class="headerlink" href="#gdchart" title="Permanent link">&para;</a></h3>
<p>Easy to use C API, high performance library to create charts  and graphs in PNG, GIF and WBMP format.</p>
<p><em>homepage</em>: <a href="http://users.fred.net/brv/chart">http://users.fred.net/brv/chart</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.5dev</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdcm">GDCM<a class="headerlink" href="#gdcm" title="Permanent link">&para;</a></h3>
<p>Grassroots DICOM: Cross-platform DICOM implementation</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/gdcm">https://sourceforge.net/projects/gdcm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.8</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.8.9</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.0.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0.20</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdgraph">GDGraph<a class="headerlink" href="#gdgraph" title="Permanent link">&para;</a></h3>
<p>GDGraph is a Perl package to generate charts</p>
<p><em>homepage</em>: <a href="https://metacpan.org/release/GDGraph">https://metacpan.org/release/GDGraph</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.54</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.54</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.56</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdist">gdist<a class="headerlink" href="#gdist" title="Permanent link">&para;</a></h3>
<p>The gdist module is a Cython interface to a C++ library (http://code.google.com/p/geodesic/) for  computing geodesic distance which is the length of shortest line between two vertices on a triangulated mesh in three  dimensions, such that the line lies on the surface.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/gdist">https://pypi.python.org/pypi/gdist</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="gdk-pixbuf">Gdk-Pixbuf<a class="headerlink" href="#gdk-pixbuf" title="Permanent link">&para;</a></h3>
<p>The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.  It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it  was distributed as part of GTK+ 2 but it was split off into a separate package  in preparation for the change to GTK+ 3. </p>
<p><em>homepage</em>: <a href="https://docs.gtk.org/gdk-pixbuf/">https://docs.gtk.org/gdk-pixbuf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.32.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.35.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.36.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.36.8</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.36.10</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.36.11</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.36.12</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.38.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.38.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.40.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.42.6</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.42.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.42.10</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gdrcopy">GDRCopy<a class="headerlink" href="#gdrcopy" title="Permanent link">&para;</a></h3>
<p>A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology.</p>
<p><em>homepage</em>: <a href="https://github.com/NVIDIA/gdrcopy">https://github.com/NVIDIA/gdrcopy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>-CUDA-11.0.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-CUDA-11.2.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="geant4">Geant4<a class="headerlink" href="#geant4" title="Permanent link">&para;</a></h3>
<p>Geant4 is a toolkit for the simulation of the passage of particles through matter.  Its areas of application include high energy, nuclear and accelerator physics,   as well as studies in medical and space science.</p>
<p><em>homepage</em>: <a href="https://geant4.cern.ch">https://geant4.cern.ch</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.5.p02</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>9.6.p04</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>10.01.p03</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>10.02.p01</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>10.03.p03</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>10.04</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>10.5</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>10.6</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>10.6.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>10.7.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>11.0.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>11.0.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>11.0.2</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="geant4-data">Geant4-data<a class="headerlink" href="#geant4-data" title="Permanent link">&para;</a></h3>
<p>Datasets for Geant4.</p>
<p><em>homepage</em>: <a href="https://geant4.cern.ch/">https://geant4.cern.ch/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20201103</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20210510</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gearshifft">gearshifft<a class="headerlink" href="#gearshifft" title="Permanent link">&para;</a></h3>
<p>Benchmark Suite for Heterogenuous FFT Implementations</p>
<p><em>homepage</em>: <a href="https://github.com/mpicbg-scicomp/gearshifft">https://github.com/mpicbg-scicomp/gearshifft</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="gegl">GEGL<a class="headerlink" href="#gegl" title="Permanent link">&para;</a></h3>
<p>GEGL (Generic Graphics Library) is a data flow based image processing framework,  providing floating point processing and non-destructive image processing capabilities to  GNU Image Manipulation Program (GIMP) and other projects.</p>
<p><em>homepage</em>: <a href="https://www.gegl.org">https://www.gegl.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.30</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gem-library">GEM-library<a class="headerlink" href="#gem-library" title="Permanent link">&para;</a></h3>
<p>Next-generation sequencing platforms (Illumina/Solexa, ABI/SOLiD, etc.)   call for powerful and very optimized tools to index/analyze huge genomes. The GEM library   strives to be a true "next-generation" tool for handling any kind of sequence data, offering   state-of-the-art algorithms and data structures specifically tailored to this demanding task.   At the moment, efficient indexing and searching algorithms based on the Burrows-Wheeler   transform (BWT) have been implemented. The library core is written in C for maximum speed,   with concise interfaces to higher-level programming languages like OCaml and Python.   Many high-performance standalone programs (mapper, splice mapper, etc.) are provided along   with the library; in general, new algorithms and tools can be easily implemented on the top of it. </p>
<p><em>homepage</em>: <a href="http://algorithms.cnag.cat/wiki/The_GEM_library">http://algorithms.cnag.cat/wiki/The_GEM_library</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20130406-045632</code></td>
<td><code>_pre-release-3_Linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gemma">GEMMA<a class="headerlink" href="#gemma" title="Permanent link">&para;</a></h3>
<p>Genome-wide Efficient Mixed Model Association</p>
<p><em>homepage</em>: <a href="https://github.com/genetics-statistics/GEMMA">https://github.com/genetics-statistics/GEMMA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.97</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>0.98.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.98.5</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gemmi">gemmi<a class="headerlink" href="#gemmi" title="Permanent link">&para;</a></h3>
<p>Gemmi is a library, accompanied by a set of programs, developed primarily for   use in macromolecular crystallography (MX). For working with:  macromolecular models (content of PDB, PDBx/mmCIF and mmJSON files), refinement restraints (CIF files), reflection data (MTZ and mmCIF formats), data on a 3D   grid (electron density maps, masks, MRC/CCP4 format) crystallographic symmetry. Parts of this library can be useful in structural bioinformatics (for symmetry- aware analysis of protein models), and in other molecular-structure sciences    that use CIF files (we have the fastest open-source CIF parser). </p>
<p><em>homepage</em>: <a href="https://gemmi.readthedocs.io/">https://gemmi.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gencore_variant_detection">gencore_variant_detection<a class="headerlink" href="#gencore_variant_detection" title="Permanent link">&para;</a></h3>
<p>This is a bundled install of many software packages for doing variant detection analysis. </p>
<p><em>homepage</em>: <a href="https://nyuad-cgsb.github.io/variant_detection/public/index.html">https://nyuad-cgsb.github.io/variant_detection/public/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="genemark-et">GeneMark-ET<a class="headerlink" href="#genemark-et" title="Permanent link">&para;</a></h3>
<p>Eukaryotic gene prediction suite with automatic training</p>
<p><em>homepage</em>: <a href="http://exon.gatech.edu/GeneMark">http://exon.gatech.edu/GeneMark</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.38</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.57</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.65</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.71</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gengetopt">gengetopt<a class="headerlink" href="#gengetopt" title="Permanent link">&para;</a></h3>
<p>Gengetopt is a tool to write command line option parsing code for C programs.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gengetopt/gengetopt.html">https://www.gnu.org/software/gengetopt/gengetopt.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.23</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="genome_profiler">Genome_Profiler<a class="headerlink" href="#genome_profiler" title="Permanent link">&para;</a></h3>
<p>Genome Profiler (GeP) is a program to perform whole-genome multilocus  sequence typing (wgMLST) analysis for bacterial isolates</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/genomeprofiler/">https://sourceforge.net/projects/genomeprofiler/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="genomemapper">GenomeMapper<a class="headerlink" href="#genomemapper" title="Permanent link">&para;</a></h3>
<p>GenomeMapper is a short read mapping tool designed for accurate read alignments.   It quickly aligns millions of reads either with ungapped or gapped alignments.   This version is used to align against a single reference.   If you are unsure which one is the appropriate GenomeMapper, you might want to use this one.</p>
<p><em>homepage</em>: <a href="https://1001genomes.org/software/genomemapper_singleref.html">https://1001genomes.org/software/genomemapper_singleref.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="genomepy">genomepy<a class="headerlink" href="#genomepy" title="Permanent link">&para;</a></h3>
<p>genomepy is designed to provide a simple and straightforward way to download and use genomic data</p>
<p><em>homepage</em>: <a href="https://github.com/vanheeringen-lab/genomepy">https://github.com/vanheeringen-lab/genomepy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="genometester4">GenomeTester4<a class="headerlink" href="#genometester4" title="Permanent link">&para;</a></h3>
<p>A toolkit for performing set operations - union, intersection and complement - on k-mer lists.</p>
<p><em>homepage</em>: <a href="https://github.com/bioinfo-ut/GenomeTester4">https://github.com/bioinfo-ut/GenomeTester4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="genomethreader">GenomeThreader<a class="headerlink" href="#genomethreader" title="Permanent link">&para;</a></h3>
<p>GenomeThreader is a software tool to compute gene structure predictions.</p>
<p><em>homepage</em>: <a href="https://genomethreader.org">https://genomethreader.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Linux_x86_64-64bit</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td><code>-Linux_x86_64-64bit</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="genometools">GenomeTools<a class="headerlink" href="#genometools" title="Permanent link">&para;</a></h3>
<p>A comprehensive software library for efficient processing of structured genome annotations.</p>
<p><em>homepage</em>: <a href="http://genometools.org">http://genometools.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.10</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.5.10</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.6.2</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="genomeworks">GenomeWorks<a class="headerlink" href="#genomeworks" title="Permanent link">&para;</a></h3>
<p>SDK for GPU accelerated genome assembly and analysis</p>
<p><em>homepage</em>: <a href="https://github.com/clara-parabricks/GenomeWorks">https://github.com/clara-parabricks/GenomeWorks</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021.02.2</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="genotypeharmonizer">GenotypeHarmonizer<a class="headerlink" href="#genotypeharmonizer" title="Permanent link">&para;</a></h3>
<p>The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data  stored using different file formats with different and potentially unknown strands.</p>
<p><em>homepage</em>: <a href="https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer">https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.14</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="genozip">genozip<a class="headerlink" href="#genozip" title="Permanent link">&para;</a></h3>
<p>Genozip is a compressor for genomic files - it compresses FASTQ, SAM/BAM/CRAM, VCF, FASTA and others.</p>
<p><em>homepage</em>: <a href="https://genozip.com">https://genozip.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>13.0.5</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gensim">gensim<a class="headerlink" href="#gensim" title="Permanent link">&para;</a></h3>
<p>Gensim is a Python library for topic modelling, document indexing and similarity retrieval with  large corpora.</p>
<p><em>homepage</em>: <a href="https://radimrehurek.com/gensim">https://radimrehurek.com/gensim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.8.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="geocube">geocube<a class="headerlink" href="#geocube" title="Permanent link">&para;</a></h3>
<p>Tool to convert geopandas vector data into rasterized xarray data.</p>
<p><em>homepage</em>: <a href="https://corteva.github.io/geocube/latest/">https://corteva.github.io/geocube/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.14</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="geopandas">geopandas<a class="headerlink" href="#geopandas" title="Permanent link">&para;</a></h3>
<p>GeoPandas is a project to add support for geographic data to pandas objects. It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and perform geometric operations.</p>
<p><em>homepage</em>: <a href="https://github.com/geopandas/geopandas">https://github.com/geopandas/geopandas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.10.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.12.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="geopy">geopy<a class="headerlink" href="#geopy" title="Permanent link">&para;</a></h3>
<p>geopy is a Python client for several popular geocoding web services. geopy makes it easy for Python developers to locate the coordinates of addresses, cities, countries, and landmarks across the globe using third-party geocoders and other data sources. </p>
<p><em>homepage</em>: <a href="https://geopy.readthedocs.io/en/stable/">https://geopy.readthedocs.io/en/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="georges">georges<a class="headerlink" href="#georges" title="Permanent link">&para;</a></h3>
<p>Georges the lemur opinionated particle accelerator modeling Python package. Also a thin wrapper over MAD-X/PTC, BDSim and G4Beamline.</p>
<p><em>homepage</em>: <a href="https://github.com/ULB-Metronu/georges">https://github.com/ULB-Metronu/georges</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2019.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="geos">GEOS<a class="headerlink" href="#geos" title="Permanent link">&para;</a></h3>
<p>GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)</p>
<p><em>homepage</em>: <a href="https://trac.osgeo.org/geos">https://trac.osgeo.org/geos</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.01</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>3.10.3</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gerris">Gerris<a class="headerlink" href="#gerris" title="Permanent link">&para;</a></h3>
<p>Gerris is a Free Software program for the solution of the partial                  differential equations describing fluid flow</p>
<p><em>homepage</em>: <a href="http://gfs.sourceforge.net/wiki/index.php/Main_Page">http://gfs.sourceforge.net/wiki/index.php/Main_Page</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20131206</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="getorb">GETORB<a class="headerlink" href="#getorb" title="Permanent link">&para;</a></h3>
<p>GETORB software package contains programs to handle the orbital data records (ODRs)</p>
<p><em>homepage</em>: <a href="http://www.deos.tudelft.nl/ers/precorbs/tools/getorb_pack.shtml">http://www.deos.tudelft.nl/ers/precorbs/tools/getorb_pack.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.2</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="getorganelle">GetOrganelle<a class="headerlink" href="#getorganelle" title="Permanent link">&para;</a></h3>
<p>This toolkit assemblies organelle genome from genomic skimming data.</p>
<p><em>homepage</em>: <a href="https://github.com/Kinggerm/GetOrganelle">https://github.com/Kinggerm/GetOrganelle</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.7.4-pre2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.7.5.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.7.6.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gettext">gettext<a class="headerlink" href="#gettext" title="Permanent link">&para;</a></h3>
<p>GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gettext/">https://www.gnu.org/software/gettext/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.19.4</code></td>
<td></td>
<td><code>GCC/4.9.2</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.19.6</code></td>
<td></td>
<td><code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.19.7</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.19.8</code></td>
<td></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016.04</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.19.8.1</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.19.8.1</code></td>
<td><code>-libxml2-2.9.7</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.20.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.21</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.21.1</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gexiv2">gexiv2<a class="headerlink" href="#gexiv2" title="Permanent link">&para;</a></h3>
<p>gexiv2 is a GObject wrapper around the Exiv2 photo metadata library.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/gexiv2">https://wiki.gnome.org/Projects/gexiv2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gfbf">gfbf<a class="headerlink" href="#gfbf" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  FlexiBLAS (BLAS and LAPACK support) and (serial) FFTW.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2022a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gffcompare">GffCompare<a class="headerlink" href="#gffcompare" title="Permanent link">&para;</a></h3>
<p>GffCompare provides classification and reference annotation mapping and  matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files.</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/stringtie/gffcompare.shtml">https://ccb.jhu.edu/software/stringtie/gffcompare.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.10.6</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.11.6</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.12.2</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>0.12.6</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gffread">gffread<a class="headerlink" href="#gffread" title="Permanent link">&para;</a></h3>
<p>GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and  more.</p>
<p><em>homepage</em>: <a href="https://github.com/gpertea/gffread">https://github.com/gpertea/gffread</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.10.6</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.11.6</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.12.7</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gflags">gflags<a class="headerlink" href="#gflags" title="Permanent link">&para;</a></h3>
<p>The gflags package contains a C++ library that implements commandline flags processing.  It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. </p>
<p><em>homepage</em>: <a href="https://github.com/gflags/gflags">https://github.com/gflags/gflags</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gfold">GFOLD<a class="headerlink" href="#gfold" title="Permanent link">&para;</a></h3>
<p>Generalized fold change for ranking differentially expressed genes from RNA-seq data</p>
<p><em>homepage</em>: <a href="http://www.tongji.edu.cn/~zhanglab/GFOLD/index.html">http://www.tongji.edu.cn/~zhanglab/GFOLD/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="gh">gh<a class="headerlink" href="#gh" title="Permanent link">&para;</a></h3>
<p>gh is GitHub on the command line.</p>
<p><em>homepage</em>: <a href="https://github.com/cli/cli">https://github.com/cli/cli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.20.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ghc">GHC<a class="headerlink" href="#ghc" title="Permanent link">&para;</a></h3>
<p>The Glorious/Glasgow Haskell Compiler</p>
<p><em>homepage</em>: <a href="https://haskell.org/ghc/">https://haskell.org/ghc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.12.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.6.5</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ghostscript">Ghostscript<a class="headerlink" href="#ghostscript" title="Permanent link">&para;</a></h3>
<p>Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to  different targets. It used to be part of the cups printing stack, but is no longer used for that.</p>
<p><em>homepage</em>: <a href="https://ghostscript.com">https://ghostscript.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.19</code></td>
<td></td>
<td><code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>9.20</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>9.21</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>9.22</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>9.22</code></td>
<td><code>-cairo-1.14.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>9.23</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>9.23</code></td>
<td><code>-cairo-1.14.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>9.27</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>9.50</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>9.52</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>9.53.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>9.54.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.56.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>10.0.0</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gibbs2">Gibbs2<a class="headerlink" href="#gibbs2" title="Permanent link">&para;</a></h3>
<p>Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary  conditions of temperature and pressure. Gibbs2 uses the results of periodic solid-state quantum-mechanical  calculations, specifically the energy-volume curve and possibly the harmonic phonon frequencies, to compute the  thermodynamic properties of the solid within the framework of the quasiharmonic approximation.</p>
<p><em>homepage</em>: <a href="https://aoterodelaroza.github.io/gibbs2/">https://aoterodelaroza.github.io/gibbs2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="giflib">giflib<a class="headerlink" href="#giflib" title="Permanent link">&para;</a></h3>
<p>giflib is a library for reading and writing gif images. It is API and ABI compatible with libungif which was in wide use while the LZW compression algorithm was patented.</p>
<p><em>homepage</em>: <a href="http://giflib.sourceforge.net/">http://giflib.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.4</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gifsicle">gifsicle<a class="headerlink" href="#gifsicle" title="Permanent link">&para;</a></h3>
<p>Gifsicle is a command-line tool for creating, editing,  and getting information about GIF images and animations.   Making a GIF animation with gifsicle is easy.</p>
<p><em>homepage</em>: <a href="https://github.com/kohler/gifsicle">https://github.com/kohler/gifsicle</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.92</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.93</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gimic">GIMIC<a class="headerlink" href="#gimic" title="Permanent link">&para;</a></h3>
<p>The GIMIC program calculates magnetically induced currents in molecules.  You need to provide this program with a density matrix in atomic-orbital (AO) basis  and three (effective) magnetically perturbed AO density matrices in the proper format. Currently ACES2, Turbomole, G09, QChem, FERMION++, and LSDalton can produce these matrices.</p>
<p><em>homepage</em>: <a href="http://gimic.readthedocs.io">http://gimic.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.04.20</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="gimkl">gimkl<a class="headerlink" href="#gimkl" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gimp">GIMP<a class="headerlink" href="#gimp" title="Permanent link">&para;</a></h3>
<p>GIMP is a cross-platform image editor available for GNU/Linux, OS X, Windows and more operating systems.</p>
<p><em>homepage</em>: <a href="https://www.gimp.org">https://www.gimp.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.24</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gimpi">gimpi<a class="headerlink" href="#gimpi" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gimpic">gimpic<a class="headerlink" href="#gimpic" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including IntelMPI for MPI support with CUDA features enabled.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gimps">GIMPS<a class="headerlink" href="#gimps" title="Permanent link">&para;</a></h3>
<p>GIMPS: Great Internet Mersenne Prime Search;  it can be useful for limited stress testing of nodes, in user space</p>
<p><em>homepage</em>: <a href="http://www.mersenne.org/">http://www.mersenne.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>p95v279</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>p95v279.linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="giolf">giolf<a class="headerlink" href="#giolf" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="giolfc">giolfc<a class="headerlink" href="#giolfc" title="Permanent link">&para;</a></h3>
<p>GCC based compiler toolchain <strong>with CUDA support</strong>, and including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="giotto-suite">Giotto-Suite<a class="headerlink" href="#giotto-suite" title="Permanent link">&para;</a></h3>
<p>Giotto Suite is focused on building a modular platform for analyzing spatial-omics technologies  and strives to be interoperable with other popular spatial analysis tools and classes. Using established packages  optimized for large(r) data, Giotto Suite adopts fast and memory efficient methods to create an interactive  analysis.</p>
<p><em>homepage</em>: <a href="https://github.com/drieslab/Giotto">https://github.com/drieslab/Giotto</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="git">git<a class="headerlink" href="#git" title="Permanent link">&para;</a></h3>
<p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.</p>
<p><em>homepage</em>: <a href="http://git-scm.com/">http://git-scm.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.5.6</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.12.2</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.13.1</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.14.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.16.1</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.19.1</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.21.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.21.0</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.23.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.23.0</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.28.0</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.32.0</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.33.1</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.36.0</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.38.1</code></td>
<td><code>-nodocs</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="git-extras">git-extras<a class="headerlink" href="#git-extras" title="Permanent link">&para;</a></h3>
<p>Extra useful scripts for git</p>
<p><em>homepage</em>: <a href="https://github.com/tj/git-extras">https://github.com/tj/git-extras</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="git-lfs">git-lfs<a class="headerlink" href="#git-lfs" title="Permanent link">&para;</a></h3>
<p>Git Large File Storage (LFS) replaces large files such as audio  samples, videos, datasets, and graphics with text pointers inside Git, while  storing the file contents on a remote server like GitHub.com</p>
<p><em>homepage</em>: <a href="https://git-lfs.github.com">https://git-lfs.github.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gitpython">GitPython<a class="headerlink" href="#gitpython" title="Permanent link">&para;</a></h3>
<p>GitPython is a python library used to interact with Git repositories </p>
<p><em>homepage</em>: <a href="https://github.com/gitpython-developers/GitPython">https://github.com/gitpython-developers/GitPython</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.11</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1.15</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.1.14</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.1.18</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.1.24</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.1.27</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="givaro">Givaro<a class="headerlink" href="#givaro" title="Permanent link">&para;</a></h3>
<p>C++ library for arithmetic and algebraic computations</p>
<p><em>homepage</em>: <a href="http://givaro.forge.imag.fr/">http://givaro.forge.imag.fr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="giza">Giza<a class="headerlink" href="#giza" title="Permanent link">&para;</a></h3>
<p>Giza is an open, lightweight scientific plotting library built on  top of cairo that provides uniform output to multiple devices.</p>
<p><em>homepage</em>: <a href="https://danieljprice.github.io/giza/">https://danieljprice.github.io/giza/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="gkeyll">GKeyll<a class="headerlink" href="#gkeyll" title="Permanent link">&para;</a></h3>
<p>Gkeyll v2.0 (pronounced as in the book The Strange Case of Dr. Jekyll  and Mr. Hyde) is a computational plasma physics code mostly written in C/C++ and LuaJIT.  Gkeyll contains solvers for gyrokinetic equations, Vlasov-Maxwell equations, and  multi-fluid equations. Gkeyll contains ab-initio and novel implementations of a number  of algorithms, and perhaps is unique in using a JIT compiled typeless dynamic language  for as its main implementation language.</p>
<p><em>homepage</em>: <a href="https://gkeyll.readthedocs.io">https://gkeyll.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220803</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="gl2ps">GL2PS<a class="headerlink" href="#gl2ps" title="Permanent link">&para;</a></h3>
<p>GL2PS: an OpenGL to PostScript printing library</p>
<p><em>homepage</em>: <a href="https://www.geuz.org/gl2ps/">https://www.geuz.org/gl2ps/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.9</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.9</code></td>
<td><code>-Mesa-11.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glade">Glade<a class="headerlink" href="#glade" title="Permanent link">&para;</a></h3>
<p>Glade is a RAD tool to enable quick &amp; easy development of user interfaces for the GTK+ toolkit  and the GNOME desktop environment.</p>
<p><em>homepage</em>: <a href="https://glade.gnome.org/">https://glade.gnome.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.8.5</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.8.6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.8.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="glew">glew<a class="headerlink" href="#glew" title="Permanent link">&para;</a></h3>
<p>The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform.</p>
<p><em>homepage</em>: <a href="https://github.com/nigels-com/glew">https://github.com/nigels-com/glew</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-egl</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-glx</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-osmesa</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glfw">GLFW<a class="headerlink" href="#glfw" title="Permanent link">&para;</a></h3>
<p>GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop</p>
<p><em>homepage</em>: <a href="https://www.glfw.org">https://www.glfw.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.3.4</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.3.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glib">GLib<a class="headerlink" href="#glib" title="Permanent link">&para;</a></h3>
<p>GLib is one of the base libraries of the GTK+ project</p>
<p><em>homepage</em>: <a href="https://www.gtk.org/">https://www.gtk.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.42.1</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.44.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.47.5</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.48.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.49.5</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.52.0</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.53.5</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.54.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.54.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.60.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.62.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.64.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.66.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.68.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.69.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.72.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.75.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glib-networking">glib-networking<a class="headerlink" href="#glib-networking" title="Permanent link">&para;</a></h3>
<p>Network extensions for GLib</p>
<p><em>homepage</em>: <a href="https://gitlab.gnome.org/GNOME/glib-networking">https://gitlab.gnome.org/GNOME/glib-networking</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.68.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.72.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glibc">glibc<a class="headerlink" href="#glibc" title="Permanent link">&para;</a></h3>
<p>The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems,  as well as many other systems that use Linux as the kernel.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libc/">https://www.gnu.org/software/libc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.17</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.26</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.30</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glibmm">GLibmm<a class="headerlink" href="#glibmm" title="Permanent link">&para;</a></h3>
<p>C++ bindings for Glib</p>
<p><em>homepage</em>: <a href="https://www.gtk.org/">https://www.gtk.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.49.7</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.66.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glimmer">GLIMMER<a class="headerlink" href="#glimmer" title="Permanent link">&para;</a></h3>
<p>Glimmer is a system for finding genes in microbial DNA, especially  the genomes of bacteria, archaea, and viruses.</p>
<p><em>homepage</em>: <a href="http://ccb.jhu.edu/software/glimmer/index.shtml">http://ccb.jhu.edu/software/glimmer/index.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.02b</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="glimmerhmm">GlimmerHMM<a class="headerlink" href="#glimmerhmm" title="Permanent link">&para;</a></h3>
<p>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model.  Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally  it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted  from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models.</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/glimmerhmm">https://ccb.jhu.edu/software/glimmerhmm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.0.4c</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glm">GLM<a class="headerlink" href="#glm" title="Permanent link">&para;</a></h3>
<p>OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on  the OpenGL Shading Language (GLSL) specifications.</p>
<p><em>homepage</em>: <a href="https://github.com/g-truc/glm">https://github.com/g-truc/glm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.7.6</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.9.8.3</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.9.9.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.9.9.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glm-aed">GLM-AED<a class="headerlink" href="#glm-aed" title="Permanent link">&para;</a></h3>
<p>The General Lake Model (GLM) is a water balance and one-dimensional vertical stratification hydrodynamic model, which is dynamically coupled with the AED water quality modelling library. GLM-AED is suitable for simulating conditions in a wide range of natural and engineered lakes, including shallow (well-mixed) and deep (stratified) systems. The model has been successfully applied to systems from the scale of individual ponds and wetlands, to actively operated reservoirs, upto the scale of the Great Lakes.</p>
<p><em>homepage</em>: <a href="https://github.com/AquaticEcoDynamics/glm-aed">https://github.com/AquaticEcoDynamics/glm-aed</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0a5</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="globalarrays">GlobalArrays<a class="headerlink" href="#globalarrays" title="Permanent link">&para;</a></h3>
<p>Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model</p>
<p><em>homepage</em>: <a href="https://hpc.pnl.gov/globalarrays">https://hpc.pnl.gov/globalarrays</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.7</code></td>
<td></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.7</code></td>
<td><code>-peigs</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>5.7.2</code></td>
<td></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.7.2</code></td>
<td><code>-peigs</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td></td>
<td><code>intel/2020a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>5.8.1</code></td>
<td></td>
<td><code>intel/2022a</code></td>
</tr>
<tr>
<td><code>5.8.2</code></td>
<td></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="globus-cli">Globus-CLI<a class="headerlink" href="#globus-cli" title="Permanent link">&para;</a></h3>
<p>A Command Line Wrapper over the Globus SDK for Python, which provides an interface to Globus services  from the shell, and is suited to both interactive and simple scripting use cases.</p>
<p><em>homepage</em>: <a href="https://docs.globus.org/cli/">https://docs.globus.org/cli/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="globusconnectpersonal">GlobusConnectPersonal<a class="headerlink" href="#globusconnectpersonal" title="Permanent link">&para;</a></h3>
<p>Globus Connect Personal turns your laptop or other personal computer into a  Globus endpoint with a just a few clicks. With Globus Connect Personal you can  share and transfer files to/from a local machinecampus server, desktop computer or laptopeven if it's behind a firewall and you don't have administrator  privileges. </p>
<p><em>homepage</em>: <a href="https://www.globus.org/globus-connect-personal">https://www.globus.org/globus-connect-personal</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="glog">glog<a class="headerlink" href="#glog" title="Permanent link">&para;</a></h3>
<p>A C++ implementation of the Google logging module.</p>
<p><em>homepage</em>: <a href="https://github.com/google/glog">https://github.com/google/glog</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.3.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glpk">GLPK<a class="headerlink" href="#glpk" title="Permanent link">&para;</a></h3>
<p>The GLPK (GNU Linear Programming Kit) package is intended for  solving large-scale linear programming (LP),  mixed integer programming (MIP), and other related problems.  It is a set of routines written in ANSI C  and organized in the form of a callable library.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/glpk/">https://www.gnu.org/software/glpk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.58</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.60</code></td>
<td><code>GCCcore/5.4.0</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.61</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.65</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="glproto">glproto<a class="headerlink" href="#glproto" title="Permanent link">&para;</a></h3>
<p>X protocol and ancillary headers</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.17</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="glucose">Glucose<a class="headerlink" href="#glucose" title="Permanent link">&para;</a></h3>
<p>Glucose is based on a new scoring scheme (well, not so new now, it was introduced in 2009) for the clause learning mechanism of so called Modern SAT solvers (it is based on our IJCAI'09 paper). It is designed to be parallel, since v4.0.</p>
<p><em>homepage</em>: <a href="https://www.labri.fr/perso/lsimon/glucose/">https://www.labri.fr/perso/lsimon/glucose/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gmap-gsnap">GMAP-GSNAP<a class="headerlink" href="#gmap-gsnap" title="Permanent link">&para;</a></h3>
<p>GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences  GSNAP: Genomic Short-read Nucleotide Alignment Program</p>
<p><em>homepage</em>: <a href="http://research-pub.gene.com/gmap/">http://research-pub.gene.com/gmap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016-05-01</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2016-11-07</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2018-05-11</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2018-07-04</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2019-03-15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2019-09-12</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2020-12-17</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2021-21-17</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2023-02-17</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gmp">GMP<a class="headerlink" href="#gmp" title="Permanent link">&para;</a></h3>
<p>GMP is a free library for arbitrary precision arithmetic, operating on signed  integers, rational numbers, and floating point numbers. </p>
<p><em>homepage</em>: <a href="https://gmplib.org/">https://gmplib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.3</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>6.0.0a</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>6.1.0</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>6.1.1</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>6.1.2</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gmpich">gmpich<a class="headerlink" href="#gmpich" title="Permanent link">&para;</a></h3>
<p>gcc and GFortran based compiler toolchain,  including MPICH for MPI support.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.08</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gmpolf">gmpolf<a class="headerlink" href="#gmpolf" title="Permanent link">&para;</a></h3>
<p>gcc and GFortran based compiler toolchain,  MPICH for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.10</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gmpy2">gmpy2<a class="headerlink" href="#gmpy2" title="Permanent link">&para;</a></h3>
<p>GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x</p>
<p><em>homepage</em>: <a href="https://github.com/aleaxit/gmpy">https://github.com/aleaxit/gmpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.8</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.8</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0.8</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0b1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0b1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0b4</code></td>
<td></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.1.0b5</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>intel-compilers/2022.1.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gmsh">gmsh<a class="headerlink" href="#gmsh" title="Permanent link">&para;</a></h3>
<p>Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor.</p>
<p><em>homepage</em>: <a href="https://gmsh.info/">https://gmsh.info/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.0.6</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.5.6</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.5.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>4.8.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.9.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.11.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gmt">GMT<a class="headerlink" href="#gmt" title="Permanent link">&para;</a></h3>
<p>GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools. </p>
<p><em>homepage</em>: <a href="https://www.generic-mapping-tools.org/">https://www.generic-mapping-tools.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.17</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.4.3</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.4.5</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>foss/2019b</code>, <code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="gmvapich2">gmvapich2<a class="headerlink" href="#gmvapich2" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain,  including MVAPICH2 for MPI support.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.20</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gmvolf">gmvolf<a class="headerlink" href="#gmvolf" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  MVAPICH2 for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.20</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gnu">GNU<a class="headerlink" href="#gnu" title="Permanent link">&para;</a></h3>
<p>Compiler-only toolchain with GCC and binutils.</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/">http://www.gnu.org/software/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9.2-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0-2.25</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gnuplot">gnuplot<a class="headerlink" href="#gnuplot" title="Permanent link">&para;</a></h3>
<p>Portable interactive, function plotting utility</p>
<p><em>homepage</em>: <a href="http://gnuplot.sourceforge.net">http://gnuplot.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.0.5</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.0.6</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.2.5</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.2.6</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>5.2.8</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.4.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.4.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.4.6</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gnutls">GnuTLS<a class="headerlink" href="#gnutls" title="Permanent link">&para;</a></h3>
<p>GnuTLS is a secure communications library implementing the SSL, TLS  and DTLS protocols and technologies around them. It provides a simple  C language application programming interface (API) to access the secure  communications protocols as well as APIs to parse and write X.509, PKCS #12,  OpenPGP and other required structures. It is aimed to be portable  and efficient with focus on security and interoperability.</p>
<p><em>homepage</em>: <a href="https://www.gnutls.org">https://www.gnutls.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.21</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.4.7</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.4.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.7.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.7.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="go">Go<a class="headerlink" href="#go" title="Permanent link">&para;</a></h3>
<p>Go is an open source programming language that makes it easy to build  simple, reliable, and efficient software.</p>
<p><em>homepage</em>: <a href="http://www.golang.org">http://www.golang.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCC/4.8.4</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>GCC/4.8.4</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.11.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.14.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.17.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.17.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.18.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.18.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="goalign">goalign<a class="headerlink" href="#goalign" title="Permanent link">&para;</a></h3>
<p>Goalign is a set of command line tools to manipulate multiple alignments.</p>
<p><em>homepage</em>: <a href="https://github.com/evolbioinfo/goalign">https://github.com/evolbioinfo/goalign</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="goatools">GOATOOLS<a class="headerlink" href="#goatools" title="Permanent link">&para;</a></h3>
<p>A Python library for Gene Ontology analyses</p>
<p><em>homepage</em>: <a href="https://github.com/tanghaibao/goatools">https://github.com/tanghaibao/goatools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.6</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gobff">gobff<a class="headerlink" href="#gobff" title="Permanent link">&para;</a></h3>
<p>GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.06</code></td>
<td><code>-amd</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gobject-introspection">GObject-Introspection<a class="headerlink" href="#gobject-introspection" title="Permanent link">&para;</a></h3>
<p>GObject introspection is a middleware layer between C libraries  (using GObject) and language bindings. The C library can be scanned at  compile time and generate a metadata file, in addition to the actual  native C library. Then at runtime, language bindings can read this  metadata and automatically provide bindings to call into the C library.</p>
<p><em>homepage</em>: <a href="https://gi.readthedocs.io/en/latest/">https://gi.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.47.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.48.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.49.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.52.0</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.53.5</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.53.5</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.53.5</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.54.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.54.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.54.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.58.3</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.60.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.63.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.64.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.66.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.68.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.72.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.74.0</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="goblf">goblf<a class="headerlink" href="#goblf" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, BLIS (BLAS support), LAPACK, FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="godon">Godon<a class="headerlink" href="#godon" title="Permanent link">&para;</a></h3>
<p>Godon is codon models software written in Go.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/Davydov/godon/">https://bitbucket.org/Davydov/godon/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210913</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gofasta">gofasta<a class="headerlink" href="#gofasta" title="Permanent link">&para;</a></h3>
<p>Some functions for dealing with alignments,  developed to handle SARS-CoV-2 data as part of the COG-UK project.</p>
<p><em>homepage</em>: <a href="https://github.com/cov-ert/gofasta">https://github.com/cov-ert/gofasta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="golf">golf<a class="headerlink" href="#golf" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including  OpenBLAS (BLAS and LAPACK support) and FFTW.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gomkl">gomkl<a class="headerlink" href="#gomkl" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain with OpenMPI and MKL</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gompi">gompi<a class="headerlink" href="#gompi" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain,  including OpenMPI for MPI support.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.04</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.06</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.07</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.08</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.10</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gompic">gompic<a class="headerlink" href="#gompic" title="Permanent link">&para;</a></h3>
<p>GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including OpenMPI for MPI support with CUDA features enabled.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="google-java-format">google-java-format<a class="headerlink" href="#google-java-format" title="Permanent link">&para;</a></h3>
<p>Reformats Java source code to comply with Google Java Style.</p>
<p><em>homepage</em>: <a href="https://github.com/google/google-java-format">https://github.com/google/google-java-format</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="googletest">googletest<a class="headerlink" href="#googletest" title="Permanent link">&para;</a></h3>
<p>Google's framework for writing C++ tests on a variety of platforms</p>
<p><em>homepage</em>: <a href="https://github.com/google/googletest">https://github.com/google/googletest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.0</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gotree">gotree<a class="headerlink" href="#gotree" title="Permanent link">&para;</a></h3>
<p>GoTree is a set of command line tools to manipulate phylogenetic trees.</p>
<p><em>homepage</em>: <a href="https://github.com/evolbioinfo/gotree">https://github.com/evolbioinfo/gotree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gp2c">GP2C<a class="headerlink" href="#gp2c" title="Permanent link">&para;</a></h3>
<p>The gp2c compiler is a package for translating GP routines into the C programming language, so that  they can  be compiled and used with the PARI system or the GP calculator. </p>
<p><em>homepage</em>: <a href="http://pari.math.u-bordeaux.fr/pub/pari/manuals/gp2c/gp2c.html">http://pari.math.u-bordeaux.fr/pub/pari/manuals/gp2c/gp2c.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.9pl5</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="gpaw">GPAW<a class="headerlink" href="#gpaw" title="Permanent link">&para;</a></h3>
<p>GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)  method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or  atom-centered basis-functions.</p>
<p><em>homepage</em>: <a href="https://wiki.fysik.dtu.dk/gpaw/">https://wiki.fysik.dtu.dk/gpaw/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>19.8.1</code></td>
<td><code>-ASE-3.18.0-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>19.8.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>20.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>20.10.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>20.10.0</code></td>
<td><code>-ASE-3.20.1-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>21.1.0</code></td>
<td><code>-ASE-3.21.1</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>21.6.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>21.6.0</code></td>
<td><code>-ASE-3.22.0</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>22.8.0</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gpaw-setups">GPAW-setups<a class="headerlink" href="#gpaw-setups" title="Permanent link">&para;</a></h3>
<p>PAW setup for the GPAW Density Functional Theory package.   Users can install setups manually using 'gpaw install-data' or use setups from this package.   The versions of GPAW and GPAW-setups can be intermixed.</p>
<p><em>homepage</em>: <a href="https://wiki.fysik.dtu.dk/gpaw/">https://wiki.fysik.dtu.dk/gpaw/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.7929</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.9.9672</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.9.11271</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.9.20000</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gperf">gperf<a class="headerlink" href="#gperf" title="Permanent link">&para;</a></h3>
<p>GNU gperf is a perfect hash function generator. For a given list of strings,  it produces a hash function and hash table, in form of C or C++ code, for  looking up a value depending on the input string. The hash function is  perfect, which means that the hash table has no collisions, and the hash  table lookup needs a single string comparison only. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gperf/">https://www.gnu.org/software/gperf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gperftools">gperftools<a class="headerlink" href="#gperftools" title="Permanent link">&para;</a></h3>
<p>gperftools is a collection of a high-performance multi-threaded malloc() implementation, plus some pretty nifty performance analysis tools. Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler. </p>
<p><em>homepage</em>: <a href="https://github.com/gperftools/gperftools">https://github.com/gperftools/gperftools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5</code></td>
<td><code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.7.90</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gpustat">gpustat<a class="headerlink" href="#gpustat" title="Permanent link">&para;</a></h3>
<p>A simple command-line utility for querying and monitoring GPU status</p>
<p><em>homepage</em>: <a href="https://github.com/wookayin/gpustat">https://github.com/wookayin/gpustat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>gcccuda/2020b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.0.0b1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gpy">GPy<a class="headerlink" href="#gpy" title="Permanent link">&para;</a></h3>
<p>GPy is a Gaussian Process (GP) framework written in Python</p>
<p><em>homepage</em>: <a href="https://sheffieldml.github.io/GPy">https://sheffieldml.github.io/GPy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gpyopt">GPyOpt<a class="headerlink" href="#gpyopt" title="Permanent link">&para;</a></h3>
<p>GPyOpt is a Python open-source library for Bayesian Optimization</p>
<p><em>homepage</em>: <a href="https://sheffieldml.github.io/GPyOpt">https://sheffieldml.github.io/GPyOpt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.6</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="gpytorch">GPyTorch<a class="headerlink" href="#gpytorch" title="Permanent link">&para;</a></h3>
<p>GPyTorch is a Gaussian process library implemented using PyTorch.</p>
<p><em>homepage</em>: <a href="https://gpytorch.ai">https://gpytorch.ai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="grace">Grace<a class="headerlink" href="#grace" title="Permanent link">&para;</a></h3>
<p>Grace is a WYSIWYG tool to make two-dimensional plots of numerical data.</p>
<p><em>homepage</em>: <a href="https://plasma-gate.weizmann.ac.il/Grace/">https://plasma-gate.weizmann.ac.il/Grace/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.25</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.1.25</code></td>
<td><code>-5build1</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>intel/2017b</code>, <code>intel/2019a</code>, <code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="gradle">Gradle<a class="headerlink" href="#gradle" title="Permanent link">&para;</a></h3>
<p>Complete Gradle install. From mobile apps to microservices, from small startups to big enterprises, Gradle helps teams build, automate and deliver better software, faster. </p>
<p><em>homepage</em>: <a href="https://gradle.org">https://gradle.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.9.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gradunwarp">gradunwarp<a class="headerlink" href="#gradunwarp" title="Permanent link">&para;</a></h3>
<p>Gradient Unwarping. This is the Human Connectome Project fork of the no longer maintained original.</p>
<p><em>homepage</em>: <a href="https://github.com/Washington-University/gradunwarp">https://github.com/Washington-University/gradunwarp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-HCP-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-HCP-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-HCP-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="graph-tool">graph-tool<a class="headerlink" href="#graph-tool" title="Permanent link">&para;</a></h3>
<p>Graph-tool is an efficient Python module for manipulation and  statistical analysis of graphs (a.k.a. networks). Contrary to  most other python modules with similar functionality, the core  data structures and algorithms are implemented in C++, making  extensive use of template metaprogramming, based heavily on  the Boost Graph Library. This confers it a level of  performance that is comparable (both in memory usage and  computation time) to that of a pure C/C++ library.</p>
<p><em>homepage</em>: <a href="https://graph-tool.skewed.de/">https://graph-tool.skewed.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.26</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.27</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="graphdb">GraphDB<a class="headerlink" href="#graphdb" title="Permanent link">&para;</a></h3>
<p>GraphDB is an enterprise ready Semantic Graph Database, compliant with W3C Standards. Semantic graph databases (also called RDF triplestores) provide the core infrastructure for solutions where modelling agility, data integration, relationship exploration and cross-enterprise data publishing and consumption are important.</p>
<p><em>homepage</em>: <a href="https://graphdb.ontotext.com/">https://graphdb.ontotext.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>10.1.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="graphene">Graphene<a class="headerlink" href="#graphene" title="Permanent link">&para;</a></h3>
<p>Graphene is a a thin layer of types for graphic libraries</p>
<p><em>homepage</em>: <a href="https://ebassi.github.io/graphene/">https://ebassi.github.io/graphene/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.10.8</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="graphicsmagick">GraphicsMagick<a class="headerlink" href="#graphicsmagick" title="Permanent link">&para;</a></h3>
<p>GraphicsMagick is the swiss army knife of image processing.</p>
<p><em>homepage</em>: <a href="https://www.graphicsmagick.org/">https://www.graphicsmagick.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.23</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.25</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.28</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.3.31</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.3.34</code></td>
<td><code>foss/2019a</code>, <code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.3.36</code></td>
<td><code>GCCcore/11.2.0</code>, <code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="graphite2">graphite2<a class="headerlink" href="#graphite2" title="Permanent link">&para;</a></h3>
<p>Graphite is a "smart font" system developed specifically to  handle the complexities of lesser-known languages of the world.</p>
<p><em>homepage</em>: <a href="https://scripts.sil.org/cms/scripts/page.php?site_id=projects&amp;item_id=graphite_home">https://scripts.sil.org/cms/scripts/page.php?site_id=projects&amp;item_id=graphite_home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.14</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="graphlan">GraPhlAn<a class="headerlink" href="#graphlan" title="Permanent link">&para;</a></h3>
<p>GraPhlAn is a software tool for producing high-quality circular representations  of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative,  and publication-ready representations of phylogenetically- and taxonomically-driven investigation. </p>
<p><em>homepage</em>: <a href="https://segatalab.github.io/tools/graphlan">https://segatalab.github.io/tools/graphlan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="graphmap">GraphMap<a class="headerlink" href="#graphmap" title="Permanent link">&para;</a></h3>
<p>A highly sensitive and accurate mapper for long, error-prone reads</p>
<p><em>homepage</em>: <a href="https://github.com/isovic/graphmap">https://github.com/isovic/graphmap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="graphmap2">GraphMap2<a class="headerlink" href="#graphmap2" title="Permanent link">&para;</a></h3>
<p>A highly sensitive and accurate mapper for long, error-prone reads</p>
<p><em>homepage</em>: <a href="https://github.com/lbcb-sci/graphmap2">https://github.com/lbcb-sci/graphmap2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="graphviz">Graphviz<a class="headerlink" href="#graphviz" title="Permanent link">&para;</a></h3>
<p>Graphviz is open source graph visualization software. Graph visualization  is a way of representing structural information as diagrams of  abstract graphs and networks. It has important applications in networking,  bioinformatics,  software engineering, database and web design, machine learning,  and in visual interfaces for other technical domains.</p>
<p><em>homepage</em>: <a href="https://www.graphviz.org/">https://www.graphviz.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.38.0</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.40.1</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.40.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.42.2</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.42.2</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.42.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.44.1</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.44.1</code></td>
<td><code>-Java-11-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.47.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.47.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.50.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="graphviz-python">graphviz-python<a class="headerlink" href="#graphviz-python" title="Permanent link">&para;</a></h3>
<p>Simple Python interface for Graphviz</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/graphviz">https://pypi.python.org/pypi/graphviz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.20.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="grasp">GRASP<a class="headerlink" href="#grasp" title="Permanent link">&para;</a></h3>
<p>The General Relativistic Atomic Structure Package (GRASP) is a set of  Fortran 90 programs for performing fully-relativistic electron structure  calculations of atoms.</p>
<p><em>homepage</em>: <a href="https://compas.github.io/grasp/">https://compas.github.io/grasp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="grass">GRASS<a class="headerlink" href="#grass" title="Permanent link">&para;</a></h3>
<p>The Geographic Resources Analysis Support System - used  for geospatial data management and analysis, image processing,  graphics and maps production, spatial modeling, and visualization</p>
<p><em>homepage</em>: <a href="https://grass.osgeo.org">https://grass.osgeo.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.6.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>7.8.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="greenlet">Greenlet<a class="headerlink" href="#greenlet" title="Permanent link">&para;</a></h3>
<p>The greenlet package is a spin-off of Stackless, a version of CPython that supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single or a few OS-level threads) and are synchronized with data exchanges on "channels". A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs. </p>
<p><em>homepage</em>: <a href="https://github.com/python-greenlet/greenlet">https://github.com/python-greenlet/greenlet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.9</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.4.9</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.4.11</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.4.12</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="grep">Grep<a class="headerlink" href="#grep" title="Permanent link">&para;</a></h3>
<p>The grep command searches one or more input files for lines containing a match to a specified pattern.  By default, grep prints the matching lines.</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/grep/">http://www.gnu.org/software/grep/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.21</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="gretl">gretl<a class="headerlink" href="#gretl" title="Permanent link">&para;</a></h3>
<p>A cross-platform software package for econometric analysis</p>
<p><em>homepage</em>: <a href="http://gretl.sourceforge.net">http://gretl.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020a</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="grib_api">grib_api<a class="headerlink" href="#grib_api" title="Permanent link">&para;</a></h3>
<p>The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python  programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of  command line tools is also provided to give quick access to GRIB messages.</p>
<p><em>homepage</em>: <a href="https://software.ecmwf.int/wiki/display/GRIB/Home">https://software.ecmwf.int/wiki/display/GRIB/Home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.16.0</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.21.0</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.24.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="grid">grid<a class="headerlink" href="#grid" title="Permanent link">&para;</a></h3>
<p>Grid is a free and open-source Python library for numerical integration,  interpolation and differentiation of interest for the quantum chemistry community.</p>
<p><em>homepage</em>: <a href="https://grid.qcdevs.org/">https://grid.qcdevs.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220610</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="grit">GRIT<a class="headerlink" href="#grit" title="Permanent link">&para;</a></h3>
<p>GRIT - A tool for the integrative analysis of RNA-seq type assays</p>
<p><em>homepage</em>: <a href="https://github.com/nboley/grit">https://github.com/nboley/grit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.5</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="grnboost">GRNBoost<a class="headerlink" href="#grnboost" title="Permanent link">&para;</a></h3>
<p>XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.</p>
<p><em>homepage</em>: <a href="https://github.com/dmlc/xgboost">https://github.com/dmlc/xgboost</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20171009</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="groff">groff<a class="headerlink" href="#groff" title="Permanent link">&para;</a></h3>
<p>Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands  and produces formatted output.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/groff">https://www.gnu.org/software/groff</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.22.4</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="groimp">GroIMP<a class="headerlink" href="#groimp" title="Permanent link">&para;</a></h3>
<p>GroIMP (Growth Grammar-related Interactive Modelling Platform) is a 3D-modelling platform.</p>
<p><em>homepage</em>: <a href="http://www.grogra.de/software/groimp">http://www.grogra.de/software/groimp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gromacs">GROMACS<a class="headerlink" href="#gromacs" title="Permanent link">&para;</a></h3>
<p>GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.</p>
<p><em>homepage</em>: <a href="http://www.gromacs.org">http://www.gromacs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.2</code></td>
<td><code>-hybrid</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-hybrid-dp</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-mt</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.1.4</code></td>
<td><code>-hybrid</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.1.4</code></td>
<td><code>-mt</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2016</code></td>
<td><code>-hybrid</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2016</code></td>
<td><code>-mt</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2016.1</code></td>
<td><code>-PLUMED</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2016.2</code></td>
<td></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2016.3</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2016.3</code></td>
<td><code>-GPU-enabled</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2016.4</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>giolf/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2016.5</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2018</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2018.1</code></td>
<td><code>-PLUMED</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2018.2</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2018.3</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2018.4</code></td>
<td><code>-PLUMED-2.5.0</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2019</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2019.2</code></td>
<td></td>
<td><code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>2019.3</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>foss/2019b</code>, <code>fosscuda/2019a</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2019.4</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2019.6</code></td>
<td></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2020</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2020.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2020.3</code></td>
<td></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2020.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2020.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>2021</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2021.2</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2021.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2021.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2021.3</code></td>
<td><code>-CUDA-11.3.1-PLUMED-2.7.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2021.3</code></td>
<td><code>-PLUMED-2.7.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2021.5</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2021.5</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2021.5</code></td>
<td><code>-CUDA-11.4.1-PLUMED-2.8.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2021.5</code></td>
<td><code>-PLUMED-2.8.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gromacswrapper">GromacsWrapper<a class="headerlink" href="#gromacswrapper" title="Permanent link">&para;</a></h3>
<p>GromacsWrapper is a python package that wraps system calls to Gromacs tools into thin classes.  This allows for fairly seamless integration of the gromacs tools into python scripts. </p>
<p><em>homepage</em>: <a href="https://gromacswrapper.readthedocs.org">https://gromacswrapper.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="groovy">Groovy<a class="headerlink" href="#groovy" title="Permanent link">&para;</a></h3>
<p>Groovy is a powerful, optionally typed and dynamic language, with static-typing and static compilation capabilities, for the Java platform aimed at improving developer productivity thanks to a concise, familiar and easy to learn syntax.</p>
<p><em>homepage</em>: <a href="https://groovy-lang.org">https://groovy-lang.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.9</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="grpc">gRPC<a class="headerlink" href="#grpc" title="Permanent link">&para;</a></h3>
<p>gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. gRPC enables client and server applications to communicate transparently, and simplifies the building of connected systems.</p>
<p><em>homepage</em>: <a href="https://grpc.io/">https://grpc.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.44.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsea">GSEA<a class="headerlink" href="#gsea" title="Permanent link">&para;</a></h3>
<p>Gene Set Enrichment Analysis (GSEA) is a computational method that  determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes). </p>
<p><em>homepage</em>: <a href="https://www.gsea-msigdb.org/gsea/index.jsp">https://www.gsea-msigdb.org/gsea/index.jsp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gsettings-desktop-schemas">gsettings-desktop-schemas<a class="headerlink" href="#gsettings-desktop-schemas" title="Permanent link">&para;</a></h3>
<p>gsettings-desktop-schemas contains a collection of GSettings schemas for  settings shared by various components of a desktop.</p>
<p><em>homepage</em>: <a href="https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas">https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.34.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsl">GSL<a class="headerlink" href="#gsl" title="Permanent link">&para;</a></h3>
<p>The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.  The library provides a wide range of mathematical routines such as random number generators, special functions  and least-squares fitting.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/gsl/">https://www.gnu.org/software/gsl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.16</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2018.3.222-GCC-7.3.0-2.30</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.5.281</code>, <code>iccifort/2020.1.217</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/12.2.0</code>, <code>GCCcore/12.2.0</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsoap">gSOAP<a class="headerlink" href="#gsoap" title="Permanent link">&para;</a></h3>
<p>The gSOAP toolkit is a C and C++ software development toolkit for  SOAP and REST XML Web services and generic C/C++ XML data bindings.  The toolkit analyzes WSDLs and XML schemas (separately or as a combined set) and maps the XML schema types  and the SOAP/REST XML messaging protocols to easy-to-use and efficient C and C++ code.  It also supports exposing (legacy) C and C++ applications as XML Web services  by auto-generating XML serialization code and WSDL specifications.  Or you can simply use it to automatically convert XML to/from C and C++ data.  The toolkit supports options to generate pure ANSI C or C++ with or without STL.</p>
<p><em>homepage</em>: <a href="http://www.cs.fsu.edu/~engelen/soap.html">http://www.cs.fsu.edu/~engelen/soap.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.48</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>2.8.100</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsport">gsport<a class="headerlink" href="#gsport" title="Permanent link">&para;</a></h3>
<p>GSPORT command-line tool for accessing GenomeScan Customer Portal</p>
<p><em>homepage</em>: <a href="https://github.com/genomescan/gsport">https://github.com/genomescan/gsport</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gst-plugins-bad">GST-plugins-bad<a class="headerlink" href="#gst-plugins-bad" title="Permanent link">&para;</a></h3>
<p>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.</p>
<p><em>homepage</em>: <a href="https://gstreamer.freedesktop.org/">https://gstreamer.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gst-plugins-base">GST-plugins-base<a class="headerlink" href="#gst-plugins-base" title="Permanent link">&para;</a></h3>
<p>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.</p>
<p><em>homepage</em>: <a href="https://gstreamer.freedesktop.org/">https://gstreamer.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.36</code></td>
<td><code>foss/2016a</code>, <code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.6.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.18.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>1.18.5</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.20.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.22.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gstreamer">GStreamer<a class="headerlink" href="#gstreamer" title="Permanent link">&para;</a></h3>
<p>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.</p>
<p><em>homepage</em>: <a href="https://gstreamer.freedesktop.org/">https://gstreamer.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.36</code></td>
<td><code>foss/2016a</code>, <code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.6.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.18.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>1.18.5</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.20.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.22.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsutil">gsutil<a class="headerlink" href="#gsutil" title="Permanent link">&para;</a></h3>
<p>gsutil is a Python application that lets you access Cloud Storage from the command line.</p>
<p><em>homepage</em>: <a href="https://cloud.google.com/storage/docs/gsutil">https://cloud.google.com/storage/docs/gsutil</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.10</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gsw">gsw<a class="headerlink" href="#gsw" title="Permanent link">&para;</a></h3>
<p>This Python implementation of the Thermodynamic Equation of  Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of the GSW-C  implementation. This library replaces the original python-gsw pure-python  implementation.. The primary reasons for this change are that by building on  the C implementation we reduce code duplication and we gain an immediate update  to the 75-term equation. Additional benefits include a major increase in speed, a reduction in memory  usage, and the inclusion of more functions. The penalty is that a C  (or MSVC C++ for Windows) compiler is required to build the package from source.</p>
<p><em>homepage</em>: <a href="https://github.com/TEOS-10/GSW-python">https://github.com/TEOS-10/GSW-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.16</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="gtdb-tk">GTDB-Tk<a class="headerlink" href="#gtdb-tk" title="Permanent link">&para;</a></h3>
<p>A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/Ecogenomics/GTDBTk">https://github.com/Ecogenomics/GTDBTk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.3.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gtk">GTK+<a class="headerlink" href="#gtk" title="Permanent link">&para;</a></h3>
<p>The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. </p>
<p><em>homepage</em>: <a href="https://developer.gnome.org/gtk+/stable/">https://developer.gnome.org/gtk+/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.24.28</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.24.30</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.24.31</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.24.32</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.24.33</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.22.30</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>3.24.8</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.24.13</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.24.17</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.24.23</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gtk2">GTK2<a class="headerlink" href="#gtk2" title="Permanent link">&para;</a></h3>
<p>The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. </p>
<p><em>homepage</em>: <a href="https://www.gtk.org">https://www.gtk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.24.33</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gtk3">GTK3<a class="headerlink" href="#gtk3" title="Permanent link">&para;</a></h3>
<p>GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. </p>
<p><em>homepage</em>: <a href="https://developer.gnome.org/gtk3/stable/">https://developer.gnome.org/gtk3/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.24.29</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.24.31</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.24.33</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.24.35</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gtk4">GTK4<a class="headerlink" href="#gtk4" title="Permanent link">&para;</a></h3>
<p>GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. </p>
<p><em>homepage</em>: <a href="https://docs.gtk.org/gtk4/">https://docs.gtk.org/gtk4/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gtksourceview">GtkSourceView<a class="headerlink" href="#gtksourceview" title="Permanent link">&para;</a></h3>
<p>GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK+  widget for multiline text editing. GtkSourceView adds support for syntax  highlighting, undo/redo, file loading and saving, search and replace, a  completion system, printing, displaying line numbers, and other features  typical of a source code editor. </p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/GtkSourceView">https://wiki.gnome.org/Projects/GtkSourceView</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.24.11</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gtool">GTOOL<a class="headerlink" href="#gtool" title="Permanent link">&para;</a></h3>
<p>GTOOL is a program for transforming sets of genotype data for use with  the programs SNPTEST and IMPUTE.  </p>
<p><em>homepage</em>: <a href="http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html">http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="gts">GTS<a class="headerlink" href="#gts" title="Permanent link">&para;</a></h3>
<p>GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.</p>
<p><em>homepage</em>: <a href="http://gts.sourceforge.net/">http://gts.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2018b</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>20121130</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="gubbins">gubbins<a class="headerlink" href="#gubbins" title="Permanent link">&para;</a></h3>
<p>Gubbins (Genealogies Unbiased By recomBinations In Nucleotide Sequences) is an algorithm that iteratively identifies loci containing elevated densities of base substitutions while concurrently constructing a phylogeny based on the putative point mutations outside of these regions. Simulations demonstrate the algorithm generates highly accurate reconstructions under realistic models of short-term bacterial evolution, and can be run in only a few hours on alignments of hundreds of bacterial genome sequences. </p>
<p><em>homepage</em>: <a href="https://sanger-pathogens.github.io/gubbins">https://sanger-pathogens.github.io/gubbins</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="guenomu">guenomu<a class="headerlink" href="#guenomu" title="Permanent link">&para;</a></h3>
<p>guenomu is a software written in C that estimates the species tree for a given set of gene families.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/leomrtns/guenomu">https://bitbucket.org/leomrtns/guenomu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.07.05</code></td>
<td><code>-mpi</code></td>
<td><code>iimpi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="guile">Guile<a class="headerlink" href="#guile" title="Permanent link">&para;</a></h3>
<p>Guile is a programming language, designed to help programmers create flexible  applications that can be extended by users or other programmers with plug-ins,  modules, or scripts. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/guile/">https://www.gnu.org/software/guile/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.8</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.11</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.7</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.0.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gulp">GULP<a class="headerlink" href="#gulp" title="Permanent link">&para;</a></h3>
<p>GULP is a program for performing a variety of types of simulation on materials  using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs  and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations</p>
<p><em>homepage</em>: <a href="https://gulp.curtin.edu.au/gulp/">https://gulp.curtin.edu.au/gulp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>6.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gurobi">Gurobi<a class="headerlink" href="#gurobi" title="Permanent link">&para;</a></h3>
<p>The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms.</p>
<p><em>homepage</em>: <a href="https://www.gurobi.com">https://www.gurobi.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.5.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.5.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.5.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.5.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>9.0.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.1.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>9.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>9.5.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.5.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="gushr">GUSHR<a class="headerlink" href="#gushr" title="Permanent link">&para;</a></h3>
<p>Assembly-free construction of UTRs from short read RNA-Seq data on the basis of coding sequence annotation. </p>
<p><em>homepage</em>: <a href="https://github.com/Gaius-Augustus/GUSHR">https://github.com/Gaius-Augustus/GUSHR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020-09-28</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="gzip">gzip<a class="headerlink" href="#gzip" title="Permanent link">&para;</a></h3>
<p>gzip (GNU zip) is a popular data compression program as a replacement for compress</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/gzip/">http://www.gnu.org/software/gzip/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="h">H<a class="headerlink" href="#h" title="Permanent link">&para;</a></h2>
<p><a href="#h4toh5">h4toh5</a> - <a href="#h5hut">H5hut</a> - <a href="#h5py">h5py</a> - <a href="#hadoop">Hadoop</a> - <a href="#hal">HAL</a> - <a href="#hanythingondemand">hanythingondemand</a> - <a href="#hapgen2">HAPGEN2</a> - <a href="#harfbuzz">HarfBuzz</a> - <a href="#harminv">Harminv</a> - <a href="#harmony">harmony</a> - <a href="#hbase">HBase</a> - <a href="#hdbscan">HDBSCAN</a> - <a href="#hddm">HDDM</a> - <a href="#hdf">HDF</a> - <a href="#hdf-eos">HDF-EOS</a> - <a href="#hdf-eos5">HDF-EOS5</a> - <a href="#hdf5">HDF5</a> - <a href="#hdf5storage">hdf5storage</a> - <a href="#hdfview">HDFView</a> - <a href="#healpix">HEALPix</a> - <a href="#health-gps">Health-GPS</a> - <a href="#heaptrack">heaptrack</a> - <a href="#hector">hector</a> - <a href="#heffte">HeFFTe</a> - <a href="#hello">Hello</a> - <a href="#help2man">help2man</a> - <a href="#hepmc3">HepMC3</a> - <a href="#hh-suite">HH-suite</a> - <a href="#hic-pro">HiC-Pro</a> - <a href="#hicexplorer">HiCExplorer</a> - <a href="#hicmatrix">HiCMatrix</a> - <a href="#hierfstat">hierfstat</a> - <a href="#hifiasm">hifiasm</a> - <a href="#highfive">HighFive</a> - <a href="#highway">Highway</a> - <a href="#hip">HIP</a> - <a href="#hipify-clang">hipify-clang</a> - <a href="#hips">HIPS</a> - <a href="#hipsycl">hipSYCL</a> - <a href="#hiredis">hiredis</a> - <a href="#hisat2">HISAT2</a> - <a href="#histolab">histolab</a> - <a href="#hivtrace">hivtrace</a> - <a href="#hl7apy">hl7apy</a> - <a href="#hlaminer">HLAminer</a> - <a href="#hmmer">HMMER</a> - <a href="#hmmer2">HMMER2</a> - <a href="#hmmlearn">hmmlearn</a> - <a href="#home">HOME</a> - <a href="#horovod">Horovod</a> - <a href="#horton">horton</a> - <a href="#how_are_we_stranded_here">how_are_we_stranded_here</a> - <a href="#hpcg">HPCG</a> - <a href="#hpcx">HPCX</a> - <a href="#hpdbscan">HPDBSCAN</a> - <a href="#hpl">HPL</a> - <a href="#htop">htop</a> - <a href="#htseq">HTSeq</a> - <a href="#htslib">HTSlib</a> - <a href="#hub">hub</a> - <a href="#humann">humann</a> - <a href="#hunspell">hunspell</a> - <a href="#hwloc">hwloc</a> - <a href="#hydra">Hydra</a> - <a href="#hyperopt">Hyperopt</a> - <a href="#hyperqueue">HyperQueue</a> - <a href="#hyperspy">hyperspy</a> - <a href="#hyphy">HyPhy</a> - <a href="#hypo">HyPo</a> - <a href="#hypothesis">hypothesis</a> - <a href="#hypre">Hypre</a></p>
<h3 id="h4toh5">h4toh5<a class="headerlink" href="#h4toh5" title="Permanent link">&para;</a></h3>
<p>The h4toh5 software consists of the h4toh5 and h5toh4 command-line utilities,  as well as a conversion library for converting between individual HDF4 and HDF5 objects.</p>
<p><em>homepage</em>: <a href="http://www.hdfgroup.org/h4toh5/">http://www.hdfgroup.org/h4toh5/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td><code>-linux-x86_64-static</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code>, <code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="h5hut">H5hut<a class="headerlink" href="#h5hut" title="Permanent link">&para;</a></h3>
<p>HDF5 Utility Toolkit: High-Performance I/O Library for Particle-based Simulations</p>
<p><em>homepage</em>: <a href="https://amas.psi.ch/H5hut/">https://amas.psi.ch/H5hut/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.99.13</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="h5py">h5py<a class="headerlink" href="#h5py" title="Permanent link">&para;</a></h3>
<p>HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,  version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous  amounts of data.</p>
<p><em>homepage</em>: <a href="https://www.h5py.org/">https://www.h5py.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-2.7.11-HDF5-1.8.16</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-2.7.11-HDF5-1.8.16-serial</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-3.5.1-HDF5-1.8.16</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.12-HDF5-1.10.0-patch1</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.12-HDF5-1.8.17</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-2.7.12-HDF5-1.8.18</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.5.2-HDF5-1.10.0-patch1</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.5.2-HDF5-1.8.17</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.5.2-HDF5-1.8.18</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-2.7.13-HDF5-1.10.1</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-2.7.13-HDF5-1.8.19</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-3.6.1-HDF5-1.10.0-patch1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-3.6.1-HDF5-1.10.1</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-3.6.1-HDF5-1.8.19</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.14-serial</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-Python-2.7.15-serial</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code>, <code>intelcuda/2019a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code>, <code>intelcuda/2020a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-serial-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>gomkl/2021a</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="hadoop">Hadoop<a class="headerlink" href="#hadoop" title="Permanent link">&para;</a></h3>
<p>Hadoop MapReduce by Cloudera</p>
<p><em>homepage</em>: <a href="https://archive.cloudera.com/cdh5/cdh/5/">https://archive.cloudera.com/cdh5/cdh/5/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>-seagate-722af1-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.0-cdh5.3.1</code></td>
<td><code>-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0-cdh5.4.5</code></td>
<td><code>-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0-cdh5.7.0</code></td>
<td><code>-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0-cdh5.8.0</code></td>
<td><code>-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0-cdh5.12.0</code></td>
<td><code>-native</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.2</code></td>
<td><code>-native</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-native</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-native-Java-1.8</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hal">HAL<a class="headerlink" href="#hal" title="Permanent link">&para;</a></h3>
<p>HAL is a structure to efficiently store and index multiple genome alignments and ancestral reconstructions. HAL is a graph-based representation which provides several advantages over matrix/block-based formats such as MAF, such as improved scalability and the ability to perform queries with respect to an arbitrary reference or subtree.  This package includes the HAL API and several analysis and conversion tools which are described below. HAL files are presently stored in either HDF5 or mmap format, but we note that the tools and most of the API are format-independent, so other databases could be implemented in the future. </p>
<p><em>homepage</em>: <a href="https://github.com/ComparativeGenomicsToolkit/hal">https://github.com/ComparativeGenomicsToolkit/hal</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="hanythingondemand">hanythingondemand<a class="headerlink" href="#hanythingondemand" title="Permanent link">&para;</a></h3>
<p>HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster. It uses the Torque resource manager to do node allocation.</p>
<p><em>homepage</em>: <a href="https://github.com/hpcugent/hanythingondemand">https://github.com/hpcugent/hanythingondemand</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>-cli</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="hapgen2">HAPGEN2<a class="headerlink" href="#hapgen2" title="Permanent link">&para;</a></h3>
<p>'HAPGEN2' simulates case control datasets at SNP markers.</p>
<p><em>homepage</em>: <a href="https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html">https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="harfbuzz">HarfBuzz<a class="headerlink" href="#harfbuzz" title="Permanent link">&para;</a></h3>
<p>HarfBuzz is an OpenType text shaping engine.</p>
<p><em>homepage</em>: <a href="https://www.freedesktop.org/wiki/Software/HarfBuzz">https://www.freedesktop.org/wiki/Software/HarfBuzz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.7</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.7.5</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.6.7</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.3.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="harminv">Harminv<a class="headerlink" href="#harminv" title="Permanent link">&para;</a></h3>
<p>Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -  given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially  decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those  sinusoids.</p>
<p><em>homepage</em>: <a href="https://github.com/stevengj/harminv">https://github.com/stevengj/harminv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2018a</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="harmony">harmony<a class="headerlink" href="#harmony" title="Permanent link">&para;</a></h3>
<p>Harmony is a general-purpose R package with an efficient algorithm for integrating multiple data sets.</p>
<p><em>homepage</em>: <a href="https://portals.broadinstitute.org/harmony">https://portals.broadinstitute.org/harmony</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0-20200224</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="hbase">HBase<a class="headerlink" href="#hbase" title="Permanent link">&para;</a></h3>
<p>Apache HBase. is the Hadoop database, a distributed, scalable, big data store. </p>
<p><em>homepage</em>: <a href="http://hbase.apache.org/">http://hbase.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="hdbscan">HDBSCAN<a class="headerlink" href="#hdbscan" title="Permanent link">&para;</a></h3>
<p>The hdbscan library is a suite of tools to use unsupervised learning to find clusters, or dense  regions, of a dataset. The primary algorithm is HDBSCAN* as proposed by Campello, Moulavi, and Sander. The library  provides a high performance implementation of this algorithm, along with tools for analysing the resulting  clustering.</p>
<p><em>homepage</em>: <a href="http://hdbscan.readthedocs.io/en/latest/">http://hdbscan.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.24</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8.27</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.29</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="hddm">HDDM<a class="headerlink" href="#hddm" title="Permanent link">&para;</a></h3>
<p>HDDM is a Python toolbox for hierarchical Bayesian parameter estimation  of the Drift Diffusion Model (via PyMC).</p>
<p><em>homepage</em>: <a href="http://ski.clps.brown.edu/hddm_docs">http://ski.clps.brown.edu/hddm_docs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.7.5</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.7.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="hdf">HDF<a class="headerlink" href="#hdf" title="Permanent link">&para;</a></h3>
<p>HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines. </p>
<p><em>homepage</em>: <a href="https://www.hdfgroup.org/products/hdf4/">https://www.hdfgroup.org/products/hdf4/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.11</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.2.12</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.2.13</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>4.2.13</code></td>
<td><code>-no-netcdf</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.2.14</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.2.15</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hdf-eos">HDF-EOS<a class="headerlink" href="#hdf-eos" title="Permanent link">&para;</a></h3>
<p>HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.</p>
<p><em>homepage</em>: <a href="https://hdfeos.org/">https://hdfeos.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.20</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hdf-eos5">HDF-EOS5<a class="headerlink" href="#hdf-eos5" title="Permanent link">&para;</a></h3>
<p>HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.</p>
<p><em>homepage</em>: <a href="https://hdfeos.org/">https://hdfeos.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.16</code></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code>, <code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="hdf5">HDF5<a class="headerlink" href="#hdf5" title="Permanent link">&para;</a></h3>
<p>HDF5 is a data model, library, and file format for storing and managing data.  It supports an unlimited variety of datatypes, and is designed for flexible  and efficient I/O and for high volume and complex data.</p>
<p><em>homepage</em>: <a href="https://portal.hdfgroup.org/display/support">https://portal.hdfgroup.org/display/support</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.10</code></td>
<td><code>-serial</code></td>
<td><code>GCC/4.8.1</code></td>
</tr>
<tr>
<td><code>1.8.11</code></td>
<td><code>-serial</code></td>
<td><code>GCC/4.8.1</code></td>
</tr>
<tr>
<td><code>1.8.12</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.13</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.8.16</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.8.16</code></td>
<td><code>-serial</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.8.17</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.17</code></td>
<td><code>-serial</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.18</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code>, <code>intel/2017.01</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.8.18</code></td>
<td><code>-serial</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.8.19</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.20</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>gmpolf/2017.10</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.10.0-patch1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017.01</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018.00</code>, <code>intel/2018.01</code>, <code>intel/2018a</code>, <code>intelcuda/2017b</code>, <code>iomkl/2017b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td></td>
<td><code>PGI/18.4-GCC-6.4.0-2.28</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>1.10.5</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompic/2019a</code>, <code>gompic/2019b</code>, <code>iimpi/2019a</code>, <code>iimpi/2019b</code>, <code>iimpic/2019a</code>, <code>iimpic/2019b</code>, <code>iompi/2019b</code></td>
</tr>
<tr>
<td><code>1.10.5</code></td>
<td><code>-serial</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.10.6</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>gompic/2020a</code>, <code>iimpi/2020a</code>, <code>iimpic/2020a</code></td>
</tr>
<tr>
<td><code>1.10.7</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code>, <code>iimpic/2020b</code></td>
</tr>
<tr>
<td><code>1.10.8</code></td>
<td></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>1.12.2</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
<tr>
<td><code>1.12.2</code></td>
<td><code>-serial</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-serial</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.14.0</code></td>
<td></td>
<td><code>gompi/2022b</code>, <code>iimpi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="hdf5storage">hdf5storage<a class="headerlink" href="#hdf5storage" title="Permanent link">&para;</a></h3>
<p>This Python package provides high level utilities to read/write a variety of Python types to/from  HDF5 (Heirarchal Data Format) formatted files. This package also provides support for MATLAB MAT v7.3 formatted  files, which are just HDF5 files with a different extension and some extra meta-data. All of this is done without  pickling data. Pickling is bad for security because it allows arbitrary code to be executed in the interpreter.  One wants to be able to read possibly HDF5 and MAT files from untrusted sources, so pickling is avoided in this  package.</p>
<p><em>homepage</em>: <a href="https://pythonhosted.org/hdf5storage/">https://pythonhosted.org/hdf5storage/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.15</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="hdfview">HDFView<a class="headerlink" href="#hdfview" title="Permanent link">&para;</a></h3>
<p>HDFView is a visual tool for browsing and editing HDF4 and HDF5 files.</p>
<p><em>homepage</em>: <a href="https://support.hdfgroup.org/products/java/hdfview/">https://support.hdfgroup.org/products/java/hdfview/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.14</code></td>
<td><code>-Java-1.8.0_152-centos6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td><code>-Java-1.8.0_152-centos7</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="healpix">HEALPix<a class="headerlink" href="#healpix" title="Permanent link">&para;</a></h3>
<p>Hierarchical Equal Area isoLatitude Pixelation of a sphere.</p>
<p><em>homepage</em>: <a href="http://healpix.sourceforge.net/">http://healpix.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.50</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="health-gps">Health-GPS<a class="headerlink" href="#health-gps" title="Permanent link">&para;</a></h3>
<p>Health-GPS microsimulation is part of the STOP project, and supports researchers  and policy makers in the analysis of the health and economic impacts of alternative  measures to tackle chronic diseases and obesity in children. The model reproduces  the characteristics of a population and simulates key individual event histories  associated with key components of relevant behaviours, such as physical activity,  and diseases such as diabetes or cancer. To run the test-jobs with HealthGPS.Tests the data-directory, found in your installation folder, must be in the current path.  </p>
<p><em>homepage</em>: <a href="https://github.com/imperialCHEPI/healthgps">https://github.com/imperialCHEPI/healthgps</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.2.2.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="heaptrack">heaptrack<a class="headerlink" href="#heaptrack" title="Permanent link">&para;</a></h3>
<p>A heap memory profiler for Linux.</p>
<p><em>homepage</em>: <a href="http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux">http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="hector">hector<a class="headerlink" href="#hector" title="Permanent link">&para;</a></h3>
<p>This is the repository for Hector, an open source, object-oriented,  simple global climate carbon-cycle model. It runs essentially instantaneously while  still representing the most critical global scale earth system processes, and is  one of a class of models heavily used for for emulating complex climate models  and uncertainty analyses.</p>
<p><em>homepage</em>: <a href="https://github.com/JGCRI/hector">https://github.com/JGCRI/hector</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="heffte">HeFFTe<a class="headerlink" href="#heffte" title="Permanent link">&para;</a></h3>
<p>Highly Efficient FFT for Exascale (HeFFTe) library</p>
<p><em>homepage</em>: <a href="https://icl.utk.edu/fft">https://icl.utk.edu/fft</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="hello">Hello<a class="headerlink" href="#hello" title="Permanent link">&para;</a></h3>
<p>The GNU Hello program produces a familiar, friendly greeting. Yes, this is another implementation of the classic program that prints "Hello, world!" when you run it.  However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior, supports greetings in many languages, and so on. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/hello/">https://www.gnu.org/software/hello/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="help2man">help2man<a class="headerlink" href="#help2man" title="Permanent link">&para;</a></h3>
<p>help2man produces simple manual pages from the '--help' and '--version' output of other commands.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/help2man/">https://www.gnu.org/software/help2man/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.47.4</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.1.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/system</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.47.5</code></td>
<td><code>GCCcore/5.5.0</code></td>
</tr>
<tr>
<td><code>1.47.6</code></td>
<td><code>GCCcore/8.1.0</code></td>
</tr>
<tr>
<td><code>1.47.7</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.47.8</code></td>
<td><code>GCCcore/7.4.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/8.4.0</code></td>
</tr>
<tr>
<td><code>1.47.10</code></td>
<td><code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code></td>
</tr>
<tr>
<td><code>1.47.12</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.47.15</code></td>
<td><code>GCCcore/10.1.0</code></td>
</tr>
<tr>
<td><code>1.47.16</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.48.3</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.4.0</code></td>
</tr>
<tr>
<td><code>1.49.2</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.5.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hepmc3">HepMC3<a class="headerlink" href="#hepmc3" title="Permanent link">&para;</a></h3>
<p>HepMC is a standard for storing Monte Carlo event data.</p>
<p><em>homepage</em>: <a href="http://hepmc.web.cern.ch/hepmc/">http://hepmc.web.cern.ch/hepmc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.5</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hh-suite">HH-suite<a class="headerlink" href="#hh-suite" title="Permanent link">&para;</a></h3>
<p>The HH-suite is an open-source software package  for sensitive protein sequence searching based on the pairwise  alignment of hidden Markov models (HMMs).</p>
<p><em>homepage</em>: <a href="https://github.com/soedinglab/hh-suite">https://github.com/soedinglab/hh-suite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0-beta.3</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompic/2020b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>gompic/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="hic-pro">HiC-Pro<a class="headerlink" href="#hic-pro" title="Permanent link">&para;</a></h3>
<p>HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data)   to the normalized contact maps.</p>
<p><em>homepage</em>: <a href="http://nservant.github.io/HiC-Pro">http://nservant.github.io/HiC-Pro</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="hicexplorer">HiCExplorer<a class="headerlink" href="#hicexplorer" title="Permanent link">&para;</a></h3>
<p>HiCexplorer addresses the common tasks of Hi-C analysis from processing to visualization.</p>
<p><em>homepage</em>: <a href="http://hicexplorer.readthedocs.org/">http://hicexplorer.readthedocs.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="hicmatrix">HiCMatrix<a class="headerlink" href="#hicmatrix" title="Permanent link">&para;</a></h3>
<p>This library implements the central class of HiCExplorer to manage Hi-C interaction matrices.</p>
<p><em>homepage</em>: <a href="https://github.com/deeptools/HiCMatrix">https://github.com/deeptools/HiCMatrix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="hierfstat">hierfstat<a class="headerlink" href="#hierfstat" title="Permanent link">&para;</a></h3>
<p>Estimates hierarchical F-statistics from haploid or diploid genetic data with any numbers of levels in the hierarchy.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=hierfstat">https://cran.r-project.org/package=hierfstat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5-7</code></td>
<td><code>-R-4.0.0-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="hifiasm">hifiasm<a class="headerlink" href="#hifiasm" title="Permanent link">&para;</a></h3>
<p>Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.</p>
<p><em>homepage</em>: <a href="https://github.com/chhylp123/hifiasm">https://github.com/chhylp123/hifiasm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.16.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="highfive">HighFive<a class="headerlink" href="#highfive" title="Permanent link">&para;</a></h3>
<p>HighFive is a modern header-only C++11 friendly interface for libhdf5.</p>
<p><em>homepage</em>: <a href="https://github.com/BlueBrain/HighFive">https://github.com/BlueBrain/HighFive</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>gompi/2021a</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="highway">Highway<a class="headerlink" href="#highway" title="Permanent link">&para;</a></h3>
<p>Highway is a C++ library for SIMD (Single Instruction, Multiple Data), i.e. applying the same operation to 'lanes'.</p>
<p><em>homepage</em>: <a href="https://github.com/google/highway">https://github.com/google/highway</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hip">HIP<a class="headerlink" href="#hip" title="Permanent link">&para;</a></h3>
<p>HIP is a C++ Runtime API and Kernel Language that allows developers to create portable applications for AMD and NVIDIA GPUs from single source code.</p>
<p><em>homepage</em>: <a href="https://github.com/ROCm-Developer-Tools/HIP">https://github.com/ROCm-Developer-Tools/HIP</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>-amd</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hipify-clang">hipify-clang<a class="headerlink" href="#hipify-clang" title="Permanent link">&para;</a></h3>
<p>Hipify-clang is a clang-based tool for translating CUDA   sources into HIP sources. It translates CUDA source into an abstract syntax   tree, which is traversed by transformation matchers. After applying all the   matchers, the output HIP source is produced.</p>
<p><em>homepage</em>: <a href="https://github.com/ROCm-Developer-Tools/HIPIFY">https://github.com/ROCm-Developer-Tools/HIPIFY</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.0</code></td>
<td><code>gcccuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="hips">HIPS<a class="headerlink" href="#hips" title="Permanent link">&para;</a></h3>
<p>HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides  an efficient parallel iterative solver for very large sparse linear systems. </p>
<p><em>homepage</em>: <a href="http://hips.gforge.inria.fr/">http://hips.gforge.inria.fr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2b-rc5</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="hipsycl">hipSYCL<a class="headerlink" href="#hipsycl" title="Permanent link">&para;</a></h3>
<p>hipSYCL is a modern SYCL implementation targeting CPUs and GPUs, with a focus on leveraging existing toolchains such as CUDA or HIP</p>
<p><em>homepage</em>: <a href="https://github.com/illuhad/hipSYCL">https://github.com/illuhad/hipSYCL</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>GCC/10.2.0</code>, <code>gcccuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="hiredis">hiredis<a class="headerlink" href="#hiredis" title="Permanent link">&para;</a></h3>
<p>Hiredis is a minimalistic C client library for the Redis database.  It is minimalistic because it just adds minimal support for the protocol, but at the same time it uses a high level printf-alike API in order to make it much higher level than otherwise suggested by its minimal code base and the lack of explicit bindings for every Redis command.</p>
<p><em>homepage</em>: <a href="https://github.com/redis/hiredis">https://github.com/redis/hiredis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hisat2">HISAT2<a class="headerlink" href="#hisat2" title="Permanent link">&para;</a></h3>
<p>HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads  (both DNA and RNA) against the general human population (as well as against a single reference genome).</p>
<p><em>homepage</em>: <a href="https://daehwankimlab.github.io/hisat2">https://daehwankimlab.github.io/hisat2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3-beta</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="histolab">histolab<a class="headerlink" href="#histolab" title="Permanent link">&para;</a></h3>
<p>Library for Digital Pathology Image Processing</p>
<p><em>homepage</em>: <a href="https://github.com/histolab/histolab">https://github.com/histolab/histolab</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="hivtrace">hivtrace<a class="headerlink" href="#hivtrace" title="Permanent link">&para;</a></h3>
<p>HIV-TRACE is an application that identifies potential transmission clusters within a  supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</p>
<p><em>homepage</em>: <a href="https://github.com/veg/hivtrace">https://github.com/veg/hivtrace</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="hl7apy">hl7apy<a class="headerlink" href="#hl7apy" title="Permanent link">&para;</a></h3>
<p>Python library to parse, create and handle HL7 v2 messages.</p>
<p><em>homepage</em>: <a href="https://github.com/crs4/hl7apy">https://github.com/crs4/hl7apy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hlaminer">HLAminer<a class="headerlink" href="#hlaminer" title="Permanent link">&para;</a></h3>
<p>HLAminer is a software for HLA predictions from next-generation shotgun (NGS) sequence read data and supports direct  read alignment and targeted de novo assembly of sequence reads. </p>
<p><em>homepage</em>: <a href="http://www.bcgsc.ca/platform/bioinfo/software/hlaminer">http://www.bcgsc.ca/platform/bioinfo/software/hlaminer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="hmmer">HMMER<a class="headerlink" href="#hmmer" title="Permanent link">&para;</a></h3>
<p>HMMER is used for searching sequence databases for homologs  of protein sequences, and for making protein sequence alignments. It  implements methods using probabilistic models called profile hidden Markov  models (profile HMMs).  Compared to BLAST, FASTA, and other sequence  alignment and database search tools based on older scoring methodology,  HMMER aims to be significantly more accurate and more able to detect remote  homologs because of the strength of its underlying mathematical models. In the  past, this strength came at significant computational expense, but in the new  HMMER3 project, HMMER is now essentially as fast as BLAST.</p>
<p><em>homepage</em>: <a href="http://hmmer.org/">http://hmmer.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1b2</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2018a</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>gompi/2019b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iimpi/2019b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code>, <code>iimpi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="hmmer2">HMMER2<a class="headerlink" href="#hmmer2" title="Permanent link">&para;</a></h3>
<p>HMMER is used for searching sequence databases for sequence homologs,  and for making sequence alignments.</p>
<p><em>homepage</em>: <a href="http://hmmer.org">http://hmmer.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.2</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hmmlearn">hmmlearn<a class="headerlink" href="#hmmlearn" title="Permanent link">&para;</a></h3>
<p>hmmlearn is a set of algorithms for unsupervised learning and inference of Hidden Markov Models</p>
<p><em>homepage</em>: <a href="https://github.com/hmmlearn/hmmlearn">https://github.com/hmmlearn/hmmlearn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="home">HOME<a class="headerlink" href="#home" title="Permanent link">&para;</a></h3>
<p>HOME (histogram of methylation) is a python package for differential methylation region (DMR) identification. The method uses histogram of methylation features and the linear Support Vector Machine (SVM) to identify DMRs  from whole genome bisulfite sequencing (WGBS) data.</p>
<p><em>homepage</em>: <a href="https://github.com/ListerLab/HOME">https://github.com/ListerLab/HOME</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="horovod">Horovod<a class="headerlink" href="#horovod" title="Permanent link">&para;</a></h3>
<p>Horovod is a distributed training framework for TensorFlow.</p>
<p><em>homepage</em>: <a href="https://github.com/uber/horovod">https://github.com/uber/horovod</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.10</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.18.2</code></td>
<td><code>-TensorFlow-1.15.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.18.2</code></td>
<td><code>-TensorFlow-1.15.2-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.18.2</code></td>
<td><code>-TensorFlow-2.0.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-TensorFlow-2.1.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.19.5</code></td>
<td><code>-TensorFlow-2.2.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.20.3</code></td>
<td><code>-TensorFlow-2.3.1-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.21.1</code></td>
<td><code>-PyTorch-1.7.1-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.21.1</code></td>
<td><code>-TensorFlow-2.4.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.21.1</code></td>
<td><code>-TensorFlow-2.4.1-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.21.3</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.21.3</code></td>
<td><code>-TensorFlow-2.3.1-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>0.22.0</code></td>
<td><code>-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.22.1</code></td>
<td><code>-CUDA-11.3.1-TensorFlow-2.6.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.22.1</code></td>
<td><code>-TensorFlow-2.5.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.22.1</code></td>
<td><code>-TensorFlow-2.5.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.23.0</code></td>
<td><code>-CUDA-11.3.1-PyTorch-1.10.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.23.0</code></td>
<td><code>-TensorFlow-2.5.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="horton">horton<a class="headerlink" href="#horton" title="Permanent link">&para;</a></h3>
<p>HORTON is a Helpful Open-source Research TOol for N-fermion systems, written  primarily in the Python programming language. (HORTON is named after the helpful  pachyderm, not the Canadian caffeine supply store.) The ultimate goal of HORTON  is to provide a platform for testing new ideas on the quantum many-body problem  at a reasonable computational cost. Although HORTON is primarily designed to be  a quantum-chemistry program, it can perform computations involving model  Hamiltonians, and could be extended for computations in nuclear physics.</p>
<p><em>homepage</em>: <a href="https://theochem.github.io/horton">https://theochem.github.io/horton</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="how_are_we_stranded_here">how_are_we_stranded_here<a class="headerlink" href="#how_are_we_stranded_here" title="Permanent link">&para;</a></h3>
<p>Python package for testing strandedness of RNA-Seq fastq files</p>
<p><em>homepage</em>: <a href="https://github.com/signalbash/how_are_we_stranded_here">https://github.com/signalbash/how_are_we_stranded_here</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="hpcg">HPCG<a class="headerlink" href="#hpcg" title="Permanent link">&para;</a></h3>
<p>The HPCG Benchmark project is an effort to create a more relevant metric for ranking HPC systems than  the High Performance LINPACK (HPL) benchmark, that is currently used by the TOP500 benchmark.</p>
<p><em>homepage</em>: <a href="https://software.sandia.gov/hpcg">https://software.sandia.gov/hpcg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.1</code></td>
<td><code>foss/2018b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2018b</code>, <code>intel/2021a</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="hpcx">HPCX<a class="headerlink" href="#hpcx" title="Permanent link">&para;</a></h3>
<p>The Mellanox HPC-X Toolkit is a comprehensive MPI and SHMEM/PGAS software suite for high performance computing environments</p>
<p><em>homepage</em>: <a href="http://www.mellanox.com/page/products_dyn?product_family=189&amp;mtag=hpc-x">http://www.mellanox.com/page/products_dyn?product_family=189&amp;mtag=hpc-x</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="hpdbscan">HPDBSCAN<a class="headerlink" href="#hpdbscan" title="Permanent link">&para;</a></h3>
<p>Highly parallel density based spatial clustering for application with noise</p>
<p><em>homepage</em>: <a href="https://github.com/Markus-Goetz/hpdbscan">https://github.com/Markus-Goetz/hpdbscan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20171110</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>20210826</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="hpl">HPL<a class="headerlink" href="#hpl" title="Permanent link">&para;</a></h3>
<p>HPL is a software package that solves a (random) dense linear system in double precision (64 bits)  arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available  implementation of the High Performance Computing Linpack Benchmark.</p>
<p><em>homepage</em>: <a href="https://www.netlib.org/benchmark/hpl/">https://www.netlib.org/benchmark/hpl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2016.04</code>, <code>foss/2016.06</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>gmpolf/2016a</code>, <code>gmvolf/1.7.20</code>, <code>gmvolf/2016a</code>, <code>intel/2016.00</code>, <code>intel/2016.01</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016.02-GCC-5.3</code>, <code>intel/2016.03-GCC-4.9</code>, <code>intel/2016.03-GCC-5.3</code>, <code>intel/2016.03-GCC-5.4</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>pomkl/2016.03</code>, <code>pomkl/2016.04</code>, <code>pomkl/2016.09</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>foss/2016.07</code>, <code>foss/2016.09</code>, <code>foss/2017a</code>, <code>foss/2017b</code>, <code>foss/2018.08</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>fosscuda/2018b</code>, <code>gimkl/2018b</code>, <code>giolf/2017b</code>, <code>giolf/2018a</code>, <code>giolfc/2017b</code>, <code>gmpolf/2017.10</code>, <code>goblf/2018b</code>, <code>gomkl/2018b</code>, <code>intel/2017.00</code>, <code>intel/2017.01</code>, <code>intel/2017.02</code>, <code>intel/2017.09</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018.00</code>, <code>intel/2018.01</code>, <code>intel/2018.02</code>, <code>intel/2018.04</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2019.00</code>, <code>intel/2019.01</code>, <code>intelcuda/2016.10</code>, <code>intelcuda/2017b</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>iomkl/2016.09-GCC-5.4.0-2.26</code>, <code>iomkl/2017.01</code>, <code>iomkl/2017a</code>, <code>iomkl/2017b</code>, <code>iomkl/2018.02</code>, <code>iomkl/2018a</code>, <code>iomkl/2018b</code>, <code>pomkl/2016.09</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>CrayCCE/19.06</code>, <code>CrayGNU/19.06</code>, <code>CrayIntel/19.06</code>, <code>Fujitsu/21.05</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022.05</code>, <code>foss/2022.10</code>, <code>foss/2022a</code>, <code>foss/2022b</code>, <code>fosscuda/2019b</code>, <code>fosscuda/2020a</code>, <code>gobff/2020.06-amd</code>, <code>gobff/2020.11</code>, <code>gobff/2020b</code>, <code>gobff/2021a</code>, <code>goblf/2020b</code>, <code>gomkl/2019a</code>, <code>gomkl/2020b</code>, <code>gomkl/2021a</code>, <code>gomkl/2022a</code>, <code>iibff/2020b</code>, <code>intel/2019.02</code>, <code>intel/2019.03</code>, <code>intel/2019a</code>, <code>intel/2019b</code>, <code>intel/2020.00</code>, <code>intel/2020.06-impi-18.5</code>, <code>intel/2020.12</code>, <code>intel/2020a</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code>, <code>intel/2022.00</code>, <code>intel/2022.09</code>, <code>intel/2022.11</code>, <code>intel/2022.12</code>, <code>intel/2022b</code>, <code>intelcuda/2019b</code>, <code>intelcuda/2020a</code>, <code>iomkl/2019.01</code>, <code>iomkl/2021a</code>, <code>iomkl/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="htop">htop<a class="headerlink" href="#htop" title="Permanent link">&para;</a></h3>
<p>An interactive process viewer for Unix</p>
<p><em>homepage</em>: <a href="https://htop.dev">https://htop.dev</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="htseq">HTSeq<a class="headerlink" href="#htseq" title="Permanent link">&para;</a></h3>
<p>HTSeq is a Python library to facilitate processing and analysis  of data from high-throughput sequencing (HTS) experiments.</p>
<p><em>homepage</em>: <a href="https://github.com/simon-anders/htseq">https://github.com/simon-anders/htseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1p1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.6.1p1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.11.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="htslib">HTSlib<a class="headerlink" href="#htslib" title="Permanent link">&para;</a></h3>
<p>PacBio fork of C library for reading/writing high-throughput sequencing data.  This package includes the utilities bgzip and tabix</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/htslib">https://github.com/PacificBiosciences/htslib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
<tr>
<td><code>20160107</code></td>
<td><code>-PacBio</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="hub">hub<a class="headerlink" href="#hub" title="Permanent link">&para;</a></h3>
<p>hub is a command-line wrapper for git that makes you better at GitHub.</p>
<p><em>homepage</em>: <a href="https://hub.github.com/">https://hub.github.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td><code>-linux-amd64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="humann">humann<a class="headerlink" href="#humann" title="Permanent link">&para;</a></h3>
<p>HUMAnN v3 is a pipeline for efficiently  and accurately determining the coverage and abundance  of microbial pathways in a community from metagenomic data. Sequencing a metagenome typically produces millions  of short DNA/RNA reads. This process, referred  to as functional profiling, aims to describe the metabolic potential  of a microbial community and its members. More generally,  functional profiling answers the question:  What are the microbes in my community-of-interest doing (or capable of doing)?</p>
<p><em>homepage</em>: <a href="http://huttenhower.sph.harvard.edu/humann">http://huttenhower.sph.harvard.edu/humann</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="hunspell">hunspell<a class="headerlink" href="#hunspell" title="Permanent link">&para;</a></h3>
<p>Hunspell is a spell checker and morphological analyzer library and program designed for languageswith rich morphology and complex word compounding or character encoding.</p>
<p><em>homepage</em>: <a href="https://hunspell.github.io/">https://hunspell.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hwloc">hwloc<a class="headerlink" href="#hwloc" title="Permanent link">&para;</a></h3>
<p>The Portable Hardware Locality (hwloc) software package provides a portable  abstraction (across OS, versions, architectures, ...) of the hierarchical  topology of modern architectures, including NUMA memory nodes, sockets, shared  caches, cores and simultaneous multithreading. It also gathers various system  attributes such as cache and memory information as well as the locality of I/O  devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily  aims at helping applications with gathering information about modern computing  hardware so as to exploit it accordingly and efficiently. </p>
<p><em>homepage</em>: <a href="https://www.open-mpi.org/projects/hwloc/">https://www.open-mpi.org/projects/hwloc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>GCC/4.8.4</code>, <code>GNU/4.9.2-2.25</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.11.1</code></td>
<td><code>GCC/4.9.3</code></td>
</tr>
<tr>
<td><code>1.11.2</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.11.3</code></td>
<td><code>GCC/5.2.0</code>, <code>GCC/5.3.0-2.26</code>, <code>GCC/5.4.0-2.26</code>, <code>GCC/6.1.0-2.27</code>, <code>PGI/16.3-GCC-4.9.3-2.25</code>, <code>PGI/16.4-GCC-5.3.0-2.26</code>, <code>iccifort/2016.3.210-GCC-4.9.3-2.25</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.11.4</code></td>
<td><code>GCC/6.2.0-2.27</code>, <code>PGI/16.7-GCC-5.4.0-2.26</code>, <code>iccifort/2017.1.132-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>1.11.5</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCC/6.3.0-2.27</code>, <code>iccifort/2017.1.132-GCC-6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>1.11.6</code></td>
<td><code>GCC/6.3.0-2.28</code></td>
</tr>
<tr>
<td><code>1.11.7</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.11.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.11.10</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.11.11</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.11.12</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCCcore/9.2.0</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hydra">Hydra<a class="headerlink" href="#hydra" title="Permanent link">&para;</a></h3>
<p>Hydra is an open-source Python framework that simplifies the development of research and other complex applications. The key feature is the ability to dynamically create a hierarchical configuration by composition and override it through config files and the command line. The name Hydra comes from its ability to run multiple similar jobs - much like a Hydra with multiple heads.</p>
<p><em>homepage</em>: <a href="https://hydra.cc/">https://hydra.cc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hyperopt">Hyperopt<a class="headerlink" href="#hyperopt" title="Permanent link">&para;</a></h3>
<p>hyperopt is a Python library for optimizing over awkward search spaces with real-valued,  discrete, and conditional dimensions.</p>
<p><em>homepage</em>: <a href="http://hyperopt.github.io/hyperopt/">http://hyperopt.github.io/hyperopt/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td><code>-Python-3.7.4-Java-1.8</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.2.5</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.2.7</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="hyperqueue">HyperQueue<a class="headerlink" href="#hyperqueue" title="Permanent link">&para;</a></h3>
<p>HyperQueue lets you build a computation plan consisting of a large amount of tasks and then execute it transparently over a system like SLURM/PBS. It dynamically groups jobs into SLURM/PBS jobs and distributes them to fully utilize allocated nodes. You thus do not have to manually aggregate your tasks into SLURM/PBS jobs.</p>
<p><em>homepage</em>: <a href="https://it4innovations.github.io/hyperqueue/">https://it4innovations.github.io/hyperqueue/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="hyperspy">hyperspy<a class="headerlink" href="#hyperspy" title="Permanent link">&para;</a></h3>
<p>HyperSpy is an open source Python library which provides tools to facilitate the interactive  data analysis of multi-dimensional datasets that can be described as multi-dimensional arrays of a given signal  (e.g. a 2D array of spectra a.k.a spectrum image)</p>
<p><em>homepage</em>: <a href="https://hyperspy.org/">https://hyperspy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="hyphy">HyPhy<a class="headerlink" href="#hyphy" title="Permanent link">&para;</a></h3>
<p>HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package   for the analysis of genetic sequences (in particular the inference of natural selection)   using techniques in phylogenetics, molecular evolution, and machine learning</p>
<p><em>homepage</em>: <a href="https://veg.github.io/hyphy-site/">https://veg.github.io/hyphy-site/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.13</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>2.5.33</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="hypo">HyPo<a class="headerlink" href="#hypo" title="Permanent link">&para;</a></h3>
<p>HyPo: Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</p>
<p><em>homepage</em>: <a href="https://github.com/kensung-lab/hypo">https://github.com/kensung-lab/hypo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hypothesis">hypothesis<a class="headerlink" href="#hypothesis" title="Permanent link">&para;</a></h3>
<p>Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized  by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets  you find more bugs in your code with less work.</p>
<p><em>homepage</em>: <a href="https://github.com/HypothesisWorks/hypothesis">https://github.com/HypothesisWorks/hypothesis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>4.23.4</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.39.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.44.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.53.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.57.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.6.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.41.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.41.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>6.7.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>6.13.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>6.14.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>6.46.7</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>6.68.2</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="hypre">Hypre<a class="headerlink" href="#hypre" title="Permanent link">&para;</a></h3>
<p>Hypre is a library for solving large, sparse linear systems of equations on massively  parallel computers. The problems of interest arise in the simulation codes being developed at LLNL  and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.</p>
<p><em>homepage</em>: <a href="https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods">https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.14.0</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.15.1</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.18.2</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.20.0</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.21.0</code></td>
<td><code>foss/2021a</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.24.0</code></td>
<td><code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.25.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>2.27.0</code></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h2 id="i">I<a class="headerlink" href="#i" title="Permanent link">&para;</a></h2>
<p><a href="#i-cistarget">i-cisTarget</a> - <a href="#i-pi">i-PI</a> - <a href="#i-tasser">I-TASSER</a> - <a href="#i7z">i7z</a> - <a href="#ica-aroma">ICA-AROMA</a> - <a href="#icc">icc</a> - <a href="#iccifort">iccifort</a> - <a href="#iccifortcuda">iccifortcuda</a> - <a href="#ichorcna">ichorCNA</a> - <a href="#icmake">icmake</a> - <a href="#icount">iCount</a> - <a href="#icu">ICU</a> - <a href="#idba-ud">IDBA-UD</a> - <a href="#idemux">idemux</a> - <a href="#ieeg-cli">ieeg-cli</a> - <a href="#ifort">ifort</a> - <a href="#igblast">IgBLAST</a> - <a href="#igmplot">IGMPlot</a> - <a href="#igraph">igraph</a> - <a href="#igv">IGV</a> - <a href="#igv-reports">igv-reports</a> - <a href="#igvtools">IGVTools</a> - <a href="#iibff">iibff</a> - <a href="#iimkl">iimkl</a> - <a href="#iimpi">iimpi</a> - <a href="#iimpic">iimpic</a> - <a href="#ijulia">IJulia</a> - <a href="#ilamb">ILAMB</a> - <a href="#ima2">IMa2</a> - <a href="#ima2p">IMa2p</a> - <a href="#imagecodecs">imagecodecs</a> - <a href="#imageio">imageio</a> - <a href="#imagej">ImageJ</a> - <a href="#imagemagick">ImageMagick</a> - <a href="#imake">imake</a> - <a href="#imath">Imath</a> - <a href="#imb">IMB</a> - <a href="#imbalanced-learn">imbalanced-learn</a> - <a href="#imgaug">imgaug</a> - <a href="#imkl">imkl</a> - <a href="#imkl-fftw">imkl-FFTW</a> - <a href="#imlib2">Imlib2</a> - <a href="#immunedeconv">immunedeconv</a> - <a href="#imod">IMOD</a> - <a href="#impi">impi</a> - <a href="#impute2">IMPUTE2</a> - <a href="#inchi">InChI</a> - <a href="#indicators">indicators</a> - <a href="#inelastica">Inelastica</a> - <a href="#infercnv">inferCNV</a> - <a href="#infercnvpy">infercnvpy</a> - <a href="#inferelator">Inferelator</a> - <a href="#infernal">Infernal</a> - <a href="#infomap">Infomap</a> - <a href="#inline">inline</a> - <a href="#inparanoid">InParanoid</a> - <a href="#inputproto">inputproto</a> - <a href="#inspector">Inspector</a> - <a href="#intarna">IntaRNA</a> - <a href="#integrate">INTEGRATE</a> - <a href="#integrate-neo">INTEGRATE-Neo</a> - <a href="#intel">intel</a> - <a href="#intel-compilers">intel-compilers</a> - <a href="#intelclusterchecker">IntelClusterChecker</a> - <a href="#intelcuda">intelcuda</a> - <a href="#inteldaal">IntelDAAL</a> - <a href="#intelpython">IntelPython</a> - <a href="#interproscan">InterProScan</a> - <a href="#interproscan_data">InterProScan_data</a> - <a href="#intervaltree">intervaltree</a> - <a href="#intervaltree-python">intervaltree-python</a> - <a href="#intltool">intltool</a> - <a href="#io_lib">io_lib</a> - <a href="#ioapi">ioapi</a> - <a href="#iodata">iodata</a> - <a href="#iomkl">iomkl</a> - <a href="#iompi">iompi</a> - <a href="#ior">IOR</a> - <a href="#iozone">IOzone</a> - <a href="#ipm">IPM</a> - <a href="#ipopt">Ipopt</a> - <a href="#ipp">ipp</a> - <a href="#ipy">IPy</a> - <a href="#ipyparallel">ipyparallel</a> - <a href="#ipyrad">ipyrad</a> - <a href="#ipython">IPython</a> - <a href="#iq-tree">IQ-TREE</a> - <a href="#iris">Iris</a> - <a href="#irkernel">IRkernel</a> - <a href="#irodsfs">irodsfs</a> - <a href="#ironpython">IronPython</a> - <a href="#isa-l">ISA-L</a> - <a href="#isl">ISL</a> - <a href="#isocirc">isoCirc</a> - <a href="#isonet">IsoNet</a> - <a href="#ispc">ispc</a> - <a href="#itac">itac</a> - <a href="#itk">ITK</a> - <a href="#itpp">itpp</a> - <a href="#itstool">ITSTool</a> - <a href="#itsx">ITSx</a> - <a href="#ivar">iVar</a></p>
<h3 id="i-cistarget">i-cisTarget<a class="headerlink" href="#i-cistarget" title="Permanent link">&para;</a></h3>
<p>An integrative genomics method for the prediction of regulatory features and cis-regulatory modules  in Human, Mouse, and Fly</p>
<p><em>homepage</em>: <a href="https://gbiomed.kuleuven.be/apps/lcb/i-cisTarget">https://gbiomed.kuleuven.be/apps/lcb/i-cisTarget</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160602</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="i-pi">i-PI<a class="headerlink" href="#i-pi" title="Permanent link">&para;</a></h3>
<p>A Python wrapper for (ab initio) (path integrals) molecular dynamics</p>
<p><em>homepage</em>: <a href="https://github.com/i-pi/i-pi">https://github.com/i-pi/i-pi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-20160213</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="i-tasser">I-TASSER<a class="headerlink" href="#i-tasser" title="Permanent link">&para;</a></h3>
<p>I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function modelling and comparison.</p>
<p><em>homepage</em>: <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/">http://zhanglab.ccmb.med.umich.edu/I-TASSER/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="i7z">i7z<a class="headerlink" href="#i7z" title="Permanent link">&para;</a></h3>
<p>A better i7 (and now i3, i5) reporting tool for Linux </p>
<p><em>homepage</em>: <a href="https://github.com/ajaiantilal/i7z">https://github.com/ajaiantilal/i7z</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20131012</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ica-aroma">ICA-AROMA<a class="headerlink" href="#ica-aroma" title="Permanent link">&para;</a></h3>
<p>ICA-AROMA (i.e. 'ICA-based Automatic Removal Of Motion Artifacts') concerns a data-driven method  to identify and remove motion-related independent components from fMRI data.</p>
<p><em>homepage</em>: <a href="https://github.com/maartenmennes/ICA-AROMA">https://github.com/maartenmennes/ICA-AROMA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.4-beta</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="icc">icc<a class="headerlink" href="#icc" title="Permanent link">&para;</a></h3>
<p>C and C++ compiler from Intel</p>
<p><em>homepage</em>: <a href="http://software.intel.com/en-us/intel-compilers/">http://software.intel.com/en-us/intel-compilers/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.0.109</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.0.109</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.1.150</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.0.098</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.2.174</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.4.196</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.5.239</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.6.256</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.7.259</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.0.128</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.2.199</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.3.222</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.5.274</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.0.117</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.1.144</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.2.187</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.3.199</code></td>
<td><code>-GCC-8.3.0-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-GCC-system-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iccifort">iccifort<a class="headerlink" href="#iccifort" title="Permanent link">&para;</a></h3>
<p>Intel C, C++ &amp; Fortran compilers</p>
<p><em>homepage</em>: <a href="http://software.intel.com/en-us/intel-cluster-toolkit-compiler/">http://software.intel.com/en-us/intel-cluster-toolkit-compiler/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.0.109</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.0.109</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.1.150</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.0.098</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.2.174</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.4.196</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.5.239</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.0.128</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.2.199</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.3.222</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.5.274</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.0.117</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.1.144</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.2.187</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.3.199</code></td>
<td><code>-GCC-8.3.0-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.4.243</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.5.281</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.0.166</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.0.166</code></td>
<td><code>-GCC-9.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.1.217</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.4.304</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-GCC-system-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iccifortcuda">iccifortcuda<a class="headerlink" href="#iccifortcuda" title="Permanent link">&para;</a></h3>
<p>Intel C, C++ &amp; Fortran compilers with CUDA toolkit</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.10</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.4.196</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ichorcna">ichorCNA<a class="headerlink" href="#ichorcna" title="Permanent link">&para;</a></h3>
<p>ichorCNA is a tool for estimating the fraction of tumor in cell-free DNA from ultra-low-pass whole genome sequencing</p>
<p><em>homepage</em>: <a href="https://github.com/broadinstitute/ichorCNA/wiki">https://github.com/broadinstitute/ichorCNA/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.3.2-20191219</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="icmake">icmake<a class="headerlink" href="#icmake" title="Permanent link">&para;</a></h3>
<p>Icmake is a hybrid between a 'make' utility and a 'shell script' language. Originally, it was written  to provide a useful tool for automatic program maintenance and system administrative tasks on old MS-DOS platforms.</p>
<p><em>homepage</em>: <a href="http://icmake.sourceforge.net/">http://icmake.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.23.02</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="icount">iCount<a class="headerlink" href="#icount" title="Permanent link">&para;</a></h3>
<p>iCount: protein-RNA interaction analysis  is a Python module and associated command-line interface (CLI),  which provides all the commands needed to process iCLIP data on protein-RNA interactions.</p>
<p><em>homepage</em>: <a href="https://github.com/tomazc/iCount">https://github.com/tomazc/iCount</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180820</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="icu">ICU<a class="headerlink" href="#icu" title="Permanent link">&para;</a></h3>
<p>ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.</p>
<p><em>homepage</em>: <a href="https://icu.unicode.org">https://icu.unicode.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>61.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>64.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>65.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>66.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>67.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>69.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>71.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>72.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="idba-ud">IDBA-UD<a class="headerlink" href="#idba-ud" title="Permanent link">&para;</a></h3>
<p>IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads  Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.  IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth  contigs are removed iteratively with cutoff threshold from low to high to reduce the errors  in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled  locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD  can iterate k value of de Bruijn graph to a very large value with less gaps and less branches  to form long contigs in both low-depth and high-depth regions.</p>
<p><em>homepage</em>: <a href="http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/">http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="idemux">idemux<a class="headerlink" href="#idemux" title="Permanent link">&para;</a></h3>
<p>idemux - inline barcode demultiplexing Idemux is a command line tool designed to demultiplex paired-end FASTQ files from QuantSeq-Pool.</p>
<p><em>homepage</em>: <a href="https://github.com/Lexogen-Tools/idemux">https://github.com/Lexogen-Tools/idemux</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ieeg-cli">ieeg-cli<a class="headerlink" href="#ieeg-cli" title="Permanent link">&para;</a></h3>
<p>IEEG.ORG is a collaborative initiative funded by the National Institutes  of Neurological Disorders and Stroke. This initiative seeks to advance research towards  the understanding of epilepsy by providing a platform for sharing data,  tools and expertise between researchers.</p>
<p><em>homepage</em>: <a href="https://www.ieeg.org/">https://www.ieeg.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14.56</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ifort">ifort<a class="headerlink" href="#ifort" title="Permanent link">&para;</a></h3>
<p>Fortran compiler from Intel</p>
<p><em>homepage</em>: <a href="http://software.intel.com/en-us/intel-compilers/">http://software.intel.com/en-us/intel-compilers/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.0.109</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.0.109</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.1.150</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.2.181</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.3.210</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.0.098</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.2.174</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.4.196</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.5.239</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.6.256</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.7.259</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.0.128</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.2.199</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.3.222</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.5.274</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.0.117</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.1.144</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.2.187</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.3.199</code></td>
<td><code>-GCC-8.3.0-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-GCC-system-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="igblast">IgBLAST<a class="headerlink" href="#igblast" title="Permanent link">&para;</a></h3>
<p>IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.</p>
<p><em>homepage</em>: <a href="https://ncbi.github.io/igblast">https://ncbi.github.io/igblast</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.15.0</code></td>
<td><code>-x64-linux</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>-x64-linux</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="igmplot">IGMPlot<a class="headerlink" href="#igmplot" title="Permanent link">&para;</a></h3>
<p>IGMPlot is a free open-source program developed to identify molecular interactions and  prepare data to build 2D and 3D representations of them in the molecular environment.</p>
<p><em>homepage</em>: <a href="http://igmplot.univ-reims.fr">http://igmplot.univ-reims.fr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.6.9b</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="igraph">igraph<a class="headerlink" href="#igraph" title="Permanent link">&para;</a></h3>
<p>igraph is a collection of network analysis tools with the emphasis on efficiency, portability and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++.</p>
<p><em>homepage</em>: <a href="https://igraph.org">https://igraph.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2016b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.8.5</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.9.4</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.9.5</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.10.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="igv">IGV<a class="headerlink" href="#igv" title="Permanent link">&para;</a></h3>
<p>This package contains command line utilities for  preprocessing, computing feature count density (coverage),  sorting, and  indexing data files.</p>
<p><em>homepage</em>: <a href="https://www.broadinstitute.org/software/igv/">https://www.broadinstitute.org/software/igv/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.68</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.80</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.4</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.12.3</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.16.0</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="igv-reports">igv-reports<a class="headerlink" href="#igv-reports" title="Permanent link">&para;</a></h3>
<p>Python application to generate self-contained igv.js pages that can be opened within a browser with "file" protocol.</p>
<p><em>homepage</em>: <a href="https://github.com/igvteam/igv-reports">https://github.com/igvteam/igv-reports</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="igvtools">IGVTools<a class="headerlink" href="#igvtools" title="Permanent link">&para;</a></h3>
<p>This package contains command line utilities for preprocessing,   computing feature count density (coverage),  sorting, and indexing data files.  See also http://www.broadinstitute.org/software/igv/igvtools_commandline. </p>
<p><em>homepage</em>: <a href="https://software.broadinstitute.org/software/igv/igvtools">https://software.broadinstitute.org/software/igv/igvtools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.68</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.72</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.75</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.18</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iibff">iibff<a class="headerlink" href="#iibff" title="Permanent link">&para;</a></h3>
<p>GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iimkl">iimkl<a class="headerlink" href="#iimkl" title="Permanent link">&para;</a></h3>
<p>Intel C/C++ and Fortran compilers, alongside Intel Math Kernel Library (MKL).</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/intel-cluster-toolkit-compiler/">https://software.intel.com/en-us/intel-cluster-toolkit-compiler/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iimpi">iimpi<a class="headerlink" href="#iimpi" title="Permanent link">&para;</a></h3>
<p>Intel C/C++ and Fortran compilers, alongside Intel MPI.</p>
<p><em>homepage</em>: <a href="http://software.intel.com/en-us/intel-cluster-toolkit-compiler/">http://software.intel.com/en-us/intel-cluster-toolkit-compiler/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.1.5</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.00</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.01</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.02</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.02</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.00</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.01</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.02</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.09</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.04</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.03</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.06-impi-18.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.12</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.09</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.12</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td><code>-GCC-system-2.29</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iimpic">iimpic<a class="headerlink" href="#iimpic" title="Permanent link">&para;</a></h3>
<p>Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ijulia">IJulia<a class="headerlink" href="#ijulia" title="Permanent link">&para;</a></h3>
<p>Julia kernel for Jupyter</p>
<p><em>homepage</em>: <a href="https://github.com/JuliaLang/IJulia.jl">https://github.com/JuliaLang/IJulia.jl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.23.3</code></td>
<td><code>-Julia-1.6.7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.24.0</code></td>
<td><code>-Julia-1.8.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ilamb">ILAMB<a class="headerlink" href="#ilamb" title="Permanent link">&para;</a></h3>
<p>The International Land Model Benchmarking (ILAMB) project is a model-data intercomparison and integration project designed to improve the performance of land models and, in parallel, improve the design of new measurement campaigns to reduce uncertainties associated with key land surface processes. </p>
<p><em>homepage</em>: <a href="https://www.ilamb.org/">https://www.ilamb.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="ima2">IMa2<a class="headerlink" href="#ima2" title="Permanent link">&para;</a></h3>
<p>IMa2 is a progam for population genetic analysis that can handle two or more populations.</p>
<p><em>homepage</em>: <a href="https://bio.cst.temple.edu/~hey/software/software.htm#IMa2">https://bio.cst.temple.edu/~hey/software/software.htm#IMa2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.27.12</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ima2p">IMa2p<a class="headerlink" href="#ima2p" title="Permanent link">&para;</a></h3>
<p>IMa2p is a parallel implementation of IMa2, using OpenMPI-C++ - a Bayesian MCMC  based method for inferring population demography under the IM (Isolation with  Migration) model. http://dx.doi.org/10.1111/1755-0998.12437 </p>
<p><em>homepage</em>: <a href="https://github.com/arunsethuraman/ima2p">https://github.com/arunsethuraman/ima2p</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20151123</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20160804</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="imagecodecs">imagecodecs<a class="headerlink" href="#imagecodecs" title="Permanent link">&para;</a></h3>
<p>Imagecodecs is a Python library that provides block-oriented, in-memory buffer transformation, compression, and decompression functions for use in the tifffile, czifile, zarr, and other scientific image input/output modules.</p>
<p><em>homepage</em>: <a href="https://github.com/cgohlke/imagecodecs">https://github.com/cgohlke/imagecodecs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021.8.26</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2022.9.26</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="imageio">imageio<a class="headerlink" href="#imageio" title="Permanent link">&para;</a></h3>
<p>Imageio is a Python library that provides an easy interface to read and write a wide range of  image data, including animated images, video, volumetric data, and scientific formats.</p>
<p><em>homepage</em>: <a href="https://imageio.github.io">https://imageio.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.10.5</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.13.5</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.22.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="imagej">ImageJ<a class="headerlink" href="#imagej" title="Permanent link">&para;</a></h3>
<p>Image Processing and Analysis in Java</p>
<p><em>homepage</em>: <a href="https://imagej.nih.gov/ij">https://imagej.nih.gov/ij</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.51a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.51i</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.51k</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.52q</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="imagemagick">ImageMagick<a class="headerlink" href="#imagemagick" title="Permanent link">&para;</a></h3>
<p>ImageMagick is a software suite to create, edit, compose, or convert bitmap images</p>
<p><em>homepage</em>: <a href="https://www.imagemagick.org/">https://www.imagemagick.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.9.4-8</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.1-6</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.1-9</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.2-9</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.3-1</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.0.5-4</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>7.0.5-10</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>7.0.7-8</code></td>
<td><code>-JasPer-1.900.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>7.0.7-14</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>7.0.7-15</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.0.7-26</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>7.0.7-30</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.0.7-30</code></td>
<td><code>-Ghostscript-9.22-cairo-1.14.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.0.7-39</code></td>
<td><code>-Ghostscript-9.23-cairo-1.14.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>7.0.8-11</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>7.0.8-46</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>7.0.9-5</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>7.0.10-1</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.0.10-35</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>7.0.11-14</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>7.1.0-4</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>7.1.0-37</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>7.1.0-53</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="imake">imake<a class="headerlink" href="#imake" title="Permanent link">&para;</a></h3>
<p>imake is a Makefile-generator that is intended to make it easier to develop software portably for multiple systems.</p>
<p><em>homepage</em>: <a href="https://www.x.org/">https://www.x.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.7</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="imath">Imath<a class="headerlink" href="#imath" title="Permanent link">&para;</a></h3>
<p>Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics </p>
<p><em>homepage</em>: <a href="https://imath.readthedocs.io/en/latest/">https://imath.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.1.6</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="imb">IMB<a class="headerlink" href="#imb" title="Permanent link">&para;</a></h3>
<p>The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and  global communication operations for a range of message sizes</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/articles/intel-mpi-benchmarks">https://software.intel.com/en-us/articles/intel-mpi-benchmarks</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1</code></td>
<td><code>foss/2016a</code>, <code>foss/2017a</code>, <code>intel/2017.02</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2018.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2019.3</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>2021.3</code></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>gompi/2022b</code>, <code>iimpi/2022a</code>, <code>iimpi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="imbalanced-learn">imbalanced-learn<a class="headerlink" href="#imbalanced-learn" title="Permanent link">&para;</a></h3>
<p>imbalanced-learn is a Python package offering a number of re-sampling techniques commonly used in  datasets showing strong between-class imbalance.</p>
<p><em>homepage</em>: <a href="https://github.com/scikit-learn-contrib/imbalanced-learn">https://github.com/scikit-learn-contrib/imbalanced-learn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.2.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.3.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.4.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="imgaug">imgaug<a class="headerlink" href="#imgaug" title="Permanent link">&para;</a></h3>
<p>This python library helps you with augmenting images for your machine learning projects.  It converts a set of input images into a new, much larger set of slightly altered images. </p>
<p><em>homepage</em>: <a href="https://imgaug.readthedocs.io/en/latest/">https://imgaug.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.8</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="imkl">imkl<a class="headerlink" href="#imkl" title="Permanent link">&para;</a></h3>
<p>Intel oneAPI Math Kernel Library</p>
<p><em>homepage</em>: <a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html">https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.2.3.187</code></td>
<td></td>
<td><code>gimpi/2.11.5</code></td>
</tr>
<tr>
<td><code>11.3.0.109</code></td>
<td></td>
<td><code>iimpi/2016.00-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>11.3.1.150</code></td>
<td></td>
<td><code>iimpi/2016.01-GCC-4.9.3-2.25</code>, <code>iimpi/8.1.5-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>11.3.2.181</code></td>
<td></td>
<td><code>iimpi/2016.02-GCC-4.9.3-2.25</code>, <code>iimpi/2016.02-GCC-5.3.0-2.26</code>, <code>pompi/2016.03</code></td>
</tr>
<tr>
<td><code>11.3.3.210</code></td>
<td></td>
<td><code>iimpi/2016.03-GCC-4.9.3-2.25</code>, <code>iimpi/2016.03-GCC-5.3.0-2.26</code>, <code>iimpi/2016.03-GCC-5.4.0-2.26</code>, <code>iimpi/2016b</code>, <code>iimpic/2016.10</code>, <code>iompi/2016.07</code>, <code>iompi/2016.09-GCC-4.9.3-2.25</code>, <code>iompi/2016.09-GCC-5.4.0-2.26</code>, <code>pompi/2016.04</code>, <code>pompi/2016.09</code></td>
</tr>
<tr>
<td><code>2017.0.098</code></td>
<td></td>
<td><code>iimpi/2017.00-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td></td>
<td><code>gimpi/2017a</code>, <code>iimpi/2017.01-GCC-5.4.0-2.26</code>, <code>iimpi/2017a</code>, <code>iompi/2017.01</code>, <code>iompi/2017a</code></td>
</tr>
<tr>
<td><code>2017.2.174</code></td>
<td></td>
<td><code>iimpi/2017.02-GCC-6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>2017.3.196</code></td>
<td></td>
<td><code>gompi/2017b</code>, <code>iimpi/2017b</code>, <code>iimpic/2017b</code>, <code>iompi/2017b</code></td>
</tr>
<tr>
<td><code>2017.4.239</code></td>
<td></td>
<td><code>iimpi/2017.09</code></td>
</tr>
<tr>
<td><code>2018.0.128</code></td>
<td></td>
<td><code>iimpi/2018.00</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td></td>
<td><code>iimpi/2018.01</code>, <code>iimpi/2018a</code>, <code>iompi/2018a</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td><code>-serial</code></td>
<td><code>iccifort/2018.1.163-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2018.2.199</code></td>
<td></td>
<td><code>iimpi/2018.02</code>, <code>iompi/2018.02</code></td>
</tr>
<tr>
<td><code>2018.3.222</code></td>
<td></td>
<td><code>gimpi/2018b</code>, <code>gompi/2018b</code>, <code>iimpi/2018b</code>, <code>iompi/2018b</code></td>
</tr>
<tr>
<td><code>2018.4.274</code></td>
<td></td>
<td><code>iimpi/2018.04</code></td>
</tr>
<tr>
<td><code>2019.0.117</code></td>
<td></td>
<td><code>iimpi/2019.00</code></td>
</tr>
<tr>
<td><code>2019.1.144</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019.01</code>, <code>iimpi/2019a</code>, <code>iimpic/2019a</code>, <code>iompi/2019.01</code></td>
</tr>
<tr>
<td><code>2019.2.187</code></td>
<td></td>
<td><code>iimpi/2019.02</code></td>
</tr>
<tr>
<td><code>2019.3.199</code></td>
<td></td>
<td><code>iimpi/2019.03</code></td>
</tr>
<tr>
<td><code>2019.5.281</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompic/2019b</code>, <code>iimpi/2019b</code>, <code>iimpic/2019b</code>, <code>iompi/2019b</code></td>
</tr>
<tr>
<td><code>2020.0.166</code></td>
<td></td>
<td><code>iimpi/2020.00</code></td>
</tr>
<tr>
<td><code>2020.1.217</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020.06-impi-18.5</code>, <code>iimpi/2020a</code>, <code>iimpic/2020a</code>, <code>iompi/2020a</code></td>
</tr>
<tr>
<td><code>2020.4.304</code></td>
<td></td>
<td><code>NVHPC/21.2</code>, <code>gompi/2020b</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code>, <code>iimpic/2020b</code>, <code>iompi/2020b</code></td>
</tr>
<tr>
<td><code>2021.1.1</code></td>
<td></td>
<td><code>iimpi/2020.12</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>iimpi/2021a</code>, <code>iompi/2021a</code></td>
</tr>
<tr>
<td><code>2021.3.0</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td></td>
<td><code>iompi/2021b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2022.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2022.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.2.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2023.0.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="imkl-fftw">imkl-FFTW<a class="headerlink" href="#imkl-fftw" title="Permanent link">&para;</a></h3>
<p>FFTW interfaces using Intel oneAPI Math Kernel Library</p>
<p><em>homepage</em>: <a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html">https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021.4.0</code></td>
<td><code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>2022.0.1</code></td>
<td><code>iimpi/2022.00</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td><code>iimpi/2022.05</code>, <code>iimpi/2022a</code></td>
</tr>
<tr>
<td><code>2022.2.0</code></td>
<td><code>iimpi/2022.09</code></td>
</tr>
<tr>
<td><code>2022.2.1</code></td>
<td><code>iimpi/2022.11</code>, <code>iimpi/2022b</code></td>
</tr>
<tr>
<td><code>2023.0.0</code></td>
<td><code>iimpi/2022.12</code></td>
</tr>
</tbody>
</table>
<h3 id="imlib2">Imlib2<a class="headerlink" href="#imlib2" title="Permanent link">&para;</a></h3>
<p>This is the Imlib 2 library - a library that does image file loading and saving as well as rendering, manipulation, arbitrary polygon support, etc. It does ALL of these operations FAST. Imlib2 also tries to be highly intelligent about doing them, so writing naive programs can be done easily, without sacrificing speed. </p>
<p><em>homepage</em>: <a href="https://docs.enlightenment.org/api/imlib2/html/">https://docs.enlightenment.org/api/imlib2/html/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="immunedeconv">immunedeconv<a class="headerlink" href="#immunedeconv" title="Permanent link">&para;</a></h3>
<p>immunedeconv is an R package for unified access to computational methods for estimating immune cell fractions from bulk RNA sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/icbi-lab/immunedeconv">https://github.com/icbi-lab/immunedeconv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="imod">IMOD<a class="headerlink" href="#imod" title="Permanent link">&para;</a></h3>
<p>IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections. The package contains tools for assembling and aligning data within multiple types and sizes of image stacks, viewing 3-D data from any orientation, and modeling and display of the image files. IMOD was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim Kremer, Quanren Xiong, and John Heumann at the University of Colorado.</p>
<p><em>homepage</em>: <a href="http://bio3d.colorado.edu/imod/">http://bio3d.colorado.edu/imod/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7.15</code></td>
<td><code>_RHEL6-64_CUDA6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11.5</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="impi">impi<a class="headerlink" href="#impi" title="Permanent link">&para;</a></h3>
<p>The Intel(R) MPI Library for Linux* OS is a multi-fabric message  passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for  Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification.</p>
<p><em>homepage</em>: <a href="http://software.intel.com/en-us/intel-mpi-library/">http://software.intel.com/en-us/intel-mpi-library/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2.006</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0.028</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0.028</code></td>
<td><code>-32bit</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.2.003</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.0.027</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.0.030</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.1.036</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.2.040</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.3.045</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.3.049</code></td>
<td></td>
<td><code>GCC/4.8.3</code>, <code>system</code></td>
</tr>
<tr>
<td><code>5.0.3.048</code></td>
<td></td>
<td><code>GCC/4.9.3</code></td>
</tr>
<tr>
<td><code>5.1.1.109</code></td>
<td></td>
<td><code>iccifort/2016.0.109-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>5.1.2.150</code></td>
<td></td>
<td><code>iccifort/2016.1.150-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>5.1.3.181</code></td>
<td></td>
<td><code>iccifort/2016.2.181-GCC-4.9.3-2.25</code>, <code>iccifort/2016.2.181-GCC-5.3.0-2.26</code>, <code>iccifort/2016.3.210-GCC-4.9.3-2.25</code>, <code>iccifort/2016.3.210-GCC-5.3.0-2.26</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>iccifortcuda/2016.10</code></td>
</tr>
<tr>
<td><code>2017.0.098</code></td>
<td></td>
<td><code>iccifort/2017.0.098-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>iccifort/2017.1.132-GCC-5.4.0-2.26</code>, <code>iccifort/2017.1.132-GCC-6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>2017.2.174</code></td>
<td></td>
<td><code>iccifort/2017.2.174-GCC-6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>2017.3.196</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>gcccuda/2017b</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>iccifortcuda/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2017.4.239</code></td>
<td></td>
<td><code>iccifort/2017.5.239-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2018.0.128</code></td>
<td></td>
<td><code>iccifort/2018.0.128-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2018.1.163</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2018.1.163-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2018.2.199</code></td>
<td></td>
<td><code>iccifort/2018.2.199-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2018.3.222</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code>, <code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2018.4.274</code></td>
<td></td>
<td><code>iccifort/2018.5.274-GCC-7.3.0-2.30</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifortcuda/2019a</code></td>
</tr>
<tr>
<td><code>2018.5.288</code></td>
<td></td>
<td><code>iccifort/2019.5.281</code>, <code>iccifort/2020.1.217</code>, <code>iccifortcuda/2019b</code></td>
</tr>
<tr>
<td><code>2019.0.117</code></td>
<td></td>
<td><code>iccifort/2019.0.117-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2019.1.144</code></td>
<td></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2019.2.187</code></td>
<td></td>
<td><code>iccifort/2019.2.187-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2019.3.199</code></td>
<td></td>
<td><code>iccifort/2019.3.199-GCC-8.3.0-2.32</code></td>
</tr>
<tr>
<td><code>2019.6.166</code></td>
<td></td>
<td><code>iccifort/2020.0.166-GCC-9.2.0</code></td>
</tr>
<tr>
<td><code>2019.7.217</code></td>
<td></td>
<td><code>iccifort/2020.1.217</code>, <code>iccifortcuda/2020a</code></td>
</tr>
<tr>
<td><code>2019.9.304</code></td>
<td></td>
<td><code>iccifort/2020.4.304</code>, <code>iccifortcuda/2020b</code></td>
</tr>
<tr>
<td><code>2019.12.320</code></td>
<td></td>
<td><code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>2021.1.1</code></td>
<td></td>
<td><code>intel-compilers/2021.1.2</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td></td>
<td><code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>2021.3.0</code></td>
<td></td>
<td><code>intel-compilers/2021.3.0</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td></td>
<td><code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>2021.5.0</code></td>
<td></td>
<td><code>intel-compilers/2022.0.1</code></td>
</tr>
<tr>
<td><code>2021.6.0</code></td>
<td></td>
<td><code>intel-compilers/2022.1.0</code></td>
</tr>
<tr>
<td><code>2021.7.0</code></td>
<td></td>
<td><code>intel-compilers/2022.2.0</code></td>
</tr>
<tr>
<td><code>2021.7.1</code></td>
<td></td>
<td><code>intel-compilers/2022.2.1</code></td>
</tr>
<tr>
<td><code>2021.8.0</code></td>
<td></td>
<td><code>intel-compilers/2023.0.0</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td></td>
<td><code>iccifort/system-GCC-system-2.29</code></td>
</tr>
</tbody>
</table>
<h3 id="impute2">IMPUTE2<a class="headerlink" href="#impute2" title="Permanent link">&para;</a></h3>
<p>IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation   and haplotype phasing program based on ideas from Howie et al. 2009 </p>
<p><em>homepage</em>: <a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html">http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>_x86_64_dynamic</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>_x86_64_static</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>_x86_64_dynamic</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>_x86_64_static</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="inchi">InChI<a class="headerlink" href="#inchi" title="Permanent link">&para;</a></h3>
<p>The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary identifier for chemical substances that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.</p>
<p><em>homepage</em>: <a href="https://www.inchi-trust.org/">https://www.inchi-trust.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.06</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="indicators">indicators<a class="headerlink" href="#indicators" title="Permanent link">&para;</a></h3>
<ul>
<li>Thread-safe progress bars and spinners - Header-only library. Grab a copy of include/indicators. - Single-header version in single_include/indicators. </li>
</ul>
<p><em>homepage</em>: <a href="https://github.com/p-ranav/indicators">https://github.com/p-ranav/indicators</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="inelastica">Inelastica<a class="headerlink" href="#inelastica" title="Permanent link">&para;</a></h3>
<p>Python package for eigenchannels, vibrations and inelastic electron transport based on SIESTA/TranSIESTA DFT.</p>
<p><em>homepage</em>: <a href="https://github.com/tfrederiksen/inelastica">https://github.com/tfrederiksen/inelastica</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="infercnv">inferCNV<a class="headerlink" href="#infercnv" title="Permanent link">&para;</a></h3>
<p>InferCNV is used to explore tumor single cell RNA-Seq data to identify evidence  for somatic large-scale chromosomal copy number alterations, such as gains or  deletions of entire chromosomes or large segments of chromosomes.</p>
<p><em>homepage</em>: <a href="https://github.com/broadinstitute/inferCNV/wiki">https://github.com/broadinstitute/inferCNV/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.7.2-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="infercnvpy">infercnvpy<a class="headerlink" href="#infercnvpy" title="Permanent link">&para;</a></h3>
<p>Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.</p>
<p><em>homepage</em>: <a href="https://github.com/icbi-lab/infercnvpy">https://github.com/icbi-lab/infercnvpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="inferelator">Inferelator<a class="headerlink" href="#inferelator" title="Permanent link">&para;</a></h3>
<p>Inferelator 3.0 is a package for gene regulatory network inference that is based on  regularized regression.</p>
<p><em>homepage</em>: <a href="https://github.com/flatironinstitute/inferelator">https://github.com/flatironinstitute/inferelator</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="infernal">Infernal<a class="headerlink" href="#infernal" title="Permanent link">&para;</a></h3>
<p>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases  for RNA structure and sequence similarities.</p>
<p><em>homepage</em>: <a href="http://eddylab.org/infernal/">http://eddylab.org/infernal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="infomap">Infomap<a class="headerlink" href="#infomap" title="Permanent link">&para;</a></h3>
<p>Multi-level network clustering based on the Map equation.</p>
<p><em>homepage</em>: <a href="https://www.mapequation.org/code.html#Linux">https://www.mapequation.org/code.html#Linux</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190308</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="inline">inline<a class="headerlink" href="#inline" title="Permanent link">&para;</a></h3>
<p>Functionality to dynamically define R functions and S4 methods  with 'inlined' C, C++ or Fortran code supporting the .C and .Call calling  conventions.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/inline">https://cran.r-project.org/web/packages/inline</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.19</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="inparanoid">InParanoid<a class="headerlink" href="#inparanoid" title="Permanent link">&para;</a></h3>
<p>InParanoid: ortholog groups with inparalogs.</p>
<p><em>homepage</em>: <a href="https://inparanoid.sbc.su.se">https://inparanoid.sbc.su.se</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0-20220607</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="inputproto">inputproto<a class="headerlink" href="#inputproto" title="Permanent link">&para;</a></h3>
<p>X.org InputProto protocol headers.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="inspector">Inspector<a class="headerlink" href="#inspector" title="Permanent link">&para;</a></h3>
<p>Intel Inspector is a dynamic memory and threading error  checking tool for users developing serial and parallel applications</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/inspector">https://software.intel.com/en-us/inspector</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2013_update6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="intarna">IntaRNA<a class="headerlink" href="#intarna" title="Permanent link">&para;</a></h3>
<p>Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites</p>
<p><em>homepage</em>: <a href="https://github.com/BackofenLab/IntaRNA">https://github.com/BackofenLab/IntaRNA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="integrate">INTEGRATE<a class="headerlink" href="#integrate" title="Permanent link">&para;</a></h3>
<p>INTEGRATE is a tool calling gene fusions with exact fusion junctions and genomic breakpoints  by combining RNA-Seq and WGS data. It is highly sensitive and accurate by applying a fast split-read  mapping algorithm based on Burrow-Wheeler transform. </p>
<p><em>homepage</em>: <a href="https://sourceforge.net/p/integrate-fusion/wiki/Home/">https://sourceforge.net/p/integrate-fusion/wiki/Home/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.6</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="integrate-neo">INTEGRATE-Neo<a class="headerlink" href="#integrate-neo" title="Permanent link">&para;</a></h3>
<p>INTEGRATE-Neo is a gene fusion neoantigen discovering tool using next-generation sequencing data.  It is written in C++ and Python. </p>
<p><em>homepage</em>: &lt;&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="intel">intel<a class="headerlink" href="#intel" title="Permanent link">&para;</a></h3>
<p>Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).</p>
<p><em>homepage</em>: <a href="https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain">https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.02</code></td>
<td><code>-GCC-4.9</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.02</code></td>
<td><code>-GCC-5.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-4.9</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-5.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.03</code></td>
<td><code>-GCC-5.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.09</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.04</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.01</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.02</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.03</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.06-impi-18.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.12</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.09</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.12</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="intel-compilers">intel-compilers<a class="headerlink" href="#intel-compilers" title="Permanent link">&para;</a></h3>
<p>Intel C, C++ &amp; Fortran compilers (classic and oneAPI)</p>
<p><em>homepage</em>: <a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html">https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021.1.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.2.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2023.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="intelclusterchecker">IntelClusterChecker<a class="headerlink" href="#intelclusterchecker" title="Permanent link">&para;</a></h3>
<p>Intel Cluster Checker verifies the configuration and  performance of Linux OS-based clusters. Anomalies and performance differences  can be identified and practical resolutions provided. </p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/cluster-checker">https://software.intel.com/en-us/cluster-checker</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017.1.016</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.5.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="intelcuda">intelcuda<a class="headerlink" href="#intelcuda" title="Permanent link">&para;</a></h3>
<p>Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,  Intel MPI &amp; Intel MKL, with CUDA toolkit</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="inteldaal">IntelDAAL<a class="headerlink" href="#inteldaal" title="Permanent link">&para;</a></h3>
<p>Intel Data Analytics Acceleration Library (Intel DAAL) is the library of  Intel architecture optimized building blocks covering all stages of data analytics: data acquisition from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.  </p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/daal">https://software.intel.com/en-us/daal</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.4.007</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="intelpython">IntelPython<a class="headerlink" href="#intelpython" title="Permanent link">&para;</a></h3>
<p>Intel Distribution for Python. Powered by Anaconda.  Accelerating Python* performance on modern architectures from Intel. </p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/intel-distribution-for-python">https://software.intel.com/en-us/intel-distribution-for-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.15</code></td>
<td><code>-2019.2.066</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.8</code></td>
<td><code>-2019.2.066</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="interproscan">InterProScan<a class="headerlink" href="#interproscan" title="Permanent link">&para;</a></h3>
<p>InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [code only: libraries and external binaries  but no data].</p>
<p><em>homepage</em>: <a href="https://www.ebi.ac.uk/interpro/">https://www.ebi.ac.uk/interpro/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.26-65.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.27-66.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.28-67.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.52-86.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.55-88.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="interproscan_data">InterProScan_data<a class="headerlink" href="#interproscan_data" title="Permanent link">&para;</a></h3>
<p>InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [data only].</p>
<p><em>homepage</em>: <a href="https://www.ebi.ac.uk/interpro/">https://www.ebi.ac.uk/interpro/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.55-88.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="intervaltree">intervaltree<a class="headerlink" href="#intervaltree" title="Permanent link">&para;</a></h3>
<p>An interval tree can be used to efficiently find a set of numeric intervals  overlapping or containing another interval. This library provides a basic implementation of an  interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. </p>
<p><em>homepage</em>: <a href="https://github.com/ekg/intervaltree">https://github.com/ekg/intervaltree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="intervaltree-python">intervaltree-python<a class="headerlink" href="#intervaltree-python" title="Permanent link">&para;</a></h3>
<p>A mutable, self-balancing interval tree. Queries may be by  point, by range overlap, or by range containment. </p>
<p><em>homepage</em>: <a href="https://github.com/chaimleib/intervaltree">https://github.com/chaimleib/intervaltree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="intltool">intltool<a class="headerlink" href="#intltool" title="Permanent link">&para;</a></h3>
<p>intltool is a set of tools to centralize translation of  many different file formats using GNU gettext-compatible PO files.</p>
<p><em>homepage</em>: <a href="http://freedesktop.org/wiki/Software/intltool/">http://freedesktop.org/wiki/Software/intltool/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.51.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.20.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.51.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="io_lib">io_lib<a class="headerlink" href="#io_lib" title="Permanent link">&para;</a></h3>
<p>Io_lib is a library of file reading and writing code to provide a general purpose trace file (and  Experiment File) reading interface. The programmer simply calls the (eg) read_reading to create a "Read" C structure  with the data loaded into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS   Windows.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/staden/files/io_lib/">http://sourceforge.net/projects/staden/files/io_lib/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14.8</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ioapi">ioapi<a class="headerlink" href="#ioapi" title="Permanent link">&para;</a></h3>
<p>The Models-3/EDSS Input/Output Applications Programming Interface (I/O API) provides the  environmental model developer with an easy-to-learn, easy-to-use programming library for data storage and  access, available from both Fortran and C. The same routines can be used for both file storage (using netCDF  files) and model coupling (using PVM mailboxes). It is the standard data access library for both the  NCSC/CMAS's EDSS project and EPA's Models-3, CMAQ, and SMOKE, as well as various other atmospheric and  hydrological modeling systems.</p>
<p><em>homepage</em>: <a href="https://www.cmascenter.org/ioapi/">https://www.cmascenter.org/ioapi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2-2020111</code></td>
<td><code>-nocpl</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="iodata">iodata<a class="headerlink" href="#iodata" title="Permanent link">&para;</a></h3>
<p>Python library for reading, writing, and converting computational chemistry file formats and  generating input files.</p>
<p><em>homepage</em>: <a href="https://github.com/theochem/iodata">https://github.com/theochem/iodata</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0a2</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="iomkl">iomkl<a class="headerlink" href="#iomkl" title="Permanent link">&para;</a></h3>
<p>Compiler toolchain including Intel compilers, Open MPI and Intel Math Kernel Library (MKL).</p>
<p><em>homepage</em>: <a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html">https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.01</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.02</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.01</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="iompi">iompi<a class="headerlink" href="#iompi" title="Permanent link">&para;</a></h3>
<p>Intel C/C++ and Fortran compilers, alongside Open MPI.</p>
<p><em>homepage</em>: <a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html">https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.07</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.01</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.02</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.01</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ior">IOR<a class="headerlink" href="#ior" title="Permanent link">&para;</a></h3>
<p>The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,  or HDF5 interfaces. </p>
<p><em>homepage</em>: <a href="https://github.com/hpc/ior">https://github.com/hpc/ior</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-mpiio</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="iozone">IOzone<a class="headerlink" href="#iozone" title="Permanent link">&para;</a></h3>
<p>IOzone is a filesystem benchmark tool. The benchmark generates and measures a variety of file  operations. Iozone has been ported to many machines and runs under many operating systems.</p>
<p><em>homepage</em>: <a href="http://www.iozone.org/">http://www.iozone.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.434</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ipm">IPM<a class="headerlink" href="#ipm" title="Permanent link">&para;</a></h3>
<p>IPM is a portable profiling infrastructure for parallel codes. It provides  a low-overhead profile of application performance and resource utilization  in a parallel program. Communication, computation, and IO are the primary  focus. </p>
<p><em>homepage</em>: <a href="https://github.com/nerscadmin/IPM">https://github.com/nerscadmin/IPM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.6</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="ipopt">Ipopt<a class="headerlink" href="#ipopt" title="Permanent link">&para;</a></h3>
<p>IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt)  is an open source software package for large-scale nonlinear optimization. </p>
<p><em>homepage</em>: <a href="https://projects.coin-or.org/Ipopt">https://projects.coin-or.org/Ipopt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.12.9</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.12.13</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ipp">ipp<a class="headerlink" href="#ipp" title="Permanent link">&para;</a></h3>
<p>Intel Integrated Performance Primitives (Intel IPP) is an extensive library  of multicore-ready, highly optimized software functions for multimedia, data processing,  and communications applications. Intel IPP offers thousands of optimized functions  covering frequently used fundamental algorithms.</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/articles/intel-ipp/">https://software.intel.com/en-us/articles/intel-ipp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.5.233</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.0.144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.0.1.150</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.132</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ipy">IPy<a class="headerlink" href="#ipy" title="Permanent link">&para;</a></h3>
<p>Class and tools for handling of IPv4 and IPv6 addresses and networks</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/IPy">https://pypi.python.org/pypi/IPy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.83</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ipyparallel">ipyparallel<a class="headerlink" href="#ipyparallel" title="Permanent link">&para;</a></h3>
<p>ipyparallel is a Python package and collection  of CLI scripts for controlling clusters for Jupyter</p>
<p><em>homepage</em>: <a href="https://ipyparallel.readthedocs.io">https://ipyparallel.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="ipyrad">ipyrad<a class="headerlink" href="#ipyrad" title="Permanent link">&para;</a></h3>
<p>ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic  data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.</p>
<p><em>homepage</em>: <a href="https://ipyrad.readthedocs.io">https://ipyrad.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.15</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="ipython">IPython<a class="headerlink" href="#ipython" title="Permanent link">&para;</a></h3>
<p>IPython provides a rich architecture for interactive computing with:  Powerful interactive shells (terminal and Qt-based).  A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.  Support for interactive data visualization and use of GUI toolkits.  Flexible, embeddable interpreters to load into your own projects.  Easy to use, high performance tools for parallel computing.</p>
<p><em>homepage</em>: <a href="https://ipython.org/index.html">https://ipython.org/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.7.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>fosscuda/2018b</code>, <code>fosscuda/2019a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>5.10.0</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>6.3.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.4.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>7.7.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>7.9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>7.13.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7.15.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7.18.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>7.25.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>7.26.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.5.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="iq-tree">IQ-TREE<a class="headerlink" href="#iq-tree" title="Permanent link">&para;</a></h3>
<p>Efficient phylogenomic software by maximum likelihood</p>
<p><em>homepage</em>: <a href="http://www.iqtree.org/">http://www.iqtree.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.5</code></td>
<td><code>-omp-mpi</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.6.6</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.6.12</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>foss/2020a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="iris">Iris<a class="headerlink" href="#iris" title="Permanent link">&para;</a></h3>
<p>A module for improving the insertion sequences of structural variant calls</p>
<p><em>homepage</em>: <a href="https://github.com/mkirsche/Iris">https://github.com/mkirsche/Iris</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>-Java-15</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="irkernel">IRkernel<a class="headerlink" href="#irkernel" title="Permanent link">&para;</a></h3>
<p>The R kernel for the 'Jupyter' environment executes R code   which the front-end (Jupyter Notebook or other front-ends) submits to the   kernel via the network.</p>
<p><em>homepage</em>: <a href="https://irkernel.github.io">https://irkernel.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.15</code></td>
<td><code>-R-3.4.3-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-R-3.6.2-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-R-3.6.3-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-R-4.0.0-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-R-4.2.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="irodsfs">irodsfs<a class="headerlink" href="#irodsfs" title="Permanent link">&para;</a></h3>
<p>FUSE implementation of iRODS Client written in Golang.</p>
<p><em>homepage</em>: <a href="https://github.com/cyverse/irodsfs">https://github.com/cyverse/irodsfs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.9</code></td>
<td><code>-linux-amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.11</code></td>
<td><code>-linux-amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.12</code></td>
<td><code>-linux-amd64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ironpython">IronPython<a class="headerlink" href="#ironpython" title="Permanent link">&para;</a></h3>
<p>IronPython is an open-source implementation of the Python programming language  which is tightly integrated with the .NET Framework. IronPython can use the .NET Framework and Python libraries, and other .NET languages can use Python code just as easily.</p>
<p><em>homepage</em>: <a href="http://ironpython.net/">http://ironpython.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="isa-l">ISA-L<a class="headerlink" href="#isa-l" title="Permanent link">&para;</a></h3>
<p>Intelligent Storage Acceleration Library</p>
<p><em>homepage</em>: <a href="https://github.com/intel/isa-l">https://github.com/intel/isa-l</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.30.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="isl">ISL<a class="headerlink" href="#isl" title="Permanent link">&para;</a></h3>
<p>isl is a library for manipulating sets and relations of integer points bounded by linear constraints.</p>
<p><em>homepage</em>: <a href="http://isl.gforge.inria.fr/">http://isl.gforge.inria.fr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.14</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>0.15</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.16</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>0.17</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.23</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.24</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="isocirc">isoCirc<a class="headerlink" href="#isocirc" title="Permanent link">&para;</a></h3>
<p>isoCirc: computational pipeline to identify high-confidence BSJs and full-length circRNA isoforms from isoCirc long-read data </p>
<p><em>homepage</em>: <a href="https://github.com/Xinglab/isoCirc">https://github.com/Xinglab/isoCirc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="isonet">IsoNet<a class="headerlink" href="#isonet" title="Permanent link">&para;</a></h3>
<p>IsoNet stands for for ISOtropic reconstructioN of Electron Tomography. It trains deep convolutional neural networks to reconstruct meaningful contents in the mis sing wedge for electron tomography, and to increase signal-to-noise ratio,       using the information learned from the original tomogram. The software requires  tomograms as input. Observing at about 30A resolution, the IsoNet generated      tomograms are largely isotropic.</p>
<p><em>homepage</em>: <a href="https://github.com/Heng-Z/IsoNet">https://github.com/Heng-Z/IsoNet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1_20210822_04_674f67f</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ispc">ispc<a class="headerlink" href="#ispc" title="Permanent link">&para;</a></h3>
<p>Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for 'single program, multiple data' (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.</p>
<p><em>homepage</em>: <a href="http://ispc.github.io/ , https://github.com/ispc/ispc/">http://ispc.github.io/ , https://github.com/ispc/ispc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="itac">itac<a class="headerlink" href="#itac" title="Permanent link">&para;</a></h3>
<p>The Intel Trace Collector is a low-overhead tracing library that performs  event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application  activities gathered by the Intel Trace Collector through graphical displays. </p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/intel-trace-analyzer/">https://software.intel.com/en-us/intel-trace-analyzer/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0.0.011</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.1.4.045</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.0.3.051</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.1.024</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.1.017</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.3.022</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.2.026</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.4.036</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.6.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="itk">ITK<a class="headerlink" href="#itk" title="Permanent link">&para;</a></h3>
<p>Insight Segmentation and Registration Toolkit (ITK) provides  an extensive suite of software tools for registering and segmenting  multidimensional imaging data.</p>
<p><em>homepage</em>: <a href="https://itk.org">https://itk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.12.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>4.13.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.13.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.13.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.13.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.13.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.13.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.13.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>5.0b01</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="itpp">itpp<a class="headerlink" href="#itpp" title="Permanent link">&para;</a></h3>
<p>IT++ is a C++ library of mathematical, signal processing and communication  classes and functions. Its main use is in simulation of communication systems and for  performing research in the area of communications.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/itpp/">https://sourceforge.net/projects/itpp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.1</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="itstool">ITSTool<a class="headerlink" href="#itstool" title="Permanent link">&para;</a></h3>
<p>ITS Tool allows you to translate your XML documents with PO files</p>
<p><em>homepage</em>: <a href="http://itstool.org/">http://itstool.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.5</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="itsx">ITSx<a class="headerlink" href="#itsx" title="Permanent link">&para;</a></h3>
<p>ITSx: Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences of fungi and other eukaryotes for use in environmental sequencing.</p>
<p><em>homepage</em>: <a href="https://microbiology.se/software/itsx/">https://microbiology.se/software/itsx/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ivar">iVar<a class="headerlink" href="#ivar" title="Permanent link">&para;</a></h3>
<p>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. </p>
<p><em>homepage</em>: <a href="https://github.com/andersen-lab/ivar">https://github.com/andersen-lab/ivar</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="j">J<a class="headerlink" href="#j" title="Permanent link">&para;</a></h2>
<p><a href="#jags">JAGS</a> - <a href="#jansson">Jansson</a> - <a href="#jasmine">Jasmine</a> - <a href="#jasper">JasPer</a> - <a href="#java">Java</a> - <a href="#javafx">JavaFX</a> - <a href="#jax">jax</a> - <a href="#jaxfrontce">JAXFrontCE</a> - <a href="#jbigkit">jbigkit</a> - <a href="#jellyfish">Jellyfish</a> - <a href="#jemalloc">jemalloc</a> - <a href="#jhbuild">jhbuild</a> - <a href="#jitcode">JiTCODE</a> - <a href="#jmodeltest">jModelTest</a> - <a href="#jorg">Jorg</a> - <a href="#joypy">joypy</a> - <a href="#jq">jq</a> - <a href="#json-c">json-c</a> - <a href="#json-fortran">json-fortran</a> - <a href="#json-glib">JSON-GLib</a> - <a href="#jsoncpp">JsonCpp</a> - <a href="#jube">JUBE</a> - <a href="#judy">Judy</a> - <a href="#julia">Julia</a> - <a href="#junit">JUnit</a> - <a href="#jupyter-contrib-nbextensions">jupyter-contrib-nbextensions</a> - <a href="#jupyter-matlab-proxy">jupyter-matlab-proxy</a> - <a href="#jupyter-resource-usage">jupyter-resource-usage</a> - <a href="#jupyter-server">jupyter-server</a> - <a href="#jupyter-server-proxy">jupyter-server-proxy</a> - <a href="#jupyterhub">JupyterHub</a> - <a href="#jupyterlab">JupyterLab</a> - <a href="#jupyterlab-lmod">jupyterlab-lmod</a> - <a href="#jupyterlmod">jupyterlmod</a> - <a href="#jwm">JWM</a> - <a href="#jxrlib">jxrlib</a></p>
<h3 id="jags">JAGS<a class="headerlink" href="#jags" title="Permanent link">&para;</a></h3>
<p>JAGS is Just Another Gibbs Sampler.  It is a program for analysis  of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  </p>
<p><em>homepage</em>: <a href="http://mcmc-jags.sourceforge.net/">http://mcmc-jags.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>fosscuda/2020b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="jansson">Jansson<a class="headerlink" href="#jansson" title="Permanent link">&para;</a></h3>
<p>Jansson is a C library for encoding, decoding and manipulating JSON data.  Its main features and design principles are:  * Simple and intuitive API and data model  * Comprehensive documentation  * No dependencies on other libraries  * Full Unicode support (UTF-8)  * Extensive test suite</p>
<p><em>homepage</em>: <a href="http://www.digip.org/jansson/">http://www.digip.org/jansson/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>2.13.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jasmine">Jasmine<a class="headerlink" href="#jasmine" title="Permanent link">&para;</a></h3>
<p>SV Merging Across Samples</p>
<p><em>homepage</em>: <a href="https://github.com/mkirsche/Jasmine">https://github.com/mkirsche/Jasmine</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Java-15</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jasper">JasPer<a class="headerlink" href="#jasper" title="Permanent link">&para;</a></h3>
<p>The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in  the JPEG-2000 Part-1 standard. </p>
<p><em>homepage</em>: <a href="https://www.ece.uvic.ca/~frodo/jasper/">https://www.ece.uvic.ca/~frodo/jasper/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.900.1</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.10</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.12</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.14</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.16</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.24</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.28</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.33</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="java">Java<a class="headerlink" href="#java" title="Permanent link">&para;</a></h3>
<p>Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers.</p>
<p><em>homepage</em>: <a href="https://java.com/">https://java.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0_24</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_10</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_15</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_21</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_40</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_45</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_60</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_75</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_76</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_79</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0_80</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_20</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_25</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_31</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_40</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_45</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_60</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_65</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_66</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_72</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_74</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_77</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_92</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_112</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_121</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_131</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_141</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_144</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_152</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_162</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_172</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_181</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_192</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_202</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_212</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_221</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_231</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_241</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_271</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_281</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_292</code></td>
<td><code>-OpenJDK</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0_311</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8_191</code></td>
<td><code>-b26-OpenJDK</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8_265</code></td>
<td><code>-b01-OpenJDK-aarch64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9.0.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.345</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.6</code></td>
<td><code>-ppc64le</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.8</code></td>
<td><code>-aarch64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.16</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>13</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>13.0.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>15</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>15.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.0.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.0.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="javafx">JavaFX<a class="headerlink" href="#javafx" title="Permanent link">&para;</a></h3>
<p>OpenJFX is an open source, next generation client application platform for desktop,  mobile and embedded systems built on Java </p>
<p><em>homepage</em>: <a href="https://openjfx.io/">https://openjfx.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.0.2</code></td>
<td><code>_linux-x64_bin-sdk</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="jax">jax<a class="headerlink" href="#jax" title="Permanent link">&para;</a></h3>
<p>Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/jax">https://pypi.python.org/pypi/jax</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.19</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.2.20</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.2.24</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.2.24</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.3.9</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.3.9</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.3.14</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.3.23</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.3.23</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.4.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.4.4</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="jaxfrontce">JAXFrontCE<a class="headerlink" href="#jaxfrontce" title="Permanent link">&para;</a></h3>
<p>JAXFront is a technology to generate graphical user interfaces  on multiple channels (Java Swing, HTML, PDF) on the basis of an XML schema.</p>
<p><em>homepage</em>: <a href="http://www.jaxfront.org/pages/free_community_edition.html">http://www.jaxfront.org/pages/free_community_edition.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.75</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="jbigkit">jbigkit<a class="headerlink" href="#jbigkit" title="Permanent link">&para;</a></h3>
<p>JBIG-KIT is a software implementation of the JBIG1 data  compression standard (ITU-T T.82), which was designed for bi-level image  data, such as scanned documents.</p>
<p><em>homepage</em>: <a href="https://www.cl.cam.ac.uk/~mgk25/jbigkit/">https://www.cl.cam.ac.uk/~mgk25/jbigkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jellyfish">Jellyfish<a class="headerlink" href="#jellyfish" title="Permanent link">&para;</a></h3>
<p>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.</p>
<p><em>homepage</em>: <a href="http://www.genome.umd.edu/jellyfish.html">http://www.genome.umd.edu/jellyfish.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.11</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.1.12</code></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.6</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.10</code></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jemalloc">jemalloc<a class="headerlink" href="#jemalloc" title="Permanent link">&para;</a></h3>
<p>jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and  scalable concurrency support.</p>
<p><em>homepage</em>: <a href="http://jemalloc.net">http://jemalloc.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.0</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.5.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jhbuild">jhbuild<a class="headerlink" href="#jhbuild" title="Permanent link">&para;</a></h3>
<p>JHBuild allows you to automatically download and compile modules (i.e. source code packages).  Modules are listed in module set files, which also include dependency information so that JHBuild can discover  what modules need to be built and in what order.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/action/show/Projects/Jhbuild">https://wiki.gnome.org/action/show/Projects/Jhbuild</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.15.92</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
</tbody>
</table>
<h3 id="jitcode">JiTCODE<a class="headerlink" href="#jitcode" title="Permanent link">&para;</a></h3>
<p>Just-in-time compilation for ordinary/delay/stochastic differential equations (DDEs)</p>
<p><em>homepage</em>: <a href="https://jitcde-common.readthedocs.io">https://jitcde-common.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="jmodeltest">jModelTest<a class="headerlink" href="#jmodeltest" title="Permanent link">&para;</a></h3>
<p>jModelTest is a tool to carry out statistical selection of best-fit models of nucleotide substitution.</p>
<p><em>homepage</em>: <a href="https://github.com/ddarriba/jmodeltest2">https://github.com/ddarriba/jmodeltest2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.10r20160303</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="jorg">Jorg<a class="headerlink" href="#jorg" title="Permanent link">&para;</a></h3>
<p>A MAG Circularization Method By Lauren Lui, Torben Nielsen, and Adam Arkin</p>
<p><em>homepage</em>: <a href="https://github.com/lmlui/Jorg">https://github.com/lmlui/Jorg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="joypy">joypy<a class="headerlink" href="#joypy" title="Permanent link">&para;</a></h3>
<p>Joyplots in Python with matplotlib &amp; pandas</p>
<p><em>homepage</em>: <a href="https://github.com/sbebo/joypy">https://github.com/sbebo/joypy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="jq">jq<a class="headerlink" href="#jq" title="Permanent link">&para;</a></h3>
<p>jq is a lightweight and flexible command-line JSON processor.</p>
<p><em>homepage</em>: <a href="https://stedolan.github.io/jq/">https://stedolan.github.io/jq/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="json-c">json-c<a class="headerlink" href="#json-c" title="Permanent link">&para;</a></h3>
<p>JSON-C implements a reference counting object model that allows you to easily construct JSON objects  in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON objects.</p>
<p><em>homepage</em>: <a href="https://github.com/json-c/json-c">https://github.com/json-c/json-c</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.16</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="json-fortran">json-fortran<a class="headerlink" href="#json-fortran" title="Permanent link">&para;</a></h3>
<p>JSON-Fortran: A Modern Fortran JSON API</p>
<p><em>homepage</em>: <a href="https://github.com/jacobwilliams/json-fortran">https://github.com/jacobwilliams/json-fortran</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.3.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="json-glib">JSON-GLib<a class="headerlink" href="#json-glib" title="Permanent link">&para;</a></h3>
<p>JSON-GLib implements a full JSON parser and generator using GLib and GObject,  and integrates JSON with GLib data types.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/JsonGlib">https://wiki.gnome.org/Projects/JsonGlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jsoncpp">JsonCpp<a class="headerlink" href="#jsoncpp" title="Permanent link">&para;</a></h3>
<p>JsonCpp is a C++ library that allows manipulating JSON values,  including serialization and deserialization to and from strings. It can also preserve existing comment in  unserialization/serialization steps, making it a convenient format to store user input files. </p>
<p><em>homepage</em>: <a href="https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html">https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.7</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.9.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jube">JUBE<a class="headerlink" href="#jube" title="Permanent link">&para;</a></h3>
<p>The JUBE benchmarking environment provides a script based framework to easily create benchmark sets, run those sets on different computer systems and evaluate the results. </p>
<p><em>homepage</em>: <a href="https://www.fz-juelich.de/jsc/jube">https://www.fz-juelich.de/jsc/jube</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="judy">Judy<a class="headerlink" href="#judy" title="Permanent link">&para;</a></h3>
<p>A C library that implements a dynamic array.</p>
<p><em>homepage</em>: <a href="http://judy.sourceforge.net/">http://judy.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="julia">Julia<a class="headerlink" href="#julia" title="Permanent link">&para;</a></h3>
<p>Julia is a high-level, high-performance dynamic programming language for numerical computing</p>
<p><em>homepage</em>: <a href="https://julialang.org">https://julialang.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.2</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.4</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.5</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.6</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.7</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.2</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.5</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="junit">JUnit<a class="headerlink" href="#junit" title="Permanent link">&para;</a></h3>
<p>A programmer-oriented testing framework for Java.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/junit">http://sourceforge.net/projects/junit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.10</code></td>
<td><code>-Java-1.7.0_10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.10</code></td>
<td><code>-Java-1.7.0_21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Java-1.7.0_15</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Java-1.7.0_21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Java-1.7.0_60</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Java-1.7.0_75</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.11</code></td>
<td><code>-Java-1.7.0_79</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_121</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_72</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_77</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyter-contrib-nbextensions">jupyter-contrib-nbextensions<a class="headerlink" href="#jupyter-contrib-nbextensions" title="Permanent link">&para;</a></h3>
<p>A collection of various notebook extensions for Jupyter</p>
<p><em>homepage</em>: <a href="https://github.com/ipython-contrib/jupyter_contrib_nbextensions">https://github.com/ipython-contrib/jupyter_contrib_nbextensions</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyter-matlab-proxy">jupyter-matlab-proxy<a class="headerlink" href="#jupyter-matlab-proxy" title="Permanent link">&para;</a></h3>
<p>MATLAB Integration for Jupyter</p>
<p><em>homepage</em>: <a href="https://github.com/mathworks/jupyter-matlab-proxy">https://github.com/mathworks/jupyter-matlab-proxy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyter-resource-usage">jupyter-resource-usage<a class="headerlink" href="#jupyter-resource-usage" title="Permanent link">&para;</a></h3>
<p>Jupyter Notebook Extension for monitoring your own Resource Usage (memory and/or CPU)</p>
<p><em>homepage</em>: <a href="https://github.com/jupyter-server/jupyter-resource-usage">https://github.com/jupyter-server/jupyter-resource-usage</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.6.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyter-server">jupyter-server<a class="headerlink" href="#jupyter-server" title="Permanent link">&para;</a></h3>
<p>The Jupyter Server provides the backend (i.e. the core services, APIs, and REST endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and Voila.</p>
<p><em>homepage</em>: <a href="https://jupyter.org/">https://jupyter.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.21.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyter-server-proxy">jupyter-server-proxy<a class="headerlink" href="#jupyter-server-proxy" title="Permanent link">&para;</a></h3>
<p>Jupyter Server Proxy lets you run arbitrary external processes (such as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc) alongside your notebook server and provide authenticated web access to them     using a path like /rstudio next to others like /lab. Alongside the python package that provides the main functionality, the JupyterLab extension (@jupyterlab/server-proxy) provides buttons in the JupyterLab launcher window to get to RStudio for example.</p>
<p><em>homepage</em>: <a href="https://github.com/jupyterhub/jupyter-server-proxy">https://github.com/jupyterhub/jupyter-server-proxy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyterhub">JupyterHub<a class="headerlink" href="#jupyterhub" title="Permanent link">&para;</a></h3>
<p>JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed  for centralized deployments in companies, university classrooms and research labs.</p>
<p><em>homepage</em>: <a href="https://jupyter.org">https://jupyter.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyterlab">JupyterLab<a class="headerlink" href="#jupyterlab" title="Permanent link">&para;</a></h3>
<p>JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar  building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,  etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter  Notebook.</p>
<p><em>homepage</em>: <a href="https://jupyter.org/">https://jupyter.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.2.8</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.0.16</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.1.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.8</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyterlab-lmod">jupyterlab-lmod<a class="headerlink" href="#jupyterlab-lmod" title="Permanent link">&para;</a></h3>
<p>JupyterLab extension that allows user to interact with environment modules before launching kernels. The extension use Lmod's Python interface to accomplish module related task like loading, unloading, saving collection, etc.</p>
<p><em>homepage</em>: <a href="https://github.com/cmd-ntrf/jupyter-lmod">https://github.com/cmd-ntrf/jupyter-lmod</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jupyterlmod">jupyterlmod<a class="headerlink" href="#jupyterlmod" title="Permanent link">&para;</a></h3>
<p>Jupyter interactive notebook server extension that allows user to interact with environment modules before launching kernels. The extension use Lmod's Python interface to accomplish module related task like loading, unloading, saving collection, etc.</p>
<p><em>homepage</em>: <a href="https://github.com/cmd-ntrf/jupyter-lmod">https://github.com/cmd-ntrf/jupyter-lmod</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="jwm">JWM<a class="headerlink" href="#jwm" title="Permanent link">&para;</a></h3>
<p>JWM is a light-weight window manager for the X11 Window System.</p>
<p><em>homepage</em>: <a href="https://joewing.net/projects/jwm/">https://joewing.net/projects/jwm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.5</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="jxrlib">jxrlib<a class="headerlink" href="#jxrlib" title="Permanent link">&para;</a></h3>
<p>Open source implementation of jpegxr</p>
<p><em>homepage</em>: <a href="https://github.com/4creators/jxrlib">https://github.com/4creators/jxrlib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="k">K<a class="headerlink" href="#k" title="Permanent link">&para;</a></h2>
<p><a href="#kahip">KaHIP</a> - <a href="#kaiju">Kaiju</a> - <a href="#kaleido">Kaleido</a> - <a href="#kalign">Kalign</a> - <a href="#kallisto">kallisto</a> - <a href="#kat">KAT</a> - <a href="#kb-python">kb-python</a> - <a href="#kbproto">kbproto</a> - <a href="#kedro">kedro</a> - <a href="#kent_tools">Kent_tools</a> - <a href="#keras">Keras</a> - <a href="#khmer">khmer</a> - <a href="#kim-api">kim-api</a> - <a href="#kineto">kineto</a> - <a href="#king">king</a> - <a href="#kite">KITE</a> - <a href="#kma">kma</a> - <a href="#kmc">KMC</a> - <a href="#kmcp">KMCP</a> - <a href="#kmergenie">KmerGenie</a> - <a href="#knime">KNIME</a> - <a href="#kpcalg">kpcalg</a> - <a href="#kraken">Kraken</a> - <a href="#kraken2">Kraken2</a> - <a href="#kratos">Kratos</a> - <a href="#krbalancing">krbalancing</a> - <a href="#kronatools">KronaTools</a> - <a href="#kwant">kwant</a> - <a href="#kwiml">KWIML</a> - <a href="#kwip">kWIP</a> - <a href="#kyotocabinet">KyotoCabinet</a></p>
<h3 id="kahip">KaHIP<a class="headerlink" href="#kahip" title="Permanent link">&para;</a></h3>
<p>The graph partitioning framework KaHIP -- Karlsruhe High Quality Partitioning.</p>
<p><em>homepage</em>: <a href="https://kahip.github.io/">https://kahip.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.14</code></td>
<td><code>gompi/2022a</code>, <code>gompi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="kaiju">Kaiju<a class="headerlink" href="#kaiju" title="Permanent link">&para;</a></h3>
<p>Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments</p>
<p><em>homepage</em>: <a href="http://kaiju.binf.ku.dk/">http://kaiju.binf.ku.dk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="kaleido">Kaleido<a class="headerlink" href="#kaleido" title="Permanent link">&para;</a></h3>
<p>Fast static image export for web-based visualization libraries with zero dependencies</p>
<p><em>homepage</em>: <a href="https://github.com/plotly/Kaleido">https://github.com/plotly/Kaleido</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.2.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kalign">Kalign<a class="headerlink" href="#kalign" title="Permanent link">&para;</a></h3>
<p>Kalign is a fast multiple sequence alignment program for biological sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/TimoLassmann/kalign">https://github.com/TimoLassmann/kalign</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.4</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kallisto">kallisto<a class="headerlink" href="#kallisto" title="Permanent link">&para;</a></h3>
<p>kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally  of target sequences using high-throughput sequencing reads.</p>
<p><em>homepage</em>: <a href="https://pachterlab.github.io/kallisto/">https://pachterlab.github.io/kallisto/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.42.5</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.43.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.43.1</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.44.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.45.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.45.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.46.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.46.1</code></td>
<td><code>foss/2019b</code>, <code>iimpi/2020a</code>, <code>iimpi/2020b</code></td>
</tr>
<tr>
<td><code>0.46.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.48.0</code></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="kat">KAT<a class="headerlink" href="#kat" title="Permanent link">&para;</a></h3>
<p>The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.</p>
<p><em>homepage</em>: <a href="https://www.earlham.ac.uk/kat-tools">https://www.earlham.ac.uk/kat-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="kb-python">kb-python<a class="headerlink" href="#kb-python" title="Permanent link">&para;</a></h3>
<p>kallisto | bustools is a workflow for pre-processing single-cell RNA-seq data.  Pre-processing single-cell RNA-seq involves: (1) association of reads with their cells of origin,  (2) collapsing of reads according to unique molecular identifiers (UMIs), and (3) generation of  gene or feature counts from the reads to generate a cell x gene matrix.</p>
<p><em>homepage</em>: <a href="https://www.kallistobus.tools/">https://www.kallistobus.tools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.27.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="kbproto">kbproto<a class="headerlink" href="#kbproto" title="Permanent link">&para;</a></h3>
<p>X.org KBProto protocol headers.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.7</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="kedro">kedro<a class="headerlink" href="#kedro" title="Permanent link">&para;</a></h3>
<p>Kedro is an open-source Python framework that applies software engineering best-practice to data and machine-learning pipelines. </p>
<p><em>homepage</em>: <a href="https://github.com/quantumblacklabs/kedro">https://github.com/quantumblacklabs/kedro</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="kent_tools">Kent_tools<a class="headerlink" href="#kent_tools" title="Permanent link">&para;</a></h3>
<p>Kent utilities: collection of tools used by the UCSC genome browser.</p>
<p><em>homepage</em>: <a href="https://genome.cse.ucsc.edu/">https://genome.cse.ucsc.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>401</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>411</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>418</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>422</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>442</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>20130806</code></td>
<td><code>-linux.x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20171107</code></td>
<td><code>-linux.x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20180716</code></td>
<td><code>-linux.x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20190326</code></td>
<td><code>-linux.x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="keras">Keras<a class="headerlink" href="#keras" title="Permanent link">&para;</a></h3>
<p>Keras is a deep learning API written in Python, running on top of the machine learning platform TensorFlow. </p>
<p><em>homepage</em>: <a href="https://keras.io/">https://keras.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.8</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.8</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="khmer">khmer<a class="headerlink" href="#khmer" title="Permanent link">&para;</a></h3>
<p>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more </p>
<p><em>homepage</em>: <a href="https://github.com/ged-lab/khmer/">https://github.com/ged-lab/khmer/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="kim-api">kim-api<a class="headerlink" href="#kim-api" title="Permanent link">&para;</a></h3>
<p>Open Knowledgebase of Interatomic Models.  KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations.  This is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  </p>
<p><em>homepage</em>: <a href="https://openkim.org/">https://openkim.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kineto">kineto<a class="headerlink" href="#kineto" title="Permanent link">&para;</a></h3>
<p>A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters</p>
<p><em>homepage</em>: <a href="https://github.com/pytorch/kineto">https://github.com/pytorch/kineto</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="king">king<a class="headerlink" href="#king" title="Permanent link">&para;</a></h3>
<p>KING is a toolset that makes use of high-throughput  SNP data typically seen in a genome-wide association study (GWAS) or  a sequencing project. Applications of KING include family relationship  inference and pedigree error checking, quality control, population  substructure identification, forensics, gene mapping, etc.</p>
<p><em>homepage</em>: <a href="https://kingrelatedness.com/">https://kingrelatedness.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="kite">KITE<a class="headerlink" href="#kite" title="Permanent link">&para;</a></h3>
<p>KITE is an open-source Python/C++ software suite for efficient real-space tight-binding (TB)  simulations of electronic structure and bulk quantum transport properties of disordered systems scalable to  multi billions of atomic orbitals.</p>
<p><em>homepage</em>: <a href="https://github.com/quantum-kite/kite">https://github.com/quantum-kite/kite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="kma">kma<a class="headerlink" href="#kma" title="Permanent link">&para;</a></h3>
<p>KMA is a mapping method designed to map raw reads directly against redundant databases,  in an ultra-fast manner using seed and extend.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/genomicepidemiology/kma">https://bitbucket.org/genomicepidemiology/kma</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.22</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="kmc">KMC<a class="headerlink" href="#kmc" title="Permanent link">&para;</a></h3>
<p>KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.</p>
<p><em>homepage</em>: <a href="http://sun.aei.polsl.pl/kmc">http://sun.aei.polsl.pl/kmc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.1.2rc1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kmcp">KMCP<a class="headerlink" href="#kmcp" title="Permanent link">&para;</a></h3>
<p>KMCP: accurate metagenomic profiling of both prokaryotic and viral populations by pseudo-mapping </p>
<p><em>homepage</em>: <a href="https://bioinf.shenwei.me/kmcp">https://bioinf.shenwei.me/kmcp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="kmergenie">KmerGenie<a class="headerlink" href="#kmergenie" title="Permanent link">&para;</a></h3>
<p>KmerGenie estimates the best k-mer length for genome de novo assembly.</p>
<p><em>homepage</em>: <a href="http://kmergenie.bx.psu.edu/">http://kmergenie.bx.psu.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7044</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.7048</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="knime">KNIME<a class="headerlink" href="#knime" title="Permanent link">&para;</a></h3>
<p>KNIME Analytics Platform is the open source software for creating data science applications and services. KNIME stands for KoNstanz Information MinEr.  </p>
<p><em>homepage</em>: <a href="https://www.knime.com/">https://www.knime.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="kpcalg">kpcalg<a class="headerlink" href="#kpcalg" title="Permanent link">&para;</a></h3>
<p>Kernel PC (kPC) algorithm for causal structure learning and  causal inference using graphical models. kPC is a version of PC algorithm that  uses kernel based independence criteria in order to be able to deal with  non-linear relationships and non-Gaussian noise.  Includes pcalg: Functions for causal structure learning and causal inference  using graphical models.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=kpcalg">https://cran.r-project.org/package=kpcalg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="kraken">Kraken<a class="headerlink" href="#kraken" title="Permanent link">&para;</a></h3>
<p>Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/kraken/">https://ccb.jhu.edu/software/kraken/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.5-beta</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.10.5-beta</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kraken2">Kraken2<a class="headerlink" href="#kraken2" title="Permanent link">&para;</a></h3>
<p>Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.</p>
<p><em>homepage</em>: <a href="https://github.com/DerrickWood/kraken2/wiki">https://github.com/DerrickWood/kraken2/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.6-beta</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.0.7-beta</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.8-beta</code></td>
<td><code>-Perl-5.30.0</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.0.9-beta</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.9-beta</code></td>
<td><code>-Perl-5.30.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="kratos">Kratos<a class="headerlink" href="#kratos" title="Permanent link">&para;</a></h3>
<p>Kratos Multiphysics (A.K.A Kratos) is a framework for building parallel multi-disciplinary  simulation software.</p>
<p><em>homepage</em>: <a href="https://www.cimne.com/kratos">https://www.cimne.com/kratos</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="krbalancing">krbalancing<a class="headerlink" href="#krbalancing" title="Permanent link">&para;</a></h3>
<p>A C++ extension for Python which computes K.R. balanced matrices.</p>
<p><em>homepage</em>: <a href="https://github.com/deeptools/Knight-Ruiz-Matrix-balancing-algorithm">https://github.com/deeptools/Knight-Ruiz-Matrix-balancing-algorithm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0b0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kronatools">KronaTools<a class="headerlink" href="#kronatools" title="Permanent link">&para;</a></h3>
<p>Krona Tools is a set of scripts to create Krona charts from  several Bioinformatics tools as well as from text and XML files.</p>
<p><em>homepage</em>: <a href="https://github.com/marbl/Krona/wiki/KronaTools">https://github.com/marbl/Krona/wiki/KronaTools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td><code>GCC/10.3.0</code>, <code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kwant">kwant<a class="headerlink" href="#kwant" title="Permanent link">&para;</a></h3>
<p>Kwant is a free (open source), powerful, and easy to use Python package for   numerical calculations on tight-binding models with a strong focus on quantum transport.</p>
<p><em>homepage</em>: <a href="https://kwant-project.org/">https://kwant-project.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="kwiml">KWIML<a class="headerlink" href="#kwiml" title="Permanent link">&para;</a></h3>
<p>The Kitware Information Macro Library</p>
<p><em>homepage</em>: <a href="https://gitlab.kitware.com/utils/kwiml">https://gitlab.kitware.com/utils/kwiml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180201</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kwip">kWIP<a class="headerlink" href="#kwip" title="Permanent link">&para;</a></h3>
<p>This software implements a de novo, alignment free measure of sample  genetic dissimilarity which operates upon raw sequencing reads.  It is able to calculate the genetic dissimilarity between samples without any reference genome,  and without assembling one.</p>
<p><em>homepage</em>: <a href="https://github.com/kdmurray91/kWIP">https://github.com/kdmurray91/kWIP</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="kyotocabinet">KyotoCabinet<a class="headerlink" href="#kyotocabinet" title="Permanent link">&para;</a></h3>
<p>Kyoto Cabinet is a library of routines for managing a database.</p>
<p><em>homepage</em>: <a href="https://fallabs.com/kyotocabinet">https://fallabs.com/kyotocabinet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.77</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="l">L<a class="headerlink" href="#l" title="Permanent link">&para;</a></h2>
<p><a href="#l_rna_scaffolder">L_RNA_scaffolder</a> - <a href="#lace">Lace</a> - <a href="#ladr">LADR</a> - <a href="#lagrangian-filtering">lagrangian-filtering</a> - <a href="#lame">LAME</a> - <a href="#lammps">LAMMPS</a> - <a href="#lancet">lancet</a> - <a href="#lapack">LAPACK</a> - <a href="#lasso-python">LASSO-Python</a> - <a href="#last">LAST</a> - <a href="#lastz">LASTZ</a> - <a href="#lavaan">lavaan</a> - <a href="#lbfgs++">LBFGS++</a> - <a href="#lcov">LCov</a> - <a href="#ldc">LDC</a> - <a href="#lddt">lDDT</a> - <a href="#leadit">LeadIT</a> - <a href="#leidenalg">leidenalg</a> - <a href="#lemon">LEMON</a> - <a href="#leptonica">Leptonica</a> - <a href="#lerc">LERC</a> - <a href="#less">less</a> - <a href="#leveldb">LevelDB</a> - <a href="#lftp">lftp</a> - <a href="#lhapdf">LHAPDF</a> - <a href="#libaec">libaec</a> - <a href="#libaio">libaio</a> - <a href="#libarchive">libarchive</a> - <a href="#libav">libav</a> - <a href="#libavif">libavif</a> - <a href="#libbigwig">libBigWig</a> - <a href="#libbitmask">libbitmask</a> - <a href="#libcdms">libcdms</a> - <a href="#libcerf">libcerf</a> - <a href="#libcint">libcint</a> - <a href="#libcircle">libcircle</a> - <a href="#libcmaes">libcmaes</a> - <a href="#libconfig">libconfig</a> - <a href="#libcpuset">libcpuset</a> - <a href="#libcroco">libcroco</a> - <a href="#libctl">libctl</a> - <a href="#libdap">libdap</a> - <a href="#libde265">libde265</a> - <a href="#libdeflate">libdeflate</a> - <a href="#libdivsufsort">libdivsufsort</a> - <a href="#libdrm">libdrm</a> - <a href="#libdrs">libdrs</a> - <a href="#libdwarf">libdwarf</a> - <a href="#libedit">libedit</a> - <a href="#libelf">libelf</a> - <a href="#libemf">libemf</a> - <a href="#libepoxy">libepoxy</a> - <a href="#libev">libev</a> - <a href="#libevent">libevent</a> - <a href="#libexif">libexif</a> - <a href="#libfabric">libfabric</a> - <a href="#libfdf">libfdf</a> - <a href="#libffcall">libffcall</a> - <a href="#libffi">libffi</a> - <a href="#libflame">libFLAME</a> - <a href="#libfontenc">libfontenc</a> - <a href="#libgcrypt">libgcrypt</a> - <a href="#libgd">libgd</a> - <a href="#libgdiplus">libgdiplus</a> - <a href="#libgdsii">libGDSII</a> - <a href="#libgeotiff">libgeotiff</a> - <a href="#libgit2">libgit2</a> - <a href="#libglade">libglade</a> - <a href="#libglu">libGLU</a> - <a href="#libglvnd">libglvnd</a> - <a href="#libgpg-error">libgpg-error</a> - <a href="#libgpuarray">libgpuarray</a> - <a href="#libgridxc">libGridXC</a> - <a href="#libgtextutils">libgtextutils</a> - <a href="#libharu">libharu</a> - <a href="#libheif">libheif</a> - <a href="#libibmad">libibmad</a> - <a href="#libibumad">libibumad</a> - <a href="#libice">libICE</a> - <a href="#libiconv">libiconv</a> - <a href="#libidn">libidn</a> - <a href="#libidn2">libidn2</a> - <a href="#libint">Libint</a> - <a href="#libis">LiBis</a> - <a href="#libjpeg-turbo">libjpeg-turbo</a> - <a href="#libjxl">libjxl</a> - <a href="#liblzf">LibLZF</a> - <a href="#libmad">libmad</a> - <a href="#libmatheval">libmatheval</a> - <a href="#libmaus2">libmaus2</a> - <a href="#libmbd">libmbd</a> - <a href="#libmemcached">libMemcached</a> - <a href="#libmicrohttpd">libmicrohttpd</a> - <a href="#libmo_unpack">libmo_unpack</a> - <a href="#libmypaint">libmypaint</a> - <a href="#libnsl">libnsl</a> - <a href="#libobjcryst">libobjcryst</a> - <a href="#libogg">libogg</a> - <a href="#libopus">libopus</a> - <a href="#libosmium">libosmium</a> - <a href="#libpci">libpci</a> - <a href="#libpciaccess">libpciaccess</a> - <a href="#libplinkio">libplinkio</a> - <a href="#libpng">libpng</a> - <a href="#libpsl">libpsl</a> - <a href="#libpsml">libPSML</a> - <a href="#libpsortb">libpsortb</a> - <a href="#libpspio">libpspio</a> - <a href="#libpthread-stubs">libpthread-stubs</a> - <a href="#libqglviewer">libQGLViewer</a> - <a href="#libreadline">libreadline</a> - <a href="#librmath">libRmath</a> - <a href="#librosa">librosa</a> - <a href="#librsb">librsb</a> - <a href="#librsvg">librsvg</a> - <a href="#librttopo">librttopo</a> - <a href="#libsamplerate">libsamplerate</a> - <a href="#libsbml">libSBML</a> - <a href="#libsigc++">libsigc++</a> - <a href="#libsigsegv">libsigsegv</a> - <a href="#libsm">libSM</a> - <a href="#libsndfile">libsndfile</a> - <a href="#libsodium">libsodium</a> - <a href="#libsoup">LibSoup</a> - <a href="#libspatialindex">libspatialindex</a> - <a href="#libspatialite">libspatialite</a> - <a href="#libssh">libssh</a> - <a href="#libstatgen">libStatGen</a> - <a href="#libsvm">LIBSVM</a> - <a href="#libtar">libtar</a> - <a href="#libtasn1">libtasn1</a> - <a href="#libtecla">libtecla</a> - <a href="#libtiff">LibTIFF</a> - <a href="#libtirpc">libtirpc</a> - <a href="#libtool">libtool</a> - <a href="#libtree">libtree</a> - <a href="#libunistring">libunistring</a> - <a href="#libunwind">libunwind</a> - <a href="#libutempter">libutempter</a> - <a href="#libuuid">LibUUID</a> - <a href="#libuv">libuv</a> - <a href="#libvdwxc">libvdwxc</a> - <a href="#libvorbis">libvorbis</a> - <a href="#libwebp">libwebp</a> - <a href="#libwpe">libwpe</a> - <a href="#libx11">libX11</a> - <a href="#libxau">libXau</a> - <a href="#libxc">libxc</a> - <a href="#libxcb">libxcb</a> - <a href="#libxcursor">libXcursor</a> - <a href="#libxdamage">libXdamage</a> - <a href="#libxdmcp">libXdmcp</a> - <a href="#libxext">libXext</a> - <a href="#libxfixes">libXfixes</a> - <a href="#libxfont">libXfont</a> - <a href="#libxft">libXft</a> - <a href="#libxi">libXi</a> - <a href="#libxinerama">libXinerama</a> - <a href="#libxkbcommon">libxkbcommon</a> - <a href="#libxml++">libxml++</a> - <a href="#libxml2">libxml2</a> - <a href="#libxml2-python">libxml2-python</a> - <a href="#libxmu">libXmu</a> - <a href="#libxp">libXp</a> - <a href="#libxpm">libXpm</a> - <a href="#libxrandr">libXrandr</a> - <a href="#libxrender">libXrender</a> - <a href="#libxslt">libxslt</a> - <a href="#libxsmm">libxsmm</a> - <a href="#libxt">libXt</a> - <a href="#libxxf86vm">libXxf86vm</a> - <a href="#libyaml">libyaml</a> - <a href="#libzeep">libzeep</a> - <a href="#libzip">libzip</a> - <a href="#lie_learn">lie_learn</a> - <a href="#lifelines">lifelines</a> - <a href="#lighter">Lighter</a> - <a href="#liknorm">liknorm</a> - <a href="#likwid">likwid</a> - <a href="#limix">limix</a> - <a href="#linbox">LinBox</a> - <a href="#line_profiler">line_profiler</a> - <a href="#lingeling">Lingeling</a> - <a href="#littlecms">LittleCMS</a> - <a href="#lldb">LLDB</a> - <a href="#llvm">LLVM</a> - <a href="#lmdb">LMDB</a> - <a href="#lmfit">LMfit</a> - <a href="#lmod">Lmod</a> - <a href="#lncloom">LncLOOM</a> - <a href="#locarna">LocARNA</a> - <a href="#lofreq">LoFreq</a> - <a href="#log-log4perl">Log-Log4perl</a> - <a href="#logaddexp">logaddexp</a> - <a href="#lohhla">LOHHLA</a> - <a href="#loki">Loki</a> - <a href="#longestrunsubsequence">longestrunsubsequence</a> - <a href="#longread_umi">longread_umi</a> - <a href="#longshot">Longshot</a> - <a href="#loompy">loompy</a> - <a href="#loomr">loomR</a> - <a href="#lpeg">LPeg</a> - <a href="#lpjml">LPJmL</a> - <a href="#lpsolve">lpsolve</a> - <a href="#lrslib">lrslib</a> - <a href="#ls-prepost">LS-PrePost</a> - <a href="#lsd2">LSD2</a> - <a href="#lsms">LSMS</a> - <a href="#ltr_retriever">LTR_retriever</a> - <a href="#ltrdetector">LtrDetector</a> - <a href="#lua">Lua</a> - <a href="#luajit">LuaJIT</a> - <a href="#luajit2-openresty">LuaJIT2-OpenResty</a> - <a href="#lucene-geo-gazetteer">Lucene-Geo-Gazetteer</a> - <a href="#lumpy">LUMPY</a> - <a href="#luscus">LUSCUS</a> - <a href="#lwgrp">lwgrp</a> - <a href="#lxml">lxml</a> - <a href="#lynx">lynx</a> - <a href="#lz4">lz4</a> - <a href="#lzo">LZO</a></p>
<h3 id="l_rna_scaffolder">L_RNA_scaffolder<a class="headerlink" href="#l_rna_scaffolder" title="Permanent link">&para;</a></h3>
<p>L_RNA_scaffolder is a novel scaffolding tool using long trancriptome reads to scaffold genome fragments</p>
<p><em>homepage</em>: <a href="http://www.fishbrowser.org/software/L_RNA_scaffolder/">http://www.fishbrowser.org/software/L_RNA_scaffolder/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20141124</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="lace">Lace<a class="headerlink" href="#lace" title="Permanent link">&para;</a></h3>
<p>Building SuperTranscripts: A linear representation of transcriptome data</p>
<p><em>homepage</em>: <a href="https://github.com/Oshlack/Lace">https://github.com/Oshlack/Lace</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ladr">LADR<a class="headerlink" href="#ladr" title="Permanent link">&para;</a></h3>
<p>Prover9, Mace4, and several related programs come packaged in  a system called LADR (Library for Automated Deduction Research). </p>
<p><em>homepage</em>: <a href="https://www.cs.unm.edu/~mccune/mace4/download">https://www.cs.unm.edu/~mccune/mace4/download</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2009-11A</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lagrangian-filtering">lagrangian-filtering<a class="headerlink" href="#lagrangian-filtering" title="Permanent link">&para;</a></h3>
<p>Temporal filtering of data in a Lagrangian frame of reference.</p>
<p><em>homepage</em>: <a href="https://github.com/angus-g/lagrangian-filtering">https://github.com/angus-g/lagrangian-filtering</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="lame">LAME<a class="headerlink" href="#lame" title="Permanent link">&para;</a></h3>
<p>LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.</p>
<p><em>homepage</em>: <a href="http://lame.sourceforge.net/">http://lame.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.99.5</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.100</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="lammps">LAMMPS<a class="headerlink" href="#lammps" title="Permanent link">&para;</a></h3>
<p>LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality. </p>
<p><em>homepage</em>: <a href="https://lammps.sandia.gov/">https://lammps.sandia.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3Mar2020</code></td>
<td><code>-Python-3.7.4-kokkos</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3Mar2020</code></td>
<td><code>-Python-3.8.2-kokkos</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7Aug2019</code></td>
<td><code>-Python-3.7.4-kokkos</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>7Aug2019</code></td>
<td><code>-Python-3.7.4-kokkos-OCTP</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>23Jun2022</code></td>
<td><code>-kokkos</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>23Jun2022</code></td>
<td><code>-kokkos-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>23Jun2022</code></td>
<td><code>-kokkos-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="lancet">lancet<a class="headerlink" href="#lancet" title="Permanent link">&para;</a></h3>
<p>Lancet is a somatic variant caller (SNVs and indels) for short read data.</p>
<p><em>homepage</em>: <a href="https://github.com/nygenome/lancet">https://github.com/nygenome/lancet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="lapack">LAPACK<a class="headerlink" href="#lapack" title="Permanent link">&para;</a></h3>
<p>LAPACK is written in Fortran90 and provides routines for solving systems of  simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue  problems, and singular value problems.</p>
<p><em>homepage</em>: <a href="https://www.netlib.org/lapack/">https://www.netlib.org/lapack/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>3.10.1</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lasso-python">LASSO-Python<a class="headerlink" href="#lasso-python" title="Permanent link">&para;</a></h3>
<p>This python library is designed for general purpose usage in the field of Computer Aided Engineering (CAE). It's name originates from the original initiator and donator of the project Lasso GmbH. The library is now maintained by an open-source community.</p>
<p><em>homepage</em>: <a href="https://open-lasso-python.github.io/lasso-python/">https://open-lasso-python.github.io/lasso-python/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="last">LAST<a class="headerlink" href="#last" title="Permanent link">&para;</a></h3>
<p>LAST finds similar regions between sequences.</p>
<p><em>homepage</em>: <a href="http://last.cbrc.jp/">http://last.cbrc.jp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>869</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>914</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1045</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1179</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lastz">LASTZ<a class="headerlink" href="#lastz" title="Permanent link">&para;</a></h3>
<p>LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle  sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS   sequencing technologies such as Roche 454. </p>
<p><em>homepage</em>: <a href="https://www.bx.psu.edu/~rsharris/lastz/">https://www.bx.psu.edu/~rsharris/lastz/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.02.00</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.04.03</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="lavaan">lavaan<a class="headerlink" href="#lavaan" title="Permanent link">&para;</a></h3>
<p>Fit a variety of latent variable models, including confirmatory factor analysis, structural  equation modeling and latent growth curve models.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/lavaan">https://cran.r-project.org/web/packages/lavaan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6-2</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.6-4.1433</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.6-9</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="lbfgs">LBFGS++<a class="headerlink" href="#lbfgs" title="Permanent link">&para;</a></h3>
<p>A header-only C++ library for L-BFGS and L-BFGS-B algorithms</p>
<p><em>homepage</em>: <a href="https://lbfgspp.statr.me">https://lbfgspp.statr.me</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="lcov">LCov<a class="headerlink" href="#lcov" title="Permanent link">&para;</a></h3>
<p>LCOV - the LTP GCOV extension</p>
<p><em>homepage</em>: <a href="http://ltp.sourceforge.net/coverage/lcov.php">http://ltp.sourceforge.net/coverage/lcov.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13</code></td>
<td><code>GCCcore/7.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ldc">LDC<a class="headerlink" href="#ldc" title="Permanent link">&para;</a></h3>
<p>The LLVM-based D Compiler</p>
<p><em>homepage</em>: <a href="https://wiki.dlang.org/LDC">https://wiki.dlang.org/LDC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.6</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.24.0</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.25.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.26.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.30.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lddt">lDDT<a class="headerlink" href="#lddt" title="Permanent link">&para;</a></h3>
<p>The local Distance Difference Test (lDDT) is a superposition-free score which evaluates local distance differences in a model compared to a reference structure.</p>
<p><em>homepage</em>: <a href="https://openstructure.org">https://openstructure.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="leadit">LeadIT<a class="headerlink" href="#leadit" title="Permanent link">&para;</a></h3>
<p>Visually Informed LeadOpt</p>
<p><em>homepage</em>: <a href="http://www.biosolveit.de/LeadIT/index.html">http://www.biosolveit.de/LeadIT/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.9</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="leidenalg">leidenalg<a class="headerlink" href="#leidenalg" title="Permanent link">&para;</a></h3>
<p>Implementation of the Leiden algorithm for various quality functions to be used with igraph in Python.</p>
<p><em>homepage</em>: <a href="https://github.com/vtraag/leidenalg">https://github.com/vtraag/leidenalg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.8.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.8.7</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.8</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="lemon">LEMON<a class="headerlink" href="#lemon" title="Permanent link">&para;</a></h3>
<p>LEMON stands for Library for Efficient Modeling and Optimization in Networks.  It is a C++ template library providing efficient implementations of common data structures and algorithms  with focus on combinatorial optimization tasks connected mainly with graphs and networks. </p>
<p><em>homepage</em>: <a href="https://lemon.cs.elte.hu">https://lemon.cs.elte.hu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="leptonica">Leptonica<a class="headerlink" href="#leptonica" title="Permanent link">&para;</a></h3>
<p>Leptonica is a collection of pedagogically-oriented open source software  that is broadly useful for image processing and image analysis applications.</p>
<p><em>homepage</em>: <a href="http://www.leptonica.org">http://www.leptonica.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.77.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.78.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.82.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.83.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lerc">LERC<a class="headerlink" href="#lerc" title="Permanent link">&para;</a></h3>
<p>LERC is an open-source image or raster format which supports rapid encoding and decoding for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding, so the precision of the original input image is preserved (within user defined error bounds).</p>
<p><em>homepage</em>: <a href="https://github.com/Esri/lerc">https://github.com/Esri/lerc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="less">less<a class="headerlink" href="#less" title="Permanent link">&para;</a></h3>
<p>Less is a free, open-source file pager. It can be found on most versions of Linux,  Unix and Mac OS, as well as on many other operating systems.</p>
<p><em>homepage</em>: <a href="http://www.greenwoodsoftware.com/less/">http://www.greenwoodsoftware.com/less/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>458</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="leveldb">LevelDB<a class="headerlink" href="#leveldb" title="Permanent link">&para;</a></h3>
<p>LevelDB is a fast key-value storage library written at Google that provides an  ordered mapping from string keys to string values.</p>
<p><em>homepage</em>: <a href="https://github.com/google/leveldb">https://github.com/google/leveldb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.18</code></td>
<td><code>foss/2016a</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.20</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.22</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lftp">lftp<a class="headerlink" href="#lftp" title="Permanent link">&para;</a></h3>
<p>LFTP is a sophisticated ftp/http client, and a file transfer program supporting a number of network protocols. Like BASH, it has job control and uses the readline library for input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel. It was designed with reliability in mind.</p>
<p><em>homepage</em>: <a href="https://lftp.yar.ru">https://lftp.yar.ru</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.4</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>4.8.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lhapdf">LHAPDF<a class="headerlink" href="#lhapdf" title="Permanent link">&para;</a></h3>
<p>Les Houches Parton Density Function  LHAPDF is the standard tool for evaluating parton distribution functions (PDFs) in high-energy physics. </p>
<p><em>homepage</em>: <a href="http://lhapdf.hepforge.org/">http://lhapdf.hepforge.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.5.3</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libaec">libaec<a class="headerlink" href="#libaec" title="Permanent link">&para;</a></h3>
<p>Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa.</p>
<p><em>homepage</em>: <a href="https://gitlab.dkrz.de/k202009/libaec">https://gitlab.dkrz.de/k202009/libaec</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libaio">libaio<a class="headerlink" href="#libaio" title="Permanent link">&para;</a></h3>
<p>Asynchronous input/output library that uses the kernels native interface.</p>
<p><em>homepage</em>: <a href="https://pagure.io/libaio">https://pagure.io/libaio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.111</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.3.112</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libarchive">libarchive<a class="headerlink" href="#libarchive" title="Permanent link">&para;</a></h3>
<p>Multi-format archive and compression library </p>
<p><em>homepage</em>: <a href="https://www.libarchive.org/">https://www.libarchive.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libav">libav<a class="headerlink" href="#libav" title="Permanent link">&para;</a></h3>
<p>Libav is a friendly and community-driven effort to provide its users with  a set of portable, functional and high-performance libraries for dealing  with multimedia formats of all sorts. </p>
<p><em>homepage</em>: <a href="https://libav.org/">https://libav.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.10</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libavif">libavif<a class="headerlink" href="#libavif" title="Permanent link">&para;</a></h3>
<p>This library aims to be a friendly, portable C implementation of the AV1 Image File Format, as described here: https://aomediacodec.github.io/av1-avif/ </p>
<p><em>homepage</em>: <a href="https://github.com/AOMediaCodec/libavif">https://github.com/AOMediaCodec/libavif</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="libbigwig">libBigWig<a class="headerlink" href="#libbigwig" title="Permanent link">&para;</a></h3>
<p>A C library for handling bigWig files</p>
<p><em>homepage</em>: <a href="https://github.com/dpryan79/libBigWig">https://github.com/dpryan79/libBigWig</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.4.6</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libbitmask">libbitmask<a class="headerlink" href="#libbitmask" title="Permanent link">&para;</a></h3>
<p>libbitmask provides a convenient, powerful bitmask data type</p>
<p><em>homepage</em>: <a href="http://oss.sgi.com/projects/cpusets/">http://oss.sgi.com/projects/cpusets/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="libcdms">libcdms<a class="headerlink" href="#libcdms" title="Permanent link">&para;</a></h3>
<p>Climate Data Management System Library.</p>
<p><em>homepage</em>: <a href="https://github.com/CDAT/libcdms/">https://github.com/CDAT/libcdms/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="libcerf">libcerf<a class="headerlink" href="#libcerf" title="Permanent link">&para;</a></h3>
<p>libcerf is a self-contained numeric library that provides an efficient and  accurate implementation of complex error functions, along with Dawson,  Faddeeva, and Voigt functions. </p>
<p><em>homepage</em>: <a href="https://jugit.fz-juelich.de/mlz/libcerf">https://jugit.fz-juelich.de/mlz/libcerf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.13</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.15</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libcint">libcint<a class="headerlink" href="#libcint" title="Permanent link">&para;</a></h3>
<p>libcint is an open source library for analytical Gaussian integrals.</p>
<p><em>homepage</em>: <a href="https://github.com/sunqm/libcint">https://github.com/sunqm/libcint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.0</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>gomkl/2021a</code></td>
</tr>
<tr>
<td><code>5.1.6</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="libcircle">libcircle<a class="headerlink" href="#libcircle" title="Permanent link">&para;</a></h3>
<p>An API to provide an efficient distributed queue on a cluster. libcircle is an  API for distributing embarrassingly parallel workloads using self-stabilization. </p>
<p><em>homepage</em>: <a href="https://github.com/hpc/libcircle/">https://github.com/hpc/libcircle/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1-rc.1</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>0.3</code></td>
<td><code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>iimpi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="libcmaes">libcmaes<a class="headerlink" href="#libcmaes" title="Permanent link">&para;</a></h3>
<p>libcmaes is a multithreaded C++11 library for high performance blackbox stochastic optimization using the CMA-ES algorithm for Covariance Matrix Adaptation Evolution Strategy.</p>
<p><em>homepage</em>: <a href="http://beniz.github.io/libcmaes/">http://beniz.github.io/libcmaes/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.5</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libconfig">libconfig<a class="headerlink" href="#libconfig" title="Permanent link">&para;</a></h3>
<p>Libconfig is a simple library for processing structured configuration files</p>
<p><em>homepage</em>: <a href="https://hyperrealm.github.io/libconfig">https://hyperrealm.github.io/libconfig</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libcpuset">libcpuset<a class="headerlink" href="#libcpuset" title="Permanent link">&para;</a></h3>
<p>libcpuset provides full access to cpuset capabilities</p>
<p><em>homepage</em>: <a href="http://oss.sgi.com/projects/cpusets/">http://oss.sgi.com/projects/cpusets/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="libcroco">libcroco<a class="headerlink" href="#libcroco" title="Permanent link">&para;</a></h3>
<p>Libcroco is a standalone css2 parsing and manipulation library.</p>
<p><em>homepage</em>: <a href="https://github.com/GNOME/libcroco">https://github.com/GNOME/libcroco</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.6.13</code></td>
<td><code>GCC/10.2.0</code>, <code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="libctl">libctl<a class="headerlink" href="#libctl" title="Permanent link">&para;</a></h3>
<p>libctl is a free Guile-based library implementing flexible control files for scientific simulations.</p>
<p><em>homepage</em>: <a href="https://libctl.readthedocs.io/en/latest/">https://libctl.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>4.1.3</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libdap">libdap<a class="headerlink" href="#libdap" title="Permanent link">&para;</a></h3>
<p>A C++ SDK which contains an implementation of DAP 2.0 and  DAP4.0. This includes both Client- and Server-side support classes.</p>
<p><em>homepage</em>: <a href="https://www.opendap.org/software/libdap">https://www.opendap.org/software/libdap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.18.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.18.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.19.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.20.3</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.20.4</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.20.6</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.20.7</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.20.8</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.20.11</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libde265">libde265<a class="headerlink" href="#libde265" title="Permanent link">&para;</a></h3>
<p>libde265 is an open source implementation of the h.265 video codec</p>
<p><em>homepage</em>: <a href="https://github.com/strukturag/libde265">https://github.com/strukturag/libde265</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.8</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libdeflate">libdeflate<a class="headerlink" href="#libdeflate" title="Permanent link">&para;</a></h3>
<p>Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.</p>
<p><em>homepage</em>: <a href="https://github.com/ebiggers/libdeflate">https://github.com/ebiggers/libdeflate</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.15</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libdivsufsort">libdivsufsort<a class="headerlink" href="#libdivsufsort" title="Permanent link">&para;</a></h3>
<p>libdivsufsort is a software library that implements a lightweight suffix  array construction algorithm.</p>
<p><em>homepage</em>: <a href="https://github.com/y-256/libdivsufsort">https://github.com/y-256/libdivsufsort</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="libdrm">libdrm<a class="headerlink" href="#libdrm" title="Permanent link">&para;</a></h3>
<p>Direct Rendering Manager runtime library.</p>
<p><em>homepage</em>: <a href="https://dri.freedesktop.org">https://dri.freedesktop.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.67</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.4.68</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.4.70</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.4.76</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.4.88</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.4.91</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.4.92</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.4.97</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.4.99</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.4.100</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4.102</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.4.106</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.4.107</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.4.110</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.4.114</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libdrs">libdrs<a class="headerlink" href="#libdrs" title="Permanent link">&para;</a></h3>
<p>PCMDI's old DRS format implementation</p>
<p><em>homepage</em>: <a href="https://github.com/CDAT/libdrs/">https://github.com/CDAT/libdrs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="libdwarf">libdwarf<a class="headerlink" href="#libdwarf" title="Permanent link">&para;</a></h3>
<p>The DWARF Debugging Information Format is of interest to programmers working on compilers and debuggers (and anyone interested in reading or writing DWARF information))</p>
<p><em>homepage</em>: <a href="https://www.prevanders.net/dwarf.html">https://www.prevanders.net/dwarf.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>20140805</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>20150310</code></td>
<td><code>GCC/4.9.2</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>20190529</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>20201201</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20210305</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>20210528</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libedit">libedit<a class="headerlink" href="#libedit" title="Permanent link">&para;</a></h3>
<p>This BSD-style licensed command line editor library provides generic line editing, history, and tokenization functions, similar to those found in GNU Readline. </p>
<p><em>homepage</em>: <a href="https://thrysoee.dk/editline/">https://thrysoee.dk/editline/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20150325</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>20180525</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20191231</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20210910</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libelf">libelf<a class="headerlink" href="#libelf" title="Permanent link">&para;</a></h3>
<p>libelf is a free ELF object file access library</p>
<p><em>homepage</em>: <a href="https://directory.fsf.org/wiki/Libelf">https://directory.fsf.org/wiki/Libelf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.13</code></td>
<td><code>GCC/4.8.3</code>, <code>GCC/4.9.2</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libemf">libemf<a class="headerlink" href="#libemf" title="Permanent link">&para;</a></h3>
<p>Library implementation of ECMA-234 API for the generation of enhanced metafiles.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/libemf/">https://sourceforge.net/projects/libemf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.13</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libepoxy">libepoxy<a class="headerlink" href="#libepoxy" title="Permanent link">&para;</a></h3>
<p>Epoxy is a library for handling OpenGL function pointer management for you</p>
<p><em>homepage</em>: <a href="https://github.com/anholt/libepoxy">https://github.com/anholt/libepoxy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>GCCcore/8.2.0</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.5.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.5.8</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.5.10</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libev">libev<a class="headerlink" href="#libev" title="Permanent link">&para;</a></h3>
<p>A full-featured and high-performance (see benchmark) event loop that is loosely modelled after libevent, but without its limitations and bugs. It is used in GNU Virtual Private Ethernet, rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many other programs.</p>
<p><em>homepage</em>: <a href="http://software.schmorp.de/pkg/libev.html">http://software.schmorp.de/pkg/libev.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.33</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libevent">libevent<a class="headerlink" href="#libevent" title="Permanent link">&para;</a></h3>
<p>The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.</p>
<p><em>homepage</em>: <a href="http://libevent.org/">http://libevent.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.22</code></td>
<td><code>GCC/4.9.2</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.1.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.1.11</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1.12</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="libexif">libexif<a class="headerlink" href="#libexif" title="Permanent link">&para;</a></h3>
<p>A library for parsing, editing, and saving EXIF data.</p>
<p><em>homepage</em>: <a href="https://libexif.github.io/">https://libexif.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.24</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libfabric">libfabric<a class="headerlink" href="#libfabric" title="Permanent link">&para;</a></h3>
<p>Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. </p>
<p><em>homepage</em>: <a href="https://ofiwg.github.io/libfabric/">https://ofiwg.github.io/libfabric/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.13.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.13.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libfdf">libfdf<a class="headerlink" href="#libfdf" title="Permanent link">&para;</a></h3>
<p>LibFDF provides a Fortran API to parse files in the  Flexible Data Format (FDF).</p>
<p><em>homepage</em>: <a href="https://launchpad.net/libfdf">https://launchpad.net/libfdf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>-serial</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libffcall">libffcall<a class="headerlink" href="#libffcall" title="Permanent link">&para;</a></h3>
<p>GNU Libffcall is a collection of four libraries which can be used to build  foreign function call interfaces in embedded interpreters </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libffcall/">https://www.gnu.org/software/libffcall/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libffi">libffi<a class="headerlink" href="#libffi" title="Permanent link">&para;</a></h3>
<p>The libffi library provides a portable, high level programming interface to  various calling conventions. This allows a programmer to call any function  specified by a call interface description at run-time.</p>
<p><em>homepage</em>: <a href="https://sourceware.org/libffi/">https://sourceware.org/libffi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/4.9.2</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.4.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libflame">libFLAME<a class="headerlink" href="#libflame" title="Permanent link">&para;</a></h3>
<p>libFLAME is a portable library for dense matrix computations, providing much of the functionality present in LAPACK.</p>
<p><em>homepage</em>: <a href="https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame">https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-amd</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-amd</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libfontenc">libfontenc<a class="headerlink" href="#libfontenc" title="Permanent link">&para;</a></h3>
<p>X11 font encoding library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs/">http://www.freedesktop.org/wiki/Software/xlibs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libgcrypt">libgcrypt<a class="headerlink" href="#libgcrypt" title="Permanent link">&para;</a></h3>
<p>Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG</p>
<p><em>homepage</em>: <a href="https://gnupg.org/related_software/libgcrypt/index.html">https://gnupg.org/related_software/libgcrypt/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.5</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.8.5</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgd">libgd<a class="headerlink" href="#libgd" title="Permanent link">&para;</a></h3>
<p>GD is an open source code library for the dynamic creation of images by programmers.</p>
<p><em>homepage</em>: <a href="https://libgd.github.io">https://libgd.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgdiplus">libgdiplus<a class="headerlink" href="#libgdiplus" title="Permanent link">&para;</a></h3>
<p>Libgdiplus is the Mono library that provides a GDI+-compatible  API on non-Windows operating systems.</p>
<p><em>homepage</em>: <a href="https://www.mono-project.com/docs/gui/libgdiplus">https://www.mono-project.com/docs/gui/libgdiplus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgdsii">libGDSII<a class="headerlink" href="#libgdsii" title="Permanent link">&para;</a></h3>
<p>libGDSII is a C++ library for working with GDSII binary data files,  intended primarily for use with the computational electromagnetism codes  scuff-em and meep but sufficiently general-purpose to allow other uses as well.</p>
<p><em>homepage</em>: <a href="https://github.com/HomerReid/libGDSII">https://github.com/HomerReid/libGDSII</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.21</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgeotiff">libgeotiff<a class="headerlink" href="#libgeotiff" title="Permanent link">&para;</a></h3>
<p>Library for reading and writing coordinate system information from/to GeoTIFF files</p>
<p><em>homepage</em>: <a href="https://directory.fsf.org/wiki/Libgeotiff">https://directory.fsf.org/wiki/Libgeotiff</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgit2">libgit2<a class="headerlink" href="#libgit2" title="Permanent link">&para;</a></h3>
<p>libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant linkable library with a solid API, allowing you to write native speed custom Git applications in any language which supports C bindings.</p>
<p><em>homepage</em>: <a href="https://libgit2.org/">https://libgit2.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libglade">libglade<a class="headerlink" href="#libglade" title="Permanent link">&para;</a></h3>
<p>Libglade is a library for constructing user interfaces dynamically from XML descriptions.</p>
<p><em>homepage</em>: <a href="https://developer.gnome.org/libglade/">https://developer.gnome.org/libglade/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.4</code></td>
<td><code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="libglu">libGLU<a class="headerlink" href="#libglu" title="Permanent link">&para;</a></h3>
<p>The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. </p>
<p><em>homepage</em>: <a href="https://mesa.freedesktop.org/archive/glu/">https://mesa.freedesktop.org/archive/glu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.0.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>fosscuda/2018b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intelcuda/2017b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>-Mesa-11.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>9.0.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libglvnd">libglvnd<a class="headerlink" href="#libglvnd" title="Permanent link">&para;</a></h3>
<p>libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.</p>
<p><em>homepage</em>: <a href="https://gitlab.freedesktop.org/glvnd/libglvnd">https://gitlab.freedesktop.org/glvnd/libglvnd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgpg-error">libgpg-error<a class="headerlink" href="#libgpg-error" title="Permanent link">&para;</a></h3>
<p>Libgpg-error is a small library that defines common error values for all GnuPG components.</p>
<p><em>homepage</em>: <a href="https://gnupg.org/related_software/libgpg-error/index.html">https://gnupg.org/related_software/libgpg-error/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.21</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.35</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.36</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.38</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.41</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.42</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.46</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libgpuarray">libgpuarray<a class="headerlink" href="#libgpuarray" title="Permanent link">&para;</a></h3>
<p>Library to manipulate tensors on the GPU.</p>
<p><em>homepage</em>: <a href="http://deeplearning.net/software/libgpuarray/">http://deeplearning.net/software/libgpuarray/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.5</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>fosscuda/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.7.5</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.7.6</code></td>
<td></td>
<td><code>fosscuda/2019a</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.7.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.7.6</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.7.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="libgridxc">libGridXC<a class="headerlink" href="#libgridxc" title="Permanent link">&para;</a></h3>
<p>A library to compute the exchange and correlation energy  and potential in spherical (i.e. atoms) or periodic systems.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/siesta-project/libraries/libgridxc">https://gitlab.com/siesta-project/libraries/libgridxc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.5</code></td>
<td><code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>0.9.6</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code>, <code>iimpi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="libgtextutils">libgtextutils<a class="headerlink" href="#libgtextutils" title="Permanent link">&para;</a></h3>
<p>ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream</p>
<p><em>homepage</em>: <a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="libharu">libharu<a class="headerlink" href="#libharu" title="Permanent link">&para;</a></h3>
<p>libHaru is a free, cross platform, open source library for generating PDF files.</p>
<p><em>homepage</em>: <a href="http://libharu.org/">http://libharu.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2017a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="libheif">libheif<a class="headerlink" href="#libheif" title="Permanent link">&para;</a></h3>
<p>libheif is an HEIF and AVIF file format decoder and encoder</p>
<p><em>homepage</em>: <a href="https://github.com/strukturag/libheif">https://github.com/strukturag/libheif</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.0</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libibmad">libibmad<a class="headerlink" href="#libibmad" title="Permanent link">&para;</a></h3>
<p>libibmad is a convenience library to encode, decode, and dump IB MAD packets. It  is implemented on top of and in conjunction with libibumad (the umad kernel  interface library.)</p>
<p><em>homepage</em>: <a href="http://www.openfabrics.org">http://www.openfabrics.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.12</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
</tbody>
</table>
<h3 id="libibumad">libibumad<a class="headerlink" href="#libibumad" title="Permanent link">&para;</a></h3>
<p>libibumad is the umad kernel interface library.</p>
<p><em>homepage</em>: <a href="http://www.openfabrics.org">http://www.openfabrics.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.10.2</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
</tbody>
</table>
<h3 id="libice">libICE<a class="headerlink" href="#libice" title="Permanent link">&para;</a></h3>
<p>X Inter-Client Exchange library for freedesktop.org</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libiconv">libiconv<a class="headerlink" href="#libiconv" title="Permanent link">&para;</a></h3>
<p>Libiconv converts from one character encoding to another through Unicode conversion</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libiconv">https://www.gnu.org/software/libiconv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.15</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libidn">libidn<a class="headerlink" href="#libidn" title="Permanent link">&para;</a></h3>
<p>GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications. Libidn's purpose is to encode and decode internationalized domain names.</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/libidn">http://www.gnu.org/software/libidn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.32</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.34</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.35</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.36</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.38</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.41</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libidn2">libidn2<a class="headerlink" href="#libidn2" title="Permanent link">&para;</a></h3>
<p>Libidn2 implements the revised algorithm for internationalized domain names called IDNA2008/TR46.</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/libidn2">http://www.gnu.org/software/libidn2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libint">Libint<a class="headerlink" href="#libint" title="Permanent link">&para;</a></h3>
<p>Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.</p>
<p><em>homepage</em>: <a href="https://github.com/evaleev/libint">https://github.com/evaleev/libint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.6</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2016b</code>, <code>foss/2018a</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2016b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-lmax-6-cp2k</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>gompi/2020a</code>, <code>iccifort/2020.4.304</code>, <code>iimpi/2020a</code>, <code>iimpi/2021a</code></td>
</tr>
<tr>
<td><code>2.7.2</code></td>
<td><code>-lmax-6-cp2k</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libis">LiBis<a class="headerlink" href="#libis" title="Permanent link">&para;</a></h3>
<p>An ultrasensitive alignment method for low input bisulfite sequencing</p>
<p><em>homepage</em>: <a href="https://github.com/Dangertrip/LiBis">https://github.com/Dangertrip/LiBis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200428</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="libjpeg-turbo">libjpeg-turbo<a class="headerlink" href="#libjpeg-turbo" title="Permanent link">&para;</a></h3>
<p>libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to  accelerate baseline JPEG compression and decompression. libjpeg is a library  that implements JPEG image encoding, decoding and transcoding. </p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/libjpeg-turbo/">https://sourceforge.net/projects/libjpeg-turbo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-NASM-2.12.01</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.1.4</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libjxl">libjxl<a class="headerlink" href="#libjxl" title="Permanent link">&para;</a></h3>
<p>JPEG XL image format reference implementation</p>
<p><em>homepage</em>: <a href="https://github.com/libjxl/libjxl">https://github.com/libjxl/libjxl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="liblzf">LibLZF<a class="headerlink" href="#liblzf" title="Permanent link">&para;</a></h3>
<p>LibLZF is a very small data compression library. It consists of only two .c and two .h files and is very easy to incorporate into your own programs. The compression algorithm is very, very fast, yet still written in portable C.</p>
<p><em>homepage</em>: <a href="http://oldhome.schmorp.de/marc/liblzf.html">http://oldhome.schmorp.de/marc/liblzf.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libmad">libmad<a class="headerlink" href="#libmad" title="Permanent link">&para;</a></h3>
<p>MAD is a high-quality MPEG audio decoder.</p>
<p><em>homepage</em>: <a href="https://www.underbit.com/products/mad/">https://www.underbit.com/products/mad/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.1b</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libmatheval">libmatheval<a class="headerlink" href="#libmatheval" title="Permanent link">&para;</a></h3>
<p>GNU libmatheval is a library (callable from C and Fortran) to parse  and evaluate symbolic expressions input as text.</p>
<p><em>homepage</em>: <a href="http://www.gnu.org/software/libmatheval/">http://www.gnu.org/software/libmatheval/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.11</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="libmaus2">libmaus2<a class="headerlink" href="#libmaus2" title="Permanent link">&para;</a></h3>
<p>libmaus2 is a collection of data structures and algorithms.</p>
<p><em>homepage</em>: <a href="https://github.com/gt1/libmaus2">https://github.com/gt1/libmaus2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.453</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="libmbd">libmbd<a class="headerlink" href="#libmbd" title="Permanent link">&para;</a></h3>
<p>Libmbd implements the many-body dispersion (MBD) method in several programming languages and frameworks:   - The Fortran implementation is the reference, most advanced implementation, with support for analytical     gradients and distributed parallelism, and additional functionality beyond the MBD method itself.     It provides a low-level and a high-level Fortran API, as well as a C API. Furthermore, Python bindings     to the C API are provided.  - The Python/Numpy implementation is intended for prototyping, and as a high-level language reference.  - The Python/Tensorflow implementation is an experiment that should enable rapid prototyping of machine     learning applications with MBD.  The Python-based implementations as well as Python bindings to the Libmbd C API are accessible from the  Python package called Pymbd. </p>
<p><em>homepage</em>: <a href="https://libmbd.github.io/index.html">https://libmbd.github.io/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.4</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>intel/2020b</code>, <code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="libmemcached">libMemcached<a class="headerlink" href="#libmemcached" title="Permanent link">&para;</a></h3>
<p>libMemcached is an open source C/C++ client library and tools for  the memcached server (http://danga.com/memcached). It has been designed to be light  on memory usage, thread safe, and provide full access to server side methods.</p>
<p><em>homepage</em>: <a href="https://memcached.org">https://memcached.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.18</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libmicrohttpd">libmicrohttpd<a class="headerlink" href="#libmicrohttpd" title="Permanent link">&para;</a></h3>
<p>GNU libmicrohttpd is a small C library that is supposed to make it easy to run  an HTTP server as part of another application. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libmicrohttpd/">https://www.gnu.org/software/libmicrohttpd/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.71</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.9.73</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libmo_unpack">libmo_unpack<a class="headerlink" href="#libmo_unpack" title="Permanent link">&para;</a></h3>
<p>A library for handling the WGDOS and RLE compression schemes used in UM files.</p>
<p><em>homepage</em>: <a href="https://github.com/SciTools/libmo_unpack">https://github.com/SciTools/libmo_unpack</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libmypaint">libmypaint<a class="headerlink" href="#libmypaint" title="Permanent link">&para;</a></h3>
<p>libmypaint, a.k.a. "brushlib", is a library for making brushstrokes which is used by MyPaint  and other projects</p>
<p><em>homepage</em>: <a href="https://github.com/mypaint/libmypaint">https://github.com/mypaint/libmypaint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libnsl">libnsl<a class="headerlink" href="#libnsl" title="Permanent link">&para;</a></h3>
<p>The libnsl package contains the public client interface for NIS(YP).</p>
<p><em>homepage</em>: <a href="https://github.com/thkukuk/libnsl">https://github.com/thkukuk/libnsl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libobjcryst">libobjcryst<a class="headerlink" href="#libobjcryst" title="Permanent link">&para;</a></h3>
<p>ObjCryst++ is Object-Oriented Crystallographic Library for C++</p>
<p><em>homepage</em>: <a href="https://github.com/diffpy/libobjcryst">https://github.com/diffpy/libobjcryst</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017.2.3</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2021.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="libogg">libogg<a class="headerlink" href="#libogg" title="Permanent link">&para;</a></h3>
<p>Ogg is a multimedia container format, and the native file and stream format for the Xiph.org multimedia codecs.</p>
<p><em>homepage</em>: <a href="https://xiph.org/ogg/">https://xiph.org/ogg/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.3.5</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libopus">libopus<a class="headerlink" href="#libopus" title="Permanent link">&para;</a></h3>
<p>Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive  speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is  standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skypes  SILK codec and Xiph.Orgs CELT codec.</p>
<p><em>homepage</em>: <a href="https://www.opus-codec.org/">https://www.opus-codec.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libosmium">libosmium<a class="headerlink" href="#libosmium" title="Permanent link">&para;</a></h3>
<p>A fast and flexible C++ library for working with OpenStreetMap data.  The Osmium Library has extensive support for all types of OSM entities:  nodes, ways, relations, and changesets.  It allows reading from and writing to OSM files in XML and PBF formats, including change files and full history files.  Osmium can store OSM data in memory and on disk in various formats and using various indexes.  Its easy to use handler interface allows you to quickly write data filtering and conversion functions.  Osmium can create WKT, WKB, OGR, GEOS and GeoJSON geometries for easy conversion into many GIS formats  and it can assemble multipolygons from ways and relations.</p>
<p><em>homepage</em>: <a href="https://osmcode.org/libosmium/">https://osmcode.org/libosmium/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.15.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.15.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="libpci">libpci<a class="headerlink" href="#libpci" title="Permanent link">&para;</a></h3>
<p>Library for portable access to PCI bus configuration registers from PCI Utils.</p>
<p><em>homepage</em>: <a href="https://github.com/pciutils/pciutils">https://github.com/pciutils/pciutils</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libpciaccess">libpciaccess<a class="headerlink" href="#libpciaccess" title="Permanent link">&para;</a></h3>
<p>Generic PCI access library.</p>
<p><em>homepage</em>: <a href="https://cgit.freedesktop.org/xorg/lib/libpciaccess/">https://cgit.freedesktop.org/xorg/lib/libpciaccess/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.4</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.14</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.16</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.17</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libplinkio">libplinkio<a class="headerlink" href="#libplinkio" title="Permanent link">&para;</a></h3>
<p>A small C and Python library for reading PLINK genotype files.</p>
<p><em>homepage</em>: <a href="https://github.com/mfranberg/libplinkio">https://github.com/mfranberg/libplinkio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.8</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libpng">libpng<a class="headerlink" href="#libpng" title="Permanent link">&para;</a></h3>
<p>libpng is the official PNG reference library</p>
<p><em>homepage</em>: <a href="http://www.libpng.org/pub/png/libpng.html">http://www.libpng.org/pub/png/libpng.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.58</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.59</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.21</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.6.23</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.24</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.26</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.27</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.28</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>1.6.29</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>1.6.32</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.6.34</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.6.36</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.6.37</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.6.38</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libpsl">libpsl<a class="headerlink" href="#libpsl" title="Permanent link">&para;</a></h3>
<p>C library for the Public Suffix List</p>
<p><em>homepage</em>: <a href="https://rockdaboot.github.io/libpsl">https://rockdaboot.github.io/libpsl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.20.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.21.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.21.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libpsml">libPSML<a class="headerlink" href="#libpsml" title="Permanent link">&para;</a></h3>
<p>A library to handle PSML, the pseudopotential markup language.</p>
<p><em>homepage</em>: <a href="https://launchpad.net/libpsml">https://launchpad.net/libpsml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.7</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.1.8</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.1.10</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libpsortb">libpsortb<a class="headerlink" href="#libpsortb" title="Permanent link">&para;</a></h3>
<p>PSORT family of programs for subcellular localization prediction as well as other datasets and  resources relevant to localization prediction.</p>
<p><em>homepage</em>: <a href="http://psort.org/">http://psort.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libpspio">libpspio<a class="headerlink" href="#libpspio" title="Permanent link">&para;</a></h3>
<p>libpspio is a library to read and write pseudopotentials in multiple formats.</p>
<p><em>homepage</em>: <a href="https://gitlab.e-cam2020.eu/esl/pspio/">https://gitlab.e-cam2020.eu/esl/pspio/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="libpthread-stubs">libpthread-stubs<a class="headerlink" href="#libpthread-stubs" title="Permanent link">&para;</a></h3>
<p>The X protocol C-language Binding (XCB) is a replacement for Xlib featuring  a small footprint, latency hiding, direct access to the protocol, improved  threading support, and extensibility. </p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libqglviewer">libQGLViewer<a class="headerlink" href="#libqglviewer" title="Permanent link">&para;</a></h3>
<p>libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers.</p>
<p><em>homepage</em>: <a href="http://libqglviewer.com/">http://libqglviewer.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>-Mesa-11.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libreadline">libreadline<a class="headerlink" href="#libreadline" title="Permanent link">&para;</a></h3>
<p>The GNU Readline library provides a set of functions for use by applications  that allow users to edit command lines as they are typed in. Both Emacs and  vi editing modes are available. The Readline library includes additional  functions to maintain a list of previously-entered command lines, to recall  and perhaps reedit those lines, and perform csh-like history expansion on  previous commands. </p>
<p><em>homepage</em>: <a href="https://tiswww.case.edu/php/chet/readline/rltop.html">https://tiswww.case.edu/php/chet/readline/rltop.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.3</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>7.0</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>8.1</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>8.1.2</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="librmath">libRmath<a class="headerlink" href="#librmath" title="Permanent link">&para;</a></h3>
<p>The routines supporting the distribution and special functions in R and a few others are declared  in C header file Rmath.h. These can be compiled into a standalone library for linking to other applications.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/doc/manuals/r-release/R-admin.html#The-standalone-Rmath-library">https://cran.r-project.org/doc/manuals/r-release/R-admin.html#The-standalone-Rmath-library</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.1.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="librosa">librosa<a class="headerlink" href="#librosa" title="Permanent link">&para;</a></h3>
<p>Python module for audio and music processing</p>
<p><em>homepage</em>: <a href="https://librosa.github.io">https://librosa.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="librsb">librsb<a class="headerlink" href="#librsb" title="Permanent link">&para;</a></h3>
<p>A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/librsb/">https://sourceforge.net/projects/librsb/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0.9</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.2.0.10</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.2.0.11</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.3.0.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.3.0.1</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="librsvg">librsvg<a class="headerlink" href="#librsvg" title="Permanent link">&para;</a></h3>
<p>Librsvg is a library to render SVG files using cairo.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/LibRsvg">https://wiki.gnome.org/Projects/LibRsvg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.40.15</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.48.4</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.51.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.52.8</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.55.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="librttopo">librttopo<a class="headerlink" href="#librttopo" title="Permanent link">&para;</a></h3>
<p>The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies using user-provided data stores.</p>
<p><em>homepage</em>: <a href="https://git.osgeo.org/gitea/rttopo/librttopo">https://git.osgeo.org/gitea/rttopo/librttopo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsamplerate">libsamplerate<a class="headerlink" href="#libsamplerate" title="Permanent link">&para;</a></h3>
<p>Secret Rabbit Code (aka libsamplerate) is a Sample Rate Converter for audio.</p>
<p><em>homepage</em>: <a href="http://www.mega-nerd.com/libsamplerate">http://www.mega-nerd.com/libsamplerate</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.9</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsbml">libSBML<a class="headerlink" href="#libsbml" title="Permanent link">&para;</a></h3>
<p>libSBML (Systems Biology Markup Language library) is a free, open-source programming library to help you read, write, manipulate, translate, and validate SBML files and data streams. It's not an application itself (though it does come with example programs), but rather a library you embed in your own applications.</p>
<p><em>homepage</em>: <a href="http://sbml.org/Software/libSBML">http://sbml.org/Software/libSBML</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.19.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsigc">libsigc++<a class="headerlink" href="#libsigc" title="Permanent link">&para;</a></h3>
<p>The libsigc++ package implements a typesafe callback system for standard C++.</p>
<p><em>homepage</em>: <a href="https://libsigcplusplus.github.io/libsigcplusplus/">https://libsigcplusplus.github.io/libsigcplusplus/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.10.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.10.8</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsigsegv">libsigsegv<a class="headerlink" href="#libsigsegv" title="Permanent link">&para;</a></h3>
<p>GNU libsigsegv is a library for handling page faults in user mode.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libsigsegv/">https://www.gnu.org/software/libsigsegv/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsm">libSM<a class="headerlink" href="#libsm" title="Permanent link">&para;</a></h3>
<p>X11 Session Management library, which allows for applications to both manage sessions,  and make use of session managers to save and restore their state for later use.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libsndfile">libsndfile<a class="headerlink" href="#libsndfile" title="Permanent link">&para;</a></h3>
<p>Libsndfile is a C library for reading and writing files containing sampled sound  (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.</p>
<p><em>homepage</em>: <a href="http://www.mega-nerd.com/libsndfile">http://www.mega-nerd.com/libsndfile</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.28</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.31</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsodium">libsodium<a class="headerlink" href="#libsodium" title="Permanent link">&para;</a></h3>
<p>Sodium is a modern, easy-to-use software library for encryption, decryption, signatures,  password hashing and more.</p>
<p><em>homepage</em>: <a href="https://doc.libsodium.org/">https://doc.libsodium.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0.8</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0.11</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.12</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.13</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.0.16</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.0.17</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.0.18</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsoup">LibSoup<a class="headerlink" href="#libsoup" title="Permanent link">&para;</a></h3>
<p>libsoup is an HTTP client/server library for GNOME. It uses GObjects and the glib main loop, to integrate well with GNOME applications, and also has a synchronous API, for use in threaded applications.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/libsoup">https://wiki.gnome.org/Projects/libsoup</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.66.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.70.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.72.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.74.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.7</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.0.8</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libspatialindex">libspatialindex<a class="headerlink" href="#libspatialindex" title="Permanent link">&para;</a></h3>
<p>C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API</p>
<p><em>homepage</em>: <a href="https://libspatialindex.org">https://libspatialindex.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libspatialite">libspatialite<a class="headerlink" href="#libspatialite" title="Permanent link">&para;</a></h3>
<p>SpatiaLite is an open source library intended to extend the SQLite core to support  fully fledged Spatial SQL capabilities.</p>
<p><em>homepage</em>: <a href="https://www.gaia-gis.it/fossil/libspatialite/home">https://www.gaia-gis.it/fossil/libspatialite/home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.0a</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.3.0a</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.3.0a</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libssh">libssh<a class="headerlink" href="#libssh" title="Permanent link">&para;</a></h3>
<p>Multiplatform C library implementing the SSHv2 protocol on client and server side</p>
<p><em>homepage</em>: <a href="https://www.libssh.org">https://www.libssh.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libstatgen">libStatGen<a class="headerlink" href="#libstatgen" title="Permanent link">&para;</a></h3>
<p>Useful set of classes for creating statistical genetic programs.</p>
<p><em>homepage</em>: <a href="https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen">https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.15</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20190330</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libsvm">LIBSVM<a class="headerlink" href="#libsvm" title="Permanent link">&para;</a></h3>
<p>LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression  (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification.</p>
<p><em>homepage</em>: <a href="https://www.csie.ntu.edu.tw/~cjlin/libsvm/">https://www.csie.ntu.edu.tw/~cjlin/libsvm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.22</code></td>
<td><code>intel/2016b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.23</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.24</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.25</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.30</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtar">libtar<a class="headerlink" href="#libtar" title="Permanent link">&para;</a></h3>
<p>C library for manipulating POSIX tar files</p>
<p><em>homepage</em>: <a href="https://repo.or.cz/libtar.git">https://repo.or.cz/libtar.git</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.20</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtasn1">libtasn1<a class="headerlink" href="#libtasn1" title="Permanent link">&para;</a></h3>
<p>Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and some other packages.  It was written by Fabio Fiorina, and has been shipped as part of GnuTLS  for some time but is now a proper GNU package.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libtasn1/">https://www.gnu.org/software/libtasn1/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7</code></td>
<td><code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>GCCcore/5.4.0</code></td>
</tr>
<tr>
<td><code>4.13</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.16.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.17.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.18.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.19.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtecla">libtecla<a class="headerlink" href="#libtecla" title="Permanent link">&para;</a></h3>
<p>The tecla library provides UNIX and LINUX programs with interactive command line editing facilities,  similar to those of the UNIX tcsh shell. In addition to simple command-line editing, it supports recall of previously  entered command lines, TAB completion of file names or other tokens, and in-line wild-card expansion of filenames.  The internal functions which perform file-name completion and wild-card expansion are also available externally for  optional use by programs. </p>
<p><em>homepage</em>: <a href="https://sites.astro.caltech.edu/~mcs/tecla/">https://sites.astro.caltech.edu/~mcs/tecla/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtiff">LibTIFF<a class="headerlink" href="#libtiff" title="Permanent link">&para;</a></h3>
<p>tiff: Library and tools for reading and writing TIFF data files</p>
<p><em>homepage</em>: <a href="https://libtiff.gitlab.io/libtiff/">https://libtiff.gitlab.io/libtiff/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.6</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.0.7</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.0.8</code></td>
<td><code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.0.9</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.01</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.10</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtirpc">libtirpc<a class="headerlink" href="#libtirpc" title="Permanent link">&para;</a></h3>
<p>Libtirpc is a port of Suns Transport-Independent RPC library to Linux.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/libtirpc/">https://sourceforge.net/projects/libtirpc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.2.6</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libtool">libtool<a class="headerlink" href="#libtool" title="Permanent link">&para;</a></h3>
<p>GNU libtool is a generic library support script. Libtool hides the complexity  of using shared libraries behind a consistent, portable interface. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libtool">https://www.gnu.org/software/libtool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.4.5</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.4.6</code></td>
<td><code>FCC/4.5.0</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>GNU/5.1.0-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.4.7</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="libtree">libtree<a class="headerlink" href="#libtree" title="Permanent link">&para;</a></h3>
<p>libtree is a tool that turns ldd into a tree, explains why  shared libraries are found and why not and optionally deploys executables and  dependencies into a single directory</p>
<p><em>homepage</em>: <a href="https://github.com/haampie/libtree">https://github.com/haampie/libtree</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="libunistring">libunistring<a class="headerlink" href="#libunistring" title="Permanent link">&para;</a></h3>
<p>This library provides functions for manipulating Unicode strings and for  manipulating C strings according to the Unicode standard.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/libunistring/">https://www.gnu.org/software/libunistring/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.3</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.9.6</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.7</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.9.10</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libunwind">libunwind<a class="headerlink" href="#libunwind" title="Permanent link">&para;</a></h3>
<p>The primary goal of libunwind is to define a portable and efficient C programming interface  (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the  preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain  (non-local goto). The API supports both local (same-process) and remote (across-process) operation.  As such, the API is useful in a number of applications</p>
<p><em>homepage</em>: <a href="https://www.nongnu.org/libunwind/">https://www.nongnu.org/libunwind/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCC/4.9.2</code>, <code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.6.2</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libutempter">libutempter<a class="headerlink" href="#libutempter" title="Permanent link">&para;</a></h3>
<p>libutempter is library that provides an interface for terminal  emulators such as screen and xterm to record user sessions to utmp and wtmp files.</p>
<p><em>homepage</em>: <a href="http://git.altlinux.org/people/ldv/packages/?p=libutempter.git">http://git.altlinux.org/people/ldv/packages/?p=libutempter.git</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.6.2</code></td>
<td><code>GCC/6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="libuuid">LibUUID<a class="headerlink" href="#libuuid" title="Permanent link">&para;</a></h3>
<p>Portable uuid C library</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/libuuid/">http://sourceforge.net/projects/libuuid/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016a</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="libuv">libuv<a class="headerlink" href="#libuv" title="Permanent link">&para;</a></h3>
<p>libuv is a multi-platform support library with a focus on asynchronous I/O.</p>
<p><em>homepage</em>: <a href="https://libuv.org">https://libuv.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.37.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libvdwxc">libvdwxc<a class="headerlink" href="#libvdwxc" title="Permanent link">&para;</a></h3>
<p>libvdwxc is a general library for evaluating energy and potential for exchange-correlation (XC) functionals from the vdW-DF family that can be used with various of density functional theory (DFT) codes.</p>
<p><em>homepage</em>: <a href="https://libvdwxc.org">https://libvdwxc.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="libvorbis">libvorbis<a class="headerlink" href="#libvorbis" title="Permanent link">&para;</a></h3>
<p>Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed audio format</p>
<p><em>homepage</em>: <a href="https://xiph.org/vorbis/">https://xiph.org/vorbis/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.7</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libwebp">libwebp<a class="headerlink" href="#libwebp" title="Permanent link">&para;</a></h3>
<p>WebP is a modern image format that provides superior lossless and lossy compression for images on the web. Using WebP, webmasters and web developers can create smaller, richer images that make the web faster.</p>
<p><em>homepage</em>: <a href="https://developers.google.com/speed/webp/">https://developers.google.com/speed/webp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libwpe">libwpe<a class="headerlink" href="#libwpe" title="Permanent link">&para;</a></h3>
<p>WPE is the reference WebKit port for embedded and low-consumption computer devices. It has been designed from the ground-up with performance, small footprint, accelerated content rendering, and simplicity of deployment in mind, bringing the excellence of the WebKit engine to countless platforms and target devices.</p>
<p><em>homepage</em>: <a href="https://webkit.org/wpe">https://webkit.org/wpe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libx11">libX11<a class="headerlink" href="#libx11" title="Permanent link">&para;</a></h3>
<p>X11 client-side library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.3</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxau">libXau<a class="headerlink" href="#libxau" title="Permanent link">&para;</a></h3>
<p>The libXau package contains a library implementing the X11 Authorization Protocol. This is useful for restricting client access to the display.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxc">libxc<a class="headerlink" href="#libxc" title="Permanent link">&para;</a></h3>
<p>Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.</p>
<p><em>homepage</em>: <a href="https://www.tddft.org/programs/libxc">https://www.tddft.org/programs/libxc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>gimkl/2017a</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>4.2.3</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>gimkl/2017a</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.3.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code>, <code>iccifort/2020.1.217</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>5.1.3</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>5.1.5</code></td>
<td><code>GCC/10.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>5.1.6</code></td>
<td><code>GCC/11.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>5.2.3</code></td>
<td><code>GCC/11.3.0</code>, <code>intel-compilers/2022.1.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxcb">libxcb<a class="headerlink" href="#libxcb" title="Permanent link">&para;</a></h3>
<p>The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.</p>
<p><em>homepage</em>: <a href="https://xcb.freedesktop.org/">https://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.1</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.13</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxcursor">libXcursor<a class="headerlink" href="#libxcursor" title="Permanent link">&para;</a></h3>
<p>X Cursor management library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.14</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxdamage">libXdamage<a class="headerlink" href="#libxdamage" title="Permanent link">&para;</a></h3>
<p>X Damage extension library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxdmcp">libXdmcp<a class="headerlink" href="#libxdmcp" title="Permanent link">&para;</a></h3>
<p>The libXdmcp package contains a library implementing the X Display Manager Control Protocol. This is useful for allowing clients to interact with the X Display Manager. </p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxext">libXext<a class="headerlink" href="#libxext" title="Permanent link">&para;</a></h3>
<p>Common X Extensions library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxfixes">libXfixes<a class="headerlink" href="#libxfixes" title="Permanent link">&para;</a></h3>
<p>X Fixes extension library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.1</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxfont">libXfont<a class="headerlink" href="#libxfont" title="Permanent link">&para;</a></h3>
<p>X font libary</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-freetype-2.6.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxft">libXft<a class="headerlink" href="#libxft" title="Permanent link">&para;</a></h3>
<p>X11 client-side library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>-fontconfig-2.11.95</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>-freetype-2.6.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxi">libXi<a class="headerlink" href="#libxi" title="Permanent link">&para;</a></h3>
<p>LibXi provides an X Window System client interface to the XINPUT extension to the X protocol.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.6</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxinerama">libXinerama<a class="headerlink" href="#libxinerama" title="Permanent link">&para;</a></h3>
<p>Xinerama multiple monitor library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxkbcommon">libxkbcommon<a class="headerlink" href="#libxkbcommon" title="Permanent link">&para;</a></h3>
<p>xkbcommon is a library to handle keyboard descriptions,  including loading them from disk, parsing them and handling their state.  It's mainly meant for client toolkits, window systems,  and other system applications.</p>
<p><em>homepage</em>: <a href="http://xkbcommon.org/">http://xkbcommon.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxml">libxml++<a class="headerlink" href="#libxml" title="Permanent link">&para;</a></h3>
<p>libxml++ is a C++ wrapper for the libxml XML parser library.</p>
<p><em>homepage</em>: <a href="http://libxmlplusplus.sourceforge.net">http://libxmlplusplus.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.40.1</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.42.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxml2">libxml2<a class="headerlink" href="#libxml2" title="Permanent link">&para;</a></h3>
<p>Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). </p>
<p><em>homepage</em>: <a href="http://xmlsoft.org/">http://xmlsoft.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.2</code></td>
<td></td>
<td><code>GCC/4.8.3</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.9.3</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.9.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.9.4</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.9.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.9.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.9.5</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>2.9.6</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.9.7</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.9.8</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.9.9</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.9.10</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.9.13</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.10.3</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxml2-python">libxml2-python<a class="headerlink" href="#libxml2-python" title="Permanent link">&para;</a></h3>
<p>Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). This is the Python binding.</p>
<p><em>homepage</em>: <a href="http://xmlsoft.org/">http://xmlsoft.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.9.8</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.9.8</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.9.13</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxmu">libXmu<a class="headerlink" href="#libxmu" title="Permanent link">&para;</a></h3>
<p>libXmu provides a set of miscellaneous utility convenience functions for X libraries to use.  libXmuu is a lighter-weight version that does not depend on libXt or libXext</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxp">libXp<a class="headerlink" href="#libxp" title="Permanent link">&para;</a></h3>
<p>libXp provides the X print library.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxpm">libXpm<a class="headerlink" href="#libxpm" title="Permanent link">&para;</a></h3>
<p>libXp provides the X print library.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxrandr">libXrandr<a class="headerlink" href="#libxrandr" title="Permanent link">&para;</a></h3>
<p>X Resize, Rotate and Reflection extension library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxrender">libXrender<a class="headerlink" href="#libxrender" title="Permanent link">&para;</a></h3>
<p>X11 client-side library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.9</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxslt">libxslt<a class="headerlink" href="#libxslt" title="Permanent link">&para;</a></h3>
<p>Libxslt is the XSLT C library developed for the GNOME project  (but usable outside of the Gnome platform).</p>
<p><em>homepage</em>: <a href="http://xmlsoft.org/">http://xmlsoft.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.28</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.28</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.29</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.1.30</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>1.1.32</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.1.33</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.34</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.37</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxsmm">libxsmm<a class="headerlink" href="#libxsmm" title="Permanent link">&para;</a></h3>
<p>LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86).</p>
<p><em>homepage</em>: <a href="https://github.com/hfp/libxsmm">https://github.com/hfp/libxsmm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.4</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.8.2</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>GCC/10.3.0</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libxt">libXt<a class="headerlink" href="#libxt" title="Permanent link">&para;</a></h3>
<p>libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are  based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation).</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.5</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libxxf86vm">libXxf86vm<a class="headerlink" href="#libxxf86vm" title="Permanent link">&para;</a></h3>
<p>X11 XFree86 video mode extension library</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="libyaml">libyaml<a class="headerlink" href="#libyaml" title="Permanent link">&para;</a></h3>
<p>LibYAML is a YAML parser and emitter written in C.</p>
<p><em>homepage</em>: <a href="https://pyyaml.org/wiki/LibYAML">https://pyyaml.org/wiki/LibYAML</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.1.7</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.2.1</code></td>
<td><code>GCCcore/7.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.2.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="libzeep">libzeep<a class="headerlink" href="#libzeep" title="Permanent link">&para;</a></h3>
<p>Libzeep was originally developed to make it easy to create SOAP servers.</p>
<p><em>homepage</em>: <a href="https://github.com/mhekkel/libzeep">https://github.com/mhekkel/libzeep</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.1</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="libzip">libzip<a class="headerlink" href="#libzip" title="Permanent link">&para;</a></h3>
<p>libzip is a C library for reading, creating, and modifying zip archives.</p>
<p><em>homepage</em>: <a href="https://libzip.org/">https://libzip.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lie_learn">lie_learn<a class="headerlink" href="#lie_learn" title="Permanent link">&para;</a></h3>
<p>lie_learn is a python package that knows how to do various tricky computations related to Lie groups and manifolds (mainly the sphere S2 and rotation group SO3).</p>
<p><em>homepage</em>: <a href="https://github.com/AMLab-Amsterdam/lie_learn">https://github.com/AMLab-Amsterdam/lie_learn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1.post1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="lifelines">lifelines<a class="headerlink" href="#lifelines" title="Permanent link">&para;</a></h3>
<p>lifelines is a complete survival analysis library, written in pure Python.</p>
<p><em>homepage</em>: <a href="https://lifelines.readthedocs.io/en/latest/">https://lifelines.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.22.8</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>0.26.3</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="lighter">Lighter<a class="headerlink" href="#lighter" title="Permanent link">&para;</a></h3>
<p>Fast and memory-efficient sequencing error corrector</p>
<p><em>homepage</em>: <a href="https://github.com/mourisl/Lighter">https://github.com/mourisl/Lighter</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCC/11.2.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="liknorm">liknorm<a class="headerlink" href="#liknorm" title="Permanent link">&para;</a></h3>
<p>Moments of the product of an exponential-family likelihood with a Normal distribution.</p>
<p><em>homepage</em>: <a href="https://github.com/limix/liknorm">https://github.com/limix/liknorm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="likwid">likwid<a class="headerlink" href="#likwid" title="Permanent link">&para;</a></h3>
<p>Likwid stands for Like I knew what I am doing. This project contributes easy to use command line tools for Linux to support programmers in developing high performance multi threaded programs. </p>
<p><em>homepage</em>: <a href="https://github.com/RRZE-HPC/likwid">https://github.com/RRZE-HPC/likwid</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.1</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.3.2</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="limix">limix<a class="headerlink" href="#limix" title="Permanent link">&para;</a></h3>
<p>Limix is a flexible and efficient linear mixed model library with interfaces to Python.  Genomic analyses require flexible models that can be adapted to the needs of the user.  Limix is smart about how particular models are fitted to save computational cost. </p>
<p><em>homepage</em>: <a href="https://github.com/limix/limix">https://github.com/limix/limix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="linbox">LinBox<a class="headerlink" href="#linbox" title="Permanent link">&para;</a></h3>
<p>C++ library for exact, high-performance linear algebra</p>
<p><em>homepage</em>: <a href="http://linalg.org/">http://linalg.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="line_profiler">line_profiler<a class="headerlink" href="#line_profiler" title="Permanent link">&para;</a></h3>
<p>line_profiler is a module for doing line-by-line profiling of functions. kernprof is a convenient script for running either line_profiler or the Python standard library's cProfile or profile modules, depending on what is available.</p>
<p><em>homepage</em>: <a href="https://github.com/pyutils/line_profiler">https://github.com/pyutils/line_profiler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="lingeling">Lingeling<a class="headerlink" href="#lingeling" title="Permanent link">&para;</a></h3>
<p>One of the design principles of the state-of-the-art SAT solver Lingeling is to use as compact data structures as possible. These reduce memory usage, increase cache efficiency and thus improve runtime, particularly, when using multiple solver instances on multi-core machines, as in our parallel portfolio solver Plingeling and our cube and conquer solver Treengeling.</p>
<p><em>homepage</em>: <a href="http://fmv.jku.at/lingeling/">http://fmv.jku.at/lingeling/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>bcp</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="littlecms">LittleCMS<a class="headerlink" href="#littlecms" title="Permanent link">&para;</a></h3>
<p>Little CMS intends to be an OPEN SOURCE small-footprint color management engine,  with special focus on accuracy and performance. </p>
<p><em>homepage</em>: <a href="http://www.littlecms.com/">http://www.littlecms.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.9</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.13.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lldb">LLDB<a class="headerlink" href="#lldb" title="Permanent link">&para;</a></h3>
<p>The debugger component of the LLVM project</p>
<p><em>homepage</em>: <a href="https://lldb.llvm.org/">https://lldb.llvm.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="llvm">LLVM<a class="headerlink" href="#llvm" title="Permanent link">&para;</a></h3>
<p>The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  ("LLVM IR"). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.</p>
<p><em>homepage</em>: <a href="https://llvm.org/">https://llvm.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.1</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td><code>GCCcore/4.9.3</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.9.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>7.0.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>10.0.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>10.0.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>11.0.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>11.1.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>12.0.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>14.0.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>15.0.5</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lmdb">LMDB<a class="headerlink" href="#lmdb" title="Permanent link">&para;</a></h3>
<p>LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance  of a pure in-memory database while retaining the persistence of standard disk-based databases.</p>
<p><em>homepage</em>: <a href="https://symas.com/lmdb">https://symas.com/lmdb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.18</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.9.21</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.22</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.9.23</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.9.24</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.9.28</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.9.29</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lmfit">LMfit<a class="headerlink" href="#lmfit" title="Permanent link">&para;</a></h3>
<p>Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python</p>
<p><em>homepage</em>: <a href="https://lmfit.github.io/lmfit-py">https://lmfit.github.io/lmfit-py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.9</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.9.14</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="lmod">Lmod<a class="headerlink" href="#lmod" title="Permanent link">&para;</a></h3>
<p>Lmod is a Lua based module system. Modules allow for dynamic modification  of a user's environment under Unix systems. See www.tacc.utexas.edu/tacc-projects/lmod  for a complete description. Lmod is a new implementation that easily handles the MODULEPATH  Hierarchical problem. It is drop-in replacement for TCL/C modules and reads TCL modulefiles directly.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/lmod/">http://sourceforge.net/projects/lmod/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.2.5</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.3</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.4</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.4.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.5</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.5.1</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.6</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.7</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.8.5</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.4</code></td>
</tr>
<tr>
<td><code>6.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="lncloom">LncLOOM<a class="headerlink" href="#lncloom" title="Permanent link">&para;</a></h3>
<p>LncLOOM is a graph-based framework that uses integer programming to identify combinations of short motifs that are deeply conserved in rapidly evolving sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/lncLOOM/lncLOOMv2/">https://github.com/lncLOOM/lncLOOMv2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="locarna">LocARNA<a class="headerlink" href="#locarna" title="Permanent link">&para;</a></h3>
<p>LocARNA is a collection of alignment tools for the structural analysis of RNA.  Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for  your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for  analyzing sets of related RNAs without known common structure.</p>
<p><em>homepage</em>: <a href="http://www.bioinf.uni-freiburg.de/Software/LocARNA/">http://www.bioinf.uni-freiburg.de/Software/LocARNA/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.2</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.9.2.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.9.2.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="lofreq">LoFreq<a class="headerlink" href="#lofreq" title="Permanent link">&para;</a></h3>
<p>Fast and sensitive variant calling from next-gen sequencing data</p>
<p><em>homepage</em>: <a href="http://csb5.github.io/lofreq">http://csb5.github.io/lofreq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.3.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="log-log4perl">Log-Log4perl<a class="headerlink" href="#log-log4perl" title="Permanent link">&para;</a></h3>
<p>Log4perl</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Log::Log4perl">https://metacpan.org/pod/Log::Log4perl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.47</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="logaddexp">logaddexp<a class="headerlink" href="#logaddexp" title="Permanent link">&para;</a></h3>
<p>C library that implements the logarithm of the sum of exponentiations. Inspired by NumPy's logaddexp function.</p>
<p><em>homepage</em>: <a href="https://github.com/horta/logaddexp">https://github.com/horta/logaddexp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lohhla">LOHHLA<a class="headerlink" href="#lohhla" title="Permanent link">&para;</a></h3>
<p>LOHHLA, Loss Of Heterozygosity in Human Leukocyte Antigen, a computational tool to evaluate HLA loss using next-generation sequencing data.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/mcgranahanlab/lohhla">https://bitbucket.org/mcgranahanlab/lohhla</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.11.05</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="loki">Loki<a class="headerlink" href="#loki" title="Permanent link">&para;</a></h3>
<p>Loki is a C++ library of designs, containing flexible implementations of common design patterns and   idioms. </p>
<p><em>homepage</em>: <a href="http://loki-lib.sourceforge.net/">http://loki-lib.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.7</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="longestrunsubsequence">longestrunsubsequence<a class="headerlink" href="#longestrunsubsequence" title="Permanent link">&para;</a></h3>
<p>Implementation of a solver for the Longest Run Subsequence Problem. Given a  sequence as input, compute a longest subsequence such that there is at most one run for every  character. </p>
<p><em>homepage</em>: <a href="https://pypi.org/project/longestrunsubsequence/">https://pypi.org/project/longestrunsubsequence/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="longread_umi">longread_umi<a class="headerlink" href="#longread_umi" title="Permanent link">&para;</a></h3>
<p>A collection of scripts for processing longread UMI data.</p>
<p><em>homepage</em>: <a href="https://github.com/SorenKarst/longread_umi">https://github.com/SorenKarst/longread_umi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="longshot">Longshot<a class="headerlink" href="#longshot" title="Permanent link">&para;</a></h3>
<p>Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific  Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs  a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can  be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).</p>
<p><em>homepage</em>: <a href="https://github.com/pjedge/longshot">https://github.com/pjedge/longshot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.4.1</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.4.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.4.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="loompy">loompy<a class="headerlink" href="#loompy" title="Permanent link">&para;</a></h3>
<p>Python implementation of the Loom file format, an efficient file format for large omics datasets</p>
<p><em>homepage</em>: <a href="https://loompy.org/">https://loompy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.6</code></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.0.7</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="loomr">loomR<a class="headerlink" href="#loomr" title="Permanent link">&para;</a></h3>
<p>An R interface for loom files</p>
<p><em>homepage</em>: <a href="https://github.com/mojaveazure/loomR">https://github.com/mojaveazure/loomR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0-20180425</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="lpeg">LPeg<a class="headerlink" href="#lpeg" title="Permanent link">&para;</a></h3>
<p>LPeg is a new pattern-matching library for Lua, based on Parsing Expression Grammars (PEGs).</p>
<p><em>homepage</em>: <a href="http://www.inf.puc-rio.br/~roberto/lpeg/">http://www.inf.puc-rio.br/~roberto/lpeg/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lpjml">LPJmL<a class="headerlink" href="#lpjml" title="Permanent link">&para;</a></h3>
<p>Dynamic global vegetation model with managed land and river routing</p>
<p><em>homepage</em>: <a href="https://github.com/PIK-LPJmL/LPJmL">https://github.com/PIK-LPJmL/LPJmL</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.003</code></td>
<td><code>iimpi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="lpsolve">lpsolve<a class="headerlink" href="#lpsolve" title="Permanent link">&para;</a></h3>
<p>Mixed Integer Linear Programming (MILP) solver</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/lpsolve/">https://sourceforge.net/projects/lpsolve/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.5.2.5</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/8.3.0</code>, <code>foss/2018a</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>iccifort/2019.5.281</code>, <code>intel/2017a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.5.2.11</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lrslib">lrslib<a class="headerlink" href="#lrslib" title="Permanent link">&para;</a></h3>
<p>lrslib is a self-contained ANSI C implementation of the reverse search algorithm for vertex enumeration/convex hull problems</p>
<p><em>homepage</em>: <a href="http://cgm.cs.mcgill.ca/~avis/C/lrs.html">http://cgm.cs.mcgill.ca/~avis/C/lrs.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>7.0a</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ls-prepost">LS-PrePost<a class="headerlink" href="#ls-prepost" title="Permanent link">&para;</a></h3>
<p>LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA.</p>
<p><em>homepage</em>: <a href="http://lstc.com/products/ls-prepost">http://lstc.com/products/ls-prepost</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6</code></td>
<td><code>-centos6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6</code></td>
<td><code>-centos7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.24</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.7.8</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.7.15</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="lsd2">LSD2<a class="headerlink" href="#lsd2" title="Permanent link">&para;</a></h3>
<p>Least-squares methods to estimate rates and dates from phylogenies</p>
<p><em>homepage</em>: <a href="https://github.com/tothuhien/lsd2">https://github.com/tothuhien/lsd2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.7</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lsms">LSMS<a class="headerlink" href="#lsms" title="Permanent link">&para;</a></h3>
<p>LSMS benchmark, part of CORAL suite</p>
<p><em>homepage</em>: <a href="https://asc.llnl.gov/CORAL-benchmarks/#lsms">https://asc.llnl.gov/CORAL-benchmarks/#lsms</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3_rev237</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="ltr_retriever">LTR_retriever<a class="headerlink" href="#ltr_retriever" title="Permanent link">&para;</a></h3>
<p>LTR_retriever is a highly accurate and sensitive program for  identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also  included in this package. </p>
<p><em>homepage</em>: <a href="https://github.com/oushujun/LTR_retriever">https://github.com/oushujun/LTR_retriever</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.0</code></td>
<td><code>foss/2020b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ltrdetector">LtrDetector<a class="headerlink" href="#ltrdetector" title="Permanent link">&para;</a></h3>
<p>A modern tool-suite for detectinglong terminal repeat retrotransposons de-novo onthe genomic scale</p>
<p><em>homepage</em>: <a href="https://github.com/TulsaBioinformaticsToolsmith/LtrDetector">https://github.com/TulsaBioinformaticsToolsmith/LtrDetector</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="lua">Lua<a class="headerlink" href="#lua" title="Permanent link">&para;</a></h3>
<p>Lua is a powerful, fast, lightweight, embeddable scripting language.  Lua combines simple procedural syntax with powerful data description constructs based  on associative arrays and extensible semantics. Lua is dynamically typed,  runs by interpreting bytecode for a register-based virtual machine,  and has automatic memory management with incremental garbage collection,  making it ideal for configuration, scripting, and rapid prototyping.</p>
<p><em>homepage</em>: <a href="https://www.lua.org/">https://www.lua.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.4-5</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.1.4-8</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.4</code>, <code>system</code></td>
</tr>
<tr>
<td><code>5.1.5</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.2.4</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.3.4</code></td>
<td><code>GCCcore/7.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>5.3.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>5.4.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.4.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.4.4</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="luajit">LuaJIT<a class="headerlink" href="#luajit" title="Permanent link">&para;</a></h3>
<p>LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. </p>
<p><em>homepage</em>: <a href="http://luajit.org/">http://luajit.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="luajit2-openresty">LuaJIT2-OpenResty<a class="headerlink" href="#luajit2-openresty" title="Permanent link">&para;</a></h3>
<p>openresty/luajit2 - OpenResty's maintained branch of LuaJIT. LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. </p>
<p><em>homepage</em>: <a href="https://github.com/openresty/luajit2">https://github.com/openresty/luajit2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1-20220411</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lucene-geo-gazetteer">Lucene-Geo-Gazetteer<a class="headerlink" href="#lucene-geo-gazetteer" title="Permanent link">&para;</a></h3>
<p>A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library  to create a searchable gazetteer.</p>
<p><em>homepage</em>: <a href="https://github.com/chrismattmann/lucene-geo-gazetteer">https://github.com/chrismattmann/lucene-geo-gazetteer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170718</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="lumpy">LUMPY<a class="headerlink" href="#lumpy" title="Permanent link">&para;</a></h3>
<p>A probabilistic framework for structural variant discovery. </p>
<p><em>homepage</em>: <a href="https://github.com/arq5x/lumpy-sv">https://github.com/arq5x/lumpy-sv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.13</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="luscus">LUSCUS<a class="headerlink" href="#luscus" title="Permanent link">&para;</a></h3>
<p>Luscus is the program for graphical display and editing of molecular systems.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/luscus/">https://sourceforge.net/projects/luscus/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="lwgrp">lwgrp<a class="headerlink" href="#lwgrp" title="Permanent link">&para;</a></h3>
<p>The light-weight group library defines data structures and collective operations to group MPI processes as an ordered set.  Such groups are useful as substitutes for MPI communicators when the overhead of communicator creation is too costly.  For example, certain sorting algorithms recursively divide processes into subgroups as the sort algorithm progresses.  These groups may be different with each invocation, so that it is inefficient to create and destroy communicators during the sort routine.</p>
<p><em>homepage</em>: <a href="https://github.com/LLNL/lwgrp">https://github.com/LLNL/lwgrp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>gompi/2019a</code>, <code>gompi/2020a</code>, <code>iimpi/2019a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="lxml">lxml<a class="headerlink" href="#lxml" title="Permanent link">&para;</a></h3>
<p>The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.</p>
<p><em>homepage</em>: <a href="https://lxml.de/">https://lxml.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.6.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.2.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.2.5</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.3.3</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.4.2</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.6.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.6.3</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.9.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lynx">lynx<a class="headerlink" href="#lynx" title="Permanent link">&para;</a></h3>
<p>lynx is an alphanumeric display oriented World-Wide Web Client</p>
<p><em>homepage</em>: <a href="http://lynx.browser.org/">http://lynx.browser.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.9</code></td>
<td><code>-develop</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="lz4">lz4<a class="headerlink" href="#lz4" title="Permanent link">&para;</a></h3>
<p>LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core.</p>
<p><em>homepage</em>: <a href="https://lz4.github.io/lz4/">https://lz4.github.io/lz4/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.9.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="lzo">LZO<a class="headerlink" href="#lzo" title="Permanent link">&para;</a></h3>
<p>Portable lossless data compression library</p>
<p><em>homepage</em>: <a href="http://www.oberhumer.com/opensource/lzo/">http://www.oberhumer.com/opensource/lzo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.09</code></td>
<td><code>intel/2016b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h2 id="m">M<a class="headerlink" href="#m" title="Permanent link">&para;</a></h2>
<p><a href="#m1qn3">M1QN3</a> - <a href="#m4">M4</a> - <a href="#mach">MACH</a> - <a href="#macs2">MACS2</a> - <a href="#macse">MACSE</a> - <a href="#maeparser">maeparser</a> - <a href="#mafft">MAFFT</a> - <a href="#mageck">MAGeCK</a> - <a href="#magick">magick</a> - <a href="#magics">Magics</a> - <a href="#magma">magma</a> - <a href="#magma-gene-analysis">MAGMA-gene-analysis</a> - <a href="#magrespython">MagresPython</a> - <a href="#mahotas">mahotas</a> - <a href="#majiq">MAJIQ</a> - <a href="#make">make</a> - <a href="#makedepend">makedepend</a> - <a href="#makedepf90">makedepf90</a> - <a href="#makeinfo">makeinfo</a> - <a href="#maker">MAKER</a> - <a href="#mako">Mako</a> - <a href="#mamba">Mamba</a> - <a href="#manta">manta</a> - <a href="#mapdamage">mapDamage</a> - <a href="#maple">Maple</a> - <a href="#mapsplice">MapSplice</a> - <a href="#maq">Maq</a> - <a href="#mariadb">MariaDB</a> - <a href="#mariadb-connector-c">MariaDB-connector-c</a> - <a href="#mars">MARS</a> - <a href="#mash">Mash</a> - <a href="#masurca">MaSuRCA</a> - <a href="#mathematica">Mathematica</a> - <a href="#mathgl">MathGL</a> - <a href="#matio">MATIO</a> - <a href="#matlab">MATLAB</a> - <a href="#matlab-engine">MATLAB-Engine</a> - <a href="#matlab-proxy">matlab-proxy</a> - <a href="#matplotlib">matplotlib</a> - <a href="#matplotlib-inline">matplotlib-inline</a> - <a href="#matsim">MATSim</a> - <a href="#maude">Maude</a> - <a href="#mauvealigner">mauveAligner</a> - <a href="#maven">Maven</a> - <a href="#mawk">mawk</a> - <a href="#maxbin">MaxBin</a> - <a href="#maxquant">MaxQuant</a> - <a href="#mayavi">mayavi</a> - <a href="#maze">maze</a> - <a href="#mbedtls">MbedTLS</a> - <a href="#mbrola">MBROLA</a> - <a href="#mbuffer">mbuffer</a> - <a href="#mc">mc</a> - <a href="#mcl">MCL</a> - <a href="#mcr">MCR</a> - <a href="#mcu">mcu</a> - <a href="#mdanalysis">MDAnalysis</a> - <a href="#mdbm">MDBM</a> - <a href="#mdsplus">MDSplus</a> - <a href="#mdsplus-java">MDSplus-Java</a> - <a href="#mdsplus-python">MDSplus-Python</a> - <a href="#mdtest">mdtest</a> - <a href="#mdtraj">MDTraj</a> - <a href="#medaka">medaka</a> - <a href="#medimgproc">medImgProc</a> - <a href="#medpy">MedPy</a> - <a href="#meep">Meep</a> - <a href="#mega">MEGA</a> - <a href="#megacc">MEGACC</a> - <a href="#megahit">MEGAHIT</a> - <a href="#megalodon">Megalodon</a> - <a href="#megan">MEGAN</a> - <a href="#meld">Meld</a> - <a href="#mem">MEM</a> - <a href="#meme">MEME</a> - <a href="#memkind">memkind</a> - <a href="#memory-profiler">memory-profiler</a> - <a href="#memote">MEMOTE</a> - <a href="#memtester">memtester</a> - <a href="#merantk">meRanTK</a> - <a href="#merckx">MERCKX</a> - <a href="#mercurial">Mercurial</a> - <a href="#mesa">Mesa</a> - <a href="#mesa-demos">Mesa-demos</a> - <a href="#meshalyzer">meshalyzer</a> - <a href="#meshio">meshio</a> - <a href="#meshtool">meshtool</a> - <a href="#meson">Meson</a> - <a href="#meson-python">meson-python</a> - <a href="#mesquite">Mesquite</a> - <a href="#mess">MESS</a> - <a href="#metabat">MetaBAT</a> - <a href="#metaboanalystr">MetaboAnalystR</a> - <a href="#metaerg">metaerg</a> - <a href="#metaeuk">MetaEuk</a> - <a href="#metageneannotator">MetaGeneAnnotator</a> - <a href="#metagenome-atlas">Metagenome-Atlas</a> - <a href="#metal">Metal</a> - <a href="#metamorpheus">MetaMorpheus</a> - <a href="#metaphlan">MetaPhlAn</a> - <a href="#metaphlan2">MetaPhlAn2</a> - <a href="#metawrap">metaWRAP</a> - <a href="#metaxa2">Metaxa2</a> - <a href="#methylartist">methylartist</a> - <a href="#methyldackel">MethylDackel</a> - <a href="#methylpy">methylpy</a> - <a href="#metis">METIS</a> - <a href="#mgltools">mgltools</a> - <a href="#mhcflurry">mhcflurry</a> - <a href="#mhcnuggets">mhcnuggets</a> - <a href="#microbeannotator">MicrobeAnnotator</a> - <a href="#microctools">microctools</a> - <a href="#migec">MiGEC</a> - <a href="#migrate-n">MIGRATE-N</a> - <a href="#miller">Miller</a> - <a href="#mimalloc">mimalloc</a> - <a href="#minc">MINC</a> - <a href="#minced">MinCED</a> - <a href="#mini-xml">Mini-XML</a> - <a href="#miniasm">miniasm</a> - <a href="#minibar">minibar</a> - <a href="#minicard">MiniCARD</a> - <a href="#miniconda2">Miniconda2</a> - <a href="#miniconda3">Miniconda3</a> - <a href="#minieigen">minieigen</a> - <a href="#minimac4">Minimac4</a> - <a href="#minimap2">minimap2</a> - <a href="#minipolish">Minipolish</a> - <a href="#minisat">MiniSat</a> - <a href="#minizip">minizip</a> - <a href="#minpack">MINPACK</a> - <a href="#minpath">MinPath</a> - <a href="#mira">MIRA</a> - <a href="#mirdeep2">miRDeep2</a> - <a href="#mish-cuda">Mish-Cuda</a> - <a href="#misha">misha</a> - <a href="#mitgcmutils">MITgcmutils</a> - <a href="#mitobim">MITObim</a> - <a href="#mitoz">MitoZ</a> - <a href="#mixcr">MiXCR</a> - <a href="#mixmhc2pred">MixMHC2pred</a> - <a href="#mkl-dnn">mkl-dnn</a> - <a href="#mkl-service">mkl-service</a> - <a href="#mkl_fft">mkl_fft</a> - <a href="#mlc">MLC</a> - <a href="#mlxtend">MLxtend</a> - <a href="#mm-common">mm-common</a> - <a href="#mmg">Mmg</a> - <a href="#mmseq">MMSEQ</a> - <a href="#mmseqs2">MMseqs2</a> - <a href="#mmtf-cpp">mmtf-cpp</a> - <a href="#mne-python">MNE-Python</a> - <a href="#moabs">MOABS</a> - <a href="#modeltest-ng">ModelTest-NG</a> - <a href="#modred">modred</a> - <a href="#molcas">Molcas</a> - <a href="#mold">mold</a> - <a href="#molden">Molden</a> - <a href="#molekel">Molekel</a> - <a href="#molmod">molmod</a> - <a href="#molpro">Molpro</a> - <a href="#mona">MONA</a> - <a href="#monai">MONAI</a> - <a href="#mongolite">mongolite</a> - <a href="#mono">Mono</a> - <a href="#monocle3">Monocle3</a> - <a href="#moonjit">moonjit</a> - <a href="#moose">MOOSE</a> - <a href="#mordecai">mordecai</a> - <a href="#moreronn">MoreRONN</a> - <a href="#mosdepth">mosdepth</a> - <a href="#mothur">Mothur</a> - <a href="#motif">motif</a> - <a href="#motioncor2">MotionCor2</a> - <a href="#motionsegmentation">motionSegmentation</a> - <a href="#moviepy">MoviePy</a> - <a href="#mpb">MPB</a> - <a href="#mpc">MPC</a> - <a href="#mpfr">MPFR</a> - <a href="#mpi4py">mpi4py</a> - <a href="#mpich">MPICH</a> - <a href="#mpich2">MPICH2</a> - <a href="#mpifileutils">mpifileutils</a> - <a href="#mpip">mpiP</a> - <a href="#mpj-express">MPJ-Express</a> - <a href="#mpmath">mpmath</a> - <a href="#mrbayes">MrBayes</a> - <a href="#mrcfile">mrcfile</a> - <a href="#mrchem">MRChem</a> - <a href="#mrcpp">MRCPP</a> - <a href="#mricron">MRIcron</a> - <a href="#mrtrix">MRtrix</a> - <a href="#msgpack-c">msgpack-c</a> - <a href="#msm">MSM</a> - <a href="#mspc">MSPC</a> - <a href="#msprime">msprime</a> - <a href="#mstore">mstore</a> - <a href="#mtl4">MTL4</a> - <a href="#mujoco">MuJoCo</a> - <a href="#mujoco-py">mujoco-py</a> - <a href="#multichoose">multichoose</a> - <a href="#multilevelestimators">MultilevelEstimators</a> - <a href="#multinest">MultiNest</a> - <a href="#multiqc">MultiQC</a> - <a href="#multiwfn">Multiwfn</a> - <a href="#mumerge">muMerge</a> - <a href="#mummer">MUMmer</a> - <a href="#mumps">MUMPS</a> - <a href="#muparser">muParser</a> - <a href="#muparserx">muparserx</a> - <a href="#mupexi">MuPeXI</a> - <a href="#muscle">MUSCLE</a> - <a href="#music">MuSiC</a> - <a href="#must">MUST</a> - <a href="#mutect">MuTect</a> - <a href="#mutil">mutil</a> - <a href="#mvapich2">MVAPICH2</a> - <a href="#mxml">mxml</a> - <a href="#mxmlplus">mxmlplus</a> - <a href="#mxnet">MXNet</a> - <a href="#mycc">MyCC</a> - <a href="#mygene">mygene</a> - <a href="#mymedialite">MyMediaLite</a> - <a href="#mympingpong">mympingpong</a> - <a href="#myokit">Myokit</a> - <a href="#mypy">mypy</a> - <a href="#mysql">MySQL</a> - <a href="#mysql-python">MySQL-python</a> - <a href="#mysqlclient">mysqlclient</a></p>
<h3 id="m1qn3">M1QN3<a class="headerlink" href="#m1qn3" title="Permanent link">&para;</a></h3>
<p>A solver of large-scale unconstrained minimization problems</p>
<p><em>homepage</em>: <a href="https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html">https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="m4">M4<a class="headerlink" href="#m4" title="Permanent link">&para;</a></h3>
<p>GNU M4 is an implementation of the traditional Unix macro processor. It is  mostly SVR4 compatible although it has some extensions (for example, handling  more than 9 positional parameters to macros). GNU M4 also has built-in  functions for including files, running shell commands, doing arithmetic, etc. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/m4/m4.html">https://www.gnu.org/software/m4/m4.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.16</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.4.17</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.2-binutils-2.25</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/4.9.3-binutils-2.25</code>, <code>GCC/5.1.0-binutils-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/4.9.4</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>GNU/5.1.0-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.4.18</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.1.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/5.5.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.1.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/7.4.0</code>, <code>GCCcore/8.1.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/8.4.0</code>, <code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code>, <code>GCCcore/system</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.4.19</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.4.0</code>, <code>GCCcore/9.5.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mach">MACH<a class="headerlink" href="#mach" title="Permanent link">&para;</a></h3>
<p>MACH 1.0 is a Markov Chain based haplotyper  that can resolve long haplotypes or infer missing genotypes  in samples of unrelated individuals.</p>
<p><em>homepage</em>: <a href="http://csg.sph.umich.edu/abecasis/MACH/">http://csg.sph.umich.edu/abecasis/MACH/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.18</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="macs2">MACS2<a class="headerlink" href="#macs2" title="Permanent link">&para;</a></h3>
<p>Model Based Analysis for ChIP-Seq data</p>
<p><em>homepage</em>: <a href="https://github.com/taoliu/MACS">https://github.com/taoliu/MACS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1.20160309</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.7.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="macse">MACSE<a class="headerlink" href="#macse" title="Permanent link">&para;</a></h3>
<p>MACSE aligns coding NT sequences with respect to their AA translation while allowing NT sequences to contain multiple frameshifts and/or stop codons. MACSE is hence the first automatic solution to align protein-coding gene datasets containing non-functional sequences (pseudogenes) without disrupting the underlying codon structure.</p>
<p><em>homepage</em>: <a href="https://bioweb.supagro.inra.fr/macse/index.php">https://bioweb.supagro.inra.fr/macse/index.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.06</code></td>
<td><code>-Java-15</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="maeparser">maeparser<a class="headerlink" href="#maeparser" title="Permanent link">&para;</a></h3>
<p>maeparser is a parser for Schrodinger Maestro files.</p>
<p><em>homepage</em>: <a href="https://github.com/schrodinger/maeparser">https://github.com/schrodinger/maeparser</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2021a</code>, <code>gompi/2022a</code>, <code>iimpi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mafft">MAFFT<a class="headerlink" href="#mafft" title="Permanent link">&para;</a></h3>
<p>MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc.</p>
<p><em>homepage</em>: <a href="https://mafft.cbrc.jp/alignment/software/source.html">https://mafft.cbrc.jp/alignment/software/source.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.305</code></td>
<td><code>-with-extensions</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>7.397</code></td>
<td><code>-with-extensions</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>7.427</code></td>
<td><code>-with-extensions</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>7.429</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>7.453</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>gompi/2020a</code>, <code>iccifort/2019.5.281</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>7.470</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/9.3.0</code>, <code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>7.471</code></td>
<td><code>-with-extensions</code></td>
<td><code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>7.475</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/10.2.0</code>, <code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>7.487</code></td>
<td><code>-with-extensions</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>7.490</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>7.505</code></td>
<td><code>-with-extensions</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mageck">MAGeCK<a class="headerlink" href="#mageck" title="Permanent link">&para;</a></h3>
<p>Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify  important genes from the recent genome-scale CRISPR-Cas9 knockout screens (or GeCKO) technology. MAGeCK is developed  by Wei Li and Han Xu from Dr. Xiaole Shirley Liu's lab at Dana-Farber Cancer Institute, and is being actively updated  by Wei Li lab from Children's National Medical Center.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/p/mageck/wiki/Home/">https://sourceforge.net/p/mageck/wiki/Home/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.9.4</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="magick">magick<a class="headerlink" href="#magick" title="Permanent link">&para;</a></h3>
<p>R bindings to the open-source image processing library ImageMagick</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/magick/">https://cran.r-project.org/web/packages/magick/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="magics">Magics<a class="headerlink" href="#magics" title="Permanent link">&para;</a></h3>
<p>Magicsis ECMWF's meteorological plotting software and can be either accessed directly through its Python orFortran interfaces or by using Metview.</p>
<p><em>homepage</em>: <a href="https://confluence.ecmwf.int/display/MAGP/">https://confluence.ecmwf.int/display/MAGP/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.13.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="magma">magma<a class="headerlink" href="#magma" title="Permanent link">&para;</a></h3>
<p>The MAGMA project aims to develop a dense linear algebra library similar to  LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.</p>
<p><em>homepage</em>: <a href="https://icl.cs.utk.edu/magma/">https://icl.cs.utk.edu/magma/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>fosscuda/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>fosscuda/2019a</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.5.4</code></td>
<td></td>
<td><code>fosscuda/2019a</code>, <code>fosscuda/2019b</code>, <code>fosscuda/2020a</code>, <code>fosscuda/2020b</code>, <code>intelcuda/2019b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="magma-gene-analysis">MAGMA-gene-analysis<a class="headerlink" href="#magma-gene-analysis" title="Permanent link">&para;</a></h3>
<p>MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous  GWAS or meta-analysis.</p>
<p><em>homepage</em>: <a href="https://ctg.cncr.nl/software/magma">https://ctg.cncr.nl/software/magma</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.07b</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.07bb</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.09b</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="magrespython">MagresPython<a class="headerlink" href="#magrespython" title="Permanent link">&para;</a></h3>
<p>MagresPython is a Python library for parsing the CCP-NC ab-initio magnetic resonance file format.  This is used in the latest version of the CASTEP and Quantum ESPRESSO (PWSCF) codes. </p>
<p><em>homepage</em>: <a href="http://tfgg.me/magres-python">http://tfgg.me/magres-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160329</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="mahotas">mahotas<a class="headerlink" href="#mahotas" title="Permanent link">&para;</a></h3>
<p>Mahotas is a computer vision and image processing library for Python</p>
<p><em>homepage</em>: <a href="https://mahotas.readthedocs.io/">https://mahotas.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="majiq">MAJIQ<a class="headerlink" href="#majiq" title="Permanent link">&para;</a></h3>
<p>MAJIQ and Voila are two software packages that together detect, quantify, and visualize  local splicing variations (LSV) from RNA-Seq data.</p>
<p><em>homepage</em>: <a href="https://majiq.biociphers.org">https://majiq.biociphers.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="make">make<a class="headerlink" href="#make" title="Permanent link">&para;</a></h3>
<p>GNU version of make utility</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/make/make.html">https://www.gnu.org/software/make/make.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.82</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="makedepend">makedepend<a class="headerlink" href="#makedepend" title="Permanent link">&para;</a></h3>
<p>The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.</p>
<p><em>homepage</em>: <a href="https://linux.die.net/man/1/makedepend">https://linux.die.net/man/1/makedepend</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="makedepf90">makedepf90<a class="headerlink" href="#makedepf90" title="Permanent link">&para;</a></h3>
<p>Makedepf90 is a program for automatic creation of  Makefile-style dependency lists for Fortran source code. </p>
<p><em>homepage</em>: <a href="https://linux.die.net/man/1/makedepf90">https://linux.die.net/man/1/makedepf90</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.8</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="makeinfo">makeinfo<a class="headerlink" href="#makeinfo" title="Permanent link">&para;</a></h3>
<p>makeinfo is part of the Texinfo project, the official documentation format of the GNU project.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/texinfo/">https://www.gnu.org/software/texinfo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.7</code></td>
<td></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.7</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.8</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="maker">MAKER<a class="headerlink" href="#maker" title="Permanent link">&para;</a></h3>
<p>MAKER is a portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases. </p>
<p><em>homepage</em>: <a href="https://yandell-lab.org/software/maker.html">https://yandell-lab.org/software/maker.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.01.04</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mako">Mako<a class="headerlink" href="#mako" title="Permanent link">&para;</a></h3>
<p>A super-fast templating language that borrows the best ideas from the existing templating languages</p>
<p><em>homepage</em>: <a href="https://www.makotemplates.org">https://www.makotemplates.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>intel/2017b</code>, <code>intel/2018.01</code>, <code>intel/2018a</code>, <code>intelcuda/2017b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.0.8</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mamba">Mamba<a class="headerlink" href="#mamba" title="Permanent link">&para;</a></h3>
<p>Mamba is a fast, robust, and cross-platform package manager. It runs on Windows, OS X and Linux  (ARM64 and PPC64LE included) and is fully compatible with conda packages and supports most of conda's commands. </p>
<p><em>homepage</em>: <a href="https://mamba.readthedocs.io/">https://mamba.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.14.0-0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="manta">manta<a class="headerlink" href="#manta" title="Permanent link">&para;</a></h3>
<p>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. It is optimized for analysis of germline variation in small sets of individuals and  somatic variation in tumor/normal sample pairs. Manta discovers, assembles and  scores large-scale SVs, medium-sized indels and large insertions within a  single efficient workflow.  </p>
<p><em>homepage</em>: <a href="https://github.com/Illumina/manta">https://github.com/Illumina/manta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code>, <code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mapdamage">mapDamage<a class="headerlink" href="#mapdamage" title="Permanent link">&para;</a></h3>
<p>mapDamage2 is a computational framework written in Python and     R, which tracks and quantifies DNA damage patterns among ancient DNA     sequencing reads generated by Next-Generation Sequencing platforms.</p>
<p><em>homepage</em>: <a href="https://ginolhac.github.io/mapDamage/">https://ginolhac.github.io/mapDamage/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="maple">Maple<a class="headerlink" href="#maple" title="Permanent link">&para;</a></h3>
<p>Maple combines the world's most powerful mathematical computation engine with an intuitive,    'clickable' user interface.</p>
<p><em>homepage</em>: <a href="http://www.maplesoft.com/products/maple/">http://www.maplesoft.com/products/maple/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mapsplice">MapSplice<a class="headerlink" href="#mapsplice" title="Permanent link">&para;</a></h3>
<p>MapSplice is a software for mapping RNA-seq data to reference genome for splice  junction discovery that depends only on reference genome, and not on any further annotations.</p>
<p><em>homepage</em>: <a href="http://www.netlab.uky.edu/p/bioinfo/MapSplice2">http://www.netlab.uky.edu/p/bioinfo/MapSplice2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="maq">Maq<a class="headerlink" href="#maq" title="Permanent link">&para;</a></h3>
<p>Maq is a software that builds mapping assemblies from short reads generated by the  next-generation sequencing  machines.</p>
<p><em>homepage</em>: <a href="http://maq.sourceforge.net/maq-man.shtml">http://maq.sourceforge.net/maq-man.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mariadb">MariaDB<a class="headerlink" href="#mariadb" title="Permanent link">&para;</a></h3>
<p>MariaDB is an enhanced, drop-in replacement for MySQL. Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.</p>
<p><em>homepage</em>: <a href="https://mariadb.org/">https://mariadb.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>10.0.21</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>10.1.13</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>10.1.14</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>10.1.17</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>10.1.24</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>10.2.11</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>10.3.7</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>10.3.10</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>10.3.14</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>10.4.13</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>10.5.8</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>10.6.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>10.9.3</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mariadb-connector-c">MariaDB-connector-c<a class="headerlink" href="#mariadb-connector-c" title="Permanent link">&para;</a></h3>
<p>MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.</p>
<p><em>homepage</em>: <a href="https://downloads.mariadb.org/connector-c/">https://downloads.mariadb.org/connector-c/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.4</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.5</code></td>
<td><code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.3.7</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.7</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.1.11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mars">MARS<a class="headerlink" href="#mars" title="Permanent link">&para;</a></h3>
<p>improving Multiple circular sequence Alignment using Refined Sequences</p>
<p><em>homepage</em>: <a href="https://github.com/lorrainea/MARS">https://github.com/lorrainea/MARS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20191101</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mash">Mash<a class="headerlink" href="#mash" title="Permanent link">&para;</a></h3>
<p>Fast genome and metagenome distance estimation using MinHash</p>
<p><em>homepage</em>: <a href="http://mash.readthedocs.org">http://mash.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="masurca">MaSuRCA<a class="headerlink" href="#masurca" title="Permanent link">&para;</a></h3>
<p>MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing  only short reads from Illumina sequencing or a mixture of short reads and long reads  (Sanger, 454, Pacbio and Nanopore).</p>
<p><em>homepage</em>: <a href="https://www.genome.umd.edu/masurca.html">https://www.genome.umd.edu/masurca.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.2.4</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.2.5</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-Perl-5.30.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.0.9</code></td>
<td><code>-Perl-5.32.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mathematica">Mathematica<a class="headerlink" href="#mathematica" title="Permanent link">&para;</a></h3>
<p>Mathematica is a computational software program used in many scientific, engineering, mathematical and computing fields.</p>
<p><em>homepage</em>: <a href="http://www.wolfram.com/mathematica">http://www.wolfram.com/mathematica</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>13.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>13.1.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mathgl">MathGL<a class="headerlink" href="#mathgl" title="Permanent link">&para;</a></h3>
<p>MathGL is ...  a library for making high-quality scientific graphics under Linux and Windows;  a library for the fast data plotting and data processing of large data arrays;  a library for working in window and console modes and for easy embedding into other programs;  a library with large and growing set of graphics. </p>
<p><em>homepage</em>: <a href="http://mathgl.sourceforge.net/doc_en/Main.html">http://mathgl.sourceforge.net/doc_en/Main.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="matio">MATIO<a class="headerlink" href="#matio" title="Permanent link">&para;</a></h3>
<p>matio is an C library for reading and writing Matlab MAT files.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/matio/">http://sourceforge.net/projects/matio/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.9</code></td>
<td><code>GCCcore/5.4.0</code></td>
</tr>
<tr>
<td><code>1.5.11</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.5.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.5.17</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.5.19</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.5.21</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.5.23</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="matlab">MATLAB<a class="headerlink" href="#matlab" title="Permanent link">&para;</a></h3>
<p>MATLAB is a high-level language and interactive environment  that enables you to perform computationally intensive tasks faster than with  traditional programming languages such as C, C++, and Fortran.</p>
<p><em>homepage</em>: <a href="https://www.mathworks.com/products/matlab">https://www.mathworks.com/products/matlab</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2012b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2015a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022a</code></td>
<td><code>-r3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="matlab-engine">MATLAB-Engine<a class="headerlink" href="#matlab-engine" title="Permanent link">&para;</a></h3>
<p>The MATLAB Engine API for Python provides a package for Python  to call MATLAB as a computational engine.</p>
<p><em>homepage</em>: <a href="https://www.mathworks.com/help/matlab/matlab-engine-for-python.html">https://www.mathworks.com/help/matlab/matlab-engine-for-python.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018b</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2018b</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2019b</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2021a-9.10.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2021b-9.11.19</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="matlab-proxy">matlab-proxy<a class="headerlink" href="#matlab-proxy" title="Permanent link">&para;</a></h3>
<p>A Python package which enables you to launch MATLAB and access it from a web browser.</p>
<p><em>homepage</em>: <a href="https://github.com/mathworks/matlab-proxy">https://github.com/mathworks/matlab-proxy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="matplotlib">matplotlib<a class="headerlink" href="#matplotlib" title="Permanent link">&para;</a></h3>
<p>matplotlib is a python 2D plotting library which produces publication quality figures in a variety of  hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python  and ipython shell, web application servers, and six graphical user interface toolkits.</p>
<p><em>homepage</em>: <a href="https://matplotlib.org">https://matplotlib.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-2.7.11-freetype-2.6.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.13-Qt-4.8.7</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.13-libpng-1.6.29</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.6.1-libpng-1.6.29</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018.02</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code>, <code>intelcuda/2019a</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021b</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code>, <code>intelcuda/2019a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code>, <code>intelcuda/2020a</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>gomkl/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td></td>
<td><code>gfbf/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="matplotlib-inline">matplotlib-inline<a class="headerlink" href="#matplotlib-inline" title="Permanent link">&para;</a></h3>
<p>Matplotlib Inline Back-end for IPython and Jupyter.</p>
<p><em>homepage</em>: <a href="https://github.com/ipython/matplotlib-inline">https://github.com/ipython/matplotlib-inline</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="matsim">MATSim<a class="headerlink" href="#matsim" title="Permanent link">&para;</a></h3>
<p>MATSim is an open-source framework to implement large-scale agent-based transport simulations.</p>
<p><em>homepage</em>: <a href="https://www.matsim.org/">https://www.matsim.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>14.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="maude">Maude<a class="headerlink" href="#maude" title="Permanent link">&para;</a></h3>
<p>Maude is a high-performance reflective language and system supporting both equational and rewriting  logic specification and programming for a wide range of applications. Maude has been influenced in important ways by  the OBJ3 language, which can be regarded as an equational logic sublanguage. Besides supporting equational  specification and programming, Maude also supports rewriting logic computation. </p>
<p><em>homepage</em>: <a href="https://maude.cs.illinois.edu/w/index.php/The_Maude_System">https://maude.cs.illinois.edu/w/index.php/The_Maude_System</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mauvealigner">mauveAligner<a class="headerlink" href="#mauvealigner" title="Permanent link">&para;</a></h3>
<p>Mauve is a system for constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. Multiple genome alignments provide a basis for research into comparative genomics and the study of genome-wide evolutionary dynamics. This version was built without Graphical User Interface. </p>
<p><em>homepage</em>: <a href="https://darlinglab.org/mauve/mauve.html">https://darlinglab.org/mauve/mauve.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4736</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="maven">Maven<a class="headerlink" href="#maven" title="Permanent link">&para;</a></h3>
<p>Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. </p>
<p><em>homepage</em>: <a href="https://maven.apache.org/index.html">https://maven.apache.org/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.9</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mawk">mawk<a class="headerlink" href="#mawk" title="Permanent link">&para;</a></h3>
<p>mawk is an interpreter for the AWK Programming Language.</p>
<p><em>homepage</em>: <a href="https://invisible-island.net/mawk/">https://invisible-island.net/mawk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.4-20141206</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.3.4-20171017</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="maxbin">MaxBin<a class="headerlink" href="#maxbin" title="Permanent link">&para;</a></h3>
<p>MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/maxbin/">https://sourceforge.net/projects/maxbin/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.6</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="maxquant">MaxQuant<a class="headerlink" href="#maxquant" title="Permanent link">&para;</a></h3>
<p>MaxQuant is a quantitative proteomics software package designed for analyzing large  mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling  techniques as well as label-free quantification are supported.</p>
<p><em>homepage</em>: <a href="https://maxquant.org/maxquant">https://maxquant.org/maxquant</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.10.43</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.3.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.2.0.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mayavi">mayavi<a class="headerlink" href="#mayavi" title="Permanent link">&para;</a></h3>
<p>The Mayavi scientific data 3-dimensional visualizer</p>
<p><em>homepage</em>: <a href="http://code.enthought.com/projects/mayavi/">http://code.enthought.com/projects/mayavi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.6.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.7.4</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="maze">maze<a class="headerlink" href="#maze" title="Permanent link">&para;</a></h3>
<p>In a similar manner to dot plots, maze highlights local sequence similarity between two DNA sequences. In particular, maximal exact substring matches are computed with MUMmer3 and visualised.</p>
<p><em>homepage</em>: <a href="https://github.com/dellytools/maze">https://github.com/dellytools/maze</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170124</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mbedtls">MbedTLS<a class="headerlink" href="#mbedtls" title="Permanent link">&para;</a></h3>
<p>An open source, portable, easy to use, readable and flexible SSL library.</p>
<p><em>homepage</em>: <a href="https://tls.mbed.org/">https://tls.mbed.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.26.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mbrola">MBROLA<a class="headerlink" href="#mbrola" title="Permanent link">&para;</a></h3>
<p>MBROLA is a speech synthesizer based on the concatenation of diphones. It takes a list of phonemes as input, together with prosodic information (duration of phonemes and a piecewise linear description of pitch), and produces speech samples on 16 bits (linear), at the sampling frequency of the diphone database.  MBROLA voices project provides list of MBROLA speech synthesizer voices. It is intended to provide easier collaboration and automatic updates for individual users and packagers. </p>
<p><em>homepage</em>: &lt;['https://github.com/numediart/MBROLA', 'https://github.com/numediart/MBROLA-voices']&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3</code></td>
<td><code>-voices-20200330</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mbuffer">mbuffer<a class="headerlink" href="#mbuffer" title="Permanent link">&para;</a></h3>
<p>mbuffer is a tool for buffering data streams with a large set of unique features. </p>
<p><em>homepage</em>: <a href="https://www.maier-komor.de/mbuffer.html">https://www.maier-komor.de/mbuffer.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20191016</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mc">mc<a class="headerlink" href="#mc" title="Permanent link">&para;</a></h3>
<p>mc-4.6.1: User-friendly file manager and visual shell</p>
<p><em>homepage</em>: <a href="https://www.midnight-commander.org/">https://www.midnight-commander.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.13</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="mcl">MCL<a class="headerlink" href="#mcl" title="Permanent link">&para;</a></h3>
<p>The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for graphs (also known as networks) based on simulation of (stochastic) flow in graphs. </p>
<p><em>homepage</em>: <a href="https://micans.org/mcl/">https://micans.org/mcl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>02.063</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>14.137</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>14.137</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>14.137</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>22.282</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mcr">MCR<a class="headerlink" href="#mcr" title="Permanent link">&para;</a></h3>
<p>The MATLAB Runtime is a standalone set of shared libraries  that enables the execution of compiled MATLAB applications  or components on computers that do not have MATLAB installed.</p>
<p><em>homepage</em>: <a href="https://www.mathworks.com/products/compiler/mcr/">https://www.mathworks.com/products/compiler/mcr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>R2013a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2013b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2014a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2014b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2015a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2015b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2016a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2016b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2018a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2018b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2019a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2019b</code></td>
<td><code>.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2020a</code></td>
<td><code>.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2020b</code></td>
<td><code>.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2021a</code></td>
<td><code>.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2021a</code></td>
<td><code>.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2021b</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2021b</code></td>
<td><code>.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2021b</code></td>
<td><code>.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2022a</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>R2022a</code></td>
<td><code>.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mcu">mcu<a class="headerlink" href="#mcu" title="Permanent link">&para;</a></h3>
<p>A package for periodic wavefunction and crystallography analysis.  mcu is designed to support large scale analysis and topological descriptions for periodic wavefunction.</p>
<p><em>homepage</em>: <a href="https://hungqpham.com/mcu/">https://hungqpham.com/mcu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021-04-06</code></td>
<td><code>gomkl/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="mdanalysis">MDAnalysis<a class="headerlink" href="#mdanalysis" title="Permanent link">&para;</a></h3>
<p>MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD) simulations in many popular formats.</p>
<p><em>homepage</em>: <a href="https://www.mdanalysis.org/">https://www.mdanalysis.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.20.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="mdbm">MDBM<a class="headerlink" href="#mdbm" title="Permanent link">&para;</a></h3>
<p>MDBM is a super-fast memory-mapped key/value store</p>
<p><em>homepage</em>: <a href="https://github.com/yahoo/mdbm">https://github.com/yahoo/mdbm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.13.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mdsplus">MDSplus<a class="headerlink" href="#mdsplus" title="Permanent link">&para;</a></h3>
<p>MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.</p>
<p><em>homepage</em>: <a href="https://mdsplus.org/">https://mdsplus.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.67</code></td>
<td><code>-Java-1.7.0_79-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>7.46.1</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>7.96.8</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.96.12</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mdsplus-java">MDSplus-Java<a class="headerlink" href="#mdsplus-java" title="Permanent link">&para;</a></h3>
<p>MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.</p>
<p><em>homepage</em>: <a href="https://mdsplus.org">https://mdsplus.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.96.12</code></td>
<td><code>-Java-13</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mdsplus-python">MDSplus-Python<a class="headerlink" href="#mdsplus-python" title="Permanent link">&para;</a></h3>
<p>MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.</p>
<p><em>homepage</em>: <a href="https://mdsplus.org">https://mdsplus.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.96.12</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mdtest">mdtest<a class="headerlink" href="#mdtest" title="Permanent link">&para;</a></h3>
<p>mdtest is an MPI-coordinated metadata benchmark test that performs   open/stat/close operations on files and directories and then reports the performance.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/mdtest/">http://sourceforge.net/projects/mdtest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.3</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="mdtraj">MDTraj<a class="headerlink" href="#mdtraj" title="Permanent link">&para;</a></h3>
<p>Read, write and analyze MD trajectories with only a few lines of Python code.</p>
<p><em>homepage</em>: <a href="https://mdtraj.org">https://mdtraj.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.9.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.9.7</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="medaka">medaka<a class="headerlink" href="#medaka" title="Permanent link">&para;</a></h3>
<p>medaka is a tool to create a consensus sequence from nanopore sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/medaka">https://github.com/nanoporetech/medaka</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.11.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="medimgproc">medImgProc<a class="headerlink" href="#medimgproc" title="Permanent link">&para;</a></h3>
<p>Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.</p>
<p><em>homepage</em>: <a href="https://github.com/WeiXuanChan/motionSegmentation">https://github.com/WeiXuanChan/motionSegmentation</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.7</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="medpy">MedPy<a class="headerlink" href="#medpy" title="Permanent link">&para;</a></h3>
<p>MedPy is a library and script collection for medical image processing in Python, providing basic  functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main  contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready  to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/MedPy/">https://pypi.org/project/MedPy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="meep">Meep<a class="headerlink" href="#meep" title="Permanent link">&para;</a></h3>
<p>Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package  developed at MIT to model electromagnetic systems.</p>
<p><em>homepage</em>: <a href="https://meep.readthedocs.io/en/latest/">https://meep.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="mega">MEGA<a class="headerlink" href="#mega" title="Permanent link">&para;</a></h3>
<p>MEGA-CC (Molecular Evolutionary Genetics Analysis Computational Core) is an integrated suite of tools  for statistics-based comparative analysis of molecular sequence data based on evolutionary principles.</p>
<p><em>homepage</em>: <a href="http://www.megasoftware.net/">http://www.megasoftware.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.20-1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>10.0.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.10</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="megacc">MEGACC<a class="headerlink" href="#megacc" title="Permanent link">&para;</a></h3>
<p>MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and   protein sequence data from species and populations.</p>
<p><em>homepage</em>: <a href="http://www.megasoftware.net">http://www.megasoftware.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.18-1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="megahit">MEGAHIT<a class="headerlink" href="#megahit" title="Permanent link">&para;</a></h3>
<p>An ultra-fast single-node solution for large and complex  metagenomics assembly via succinct de Bruijn graph</p>
<p><em>homepage</em>: <a href="https://github.com/voutcn/megahit">https://github.com/voutcn/megahit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.8</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.2.9</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.9</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="megalodon">Megalodon<a class="headerlink" href="#megalodon" title="Permanent link">&para;</a></h3>
<p>Megalodon is a research command line tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads by anchoring the information rich basecalling neural network output to a reference genome/transriptome.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/megalodon">https://github.com/nanoporetech/megalodon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.5</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="megan">MEGAN<a class="headerlink" href="#megan" title="Permanent link">&para;</a></h3>
<p>MEGAN is a comprehensive toolbox for interactively analyzing microbiome data</p>
<p><em>homepage</em>: <a href="http://megan.informatik.uni-tuebingen.de/">http://megan.informatik.uni-tuebingen.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.22.0</code></td>
<td><code>-Java-17</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="meld">Meld<a class="headerlink" href="#meld" title="Permanent link">&para;</a></h3>
<p>Meld is a visual diff and merge tool targeted at developers.   Meld helps you compare files, directories, and version controlled projects.   It provides two- and three-way comparison of both files and directories, and has support   for many popular version control systems.   </p>
<p><em>homepage</em>: <a href="https://meldmerge.org/">https://meldmerge.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.20.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mem">MEM<a class="headerlink" href="#mem" title="Permanent link">&para;</a></h3>
<p>Marker Enrichment Modeling (MEM) is a tool designed to calculate enrichment scores.</p>
<p><em>homepage</em>: <a href="https://github.com/cytolab/mem">https://github.com/cytolab/mem</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20191023</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>20191023</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="meme">MEME<a class="headerlink" href="#meme" title="Permanent link">&para;</a></h3>
<p>The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or  GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using  MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate  motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment  using SpaMo or CentriMo.</p>
<p><em>homepage</em>: <a href="https://meme-suite.org/meme/index.html">https://meme-suite.org/meme/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.4</code></td>
<td><code>-Perl-5.26.0-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0.4</code></td>
<td><code>-Perl-5.26.0-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="memkind">memkind<a class="headerlink" href="#memkind" title="Permanent link">&para;</a></h3>
<p>User Extensible Heap Manager built on top of jemalloc which enables control of memory characteristics  and a partitioning of the heap between kinds of memory. </p>
<p><em>homepage</em>: <a href="http://memkind.github.io">http://memkind.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/5.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="memory-profiler">memory-profiler<a class="headerlink" href="#memory-profiler" title="Permanent link">&para;</a></h3>
<p>memory-profiler is a Python module for monitoring memory consumption of a process  as well as line-by-line analysis of memory consumption for python programs.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/memory-profiler">https://pypi.org/project/memory-profiler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.55.0</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="memote">MEMOTE<a class="headerlink" href="#memote" title="Permanent link">&para;</a></h3>
<p>The genome-scale metabolic model test suite</p>
<p><em>homepage</em>: <a href="https://github.com/opencobra/memote">https://github.com/opencobra/memote</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="memtester">memtester<a class="headerlink" href="#memtester" title="Permanent link">&para;</a></h3>
<p>A userspace utility for testing the memory subsystem for faults</p>
<p><em>homepage</em>: <a href="https://pyropus.ca./software/memtester/">https://pyropus.ca./software/memtester/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="merantk">meRanTK<a class="headerlink" href="#merantk" title="Permanent link">&para;</a></h3>
<p>meRanTK is a versatile high performance toolkit for complete analysis of methylated RNA data.</p>
<p><em>homepage</em>: <a href="http://www.icbi.at/index.html">http://www.icbi.at/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="merckx">MERCKX<a class="headerlink" href="#merckx" title="Permanent link">&para;</a></h3>
<p>Multilingual Entity/Resource Combiner &amp; Knowledge eXtractor</p>
<p><em>homepage</em>: <a href="https://github.com/madewild/MERCKX">https://github.com/madewild/MERCKX</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170330</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="mercurial">Mercurial<a class="headerlink" href="#mercurial" title="Permanent link">&para;</a></h3>
<p>Mercurial is a free, distributed source control management tool. It efficiently handles projects of any size and offers an easy and intuitive interface. </p>
<p><em>homepage</em>: <a href="https://www.mercurial-scm.org">https://www.mercurial-scm.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>6.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mesa">Mesa<a class="headerlink" href="#mesa" title="Permanent link">&para;</a></h3>
<p>Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.</p>
<p><em>homepage</em>: <a href="https://www.mesa3d.org/">https://www.mesa3d.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.1.2</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>11.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>12.0.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>17.0.2</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>17.2.4</code></td>
<td><code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>17.2.5</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>17.3.6</code></td>
<td><code>foss/2018a</code>, <code>fosscuda/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>18.1.1</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>19.0.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>19.1.7</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>19.2.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20.0.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20.2.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>21.1.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>21.1.7</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>22.0.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>22.2.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mesa-demos">Mesa-demos<a class="headerlink" href="#mesa-demos" title="Permanent link">&para;</a></h3>
<p>Mesa utility and demo programs, including glxinfo and eglinfo.</p>
<p><em>homepage</em>: <a href="https://www.mesa3d.org/">https://www.mesa3d.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.4.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="meshalyzer">meshalyzer<a class="headerlink" href="#meshalyzer" title="Permanent link">&para;</a></h3>
<p>Graphical program for display time dependent data on 3D finite element meshes</p>
<p><em>homepage</em>: <a href="https://git.opencarp.org/openCARP/meshalyzer">https://git.opencarp.org/openCARP/meshalyzer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>20200308</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="meshio">meshio<a class="headerlink" href="#meshio" title="Permanent link">&para;</a></h3>
<p>meshio is a tool for reading/writing various mesh formats representing unstructured meshes</p>
<p><em>homepage</em>: <a href="https://github.com/nschloe/meshio">https://github.com/nschloe/meshio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="meshtool">meshtool<a class="headerlink" href="#meshtool" title="Permanent link">&para;</a></h3>
<p>Meshtool is a comand-line tool written in C++. It is designed to apply various manipulations to volumetric meshes.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/aneic/meshtool">https://bitbucket.org/aneic/meshtool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>16</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="meson">Meson<a class="headerlink" href="#meson" title="Permanent link">&para;</a></h3>
<p>Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.</p>
<p><em>homepage</em>: <a href="https://mesonbuild.com">https://mesonbuild.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.43.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.46.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.48.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.48.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.50.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.51.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.53.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.53.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.55.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.55.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.58.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.58.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.59.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.62.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>0.64.0</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="meson-python">meson-python<a class="headerlink" href="#meson-python" title="Permanent link">&para;</a></h3>
<p>Python build backend (PEP 517) for Meson projects</p>
<p><em>homepage</em>: <a href="https://github.com/mesonbuild/meson-python">https://github.com/mesonbuild/meson-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mesquite">Mesquite<a class="headerlink" href="#mesquite" title="Permanent link">&para;</a></h3>
<p>Mesh-Quality Improvement Library</p>
<p><em>homepage</em>: <a href="https://software.sandia.gov/mesquite/">https://software.sandia.gov/mesquite/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/8.3.0</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="mess">MESS<a class="headerlink" href="#mess" title="Permanent link">&para;</a></h3>
<p>Master Equation System Solver (MESS)</p>
<p><em>homepage</em>: <a href="https://github.com/PACChem/MESS">https://github.com/PACChem/MESS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="metabat">MetaBAT<a class="headerlink" href="#metabat" title="Permanent link">&para;</a></h3>
<p>An efficient tool for accurately reconstructing single genomes from complex microbial communities</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/berkeleylab/metabat">https://bitbucket.org/berkeleylab/metabat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.12.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>2.15</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.15</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="metaboanalystr">MetaboAnalystR<a class="headerlink" href="#metaboanalystr" title="Permanent link">&para;</a></h3>
<p>MetaboAnalystR contains the R functions and libraries underlying the popular MetaboAnalyst web  server, including &gt; 500 functions for metabolomic data analysis, visualization, and functional interpretation.</p>
<p><em>homepage</em>: <a href="https://www.metaboanalyst.ca">https://www.metaboanalyst.ca</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1-20190827</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="metaerg">metaerg<a class="headerlink" href="#metaerg" title="Permanent link">&para;</a></h3>
<p>MetaErg is a stand-alone and fully automated metagenomic and metaproteomic data annotation pipeline.</p>
<p><em>homepage</em>: <a href="https://github.com/xiaoli-dong/metaerg">https://github.com/xiaoli-dong/metaerg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="metaeuk">MetaEuk<a class="headerlink" href="#metaeuk" title="Permanent link">&para;</a></h3>
<p>MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic  metagenomic contigs.</p>
<p><em>homepage</em>: <a href="https://metaeuk.soedinglab.org">https://metaeuk.soedinglab.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>5</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>6</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="metageneannotator">MetaGeneAnnotator<a class="headerlink" href="#metageneannotator" title="Permanent link">&para;</a></h3>
<p>MetaGeneAnnotator is a gene-finding program for prokaryote and phage.</p>
<p><em>homepage</em>: <a href="http://metagene.nig.ac.jp/">http://metagene.nig.ac.jp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20080819</code></td>
<td><code>-x86-64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="metagenome-atlas">Metagenome-Atlas<a class="headerlink" href="#metagenome-atlas" title="Permanent link">&para;</a></h3>
<p>Metagenome-atlas is a easy-to-use metagenomic pipeline based on snakemake.  It handles all steps from QC, Assembly, Binning, to Annotation.</p>
<p><em>homepage</em>: <a href="https://github.com/metagenome-atlas/atlas">https://github.com/metagenome-atlas/atlas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="metal">Metal<a class="headerlink" href="#metal" title="Permanent link">&para;</a></h3>
<p>Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large  datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.</p>
<p><em>homepage</em>: <a href="http://csg.sph.umich.edu/abecasis/Metal/">http://csg.sph.umich.edu/abecasis/Metal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2011-03-25</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2020-05-05</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="metamorpheus">MetaMorpheus<a class="headerlink" href="#metamorpheus" title="Permanent link">&para;</a></h3>
<p>MetaMorpheus is a bottom-up proteomics database search software  with integrated post-translational modification (PTM) discovery capability.  This program combines features of Morpheus and G-PTM-D in a single tool. </p>
<p><em>homepage</em>: <a href="https://github.com/smith-chem-wisc/MetaMorpheus">https://github.com/smith-chem-wisc/MetaMorpheus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.320</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="metaphlan">MetaPhlAn<a class="headerlink" href="#metaphlan" title="Permanent link">&para;</a></h3>
<p>MetaPhlAn is a computational tool for profiling the composition  of microbial communities from metagenomic shotgun sequencing data </p>
<p><em>homepage</em>: <a href="https://github.com/biobakery/MetaPhlAn">https://github.com/biobakery/MetaPhlAn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="metaphlan2">MetaPhlAn2<a class="headerlink" href="#metaphlan2" title="Permanent link">&para;</a></h3>
<p>MetaPhlAn is a computational tool for profiling the composition of microbial  communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing  data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now  possible to perform accurate strain-level microbial profiling.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/biobakery/metaphlan2/">https://bitbucket.org/biobakery/metaphlan2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.8</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.7.8</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7.8</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="metawrap">metaWRAP<a class="headerlink" href="#metawrap" title="Permanent link">&para;</a></h3>
<p>MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.</p>
<p><em>homepage</em>: <a href="https://github.com/bxlab/metaWRAP">https://github.com/bxlab/metaWRAP</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="metaxa2">Metaxa2<a class="headerlink" href="#metaxa2" title="Permanent link">&para;</a></h3>
<p>Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically</p>
<p><em>homepage</em>: <a href="https://microbiology.se/software/metaxa2/">https://microbiology.se/software/metaxa2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="methylartist">methylartist<a class="headerlink" href="#methylartist" title="Permanent link">&para;</a></h3>
<p>Tools for plotting methylation data in various ways </p>
<p><em>homepage</em>: <a href="https://github.com/adamewing/methylartist">https://github.com/adamewing/methylartist</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.6</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="methyldackel">MethylDackel<a class="headerlink" href="#methyldackel" title="Permanent link">&para;</a></h3>
<p>A (mostly) universal methylation extractor for BS-seq experiments.</p>
<p><em>homepage</em>: <a href="https://github.com/dpryan79/MethylDackel">https://github.com/dpryan79/MethylDackel</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="methylpy">methylpy<a class="headerlink" href="#methylpy" title="Permanent link">&para;</a></h3>
<p>Bisulfite sequencing data processing and differential methylation analysis.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/methylpy">https://pypi.python.org/pypi/methylpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.9</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.2.9</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="metis">METIS<a class="headerlink" href="#metis" title="Permanent link">&para;</a></h3>
<p>METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes.</p>
<p><em>homepage</em>: <a href="http://glaros.dtc.umn.edu/gkhome/metis/metis/overview">http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.2</code></td>
<td></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2018b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>-32bitIDX</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="mgltools">mgltools<a class="headerlink" href="#mgltools" title="Permanent link">&para;</a></h3>
<p>The MGLTools software suite can be used for visualization     and analysis of molecular structures and comprises     the Python Molecular Viewer (PMV, a general purpose molecular viewer),     AutoDockTools (ADT, a set of PMV commands specifically developed to support AutoDock users)     and Vision (a visual programming environment). </p>
<p><em>homepage</em>: <a href="https://ccsb.scripps.edu/mgltools/">https://ccsb.scripps.edu/mgltools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.7</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mhcflurry">mhcflurry<a class="headerlink" href="#mhcflurry" title="Permanent link">&para;</a></h3>
<p>MHCflurry implements class I peptide/MHC binding affinity  prediction. By default it supports 112 MHC alleles using ensembles of  allele-specific models. Pan-allele predictors supporting virtually any MHC  allele of known sequence are available for testing (see below). MHCflurry  runs on Python 2.7 and 3.4+ using the keras neural network library. It  exposes command-line and Python library interfaces.</p>
<p><em>homepage</em>: <a href="https://github.com/openvax/mhcflurry">https://github.com/openvax/mhcflurry</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="mhcnuggets">mhcnuggets<a class="headerlink" href="#mhcnuggets" title="Permanent link">&para;</a></h3>
<p>MHCnuggets: Neoantigen peptide MHC binding prediction for class I and II.</p>
<p><em>homepage</em>: <a href="https://github.com/KarchinLab/mhcnuggets">https://github.com/KarchinLab/mhcnuggets</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="microbeannotator">MicrobeAnnotator<a class="headerlink" href="#microbeannotator" title="Permanent link">&para;</a></h3>
<p>Easy-to-use pipeline for the comprehensive metabolic annotation of microbial genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/cruizperez/MicrobeAnnotator">https://github.com/cruizperez/MicrobeAnnotator</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.5</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="microctools">microctools<a class="headerlink" href="#microctools" title="Permanent link">&para;</a></h3>
<p>Various worker functions for microclimc package</p>
<p><em>homepage</em>: <a href="https://github.com/ilyamaclean/microctools">https://github.com/ilyamaclean/microctools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0-20201209</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="migec">MiGEC<a class="headerlink" href="#migec" title="Permanent link">&para;</a></h3>
<p>MIGEC is a software pipeline that facilitates processing and analysis of   immune repertoire sequencing data generated using molecular barcoding technique</p>
<p><em>homepage</em>: <a href="https://milaboratory.com/software/migec/">https://milaboratory.com/software/migec/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.8</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.9</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="migrate-n">MIGRATE-N<a class="headerlink" href="#migrate-n" title="Permanent link">&para;</a></h3>
<p>Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data.  It  uses a coalescent theory approach taking into account history of  mutations and uncertainty of the genealogy.  </p>
<p><em>homepage</em>: <a href="https://peterbeerli.com/migrate-html5/index.html">https://peterbeerli.com/migrate-html5/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.8</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.2.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.0.4</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="miller">Miller<a class="headerlink" href="#miller" title="Permanent link">&para;</a></h3>
<p>Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON</p>
<p><em>homepage</em>: <a href="https://miller.readthedocs.io/">https://miller.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.4.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mimalloc">mimalloc<a class="headerlink" href="#mimalloc" title="Permanent link">&para;</a></h3>
<p>mimalloc is a general purpose allocator with excellent performance characteristics.</p>
<p><em>homepage</em>: <a href="https://microsoft.github.io/mimalloc/">https://microsoft.github.io/mimalloc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minc">MINC<a class="headerlink" href="#minc" title="Permanent link">&para;</a></h3>
<p>Medical Image NetCDF or MINC isn't netCDF.</p>
<p><em>homepage</em>: <a href="https://github.com/BIC-MNI/libminc">https://github.com/BIC-MNI/libminc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.03</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="minced">MinCED<a class="headerlink" href="#minced" title="Permanent link">&para;</a></h3>
<p>Mining CRISPRs in Environmental Datasets</p>
<p><em>homepage</em>: <a href="https://github.com/ctSkennerton/minced">https://github.com/ctSkennerton/minced</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mini-xml">Mini-XML<a class="headerlink" href="#mini-xml" title="Permanent link">&para;</a></h3>
<p>Mini-XML is a small XML parsing library that you can use to read XML data files or strings in your  application without requiring large non-standard libraries</p>
<p><em>homepage</em>: <a href="https://www.msweet.org/mxml">https://www.msweet.org/mxml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="miniasm">miniasm<a class="headerlink" href="#miniasm" title="Permanent link">&para;</a></h3>
<p>Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format. Different from mainstream assemblers, miniasm does not have a consensus step. It simply concatenates pieces of read sequences to generate the final unitig sequences. Thus the per-base error rate is similar to the raw input reads.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/minimap2">https://github.com/lh3/minimap2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3-20191007</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minibar">minibar<a class="headerlink" href="#minibar" title="Permanent link">&para;</a></h3>
<p>Dual barcode and primer demultiplexing for MinION sequenced reads</p>
<p><em>homepage</em>: <a href="https://github.com/calacademy-research/minibar">https://github.com/calacademy-research/minibar</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200326</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>20200326</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
</tbody>
</table>
<h3 id="minicard">MiniCARD<a class="headerlink" href="#minicard" title="Permanent link">&para;</a></h3>
<p>MiniCARD is a <em>cardinality solver</em> based on MiniSAT [www.minisat.se].  MiniCARD handles cardinality constraints natively, using the same efficient data structures and techniques MiniSAT uses for clauses, giving it much better performance on cardinality constraints than CNF encodings of those constraints passed to a typical SAT solver.  It can read the standard DIMACS CNF format, the OPB pseudo-boolean format (with linear cardinality constraints only), and CNF+, a format that extends CNF to include cardinality constraints.</p>
<p><em>homepage</em>: <a href="https://github.com/liffiton/minicard">https://github.com/liffiton/minicard</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="miniconda2">Miniconda2<a class="headerlink" href="#miniconda2" title="Permanent link">&para;</a></h3>
<p>Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.</p>
<p><em>homepage</em>: <a href="https://docs.conda.io/en/latest/miniconda.html">https://docs.conda.io/en/latest/miniconda.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.14</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.7.10</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="miniconda3">Miniconda3<a class="headerlink" href="#miniconda3" title="Permanent link">&para;</a></h3>
<p>Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.</p>
<p><em>homepage</em>: <a href="https://docs.conda.io/en/latest/miniconda.html">https://docs.conda.io/en/latest/miniconda.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.5.12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.14</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.7.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.12.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.11.1-1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="minieigen">minieigen<a class="headerlink" href="#minieigen" title="Permanent link">&para;</a></h3>
<p>A small wrapper for core parts of EIgen, c++ library for linear algebra.</p>
<p><em>homepage</em>: <a href="https://launchpad.net/minieigen/">https://launchpad.net/minieigen/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.5.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.5.3</code></td>
<td><code>-Python-2.7.12-Boost-1.63.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.5.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="minimac4">Minimac4<a class="headerlink" href="#minimac4" title="Permanent link">&para;</a></h3>
<p>Minimac4 is a latest version in the series of genotype imputation software  - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010).  Minimac4 is a lower memory and more computationally efficient implementation  of the original algorithms with comparable imputation quality.</p>
<p><em>homepage</em>: <a href="https://genome.sph.umich.edu/wiki/Minimac4">https://genome.sph.umich.edu/wiki/Minimac4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="minimap2">minimap2<a class="headerlink" href="#minimap2" title="Permanent link">&para;</a></h3>
<p>Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/minimap2">https://github.com/lh3/minimap2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0rc1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.11</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.17</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.18</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.20</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.22</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.24</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minipolish">Minipolish<a class="headerlink" href="#minipolish" title="Permanent link">&para;</a></h3>
<p>A tool for Racon polishing of miniasm assemblies</p>
<p><em>homepage</em>: <a href="https://github.com/rrwick/Minipolish">https://github.com/rrwick/Minipolish</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minisat">MiniSat<a class="headerlink" href="#minisat" title="Permanent link">&para;</a></h3>
<p>MiniSat is a minimalistic, open-source SAT solver, developed to help researchers and developers alike to get started on SAT.</p>
<p><em>homepage</em>: <a href="http://minisat.se/">http://minisat.se/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>20130925</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minizip">minizip<a class="headerlink" href="#minizip" title="Permanent link">&para;</a></h3>
<p>Mini zip and unzip based on zlib</p>
<p><em>homepage</em>: <a href="https://www.winimage.com/zLibDll/minizip.html">https://www.winimage.com/zLibDll/minizip.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minpack">MINPACK<a class="headerlink" href="#minpack" title="Permanent link">&para;</a></h3>
<p>Minpack includes software for solving nonlinear equations and  nonlinear least squares problems.  Five algorithmic paths each include  a core subroutine and an easy-to-use driver.  The algorithms proceed  either from an analytic specification of the Jacobian matrix or  directly from the problem functions.  The paths include facilities for  systems of equations with a banded Jacobian matrix, for least squares  problems with a large amount of data, and for checking the consistency  of the Jacobian matrix with the functions. </p>
<p><em>homepage</em>: <a href="https://www.netlib.org/minpack/">https://www.netlib.org/minpack/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19961126</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="minpath">MinPath<a class="headerlink" href="#minpath" title="Permanent link">&para;</a></h3>
<p>MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions  using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological  pathways for a query dataset.</p>
<p><em>homepage</em>: <a href="https://omics.informatics.indiana.edu/MinPath">https://omics.informatics.indiana.edu/MinPath</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mira">MIRA<a class="headerlink" href="#mira" title="Permanent link">&para;</a></h3>
<p>MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),  IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads).</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/p/mira-assembler/wiki/Home/">https://sourceforge.net/p/mira-assembler/wiki/Home/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.9.6</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0rc2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mirdeep2">miRDeep2<a class="headerlink" href="#mirdeep2" title="Permanent link">&para;</a></h3>
<p>miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs</p>
<p><em>homepage</em>: <a href="https://www.mdc-berlin.de/8551903/en/">https://www.mdc-berlin.de/8551903/en/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.0.8</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="mish-cuda">Mish-Cuda<a class="headerlink" href="#mish-cuda" title="Permanent link">&para;</a></h3>
<p>Mish-Cuda: Self Regularized Non-Monotonic Activation Function</p>
<p><em>homepage</em>: <a href="https://github.com/JunnYu/mish-cuda">https://github.com/JunnYu/mish-cuda</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210309</code></td>
<td><code>-PyTorch-1.9.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="misha">misha<a class="headerlink" href="#misha" title="Permanent link">&para;</a></h3>
<p>The misha package is intended to help users to efficiently analyze genomic data achieved  from various experiments.</p>
<p><em>homepage</em>: <a href="https://github.com/tanaylab/misha">https://github.com/tanaylab/misha</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.10</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mitgcmutils">MITgcmutils<a class="headerlink" href="#mitgcmutils" title="Permanent link">&para;</a></h3>
<p>A numerical model designed for study of the atmosphere, ocean,  and climate, MITgcms flexible non-hydrostatic formulation enables it to  efficiently simulate fluid phenomena over a wide range of scales; its adjoint  capabilities enable it to be applied to sensitivity questions and to parameter  and state estimation problems. By employing fluid equation isomorphisms, a  single dynamical kernel can be used to simulate flow of both the atmosphere  and ocean. The model is developed to perform efficiently on a wide variety of  computational platforms.</p>
<p><em>homepage</em>: <a href="https://mitgcm.org/">https://mitgcm.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mitobim">MITObim<a class="headerlink" href="#mitobim" title="Permanent link">&para;</a></h3>
<p>The MITObim procedure (mitochondrial baiting and iterative mapping) represents  a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms  directly from total genomic DNA derived NGS reads.</p>
<p><em>homepage</em>: <a href="https://github.com/chrishah/MITObim">https://github.com/chrishah/MITObim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="mitoz">MitoZ<a class="headerlink" href="#mitoz" title="Permanent link">&para;</a></h3>
<p>MitoZ is a Python3-based toolkit which aims to automatically filter pair-end raw data (fastq files),  assemble genome, search for mitogenome sequences from the genome assembly result, annotate mitogenome  (genbank file as result), and mitogenome visualization.</p>
<p><em>homepage</em>: <a href="https://github.com/linzhi2013/MitoZ">https://github.com/linzhi2013/MitoZ</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="mixcr">MiXCR<a class="headerlink" href="#mixcr" title="Permanent link">&para;</a></h3>
<p>MiXCR processes big immunome data from raw sequences  to quantitated clonotypes </p>
<p><em>homepage</em>: <a href="https://milaboratory.com/software/mixcr/">https://milaboratory.com/software/mixcr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.9</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.13</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.13</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mixmhc2pred">MixMHC2pred<a class="headerlink" href="#mixmhc2pred" title="Permanent link">&para;</a></h3>
<p>MixMHC2pred is a predictor of HLA class II ligands and epitopes.  It is described in publication Racle, J., et al. Robust prediction of HLA class II  epitopes by deep motif deconvolution of immunopeptidomes</p>
<p><em>homepage</em>: <a href="https://github.com/GfellerLab/MixMHC2pred">https://github.com/GfellerLab/MixMHC2pred</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mkl-dnn">mkl-dnn<a class="headerlink" href="#mkl-dnn" title="Permanent link">&para;</a></h3>
<p>Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) MKL-DNN)</p>
<p><em>homepage</em>: <a href="https://01.org/mkl-dnn">https://01.org/mkl-dnn</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.13</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.16</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.17.2</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="mkl-service">mkl-service<a class="headerlink" href="#mkl-service" title="Permanent link">&para;</a></h3>
<p>Python hooks for Intel(R) Math Kernel Library runtime control settings.</p>
<p><em>homepage</em>: <a href="https://github.com/IntelPython/mkl-service">https://github.com/IntelPython/mkl-service</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>intel/2020b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="mkl_fft">mkl_fft<a class="headerlink" href="#mkl_fft" title="Permanent link">&para;</a></h3>
<p>NumPy-based Python interface to Intel(R) MKL FFT functionality</p>
<p><em>homepage</em>: <a href="https://github.com/IntelPython/mkl_fft">https://github.com/IntelPython/mkl_fft</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.14</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="mlc">MLC<a class="headerlink" href="#mlc" title="Permanent link">&para;</a></h3>
<p>Intel Memory Latency Checker (Intel MLC) is a tool used to measure memory latencies and b/w,  and how they change with increasing load on the system.</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/articles/intelr-memory-latency-checker">https://software.intel.com/en-us/articles/intelr-memory-latency-checker</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mlxtend">MLxtend<a class="headerlink" href="#mlxtend" title="Permanent link">&para;</a></h3>
<p>Mlxtend (machine learning extensions) is a Python library of useful tools for the day-to-day data  science tasks.</p>
<p><em>homepage</em>: <a href="https://rasbt.github.io/mlxtend/">https://rasbt.github.io/mlxtend/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mm-common">mm-common<a class="headerlink" href="#mm-common" title="Permanent link">&para;</a></h3>
<p>The mm-common module provides the build infrastructure and  utilities shared among the GNOME C++ binding libraries.</p>
<p><em>homepage</em>: <a href="https://gitlab.gnome.org/GNOME/mm-common">https://gitlab.gnome.org/GNOME/mm-common</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mmg">Mmg<a class="headerlink" href="#mmg" title="Permanent link">&para;</a></h3>
<p>Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries: the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation and generation of a triangulation from a set of points or from given boundary edges the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries.</p>
<p><em>homepage</em>: <a href="https://www.mmgtools.org/">https://www.mmgtools.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.3.9</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>5.6.0</code></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="mmseq">MMSEQ<a class="headerlink" href="#mmseq" title="Permanent link">&para;</a></h3>
<p>The MMSEQ package contains a collection of statistical tools  for analysing RNA-seq expression data. </p>
<p><em>homepage</em>: <a href="https://github.com/eturro/mmseq">https://github.com/eturro/mmseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.8</code></td>
<td><code>-linux64-static</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mmseqs2">MMseqs2<a class="headerlink" href="#mmseqs2" title="Permanent link">&para;</a></h3>
<p>MMseqs2: ultra fast and sensitive search and clustering suite</p>
<p><em>homepage</em>: <a href="https://mmseqs.com">https://mmseqs.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1-c7a89</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5-9375b</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8-fac81</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>10-6d92c</code></td>
<td><code>gompi/2019b</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>11-e1a1c</code></td>
<td><code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>13-45111</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>13-45111-20211006</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>13-45111-20211019</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mmtf-cpp">mmtf-cpp<a class="headerlink" href="#mmtf-cpp" title="Permanent link">&para;</a></h3>
<p>The Macromolecular Transmission Format (MMTF) is a new compact binary format to  transmit and store biomolecular structures for fast 3D visualization and        analysis. </p>
<p><em>homepage</em>: <a href="https://mmtf.rcsb.org/">https://mmtf.rcsb.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mne-python">MNE-Python<a class="headerlink" href="#mne-python" title="Permanent link">&para;</a></h3>
<p>MNE-Python software is an open-source Python package for exploring, visualizing, and analyzing human neurophysiological data such as MEG, EEG, sEEG, ECoG, and more. It includes modules for data input/output, preprocessing, visualization, source estimation, time-frequency analysis, connectivity analysis, machine learning, and statistics.</p>
<p><em>homepage</em>: <a href="https://mne.tools/stable/index.html">https://mne.tools/stable/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.24.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="moabs">MOABS<a class="headerlink" href="#moabs" title="Permanent link">&para;</a></h3>
<p>MOABS: MOdel based Analysis of Bisulfite Sequencing data</p>
<p><em>homepage</em>: <a href="https://github.com/sunnyisgalaxy/moabs">https://github.com/sunnyisgalaxy/moabs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.9.6</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="modeltest-ng">ModelTest-NG<a class="headerlink" href="#modeltest-ng" title="Permanent link">&para;</a></h3>
<p>ModelTest-NG is a tool for selecting the best-fit model of evolution for DNA and protein alignments. ModelTest-NG supersedes jModelTest and ProtTest in one single tool, with graphical and command console interfaces. </p>
<p><em>homepage</em>: <a href="https://github.com/ddarriba/modeltest">https://github.com/ddarriba/modeltest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.7</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="modred">modred<a class="headerlink" href="#modred" title="Permanent link">&para;</a></h3>
<p>Compute modal decompositions and reduced-order models, easily, efficiently, and in parallel.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/modred/">https://pypi.python.org/pypi/modred/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="molcas">Molcas<a class="headerlink" href="#molcas" title="Permanent link">&para;</a></h3>
<p>Molcas is an ab initio quantum chemistry software package developed by scientists to be used by scientists. The basic philosophy is is to be able to treat general electronic structures for molecules consisting of atoms from most of the periodic table. As such, the primary focus of the package is on multiconfigurational methods with applications typically connected to the treatment of highly degenerate states.</p>
<p><em>homepage</em>: <a href="http://www.molcas.org">http://www.molcas.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0-15.06.18</code></td>
<td><code>_CentOS_6.6_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td><code>-centos-mkl-par</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td><code>-centos-par</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mold">mold<a class="headerlink" href="#mold" title="Permanent link">&para;</a></h3>
<p>mold is a high-performance drop-in replacement for existing Unix linkers.</p>
<p><em>homepage</em>: <a href="https://github.com/rui314/mold">https://github.com/rui314/mold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.6</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="molden">Molden<a class="headerlink" href="#molden" title="Permanent link">&para;</a></h3>
<p>Molden is a package for displaying Molecular Density from the  Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical  packages Mopac/Ampac</p>
<p><em>homepage</em>: <a href="https://www.theochem.ru.nl/molden/">https://www.theochem.ru.nl/molden/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.7</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>6.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="molekel">Molekel<a class="headerlink" href="#molekel" title="Permanent link">&para;</a></h3>
<p>Molekel is an open-source multi-platform molecular visualization program. </p>
<p><em>homepage</em>: <a href="http://ugovaretto.github.io/molekel/wiki/pmwiki.php/Main/HomePage.html">http://ugovaretto.github.io/molekel/wiki/pmwiki.php/Main/HomePage.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.0</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="molmod">molmod<a class="headerlink" href="#molmod" title="Permanent link">&para;</a></h3>
<p>MolMod is a Python library with many compoments that are useful to write molecular modeling programs.</p>
<p><em>homepage</em>: <a href="https://molmod.github.io/molmod/">https://molmod.github.io/molmod/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.4.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.4.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.4.8</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="molpro">Molpro<a class="headerlink" href="#molpro" title="Permanent link">&para;</a></h3>
<p>Molpro is a complete system of ab initio programs for molecular electronic structure calculations.</p>
<p><em>homepage</em>: <a href="https://www.molpro.net">https://www.molpro.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2010.1.23</code></td>
<td><code>.Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2015.1.0</code></td>
<td><code>.linux_x86_64_intel</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2015.1.3</code></td>
<td><code>.linux_x86_64_openmp</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2015.1.10</code></td>
<td><code>.linux_x86_64_openmp</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mona">MONA<a class="headerlink" href="#mona" title="Permanent link">&para;</a></h3>
<p>MONA is a tool that translates formulas to finite-state automata. The formulas may express search patterns, temporal properties of reactive systems, parse tree constraints, etc. MONA analyses the automaton resulting from the compilation and prints out "valid" or a counter-example.</p>
<p><em>homepage</em>: <a href="https://www.brics.dk/mona/index.html">https://www.brics.dk/mona/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4-18</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="monai">MONAI<a class="headerlink" href="#monai" title="Permanent link">&para;</a></h3>
<p>MONAI is a PyTorch-based, open-source framework for deep learning in healthcare imaging, part of PyTorch Ecosystem. </p>
<p><em>homepage</em>: <a href="https://monai.io/">https://monai.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mongolite">mongolite<a class="headerlink" href="#mongolite" title="Permanent link">&para;</a></h3>
<p>High-performance MongoDB client based on 'mongo-c-driver' and 'jsonlite'.  Includes support for aggregation, indexing, map-reduce, streaming, encryption,  enterprise authentication, and GridFS. The online user manual provides an overview  of the available methods in the package: <a href="https://jeroen.github.io/mongolite/">https://jeroen.github.io/mongolite/</a>.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/mongolite">https://cran.r-project.org/web/packages/mongolite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mono">Mono<a class="headerlink" href="#mono" title="Permanent link">&para;</a></h3>
<p>An open source, cross-platform, implementation of C# and the CLR that is  binary compatible with Microsoft.NET.</p>
<p><em>homepage</em>: <a href="https://www.mono-project.com/">https://www.mono-project.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.6</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.6.2.7</code></td>
<td><code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>4.8.0.495</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.4.1.6</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.10.0.160</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.18.1.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>6.4.0.198</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.8.0.105</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>6.12.0.122</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="monocle3">Monocle3<a class="headerlink" href="#monocle3" title="Permanent link">&para;</a></h3>
<p>An analysis toolkit for single-cell RNA-seq. </p>
<p><em>homepage</em>: <a href="https://cole-trapnell-lab.github.io/monocle3/">https://cole-trapnell-lab.github.io/monocle3/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.7.2-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.2.3</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="moonjit">moonjit<a class="headerlink" href="#moonjit" title="Permanent link">&para;</a></h3>
<p>Moonjit is a Just-In-Time Compiler (JIT) for the Lua programming  language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language.</p>
<p><em>homepage</em>: <a href="https://github.com/moonjit/moonjit">https://github.com/moonjit/moonjit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="moose">MOOSE<a class="headerlink" href="#moose" title="Permanent link">&para;</a></h3>
<p>The Multiphysics Object-Oriented Simulation Environment (MOOSE) is a finite-element, multiphysics framework primarily developed by Idaho National Laboratory</p>
<p><em>homepage</em>: <a href="https://mooseframework.inl.gov">https://mooseframework.inl.gov</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021-05-18</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="mordecai">mordecai<a class="headerlink" href="#mordecai" title="Permanent link">&para;</a></h3>
<p>mordecai is a full text geoparsing as a Python library. Extract the place names from a piece of text, resolve them to the correct place, and return their coordinates and structured geographic information.</p>
<p><em>homepage</em>: <a href="https://github.com/openeventdata/mordecai">https://github.com/openeventdata/mordecai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="moreronn">MoreRONN<a class="headerlink" href="#moreronn" title="Permanent link">&para;</a></h3>
<p>MoreRONN is the spiritual successor of RONN and is useful for surveying  disorder in proteins as well as designing expressible constructs for X-ray crystallography.</p>
<p><em>homepage</em>: <a href="https://github.com/varun-ramraj/MoreRONN">https://github.com/varun-ramraj/MoreRONN</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mosdepth">mosdepth<a class="headerlink" href="#mosdepth" title="Permanent link">&para;</a></h3>
<p>Fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing</p>
<p><em>homepage</em>: <a href="https://github.com/brentp/mosdepth">https://github.com/brentp/mosdepth</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.2.3</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.3.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mothur">Mothur<a class="headerlink" href="#mothur" title="Permanent link">&para;</a></h3>
<p>Mothur is a single piece of open-source, expandable software  to fill the bioinformatics needs of the microbial ecology community.</p>
<p><em>homepage</em>: <a href="https://www.mothur.org/">https://www.mothur.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.39.5</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.41.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.43.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="motif">motif<a class="headerlink" href="#motif" title="Permanent link">&para;</a></h3>
<p>Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building  applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.  It was the standard toolkit for the Common Desktop Environment and thus for Unix.</p>
<p><em>homepage</em>: <a href="https://motif.ics.com/">https://motif.ics.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.5</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.3.7</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.3.8</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="motioncor2">MotionCor2<a class="headerlink" href="#motioncor2" title="Permanent link">&para;</a></h3>
<p>MotionCor2 correct anisotropic image motion at the single pixel level across the whole frame, suitable for both single particle and tomographic images. Iterative, patch-based motion detection is combined with spatial and temporal constraints and dose weighting.  Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache, Yifan Cheng and David A. Agard (2016) Anisotropic Correction of Beam-induced Motion for Improved Single-particle Electron Cryo-microscopy, Nature Methods, submitted. BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960 </p>
<p><em>homepage</em>: <a href="https://msg.ucsf.edu/">https://msg.ucsf.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.6</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="motionsegmentation">motionSegmentation<a class="headerlink" href="#motionsegmentation" title="Permanent link">&para;</a></h3>
<p>Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.</p>
<p><em>homepage</em>: <a href="https://github.com/WeiXuanChan/motionSegmentation">https://github.com/WeiXuanChan/motionSegmentation</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.9</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="moviepy">MoviePy<a class="headerlink" href="#moviepy" title="Permanent link">&para;</a></h3>
<p>MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,  title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects.</p>
<p><em>homepage</em>: <a href="https://zulko.github.io/moviepy/">https://zulko.github.io/moviepy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="mpb">MPB<a class="headerlink" href="#mpb" title="Permanent link">&para;</a></h3>
<p>MPB is a free and open-source software package for computing  the band structures, or dispersion relations, and electromagnetic  modes of periodic dielectric structures, on both serial  and parallel computers. MPB is an acronym for MIT Photonic Bands.</p>
<p><em>homepage</em>: <a href="https://mpb.readthedocs.io/en/latest/">https://mpb.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="mpc">MPC<a class="headerlink" href="#mpc" title="Permanent link">&para;</a></h3>
<p>Gnu Mpc is a C library for the arithmetic of  complex numbers with arbitrarily high precision and correct  rounding of the result. It extends the principles of the IEEE-754  standard for fixed precision real floating point numbers to  complex numbers, providing well-defined semantics for every  operation. At the same time, speed of operation at high precision  is a major design goal.</p>
<p><em>homepage</em>: <a href="http://www.multiprecision.org/">http://www.multiprecision.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-MPFR-3.1.6</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mpfr">MPFR<a class="headerlink" href="#mpfr" title="Permanent link">&para;</a></h3>
<p>The MPFR library is a C library for multiple-precision floating-point  computations with correct rounding. </p>
<p><em>homepage</em>: <a href="https://www.mpfr.org">https://www.mpfr.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.1.6</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mpi4py">mpi4py<a class="headerlink" href="#mpi4py" title="Permanent link">&para;</a></h3>
<p>MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for  the Python programming language, allowing any Python program to exploit multiple processors.</p>
<p><em>homepage</em>: <a href="https://github.com/mpi4py/mpi4py">https://github.com/mpi4py/mpi4py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-2.7.11-timed-pingpong</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-2.7.12-timed-pingpong</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.13-timed-pingpong</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-2.7.14-timed-pingpong</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-timed-pingpong</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td></td>
<td><code>gompi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="mpich">MPICH<a class="headerlink" href="#mpich" title="Permanent link">&para;</a></h3>
<p>MPICH is a high-performance and widely portable implementation  of the Message Passing Interface (MPI) standard (MPI-1, MPI-2 and MPI-3).</p>
<p><em>homepage</em>: <a href="https://www.mpich.org/">https://www.mpich.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCC/4.8.1</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GCC/7.2.0-2.29</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/7.2.0-2.29</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mpich2">MPICH2<a class="headerlink" href="#mpich2" title="Permanent link">&para;</a></h3>
<p>MPICH v3.x is an open source high-performance MPI 3.0 implementation. It does not support InfiniBand (use MVAPICH2 with InfiniBand devices).</p>
<p><em>homepage</em>: <a href="http://www.mpich.org/">http://www.mpich.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCC/4.8.1</code></td>
</tr>
</tbody>
</table>
<h3 id="mpifileutils">mpifileutils<a class="headerlink" href="#mpifileutils" title="Permanent link">&para;</a></h3>
<p>MPI-Based File Utilities For Distributed Systems </p>
<p><em>homepage</em>: <a href="https://hpc.github.io/mpifileutils/">https://hpc.github.io/mpifileutils/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>0.10</code></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="mpip">mpiP<a class="headerlink" href="#mpip" title="Permanent link">&para;</a></h3>
<p>mpiP is a lightweight profiling library for MPI applications.  Because it  only collects statistical information about MPI functions, mpiP generates  considerably less overhead and much less data than tracing tools.  All the  information captured by mpiP is task-local. It only uses communication during  report generation, typically at the end of the experiment, to merge results  from all of the tasks into one output file. </p>
<p><em>homepage</em>: <a href="http://mpip.sourceforge.net/">http://mpip.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code>, <code>iompi/2019.01</code></td>
</tr>
</tbody>
</table>
<h3 id="mpj-express">MPJ-Express<a class="headerlink" href="#mpj-express" title="Permanent link">&para;</a></h3>
<p>MPJ Express is an open source Java message passing library that allows application   developers to write and execute parallel applications for multicore processors and compute clusters/clouds.</p>
<p><em>homepage</em>: <a href="http://mpj-express.org/">http://mpj-express.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.44</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="mpmath">mpmath<a class="headerlink" href="#mpmath" title="Permanent link">&para;</a></h3>
<p>mpmath can be used as an arbitrary-precision substitute for Python's float/complex  types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation  can be performed just as well at 10-digit or 1000-digit precision, with either real or complex  numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely  high precision work.</p>
<p><em>homepage</em>: <a href="https://mpmath.org/">https://mpmath.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.19</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mrbayes">MrBayes<a class="headerlink" href="#mrbayes" title="Permanent link">&para;</a></h3>
<p>MrBayes is a program for Bayesian inference and model choice across  a wide range of phylogenetic and evolutionary models.</p>
<p><em>homepage</em>: <a href="https://nbisweden.github.io/MrBayes/">https://nbisweden.github.io/MrBayes/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.6</code></td>
<td><code>foss/2016a</code>, <code>foss/2017a</code>, <code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>3.2.7</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2022a</code>, <code>gompic/2019b</code></td>
</tr>
<tr>
<td><code>3.2.7a</code></td>
<td><code>foss/2020a</code>, <code>iimpi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="mrcfile">mrcfile<a class="headerlink" href="#mrcfile" title="Permanent link">&para;</a></h3>
<p>mrcfile is a Python implementation of the MRC2014 file format, which is used in  structural biology to store image and volume data. It allows MRC files to be created and opened easily using a very simple API,    which exposes the files header and data as numpy arrays. The code runs in      Python 2 and 3 and is fully unit-tested. This library aims to allow users and developers to read and write standard- compliant MRC files in Python as easily as possible, and with no dependencies on any compiled libraries except numpy. You can use it interactively to inspect  files, correct headers and so on, or in scripts and larger software packages to  provide basic MRC file I/O functions. </p>
<p><em>homepage</em>: <a href="https://github.com/ccpem/mrcfile">https://github.com/ccpem/mrcfile</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mrchem">MRChem<a class="headerlink" href="#mrchem" title="Permanent link">&para;</a></h3>
<p>MRChem is a numerical real-space code for molecular electronic structure calculations within the self-consistent field (SCF) approximations of quantum chemistry: Hartree-Fock and Density Functional Theory.</p>
<p><em>homepage</em>: <a href="https://mrchem.readthedocs.io">https://mrchem.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mrcpp">MRCPP<a class="headerlink" href="#mrcpp" title="Permanent link">&para;</a></h3>
<p>MultiResolution Computation Program Package</p>
<p><em>homepage</em>: <a href="https://mrcpp.readthedocs.io">https://mrcpp.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.6</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mricron">MRIcron<a class="headerlink" href="#mricron" title="Permanent link">&para;</a></h3>
<p>MRIcron allows viewing of medical images. It  includes tools to complement SPM and FSL. Native format is NIFTI  but includes a conversion program (see dcm2nii) for converting DICOM images.  Features layers, ROIs, and volume rendering.  </p>
<p><em>homepage</em>: <a href="http://www.mccauslandcenter.sc.edu/mricro/mricron/">http://www.mccauslandcenter.sc.edu/mricro/mricron/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.20180614</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20150601</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mrtrix">MRtrix<a class="headerlink" href="#mrtrix" title="Permanent link">&para;</a></h3>
<p>MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner  robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines.</p>
<p><em>homepage</em>: <a href="http://www.brain.org.au/software/index.html#mrtrix">http://www.brain.org.au/software/index.html#mrtrix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.14</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.3.15</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0-rc-20191217</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.0-rc-20191217</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.0_RC2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>3.0_RC3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="msgpack-c">msgpack-c<a class="headerlink" href="#msgpack-c" title="Permanent link">&para;</a></h3>
<p>MessagePack is an efficient binary serialization format, which lets you exchange data among multiple languages like JSON, except that it's faster and smaller.    Small integers are encoded into a single byte while typical short strings        require only one extra byte in addition to the strings themselves.</p>
<p><em>homepage</em>: <a href="http://msgpack.org/">http://msgpack.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="msm">MSM<a class="headerlink" href="#msm" title="Permanent link">&para;</a></h3>
<p>Multimodal Surface Matching with Higher order Clique Reduction</p>
<p><em>homepage</em>: <a href="https://github.com/ecr05/MSM_HOCR">https://github.com/ecr05/MSM_HOCR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="mspc">MSPC<a class="headerlink" href="#mspc" title="Permanent link">&para;</a></h3>
<p>Using combined evidence from replicates to evaluate ChIP-seq peaks</p>
<p><em>homepage</em>: <a href="https://genometric.github.io/MSPC/">https://genometric.github.io/MSPC/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="msprime">msprime<a class="headerlink" href="#msprime" title="Permanent link">&para;</a></h3>
<p>msprime is a coalescent simulator and library for processing tree-based genetic data.</p>
<p><em>homepage</em>: <a href="https://msprime.readthedocs.io">https://msprime.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="mstore">mstore<a class="headerlink" href="#mstore" title="Permanent link">&para;</a></h3>
<p>Molecular structure store for testing</p>
<p><em>homepage</em>: <a href="https://github.com/grimme-lab/mstore">https://github.com/grimme-lab/mstore</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mtl4">MTL4<a class="headerlink" href="#mtl4" title="Permanent link">&para;</a></h3>
<p>The Matrix Template Library 4 incorporates the most modern programming techniques  to provide an easy and intuitive interface to users while enabling optimal performance. The natural  mathematical notation in MTL4 empowers all engineers and scientists to implement their algorithms and  models in minimal time. All technical aspects are encapsulated in the library.</p>
<p><em>homepage</em>: <a href="http://www.simunova.com/mtl4">http://www.simunova.com/mtl4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.8878</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.9555</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mujoco">MuJoCo<a class="headerlink" href="#mujoco" title="Permanent link">&para;</a></h3>
<p>MuJoCo stands for Multi-Joint dynamics with Contact. It is a general purpose physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, machine learning, and other areas which demand fast and accurate simulation of articulated structures interacting with their environment.</p>
<p><em>homepage</em>: <a href="https://mujoco.org/">https://mujoco.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mujoco-py">mujoco-py<a class="headerlink" href="#mujoco-py" title="Permanent link">&para;</a></h3>
<p>MuJoCo is a physics engine for detailed, efficient rigid body simulations with contacts. mujoco-py allows using MuJoCo from Python 3.</p>
<p><em>homepage</em>: <a href="https://github.com/openai/mujoco-py">https://github.com/openai/mujoco-py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.2.14</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="multichoose">multichoose<a class="headerlink" href="#multichoose" title="Permanent link">&para;</a></h3>
<p>generate multiset combinations (n multichoose k).</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/multichoose">https://github.com/ekg/multichoose</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="multilevelestimators">MultilevelEstimators<a class="headerlink" href="#multilevelestimators" title="Permanent link">&para;</a></h3>
<p>The Julia module for Multilevel Monte Carlo methods</p>
<p><em>homepage</em>: <a href="https://github.com/PieterjanRobbe/MultilevelEstimators.jl">https://github.com/PieterjanRobbe/MultilevelEstimators.jl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-Julia-1.7.2</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="multinest">MultiNest<a class="headerlink" href="#multinest" title="Permanent link">&para;</a></h3>
<p>MultiNest is a Bayesian inference tool which calculates the evidence and explores the parameter space  which may contain multiple posterior modes and pronounced (curving) degeneracies in moderately high dimensions.</p>
<p><em>homepage</em>: <a href="https://ccpforge.cse.rl.ac.uk/gf/project/multinest/">https://ccpforge.cse.rl.ac.uk/gf/project/multinest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.10</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="multiqc">MultiQC<a class="headerlink" href="#multiqc" title="Permanent link">&para;</a></h3>
<p>Aggregate results from bioinformatics analyses across many samples into a single   report.   MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general  use tool, perfect for summarising the output from numerous bioinformatics tools.</p>
<p><em>homepage</em>: <a href="https://multiqc.info">https://multiqc.info</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="multiwfn">Multiwfn<a class="headerlink" href="#multiwfn" title="Permanent link">&para;</a></h3>
<p>Multiwfn is an extremely powerful program for realizingi  electronic wavefunction analysis, which is a key ingredient of quantum  chemistry. Multiwfn is free, open-source, high-efficient, very user-friendly  and flexible, it supports almost all of the most important wavefunction  analysis methods.</p>
<p><em>homepage</em>: <a href="http://sobereva.com/multiwfn/">http://sobereva.com/multiwfn/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>intel/2019a</code>, <code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="mumerge">muMerge<a class="headerlink" href="#mumerge" title="Permanent link">&para;</a></h3>
<p>muMerge is a tool for combining bed regions from multiple bed files that overlap.</p>
<p><em>homepage</em>: <a href="https://github.com/Dowell-Lab/mumerge">https://github.com/Dowell-Lab/mumerge</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mummer">MUMmer<a class="headerlink" href="#mummer" title="Permanent link">&para;</a></h3>
<p>MUMmer is a system for rapidly aligning entire genomes,  whether in complete or draft form. AMOS makes use of it. </p>
<p><em>homepage</em>: <a href="https://mummer.sourceforge.net/">https://mummer.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.23</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>4.0.0beta2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.0rc1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mumps">MUMPS<a class="headerlink" href="#mumps" title="Permanent link">&para;</a></h3>
<p>A parallel sparse direct solver</p>
<p><em>homepage</em>: <a href="https://graal.ens-lyon.fr/MUMPS/">https://graal.ens-lyon.fr/MUMPS/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.2</code></td>
<td><code>-metis</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>-metis</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019a</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>-metis-seq</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>5.3.5</code></td>
<td><code>-metis</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>-metis</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td><code>-metis</code></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>5.5.0</code></td>
<td><code>-metis</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.5.1</code></td>
<td><code>-metis</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="muparser">muParser<a class="headerlink" href="#muparser" title="Permanent link">&para;</a></h3>
<p>muParser is an extensible high performance math expression parser library  written in C++. It works by transforming a mathematical expression into  bytecode and precalculating constant parts of the expression. </p>
<p><em>homepage</em>: <a href="https://beltoforion.de/article.php?a=muparser">https://beltoforion.de/article.php?a=muparser</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.5</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="muparserx">muparserx<a class="headerlink" href="#muparserx" title="Permanent link">&para;</a></h3>
<p>A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more.</p>
<p><em>homepage</em>: <a href="https://beltoforion.de/en/muparserx/">https://beltoforion.de/en/muparserx/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.8</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mupexi">MuPeXI<a class="headerlink" href="#mupexi" title="Permanent link">&para;</a></h3>
<p>MuPeXI: Mutant Peptide eXtractor and Informer. Given a list of somatic mutations (VCF file) as input,  MuPeXI returns a table containing all mutated peptides (neo-peptides) of user-defined lengths,  along with several pieces of information relevant for identifying  which of these neo-peptides are likely to serve as neo-epitopes.</p>
<p><em>homepage</em>: <a href="https://github.com/ambj/MuPeXI">https://github.com/ambj/MuPeXI</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Perl-5.28.0-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="muscle">MUSCLE<a class="headerlink" href="#muscle" title="Permanent link">&para;</a></h3>
<p>MUSCLE is one of the best-performing multiple alignment programs  according to published benchmark tests, with accuracy and speed that are consistently  better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users  learn everything they need to know about MUSCLE in a few minutes-only a handful of  command-line options are needed to perform common alignment tasks.</p>
<p><em>homepage</em>: <a href="https://drive5.com/muscle/">https://drive5.com/muscle/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.31</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2016a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.8.31</code></td>
<td><code>-i86linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.8.1551</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>5.0.1428</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="music">MuSiC<a class="headerlink" href="#music" title="Permanent link">&para;</a></h3>
<p>Multi-subject Single Cell deconvolution (MuSiC) is a deconvolution method that utilizes  cross-subject scRNA-seq to estimate cell type proportions in bulk RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/xuranw/MuSiC">https://github.com/xuranw/MuSiC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.2</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="must">MUST<a class="headerlink" href="#must" title="Permanent link">&para;</a></h3>
<p>MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.  As MPI calls are complex and usage errors common, this functionality is extremely helpful for application developers  that want to develop correct MPI applications.  This includes errors that already manifest  segmentation faults or incorrect results  as well as many errors  that are not visible to the application developer or do not manifest on a certain system or MPI implementation.</p>
<p><em>homepage</em>: <a href="https://hpc.rwth-aachen.de/must/">https://hpc.rwth-aachen.de/must/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="mutect">MuTect<a class="headerlink" href="#mutect" title="Permanent link">&para;</a></h3>
<p>MuTect is a method developed at the Broad Institute for the reliable  and accurate identification of somatic point mutations in next generation sequencing  data of cancer genomes. </p>
<p><em>homepage</em>: <a href="http://www.broadinstitute.org/cancer/cga/mutect">http://www.broadinstitute.org/cancer/cga/mutect</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Java-1.7.0_76</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="mutil">mutil<a class="headerlink" href="#mutil" title="Permanent link">&para;</a></h3>
<p>Mutil is a set of standard utilities that have been parallelized to maximize performance on  modern file systems. These currently include multi-threaded drop-in replacements for cp and md5sum from GNU coreutils, which have achieved 10/30x rates on one/many nodes. </p>
<p><em>homepage</em>: <a href="http://people.nas.nasa.gov/~kolano/projects/mutil.html">http://people.nas.nasa.gov/~kolano/projects/mutil.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.822.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="mvapich2">MVAPICH2<a class="headerlink" href="#mvapich2" title="Permanent link">&para;</a></h3>
<p>This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH.</p>
<p><em>homepage</em>: <a href="http://mvapich.cse.ohio-state.edu/overview/mvapich2/">http://mvapich.cse.ohio-state.edu/overview/mvapich2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>GCC/4.8.4</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.2b</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
</tbody>
</table>
<h3 id="mxml">mxml<a class="headerlink" href="#mxml" title="Permanent link">&para;</a></h3>
<p>Mini-XML is a tiny XML library that you can use to read and write XML and  XML-like data files in your application without requiring large non-standard  libraries. </p>
<p><em>homepage</em>: <a href="https://github.com/michaelrsweet/mxml">https://github.com/michaelrsweet/mxml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mxmlplus">mxmlplus<a class="headerlink" href="#mxmlplus" title="Permanent link">&para;</a></h3>
<p>Mxml is a pure C library (yet having an object oriented layout) that is meant to help developers  implementing XML file interpretation in their projects.</p>
<p><em>homepage</em>: <a href="http://mxml.sourceforge.net">http://mxml.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mxnet">MXNet<a class="headerlink" href="#mxnet" title="Permanent link">&para;</a></h3>
<p>Flexible and Efficient Library for Deep Learning</p>
<p><em>homepage</em>: <a href="https://mxnet.io/">https://mxnet.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.3</code></td>
<td><code>-Python-2.7.12-R-3.3.3</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="mycc">MyCC<a class="headerlink" href="#mycc" title="Permanent link">&para;</a></h3>
<p>MyCC is built and delivered as a tailored solution for metagenomics sequencesclassfication.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/sb2nhri/files/MyCC/">https://sourceforge.net/projects/sb2nhri/files/MyCC/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017-03-01</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="mygene">mygene<a class="headerlink" href="#mygene" title="Permanent link">&para;</a></h3>
<p>Python Client for MyGene.Info services.</p>
<p><em>homepage</em>: <a href="https://github.com/biothings/mygene.py">https://github.com/biothings/mygene.py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="mymedialite">MyMediaLite<a class="headerlink" href="#mymedialite" title="Permanent link">&para;</a></h3>
<p>MyMediaLite is a lightweight, multi-purpose library of recommender system algorithms.</p>
<p><em>homepage</em>: <a href="http://www.ismll.uni-hildesheim.de/mymedialite/">http://www.ismll.uni-hildesheim.de/mymedialite/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.10</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.11</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="mympingpong">mympingpong<a class="headerlink" href="#mympingpong" title="Permanent link">&para;</a></h3>
<p>A mpi4py based random pair pingpong network stress test.</p>
<p><em>homepage</em>: <a href="https://github.com/hpcugent/mympingpong">https://github.com/hpcugent/mympingpong</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="myokit">Myokit<a class="headerlink" href="#myokit" title="Permanent link">&para;</a></h3>
<p>Myokit is an open-source Python-based toolkit that facilitates modeling and simulation of cardiac cellular electrophysiology.</p>
<p><em>homepage</em>: <a href="http://myokit.org">http://myokit.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.32.0</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="mypy">mypy<a class="headerlink" href="#mypy" title="Permanent link">&para;</a></h3>
<p>Optional static typing for Python</p>
<p><em>homepage</em>: <a href="http://www.mypy-lang.org/">http://www.mypy-lang.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.5</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="mysql">MySQL<a class="headerlink" href="#mysql" title="Permanent link">&para;</a></h3>
<p>MySQL is one of the world's most widely used open-source  relational database management system (RDBMS).</p>
<p><em>homepage</em>: <a href="http://www.mysql.com/">http://www.mysql.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6.26</code></td>
<td><code>-clientonly</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>5.7.21</code></td>
<td><code>-clientonly</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="mysql-python">MySQL-python<a class="headerlink" href="#mysql-python" title="Permanent link">&para;</a></h3>
<p>MySQL database connector for Python</p>
<p><em>homepage</em>: <a href="https://github.com/farcepest/MySQLdb1">https://github.com/farcepest/MySQLdb1</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.5</code></td>
<td><code>-Python-2.7.11-MariaDB-10.1.14</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="mysqlclient">mysqlclient<a class="headerlink" href="#mysqlclient" title="Permanent link">&para;</a></h3>
<p>Python interface to MySQL</p>
<p><em>homepage</em>: <a href="https://github.com/PyMySQL/mysqlclient-python">https://github.com/PyMySQL/mysqlclient-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h2 id="n">N<a class="headerlink" href="#n" title="Permanent link">&para;</a></h2>
<p><a href="#n2v">n2v</a> - <a href="#nag">NAG</a> - <a href="#nagfor">NAGfor</a> - <a href="#namd">NAMD</a> - <a href="#namedlist">namedlist</a> - <a href="#nano">nano</a> - <a href="#nanocomp">NanoComp</a> - <a href="#nanocompore">nanocompore</a> - <a href="#nanofilt">NanoFilt</a> - <a href="#nanoflann">nanoflann</a> - <a href="#nanoget">nanoget</a> - <a href="#nanolyse">NanoLyse</a> - <a href="#nanomath">nanomath</a> - <a href="#nanomax-analysis-utils">nanomax-analysis-utils</a> - <a href="#nanonet">nanonet</a> - <a href="#nanoplot">NanoPlot</a> - <a href="#nanopolish">nanopolish</a> - <a href="#nanopolishcomp">NanopolishComp</a> - <a href="#napari">napari</a> - <a href="#nasm">NASM</a> - <a href="#nauty">nauty</a> - <a href="#nbo">NBO</a> - <a href="#ncbi-toolkit">NCBI-Toolkit</a> - <a href="#ncbi-vdb">ncbi-vdb</a> - <a href="#nccl">NCCL</a> - <a href="#nccl-tests">NCCL-tests</a> - <a href="#ncdf4">ncdf4</a> - <a href="#ncdu">ncdu</a> - <a href="#nciplot">NCIPLOT</a> - <a href="#ncl">NCL</a> - <a href="#nco">NCO</a> - <a href="#ncompress">ncompress</a> - <a href="#ncurses">ncurses</a> - <a href="#ncview">ncview</a> - <a href="#nd2reader">nd2reader</a> - <a href="#ne">ne</a> - <a href="#neci">NECI</a> - <a href="#nedit">NEdit</a> - <a href="#nek5000">Nek5000</a> - <a href="#nektar++">Nektar++</a> - <a href="#neon">neon</a> - <a href="#neptune-client">neptune-client</a> - <a href="#net-core">Net-core</a> - <a href="#netcdf">netCDF</a> - <a href="#netcdf-c++">netCDF-C++</a> - <a href="#netcdf-c++4">netCDF-C++4</a> - <a href="#netcdf-fortran">netCDF-Fortran</a> - <a href="#netcdf4-python">netcdf4-python</a> - <a href="#netloc">netloc</a> - <a href="#netlogo">NetLogo</a> - <a href="#netmhc">netMHC</a> - <a href="#netmhcii">netMHCII</a> - <a href="#netmhciipan">netMHCIIpan</a> - <a href="#netmhcpan">netMHCpan</a> - <a href="#netpipe">NetPIPE</a> - <a href="#netpyne">NetPyNE</a> - <a href="#nettle">nettle</a> - <a href="#networktools">networkTools</a> - <a href="#networkx">networkx</a> - <a href="#neurokit">NeuroKit</a> - <a href="#neuron">NEURON</a> - <a href="#newhybrids">NewHybrids</a> - <a href="#nextflow">Nextflow</a> - <a href="#nextgenmap">NextGenMap</a> - <a href="#nexus-cl">NEXUS-CL</a> - <a href="#nf-core-mag">nf-core-mag</a> - <a href="#nfft">NFFT</a> - <a href="#nghttp2">nghttp2</a> - <a href="#nghttp3">nghttp3</a> - <a href="#ngless">NGLess</a> - <a href="#nglview">nglview</a> - <a href="#ngs">NGS</a> - <a href="#ngs-python">NGS-Python</a> - <a href="#ngsadmix">NGSadmix</a> - <a href="#ngspeciesid">NGSpeciesID</a> - <a href="#ngspice">ngspice</a> - <a href="#ngtcp2">ngtcp2</a> - <a href="#nibabel">NiBabel</a> - <a href="#nichenetr">nichenetr</a> - <a href="#nifti">NIfTI</a> - <a href="#nifti2dicom">nifti2dicom</a> - <a href="#nilearn">Nilearn</a> - <a href="#nim">Nim</a> - <a href="#nimble">NIMBLE</a> - <a href="#ninja">Ninja</a> - <a href="#nipype">Nipype</a> - <a href="#nlmpy">NLMpy</a> - <a href="#nlohmann_json">nlohmann_json</a> - <a href="#nlopt">NLopt</a> - <a href="#nltk">NLTK</a> - <a href="#nnu-net">nnU-Net</a> - <a href="#node-red">Node-RED</a> - <a href="#nodejs">nodejs</a> - <a href="#normaliz">Normaliz</a> - <a href="#nose-parameterized">nose-parameterized</a> - <a href="#novoalign">novoalign</a> - <a href="#novoplasty">NOVOPlasty</a> - <a href="#npstat">npstat</a> - <a href="#nrgljubljana">NRGLjubljana</a> - <a href="#nsight-compute">Nsight-Compute</a> - <a href="#nsight-systems">Nsight-Systems</a> - <a href="#nspr">NSPR</a> - <a href="#nss">NSS</a> - <a href="#nsync">nsync</a> - <a href="#ntcard">ntCard</a> - <a href="#ntedit">ntEdit</a> - <a href="#nthits">ntHits</a> - <a href="#ntl">NTL</a> - <a href="#ntpoly">NTPoly</a> - <a href="#num2words">num2words</a> - <a href="#numactl">numactl</a> - <a href="#numba">numba</a> - <a href="#numdiff">numdiff</a> - <a href="#numexpr">numexpr</a> - <a href="#numpy">numpy</a> - <a href="#nvhpc">NVHPC</a> - <a href="#nvofbf">nvofbf</a> - <a href="#nvompi">nvompi</a> - <a href="#nvshmem">NVSHMEM</a> - <a href="#nvtop">nvtop</a> - <a href="#nwchem">NWChem</a> - <a href="#nxtrim">NxTrim</a></p>
<h3 id="n2v">n2v<a class="headerlink" href="#n2v" title="Permanent link">&para;</a></h3>
<p>Noise2Void - Learning Denoising from Single Noisy Images The field of image denoising is currently dominated by discriminative deep       learning methods that are trained on pairs of noisy input and clean target       images. Recently it has been shown that such methods can also be trained without clean targets. Instead, independent pairs of noisy images can be used, in an     approach known as NOISE2NOISE (N2N). Here, we introduce NOISE2VOID (N2V), a      training scheme that takes this idea one step further. It does not require noisy image pairs, nor clean target images. Consequently, N2V allows us to train       directly on the body of data to be denoised and can therefore be applied when    other methods cannot. Especially interesting is the application to biomedical    image data, where the acquisition of training targets, clean or noisy, is        frequently not possible. We compare the performance of N2V to approaches that    have either clean target images and/or noisy image pairs available. Intuitively, N2V cannot be expected to outperform methods that have more information          available during training. Still, we observe that the denoising performance of   NOISE2VOID drops in moderation and compares favorably to training-free denoising methods.</p>
<p><em>homepage</em>: <a href="https://github.com/juglab/n2v">https://github.com/juglab/n2v</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nag">NAG<a class="headerlink" href="#nag" title="Permanent link">&para;</a></h3>
<p>The worlds largest collection of robust, documented, tested and maintained numerical algorithms.</p>
<p><em>homepage</em>: <a href="http://www.nag.co.uk">http://www.nag.co.uk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>24</code></td>
<td><code>PGI/17.4-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>26</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="nagfor">NAGfor<a class="headerlink" href="#nagfor" title="Permanent link">&para;</a></h3>
<p>The checking compiler for improved code portability and detailed error reporting.</p>
<p><em>homepage</em>: <a href="http://www.nag.co.uk">http://www.nag.co.uk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2.14</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="namd">NAMD<a class="headerlink" href="#namd" title="Permanent link">&para;</a></h3>
<p>NAMD is a parallel molecular dynamics code designed for high-performance simulation of  large biomolecular systems.</p>
<p><em>homepage</em>: <a href="https://www.ks.uiuc.edu/Research/namd/">https://www.ks.uiuc.edu/Research/namd/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>-mpi</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>-CUDA-8.0.61</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2018b</code>, <code>foss/2019b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td></td>
<td><code>fosscuda/2019b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.14</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="namedlist">namedlist<a class="headerlink" href="#namedlist" title="Permanent link">&para;</a></h3>
<p>A Python object, similar to namedtuple, but for lists.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/ericvsmith/namedlist">https://gitlab.com/ericvsmith/namedlist</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nano">nano<a class="headerlink" href="#nano" title="Permanent link">&para;</a></h3>
<p>a simple editor, inspired by Pico</p>
<p><em>homepage</em>: <a href="https://www.nano-editor.org/">https://www.nano-editor.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nanocomp">NanoComp<a class="headerlink" href="#nanocomp" title="Permanent link">&para;</a></h3>
<p>Comparing runs of Oxford Nanopore sequencing data and alignments</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/NanoComp">https://github.com/wdecoster/NanoComp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="nanocompore">nanocompore<a class="headerlink" href="#nanocompore" title="Permanent link">&para;</a></h3>
<p>Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples</p>
<p><em>homepage</em>: <a href="https://nanocompore.rna.rocks/">https://nanocompore.rna.rocks/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0rc3-2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="nanofilt">NanoFilt<a class="headerlink" href="#nanofilt" title="Permanent link">&para;</a></h3>
<p>Filtering and trimming of long read sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/nanofilt">https://github.com/wdecoster/nanofilt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="nanoflann">nanoflann<a class="headerlink" href="#nanoflann" title="Permanent link">&para;</a></h3>
<p>nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies.</p>
<p><em>homepage</em>: <a href="https://github.com/jlblancoc/nanoflann">https://github.com/jlblancoc/nanoflann</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nanoget">nanoget<a class="headerlink" href="#nanoget" title="Permanent link">&para;</a></h3>
<p>Functions to extract information from Oxford Nanopore sequencing data and alignments</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/nanoget">https://github.com/wdecoster/nanoget</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.15.0</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.18.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nanolyse">NanoLyse<a class="headerlink" href="#nanolyse" title="Permanent link">&para;</a></h3>
<p>Remove reads mapping to the lambda phage genome from a fastq file.</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/nanolyse">https://github.com/wdecoster/nanolyse</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nanomath">nanomath<a class="headerlink" href="#nanomath" title="Permanent link">&para;</a></h3>
<p>A few simple math function for other Oxford Nanopore processing scripts</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/nanomath">https://github.com/wdecoster/nanomath</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.23.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nanomax-analysis-utils">nanomax-analysis-utils<a class="headerlink" href="#nanomax-analysis-utils" title="Permanent link">&para;</a></h3>
<p>A set of tools for handling and analysing data at the NanoMAX beamline.</p>
<p><em>homepage</em>: <a href="https://github.com/maxiv-science/nanomax-analysis-utils">https://github.com/maxiv-science/nanomax-analysis-utils</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.4.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.4.4</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="nanonet">nanonet<a class="headerlink" href="#nanonet" title="Permanent link">&para;</a></h3>
<p>Nanonet provides recurrent neural network basecalling for Oxford Nanopore MinION data.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/nanonet">https://github.com/nanoporetech/nanonet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="nanoplot">NanoPlot<a class="headerlink" href="#nanoplot" title="Permanent link">&para;</a></h3>
<p>Plotting suite for long read sequencing data and alignments</p>
<p><em>homepage</em>: <a href="https://github.com/wdecoster/NanoPlot">https://github.com/wdecoster/NanoPlot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.28.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.33.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="nanopolish">nanopolish<a class="headerlink" href="#nanopolish" title="Permanent link">&para;</a></h3>
<p>Software package for signal-level analysis of Oxford Nanopore sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/jts/nanopolish">https://github.com/jts/nanopolish</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.2</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.10.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.13.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.13.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="nanopolishcomp">NanopolishComp<a class="headerlink" href="#nanopolishcomp" title="Permanent link">&para;</a></h3>
<p>NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files</p>
<p><em>homepage</em>: <a href="https://adrienleger.com/NanopolishComp/">https://adrienleger.com/NanopolishComp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.11</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="napari">napari<a class="headerlink" href="#napari" title="Permanent link">&para;</a></h3>
<p>napari is a fast, interactive, multi-dimensional image viewer for Python. It's   designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based           rendering), and the scientific Python stack (numpy, scipy).</p>
<p><em>homepage</em>: <a href="https://napari.org/">https://napari.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.15</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="nasm">NASM<a class="headerlink" href="#nasm" title="Permanent link">&para;</a></h3>
<p>NASM: General-purpose x86 assembler</p>
<p><em>homepage</em>: <a href="https://www.nasm.us/">https://www.nasm.us/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11.08</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.12.01</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.12.02</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.13.01</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.13.03</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.14.02</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.15.05</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nauty">nauty<a class="headerlink" href="#nauty" title="Permanent link">&para;</a></h3>
<p>nauty and Traces are programs for computing automorphism groups of graphs and digraphs. They can also produce a canonical label.</p>
<p><em>homepage</em>: <a href="https://pallini.di.uniroma1.it/">https://pallini.di.uniroma1.it/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6r12</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.7rc2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.7rc5</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nbo">NBO<a class="headerlink" href="#nbo" title="Permanent link">&para;</a></h3>
<p>The Natural Bond Orbital (NBO) program is a discovery tool for chemical insights from complex wavefunctions. </p>
<p><em>homepage</em>: <a href="http://nbo.chem.wisc.edu/">http://nbo.chem.wisc.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="ncbi-toolkit">NCBI-Toolkit<a class="headerlink" href="#ncbi-toolkit" title="Permanent link">&para;</a></h3>
<p>The NCBI Toolkit is a collection of utilities developed for the   production and distribution of GenBank, Entrez, BLAST, and related services   by the National Center for Biotechnology Information.</p>
<p><em>homepage</em>: <a href="http://ncbi.github.io/cxx-toolkit/">http://ncbi.github.io/cxx-toolkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>18.0.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="ncbi-vdb">ncbi-vdb<a class="headerlink" href="#ncbi-vdb" title="Permanent link">&para;</a></h3>
<p>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for  using data in the INSDC Sequence Read Archives.</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/ncbi-vdb">https://github.com/ncbi/ncbi-vdb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.8-1</code></td>
<td><code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.9.1-1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.9.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.10.4</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.10.7</code></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.10.9</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>2.11.2</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nccl">NCCL<a class="headerlink" href="#nccl" title="Permanent link">&para;</a></h3>
<p>The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/nccl">https://developer.nvidia.com/nccl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.4</code></td>
<td><code>-CUDA-9.0.176</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.13</code></td>
<td><code>-CUDA-9.2.148.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.7</code></td>
<td></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>gcccuda/2019a</code></td>
</tr>
<tr>
<td><code>2.4.8</code></td>
<td></td>
<td><code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>2.8.3</code></td>
<td><code>-CUDA-11.0.2</code></td>
<td><code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.8.3</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.9.9</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.10.3</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.11.4</code></td>
<td></td>
<td><code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>2.12.12</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.16.2</code></td>
<td><code>-CUDA-12.0.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nccl-tests">NCCL-tests<a class="headerlink" href="#nccl-tests" title="Permanent link">&para;</a></h3>
<p>Tests check both the performance and the correctness of NCCL operations.</p>
<p><em>homepage</em>: <a href="https://github.com/NVIDIA/nccl-tests">https://github.com/NVIDIA/nccl-tests</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>gompic/2019b</code></td>
</tr>
<tr>
<td><code>2.13.6</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ncdf4">ncdf4<a class="headerlink" href="#ncdf4" title="Permanent link">&para;</a></h3>
<p>ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/ncdf4">https://cran.r-project.org/web/packages/ncdf4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.16</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="ncdu">ncdu<a class="headerlink" href="#ncdu" title="Permanent link">&para;</a></h3>
<p>Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a  remote server where you don't have an entire graphical setup available, but it is a useful tool even on regular  desktop systems. Ncdu aims to be fast, simple and easy to use, and should be able to run in any minimal POSIX-like  environment with ncurses installed.</p>
<p><em>homepage</em>: <a href="https://dev.yorhel.nl/ncdu">https://dev.yorhel.nl/ncdu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nciplot">NCIPLOT<a class="headerlink" href="#nciplot" title="Permanent link">&para;</a></h3>
<p>NCIPLOT is a program for revealing non covalent interactions  based on the reduced density gradient. </p>
<p><em>homepage</em>: <a href="https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html">https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0-20190718</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>4.0-20200106</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>4.0-20200624</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ncl">NCL<a class="headerlink" href="#ncl" title="Permanent link">&para;</a></h3>
<p>NCL is an interpreted language designed specifically for scientific data analysis and visualization.</p>
<p><em>homepage</em>: <a href="https://www.ncl.ucar.edu">https://www.ncl.ucar.edu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.4.0</code></td>
<td><code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.5.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.6.2</code></td>
<td><code>foss/2018b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>intel/2018b</code>, <code>intel/2019b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="nco">NCO<a class="headerlink" href="#nco" title="Permanent link">&para;</a></h3>
<p>The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats,  including DAP, HDF4, and HDF5.</p>
<p><em>homepage</em>: <a href="https://github.com/nco/nco">https://github.com/nco/nco</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.6.6</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.7.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.7.4</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>4.7.6</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.7.9</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.9.7</code></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.0.3</code></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>5.1.3</code></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ncompress">ncompress<a class="headerlink" href="#ncompress" title="Permanent link">&para;</a></h3>
<p>Compress is a fast, simple LZW file compressor. Compress does not have the  highest compression rate, but it is one of the fastest programs to compress  data. Compress is the defacto standard in the UNIX community for compressing  files. </p>
<p><em>homepage</em>: <a href="http://ncompress.sourceforge.net/">http://ncompress.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.4.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ncurses">ncurses<a class="headerlink" href="#ncurses" title="Permanent link">&para;</a></h3>
<p>The Ncurses (new curses) library is a free software emulation of curses in  System V Release 4.0, and more. It uses Terminfo format, supports pads and  color and multiple highlights and forms characters and function-key mapping,  and has all the other SYSV-curses enhancements over BSD Curses. </p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/ncurses/">https://www.gnu.org/software/ncurses/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.9</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code>, <code>GCC/4.8.3</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GNU/4.9.3-2.25</code>, <code>gimkl/2.11.5</code>, <code>system</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>6.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>6.2</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>6.3</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ncview">ncview<a class="headerlink" href="#ncview" title="Permanent link">&para;</a></h3>
<p>Ncview is a visual browser for netCDF format files.  Typically you would use ncview to get a quick and easy, push-button  look at your netCDF files. You can view simple movies of the data,  view along various dimensions, take a look at the actual data values,  change color maps, invert the data, etc.</p>
<p><em>homepage</em>: <a href="http://meteora.ucsd.edu/~pierce/ncview_home_page.html">http://meteora.ucsd.edu/~pierce/ncview_home_page.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.7</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>foss/2019b</code>, <code>gompi/2019a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2019b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>2.1.8</code></td>
<td><code>gompi/2020a</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nd2reader">nd2reader<a class="headerlink" href="#nd2reader" title="Permanent link">&para;</a></h3>
<p>nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+.</p>
<p><em>homepage</em>: <a href="https://www.lighthacking.nl/nd2reader/">https://www.lighthacking.nl/nd2reader/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="ne">ne<a class="headerlink" href="#ne" title="Permanent link">&para;</a></h3>
<p>ne is a free (GPL'd) text editor based on the POSIX standard that runs (we hope) on almost any UN*X machine. ne is easy to use for the beginner, but powerful and fully configurable for the wizard, and most sparing in its resource usage.</p>
<p><em>homepage</em>: <a href="http://ne.di.unimi.it/">http://ne.di.unimi.it/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>gimkl/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="neci">NECI<a class="headerlink" href="#neci" title="Permanent link">&para;</a></h3>
<p>Standalone NECI codebase designed for FCIQMC and other stochastic quantum chemistry methods.</p>
<p><em>homepage</em>: <a href="https://github.com/ghb24/NECI_STABLE">https://github.com/ghb24/NECI_STABLE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220711</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nedit">NEdit<a class="headerlink" href="#nedit" title="Permanent link">&para;</a></h3>
<p>NEdit is a multi-purpose text editor for the X Window System,  which combines a standard, easy to use, graphical user interface with the thorough  functionality and stability required by users who edit text eight hours a day.</p>
<p><em>homepage</em>: <a href="http://www.nedit.org">http://www.nedit.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.5</code></td>
<td><code>-Linux-x86</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nek5000">Nek5000<a class="headerlink" href="#nek5000" title="Permanent link">&para;</a></h3>
<p>a fast and scalable high-order solver for computational fluid dynamics</p>
<p><em>homepage</em>: <a href="https://nek5000.mcs.anl.gov/">https://nek5000.mcs.anl.gov/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="nektar">Nektar++<a class="headerlink" href="#nektar" title="Permanent link">&para;</a></h3>
<p>Nektar++ is a tensor product based finite element package designed to allow one to construct efficient classical low polynomial order h-type solvers (where h is the size of the finite element) as well as higher p-order piecewise polynomial order solvers.</p>
<p><em>homepage</em>: <a href="https://www.nektar.info">https://www.nektar.info</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="neon">neon<a class="headerlink" href="#neon" title="Permanent link">&para;</a></h3>
<p>neon is an HTTP/1.1 and WebDAV client library, with a C interface. </p>
<p><em>homepage</em>: <a href="https://github.com/notroj/neon">https://github.com/notroj/neon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.31.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="neptune-client">neptune-client<a class="headerlink" href="#neptune-client" title="Permanent link">&para;</a></h3>
<p>Neptune is an experiment tracking hub that brings organization and collaboration to your data science team. </p>
<p><em>homepage</em>: <a href="https://docs.neptune.ai">https://docs.neptune.ai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.129</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.10.5</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.16.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="net-core">Net-core<a class="headerlink" href="#net-core" title="Permanent link">&para;</a></h3>
<p>.NET Core is a free and open-source managed computer software framework for the Windows,  Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic</p>
<p><em>homepage</em>: <a href="https://dotnet.microsoft.com/">https://dotnet.microsoft.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="netcdf">netCDF<a class="headerlink" href="#netcdf" title="Permanent link">&para;</a></h3>
<p>NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.</p>
<p><em>homepage</em>: <a href="https://www.unidata.ucar.edu/software/netcdf/">https://www.unidata.ucar.edu/software/netcdf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.3.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>4.4.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.4.1.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>4.4.1.1</code></td>
<td><code>-HDF5-1.10.1</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.4.1.1</code></td>
<td><code>-HDF5-1.8.18</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.4.1.1</code></td>
<td><code>-HDF5-1.8.19</code></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.5.0</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.00</code>, <code>intel/2018.01</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>4.6.1</code></td>
<td></td>
<td><code>PGI/18.4-GCC-6.4.0-2.28</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>4.6.2</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>4.7.1</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompic/2019b</code>, <code>iimpi/2019b</code>, <code>iimpic/2019b</code></td>
</tr>
<tr>
<td><code>4.7.4</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompic/2020a</code>, <code>gompic/2020b</code>, <code>iimpi/2020a</code>, <code>iimpi/2020b</code>, <code>iimpic/2020b</code></td>
</tr>
<tr>
<td><code>4.8.0</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>iimpi/2021a</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td></td>
<td><code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>4.9.0</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>gompi/2022b</code>, <code>iimpi/2022a</code>, <code>iimpi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="netcdf-c">netCDF-C++<a class="headerlink" href="#netcdf-c" title="Permanent link">&para;</a></h3>
<p>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.</p>
<p><em>homepage</em>: <a href="http://www.unidata.ucar.edu/software/netcdf/">http://www.unidata.ucar.edu/software/netcdf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="netcdf-c4">netCDF-C++4<a class="headerlink" href="#netcdf-c4" title="Permanent link">&para;</a></h3>
<p>NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.</p>
<p><em>homepage</em>: <a href="https://www.unidata.ucar.edu/software/netcdf/">https://www.unidata.ucar.edu/software/netcdf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.0</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019a</code>, <code>iimpi/2019a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-HDF5-1.8.18</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-HDF5-1.8.19</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code>, <code>iimpi/2021b</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="netcdf-fortran">netCDF-Fortran<a class="headerlink" href="#netcdf-fortran" title="Permanent link">&para;</a></h3>
<p>NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.</p>
<p><em>homepage</em>: <a href="https://www.unidata.ucar.edu/software/netcdf/">https://www.unidata.ucar.edu/software/netcdf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.4.2</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>4.4.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.4.4</code></td>
<td></td>
<td><code>PGI/18.4-GCC-6.4.0-2.28</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intelcuda/2017b</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>4.4.4</code></td>
<td><code>-HDF5-1.8.18</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.4.4</code></td>
<td><code>-HDF5-1.8.19</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.4.5</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>iimpi/2019a</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompic/2019b</code>, <code>gompic/2020a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iimpic/2019b</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="netcdf4-python">netcdf4-python<a class="headerlink" href="#netcdf4-python" title="Permanent link">&para;</a></h3>
<p>Python/numpy interface to netCDF.</p>
<p><em>homepage</em>: <a href="https://unidata.github.io/netcdf4-python/">https://unidata.github.io/netcdf4-python/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.9</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.6.3-HDF5-1.8.19</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.6.2-HDF5-1.8.19</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.5.5.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.5.7</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="netloc">netloc<a class="headerlink" href="#netloc" title="Permanent link">&para;</a></h3>
<p>The Portable Network Locality (netloc) software package provides  network topology discovery tools, and an abstract representation of those networks  topologies for a range of network types and configurations.  It is provided as a companion to the Portable Hardware Locality (hwloc) package.</p>
<p><em>homepage</em>: <a href="http://www.open-mpi.org/software/netloc">http://www.open-mpi.org/software/netloc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
</tbody>
</table>
<h3 id="netlogo">NetLogo<a class="headerlink" href="#netlogo" title="Permanent link">&para;</a></h3>
<p>NetLogo is a multi-agent programmable modeling environment. It is used by tens of thousands of students, teachers and researchers worldwide. It also powers HubNet participatory simulations. It is authored by Uri Wilensky and developed at the CCL.</p>
<p><em>homepage</em>: <a href="https://ccl.northwestern.edu/netlogo/">https://ccl.northwestern.edu/netlogo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.3.1</code></td>
<td><code>-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.4</code></td>
<td><code>-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.2.2</code></td>
<td><code>-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>-64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="netmhc">netMHC<a class="headerlink" href="#netmhc" title="Permanent link">&para;</a></h3>
<p>NetMHC 4.0 predicts binding of peptides to a number of different  HLA alleles using artificial neural networks (ANN).</p>
<p><em>homepage</em>: <a href="https://www.cbs.dtu.dk/services/NetMHC/">https://www.cbs.dtu.dk/services/NetMHC/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="netmhcii">netMHCII<a class="headerlink" href="#netmhcii" title="Permanent link">&para;</a></h3>
<p>NetMHCII 2.3 server predicts binding of peptides to HLA-DR,  HLA-DQ, HLA-DP and mouse MHC class II alleles using articial neuron networks.  Predictions can be obtained for 25 HLA-DR alleles, 20 HLA-DQ, 9 HLA-DP, and  7 mouse H2 class II alleles.</p>
<p><em>homepage</em>: <a href="https://www.cbs.dtu.dk/services/NetMHCII/">https://www.cbs.dtu.dk/services/NetMHCII/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="netmhciipan">netMHCIIpan<a class="headerlink" href="#netmhciipan" title="Permanent link">&para;</a></h3>
<p>NetMHCIIpan 3.2 server predicts binding of peptides to MHC  class II molecules. The predictions are available for the three human MHC  class II isotypes HLA-DR, HLA-DP and HLA-DQ, as well as mouse molecules (H-2).</p>
<p><em>homepage</em>: <a href="https://www.cbs.dtu.dk/services/NetMHCIIpan/">https://www.cbs.dtu.dk/services/NetMHCIIpan/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="netmhcpan">netMHCpan<a class="headerlink" href="#netmhcpan" title="Permanent link">&para;</a></h3>
<p>The NetMHCpan software predicts binding of peptides to any known MHC  molecule using artificial neural networks (ANNs).</p>
<p><em>homepage</em>: <a href="https://www.cbs.dtu.dk/services/NetMHCpan/">https://www.cbs.dtu.dk/services/NetMHCpan/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0a</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="netpipe">NetPIPE<a class="headerlink" href="#netpipe" title="Permanent link">&para;</a></h3>
<p>NetPIPE is a protocol independent communication performance benchmark that visually represents  the network performance under a variety of conditions.</p>
<p><em>homepage</em>: <a href="https://netpipe.cs.ksu.edu/">https://netpipe.cs.ksu.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.1.4</code></td>
<td><code>gompi/2020b</code>, <code>iimpi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="netpyne">NetPyNE<a class="headerlink" href="#netpyne" title="Permanent link">&para;</a></h3>
<p>NetPyNE is an open-source Python package to facilitate the development, parallel simulation, analysis, and optimization of biological neuronal networks using the NEURON simulator.</p>
<p><em>homepage</em>: <a href="http://www.netpyne.org">http://www.netpyne.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="nettle">nettle<a class="headerlink" href="#nettle" title="Permanent link">&para;</a></h3>
<p>Nettle is a cryptographic library that is designed to fit easily  in more or less any context: In crypto toolkits for object-oriented  languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,  or even in kernel space.</p>
<p><em>homepage</em>: <a href="https://www.lysator.liu.se/~nisse/nettle/">https://www.lysator.liu.se/~nisse/nettle/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.1</code></td>
<td><code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.7.3</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.8.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="networktools">networkTools<a class="headerlink" href="#networktools" title="Permanent link">&para;</a></h3>
<p>Dynamical Network Analysis is a method of characterizing allosteric signalling through biomolecular complexes.</p>
<p><em>homepage</em>: <a href="http://faculty.scs.illinois.edu/schulten/software/networkTools/">http://faculty.scs.illinois.edu/schulten/software/networkTools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="networkx">networkx<a class="headerlink" href="#networkx" title="Permanent link">&para;</a></h3>
<p>NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics, and functions of complex networks.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/networkx">https://pypi.python.org/pypi/networkx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.8.4</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="neurokit">NeuroKit<a class="headerlink" href="#neurokit" title="Permanent link">&para;</a></h3>
<p>NeuroKit is a Python module that provides high-level integrative functions with good and  flexible defaults, allowing users to focus on whats important.</p>
<p><em>homepage</em>: <a href="https://neurokit.readthedocs.io">https://neurokit.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.7</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="neuron">NEURON<a class="headerlink" href="#neuron" title="Permanent link">&para;</a></h3>
<p>Empirically-based simulations of neurons and networks of neurons.</p>
<p><em>homepage</em>: <a href="https://www.neuron.yale.edu/neuron">https://www.neuron.yale.edu/neuron</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.4</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.6.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>7.8.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="newhybrids">NewHybrids<a class="headerlink" href="#newhybrids" title="Permanent link">&para;</a></h3>
<p>This implements a Gibbs sampler to estimate the posterior probability that genetically sampled individuals fall into each of a set of user-defined hybrid categories.</p>
<p><em>homepage</em>: <a href="https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs">https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1_Beta3</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nextflow">Nextflow<a class="headerlink" href="#nextflow" title="Permanent link">&para;</a></h3>
<p>Nextflow is a reactive workflow framework and a programming DSL   that eases writing computational pipelines with complex data</p>
<p><em>homepage</em>: <a href="https://www.nextflow.io/">https://www.nextflow.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.04.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.07.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.12.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20.01.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20.04.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20.10.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.03.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.08.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.10.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.04.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.10.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.10.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.10.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.10.6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nextgenmap">NextGenMap<a class="headerlink" href="#nextgenmap" title="Permanent link">&para;</a></h3>
<p>NextGenMap is a flexible highly sensitive short read mapping tool that  handles much higher mismatch rates than comparable algorithms while still outperforming  them in terms of runtime.</p>
<p><em>homepage</em>: <a href="http://cibiv.github.io/NextGenMap/">http://cibiv.github.io/NextGenMap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.5</code></td>
<td><code>GCC/11.2.0</code>, <code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="nexus-cl">NEXUS-CL<a class="headerlink" href="#nexus-cl" title="Permanent link">&para;</a></h3>
<p>The NEXUS Class Library is a C++ library for parsing NEXUS files.</p>
<p><em>homepage</em>: <a href="https://ncl.sourceforge.net/">https://ncl.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.18</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="nf-core-mag">nf-core-mag<a class="headerlink" href="#nf-core-mag" title="Permanent link">&para;</a></h3>
<p>The Nextflow pipeline 'mag' ported to EasyBuild/EESSI.</p>
<p><em>homepage</em>: <a href="https://nf-co.re/mag">https://nf-co.re/mag</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20221110</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nfft">NFFT<a class="headerlink" href="#nfft" title="Permanent link">&para;</a></h3>
<p>The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine  library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more  dimensions, of arbitrary input size, and of complex data.</p>
<p><em>homepage</em>: <a href="https://www-user.tu-chemnitz.de/~potts/nfft/">https://www-user.tu-chemnitz.de/~potts/nfft/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.3</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>foss/2020a</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.5.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nghttp2">nghttp2<a class="headerlink" href="#nghttp2" title="Permanent link">&para;</a></h3>
<p>This is an implementation of the Hypertext Transfer Protocol version 2 in C.  The framing layer of HTTP/2 is implemented as a reusable C library.  On top of that, we have implemented an HTTP/2 client, server and proxy.  We have also developed load test and benchmarking tools for HTTP/2.  An HPACK encoder and decoder are available as a public API.</p>
<p><em>homepage</em>: <a href="https://github.com/nghttp2/nghttp2">https://github.com/nghttp2/nghttp2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.48.0</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nghttp3">nghttp3<a class="headerlink" href="#nghttp3" title="Permanent link">&para;</a></h3>
<p>nghttp3 is an implementation of RFC 9114 HTTP/3  mapping over QUIC and RFC 9204 QPACK in C. It does not depend on any particular QUIC transport implementation.</p>
<p><em>homepage</em>: <a href="https://github.com/ngtcp2/nghttp3">https://github.com/ngtcp2/nghttp3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ngless">NGLess<a class="headerlink" href="#ngless" title="Permanent link">&para;</a></h3>
<p>NGLess is a domain-specific language for NGS (next-generation sequencing data) processing.</p>
<p><em>homepage</em>: <a href="https://ngless.embl.de/">https://ngless.embl.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-static-Linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nglview">nglview<a class="headerlink" href="#nglview" title="Permanent link">&para;</a></h3>
<p>IPython widget to interactively view molecular structures and trajectories.</p>
<p><em>homepage</em>: <a href="https://github.com/arose/nglview">https://github.com/arose/nglview</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.7.7</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="ngs">NGS<a class="headerlink" href="#ngs" title="Permanent link">&para;</a></h3>
<p>NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/ngs">https://github.com/ncbi/ngs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.5</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.9.3</code></td>
<td><code>-Java-1.8</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Java-1.8</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.10.4</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.10.5</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.10.9</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.11.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ngs-python">NGS-Python<a class="headerlink" href="#ngs-python" title="Permanent link">&para;</a></h3>
<p>NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/ngs">https://github.com/ncbi/ngs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.9.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.10.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="ngsadmix">NGSadmix<a class="headerlink" href="#ngsadmix" title="Permanent link">&para;</a></h3>
<p>NGSadmix is a tool for finding admixture proportions  from NGS data, based on genotype likelihoods.</p>
<p><em>homepage</em>: <a href="http://www.popgen.dk/software/index.php/NgsAdmix">http://www.popgen.dk/software/index.php/NgsAdmix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>32</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
</tbody>
</table>
<h3 id="ngspeciesid">NGSpeciesID<a class="headerlink" href="#ngspeciesid" title="Permanent link">&para;</a></h3>
<p>NGSpeciesID is a tool for clustering and consensus forming of targeted ONT reads.</p>
<p><em>homepage</em>: <a href="https://github.com/ksahlin/NGSpeciesID">https://github.com/ksahlin/NGSpeciesID</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.1.2.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="ngspice">ngspice<a class="headerlink" href="#ngspice" title="Permanent link">&para;</a></h3>
<p>Ngspice is a mixed-level/mixed-signal circuit simulator. Its code  is based on three open source software packages: Spice3f5, Cider1b1  and Xspice. </p>
<p><em>homepage</em>: <a href="https://ngspice.sourceforge.net">https://ngspice.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>31</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="ngtcp2">ngtcp2<a class="headerlink" href="#ngtcp2" title="Permanent link">&para;</a></h3>
<p>'Call it TCP/2. One More Time.'  ngtcp2 project is an effort to implement RFC9000 QUIC protocol.</p>
<p><em>homepage</em>: <a href="https://github.com/ngtcp2/ngtcp2">https://github.com/ngtcp2/ngtcp2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nibabel">NiBabel<a class="headerlink" href="#nibabel" title="Permanent link">&para;</a></h3>
<p>NiBabel provides read/write access to some common medical and neuroimaging file formats,  including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT  as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and  annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI.</p>
<p><em>homepage</em>: <a href="https://nipy.github.io/nibabel">https://nipy.github.io/nibabel</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-2.7.11-freetype-2.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nichenetr">nichenetr<a class="headerlink" href="#nichenetr" title="Permanent link">&para;</a></h3>
<p>R implementation of the NicheNet method,  to predict active ligand-target links between interacting cells</p>
<p><em>homepage</em>: <a href="https://github.com/saeyslab/nichenetr">https://github.com/saeyslab/nichenetr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1-20230223</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nifti">NIfTI<a class="headerlink" href="#nifti" title="Permanent link">&para;</a></h3>
<p>Niftilib is a set of i/o libraries for reading and writing files in the nifti-1 data format.</p>
<p><em>homepage</em>: <a href="http://niftilib.sourceforge.net/">http://niftilib.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nifti2dicom">nifti2dicom<a class="headerlink" href="#nifti2dicom" title="Permanent link">&para;</a></h3>
<p>Nifti2Dicom is a conversion tool that converts 3D NIfTI files (and other formats supported by ITK, including Analyze, MetaImage Nrrd and VTK) to DICOM. Unlike other conversion tools, it can import a DICOM file that is used to import the patient and study DICOM tags, and allows you to edit the accession number and other DICOM tags, in order to create a valid DICOM that can be imported in a PACS.</p>
<p><em>homepage</em>: <a href="https://github.com/biolab-unige/nifti2dicom">https://github.com/biolab-unige/nifti2dicom</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.11</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="nilearn">Nilearn<a class="headerlink" href="#nilearn" title="Permanent link">&para;</a></h3>
<p>Nilearn is a Python module for fast and easy statistical learning on NeuroImaging data.</p>
<p><em>homepage</em>: <a href="https://nilearn.github.io/">https://nilearn.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.2</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.5.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.5.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="nim">Nim<a class="headerlink" href="#nim" title="Permanent link">&para;</a></h3>
<p>Nim is a systems and applications programming language.</p>
<p><em>homepage</em>: <a href="https://nim-lang.org/">https://nim-lang.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>0.19.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.4.6</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.4.8</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.6.6</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nimble">NIMBLE<a class="headerlink" href="#nimble" title="Permanent link">&para;</a></h3>
<p>NIMBLE is a system for building and sharing analysis methods for statistical models,  especially for hierarchical models and computationally-intensive methods.</p>
<p><em>homepage</em>: <a href="https://r-nimble.org">https://r-nimble.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="ninja">Ninja<a class="headerlink" href="#ninja" title="Permanent link">&para;</a></h3>
<p>Ninja is a small build system with a focus on speed.</p>
<p><em>homepage</em>: <a href="https://ninja-build.org/">https://ninja-build.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.2</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.11.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nipype">Nipype<a class="headerlink" href="#nipype" title="Permanent link">&para;</a></h3>
<p>Nipype is a Python project that provides a uniform interface to existing neuroimaging software and  facilitates interaction between these packages within a single workflow.</p>
<p><em>homepage</em>: <a href="https://nipype.readthedocs.io">https://nipype.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.8.5</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="nlmpy">NLMpy<a class="headerlink" href="#nlmpy" title="Permanent link">&para;</a></h3>
<p>NLMpy is a Python package for the creation of neutral landscape models that are widely used  in the modelling of ecological patterns and processes across landscapes.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/nlmpy">https://pypi.org/project/nlmpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="nlohmann_json">nlohmann_json<a class="headerlink" href="#nlohmann_json" title="Permanent link">&para;</a></h3>
<p>JSON for Modern C++</p>
<p><em>homepage</em>: <a href="https://github.com/nlohmann/json">https://github.com/nlohmann/json</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.10.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.10.4</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.10.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.11.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nlopt">NLopt<a class="headerlink" href="#nlopt" title="Permanent link">&para;</a></h3>
<p>NLopt is a free/open-source library for nonlinear optimization,  providing a common interface for a number of different free optimization routines  available online as well as original implementations of various other algorithms. </p>
<p><em>homepage</em>: <a href="http://ab-initio.mit.edu/wiki/index.php/NLopt">http://ab-initio.mit.edu/wiki/index.php/NLopt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nltk">NLTK<a class="headerlink" href="#nltk" title="Permanent link">&para;</a></h3>
<p>NLTK is a leading platform for building Python programs to work with human language data.</p>
<p><em>homepage</em>: <a href="https://www.nltk.org/">https://www.nltk.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.2.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>3.6.7</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.7</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="nnu-net">nnU-Net<a class="headerlink" href="#nnu-net" title="Permanent link">&para;</a></h3>
<p>nnU-Net is the first segmentation method that is designed to deal with the dataset diversity found  in the domain It condenses and automates the keys decisions for designing a successful segmentation pipeline for  any given dataset.</p>
<p><em>homepage</em>: <a href="https://github.com/MIC-DKFZ/nnUNet">https://github.com/MIC-DKFZ/nnUNet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="node-red">Node-RED<a class="headerlink" href="#node-red" title="Permanent link">&para;</a></h3>
<p>Node-RED is a programming tool for wiring together hardware devices, APIs and online services  in new and interesting ways.</p>
<p><em>homepage</em>: <a href="http://nodered.org">http://nodered.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.2</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="nodejs">nodejs<a class="headerlink" href="#nodejs" title="Permanent link">&para;</a></h3>
<p>Node.js is a platform built on Chrome's JavaScript runtime   for easily building fast, scalable network applications. Node.js uses an   event-driven, non-blocking I/O model that makes it lightweight and efficient,   perfect for data-intensive real-time applications that run across distributed devices.</p>
<p><em>homepage</em>: <a href="http://nodejs.org">http://nodejs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.7</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.10.3</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>8.9.4</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>10.15.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>10.15.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>12.16.1</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>12.19.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>14.17.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>14.17.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>14.17.6</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>16.15.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>18.12.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="normaliz">Normaliz<a class="headerlink" href="#normaliz" title="Permanent link">&para;</a></h3>
<p>Normaliz is a open source tool for computations in affine monoids, vector configurations, rational polyhedra and rational cones. Normaliz now computes rational and algebraic polyhedra, i.e., polyhedra defined over real algebraic extensions of QQ.</p>
<p><em>homepage</em>: <a href="https://www.normaliz.uni-osnabrueck.de/">https://www.normaliz.uni-osnabrueck.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.3</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.7.4</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>3.8.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nose-parameterized">nose-parameterized<a class="headerlink" href="#nose-parameterized" title="Permanent link">&para;</a></h3>
<p>Parameterized testing with any Python test framework.</p>
<p><em>homepage</em>: <hmat://github.com/wolever/nose-parameterized></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="novoalign">novoalign<a class="headerlink" href="#novoalign" title="Permanent link">&para;</a></h3>
<p>NovoCraft is a software bundle.  NovoAlign:  Markets leading aligner with fully packed features designed for mapping  of short reads onto a reference genome from Illumina, Ion Torrent, 454,  and Color Spance NGS platforms.  NovoAlignCS:  Leading aligner packed with features designed to fully support reads from  ABI SOLiD Color Space.  NovoSort:  Custom designed multi-threaded sort/merge tools for BAM files.  NovoMethyl:  It can analyse a set of alignments to identify methylated cytosines.  WARNING!  You can only use the sofware without a license (with some features disabled)  if you are a non-profit organisation and use is for your own research or for  use by students as part of their course. A license is required for use where  these programs are part of a service where a third party is billed.</p>
<p><em>homepage</em>: <a href="http://www.novocraft.com/">http://www.novocraft.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.09.00</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.09.01</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="novoplasty">NOVOPlasty<a class="headerlink" href="#novoplasty" title="Permanent link">&para;</a></h3>
<p>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/ndierckx/NOVOPlasty">https://github.com/ndierckx/NOVOPlasty</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="npstat">npstat<a class="headerlink" href="#npstat" title="Permanent link">&para;</a></h3>
<p>npstat implements some population genetics tests and estimators that can be applied  to pooled sequences from Next Generation Sequencing experiments.</p>
<p><em>homepage</em>: <a href="https://code.google.com/archive/p/npstat/">https://code.google.com/archive/p/npstat/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="nrgljubljana">NRGLjubljana<a class="headerlink" href="#nrgljubljana" title="Permanent link">&para;</a></h3>
<p>NRG Ljubljana is an efficient implementation of the numerical renormalization group (NRG)   technique for solving quantum impurity problems that arise as simplified models of   magnetic impurities and as effective models in the dynamical mean field theory (DMFT)   approach to bulk correlated materials. </p>
<p><em>homepage</em>: <a href="http://nrgljubljana.ijs.si/">http://nrgljubljana.ijs.si/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.3.23</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="nsight-compute">Nsight-Compute<a class="headerlink" href="#nsight-compute" title="Permanent link">&para;</a></h3>
<p>NVIDIA Nsight Compute is an interactive kernel profiler for CUDA applications. It provides detailed performance metrics and API debugging via a user interface and a command line tool. In addition, its baseline feature allows users to compare results within the tool. Nsight Compute provides a customizable and data-driven user interface and fast metric collection, which can be extended with rules-based analysis scripts for post-processing results. The rules-based guided analysis helps isolate and fix memory throughput, compute, and occupancy inefficiencies in the target application.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/nsight-compute">https://developer.nvidia.com/nsight-compute</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nsight-systems">Nsight-Systems<a class="headerlink" href="#nsight-systems" title="Permanent link">&para;</a></h3>
<p>NVIDIA Nsight Systems is a system-wide performance analysis tool designed to visualize an applications algorithm, help you select the largest opportunities to optimize, and tune to scale efficiently across any quantity of CPUs and GPUs in your computer; from laptops to DGX servers.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/nsight-compute">https://developer.nvidia.com/nsight-compute</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.5.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nspr">NSPR<a class="headerlink" href="#nspr" title="Permanent link">&para;</a></h3>
<p>Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level  and libc-like functions.</p>
<p><em>homepage</em>: <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR">https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.20</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>4.21</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.25</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.29</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.30</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.32</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.34</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.35</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nss">NSS<a class="headerlink" href="#nss" title="Permanent link">&para;</a></h3>
<p>Network Security Services (NSS) is a set of libraries designed to support cross-platform development  of security-enabled client and server applications.</p>
<p><em>homepage</em>: <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS">https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.39</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.42.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.45</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.51</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.57</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.65</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.69</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.79</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.85</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nsync">nsync<a class="headerlink" href="#nsync" title="Permanent link">&para;</a></h3>
<p>nsync is a C library that exports various synchronization primitives, such as mutexes</p>
<p><em>homepage</em>: <a href="https://github.com/google/nsync">https://github.com/google/nsync</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.24.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.25.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ntcard">ntCard<a class="headerlink" href="#ntcard" title="Permanent link">&para;</a></h3>
<p>ntCard is a streaming algorithm for estimating the frequencies of k-mers in genomics datasets.</p>
<p><em>homepage</em>: <a href="https://www.bcgsc.ca/resources/software/ntcard">https://www.bcgsc.ca/resources/software/ntcard</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ntedit">ntEdit<a class="headerlink" href="#ntedit" title="Permanent link">&para;</a></h3>
<p>ntEdit is a fast and scalable genomics application for polishing genome assembly drafts.</p>
<p><em>homepage</em>: <a href="https://github.com/bcgsc/ntEdit">https://github.com/bcgsc/ntEdit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
</tbody>
</table>
<h3 id="nthits">ntHits<a class="headerlink" href="#nthits" title="Permanent link">&para;</a></h3>
<p>ntHits is a method for identifying repeats in high-throughput DNA sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/bcgsc/ntHits">https://github.com/bcgsc/ntHits</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1</code></td>
<td><code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
</tbody>
</table>
<h3 id="ntl">NTL<a class="headerlink" href="#ntl" title="Permanent link">&para;</a></h3>
<p>NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields.</p>
<p><em>homepage</em>: <a href="https://shoup.net/ntl/">https://shoup.net/ntl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.3.4</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="ntpoly">NTPoly<a class="headerlink" href="#ntpoly" title="Permanent link">&para;</a></h3>
<p>is a massively parallel library for computing the functions of sparse, symmetric matrices based on  polynomial expansions. For sufficiently sparse matrices, most of the matrix functions  in NTPoly can be computed in linear time.</p>
<p><em>homepage</em>: <a href="https://github.com/william-dawson/NTPoly">https://github.com/william-dawson/NTPoly</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.1</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="num2words">num2words<a class="headerlink" href="#num2words" title="Permanent link">&para;</a></h3>
<p>Modules to convert numbers to words. 42 --&gt; forty-two</p>
<p><em>homepage</em>: <a href="https://github.com/savoirfairelinux/num2words">https://github.com/savoirfairelinux/num2words</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.10</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="numactl">numactl<a class="headerlink" href="#numactl" title="Permanent link">&para;</a></h3>
<p>The numactl program allows you to run your application program on specific cpu's and memory nodes. It does this by supplying a NUMA memory policy to the operating system before running your program. The libnuma library provides convenient ways for you to add NUMA memory policies into your own program.</p>
<p><em>homepage</em>: <a href="http://oss.sgi.com/projects/libnuma/">http://oss.sgi.com/projects/libnuma/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.9</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>2.0.10</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.0.11</code></td>
<td><code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/5.2.0</code>, <code>GCC/5.3.0-2.26</code>, <code>GCC/5.4.0-2.26</code>, <code>GCC/6.1.0-2.27</code>, <code>GCC/6.2.0-2.27</code>, <code>GCC/6.3.0-2.27</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>iccifort/2016.3.210-GCC-4.9.3-2.25</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.12</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.13</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.14</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.0.16</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="numba">numba<a class="headerlink" href="#numba" title="Permanent link">&para;</a></h3>
<p>Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.</p>
<p><em>homepage</em>: <a href="https://numba.pydata.org/">https://numba.pydata.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.24.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.24.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.26.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.32.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.37.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.37.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.39.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.43.1</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.46.0</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.47.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.50.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.50.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.52.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.52.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.53.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.54.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>0.54.1</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.56.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.56.4</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="numdiff">numdiff<a class="headerlink" href="#numdiff" title="Permanent link">&para;</a></h3>
<p>Numdiff (which I will also write numdiff) is a little program that can be used to compare putatively  similar files line by line and field by field, ignoring small numeric differences or/and different numeric formats.  Equivalently, Numdiff is a program with the capability to appropriately compare files containing numerical fields  (and not only).</p>
<p><em>homepage</em>: <a href="https://www.nongnu.org/numdiff/">https://www.nongnu.org/numdiff/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.9.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="numexpr">numexpr<a class="headerlink" href="#numexpr" title="Permanent link">&para;</a></h3>
<p>The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.  It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into  code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a  compiler at runtime.</p>
<p><em>homepage</em>: <a href="https://numexpr.readthedocs.io/en/latest/">https://numexpr.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.6.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.6.5</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="numpy">numpy<a class="headerlink" href="#numpy" title="Permanent link">&para;</a></h3>
<p>NumPy is the fundamental package for scientific computing with Python. It contains among other things:  a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran  code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,  NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be   defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases.</p>
<p><em>homepage</em>: <a href="http://www.numpy.org">http://www.numpy.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.10.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.13.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="nvhpc">NVHPC<a class="headerlink" href="#nvhpc" title="Permanent link">&para;</a></h3>
<p>C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/hpc-sdk/">https://developer.nvidia.com/hpc-sdk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20.7</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20.9</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.7</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.9</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>21.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.7</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.9</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>22.11</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>23.1</code></td>
<td><code>-CUDA-12.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nvofbf">nvofbf<a class="headerlink" href="#nvofbf" title="Permanent link">&para;</a></h3>
<p>NVHPC based toolchain, including OpenMPI for MPI support, OpenBLAS (via FlexiBLAS for BLAS and LAPACK support), FFTW and ScaLAPACK.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2022.07</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nvompi">nvompi<a class="headerlink" href="#nvompi" title="Permanent link">&para;</a></h3>
<p>NVHPC based compiler toolchain, including OpenMPI for MPI support.</p>
<p><em>homepage</em>: &lt;(none)&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2022.07</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="nvshmem">NVSHMEM<a class="headerlink" href="#nvshmem" title="Permanent link">&para;</a></h3>
<p>NVSHMEM is a parallel programming interface based on OpenSHMEM that provides  efficient and scalable communication for NVIDIA GPU clusters. NVSHMEM creates a  global address space for data that spans the memory of multiple GPUs and can be  accessed with fine-grained GPU-initiated operations, CPU-initiated operations,  and operations on CUDA streams. </p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/nvshmem">https://developer.nvidia.com/nvshmem</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="nvtop">nvtop<a class="headerlink" href="#nvtop" title="Permanent link">&para;</a></h3>
<p>htop-like GPU usage monitor</p>
<p><em>homepage</em>: <a href="https://github.com/Syllo/nvtop">https://github.com/Syllo/nvtop</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="nwchem">NWChem<a class="headerlink" href="#nwchem" title="Permanent link">&para;</a></h3>
<p>NWChem aims to provide its users with computational chemistry tools that are scalable both in  their ability to treat large scientific computational chemistry problems efficiently, and in their use of available  parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.  NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all  combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties  and relativity.</p>
<p><em>homepage</em>: <a href="https://www.nwchem-sw.org">https://www.nwchem-sw.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.6.revision27746</code></td>
<td><code>-2015-10-20-Python-2.7.12</code></td>
<td><code>iomkl/2017a</code></td>
</tr>
<tr>
<td><code>6.6.revision27746</code></td>
<td><code>-2015-10-20-patches-20170814-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.8.revision47</code></td>
<td><code>-2017-12-14-Python-2.7.14</code></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>7.0.2</code></td>
<td></td>
<td><code>intel/2021a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>7.0.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="nxtrim">NxTrim<a class="headerlink" href="#nxtrim" title="Permanent link">&para;</a></h3>
<p>NxTrim is a software to remove Nextera Mate Pair junction adapters and categorise reads according to the orientation implied by the adapter location.</p>
<p><em>homepage</em>: <a href="https://github.com/sequencing/NxTrim">https://github.com/sequencing/NxTrim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h2 id="o">O<a class="headerlink" href="#o" title="Permanent link">&para;</a></h2>
<p><a href="#oases">Oases</a> - <a href="#obitools">OBITools</a> - <a href="#obitools3">OBITools3</a> - <a href="#ocaml">OCaml</a> - <a href="#occt">occt</a> - <a href="#oceanspy">oceanspy</a> - <a href="#ocnet">OCNet</a> - <a href="#octave">Octave</a> - <a href="#octopus-vcf">Octopus-vcf</a> - <a href="#ogdf">OGDF</a> - <a href="#olaflow">olaFlow</a> - <a href="#olego">olego</a> - <a href="#oma">OMA</a> - <a href="#omegafold">OmegaFold</a> - <a href="#onedrive">onedrive</a> - <a href="#onnx">ONNX</a> - <a href="#onnx-runtime">ONNX-Runtime</a> - <a href="#ont-fast5-api">ont-fast5-api</a> - <a href="#ont-remora">ont-remora</a> - <a href="#oompa">OOMPA</a> - <a href="#opari2">OPARI2</a> - <a href="#open-data-cube-core">Open-Data-Cube-Core</a> - <a href="#openai-gym">OpenAI-Gym</a> - <a href="#openbabel">OpenBabel</a> - <a href="#openblas">OpenBLAS</a> - <a href="#opencarp">openCARP</a> - <a href="#opencensus-python">OpenCensus-python</a> - <a href="#opencoarrays">OpenCoarrays</a> - <a href="#opencolorio">OpenColorIO</a> - <a href="#opencv">OpenCV</a> - <a href="#openexr">OpenEXR</a> - <a href="#openface">OpenFace</a> - <a href="#openfast">OpenFAST</a> - <a href="#openfoam">OpenFOAM</a> - <a href="#openfoam-extend">OpenFOAM-Extend</a> - <a href="#openfold">OpenFold</a> - <a href="#openforcefield">OpenForceField</a> - <a href="#openimageio">OpenImageIO</a> - <a href="#openjpeg">OpenJPEG</a> - <a href="#openkim-api">OpenKIM-API</a> - <a href="#openkim-models">openkim-models</a> - <a href="#openmm">OpenMM</a> - <a href="#openmm-plumed">OpenMM-PLUMED</a> - <a href="#openmmtools">OpenMMTools</a> - <a href="#openmolcas">OpenMolcas</a> - <a href="#openmpi">OpenMPI</a> - <a href="#openms">OpenMS</a> - <a href="#opennlp">OpenNLP</a> - <a href="#openpgm">OpenPGM</a> - <a href="#openpiv">OpenPIV</a> - <a href="#openpyxl">openpyxl</a> - <a href="#openrefine">OpenRefine</a> - <a href="#openscenegraph">OpenSceneGraph</a> - <a href="#opensees">OpenSees</a> - <a href="#openslide">OpenSlide</a> - <a href="#openslide-python">openslide-python</a> - <a href="#openssl">OpenSSL</a> - <a href="#openstackclient">OpenStackClient</a> - <a href="#opera">OPERA</a> - <a href="#opera-ms">OPERA-MS</a> - <a href="#optitype">OptiType</a> - <a href="#optix">OptiX</a> - <a href="#optuna">Optuna</a> - <a href="#or-tools">OR-Tools</a> - <a href="#orca">ORCA</a> - <a href="#orffinder">ORFfinder</a> - <a href="#orfm">OrfM</a> - <a href="#orthofinder">OrthoFinder</a> - <a href="#orthomcl">OrthoMCL</a> - <a href="#osi">Osi</a> - <a href="#ospray">OSPRay</a> - <a href="#osu-micro-benchmarks">OSU-Micro-Benchmarks</a> - <a href="#otf2">OTF2</a> - <a href="#ovito">OVITO</a> - <a href="#owncloud">ownCloud</a> - <a href="#oxford_asl">oxford_asl</a></p>
<h3 id="oases">Oases<a class="headerlink" href="#oases" title="Permanent link">&para;</a></h3>
<p>Oases is a de novo transcriptome assembler designed to produce transcripts from  short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly.</p>
<p><em>homepage</em>: <a href="http://www.ebi.ac.uk/~zerbino/oases/">http://www.ebi.ac.uk/~zerbino/oases/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.08</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.2.08</code></td>
<td><code>-kmer_101</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="obitools">OBITools<a class="headerlink" href="#obitools" title="Permanent link">&para;</a></h3>
<p>The OBITools programs aims to help you to manipulate various data and sequence files in a  convenient way using the Unix command line interface. They follow the standard Unix interface for command line  program, allowing to chain a set of commands using the pipe mecanism.</p>
<p><em>homepage</em>: <a href="http://www.grenoble.prabi.fr/trac/OBITools">http://www.grenoble.prabi.fr/trac/OBITools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.9</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.2.13</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="obitools3">OBITools3<a class="headerlink" href="#obitools3" title="Permanent link">&para;</a></h3>
<p>A package for the management of analyses and data in DNA metabarcoding.</p>
<p><em>homepage</em>: <a href="https://metabarcoding.org/obitools3">https://metabarcoding.org/obitools3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1b8</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ocaml">OCaml<a class="headerlink" href="#ocaml" title="Permanent link">&para;</a></h3>
<p>OCaml is a general purpose industrial-strength programming language  with an emphasis on expressiveness and safety. Developed for more than 20 years at Inria  it benefits from one of the most advanced type systems and supports functional,  imperative and object-oriented styles of programming.</p>
<p><em>homepage</em>: <a href="http://ocaml.org/">http://ocaml.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.02.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.07.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="occt">occt<a class="headerlink" href="#occt" title="Permanent link">&para;</a></h3>
<p>Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications.</p>
<p><em>homepage</em>: <a href="https://www.opencascade.com/">https://www.opencascade.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.3.0p4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>7.5.0p1</code></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="oceanspy">oceanspy<a class="headerlink" href="#oceanspy" title="Permanent link">&para;</a></h3>
<p>OceanSpy - A Python package to facilitate ocean model data analysis  and visualization.</p>
<p><em>homepage</em>: <a href="https://github.com/hainegroup/oceanspy">https://github.com/hainegroup/oceanspy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ocnet">OCNet<a class="headerlink" href="#ocnet" title="Permanent link">&para;</a></h3>
<p>Generate and analyze Optimal Channel Networks (OCNs): oriented spanning trees reproducing all scaling features characteristic of real, natural river networks. As such, they can be used in a variety of numerical experiments in the fields of hydrology, ecology and epidemiology.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/OCNet">https://cran.r-project.org/web/packages/OCNet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="octave">Octave<a class="headerlink" href="#octave" title="Permanent link">&para;</a></h3>
<p>GNU Octave is a high-level interpreted language, primarily intended for numerical computations.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/octave/">https://www.gnu.org/software/octave/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>-mt</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.4.1</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>foss/2019b</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>7.1.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="octopus-vcf">Octopus-vcf<a class="headerlink" href="#octopus-vcf" title="Permanent link">&para;</a></h3>
<p>Octopus is a mapping-based variant caller that implements several calling models within  a unified haplotype-aware framework. Octopus takes inspiration from particle filtering by constructing a  tree of haplotypes and dynamically pruning and extending the tree based on haplotype posterior probabilities  in a sequential manner. This allows octopus to implicitly consider all possible haplotypes at a given loci in  reasonable time.</p>
<p><em>homepage</em>: <a href="https://github.com/luntergroup/octopus">https://github.com/luntergroup/octopus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.7.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ogdf">OGDF<a class="headerlink" href="#ogdf" title="Permanent link">&para;</a></h3>
<p>OGDF is a self-contained C++ library for graph algorithms, in particular for (but not restricted to) automatic graph drawing. It offers sophisticated algorithms and data structures to use within your own applications or scientific projects.</p>
<p><em>homepage</em>: <a href="https://ogdf.uos.de/">https://ogdf.uos.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>dogwood-202202</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="olaflow">olaFlow<a class="headerlink" href="#olaflow" title="Permanent link">&para;</a></h3>
<p>olaFlow CFD Suite is a free and open source project committed to bringing the latest advances for the simulation of wave dynamics to the OpenFOAM and FOAM-extend communities.</p>
<p><em>homepage</em>: <a href="https://olaflow.github.io/">https://olaflow.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210820</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="olego">olego<a class="headerlink" href="#olego" title="Permanent link">&para;</a></h3>
<p>OLego is a program specifically designed for de novo spliced mapping of mRNA-seq reads. OLego adopts a seed-and-extend scheme, and does not rely on a separate external mapper.</p>
<p><em>homepage</em>: <a href="https://github.com/chaolinzhanglab/olego">https://github.com/chaolinzhanglab/olego</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.9</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="oma">OMA<a class="headerlink" href="#oma" title="Permanent link">&para;</a></h3>
<p>The OMA ('Orthologous MAtrix') project is a method and database for the inference   of orthologs among complete genomes</p>
<p><em>homepage</em>: <a href="https://omabrowser.org/standalone/">https://omabrowser.org/standalone/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="omegafold">OmegaFold<a class="headerlink" href="#omegafold" title="Permanent link">&para;</a></h3>
<p>OmegaFold: High-resolution de novo Structure Prediction from Primary Sequence</p>
<p><em>homepage</em>: <a href="https://github.com/HeliXonProtein/OmegaFold">https://github.com/HeliXonProtein/OmegaFold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="onedrive">onedrive<a class="headerlink" href="#onedrive" title="Permanent link">&para;</a></h3>
<p>A free Microsoft OneDrive Client which supports OneDrive Personal, OneDrive for Business, OneDrive for Office365 and SharePoint.</p>
<p><em>homepage</em>: <a href="https://abraunegg.github.io/">https://abraunegg.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.4.21</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="onnx">ONNX<a class="headerlink" href="#onnx" title="Permanent link">&para;</a></h3>
<p>Open Neural Network Exchange (ONNX) is an open ecosystem that empowers AI developers to choose the right tools as their project evolves. ONNX provides an open source format for AI models, both deep learning and traditional ML. It defines an extensible computation graph model, as well as definitions of built-in operators and standard data types. Currently we focus on the capabilities needed for inferencing (scoring).</p>
<p><em>homepage</em>: <a href="https://onnx.ai/">https://onnx.ai/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="onnx-runtime">ONNX-Runtime<a class="headerlink" href="#onnx-runtime" title="Permanent link">&para;</a></h3>
<p>ONNX Runtime inference can enable faster customer experiences and lower costs, supporting models from deep learning frameworks such as PyTorch and TensorFlow/Keras as well as classical machine learning libraries such as scikit-learn, LightGBM, XGBoost, etc. ONNX Runtime is compatible with different hardware, drivers, and operating systems, and provides optimal performance by leveraging hardware accelerators where applicable alongside graph optimizations and transforms.</p>
<p><em>homepage</em>: <a href="https://onnxruntime.ai/">https://onnxruntime.ai/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="ont-fast5-api">ont-fast5-api<a class="headerlink" href="#ont-fast5-api" title="Permanent link">&para;</a></h3>
<p>ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore .fast5 file format.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/ont_fast5_api">https://github.com/nanoporetech/ont_fast5_api</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="ont-remora">ont-remora<a class="headerlink" href="#ont-remora" title="Permanent link">&para;</a></h3>
<p>Methylation/modified base calling separated from basecalling. Remora primarily provides an API to call modified bases for basecaller programs such as Bonito. Remora also provides the tools to prepare datasets, train modified base models and run simple inference.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/remora">https://github.com/nanoporetech/remora</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.1.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="oompa">OOMPA<a class="headerlink" href="#oompa" title="Permanent link">&para;</a></h3>
<p>OOMPA is a suite of R packages for the analysis of gene expression (RNA), proteomics profiling,  and other high throughput molecular biology data.</p>
<p><em>homepage</em>: <a href="https://r-forge.r-project.org/projects/oompa">https://r-forge.r-project.org/projects/oompa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="opari2">OPARI2<a class="headerlink" href="#opari2" title="Permanent link">&para;</a></h3>
<p>OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a  source-to-source instrumentation tool for OpenMP and hybrid codes.  It surrounds OpenMP directives and runtime library calls with calls  to the POMP2 measurement interface. </p>
<p><em>homepage</em>: <a href="https://www.score-p.org">https://www.score-p.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.0.5</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.7</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="open-data-cube-core">Open-Data-Cube-Core<a class="headerlink" href="#open-data-cube-core" title="Permanent link">&para;</a></h3>
<p>The Open Data Cube Core provides an integrated gridded data analysis environment for decades of analysis ready earth observation satellite and related data from multiple satellite and other acquisition systems.</p>
<p><em>homepage</em>: <a href="https://github.com/opendatacube/datacube-core">https://github.com/opendatacube/datacube-core</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openai-gym">OpenAI-Gym<a class="headerlink" href="#openai-gym" title="Permanent link">&para;</a></h3>
<p>A toolkit for developing and comparing reinforcement learning algorithms.</p>
<p><em>homepage</em>: <a href="https://gym.openai.com">https://gym.openai.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.21.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="openbabel">OpenBabel<a class="headerlink" href="#openbabel" title="Permanent link">&para;</a></h3>
<p>Open Babel is a chemical toolbox designed to speak the many  languages of chemical data. It's an open, collaborative project allowing anyone  to search, convert, analyze, or store data from molecular modeling, chemistry,  solid-state materials, biochemistry, or related areas.</p>
<p><em>homepage</em>: <a href="https://openbabel.org">https://openbabel.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>iimpi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openblas">OpenBLAS<a class="headerlink" href="#openblas" title="Permanent link">&para;</a></h3>
<p>OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.</p>
<p><em>homepage</em>: <a href="https://xianyi.github.com/OpenBLAS/">https://xianyi.github.com/OpenBLAS/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.9</code></td>
<td><code>-LAPACK-3.5.0</code></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>0.2.12</code></td>
<td><code>-LAPACK-3.5.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>0.2.13</code></td>
<td><code>-LAPACK-3.5.0</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>0.2.14</code></td>
<td><code>-LAPACK-3.5.0</code></td>
<td><code>GNU/4.9.2-2.25</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>0.2.15</code></td>
<td><code>-LAPACK-3.6.0</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>0.2.18</code></td>
<td><code>-LAPACK-3.6.0</code></td>
<td><code>GCC/4.9.4-2.25</code>, <code>GCC/5.3.0-2.26</code>, <code>GCC/5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>0.2.18</code></td>
<td><code>-LAPACK-3.6.1</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>gompi/2016.07</code></td>
</tr>
<tr>
<td><code>0.2.19</code></td>
<td><code>-LAPACK-3.6.1</code></td>
<td><code>gompi/2016.09</code></td>
</tr>
<tr>
<td><code>0.2.19</code></td>
<td><code>-LAPACK-3.7.0</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCC/6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>0.2.20</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCC/6.4.0-2.28</code>, <code>GCC/7.2.0-2.29</code></td>
</tr>
<tr>
<td><code>0.3.0</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>0.3.3</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.3.4</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.3.5</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.3.6</code></td>
<td></td>
<td><code>GCC/8.3.0-2.32</code></td>
</tr>
<tr>
<td><code>0.3.7</code></td>
<td></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>0.3.8</code></td>
<td></td>
<td><code>GCC/9.2.0</code></td>
</tr>
<tr>
<td><code>0.3.9</code></td>
<td></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>0.3.12</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.3.15</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>0.3.17</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>0.3.18</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>0.3.20</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>NVHPC/22.7-CUDA-11.7.0</code></td>
</tr>
<tr>
<td><code>0.3.21</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="opencarp">openCARP<a class="headerlink" href="#opencarp" title="Permanent link">&para;</a></h3>
<p>openCARP is an open cardiac electrophysiology simulator for in-silico experiments.</p>
<p><em>homepage</em>: <a href="https://opencarp.org">https://opencarp.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>8.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="opencensus-python">OpenCensus-python<a class="headerlink" href="#opencensus-python" title="Permanent link">&para;</a></h3>
<p>OpenCensus for Python. OpenCensus provides a framework to measure a server's resource usage and collect performance stats. This repository contains Python related utilities and supporting software needed by OpenCensus.</p>
<p><em>homepage</em>: <a href="https://opencensus.io/">https://opencensus.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="opencoarrays">OpenCoarrays<a class="headerlink" href="#opencoarrays" title="Permanent link">&para;</a></h3>
<p>OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran.</p>
<p><em>homepage</em>: <a href="https://github.com/sourceryinstitute/opencoarrays">https://github.com/sourceryinstitute/opencoarrays</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.0</code></td>
<td><code>gompi/2017a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>gompi/2018b</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.9.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="opencolorio">OpenColorIO<a class="headerlink" href="#opencolorio" title="Permanent link">&para;</a></h3>
<p>OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production  with an emphasis on visual effects and computer animation.</p>
<p><em>homepage</em>: <a href="http://opencolorio.org/">http://opencolorio.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="opencv">OpenCV<a class="headerlink" href="#opencv" title="Permanent link">&para;</a></h3>
<p>OpenCV (Open Source Computer Vision Library) is an open source computer vision  and machine learning software library. OpenCV was built to provide  a common infrastructure for computer vision applications and to accelerate  the use of machine perception in the commercial products.  Includes extra modules for OpenCV from the contrib repository.</p>
<p><em>homepage</em>: <a href="https://opencv.org/">https://opencv.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.12</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-Python-3.6.4-contrib</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.4.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.4.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>3.4.7</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-3.8.2-contrib</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.5.1</code></td>
<td><code>-contrib</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td><code>-CUDA-11.3.1-contrib</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td><code>-contrib</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.5.5</code></td>
<td><code>-CUDA-11.4.1-contrib</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.5.5</code></td>
<td><code>-contrib</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-CUDA-11.7.0-contrib</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-contrib</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="openexr">OpenEXR<a class="headerlink" href="#openexr" title="Permanent link">&para;</a></h3>
<p>OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light &amp; Magic  for use in computer imaging applications</p>
<p><em>homepage</em>: <a href="https://www.openexr.com/">https://www.openexr.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.5.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openface">OpenFace<a class="headerlink" href="#openface" title="Permanent link">&para;</a></h3>
<p>OpenFace  a state-of-the art tool intended for facial landmark detection, head pose estimation, facial action unit recognition, and eye-gaze estimation.</p>
<p><em>homepage</em>: <a href="https://github.com/TadasBaltrusaitis/OpenFace">https://github.com/TadasBaltrusaitis/OpenFace</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="openfast">OpenFAST<a class="headerlink" href="#openfast" title="Permanent link">&para;</a></h3>
<p>OpenFAST is a wind turbine simulation tool which builds on FAST v8. FAST.Farm extends the capability of OpenFAST to simulate multi-turbine wind farms</p>
<p><em>homepage</em>: <a href="http://openfast.readthedocs.io/">http://openfast.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="openfoam">OpenFOAM<a class="headerlink" href="#openfoam" title="Permanent link">&para;</a></h3>
<p>OpenFOAM is a free, open source CFD software package.  OpenFOAM has an extensive range of features to solve anything from complex fluid flows  involving chemical reactions, turbulence and heat transfer,  to solid dynamics and electromagnetics.</p>
<p><em>homepage</em>: <a href="https://www.openfoam.com/">https://www.openfoam.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>intel/2016a</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.x</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td></td>
<td><code>intel/2017a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>intel/2017a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td></td>
<td><code>intel/2016b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.0</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.x-20170904</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0-20180108</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>foss/2019b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>7</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>7</code></td>
<td><code>-20200508</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>8</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code></td>
</tr>
<tr>
<td><code>8</code></td>
<td><code>-20210316</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>9</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>10</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>v1606+</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>v1612+</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>v1712</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>v1806</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>v1812</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>v1906</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>v1912</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>v2006</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>v2012</code></td>
<td></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>v2106</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>v2112</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>v2206</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="openfoam-extend">OpenFOAM-Extend<a class="headerlink" href="#openfoam-extend" title="Permanent link">&para;</a></h3>
<p>OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.</p>
<p><em>homepage</em>: <a href="http://www.extend-project.de/">http://www.extend-project.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1</code></td>
<td></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td></td>
<td><code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.0</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.0</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.1-20191120</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.1-20200408</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="openfold">OpenFold<a class="headerlink" href="#openfold" title="Permanent link">&para;</a></h3>
<p>A faithful PyTorch reproduction of DeepMind's AlphaFold 2</p>
<p><em>homepage</em>: <a href="https://github.com/aqlaboratory/openfold">https://github.com/aqlaboratory/openfold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="openforcefield">OpenForceField<a class="headerlink" href="#openforcefield" title="Permanent link">&para;</a></h3>
<p>Simulation and Parameter Estimation in Geophysics  - A python package for simulation and gradient based parameter estimation in the context of geophysical applications.</p>
<p><em>homepage</em>: <a href="https://github.com/openforcefield/openforcefield/releases">https://github.com/openforcefield/openforcefield/releases</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openimageio">OpenImageIO<a class="headerlink" href="#openimageio" title="Permanent link">&para;</a></h3>
<p>OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities,  and applications.</p>
<p><em>homepage</em>: <a href="https://openimageio.org/">https://openimageio.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.17</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.7.17</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.8.16</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.0.12</code></td>
<td><code>gompi/2019b</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>2.1.12.0</code></td>
<td><code>gompi/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>2.3.17.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openjpeg">OpenJPEG<a class="headerlink" href="#openjpeg" title="Permanent link">&para;</a></h3>
<p>OpenJPEG is an open-source JPEG 2000 codec written in  C language. It has been developed in order to promote the use of JPEG 2000,  a still-image compression standard from the Joint Photographic Experts Group  (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as  a JPEG 2000 Reference Software.</p>
<p><em>homepage</em>: <a href="https://www.openjpeg.org/">https://www.openjpeg.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openkim-api">OpenKIM-API<a class="headerlink" href="#openkim-api" title="Permanent link">&para;</a></h3>
<p>Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models have to be installed by the user by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  </p>
<p><em>homepage</em>: <a href="https://openkim.org/">https://openkim.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.2</code></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.9.7</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="openkim-models">openkim-models<a class="headerlink" href="#openkim-models" title="Permanent link">&para;</a></h3>
<p>Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild installs the models.  The API itself is in the kim-api package.  </p>
<p><em>homepage</em>: <a href="https://openkim.org/">https://openkim.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190725</code></td>
<td><code>GCCcore/10.2.0</code>, <code>foss/2019a</code>, <code>foss/2019b</code>, <code>intel/2019a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>20210128</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openmm">OpenMM<a class="headerlink" href="#openmm" title="Permanent link">&para;</a></h3>
<p>OpenMM is a toolkit for molecular simulation.</p>
<p><em>homepage</em>: <a href="https://openmm.org">https://openmm.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.1.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>7.4.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>7.4.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7.5.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>7.5.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>fosscuda/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>7.5.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>7.5.1</code></td>
<td><code>-CUDA-11.3.1-DeepMind-patch</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>7.5.1</code></td>
<td><code>-DeepMind-patch</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>7.7.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>7.7.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="openmm-plumed">OpenMM-PLUMED<a class="headerlink" href="#openmm-plumed" title="Permanent link">&para;</a></h3>
<p>This project provides a connection between OpenMM and PLUMED. It allows you to bias or analyze an OpenMM simulation based on collective variables.</p>
<p><em>homepage</em>: <a href="https://github.com/openmm/openmm-plumed/">https://github.com/openmm/openmm-plumed/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openmmtools">OpenMMTools<a class="headerlink" href="#openmmtools" title="Permanent link">&para;</a></h3>
<p>A batteries-included toolkit for the GPU-accelerated OpenMM molecular simulation engine. openmmtools is a Python library layer that sits on top of OpenMM to provide access to a variety of useful tools  for building full-featured molecular simulation packages. </p>
<p><em>homepage</em>: <a href="https://github.com/choderalab/openmmtools">https://github.com/choderalab/openmmtools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.20.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openmolcas">OpenMolcas<a class="headerlink" href="#openmolcas" title="Permanent link">&para;</a></h3>
<p>OpenMolcas is a quantum chemistry software package.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/Molcas/OpenMolcas">https://gitlab.com/Molcas/OpenMolcas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>18.09</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>20.10</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>20.10</code></td>
<td><code>-Python-3.8.2-noGA</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>21.06</code></td>
<td></td>
<td><code>intel/2021a</code></td>
</tr>
<tr>
<td><code>22.06</code></td>
<td></td>
<td><code>intel/2022a</code></td>
</tr>
<tr>
<td><code>22.10</code></td>
<td></td>
<td><code>intel/2022a</code></td>
</tr>
<tr>
<td><code>22.10</code></td>
<td><code>-noGA</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="openmpi">OpenMPI<a class="headerlink" href="#openmpi" title="Permanent link">&para;</a></h3>
<p>The Open MPI Project is an open source MPI implementation.</p>
<p><em>homepage</em>: <a href="http://www.open-mpi.org/">http://www.open-mpi.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.5</code></td>
<td></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code>, <code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>1.6.5</code></td>
<td><code>-no-OFED</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td></td>
<td><code>GCC/4.8.3</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.8.5</code></td>
<td></td>
<td><code>GNU/4.9.2-2.25</code></td>
</tr>
<tr>
<td><code>1.8.6</code></td>
<td></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.8.8</code></td>
<td></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GCC/5.3.0-2.26</code>, <code>GCC/6.1.0-2.27</code>, <code>PGI/16.3-GCC-4.9.3-2.25</code>, <code>PGI/16.4-GCC-5.3.0-2.26</code></td>
</tr>
<tr>
<td><code>1.10.3</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCC/6.1.0-2.27</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>1.10.4</code></td>
<td></td>
<td><code>PGI/16.7-GCC-5.4.0-2.26</code>, <code>iccifort/2016.3.210-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>GCC/5.2.0</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td></td>
<td><code>GCC/6.2.0-2.27</code>, <code>iccifort/2017.1.132-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>GCC/6.3.0-2.27</code>, <code>iccifort/2017.1.132-GCC-6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-opa</code></td>
<td><code>GCC/6.3.0-2.27</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>GCC/6.3.0-2.28</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>gcccuda/2017b</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>gcccuda/2018a</code>, <code>iccifort/2018.1.163-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td></td>
<td><code>iccifort/2018.2.199-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>GCC/7.2.0-2.29</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>GCC/7.3.0-2.30</code>, <code>gcccuda/2018b</code>, <code>iccifort/2018.3.222-GCC-7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>gcccuda/2019a</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td></td>
<td><code>GCC/8.3.0</code>, <code>GCC/8.3.0-2.32</code>, <code>gcccuda/2019b</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>-hpcx</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td></td>
<td><code>GCC/8.3.0-2.32</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>GCC/9.2.0-2.32</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td></td>
<td><code>GCC/9.3.0</code>, <code>gcccuda/2020a</code>, <code>iccifort/2020.1.217</code>, <code>iccifortcuda/2020a</code></td>
</tr>
<tr>
<td><code>4.0.5</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code>, <code>gcccuda/2020b</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>4.0.5</code></td>
<td><code>-CUDA-11.2.1</code></td>
<td><code>NVHPC/21.2</code></td>
</tr>
<tr>
<td><code>4.0.6</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>4.0.7</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
<tr>
<td><code>4.1.2</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>4.1.4</code></td>
<td></td>
<td><code>GCC/11.3.0</code>, <code>GCC/12.2.0</code>, <code>NVHPC/22.7-CUDA-11.7.0</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
<tr>
<td><code>system</code></td>
<td></td>
<td><code>GCC/system-2.29</code></td>
</tr>
</tbody>
</table>
<h3 id="openms">OpenMS<a class="headerlink" href="#openms" title="Permanent link">&para;</a></h3>
<p>As part of the deNBI Center for integrative Bioinformatics, OpenMS offers an open-source software  C++ library (+ python bindings) for LC/MS data management and analyses. It provides an infrastructure  for the rapid development of mass spectrometry related software as well as a rich toolset built on top of it.</p>
<p><em>homepage</em>: <a href="https://www.openms.de/">https://www.openms.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="opennlp">OpenNLP<a class="headerlink" href="#opennlp" title="Permanent link">&para;</a></h3>
<p>The Apache OpenNLP library is a machine learning based toolkit for the processing of  natural language text.</p>
<p><em>homepage</em>: <a href="https://opennlp.apache.org/">https://opennlp.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="openpgm">OpenPGM<a class="headerlink" href="#openpgm" title="Permanent link">&para;</a></h3>
<p>OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM)  specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol  that enables receivers to detect loss, request retransmission of lost data, or notify an application  of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible  for ensuring all data is received, absolving the sender of reception responsibility.</p>
<p><em>homepage</em>: <a href="http://code.google.com/p/openpgm/">http://code.google.com/p/openpgm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.122</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="openpiv">OpenPIV<a class="headerlink" href="#openpiv" title="Permanent link">&para;</a></h3>
<p>OpenPIV is an open source Particle Image Velocimetry analysis software</p>
<p><em>homepage</em>: <a href="http://www.openpiv.net">http://www.openpiv.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.21.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="openpyxl">openpyxl<a class="headerlink" href="#openpyxl" title="Permanent link">&para;</a></h3>
<p>A Python library to read/write Excel 2010 xlsx/xlsm files</p>
<p><em>homepage</em>: <a href="https://openpyxl.readthedocs.io">https://openpyxl.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.0.7</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.9</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.0.10</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openrefine">OpenRefine<a class="headerlink" href="#openrefine" title="Permanent link">&para;</a></h3>
<p>OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it,  and augment it with data coming from the web.</p>
<p><em>homepage</em>: <a href="https://github.com/OpenRefine/OpenRefine">https://github.com/OpenRefine/OpenRefine</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="openscenegraph">OpenSceneGraph<a class="headerlink" href="#openscenegraph" title="Permanent link">&para;</a></h3>
<p>The OpenSceneGraph is an open source high performance 3D graphics toolkit,  used by application developers in fields such as visual simulation, games, virtual reality,  scientific visualization and modelling. Written entirely in Standard C++ and OpenGL it runs  on all Windows platforms, OSX, GNU/Linux, IRIX, Solaris, HP-Ux, AIX and FreeBSD operating  systems. The OpenSceneGraph is now well established as the world leading scene graph  technology, used widely in the vis-sim, space, scientific, oil-gas, games and virtual  reality industries.</p>
<p><em>homepage</em>: <a href="http://www.openscenegraph.org/">http://www.openscenegraph.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.5</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="opensees">OpenSees<a class="headerlink" href="#opensees" title="Permanent link">&para;</a></h3>
<p>Open System for Earthquake Engineering Simulation</p>
<p><em>homepage</em>: <a href="https://opensees.berkeley.edu/index.php">https://opensees.berkeley.edu/index.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.8.2-parallel</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="openslide">OpenSlide<a class="headerlink" href="#openslide" title="Permanent link">&para;</a></h3>
<p>OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).</p>
<p><em>homepage</em>: <a href="https://openslide.org/">https://openslide.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-largefiles</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openslide-python">openslide-python<a class="headerlink" href="#openslide-python" title="Permanent link">&para;</a></h3>
<p>OpenSlide Python is a Python interface to the OpenSlide library.</p>
<p><em>homepage</em>: <a href="https://github.com/openslide/openslide-python">https://github.com/openslide/openslide-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openssl">OpenSSL<a class="headerlink" href="#openssl" title="Permanent link">&para;</a></h3>
<p>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,  and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)  protocols as well as a full-strength general purpose cryptography library. </p>
<p><em>homepage</em>: <a href="https://www.openssl.org/">https://www.openssl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.0.1f</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.0.1k</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.0.1s</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0.2g</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
<tr>
<td><code>1.0.2h</code></td>
<td><code>foss/2016.04</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.1.0c</code></td>
<td><code>GCC/5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>1.1.0e</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.0h</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1b</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.1d</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1e</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1h</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.1.1k</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.1.1n</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.1.1q</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="openstackclient">OpenStackClient<a class="headerlink" href="#openstackclient" title="Permanent link">&para;</a></h3>
<p>OpenStackClient (aka OSC) is a command-line client for OpenStack that brings the command set for Compute, Identity, Image, Network, Object Store and Block        Storage APIs together in a single shell with a uniform command structure.</p>
<p><em>homepage</em>: <a href="https://docs.openstack.org/python-openstackclient/latest/">https://docs.openstack.org/python-openstackclient/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.5.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="opera">OPERA<a class="headerlink" href="#opera" title="Permanent link">&para;</a></h3>
<p>An optimal genome scaffolding program</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/operasf">https://sourceforge.net/projects/operasf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.6</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="opera-ms">OPERA-MS<a class="headerlink" href="#opera-ms" title="Permanent link">&para;</a></h3>
<p>OPERA-MS is a hybrid metagenomic assembler which combines the  advantages of short and long-read technologies to provide high quality  assemblies, addressing issues of low contiguity for short-read only assemblies,  and low base-pair quality for long-read only assemblies.</p>
<p><em>homepage</em>: <a href="https://github.com/CSB5/OPERA-MS">https://github.com/CSB5/OPERA-MS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0-20200802</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="optitype">OptiType<a class="headerlink" href="#optitype" title="Permanent link">&para;</a></h3>
<p>OptiType is a novel HLA genotyping algorithm based on integer linear programming,  capable of producing accurate 4-digit HLA genotyping predictions from NGS data  by simultaneously selecting all major and minor HLA Class I alleles. </p>
<p><em>homepage</em>: <a href="https://github.com/FRED-2/OptiType">https://github.com/FRED-2/OptiType</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="optix">OptiX<a class="headerlink" href="#optix" title="Permanent link">&para;</a></h3>
<p>OptiX is NVIDIA SDK for easy ray tracing performance.  It provides a simple framework for accessing the GPUs massive ray tracing  power using state-of-the-art GPU algorithms.</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/optix">https://developer.nvidia.com/optix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.9.0</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>6.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>7.2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="optuna">Optuna<a class="headerlink" href="#optuna" title="Permanent link">&para;</a></h3>
<p>Optuna is an automatic hyperparameter optimization software framework, particularly designed for machine learning. It features an imperative, define-by-run style user API. Thanks to our define-by-run API, the code written with Optuna enjoys high modularity, and the user of Optuna can dynamically construct the search spaces for the hyperparameters.</p>
<p><em>homepage</em>: <a href="https://optuna.org/">https://optuna.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="or-tools">OR-Tools<a class="headerlink" href="#or-tools" title="Permanent link">&para;</a></h3>
<p>Google Optimization Tools (a.k.a., OR-Tools) is an open-source, fast and portable software suite for solving combinatorial optimization problems.</p>
<p><em>homepage</em>: <a href="https://developers.google.com/optimization/">https://developers.google.com/optimization/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="orca">ORCA<a class="headerlink" href="#orca" title="Permanent link">&para;</a></h3>
<p>ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects.</p>
<p><em>homepage</em>: <a href="https://orcaforum.kofo.mpg.de">https://orcaforum.kofo.mpg.de</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3_0_2-linux_x86-64</code></td>
<td><code>-OpenMPI-1.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0.2</code></td>
<td><code>-OpenMPI-2.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-OpenMPI-2.0.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>-OpenMPI-3.1.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-static</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.0.1</code></td>
<td><code>-static</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-static</code></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>5.0.3</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>5.0.4</code></td>
<td></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="orffinder">ORFfinder<a class="headerlink" href="#orffinder" title="Permanent link">&para;</a></h3>
<p>ORF finder searches for open reading frames (ORFs) in the  DNA sequence you enter. The program returns the range of each ORF,  along with its protein translation. Use ORF finder to search newly  sequenced DNA for potential protein encoding segments, verify  predicted protein using newly developed SMART BLAST or regular BLASTP.</p>
<p><em>homepage</em>: <a href="https://www.ncbi.nlm.nih.gov/orffinder/">https://www.ncbi.nlm.nih.gov/orffinder/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="orfm">OrfM<a class="headerlink" href="#orfm" title="Permanent link">&para;</a></h3>
<p>A simple and not slow open reading frame (ORF) caller.</p>
<p><em>homepage</em>: <a href="https://github.com/wwood/OrfM">https://github.com/wwood/OrfM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="orthofinder">OrthoFinder<a class="headerlink" href="#orthofinder" title="Permanent link">&para;</a></h3>
<p>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics</p>
<p><em>homepage</em>: <a href="https://github.com/davidemms/OrthoFinder">https://github.com/davidemms/OrthoFinder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.3.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.3.11</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.5.4</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="orthomcl">OrthoMCL<a class="headerlink" href="#orthomcl" title="Permanent link">&para;</a></h3>
<p>OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences.</p>
<p><em>homepage</em>: <a href="http://orthomcl.org/">http://orthomcl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.9</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="osi">Osi<a class="headerlink" href="#osi" title="Permanent link">&para;</a></h3>
<p>Osi (Open Solver Interface) provides an abstract base class to a generic linear programming (LP) solver, along with derived classes for specific solvers. Many applications may be able to use the Osi to insulate themselves from a specific LP solver. That is, programs written to the OSI standard may be linked to any solver with an OSI interface and should produce correct results. The OSI has been significantly extended compared to its first incarnation. Currently, the OSI supports linear programming solvers and has rudimentary support for integer programming.</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/Osi">https://github.com/coin-or/Osi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.108.5</code></td>
<td><code>GCCcore/7.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.108.6</code></td>
<td><code>GCC/10.3.0</code>, <code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.108.7</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ospray">OSPRay<a class="headerlink" href="#ospray" title="Permanent link">&para;</a></h3>
<p>Open, Scalable, and Portable Ray Tracing Engine</p>
<p><em>homepage</em>: <a href="https://www.ospray.org">https://www.ospray.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="osu-micro-benchmarks">OSU-Micro-Benchmarks<a class="headerlink" href="#osu-micro-benchmarks" title="Permanent link">&para;</a></h3>
<p>OSU Micro-Benchmarks</p>
<p><em>homepage</em>: <a href="https://mvapich.cse.ohio-state.edu/benchmarks/">https://mvapich.cse.ohio-state.edu/benchmarks/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.3.2</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>5.6.2</code></td>
<td></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>5.6.3</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompic/2019b</code>, <code>gompic/2020a</code>, <code>iimpi/2019a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iimpi/2020b</code>, <code>iimpic/2019b</code>, <code>iimpic/2020a</code></td>
</tr>
<tr>
<td><code>5.7</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompic/2020b</code>, <code>iimpi/2020b</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td></td>
<td><code>ffmpi/4.5.0</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>iimpi/2021a</code>, <code>iompi/2021a</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td></td>
<td><code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td></td>
<td><code>gompi/2022.05</code>, <code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>5.9</code></td>
<td><code>-ROCm-4.5.0</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>6.2</code></td>
<td></td>
<td><code>gompi/2022.10</code>, <code>gompi/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="otf2">OTF2<a class="headerlink" href="#otf2" title="Permanent link">&para;</a></h3>
<p>The Open Trace Format 2 is a highly scalable, memory efficient event trace  data format plus support library. It is the new standard trace format for  Scalasca, Vampir, and TAU and is open for other tools.  </p>
<p><em>homepage</em>: <a href="https://www.score-p.org">https://www.score-p.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2016a</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ovito">OVITO<a class="headerlink" href="#ovito" title="Permanent link">&para;</a></h3>
<p>OVITO is a scientific visualization and data analysis solution for atomistic and other particle-based models. It helps scientists gain meaningful and quick insights from numerical simulation results.</p>
<p><em>homepage</em>: <a href="https://www.ovito.org">https://www.ovito.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.11</code></td>
<td><code>-basic</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="owncloud">ownCloud<a class="headerlink" href="#owncloud" title="Permanent link">&para;</a></h3>
<p>The ownCloud Desktop Client is a tool to synchronize files from ownCloud Server with your computer.</p>
<p><em>homepage</em>: <a href="https://github.com/owncloud/client">https://github.com/owncloud/client</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.5.4</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="oxford_asl">oxford_asl<a class="headerlink" href="#oxford_asl" title="Permanent link">&para;</a></h3>
<p>A command line tool for quantification of perfusion from ASL data</p>
<p><em>homepage</em>: <a href="https://github.com/ibme-qubic/oxford_asl">https://github.com/ibme-qubic/oxford_asl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.6</code></td>
<td><code>-centos7-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h2 id="p">P<a class="headerlink" href="#p" title="Permanent link">&para;</a></h2>
<p><a href="#p11-kit">p11-kit</a> - <a href="#p4-phylogenetics">p4-phylogenetics</a> - <a href="#p4est">p4est</a> - <a href="#p4vasp">p4vasp</a> - <a href="#p7zip">p7zip</a> - <a href="#packmol">packmol</a> - <a href="#pagmo">pagmo</a> - <a href="#pairsnp">pairsnp</a> - <a href="#pal2nal">PAL2NAL</a> - <a href="#paladin">paladin</a> - <a href="#paml">PAML</a> - <a href="#panaroo">panaroo</a> - <a href="#pandapower">pandapower</a> - <a href="#pandas">pandas</a> - <a href="#pandas-datareader">pandas-datareader</a> - <a href="#pandaseq">PANDAseq</a> - <a href="#pandoc">Pandoc</a> - <a href="#panedr">Panedr</a> - <a href="#pango">Pango</a> - <a href="#pangolin">pangolin</a> - <a href="#panito">panito</a> - <a href="#papi">PAPI</a> - <a href="#parallel">parallel</a> - <a href="#parallel-fastq-dump">parallel-fastq-dump</a> - <a href="#parallel-hashmap">Parallel-Hashmap</a> - <a href="#parallelio">ParallelIO</a> - <a href="#parameterized">parameterized</a> - <a href="#parasail">parasail</a> - <a href="#paraver">Paraver</a> - <a href="#paraview">ParaView</a> - <a href="#parcels">Parcels</a> - <a href="#pari-gp">PARI-GP</a> - <a href="#parmed">ParmEd</a> - <a href="#parmetis">ParMETIS</a> - <a href="#parmgridgen">ParMGridGen</a> - <a href="#partitionfinder">PartitionFinder</a> - <a href="#pasta">pasta</a> - <a href="#pastix">PaStiX</a> - <a href="#patch">patch</a> - <a href="#patchelf">patchelf</a> - <a href="#path.py">path.py</a> - <a href="#paup">PAUP</a> - <a href="#pauvre">pauvre</a> - <a href="#pbbam">pbbam</a> - <a href="#pbcopper">pbcopper</a> - <a href="#pbdagcon">pbdagcon</a> - <a href="#pbmm2">pbmm2</a> - <a href="#pbs_python">pbs_python</a> - <a href="#pbsuite">PBSuite</a> - <a href="#pcangsd">PCAngsd</a> - <a href="#pcc">PCC</a> - <a href="#pcl">PCL</a> - <a href="#pcmsolver">PCMSolver</a> - <a href="#pcraster">PCRaster</a> - <a href="#pcre">PCRE</a> - <a href="#pcre2">PCRE2</a> - <a href="#pdsh">pdsh</a> - <a href="#pdt">PDT</a> - <a href="#pear">PEAR</a> - <a href="#penncnv">PennCNV</a> - <a href="#pept">PEPT</a> - <a href="#percolator">Percolator</a> - <a href="#perl">Perl</a> - <a href="#perl-app-cpanminus">perl-app-cpanminus</a> - <a href="#perl4-corelibs">Perl4-CoreLibs</a> - <a href="#perseus">Perseus</a> - <a href="#pest++">PEST++</a> - <a href="#petsc">PETSc</a> - <a href="#petsc4py">petsc4py</a> - <a href="#pfamscan">PfamScan</a> - <a href="#pfind">pfind</a> - <a href="#pftoolsv3">pftoolsV3</a> - <a href="#pfunit">pFUnit</a> - <a href="#pgdspider">PGDSpider</a> - <a href="#pgi">PGI</a> - <a href="#phanotate">PHANOTATE</a> - <a href="#phantompeakqualtools">Phantompeakqualtools</a> - <a href="#phase">PHASE</a> - <a href="#phast">PHAST</a> - <a href="#phenoflow">Phenoflow</a> - <a href="#phewas">PheWAS</a> - <a href="#phipack">PhiPack</a> - <a href="#phlat">PHLAT</a> - <a href="#phonemizer">phonemizer</a> - <a href="#phono3py">phono3py</a> - <a href="#phonopy">phonopy</a> - <a href="#photontorch">photontorch</a> - <a href="#phototonic">phototonic</a> - <a href="#phylip">PHYLIP</a> - <a href="#phylobayes-mpi">PhyloBayes-MPI</a> - <a href="#phylokit">phylokit</a> - <a href="#phylonaut">phylonaut</a> - <a href="#phylophlan">PhyloPhlAn</a> - <a href="#phyml">PhyML</a> - <a href="#phyx">phyx</a> - <a href="#picard">picard</a> - <a href="#pigz">pigz</a> - <a href="#pil">PIL</a> - <a href="#pileometh">PileOMeth</a> - <a href="#pillow">Pillow</a> - <a href="#pillow-simd">Pillow-SIMD</a> - <a href="#pilon">Pilon</a> - <a href="#pims">PIMS</a> - <a href="#pindel">Pindel</a> - <a href="#pingouin">Pingouin</a> - <a href="#pint">Pint</a> - <a href="#pip">pip</a> - <a href="#pipits">PIPITS</a> - <a href="#pirate">PIRATE</a> - <a href="#pirs">pIRS</a> - <a href="#pisces">Pisces</a> - <a href="#pisvm">piSvM</a> - <a href="#pisvm-jsc">piSvM-JSC</a> - <a href="#pixman">pixman</a> - <a href="#pizzly">pizzly</a> - <a href="#pkg-config">pkg-config</a> - <a href="#pkgconf">pkgconf</a> - <a href="#pkgconfig">pkgconfig</a> - <a href="#plams">PLAMS</a> - <a href="#plantcv">plantcv</a> - <a href="#plascope">PlaScope</a> - <a href="#plasmapy">PlasmaPy</a> - <a href="#plast">PLAST</a> - <a href="#platanus">Platanus</a> - <a href="#platypus">Platypus</a> - <a href="#plc">plc</a> - <a href="#plink">PLINK</a> - <a href="#plinkqc">plinkQC</a> - <a href="#plinkseq">PLINKSEQ</a> - <a href="#plot1cell">plot1cell</a> - <a href="#ploticus">Ploticus</a> - <a href="#plotly">plotly</a> - <a href="#plotly-orca">plotly-orca</a> - <a href="#plotly.py">plotly.py</a> - <a href="#plplot">PLplot</a> - <a href="#plumed">PLUMED</a> - <a href="#ply">PLY</a> - <a href="#pmix">PMIx</a> - <a href="#pmx">pmx</a> - <a href="#pnetcdf">PnetCDF</a> - <a href="#pocl">pocl</a> - <a href="#poetry">poetry</a> - <a href="#polars">polars</a> - <a href="#polymake">polymake</a> - <a href="#pomkl">pomkl</a> - <a href="#pompi">pompi</a> - <a href="#poppler">poppler</a> - <a href="#popscle">popscle</a> - <a href="#popt">popt</a> - <a href="#porechop">Porechop</a> - <a href="#porefoam">porefoam</a> - <a href="#poretools">poretools</a> - <a href="#portcullis">Portcullis</a> - <a href="#postgres-xl">Postgres-XL</a> - <a href="#postgresql">PostgreSQL</a> - <a href="#pot">POT</a> - <a href="#pov-ray">POV-Ray</a> - <a href="#ppanggolin">PPanGGOLiN</a> - <a href="#ppfold">PPfold</a> - <a href="#ppl">ppl</a> - <a href="#pplacer">pplacer</a> - <a href="#pplpy">pplpy</a> - <a href="#prank">PRANK</a> - <a href="#prc">PRC</a> - <a href="#precice">preCICE</a> - <a href="#prequal">PREQUAL</a> - <a href="#preseq">preseq</a> - <a href="#presto">presto</a> - <a href="#pretty-yaml">pretty-yaml</a> - <a href="#primer3">Primer3</a> - <a href="#prinseq">PRINSEQ</a> - <a href="#printproto">printproto</a> - <a href="#prisms-pf">PRISMS-PF</a> - <a href="#probabel">ProbABEL</a> - <a href="#prodigal">prodigal</a> - <a href="#profit">ProFit</a> - <a href="#proj">PROJ</a> - <a href="#projectq">ProjectQ</a> - <a href="#prokka">prokka</a> - <a href="#prompt-toolkit">prompt-toolkit</a> - <a href="#proovread">proovread</a> - <a href="#propy">propy</a> - <a href="#proteinortho">Proteinortho</a> - <a href="#prothint">ProtHint</a> - <a href="#protobuf">protobuf</a> - <a href="#protobuf-python">protobuf-python</a> - <a href="#protozero">protozero</a> - <a href="#prsice">PRSice</a> - <a href="#pscom">pscom</a> - <a href="#psi">PSI</a> - <a href="#psi4">PSI4</a> - <a href="#psiclass">PsiCLASS</a> - <a href="#psipred">PSIPRED</a> - <a href="#psmc">psmc</a> - <a href="#psmpi">psmpi</a> - <a href="#psmpi2">psmpi2</a> - <a href="#psolver">PSolver</a> - <a href="#psortb">PSORTb</a> - <a href="#psrecord">psrecord</a> - <a href="#pstoedit">pstoedit</a> - <a href="#psutil">psutil</a> - <a href="#psycopg2">psycopg2</a> - <a href="#ptemcee">ptemcee</a> - <a href="#ptesfinder">PTESFinder</a> - <a href="#pubtcrs">pubtcrs</a> - <a href="#pugixml">pugixml</a> - <a href="#pullseq">pullseq</a> - <a href="#pulp">PuLP</a> - <a href="#purge_dups">purge_dups</a> - <a href="#py">py</a> - <a href="#py-aiger">py-aiger</a> - <a href="#py-aiger-bdd">py-aiger-bdd</a> - <a href="#py-cpuinfo">py-cpuinfo</a> - <a href="#pyabc">pyABC</a> - <a href="#pyamg">PyAMG</a> - <a href="#pyaps3">PyAPS3</a> - <a href="#pybedtools">pybedtools</a> - <a href="#pyberny">PyBerny</a> - <a href="#pybigwig">pyBigWig</a> - <a href="#pybind11">pybind11</a> - <a href="#pybinding">pybinding</a> - <a href="#pycairo">PyCairo</a> - <a href="#pycalib">PyCalib</a> - <a href="#pyccel">pyccel</a> - <a href="#pycharm">PyCharm</a> - <a href="#pychemps2">PyCheMPS2</a> - <a href="#pycifrw">PyCifRW</a> - <a href="#pyclone">PyClone</a> - <a href="#pycma">pycma</a> - <a href="#pycocotools">pycocotools</a> - <a href="#pycodestyle">pycodestyle</a> - <a href="#pycogent">PyCogent</a> - <a href="#pycoqc">pycoQC</a> - <a href="#pycuda">PyCUDA</a> - <a href="#pycurl">PycURL</a> - <a href="#pydamage">PyDamage</a> - <a href="#pydantic">pydantic</a> - <a href="#pydatastream">PyDatastream</a> - <a href="#pydicom">pydicom</a> - <a href="#pydlpoly">pydlpoly</a> - <a href="#pydot">pydot</a> - <a href="#pyega3">pyEGA3</a> - <a href="#pyenchant">pyenchant</a> - <a href="#pyevtk">PyEVTK</a> - <a href="#pyfai">pyFAI</a> - <a href="#pyfaidx">pyfaidx</a> - <a href="#pyfasta">pyfasta</a> - <a href="#pyffmpeg">PyFFmpeg</a> - <a href="#pyfftw">pyFFTW</a> - <a href="#pyfits">pyfits</a> - <a href="#pyfmi">PyFMI</a> - <a href="#pyfoam">PyFoam</a> - <a href="#pyfr">PyFR</a> - <a href="#pyfrag">PyFrag</a> - <a href="#pygccxml">pygccxml</a> - <a href="#pygenometracks">pyGenomeTracks</a> - <a href="#pygeos">PyGEOS</a> - <a href="#pygimli">pyGIMLi</a> - <a href="#pygments">Pygments</a> - <a href="#pygmo">pygmo</a> - <a href="#pygobject">PyGObject</a> - <a href="#pygraphviz">pygraphviz</a> - <a href="#pygrib">pygrib</a> - <a href="#pygtk">PyGTK</a> - <a href="#pygts">PyGTS</a> - <a href="#pygwas">PyGWAS</a> - <a href="#pyhdf">pyhdf</a> - <a href="#pyimagej">PyImageJ</a> - <a href="#pyiron">pyiron</a> - <a href="#pyke3">Pyke3</a> - <a href="#pylift">pylift</a> - <a href="#pylint">Pylint</a> - <a href="#pymatgen">pymatgen</a> - <a href="#pymatgen-db">pymatgen-db</a> - <a href="#pymbar">pymbar</a> - <a href="#pymc3">PyMC3</a> - <a href="#pymca">pymca</a> - <a href="#pymemcache">pymemcache</a> - <a href="#pymol">PyMOL</a> - <a href="#pynast">PyNAST</a> - <a href="#pyobjcryst">pyobjcryst</a> - <a href="#pyod">PyOD</a> - <a href="#pyomo">Pyomo</a> - <a href="#pyopencl">PyOpenCL</a> - <a href="#pyopengl">PyOpenGL</a> - <a href="#pyparsing">pyparsing</a> - <a href="#pyperf">pyperf</a> - <a href="#pyplusplus">pyplusplus</a> - <a href="#pypower">PYPOWER</a> - <a href="#pyproj">pyproj</a> - <a href="#pypsa">PyPSA</a> - <a href="#pyqstem">pyqstem</a> - <a href="#pyqt">PyQt</a> - <a href="#pyqt5">PyQt5</a> - <a href="#pyqtgraph">PyQtGraph</a> - <a href="#pyradiomics">pyradiomics</a> - <a href="#pyre">PyRe</a> - <a href="#pyretis">PyRETIS</a> - <a href="#pyringe">pyringe</a> - <a href="#pyro-api">pyro-api</a> - <a href="#pyro-ppl">pyro-ppl</a> - <a href="#pyro4">Pyro4</a> - <a href="#pyrosetta">PyRosetta</a> - <a href="#pysam">Pysam</a> - <a href="#pysamstats">pysamstats</a> - <a href="#pysat">PySAT</a> - <a href="#pyscaf">pyScaf</a> - <a href="#pyscenic">pySCENIC</a> - <a href="#pyscf">PySCF</a> - <a href="#pysheds">pysheds</a> - <a href="#pyshp">pyshp</a> - <a href="#pyside2">PySide2</a> - <a href="#pyslim">pyslim</a> - <a href="#pysndfx">pysndfx</a> - <a href="#pysolar">Pysolar</a> - <a href="#pyspoa">pyspoa</a> - <a href="#pysqlite">pysqlite</a> - <a href="#pystan">PyStan</a> - <a href="#pysteps">pysteps</a> - <a href="#pystran">pystran</a> - <a href="#pytables">PyTables</a> - <a href="#pytest">pytest</a> - <a href="#pytest-benchmark">pytest-benchmark</a> - <a href="#pytest-rerunfailures">pytest-rerunfailures</a> - <a href="#pytest-shard">pytest-shard</a> - <a href="#pytest-xdist">pytest-xdist</a> - <a href="#pythia">PYTHIA</a> - <a href="#python">Python</a> - <a href="#python-docx">python-docx</a> - <a href="#python-hl7">python-hl7</a> - <a href="#python-igraph">python-igraph</a> - <a href="#python-irodsclient">python-irodsclient</a> - <a href="#python-isal">python-isal</a> - <a href="#python-levenshtein">python-Levenshtein</a> - <a href="#python-libsbml">python-libsbml</a> - <a href="#python-louvain">python-louvain</a> - <a href="#python-parasail">python-parasail</a> - <a href="#python-telegram-bot">python-telegram-bot</a> - <a href="#python-weka-wrapper3">python-weka-wrapper3</a> - <a href="#pythran">pythran</a> - <a href="#pytorch">PyTorch</a> - <a href="#pytorch-geometric">PyTorch-Geometric</a> - <a href="#pytorch-ignite">PyTorch-Ignite</a> - <a href="#pytorch-lightning">PyTorch-Lightning</a> - <a href="#pytorch3d">PyTorch3D</a> - <a href="#pyvcf">PyVCF</a> - <a href="#pyvcf3">PyVCF3</a> - <a href="#pywannier90">pyWannier90</a> - <a href="#pywavelets">PyWavelets</a> - <a href="#pyyaml">PyYAML</a> - <a href="#pyzmq">PyZMQ</a></p>
<h3 id="p11-kit">p11-kit<a class="headerlink" href="#p11-kit" title="Permanent link">&para;</a></h3>
<p>Provides a way to load and enumerate PKCS#11 modules.  Provides a standard configuration setup for installing  PKCS#11 modules in such a way that they're discoverable.  Also solves problems with coordinating the use of PKCS#11  by different components or libraries living in the same process.</p>
<p><em>homepage</em>: <a href="https://p11-glue.freedesktop.org/p11-kit.html">https://p11-glue.freedesktop.org/p11-kit.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.23.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.24.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.24.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="p4-phylogenetics">p4-phylogenetics<a class="headerlink" href="#p4-phylogenetics" title="Permanent link">&para;</a></h3>
<p>A Python phyloinformatic toolkit, and an implementation of tree-heterogeneous models of evolution.</p>
<p><em>homepage</em>: <a href="http://p4.nhm.ac.uk/">http://p4.nhm.ac.uk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4-20210322</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="p4est">p4est<a class="headerlink" href="#p4est" title="Permanent link">&para;</a></h3>
<p>p4est is a C library to manage a collection (a forest) of multiple connected adaptive quadtrees or octrees in parallel.</p>
<p><em>homepage</em>: <a href="https://www.p4est.org">https://www.p4est.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="p4vasp">p4vasp<a class="headerlink" href="#p4vasp" title="Permanent link">&para;</a></h3>
<p>Visualization suite for VASP</p>
<p><em>homepage</em>: <a href="http://www.p4vasp.at/">http://www.p4vasp.at/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.29</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.3.30</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.3.30</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="p7zip">p7zip<a class="headerlink" href="#p7zip" title="Permanent link">&para;</a></h3>
<p>p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.  7-Zip is a file archiver with highest compression ratio.</p>
<p><em>homepage</em>: <a href="http://p7zip.sourceforge.net/">http://p7zip.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.38.1</code></td>
<td><code>GCC/4.9.2</code>, <code>system</code></td>
</tr>
<tr>
<td><code>16.02</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>17.03</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>17.04</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="packmol">packmol<a class="headerlink" href="#packmol" title="Permanent link">&para;</a></h3>
<p>Packing Optimization for Molecular Dynamics Simulations</p>
<p><em>homepage</em>: <a href="http://m3g.iqm.unicamp.br/packmol">http://m3g.iqm.unicamp.br/packmol</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>16.103</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>18.013</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>20.2.2</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>v20.2.2</code></td>
<td><code>iccifort/2020.1.217</code></td>
</tr>
</tbody>
</table>
<h3 id="pagmo">pagmo<a class="headerlink" href="#pagmo" title="Permanent link">&para;</a></h3>
<p>pagmo is a C++ scientific library for massively parallel optimization.</p>
<p><em>homepage</em>: <a href="https://esa.github.io/pagmo2">https://esa.github.io/pagmo2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.17.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pairsnp">pairsnp<a class="headerlink" href="#pairsnp" title="Permanent link">&para;</a></h3>
<p>A set of scripts for very quickly obtaining pairwise SNP distance matrices from multiple sequence alignments  using sparse matrix libraries to improve performance. </p>
<p><em>homepage</em>: <a href="https://github.com/gtonkinhill/pairsnp">https://github.com/gtonkinhill/pairsnp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.7</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pal2nal">PAL2NAL<a class="headerlink" href="#pal2nal" title="Permanent link">&para;</a></h3>
<p>PAL2NAL is a program that converts a multiple sequence alignment of proteins and  the corresponding DNA (or mRNA) sequences into a codon alignment. The program automatically  assigns the corresponding codon sequence even if the input DNA sequence has mismatches with the  input protein sequence, or contains UTRs, polyA tails. It can also deal with frame shifts in the  input alignment, which is suitable for the analysis of pseudogenes. The resulting codon alignment  can further be subjected to the calculation of synonymous (d_S) and non-synonymous (d_N) subs-  titution rates. </p>
<p><em>homepage</em>: <a href="http://www.bork.embl.de/pal2nal/">http://www.bork.embl.de/pal2nal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>14</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="paladin">paladin<a class="headerlink" href="#paladin" title="Permanent link">&para;</a></h3>
<p>Protein ALignment And Detection INterface  PALADIN is a protein sequence alignment tool designed for the accurate functional characterization of metagenomes. </p>
<p><em>homepage</em>: <a href="https://github.com/ToniWestbrook/paladin">https://github.com/ToniWestbrook/paladin</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.6</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="paml">PAML<a class="headerlink" href="#paml" title="Permanent link">&para;</a></h3>
<p>PAML is a package of programs for phylogenetic   analyses of DNA or protein sequences using maximum likelihood.</p>
<p><em>homepage</em>: <a href="http://abacus.gene.ucl.ac.uk/software/paml.html">http://abacus.gene.ucl.ac.uk/software/paml.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9i</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>4.9j</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.10.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="panaroo">panaroo<a class="headerlink" href="#panaroo" title="Permanent link">&para;</a></h3>
<p>A pangenome analysis pipeline.</p>
<p><em>homepage</em>: <a href="https://gtonkinhill.github.io/panaroo/">https://gtonkinhill.github.io/panaroo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.8</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.2.9</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pandapower">pandapower<a class="headerlink" href="#pandapower" title="Permanent link">&para;</a></h3>
<p>An easy to use open source tool for power system modeling, analysis and optimization with a high degree of automation</p>
<p><em>homepage</em>: <a href="http://www.pandapower.org/">http://www.pandapower.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pandas">pandas<a class="headerlink" href="#pandas" title="Permanent link">&para;</a></h3>
<p>pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures  and data analysis tools for the Python programming language.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pandas/">https://pypi.python.org/pypi/pandas/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.18.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.18.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.19.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.19.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.20.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.21.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.21.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="pandas-datareader">pandas-datareader<a class="headerlink" href="#pandas-datareader" title="Permanent link">&para;</a></h3>
<p>Up to date remote data access for pandas, works for multiple versions of pandas.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/pandas-datareader">https://pypi.org/project/pandas-datareader</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pandaseq">PANDAseq<a class="headerlink" href="#pandaseq" title="Permanent link">&para;</a></h3>
<p>PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded     in the sequence, and reconstruct an overlapping sequence.</p>
<p><em>homepage</em>: <a href="https://github.com/neufeld/pandaseq">https://github.com/neufeld/pandaseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>2.11</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pandoc">Pandoc<a class="headerlink" href="#pandoc" title="Permanent link">&para;</a></h3>
<p>If you need to convert files from one markup format into another, pandoc is your swiss-army knife</p>
<p><em>homepage</em>: <a href="http://pandoc.org">http://pandoc.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="panedr">Panedr<a class="headerlink" href="#panedr" title="Permanent link">&para;</a></h3>
<p>Panedr uses the Pyedr library to read a Gromacs EDR binary energy XDR file and returns its contents as a pandas dataframe.</p>
<p><em>homepage</em>: <a href="https://github.com/MDAnalysis/panedr">https://github.com/MDAnalysis/panedr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pango">Pango<a class="headerlink" href="#pango" title="Permanent link">&para;</a></h3>
<p>Pango is a library for laying out and rendering of text, with an emphasis on internationalization. Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.</p>
<p><em>homepage</em>: <a href="https://www.pango.org/">https://www.pango.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.39.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.40.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.40.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.40.5</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.40.12</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.40.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.41.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.41.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.42.4</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.43.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.44.7</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.47.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.48.5</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.48.8</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.50.7</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.50.12</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pangolin">pangolin<a class="headerlink" href="#pangolin" title="Permanent link">&para;</a></h3>
<p>Software package for assigning SARS-CoV-2 genome sequences to global lineages. This module also contains the faToVcf tool</p>
<p><em>homepage</em>: <a href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.11</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>3.1.16</code></td>
<td><code>-pangoLEARN-2021-10-18</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.1.16</code></td>
<td><code>-pangoLEARN-2021-11-25</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="panito">panito<a class="headerlink" href="#panito" title="Permanent link">&para;</a></h3>
<p>Calculate genome wide average nucleotide identity (gwANI) for a multiFASTA alignment.</p>
<p><em>homepage</em>: <a href="https://github.com/sanger-pathogens/panito">https://github.com/sanger-pathogens/panito</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="papi">PAPI<a class="headerlink" href="#papi" title="Permanent link">&para;</a></h3>
<p>PAPI provides the tool designer and application engineer with a consistent  interface and methodology for use of the performance counter hardware found  in most major microprocessors. PAPI enables software engineers to see, in near  real time, the relation between software performance and processor events.  In addition Component PAPI provides access to a collection of components  that expose performance measurement opportunites across the hardware and  software stack. </p>
<p><em>homepage</em>: <a href="https://icl.cs.utk.edu/projects/papi/">https://icl.cs.utk.edu/projects/papi/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.5.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>5.6.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>5.7.0</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.0.0.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="parallel">parallel<a class="headerlink" href="#parallel" title="Permanent link">&para;</a></h3>
<p>parallel: Build and execute shell commands in parallel</p>
<p><em>homepage</em>: <a href="https://savannah.gnu.org/projects/parallel/">https://savannah.gnu.org/projects/parallel/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20141122</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>20150322</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>20150822</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>20160622</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20170822</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>20171022</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20171122</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20180422</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>20180822</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>20181222</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>20190222</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>20190622</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>20190922</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20200422</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20200522</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20210322</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20210622</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>20210722</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>20220722</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="parallel-fastq-dump">parallel-fastq-dump<a class="headerlink" href="#parallel-fastq-dump" title="Permanent link">&para;</a></h3>
<p>parallel fastq-dump wrapper</p>
<p><em>homepage</em>: <a href="https://github.com/rvalieris/parallel-fastq-dump">https://github.com/rvalieris/parallel-fastq-dump</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.6.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.6.7</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="parallel-hashmap">Parallel-Hashmap<a class="headerlink" href="#parallel-hashmap" title="Permanent link">&para;</a></h3>
<p>Parallel Hashmap is built on a modified version of Abseil's flat_hash_map. Parallel Hashmap has lower space requirements, is nearly as fast as the underlying flat_hash_map, and can be used from multiple threads with high levels of concurrency.</p>
<p><em>homepage</em>: <a href="https://github.com/greg7mdp/parallel-hashmap">https://github.com/greg7mdp/parallel-hashmap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.33</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="parallelio">ParallelIO<a class="headerlink" href="#parallelio" title="Permanent link">&para;</a></h3>
<p>A high-level Parallel I/O Library for structured grid applications</p>
<p><em>homepage</em>: <a href="https://github.com/NCAR/ParallelIO">https://github.com/NCAR/ParallelIO</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2a</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.5.10</code></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="parameterized">parameterized<a class="headerlink" href="#parameterized" title="Permanent link">&para;</a></h3>
<p>Parameterized testing with any Python test framework </p>
<p><em>homepage</em>: <a href="https://github.com/wolever/parameterized">https://github.com/wolever/parameterized</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="parasail">parasail<a class="headerlink" href="#parasail" title="Permanent link">&para;</a></h3>
<p>parasail is a SIMD C (C99) library containing implementations  of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global  pairwise sequence alignment algorithms. </p>
<p><em>homepage</em>: <a href="https://github.com/jeffdaily/parasail">https://github.com/jeffdaily/parasail</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>GCC/8.3.0</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="paraver">Paraver<a class="headerlink" href="#paraver" title="Permanent link">&para;</a></h3>
<p>A very powerful performance visualization and analysis tool based on  traces that can be used to analyse any information that is expressed on its input trace format.  Traces for parallel MPI, OpenMP and other programs can be genereated with Extrae.</p>
<p><em>homepage</em>: <a href="https://tools.bsc.es/paraver">https://tools.bsc.es/paraver</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.9.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="paraview">ParaView<a class="headerlink" href="#paraview" title="Permanent link">&para;</a></h3>
<p>ParaView is a scientific parallel visualizer.</p>
<p><em>homepage</em>: <a href="https://www.paraview.org">https://www.paraview.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>-mpi</code></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.5.2</code></td>
<td><code>-Python-2.7.15-mpi</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.6.2</code></td>
<td><code>-Python-3.7.4-mpi</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>-Python-3.8.2-mpi</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>5.8.1</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.9.1</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>5.10.1</code></td>
<td><code>-mpi</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="parcels">Parcels<a class="headerlink" href="#parcels" title="Permanent link">&para;</a></h3>
<p>Parcels (Probably A Really Computationally Efficient Lagrangian Simulator) is a set of Python classes and methods to create customisable particle tracking simulations using output from Ocean Circulation models. Parcels can be used to track passive and active particulates such as water, plankton, plastic and fish.</p>
<p><em>homepage</em>: <a href="http://www.oceanparcels.org/">http://www.oceanparcels.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pari-gp">PARI-GP<a class="headerlink" href="#pari-gp" title="Permanent link">&para;</a></h3>
<p>PARI/GP is a widely used computer algebra system designed for fast computations in number theory  (factorizations, algebraic  number theory, elliptic curves...), but also contains a large number of other useful  functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc.,  and a lot of transcendental functions. PARI is  also available as a C library to allow for faster computations. </p>
<p><em>homepage</em>: <a href="http://pari.math.u-bordeaux.fr">http://pari.math.u-bordeaux.fr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.6</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="parmed">ParmEd<a class="headerlink" href="#parmed" title="Permanent link">&para;</a></h3>
<p>ParmEd is a general tool for aiding in investigations of biomolecular systems using popular  molecular simulation packages, like Amber, CHARMM, and OpenMM written in Python.</p>
<p><em>homepage</em>: <a href="https://parmed.github.io/ParmEd">https://parmed.github.io/ParmEd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="parmetis">ParMETIS<a class="headerlink" href="#parmetis" title="Permanent link">&para;</a></h3>
<p>ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning  unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the  functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and  large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way  graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.</p>
<p><em>homepage</em>: <a href="http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview">http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.3</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>gimkl/2.11.5</code>, <code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>gompi/2022b</code>, <code>iimpi/2019a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code>, <code>iimpi/2021b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="parmgridgen">ParMGridGen<a class="headerlink" href="#parmgridgen" title="Permanent link">&para;</a></h3>
<p>ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen,  that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited  for geometric multigrid methods.</p>
<p><em>homepage</em>: <a href="http://www-users.cs.umn.edu/~moulitsa/software.html">http://www-users.cs.umn.edu/~moulitsa/software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>gimkl/2.11.5</code>, <code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>intel/2016a</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="partitionfinder">PartitionFinder<a class="headerlink" href="#partitionfinder" title="Permanent link">&para;</a></h3>
<p>PartitionFinder 2 is a Python program for simultaneously choosing partitioning schemes and models of molecular evolution for phylogenetic analyses of DNA, protein, and morphological data. You can PartitionFinder 2 before running a phylogenetic analysis, in order to decide how to divide up your sequence data into separate blocks before analysis, and to simultaneously perform model selection on each of those blocks.</p>
<p><em>homepage</em>: <a href="https://www.robertlanfear.com/partitionfinder">https://www.robertlanfear.com/partitionfinder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pasta">pasta<a class="headerlink" href="#pasta" title="Permanent link">&para;</a></h3>
<p>PASTA (Practical Alignment using SATe and Transitivity)</p>
<p><em>homepage</em>: <a href="https://github.com/smirarab/pasta">https://github.com/smirarab/pasta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="pastix">PaStiX<a class="headerlink" href="#pastix" title="Permanent link">&para;</a></h3>
<p>PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance  parallel solver for very large sparse linear systems based on direct methods.</p>
<p><em>homepage</em>: <a href="http://pastix.gforge.inria.fr/">http://pastix.gforge.inria.fr/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="patch">patch<a class="headerlink" href="#patch" title="Permanent link">&para;</a></h3>
<p>Patch takes a patch file containing a difference listing produced by the diff program  and applies those differences to one or more original files, producing patched versions.</p>
<p><em>homepage</em>: <a href="https://savannah.gnu.org/projects/patch">https://savannah.gnu.org/projects/patch</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="patchelf">patchelf<a class="headerlink" href="#patchelf" title="Permanent link">&para;</a></h3>
<p>PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.</p>
<p><em>homepage</em>: <a href="http://nixos.org/patchelf.html">http://nixos.org/patchelf.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>0.9</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.10</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.12</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.13</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.15.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pathpy">path.py<a class="headerlink" href="#pathpy" title="Permanent link">&para;</a></h3>
<p>path.py is a Python library implementing path objects as first-class entities,  allowing common operations on files to be invoked on those path objects directly.</p>
<p><em>homepage</em>: <a href="https://github.com/jaraco/path.py">https://github.com/jaraco/path.py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.2.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.2.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.2.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.2.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>10.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="paup">PAUP<a class="headerlink" href="#paup" title="Permanent link">&para;</a></h3>
<p>PAUP<em> (Phylogenetic Analysis Using Parsimony </em>and other methods) is a computational phylogenetics program for inferring evolutionary trees.</p>
<p><em>homepage</em>: <a href="https://paup.phylosolutions.com/">https://paup.phylosolutions.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0a166</code></td>
<td><code>-centos64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0a168</code></td>
<td><code>-centos64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pauvre">pauvre<a class="headerlink" href="#pauvre" title="Permanent link">&para;</a></h3>
<p>Tools for plotting Oxford Nanopore and other long-read data</p>
<p><em>homepage</em>: <a href="https://github.com/conchoecia/pauvre">https://github.com/conchoecia/pauvre</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.1923</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.1924</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pbbam">pbbam<a class="headerlink" href="#pbbam" title="Permanent link">&para;</a></h3>
<p>The pbbam software package provides components to create, query, &amp; edit PacBio BAM files  and associated indices.</p>
<p><em>homepage</em>: <a href="http://pbbam.readthedocs.io/">http://pbbam.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>20170508</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pbcopper">pbcopper<a class="headerlink" href="#pbcopper" title="Permanent link">&para;</a></h3>
<p>The pbcopper library provides a suite of data structures, algorithms,  and utilities for C++ applications.</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/pbcopper">https://github.com/PacificBiosciences/pbcopper</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pbdagcon">pbdagcon<a class="headerlink" href="#pbdagcon" title="Permanent link">&para;</a></h3>
<p>pbdagcon is a tool that implements DAGCon (Directed Acyclic Graph Consensus) which is a sequence  consensus algorithm based on using directed acyclic graphs to encode multiple sequence alignment.</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/pbdagcon">https://github.com/PacificBiosciences/pbdagcon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170330</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pbmm2">pbmm2<a class="headerlink" href="#pbmm2" title="Permanent link">&para;</a></h3>
<p>A minimap2 frontend for PacBio native data formats</p>
<p><em>homepage</em>: <a href="https://github.com/PacificBiosciences/pbmm2">https://github.com/PacificBiosciences/pbmm2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pbs_python">pbs_python<a class="headerlink" href="#pbs_python" title="Permanent link">&para;</a></h3>
<p>The pbs_python package is a wrapper class for the Torque C library. With this package you now can  write utilities/extensions in Python instead of C. We developed this package because we want to replace xpbsmon by  an ascii version named pbsmon. PBSQuery is also included in this package. This is a python module build on top of  the pbs python module to simplify querying the batch server, eg: how many jobs, how many nodes, ...</p>
<p><em>homepage</em>: <a href="https://oss.trac.surfsara.nl/pbs_python">https://oss.trac.surfsara.nl/pbs_python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pbsuite">PBSuite<a class="headerlink" href="#pbsuite" title="Permanent link">&para;</a></h3>
<p>PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or  long 454 reads in fasta format) to high-confidence draft assembles.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/p/pb-jelly/wiki/Home/">https://sourceforge.net/p/pb-jelly/wiki/Home/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15.8.24</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="pcangsd">PCAngsd<a class="headerlink" href="#pcangsd" title="Permanent link">&para;</a></h3>
<p>PCAngsd, which estimates the covariance matrix for low depth  NGS data in an iterative procedure based on genotype likelihoods and is able  to perform multiple population genetic analyses in heterogeneous populations.</p>
<p><em>homepage</em>: <a href="http://www.popgen.dk/software/index.php/PCAngsd">http://www.popgen.dk/software/index.php/PCAngsd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.97</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pcc">PCC<a class="headerlink" href="#pcc" title="Permanent link">&para;</a></h3>
<p>The compiler is based on the original Portable C Compiler by S. C. Johnson,  written in the late 70's. About 50% of the frontend code and 80% of the backend code has been modified.</p>
<p><em>homepage</em>: <a href="http://pcc.ludd.ltu.se/">http://pcc.ludd.ltu.se/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20131024</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pcl">PCL<a class="headerlink" href="#pcl" title="Permanent link">&para;</a></h3>
<p>The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and  point cloud processing.</p>
<p><em>homepage</em>: <a href="http://pointclouds.org/">http://pointclouds.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pcmsolver">PCMSolver<a class="headerlink" href="#pcmsolver" title="Permanent link">&para;</a></h3>
<p>An API for the Polarizable Continuum Model.</p>
<p><em>homepage</em>: <a href="https://pcmsolver.readthedocs.org">https://pcmsolver.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>20160205</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pcraster">PCRaster<a class="headerlink" href="#pcraster" title="Permanent link">&para;</a></h3>
<p>PCRaster Is a collection of software targeted at the development  and deployment of spatio-temporal environmental models.</p>
<p><em>homepage</em>: <a href="http://pcraster.geo.uu.nl/">http://pcraster.geo.uu.nl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pcre">PCRE<a class="headerlink" href="#pcre" title="Permanent link">&para;</a></h3>
<p>The PCRE library is a set of functions that implement regular expression  pattern matching using the same syntax and semantics as Perl 5. </p>
<p><em>homepage</em>: <a href="https://www.pcre.org/">https://www.pcre.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.38</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.39</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.40</code></td>
<td><code>GCCcore/6.3.0</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>8.41</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>8.42</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>8.43</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>8.44</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.45</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pcre2">PCRE2<a class="headerlink" href="#pcre2" title="Permanent link">&para;</a></h3>
<p>The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5. </p>
<p><em>homepage</em>: <a href="https://www.pcre.org/">https://www.pcre.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>10.21</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>10.31</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>10.33</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>10.34</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>10.35</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>10.36</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>10.37</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>10.40</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pdsh">pdsh<a class="headerlink" href="#pdsh" title="Permanent link">&para;</a></h3>
<p>A high performance, parallel remote shell utility</p>
<p><em>homepage</em>: <a href="https://github.com/chaos/pdsh">https://github.com/chaos/pdsh</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.34</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pdt">PDT<a class="headerlink" href="#pdt" title="Permanent link">&para;</a></h3>
<p>Program Database Toolkit (PDT) is a framework for analyzing source code  written in several programming languages and for making rich program  knowledge accessible to developers of static and dynamic analysis tools.  PDT implements a standard program representation, the program database  (PDB), that can be accessed in a uniform way through a class library  supporting common PDB operations. </p>
<p><em>homepage</em>: <a href="https://www.cs.uoregon.edu/research/pdt/">https://www.cs.uoregon.edu/research/pdt/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.22</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.25</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.25.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pear">PEAR<a class="headerlink" href="#pear" title="Permanent link">&para;</a></h3>
<p>PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger.   It is fully parallelized and can run with as low as just a few kilobytes of memory.</p>
<p><em>homepage</em>: <a href="http://sco.h-its.org/exelixis/web/software/pear/">http://sco.h-its.org/exelixis/web/software/pear/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.8</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.10</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>0.9.11</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="penncnv">PennCNV<a class="headerlink" href="#penncnv" title="Permanent link">&para;</a></h3>
<p>A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently  it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format,  it can also handle other types of SNP arrays and oligonucleotide arrays.</p>
<p><em>homepage</em>: <a href="https://penncnv.openbioinformatics.org/">https://penncnv.openbioinformatics.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pept">PEPT<a class="headerlink" href="#pept" title="Permanent link">&para;</a></h3>
<p>A Python library that integrates all the tools necessary to perform research using Positron Emission  Particle Tracking (PEPT). The library includes algorithms for the location, identification and tracking of particles,  in addition to tools for visualisation and analysis, and utilities allowing the realistic simulation of PEPT data.</p>
<p><em>homepage</em>: <a href="https://pept.readthedocs.io/en/latest/">https://pept.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="percolator">Percolator<a class="headerlink" href="#percolator" title="Permanent link">&para;</a></h3>
<p>Semi-supervised learning for peptide identification from shotgun proteomics datasets</p>
<p><em>homepage</em>: <a href="https://github.com/percolator/percolator">https://github.com/percolator/percolator</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="perl">Perl<a class="headerlink" href="#perl" title="Permanent link">&para;</a></h3>
<p>Larry Wall's Practical Extraction and Report Language</p>
<p><em>homepage</em>: <a href="https://www.perl.org/">https://www.perl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.20.1</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.20.2</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.20.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.22.0</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.22.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.22.1</code></td>
<td><code>-bare</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.22.2</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.24.0</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.24.0</code></td>
<td><code>-bare</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.24.1</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.26.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.00</code>, <code>intel/2018.01</code></td>
</tr>
<tr>
<td><code>5.26.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.26.1</code></td>
<td><code>-bare</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.28.0</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.28.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.30.0</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.30.0</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.30.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.30.2</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.32.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.32.0</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.32.1</code></td>
<td></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.32.1</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.34.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.34.0</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.34.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.34.1</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.36.0</code></td>
<td></td>
<td><code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
<tr>
<td><code>5.36.0</code></td>
<td><code>-minimal</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="perl-app-cpanminus">perl-app-cpanminus<a class="headerlink" href="#perl-app-cpanminus" title="Permanent link">&para;</a></h3>
<p>cpanm - get, unpack build and install modules from CPAN </p>
<p><em>homepage</em>: <a href="https://github.com/miyagawa/cpanminus">https://github.com/miyagawa/cpanminus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7039</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="perl4-corelibs">Perl4-CoreLibs<a class="headerlink" href="#perl4-corelibs" title="Permanent link">&para;</a></h3>
<p>Libraries historically supplied with Perl 4</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Perl4::CoreLibs">https://metacpan.org/pod/Perl4::CoreLibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.003</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="perseus">Perseus<a class="headerlink" href="#perseus" title="Permanent link">&para;</a></h3>
<p>The Perseus software platform supports biological and biomedical researchers     in interpreting protein quantification, interaction and post-translational modification data.</p>
<p><em>homepage</em>: <a href="https://maxquant.org/perseus">https://maxquant.org/perseus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.7.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pest">PEST++<a class="headerlink" href="#pest" title="Permanent link">&para;</a></h3>
<p>PEST++ is a software suite aimed at supporting  complex numerical models in the decision-support context.  Much focus has been devoted to supporting environmental models  (groundwater, surface water, etc) but these tools are readily  applicable to any computer model. </p>
<p><em>homepage</em>: <a href="https://github.com/usgs/pestpp">https://github.com/usgs/pestpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.5</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="petsc">PETSc<a class="headerlink" href="#petsc" title="Permanent link">&para;</a></h3>
<p>PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the  scalable (parallel) solution of scientific applications modeled by partial differential equations.</p>
<p><em>homepage</em>: <a href="http://www.mcs.anl.gov/petsc">http://www.mcs.anl.gov/petsc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.7.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.7.5</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.8.3</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.9.3</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.11.0</code></td>
<td><code>-downloaded-deps</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.12.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.12.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.12.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.14.4</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.15.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.17.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="petsc4py">petsc4py<a class="headerlink" href="#petsc4py" title="Permanent link">&para;</a></h3>
<p>petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/petsc/petsc4py">https://bitbucket.org/petsc/petsc4py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.12.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.15.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pfamscan">PfamScan<a class="headerlink" href="#pfamscan" title="Permanent link">&para;</a></h3>
<p>PfamScan is used to search a FASTA sequence against a library of Pfam HMM.</p>
<p><em>homepage</em>: <a href="https://www.ebi.ac.uk/seqdb/confluence/display/THD/PfamScan">https://www.ebi.ac.uk/seqdb/confluence/display/THD/PfamScan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pfind">pfind<a class="headerlink" href="#pfind" title="Permanent link">&para;</a></h3>
<p>Drop-in replacement for find, implemented for using parallel access and MPI.</p>
<p><em>homepage</em>: <a href="https://github.com/VI4IO/pfind">https://github.com/VI4IO/pfind</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220613</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pftoolsv3">pftoolsV3<a class="headerlink" href="#pftoolsv3" title="Permanent link">&para;</a></h3>
<p>A suite of tools to build and search generalized profiles (protein and DNA).</p>
<p><em>homepage</em>: <a href="https://github.com/sib-swiss/pftools3">https://github.com/sib-swiss/pftools3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.11</code></td>
<td><code>GCCcore/10.3.0</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.2.12</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>20160324</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pfunit">pFUnit<a class="headerlink" href="#pfunit" title="Permanent link">&para;</a></h3>
<p>pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.</p>
<p><em>homepage</em>: <a href="https://github.com/Goddard-Fortran-Ecosystem/pFUnit">https://github.com/Goddard-Fortran-Ecosystem/pFUnit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.9</code></td>
<td><code>gompi/2018b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pgdspider">PGDSpider<a class="headerlink" href="#pgdspider" title="Permanent link">&para;</a></h3>
<p>An automated data conversion tool for connecting population genetics and genomics programs</p>
<p><em>homepage</em>: <a href="http://cmpg.unibe.ch/software/PGDSpider/">http://cmpg.unibe.ch/software/PGDSpider/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0.3</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pgi">PGI<a class="headerlink" href="#pgi" title="Permanent link">&para;</a></h3>
<p>C, C++ and Fortran compilers from The Portland Group - PGI</p>
<p><em>homepage</em>: <a href="https://www.pgroup.com/">https://www.pgroup.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15.7</code></td>
<td><code>-GNU-4.9.2-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>15.7</code></td>
<td><code>-GNU-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>15.10</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.1</code></td>
<td><code>-CDK-GCC-4.9.2-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.3</code></td>
<td><code>-GCC-4.9.3-2.25</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.4</code></td>
<td><code>-GCC-5.3.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.7</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16.10</code></td>
<td><code>-GCC-5.4.0-2.26</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.1</code></td>
<td><code>-GCC-6.3.0-2.27</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.3</code></td>
<td><code>-GCC-6.3.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.4</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.10</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.1</code></td>
<td><code>-GCC-7.2.0-2.29</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.4</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.7</code></td>
<td><code>-GCC-7.3.0-2.30</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.10</code></td>
<td><code>-GCC-6.4.0-2.28</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.1</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.4</code></td>
<td><code>-GCC-8.2.0-2.31.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.7</code></td>
<td><code>-GCC-8.3.0-2.32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>19.10</code></td>
<td><code>-GCC-8.3.0-2.32</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="phanotate">PHANOTATE<a class="headerlink" href="#phanotate" title="Permanent link">&para;</a></h3>
<p>PHANOTATE: a tool to annotate phage genomes</p>
<p><em>homepage</em>: <a href="https://github.com/deprekate/PHANOTATE">https://github.com/deprekate/PHANOTATE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190724</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="phantompeakqualtools">Phantompeakqualtools<a class="headerlink" href="#phantompeakqualtools" title="Permanent link">&para;</a></h3>
<p>It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/kundajelab/phantompeakqualtools">https://github.com/kundajelab/phantompeakqualtools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="phase">PHASE<a class="headerlink" href="#phase" title="Permanent link">&para;</a></h3>
<p>The program PHASE implements a Bayesian statistical  method for reconstructing haplotypes from population genotype data. Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf</p>
<p><em>homepage</em>: <a href="http://stephenslab.uchicago.edu/phase/download.html">http://stephenslab.uchicago.edu/phase/download.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="phast">PHAST<a class="headerlink" href="#phast" title="Permanent link">&para;</a></h3>
<p>PHAST is a freely available software package for comparative and evolutionary genomics.</p>
<p><em>homepage</em>: <a href="http://compgen.cshl.edu/phast/">http://compgen.cshl.edu/phast/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="phenoflow">Phenoflow<a class="headerlink" href="#phenoflow" title="Permanent link">&para;</a></h3>
<p>R package offering functionality for the advanced analysis of microbial flow cytometry data</p>
<p><em>homepage</em>: <a href="https://github.com/rprops/Phenoflow_package">https://github.com/rprops/Phenoflow_package</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2-20200917</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="phewas">PheWAS<a class="headerlink" href="#phewas" title="Permanent link">&para;</a></h3>
<p>Provides an accessible R interface to the phenome wide association study.</p>
<p><em>homepage</em>: <a href="https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package">https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12</code></td>
<td><code>-R-3.3.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.99.5-2</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="phipack">PhiPack<a class="headerlink" href="#phipack" title="Permanent link">&para;</a></h3>
<p>The PhiPack software package implements (in C) a few tests for recombination and can produce refined incompatibility matrices as well.</p>
<p><em>homepage</em>: <a href="http://www.maths.otago.ac.nz/~dbryant/software.html">http://www.maths.otago.ac.nz/~dbryant/software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.06.14</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="phlat">PHLAT<a class="headerlink" href="#phlat" title="Permanent link">&para;</a></h3>
<p>PHLAT is a bioinformatics algorithm that offers HLA typing at four-digit resolution (or higher)  using genome-wide transcriptome and exome sequencing data over a wide range of read lengths and sequencing depths.</p>
<p><em>homepage</em>: <a href="https://sites.google.com/site/phlatfortype/">https://sites.google.com/site/phlatfortype/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="phonemizer">phonemizer<a class="headerlink" href="#phonemizer" title="Permanent link">&para;</a></h3>
<p>The phonemizer allows simple phonemization of words and texts in many languages. Provides both the phonemize command-line tool and the Python function phonemizer.phonemize. It is using four backends: espeak, espeak-mbrola, festival and segments. </p>
<p><em>homepage</em>: <a href="https://github.com/bootphon/phonemizer">https://github.com/bootphon/phonemizer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="phono3py">phono3py<a class="headerlink" href="#phono3py" title="Permanent link">&para;</a></h3>
<p>phono3py calculates phonon-phonon interaction and related properties using the supercell approach.</p>
<p><em>homepage</em>: <a href="https://atztogo.github.io/phono3py/">https://atztogo.github.io/phono3py/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12.5.35</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.12.7.55</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="phonopy">phonopy<a class="headerlink" href="#phonopy" title="Permanent link">&para;</a></h3>
<p>Phonopy is an open source package of phonon calculations based on the supercell approach.</p>
<p><em>homepage</em>: <a href="https://atztogo.github.io/phonopy/">https://atztogo.github.io/phonopy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.12.2.20</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.12.6.66</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.13.0.64</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.14.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.12.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.16.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="photontorch">photontorch<a class="headerlink" href="#photontorch" title="Permanent link">&para;</a></h3>
<p>Photontorch is a photonic simulator for highly parallel simulation and optimization of photonic circuits in time and frequency domain. Photontorch features CUDA enabled simulation and optimization of photonic circuits. It leverages the deep learning framework PyTorch to view the photonic circuit as essentially a recurrent neural network. This enables the use of native PyTorch optimizers to optimize the (physical) parameters of the circuit. </p>
<p><em>homepage</em>: <a href="https://docs.photontorch.com/">https://docs.photontorch.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="phototonic">phototonic<a class="headerlink" href="#phototonic" title="Permanent link">&para;</a></h3>
<p>Phototonic is an image viewer and organizer</p>
<p><em>homepage</em>: <a href="https://github.com/oferkv/phototonic">https://github.com/oferkv/phototonic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="phylip">PHYLIP<a class="headerlink" href="#phylip" title="Permanent link">&para;</a></h3>
<p>PHYLIP is a free package of programs for inferring phylogenies.</p>
<p><em>homepage</em>: <a href="http://evolution.genetics.washington.edu/phylip">http://evolution.genetics.washington.edu/phylip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.696</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.697</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/9.3.0</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="phylobayes-mpi">PhyloBayes-MPI<a class="headerlink" href="#phylobayes-mpi" title="Permanent link">&para;</a></h3>
<p>A Bayesian software for phylogenetic reconstruction using mixture models</p>
<p><em>homepage</em>: <a href="https://github.com/bayesiancook/pbmpi">https://github.com/bayesiancook/pbmpi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20161021</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="phylokit">phylokit<a class="headerlink" href="#phylokit" title="Permanent link">&para;</a></h3>
<p>C++ library for high performance phylogenetics</p>
<p><em>homepage</em>: <a href="https://github.com/pranjalv123/phylokit">https://github.com/pranjalv123/phylokit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="phylonaut">phylonaut<a class="headerlink" href="#phylonaut" title="Permanent link">&para;</a></h3>
<p>Dynamic programming for phylogenetics applications</p>
<p><em>homepage</em>: <a href="https://github.com/pranjalv123/phylonaut">https://github.com/pranjalv123/phylonaut</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190626</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="phylophlan">PhyloPhlAn<a class="headerlink" href="#phylophlan" title="Permanent link">&para;</a></h3>
<p>PhyloPhlAn is an integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes. PhyloPhlAn is an accurate, rapid, and easy-to-use method for large-scale microbial genome characterization and phylogenetic analysis at multiple levels of resolution. </p>
<p><em>homepage</em>: <a href="https://github.com/biobakery/phylophlan">https://github.com/biobakery/phylophlan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="phyml">PhyML<a class="headerlink" href="#phyml" title="Permanent link">&para;</a></h3>
<p>Phylogenetic estimation using (Maximum) Likelihood</p>
<p><em>homepage</em>: <a href="https://github.com/stephaneguindon/phyml">https://github.com/stephaneguindon/phyml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.20190321</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.3.20200621</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="phyx">phyx<a class="headerlink" href="#phyx" title="Permanent link">&para;</a></h3>
<p>phyx performs phylogenetics analyses on trees and sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/FePhyFoFum/phyx">https://github.com/FePhyFoFum/phyx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.01</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="picard">picard<a class="headerlink" href="#picard" title="Permanent link">&para;</a></h3>
<p>A set of tools (in Java) for working with next generation sequencing data in the BAM format.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/picard">https://sourceforge.net/projects/picard</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.39</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.100</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.109</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.119</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.119</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.120</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.141</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Java-1.8.0_112</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Java-1.8.0_74</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Java-1.8.0_131</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td><code>-Java-1.8.0_131</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.18.5</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.18.11</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.18.14</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.18.17</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.18.27</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.20.6</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.21.1</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.21.6</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.22.1</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.25.0</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.25.1</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.25.5</code></td>
<td><code>-Java-13</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.26.10</code></td>
<td><code>-Java-15</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pigz">pigz<a class="headerlink" href="#pigz" title="Permanent link">&para;</a></h3>
<p>pigz, which stands for parallel implementation of gzip, is a fully  functional replacement for gzip that exploits multiple processors and multiple  cores to the hilt when compressing data. pigz was written by Mark Adler, and  uses the zlib and pthread libraries. </p>
<p><em>homepage</em>: <a href="https://zlib.net/pigz/">https://zlib.net/pigz/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.3</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pil">PIL<a class="headerlink" href="#pil" title="Permanent link">&para;</a></h3>
<p>The Python Imaging Library (PIL) adds image processing capabilities to your Python interpreter.  This library supports many file formats, and provides powerful image processing and graphics capabilities.</p>
<p><em>homepage</em>: <a href="http://www.pythonware.com/products/pil">http://www.pythonware.com/products/pil</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-2.7.11-freetype-2.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pileometh">PileOMeth<a class="headerlink" href="#pileometh" title="Permanent link">&para;</a></h3>
<p>PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments.  PileOMeth extracts per-base methylation metrics from them.  PileOMeth requires an indexed fasta file containing the reference genome as well. </p>
<p><em>homepage</em>: <a href="https://github.com/dpryan79/PileOMeth">https://github.com/dpryan79/PileOMeth</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.11</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="pillow">Pillow<a class="headerlink" href="#pillow" title="Permanent link">&para;</a></h3>
<p>Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.</p>
<p><em>homepage</em>: <a href="https://pillow.readthedocs.org/">https://pillow.readthedocs.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-2.7.11-freetype-2.6.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.5.2-freetype-2.6.5</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>6.2.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>8.3.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.3.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>9.1.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>9.2.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>9.4.0</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pillow-simd">Pillow-SIMD<a class="headerlink" href="#pillow-simd" title="Permanent link">&para;</a></h3>
<p>Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.</p>
<p><em>homepage</em>: <a href="https://github.com/uploadcare/pillow-simd">https://github.com/uploadcare/pillow-simd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.3.0.post0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>6.0.x.post0</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>7.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>7.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>8.3.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.3.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.2.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pilon">Pilon<a class="headerlink" href="#pilon" title="Permanent link">&para;</a></h3>
<p>Pilon is an automated genome assembly improvement and variant detection tool</p>
<p><em>homepage</em>: <a href="https://github.com/broadinstitute/pilon">https://github.com/broadinstitute/pilon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.22</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.22</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.23</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.23</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pims">PIMS<a class="headerlink" href="#pims" title="Permanent link">&para;</a></h3>
<p>PIMS is a lazy-loading interface to sequential data with numpy-like slicing.</p>
<p><em>homepage</em>: <a href="https://soft-matter.github.io/pims">https://soft-matter.github.io/pims</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pindel">Pindel<a class="headerlink" href="#pindel" title="Permanent link">&para;</a></h3>
<p>Pindel can detect breakpoints of large deletions, medium sized  insertions, inversions, tandem duplications and other structural variants at single-based  resolution from next-gen sequence data. It uses a pattern growth approach to identify the  breakpoints of these variants from paired-end short reads. </p>
<p><em>homepage</em>: <a href="http://gmt.genome.wustl.edu/packages/pindel/">http://gmt.genome.wustl.edu/packages/pindel/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.5b8</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.2.5b9-20170508</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="pingouin">Pingouin<a class="headerlink" href="#pingouin" title="Permanent link">&para;</a></h3>
<p>Pingouin is an open-source statistical package written in Python 3 and based mostly on Pandas and  NumPy.</p>
<p><em>homepage</em>: <a href="https://pingouin-stats.org/">https://pingouin-stats.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pint">Pint<a class="headerlink" href="#pint" title="Permanent link">&para;</a></h3>
<p>Pint is a Python package to define, operate and manipulate physical quantities: the product of a numerical value and a unit of measurement. It allows arithmetic operations between them and conversions from and to different units.</p>
<p><em>homepage</em>: <a href="https://github.com/hgrecco/pint">https://github.com/hgrecco/pint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.14</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.19.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pip">pip<a class="headerlink" href="#pip" title="Permanent link">&para;</a></h3>
<p>The PyPA recommended tool for installing Python packages.</p>
<p><em>homepage</em>: <a href="https://pip.pypa.io">https://pip.pypa.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>8.1.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.1.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="pipits">PIPITS<a class="headerlink" href="#pipits" title="Permanent link">&para;</a></h3>
<p>An automated pipeline for analyses of fungal internal transcribed spacer (ITS) sequences  from the Illumina sequencing platform.</p>
<p><em>homepage</em>: <a href="https://github.com/hsgweon/pipits">https://github.com/hsgweon/pipits</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.8</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pirate">PIRATE<a class="headerlink" href="#pirate" title="Permanent link">&para;</a></h3>
<p>A toolbox for pangenome analysis and threshold evaluation.</p>
<p><em>homepage</em>: <a href="https://github.com/SionBayliss/PIRATE">https://github.com/SionBayliss/PIRATE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pirs">pIRS<a class="headerlink" href="#pirs" title="Permanent link">&para;</a></h3>
<p>pIRS (profile based Illumina pair-end Reads Simulator) is a program for simulating paired-end reads from a reference genome. It is optimized for simulating reads similar to those generated from the Illumina platform.</p>
<p><em>homepage</em>: <a href="https://github.com/galaxy001/pirs">https://github.com/galaxy001/pirs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pisces">Pisces<a class="headerlink" href="#pisces" title="Permanent link">&para;</a></h3>
<p>Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.</p>
<p><em>homepage</em>: <a href="https://github.com/Illumina/Pisces">https://github.com/Illumina/Pisces</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.7.47</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pisvm">piSvM<a class="headerlink" href="#pisvm" title="Permanent link">&para;</a></h3>
<p>piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system.</p>
<p><em>homepage</em>: <a href="http://pisvm.sourceforge.net/">http://pisvm.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pisvm-jsc">piSvM-JSC<a class="headerlink" href="#pisvm-jsc" title="Permanent link">&para;</a></h3>
<p>piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system. This version is a fork of the original PiSvM to increase scalability.</p>
<p><em>homepage</em>: <a href="https://github.com/mricherzhagen/pisvm">https://github.com/mricherzhagen/pisvm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2-20150622</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pixman">pixman<a class="headerlink" href="#pixman" title="Permanent link">&para;</a></h3>
<p>Pixman is a low-level software library for pixel manipulation, providing  features such as image compositing and trapezoid rasterization. Important  users of pixman are the cairo graphics library and the X server. </p>
<p><em>homepage</em>: <a href="http://www.pixman.org/">http://www.pixman.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.34.0</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.38.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.38.4</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.40.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>0.42.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pizzly">pizzly<a class="headerlink" href="#pizzly" title="Permanent link">&para;</a></h3>
<p>Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.</p>
<p><em>homepage</em>: <a href="https://github.com/pmelsted/pizzly">https://github.com/pmelsted/pizzly</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.37.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pkg-config">pkg-config<a class="headerlink" href="#pkg-config" title="Permanent link">&para;</a></h3>
<p>pkg-config is a helper tool used when compiling applications and libraries.  It helps you insert the correct compiler options on the command line so an  application can use gcc -o test test.c <code>pkg-config --libs --cflags glib-2.0</code>  for instance, rather than hard-coding values on where to find glib (or other  libraries). </p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/pkg-config/">http://www.freedesktop.org/wiki/Software/pkg-config/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.28</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.9.2</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>0.29</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.29.1</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>gimkl/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.29.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2017a</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pkgconf">pkgconf<a class="headerlink" href="#pkgconf" title="Permanent link">&para;</a></h3>
<p>pkgconf is a program which helps to configure compiler and linker flags for development libraries.  It is similar to pkg-config from freedesktop.org.</p>
<p><em>homepage</em>: <a href="https://github.com/pkgconf/pkgconf">https://github.com/pkgconf/pkgconf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pkgconfig">pkgconfig<a class="headerlink" href="#pkgconfig" title="Permanent link">&para;</a></h3>
<p>pkgconfig is a Python module to interface with the pkg-config command line tool</p>
<p><em>homepage</em>: <a href="https://github.com/matze/pkgconfig">https://github.com/matze/pkgconfig</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-python</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.5.4</code></td>
<td><code>-python</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.5.5</code></td>
<td><code>-python</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="plams">PLAMS<a class="headerlink" href="#plams" title="Permanent link">&para;</a></h3>
<p>The Python Library for Automating Molecular Simulation (PLAMS) is powerful and flexible Python tool interfaced to the Amsterdam Modeling Suite engines ADF, BAND, DFTB, MOPAC, ReaxFF, and UFF.</p>
<p><em>homepage</em>: <a href="https://www.scm.com/product/plams">https://www.scm.com/product/plams</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td><code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="plantcv">plantcv<a class="headerlink" href="#plantcv" title="Permanent link">&para;</a></h3>
<p>PlantCV: Plant phenotyping using computer vision.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/plantcv/">https://pypi.org/project/plantcv/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="plascope">PlaScope<a class="headerlink" href="#plascope" title="Permanent link">&para;</a></h3>
<p>Plasmid exploration of bacterial genomes</p>
<p><em>homepage</em>: <a href="https://github.com/GuilhemRoyer/PlaScope">https://github.com/GuilhemRoyer/PlaScope</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="plasmapy">PlasmaPy<a class="headerlink" href="#plasmapy" title="Permanent link">&para;</a></h3>
<p>Open source Python ecosystem for plasma research and education</p>
<p><em>homepage</em>: <a href="https://www.plasmapy.org">https://www.plasmapy.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="plast">PLAST<a class="headerlink" href="#plast" title="Permanent link">&para;</a></h3>
<p>PLAST is a parallel alignment search tool for comparing large protein banks</p>
<p><em>homepage</em>: <a href="http://www.irisa.fr/symbiose/projects/plast/">http://www.irisa.fr/symbiose/projects/plast/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="platanus">Platanus<a class="headerlink" href="#platanus" title="Permanent link">&para;</a></h3>
<p>PLATform for Assembling NUcleotide Sequences</p>
<p><em>homepage</em>: <a href="http://platanus.bio.titech.ac.jp/">http://platanus.bio.titech.ac.jp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>-linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="platypus">Platypus<a class="headerlink" href="#platypus" title="Permanent link">&para;</a></h3>
<p>Platypus is a tool designed for efficient and accurate variant-detection  in high-throughput sequencing data.</p>
<p><em>homepage</em>: <a href="http://www.well.ox.ac.uk/platypus">http://www.well.ox.ac.uk/platypus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="plc">plc<a class="headerlink" href="#plc" title="Permanent link">&para;</a></h3>
<p>plc is the public Planck Likelihood Code.  It provides C and Fortran libraries that allow   users to compute the log likelihoods of the temperature,   polarization, and lensing maps.  Optionally, it also provides a python version of this library,   as well as tools to modify the predetermined options for some likelihoods   (e.g. changing the high-ell and low-ell lmin and lmax values of the temperature). </p>
<p><em>homepage</em>: <a href="http://pla.esac.esa.int/pla/#home">http://pla.esac.esa.int/pla/#home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="plink">PLINK<a class="headerlink" href="#plink" title="Permanent link">&para;</a></h3>
<p>PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</p>
<p><em>homepage</em>: <a href="https://www.cog-genomics.org/plink/2.0/">https://www.cog-genomics.org/plink/2.0/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.07</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.07-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9b5</code></td>
<td><code>golf/2020a</code></td>
</tr>
<tr>
<td><code>1.9b_4.1-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9b_6.17-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9b_6.21-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.00-alpha1-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.00-alpha2-x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.00-alpha2-x86_64_avx2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.00a2.3</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.00a2.3_x86_64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.00a3.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.00a3.6</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>2.00a3.7</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="plinkqc">plinkQC<a class="headerlink" href="#plinkqc" title="Permanent link">&para;</a></h3>
<p>plinkQC is a R/CRAN package for genotype quality control in genetic association studies.  It makes PLINK basic statistics (e.g.missing genotyping rates per individual, allele frequencies per genetic  marker) and relationship functions easily accessible from within R and allows for automatic evaluation of the  results.</p>
<p><em>homepage</em>: <a href="https://meyer-lab-cshl.github.io/plinkQC/">https://meyer-lab-cshl.github.io/plinkQC/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.3</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="plinkseq">PLINKSEQ<a class="headerlink" href="#plinkseq" title="Permanent link">&para;</a></h3>
<p>PLINK/SEQ is an open-source C/C++ library for working with human   genetic variation data. The specific focus is to provide a platform for analytic tool   development for variation data from large-scale resequencing and genotyping projects,   particularly whole-exome and whole-genome studies. It is independent of (but designed   to be complementary to) the existing PLINK package. </p>
<p><em>homepage</em>: <a href="https://atgu.mgh.harvard.edu/plinkseq/">https://atgu.mgh.harvard.edu/plinkseq/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="plot1cell">plot1cell<a class="headerlink" href="#plot1cell" title="Permanent link">&para;</a></h3>
<p>plot1cell: a package for advanced single cell data visualization</p>
<p><em>homepage</em>: <a href="https://github.com/HaojiaWu/plot1cell">https://github.com/HaojiaWu/plot1cell</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ploticus">Ploticus<a class="headerlink" href="#ploticus" title="Permanent link">&para;</a></h3>
<p>Ploticus is a free GPL software utility that can produce various types of plots and graphs</p>
<p><em>homepage</em>: <a href="http://ploticus.sourceforge.net/doc/welcome.html">http://ploticus.sourceforge.net/doc/welcome.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.42</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="plotly">plotly<a class="headerlink" href="#plotly" title="Permanent link">&para;</a></h3>
<p>Easily translate 'ggplot2' graphs to an interactive web-based version and/or  create custom web-based visualizations directly from R.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/plotly">https://cran.r-project.org/web/packages/plotly</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7.1</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.8.0</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="plotly-orca">plotly-orca<a class="headerlink" href="#plotly-orca" title="Permanent link">&para;</a></h3>
<p>Orca is an Electron app that generates images and reports of Plotly things like  plotly.js graphs, dash apps, dashboards from the command line.</p>
<p><em>homepage</em>: <a href="https://github.com/plotly/orca">https://github.com/plotly/orca</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="plotlypy">plotly.py<a class="headerlink" href="#plotlypy" title="Permanent link">&para;</a></h3>
<p>An open-source, interactive graphing library for Python</p>
<p><em>homepage</em>: <a href="https://plot.ly/python">https://plot.ly/python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>4.4.1</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.8.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.14.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.12.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.13.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="plplot">PLplot<a class="headerlink" href="#plplot" title="Permanent link">&para;</a></h3>
<p>PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols  and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer.</p>
<p><em>homepage</em>: <a href="http://plplot.sourceforge.net">http://plplot.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.11.1</code></td>
<td><code>-Java-1.7.0_80-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.11.1</code></td>
<td><code>-Java-1.7.0_80-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="plumed">PLUMED<a class="headerlink" href="#plumed" title="Permanent link">&para;</a></h3>
<p>PLUMED is an open source library for free energy calculations in molecular systems which  works together with some of the most popular molecular dynamics engines. Free energy calculations can be  performed as a function of many order parameters with a particular  focus on biological problems, using  state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.  The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. </p>
<p><em>homepage</em>: <a href="https://www.plumed.org">https://www.plumed.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-PathCV</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td></td>
<td><code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-PathCV</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.5.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2.5b</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.7.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>2.7.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="ply">PLY<a class="headerlink" href="#ply" title="Permanent link">&para;</a></h3>
<p>PLY is yet another implementation of lex and yacc for Python.</p>
<p><em>homepage</em>: <a href="http://www.dabeaz.com/ply/">http://www.dabeaz.com/ply/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.11</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.11</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pmix">PMIx<a class="headerlink" href="#pmix" title="Permanent link">&para;</a></h3>
<p>Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. </p>
<p><em>homepage</em>: <a href="https://pmix.org/">https://pmix.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2.3</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.1.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pmx">pmx<a class="headerlink" href="#pmx" title="Permanent link">&para;</a></h3>
<p>pmx (formerly pymacs) is a small bunch of classes to read structure files such as pdb or gro and trajectory data in gromacs xtc format. Over the years it has been extended towards a versatile (bio-) molecular structure manipulation package with some additional functionalities, e.g. gromacs file parsers and scripts for setup and analysis of free energy calculations.</p>
<p><em>homepage</em>: <a href="https://github.com/deGrootLab/pmx">https://github.com/deGrootLab/pmx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pnetcdf">PnetCDF<a class="headerlink" href="#pnetcdf" title="Permanent link">&para;</a></h3>
<p>Parallel netCDF: A Parallel I/O Library for NetCDF File Access</p>
<p><em>homepage</em>: <a href="https://trac.mcs.anl.gov/projects/parallel-netcdf">https://trac.mcs.anl.gov/projects/parallel-netcdf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompic/2019b</code>, <code>gompic/2020a</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>1.12.2</code></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompic/2020b</code>, <code>iimpi/2021a</code></td>
</tr>
<tr>
<td><code>1.12.3</code></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pocl">pocl<a class="headerlink" href="#pocl" title="Permanent link">&para;</a></h3>
<p>Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard</p>
<p><em>homepage</em>: <a href="http://portablecl.org">http://portablecl.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>gcccuda/2019a</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td><code>GCC/8.3.0</code>, <code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td><code>GCC/10.2.0</code>, <code>gcccuda/2020b</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="poetry">poetry<a class="headerlink" href="#poetry" title="Permanent link">&para;</a></h3>
<p>Python packaging and dependency management made easy. Poetry helps you declare, manage and install  dependencies of Python projects, ensuring you have the right stack everywhere.</p>
<p><em>homepage</em>: <a href="https://python-poetry.org">https://python-poetry.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="polars">polars<a class="headerlink" href="#polars" title="Permanent link">&para;</a></h3>
<p>Lightning-fast DataFrame library for Rust and Python.</p>
<p><em>homepage</em>: <a href="https://pola.rs/">https://pola.rs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.6</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="polymake">polymake<a class="headerlink" href="#polymake" title="Permanent link">&para;</a></h3>
<p>polymake is open source software for research in polyhedral geometry. It deals with polytopes, polyhedra and fans as well as simplicial complexes, matroids, graphs, tropical hypersurfaces, and other objects.</p>
<p><em>homepage</em>: <a href="https://polymake.org">https://polymake.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0r1</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.8</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pomkl">pomkl<a class="headerlink" href="#pomkl" title="Permanent link">&para;</a></h3>
<p>Toolchain with PGI C, C++ and Fortran compilers, alongside Intel MKL &amp; OpenMPI.</p>
<p><em>homepage</em>: <a href="http://www.pgroup.com/index.htm">http://www.pgroup.com/index.htm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.03</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.04</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pompi">pompi<a class="headerlink" href="#pompi" title="Permanent link">&para;</a></h3>
<p>Toolchain with PGI C, C++ and Fortran compilers, alongside OpenMPI.</p>
<p><em>homepage</em>: <a href="http://www.pgroup.com/index.htm">http://www.pgroup.com/index.htm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.03</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.04</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016.09</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="poppler">poppler<a class="headerlink" href="#poppler" title="Permanent link">&para;</a></h3>
<p>Poppler is a PDF rendering library</p>
<p><em>homepage</em>: <a href="https://poppler.freedesktop.org">https://poppler.freedesktop.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.70.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.90.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>21.06.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>22.01.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>22.12.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="popscle">popscle<a class="headerlink" href="#popscle" title="Permanent link">&para;</a></h3>
<p>A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools </p>
<p><em>homepage</em>: <a href="https://github.com/statgen/popscle">https://github.com/statgen/popscle</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1-beta</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="popt">popt<a class="headerlink" href="#popt" title="Permanent link">&para;</a></h3>
<p>Popt is a C library for parsing command line parameters.</p>
<p><em>homepage</em>: <a href="http://freecode.com/projects/popt">http://freecode.com/projects/popt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.14</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.16</code></td>
<td><code>GCC/4.9.2</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="porechop">Porechop<a class="headerlink" href="#porechop" title="Permanent link">&para;</a></h3>
<p>Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.  Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,  it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments  to effectively find adapters, even at low sequence identity</p>
<p><em>homepage</em>: <a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.2.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="porefoam">porefoam<a class="headerlink" href="#porefoam" title="Permanent link">&para;</a></h3>
<p>Direct pore-scale simulation of single- and two-phase flow through confined media</p>
<p><em>homepage</em>: <a href="https://github.com/ImperialCollegeLondon/porefoam">https://github.com/ImperialCollegeLondon/porefoam</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021-09-21</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="poretools">poretools<a class="headerlink" href="#poretools" title="Permanent link">&para;</a></h3>
<p>A toolkit for working with nanopore sequencing data from Oxford Nanopore.</p>
<p><em>homepage</em>: <a href="https://poretools.readthedocs.io/en/latest/">https://poretools.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="portcullis">Portcullis<a class="headerlink" href="#portcullis" title="Permanent link">&para;</a></h3>
<p>Portcullis stands for PORTable CULLing of Invalid Splice junctions from pre-aligned RNA-seq data. It is known that RNAseq mapping tools generate many invalid junction predictions, particularly in deep datasets with high coverage over splice sites. In order to address this, instead for creating a new RNAseq mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file generated by an RNAseq mapper of the user's own choice (e.g. Tophat2, Gsnap, STAR2 or HISAT2) as input (i.e. it's portable). It then, analyses and quantifies all splice junctions in the file before, filtering (culling) those which are unlikely to be genuine. Portcullis output's junctions in a variety of formats making it suitable for downstream analysis (such as differential splicing analysis and gene modelling) without additional work. Portcullis can also filter the original BAM file removing alignments associated with bad junctions.</p>
<p><em>homepage</em>: <a href="https://github.com/maplesond/portcullis">https://github.com/maplesond/portcullis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="postgres-xl">Postgres-XL<a class="headerlink" href="#postgres-xl" title="Permanent link">&para;</a></h3>
<p>Postgres-XL is a horizontally scalable open source SQL database cluster, flexible enough to handle varying database workloads:</p>
<p><em>homepage</em>: <a href="http://www.postgres-xl.org">http://www.postgres-xl.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.5r1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="postgresql">PostgreSQL<a class="headerlink" href="#postgresql" title="Permanent link">&para;</a></h3>
<p>PostgreSQL is a powerful, open source object-relational database system.  It is fully ACID compliant, has full support for foreign keys,  joins, views, triggers, and stored procedures (in multiple languages).  It includes most SQL:2008 data types, including INTEGER,  NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.  It also supports storage of binary large objects, including pictures,  sounds, or video. It has native programming interfaces for C/C++, Java,  .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.</p>
<p><em>homepage</em>: <a href="https://www.postgresql.org/">https://www.postgresql.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.4.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>9.5.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>9.6.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>9.6.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>10.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>10.3</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>10.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>11.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>11.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>12.4</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>13.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>13.3</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>13.4</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>14.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pot">POT<a class="headerlink" href="#pot" title="Permanent link">&para;</a></h3>
<p>POT (Python Optimal Transport) is a Python library provide several solvers for optimization problems  related to Optimal Transport for signal, image processing and machine learning.</p>
<p><em>homepage</em>: <a href="https://github.com/rflamary/POT">https://github.com/rflamary/POT</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pov-ray">POV-Ray<a class="headerlink" href="#pov-ray" title="Permanent link">&para;</a></h3>
<p>The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program  which generates images from a text-based scene description, and is available for a variety  of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning  three-dimensional graphics. The source code is available for those wanting to do their own ports.</p>
<p><em>homepage</em>: <a href="https://www.povray.org/">https://www.povray.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.0.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.7.0.7</code></td>
<td><code>foss/2017b</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.7.0.8</code></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>3.7.0.10</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ppanggolin">PPanGGOLiN<a class="headerlink" href="#ppanggolin" title="Permanent link">&para;</a></h3>
<p>PPanGGOLiN is a software suite used to create and manipulate prokaryotic pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage of large scale environmental studies and letting users study the pangenome of uncultivable species. </p>
<p><em>homepage</em>: <a href="https://github.com/labgem/PPanGGOLiN">https://github.com/labgem/PPanGGOLiN</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.136</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="ppfold">PPfold<a class="headerlink" href="#ppfold" title="Permanent link">&para;</a></h3>
<p>PPfold is a new implementation of pfold, written in Java 6.0. It can predict the consensus secondary  structure of RNA alignments through a stochastic context-free grammar coupled to an evolutionary model.  It can also use data from chemical probing experiments to predict RNA secondary structure. PPfold is multithreaded,  and can solve the structure of much longer alignments than pfold.</p>
<p><em>homepage</em>: <ihttp://www.daimi.au.dk/~compbio/pfold/downloads.html></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Java-1.8.0_66</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ppl">ppl<a class="headerlink" href="#ppl" title="Permanent link">&para;</a></h3>
<p>The Parma Polyhedra Library (PPL) provides numerical abstractions especially  targeted at applications in the field of analysis and verification of complex systems.</p>
<p><em>homepage</em>: <a href="https://www.bugseng.com/parma-polyhedra-library">https://www.bugseng.com/parma-polyhedra-library</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pplacer">pplacer<a class="headerlink" href="#pplacer" title="Permanent link">&para;</a></h3>
<p>Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic  likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give  useful information about uncertainty, and to offer advanced visualization and downstream analysis.</p>
<p><em>homepage</em>: <a href="http://matsen.fhcrc.org/pplacer/">http://matsen.fhcrc.org/pplacer/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.alpha19</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pplpy">pplpy<a class="headerlink" href="#pplpy" title="Permanent link">&para;</a></h3>
<p>This Python package provides a wrapper to the C++ Parma Polyhedra Library (PPL).</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/pplpy/">https://pypi.org/project/pplpy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.8.4</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="prank">PRANK<a class="headerlink" href="#prank" title="Permanent link">&para;</a></h3>
<p>PRANK is a probabilistic multiple alignment program for DNA,   codon and amino-acid sequences. PRANK is based on a novel algorithm that treats   insertions correctly and avoids over-estimation of the number of deletion events.</p>
<p><em>homepage</em>: <a href="http://wasabiapp.org/software/prank/">http://wasabiapp.org/software/prank/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>170427</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/9.3.0</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="prc">PRC<a class="headerlink" href="#prc" title="Permanent link">&para;</a></h3>
<p>PRC is a stand-alone program for aligning and scoring two profile hidden Markov models.  This can be used to detect remote relationships between profiles more effectively than by doing simple  profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both  hidden Markov models. </p>
<p><em>homepage</em>: <http//supfam.org/PRC/></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.6</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="precice">preCICE<a class="headerlink" href="#precice" title="Permanent link">&para;</a></h3>
<p>preCICE (Precise Code Interaction Coupling Environment) is a coupling library for partitioned multi-physics simulations, including, but not restricted to fluid-structure interaction and conjugate heat transfer simulations. Partitioned means that preCICE couples existing programs (solvers) capable of simulating a subpart of the complete physics involved in a simulation. This allows for the high flexibility that is needed to keep a decent time-to-solution for complex multi-physics scenarios.</p>
<p><em>homepage</em>: <a href="https://www.precice.org/">https://www.precice.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="prequal">PREQUAL<a class="headerlink" href="#prequal" title="Permanent link">&para;</a></h3>
<p>A program to identify and mask regions with non-homologous adjacent characters in FASTA files.</p>
<p><em>homepage</em>: <a href="https://github.com/simonwhelan/prequal">https://github.com/simonwhelan/prequal</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.02</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="preseq">preseq<a class="headerlink" href="#preseq" title="Permanent link">&para;</a></h3>
<p>Software for predicting library complexity and genome coverage in high-throughput sequencing.</p>
<p><em>homepage</em>: <a href="https://smithlabresearch.org/software/preseq">https://smithlabresearch.org/software/preseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>foss/2018b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="presto">presto<a class="headerlink" href="#presto" title="Permanent link">&para;</a></h3>
<p>Presto performs a fast Wilcoxon rank sum test and auROC analysis.</p>
<p><em>homepage</em>: <a href="https://github.com/immunogenomics/presto">https://github.com/immunogenomics/presto</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0-20200718</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.0.0-20230113</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pretty-yaml">pretty-yaml<a class="headerlink" href="#pretty-yaml" title="Permanent link">&para;</a></h3>
<p>PyYAML-based python module to produce pretty and readable YAML-serialized data. This module is for serialization only, see ruamel.yaml module for literate YAML parsing (keeping track of comments, spacing, line/column numbers of values, etc).</p>
<p><em>homepage</em>: <a href="https://github.com/mk-fg/pretty-yaml">https://github.com/mk-fg/pretty-yaml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>19.12.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20.4.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>21.10.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="primer3">Primer3<a class="headerlink" href="#primer3" title="Permanent link">&para;</a></h3>
<p>Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').  PCR is an essential and ubiquitous tool in genetics and molecular biology.  Primer3 can also design hybridization probes and sequencing primers.</p>
<p><em>homepage</em>: <a href="https://primer3.org/">https://primer3.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.7</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="prinseq">PRINSEQ<a class="headerlink" href="#prinseq" title="Permanent link">&para;</a></h3>
<p>A bioinformatics tool to PRe-process and show INformation of SEQuence data.</p>
<p><em>homepage</em>: <a href="http://prinseq.sourceforge.net">http://prinseq.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.20.4</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.20.4</code></td>
<td><code>-Perl-5.32.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.20.4</code></td>
<td><code>-Perl-5.34.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="printproto">printproto<a class="headerlink" href="#printproto" title="Permanent link">&para;</a></h3>
<p>X.org PrintProto protocol headers.</p>
<p><em>homepage</em>: <a href="http://xorg.freedesktop.org/">http://xorg.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="prisms-pf">PRISMS-PF<a class="headerlink" href="#prisms-pf" title="Permanent link">&para;</a></h3>
<p>PRISMS-PF is a powerful, massively parallel finite element code for conducting phase field and  other related simulations of microstructural evolution.</p>
<p><em>homepage</em>: <a href="https://prisms-center.github.io/phaseField">https://prisms-center.github.io/phaseField</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="probabel">ProbABEL<a class="headerlink" href="#probabel" title="Permanent link">&para;</a></h3>
<p>Tool for genome-wide association  analysis of imputed genetic data.</p>
<p><em>homepage</em>: <a href="http://www.genabel.org/packages/ProbABEL">http://www.genabel.org/packages/ProbABEL</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="prodigal">prodigal<a class="headerlink" href="#prodigal" title="Permanent link">&para;</a></h3>
<p>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee.</p>
<p><em>homepage</em>: <a href="https://github.com/hyattpd/Prodigal/">https://github.com/hyattpd/Prodigal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCC/4.9.3-binutils-2.25</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="profit">ProFit<a class="headerlink" href="#profit" title="Permanent link">&para;</a></h3>
<p>ProFit (pronounced Pro-Fit, not profit!) is designed to be the ultimate program for  performing least squares fits of two or more protein structures. It performs a very simple and  basic function, but allows as much flexibility as possible in performing this procedure. Thus one  can specify subsets of atoms to be considered, specify zones to be fitted by number, sequence, or  by sequence alignment. </p>
<p><em>homepage</em>: <a href="http://www.bioinf.org.uk/software/profit">http://www.bioinf.org.uk/software/profit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="proj">PROJ<a class="headerlink" href="#proj" title="Permanent link">&para;</a></h3>
<p>Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates</p>
<p><em>homepage</em>: <a href="https://proj.org">https://proj.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.9.2</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.9.3</code></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>6.0.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>7.2.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>9.1.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="projectq">ProjectQ<a class="headerlink" href="#projectq" title="Permanent link">&para;</a></h3>
<p>An open source software framework for quantum computing</p>
<p><em>homepage</em>: <a href="https://projectq.ch">https://projectq.ch</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="prokka">prokka<a class="headerlink" href="#prokka" title="Permanent link">&para;</a></h3>
<p>Prokka is a software tool for the rapid annotation of prokaryotic genomes.</p>
<p><em>homepage</em>: <a href="https://www.vicbioinformatics.com/software.prokka.shtml">https://www.vicbioinformatics.com/software.prokka.shtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11</code></td>
<td><code>-BioPerl-1.7.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.13</code></td>
<td><code>-BioPerl-1.7.2</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.13.4</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.13.7</code></td>
<td></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>1.14.5</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="prompt-toolkit">prompt-toolkit<a class="headerlink" href="#prompt-toolkit" title="Permanent link">&para;</a></h3>
<p>prompt_toolkit is a Python library for building powerful interactive command lines and  terminal applications.</p>
<p><em>homepage</em>: <a href="https://github.com/jonathanslenders/python-prompt-toolkit">https://github.com/jonathanslenders/python-prompt-toolkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.13</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="proovread">proovread<a class="headerlink" href="#proovread" title="Permanent link">&para;</a></h3>
<p>PacBio hybrid error correction through iterative short read consensus</p>
<p><em>homepage</em>: <a href="https://github.com/BioInf-Wuerzburg/proovread">https://github.com/BioInf-Wuerzburg/proovread</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.14.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="propy">propy<a class="headerlink" href="#propy" title="Permanent link">&para;</a></h3>
<p>Propy is a protein description software. It allows analyzing sequence-derived structural and physicochemical features, which are very useful in representing and distinguishing proteins or peptides of different structural, functional and interaction properties. These have been widely used in developing methods and software for predicting protein structural and functional classes, protein-protein interactions, drug-target interactions, protein substrates, among others.</p>
<p><em>homepage</em>: <a href="https://code.google.com/archive/p/protpy">https://code.google.com/archive/p/protpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="proteinortho">Proteinortho<a class="headerlink" href="#proteinortho" title="Permanent link">&para;</a></h3>
<p>Proteinortho is a tool to detect orthologous genes within different species.</p>
<p><em>homepage</em>: <a href="http://www.bioinf.uni-leipzig.de/Software/proteinortho/">http://www.bioinf.uni-leipzig.de/Software/proteinortho/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.16b</code></td>
<td><code>-Python-3.6.4-Perl-5.26.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="prothint">ProtHint<a class="headerlink" href="#prothint" title="Permanent link">&para;</a></h3>
<p>ProtHint is a pipeline for predicting and scoring hints (in the form of introns, start and  stop codons) in the genome of interest by mapping and spliced aligning predicted genes to a database of  reference protein sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/gatech-genemark/ProtHint">https://github.com/gatech-genemark/ProtHint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="protobuf">protobuf<a class="headerlink" href="#protobuf" title="Permanent link">&para;</a></h3>
<p>Google Protocol Buffers</p>
<p><em>homepage</em>: <a href="https://github.com/google/protobuf/">https://github.com/google/protobuf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.6.1.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.7.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.10.0</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.14.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.17.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.19.4</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.21.9</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="protobuf-python">protobuf-python<a class="headerlink" href="#protobuf-python" title="Permanent link">&para;</a></h3>
<p>Python Protocol Buffers runtime library.</p>
<p><em>homepage</em>: <a href="https://github.com/google/protobuf/">https://github.com/google/protobuf/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>3.10.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>3.14.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.17.3</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.19.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="protozero">protozero<a class="headerlink" href="#protozero" title="Permanent link">&para;</a></h3>
<p>Minimalistic protocol buffer decoder and encoder in C++.</p>
<p><em>homepage</em>: <a href="https://github.com/mapbox/protozero">https://github.com/mapbox/protozero</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.8</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="prsice">PRSice<a class="headerlink" href="#prsice" title="Permanent link">&para;</a></h3>
<p>PRSice (pronounced 'precise') is a Polygenic Risk  Score software for calculating, applying, evaluating and  plotting the results of polygenic risk scores (PRS) analyses.</p>
<p><em>homepage</em>: <a href="https://www.prsice.info/">https://www.prsice.info/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.12</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.3.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pscom">pscom<a class="headerlink" href="#pscom" title="Permanent link">&para;</a></h3>
<p>ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.</p>
<p><em>homepage</em>: <a href="http://www.par-tec.com">http://www.par-tec.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.43</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.0.44-1</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.0.48-1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="psi">PSI<a class="headerlink" href="#psi" title="Permanent link">&para;</a></h3>
<p>PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.</p>
<p><em>homepage</em>: <a href="http://www.psicode.org/">http://www.psicode.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0b6-20160201</code></td>
<td><code>-mt-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="psi4">PSI4<a class="headerlink" href="#psi4" title="Permanent link">&para;</a></h3>
<p>PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.</p>
<p><em>homepage</em>: <a href="http://www.psicode.org/">http://www.psicode.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-mt-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-2.7.15-maxam8</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="psiclass">PsiCLASS<a class="headerlink" href="#psiclass" title="Permanent link">&para;</a></h3>
<p>PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.</p>
<p><em>homepage</em>: <a href="https://github.com/splicebox/PsiCLASS">https://github.com/splicebox/PsiCLASS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="psipred">PSIPRED<a class="headerlink" href="#psipred" title="Permanent link">&para;</a></h3>
<p>Accurate protein secondary structure prediction</p>
<p><em>homepage</em>: <a href="http://bioinf.cs.ucl.ac.uk">http://bioinf.cs.ucl.ac.uk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.02</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="psmc">psmc<a class="headerlink" href="#psmc" title="Permanent link">&para;</a></h3>
<p>This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/psmc">https://github.com/lh3/psmc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>foss/2016a</code>, <code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="psmpi">psmpi<a class="headerlink" href="#psmpi" title="Permanent link">&para;</a></h3>
<p>ParaStation MPI is an open source high-performance MPI 3.0 implementation, based on MPICH v3. It provides extra low level communication libraries and integration with  various batch systems for tighter process control. </p>
<p><em>homepage</em>: <a href="https://github.com/ParaStation/psmpi2">https://github.com/ParaStation/psmpi2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.0-1</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.1.5-1</code></td>
<td></td>
<td><code>GCC/4.9.3</code></td>
</tr>
<tr>
<td><code>5.1.5-1</code></td>
<td><code>-mt</code></td>
<td><code>GCC/4.9.3</code></td>
</tr>
</tbody>
</table>
<h3 id="psmpi2">psmpi2<a class="headerlink" href="#psmpi2" title="Permanent link">&para;</a></h3>
<p>ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.</p>
<p><em>homepage</em>: <a href="http://www.par-tec.com">http://www.par-tec.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.29</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.0.29</code></td>
<td><code>-mt</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="psolver">PSolver<a class="headerlink" href="#psolver" title="Permanent link">&para;</a></h3>
<p>Interpolating scaling function Poisson Solver Library  </p>
<p><em>homepage</em>: <a href="http://bigdft.org/devel-doc/d1/d81/group__PSOLVER.html">http://bigdft.org/devel-doc/d1/d81/group__PSOLVER.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.6</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="psortb">PSORTb<a class="headerlink" href="#psortb" title="Permanent link">&para;</a></h3>
<p>PSORTb v3.0.4 is the most precise bacterial localization prediction tool available.</p>
<p><em>homepage</em>: <a href="http://psort.org/psortb/index.html">http://psort.org/psortb/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.4</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="psrecord">psrecord<a class="headerlink" href="#psrecord" title="Permanent link">&para;</a></h3>
<p>psrecord is a small utility that uses the psutil library to record the CPU and  memory activity of a process.</p>
<p><em>homepage</em>: <a href="https://github.com/astrofrog/psrecord">https://github.com/astrofrog/psrecord</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pstoedit">pstoedit<a class="headerlink" href="#pstoedit" title="Permanent link">&para;</a></h3>
<p>pstoedit translates PostScript and PDF graphics into other vector formats</p>
<p><em>homepage</em>: <a href="http://pstoedit.net/">http://pstoedit.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.70</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="psutil">psutil<a class="headerlink" href="#psutil" title="Permanent link">&para;</a></h3>
<p>A cross-platform process and system utilities module for Python</p>
<p><em>homepage</em>: <a href="https://github.com/giampaolo/psutil">https://github.com/giampaolo/psutil</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.4.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.4.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.4.7</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.6.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>5.6.3</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.9.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="psycopg2">psycopg2<a class="headerlink" href="#psycopg2" title="Permanent link">&para;</a></h3>
<p>Psycopg is the most popular PostgreSQL adapter for the Python programming language.</p>
<p><em>homepage</em>: <a href="https://psycopg.org/">https://psycopg.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.8.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.8.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.9.5</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ptemcee">ptemcee<a class="headerlink" href="#ptemcee" title="Permanent link">&para;</a></h3>
<p>ptemcee, pronounced "tem-cee", is fork of Daniel Foreman-Mackey's wonderful emcee to implement  parallel tempering more robustly. If you're trying to characterise awkward, multi-model probability  distributions, then ptemcee is your friend.</p>
<p><em>homepage</em>: <a href="https://github.com/willvousden/ptemcee">https://github.com/willvousden/ptemcee</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ptesfinder">PTESFinder<a class="headerlink" href="#ptesfinder" title="Permanent link">&para;</a></h3>
<p>Post-Transcriptional Exon Shuffling (PTES) Identification Pipeline</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/ptesfinder-v1/">https://sourceforge.net/projects/ptesfinder-v1/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pubtcrs">pubtcrs<a class="headerlink" href="#pubtcrs" title="Permanent link">&para;</a></h3>
<p>This repository contains C++ source code for the TCR clustering  and correlation analyses described in the manuscript "Human T cell receptor  occurrence patterns encode immune history, genetic background, and receptor  specificity" by William S DeWitt III, Anajane Smith, Gary Schoch, John A Hansen,  Frederick A Matsen IV and Philip Bradley, available on bioRxiv.</p>
<p><em>homepage</em>: <a href="https://github.com/phbradley/pubtcrs">https://github.com/phbradley/pubtcrs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180622</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pugixml">pugixml<a class="headerlink" href="#pugixml" title="Permanent link">&para;</a></h3>
<p>pugixml is a light-weight C++ XML processing library</p>
<p><em>homepage</em>: <a href="https://pugixml.org/">https://pugixml.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pullseq">pullseq<a class="headerlink" href="#pullseq" title="Permanent link">&para;</a></h3>
<p>Utility program for extracting sequences from a fasta/fastq file</p>
<p><em>homepage</em>: <a href="https://github.com/bcthomas/pullseq">https://github.com/bcthomas/pullseq</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pulp">PuLP<a class="headerlink" href="#pulp" title="Permanent link">&para;</a></h3>
<p>PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to solve linear problems. </p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/pulp">https://github.com/coin-or/pulp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="purge_dups">purge_dups<a class="headerlink" href="#purge_dups" title="Permanent link">&para;</a></h3>
<p>purge haplotigs and overlaps in an assembly based on read depth</p>
<p><em>homepage</em>: <a href="https://github.com/dfguan/purge_dups">https://github.com/dfguan/purge_dups</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="py">py<a class="headerlink" href="#py" title="Permanent link">&para;</a></h3>
<p>library with cross-python path, ini-parsing, io, code, log facilities</p>
<p><em>homepage</em>: &lt; https://pylib.readthedocs.org/&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.31</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.4.31</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="py-aiger">py-aiger<a class="headerlink" href="#py-aiger" title="Permanent link">&para;</a></h3>
<p>A python library for manipulating sequential and combinatorial circuits. This module provides the py-aiger extensions: aiger_bv, aiger_cnf, aiger_ptltl, aiger_coins, aiger_gridworld, aiger_dfa</p>
<p><em>homepage</em>: <a href="https://github.com/mvcisback/py-aiger">https://github.com/mvcisback/py-aiger</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.1.14</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="py-aiger-bdd">py-aiger-bdd<a class="headerlink" href="#py-aiger-bdd" title="Permanent link">&para;</a></h3>
<p>Aiger to BDD bridge.</p>
<p><em>homepage</em>: <a href="https://github.com/mvcisback/py-aiger-bdd">https://github.com/mvcisback/py-aiger-bdd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="py-cpuinfo">py-cpuinfo<a class="headerlink" href="#py-cpuinfo" title="Permanent link">&para;</a></h3>
<p>py-cpuinfo gets CPU info with pure Python.</p>
<p><em>homepage</em>: <a href="https://github.com/workhorsy/py-cpuinfo">https://github.com/workhorsy/py-cpuinfo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.0.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>9.0.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyabc">pyABC<a class="headerlink" href="#pyabc" title="Permanent link">&para;</a></h3>
<p>Massively parallel, distributed and scalable ABC-SMC (Approximate Bayesian Computation - Sequential Monte Carlo) for parameter estimation of complex stochastic models.  Implemented in Python with support of the R language. </p>
<p><em>homepage</em>: <a href="https://github.com/icb-dcm/pyabc">https://github.com/icb-dcm/pyabc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyamg">PyAMG<a class="headerlink" href="#pyamg" title="Permanent link">&para;</a></h3>
<p>PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface.</p>
<p><em>homepage</em>: <a href="https://pyamg.github.io">https://pyamg.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>4.2.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyaps3">PyAPS3<a class="headerlink" href="#pyaps3" title="Permanent link">&para;</a></h3>
<p>Python 3 Atmospheric Phase Screen</p>
<p><em>homepage</em>: <a href="https://github.com/AngeliqueBenoit/pyaps3">https://github.com/AngeliqueBenoit/pyaps3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190407</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pybedtools">pybedtools<a class="headerlink" href="#pybedtools" title="Permanent link">&para;</a></h3>
<p>pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.</p>
<p><em>homepage</em>: <a href="https://daler.github.io/pybedtools">https://daler.github.io/pybedtools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.10</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.7.10</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyberny">PyBerny<a class="headerlink" href="#pyberny" title="Permanent link">&para;</a></h3>
<p>PyBerny is an optimizer of molecular geometries with respect to the total energy, using nuclear gradient information.</p>
<p><em>homepage</em>: <a href="https://github.com/jhrmnn/pyberny">https://github.com/jhrmnn/pyberny</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.6.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pybigwig">pyBigWig<a class="headerlink" href="#pybigwig" title="Permanent link">&para;</a></h3>
<p>A python extension, written in C, for quick access to bigBed  files and access to and creation of bigWig files.</p>
<p><em>homepage</em>: <a href="https://github.com/deeptools/pyBigWig">https://github.com/deeptools/pyBigWig</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.13</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.3.17</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.3.18</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pybind11">pybind11<a class="headerlink" href="#pybind11" title="Permanent link">&para;</a></h3>
<p>pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,  mainly to create Python bindings of existing C++ code.</p>
<p><em>homepage</em>: <a href="https://pybind11.readthedocs.io">https://pybind11.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.4.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.9.2</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.10.3</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pybinding">pybinding<a class="headerlink" href="#pybinding" title="Permanent link">&para;</a></h3>
<p>Pybinding is a Python package for numerical tight-binding  calculations in solid state physics.</p>
<p><em>homepage</em>: <a href="https://github.com/dean0x7d/pybinding">https://github.com/dean0x7d/pybinding</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.5</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycairo">PyCairo<a class="headerlink" href="#pycairo" title="Permanent link">&para;</a></h3>
<p>Python bindings for the cairo library</p>
<p><em>homepage</em>: <a href="https://pycairo.readthedocs.io/">https://pycairo.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.18.2</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.20.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.20.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.21.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pycalib">PyCalib<a class="headerlink" href="#pycalib" title="Permanent link">&para;</a></h3>
<p>Python library for classifier calibration</p>
<p><em>homepage</em>: <a href="https://github.com/classifier-calibration/PyCalib">https://github.com/classifier-calibration/PyCalib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0.dev0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyccel">pyccel<a class="headerlink" href="#pyccel" title="Permanent link">&para;</a></h3>
<p>Python extension language using accelerators</p>
<p><em>homepage</em>: <a href="https://github.com/pyccel/pyccel">https://github.com/pyccel/pyccel</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycharm">PyCharm<a class="headerlink" href="#pycharm" title="Permanent link">&para;</a></h3>
<p>PyCharm Community Edition: Python IDE for Professional Developers</p>
<p><em>homepage</em>: <a href="https://www.jetbrains.com/pycharm/">https://www.jetbrains.com/pycharm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017.2.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.1.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.2.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.3.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="pychemps2">PyCheMPS2<a class="headerlink" href="#pychemps2" title="Permanent link">&para;</a></h3>
<p>PyCheMPS2 is a python interface to CheMPS2, for compilation without MPI. CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.</p>
<p><em>homepage</em>: <a href="https://sebwouters.github.io/CheMPS2">https://sebwouters.github.io/CheMPS2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.12</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycifrw">PyCifRW<a class="headerlink" href="#pycifrw" title="Permanent link">&para;</a></h3>
<p>PyCIFRW provides support for reading and writing CIF (Crystallographic Information Format) files using Python.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/jamesrhester/pycifrw/src/development">https://bitbucket.org/jamesrhester/pycifrw/src/development</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyclone">PyClone<a class="headerlink" href="#pyclone" title="Permanent link">&para;</a></h3>
<p>PyClone is a Python package that wraps rclone and provides a threaded interface for  an installation at the host or container level.</p>
<p><em>homepage</em>: <a href="https://gitlab.com/ltgiv/pyclone">https://gitlab.com/ltgiv/pyclone</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.9b2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pycma">pycma<a class="headerlink" href="#pycma" title="Permanent link">&para;</a></h3>
<p>A stochastic numerical optimization algorithm for difficult (non-convex, ill-conditioned,  multi-modal, rugged, noisy) optimization problems in continuous search spaces, implemented in Python.</p>
<p><em>homepage</em>: <a href="https://github.com/CMA-ES/pycma">https://github.com/CMA-ES/pycma</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycocotools">pycocotools<a class="headerlink" href="#pycocotools" title="Permanent link">&para;</a></h3>
<p>Official APIs for the MS-COCO dataset</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/pycocotools">https://pypi.org/project/pycocotools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycodestyle">pycodestyle<a class="headerlink" href="#pycodestyle" title="Permanent link">&para;</a></h3>
<p>pycodestyle is a tool to check your Python code against some of the style conventions in PEP 8.</p>
<p><em>homepage</em>: <a href="https://pycodestyle.readthedocs.io">https://pycodestyle.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycogent">PyCogent<a class="headerlink" href="#pycogent" title="Permanent link">&para;</a></h3>
<p>PyCogent is a software library for genomic biology. It is a fully  integrated and thoroughly tested framework for: controlling third-party applications;  devising workflows; querying databases; conducting novel probabilistic analyses of  biological sequence evolution; and generating publication quality graphics.</p>
<p><em>homepage</em>: <a href="http://pycogent.org">http://pycogent.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pycoqc">pycoQC<a class="headerlink" href="#pycoqc" title="Permanent link">&para;</a></h3>
<p>PycoQC computes metrics and generates interactive QC plots for  Oxford Nanopore technologies sequencing data.</p>
<p><em>homepage</em>: <a href="https://tleonardi.github.io/pycoQC">https://tleonardi.github.io/pycoQC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pycuda">PyCUDA<a class="headerlink" href="#pycuda" title="Permanent link">&para;</a></h3>
<p>PyCUDA lets you access Nvidias CUDA parallel computation API from Python.</p>
<p><em>homepage</em>: <a href="https://mathema.tician.de/software/pycuda">https://mathema.tician.de/software/pycuda</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2016.1.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2017.1.1</code></td>
<td><code>-CUDA-9.0.176-Python-2.7.14</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2017.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2018.1</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2019.1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>2020.1</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pycurl">PycURL<a class="headerlink" href="#pycurl" title="Permanent link">&para;</a></h3>
<p>PycURL is a Python interface to libcurl. PycURL can be used to fetch objects identified by a URL  from a Python program, similar to the urllib Python module. PycURL is mature, very fast, and supports a lot of  features.</p>
<p><em>homepage</em>: <a href="http://pycurl.io/">http://pycurl.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.43.0.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pydamage">PyDamage<a class="headerlink" href="#pydamage" title="Permanent link">&para;</a></h3>
<p>Pydamage, is a Python software to automate the process of contig damage identification and estimation. After modelling the ancient DNA damage using the C to T transitions, Pydamage uses a likelihood ratio test to discriminate between truly ancient, and modern contigs originating from sample contamination.</p>
<p><em>homepage</em>: <a href="https://github.com/maxibor/pydamage">https://github.com/maxibor/pydamage</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.70</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pydantic">pydantic<a class="headerlink" href="#pydantic" title="Permanent link">&para;</a></h3>
<p>Data validation and settings management using Python type hinting.</p>
<p><em>homepage</em>: <a href="https://github.com/samuelcolvin/pydantic">https://github.com/samuelcolvin/pydantic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.10.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pydatastream">PyDatastream<a class="headerlink" href="#pydatastream" title="Permanent link">&para;</a></h3>
<p>Lightweight SOAP client</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/suds-py3">https://pypi.python.org/pypi/suds-py3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pydicom">pydicom<a class="headerlink" href="#pydicom" title="Permanent link">&para;</a></h3>
<p>Pure python package for DICOM medical file reading and writing.</p>
<p><em>homepage</em>: <a href="https://pydicom.github.io/">https://pydicom.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.9</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pydlpoly">pydlpoly<a class="headerlink" href="#pydlpoly" title="Permanent link">&para;</a></h3>
<p>Pydlpoly is a molecular dynamics simulation package which is a modified version of DL-POLY with  a Python language interface.</p>
<p><em>homepage</em>: <a href="http://cmc.aci.ruhr-uni-bochum.de/cmc/">http://cmc.aci.ruhr-uni-bochum.de/cmc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20150225</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20150225</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pydot">pydot<a class="headerlink" href="#pydot" title="Permanent link">&para;</a></h3>
<p>Python interface to Graphviz's Dot language.</p>
<p><em>homepage</em>: <a href="https://github.com/pydot/pydot">https://github.com/pydot/pydot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>GCCcore/9.3.0</code>, <code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyega3">pyEGA3<a class="headerlink" href="#pyega3" title="Permanent link">&para;</a></h3>
<p>A basic Python-based EGA download client </p>
<p><em>homepage</em>: <a href="https://github.com/EGA-archive/ega-download-client">https://github.com/EGA-archive/ega-download-client</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.33</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyenchant">pyenchant<a class="headerlink" href="#pyenchant" title="Permanent link">&para;</a></h3>
<p>PyEnchant is a spellchecking library for Python, based on the excellent Enchant library.</p>
<p><em>homepage</em>: <a href="https://pythonhosted.org/pyenchant/">https://pythonhosted.org/pyenchant/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.8</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyevtk">PyEVTK<a class="headerlink" href="#pyevtk" title="Permanent link">&para;</a></h3>
<p>EVTK (Export VTK) package allows exporting data to binary VTK files for visualization and data analysis with any of the visualization packages that support VTK files</p>
<p><em>homepage</em>: <a href="https://github.com/paulo-herrera/PyEVTK">https://github.com/paulo-herrera/PyEVTK</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfai">pyFAI<a class="headerlink" href="#pyfai" title="Permanent link">&para;</a></h3>
<p>Python implementation of fast azimuthal integration.</p>
<p><em>homepage</em>: <a href="https://github.com/silx-kit/pyFAI">https://github.com/silx-kit/pyFAI</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.19.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.21.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfaidx">pyfaidx<a class="headerlink" href="#pyfaidx" title="Permanent link">&para;</a></h3>
<p>pyfaidx: efficient pythonic random access to fasta subsequences</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pyfaidx">https://pypi.python.org/pypi/pyfaidx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.9.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>0.5.9.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.6.3.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfasta">pyfasta<a class="headerlink" href="#pyfasta" title="Permanent link">&para;</a></h3>
<p>fast, memory-efficient, pythonic (and command-line) access to fasta sequence files</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/pyfasta/">https://pypi.org/project/pyfasta/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyffmpeg">PyFFmpeg<a class="headerlink" href="#pyffmpeg" title="Permanent link">&para;</a></h3>
<p>Python FFmpeg wrapper</p>
<p><em>homepage</em>: <a href="https://github.com/mhaller/pyffmpeg">https://github.com/mhaller/pyffmpeg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1beta</code></td>
<td><code>-Python-2.7.10</code></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>2.1beta</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfftw">pyFFTW<a class="headerlink" href="#pyfftw" title="Permanent link">&para;</a></h3>
<p>A pythonic wrapper around FFTW, the FFT library, presenting a unified interface  for all the supported transforms.</p>
<p><em>homepage</em>: <a href="https://github.com/pyFFTW/pyFFTW">https://github.com/pyFFTW/pyFFTW</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfits">pyfits<a class="headerlink" href="#pyfits" title="Permanent link">&para;</a></h3>
<p>The PyFITS module is a Python library providing access to FITS (Flexible Image Transport System)</p>
<p><em>homepage</em>: <a href="https://pythonhosted.org/pyfits/">https://pythonhosted.org/pyfits/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfmi">PyFMI<a class="headerlink" href="#pyfmi" title="Permanent link">&para;</a></h3>
<p>PyFMI is a package for loading and interacting with Functional Mock-Up Units (FMUs),  which are compiled dynamic models compliant with the Functional Mock-Up Interface (FMI)</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/PyFMI/">https://pypi.org/project/PyFMI/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfoam">PyFoam<a class="headerlink" href="#pyfoam" title="Permanent link">&para;</a></h3>
<p>A Python library to control OpenFOAM-runs and manipulate OpenFOAM-data.</p>
<p><em>homepage</em>: <a href="http://openfoamwiki.net/index.php/Contrib/PyFoam">http://openfoamwiki.net/index.php/Contrib/PyFoam</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.5</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfr">PyFR<a class="headerlink" href="#pyfr" title="Permanent link">&para;</a></h3>
<p>PyFR is an open-source Python based framework for solving advection-diffusion type problems on streaming architectures using the Flux Reconstruction approach of Huynh. The framework is designed to solve a range of governing systems on mixed unstructured grids containing various element types. It is also designed to target a range of hardware platforms via use of an in-built domain specific language derived from the Mako templating engine.</p>
<p><em>homepage</em>: <a href="http://www.pyfr.org">http://www.pyfr.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.6</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intelcuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyfrag">PyFrag<a class="headerlink" href="#pyfrag" title="Permanent link">&para;</a></h3>
<p>PyFrag 2019 resolves three main challenges associated with the automatized computational exploration of reaction mechanisms: 1) the management of multiple parallel calculations to automatically find a reaction path; 2) the monitoring of the entire computational process along with the extraction and plotting of relevant information from large amounts of data; and 3) the analysis and presentation of these data in a clear and informative way.  This module provides the Activation Strain Analysis (ASA) Module of PyFrag 2019 </p>
<p><em>homepage</em>: <a href="https://pyfragdocument.readthedocs.io/en/latest/includeme.html">https://pyfragdocument.readthedocs.io/en/latest/includeme.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019-20220216</code></td>
<td><code>-ASA</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pygccxml">pygccxml<a class="headerlink" href="#pygccxml" title="Permanent link">&para;</a></h3>
<p>Python package for easy C++ declarations navigation.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pygccxml">https://pypi.python.org/pypi/pygccxml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160706</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20160706</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygenometracks">pyGenomeTracks<a class="headerlink" href="#pygenometracks" title="Permanent link">&para;</a></h3>
<p>pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable.</p>
<p><em>homepage</em>: <a href="https://pygenometracks.readthedocs.io">https://pygenometracks.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.8</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygeos">PyGEOS<a class="headerlink" href="#pygeos" title="Permanent link">&para;</a></h3>
<p>PyGEOS is a C/Python library with vectorized geometry functions. The geometry operations are done in  the open-source geometry library GEOS. PyGEOS wraps these operations in NumPy ufuncs providing a performance  improvement when operating on arrays of geometries.</p>
<p><em>homepage</em>: <a href="https://pygeos.readthedocs.io">https://pygeos.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.10.2</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pygimli">pyGIMLi<a class="headerlink" href="#pygimli" title="Permanent link">&para;</a></h3>
<p>pyGIMLi is an open-source multi-method library for solving inverse  and forward tasks related to geophysical problems. Written in C++ and Python,  it offers both efficiency and flexibility allowing you to quickly build  your own robust inversion applications for the geophysical problem at hand.</p>
<p><em>homepage</em>: <a href="http://www.pygimli.org/">http://www.pygimli.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160803</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20160803</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygments">Pygments<a class="headerlink" href="#pygments" title="Permanent link">&para;</a></h3>
<p>Generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications  that need to prettify source code.</p>
<p><em>homepage</em>: <a href="https://pygments.org/">https://pygments.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygmo">pygmo<a class="headerlink" href="#pygmo" title="Permanent link">&para;</a></h3>
<p>pygmo is a scientific Python library for massively parallel optimization.</p>
<p><em>homepage</em>: <a href="https://esa.github.io/pygmo2">https://esa.github.io/pygmo2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.16.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pygobject">PyGObject<a class="headerlink" href="#pygobject" title="Permanent link">&para;</a></h3>
<p>PyGObject is a Python package which provides bindings for GObject based libraries such as GTK, GStreamer, WebKitGTK, GLib, GIO and many more.</p>
<p><em>homepage</em>: <a href="https://pygobject.readthedocs.io/">https://pygobject.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.28.6</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.28.6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.28.7</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.28.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.34.0</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.34.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.34.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.42.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pygraphviz">pygraphviz<a class="headerlink" href="#pygraphviz" title="Permanent link">&para;</a></h3>
<p>PyGraphviz is a Python interface to the Graphviz graph layout and visualization package.  With PyGraphviz you can create, edit, read, write, and draw graphs using Python to access the Graphviz  graph data structure and layout algorithms.</p>
<p><em>homepage</em>: <a href="https://pygraphviz.github.io/">https://pygraphviz.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pygrib">pygrib<a class="headerlink" href="#pygrib" title="Permanent link">&para;</a></h3>
<p>Python interface for reading and writing GRIB data</p>
<p><em>homepage</em>: <a href="https://jswhit.github.io/pygrib">https://jswhit.github.io/pygrib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.4</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygtk">PyGTK<a class="headerlink" href="#pygtk" title="Permanent link">&para;</a></h3>
<p>PyGTK lets you to easily create programs with a graphical user interface  using the Python programming language.</p>
<p><em>homepage</em>: <a href="http://www.pygtk.org/">http://www.pygtk.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.24.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.24.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.24.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pygts">PyGTS<a class="headerlink" href="#pygts" title="Permanent link">&para;</a></h3>
<p>PyGTS is a python package used to construct, manipulate,   and perform computations on triangulated surfaces.   It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. </p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/pygts/">https://sourceforge.net/projects/pygts/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pygwas">PyGWAS<a class="headerlink" href="#pygwas" title="Permanent link">&para;</a></h3>
<p>PyGWAS is a library for running Genome Wide Association studies.</p>
<p><em>homepage</em>: <a href="https://github.com/timeu/pygwas">https://github.com/timeu/pygwas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyhdf">pyhdf<a class="headerlink" href="#pyhdf" title="Permanent link">&para;</a></h3>
<p>Python wrapper around the NCSA HDF version 4 library</p>
<p><em>homepage</em>: <a href="https://github.com/fhs/pyhdf">https://github.com/fhs/pyhdf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyimagej">PyImageJ<a class="headerlink" href="#pyimagej" title="Permanent link">&para;</a></h3>
<p>PyImageJ provides a set of wrapper functions for integration between ImageJ2 and Python. It also  supports the original ImageJ API and data structures. A major advantage of this approach is the ability to combine  ImageJ and ImageJ2 with other tools available from the Python software ecosystem, including NumPy, SciPy,  scikit-image, CellProfiler, OpenCV, ITK and many more.</p>
<p><em>homepage</em>: <a href="https://pyimagej.readthedocs.io/">https://pyimagej.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyiron">pyiron<a class="headerlink" href="#pyiron" title="Permanent link">&para;</a></h3>
<p>An integrated development environment (IDE) for computational materials science.</p>
<p><em>homepage</em>: <a href="https://github.com/pyiron/pyiron">https://github.com/pyiron/pyiron</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyke3">Pyke3<a class="headerlink" href="#pyke3" title="Permanent link">&para;</a></h3>
<p>Pyke introduces a form of Logic Programming (inspired by Prolog)  to the Python community by providing a knowledge-based inference engine  (expert system) written in 100% Python.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/pyke/">http://sourceforge.net/projects/pyke/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pylift">pylift<a class="headerlink" href="#pylift" title="Permanent link">&para;</a></h3>
<p>pylift is an uplift library that provides, primarily:  (1) Fast uplift modeling implementations and (2) Evaluation tools (UpliftEval class).</p>
<p><em>homepage</em>: <a href="https://github.com/df-foundation/pylift">https://github.com/df-foundation/pylift</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.1.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="pylint">Pylint<a class="headerlink" href="#pylint" title="Permanent link">&para;</a></h3>
<p>Pylint is a tool that checks for errors in Python code, tries to enforce  a coding standard and looks for code smells. It can also look for certain type errors,  it can recommend suggestions about how particular blocks can be refactored and  can offer you details about the code's complexity.</p>
<p><em>homepage</em>: <a href="https://www.pylint.org/">https://www.pylint.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.4</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pymatgen">pymatgen<a class="headerlink" href="#pymatgen" title="Permanent link">&para;</a></h3>
<p>Python Materials Genomics is a robust materials analysis code that defines core object  representations for structures and molecules with support for many electronic structure codes.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pymatgen">https://pypi.python.org/pypi/pymatgen</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.3.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.7.3</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2017.10.16</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2017.10.16</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2022.0.4</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pymatgen-db">pymatgen-db<a class="headerlink" href="#pymatgen-db" title="Permanent link">&para;</a></h3>
<p>Pymatgen-db is a database add-on for the Python Materials Genomics (pymatgen)  materials analysis library.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pymatgen-db">https://pypi.python.org/pypi/pymatgen-db</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.5</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="pymbar">pymbar<a class="headerlink" href="#pymbar" title="Permanent link">&para;</a></h3>
<p>The pymbar package contains the pymbar suite of tools for the analysis of simulated and experimental data with the multistate Bennett acceptance ratio (MBAR) estimator.</p>
<p><em>homepage</em>: <a href="http://pymbar.readthedocs.io/en/master/">http://pymbar.readthedocs.io/en/master/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="pymc3">PyMC3<a class="headerlink" href="#pymc3" title="Permanent link">&para;</a></h3>
<p>Probabilistic Programming in Python:  Bayesian Modeling and Probabilistic Machine Learning with Theano</p>
<p><em>homepage</em>: <a href="https://docs.pymc.io/">https://docs.pymc.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pymca">pymca<a class="headerlink" href="#pymca" title="Permanent link">&para;</a></h3>
<p>The PyMca X-Ray Fluorescence Toolkit, including PyMca5 and fisx.</p>
<p><em>homepage</em>: <a href="https://github.com/vasole/pymca">https://github.com/vasole/pymca</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>5.6.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>5.7.6</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pymemcache">pymemcache<a class="headerlink" href="#pymemcache" title="Permanent link">&para;</a></h3>
<p>A comprehensive, fast, pure-Python memcached client.</p>
<p><em>homepage</em>: <a href="https://github.com/pinterest/pymemcache">https://github.com/pinterest/pymemcache</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="pymol">PyMOL<a class="headerlink" href="#pymol" title="Permanent link">&para;</a></h3>
<p>PyMOL is a Python-enhanced molecular graphics tool. It excels at 3D              visualization of proteins, small molecules, density, surfaces, and trajectories. It also includes molecular editing, ray tracing, and movies. Open Source PyMOL   is free to everyone! </p>
<p><em>homepage</em>: <a href="https://github.com/schrodinger/pymol-open-source">https://github.com/schrodinger/pymol-open-source</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pynast">PyNAST<a class="headerlink" href="#pynast" title="Permanent link">&para;</a></h3>
<p>PyNAST is a reimplementation of the NAST sequence aligner, which has  become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.  This reimplementation is more flexible, faster, and easier to install and maintain than  the original NAST implementation.</p>
<p><em>homepage</em>: <a href="https://biocore.github.io/pynast">https://biocore.github.io/pynast</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyobjcryst">pyobjcryst<a class="headerlink" href="#pyobjcryst" title="Permanent link">&para;</a></h3>
<p>Python bindings to ObjCryst++, the Object-Oriented Crystallographic Library.</p>
<p><em>homepage</em>: <a href="https://github.com/diffpy/pyobjcryst">https://github.com/diffpy/pyobjcryst</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0.post2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyod">PyOD<a class="headerlink" href="#pyod" title="Permanent link">&para;</a></h3>
<p>PyOD is a comprehensive and scalable Python toolkit for detecting outlying objects in multivariate data.</p>
<p><em>homepage</em>: <a href="https://pyod.readthedocs.io">https://pyod.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.7</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyomo">Pyomo<a class="headerlink" href="#pyomo" title="Permanent link">&para;</a></h3>
<p>Pyomo is a Python-based open-source software package that supports a diverse set of optimization  capabilities for formulating and analyzing optimization models. </p>
<p><em>homepage</em>: <a href="https://www.pyomo.org/">https://www.pyomo.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.5.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.5.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>6.4.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyopencl">PyOpenCL<a class="headerlink" href="#pyopencl" title="Permanent link">&para;</a></h3>
<p>PyOpenCL lets you access GPUs and other massively parallel compute devices from Python.</p>
<p><em>homepage</em>: <a href="https://mathema.tician.de/software/pyopencl/">https://mathema.tician.de/software/pyopencl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020.2.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2021.1.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2021.2.13</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyopengl">PyOpenGL<a class="headerlink" href="#pyopengl" title="Permanent link">&para;</a></h3>
<p>PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.</p>
<p><em>homepage</em>: <a href="http://pyopengl.sourceforge.net">http://pyopengl.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.1a1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.1.1a1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.1.1a1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.1.1a1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.1.3b2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.1.3b2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyparsing">pyparsing<a class="headerlink" href="#pyparsing" title="Permanent link">&para;</a></h3>
<p>The pyparsing module is an alternative approach to creating and executing simple grammars, vs. the traditional lex/yacc approach, or the use of regular expressions. The pyparsing module provides a library of classes that client code uses to construct the grammar directly in Python code.</p>
<p><em>homepage</em>: <a href="https://github.com/pyparsing/pyparsing">https://github.com/pyparsing/pyparsing</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.6</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyperf">pyperf<a class="headerlink" href="#pyperf" title="Permanent link">&para;</a></h3>
<p>The Python pyperf module is a toolkit to write, run and analyze benchmarks</p>
<p><em>homepage</em>: <a href="https://github.com/psf/pyperf">https://github.com/psf/pyperf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyplusplus">pyplusplus<a class="headerlink" href="#pyplusplus" title="Permanent link">&para;</a></h3>
<p>Py++ is a code generator for Boost.Python that simplifies  writing Python bindings of a C/C++ library  The tool is implemented as a Python module which is controlled by a user script.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/ompl/pyplusplus">https://bitbucket.org/ompl/pyplusplus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160707</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>20160707</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pypower">PYPOWER<a class="headerlink" href="#pypower" title="Permanent link">&para;</a></h3>
<p>PYPOWER is a power flow and Optimal Power Flow (OPF) solver. It is a port of MATPOWER to the Python programming language.</p>
<p><em>homepage</em>: <a href="https://github.com/rwl/PYPOWER">https://github.com/rwl/PYPOWER</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.15</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyproj">pyproj<a class="headerlink" href="#pyproj" title="Permanent link">&para;</a></h3>
<p>Python interface to PROJ4 library for cartographic transformations</p>
<p><em>homepage</em>: <a href="https://pyproj4.github.io/pyproj">https://pyproj4.github.io/pyproj</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.3</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.6.1.post1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pypsa">PyPSA<a class="headerlink" href="#pypsa" title="Permanent link">&para;</a></h3>
<p>PyPSA is an open source toolbox for simulating and optimising modern power systems that include features such as conventional generators with unit commitment, variable wind and solar generation, storage units, coupling to other energy sectors, and mixed alternating and direct current networks. PyPSA is designed to scale well with large networks and long time series.</p>
<p><em>homepage</em>: <a href="https://pypsa.readthedocs.io/">https://pypsa.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyqstem">pyqstem<a class="headerlink" href="#pyqstem" title="Permanent link">&para;</a></h3>
<p>QSTEM is a program for quantitative image simulation in electron microscopy, including TEM, STEM and CBED image simulation.  This project interfaces the QSTEM code with Python and the Atomic Simulation Environment (ASE) to provide a single environment for building models, simulating and analysing images.</p>
<p><em>homepage</em>: <a href="https://github.com/jacobjma/PyQSTEM">https://github.com/jacobjma/PyQSTEM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-ASE-3.22.0</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyqt">PyQt<a class="headerlink" href="#pyqt" title="Permanent link">&para;</a></h3>
<p>PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework.</p>
<p><em>homepage</em>: <a href="http://www.riverbankcomputing.co.uk/software/pyqt">http://www.riverbankcomputing.co.uk/software/pyqt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.11.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.11.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.12</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.12.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>4.12.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyqt5">PyQt5<a class="headerlink" href="#pyqt5" title="Permanent link">&para;</a></h3>
<p>PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.</p>
<p><em>homepage</em>: <a href="http://www.riverbankcomputing.co.uk/software/pyqt">http://www.riverbankcomputing.co.uk/software/pyqt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.8.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>5.9.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.9.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.11.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.12.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.12.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.13.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.15.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.15.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.15.4</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.15.5</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyqtgraph">PyQtGraph<a class="headerlink" href="#pyqtgraph" title="Permanent link">&para;</a></h3>
<p>PyQtGraph is a pure-python graphics and GUI library built on PyQt5/PySide2 and numpy.</p>
<p><em>homepage</em>: <a href="http://www.pyqtgraph.org/">http://www.pyqtgraph.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.12.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyradiomics">pyradiomics<a class="headerlink" href="#pyradiomics" title="Permanent link">&para;</a></h3>
<p>Open-source python package for the extraction of Radiomics features from 2D and 3D   images and binary masks.</p>
<p><em>homepage</em>: <a href="https://pyradiomics.readthedocs.io/">https://pyradiomics.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyre">PyRe<a class="headerlink" href="#pyre" title="Permanent link">&para;</a></h3>
<p>PyRe (Python Reliability) is a Python module for structural reliability analysis.</p>
<p><em>homepage</em>: <a href="https://hackl.science/pyre">https://hackl.science/pyre</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.3-20190221</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyretis">PyRETIS<a class="headerlink" href="#pyretis" title="Permanent link">&para;</a></h3>
<p>PyRETIS is a Python library for rare event molecular simulations with emphasis on methods based  on transition interface sampling and replica exchange transition interface sampling.</p>
<p><em>homepage</em>: <a href="http://www.pyretis.org">http://www.pyretis.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyringe">pyringe<a class="headerlink" href="#pyringe" title="Permanent link">&para;</a></h3>
<p>Debugger capable of attaching to and injecting code into python processes.</p>
<p><em>homepage</em>: <a href="https://github.com/google/pyringe">https://github.com/google/pyringe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyro-api">pyro-api<a class="headerlink" href="#pyro-api" title="Permanent link">&para;</a></h3>
<p>Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.</p>
<p><em>homepage</em>: <a href="https://github.com/pyro-ppl/pyro-api">https://github.com/pyro-ppl/pyro-api</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyro-ppl">pyro-ppl<a class="headerlink" href="#pyro-ppl" title="Permanent link">&para;</a></h3>
<p>Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.</p>
<p><em>homepage</em>: <a href="https://github.com/pyro-ppl/pyro">https://github.com/pyro-ppl/pyro</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyro4">Pyro4<a class="headerlink" href="#pyro4" title="Permanent link">&para;</a></h3>
<p>Pyro means PYthon Remote Objects. It is a library that enables you to build applications in which  objects can talk to eachother over the network, with minimal programming effort.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/Pyro4">https://pypi.python.org/pypi/Pyro4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.47</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyrosetta">PyRosetta<a class="headerlink" href="#pyrosetta" title="Permanent link">&para;</a></h3>
<p>PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling suite. It enables users to design their own custom molecular modeling algorithms using Rosetta sampling methods and energy functions.</p>
<p><em>homepage</em>: <a href="https://www.pyrosetta.org/">https://www.pyrosetta.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.release-292</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pysam">Pysam<a class="headerlink" href="#pysam" title="Permanent link">&para;</a></h3>
<p>Pysam is a python module for reading and manipulating Samfiles.   It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.</p>
<p><em>homepage</em>: <a href="https://github.com/pysam-developers/pysam">https://github.com/pysam-developers/pysam</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.1.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.0.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.12.0.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.12.0.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.13</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.14</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.14</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.14</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.14.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.14.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.15.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.15.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.15.2</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.15.3</code></td>
<td></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.16.0.1</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>0.16.0.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.17.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>0.18.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pysamstats">pysamstats<a class="headerlink" href="#pysamstats" title="Permanent link">&para;</a></h3>
<p>A Python utility for calculating statistics against genome positions based on sequence alignments from a SAM or BAM file.</p>
<p><em>homepage</em>: <a href="https://github.com/alimanfoo/pysamstats">https://github.com/alimanfoo/pysamstats</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pysat">PySAT<a class="headerlink" href="#pysat" title="Permanent link">&para;</a></h3>
<p>PySAT is a Python toolkit, which aims at providing a simple and unified interface to a number of state-of-art Boolean satisfiability (SAT) solvers as well as to a variety of cardinality and pseudo-Boolean encodings.</p>
<p><em>homepage</em>: <a href="https://pysathq.github.io/">https://pysathq.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6.dev11</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>0.1.7.dev1</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyscaf">pyScaf<a class="headerlink" href="#pyscaf" title="Permanent link">&para;</a></h3>
<p>pyScaf orders contigs from genome assemblies utilising several types of information</p>
<p><em>homepage</em>: <a href="https://github.com/lpryszcz/pyScaf">https://github.com/lpryszcz/pyScaf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12a4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyscenic">pySCENIC<a class="headerlink" href="#pyscenic" title="Permanent link">&para;</a></h3>
<p>pySCENIC is a lightning-fast python implementation of the SCENIC pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/aertslab/pySCENIC">https://github.com/aertslab/pySCENIC</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="pyscf">PySCF<a class="headerlink" href="#pyscf" title="Permanent link">&para;</a></h3>
<p>PySCF is an open-source collection of electronic structure modules powered by Python.</p>
<p><em>homepage</em>: <a href="http://www.pyscf.org">http://www.pyscf.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.3</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.7.6</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>gomkl/2021a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pysheds">pysheds<a class="headerlink" href="#pysheds" title="Permanent link">&para;</a></h3>
<p>Simple and fast watershed delineation in python.</p>
<p><em>homepage</em>: <a href="https://mattbartos.com/pysheds/">https://mattbartos.com/pysheds/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.7.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyshp">pyshp<a class="headerlink" href="#pyshp" title="Permanent link">&para;</a></h3>
<p>Pure Python read/write support for ESRI Shapefile format</p>
<p><em>homepage</em>: <a href="https://github.com/GeospatialPython/pyshp">https://github.com/GeospatialPython/pyshp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.12</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyside2">PySide2<a class="headerlink" href="#pyside2" title="Permanent link">&para;</a></h3>
<p>PySide2 is the official Python module from the Qt for Python project,     which provides access to the complete Qt 5.12+ framework.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/PySide2">https://pypi.python.org/pypi/PySide2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.14.2.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyslim">pyslim<a class="headerlink" href="#pyslim" title="Permanent link">&para;</a></h3>
<p>A Python API for reading and modifying tskit tree sequence files produced by SLiM, or modifying files produced by other programs (e.g., msprime, fwdpy11, and tsinfer) for use in SLiM.</p>
<p><em>homepage</em>: <a href="https://tskit.dev/pyslim/docs/stable/introduction.html">https://tskit.dev/pyslim/docs/stable/introduction.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pysndfx">pysndfx<a class="headerlink" href="#pysndfx" title="Permanent link">&para;</a></h3>
<p>A lightweight Python wrapper for SoX - Sound eXchange.  Supported effects range from EQ and compression to phasers, reverb and pitch shifters.</p>
<p><em>homepage</em>: <a href="https://github.com/carlthome/python-audio-effects">https://github.com/carlthome/python-audio-effects</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pysolar">Pysolar<a class="headerlink" href="#pysolar" title="Permanent link">&para;</a></h3>
<p>Pysolar is a collection of Python libraries for simulating the irradiation  of any point on earth by the sun.</p>
<p><em>homepage</em>: <a href="https://pysolar.org/">https://pysolar.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.7</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.8</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.8</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyspoa">pyspoa<a class="headerlink" href="#pyspoa" title="Permanent link">&para;</a></h3>
<p>Python bindings to spoa.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/pyspoa">https://github.com/nanoporetech/pyspoa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>0.0.8</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pysqlite">pysqlite<a class="headerlink" href="#pysqlite" title="Permanent link">&para;</a></h3>
<p>pysqlite is an interface to the SQLite 3.x embedded relational database engine.  It is almost fully compliant with the Python database API version 2.0 also exposes the unique features of SQLite.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/pysqlite">https://pypi.python.org/pypi/pysqlite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="pystan">PyStan<a class="headerlink" href="#pystan" title="Permanent link">&para;</a></h3>
<p>Python interface to Stan, a package for Bayesian inference  using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo.</p>
<p><em>homepage</em>: <a href="https://github.com/stan-dev/pystan">https://github.com/stan-dev/pystan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.19.0.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.19.1.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="pysteps">pysteps<a class="headerlink" href="#pysteps" title="Permanent link">&para;</a></h3>
<p>Pysteps is an open-source and community-driven Python library for probabilistic precipitation nowcasting, i.e. short-term ensemble prediction systems.</p>
<p><em>homepage</em>: <a href="https://pysteps.github.io/">https://pysteps.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pystran">pystran<a class="headerlink" href="#pystran" title="Permanent link">&para;</a></h3>
<p>Toolset of dynamical model STRucture ANalysis algorithms</p>
<p><em>homepage</em>: <a href="https://stijnvanhoey.github.io/pystran/">https://stijnvanhoey.github.io/pystran/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017.04.20</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="pytables">PyTables<a class="headerlink" href="#pytables" title="Permanent link">&para;</a></h3>
<p>PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope  with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the  NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical  parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browsing,  processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and   disk resources so that data takes much less space (specially if on-flight compression is used) than other solutions   such as relational or object oriented databases.</p>
<p><em>homepage</em>: <a href="https://www.pytables.org">https://www.pytables.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.2.3.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.4.4</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.4.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pytest">pytest<a class="headerlink" href="#pytest" title="Permanent link">&para;</a></h3>
<p>pytest: simple powerful testing with Python</p>
<p><em>homepage</em>: <a href="https://pytest.org">https://pytest.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>6.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>7.1.3</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pytest-benchmark">pytest-benchmark<a class="headerlink" href="#pytest-benchmark" title="Permanent link">&para;</a></h3>
<p>A pytest fixture for benchmarking code.</p>
<p><em>homepage</em>: <a href="https://github.com/ionelmc/pytest-benchmark">https://github.com/ionelmc/pytest-benchmark</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pytest-rerunfailures">pytest-rerunfailures<a class="headerlink" href="#pytest-rerunfailures" title="Permanent link">&para;</a></h3>
<p>pytest plugin to re-run tests to eliminate flaky failures.</p>
<p><em>homepage</em>: <a href="https://github.com/pytest-dev/pytest-rerunfailures">https://github.com/pytest-dev/pytest-rerunfailures</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>11.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pytest-shard">pytest-shard<a class="headerlink" href="#pytest-shard" title="Permanent link">&para;</a></h3>
<p>pytest plugin to support parallelism across multiple machines.  Shards tests based on a hash of their test name enabling easy parallelism across machines, suitable for a wide variety of continuous integration services. Tests are split at the finest level of granularity, individual test cases, enabling parallelism even if all of your tests are in a single file (or even single parameterized test method). </p>
<p><em>homepage</em>: <a href="https://github.com/AdamGleave/pytest-shard">https://github.com/AdamGleave/pytest-shard</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pytest-xdist">pytest-xdist<a class="headerlink" href="#pytest-xdist" title="Permanent link">&para;</a></h3>
<p>xdist: pytest distributed testing plugin  The pytest-xdist plugin extends pytest with some unique test execution modes:      * test run parallelization: if you have multiple CPUs or hosts you       can use those for a combined test run. This allows to speed up       development or to use special resources of remote machines.      * --looponfail: run your tests repeatedly in a subprocess. After         each run pytest waits until a file in your project changes and         then re-runs the previously failing tests. This is repeated         until all tests pass after which again a full run is         performed.      * Multi-Platform coverage: you can specify different Python       interpreters or different platforms and run tests in parallel on       all of them.  Before running tests remotely, pytest efficiently rsyncs your program source code to the remote place. All test results are reported back and displayed to your local terminal. You may specify different Python versions and interpreters.</p>
<p><em>homepage</em>: <a href="https://github.com/pytest-dev/pytest-xdist">https://github.com/pytest-dev/pytest-xdist</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pythia">PYTHIA<a class="headerlink" href="#pythia" title="Permanent link">&para;</a></h3>
<p>PYTHIA is a standard tool for the generation of events in high-energy collisions,  comprising a coherent set of physics models for the evolution from a few-body hard  process to a complex multiparticle final state.</p>
<p><em>homepage</em>: <a href="http://home.thep.lu.se/~torbjorn/Pythia.html">http://home.thep.lu.se/~torbjorn/Pythia.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.226</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="python">Python<a class="headerlink" href="#python" title="Permanent link">&para;</a></h3>
<p>Python is a programming language that lets you work more quickly and integrate your systems  more effectively.</p>
<p><em>homepage</em>: <a href="https://python.org/">https://python.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.9</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.7.10</code></td>
<td></td>
<td><code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>2.7.10</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.7.11</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.7.11</code></td>
<td><code>-bare</code></td>
<td><code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.7.11</code></td>
<td><code>-libX11-1.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.7.12</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>iomkl/2017a</code></td>
</tr>
<tr>
<td><code>2.7.12</code></td>
<td><code>-bare</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>2.7.13</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.13</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>2.7.14</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>intel/2017b</code>, <code>intel/2018.01</code>, <code>intel/2018a</code>, <code>intelcuda/2017b</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>2.7.14</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.7.15</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>2.7.15</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.7.16</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.16</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.18</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.18</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td></td>
<td><code>foss/2016.04</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>-bare</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.00</code></td>
</tr>
<tr>
<td><code>3.6.3</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.01</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>3.6.4</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>foss/2018a</code>, <code>fosscuda/2018a</code>, <code>golf/2018a</code>, <code>iimkl/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018.02</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.6.6</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.7.4</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.8.6</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.9.5</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.9.5</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.10.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.10.4</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
<tr>
<td><code>3.11.2</code></td>
<td><code>-bare</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-docx">python-docx<a class="headerlink" href="#python-docx" title="Permanent link">&para;</a></h3>
<p>python-docx is a Python library for creating and updating Microsoft Word (.docx) files</p>
<p><em>homepage</em>: <a href="https://python-docx.readthedocs.io/en/latest/">https://python-docx.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.11</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-hl7">python-hl7<a class="headerlink" href="#python-hl7" title="Permanent link">&para;</a></h3>
<p>A simple library for parsing messages of Health Level 7 (HL7) version 2.x into Python objects.</p>
<p><em>homepage</em>: <a href="https://github.com/johnpaulett/python-hl7">https://github.com/johnpaulett/python-hl7</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-igraph">python-igraph<a class="headerlink" href="#python-igraph" title="Permanent link">&para;</a></h3>
<p>Python interface to the igraph high performance graph library, primarily aimed at complex network  research and analysis.</p>
<p><em>homepage</em>: <a href="https://igraph.org/python">https://igraph.org/python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1.post6</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.7.1.post6</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.9.6</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.9.8</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.10.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="python-irodsclient">python-irodsclient<a class="headerlink" href="#python-irodsclient" title="Permanent link">&para;</a></h3>
<p>A python API for iRODS</p>
<p><em>homepage</em>: <a href="https://github.com/irods/python-irodsclient">https://github.com/irods/python-irodsclient</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-isal">python-isal<a class="headerlink" href="#python-isal" title="Permanent link">&para;</a></h3>
<p>Faster zlib and gzip compatible compression and decompression  by providing python bindings for the isa-l library. </p>
<p><em>homepage</em>: <a href="https://github.com/pycompression/python-isal">https://github.com/pycompression/python-isal</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.0</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-levenshtein">python-Levenshtein<a class="headerlink" href="#python-levenshtein" title="Permanent link">&para;</a></h3>
<p>Python extension for computing string edit distances and similarities.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/python-Levenshtein/">https://pypi.org/project/python-Levenshtein/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="python-libsbml">python-libsbml<a class="headerlink" href="#python-libsbml" title="Permanent link">&para;</a></h3>
<p>LibSBML Python API.</p>
<p><em>homepage</em>: <a href="https://sbml.org/">https://sbml.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.19.7</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="python-louvain">python-louvain<a class="headerlink" href="#python-louvain" title="Permanent link">&para;</a></h3>
<p>Louvain algorithm for community detection</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/python-louvain">https://pypi.org/project/python-louvain</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="python-parasail">python-parasail<a class="headerlink" href="#python-parasail" title="Permanent link">&para;</a></h3>
<p>Python Bindings for the Parasail C Library</p>
<p><em>homepage</em>: <a href="https://github.com/jeffdaily/parasail-python">https://github.com/jeffdaily/parasail-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.12</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.1.16</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="python-telegram-bot">python-telegram-bot<a class="headerlink" href="#python-telegram-bot" title="Permanent link">&para;</a></h3>
<p>This library provides a pure Python, asynchronous interface for the Telegram Bot API.  It's compatible with Python versions 3.7+.</p>
<p><em>homepage</em>: <a href="https://python-telegram-bot.org/">https://python-telegram-bot.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20.0a0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="python-weka-wrapper3">python-weka-wrapper3<a class="headerlink" href="#python-weka-wrapper3" title="Permanent link">&para;</a></h3>
<p>Python3 wrapper for the Weka Machine Learning Workbench</p>
<p><em>homepage</em>: <a href="https://github.com/fracpete/python-weka-wrapper3">https://github.com/fracpete/python-weka-wrapper3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.11</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pythran">pythran<a class="headerlink" href="#pythran" title="Permanent link">&para;</a></h3>
<p>Pythran is an ahead of time compiler for a subset of the Python language, with a focus on scientific computing. It takes a Python module annotated with a few interface description and turns it into a native Python module with the same interface, but (hopefully) faster. </p>
<p><em>homepage</em>: <a href="https://pythran.readthedocs.io">https://pythran.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.4.post1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pytorch">PyTorch<a class="headerlink" href="#pytorch" title="Permanent link">&para;</a></h3>
<p>Tensors and Dynamic neural networks in Python with strong GPU acceleration. PyTorch is a deep learning framework that puts Python first.</p>
<p><em>homepage</em>: <a href="https://pytorch.org/">https://pytorch.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.4.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.7.4-imkl</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-imkl</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pytorch-geometric">PyTorch-Geometric<a class="headerlink" href="#pytorch-geometric" title="Permanent link">&para;</a></h3>
<p>PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.</p>
<p><em>homepage</em>: <a href="https://github.com/rusty1s/pytorch_geometric">https://github.com/rusty1s/pytorch_geometric</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-3.7.4-PyTorch-1.4.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td><code>-Python-3.7.4-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-PyTorch-1.9.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-PyTorch-1.12.1-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pytorch-ignite">PyTorch-Ignite<a class="headerlink" href="#pytorch-ignite" title="Permanent link">&para;</a></h3>
<p>Ignite is a high-level library to help with training and evaluating neural networks in PyTorch flexibly and transparently.</p>
<p><em>homepage</em>: <a href="https://pytorch-ignite.ai/">https://pytorch-ignite.ai/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.9</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pytorch-lightning">PyTorch-Lightning<a class="headerlink" href="#pytorch-lightning" title="Permanent link">&para;</a></h3>
<p>PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers.</p>
<p><em>homepage</em>: <a href="https://pytorchlightning.ai">https://pytorchlightning.ai</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.9</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.5.9</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.7.7</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.7.7</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="pytorch3d">PyTorch3D<a class="headerlink" href="#pytorch3d" title="Permanent link">&para;</a></h3>
<p>PyTorch3D is FAIR's library of reusable components for deep learning with 3D data.</p>
<p><em>homepage</em>: <a href="https://pytorch3d.org/">https://pytorch3d.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyvcf">PyVCF<a class="headerlink" href="#pyvcf" title="Permanent link">&para;</a></h3>
<p>A Variant Call Format reader for Python.</p>
<p><em>homepage</em>: <a href="https://github.com/jamescasbon/PyVCF">https://github.com/jamescasbon/PyVCF</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyvcf3">PyVCF3<a class="headerlink" href="#pyvcf3" title="Permanent link">&para;</a></h3>
<p>A VCFv4.0 and 4.1 parser for Python. The intent of this module is to mimic the csv module in the Python stdlib, as opposed to more flexible serialization formats like JSON or YAML. vcf will attempt to parse the content of each record based on the data types specified in the meta-information lines -- specifically the ##INFO and ##FORMAT lines. If these lines are missing or incomplete, it will check against the reserved types mentioned in the spec. Failing that, it will just return strings. PyVCF3 has been created because the Official PyVCF repository is no longer maintained and do not accept any pull requests. This fork is for python 3 only and has been published on pyPI as PyVCF3.</p>
<p><em>homepage</em>: <a href="https://github.com/dridk/PyVCF3">https://github.com/dridk/PyVCF3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pywannier90">pyWannier90<a class="headerlink" href="#pywannier90" title="Permanent link">&para;</a></h3>
<p>A Wannier90 Python interface for VASP and PySCF</p>
<p><em>homepage</em>: <a href="https://github.com/hungpham2017/pyWannier90">https://github.com/hungpham2017/pyWannier90</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2021-12-07</code></td>
<td><code>foss/2021a</code>, <code>gomkl/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="pywavelets">PyWavelets<a class="headerlink" href="#pywavelets" title="Permanent link">&para;</a></h3>
<p>PyWavelets is open source wavelet transform software for Python.</p>
<p><em>homepage</em>: <a href="https://pywavelets.readthedocs.io">https://pywavelets.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td></td>
<td><code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="pyyaml">PyYAML<a class="headerlink" href="#pyyaml" title="Permanent link">&para;</a></h3>
<p>PyYAML is a YAML parser and emitter for the Python programming language.</p>
<p><em>homepage</em>: <a href="https://github.com/yaml/pyyaml">https://github.com/yaml/pyyaml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.11</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.13</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.13</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.13</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.3</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.3.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.4.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="pyzmq">PyZMQ<a class="headerlink" href="#pyzmq" title="Permanent link">&para;</a></h3>
<p>Python bindings for ZeroMQ</p>
<p><em>homepage</em>: <a href="https://www.zeromq.org/bindings:python">https://www.zeromq.org/bindings:python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15.2.0</code></td>
<td><code>-Python-2.7.11-zmq4</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>15.2.0</code></td>
<td><code>-Python-3.5.1-zmq4</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>15.3.0</code></td>
<td><code>-Python-2.7.11-zmq4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>15.3.0</code></td>
<td><code>-Python-3.5.1-zmq4</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>15.4.0</code></td>
<td><code>-Python-2.7.12-zmq4</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>15.4.0</code></td>
<td><code>-Python-3.5.2-zmq4</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>16.0.2</code></td>
<td><code>-Python-2.7.12-zmq4</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>16.0.2</code></td>
<td><code>-Python-2.7.13-zmq4</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>16.0.2</code></td>
<td><code>-Python-3.5.2-zmq4</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>16.0.3</code></td>
<td><code>-Python-2.7.14-zmq4</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>17.0.0</code></td>
<td><code>-Python-2.7.14-zmq4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>17.0.0</code></td>
<td><code>-Python-3.6.4-zmq4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>18.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>22.3.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>24.0.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="q">Q<a class="headerlink" href="#q" title="Permanent link">&para;</a></h2>
<p><a href="#q6">Q6</a> - <a href="#qca">QCA</a> - <a href="#qcat">qcat</a> - <a href="#qcg-pilotjob">QCG-PilotJob</a> - <a href="#qcint">qcint</a> - <a href="#qcxms">QCxMS</a> - <a href="#qd">QD</a> - <a href="#qdd">QDD</a> - <a href="#qemu">QEMU</a> - <a href="#qgis">QGIS</a> - <a href="#qhull">Qhull</a> - <a href="#qiime">QIIME</a> - <a href="#qiime2">QIIME2</a> - <a href="#qiskit">Qiskit</a> - <a href="#qjson">QJson</a> - <a href="#qml">QML</a> - <a href="#qpth">qpth</a> - <a href="#qrupdate">qrupdate</a> - <a href="#qscintilla">QScintilla</a> - <a href="#qt">Qt</a> - <a href="#qt5">Qt5</a> - <a href="#qt5webkit">Qt5Webkit</a> - <a href="#qtconsole">Qtconsole</a> - <a href="#qtkeychain">QtKeychain</a> - <a href="#qtltools">QTLtools</a> - <a href="#qtop">qtop</a> - <a href="#qtpy">QtPy</a> - <a href="#qualimap">Qualimap</a> - <a href="#quandl">Quandl</a> - <a href="#quantumespresso">QuantumESPRESSO</a> - <a href="#quast">QUAST</a> - <a href="#quazip">QuaZIP</a> - <a href="#quickff">QuickFF</a> - <a href="#quickpic">QuickPIC</a> - <a href="#quip">Quip</a> - <a href="#quorum">Quorum</a> - <a href="#qutip">QuTiP</a> - <a href="#qwt">Qwt</a> - <a href="#qwtpolar">QwtPolar</a></p>
<h3 id="q6">Q6<a class="headerlink" href="#q6" title="Permanent link">&para;</a></h3>
<p>EVB, FEP and LIE simulator.</p>
<p><em>homepage</em>: <a href="https://github.com/qusers/Q6">https://github.com/qusers/Q6</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180205</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="qca">QCA<a class="headerlink" href="#qca" title="Permanent link">&para;</a></h3>
<p>Taking a hint from the similarly-named Java Cryptography Architecture,  QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes  and conventions. QCA separates the API from the implementation, using plugins known as  Providers. The advantage of this model is to allow applications to avoid linking to or  explicitly depending on any particular cryptographic library. This allows one to easily  change or upgrade crypto implementations without even needing to recompile the  application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX.</p>
<p><em>homepage</em>: <a href="https://userbase.kde.org/QCA">https://userbase.kde.org/QCA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.5</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qcat">qcat<a class="headerlink" href="#qcat" title="Permanent link">&para;</a></h3>
<p>qcat is a Python command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/qcat/releases">https://github.com/nanoporetech/qcat/releases</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="qcg-pilotjob">QCG-PilotJob<a class="headerlink" href="#qcg-pilotjob" title="Permanent link">&para;</a></h3>
<p>A python service for easy execution of many tasks inside a single allocation.</p>
<p><em>homepage</em>: <a href="https://qcg-pilotjob.readthedocs.org">https://qcg-pilotjob.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.3</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="qcint">qcint<a class="headerlink" href="#qcint" title="Permanent link">&para;</a></h3>
<p>libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform.</p>
<p><em>homepage</em>: <a href="http://wiki.sunqm.net/libcint">http://wiki.sunqm.net/libcint</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.18</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="qcxms">QCxMS<a class="headerlink" href="#qcxms" title="Permanent link">&para;</a></h3>
<p>QCxMS is a quantum chemical based program to calculate electron ionization (EI)  and collision induced dissociation (CID) mass spectra using Born-Oppenheimer Molecular Dynamics  (BO-MD). It is the successor of the QCEIMS program, in which the EI part is exchanged to x to  account for the greater general applicibility of the program. </p>
<p><em>homepage</em>: <a href="https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html">https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="qd">QD<a class="headerlink" href="#qd" title="Permanent link">&para;</a></h3>
<p>Quad Double computation package</p>
<p><em>homepage</em>: <a href="https://github.com/scibuilder/QD">https://github.com/scibuilder/QD</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.9</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.3.17</code></td>
<td><code>-20160110</code></td>
<td><code>NVHPC/21.2</code></td>
</tr>
</tbody>
</table>
<h3 id="qdd">QDD<a class="headerlink" href="#qdd" title="Permanent link">&para;</a></h3>
<p>A user-friendly program to select microsatellite markers and design primers  from large sequencing projects.</p>
<p><em>homepage</em>: <a href="http://net.imbe.fr/~emeglecz/qdd.html">http://net.imbe.fr/~emeglecz/qdd.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.2</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="qemu">QEMU<a class="headerlink" href="#qemu" title="Permanent link">&para;</a></h3>
<p>QEMU is a generic and open source machine emulator and virtualizer.</p>
<p><em>homepage</em>: <a href="https://www.qemu.org/">https://www.qemu.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.1</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="qgis">QGIS<a class="headerlink" href="#qgis" title="Permanent link">&para;</a></h3>
<p>QGIS is a user friendly Open Source Geographic Information System (GIS)</p>
<p><em>homepage</em>: <a href="http://www.qgis.org/">http://www.qgis.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.14.12</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.18.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.4.12</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.28.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="qhull">Qhull<a class="headerlink" href="#qhull" title="Permanent link">&para;</a></h3>
<p>Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,  halfspace intersection about a point, furthest-site Delaunay triangulation,  and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and  higher dimensions. Qhull implements the Quickhull algorithm for computing the  convex hull. </p>
<p><em>homepage</em>: <a href="http://www.qhull.org">http://www.qhull.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2015.2</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2019.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2020.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qiime">QIIME<a class="headerlink" href="#qiime" title="Permanent link">&para;</a></h3>
<p>QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.</p>
<p><em>homepage</em>: <a href="http://qiime.org/">http://qiime.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="qiime2">QIIME2<a class="headerlink" href="#qiime2" title="Permanent link">&para;</a></h3>
<p>QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.</p>
<p><em>homepage</em>: <a href="http://qiime2.org/">http://qiime2.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2017.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019.7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020.11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="qiskit">Qiskit<a class="headerlink" href="#qiskit" title="Permanent link">&para;</a></h3>
<p>Qiskit is an open-source framework for working with noisy quantum computers  at the level of pulses, circuits, and algorithms.</p>
<p><em>homepage</em>: <a href="https://qiskit.org">https://qiskit.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.31.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="qjson">QJson<a class="headerlink" href="#qjson" title="Permanent link">&para;</a></h3>
<p>QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa.</p>
<p><em>homepage</em>: <a href="http://qjson.sourceforge.net/">http://qjson.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="qml">QML<a class="headerlink" href="#qml" title="Permanent link">&para;</a></h3>
<p>QML is a Python2/3-compatible toolkit for representation learning of properties  of molecules and solids.</p>
<p><em>homepage</em>: <a href="https://www.qmlcode.org">https://www.qmlcode.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.10</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="qpth">qpth<a class="headerlink" href="#qpth" title="Permanent link">&para;</a></h3>
<p>A fast and differentiable QP solver for PyTorch. </p>
<p><em>homepage</em>: <a href="https://locuslab.github.io/qpth/">https://locuslab.github.io/qpth/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.13-20190626</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="qrupdate">qrupdate<a class="headerlink" href="#qrupdate" title="Permanent link">&para;</a></h3>
<p>qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/qrupdate/">https://sourceforge.net/projects/qrupdate/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016a</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="qscintilla">QScintilla<a class="headerlink" href="#qscintilla" title="Permanent link">&para;</a></h3>
<p>QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control</p>
<p><em>homepage</em>: <a href="https://www.riverbankcomputing.com/software/qscintilla">https://www.riverbankcomputing.com/software/qscintilla</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.10</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.11.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.11.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qt">Qt<a class="headerlink" href="#qt" title="Permanent link">&para;</a></h3>
<p>Qt is a comprehensive cross-platform C++ application framework.</p>
<p><em>homepage</em>: <a href="http://qt.io/">http://qt.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.8</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.8.6</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.8.7</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.8.7</code></td>
<td><code>-GLib-2.48.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="qt5">Qt5<a class="headerlink" href="#qt5" title="Permanent link">&para;</a></h3>
<p>Qt is a comprehensive cross-platform C++ application framework.</p>
<p><em>homepage</em>: <a href="https://qt.io/">https://qt.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.7.0</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.8.0</code></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.9.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>5.9.8</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>5.10.1</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.11.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>5.12.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.13.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>5.14.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.14.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.15.2</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.15.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>5.15.7</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qt5webkit">Qt5Webkit<a class="headerlink" href="#qt5webkit" title="Permanent link">&para;</a></h3>
<p>Qt Port of WebKit. WebKit is an open source web browser engine.</p>
<p><em>homepage</em>: <a href="https://github.com/qt/qtwebkit">https://github.com/qt/qtwebkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.212.0-alpha3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.212.0-alpha4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qtconsole">Qtconsole<a class="headerlink" href="#qtconsole" title="Permanent link">&para;</a></h3>
<p>A rich Qt-based console for working with Jupyter kernels, supporting rich media  output, session export, and more. The Qtconsole is a very lightweight application that largely feels like a        terminal, but provides a number of enhancements only possible in a GUI, such as  inline figures, proper multiline editing with syntax highlighting, graphical     calltips, and more.</p>
<p><em>homepage</em>: <a href="https://jupyter.org/">https://jupyter.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7.7</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>5.3.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qtkeychain">QtKeychain<a class="headerlink" href="#qtkeychain" title="Permanent link">&para;</a></h3>
<p>Platform-independent Qt API for storing passwords securely.</p>
<p><em>homepage</em>: <a href="https://github.com/frankosterfeld/qtkeychain">https://github.com/frankosterfeld/qtkeychain</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.1</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.13.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qtltools">QTLtools<a class="headerlink" href="#qtltools" title="Permanent link">&para;</a></h3>
<p>QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs)  in few easy-to-perform steps.</p>
<p><em>homepage</em>: <a href="https://qtltools.github.io/qtltools/">https://qtltools.github.io/qtltools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="qtop">qtop<a class="headerlink" href="#qtop" title="Permanent link">&para;</a></h3>
<p>qtop is a nifty command-line tool for monitoring queueing systems, esp. PBS/torque.  It tries to fit as much information as possible in your screen's real estate,  by stitching together the output of commands like pbsnodes -a, qstat &amp; qstat -q.  It is possible to write wrappers for other platforms -people have done so for SGE, OAR etc-  or, even examine traces offline and present the sampled information.</p>
<p><em>homepage</em>: <a href="http://cern.ch/fotis/QTOP/">http://cern.ch/fotis/QTOP/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>53</code></td>
<td><code>-1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="qtpy">QtPy<a class="headerlink" href="#qtpy" title="Permanent link">&para;</a></h3>
<p>QtPy is a small abstraction layer that lets you write applications using a single API call to  either PyQt or PySide. It provides support for PyQt5, PyQt4, PySide2 and PySide.</p>
<p><em>homepage</em>: <a href="https://github.com/spyder-ide/qtpy">https://github.com/spyder-ide/qtpy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qualimap">Qualimap<a class="headerlink" href="#qualimap" title="Permanent link">&para;</a></h3>
<p>Qualimap 2 is a platform-independent application written in Java and R that provides both  a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of  alignment sequencing data and its derivatives like feature counts.</p>
<p><em>homepage</em>: <a href="http://qualimap.bioinfo.cipf.es/">http://qualimap.bioinfo.cipf.es/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.1</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="quandl">Quandl<a class="headerlink" href="#quandl" title="Permanent link">&para;</a></h3>
<p>A Python library for Quandls RESTful API.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/Quandl">https://pypi.python.org/pypi/Quandl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.4.8</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="quantumespresso">QuantumESPRESSO<a class="headerlink" href="#quantumespresso" title="Permanent link">&para;</a></h3>
<p>Quantum ESPRESSO  is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). </p>
<p><em>homepage</em>: <a href="https://www.quantum-espresso.org">https://www.quantum-espresso.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.3.0</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>-hybrid</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>6.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.2</code></td>
<td></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>6.2.1</code></td>
<td></td>
<td><code>iomkl/2017b</code></td>
</tr>
<tr>
<td><code>6.3</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>6.4.1</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>6.5</code></td>
<td></td>
<td><code>intel/2019a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>6.6</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>6.7</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>intel/2019b</code>, <code>intel/2021a</code>, <code>iomkl/2019b</code></td>
</tr>
<tr>
<td><code>6.8</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>7.0</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="quast">QUAST<a class="headerlink" href="#quast" title="Permanent link">&para;</a></h3>
<p>QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison.</p>
<p><em>homepage</em>: <a href="https://github.com/ablab/quast">https://github.com/ablab/quast</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>4.6.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>5.0.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="quazip">QuaZIP<a class="headerlink" href="#quazip" title="Permanent link">&para;</a></h3>
<p>QuaZIP is the C++ wrapper for Gilles Vollant's ZIP/UNZIP   package (AKA Minizip) using Trolltech's Qt library.</p>
<p><em>homepage</em>: <a href="https://stachenov.github.io/quazip">https://stachenov.github.io/quazip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="quickff">QuickFF<a class="headerlink" href="#quickff" title="Permanent link">&para;</a></h3>
<p>QuickFF is a Python package developed at the Center for Molecular Modeling (CMM) to quickly derive accurate force fields from ab initio calculations.</p>
<p><em>homepage</em>: <a href="https://molmod.github.io/QuickFF/">https://molmod.github.io/QuickFF/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.2.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.2.7</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="quickpic">QuickPIC<a class="headerlink" href="#quickpic" title="Permanent link">&para;</a></h3>
<p>QuickPIC is a 3D parallel (MPI &amp; OpenMP Hybrid) Quasi-Static PIC code,  which is developed based on the framework UPIC. QuickPIC can efficiently simulate plasma  based accelerator problems. This is the UCLA Plasma Simulation Group's official open-source  repository for QuickPIC.</p>
<p><em>homepage</em>: <a href="https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource">https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210224</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="quip">Quip<a class="headerlink" href="#quip" title="Permanent link">&para;</a></h3>
<p>Quip compresses next-generation sequencing data with extreme prejudice.  It supports input and output in the FASTQ and SAM/BAM formats,  compressing large datasets to as little as 15% of their original size.</p>
<p><em>homepage</em>: <a href="http://homes.cs.washington.edu/~dcjones/quip">http://homes.cs.washington.edu/~dcjones/quip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.8</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="quorum">Quorum<a class="headerlink" href="#quorum" title="Permanent link">&para;</a></h3>
<p>QuorUM is an error corrector for Illumina reads</p>
<p><em>homepage</em>: <a href="http://www.genome.umd.edu/quorum.html">http://www.genome.umd.edu/quorum.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="qutip">QuTiP<a class="headerlink" href="#qutip" title="Permanent link">&para;</a></h3>
<p>QuTiP is open-source software for simulating the dynamics of open quantum systems.</p>
<p><em>homepage</em>: <a href="http://qutip.org">http://qutip.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="qwt">Qwt<a class="headerlink" href="#qwt" title="Permanent link">&para;</a></h3>
<p>The Qwt library contains GUI Components and utility classes which are primarily useful for programs  with a technical background.</p>
<p><em>homepage</em>: <a href="https://qwt.sourceforge.io/">https://qwt.sourceforge.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>6.1.3</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>6.1.4</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.1.5</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="qwtpolar">QwtPolar<a class="headerlink" href="#qwtpolar" title="Permanent link">&para;</a></h3>
<p>The QwtPolar library contains classes for displaying values on a polar coordinate system.</p>
<p><em>homepage</em>: <a href="http://qwtpolar.sourceforge.net/">http://qwtpolar.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h2 id="r">R<a class="headerlink" href="#r" title="Permanent link">&para;</a></h2>
<p><a href="#r">R</a> - <a href="#r-bundle-bioconductor">R-bundle-Bioconductor</a> - <a href="#r-inla">R-INLA</a> - <a href="#r-keras">R-keras</a> - <a href="#r-opencv">R-opencv</a> - <a href="#r-tesseract">R-tesseract</a> - <a href="#r2jags">R2jags</a> - <a href="#racon">Racon</a> - <a href="#radeontop">radeontop</a> - <a href="#ragoo">RaGOO</a> - <a href="#ragout">Ragout</a> - <a href="#ragtag">RagTag</a> - <a href="#rampart">rampart</a> - <a href="#randfold">randfold</a> - <a href="#randrproto">randrproto</a> - <a href="#rapidcsv">rapidcsv</a> - <a href="#rapidjson">RapidJSON</a> - <a href="#rapidnj">rapidNJ</a> - <a href="#rapidtide">rapidtide</a> - <a href="#rapsearch2">RAPSearch2</a> - <a href="#rascaf">Rascaf</a> - <a href="#raspa2">RASPA2</a> - <a href="#rasterio">rasterio</a> - <a href="#rasterstats">rasterstats</a> - <a href="#ratatosk">Ratatosk</a> - <a href="#raxml">RAxML</a> - <a href="#raxml-ng">RAxML-NG</a> - <a href="#ray-assembler">Ray-assembler</a> - <a href="#ray-project">Ray-project</a> - <a href="#raysect">Raysect</a> - <a href="#rbfopt">RBFOpt</a> - <a href="#rclone">rclone</a> - <a href="#rcorrector">Rcorrector</a> - <a href="#rcppgsl">RcppGSL</a> - <a href="#rcuda">rCUDA</a> - <a href="#rdflib">RDFlib</a> - <a href="#rdkit">RDKit</a> - <a href="#rdp-classifier">RDP-Classifier</a> - <a href="#re2">RE2</a> - <a href="#re2c">re2c</a> - <a href="#reads2snp">Reads2snp</a> - <a href="#reapr">Reapr</a> - <a href="#recon">RECON</a> - <a href="#red">Red</a> - <a href="#redis">Redis</a> - <a href="#redis-py">redis-py</a> - <a href="#redundans">Redundans</a> - <a href="#reframe">ReFrame</a> - <a href="#regionmask">regionmask</a> - <a href="#regtools">RegTools</a> - <a href="#relate">Relate</a> - <a href="#relion">RELION</a> - <a href="#rematch">ReMatCh</a> - <a href="#remora">REMORA</a> - <a href="#renderproto">renderproto</a> - <a href="#repasthpc">RepastHPC</a> - <a href="#repeatmasker">RepeatMasker</a> - <a href="#repeatmodeler">RepeatModeler</a> - <a href="#repeatscout">RepeatScout</a> - <a href="#request">request</a> - <a href="#requests">requests</a> - <a href="#rerconverge">RERconverge</a> - <a href="#resolos">resolos</a> - <a href="#retworkx">retworkx</a> - <a href="#revbayes">RevBayes</a> - <a href="#rgdal">rgdal</a> - <a href="#rgeos">rgeos</a> - <a href="#rgurobi">Rgurobi</a> - <a href="#rhdf5">rhdf5</a> - <a href="#rheotool">RheoTool</a> - <a href="#rickflow">rickflow</a> - <a href="#rinchi">RInChI</a> - <a href="#rioxarray">rioxarray</a> - <a href="#rising">rising</a> - <a href="#rivet">Rivet</a> - <a href="#rjags">rjags</a> - <a href="#rlcard">RLCard</a> - <a href="#rmarkdown">rmarkdown</a> - <a href="#rmath">Rmath</a> - <a href="#rmats-turbo">rMATS-turbo</a> - <a href="#rmblast">RMBlast</a> - <a href="#rna-bloom">RNA-Bloom</a> - <a href="#rna-seqc">RNA-SeQC</a> - <a href="#rnaclust">RNAclust</a> - <a href="#rnacode">RNAcode</a> - <a href="#rnaindel">RNAIndel</a> - <a href="#rnammer">RNAmmer</a> - <a href="#rnaquast">rnaQUAST</a> - <a href="#rnaz">RNAz</a> - <a href="#rnbeads">RnBeads</a> - <a href="#roary">Roary</a> - <a href="#rocm">ROCm</a> - <a href="#rocm-cmake">rocm-cmake</a> - <a href="#rocm-compilersupport">ROCm-CompilerSupport</a> - <a href="#rocm-smi">rocm-smi</a> - <a href="#rocminfo">rocminfo</a> - <a href="#rocr-runtime">ROCR-Runtime</a> - <a href="#roct-thunk-interface">ROCT-Thunk-Interface</a> - <a href="#roi_pac">ROI_PAC</a> - <a href="#rome">ROME</a> - <a href="#root">ROOT</a> - <a href="#root_numpy">root_numpy</a> - <a href="#rootpy">rootpy</a> - <a href="#rosetta">Rosetta</a> - <a href="#rpmrebuild">rpmrebuild</a> - <a href="#rpy2">rpy2</a> - <a href="#rqgis3">RQGIS3</a> - <a href="#rsem">RSEM</a> - <a href="#rseqc">RSeQC</a> - <a href="#rstan">RStan</a> - <a href="#rstanarm">rstanarm</a> - <a href="#rstudio-server">RStudio-Server</a> - <a href="#rtg-tools">RTG-Tools</a> - <a href="#rtree">Rtree</a> - <a href="#ruamel.yaml">ruamel.yaml</a> - <a href="#ruby">Ruby</a> - <a href="#ruby-tk">Ruby-Tk</a> - <a href="#ruffus">ruffus</a> - <a href="#rust">Rust</a></p>
<h3 id="r_1">R<a class="headerlink" href="#r_1" title="Permanent link">&para;</a></h3>
<p>R is a free software environment for statistical computing  and graphics.</p>
<p><em>homepage</em>: <a href="https://www.r-project.org/">https://www.r-project.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2.3</code></td>
<td><code>-bare</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.2.3</code></td>
<td><code>-libX11-1.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td><code>-X11-20160819</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.3.3</code></td>
<td><code>-X11-20170314</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-X11-20170314</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>-X11-20160819</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td><code>-X11-20171023</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td><code>-X11-20171023-HDF5-1.8.19</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.4.4</code></td>
<td><code>-X11-20180131</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>-X11-20180131</code></td>
<td><code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>-bare</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.6.0</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.6.3</code></td>
<td></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4.0.4</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4.0.5</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.1.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.2.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td></td>
<td><code>foss/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="r-bundle-bioconductor">R-bundle-Bioconductor<a class="headerlink" href="#r-bundle-bioconductor" title="Permanent link">&para;</a></h3>
<p>Bioconductor provides tools for the analysis and coprehension  of high-throughput genomic data.</p>
<p><em>homepage</em>: <a href="https://bioconductor.org">https://bioconductor.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>-R-3.2.3</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.6</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.7</code></td>
<td><code>-R-3.5.0</code></td>
<td><code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.7</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.8</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.9</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.10</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.11</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.12</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>3.13</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.14</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.15</code></td>
<td><code>-R-4.2.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.15</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="r-inla">R-INLA<a class="headerlink" href="#r-inla" title="Permanent link">&para;</a></h3>
<p>R-INLA is a package in R that do approximate Bayesian inference for Latent Gaussian Models.</p>
<p><em>homepage</em>: <a href="https://www.r-inla.org">https://www.r-inla.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>21.05.02</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="r-keras">R-keras<a class="headerlink" href="#r-keras" title="Permanent link">&para;</a></h3>
<p>Interface to 'Keras' <a href="https://keras.io">https://keras.io</a>, a high-level neural networks 'API'. </p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/keras">https://cran.r-project.org/web/packages/keras</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.6</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.2.5.0</code></td>
<td><code>-Python-3.7.2-R-3.6.0</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>2.2.5.0</code></td>
<td><code>-Python-3.7.4-R-3.6.2</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="r-opencv">R-opencv<a class="headerlink" href="#r-opencv" title="Permanent link">&para;</a></h3>
<p>Experimenting with computer vision and machine learning in R. This package exposes some of the available OpenCV algorithms, such as edge, body or face detection. These can either be applied to analyze static images, or to filter live video footage from a camera device.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/opencv/">https://cran.r-project.org/web/packages/opencv/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="r-tesseract">R-tesseract<a class="headerlink" href="#r-tesseract" title="Permanent link">&para;</a></h3>
<p>The R extension for using tesseract</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=tesseract">https://cran.r-project.org/package=tesseract</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="r2jags">R2jags<a class="headerlink" href="#r2jags" title="Permanent link">&para;</a></h3>
<p>Providing wrapper functions to implement Bayesian analysis in JAGS.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/R2jags">https://cran.r-project.org/web/packages/R2jags</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7-1</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="racon">Racon<a class="headerlink" href="#racon" title="Permanent link">&para;</a></h3>
<p>Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.</p>
<p><em>homepage</em>: <a href="https://github.com/lbcb-sci/racon">https://github.com/lbcb-sci/racon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.4.7</code></td>
<td><code>GCCcore/8.2.0</code>, <code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>1.4.10</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>1.4.13</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.4.21</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="radeontop">radeontop<a class="headerlink" href="#radeontop" title="Permanent link">&para;</a></h3>
<p>View your GPU utilization, both for the total activity percent and individual blocks.</p>
<p><em>homepage</em>: <a href="https://github.com/clbr/radeontop">https://github.com/clbr/radeontop</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ragoo">RaGOO<a class="headerlink" href="#ragoo" title="Permanent link">&para;</a></h3>
<p>A tool to order and orient genome assembly contigs via Minimap2 alignments to a reference genome</p>
<p><em>homepage</em>: <a href="https://github.com/malonge/RaGOO">https://github.com/malonge/RaGOO</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="ragout">Ragout<a class="headerlink" href="#ragout" title="Permanent link">&para;</a></h3>
<p>Ragout (Reference-Assisted Genome Ordering UTility) is a tool for chromosome-level scaffolding using multiple references. Given initial assembly fragments (contigs/scaffolds) and one or multiple related references (complete or draft), it produces a chromosome-scale assembly (as a set of scaffolds).</p>
<p><em>homepage</em>: <a href="https://github.com/fenderglass/Ragout">https://github.com/fenderglass/Ragout</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ragtag">RagTag<a class="headerlink" href="#ragtag" title="Permanent link">&para;</a></h3>
<p>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies. Tasks include: homology-based misassembly correction, homology-based assembly scaffolding and patching, and scaffold merging. RagTag also provides command line utilities for working with common genome assembly file formats.</p>
<p><em>homepage</em>: <a href="https://github.com/malonge/RagTag">https://github.com/malonge/RagTag</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rampart">rampart<a class="headerlink" href="#rampart" title="Permanent link">&para;</a></h3>
<p>Read Assignment, Mapping, and Phylogenetic Analysis in Real Time.</p>
<p><em>homepage</em>: <a href="https://github.com/artic-network/rampart/">https://github.com/artic-network/rampart/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.2.0rc3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="randfold">randfold<a class="headerlink" href="#randfold" title="Permanent link">&para;</a></h3>
<p>Minimum free energy of folding randomization test software</p>
<p><em>homepage</em>: <a href="http://bioinformatics.psb.ugent.be/software/details/Randfold">http://bioinformatics.psb.ugent.be/software/details/Randfold</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="randrproto">randrproto<a class="headerlink" href="#randrproto" title="Permanent link">&para;</a></h3>
<p>Xrandr protocol and ancillary headers</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="rapidcsv">rapidcsv<a class="headerlink" href="#rapidcsv" title="Permanent link">&para;</a></h3>
<p>Rapidcsv is a C++ header-only library for CSV parsing.  While the name admittedly was inspired by the rapidjson project,  the objectives are not the same. The goal of rapidcsv is to be  an easy-to-use CSV library enabling rapid development.  For optimal performance (be it CPU or memory usage) a CSV parser  implemented for the specific use-case is likely to be more performant. </p>
<p><em>homepage</em>: <a href="https://github.com/d99kris/rapidcsv">https://github.com/d99kris/rapidcsv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.64</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rapidjson">RapidJSON<a class="headerlink" href="#rapidjson" title="Permanent link">&para;</a></h3>
<p>A fast JSON parser/generator for C++ with both SAX/DOM style API</p>
<p><em>homepage</em>: <a href="https://rapidjson.org">https://rapidjson.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rapidnj">rapidNJ<a class="headerlink" href="#rapidnj" title="Permanent link">&para;</a></h3>
<p>RapidNJ is an algorithmic engineered implementation of canonical  neighbour-joining. It uses an efficient search heuristic to speed-up the core  computations of the neighbour-joining method that enables RapidNJ to  outperform other state-of-the-art neighbour-joining implementations.</p>
<p><em>homepage</em>: <a href="https://github.com/somme89/rapidNJ">https://github.com/somme89/rapidNJ</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rapidtide">rapidtide<a class="headerlink" href="#rapidtide" title="Permanent link">&para;</a></h3>
<p>Rapidtide is a suite of python programs used to perform time delay analysis on functional imaging  data to find time lagged correlations between the voxelwise time series and other time series.</p>
<p><em>homepage</em>: <a href="https://rapidtide.readthedocs.io">https://rapidtide.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="rapsearch2">RAPSearch2<a class="headerlink" href="#rapsearch2" title="Permanent link">&para;</a></h3>
<p>RAPSearch stands for Reduced Alphabet based Protein similarity Search </p>
<p><em>homepage</em>: <a href="https://omics.informatics.indiana.edu/mg/RAPSearch2">https://omics.informatics.indiana.edu/mg/RAPSearch2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.24</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rascaf">Rascaf<a class="headerlink" href="#rascaf" title="Permanent link">&para;</a></h3>
<p>Rascaf (RnA-seq SCAFfolder) uses continuity and order information from paired-end RNA-seq reads  to improve a draft assembly, particularly in the gene regions.</p>
<p><em>homepage</em>: <a href="https://github.com/mourisl/Rascaf">https://github.com/mourisl/Rascaf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="raspa2">RASPA2<a class="headerlink" href="#raspa2" title="Permanent link">&para;</a></h3>
<p>RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials.</p>
<p><em>homepage</em>: <a href="https://iraspa.org/raspa/">https://iraspa.org/raspa/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.41</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rasterio">rasterio<a class="headerlink" href="#rasterio" title="Permanent link">&para;</a></h3>
<p>Rasterio reads and writes geospatial raster data.</p>
<p><em>homepage</em>: <a href="https://github.com/mapbox/rasterio">https://github.com/mapbox/rasterio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.7</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.2.10</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.3.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rasterstats">rasterstats<a class="headerlink" href="#rasterstats" title="Permanent link">&para;</a></h3>
<p>rasterstats is a Python module for summarizing geospatial raster datasets based on vector geometries.</p>
<p><em>homepage</em>: <a href="https://github.com/perrygeo/python-rasterstats">https://github.com/perrygeo/python-rasterstats</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.15.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="ratatosk">Ratatosk<a class="headerlink" href="#ratatosk" title="Permanent link">&para;</a></h3>
<p>Phased hybrid error correction of long reads using colored de Bruijn graphs</p>
<p><em>homepage</em>: <a href="https://github.com/DecodeGenetics/Ratatosk">https://github.com/DecodeGenetics/Ratatosk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="raxml">RAxML<a class="headerlink" href="#raxml" title="Permanent link">&para;</a></h3>
<p>RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.</p>
<p><em>homepage</em>: <a href="https://github.com/stamatak/standard-RAxML">https://github.com/stamatak/standard-RAxML</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.2.4</code></td>
<td><code>-hybrid-avx2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.2.9</code></td>
<td><code>-hybrid-avx2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.2.10</code></td>
<td><code>-hybrid-avx2</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>8.2.11</code></td>
<td><code>-hybrid-avx</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.2.11</code></td>
<td><code>-hybrid-avx2</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.2.11</code></td>
<td><code>-hybrid-sse3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.2.12</code></td>
<td><code>-hybrid-avx2</code></td>
<td><code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompi/2021b</code>, <code>iimpi/2019b</code>, <code>intel/2018b</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>8.2.12</code></td>
<td><code>-pthreads-avx2</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="raxml-ng">RAxML-NG<a class="headerlink" href="#raxml-ng" title="Permanent link">&para;</a></h3>
<p>RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)  optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree  Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree.</p>
<p><em>homepage</em>: <a href="https://github.com/amkozlov/raxml-ng">https://github.com/amkozlov/raxml-ng</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>GCC/8.3.0</code>, <code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ray-assembler">Ray-assembler<a class="headerlink" href="#ray-assembler" title="Permanent link">&para;</a></h3>
<p>Parallel genome assemblies for parallel DNA sequencing</p>
<p><em>homepage</em>: <a href="http://denovoassembler.sourceforge.net/">http://denovoassembler.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>iimpi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ray-project">Ray-project<a class="headerlink" href="#ray-project" title="Permanent link">&para;</a></h3>
<p>Ray is a fast and simple framework for building and running distributed applications.</p>
<p><em>homepage</em>: <a href="https://docs.ray.io/en/latest/">https://docs.ray.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.9.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.13.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="raysect">Raysect<a class="headerlink" href="#raysect" title="Permanent link">&para;</a></h3>
<p>Raysect is an OOP ray-tracing framework for Python</p>
<p><em>homepage</em>: <a href="https://raysect.org">https://raysect.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rbfopt">RBFOpt<a class="headerlink" href="#rbfopt" title="Permanent link">&para;</a></h3>
<p>RBFOpt is a Python library for black-box optimization (also known as derivative-free optimization).</p>
<p><em>homepage</em>: <a href="https://github.com/coin-or/rbfopt">https://github.com/coin-or/rbfopt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.1</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rclone">rclone<a class="headerlink" href="#rclone" title="Permanent link">&para;</a></h3>
<p>Rclone is a command line program to sync files and directories to and from  a variety of online storage services </p>
<p><em>homepage</em>: <a href="https://rclone.org">https://rclone.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.42</code></td>
<td><code>-amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.54.1</code></td>
<td><code>-amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.56.0</code></td>
<td><code>-amd64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.57.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="rcorrector">Rcorrector<a class="headerlink" href="#rcorrector" title="Permanent link">&para;</a></h3>
<p>Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/mourisl/Rcorrector">https://github.com/mourisl/Rcorrector</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="rcppgsl">RcppGSL<a class="headerlink" href="#rcppgsl" title="Permanent link">&para;</a></h3>
<p>The 'RcppGSL' package provides an easy-to-use interface  between 'GSL' data structures and R using concepts from 'Rcpp' which is itself  a package that eases the interfaces between R and C++.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/RcppGSL">https://cran.r-project.org/web/packages/RcppGSL</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.8</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rcuda">rCUDA<a class="headerlink" href="#rcuda" title="Permanent link">&para;</a></h3>
<p>The rCUDA Framework enables the concurrent usage of CUDA-compatible devices remotely.</p>
<p><em>homepage</em>: <a href="http://www.rcuda.net/">http://www.rcuda.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.1</code></td>
<td><code>_linux_64_Ubuntu10.04</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td><code>_linux_64_scientificLinux6</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="rdflib">RDFlib<a class="headerlink" href="#rdflib" title="Permanent link">&para;</a></h3>
<p>RDFLib is a Python library for working with RDF, a simple yet powerful language  for representing information.</p>
<p><em>homepage</em>: <a href="https://github.com/RDFLib/rdflib">https://github.com/RDFLib/rdflib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.2</code></td>
<td><code>GCCcore/8.3.0</code>, <code>foss/2019a</code></td>
</tr>
<tr>
<td><code>5.0.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>6.2.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rdkit">RDKit<a class="headerlink" href="#rdkit" title="Permanent link">&para;</a></h3>
<p>RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.</p>
<p><em>homepage</em>: <a href="https://www.rdkit.org">https://www.rdkit.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.09.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2019.09.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2020.03.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2020.09.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2021.03.4</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2022.03.5</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2022.09.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rdp-classifier">RDP-Classifier<a class="headerlink" href="#rdp-classifier" title="Permanent link">&para;</a></h3>
<p>The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic  assignments from domain to genus, with confidence estimates for each assignment.</p>
<p><em>homepage</em>: <a href="http://sourceforge.net/projects/rdp-classifier">http://sourceforge.net/projects/rdp-classifier</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>-Java-1.7.0_60</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.12</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.13</code></td>
<td><code>-Java-17</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="re2">RE2<a class="headerlink" href="#re2" title="Permanent link">&para;</a></h3>
<p>RE2 is a fast, safe, thread-friendly alternative to backtracking regular expression engines like those used in PCRE, Perl, and Python. It is a C++ library. </p>
<p><em>homepage</em>: <a href="https://github.com/google/re2">https://github.com/google/re2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020-07-01</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2021-06-01</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2022-02-01</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2022-06-01</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="re2c">re2c<a class="headerlink" href="#re2c" title="Permanent link">&para;</a></h3>
<p>re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons.</p>
<p><em>homepage</em>: <a href="https://re2c.org">https://re2c.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="reads2snp">Reads2snp<a class="headerlink" href="#reads2snp" title="Permanent link">&para;</a></h3>
<p>reads2snp is a SNP and genotype caller: it predicts the genotype of distinct individuals   at distinct positions of a set of sequences based on read mapping / read counts. Its typical input is a bam file.   Its typical output is a vcf file. It is written in C++, based on the bio++ libraries, multi-threaded with openMP,   available under Linux and MacOS</p>
<p><em>homepage</em>: <a href="http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools">http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="reapr">Reapr<a class="headerlink" href="#reapr" title="Permanent link">&para;</a></h3>
<p>A tool that evaluates the accuracy of a genome assembly using mapped paired end reads,  without the use of a reference genome for comparison.</p>
<p><em>homepage</em>: <a href="https://www.sanger.ac.uk/science/tools/reapr">https://www.sanger.ac.uk/science/tools/reapr</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.18</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="recon">RECON<a class="headerlink" href="#recon" title="Permanent link">&para;</a></h3>
<p>Patched version of RECON to be used with RepeatModeler.</p>
<p><em>homepage</em>: <a href="https://www.repeatmasker.org/RepeatModeler/">https://www.repeatmasker.org/RepeatModeler/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.08</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="red">Red<a class="headerlink" href="#red" title="Permanent link">&para;</a></h3>
<p>Red (REpeat Detector)</p>
<p><em>homepage</em>: <a href="http://toolsmith.ens.utulsa.edu/">http://toolsmith.ens.utulsa.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2015-05-22</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="redis">Redis<a class="headerlink" href="#redis" title="Permanent link">&para;</a></h3>
<p>Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes, and streams. Redis has built-in replication, Lua scripting, LRU eviction, transactions, and different levels of on-disk persistence, and provides high availability via Redis Sentinel and automatic partitioning with Redis Cluster.</p>
<p><em>homepage</em>: <a href="https://redis.io">https://redis.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2.6</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>7.0.8</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="redis-py">redis-py<a class="headerlink" href="#redis-py" title="Permanent link">&para;</a></h3>
<p>The Python interface to the Redis key-value store.</p>
<p><em>homepage</em>: <a href="https://github.com/redis/redis-py">https://github.com/redis/redis-py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.3.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.5.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="redundans">Redundans<a class="headerlink" href="#redundans" title="Permanent link">&para;</a></h3>
<p>Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/lpryszcz/redundans">https://github.com/lpryszcz/redundans</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13c</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="reframe">ReFrame<a class="headerlink" href="#reframe" title="Permanent link">&para;</a></h3>
<p>ReFrame is a framework for writing regression tests for HPC systems.</p>
<p><em>homepage</em>: <a href="https://github.com/reframe-hpc/reframe">https://github.com/reframe-hpc/reframe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.18</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.19</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.20</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.21</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.5.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.6.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.8.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.10.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.11.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.11.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.12.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.5</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="regionmask">regionmask<a class="headerlink" href="#regionmask" title="Permanent link">&para;</a></h3>
<p>regionmask creates masks of geographical regions. It determines to which geographic region each grid point belongs.</p>
<p><em>homepage</em>: <a href="https://regionmask.readthedocs.io">https://regionmask.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="regtools">RegTools<a class="headerlink" href="#regtools" title="Permanent link">&para;</a></h3>
<p>RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context.</p>
<p><em>homepage</em>: <a href="https://regtools.readthedocs.org">https://regtools.readthedocs.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.5.2</code></td>
<td><code>foss/2020b</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="relate">Relate<a class="headerlink" href="#relate" title="Permanent link">&para;</a></h3>
<p>Software for estimating genome-wide genealogies for thousands of samples</p>
<p><em>homepage</em>: <a href="https://myersgroup.github.io/relate/">https://myersgroup.github.io/relate/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20211123</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="relion">RELION<a class="headerlink" href="#relion" title="Permanent link">&para;</a></h3>
<p>RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer  program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class  averages in electron cryo-microscopy (cryo-EM).</p>
<p><em>homepage</em>: <a href="http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page">http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td><code>-single</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>fosscuda/2018a</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0_beta.2018.08.02</code></td>
<td></td>
<td><code>fosscuda/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="rematch">ReMatCh<a class="headerlink" href="#rematch" title="Permanent link">&para;</a></h3>
<p>Reads mapping against target sequences, checking mapping and consensus sequences production</p>
<p><em>homepage</em>: <a href="https://github.com/B-UMMI/ReMatCh">https://github.com/B-UMMI/ReMatCh</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="remora">REMORA<a class="headerlink" href="#remora" title="Permanent link">&para;</a></h3>
<p>REsource MOnitoring for Remote Applications</p>
<p><em>homepage</em>: <a href="https://github.com/TACC/remora">https://github.com/TACC/remora</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.2</code></td>
<td><code>foss/2017a</code>, <code>foss/2018a</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="renderproto">renderproto<a class="headerlink" href="#renderproto" title="Permanent link">&para;</a></h3>
<p>Xrender protocol and ancillary headers</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="repasthpc">RepastHPC<a class="headerlink" href="#repasthpc" title="Permanent link">&para;</a></h3>
<p>The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years:  Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based  modeling system that is designed for use on large computing clusters and supercomputers.</p>
<p><em>homepage</em>: <a href="https://repast.github.io/">https://repast.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="repeatmasker">RepeatMasker<a class="headerlink" href="#repeatmasker" title="Permanent link">&para;</a></h3>
<p>RepeatMasker is a program that screens DNA sequences for interspersed repeats  and low complexity DNA sequences.</p>
<p><em>homepage</em>: <a href="https://www.repeatmasker.org/">https://www.repeatmasker.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.8</code></td>
<td><code>-Perl-5.26.0-HMMER</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>4.0.8</code></td>
<td><code>-Perl-5.28.0-HMMER</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.9-p2</code></td>
<td><code>-HMMER</code></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>4.1.2-p1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.1.4</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="repeatmodeler">RepeatModeler<a class="headerlink" href="#repeatmodeler" title="Permanent link">&para;</a></h3>
<p>RepeatModeler is a de novo transposable element (TE) family identification and modeling package. </p>
<p><em>homepage</em>: <a href="https://www.repeatmasker.org/">https://www.repeatmasker.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2a</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="repeatscout">RepeatScout<a class="headerlink" href="#repeatscout" title="Permanent link">&para;</a></h3>
<p>De Novo Repeat Finder, Price A.L., Jones N.C. and Pevzner P.A. Developed and  tested with our multiple sequence version of RepeatScout ( 1.0.6 )</p>
<p><em>homepage</em>: <a href="https://www.repeatmasker.org/">https://www.repeatmasker.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="request">request<a class="headerlink" href="#request" title="Permanent link">&para;</a></h3>
<p>Request is designed to be the simplest way possible to make http calls.      It supports HTTPS and follows redirects by default.</p>
<p><em>homepage</em>: <a href="https://github.com/request/request/">https://github.com/request/request/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.88.1</code></td>
<td><code>-nodejs-12.19.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="requests">requests<a class="headerlink" href="#requests" title="Permanent link">&para;</a></h3>
<p>Python http for humans</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/requests">https://pypi.python.org/pypi/requests</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.11.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.11.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.13.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="rerconverge">RERconverge<a class="headerlink" href="#rerconverge" title="Permanent link">&para;</a></h3>
<p>RERconverge is a set of software written in R that  estimates the correlation between relative evolutionary rates of  genes and the evolution of a convergent binary or continuous trait  across a phylogeny. </p>
<p><em>homepage</em>: <a href="https://github.com/nclark-lab/RERconverge">https://github.com/nclark-lab/RERconverge</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="resolos">resolos<a class="headerlink" href="#resolos" title="Permanent link">&para;</a></h3>
<p>Resolos is a toolkit written in Python for maintaining reproducible environments  for scientific computations. It's main goal is to enable researchers to easily replicate environments through space (running code on HPC environment) and time (environment preservation for long term archival). For installation and detailed usage, check out the documentation.</p>
<p><em>homepage</em>: <a href="https://gitlab.unige.ch/resolos/resolos">https://gitlab.unige.ch/resolos/resolos</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="retworkx">retworkx<a class="headerlink" href="#retworkx" title="Permanent link">&para;</a></h3>
<p>retworkx is a general purpose graph library for python3 written in Rust to take advantage of the performance and safety that Rust provides. It was built as a replacement for qiskit's previous (and current) networkx usage (hence the name) but is designed to provide a high performance general purpose graph library for any python application. The project was originally started to build a faster directed graph to use as the underlying data structure for the DAG at the center of qiskit-terra's transpiler, but it has since grown to cover all the graph usage in Qiskit and other applications.</p>
<p><em>homepage</em>: <a href="https://github.com/Qiskit/retworkx">https://github.com/Qiskit/retworkx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="revbayes">RevBayes<a class="headerlink" href="#revbayes" title="Permanent link">&para;</a></h3>
<p>RevBayes provides an interactive environment for statistical computation in  phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in  evolutionary biology, particularly phylogenetics.</p>
<p><em>homepage</em>: <a href="https://revbayes.github.io/">https://revbayes.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rgdal">rgdal<a class="headerlink" href="#rgdal" title="Permanent link">&para;</a></h3>
<p>Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (&gt;= 1.11.4 and &lt;= 2.5.0) and  access to projection/transformation operations from the 'PROJ.4' library.</p>
<p><em>homepage</em>: <a href="https://rgdal.r-forge.r-project.org/">https://rgdal.r-forge.r-project.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4-4</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.4-8</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.4-8</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.5-16</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.5-23</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.5-23</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="rgeos">rgeos<a class="headerlink" href="#rgeos" title="Permanent link">&para;</a></h3>
<p>R interface to Geometry Engine - Open Source (GEOS) using the C API for topology operations  on geometries</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/rgeos/">https://cran.r-project.org/web/packages/rgeos/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3-17</code></td>
<td><code>-R-3.2.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.5-1</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.5-2</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.5-5</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.5-5</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="rgurobi">Rgurobi<a class="headerlink" href="#rgurobi" title="Permanent link">&para;</a></h3>
<p>Gurobi Optimizer 9.5 interface</p>
<p><em>homepage</em>: <a href="https://www.gurobi.com">https://www.gurobi.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.1.2</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>9.5.0</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="rhdf5">rhdf5<a class="headerlink" href="#rhdf5" title="Permanent link">&para;</a></h3>
<p>This R/Bioconductor package provides an interface between HDF5 and R.</p>
<p><em>homepage</em>: <a href="https://bioconductor.org/packages/release/bioc/html/rhdf5.html">https://bioconductor.org/packages/release/bioc/html/rhdf5.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.16.0</code></td>
<td><code>-R-3.2.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="rheotool">RheoTool<a class="headerlink" href="#rheotool" title="Permanent link">&para;</a></h3>
<p>RheoTool is an open-source toolbox based on OpenFOAM to simulate Generalized Newtonian Fluids (GNF) and viscoelastic fluids under pressure-driven and/or electrically-driven flows.</p>
<p><em>homepage</em>: <a href="https://github.com/fppimenta/rheoTool">https://github.com/fppimenta/rheoTool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="rickflow">rickflow<a class="headerlink" href="#rickflow" title="Permanent link">&para;</a></h3>
<p>Running and Analyzing OpenMM Jobs</p>
<p><em>homepage</em>: <a href="https://gitlab.com/Olllom/rickflow">https://gitlab.com/Olllom/rickflow</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.7.0-20200529</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="rinchi">RInChI<a class="headerlink" href="#rinchi" title="Permanent link">&para;</a></h3>
<p>The reaction IUPAC International Chemical Identifier (RInChI TM) is a non-proprietary identifier for chemical reactions that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.</p>
<p><em>homepage</em>: <a href="https://www.inchi-trust.org/">https://www.inchi-trust.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.00</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="rioxarray">rioxarray<a class="headerlink" href="#rioxarray" title="Permanent link">&para;</a></h3>
<p>geospatial xarray extension powered by rasterio</p>
<p><em>homepage</em>: <a href="https://github.com/corteva/rioxarray">https://github.com/corteva/rioxarray</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.12</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.0.24</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.1.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="rising">rising<a class="headerlink" href="#rising" title="Permanent link">&para;</a></h3>
<p>Provides everything needed for high performance data loading and augmentation in PyTorch.</p>
<p><em>homepage</em>: <a href="https://rising.rtfd.io/">https://rising.rtfd.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rivet">Rivet<a class="headerlink" href="#rivet" title="Permanent link">&para;</a></h3>
<p>Rivet toolkit (Robust Independent Validation of Experiment and Theory)  To use your own analysis you must append the path to <code>RIVET_ANALYSIS_PATH</code>. </p>
<p><em>homepage</em>: <a href="https://gitlab.com/hepcedar/rivet">https://gitlab.com/hepcedar/rivet</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.7</code></td>
<td><code>-HepMC3-3.2.5</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rjags">rjags<a class="headerlink" href="#rjags" title="Permanent link">&para;</a></h3>
<p>The rjags package is an interface to the JAGS library.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/rjags">https://cran.r-project.org/web/packages/rjags</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4-6</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4-6</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4-8</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4-9</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td><code>-R-4.0.4</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td><code>-R-4.0.5</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>4-10</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4-12</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4-13</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rlcard">RLCard<a class="headerlink" href="#rlcard" title="Permanent link">&para;</a></h3>
<p>RLCard is a toolkit for Reinforcement Learning (RL) in card games.</p>
<p><em>homepage</em>: <a href="https://www.rlcard.org">https://www.rlcard.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rmarkdown">rmarkdown<a class="headerlink" href="#rmarkdown" title="Permanent link">&para;</a></h3>
<p>Convert R Markdown documents into a variety of formats.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/rmarkdown">https://cran.r-project.org/web/packages/rmarkdown</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.20</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="rmath">Rmath<a class="headerlink" href="#rmath" title="Permanent link">&para;</a></h3>
<p>Rmath is the standalone version of the R math library.  Rmath can be used in your own C/C++ routines.</p>
<p><em>homepage</em>: <a href="https://www.r-project.org/">https://www.r-project.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.0.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rmats-turbo">rMATS-turbo<a class="headerlink" href="#rmats-turbo" title="Permanent link">&para;</a></h3>
<p>rMATS turbo is the C/Cython version of rMATS (refer to http://rnaseq-mats.sourceforge.net).</p>
<p><em>homepage</em>: <a href="https://github.com/Xinglab/rmats-turbo">https://github.com/Xinglab/rmats-turbo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="rmblast">RMBlast<a class="headerlink" href="#rmblast" title="Permanent link">&para;</a></h3>
<p>RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary  difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn'  for use with RepeatMasker and RepeatModeler.</p>
<p><em>homepage</em>: <a href="http://www.repeatmasker.org/RMBlast.html">http://www.repeatmasker.org/RMBlast.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.28</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>2.13.0</code></td>
<td></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rna-bloom">RNA-Bloom<a class="headerlink" href="#rna-bloom" title="Permanent link">&para;</a></h3>
<p>RNA-Bloom is a fast and memory-efficient de novo transcript sequence assembler.</p>
<p><em>homepage</em>: <a href="https://github.com/bcgsc/RNA-Bloom">https://github.com/bcgsc/RNA-Bloom</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.3</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>1.4.3</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rna-seqc">RNA-SeQC<a class="headerlink" href="#rna-seqc" title="Permanent link">&para;</a></h3>
<p>Fast, efficient RNA-Seq metrics for quality control and process optimization</p>
<p><em>homepage</em>: <a href="https://github.com/getzlab/rnaseqc">https://github.com/getzlab/rnaseqc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.8</code></td>
<td><code>-Java-1.8</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.8</code></td>
<td><code>-Java-1.8.0_121</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnaclust">RNAclust<a class="headerlink" href="#rnaclust" title="Permanent link">&para;</a></h3>
<p>RNAclust is a perl script summarizing all the single steps required for clustering  of structured RNA motifs, i.e. identifying groups of RNA sequences sharing a secondary structure  motif. It requires as input a multiple FASTA file.</p>
<p><em>homepage</em>: <a href="http://www.bioinf.uni-leipzig.de/~kristin/Software/RNAclust/">http://www.bioinf.uni-leipzig.de/~kristin/Software/RNAclust/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnacode">RNAcode<a class="headerlink" href="#rnacode" title="Permanent link">&para;</a></h3>
<p>RNAcode - Analyze the protein coding potential in multiple sequence alignments</p>
<p><em>homepage</em>: <a href="https://wash.github.io/rnacode/">https://wash.github.io/rnacode/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3</code></td>
<td><code>foss/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="rnaindel">RNAIndel<a class="headerlink" href="#rnaindel" title="Permanent link">&para;</a></h3>
<p>RNAIndel calls coding indels and classifies them into somatic, germline,  and artifact from tumor RNA-Seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/stjude/RNAIndel">https://github.com/stjude/RNAIndel</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnammer">RNAmmer<a class="headerlink" href="#rnammer" title="Permanent link">&para;</a></h3>
<p>This is an example description.</p>
<p><em>homepage</em>: <a href="https://www.cbs.dtu.dk/services/RNAmmer">https://www.cbs.dtu.dk/services/RNAmmer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnaquast">rnaQUAST<a class="headerlink" href="#rnaquast" title="Permanent link">&para;</a></h3>
<p>rnaQUAST is a tool for evaluating RNA-Seq assemblies using reference genome and gene database. In addition, rnaQUAST is also capable of estimating gene database coverage by raw reads and de novo quality assessment using third-party software.</p>
<p><em>homepage</em>: <a href="https://cab.spbu.ru/software/rnaquast/">https://cab.spbu.ru/software/rnaquast/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnaz">RNAz<a class="headerlink" href="#rnaz" title="Permanent link">&para;</a></h3>
<p>RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.</p>
<p><em>homepage</em>: <a href="http://www.tbi.univie.ac.at/~wash/RNAz/">http://www.tbi.univie.ac.at/~wash/RNAz/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2016b</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rnbeads">RnBeads<a class="headerlink" href="#rnbeads" title="Permanent link">&para;</a></h3>
<p>RnBeads is an R package for comprehensive analysis of DNA methylation data obtained with any experimental protocol that provides single-CpG resolution.</p>
<p><em>homepage</em>: <a href="https://rnbeads.org">https://rnbeads.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.0</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="roary">Roary<a class="headerlink" href="#roary" title="Permanent link">&para;</a></h3>
<p>Rapid large-scale prokaryote pan genome analysis</p>
<p><em>homepage</em>: <a href="https://github.com/sanger-pathogens/Roary">https://github.com/sanger-pathogens/Roary</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.12.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.12.0</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rocm">ROCm<a class="headerlink" href="#rocm" title="Permanent link">&para;</a></h3>
<p>AMD ROCm is the first open-source software development platform for HPC/Hyperscale-class GPU computing. AMD ROCm brings the UNIX philosophy of choice, minimalism and modular software development to GPU computing.</p>
<p><em>homepage</em>: <a href="https://rocmdocs.amd.com/en/latest/">https://rocmdocs.amd.com/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rocm-cmake">rocm-cmake<a class="headerlink" href="#rocm-cmake" title="Permanent link">&para;</a></h3>
<p>ROCM cmake modules provides cmake modules for common build tasks needed for the ROCM software stack</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/rocm-cmake">https://github.com/RadeonOpenCompute/rocm-cmake</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rocm-compilersupport">ROCm-CompilerSupport<a class="headerlink" href="#rocm-compilersupport" title="Permanent link">&para;</a></h3>
<p>The compiler support repository provides various Lightning Compiler related services</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/ROCm-CompilerSupport">https://github.com/RadeonOpenCompute/ROCm-CompilerSupport</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rocm-smi">rocm-smi<a class="headerlink" href="#rocm-smi" title="Permanent link">&para;</a></h3>
<p>The ROCm System Management Interface Library, or ROCm SMI library, is part of the Radeon Open Compute ROCm software stack. It is a C library for Linux that provides a user space interface for applications to monitor and control GPU applications.</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/rocm_smi_lib">https://github.com/RadeonOpenCompute/rocm_smi_lib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rocminfo">rocminfo<a class="headerlink" href="#rocminfo" title="Permanent link">&para;</a></h3>
<p>ROCm Application for Reporting System Info</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/rocminfo">https://github.com/RadeonOpenCompute/rocminfo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="rocr-runtime">ROCR-Runtime<a class="headerlink" href="#rocr-runtime" title="Permanent link">&para;</a></h3>
<p>The user-mode API interfaces and libraries necessary for host applications to launch compute kernels to available HSA ROCm kernel agents</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/ROCR-Runtime">https://github.com/RadeonOpenCompute/ROCR-Runtime</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="roct-thunk-interface">ROCT-Thunk-Interface<a class="headerlink" href="#roct-thunk-interface" title="Permanent link">&para;</a></h3>
<p>The user-mode API interfaces used to interact with the ROCk driver</p>
<p><em>homepage</em>: <a href="https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface">https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="roi_pac">ROI_PAC<a class="headerlink" href="#roi_pac" title="Permanent link">&para;</a></h3>
<p>Repeat Orbit Interferometry PACkage (ROI_PAC), software for processing synthetic aperture radar data  to produce differential interferograms</p>
<p><em>homepage</em>: <a href="http://roipac.org/">http://roipac.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="rome">ROME<a class="headerlink" href="#rome" title="Permanent link">&para;</a></h3>
<p>The ROME (Refinement and Optimization via Machine Learning for cryo-EM)  Software package is one of the major research products at the Intel PCCSB.</p>
<p><em>homepage</em>: <a href="https://ipccsb.dfci.harvard.edu/rome">https://ipccsb.dfci.harvard.edu/rome</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>intel/2019.02</code></td>
</tr>
</tbody>
</table>
<h3 id="root">ROOT<a class="headerlink" href="#root" title="Permanent link">&para;</a></h3>
<p>The ROOT system provides a set of OO frameworks with all the functionality     needed to handle and analyze large amounts of data in a very efficient way.</p>
<p><em>homepage</em>: <a href="http://root.cern.ch/drupal/">http://root.cern.ch/drupal/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.10.02</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>6.10.04</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>6.10.08</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>6.14.06</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.14.06</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>6.20.04</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>6.22.08</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>6.24.06</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>v5.34.34</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>v5.34.36</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>v6.06.02</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>v6.08.02</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="root_numpy">root_numpy<a class="headerlink" href="#root_numpy" title="Permanent link">&para;</a></h3>
<p>root_numpy is a Python extension module that provides an efficient interface between ROOT and  NumPy. root_numpys internals are compiled C++ and can therefore handle large amounts of data much faster than  equivalent pure Python implementations.</p>
<p><em>homepage</em>: <a href="https://scikit-hep.org/root_numpy/">https://scikit-hep.org/root_numpy/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rootpy">rootpy<a class="headerlink" href="#rootpy" title="Permanent link">&para;</a></h3>
<p>The rootpy project is a community-driven initiative aiming to provide a more pythonic interface with ROOT on top of the existing PyROOT bindings. Given Pythons reflective and dynamic nature, rootpy also aims to improve ROOT design flaws and supplement existing ROOT functionality. The scientific Python community also offers a multitude of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn, and PyTables, but a suitable interface between them and ROOT has been lacking. rootpy provides the interfaces and conversion mechanisms required to liberate your data and to take advantage of these alternatives if needed.</p>
<p><em>homepage</em>: <a href="http://www.rootpy.org/">http://www.rootpy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rosetta">Rosetta<a class="headerlink" href="#rosetta" title="Permanent link">&para;</a></h3>
<p>Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.</p>
<p><em>homepage</em>: <a href="https://www.rosettacommons.org">https://www.rosettacommons.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2016.13.58602</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2016.46.59086</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="rpmrebuild">rpmrebuild<a class="headerlink" href="#rpmrebuild" title="Permanent link">&para;</a></h3>
<p>rpmrebuild is a tool to build an RPM file from a package that has already been     installed in a basic use</p>
<p><em>homepage</em>: <a href="http://rpmrebuild.sourceforge.net/">http://rpmrebuild.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="rpy2">rpy2<a class="headerlink" href="#rpy2" title="Permanent link">&para;</a></h3>
<p>rpy2 is an interface to R running embedded in a Python process.</p>
<p><em>homepage</em>: <a href="https://rpy2.bitbucket.io/">https://rpy2.bitbucket.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.9</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.8.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.2.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.4.5</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="rqgis3">RQGIS3<a class="headerlink" href="#rqgis3" title="Permanent link">&para;</a></h3>
<p>RQGIS3 establishes an interface between R and QGIS3, i.e.,  it allows the user to access QGIS3 functionalities from within R.</p>
<p><em>homepage</em>: <a href="https://github.com/r-spatial/RQGIS3">https://github.com/r-spatial/RQGIS3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190903</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="rsem">RSEM<a class="headerlink" href="#rsem" title="Permanent link">&para;</a></h3>
<p>RNA-Seq by Expectation-Maximization</p>
<p><em>homepage</em>: <a href="https://deweylab.github.io/RSEM/">https://deweylab.github.io/RSEM/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.26</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.2.30</code></td>
<td><code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td><code>foss/2019b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rseqc">RSeQC<a class="headerlink" href="#rseqc" title="Permanent link">&para;</a></h3>
<p>RSeQC provides a number of useful modules that can  comprehensively evaluate high throughput sequence data especially RNA-seq  data. Some basic modules quickly inspect sequence quality, nucleotide  composition bias, PCR bias and GC bias, while RNA-seq specific modules  evaluate sequencing saturation, mapped reads distribution, coverage  uniformity, strand specificity, transcript level RNA integrity etc.</p>
<p><em>homepage</em>: <a href="http://rseqc.sourceforge.net/">http://rseqc.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-2.7.12-R-3.3.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.6.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="rstan">RStan<a class="headerlink" href="#rstan" title="Permanent link">&para;</a></h3>
<p>RStan is the R interface to Stan. Stan is a state-of-the-art   platform for statistical modeling and high-performance statistical computation.</p>
<p><em>homepage</em>: <a href="https://github.com/stan-dev/rstan">https://github.com/stan-dev/rstan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.18.2</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.18.2</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="rstanarm">rstanarm<a class="headerlink" href="#rstanarm" title="Permanent link">&para;</a></h3>
<p>Estimates previously compiled regression models using the 'rstan' package, which provides  the R interface to the Stan C++ library for Bayesian estimation.</p>
<p><em>homepage</em>: <a href="https://r-forge.r-project.org/projects/rstanarm">https://r-forge.r-project.org/projects/rstanarm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.19.3</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="rstudio-server">RStudio-Server<a class="headerlink" href="#rstudio-server" title="Permanent link">&para;</a></h3>
<p>This is the RStudio Server version. RStudio is a set of integrated tools designed to help you be more productive with R.  The server can be started with:   rserver --server-daemonize=0 --www-port=8787  If you need a database config one can be created with:   MYTMP=<code>mktemp -d</code> &amp;&amp; echo -e "provider=sqlite\ndirectory=${MYTMP}/sqlite" &gt; "${MYTMP}/db.conf" and then used with:   rserver ... --database-config-file="${MYTMP}/db.conf" </p>
<p><em>homepage</em>: <a href="https://www.rstudio.com/">https://www.rstudio.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5033</code></td>
<td><code>-Java-11</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.2.5042</code></td>
<td><code>-Java-11</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.3.959</code></td>
<td><code>-Java-11-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.3.1093</code></td>
<td><code>-Java-11-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.4.1717</code></td>
<td><code>-Java-11-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2022.07.2+576</code></td>
<td><code>-Java-11-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="rtg-tools">RTG-Tools<a class="headerlink" href="#rtg-tools" title="Permanent link">&para;</a></h3>
<p>RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,  as well as utilities for processing other common NGS data formats. </p>
<p><em>homepage</em>: <a href="https://www.realtimegenomics.com/products/rtg-tools">https://www.realtimegenomics.com/products/rtg-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.1</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="rtree">Rtree<a class="headerlink" href="#rtree" title="Permanent link">&para;</a></h3>
<p>Rtree is a ctypes Python wrapper of libspatialindex that provides a number of advanced spatial  indexing features for the spatially curious Python user.</p>
<p><em>homepage</em>: <a href="https://toblerity.org/rtree/">https://toblerity.org/rtree/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="ruamelyaml">ruamel.yaml<a class="headerlink" href="#ruamelyaml" title="Permanent link">&para;</a></h3>
<p>ruamel.yaml is a YAML 1.2 loader/dumper package for Python.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/ruamel-yaml">https://sourceforge.net/projects/ruamel-yaml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.21</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ruby">Ruby<a class="headerlink" href="#ruby" title="Permanent link">&para;</a></h3>
<p>Ruby is a dynamic, open source programming language with  a focus on simplicity and productivity. It has an elegant syntax that is  natural to read and easy to write.</p>
<p><em>homepage</em>: <a href="https://www.ruby-lang.org">https://www.ruby-lang.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>intel/2016b</code>, <code>system</code></td>
</tr>
<tr>
<td><code>2.3.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>foss/2018a</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.6.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ruby-tk">Ruby-Tk<a class="headerlink" href="#ruby-tk" title="Permanent link">&para;</a></h3>
<p>Ruby Tk interface module using tcltklib</p>
<p><em>homepage</em>: <a href="https://github.com/ruby/tk">https://github.com/ruby/tk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Ruby-2.5.1</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="ruffus">ruffus<a class="headerlink" href="#ruffus" title="Permanent link">&para;</a></h3>
<p>Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and user-friendly,  and widely used in science and bioinformatics.</p>
<p><em>homepage</em>: <a href="http://www.ruffus.org.uk/">http://www.ruffus.org.uk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.4</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="rust">Rust<a class="headerlink" href="#rust" title="Permanent link">&para;</a></h3>
<p>Rust is a systems programming language that runs blazingly fast, prevents segfaults,  and guarantees thread safety.</p>
<p><em>homepage</em>: <a href="https://www.rust-lang.org">https://www.rust-lang.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.0</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.18.0</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.21.0</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.22.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.29.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.30.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.35.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.37.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.42.0</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.52.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.54.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.56.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.60.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.65.0</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="s">S<a class="headerlink" href="#s" title="Permanent link">&para;</a></h2>
<p><a href="#s-lang">S-Lang</a> - <a href="#s4">S4</a> - <a href="#sage">SAGE</a> - <a href="#sailfish">Sailfish</a> - <a href="#salib">SALib</a> - <a href="#salmon">Salmon</a> - <a href="#salmon-tddft">SALMON-TDDFT</a> - <a href="#sambamba">Sambamba</a> - <a href="#samblaster">samblaster</a> - <a href="#samcef">Samcef</a> - <a href="#samclip">samclip</a> - <a href="#samplot">samplot</a> - <a href="#samtools">SAMtools</a> - <a href="#sansa">sansa</a> - <a href="#sap">SAP</a> - <a href="#sas">SAS</a> - <a href="#satsuma2">Satsuma2</a> - <a href="#savvy">savvy</a> - <a href="#saxon-he">Saxon-HE</a> - <a href="#sbcl">SBCL</a> - <a href="#sbt">sbt</a> - <a href="#scafacos">ScaFaCoS</a> - <a href="#scalapack">ScaLAPACK</a> - <a href="#scalasca">Scalasca</a> - <a href="#scalce">SCALCE</a> - <a href="#scanpy">scanpy</a> - <a href="#scarches">scArches</a> - <a href="#scenic">SCENIC</a> - <a href="#scgenefit">scGeneFit</a> - <a href="#scgid">SCGid</a> - <a href="#schicexplorer">scHiCExplorer</a> - <a href="#schrodinger">Schrodinger</a> - <a href="#scib">scib</a> - <a href="#sciclone">sciClone</a> - <a href="#scientificpython">ScientificPython</a> - <a href="#scikit-allel">scikit-allel</a> - <a href="#scikit-bio">scikit-bio</a> - <a href="#scikit-build">scikit-build</a> - <a href="#scikit-cuda">scikit-cuda</a> - <a href="#scikit-image">scikit-image</a> - <a href="#scikit-learn">scikit-learn</a> - <a href="#scikit-misc">scikit-misc</a> - <a href="#scikit-multilearn">scikit-multilearn</a> - <a href="#scikit-optimize">scikit-optimize</a> - <a href="#scikit-uplift">scikit-uplift</a> - <a href="#sciphi">SCIPhI</a> - <a href="#scipy">scipy</a> - <a href="#scipy-bundle">SciPy-bundle</a> - <a href="#scnorm">SCnorm</a> - <a href="#scoary">Scoary</a> - <a href="#scons">SCons</a> - <a href="#scoop">SCOOP</a> - <a href="#scopeloomr">SCopeLoomR</a> - <a href="#score-p">Score-P</a> - <a href="#scotch">SCOTCH</a> - <a href="#scp">scp</a> - <a href="#scpred">scPred</a> - <a href="#scrappie">Scrappie</a> - <a href="#scvelo">scVelo</a> - <a href="#scvi-tools">scvi-tools</a> - <a href="#scythe">Scythe</a> - <a href="#sdcc">SDCC</a> - <a href="#sdl">SDL</a> - <a href="#sdl2">SDL2</a> - <a href="#sdl2_image">SDL2_image</a> - <a href="#sdl_image">SDL_image</a> - <a href="#sdsl">SDSL</a> - <a href="#seaborn">Seaborn</a> - <a href="#searchgui">SearchGUI</a> - <a href="#seaview">SeaView</a> - <a href="#secapr">SECAPR</a> - <a href="#seeder">Seeder</a> - <a href="#segemehl">segemehl</a> - <a href="#segmentation-models">segmentation-models</a> - <a href="#seissol">SeisSol</a> - <a href="#selestim">SelEstim</a> - <a href="#selfies">SELFIES</a> - <a href="#sentencepiece">SentencePiece</a> - <a href="#sep">sep</a> - <a href="#sepp">SEPP</a> - <a href="#seq-gen">Seq-Gen</a> - <a href="#seq2hla">seq2HLA</a> - <a href="#seqan">SeqAn</a> - <a href="#seqan3">SeqAn3</a> - <a href="#seqkit">SeqKit</a> - <a href="#seqlib">SeqLib</a> - <a href="#seqmagick">Seqmagick</a> - <a href="#seqprep">SeqPrep</a> - <a href="#seqtk">seqtk</a> - <a href="#serf">Serf</a> - <a href="#setuptools">setuptools</a> - <a href="#seurat">Seurat</a> - <a href="#seuratdata">SeuratData</a> - <a href="#seuratdisk">SeuratDisk</a> - <a href="#seuratwrappers">SeuratWrappers</a> - <a href="#sf">sf</a> - <a href="#sfftk">sfftk</a> - <a href="#shannon">Shannon</a> - <a href="#shap">SHAP</a> - <a href="#shapaar">shapAAR</a> - <a href="#shapeit">SHAPEIT</a> - <a href="#shapeit4">SHAPEIT4</a> - <a href="#shapely">Shapely</a> - <a href="#sharutils">sharutils</a> - <a href="#shasta">Shasta</a> - <a href="#shengbte">ShengBTE</a> - <a href="#shift">shift</a> - <a href="#shore">SHORE</a> - <a href="#short-pair">Short-Pair</a> - <a href="#shovill">shovill</a> - <a href="#shrinkwrap">shrinkwrap</a> - <a href="#shtns">SHTns</a> - <a href="#sibelia">Sibelia</a> - <a href="#sicer2">SICER2</a> - <a href="#sickle">sickle</a> - <a href="#siesta">Siesta</a> - <a href="#signalp">SignalP</a> - <a href="#silhouetterank">silhouetteRank</a> - <a href="#silx">silx</a> - <a href="#simint">simint</a> - <a href="#simnibs">SimNIBS</a> - <a href="#simpeg">SimPEG</a> - <a href="#simple">SIMPLE</a> - <a href="#simpleelastix">SimpleElastix</a> - <a href="#simpleitk">SimpleITK</a> - <a href="#simpy">simpy</a> - <a href="#simvascular">SimVascular</a> - <a href="#singlem">SingleM</a> - <a href="#singular">Singular</a> - <a href="#sinvict">SiNVICT</a> - <a href="#sionlib">SIONlib</a> - <a href="#sip">SIP</a> - <a href="#siscone">siscone</a> - <a href="#sisso">SISSO</a> - <a href="#sisso++">SISSO++</a> - <a href="#skesa">SKESA</a> - <a href="#sketchmap">sketchmap</a> - <a href="#skewer">skewer</a> - <a href="#sklearn-pandas">sklearn-pandas</a> - <a href="#sklearn-som">sklearn-som</a> - <a href="#skorch">skorch</a> - <a href="#slamdunk">SlamDunk</a> - <a href="#slatec">SLATEC</a> - <a href="#slepc">SLEPc</a> - <a href="#slepc4py">slepc4py</a> - <a href="#sleuth">sleuth</a> - <a href="#slidingwindow">slidingwindow</a> - <a href="#slim">SLiM</a> - <a href="#slow5tools">slow5tools</a> - <a href="#smafa">smafa</a> - <a href="#smallgenomeutilities">smallgenomeutilities</a> - <a href="#smartdenovo">SMARTdenovo</a> - <a href="#smc++">SMC++</a> - <a href="#smfishhmrf">smfishHmrf</a> - <a href="#smithwaterman">smithwaterman</a> - <a href="#smoldyn">Smoldyn</a> - <a href="#smooth-topk">smooth-topk</a> - <a href="#smrt-link">SMRT-Link</a> - <a href="#smv">SMV</a> - <a href="#snakemake">snakemake</a> - <a href="#snap">SNAP</a> - <a href="#snap-hmm">SNAP-HMM</a> - <a href="#snape-pooled">SNAPE-pooled</a> - <a href="#snaphu">snaphu</a> - <a href="#snappy">snappy</a> - <a href="#snippy">snippy</a> - <a href="#snp-sites">snp-sites</a> - <a href="#snpeff">snpEff</a> - <a href="#snphylo">SNPhylo</a> - <a href="#snpomatic">SNPomatic</a> - <a href="#soapaligner">SOAPaligner</a> - <a href="#soapdenovo-trans">SOAPdenovo-Trans</a> - <a href="#soapdenovo2">SOAPdenovo2</a> - <a href="#soapfuse">SOAPfuse</a> - <a href="#socat">socat</a> - <a href="#soci">SOCI</a> - <a href="#solexaqa++">SolexaQA++</a> - <a href="#sonic">sonic</a> - <a href="#soqt">SoQt</a> - <a href="#sortmerna">SortMeRNA</a> - <a href="#sox">SoX</a> - <a href="#soxt">SoXt</a> - <a href="#spaceranger">SpaceRanger</a> - <a href="#spack">Spack</a> - <a href="#spacy">spaCy</a> - <a href="#spades">SPAdes</a> - <a href="#spaln">spaln</a> - <a href="#spark">Spark</a> - <a href="#sparse-neighbors-search">sparse-neighbors-search</a> - <a href="#sparsehash">sparsehash</a> - <a href="#spatialde">SpatialDE</a> - <a href="#spatialreg">spatialreg</a> - <a href="#spdlog">spdlog</a> - <a href="#spectra">SpectrA</a> - <a href="#spectral.methods">spectral.methods</a> - <a href="#speech_tools">speech_tools</a> - <a href="#spglib">spglib</a> - <a href="#spglib-python">spglib-python</a> - <a href="#sphinx">Sphinx</a> - <a href="#sphinx-rtd-theme">Sphinx-RTD-Theme</a> - <a href="#spiceypy">SpiceyPy</a> - <a href="#spieceasi">SpiecEasi</a> - <a href="#spladder">SplAdder</a> - <a href="#splash">SPLASH</a> - <a href="#splicemap">SpliceMap</a> - <a href="#split-seq">split-seq</a> - <a href="#splitref">splitRef</a> - <a href="#spm">SPM</a> - <a href="#spoa">spoa</a> - <a href="#spooles">SPOOLES</a> - <a href="#spotpy">SPOTPY</a> - <a href="#sprng">SPRNG</a> - <a href="#spyder">Spyder</a> - <a href="#sqlite">SQLite</a> - <a href="#squeezemeta">SqueezeMeta</a> - <a href="#squidpy">Squidpy</a> - <a href="#sra-toolkit">SRA-Toolkit</a> - <a href="#sradownloader">sradownloader</a> - <a href="#srprism">SRPRISM</a> - <a href="#srst2">SRST2</a> - <a href="#ssaha2">SSAHA2</a> - <a href="#ssn">SSN</a> - <a href="#sspace_basic">SSPACE_Basic</a> - <a href="#ssw">SSW</a> - <a href="#stacey">STACEY</a> - <a href="#stack">Stack</a> - <a href="#stacks">Stacks</a> - <a href="#stamp">STAMP</a> - <a href="#stamps">StaMPS</a> - <a href="#stampy">Stampy</a> - <a href="#star">STAR</a> - <a href="#star-ccm+">STAR-CCM+</a> - <a href="#star-fusion">STAR-Fusion</a> - <a href="#stardist">stardist</a> - <a href="#stars">stars</a> - <a href="#stata">Stata</a> - <a href="#statistics-r">Statistics-R</a> - <a href="#statsmodels">statsmodels</a> - <a href="#steak">STEAK</a> - <a href="#stir">STIR</a> - <a href="#stpipeline">stpipeline</a> - <a href="#strainberry">Strainberry</a> - <a href="#stream">STREAM</a> - <a href="#strelka">strelka</a> - <a href="#stringtie">StringTie</a> - <a href="#stripy">stripy</a> - <a href="#strique">STRique</a> - <a href="#structure">Structure</a> - <a href="#strumpack">STRUMPACK</a> - <a href="#suave">suave</a> - <a href="#subread">Subread</a> - <a href="#subversion">Subversion</a> - <a href="#suds">suds</a> - <a href="#suitesparse">SuiteSparse</a> - <a href="#sumaclust">SUMACLUST</a> - <a href="#sumatra">SUMATRA</a> - <a href="#sumo">SUMO</a> - <a href="#sundials">SUNDIALS</a> - <a href="#sunpy">SunPy</a> - <a href="#superlu">SuperLU</a> - <a href="#superlu_dist">SuperLU_DIST</a> - <a href="#supermagic">supermagic</a> - <a href="#supernova">supernova</a> - <a href="#survivor">SURVIVOR</a> - <a href="#svdetect">SVDetect</a> - <a href="#svdquest">SVDquest</a> - <a href="#svg">SVG</a> - <a href="#svist4get">svist4get</a> - <a href="#swarm">swarm</a> - <a href="#swash">SWASH</a> - <a href="#swat+">SWAT+</a> - <a href="#swifter">swifter</a> - <a href="#swig">SWIG</a> - <a href="#swipe">SWIPE</a> - <a href="#swissknife">swissknife</a> - <a href="#symengine">SymEngine</a> - <a href="#symengine-python">SymEngine-python</a> - <a href="#symphony">SYMPHONY</a> - <a href="#sympy">sympy</a> - <a href="#syri">SyRI</a> - <a href="#szip">Szip</a></p>
<h3 id="s-lang">S-Lang<a class="headerlink" href="#s-lang" title="Permanent link">&para;</a></h3>
<p>S-Lang is a multi-platform programmer's library designed to allow  a developer to create robust multi-platform software. It provides facilities  required by interactive applications such as display/screen management, keyboard  input, keymaps, and so on.</p>
<p><em>homepage</em>: <a href="http://www.jedsoft.org/slang/">http://www.jedsoft.org/slang/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="s4">S4<a class="headerlink" href="#s4" title="Permanent link">&para;</a></h3>
<p>S4 stands for Stanford Stratified Structure Solver, a frequency domain code  to solve the linear Maxwells equations in layered periodic structures. Internally, it uses  Rigorous Coupled Wave Analysis (RCWA, also called the Fourier Modal Method (FMM)) and the  S-matrix algorithm.</p>
<p><em>homepage</em>: <a href="https://web.stanford.edu/group/fan/S4">https://web.stanford.edu/group/fan/S4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1-20180610</code></td>
<td><code>foss/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="sage">SAGE<a class="headerlink" href="#sage" title="Permanent link">&para;</a></h3>
<p>S.A.G.E. (Statistical Analysis for Genetic  Epidemiology) is free software package containing programs for  use in the genetic analysis of family, pedigree and individual data.</p>
<p><em>homepage</em>: <a href="http://darwin.cwru.edu/sage/">http://darwin.cwru.edu/sage/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="sailfish">Sailfish<a class="headerlink" href="#sailfish" title="Permanent link">&para;</a></h3>
<p>Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of  isoform abundances directly from a set of reference sequences and RNA-seq reads. </p>
<p><em>homepage</em>: <a href="http://www.cs.cmu.edu/~ckingsf/software/sailfish/">http://www.cs.cmu.edu/~ckingsf/software/sailfish/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.1</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="salib">SALib<a class="headerlink" href="#salib" title="Permanent link">&para;</a></h3>
<p>Sensitivity Analysis Library in Python (Numpy). Contains Sobol, Morris, Fractional Factorial and FAST methods.</p>
<p><em>homepage</em>: <a href="https://SALib.github.io/SALib/">https://SALib.github.io/SALib/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="salmon">Salmon<a class="headerlink" href="#salmon" title="Permanent link">&para;</a></h3>
<p>Salmon is a wicked-fast program to produce a highly-accurate,  transcript-level quantification estimates from RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/COMBINE-lab/salmon">https://github.com/COMBINE-lab/salmon</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.14.1</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.14.2</code></td>
<td></td>
<td><code>gompi/2019a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="salmon-tddft">SALMON-TDDFT<a class="headerlink" href="#salmon-tddft" title="Permanent link">&para;</a></h3>
<p>SALMON is an open-source computer program for ab-initio quantum-mechanical calculations of  electron dynamics at the nanoscale that takes place in various situations of light-matter interactions.  It is based on time-dependent density functional theory, solving time-dependent Kohn-Sham equation in  real time and real space with norm-conserving pseudopotentials.</p>
<p><em>homepage</em>: <a href="https://salmon-tddft.jp">https://salmon-tddft.jp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="sambamba">Sambamba<a class="headerlink" href="#sambamba" title="Permanent link">&para;</a></h3>
<p>Sambamba is a high performance modern robust and fast tool  (and library), written in the D programming language, for working with SAM  and BAM files. Current functionality is an important subset of samtools  functionality, including view, index, sort, markdup, and depth.</p>
<p><em>homepage</em>: <a href="https://lomereiter.github.io/sambamba/">https://lomereiter.github.io/sambamba/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.7.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="samblaster">samblaster<a class="headerlink" href="#samblaster" title="Permanent link">&para;</a></h3>
<p>samblaster is a fast and flexible program for marking  duplicates in read-id grouped1 paired-end SAM files.</p>
<p><em>homepage</em>: <a href="https://github.com/GregoryFaust/samblaster">https://github.com/GregoryFaust/samblaster</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.24</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.1.26</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="samcef">Samcef<a class="headerlink" href="#samcef" title="Permanent link">&para;</a></h3>
<p>FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and  thermal applications.</p>
<p><em>homepage</em>: <a href="http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/">http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17.0-03</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="samclip">samclip<a class="headerlink" href="#samclip" title="Permanent link">&para;</a></h3>
<p>Filter SAM file for soft and hard clipped alignments.</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/samclip">https://github.com/tseemann/samclip</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="samplot">samplot<a class="headerlink" href="#samplot" title="Permanent link">&para;</a></h3>
<p>Plot structural variant signals from many BAMs and CRAMs. </p>
<p><em>homepage</em>: <a href="https://github.com/ryanlayer/samplot">https://github.com/ryanlayer/samplot</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="samtools">SAMtools<a class="headerlink" href="#samtools" title="Permanent link">&para;</a></h3>
<p>SAM Tools provide various utilities for manipulating alignments in the SAM format,   including sorting, merging, indexing and generating alignments in a per-position format.</p>
<p><em>homepage</em>: <a href="http://www.htslib.org/">http://www.htslib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.17</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.1.19</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.1.20</code></td>
<td></td>
<td><code>GCC/8.3.0</code>, <code>foss/2018b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.2</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-HTSlib-1.3.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.6</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>foss/2017a</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.7</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.10</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.11</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>1.12</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.13</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.14</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.15</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.15.1</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>1.17</code></td>
<td></td>
<td><code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sansa">sansa<a class="headerlink" href="#sansa" title="Permanent link">&para;</a></h3>
<p>Structural variant (SV) annotation, a companion to the 'dolly' tool.</p>
<p><em>homepage</em>: <a href="https://github.com/dellytools/sansa/">https://github.com/dellytools/sansa/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.7</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="sap">SAP<a class="headerlink" href="#sap" title="Permanent link">&para;</a></h3>
<p>SAP is a pairwise structure alignment via double dynamic programming</p>
<p><em>homepage</em>: <a href="https://github.com/mathbio-nimr-mrc-ac-uk/SAP">https://github.com/mathbio-nimr-mrc-ac-uk/SAP</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sas">SAS<a class="headerlink" href="#sas" title="Permanent link">&para;</a></h3>
<p>SAS is a software suite for advanced analytics, multivariate analyses, business intelligence,  data management, and predictive analytics.</p>
<p><em>homepage</em>: <a href="https://www.sas.com">https://www.sas.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="satsuma2">Satsuma2<a class="headerlink" href="#satsuma2" title="Permanent link">&para;</a></h3>
<p>Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to accommodate the billions of base pairs in vertebrate genomes).</p>
<p><em>homepage</em>: <a href="https://github.com/bioinfologics/satsuma2">https://github.com/bioinfologics/satsuma2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220304</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="savvy">savvy<a class="headerlink" href="#savvy" title="Permanent link">&para;</a></h3>
<p>Interface to various variant calling formats.</p>
<p><em>homepage</em>: <a href="https://github.com/statgen/savvy">https://github.com/statgen/savvy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="saxon-he">Saxon-HE<a class="headerlink" href="#saxon-he" title="Permanent link">&para;</a></h3>
<p>Open Source SAXON XSLT processor developed by Saxonica Limited.</p>
<p><em>homepage</em>: <a href="http://saxon.sourceforge.net">http://saxon.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.7.0.4</code></td>
<td><code>-Java-1.7.0_79</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.7.0.21</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>9.9.1.7</code></td>
<td><code>-Java-13</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="sbcl">SBCL<a class="headerlink" href="#sbcl" title="Permanent link">&para;</a></h3>
<p>Steel Bank Common Lisp (SBCL) is a high performance Common Lisp compiler. It is open source / free software, with a permissive license. In addition to the compiler and runtime system for ANSI Common Lisp, it provides an interactive environment including a debugger, a statistical profiler, a code coverage tool, and many other extensions.</p>
<p><em>homepage</em>: <a href="http://sbcl.sourceforge.net/">http://sbcl.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.9</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sbt">sbt<a class="headerlink" href="#sbt" title="Permanent link">&para;</a></h3>
<p>sbt is a build tool for Scala, Java, and more.</p>
<p><em>homepage</em>: <a href="http://www.scala-sbt.org/">http://www.scala-sbt.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.3.13</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="scafacos">ScaFaCoS<a class="headerlink" href="#scafacos" title="Permanent link">&para;</a></h3>
<p>ScaFaCoS is a library of scalable fast coulomb solvers.</p>
<p><em>homepage</em>: <a href="http://www.scafacos.de/">http://www.scafacos.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="scalapack">ScaLAPACK<a class="headerlink" href="#scalapack" title="Permanent link">&para;</a></h3>
<p>The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines  redesigned for distributed memory MIMD parallel computers.</p>
<p><em>homepage</em>: <a href="https://www.netlib.org/scalapack/">https://www.netlib.org/scalapack/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompic/2019b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-BLIS-0.3.2</code></td>
<td><code>gompi/2018b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.13-LAPACK-3.5.0</code></td>
<td><code>gmvapich2/1.7.20</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.15-LAPACK-3.6.0</code></td>
<td><code>gmpich/2016a</code>, <code>gmvapich2/2016a</code>, <code>gompi/2016a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.18-LAPACK-3.6.0</code></td>
<td><code>gompi/2016.04</code>, <code>gompi/2016.06</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.18-LAPACK-3.6.1</code></td>
<td><code>gompi/2016.07</code>, <code>gompi/2016b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.19-LAPACK-3.6.1</code></td>
<td><code>gompi/2016.09</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.19-LAPACK-3.7.0</code></td>
<td><code>gompi/2017a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.2.20</code></td>
<td><code>gimpi/2017b</code>, <code>gimpi/2018a</code>, <code>gimpic/2017b</code>, <code>gmpich/2017.08</code>, <code>gompi/2017b</code>, <code>gompi/2018a</code>, <code>gompic/2017b</code>, <code>gompic/2018a</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.3.1</code></td>
<td><code>gompi/2018b</code>, <code>gompic/2018b</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.3.3</code></td>
<td><code>gompi/2018.08</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>-OpenBLAS-0.3.5</code></td>
<td><code>gompi/2019a</code>, <code>gompic/2019a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompic/2020a</code>, <code>gompic/2020b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-bf</code></td>
<td><code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>iimpi/2020b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-bl</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-fb</code></td>
<td><code>gompi/2021a</code>, <code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-amd</code></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-fb</code></td>
<td><code>gompi/2022.05</code>, <code>gompi/2022.10</code>, <code>gompi/2022a</code>, <code>gompi/2022b</code>, <code>nvompi/2022.07</code></td>
</tr>
</tbody>
</table>
<h3 id="scalasca">Scalasca<a class="headerlink" href="#scalasca" title="Permanent link">&para;</a></h3>
<p>Scalasca is a software tool that supports the performance optimization of  parallel programs by measuring and analyzing their runtime behavior. The  analysis identifies potential performance bottlenecks -- in particular  those concerning communication and synchronization -- and offers guidance  in exploring their causes. </p>
<p><em>homepage</em>: <a href="https://www.scalasca.org/">https://www.scalasca.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>gompi/2019a</code>, <code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>gompi/2021a</code>, <code>gompic/2020b</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scalce">SCALCE<a class="headerlink" href="#scalce" title="Permanent link">&para;</a></h3>
<p>SCALCE [skelz] is a FASTQ compression tool that uses locally consistet parsing  to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can  handle other technologies (that generate base pair reads) if the read length is the same throughout the file.</p>
<p><em>homepage</em>: <a href="http://scalce.sourceforge.net/Home">http://scalce.sourceforge.net/Home</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="scanpy">scanpy<a class="headerlink" href="#scanpy" title="Permanent link">&para;</a></h3>
<p>Scanpy is a scalable toolkit for analyzing single-cell gene expression data built  jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference  and differential expression testing. The Python-based implementation efficiently deals with  datasets of more than one million cells. </p>
<p><em>homepage</em>: <a href="https://scanpy.readthedocs.io/en/stable/">https://scanpy.readthedocs.io/en/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.8.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.9.1</code></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scarches">scArches<a class="headerlink" href="#scarches" title="Permanent link">&para;</a></h3>
<p>Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell  data.</p>
<p><em>homepage</em>: <a href="https://github.com/theislab/scarches">https://github.com/theislab/scarches</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.6</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scenic">SCENIC<a class="headerlink" href="#scenic" title="Permanent link">&para;</a></h3>
<p>SCENIC Suite is a set of tools to study and decipher gene regulation.</p>
<p><em>homepage</em>: <a href="https://scenic.aertslab.org">https://scenic.aertslab.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.4</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scgenefit">scGeneFit<a class="headerlink" href="#scgenefit" title="Permanent link">&para;</a></h3>
<p>Python code for genetic marker selection using linear programming.</p>
<p><em>homepage</em>: <a href="https://github.com/solevillar/scGeneFit-python">https://github.com/solevillar/scGeneFit-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scgid">SCGid<a class="headerlink" href="#scgid" title="Permanent link">&para;</a></h3>
<p>A consensus approach to contig filtering and genome prediction from single-cell sequencing libraries</p>
<p><em>homepage</em>: <a href="https://github.com/amsesk/SCGid">https://github.com/amsesk/SCGid</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9b0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="schicexplorer">scHiCExplorer<a class="headerlink" href="#schicexplorer" title="Permanent link">&para;</a></h3>
<p>The scHiCExplorer is a software to demultiplex, process, correct, normalize, manipulate, analyse and visualize single-cell Hi-C data.</p>
<p><em>homepage</em>: <a href="https://schicexplorer.readthedocs.io">https://schicexplorer.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="schrodinger">Schrodinger<a class="headerlink" href="#schrodinger" title="Permanent link">&para;</a></h3>
<p>Schrodinger aims to provide integrated software solutions and services   that truly meet its customers needs. We want to empower researchers around the world to   achieve their goals of improving human health and quality of life through advanced computational   techniques that transform the way chemists design compounds and materials. </p>
<p><em>homepage</em>: <a href="https://www.schrodinger.com/">https://www.schrodinger.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2020-4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021-4</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022-1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022-2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022-3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="scib">scib<a class="headerlink" href="#scib" title="Permanent link">&para;</a></h3>
<p>Benchmarking atlas-level data integration in single-cell genomics.</p>
<p><em>homepage</em>: <a href="https://github.com/theislab/scib">https://github.com/theislab/scib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sciclone">sciClone<a class="headerlink" href="#sciclone" title="Permanent link">&para;</a></h3>
<p>An R package for inferring the subclonal architecture of tumors </p>
<p><em>homepage</em>: <a href="https://github.com/genome/sciclone">https://github.com/genome/sciclone</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="scientificpython">ScientificPython<a class="headerlink" href="#scientificpython" title="Permanent link">&para;</a></h3>
<p>ScientificPython is a collection of Python modules for scientific computing.  It contains support for geometry, mathematical functions, statistics, physical units, IO, visualization,  and parallelization.</p>
<p><em>homepage</em>: <a href="http://dirac.cnrs-orleans.fr/ScientificPython">http://dirac.cnrs-orleans.fr/ScientificPython</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.4</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-allel">scikit-allel<a class="headerlink" href="#scikit-allel" title="Permanent link">&para;</a></h3>
<p>This package provides utilities for exploratory analysis of large scale genetic variation data.  It is based on numpy, scipy and other general-purpose Python scientific libraries.</p>
<p><em>homepage</em>: <a href="https://scikit-allel.readthedocs.io/en/latest/">https://scikit-allel.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.8</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.3.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-bio">scikit-bio<a class="headerlink" href="#scikit-bio" title="Permanent link">&para;</a></h3>
<p>scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms and educational resources for bioinformatics.</p>
<p><em>homepage</em>: <a href="http://scikit-bio.org">http://scikit-bio.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.5.7</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-build">scikit-build<a class="headerlink" href="#scikit-build" title="Permanent link">&para;</a></h3>
<p>Scikit-Build, or skbuild, is an improved build system generator for CPython C/C++/Fortran/Cython extensions.</p>
<p><em>homepage</em>: <a href="https://scikit-build.readthedocs.io/en/latest">https://scikit-build.readthedocs.io/en/latest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>0.15.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-cuda">scikit-cuda<a class="headerlink" href="#scikit-cuda" title="Permanent link">&para;</a></h3>
<p>SciKit-cuda, a.k.a. skcuda, provides Python interfaces to many of the functions in the CUDA  device/runtime, CUBLAS, CUFFT, and CUSOLVER libraries distributed as part of NVIDIA's CUDA Programming Toolkit.</p>
<p><em>homepage</em>: <a href="https://github.com/lebedov/scikit-cuda">https://github.com/lebedov/scikit-cuda</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-image">scikit-image<a class="headerlink" href="#scikit-image" title="Permanent link">&para;</a></h3>
<p>scikit-image is a collection of algorithms for image processing.</p>
<p><em>homepage</em>: <a href="https://scikit-image.org/">https://scikit-image.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.14.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.14.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.15.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.16.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>0.18.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.19.3</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-learn">scikit-learn<a class="headerlink" href="#scikit-learn" title="Permanent link">&para;</a></h3>
<p>Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.</p>
<p><em>homepage</em>: <a href="https://scikit-learn.org/stable/index.html">https://scikit-learn.org/stable/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-2.7.11-freetype-2.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.17.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.18</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.18</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.18.2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.19.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.19.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.20.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.20.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.20.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.20.3</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.20.4</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.20.4</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.21.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.23.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code>, <code>intelcuda/2020a</code></td>
</tr>
<tr>
<td><code>0.23.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>0.24.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td></td>
<td><code>gfbf/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-misc">scikit-misc<a class="headerlink" href="#scikit-misc" title="Permanent link">&para;</a></h3>
<p>Miscellaneous tools for data analysis and scientific computing</p>
<p><em>homepage</em>: <a href="https://github.com/has2k1/scikit-misc">https://github.com/has2k1/scikit-misc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.4</code></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-multilearn">scikit-multilearn<a class="headerlink" href="#scikit-multilearn" title="Permanent link">&para;</a></h3>
<p>Scikit-multilearn is a BSD-licensed library for multi-label classification that is built on top of the well-known scikit-learn ecosystem.</p>
<p><em>homepage</em>: <a href="http://scikit.ml">http://scikit.ml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-optimize">scikit-optimize<a class="headerlink" href="#scikit-optimize" title="Permanent link">&para;</a></h3>
<p>Scikit-Optimize, or skopt, is a simple and efficient library to minimize (very) expensive  and noisy black-box functions.</p>
<p><em>homepage</em>: <a href="https://scikit-optimize.github.io">https://scikit-optimize.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.7.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scikit-uplift">scikit-uplift<a class="headerlink" href="#scikit-uplift" title="Permanent link">&para;</a></h3>
<p>scikit-uplift is a Python module for classic approaches for uplift modeling built on top of scikit-learn. Uplift prediction aims to estimate the causal impact of a treatment at the individual level. </p>
<p><em>homepage</em>: <a href="https://github.com/maks-sh/scikit-uplift">https://github.com/maks-sh/scikit-uplift</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="sciphi">SCIPhI<a class="headerlink" href="#sciphi" title="Permanent link">&para;</a></h3>
<p>Single-cell mutation identification via phylogenetic inference (SCIPhI) is a new approach  to mutation detection in individual tumor cells by leveraging the evolutionary relationship among cells.</p>
<p><em>homepage</em>: <a href="https://github.com/cbg-ethz/SCIPhI">https://github.com/cbg-ethz/SCIPhI</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.3</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="scipy">scipy<a class="headerlink" href="#scipy" title="Permanent link">&para;</a></h3>
<p>SciPy is a collection of mathematical algorithms and convenience  functions built on the Numpy extension for Python.</p>
<p><em>homepage</em>: <a href="https://www.scipy.org">https://www.scipy.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.17.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.19.0</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="scipy-bundle">SciPy-bundle<a class="headerlink" href="#scipy-bundle" title="Permanent link">&para;</a></h3>
<p>Bundle of Python packages for scientific software</p>
<p><em>homepage</em>: <a href="https://python.org/">https://python.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.03</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code>, <code>intel/2019a</code>, <code>intelcuda/2019a</code></td>
</tr>
<tr>
<td><code>2019.10</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>2019.10</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2019.10</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>2020.03</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2020.03</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code>, <code>intelcuda/2020a</code></td>
</tr>
<tr>
<td><code>2020.11</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intelcuda/2020b</code></td>
</tr>
<tr>
<td><code>2020.11</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2021.05</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>gomkl/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>2021.10</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2021.10</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2022.05</code></td>
<td></td>
<td><code>foss/2022.05</code>, <code>foss/2022a</code>, <code>intel/2022.05</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>2023.02</code></td>
<td></td>
<td><code>gfbf/2022b</code></td>
</tr>
</tbody>
</table>
<h3 id="scnorm">SCnorm<a class="headerlink" href="#scnorm" title="Permanent link">&para;</a></h3>
<p>This package implements SCnorm  a method to normalize single-cell RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://bioconductor.org/packages/SCnorm">https://bioconductor.org/packages/SCnorm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99.7</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="scoary">Scoary<a class="headerlink" href="#scoary" title="Permanent link">&para;</a></h3>
<p>Microbial pan-GWAS using the output from Roary</p>
<p><em>homepage</em>: <a href="https://github.com/AdmiralenOla/Scoary">https://github.com/AdmiralenOla/Scoary</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.16</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.6.16</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="scons">SCons<a class="headerlink" href="#scons" title="Permanent link">&para;</a></h3>
<p>SCons is a software construction tool.</p>
<p><em>homepage</em>: <a href="https://www.scons.org/">https://www.scons.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-2.7.15-bare</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.0.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.1.0.post1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="scoop">SCOOP<a class="headerlink" href="#scoop" title="Permanent link">&para;</a></h3>
<p>SCOOP (Scalable COncurrent Operations in Python) is a distributed task module  allowing concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.</p>
<p><em>homepage</em>: <a href="http://code.google.com/p/scoop/">http://code.google.com/p/scoop/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.7.1.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.7.1.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.7.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.7.1.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="scopeloomr">SCopeLoomR<a class="headerlink" href="#scopeloomr" title="Permanent link">&para;</a></h3>
<p>An R package (compatible with SCope) to create generic .loom files and extend them with other data e.g.: SCENIC regulons, Seurat clusters and markers, ...</p>
<p><em>homepage</em>: <a href="https://github.com/aertslab/SCopeLoomR">https://github.com/aertslab/SCopeLoomR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.0</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="score-p">Score-P<a class="headerlink" href="#score-p" title="Permanent link">&para;</a></h3>
<p>The Score-P measurement infrastructure is a highly scalable and easy-to-use  tool suite for profiling, event tracing, and online analysis of HPC  applications. </p>
<p><em>homepage</em>: <a href="https://www.score-p.org">https://www.score-p.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.0</code></td>
<td></td>
<td><code>gompi/2019a</code>, <code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompic/2019a</code>, <code>gompic/2019b</code>, <code>gompic/2020a</code></td>
</tr>
<tr>
<td><code>7.0</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>gompic/2020b</code></td>
</tr>
<tr>
<td><code>7.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td></td>
<td><code>gompi/2021b</code>, <code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>8.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scotch">SCOTCH<a class="headerlink" href="#scotch" title="Permanent link">&para;</a></h3>
<p>Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.</p>
<p><em>homepage</em>: <a href="https://www.labri.fr/perso/pelegrin/scotch/">https://www.labri.fr/perso/pelegrin/scotch/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.12b_esmumps</code></td>
<td></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>6.0.4</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.0.4</code></td>
<td><code>-64bitint</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>6.0.6</code></td>
<td></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>gompi/2019a</code>, <code>iimpi/2019a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>6.0.9</code></td>
<td></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>iimpi/2019b</code>, <code>iimpi/2020a</code>, <code>iimpic/2019b</code></td>
</tr>
<tr>
<td><code>6.1.0</code></td>
<td></td>
<td><code>gompi/2020b</code>, <code>gompi/2021a</code>, <code>iimpi/2020b</code>, <code>iimpi/2021a</code></td>
</tr>
<tr>
<td><code>6.1.2</code></td>
<td></td>
<td><code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>7.0.1</code></td>
<td></td>
<td><code>gompi/2022a</code>, <code>iimpi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="scp">scp<a class="headerlink" href="#scp" title="Permanent link">&para;</a></h3>
<p>The scp.py module uses a paramiko transport to send and recieve files via the scp1 protocol.</p>
<p><em>homepage</em>: <a href="https://github.com/jbardin/scp.py">https://github.com/jbardin/scp.py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.13.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="scpred">scPred<a class="headerlink" href="#scpred" title="Permanent link">&para;</a></h3>
<p>scPred package for cell type prediction from scRNA-seq data </p>
<p><em>homepage</em>: <a href="https://github.com/powellgenomicslab/scPred">https://github.com/powellgenomicslab/scPred</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.2</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="scrappie">Scrappie<a class="headerlink" href="#scrappie" title="Permanent link">&para;</a></h3>
<p>Scrappie is a technology demonstrator for the Oxford Nanopore Research Algorithms group.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/scrappie">https://github.com/nanoporetech/scrappie</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="scvelo">scVelo<a class="headerlink" href="#scvelo" title="Permanent link">&para;</a></h3>
<p>scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using  dynamical modeling.</p>
<p><em>homepage</em>: <a href="https://scvelo.org">https://scvelo.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.24</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.2.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scvi-tools">scvi-tools<a class="headerlink" href="#scvi-tools" title="Permanent link">&para;</a></h3>
<p>scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and analysis of single-cell omics data, built on top of PyTorch and AnnData.</p>
<p><em>homepage</em>: <a href="https://github.com/scverse/scvi-tools">https://github.com/scverse/scvi-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.16.4</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.16.4</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="scythe">Scythe<a class="headerlink" href="#scythe" title="Permanent link">&para;</a></h3>
<p>Scythe uses a Naive Bayesian approach to classify contaminant substrings in  sequence reads. It considers quality information, which can make it robust in  picking out 3'-end adapters, which often include poor quality bases. </p>
<p><em>homepage</em>: <a href="https://github.com/ucdavis-bioinformatics/scythe">https://github.com/ucdavis-bioinformatics/scythe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.994</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sdcc">SDCC<a class="headerlink" href="#sdcc" title="Permanent link">&para;</a></h3>
<p>SDCC is a retargettable, optimizing ANSI - C compiler suite that targets  the Intel MCS51 based microprocessors (8031, 8032, 8051, 8052, etc.), Maxim (formerly Dallas)  DS80C390 variants, Freescale (formerly Motorola) HC08 based (hc08, s08) and Zilog Z80 based  MCUs (z80, z180, gbz80, Rabbit 2000/3000, Rabbit 3000A). Work is in progress on supporting the  Microchip PIC16 and PIC18 targets. It can be retargeted for other microprocessors.</p>
<p><em>homepage</em>: <a href="http://sdcc.sourceforge.net/">http://sdcc.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="sdl">SDL<a class="headerlink" href="#sdl" title="Permanent link">&para;</a></h3>
<p>SDL: Simple DirectMedia Layer, a cross-platform multimedia library</p>
<p><em>homepage</em>: <a href="https://www.libsdl.org/">https://www.libsdl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.15</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sdl2">SDL2<a class="headerlink" href="#sdl2" title="Permanent link">&para;</a></h3>
<p>SDL: Simple DirectMedia Layer, a cross-platform multimedia library</p>
<p><em>homepage</em>: <a href="https://www.libsdl.org/">https://www.libsdl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.4</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0.9</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.0.10</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.14</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.0.20</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.0.22</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.26.3</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sdl2_image">SDL2_image<a class="headerlink" href="#sdl2_image" title="Permanent link">&para;</a></h3>
<p>SDL_image is an image file loading library.</p>
<p><em>homepage</em>: <a href="https://www.libsdl.org/projects/SDL_image/">https://www.libsdl.org/projects/SDL_image/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sdl_image">SDL_image<a class="headerlink" href="#sdl_image" title="Permanent link">&para;</a></h3>
<p>SDL_image is an image file loading library.</p>
<p><em>homepage</em>: <a href="https://www.libsdl.org/projects/SDL_image/release-1.2.html">https://www.libsdl.org/projects/SDL_image/release-1.2.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.12</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sdsl">SDSL<a class="headerlink" href="#sdsl" title="Permanent link">&para;</a></h3>
<p>The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing  succinct data structures.</p>
<p><em>homepage</em>: <a href="https://github.com/simongog/sdsl-lite">https://github.com/simongog/sdsl-lite</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1-20191211</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="seaborn">Seaborn<a class="headerlink" href="#seaborn" title="Permanent link">&para;</a></h3>
<p>Seaborn is a Python visualization library based on matplotlib.  It provides a high-level interface for drawing attractive statistical graphics. </p>
<p><em>homepage</em>: <a href="https://seaborn.pydata.org/">https://seaborn.pydata.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>foss/2019a</code>, <code>intel/2018b</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="searchgui">SearchGUI<a class="headerlink" href="#searchgui" title="Permanent link">&para;</a></h3>
<p>SearchGUI is a user-friendly open-source graphical user interface for configuring and  running proteomics identification search engines and de novo sequencing algorithms,  currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,  Novor and DirecTag.</p>
<p><em>homepage</em>: <a href="https://github.com/compomics/searchgui">https://github.com/compomics/searchgui</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.3</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="seaview">SeaView<a class="headerlink" href="#seaview" title="Permanent link">&para;</a></h3>
<p>SeaView is a multiplatform, graphical user interface for multiple sequence alignment  and molecular phylogeny.</p>
<p><em>homepage</em>: <a href="https://doua.prabi.fr/software/seaview">https://doua.prabi.fr/software/seaview</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.5</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="secapr">SECAPR<a class="headerlink" href="#secapr" title="Permanent link">&para;</a></h3>
<p>SECAPR is a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments</p>
<p><em>homepage</em>: <a href="https://github.com/AntonelliLab/seqcap_processor">https://github.com/AntonelliLab/seqcap_processor</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.15</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="seeder">Seeder<a class="headerlink" href="#seeder" title="Permanent link">&para;</a></h3>
<p>Seeder is a framework for DNA motif discovery.</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Seeder">https://metacpan.org/pod/Seeder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.01</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="segemehl">segemehl<a class="headerlink" href="#segemehl" title="Permanent link">&para;</a></h3>
<p>segemehl is a software to map short sequencer reads to reference genomes.   Unlike other methods, segemehl is able to detect not only mismatches but also insertions   and deletions. Furthermore, segemehl is not limited to a specific read length and is able   to mapprimer- or polyadenylation contaminated reads correctly. segemehl implements a matching   strategy based on enhanced suffix arrays (ESA). Segemehl now supports the SAM format, reads   gziped queries to save both disk and memory space and allows bisulfite sequencing mapping   and split read mapping. </p>
<p><em>homepage</em>: <a href="http://www.bioinf.uni-leipzig.de/Software/segemehl/">http://www.bioinf.uni-leipzig.de/Software/segemehl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.3.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/8.3.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="segmentation-models">segmentation-models<a class="headerlink" href="#segmentation-models" title="Permanent link">&para;</a></h3>
<p>Python library with Neural Networks for Image Segmentation based on Keras and TensorFlow.</p>
<p><em>homepage</em>: <a href="https://github.com/qubvel/segmentation_models">https://github.com/qubvel/segmentation_models</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="seissol">SeisSol<a class="headerlink" href="#seissol" title="Permanent link">&para;</a></h3>
<p>SeisSol is a software package for simulating wave propagation and dynamic rupture based on  the arbitrary high-order accurate derivative discontinuous Galerkin method (ADER-DG).</p>
<p><em>homepage</em>: <a href="http://www.seissol.org">http://www.seissol.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>201703</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="selestim">SelEstim<a class="headerlink" href="#selestim" title="Permanent link">&para;</a></h3>
<p>SelEstim is aimed at distinguishing neutral from selected polymorphisms and estimate   the intensity of selection at the latter. The SelEstim model accounts explicitly for positive selection,   and it is assumed that all marker loci in the dataset are responding to selection, to some extent</p>
<p><em>homepage</em>: <a href="http://www1.montpellier.inra.fr/CBGP/software/selestim/index.html">http://www1.montpellier.inra.fr/CBGP/software/selestim/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>-Linux-64bits</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="selfies">SELFIES<a class="headerlink" href="#selfies" title="Permanent link">&para;</a></h3>
<p>Robust representation of semantically constrained graphs, in particular for molecules in chemistry</p>
<p><em>homepage</em>: <a href="https://github.com/aspuru-guzik-group/selfies">https://github.com/aspuru-guzik-group/selfies</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sentencepiece">SentencePiece<a class="headerlink" href="#sentencepiece" title="Permanent link">&para;</a></h3>
<p>Unsupervised text tokenizer for Neural Network-based text generation.</p>
<p><em>homepage</em>: <a href="https://github.com/google/sentencepiece">https://github.com/google/sentencepiece</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.85</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>0.1.94</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>0.1.96</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>0.1.97</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sep">sep<a class="headerlink" href="#sep" title="Permanent link">&para;</a></h3>
<p>Python and C library for Source Extraction and Photometry.  (this easyconfig provides python library only)</p>
<p><em>homepage</em>: <a href="https://sep.readthedocs.io">https://sep.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="sepp">SEPP<a class="headerlink" href="#sepp" title="Permanent link">&para;</a></h3>
<p>SATe-enabled Phylogenetic Placement - addresses the problem of phylogenetic placement of short reads into reference alignments and trees.</p>
<p><em>homepage</em>: <a href="https://github.com/smirarab/sepp">https://github.com/smirarab/sepp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.3.10</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.3.10</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.5.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.5.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="seq-gen">Seq-Gen<a class="headerlink" href="#seq-gen" title="Permanent link">&para;</a></h3>
<p>Seq-Gen is a program that will simulate the evolution of nucleotide or amino acid sequences  along a phylogeny, using common models of the substitution process.</p>
<p><em>homepage</em>: <a href="http://tree.bio.ed.ac.uk/software/Seq-Gen">http://tree.bio.ed.ac.uk/software/Seq-Gen</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.4</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="seq2hla">seq2HLA<a class="headerlink" href="#seq2hla" title="Permanent link">&para;</a></h3>
<p>In-silico method written in Python and R to determine HLA genotypes of a sample.  seq2HLA takes standard RNA-Seq sequence reads in fastq format as input,  uses a bowtie index comprising all HLA alleles and outputs the most likely  HLA class I and class II genotypes (in 4 digit resolution),  a p-value for each call, and the expression of each class. </p>
<p><em>homepage</em>: <a href="https://github.com/TRON-Bioinformatics/seq2HLA">https://github.com/TRON-Bioinformatics/seq2HLA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="seqan">SeqAn<a class="headerlink" href="#seqan" title="Permanent link">&para;</a></h3>
<p>SeqAn is an open source C++ library of efficient algorithms and data structures  for the analysis of sequences with the focus on biological data</p>
<p><em>homepage</em>: <a href="https://www.seqan.de/">https://www.seqan.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>-library</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.2</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="seqan3">SeqAn3<a class="headerlink" href="#seqan3" title="Permanent link">&para;</a></h3>
<p>SeqAn is an open source C++ library   of efficient algorithms and data structures for the   analysis of sequences with the focus on biological   data. Our library applies a unique generic design   that guarantees high performance, generality,   extensibility, and integration with other libraries.   SeqAn is easy to use and simplifies the development   of new software tools with a minimal loss of   performance. </p>
<p><em>homepage</em>: <a href="https://github.com/seqan/seqan3">https://github.com/seqan/seqan3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="seqkit">SeqKit<a class="headerlink" href="#seqkit" title="Permanent link">&para;</a></h3>
<p>SeqKit - a cross-platform and ultrafast toolkit for FASTA/Q file manipulation</p>
<p><em>homepage</em>: <a href="https://bioinf.shenwei.me/seqkit/">https://bioinf.shenwei.me/seqkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.13.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="seqlib">SeqLib<a class="headerlink" href="#seqlib" title="Permanent link">&para;</a></h3>
<p>C++ interface to HTSlib, BWA-MEM and Fermi.</p>
<p><em>homepage</em>: <a href="https://github.com/walaj/SeqLib">https://github.com/walaj/SeqLib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="seqmagick">Seqmagick<a class="headerlink" href="#seqmagick" title="Permanent link">&para;</a></h3>
<p>We often have to convert between sequence formats and do  little tasks on them, and it's not worth writing scripts for that. Seqmagick  is a kickass little utility built in the spirit of imagemagick to expose the  file format conversion in Biopython in a convenient way. Instead of having a  big mess of scripts, there is one that takes arguments.</p>
<p><em>homepage</em>: <a href="https://fhcrc.github.io/seqmagick/">https://fhcrc.github.io/seqmagick/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="seqprep">SeqPrep<a class="headerlink" href="#seqprep" title="Permanent link">&para;</a></h3>
<p>Tool for stripping adaptors and/or merging paired reads with overlap into single reads.</p>
<p><em>homepage</em>: <a href="https://github.com/jstjohn/SeqPrep">https://github.com/jstjohn/SeqPrep</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="seqtk">seqtk<a class="headerlink" href="#seqtk" title="Permanent link">&para;</a></h3>
<p>Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.   It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/seqtk/">https://github.com/lh3/seqtk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="serf">Serf<a class="headerlink" href="#serf" title="Permanent link">&para;</a></h3>
<p>The serf library is a high performance C-based HTTP client library  built upon the Apache Portable Runtime (APR) library</p>
<p><em>homepage</em>: <a href="http://serf.apache.org/">http://serf.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2017b</code>, <code>intel/2017b</code>, <code>iomkl/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="setuptools">setuptools<a class="headerlink" href="#setuptools" title="Permanent link">&para;</a></h3>
<p>Easily download, build, install, upgrade, and uninstall Python packages</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/setuptools">https://pypi.org/project/setuptools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>41.0.1</code></td>
<td><code>-py3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="seurat">Seurat<a class="headerlink" href="#seurat" title="Permanent link">&para;</a></h3>
<p>Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.</p>
<p><em>homepage</em>: <a href="https://satijalab.org/seurat">https://satijalab.org/seurat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0.16</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="seuratdata">SeuratData<a class="headerlink" href="#seuratdata" title="Permanent link">&para;</a></h3>
<p>SeuratData is a mechanism for distributing datasets in the form of Seurat objects using R's internal package and data management systems.</p>
<p><em>homepage</em>: <a href="https://github.com/satijalab/seurat-data">https://github.com/satijalab/seurat-data</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210514</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="seuratdisk">SeuratDisk<a class="headerlink" href="#seuratdisk" title="Permanent link">&para;</a></h3>
<p>Interfaces for HDF5-based Single Cell File Formats</p>
<p><em>homepage</em>: <a href="https://github.com/mojaveazure/seurat-disk">https://github.com/mojaveazure/seurat-disk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.0.9020</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="seuratwrappers">SeuratWrappers<a class="headerlink" href="#seuratwrappers" title="Permanent link">&para;</a></h3>
<p>SeuratWrappers is a collection of community-provided methods and extensions for Seurat</p>
<p><em>homepage</em>: <a href="https://github.com/satijalab/seurat-wrappers">https://github.com/satijalab/seurat-wrappers</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20210528</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="sf">sf<a class="headerlink" href="#sf" title="Permanent link">&para;</a></h3>
<p>Support for simple features, a standardized way to encode spatial vector data. Binds to GDAL for reading and writing data, to GEOS for geometrical operations, and to PROJ for projection conversions and datum transformations.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=sf">https://cran.r-project.org/package=sf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9-5</code></td>
<td><code>-R-4.0.0-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="sfftk">sfftk<a class="headerlink" href="#sfftk" title="Permanent link">&para;</a></h3>
<p>sfftk is a set of utilities that facilitate creation, conversion and             modification of Electron Microscopy Data Bank - Segmentation File Format         (EMDB-SFF) files. EMDB-SFF is an open, community-driven file format to handle    annotated segmentations and subtomogram averages that facilitates segmentation   file interchange. It is written in Python and provides both a command-line suite of commands and a Python API.</p>
<p><em>homepage</em>: <a href="https://emdb-empiar.github.io/EMDB-SFF/">https://emdb-empiar.github.io/EMDB-SFF/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.4</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="shannon">Shannon<a class="headerlink" href="#shannon" title="Permanent link">&para;</a></h3>
<p>Shannon is a program for assembling transcripts from RNA-Seq data</p>
<p><em>homepage</em>: <a href="https://sreeramkannan.github.io/Shannon/">https://sreeramkannan.github.io/Shannon/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170511</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="shap">SHAP<a class="headerlink" href="#shap" title="Permanent link">&para;</a></h3>
<p>SHAP (SHapley Additive exPlanations) is a game theoretic approach to explain the output of any  machine learning model. It connects optimal credit allocation with local explanations using the classic Shapley  values from game theory and their related extensions.</p>
<p><em>homepage</em>: <a href="https://github.com/slundberg/shap">https://github.com/slundberg/shap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.35.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="shapaar">shapAAR<a class="headerlink" href="#shapaar" title="Permanent link">&para;</a></h3>
<p>An R package for the extraction, analysis and classification of (not only) archaeological objects  from scanned images.</p>
<p><em>homepage</em>: <a href="https://github.com/ISAAKiel/shapAAR">https://github.com/ISAAKiel/shapAAR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0-20180425</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="shapeit">SHAPEIT<a class="headerlink" href="#shapeit" title="Permanent link">&para;</a></h3>
<p>SHAPEIT is a fast and accurate method for estimation of  haplotypes (aka phasing) from genotype or sequencing data.</p>
<p><em>homepage</em>: <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.r837</code></td>
<td><code>.GLIBCv2.12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.r904</code></td>
<td><code>.glibcv2.17</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="shapeit4">SHAPEIT4<a class="headerlink" href="#shapeit4" title="Permanent link">&para;</a></h3>
<p>SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. </p>
<p><em>homepage</em>: <a href="https://odelaneau.github.io/shapeit4/">https://odelaneau.github.io/shapeit4/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.3</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="shapely">Shapely<a class="headerlink" href="#shapely" title="Permanent link">&para;</a></h3>
<p>Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects. It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.</p>
<p><em>homepage</em>: <a href="https://github.com/Toblerity/Shapely">https://github.com/Toblerity/Shapely</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.8.1.post1</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.8.2</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.8a1</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="sharutils">sharutils<a class="headerlink" href="#sharutils" title="Permanent link">&para;</a></h3>
<p>GNU shar makes so-called shell archives out of many files, preparing them for transmission by  electronic mail services, while unshar helps unpacking shell archives after reception.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/sharutils/">https://www.gnu.org/software/sharutils/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.15</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="shasta">Shasta<a class="headerlink" href="#shasta" title="Permanent link">&para;</a></h3>
<p>The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include: Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k  10). </p>
<p><em>homepage</em>: <a href="https://github.com/chanzuckerberg/shasta">https://github.com/chanzuckerberg/shasta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="shengbte">ShengBTE<a class="headerlink" href="#shengbte" title="Permanent link">&para;</a></h3>
<p>A solver for the Boltzmann transport equation for phonons.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/sousaw/shengbte">https://bitbucket.org/sousaw/shengbte</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="shift">shift<a class="headerlink" href="#shift" title="Permanent link">&para;</a></h3>
<p>Shift is a framework for Self-Healing Independent File Transfer that provides high performance and resilience for local and remote transfers through a variety of techniques. </p>
<p><em>homepage</em>: <a href="http://people.nas.nasa.gov/~kolano/projects/shift.html">http://people.nas.nasa.gov/~kolano/projects/shift.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="shore">SHORE<a class="headerlink" href="#shore" title="Permanent link">&para;</a></h3>
<p>SHORE, for Short Read, is a mapping and analysis pipeline for short read data produced on the Illumina platform. </p>
<p><em>homepage</em>: <a href="http://1001genomes.org/software/shore.html">http://1001genomes.org/software/shore.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="short-pair">Short-Pair<a class="headerlink" href="#short-pair" title="Permanent link">&para;</a></h3>
<p>Sensitive Short Read Homology Search for Paired-End Reads</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/short-pair">https://sourceforge.net/projects/short-pair</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170125</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>20170125</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="shovill">shovill<a class="headerlink" href="#shovill" title="Permanent link">&para;</a></h3>
<p>Faster SPAdes assembly of Illumina reads</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/shovill">https://github.com/tseemann/shovill</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="shrinkwrap">shrinkwrap<a class="headerlink" href="#shrinkwrap" title="Permanent link">&para;</a></h3>
<p>A std::streambuf wrapper for compression formats.</p>
<p><em>homepage</em>: <a href="https://github.com/jonathonl/shrinkwrap">https://github.com/jonathonl/shrinkwrap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0-beta</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="shtns">SHTns<a class="headerlink" href="#shtns" title="Permanent link">&para;</a></h3>
<p>Spherical Harmonic Transform library aimed at high performance numerical simulations in spherical geometries.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/nschaeff/shtns">https://bitbucket.org/nschaeff/shtns</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.5.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sibelia">Sibelia<a class="headerlink" href="#sibelia" title="Permanent link">&para;</a></h3>
<p>Sibelia: A comparative genomics tool: It assists biologists in analysing   the genomic variations that correlate with pathogens, or the genomic changes that help   microorganisms adapt in different environments. Sibelia will also be helpful for the   evolutionary and genome rearrangement studies for multiple strains of microorganisms.</p>
<p><em>homepage</em>: <a href="http://bioinf.spbau.ru/en/sibelia">http://bioinf.spbau.ru/en/sibelia</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.6</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.0.7</code></td>
<td><code>foss/2018b</code>, <code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="sicer2">SICER2<a class="headerlink" href="#sicer2" title="Permanent link">&para;</a></h3>
<p>Redesigned and improved ChIP-seq broad peak calling tool SICER</p>
<p><em>homepage</em>: <a href="https://zanglab.github.io/SICER2/">https://zanglab.github.io/SICER2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="sickle">sickle<a class="headerlink" href="#sickle" title="Permanent link">&para;</a></h3>
<p>Windowed Adaptive Trimming for fastq files using quality </p>
<p><em>homepage</em>: <a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.33</code></td>
<td><code>foss/2017a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="siesta">Siesta<a class="headerlink" href="#siesta" title="Permanent link">&para;</a></h3>
<p>SIESTA is both a method and its computer program implementation, to perform efficient electronic  structure calculations and ab initio molecular dynamics simulations of molecules and solids.</p>
<p><em>homepage</em>: <a href="http://departments.icmab.es/leem/siesta">http://departments.icmab.es/leem/siesta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.1-b2</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.1-b3</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.1-b4</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.1-MaX-1.0</code></td>
<td></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.1-MaX-1.0</code></td>
<td><code>-PEXSI</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2020a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="signalp">SignalP<a class="headerlink" href="#signalp" title="Permanent link">&para;</a></h3>
<p>SignalP predicts the presence  and  location of signal peptide  cleavage sites in  amino acid  sequences  from  different organisms</p>
<p><em>homepage</em>: <a href="https://services.healthtech.dtu.dk/software.php">https://services.healthtech.dtu.dk/software.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0b</code></td>
<td><code>-Linux</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0g</code></td>
<td><code>-fast</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>6.0g</code></td>
<td><code>-fast-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="silhouetterank">silhouetteRank<a class="headerlink" href="#silhouetterank" title="Permanent link">&para;</a></h3>
<p>silhouetteRank is a tool for finding spatially variable genes based on computing silhouette  coefficient from binarized spatial gene expression data</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/silhouetteRank">https://pypi.org/project/silhouetteRank</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5.13</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="silx">silx<a class="headerlink" href="#silx" title="Permanent link">&para;</a></h3>
<p>The silx project provides a collection of Python packages to support the  development of data assessment, reduction and analysis applications at synchrotron radiation facilities.</p>
<p><em>homepage</em>: <a href="http://www.silx.org/">http://www.silx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.13.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.14.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="simint">simint<a class="headerlink" href="#simint" title="Permanent link">&para;</a></h3>
<p>Simint is a vectorized implementation of the Obara-Saika (OS)  method of calculating electron repulsion integrals. Speedup is gained by  vectorizing the primitive loop of the OS algorithm, with additional vectorization  and optimizations left to the compiler.</p>
<p><em>homepage</em>: <a href="https://www.bennyp.org/research/simint/">https://www.bennyp.org/research/simint/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>-lmax-5-vec-avx-psi4</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="simnibs">SimNIBS<a class="headerlink" href="#simnibs" title="Permanent link">&para;</a></h3>
<p>SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation.</p>
<p><em>homepage</em>: <a href="https://simnibs.github.io/simnibs">https://simnibs.github.io/simnibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.4</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="simpeg">SimPEG<a class="headerlink" href="#simpeg" title="Permanent link">&para;</a></h3>
<p>Simulation and Parameter Estimation in Geophysics: a python package for simulation and gradient based parameter estimation in the context of geophysical applications.</p>
<p><em>homepage</em>: <a href="https://simpeg.xyz">https://simpeg.xyz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.14.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.18.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="simple">SIMPLE<a class="headerlink" href="#simple" title="Permanent link">&para;</a></h3>
<p>Single-particle IMage Processing Linux Engine SIMPLE is an open-source software package for analysis of cryogenic transmission electron microscopy (cryo-EM)  movies of single-particles (Single-Particle Analysis, SPA).</p>
<p><em>homepage</em>: <a href="https://simplecryoem.com/">https://simplecryoem.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="simpleelastix">SimpleElastix<a class="headerlink" href="#simpleelastix" title="Permanent link">&para;</a></h3>
<p>Multi-lingual medical image registration library.</p>
<p><em>homepage</em>: <a href="https://simpleelastix.github.io">https://simpleelastix.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="simpleitk">SimpleITK<a class="headerlink" href="#simpleitk" title="Permanent link">&para;</a></h3>
<p>SimpleITK is a simplified programming interface to the algorithms and data structures of  the Insight Toolkit (ITK).</p>
<p><em>homepage</em>: <a href="https://www.simpleitk.org">https://www.simpleitk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.1.1.2</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="simpy">simpy<a class="headerlink" href="#simpy" title="Permanent link">&para;</a></h3>
<p>SimPy is a process-based discrete-event simulation framework based on standard Python.</p>
<p><em>homepage</em>: <a href="https://simpy.readthedocs.io">https://simpy.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.11</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.0.11</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="simvascular">SimVascular<a class="headerlink" href="#simvascular" title="Permanent link">&para;</a></h3>
<p>SimVascular is an open source software suite for cardiovascular simulation, providing a complete  pipeline from medical image data to 3D model construction, meshing, and blood flow simulation.</p>
<p><em>homepage</em>: <a href="http://simvascular.github.io/">http://simvascular.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.16.0406</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="singlem">SingleM<a class="headerlink" href="#singlem" title="Permanent link">&para;</a></h3>
<p>SingleM is a tool to find the abundances of discrete operational taxonomic units (OTUs) directly  from shotgun metagenome data, without heavy reliance on reference sequence databases.</p>
<p><em>homepage</em>: <a href="https://github.com/wwood/singlem">https://github.com/wwood/singlem</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="singular">Singular<a class="headerlink" href="#singular" title="Permanent link">&para;</a></h3>
<p>Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory.</p>
<p><em>homepage</em>: <a href="https://www.singular.uni-kl.de/">https://www.singular.uni-kl.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="sinvict">SiNVICT<a class="headerlink" href="#sinvict" title="Permanent link">&para;</a></h3>
<p>SiNVICT is a tool for the detection of SNVs and indels from cfDNA/ctDNA samples obtained  by ultra-deep sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/sfu-compbio/sinvict">https://github.com/sfu-compbio/sinvict</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-20180817</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sionlib">SIONlib<a class="headerlink" href="#sionlib" title="Permanent link">&para;</a></h3>
<p>SIONlib is a scalable I/O library for parallel access to task-local files.  The library not only supports writing and reading binary data to or from  several thousands of processors into a single or a small number of physical  files, but also provides global open and close functions to access SIONlib  files in parallel. This package provides a stripped-down installation of  SIONlib for use with performance tools (e.g., Score-P), with renamed symbols  to avoid conflicts when an application using SIONlib itself is linked against  a tool requiring a different SIONlib version. </p>
<p><em>homepage</em>: <a href="https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html">https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td><code>-tools</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td><code>-tools</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.7.4</code></td>
<td><code>-tools</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.7.6</code></td>
<td><code>-tools</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.7.7</code></td>
<td><code>-tools</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sip">SIP<a class="headerlink" href="#sip" title="Permanent link">&para;</a></h3>
<p>SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.</p>
<p><em>homepage</em>: <a href="http://www.riverbankcomputing.com/software/sip/">http://www.riverbankcomputing.com/software/sip/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.18</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.18.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.18.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.19</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.19.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.19.8</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.19.8</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.19.13</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="siscone">siscone<a class="headerlink" href="#siscone" title="Permanent link">&para;</a></h3>
<p>Hadron Seedless Infrared-Safe Cone jet algorithm</p>
<p><em>homepage</em>: <a href="https://siscone.hepforge.org/">https://siscone.hepforge.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.5</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sisso">SISSO<a class="headerlink" href="#sisso" title="Permanent link">&para;</a></h3>
<p>A data-driven method combining symbolic regression and compressed sensing toward accurate &amp;  interpretable models.</p>
<p><em>homepage</em>: <a href="https://github.com/rouyang2017/SISSO">https://github.com/rouyang2017/SISSO</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>3.1-20220324</code></td>
<td><code>iimpi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sisso_1">SISSO++<a class="headerlink" href="#sisso_1" title="Permanent link">&para;</a></h3>
<p>C++ implementation of SISSO with built in Python bindings for an efficient python interface</p>
<p><em>homepage</em>: <a href="https://gitlab.com/sissopp_developers/sissopp">https://gitlab.com/sissopp_developers/sissopp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="skesa">SKESA<a class="headerlink" href="#skesa" title="Permanent link">&para;</a></h3>
<p>SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/SKESA">https://github.com/ncbi/SKESA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>_saute.1.3.0_1</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sketchmap">sketchmap<a class="headerlink" href="#sketchmap" title="Permanent link">&para;</a></h3>
<p>Sketch-map is a dimensionality reduction algorithm that is particularly well suited to examining  the high-dimensionality data that is routinely produced in atomistic simulations.</p>
<p><em>homepage</em>: <a href="http://sketchmap.org/">http://sketchmap.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170130</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="skewer">skewer<a class="headerlink" href="#skewer" title="Permanent link">&para;</a></h3>
<p>skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter  trimming and it is specially designed for processing next-generation sequencing (NGS) paired-end sequences.</p>
<p><em>homepage</em>: <a href="https://github.com/relipmoc/skewer">https://github.com/relipmoc/skewer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="sklearn-pandas">sklearn-pandas<a class="headerlink" href="#sklearn-pandas" title="Permanent link">&para;</a></h3>
<p>This module provides a bridge between Scikit-Learn's machine learning methods and pandas-style Data Frames. In particular, it provides a way to map DataFrame columns to transformations, which are later recombined into features.</p>
<p><em>homepage</em>: <a href="https://github.com/scikit-learn-contrib/sklearn-pandas">https://github.com/scikit-learn-contrib/sklearn-pandas</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sklearn-som">sklearn-som<a class="headerlink" href="#sklearn-som" title="Permanent link">&para;</a></h3>
<p>A simple, planar self-organizing map with methods similar to clustering methods in Scikit Learn.</p>
<p><em>homepage</em>: <a href="https://github.com/rileypsmith/sklearn-som">https://github.com/rileypsmith/sklearn-som</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="skorch">skorch<a class="headerlink" href="#skorch" title="Permanent link">&para;</a></h3>
<p>A scikit-learn compatible neural network library that wraps PyTorch.</p>
<p><em>homepage</em>: <a href="https://skorch.readthedocs.io/">https://skorch.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.0</code></td>
<td><code>-PyTorch-1.10.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="slamdunk">SlamDunk<a class="headerlink" href="#slamdunk" title="Permanent link">&para;</a></h3>
<p>SlamDunk is a novel, fully automated software tool for automated, robust, scalable and reproducible SLAMseq data analysis.</p>
<p><em>homepage</em>: <a href="https://t-neumann.github.io/slamdunk">https://t-neumann.github.io/slamdunk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="slatec">SLATEC<a class="headerlink" href="#slatec" title="Permanent link">&para;</a></h3>
<p>SLATEC Common Mathematical Library, a comprehensive software library containing     over 1400 general purpose mathematical and statistical routines written in Fortran 77.</p>
<p><em>homepage</em>: <a href="https://www.netlib.org/slatec/">https://www.netlib.org/slatec/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/8.3.0</code>, <code>iccifort/2018.1.163-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="slepc">SLEPc<a class="headerlink" href="#slepc" title="Permanent link">&para;</a></h3>
<p>SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution  of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for  either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a  partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.</p>
<p><em>homepage</em>: <a href="https://www.grycap.upv.es/slepc/">https://www.grycap.upv.es/slepc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.3</code></td>
<td></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.9.2</code></td>
<td></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.11.0</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.12.2</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.12.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.12.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>3.14.2</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>3.15.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>3.17.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="slepc4py">slepc4py<a class="headerlink" href="#slepc4py" title="Permanent link">&para;</a></h3>
<p>Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/slepc/slepc4py">https://bitbucket.org/slepc/slepc4py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.12.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.15.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="sleuth">sleuth<a class="headerlink" href="#sleuth" title="Permanent link">&para;</a></h3>
<p>Investigate RNA-Seq transcript abundance from kallisto and perform differential expression analysis.</p>
<p><em>homepage</em>: <a href="http://pachterlab.github.io/sleuth">http://pachterlab.github.io/sleuth</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.29.0</code></td>
<td><code>-R-3.4.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.30.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="slidingwindow">slidingwindow<a class="headerlink" href="#slidingwindow" title="Permanent link">&para;</a></h3>
<p>slidingwindow is a simple little Python library for computing a set of windows into a larger dataset,  designed for use with image-processing algorithms that utilise a sliding window to break the processing up  into a series of smaller chunks.</p>
<p><em>homepage</em>: <a href="https://github.com/adamrehn/slidingwindow">https://github.com/adamrehn/slidingwindow</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.13</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.0.13</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="slim">SLiM<a class="headerlink" href="#slim" title="Permanent link">&para;</a></h3>
<p>SLiM is an evolutionary simulation framework that combines a powerful engine for population  genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios.</p>
<p><em>homepage</em>: <a href="https://messerlab.org/slim">https://messerlab.org/slim</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="slow5tools">slow5tools<a class="headerlink" href="#slow5tools" title="Permanent link">&para;</a></h3>
<p>slow5tools is a toolkit for converting (FAST5 &lt;-&gt; SLOW5), compressing, viewing, indexing  and manipulating data in SLOW5 format.</p>
<p><em>homepage</em>: <a href="https://hasindu2008.github.io/slow5tools">https://hasindu2008.github.io/slow5tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="smafa">smafa<a class="headerlink" href="#smafa" title="Permanent link">&para;</a></h3>
<p>Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences  which are all the same length.</p>
<p><em>homepage</em>: <a href="https://github.com/wwood/smafa">https://github.com/wwood/smafa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="smallgenomeutilities">smallgenomeutilities<a class="headerlink" href="#smallgenomeutilities" title="Permanent link">&para;</a></h3>
<p>The smallgenomeutilities are a collection of scripts that is  useful for dealing and manipulating NGS data of small viral genomes. They  are written in Python 3 with a small number of dependencies.</p>
<p><em>homepage</em>: <a href="https://github.com/cbg-ethz/smallgenomeutilities">https://github.com/cbg-ethz/smallgenomeutilities</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="smartdenovo">SMARTdenovo<a class="headerlink" href="#smartdenovo" title="Permanent link">&para;</a></h3>
<p>SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy.</p>
<p><em>homepage</em>: <a href="https://github.com/ruanjue/smartdenovo">https://github.com/ruanjue/smartdenovo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180219</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="smc">SMC++<a class="headerlink" href="#smc" title="Permanent link">&para;</a></h3>
<p>SMC++ is a program for estimating the size history of populations from whole genome sequence data.</p>
<p><em>homepage</em>: <a href="https://github.com/popgenmethods/smcpp">https://github.com/popgenmethods/smcpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.15.4</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="smfishhmrf">smfishHmrf<a class="headerlink" href="#smfishhmrf" title="Permanent link">&para;</a></h3>
<p>smFish spatial pattern mining and cell type prediction</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/smfishHmrf">https://pypi.org/project/smfishHmrf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="smithwaterman">smithwaterman<a class="headerlink" href="#smithwaterman" title="Permanent link">&para;</a></h3>
<p>smith-waterman-gotoh alignment algorithm.</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/smithwaterman">https://github.com/ekg/smithwaterman</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160702</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="smoldyn">Smoldyn<a class="headerlink" href="#smoldyn" title="Permanent link">&para;</a></h3>
<p>Smoldyn is a computer program for cell-scale biochemical simulations. It simulates each molecule of  interest individually to capture natural stochasticity and to yield nanometer-scale spatial resolution. It treats  other molecules implicitly, enabling it to simulate hundreds of thousands of molecules over several minutes of real  time. Simulated molecules diffuse, react, are confined by surfaces, and bind to membranes much as they would in a real  biological system. </p>
<p><em>homepage</em>: <a href="http://www.smoldyn.org/">http://www.smoldyn.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.48</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="smooth-topk">smooth-topk<a class="headerlink" href="#smooth-topk" title="Permanent link">&para;</a></h3>
<p>Smooth Loss Functions for Deep Top-k Classification</p>
<p><em>homepage</em>: <a href="https://github.com/oval-group/smooth-topk">https://github.com/oval-group/smooth-topk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0-20210817</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0-20210817</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="smrt-link">SMRT-Link<a class="headerlink" href="#smrt-link" title="Permanent link">&para;</a></h3>
<p>PacBios open-source SMRT Analysis software suite is designed for use with Single Molecule,   Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI   or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through   the extensive set of APIs provided</p>
<p><em>homepage</em>: <a href="https://www.pacb.com/support/software-downloads/">https://www.pacb.com/support/software-downloads/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.0.26412</code></td>
<td><code>-cli-tools-only</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.0.0.47841</code></td>
<td><code>-cli-tools-only</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="smv">SMV<a class="headerlink" href="#smv" title="Permanent link">&para;</a></h3>
<p>Smokeview is a visualization program that displays output of FDS and CFAST simulations.</p>
<p><em>homepage</em>: <a href="https://github.com/firemodels/smv">https://github.com/firemodels/smv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.7.17</code></td>
<td><code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="snakemake">snakemake<a class="headerlink" href="#snakemake" title="Permanent link">&para;</a></h3>
<p>The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.</p>
<p><em>homepage</em>: <a href="https://snakemake.readthedocs.io">https://snakemake.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.2.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>5.26.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.1.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>6.6.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>6.10.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>7.18.2</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>7.22.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="snap">SNAP<a class="headerlink" href="#snap" title="Permanent link">&para;</a></h3>
<p>Scalable Nucleotide Alignment Program -- a fast and accurate read aligner for  high-throughput sequencing data</p>
<p><em>homepage</em>: <a href="https://www.microsoft.com/en-us/research/project/snap">https://www.microsoft.com/en-us/research/project/snap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0beta.23</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>GCC/11.3.0</code>, <code>GCC/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="snap-hmm">SNAP-HMM<a class="headerlink" href="#snap-hmm" title="Permanent link">&para;</a></h3>
<p>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser. </p>
<p><em>homepage</em>: <a href="https://korflab.github.io/">https://korflab.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2013-11-29</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>GCC/8.3.0</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
<tr>
<td><code>20190603</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>20221022</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="snape-pooled">SNAPE-pooled<a class="headerlink" href="#snape-pooled" title="Permanent link">&para;</a></h3>
<p>SNAPE-pooled computes the probability distribution for the frequency of the minor allele  in a certain population, at a certain position in the genome.</p>
<p><em>homepage</em>: <a href="https://code.google.com/archive/p/snape-pooled/">https://code.google.com/archive/p/snape-pooled/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>r32</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="snaphu">snaphu<a class="headerlink" href="#snaphu" title="Permanent link">&para;</a></h3>
<p>SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping  proposed by Chen and Zebker</p>
<p><em>homepage</em>: <a href="https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/">https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="snappy">snappy<a class="headerlink" href="#snappy" title="Permanent link">&para;</a></h3>
<p>Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.</p>
<p><em>homepage</em>: <a href="https://github.com/google/snappy">https://github.com/google/snappy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.2</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>1.1.6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.1.7</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.1.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.1.9</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="snippy">snippy<a class="headerlink" href="#snippy" title="Permanent link">&para;</a></h3>
<p>Rapid haploid variant calling and core genome alignment</p>
<p><em>homepage</em>: <a href="https://github.com/tseemann/snippy">https://github.com/tseemann/snippy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.4.1</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Java-13-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>4.6.0</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="snp-sites">snp-sites<a class="headerlink" href="#snp-sites" title="Permanent link">&para;</a></h3>
<p>Finds SNP sites from a multi-FASTA alignment file.</p>
<p><em>homepage</em>: <a href="https://sanger-pathogens.github.io/snp-sites/">https://sanger-pathogens.github.io/snp-sites/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="snpeff">snpEff<a class="headerlink" href="#snpeff" title="Permanent link">&para;</a></h3>
<p>SnpEff is a variant annotation and effect prediction tool.  It annotates and predicts the effects of genetic variants (such as amino acid changes).</p>
<p><em>homepage</em>: <a href="https://pcingola.github.io/SnpEff/">https://pcingola.github.io/SnpEff/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1d</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3t</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td><code>-Java-13</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.0</code></td>
<td><code>-Java-13-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.0e</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>5.0e</code></td>
<td><code>-Java-13</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="snphylo">SNPhylo<a class="headerlink" href="#snphylo" title="Permanent link">&para;</a></h3>
<p>SNPhylo: a pipeline to generate a phylogenetic tree from huge SNP data</p>
<p><em>homepage</em>: <a href="http://chibba.pgml.uga.edu/snphylo/">http://chibba.pgml.uga.edu/snphylo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20140701</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20160204</code></td>
<td><code>-Python-2.7.14-R-3.4.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="snpomatic">SNPomatic<a class="headerlink" href="#snpomatic" title="Permanent link">&para;</a></h3>
<p>High throughput sequencing technologies generate large amounts of short reads.  Mapping these to a reference sequence consumes large amounts of processing  time and memory, and read mapping errors can lead to noisy or incorrect  alignments. SNP-o-matic is a fast, memory-efficient, and stringent read  mapping tool offering a variety of analytical output functions, with an  emphasis on genotyping. </p>
<p><em>homepage</em>: <a href="https://github.com/magnusmanske/snpomatic">https://github.com/magnusmanske/snpomatic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="soapaligner">SOAPaligner<a class="headerlink" href="#soapaligner" title="Permanent link">&para;</a></h3>
<p>SOAPaligner/soap2 is a member of the SOAP (Short Oligonucleotide   Analysis Package). It is an updated version of SOAP software for short oligonucleotide   alignment. </p>
<p><em>homepage</em>: <a href="http://soap.genomics.org.cn/soapaligner.html">http://soap.genomics.org.cn/soapaligner.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.21</code></td>
<td><code>_Linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="soapdenovo-trans">SOAPdenovo-Trans<a class="headerlink" href="#soapdenovo-trans" title="Permanent link">&para;</a></h3>
<p>SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. </p>
<p><em>homepage</em>: <a href="http://soap.genomics.org.cn/SOAPdenovo-Trans.html">http://soap.genomics.org.cn/SOAPdenovo-Trans.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.4</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="soapdenovo2">SOAPdenovo2<a class="headerlink" href="#soapdenovo2" title="Permanent link">&para;</a></h3>
<p>SOAPdenovo is a novel short-read assembly method that can build a   de novo draft assembly for human-sized genomes. The program is specially designed to   assemble Illumina short reads. It creates new opportunities for building reference   sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.   SOAPdenovo2 is the successor of SOAPdenovo.</p>
<p><em>homepage</em>: <a href="http://soap.genomics.org.cn/index.html">http://soap.genomics.org.cn/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>r240</code></td>
<td><code>GCC/5.4.0-2.26</code></td>
</tr>
<tr>
<td><code>r241</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>foss/2018a</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="soapfuse">SOAPfuse<a class="headerlink" href="#soapfuse" title="Permanent link">&para;</a></h3>
<p>SOAPfuse is an open source tool developed for genome-wide detection of fusion  transcripts from paired-end RNA-Seq data.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/p/soapfuse/wiki/Home/">https://sourceforge.net/p/soapfuse/wiki/Home/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.27</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.27</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="socat">socat<a class="headerlink" href="#socat" title="Permanent link">&para;</a></h3>
<p>socat is a relay for bidirectional data transfer between two independent data channels.</p>
<p><em>homepage</em>: <a href="http://www.dest-unreach.org/socat">http://www.dest-unreach.org/socat</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.3.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="soci">SOCI<a class="headerlink" href="#soci" title="Permanent link">&para;</a></h3>
<p>SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the  regular C++ code, staying entirely within the Standard C++.</p>
<p><em>homepage</em>: <a href="http://soci.sourceforge.net/">http://soci.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.1</code></td>
<td><code>GCC/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>4.0.3</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="solexaqa">SolexaQA++<a class="headerlink" href="#solexaqa" title="Permanent link">&para;</a></h3>
<p>SolexaQA calculates sequence quality statistics and creates visual representations  of data quality for second-generation sequencing data. Originally developed for the Illumina system  (historically known as Solexa), SolexaQA now also supports Ion Torrent and 454 data.</p>
<p><em>homepage</em>: <a href="http://solexaqa.sourceforge.net">http://solexaqa.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.5</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="sonic">sonic<a class="headerlink" href="#sonic" title="Permanent link">&para;</a></h3>
<p>Sonic is a simple algorithm for speeding up or slowing down speech.  However, it's optimized for speed ups of over 2X, unlike previous algorithms for changing speech rate.  The Sonic library is a very simple ANSI C library that is designed to easily be integrated into streaming voice applications, like TTS back ends. </p>
<p><em>homepage</em>: <a href="https://github.com/espeak-ng/sonic">https://github.com/espeak-ng/sonic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180202</code></td>
<td><code>gompi/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="soqt">SoQt<a class="headerlink" href="#soqt" title="Permanent link">&para;</a></h3>
<p>SoQt is a library which provides the glue between Systems in  Motion's Coin high-level 3D visualization library and the Qt 2D user interface  library. </p>
<p><em>homepage</em>: <a href="https://coin3d.github.io/SoQt/html/">https://coin3d.github.io/SoQt/html/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.0</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="sortmerna">SortMeRNA<a class="headerlink" href="#sortmerna" title="Permanent link">&para;</a></h3>
<p>SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</p>
<p><em>homepage</em>: <a href="http://bioinfo.lifl.fr/RNA/sortmerna/">http://bioinfo.lifl.fr/RNA/sortmerna/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="sox">SoX<a class="headerlink" href="#sox" title="Permanent link">&para;</a></h3>
<p>Sound eXchange, the Swiss Army knife of audio manipulation</p>
<p><em>homepage</em>: <a href="http://sox.sourceforge.net/">http://sox.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>14.4.2</code></td>
<td><code>GCC/8.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="soxt">SoXt<a class="headerlink" href="#soxt" title="Permanent link">&para;</a></h3>
<p>SoXt is an Xt/Motif glue library for Coin. It can also be used  on top of the SGI or TGS implementation of Open Inventor, and is designed to  be source code compatible with SGI's InventorXt library. </p>
<p><em>homepage</em>: <a href="https://coin3d.github.io/SoXt/html/">https://coin3d.github.io/SoXt/html/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spaceranger">SpaceRanger<a class="headerlink" href="#spaceranger" title="Permanent link">&para;</a></h3>
<p>Space Ranger is a set of analysis pipelines that process Visium spatial RNA-seq output and brightfield microscope images in order to detect tissue, align reads, generate feature-spot matrices, perform clustering and gene expression analysis, and place spots in spatial context on the slide image. </p>
<p><em>homepage</em>: <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spack">Spack<a class="headerlink" href="#spack" title="Permanent link">&para;</a></h3>
<p>Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific  software easy. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,  and all of these builds can coexist on the same machine.</p>
<p><em>homepage</em>: <a href="https://spack.io/">https://spack.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.10.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.16.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.17.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.17.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="spacy">spaCy<a class="headerlink" href="#spacy" title="Permanent link">&para;</a></h3>
<p>Industrial-strength Natural Language Processing (NLP) in Python.</p>
<p><em>homepage</em>: <a href="https://spacy.io/">https://spacy.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.4</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="spades">SPAdes<a class="headerlink" href="#spades" title="Permanent link">&para;</a></h3>
<p>Genome assembler for single-cell and isolates data sets</p>
<p><em>homepage</em>: <a href="http://bioinf.spbau.ru/en/spades">http://bioinf.spbau.ru/en/spades</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9.0</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>3.10.1</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>3.11.1</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.12.0</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.13.1</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.14.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.14.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>3.14.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>3.15.2</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>3.15.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>3.15.3</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.15.5</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spaln">spaln<a class="headerlink" href="#spaln" title="Permanent link">&para;</a></h3>
<p>Spaln (space-efficient spliced alignment) is a stand-alone program that maps  and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.</p>
<p><em>homepage</em>: <a href="https://github.com/ogotoh/spaln">https://github.com/ogotoh/spaln</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.3c</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.4.03</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.4.12</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.4.13f</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spark">Spark<a class="headerlink" href="#spark" title="Permanent link">&para;</a></h3>
<p>Spark is Hadoop MapReduce done in memory</p>
<p><em>homepage</em>: <a href="https://spark.apache.org">https://spark.apache.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Hadoop-2.6-Java-1.8.0_144</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Hadoop-2.6-Java-1.8.0_152</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Hadoop-2.6-Java-1.8.0_152-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Hadoop-2.7-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Hadoop-2.7-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Hadoop-2.7-Java-1.8-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.5</code></td>
<td><code>-Python-3.7.4-Java-1.8</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sparse-neighbors-search">sparse-neighbors-search<a class="headerlink" href="#sparse-neighbors-search" title="Permanent link">&para;</a></h3>
<p>A Python/C++ implementation of an approximate nearest neighbor search for sparse data structures  based on the idea of local sensitive hash functions.</p>
<p><em>homepage</em>: <a href="https://github.com/joachimwolff/sparse-neighbors-search">https://github.com/joachimwolff/sparse-neighbors-search</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sparsehash">sparsehash<a class="headerlink" href="#sparsehash" title="Permanent link">&para;</a></h3>
<p>An extremely memory-efficient hash_map implementation. 2 bits/entry overhead!  The SparseHash library contains several hash-map implementations, including  implementations that optimize for space or speed. </p>
<p><em>homepage</em>: <a href="https://github.com/sparsehash/sparsehash">https://github.com/sparsehash/sparsehash</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spatialde">SpatialDE<a class="headerlink" href="#spatialde" title="Permanent link">&para;</a></h3>
<p>SpatialDE is a method to identify genes which significantly depend on spatial coordinates in  non-linear and non-parametric ways. The intended applications are spatially resolved RNA-sequencing from e.g.  Spatial Transcriptomics, or in situ gene expression measurements from e.g. SeqFISH or MERFISH.</p>
<p><em>homepage</em>: <a href="https://pypi.org/project/SpatialDE">https://pypi.org/project/SpatialDE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="spatialreg">spatialreg<a class="headerlink" href="#spatialreg" title="Permanent link">&para;</a></h3>
<p>A collection of all the estimation functions for spatial cross-sectional models (on lattice/areal  data using spatial weights matrices) contained up to now in 'spdep', 'sphet' and 'spse'.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=spatialreg">https://cran.r-project.org/package=spatialreg</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1-5</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.1-8</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="spdlog">spdlog<a class="headerlink" href="#spdlog" title="Permanent link">&para;</a></h3>
<p>Very fast, header-only/compiled, C++ logging library.</p>
<p><em>homepage</em>: <a href="https://github.com/gabime/spdlog">https://github.com/gabime/spdlog</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spectra">SpectrA<a class="headerlink" href="#spectra" title="Permanent link">&para;</a></h3>
<p>Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++  library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library.</p>
<p><em>homepage</em>: <a href="https://spectralib.org/">https://spectralib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spectralmethods">spectral.methods<a class="headerlink" href="#spectralmethods" title="Permanent link">&para;</a></h3>
<p>Contains some implementations of Singular Spectrum Analysis (SSA) for the gapfilling and  spectral decomposition of time series.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/spectral.methods">https://cran.r-project.org/web/packages/spectral.methods</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.2.133</code></td>
<td><code>-R-3.4.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.7.2.133</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="speech_tools">speech_tools<a class="headerlink" href="#speech_tools" title="Permanent link">&para;</a></h3>
<p>The Edinburgh Speech Tools Library is a collection of C++ class, functions and related programs for manipulating the sorts of objects used in speech processing. It includes support for reading and writing waveforms, parameter files (LPC, Ceptra, F0) in various formats and converting between them. It also includes support for linguistic type objects and support for various label files and ngrams (with smoothing). </p>
<p><em>homepage</em>: &lt;['http://festvox.org/festival/']&gt;</p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spglib">spglib<a class="headerlink" href="#spglib" title="Permanent link">&para;</a></h3>
<p>Spglib is a library for finding and handling crystal symmetries written in C.</p>
<p><em>homepage</em>: <a href="https://atztogo.github.io/spglib/">https://atztogo.github.io/spglib/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.2</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>1.9.9</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.16.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spglib-python">spglib-python<a class="headerlink" href="#spglib-python" title="Permanent link">&para;</a></h3>
<p>Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/spglib">https://pypi.python.org/pypi/spglib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.4.2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9.5</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9.9.38</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.10.0.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.10.0.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.14.1.post0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code>, <code>intel/2019b</code>, <code>intelcuda/2019b</code></td>
</tr>
<tr>
<td><code>1.16.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>1.16.1</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>gomkl/2021a</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>1.16.3</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sphinx">Sphinx<a class="headerlink" href="#sphinx" title="Permanent link">&para;</a></h3>
<p>Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.  It was originally created for the new Python documentation, and it has excellent facilities for the   documentation of Python projects, but C/C++ is already supported as well, and it is planned to add   special support for other languages as well.</p>
<p><em>homepage</em>: <a href="http://sphinx.pocoo.org/">http://sphinx.pocoo.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.8</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.4.8</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.8.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="sphinx-rtd-theme">Sphinx-RTD-Theme<a class="headerlink" href="#sphinx-rtd-theme" title="Permanent link">&para;</a></h3>
<p>Sphinx theme designed to provide a great reader experience  for documentation users on both desktop and mobile devices.</p>
<p><em>homepage</em>: <a href="https://sphinx-rtd-theme.readthedocs.io">https://sphinx-rtd-theme.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spiceypy">SpiceyPy<a class="headerlink" href="#spiceypy" title="Permanent link">&para;</a></h3>
<p>SpiceyPy is a Python wrapper for the NAIF C SPICE Toolkit (N65)</p>
<p><em>homepage</em>: <a href="https://github.com/AndrewAnnex/SpiceyPy">https://github.com/AndrewAnnex/SpiceyPy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="spieceasi">SpiecEasi<a class="headerlink" href="#spieceasi" title="Permanent link">&para;</a></h3>
<p>Sparse InversE Covariance estimation for Ecological Association and Statistical Inference</p>
<p><em>homepage</em>: <a href="https://github.com/zdk123/SpiecEasi">https://github.com/zdk123/SpiecEasi</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-R-3.4.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>20160830</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="spladder">SplAdder<a class="headerlink" href="#spladder" title="Permanent link">&para;</a></h3>
<p>Splicing Adder is a toolbox for alternative splicing analysis based on RNA-Seq alignment data. Briefly, the software takes a given annotation and RNA-Seq read alignments in standardized formats, transforms the annotation into a splicing graph representation, augments the splicing graph with additional information extracted from the read data, extracts alternative splicing events from the graph and quantifies the events based on the alignment data.</p>
<p><em>homepage</em>: <a href="https://github.com/ratschlab/spladder">https://github.com/ratschlab/spladder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="splash">SPLASH<a class="headerlink" href="#splash" title="Permanent link">&para;</a></h3>
<p>SPLASH is a free and open source visualisation tool  for Smoothed Particle Hydrodynamics (SPH) simulations.</p>
<p><em>homepage</em>: <a href="http://users.monash.edu.au/~dprice/splash/index.html">http://users.monash.edu.au/~dprice/splash/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="splicemap">SpliceMap<a class="headerlink" href="#splicemap" title="Permanent link">&para;</a></h3>
<p>SpliceMap is a de novo splice junction discovery and alignment tool.  It offers high sensitivity and support for arbitrary RNA-seq read lengths.</p>
<p><em>homepage</em>: <a href="https://web.stanford.edu/group/wonglab/SpliceMap/">https://web.stanford.edu/group/wonglab/SpliceMap/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3.5.2</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
</tbody>
</table>
<h3 id="split-seq">split-seq<a class="headerlink" href="#split-seq" title="Permanent link">&para;</a></h3>
<p>Analysis tools for split-seq.</p>
<p><em>homepage</em>: <a href="https://github.com/yjzhang/split-seq-pipeline">https://github.com/yjzhang/split-seq-pipeline</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190717</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="splitref">splitRef<a class="headerlink" href="#splitref" title="Permanent link">&para;</a></h3>
<p>splitRef splits a reference haplotype file into smaller files   with subsets of markers. The current version is a pre-release. </p>
<p><em>homepage</em>: <a href="http://www.sph.umich.edu/csg/yli/splitRef/index.html">http://www.sph.umich.edu/csg/yli/splitRef/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="spm">SPM<a class="headerlink" href="#spm" title="Permanent link">&para;</a></h3>
<p>SPM (Statistical Parametric Mapping) refers to the construction  and assessment of spatially extended statistical processes used to test hypo-  theses about functional imaging data. These ideas have been instantiated in  software that is called SPM.  The SPM software package has been designed for the analysis of brain imaging  data sequences. The sequences can be a series of images from different cohorts,  or time-series from the same subject. The current release is designed for the  analysis of fMRI, PET, SPECT, EEG and MEG. </p>
<p><em>homepage</em>: <a href="https://www.fil.ion.ucl.ac.uk/spm">https://www.fil.ion.ucl.ac.uk/spm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>12.5_r7771</code></td>
<td><code>-MATLAB-2021a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>12.5_r7771</code></td>
<td><code>-MATLAB-2021b</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="spoa">spoa<a class="headerlink" href="#spoa" title="Permanent link">&para;</a></h3>
<p>Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm  which is used to generate consensus sequences</p>
<p><em>homepage</em>: <a href="https://github.com/rvaser/spoa">https://github.com/rvaser/spoa</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>4.0.7</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="spooles">SPOOLES<a class="headerlink" href="#spooles" title="Permanent link">&para;</a></h3>
<p>SPOOLES is a library for solving sparse real and complex linear  systems of equations, written in the C language using object oriented design. </p>
<p><em>homepage</em>: <a href="https://netlib.org/linalg/spooles/spooles.2.2.html">https://netlib.org/linalg/spooles/spooles.2.2.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>gompi/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="spotpy">SPOTPY<a class="headerlink" href="#spotpy" title="Permanent link">&para;</a></h3>
<p>SPOTPY is a Python framework that enables the use of Computational optimization techniques for calibration, uncertainty and sensitivity analysis techniques of almost every (environmental-) model.</p>
<p><em>homepage</em>: <a href="https://spotpy.readthedocs.io">https://spotpy.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.14</code></td>
<td><code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sprng">SPRNG<a class="headerlink" href="#sprng" title="Permanent link">&para;</a></h3>
<p>Scalable Parallel Pseudo Random Number Generators Library</p>
<p><em>homepage</em>: <a href="http://www.sprng.org/">http://www.sprng.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0b</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="spyder">Spyder<a class="headerlink" href="#spyder" title="Permanent link">&para;</a></h3>
<p>Spyder is an interactive Python development environment providing MATLAB-like features in a simple  and light-weighted software.</p>
<p><em>homepage</em>: <a href="https://github.com/spyder-ide/spyder">https://github.com/spyder-ide/spyder</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.3.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.3.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="sqlite">SQLite<a class="headerlink" href="#sqlite" title="Permanent link">&para;</a></h3>
<p>SQLite: SQL Database Engine in a C Library</p>
<p><em>homepage</em>: <a href="http://www.sqlite.org/">http://www.sqlite.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.8.1</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>3.8.10.2</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>gimkl/2.11.5</code></td>
</tr>
<tr>
<td><code>3.9.2</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.14.1</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
<tr>
<td><code>3.17.0</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>3.20.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.21.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.23.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.24.0</code></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.26.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.27.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.29.0</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.31.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.33.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.35.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.36</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.38.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.39.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="squeezemeta">SqueezeMeta<a class="headerlink" href="#squeezemeta" title="Permanent link">&para;</a></h3>
<p>SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,  covering all steps of the analysis.</p>
<p><em>homepage</em>: <a href="https://github.com/jtamames/SqueezeMeta">https://github.com/jtamames/SqueezeMeta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="squidpy">Squidpy<a class="headerlink" href="#squidpy" title="Permanent link">&para;</a></h3>
<p>Squidpy is a tool for the analysis and visualization of spatial molecular data.</p>
<p><em>homepage</em>: <a href="https://squidpy.readthedocs.io">https://squidpy.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sra-toolkit">SRA-Toolkit<a class="headerlink" href="#sra-toolkit" title="Permanent link">&para;</a></h3>
<p>The SRA Toolkit, and the source-code SRA System Development  Kit (SDK), will allow you to programmatically access data housed within SRA  and convert it from the SRA format</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/sra-tools">https://github.com/ncbi/sra-tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.5</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.4-1</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.7</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.2-1</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.0</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.2</code></td>
<td><code>-ubuntu64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.4</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.9.6-1</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.4</code></td>
<td></td>
<td><code>gompi/2019b</code></td>
</tr>
<tr>
<td><code>2.10.5</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.10.8</code></td>
<td></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>2.10.9</code></td>
<td></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-centos_linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.0.3</code></td>
<td></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sradownloader">sradownloader<a class="headerlink" href="#sradownloader" title="Permanent link">&para;</a></h3>
<p>SRAdownloader takes the annotation table from the SRA run selector tool  and retrieves the raw fastq files for the selected samples</p>
<p><em>homepage</em>: <a href="https://github.com/s-andrews/sradownloader">https://github.com/s-andrews/sradownloader</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.9</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="srprism">SRPRISM<a class="headerlink" href="#srprism" title="Permanent link">&para;</a></h3>
<p>Single Read Paired Read Indel Substitution Minimizer</p>
<p><em>homepage</em>: <a href="https://github.com/ncbi/SRPRISM">https://github.com/ncbi/SRPRISM</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.1.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="srst2">SRST2<a class="headerlink" href="#srst2" title="Permanent link">&para;</a></h3>
<p>Short Read Sequence Typing for Bacterial Pathogens</p>
<p><em>homepage</em>: <a href="https://katholt.github.io/srst2">https://katholt.github.io/srst2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0-20210620</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ssaha2">SSAHA2<a class="headerlink" href="#ssaha2" title="Permanent link">&para;</a></h3>
<p>SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a   pairwise sequence alignment program designed for the efficient mapping of sequencing   reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms   (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.)   are supported. A pile-up pipeline for analysis and genotype calling is available as   a separate package. </p>
<p><em>homepage</em>: <a href="http://www.sanger.ac.uk/resources/software/ssaha2/">http://www.sanger.ac.uk/resources/software/ssaha2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.5.5</code></td>
<td><code>-i686</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.5</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ssn">SSN<a class="headerlink" href="#ssn" title="Permanent link">&para;</a></h3>
<p>Spatial statistical modeling and prediction for data on stream networks, including models based on in-stream distance. Models are created using moving average constructions. Spatial linear models, including explanatory variables, can be fit with (restricted) maximum likelihood. Mapping and other graphical functions are included.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/SSN">https://cran.r-project.org/web/packages/SSN</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.14</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="sspace_basic">SSPACE_Basic<a class="headerlink" href="#sspace_basic" title="Permanent link">&para;</a></h3>
<p>SSPACE Basic, SSAKE-based Scaffolding of Pre-Assembled Contigs after Extension</p>
<p><em>homepage</em>: <a href="https://github.com/nsoranzo/sspace_basic">https://github.com/nsoranzo/sspace_basic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ssw">SSW<a class="headerlink" href="#ssw" title="Permanent link">&para;</a></h3>
<p>SSW is a fast implementation of the Smith-Waterman algorithm, which uses the  Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the  instruction level. SSW library provides an API that can be flexibly used by programs written in  C, C++ and other languages. We also provide a software that can do protein and genome alignment  directly. Current version of our implementation is ~50 times faster than an ordinary  Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path  (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and  location heuristically. </p>
<p><em>homepage</em>: <a href="https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library">https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="stacey">STACEY<a class="headerlink" href="#stacey" title="Permanent link">&para;</a></h3>
<p>The BEAST2 package STACEY can be used for species delimitation and species tree estimation, based on the multispecies coalescent model.</p>
<p><em>homepage</em>: <a href="http://www.indriid.com/software.html">http://www.indriid.com/software.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.5</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="stack">Stack<a class="headerlink" href="#stack" title="Permanent link">&para;</a></h3>
<p>Stack is a cross-platform program for developing Haskell projects. It is intended for Haskellers both new and experienced.</p>
<p><em>homepage</em>: <a href="https://docs.haskellstack.org">https://docs.haskellstack.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.3</code></td>
<td><code>-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="stacks">Stacks<a class="headerlink" href="#stacks" title="Permanent link">&para;</a></h3>
<p>Stacks is a software pipeline for building loci from short-read sequences, such as those generated on  the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,  for the purpose of building genetic maps and conducting population genomics and phylogeography. </p>
<p><em>homepage</em>: <a href="https://catchenlab.life.illinois.edu/stacks/">https://catchenlab.life.illinois.edu/stacks/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.40</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.42</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.44</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.45</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.46</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.47</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>1.48</code></td>
<td><code>intel/2017b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.0</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0Beta7c</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0Beta8c</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0Beta9</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.0Beta10a</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.3b</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2.3e</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.41</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code>, <code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.53</code></td>
<td><code>foss/2019b</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>2.54</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.62</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="stamp">STAMP<a class="headerlink" href="#stamp" title="Permanent link">&para;</a></h3>
<p>STAMP is a tool for characterizing similarities between transcription factor binding motifs</p>
<p><em>homepage</em>: <a href="http://www.benoslab.pitt.edu/stamp/">http://www.benoslab.pitt.edu/stamp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="stamps">StaMPS<a class="headerlink" href="#stamps" title="Permanent link">&para;</a></h3>
<p>A software package to extract ground displacements from time series of synthetic aperture radar  (SAR) acquisitions.</p>
<p><em>homepage</em>: <a href="http://homepages.see.leeds.ac.uk/~earahoo/stamps/">http://homepages.see.leeds.ac.uk/~earahoo/stamps/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.3b1</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="stampy">Stampy<a class="headerlink" href="#stampy" title="Permanent link">&para;</a></h3>
<p>Stampy is a package for the mapping of short reads from illumina sequencing machines onto  a reference genome.</p>
<p><em>homepage</em>: <a href="http://www.well.ox.ac.uk/stampy">http://www.well.ox.ac.uk/stampy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.31</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.32</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="star">STAR<a class="headerlink" href="#star" title="Permanent link">&para;</a></h3>
<p>STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.</p>
<p><em>homepage</em>: <a href="https://github.com/alexdobin/STAR">https://github.com/alexdobin/STAR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4.2a</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0a</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.5.2a</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.5.2b</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.5.3a</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.5.4b</code></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.6.0c</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.6.1c</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7.0d</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7.0f</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7.1a</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.7.2b</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.3a</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.4a</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.5b</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.6a</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.7a</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.7.8a</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>2.7.9a</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7.10a_alpha_220601</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.7.10b</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="star-ccm">STAR-CCM+<a class="headerlink" href="#star-ccm" title="Permanent link">&para;</a></h3>
<p>STAR-CCM+ is a multiphysics computational fluid dynamics (CFD) software for the simulation of products operating under real-world conditions.</p>
<p><em>homepage</em>: <a href="https://mdx.plm.automation.siemens.com/star-ccm-plus">https://mdx.plm.automation.siemens.com/star-ccm-plus</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>13.04.011</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.02.008</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.02.008</code></td>
<td><code>-r8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.04.008</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.04.008</code></td>
<td><code>-r8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.06.007</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17.06.007</code></td>
<td><code>-r8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.02.008</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>18.02.008</code></td>
<td><code>-r8</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="star-fusion">STAR-Fusion<a class="headerlink" href="#star-fusion" title="Permanent link">&para;</a></h3>
<p>STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts  supported by Illumina reads. STAR-Fusion further processes the output generated  by the STAR aligner to map junction reads and spanning reads to a reference  annotation set.</p>
<p><em>homepage</em>: <a href="https://github.com/STAR-Fusion/STAR-Fusion">https://github.com/STAR-Fusion/STAR-Fusion</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Perl-5.28.1-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>1.8.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="stardist">stardist<a class="headerlink" href="#stardist" title="Permanent link">&para;</a></h3>
<p>Object Detection with Star-convex Shapes.</p>
<p><em>homepage</em>: <a href="https://github.com/stardist/stardist">https://github.com/stardist/stardist</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.3</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="stars">stars<a class="headerlink" href="#stars" title="Permanent link">&para;</a></h3>
<p>Reading, manipulating, writing and plotting spatiotemporal arrays (raster and vector data cubes)  in R, using GDAL bindings provided by sf, and NetCDF bindings by ncmeta and RNetCDF.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=stars">https://cran.r-project.org/package=stars</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4-3</code></td>
<td><code>-R-4.0.0-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="stata">Stata<a class="headerlink" href="#stata" title="Permanent link">&para;</a></h3>
<p>Stata is a complete, integrated statistical software package that provides everything you need  for data analysis, data management, and graphics.</p>
<p><em>homepage</em>: <a href="https://www.stata.com/">https://www.stata.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>15</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>16</code></td>
<td><code>-legacy</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>17</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="statistics-r">Statistics-R<a class="headerlink" href="#statistics-r" title="Permanent link">&para;</a></h3>
<p>Perl interface with the R statistical program</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Statistics::R">https://metacpan.org/pod/Statistics::R</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.34</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="statsmodels">statsmodels<a class="headerlink" href="#statsmodels" title="Permanent link">&para;</a></h3>
<p>Statsmodels is a Python module that provides classes and functions  for the estimation of many different statistical models, as well as for conducting   statistical tests, and statistical data exploration.</p>
<p><em>homepage</em>: <a href="https://www.statsmodels.org">https://www.statsmodels.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.8.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.10.1</code></td>
<td></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.11.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.12.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="steak">STEAK<a class="headerlink" href="#steak" title="Permanent link">&para;</a></h3>
<p>Detects integrations of any sort in high-throughput  sequencing (HTS) data. STEAK was built for validating and discovering  transposable element (TE) and retroviral integrations in a variety of  HTS data. The software performs on both single-end (SE) and paired-end ( PE) libraries and on a variety of HTS sequencing strategies. It can  be applied to a broad range of research interests and clinical uses  such as population genetic studies and detecting polymorphic integrations.</p>
<p><em>homepage</em>: <a href="https://omictools.com/steak-tool">https://omictools.com/steak-tool</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2019.09.12</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2019.09.12</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="stir">STIR<a class="headerlink" href="#stir" title="Permanent link">&para;</a></h3>
<p>Software for Tomographic Image Reconstruction</p>
<p><em>homepage</em>: <a href="http://stir.sourceforge.net/">http://stir.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="stpipeline">stpipeline<a class="headerlink" href="#stpipeline" title="Permanent link">&para;</a></h3>
<p>The ST Pipeline contains the tools and scripts needed to process  and analyze the raw files generated with the Spatial Transcriptomics method  in FASTQ format to generated datasets for down-stream analysis. The ST  pipeline can also be used to process single cell data as long as a file  with barcodes identifying each cell is provided. The ST Pipeline can also  process RNA-Seq datasets generated with or without UMIs.</p>
<p><em>homepage</em>: <a href="https://github.com/SpatialTranscriptomicsResearch/st_pipeline">https://github.com/SpatialTranscriptomicsResearch/st_pipeline</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.7.6</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>1.7.6</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="strainberry">Strainberry<a class="headerlink" href="#strainberry" title="Permanent link">&para;</a></h3>
<p>Strainberry is a method that performs strain separation in low-complexity metagenomes using error-prone long-read technologies. It exploits state-of-the-art tools for variant calling, haplotype phasing, and genome assembly, in order to achieve single-sample assembly of strains with higher quality than other state-of-the-art long-read assemblers.</p>
<p><em>homepage</em>: <a href="https://github.com/rvicedomini/strainberry">https://github.com/rvicedomini/strainberry</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="stream">STREAM<a class="headerlink" href="#stream" title="Permanent link">&para;</a></h3>
<p>The STREAM benchmark is a simple synthetic benchmark program that measures sustainable  memory bandwidth (in MB/s) and the corresponding computation rate for simple vector kernels.</p>
<p><em>homepage</em>: <a href="https://www.cs.virginia.edu/stream/">https://www.cs.virginia.edu/stream/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.10</code></td>
<td><code>GCC/11.3.0</code>, <code>GCC/7.3.0-2.30</code>, <code>GCC/8.2.0-2.31.1</code>, <code>GCC/9.3.0</code>, <code>iccifort/2020.1.217</code>, <code>intel-compilers/2022.2.1</code>, <code>intel/2016b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="strelka">strelka<a class="headerlink" href="#strelka" title="Permanent link">&para;</a></h3>
<p>Strelka2 is a fast and accurate small variant caller  optimized for analysis of germline variation in small cohorts and  somatic variation in tumor/normal sample pairs.</p>
<p><em>homepage</em>: <a href="https://github.com/Illumina/strelka">https://github.com/Illumina/strelka</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.7</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.9.9</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.9.10</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="stringtie">StringTie<a class="headerlink" href="#stringtie" title="Permanent link">&para;</a></h3>
<p>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts</p>
<p><em>homepage</em>: <a href="https://ccb.jhu.edu/software/stringtie/">https://ccb.jhu.edu/software/stringtie/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.3b</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3.5</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.1.3</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1.4</code></td>
<td><code>GCC/8.3.0</code>, <code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1.7</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="stripy">stripy<a class="headerlink" href="#stripy" title="Permanent link">&para;</a></h3>
<p>A Python interface to TRIPACK and STRIPACK Fortran code for (constrained) triangulation in Cartesian  coordinates and on a sphere. Stripy is an object-oriented package and includes routines from SRFPACK and SSRFPACK for  interpolation (nearest neighbor, linear and hermite cubic) and to evaluate derivatives (Renka 1996a,b and 1997a,b).</p>
<p><em>homepage</em>: <a href="https://github.com/underworldcode/stripy">https://github.com/underworldcode/stripy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="strique">STRique<a class="headerlink" href="#strique" title="Permanent link">&para;</a></h3>
<p>STRique is a python package to analyze repeat expansion and methylation states of short tandem repeats (STR) in Oxford Nanopore Technology (ONT) long read sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/giesselmann/STRique">https://github.com/giesselmann/STRique</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="structure">Structure<a class="headerlink" href="#structure" title="Permanent link">&para;</a></h3>
<p>The program structure is a free software package for using multi-locus genotype data to investigate  population structure.</p>
<p><em>homepage</em>: <a href="https://web.stanford.edu/group/pritchardlab/structure.html">https://web.stanford.edu/group/pritchardlab/structure.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.4</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>iccifort/2019.3.199-GCC-8.3.0-2.32</code>, <code>iccifort/2019.5.281</code></td>
</tr>
</tbody>
</table>
<h3 id="strumpack">STRUMPACK<a class="headerlink" href="#strumpack" title="Permanent link">&para;</a></h3>
<p>STRUMPACK - STRUctured Matrix PACKage - Fast linear solvers and preconditioner  for both dense and sparse systems using low-rank structured factorization with randomized sampling.</p>
<p><em>homepage</em>: <a href="https://fastmath-scidac.llnl.gov/software/strumpack.html">https://fastmath-scidac.llnl.gov/software/strumpack.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.1.0</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="suave">suave<a class="headerlink" href="#suave" title="Permanent link">&para;</a></h3>
<p>suave is an interactive web application to visualize read depth ratios between two samples and the structural variants of one of the samples (typically the "case" sample in a case/control setup such as tumor/normal comparison).</p>
<p><em>homepage</em>: <a href="https://github.com/dellytools/suave">https://github.com/dellytools/suave</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160529</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="subread">Subread<a class="headerlink" href="#subread" title="Permanent link">&para;</a></h3>
<p>High performance read alignment, quantification and mutation discovery</p>
<p><em>homepage</em>: <a href="http://subread.sourceforge.net/">http://subread.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0-p1</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.6.3</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.6.4</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>GCC/7.3.0-2.30</code>, <code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0.2</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.3</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="subversion">Subversion<a class="headerlink" href="#subversion" title="Permanent link">&para;</a></h3>
<p>Subversion is an open source version control system.</p>
<p><em>homepage</em>: <a href="http://subversion.apache.org/">http://subversion.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.7</code></td>
<td><code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>1.9.9</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.14.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.14.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="suds">suds<a class="headerlink" href="#suds" title="Permanent link">&para;</a></h3>
<p>Lightweight SOAP client</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/suds-py3">https://pypi.python.org/pypi/suds-py3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="suitesparse">SuiteSparse<a class="headerlink" href="#suitesparse" title="Permanent link">&para;</a></h3>
<p>SuiteSparse is a collection of libraries manipulate sparse matrices.</p>
<p><em>homepage</em>: <a href="https://faculty.cse.tamu.edu/davis/suitesparse.html">https://faculty.cse.tamu.edu/davis/suitesparse.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.1</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.5.2</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.5.3</code></td>
<td><code>-ParMETIS-4.0.3</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.5.5</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.5.5</code></td>
<td><code>-ParMETIS-4.0.3</code></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>4.5.6</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>5.1.2</code></td>
<td><code>-ParMETIS-4.0.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.4.0</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2019a</code>, <code>intel/2018b</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>5.6.0</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.7.1</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>5.8.1</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>5.10.1</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2021a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>5.10.1</code></td>
<td><code>-METIS-5.1.0-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>5.13.0</code></td>
<td><code>-METIS-5.1.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sumaclust">SUMACLUST<a class="headerlink" href="#sumaclust" title="Permanent link">&para;</a></h3>
<p>SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.</p>
<p><em>homepage</em>: <a href="http://metabarcoding.org/sumatra">http://metabarcoding.org/sumatra</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.20</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="sumatra">SUMATRA<a class="headerlink" href="#sumatra" title="Permanent link">&para;</a></h3>
<p>SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.</p>
<p><em>homepage</em>: <a href="http://metabarcoding.org/sumatra">http://metabarcoding.org/sumatra</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.20</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="sumo">SUMO<a class="headerlink" href="#sumo" title="Permanent link">&para;</a></h3>
<p>Simulation of Urban MObility" (SUMO) is an open source, highly portable, microscopic and continuous traffic simulation package designed to handle large networks. It allows for intermodal simulation including pedestrians and comes with a large set of tools for scenario creation.</p>
<p><em>homepage</em>: <a href="https://www.eclipse.org/sumo/">https://www.eclipse.org/sumo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.14.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="sundials">SUNDIALS<a class="headerlink" href="#sundials" title="Permanent link">&para;</a></h3>
<p>SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers</p>
<p><em>homepage</em>: <a href="https://computation.llnl.gov/projects/sundials">https://computation.llnl.gov/projects/sundials</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>intel/2016b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>foss/2016b</code>, <code>foss/2017b</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>5.7.0</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="sunpy">SunPy<a class="headerlink" href="#sunpy" title="Permanent link">&para;</a></h3>
<p>The community-developed, free and open-source solar data analysis environment for Python.</p>
<p><em>homepage</em>: <a href="https://sunpy.org/">https://sunpy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="superlu">SuperLU<a class="headerlink" href="#superlu" title="Permanent link">&para;</a></h3>
<p>SuperLU is a general purpose library for the  direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.</p>
<p><em>homepage</em>: <a href="https://crd-legacy.lbl.gov/~xiaoye/SuperLU/">https://crd-legacy.lbl.gov/~xiaoye/SuperLU/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.2.1</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2022a</code>, <code>intel/2020b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="superlu_dist">SuperLU_DIST<a class="headerlink" href="#superlu_dist" title="Permanent link">&para;</a></h3>
<p>SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.</p>
<p><em>homepage</em>: <a href="https://crd-legacy.lbl.gov/~xiaoye/SuperLU/">https://crd-legacy.lbl.gov/~xiaoye/SuperLU/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.0</code></td>
<td><code>-trisolve-merge</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>6.4.0</code></td>
<td></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="supermagic">supermagic<a class="headerlink" href="#supermagic" title="Permanent link">&para;</a></h3>
<p>Very simple MPI sanity code. Nothing more, nothing less.</p>
<p><em>homepage</em>: <a href="https://github.com/hpc/supermagic">https://github.com/hpc/supermagic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170824</code></td>
<td><code>foss/2017a</code>, <code>gompi/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="supernova">supernova<a class="headerlink" href="#supernova" title="Permanent link">&para;</a></h3>
<p>Supernova is a software package for de novo assembly from Chromium Linked-Reads  that are made from a single whole-genome library from an individual DNA source</p>
<p><em>homepage</em>: <a href="https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome">https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="survivor">SURVIVOR<a class="headerlink" href="#survivor" title="Permanent link">&para;</a></h3>
<p>Toolset for SV simulation, comparison and filtering</p>
<p><em>homepage</em>: <a href="https://github.com/fritzsedlazeck/SURVIVOR">https://github.com/fritzsedlazeck/SURVIVOR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.7-19-ged1ca51</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="svdetect">SVDetect<a class="headerlink" href="#svdetect" title="Permanent link">&para;</a></h3>
<p>SVDetect is a application for the isolation and the type prediction of intra- and inter-chromosomal  rearrangements from paired-end/mate-pair sequencing data provided by the high-throughput sequencing technologies.  This tool aims to identifying structural variations with both clustering and sliding-window strategies, and  helping in their visualization at the genome scale.</p>
<p><em>homepage</em>: <a href="http://svdetect.sourceforge.net/Site/Home.html">http://svdetect.sourceforge.net/Site/Home.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8b</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="svdquest">SVDquest<a class="headerlink" href="#svdquest" title="Permanent link">&para;</a></h3>
<p>SVDquartets-based species trees</p>
<p><em>homepage</em>: <a href="https://github.com/pranjalv123/SVDquest">https://github.com/pranjalv123/SVDquest</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20190627</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="svg">SVG<a class="headerlink" href="#svg" title="Permanent link">&para;</a></h3>
<p>Perl binding for SVG</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/SVG">https://metacpan.org/pod/SVG</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.84</code></td>
<td><code>-Perl-5.30.0</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.87</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="svist4get">svist4get<a class="headerlink" href="#svist4get" title="Permanent link">&para;</a></h3>
<p>Svist4get is a simple bioinformatics tool for visualization of  genomic signal tracks in user-defined genomic windows, either arbitrary selected  by genomic coordinates or anchored to particular transcripts or genes.</p>
<p><em>homepage</em>: <a href="https://github.com/art-egorov/svist4get">https://github.com/art-egorov/svist4get</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="swarm">swarm<a class="headerlink" href="#swarm" title="Permanent link">&para;</a></h3>
<p>A robust and fast clustering method for amplicon-based studies </p>
<p><em>homepage</em>: <a href="https://github.com/torognes/swarm">https://github.com/torognes/swarm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.2</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="swash">SWASH<a class="headerlink" href="#swash" title="Permanent link">&para;</a></h3>
<p>SWASH is a general-purpose numerical tool for simulating unsteady, non-hydrostatic, free-surface,  rotational flow and transport phenomena in coastal waters as driven by waves, tides, buoyancy and wind forces.</p>
<p><em>homepage</em>: <a href="http://swash.sourceforge.net/">http://swash.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.14</code></td>
<td><code>-mpi</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>4.01</code></td>
<td><code>-mpi</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="swat">SWAT+<a class="headerlink" href="#swat" title="Permanent link">&para;</a></h3>
<p>The Soil &amp; Water Assessment Tool (SWAT) is a small watershed to river basin-scale model used to simulate the quality and quantity of surface and ground water and predict the environmental impact of land use, land management practices, and climate change. In order to face present and future challenges in water resources modeling SWAT code has undergone major modifications over the past few years, resulting in SWAT+, a completely revised version of the model. SWAT+ provides a more flexible spatial representation of interactions and processes within a watershed.</p>
<p><em>homepage</em>: <a href="https://swatplus.gitbook.io">https://swatplus.gitbook.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>60.4.1</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>60.5.1</code></td>
<td><code>iccifort/2020.4.304</code></td>
</tr>
</tbody>
</table>
<h3 id="swifter">swifter<a class="headerlink" href="#swifter" title="Permanent link">&para;</a></h3>
<p>A package which efficiently applies any function to a pandas dataframe or series in the fastest available manner.</p>
<p><em>homepage</em>: <a href="https://github.com/jmcarpenter2/swifter">https://github.com/jmcarpenter2/swifter</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="swig">SWIG<a class="headerlink" href="#swig" title="Permanent link">&para;</a></h3>
<p>SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.</p>
<p><em>homepage</em>: <a href="http://www.swig.org/">http://www.swig.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.8</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.0.8</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>3.0.10</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0.10</code></td>
<td><code>-Python-2.7.12-PCRE-8.39</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0.11</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-2.7.14-bare</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCCcore/8.2.0</code>, <code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code>, <code>intel/2018.00</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code>, <code>intel/2018.01</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.0.12</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="swipe">SWIPE<a class="headerlink" href="#swipe" title="Permanent link">&para;</a></h3>
<p>Smith-Waterman database searches with inter-sequence SIMD parallelisation</p>
<p><em>homepage</em>: <a href="https://github.com/torognes/swipe">https://github.com/torognes/swipe</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="swissknife">swissknife<a class="headerlink" href="#swissknife" title="Permanent link">&para;</a></h3>
<p>Perl module for reading and writing UniProtKB data in plain text format.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/swissknife/">https://sourceforge.net/projects/swissknife/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.80</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="symengine">SymEngine<a class="headerlink" href="#symengine" title="Permanent link">&para;</a></h3>
<p>SymEngine is a standalone fast C++ symbolic manipulation library</p>
<p><em>homepage</em>: <a href="https://github.com/symengine/symengine">https://github.com/symengine/symengine</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td><code>-20181006</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.4.0</code></td>
<td></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="symengine-python">SymEngine-python<a class="headerlink" href="#symengine-python" title="Permanent link">&para;</a></h3>
<p>Python wrappers to the C++ library SymEngine, a fast C++ symbolic manipulation library.</p>
<p><em>homepage</em>: <a href="https://github.com/symengine/symengine.py">https://github.com/symengine/symengine.py</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7.2</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="symphony">SYMPHONY<a class="headerlink" href="#symphony" title="Permanent link">&para;</a></h3>
<p>SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.</p>
<p><em>homepage</em>: <a href="https://projects.coin-or.org/SYMPHONY">https://projects.coin-or.org/SYMPHONY</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6.16</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="sympy">sympy<a class="headerlink" href="#sympy" title="Permanent link">&para;</a></h3>
<p>SymPy is a Python library for symbolic mathematics. It aims to  become a full-featured computer algebra system (CAS) while keeping the code as  simple as possible in order to be comprehensible and easily extensible. SymPy  is written entirely in Python and does not require any external libraries.</p>
<p><em>homepage</em>: <a href="https://sympy.org/">https://sympy.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.6.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.7.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>1.8</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>1.11.1</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="syri">SyRI<a class="headerlink" href="#syri" title="Permanent link">&para;</a></h3>
<p>Synteny and Rearrangement Identifier (SyRI).</p>
<p><em>homepage</em>: <a href="https://schneebergerlab.github.io/syri/">https://schneebergerlab.github.io/syri/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="szip">Szip<a class="headerlink" href="#szip" title="Permanent link">&para;</a></h3>
<p>Szip compression software, providing lossless compression of scientific data </p>
<p><em>homepage</em>: <a href="https://www.hdfgroup.org/doc_resource/SZIP/">https://www.hdfgroup.org/doc_resource/SZIP/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1</code></td>
<td><code>GCC/4.8.1</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>gmpolf/2017.10</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017.01</code>, <code>intel/2017a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="t">T<a class="headerlink" href="#t" title="Permanent link">&para;</a></h2>
<p><a href="#tabix">tabix</a> - <a href="#tabixpp">tabixpp</a> - <a href="#taco">taco</a> - <a href="#tagdust">TagDust</a> - <a href="#taglib">TagLib</a> - <a href="#taiyaki">Taiyaki</a> - <a href="#talon">TALON</a> - <a href="#talys">TALYS</a> - <a href="#tamkin">TAMkin</a> - <a href="#task-spooler">task-spooler</a> - <a href="#taxator-tk">taxator-tk</a> - <a href="#tba">TBA</a> - <a href="#tbb">tbb</a> - <a href="#tbl2asn">tbl2asn</a> - <a href="#tcc">TCC</a> - <a href="#tcl">Tcl</a> - <a href="#tclap">TCLAP</a> - <a href="#tcsh">tcsh</a> - <a href="#tecplot360ex">tecplot360ex</a> - <a href="#telemac-mascaret">TELEMAC-MASCARET</a> - <a href="#telescope">Telescope</a> - <a href="#teneto">Teneto</a> - <a href="#tensorboard">tensorboard</a> - <a href="#tensorboardx">tensorboardX</a> - <a href="#tensorflow">TensorFlow</a> - <a href="#tensorflow-datasets">TensorFlow-Datasets</a> - <a href="#tensorflow-probability">tensorflow-probability</a> - <a href="#tensorrt">TensorRT</a> - <a href="#terastructure">terastructure</a> - <a href="#termcolor">termcolor</a> - <a href="#tesla-deployment-kit">Tesla-Deployment-Kit</a> - <a href="#tesseract">tesseract</a> - <a href="#testpath">testpath</a> - <a href="#tetgen">TetGen</a> - <a href="#tetoolkit">TEToolkit</a> - <a href="#tetranscripts">TEtranscripts</a> - <a href="#texinfo">texinfo</a> - <a href="#texlive">texlive</a> - <a href="#text-csv">Text-CSV</a> - <a href="#tfea">TFEA</a> - <a href="#theano">Theano</a> - <a href="#themispy">ThemisPy</a> - <a href="#theta">THetA</a> - <a href="#thurstonianirt">thurstonianIRT</a> - <a href="#ticcutils">TiCCutils</a> - <a href="#tidybayes">tidybayes</a> - <a href="#tidymodels">tidymodels</a> - <a href="#tika">Tika</a> - <a href="#timbl">TiMBL</a> - <a href="#time">time</a> - <a href="#tinker">TINKER</a> - <a href="#tinydb">TinyDB</a> - <a href="#tinyxml">TinyXML</a> - <a href="#tk">Tk</a> - <a href="#tkinter">Tkinter</a> - <a href="#tm-align">TM-align</a> - <a href="#tmae">tMAE</a> - <a href="#tmap">tmap</a> - <a href="#tmux">tmux</a> - <a href="#tn93">TN93</a> - <a href="#tobias">TOBIAS</a> - <a href="#tofu">ToFu</a> - <a href="#togl">Togl</a> - <a href="#toil">toil</a> - <a href="#tokenizers">tokenizers</a> - <a href="#tombo">Tombo</a> - <a href="#toml-fortran">TOML-Fortran</a> - <a href="#topaz">topaz</a> - <a href="#tophat">TopHat</a> - <a href="#torchaudio">torchaudio</a> - <a href="#torchdata">torchdata</a> - <a href="#torchinfo">torchinfo</a> - <a href="#torchsampler">torchsampler</a> - <a href="#torchtext">torchtext</a> - <a href="#torchvision">torchvision</a> - <a href="#totalview">TotalView</a> - <a href="#tqdm">tqdm</a> - <a href="#tracer">Tracer</a> - <a href="#transcriptclean">TranscriptClean</a> - <a href="#transdecoder">TransDecoder</a> - <a href="#transformers">Transformers</a> - <a href="#transrate">Transrate</a> - <a href="#travis">travis</a> - <a href="#travis-analyzer">TRAVIS-Analyzer</a> - <a href="#treatsens">treatSens</a> - <a href="#treemix">TreeMix</a> - <a href="#treeshrink">TreeShrink</a> - <a href="#trf">TRF</a> - <a href="#triangle">Triangle</a> - <a href="#trilinos">Trilinos</a> - <a href="#trim_galore">Trim_Galore</a> - <a href="#trimal">trimAl</a> - <a href="#trimesh">trimesh</a> - <a href="#trimmomatic">Trimmomatic</a> - <a href="#trinity">Trinity</a> - <a href="#triqs">TRIQS</a> - <a href="#triqs-cthyb">TRIQS-cthyb</a> - <a href="#triqs-dft_tools">TRIQS-dft_tools</a> - <a href="#triqs-tprf">TRIQS-tprf</a> - <a href="#trust">TRUST</a> - <a href="#trust4">TRUST4</a> - <a href="#trycycler">Trycycler</a> - <a href="#tsne">tsne</a> - <a href="#turbovnc">TurboVNC</a> - <a href="#tvb">TVB</a> - <a href="#tvb-data">tvb-data</a> - <a href="#tvb-deps">TVB-deps</a> - <a href="#tvb-framework">tvb-framework</a> - <a href="#tvb-library">tvb-library</a> - <a href="#twl-ninja">TWL-NINJA</a> - <a href="#typing-extensions">typing-extensions</a></p>
<h3 id="tabix">tabix<a class="headerlink" href="#tabix" title="Permanent link">&para;</a></h3>
<p>Generic indexer for TAB-delimited genome position files </p>
<p><em>homepage</em>: <a href="http://samtools.sourceforge.net">http://samtools.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>foss/2016a</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="tabixpp">tabixpp<a class="headerlink" href="#tabixpp" title="Permanent link">&para;</a></h3>
<p>C++ wrapper to tabix indexer</p>
<p><em>homepage</em>: <a href="https://github.com/ekg/tabixpp">https://github.com/ekg/tabixpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="taco">taco<a class="headerlink" href="#taco" title="Permanent link">&para;</a></h3>
<p>Multi-sample transcriptome assembly from RNA-Seq</p>
<p><em>homepage</em>: <a href="http://tacorna.github.io">http://tacorna.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="tagdust">TagDust<a class="headerlink" href="#tagdust" title="Permanent link">&para;</a></h3>
<p>Raw sequences produced by next generation sequencing (NGS) machines may contain adapter,  linker, barcode and fingerprint sequences. TagDust2 is a program to extract and correctly label the  sequences to be mapped in downstream pipelines.</p>
<p><em>homepage</em>: <a href="http://tagdust.sourceforge.net/">http://tagdust.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.33</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="taglib">TagLib<a class="headerlink" href="#taglib" title="Permanent link">&para;</a></h3>
<p>TagLib is a library for reading and editing the meta-data of several popular audio formats.</p>
<p><em>homepage</em>: <a href="https://taglib.org/">https://taglib.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="taiyaki">Taiyaki<a class="headerlink" href="#taiyaki" title="Permanent link">&para;</a></h3>
<p>Taiyaki is research software for training models for basecalling Oxford Nanopore reads.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/taiyaki">https://github.com/nanoporetech/taiyaki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.0-20200617</code></td>
<td><code>-Python-3.7.2-PyTorch-1.2.0</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="talon">TALON<a class="headerlink" href="#talon" title="Permanent link">&para;</a></h3>
<p>TALON is a Python package for identifying and quantifying known and novel genes/isoforms in long-read transcriptome data sets. TALON is technology-agnostic in that it works from mapped SAM files, allowing data from different sequencing platforms (i.e. PacBio and Oxford Nanopore) to be analyzed side by side.</p>
<p><em>homepage</em>: <a href="https://github.com/mortazavilab/TALON">https://github.com/mortazavilab/TALON</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="talys">TALYS<a class="headerlink" href="#talys" title="Permanent link">&para;</a></h3>
<p>TALYS is a nuclear reaction program.</p>
<p><em>homepage</em>: <a href="https://tendl.web.psi.ch/tendl_2019/talys.html">https://tendl.web.psi.ch/tendl_2019/talys.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.95</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tamkin">TAMkin<a class="headerlink" href="#tamkin" title="Permanent link">&para;</a></h3>
<p>TAMkin is a post-processing toolkit for normal mode analysis,  thermochemistry and reaction kinetics. It uses a Hessian computation from a  standard computational chemistry program as its input.</p>
<p><em>homepage</em>: <a href="https://molmod.github.io/tamkin/">https://molmod.github.io/tamkin/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.9</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.2.6</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.2.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="task-spooler">task-spooler<a class="headerlink" href="#task-spooler" title="Permanent link">&para;</a></h3>
<p>task spooler is a Unix batch system where the tasks spooled run one after the other.</p>
<p><em>homepage</em>: <a href="http://viric.name/soft/ts/">http://viric.name/soft/ts/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="taxator-tk">taxator-tk<a class="headerlink" href="#taxator-tk" title="Permanent link">&para;</a></h3>
<p>A set of programs for the taxonomic analysis of nucleotide sequence data</p>
<p><em>homepage</em>: <a href="https://github.com/fungs/taxator-tk">https://github.com/fungs/taxator-tk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.3</code></td>
<td><code>GCC/10.2.0</code>, <code>foss/2018b</code>, <code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="tba">TBA<a class="headerlink" href="#tba" title="Permanent link">&para;</a></h3>
<p>TBA (a Transcription factor Binding Analysis): TBA is a multi-functional machine learning tool for identifying  transcription factors associated with genomic features</p>
<p><em>homepage</em>: <a href="https://github.com/jenhantao/tba">https://github.com/jenhantao/tba</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="tbb">tbb<a class="headerlink" href="#tbb" title="Permanent link">&para;</a></h3>
<p>Intel Threading Building Blocks (Intel TBB)  is a widely used, award-winning C++ template library for creating reliable,  portable, and scalable parallel applications.  Use Intel TBB for a simple and rapid way of developing robust task-based  parallel applications that scale to available processor cores, are compatible  with multiple environments, and are easier to maintain.  Intel TBB is the most proficient way to implement future-proof parallel applications  that tap into the power and performance of multicore and manycore hardware platforms.</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/articles/intel-tbb/">https://software.intel.com/en-us/articles/intel-tbb/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0.233</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.5.339</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.3.6.211</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.4.2.152</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.2.132</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.4.174</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017.6.196</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_U5</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2017_U6</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2018_U1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2018_U2</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2018_U3</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2018_U5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2019_U4</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2019_U9</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2020.1</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2020.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2020.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2021.4.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2021.5.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tbl2asn">tbl2asn<a class="headerlink" href="#tbl2asn" title="Permanent link">&para;</a></h3>
<p>Tbl2asn is a command-line program that automates the creation of  sequence records for submission to GenBank</p>
<p><em>homepage</em>: <a href="https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/">https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>25.8</code></td>
<td><code>-linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20180227</code></td>
<td><code>-linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20200302</code></td>
<td><code>-linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>20220427</code></td>
<td><code>-linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="tcc">TCC<a class="headerlink" href="#tcc" title="Permanent link">&para;</a></h3>
<p>The Tiny C Compiler (aka TCC, tCc, or TinyCC) is an x86 and x86-64 C compiler  created by Fabrice Bellard. It is designed to work for slow computers with little disk space  and can run shebang style !/usr/bin/tcc . TCC is distributed under the LGPL. TCC claims to  implement all of ANSI C (C89/C90),[1] much of the new ISO C99 standard,  and many GNU C extensions including inline assembly.</p>
<p><em>homepage</em>: <a href="http://bellard.org/tcc/">http://bellard.org/tcc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.26</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="tcl">Tcl<a class="headerlink" href="#tcl" title="Permanent link">&para;</a></h3>
<p>Tcl (Tool Command Language) is a very powerful but easy to learn dynamic  programming language, suitable for a very wide range of uses, including web  and desktop applications, networking, administration, testing and many more. </p>
<p><em>homepage</em>: <a href="https://www.tcl.tk/">https://www.tcl.tk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.5.19</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>8.6.3</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>8.6.4</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>8.6.5</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.6.6</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>8.6.7</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>8.6.8</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>8.6.9</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>8.6.10</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.6.11</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.6.12</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tclap">TCLAP<a class="headerlink" href="#tclap" title="Permanent link">&para;</a></h3>
<p>TCLAP is a small, flexible library that provides a simple interface for defining and accessing command line arguments. It was intially inspired by the user friendly CLAP libary.</p>
<p><em>homepage</em>: <a href="http://tclap.sourceforge.net/">http://tclap.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.5</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tcsh">tcsh<a class="headerlink" href="#tcsh" title="Permanent link">&para;</a></h3>
<p>Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh).  It is a command language interpreter usable both as an interactive login shell and a shell script command  processor. It includes a command-line editor, programmable word completion, spelling correction, a history  mechanism, job control and a C-like syntax.</p>
<p><em>homepage</em>: <a href="https://www.tcsh.org">https://www.tcsh.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.19.00</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>6.20.00</code></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>6.22.02</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.22.03</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>6.22.04</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>6.24.01</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tecplot360ex">tecplot360ex<a class="headerlink" href="#tecplot360ex" title="Permanent link">&para;</a></h3>
<p>Quickly plot and animate your CFD results exactly the way you want. Analyze complex solutions, arrange multiple layouts, and communicate your results with professional images and animations.</p>
<p><em>homepage</em>: <a href="http://www.tecplot.com/products/tecplot-360/">http://www.tecplot.com/products/tecplot-360/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="telemac-mascaret">TELEMAC-MASCARET<a class="headerlink" href="#telemac-mascaret" title="Permanent link">&para;</a></h3>
<p>TELEMAC-MASCARET is an integrated suite of solvers for use in the field of free-surface flow. Having been used in the context of many studies throughout the world, it has become one of the major standards in its field.</p>
<p><em>homepage</em>: <a href="http://www.opentelemac.org">http://www.opentelemac.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8p3r1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="telescope">Telescope<a class="headerlink" href="#telescope" title="Permanent link">&para;</a></h3>
<p>Single locus resolution of Transposable ELEment expression using next-generation sequencing.</p>
<p><em>homepage</em>: <a href="https://github.com/mlbendall/telescope">https://github.com/mlbendall/telescope</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="teneto">Teneto<a class="headerlink" href="#teneto" title="Permanent link">&para;</a></h3>
<p>Teneto is package for deriving, analysing and plotting temporal network representations. Additional tools for temporal network analysis with neuroimaging contexts.</p>
<p><em>homepage</em>: <a href="https://teneto.readthedocs.io">https://teneto.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorboard">tensorboard<a class="headerlink" href="#tensorboard" title="Permanent link">&para;</a></h3>
<p>TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs.</p>
<p><em>homepage</em>: <a href="https://github.com/tensorflow/tensorboard">https://github.com/tensorflow/tensorboard</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorboardx">tensorboardX<a class="headerlink" href="#tensorboardx" title="Permanent link">&para;</a></h3>
<p>Tensorboard for PyTorch.</p>
<p><em>homepage</em>: <a href="https://github.com/lanpa/tensorboardX">https://github.com/lanpa/tensorboardX</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.1</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorflow">TensorFlow<a class="headerlink" href="#tensorflow" title="Permanent link">&para;</a></h3>
<p>An open-source software library for Machine Intelligence</p>
<p><em>homepage</em>: <a href="https://www.tensorflow.org/">https://www.tensorflow.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-GPU-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.2.1</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.3.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.7.0</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>fosscuda/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.10.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.12.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>1.14.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>1.15.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.15.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.15.5</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.15.5</code></td>
<td><code>-Python-3.7.4-nompi</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.0.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.2.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.2.3</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.3.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.4.4</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>2.5.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.5.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.6.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.7.1</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.8.4</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>2.11.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorflow-datasets">TensorFlow-Datasets<a class="headerlink" href="#tensorflow-datasets" title="Permanent link">&para;</a></h3>
<p>TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as Jax. All datasets are exposed as tf.data.Datasets , enabling easy-to-use and high-performance input pipelines. </p>
<p><em>homepage</em>: <a href="https://www.tensorflow.org/datasets">https://www.tensorflow.org/datasets</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.7.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.8.3</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorflow-probability">tensorflow-probability<a class="headerlink" href="#tensorflow-probability" title="Permanent link">&para;</a></h3>
<p>TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis.</p>
<p><em>homepage</em>: <a href="https://www.tensorflow.org/probability">https://www.tensorflow.org/probability</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.16.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="tensorrt">TensorRT<a class="headerlink" href="#tensorrt" title="Permanent link">&para;</a></h3>
<p>NVIDIA TensorRT is a platform for high-performance deep learning inference</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/tensorrt">https://developer.nvidia.com/tensorrt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.1.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="terastructure">terastructure<a class="headerlink" href="#terastructure" title="Permanent link">&para;</a></h3>
<p>TeraStructure is a new algorithm to fit Bayesian models of genetic variation in human  populations on tera-sample-sized data sets (10^12 observed genotypes, i.e., 1M individuals at 1M SNPs).  This package provides a scalable, multi-threaded C++ implementation that can be run on a single computer. </p>
<p><em>homepage</em>: <a href="https://github.com/StoreyLab/terastructure">https://github.com/StoreyLab/terastructure</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="termcolor">termcolor<a class="headerlink" href="#termcolor" title="Permanent link">&para;</a></h3>
<p>Termcolor is a header-only C++ library for printing colored  messages to the terminal.</p>
<p><em>homepage</em>: <a href="https://termcolor.readthedocs.io/">https://termcolor.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="tesla-deployment-kit">Tesla-Deployment-Kit<a class="headerlink" href="#tesla-deployment-kit" title="Permanent link">&para;</a></h3>
<p>The Tesla Deployment Kit is a set of tools provided primarily for the NVIDIA Tesla  range of GPUs. They aim to empower users to better manage their NVIDIA GPUs by providing a broad  range of functionalities. The kit contains:  * NVIDIA Management Library (NVML),  * Tesla Deployment Kit - Linux Edition (Aug 1st, 2013)</p>
<p><em>homepage</em>: <a href="https://developer.nvidia.com/tesla-deployment-kit">https://developer.nvidia.com/tesla-deployment-kit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.319.43</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="tesseract">tesseract<a class="headerlink" href="#tesseract" title="Permanent link">&para;</a></h3>
<p>Tesseract is an optical character recognition engine</p>
<p><em>homepage</em>: <a href="https://github.com/tesseract-ocr/tesseract">https://github.com/tesseract-ocr/tesseract</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>4.1.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="testpath">testpath<a class="headerlink" href="#testpath" title="Permanent link">&para;</a></h3>
<p>Test utilities for code working with files and commands</p>
<p><em>homepage</em>: <a href="https://github.com/jupyter/testpath">https://github.com/jupyter/testpath</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.3</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.3</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="tetgen">TetGen<a class="headerlink" href="#tetgen" title="Permanent link">&para;</a></h3>
<p>A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator </p>
<p><em>homepage</em>: <a href="http://wias-berlin.de/software/index.jsp?id=TetGen">http://wias-berlin.de/software/index.jsp?id=TetGen</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tetoolkit">TEToolkit<a class="headerlink" href="#tetoolkit" title="Permanent link">&para;</a></h3>
<p>Tools for estimating differential enrichment of Transposable  Elements and other highly repetitive regions</p>
<p><em>homepage</em>: <a href="http://hammelllab.labsites.cshl.edu/software/#TEToolkit">http://hammelllab.labsites.cshl.edu/software/#TEToolkit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tetranscripts">TEtranscripts<a class="headerlink" href="#tetranscripts" title="Permanent link">&para;</a></h3>
<p>TEtranscripts and TEcount takes RNA-seq (and similar data) and annotates reads to both genes &amp; transposable elements. TEtranscripts then performs differential analysis using DESeq2.</p>
<p><em>homepage</em>: <a href="https://github.com/mhammell-laboratory/TEtranscripts">https://github.com/mhammell-laboratory/TEtranscripts</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="texinfo">texinfo<a class="headerlink" href="#texinfo" title="Permanent link">&para;</a></h3>
<p>Texinfo is the official documentation format of the GNU project.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/texinfo/">https://www.gnu.org/software/texinfo/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.13a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>6.4</code></td>
<td><code>GCCcore/5.4.0</code></td>
</tr>
<tr>
<td><code>6.5</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>6.6</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.7</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>6.8</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>7.0.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="texlive">texlive<a class="headerlink" href="#texlive" title="Permanent link">&para;</a></h3>
<p>TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript  text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a  PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to  be a representation of the final typeset output, and so can be easily edited and manipulated.</p>
<p><em>homepage</em>: <a href="https://tug.org">https://tug.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200406</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20210324</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>20220321</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="text-csv">Text-CSV<a class="headerlink" href="#text-csv" title="Permanent link">&para;</a></h3>
<p>Text-CSV parser</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/Text::CSV">https://metacpan.org/pod/Text::CSV</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.33</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tfea">TFEA<a class="headerlink" href="#tfea" title="Permanent link">&para;</a></h3>
<p>Transcription Factor Enrichment Analysis </p>
<p><em>homepage</em>: <a href="https://github.com/Dowell-Lab/TFEA">https://github.com/Dowell-Lab/TFEA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.4</code></td>
<td><code>-muMerge-1.1.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="theano">Theano<a class="headerlink" href="#theano" title="Permanent link">&para;</a></h3>
<p>Theano is a Python library that allows you to define, optimize, and evaluate mathematical expressions involving multi-dimensional arrays efficiently.</p>
<p><em>homepage</em>: <a href="https://deeplearning.net/software/theano">https://deeplearning.net/software/theano</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>fosscuda/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>fosscuda/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.0.4</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-PyMC</code></td>
<td><code>foss/2020b</code>, <code>foss/2021b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code>, <code>intel/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="themispy">ThemisPy<a class="headerlink" href="#themispy" title="Permanent link">&para;</a></h3>
<p>A header-only C++ library for L-BFGS and L-BFGS-B algorithms</p>
<p><em>homepage</em>: <a href="https://github.com/aeb/ThemisPy">https://github.com/aeb/ThemisPy</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="theta">THetA<a class="headerlink" href="#theta" title="Permanent link">&para;</a></h3>
<p>Tumor Heterogeneity Analysis (THetA) and THetA2 are algorithms  that estimate the tumor purity and clonal/subclonal copy number aberrations  directly from high-throughput DNA sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/raphael-group/THetA">https://github.com/raphael-group/THetA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.7</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="thurstonianirt">thurstonianIRT<a class="headerlink" href="#thurstonianirt" title="Permanent link">&para;</a></h3>
<p>Fit Thurstonian IRT models in R using Stan, lavaan, or Mplus</p>
<p><em>homepage</em>: <a href="https://github.com/paul-buerkner/thurstonianIRT">https://github.com/paul-buerkner/thurstonianIRT</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.0</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="ticcutils">TiCCutils<a class="headerlink" href="#ticcutils" title="Permanent link">&para;</a></h3>
<p>TiCC utils is a collection of generic C++ software which is used in a lot of programs produced at  Tilburg centre for Cognition and Communication (TiCC) at Tilburg University and  Centre for Dutch Language and Speech at University of Antwerp.</p>
<p><em>homepage</em>: <a href="https://github.com/LanguageMachines/ticcutils">https://github.com/LanguageMachines/ticcutils</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.21</code></td>
<td><code>iimpi/2019a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="tidybayes">tidybayes<a class="headerlink" href="#tidybayes" title="Permanent link">&para;</a></h3>
<p>Compose data for and extract, manipulate, and visualize posterior draws from Bayesian models ('JAGS', 'Stan', 'rstanarm', 'brms', 'MCMCglmm', 'coda', ...) in a tidy data format. </p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=tidybayes">https://cran.r-project.org/package=tidybayes</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.1</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="tidymodels">tidymodels<a class="headerlink" href="#tidymodels" title="Permanent link">&para;</a></h3>
<p>The tidy modeling "verse" is a collection of packages for modeling and statistical analysis that share the underlying design philosophy, grammar, and data structures of the tidyverse.</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/package=tidymodels">https://cran.r-project.org/package=tidymodels</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.0</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="tika">Tika<a class="headerlink" href="#tika" title="Permanent link">&para;</a></h3>
<p>The Apache Tika toolkit detects and extracts metadata and text from over a thousand different  file types (such as PPT, XLS, and PDF).</p>
<p><em>homepage</em>: <a href="https://tika.apache.org/">https://tika.apache.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.16</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="timbl">TiMBL<a class="headerlink" href="#timbl" title="Permanent link">&para;</a></h3>
<p>TiMBL (Tilburg Memory Based Learner)  is an open source software package implementing several memory-based learning algorithms,  among which IB1-IG, an implementation of k-nearest neighbor classification with feature weighting suitable for  symbolic feature spaces, and IGTree, a decision-tree approximation of IB1-IG. All implemented algorithms have in  common that they store some representation of the training set explicitly in memory. During testing, new cases are  classified by extrapolation from the most similar stored cases.</p>
<p><em>homepage</em>: <a href="http://ilk.uvt.nl/timbl/">http://ilk.uvt.nl/timbl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.4.7</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>6.4.13</code></td>
<td><code>iimpi/2019a</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="time">time<a class="headerlink" href="#time" title="Permanent link">&para;</a></h3>
<p>The `time' command runs another program, then displays information about the resources used by that  program, collected by the system while the program was running.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/time/">https://www.gnu.org/software/time/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tinker">TINKER<a class="headerlink" href="#tinker" title="Permanent link">&para;</a></h3>
<p>The TINKER molecular modeling software is a complete and general package for molecular mechanics  and dynamics, with some special features for biopolymers.</p>
<p><em>homepage</em>: <a href="https://dasher.wustl.edu/tinker">https://dasher.wustl.edu/tinker</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.6.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>8.7.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>8.8.1</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="tinydb">TinyDB<a class="headerlink" href="#tinydb" title="Permanent link">&para;</a></h3>
<p>TinyDB is a lightweight document oriented database optimized for your happiness :) It's written in  pure Python and has no external dependencies. The target are small apps that would be blown away by a SQL-DB or an  external database server.</p>
<p><em>homepage</em>: <a href="https://tinydb.readthedocs.io/">https://tinydb.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.15.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tinyxml">TinyXML<a class="headerlink" href="#tinyxml" title="Permanent link">&para;</a></h3>
<p>TinyXML is a simple, small, minimal, C++ XML parser that can be easily integrating  into other programs. It reads XML and creates C++ objects representing the XML document. The  objects can be manipulated, changed, and saved again as XML. </p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/tinyxml">https://sourceforge.net/projects/tinyxml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tk">Tk<a class="headerlink" href="#tk" title="Permanent link">&para;</a></h3>
<p>Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for  building a graphical user interface (GUI) in many different programming languages.</p>
<p><em>homepage</em>: <a href="https://www.tcl.tk/">https://www.tcl.tk/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.6.3</code></td>
<td><code>-no-X11</code></td>
<td><code>GCC/4.8.4</code>, <code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>8.6.4</code></td>
<td></td>
<td><code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>8.6.4</code></td>
<td><code>-libX11-1.6.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>8.6.4</code></td>
<td><code>-no-X11</code></td>
<td><code>GCC/4.9.3-2.25</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>8.6.5</code></td>
<td></td>
<td><code>GCC/5.4.0-2.26</code>, <code>GCCcore/6.3.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>iccifort/2016.3.210-GCC-5.4.0-2.26</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>8.6.6</code></td>
<td></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>8.6.7</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>8.6.8</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>foss/2018a</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>8.6.9</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>8.6.10</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>8.6.11</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>8.6.12</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tkinter">Tkinter<a class="headerlink" href="#tkinter" title="Permanent link">&para;</a></h3>
<p>Tkinter module, built with the Python buildsystem</p>
<p><em>homepage</em>: <a href="https://python.org/">https://python.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.7.13</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.7.14</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>2.7.15</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.7.15</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.7.16</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.7.18</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.6.1</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>foss/2017a</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.6.2</code></td>
<td><code>-Python-3.6.2</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>3.6.3</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>3.6.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code>, <code>iomkl/2018.02</code>, <code>iomkl/2018a</code></td>
</tr>
<tr>
<td><code>3.6.6</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>3.7.2</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.7.4</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.8.2</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.8.6</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.9.5</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.9.6</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.10.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>3.10.8</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tm-align">TM-align<a class="headerlink" href="#tm-align" title="Permanent link">&para;</a></h3>
<p>This package unifies protein structure alignment and RNA structure alignment  into the standard TM-align program for single chain structure alignment, MM-align program for  multi-chain structure alignment, and TM-score program for sequence dependent structure superposition.</p>
<p><em>homepage</em>: <a href="https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/">https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180426</code></td>
<td><code>foss/2018b</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>20190822</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tmae">tMAE<a class="headerlink" href="#tmae" title="Permanent link">&para;</a></h3>
<p>Package containing functions to:      perform a negative binomial test on allele-specific counts     add gnomAD minor allele frequencies     MAplot (FC vs total counts) of allele-specific counts and results     allelic counts (ALT vs REF)</p>
<p><em>homepage</em>: <a href="https://github.com/mumichae/tMAE">https://github.com/mumichae/tMAE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-R-4.0.3</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="tmap">tmap<a class="headerlink" href="#tmap" title="Permanent link">&para;</a></h3>
<p>tmap is a very fast visualization library for large, high-dimensional data sets. Currently, tmap is available for Python. tmap's graph layouts are based on the OGDF library.</p>
<p><em>homepage</em>: <a href="https://tmap.gdb.tools">https://tmap.gdb.tools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20220502</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tmux">tmux<a class="headerlink" href="#tmux" title="Permanent link">&para;</a></h3>
<p>tmux is a terminal multiplexer: it enables a number of terminals to be created, accessed, and controlled from a single screen. tmux may be detached from a screen and continue running in the background, then later reattached.</p>
<p><em>homepage</em>: <a href="https://github.com/tmux/tmux/">https://github.com/tmux/tmux/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9a</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>2.2</code></td>
<td><code>GCCcore/4.9.3</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>GCC/5.4.0-2.26</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.1c</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2a</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>system</code></td>
</tr>
<tr>
<td><code>3.3a</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tn93">TN93<a class="headerlink" href="#tn93" title="Permanent link">&para;</a></h3>
<p>This is a simple program meant to compute pairwise distances between  aligned nucleotide sequences in sequential FASTA format using the Tamura Nei 93 distance.</p>
<p><em>homepage</em>: <a href="https://github.com/veg/tn93">https://github.com/veg/tn93</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.7</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tobias">TOBIAS<a class="headerlink" href="#tobias" title="Permanent link">&para;</a></h3>
<p>TOBIAS is a collection of command-line bioinformatics tools  for performing footprinting analysis on ATAC-seq data.</p>
<p><em>homepage</em>: <a href="https://github.com/loosolab/TOBIAS">https://github.com/loosolab/TOBIAS</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.12</code></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>0.14.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="tofu">ToFu<a class="headerlink" href="#tofu" title="Permanent link">&para;</a></h3>
<p>Tomography for Fusion.</p>
<p><em>homepage</em>: <a href="https://github.com/ToFuProject/tofu">https://github.com/ToFuProject/tofu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.17</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.3.17</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="togl">Togl<a class="headerlink" href="#togl" title="Permanent link">&para;</a></h3>
<p>A Tcl/Tk widget for OpenGL rendering.</p>
<p><em>homepage</em>: <a href="https://sourceforge.net/projects/togl/">https://sourceforge.net/projects/togl/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="toil">toil<a class="headerlink" href="#toil" title="Permanent link">&para;</a></h3>
<p>A scalable, efficient, cross-platform (Linux/macOS) and easy-to-use workflow engine in pure Python.</p>
<p><em>homepage</em>: <a href="https://github.com/DataBiosphere/toil">https://github.com/DataBiosphere/toil</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.8.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="tokenizers">tokenizers<a class="headerlink" href="#tokenizers" title="Permanent link">&para;</a></h3>
<p>Fast State-of-the-Art Tokenizers optimized for Research and Production</p>
<p><em>homepage</em>: <a href="https://github.com/huggingface/tokenizers">https://github.com/huggingface/tokenizers</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tombo">Tombo<a class="headerlink" href="#tombo" title="Permanent link">&para;</a></h3>
<p>Tombo is a suite of tools primarily for the identification of modified nucleotides  from raw nanopore sequencing data.</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/tombo">https://github.com/nanoporetech/tombo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="toml-fortran">TOML-Fortran<a class="headerlink" href="#toml-fortran" title="Permanent link">&para;</a></h3>
<p>TOML parser for Fortran projects</p>
<p><em>homepage</em>: <a href="https://github.com/toml-f/toml-f">https://github.com/toml-f/toml-f</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.2</code></td>
<td><code>GCC/10.2.0</code>, <code>iccifort/2020.4.304</code></td>
</tr>
<tr>
<td><code>0.3.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="topaz">topaz<a class="headerlink" href="#topaz" title="Permanent link">&para;</a></h3>
<p>A pipeline for particle detection in cryo-electron microscopy  images using convolutional neural networks trained from positive and unlabeled  examples. Topaz also includes methods for micrograph denoising using deep de-  noising models. </p>
<p><em>homepage</em>: <a href="https://github.com/tbepler/topaz">https://github.com/tbepler/topaz</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.5</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.2.5</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="tophat">TopHat<a class="headerlink" href="#tophat" title="Permanent link">&para;</a></h3>
<p>TopHat is a fast splice junction mapper for RNA-Seq reads.</p>
<p><em>homepage</em>: <a href="http://ccb.jhu.edu/software/tophat/">http://ccb.jhu.edu/software/tophat/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="torchaudio">torchaudio<a class="headerlink" href="#torchaudio" title="Permanent link">&para;</a></h3>
<p>Data manipulation and transformation for audio signal processing, powered by PyTorch </p>
<p><em>homepage</em>: <a href="https://github.com/pytorch/audio">https://github.com/pytorch/audio</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-3.7.4-PyTorch-1.4.0</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="torchdata">torchdata<a class="headerlink" href="#torchdata" title="Permanent link">&para;</a></h3>
<p>TorchData is a prototype library of common modular data loading primitives for easily constructing flexible and performant data pipelines." </p>
<p><em>homepage</em>: <a href="https://github.com/pytorch/data">https://github.com/pytorch/data</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td><code>-PyTorch-1.11.0-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="torchinfo">torchinfo<a class="headerlink" href="#torchinfo" title="Permanent link">&para;</a></h3>
<p>" Torchinfo provides information complementary to what is provided by print(your_model) in PyTorch, similar to Tensorflow's model.summary() API to view the visualization of the model, which is helpful while debugging your network. </p>
<p><em>homepage</em>: <a href="https://github.com/TylerYep/torchinfo">https://github.com/TylerYep/torchinfo</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.2</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="torchsampler">torchsampler<a class="headerlink" href="#torchsampler" title="Permanent link">&para;</a></h3>
<p>A (PyTorch) imbalanced dataset sampler for oversampling low classes and undersampling high frequent ones.</p>
<p><em>homepage</em>: <a href="https://github.com/ufoym/imbalanced-dataset-sampler">https://github.com/ufoym/imbalanced-dataset-sampler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.1.2</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="torchtext">torchtext<a class="headerlink" href="#torchtext" title="Permanent link">&para;</a></h3>
<p>Data loaders and abstractions for text and NLP</p>
<p><em>homepage</em>: <a href="https://github.com/pytorch/text">https://github.com/pytorch/text</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.0</code></td>
<td><code>-PyTorch-1.4.0-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4-PyTorch-1.6.0</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.8.1</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-PyTorch-1.9.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="torchvision">torchvision<a class="headerlink" href="#torchvision" title="Permanent link">&para;</a></h3>
<p>Datasets, Transforms and Models specific to Computer Vision</p>
<p><em>homepage</em>: <a href="https://github.com/pytorch/vision">https://github.com/pytorch/vision</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.2.0</code></td>
<td><code>-Python-3.6.4-CUDA-9.1.85</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.2.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.3.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.4.2</code></td>
<td><code>-PyTorch-1.3.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.4.2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-3.7.4-PyTorch-1.4.0</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4-PyTorch-1.6.0</code></td>
<td><code>foss/2019b</code>, <code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>-Python-3.7.4-PyTorch-1.6.0-imkl</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-PyTorch-1.7.1</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.8.2</code></td>
<td><code>-Python-3.7.4-PyTorch-1.7.1</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>-Python-3.7.4-PyTorch-1.8.1</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-PyTorch-1.9.0</code></td>
<td><code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.11.3</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.11.3</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.12.0</code></td>
<td><code>-PyTorch-1.11.0-CUDA-11.3.1</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.13.1</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="totalview">TotalView<a class="headerlink" href="#totalview" title="Permanent link">&para;</a></h3>
<p>TotalView is a GUI-based source code defect analysis tool that gives you unprecedented  control over processes and thread execution and visibility into program state and variables. It allows  you to debug one or many processes and/or threads in a single window with complete control over program  execution. This allows you to set breakpoints, stepping line by line through the code on a single thread,  or with coordinated groups of processes or threads, and run or halt arbitrary sets of processes or threads.  You can reproduce and troubleshoot difficult problems that can occur in concurrent programs that take  advantage of threads, OpenMP, MPI, GPUs or coprocessors.</p>
<p><em>homepage</em>: <a href="http://www.roguewave.com/products/totalview.aspx">http://www.roguewave.com/products/totalview.aspx</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.11.0-0</code></td>
<td><code>-linux-x86-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.11.0-2</code></td>
<td><code>-linux-x86-64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.12.0-0</code></td>
<td><code>-linux-x86-64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="tqdm">tqdm<a class="headerlink" href="#tqdm" title="Permanent link">&para;</a></h3>
<p>A fast, extensible progress bar for Python and CLI</p>
<p><em>homepage</em>: <a href="https://github.com/tqdm/tqdm">https://github.com/tqdm/tqdm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.24.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.24.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>4.29.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>4.32.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>4.41.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>4.41.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.47.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.51.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>4.56.2</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.60.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.61.1</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.61.2</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.62.3</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.64.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.64.1</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tracer">Tracer<a class="headerlink" href="#tracer" title="Permanent link">&para;</a></h3>
<p>Tracer is a graphical tool for visualization and diagnostics of MCMC output.</p>
<p><em>homepage</em>: <a href="https://github.com/beast-dev/tracer">https://github.com/beast-dev/tracer</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.7.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="transcriptclean">TranscriptClean<a class="headerlink" href="#transcriptclean" title="Permanent link">&para;</a></h3>
<p>TranscriptClean is a Python program that corrects mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome.</p>
<p><em>homepage</em>: <a href="https://github.com/mortazavilab/TranscriptClean">https://github.com/mortazavilab/TranscriptClean</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="transdecoder">TransDecoder<a class="headerlink" href="#transdecoder" title="Permanent link">&para;</a></h3>
<p>TransDecoder identifies candidate coding regions within transcript sequences,  such as those generated by de novo RNA-Seq transcript assembly using Trinity,  or constructed based on RNA-Seq alignments to the genome using  Tophat and Cufflinks.</p>
<p><em>homepage</em>: <a href="https://github.com/TransDecoder/TransDecoder/wiki">https://github.com/TransDecoder/TransDecoder/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="transformers">Transformers<a class="headerlink" href="#transformers" title="Permanent link">&para;</a></h3>
<p>State-of-the-art Natural Language Processing for PyTorch and TensorFlow 2.0</p>
<p><em>homepage</em>: <a href="https://github.com/huggingface/transformers">https://github.com/huggingface/transformers</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.2.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>4.20.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.21.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>4.24.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="transrate">Transrate<a class="headerlink" href="#transrate" title="Permanent link">&para;</a></h3>
<p>Transrate is software for de-novo transcriptome assembly quality analysis.  It examines your assembly in detail and compares it to experimental evidence  such as the sequencing reads, reporting quality scores for contigs and  assemblies. This allows you to choose between assemblers and parameters,  filter out the bad contigs from an assembly, and help decide when  to stop trying to improve the assembly.</p>
<p><em>homepage</em>: <a href="http://hibberdlab.com/transrate/">http://hibberdlab.com/transrate/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="travis">travis<a class="headerlink" href="#travis" title="Permanent link">&para;</a></h3>
<p>Travis CI Client (CLI and Ruby library)</p>
<p><em>homepage</em>: <a href="https://github.com/travis-ci/travis.rb">https://github.com/travis-ci/travis.rb</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.2</code></td>
<td><code>-Ruby-2.3.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.8.4</code></td>
<td><code>-Ruby-2.3.3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="travis-analyzer">TRAVIS-Analyzer<a class="headerlink" href="#travis-analyzer" title="Permanent link">&para;</a></h3>
<p>TRAVIS is a free tool for analyzing and visualizing trajectories from all kinds of Molecular Dynamics or Monte Carlo simulations.</p>
<p><em>homepage</em>: <a href="http://www.travis-analyzer.de/">http://www.travis-analyzer.de/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>210521</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="treatsens">treatSens<a class="headerlink" href="#treatsens" title="Permanent link">&para;</a></h3>
<p>Utilities to investigate sensitivity to unmeasured confounding in parametric models  with either binary or continuous treatment.</p>
<p><em>homepage</em>: <a href="https://github.com/vdorie/treatSens">https://github.com/vdorie/treatSens</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0-20201002</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="treemix">TreeMix<a class="headerlink" href="#treemix" title="Permanent link">&para;</a></h3>
<p>TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a  set of populations.</p>
<p><em>homepage</em>: <a href="http://bitbucket.org/nygcresearch/treemix">http://bitbucket.org/nygcresearch/treemix</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="treeshrink">TreeShrink<a class="headerlink" href="#treeshrink" title="Permanent link">&para;</a></h3>
<p>TreeShrink is an algorithm for detecting abnormally long branches in one or more phylogenetic trees.</p>
<p><em>homepage</em>: <a href="https://github.com/uym2/TreeShrink">https://github.com/uym2/TreeShrink</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.2</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
</tbody>
</table>
<h3 id="trf">TRF<a class="headerlink" href="#trf" title="Permanent link">&para;</a></h3>
<p>Tandem Repeats Finder: a program to analyze DNA sequences.</p>
<p><em>homepage</em>: <a href="https://tandem.bu.edu/trf/trf.html">https://tandem.bu.edu/trf/trf.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.09</code></td>
<td><code>-linux64</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.09.1</code></td>
<td></td>
<td><code>GCC/10.3.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="triangle">Triangle<a class="headerlink" href="#triangle" title="Permanent link">&para;</a></h3>
<p>Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,  conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.  The latter can be generated with no small or large angles,  and are thus suitable for finite element analysis.</p>
<p><em>homepage</em>: <a href="http://www.cs.cmu.edu/~quake/triangle.html">http://www.cs.cmu.edu/~quake/triangle.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>foss/2018a</code>, <code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="trilinos">Trilinos<a class="headerlink" href="#trilinos" title="Permanent link">&para;</a></h3>
<p>The Trilinos Project is an effort to develop algorithms and enabling technologies  within an object-oriented software framework for the solution of large-scale, complex multi-physics  engineering and scientific problems. A unique design feature of Trilinos is its focus on packages.</p>
<p><em>homepage</em>: <a href="https://trilinos.org">https://trilinos.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>12.12.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>12.12.1</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>12.12.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>13.4.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="trim_galore">Trim_Galore<a class="headerlink" href="#trim_galore" title="Permanent link">&para;</a></h3>
<p>Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control, with some added functionality to remove biased methylation positions for RRBS sequence files (for directional, non-directional  (or paired-end) sequencing).</p>
<p><em>homepage</em>: <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.4.4</code></td>
<td></td>
<td><code>foss/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Java-11</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.2</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>0.6.5</code></td>
<td><code>-Java-11-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.6.6</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.6.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>0.6.7</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>0.6.10</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="trimal">trimAl<a class="headerlink" href="#trimal" title="Permanent link">&para;</a></h3>
<p>A tool for automated alignment trimming in large-scale phylogenetic analyses</p>
<p><em>homepage</em>: <a href="http://trimal.cgenomics.org/">http://trimal.cgenomics.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code>, <code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="trimesh">trimesh<a class="headerlink" href="#trimesh" title="Permanent link">&para;</a></h3>
<p>Trimesh is a Python (2.7- 3.3+) library for loading and using triangular meshes with an emphasis on watertight meshes. The goal of the library is to provide a fully featured Trimesh object which allows for easy manipulation and analysis, in the style of the excellent Polygon object in the Shapely library.</p>
<p><em>homepage</em>: <a href="https://trimsh.org/">https://trimsh.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.17.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="trimmomatic">Trimmomatic<a class="headerlink" href="#trimmomatic" title="Permanent link">&para;</a></h3>
<p>Trimmomatic performs a variety of useful trimming tasks for illumina   paired-end and single ended data.The selection of trimming steps and their associated   parameters are supplied on the command line. </p>
<p><em>homepage</em>: <a href="http://www.usadellab.org/cms/?page=trimmomatic">http://www.usadellab.org/cms/?page=trimmomatic</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.32</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.36</code></td>
<td><code>-Java-1.8.0_92</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.38</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.38</code></td>
<td><code>-Java-1.8.0_162</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.39</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.39</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="trinity">Trinity<a class="headerlink" href="#trinity" title="Permanent link">&para;</a></h3>
<p>Trinity represents a novel method for the efficient and robust de novo reconstruction  of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,  Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.</p>
<p><em>homepage</em>: <a href="https://trinityrnaseq.github.io">https://trinityrnaseq.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.6.6</code></td>
<td></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.8.4</code></td>
<td></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.8.5</code></td>
<td></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.8.5</code></td>
<td><code>-Java-11</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.9.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.10.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>2.15.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="triqs">TRIQS<a class="headerlink" href="#triqs" title="Permanent link">&para;</a></h3>
<p>TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems. </p>
<p><em>homepage</em>: <a href="https://triqs.github.io/triqs">https://triqs.github.io/triqs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>2.2.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="triqs-cthyb">TRIQS-cthyb<a class="headerlink" href="#triqs-cthyb" title="Permanent link">&para;</a></h3>
<p>TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   cthyb = continuous-time hybridisation-expansion quantum Monte Carlo   The TRIQS-based hybridization-expansion solver allows to solve the  generic problem of a quantum impurity embedded in a conduction bath  for an arbitrary local interaction vertex. The impurity can be any  set of orbitals, on one or several atoms. </p>
<p><em>homepage</em>: <a href="https://triqs.github.io/cthyb/">https://triqs.github.io/cthyb/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="triqs-dft_tools">TRIQS-dft_tools<a class="headerlink" href="#triqs-dft_tools" title="Permanent link">&para;</a></h3>
<p>TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   This TRIQS-based-based application is aimed at ab-initio calculations  for correlated materials, combining realistic DFT band-structure calculation   with the dynamical mean-field theory. Together with the necessary tools to  perform the DMFT self-consistency loop for realistic multi-band problems,  the package provides a full-fledged charge self-consistent interface to the  Wien2K package. In addition, if Wien2k is not available, it provides a generic  interface for one-shot DFT+DMFT calculations, where only the single-particle  Hamiltonian in orbital space has to be provided. </p>
<p><em>homepage</em>: <a href="https://triqs.github.io/dft_tools/">https://triqs.github.io/dft_tools/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="triqs-tprf">TRIQS-tprf<a class="headerlink" href="#triqs-tprf" title="Permanent link">&para;</a></h3>
<p>TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   TPRF is a TRIQS-based two-particle response function tool box that  implements basic operations for higher order response functions such  as inversion, products, the random phase approximation, the bethe  salpeter equation (in the local vertex approximation), etc..   The aim is to provide efficient (C++/OpenMP/MPI) implementations of  the basic operations needed to compute the two-particle response in  the different two-particle channels (particle-hole, particle-particle). </p>
<p><em>homepage</em>: <a href="https://triqs.github.io/tprf">https://triqs.github.io/tprf</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>3.1.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="trust">TRUST<a class="headerlink" href="#trust" title="Permanent link">&para;</a></h3>
<p>Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze  TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. </p>
<p><em>homepage</em>: <a href="https://bitbucket.org/liulab/trust">https://bitbucket.org/liulab/trust</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="trust4">TRUST4<a class="headerlink" href="#trust4" title="Permanent link">&para;</a></h3>
<p>Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. TRUST4 performs de novo assembly on V, J, C genes including the hypervariable complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to report the corresponding information. TRUST4 supports both single-end and paired-end sequencing data with any read length.</p>
<p><em>homepage</em>: <a href="https://github.com/liulab-dfci/TRUST4">https://github.com/liulab-dfci/TRUST4</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.5.1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.0.6</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>1.0.7</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="trycycler">Trycycler<a class="headerlink" href="#trycycler" title="Permanent link">&para;</a></h3>
<p>Trycycler is a tool for generating consensus long-read assemblies for bacterial genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/rrwick/Trycycler">https://github.com/rrwick/Trycycler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>0.5.2</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.5.3</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="tsne">tsne<a class="headerlink" href="#tsne" title="Permanent link">&para;</a></h3>
<p>Python library containing T-SNE algorithms.</p>
<p><em>homepage</em>: <a href="https://github.com/danielfrg/tsne">https://github.com/danielfrg/tsne</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.8</code></td>
<td><code>-Python-2.7.16</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="turbovnc">TurboVNC<a class="headerlink" href="#turbovnc" title="Permanent link">&para;</a></h3>
<p>TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide  peak performance for 3D and video workloads.</p>
<p><em>homepage</em>: <a href="https://www.turbovnc.org">https://www.turbovnc.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.0.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="tvb">TVB<a class="headerlink" href="#tvb" title="Permanent link">&para;</a></h3>
<p>The Virtual Brain will deliver the first open simulation of the human brain  based on individual large-scale connectivity.</p>
<p><em>homepage</em>: <a href="http://thevirtualbrain.org">http://thevirtualbrain.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tvb-data">tvb-data<a class="headerlink" href="#tvb-data" title="Permanent link">&para;</a></h3>
<p>The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  Various demonstration data for use with TVB.</p>
<p><em>homepage</em>: <a href="http://www.thevirtualbrain.org/">http://www.thevirtualbrain.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20150915</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tvb-deps">TVB-deps<a class="headerlink" href="#tvb-deps" title="Permanent link">&para;</a></h3>
<p>Bundle of dependency Python packages for TVB (The Virtual Brain)</p>
<p><em>homepage</em>: <a href="https://thevirtualbrain.org">https://thevirtualbrain.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160618</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tvb-framework">tvb-framework<a class="headerlink" href="#tvb-framework" title="Permanent link">&para;</a></h3>
<p>The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.</p>
<p><em>homepage</em>: <a href="http://www.thevirtualbrain.org/">http://www.thevirtualbrain.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20150921</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="tvb-library">tvb-library<a class="headerlink" href="#tvb-library" title="Permanent link">&para;</a></h3>
<p>The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.</p>
<p><em>homepage</em>: <a href="http://www.thevirtualbrain.org/">http://www.thevirtualbrain.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20150922</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="twl-ninja">TWL-NINJA<a class="headerlink" href="#twl-ninja" title="Permanent link">&para;</a></h3>
<p>Nearly Infinite Neighbor Joining Application.</p>
<p><em>homepage</em>: <a href="https://github.com/TravisWheelerLab/NINJA">https://github.com/TravisWheelerLab/NINJA</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.97-cluster_only</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>0.98-cluster_only</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="typing-extensions">typing-extensions<a class="headerlink" href="#typing-extensions" title="Permanent link">&para;</a></h3>
<p>Typing Extensions - Backported and Experimental Type Hints for Python</p>
<p><em>homepage</em>: <a href="https://github.com/python/typing/blob/master/typing_extensions/README.rst">https://github.com/python/typing/blob/master/typing_extensions/README.rst</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.7.4.3</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>3.7.4.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.7.4.3</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.10.0.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.10.0.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.3.0</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.4.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="u">U<a class="headerlink" href="#u" title="Permanent link">&para;</a></h2>
<p><a href="#ucc">UCC</a> - <a href="#ucc-cuda">UCC-CUDA</a> - <a href="#uclust">UCLUST</a> - <a href="#ucx">UCX</a> - <a href="#ucx-cuda">UCX-CUDA</a> - <a href="#ucx-py">ucx-py</a> - <a href="#ucx-rocm">UCX-ROCm</a> - <a href="#udocker">udocker</a> - <a href="#udunits">UDUNITS</a> - <a href="#ufl">UFL</a> - <a href="#umap-learn">umap-learn</a> - <a href="#umi-tools">UMI-tools</a> - <a href="#umi4cpackage">umi4cPackage</a> - <a href="#umis">umis</a> - <a href="#unafold">UNAFold</a> - <a href="#uncertainty-calibration">uncertainty-calibration</a> - <a href="#unicycler">Unicycler</a> - <a href="#unidecode">Unidecode</a> - <a href="#unimap">unimap</a> - <a href="#units">units</a> - <a href="#unixodbc">unixODBC</a> - <a href="#unrar">unrar</a> - <a href="#unzip">UnZip</a> - <a href="#uqtk">UQTk</a> - <a href="#usearch">USEARCH</a> - <a href="#usher">UShER</a> - <a href="#uspex">USPEX</a> - <a href="#utf8proc">utf8proc</a> - <a href="#util-linux">util-linux</a></p>
<h3 id="ucc">UCC<a class="headerlink" href="#ucc" title="Permanent link">&para;</a></h3>
<p>UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes. </p>
<p><em>homepage</em>: <a href="https://www.openucx.org/">https://www.openucx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ucc-cuda">UCC-CUDA<a class="headerlink" href="#ucc-cuda" title="Permanent link">&para;</a></h3>
<p>UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes.  This module adds the UCC CUDA support. </p>
<p><em>homepage</em>: <a href="https://www.openucx.org/">https://www.openucx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="uclust">UCLUST<a class="headerlink" href="#uclust" title="Permanent link">&para;</a></h3>
<p>UCLUST: Extreme high-speed sequence clustering, alignment and database search.</p>
<p><em>homepage</em>: <a href="https://www.drive5.com/usearch/index.html">https://www.drive5.com/usearch/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.22q</code></td>
<td><code>-i86linux64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="ucx">UCX<a class="headerlink" href="#ucx" title="Permanent link">&para;</a></h3>
<p>Unified Communication X An open-source production grade communication framework for data centric and high-performance applications </p>
<p><em>homepage</em>: <a href="http://www.openucx.org/">http://www.openucx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.1</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.5.0rc1</code></td>
<td><code>-hpcx</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.5.1</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>1.6.1</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.8.0</code></td>
<td><code>-CUDA-11.0.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-CUDA-11.1.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.9.0</code></td>
<td><code>-CUDA-11.2.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.10.0</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.11.2</code></td>
<td></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ucx-cuda">UCX-CUDA<a class="headerlink" href="#ucx-cuda" title="Permanent link">&para;</a></h3>
<p>Unified Communication X An open-source production grade communication framework for data centric and high-performance applications  This module adds the UCX CUDA support. </p>
<p><em>homepage</em>: <a href="http://www.openucx.org/">http://www.openucx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.10.0</code></td>
<td><code>-CUDA-11.3.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.11.0</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.11.2</code></td>
<td><code>-CUDA-11.4.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.12.1</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.13.1</code></td>
<td><code>-CUDA-12.0.0</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ucx-py">ucx-py<a class="headerlink" href="#ucx-py" title="Permanent link">&para;</a></h3>
<p>UCX-Py is the Python interface for UCX, a low-level high-performance networking library. UCX and UCX-Py supports several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.</p>
<p><em>homepage</em>: <a href="https://ucx-py.readthedocs.io">https://ucx-py.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.21.0</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="ucx-rocm">UCX-ROCm<a class="headerlink" href="#ucx-rocm" title="Permanent link">&para;</a></h3>
<p>Unified Communication X An open-source production grade communication framework for data centric and high-performance applications  This module adds the UCX ROCm support. </p>
<p><em>homepage</em>: <a href="http://www.openucx.org/">http://www.openucx.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11.2</code></td>
<td><code>-ROCm-4.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="udocker">udocker<a class="headerlink" href="#udocker" title="Permanent link">&para;</a></h3>
<p>A basic user tool to execute simple docker containers in batch  or interactive systems without root privileges.</p>
<p><em>homepage</em>: <a href="https://github.com/indigo-dc/udocker">https://github.com/indigo-dc/udocker</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="udunits">UDUNITS<a class="headerlink" href="#udunits" title="Permanent link">&para;</a></h3>
<p>UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.</p>
<p><em>homepage</em>: <a href="https://www.unidata.ucar.edu/software/udunits/">https://www.unidata.ucar.edu/software/udunits/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.20</code></td>
<td><code>foss/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.2.24</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.2.25</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.2.26</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2018a</code>, <code>foss/2018b</code>, <code>foss/2020a</code>, <code>intel/2017b</code>, <code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>2.2.28</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="ufl">UFL<a class="headerlink" href="#ufl" title="Permanent link">&para;</a></h3>
<p>The Unified Form Language (UFL) is a domain specific language   for declaration of finite element discretizations of variational forms.   More precisely, it defines a flexible interface for choosing finite element   spaces and defining expressions for weak forms in a notation close to   mathematical notation.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/fenics-project/ufl">https://bitbucket.org/fenics-project/ufl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>2019.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="umap-learn">umap-learn<a class="headerlink" href="#umap-learn" title="Permanent link">&para;</a></h3>
<p>Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique  that can be used for visualisation similarly to t-SNE, but also for general non-linear  dimension reduction. </p>
<p><em>homepage</em>: <a href="https://umap-learn.readthedocs.io/en/latest/">https://umap-learn.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.6</code></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>0.5.3</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="umi-tools">UMI-tools<a class="headerlink" href="#umi-tools" title="Permanent link">&para;</a></h3>
<p>Tools for handling Unique Molecular Identifiers in NGS data sets</p>
<p><em>homepage</em>: <a href="https://umi-tools.readthedocs.io">https://umi-tools.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.0.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="umi4cpackage">umi4cPackage<a class="headerlink" href="#umi4cpackage" title="Permanent link">&para;</a></h3>
<p>umi4cPackage is a processing and analysis pipeline for UMI-4C experiment.</p>
<p><em>homepage</em>: <a href="https://github.com/tanaylab/umi4cpackage">https://github.com/tanaylab/umi4cpackage</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20200116</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="umis">umis<a class="headerlink" href="#umis" title="Permanent link">&para;</a></h3>
<p>Package for estimating UMI counts in Transcript Tag Counting data.</p>
<p><em>homepage</em>: <a href="https://github.com/vals/umis">https://github.com/vals/umis</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="unafold">UNAFold<a class="headerlink" href="#unafold" title="Permanent link">&para;</a></h3>
<p>The UNAFold package contains several programs for performing energy minimization and partition  function calculations on nucleic acid sequences.</p>
<p><em>homepage</em>: <a href="http://mfold.rna.albany.edu/?q=DINAMelt/software">http://mfold.rna.albany.edu/?q=DINAMelt/software</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="uncertainty-calibration">uncertainty-calibration<a class="headerlink" href="#uncertainty-calibration" title="Permanent link">&para;</a></h3>
<p>Python library to measure the calibration error of models, including confidence intervals computed by Bootstrap resampling, and code to recalibrate models.</p>
<p><em>homepage</em>: <a href="https://github.com/p-lambda/verified_calibration">https://github.com/p-lambda/verified_calibration</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.9</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="unicycler">Unicycler<a class="headerlink" href="#unicycler" title="Permanent link">&para;</a></h3>
<p>Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets  where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore)  where it runs a miniasm+Racon pipeline. </p>
<p><em>homepage</em>: <a href="https://github.com/rrwick/Unicycler">https://github.com/rrwick/Unicycler</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.7</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>0.4.8</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>gompi/2020a</code></td>
</tr>
<tr>
<td><code>0.4.9</code></td>
<td></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="unidecode">Unidecode<a class="headerlink" href="#unidecode" title="Permanent link">&para;</a></h3>
<p>Python library for lossy ASCII transliterations of Unicode text (port of Text::Unidecode Perl module)</p>
<p><em>homepage</em>: <a href="https://github.com/avian2/unidecode">https://github.com/avian2/unidecode</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.3.6</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="unimap">unimap<a class="headerlink" href="#unimap" title="Permanent link">&para;</a></h3>
<p>Unimap is a fork of minimap2 optimized for assembly-to-reference alignment. It integrates the minigraph chaining algorithm and can align through long INDELs (up to 100kb by default) much faster than minimap2. Unimap is a better fit for resolving segmental duplications and is recommended over minimap2 for alignment between high-quality assemblies.  Unimap does not replace minimap2 for other types of alignment. It drops the support of multi-part index and short-read mapping. Its long-read alignment is different from minimap2 but is not necessarily better. Unimap is more of a specialized minimap2 at the moment.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/unimap">https://github.com/lh3/unimap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="units">units<a class="headerlink" href="#units" title="Permanent link">&para;</a></h3>
<p>GNU Units converts quantities expressed in various systems of measurement to their equivalents in other systems of measurement. Like many similar programs, it can handle multiplicative scale changes. It can also handle nonlinear conversions such as Fahrenheit to Celsius or wire gauge, and it can convert from and to sums of units, such as converting between meters and feet plus inches.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/units/">https://www.gnu.org/software/units/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.19</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="unixodbc">unixODBC<a class="headerlink" href="#unixodbc" title="Permanent link">&para;</a></h3>
<p>unixODBC provides a uniform interface between application and database driver</p>
<p><em>homepage</em>: <a href="https://www.unixodbc.org">https://www.unixodbc.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.7</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="unrar">unrar<a class="headerlink" href="#unrar" title="Permanent link">&para;</a></h3>
<p>RAR is a powerful archive manager.</p>
<p><em>homepage</em>: <a href="https://www.rarlab.com/">https://www.rarlab.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.6.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>5.7.3</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>6.0.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="unzip">UnZip<a class="headerlink" href="#unzip" title="Permanent link">&para;</a></h3>
<p>UnZip is an extraction utility for archives compressed in .zip format (also called "zipfiles"). Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own Zip program, our primary objectives have been portability and non-MSDOS functionality.</p>
<p><em>homepage</em>: <a href="http://www.info-zip.org/UnZip.html">http://www.info-zip.org/UnZip.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="uqtk">UQTk<a class="headerlink" href="#uqtk" title="Permanent link">&para;</a></h3>
<p>The UQ Toolkit (UQTk) is a collection of libraries and tools  for the quantification of uncertainty in numerical model predictions.</p>
<p><em>homepage</em>: <a href="https://www.sandia.gov/UQToolkit/">https://www.sandia.gov/UQToolkit/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="usearch">USEARCH<a class="headerlink" href="#usearch" title="Permanent link">&para;</a></h3>
<p>USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are  often orders of magnitude faster than BLAST.</p>
<p><em>homepage</em>: <a href="http://www.drive5.com/usearch/index.html">http://www.drive5.com/usearch/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.2.236</code></td>
<td><code>-i86linux32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.236-6.1.544</code></td>
<td><code>-i86linux32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>6.1.544</code></td>
<td><code>-i86linux32</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>11.0.667</code></td>
<td><code>-i86linux32</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="usher">UShER<a class="headerlink" href="#usher" title="Permanent link">&para;</a></h3>
<p>UShER is now a package consisting of a family of programs for  rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/yatisht/usher">https://github.com/yatisht/usher</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>0.5.0</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="uspex">USPEX<a class="headerlink" href="#uspex" title="Permanent link">&para;</a></h3>
<p>USPEX (Universal Structure Predictor: Evolutionary Xtallography...  and in Russian "uspekh" means "success" - owing to the high success rate and many  useful results produced by this method) is a method developed by the Oganov  laboratory since 2004. The problem of crystal structure prediction is very old and  does, in fact, constitute the central problem of theoretical crystal chemistry.  USPEX can also be used for finding low-energy metastable phases, as well as stable  structures of nanoparticles, surface reconstructions, molecular packings in organic  crystals, and for searching for materials with desired physical (mechanical,  electronic) properties. The USPEX code is based on an efficient evolutionary  algorithm developed by A.R. Oganov's group, but also has options for using  alternative methods (random sampling, metadynamics, corrected particle swarm  optimization algorithms). USPEX is interfaced with many ab initio codes, such as  VASP, SIESTA, GULP, Quantum Espresso, CP2K, CASTEP, LAMMPS, and so on. </p>
<p><em>homepage</em>: <a href="http://uspex-team.org/uspex/overview">http://uspex-team.org/uspex/overview</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>9.4.4</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="utf8proc">utf8proc<a class="headerlink" href="#utf8proc" title="Permanent link">&para;</a></h3>
<p>utf8proc is a small, clean C library that provides Unicode normalization, case-folding,  and other operations for data in the UTF-8 encoding.</p>
<p><em>homepage</em>: <a href="https://github.com/JuliaStrings/utf8proc">https://github.com/JuliaStrings/utf8proc</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.3.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.6.1</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="util-linux">util-linux<a class="headerlink" href="#util-linux" title="Permanent link">&para;</a></h3>
<p>Set of Linux utilities</p>
<p><em>homepage</em>: <a href="https://www.kernel.org/pub/linux/utils/util-linux">https://www.kernel.org/pub/linux/utils/util-linux</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.27.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.28</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.28.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.29</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.29.2</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.30</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.30.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.31</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.31.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.32</code></td>
<td><code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.32.1</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.33</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.34</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.35</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.36</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.37</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>2.38</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>2.38.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="v">V<a class="headerlink" href="#v" title="Permanent link">&para;</a></h2>
<p><a href="#v8">V8</a> - <a href="#vala">Vala</a> - <a href="#valgrind">Valgrind</a> - <a href="#vampir">Vampir</a> - <a href="#vampire">Vampire</a> - <a href="#vardict">VarDict</a> - <a href="#variant_tools">variant_tools</a> - <a href="#variantmetacaller">VariantMetaCaller</a> - <a href="#varscan">VarScan</a> - <a href="#vartools">vartools</a> - <a href="#vasp">VASP</a> - <a href="#vatools">VAtools</a> - <a href="#vawk">vawk</a> - <a href="#vbz-compression">VBZ-Compression</a> - <a href="#vcf-kit">VCF-kit</a> - <a href="#vcflib">vcflib</a> - <a href="#vcfnp">vcfnp</a> - <a href="#vcftools">VCFtools</a> - <a href="#vcontact2">vConTACT2</a> - <a href="#vegas">VEGAS</a> - <a href="#velocyto">velocyto</a> - <a href="#velvet">Velvet</a> - <a href="#vep">VEP</a> - <a href="#verifybamid">verifyBamID</a> - <a href="#verse">VERSE</a> - <a href="#vesta">VESTA</a> - <a href="#viennarna">ViennaRNA</a> - <a href="#vim">Vim</a> - <a href="#virsorter">VirSorter</a> - <a href="#virsorter2">VirSorter2</a> - <a href="#virtualgl">VirtualGL</a> - <a href="#virtuoso-opensource">Virtuoso-opensource</a> - <a href="#vispy">VisPy</a> - <a href="#vitessce-python">vitessce-python</a> - <a href="#vitesscer">vitessceR</a> - <a href="#vmd">VMD</a> - <a href="#vmtk">VMTK</a> - <a href="#voltools">voltools</a> - <a href="#vorbis-tools">vorbis-tools</a> - <a href="#voro++">Voro++</a> - <a href="#vsc-base">vsc-base</a> - <a href="#vsc-install">vsc-install</a> - <a href="#vsc-mympirun">vsc-mympirun</a> - <a href="#vsc-mympirun-scoop">vsc-mympirun-scoop</a> - <a href="#vsc-processcontrol">vsc-processcontrol</a> - <a href="#vsearch">VSEARCH</a> - <a href="#vt">vt</a> - <a href="#vtk">VTK</a> - <a href="#vtune">VTune</a> - <a href="#vv">VV</a> - <a href="#vxl">VXL</a></p>
<h3 id="v8">V8<a class="headerlink" href="#v8" title="Permanent link">&para;</a></h3>
<p>R interface to Google's open source JavaScript engine</p>
<p><em>homepage</em>: <a href="https://cran.r-project.org/web/packages/V8/">https://cran.r-project.org/web/packages/V8/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.3</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>-R-3.6.2</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>3.4.0</code></td>
<td><code>-R-4.0.0</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="vala">Vala<a class="headerlink" href="#vala" title="Permanent link">&para;</a></h3>
<p>Vala is a programming language using modern high level abstractions without imposing additional runtime requirements and without using a different ABI compared to applications and libraries written in C.</p>
<p><em>homepage</em>: <a href="https://wiki.gnome.org/Projects/Vala">https://wiki.gnome.org/Projects/Vala</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.52.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="valgrind">Valgrind<a class="headerlink" href="#valgrind" title="Permanent link">&para;</a></h3>
<p>Valgrind: Debugging and profiling tools</p>
<p><em>homepage</em>: <a href="https://valgrind.org">https://valgrind.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.11.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.13.0</code></td>
<td><code>foss/2017b</code>, <code>foss/2018a</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>3.14.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>3.16.1</code></td>
<td><code>gompi/2019b</code>, <code>gompi/2020a</code>, <code>gompi/2020b</code>, <code>iimpi/2020a</code></td>
</tr>
<tr>
<td><code>3.17.0</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>3.18.1</code></td>
<td><code>gompi/2021b</code>, <code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>3.19.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
<tr>
<td><code>3.20.0</code></td>
<td><code>gompi/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vampir">Vampir<a class="headerlink" href="#vampir" title="Permanent link">&para;</a></h3>
<p>The Vampir software tool provides an easy-to-use framework that enables  developers to quickly display and analyze arbitrary program behavior at any level of detail.  The tool suite implements optimized event analysis algorithms and customizable displays that  enable fast and interactive rendering of very complex performance monitoring data.</p>
<p><em>homepage</em>: <a href="http://www.vampir.eu">http://www.vampir.eu</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.4.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>8.4.1</code></td>
<td><code>-demo</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vampire">Vampire<a class="headerlink" href="#vampire" title="Permanent link">&para;</a></h3>
<p>The Vampire Theorem Prover.</p>
<p><em>homepage</em>: <a href="https://vprover.github.io/">https://vprover.github.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.5.1</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="vardict">VarDict<a class="headerlink" href="#vardict" title="Permanent link">&para;</a></h3>
<p>VarDict is an ultra sensitive variant caller for both single  and paired sample variant calling from BAM files.</p>
<p><em>homepage</em>: <a href="https://github.com/AstraZeneca-NGS/VarDict">https://github.com/AstraZeneca-NGS/VarDict</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.7</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="variant_tools">variant_tools<a class="headerlink" href="#variant_tools" title="Permanent link">&para;</a></h3>
<p>Variant tools is a software tool for the manipulation, annotation, selection, simulation, and  analysis of variants in the context of next-gen sequencing analysis. Unlike some other tools used for Next-Gen  sequencing analysis, variant tools is project based and provides a whole set of tools to manipulate and analyze  genetic variants.</p>
<p><em>homepage</em>: <a href="https://vatlab.github.io/vat-docs/">https://vatlab.github.io/vat-docs/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.3</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="variantmetacaller">VariantMetaCaller<a class="headerlink" href="#variantmetacaller" title="Permanent link">&para;</a></h3>
<p>VariantMetaCaller automatically integrates variant calling pipelines into a better performing  overall model that also predicts accurate variant probabilities.</p>
<p><em>homepage</em>: <a href="http://bioinformatics.mit.bme.hu/VariantMetaCaller/">http://bioinformatics.mit.bme.hu/VariantMetaCaller/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="varscan">VarScan<a class="headerlink" href="#varscan" title="Permanent link">&para;</a></h3>
<p>Variant calling and somatic mutation/CNV detection for next-generation sequencing data</p>
<p><em>homepage</em>: <a href="https://github.com/dkoboldt/varscan">https://github.com/dkoboldt/varscan</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.6</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.1</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.4</code></td>
<td><code>-Java-1.8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.4</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vartools">vartools<a class="headerlink" href="#vartools" title="Permanent link">&para;</a></h3>
<p>Command line utility that provides tools for processing   and analyzing astronomical time series data.</p>
<p><em>homepage</em>: <a href="https://www.astro.princeton.edu/~jhartman/vartools.html">https://www.astro.princeton.edu/~jhartman/vartools.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.35</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="vasp">VASP<a class="headerlink" href="#vasp" title="Permanent link">&para;</a></h3>
<p>The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.</p>
<p><em>homepage</em>: <a href="http://www.vasp.at">http://www.vasp.at</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.4.1</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
</tbody>
</table>
<h3 id="vatools">VAtools<a class="headerlink" href="#vatools" title="Permanent link">&para;</a></h3>
<p>VAtools is a python package that includes several tools to annotate VCF files  with data from other tools. </p>
<p><em>homepage</em>: <a href="https://vatools.readthedocs.io/en/latest/">https://vatools.readthedocs.io/en/latest/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="vawk">vawk<a class="headerlink" href="#vawk" title="Permanent link">&para;</a></h3>
<p>An awk-like VCF parser  </p>
<p><em>homepage</em>: <a href="https://github.com/cc2qe/vawk">https://github.com/cc2qe/vawk</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.0.1</code></td>
<td><code>-Python-2.7.18</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="vbz-compression">VBZ-Compression<a class="headerlink" href="#vbz-compression" title="Permanent link">&para;</a></h3>
<p>VBZ compression HDF5 plugin for nanopolish</p>
<p><em>homepage</em>: <a href="https://github.com/nanoporetech/vbz_compression">https://github.com/nanoporetech/vbz_compression</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.1</code></td>
<td><code>gompi/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="vcf-kit">VCF-kit<a class="headerlink" href="#vcf-kit" title="Permanent link">&para;</a></h3>
<p>VCF-kit is a command-line based collection of utilities for performing analysis on  Variant Call Format (VCF) files.</p>
<p><em>homepage</em>: <a href="https://github.com/AndersenLab/VCF-kit">https://github.com/AndersenLab/VCF-kit</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.6</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="vcflib">vcflib<a class="headerlink" href="#vcflib" title="Permanent link">&para;</a></h3>
<p>vcflib provides methods to manipulate and interpret sequence variation as it can be  described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended  to concisely describe reference-indexed genetic variations between individuals.</p>
<p><em>homepage</em>: <a href="https://github.com/vcflib/vcflib">https://github.com/vcflib/vcflib</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.2</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>1.0.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-R-4.1.0</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.0.3</code></td>
<td><code>-R-4.1.2</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="vcfnp">vcfnp<a class="headerlink" href="#vcfnp" title="Permanent link">&para;</a></h3>
<p>Load data from a VCF (variant call format) file into numpy arrays,   and (optionally) from there into an HDF5 file.</p>
<p><em>homepage</em>: <a href="https://github.com/alimanfoo/vcfnp">https://github.com/alimanfoo/vcfnp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="vcftools">VCFtools<a class="headerlink" href="#vcftools" title="Permanent link">&para;</a></h3>
<p>The aim of VCFtools is to provide   easily accessible methods for working with complex   genetic variation data in the form of VCF files.</p>
<p><em>homepage</em>: <a href="https://vcftools.github.io">https://vcftools.github.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.14</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.1.15</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>0.1.15</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.1.15</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>0.1.16</code></td>
<td></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/11.3.0</code>, <code>GCC/8.3.0</code>, <code>GCC/9.3.0</code>, <code>iccifort/2019.5.281</code></td>
</tr>
<tr>
<td><code>0.1.16</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="vcontact2">vConTACT2<a class="headerlink" href="#vcontact2" title="Permanent link">&para;</a></h3>
<p>vConTACT2 is a tool to perform guilt-by-contig-association classification of  viral genomic sequence data.</p>
<p><em>homepage</em>: <a href="https://bitbucket.org/MAVERICLab/vcontact2">https://bitbucket.org/MAVERICLab/vcontact2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.11.3</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vegas">VEGAS<a class="headerlink" href="#vegas" title="Permanent link">&para;</a></h3>
<p>VEGAS (Versatile Gene-based Association Study) is  a free program for performing gene-based tests for association using  the results from genetic association studies </p>
<p><em>homepage</em>: <a href="http://gump.qimr.edu.au/VEGAS/">http://gump.qimr.edu.au/VEGAS/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.27</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="velocyto">velocyto<a class="headerlink" href="#velocyto" title="Permanent link">&para;</a></h3>
<p>Velocyto is a library for the analysis of RNA velocity.</p>
<p><em>homepage</em>: <a href="https://velocyto.org/velocyto.py/">https://velocyto.org/velocyto.py/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.17.17</code></td>
<td></td>
<td><code>foss/2021a</code>, <code>foss/2022a</code></td>
</tr>
<tr>
<td><code>0.17.17</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="velvet">Velvet<a class="headerlink" href="#velvet" title="Permanent link">&para;</a></h3>
<p>Sequence assembler for very short reads</p>
<p><em>homepage</em>: <a href="http://www.ebi.ac.uk/~zerbino/velvet/">http://www.ebi.ac.uk/~zerbino/velvet/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.10</code></td>
<td><code>-mt-kmer_191</code></td>
<td><code>GCC/11.2.0</code>, <code>GCC/8.3.0</code>, <code>foss/2018a</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.2.10</code></td>
<td><code>-mt-kmer_37</code></td>
<td><code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="vep">VEP<a class="headerlink" href="#vep" title="Permanent link">&para;</a></h3>
<p>Variant Effect Predictor (VEP) determines the effect of your  variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,  transcripts, and protein sequence, as well as regulatory regions.  Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently  used routines in VEP.</p>
<p><em>homepage</em>: <a href="https://www.ensembl.org/info/docs/tools/vep">https://www.ensembl.org/info/docs/tools/vep</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>93.4</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>94.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>94.5</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>95.0</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>96.0</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>103.1</code></td>
<td></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>105</code></td>
<td></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>107</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="verifybamid">verifyBamID<a class="headerlink" href="#verifybamid" title="Permanent link">&para;</a></h3>
<p>verifyBamID is a software that verifies whether the reads in particular file match previously known   genotypes for an individual (or group of individuals), and checks whether the reads are contaminated as a mixture of   two samples. verifyBamID can detect sample contamination and swaps when external genotypes are available. When   external genotypes are not available, verifyBamID still robustly detects sample swaps. </p>
<p><em>homepage</em>: <a href="http://csg.sph.umich.edu/kang/verifyBamID/index.html">http://csg.sph.umich.edu/kang/verifyBamID/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.3</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="verse">VERSE<a class="headerlink" href="#verse" title="Permanent link">&para;</a></h3>
<p>A versatile and efficient RNA-Seq read counting tool</p>
<p><em>homepage</em>: <a href="https://github.com/qinzhu/VERSE">https://github.com/qinzhu/VERSE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.5</code></td>
<td><code>foss/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="vesta">VESTA<a class="headerlink" href="#vesta" title="Permanent link">&para;</a></h3>
<p>VESTA is a 3D visualization program for structured models,    volumetric data such as electron/nuclear densities,    and crystal morphologies. </p>
<p><em>homepage</em>: <a href="http://www.jp-minerals.org/vesta">http://www.jp-minerals.org/vesta</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.5.8</code></td>
<td><code>-gtk3</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="viennarna">ViennaRNA<a class="headerlink" href="#viennarna" title="Permanent link">&para;</a></h3>
<p>The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.</p>
<p><em>homepage</em>: <a href="https://www.tbi.univie.ac.at/RNA/">https://www.tbi.univie.ac.at/RNA/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.3</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.3.4</code></td>
<td></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.3.5</code></td>
<td></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.4.10</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.4.11</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.4.14</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.5.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="vim">Vim<a class="headerlink" href="#vim" title="Permanent link">&para;</a></h3>
<p>Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set. </p>
<p><em>homepage</em>: <a href="http://www.vim.org">http://www.vim.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>8.1.0483</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>8.1.1209</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>9.0.0950</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="virsorter">VirSorter<a class="headerlink" href="#virsorter" title="Permanent link">&para;</a></h3>
<p>VirSorter: mining viral signal from microbial genomic data.</p>
<p><em>homepage</em>: <a href="https://github.com/simroux/VirSorter">https://github.com/simroux/VirSorter</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160601</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="virsorter2">VirSorter2<a class="headerlink" href="#virsorter2" title="Permanent link">&para;</a></h3>
<p>VirSorter2 applies a multi-classifier, expert-guided approach to detect diverse DNA and  RNA virus genomes.</p>
<p><em>homepage</em>: <a href="https://github.com/jiarong/VirSorter2">https://github.com/jiarong/VirSorter2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.4</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="virtualgl">VirtualGL<a class="headerlink" href="#virtualgl" title="Permanent link">&para;</a></h3>
<p>VirtualGL is an open source toolkit that gives any Linux or  Unix remote display software the ability to run OpenGL applications with full  hardware acceleration.</p>
<p><em>homepage</em>: <a href="https://virtualgl.org/">https://virtualgl.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="virtuoso-opensource">Virtuoso-opensource<a class="headerlink" href="#virtuoso-opensource" title="Permanent link">&para;</a></h3>
<p>Virtuoso is a high-performance and scalable Multi-Model RDBMS, Data Integration Middleware, Linked Data Deployment, and HTTP Application Server Platform.</p>
<p><em>homepage</em>: <a href="https://github.com/openlink/virtuoso-opensource">https://github.com/openlink/virtuoso-opensource</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.2.6.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="vispy">VisPy<a class="headerlink" href="#vispy" title="Permanent link">&para;</a></h3>
<p>VisPy is a high-performance interactive 2D/3D data visualization library  leveraging the computational power of modern Graphics Processing Units (GPUs) through the  OpenGL library to display very large datasets.</p>
<p><em>homepage</em>: <a href="https://vispy.org">https://vispy.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.6</code></td>
<td><code>foss/2020b</code>, <code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="vitessce-python">vitessce-python<a class="headerlink" href="#vitessce-python" title="Permanent link">&para;</a></h3>
<p>Python API and Jupyter widget facilitating interactive visualization of spatial single-cell data with Vitessce.</p>
<p><em>homepage</em>: <a href="https://github.com/vitessce/vitessce-python">https://github.com/vitessce/vitessce-python</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20230222</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vitesscer">vitessceR<a class="headerlink" href="#vitesscer" title="Permanent link">&para;</a></h3>
<p>Vitessce is a visual integration tool for exploration of spatial single-cell experiments.</p>
<p><em>homepage</em>: <a href="https://github.com/vitessce/vitessceR">https://github.com/vitessce/vitessceR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.99.0-20230110</code></td>
<td><code>-R-4.2.1</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vmd">VMD<a class="headerlink" href="#vmd" title="Permanent link">&para;</a></h3>
<p>VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular  systems using 3-D graphics and built-in scripting.</p>
<p><em>homepage</em>: <a href="https://www.ks.uiuc.edu/Research/vmd">https://www.ks.uiuc.edu/Research/vmd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.9.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.9.4a43</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>fosscuda/2019b</code></td>
</tr>
<tr>
<td><code>1.9.4a51</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code></td>
</tr>
<tr>
<td><code>1.9.4a57</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vmtk">VMTK<a class="headerlink" href="#vmtk" title="Permanent link">&para;</a></h3>
<p>vmtk is a collection of libraries and tools for 3D reconstruction, geometric analysis, mesh generation and surface data analysis for image-based modeling of blood vessels. </p>
<p><em>homepage</em>: <a href="https://vmtk.org">https://vmtk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="voltools">voltools<a class="headerlink" href="#voltools" title="Permanent link">&para;</a></h3>
<p>CUDA-accelerated numpy 3D affine transformations</p>
<p><em>homepage</em>: <a href="https://github.com/the-lay/voltools">https://github.com/the-lay/voltools</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.2</code></td>
<td><code>foss/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="vorbis-tools">vorbis-tools<a class="headerlink" href="#vorbis-tools" title="Permanent link">&para;</a></h3>
<p>Command-line tools for creating and playing Ogg Vorbis files.</p>
<p><em>homepage</em>: <a href="https://xiph.org/vorbis/">https://xiph.org/vorbis/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="voro">Voro++<a class="headerlink" href="#voro" title="Permanent link">&para;</a></h3>
<p>Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles.</p>
<p><em>homepage</em>: <a href="http://math.lbl.gov/voro++/">http://math.lbl.gov/voro++/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.6</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2019b</code>, <code>intel/2016a</code>, <code>intel/2019b</code></td>
</tr>
</tbody>
</table>
<h3 id="vsc-base">vsc-base<a class="headerlink" href="#vsc-base" title="Permanent link">&para;</a></h3>
<p>Basic Python libraries used by UGent's HPC group</p>
<p><em>homepage</em>: <a href="http://hpcugent.github.com/vsc-base/">http://hpcugent.github.com/vsc-base/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.9</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.7.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.0.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.1.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.4.17</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.5.8</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2.8.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vsc-install">vsc-install<a class="headerlink" href="#vsc-install" title="Permanent link">&para;</a></h3>
<p>vsc-install provides shared setuptools functions and classes for Python libraries  developed by UGent's HPC group</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/vsc-install">https://pypi.python.org/pypi/vsc-install</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.18</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.10.6</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.10.6</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.10.11</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>0.10.11</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.10.25</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.10.26</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>0.11.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vsc-mympirun">vsc-mympirun<a class="headerlink" href="#vsc-mympirun" title="Permanent link">&para;</a></h3>
<p>mympirun is a tool to make it easier for users of HPC clusters to run MPI programs with good performance.</p>
<p><em>homepage</em>: <a href="https://github.com/hpcugent/vsc-mympirun">https://github.com/hpcugent/vsc-mympirun</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.2.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-2.7.11-vsc-base-2.4.17</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-Python-2.7.11-vsc-base-2.5.1</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.4.2</code></td>
<td><code>-vsc-base-2.4.2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.4.3</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.0.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.0b0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.6</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.8</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>4.1.9</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.3</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.4</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.5</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.6</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.7</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.9</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.10</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.2.11</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>5.3.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vsc-mympirun-scoop">vsc-mympirun-scoop<a class="headerlink" href="#vsc-mympirun-scoop" title="Permanent link">&para;</a></h3>
<p>VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent.</p>
<p><em>homepage</em>: <a href="http://hpcugent.github.com/vsc-mympirun/">http://hpcugent.github.com/vsc-mympirun/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.1</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="vsc-processcontrol">vsc-processcontrol<a class="headerlink" href="#vsc-processcontrol" title="Permanent link">&para;</a></h3>
<p>vsc-processcontrol is a module to abstract process control like scheduler settings and affinity  from actual implementations like vsc.affinity and psutil.</p>
<p><em>homepage</em>: <a href="http://hpcugent.github.com/vsc-processcontrol/">http://hpcugent.github.com/vsc-processcontrol/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.0</code></td>
<td><code>-vsc-base-2.1.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vsearch">VSEARCH<a class="headerlink" href="#vsearch" title="Permanent link">&para;</a></h3>
<p>VSEARCH supports de novo and reference based chimera detection,  clustering, full-length and prefix dereplication, rereplication,  reverse complementation, masking, all-vs-all pairwise global alignment,  exact and global alignment searching, shuffling, subsampling and sorting.  It also supports FASTQ file analysis, filtering,  conversion and merging of paired-end reads.</p>
<p><em>homepage</em>: <a href="https://github.com/torognes/vsearch">https://github.com/torognes/vsearch</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.9.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>2.13.4</code></td>
<td><code>iccifort/2019.1.144-GCC-8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.15.0</code></td>
<td><code>GCC/9.3.0</code></td>
</tr>
<tr>
<td><code>2.18.0</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>2.21.1</code></td>
<td><code>GCC/10.3.0</code></td>
</tr>
<tr>
<td><code>2.22.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="vt">vt<a class="headerlink" href="#vt" title="Permanent link">&para;</a></h3>
<p>A tool set for short variant discovery in genetic sequence data.</p>
<p><em>homepage</em>: <a href="http://genome.sph.umich.edu/wiki/Vt">http://genome.sph.umich.edu/wiki/Vt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.57721</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>GCC/9.3.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="vtk">VTK<a class="headerlink" href="#vtk" title="Permanent link">&para;</a></h3>
<p>The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.</p>
<p><em>homepage</em>: <a href="https://www.vtk.org">https://www.vtk.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.3.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>6.3.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.0.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.1.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>7.1.1</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>8.0.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.1.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>8.1.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code>, <code>intel/2019a</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>8.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>fosscuda/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>9.0.1</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>9.1.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>9.2.0.rc2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>9.2.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="vtune">VTune<a class="headerlink" href="#vtune" title="Permanent link">&para;</a></h3>
<p>Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.</p>
<p><em>homepage</em>: <a href="https://software.intel.com/en-us/vtune">https://software.intel.com/en-us/vtune</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2013_update6</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update8</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update9</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2013_update12</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2016_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2017_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update1</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2018_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update2</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2019_update5</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2020_update3</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.6.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2021.9.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.0.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.2.0</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>2022.3.0</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="vv">VV<a class="headerlink" href="#vv" title="Permanent link">&para;</a></h3>
<p>VV is an open-source and cross platform image viewer, designed for fast and  simple visualization of spatio-temporal images: 2D, 2D+t, 3D and 3D+t (or 4D)  images. Only the command-line (clitk) tools are build.</p>
<p><em>homepage</em>: <a href="https://www.creatis.insa-lyon.fr/rio/vv">https://www.creatis.insa-lyon.fr/rio/vv</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2018.09.19</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="vxl">VXL<a class="headerlink" href="#vxl" title="Permanent link">&para;</a></h3>
<p>A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.</p>
<p><em>homepage</em>: <a href="https://sf.net/projects/vxl">https://sf.net/projects/vxl</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.18.0</code></td>
<td><code>foss/2018a</code></td>
</tr>
</tbody>
</table>
<h2 id="w">W<a class="headerlink" href="#w" title="Permanent link">&para;</a></h2>
<p><a href="#wandb">wandb</a> - <a href="#wannier90">Wannier90</a> - <a href="#wanniertools">WannierTools</a> - <a href="#wayland">Wayland</a> - <a href="#waylandpp">Waylandpp</a> - <a href="#wct">WCT</a> - <a href="#wcwidth">wcwidth</a> - <a href="#webin-cli">webin-cli</a> - <a href="#webkitgtk+">WebKitGTK+</a> - <a href="#websocket++">WebSocket++</a> - <a href="#weka">WEKA</a> - <a href="#wget">wget</a> - <a href="#wgsim">wgsim</a> - <a href="#wham">WHAM</a> - <a href="#whatshap">WhatsHap</a> - <a href="#wheel">wheel</a> - <a href="#wien2k">WIEN2k</a> - <a href="#wildmagic">WildMagic</a> - <a href="#winnowmap">Winnowmap</a> - <a href="#wisecondorx">WisecondorX</a> - <a href="#wisexome">WISExome</a> - <a href="#wkhtmltopdf">wkhtmltopdf</a> - <a href="#worker">worker</a> - <a href="#wpebackend-fdo">wpebackend-fdo</a> - <a href="#wps">WPS</a> - <a href="#wrf">WRF</a> - <a href="#wrf-fire">WRF-Fire</a> - <a href="#wrf-python">wrf-python</a> - <a href="#wtdbg2">wtdbg2</a> - <a href="#wxpropertygrid">wxPropertyGrid</a> - <a href="#wxpython">wxPython</a> - <a href="#wxwidgets">wxWidgets</a></p>
<h3 id="wandb">wandb<a class="headerlink" href="#wandb" title="Permanent link">&para;</a></h3>
<p>CLI and Python API for Weights and Biases (wandb), a tool for visualizing and tracking your machine learning experiments.</p>
<p><em>homepage</em>: <a href="https://www.wandb.com/">https://www.wandb.com/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.30</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>gcccuda/2019b</code></td>
</tr>
<tr>
<td><code>0.13.4</code></td>
<td></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>0.13.6</code></td>
<td></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wannier90">Wannier90<a class="headerlink" href="#wannier90" title="Permanent link">&para;</a></h3>
<p>A tool for obtaining maximally-localised Wannier functions</p>
<p><em>homepage</em>: <a href="http://www.wannier.org">http://www.wannier.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>2.0.1</code></td>
<td></td>
<td><code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.0.1.1</code></td>
<td><code>-abinit</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>foss/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.0</code></td>
<td></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>3.1.0</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code>, <code>foss/2021b</code>, <code>foss/2022a</code>, <code>gomkl/2021a</code>, <code>intel/2020a</code>, <code>intel/2020b</code>, <code>intel/2021a</code>, <code>intel/2021b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="wanniertools">WannierTools<a class="headerlink" href="#wanniertools" title="Permanent link">&para;</a></h3>
<p>WannierTools is an open source software that studies the physical properties of given tight-binding  model.</p>
<p><em>homepage</em>: <a href="http://www.wanniertools.com">http://www.wanniertools.com</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.0</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>2.5.1</code></td>
<td><code>intel/2020b</code></td>
</tr>
</tbody>
</table>
<h3 id="wayland">Wayland<a class="headerlink" href="#wayland" title="Permanent link">&para;</a></h3>
<p>Wayland is a project to define a protocol for a compositor to talk to  its clients as well as a library implementation of the protocol.  The  compositor can be a standalone display server running on Linux kernel  modesetting and evdev input devices, an X application, or a wayland  client itself.  The clients can be traditional applications, X servers  (rootless or fullscreen) or other display servers. </p>
<p><em>homepage</em>: <a href="https://wayland.freedesktop.org/">https://wayland.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>1.21.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="waylandpp">Waylandpp<a class="headerlink" href="#waylandpp" title="Permanent link">&para;</a></h3>
<p>Wayland is an object oriented display protocol, which features request and events. Requests can be seen as method calls on certain objects, whereas events can be seen as signals of an object. This makes the Wayland protocol a perfect candidate for a C++ binding.  The goal of this library is to create such a C++ binding for Wayland using the most modern C++ technology currently available, providing an easy to use C++ API to Wayland.</p>
<p><em>homepage</em>: <a href="https://github.com/NilsBrause/waylandpp">https://github.com/NilsBrause/waylandpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wct">WCT<a class="headerlink" href="#wct" title="Permanent link">&para;</a></h3>
<p>NOAA's Weather and Climate Toolkit (WCT) is free, platform independent software  distributed from NOAA's National Centers for Environmental Information (NCEI).  The WCT allows the visualization and data export of weather and climate data,  including Radar, Satellite and Model data. The WCT also provides access to  weather/climate web services provided from NCEI and other organizations. </p>
<p><em>homepage</em>: <a href="https://www.ncdc.noaa.gov/wct/index.php">https://www.ncdc.noaa.gov/wct/index.php</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.6.0</code></td>
<td><code>-Java-11</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="wcwidth">wcwidth<a class="headerlink" href="#wcwidth" title="Permanent link">&para;</a></h3>
<p>wcwidth is a low-level Python library to simplify Terminal emulation.</p>
<p><em>homepage</em>: <a href="https://github.com/jquast/wcwidth">https://github.com/jquast/wcwidth</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.7</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.1.7</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.1.7</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.1.7</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="webin-cli">webin-cli<a class="headerlink" href="#webin-cli" title="Permanent link">&para;</a></h3>
<p>The Webin command line submission interface can be used to validate,  upload and submit files to the European Nucleotide Archive (ENA)</p>
<p><em>homepage</em>: <a href="https://github.com/enasequence/webin-cli">https://github.com/enasequence/webin-cli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.8.9</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="webkitgtk">WebKitGTK+<a class="headerlink" href="#webkitgtk" title="Permanent link">&para;</a></h3>
<p>WebKitGTK+ is a full-featured port of the WebKit rendering engine, suitable for projects requiring any kind of web integration, from hybrid HTML/CSS applications to full-fledged web browsers. It offers WebKit's full functionality and is useful in a wide range of systems from desktop computers to embedded systems like phones, tablets, and televisions.</p>
<p><em>homepage</em>: <a href="https://webkitgtk.org/">https://webkitgtk.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.24.1</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>2.27.4</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>2.37.1</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="websocket">WebSocket++<a class="headerlink" href="#websocket" title="Permanent link">&para;</a></h3>
<p>WebSocket++ is an open source (BSD license) header only C++ library  that implements RFC6455 The WebSocket Protocol. </p>
<p><em>homepage</em>: <a href="https://www.zaphoyd.com/websocketpp/">https://www.zaphoyd.com/websocketpp/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.1</code></td>
<td><code>gompi/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="weka">WEKA<a class="headerlink" href="#weka" title="Permanent link">&para;</a></h3>
<p>Weka is a collection of machine learning algorithms for data mining tasks.   The algorithms can either be applied directly to a dataset or called from your own Java code.   Weka contains tools for data pre-processing, classification, regression, clustering,   association rules, and visualization. It is also well-suited for developing new machine   learning schemes.</p>
<p><em>homepage</em>: <a href="http://www.cs.waikato.ac.nz/ml/weka/index.html">http://www.cs.waikato.ac.nz/ml/weka/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.6.12</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>3.7.0</code></td>
<td><code>-Java-1.7.0_80</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="wget">wget<a class="headerlink" href="#wget" title="Permanent link">&para;</a></h3>
<p>GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,  the most widely-used Internet protocols. It is a non-interactive commandline tool,  so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.</p>
<p><em>homepage</em>: <a href="https://www.gnu.org/software/wget">https://www.gnu.org/software/wget</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.17.1</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.19.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.20.1</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.20.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.21.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.21.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.21.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wgsim">wgsim<a class="headerlink" href="#wgsim" title="Permanent link">&para;</a></h3>
<p>Wgsim is a small tool for simulating sequence reads from a reference genome.</p>
<p><em>homepage</em>: <a href="https://github.com/lh3/wgsim/">https://github.com/lh3/wgsim/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20111017</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wham">WHAM<a class="headerlink" href="#wham" title="Permanent link">&para;</a></h3>
<p>An implementation of WHAM:  the Weighted Histogram Analysis Method</p>
<p><em>homepage</em>: <a href="http://membrane.urmc.rochester.edu/?page_id=126">http://membrane.urmc.rochester.edu/?page_id=126</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.9.1</code></td>
<td></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.0.9.1</code></td>
<td><code>-kj_mol</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>2.0.10.2</code></td>
<td></td>
<td><code>intel/2020a</code></td>
</tr>
<tr>
<td><code>2.0.10.2</code></td>
<td><code>-kj_mol</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="whatshap">WhatsHap<a class="headerlink" href="#whatshap" title="Permanent link">&para;</a></h3>
<p>WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly. It is especially suitable for long reads, but works also well with short reads.</p>
<p><em>homepage</em>: <a href="https://whatshap.readthedocs.io">https://whatshap.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1</code></td>
<td><code>foss/2020b</code>, <code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.4</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="wheel">wheel<a class="headerlink" href="#wheel" title="Permanent link">&para;</a></h3>
<p>A built-package format for Python.</p>
<p><em>homepage</em>: <a href="https://pypi.python.org/pypi/wheel">https://pypi.python.org/pypi/wheel</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.29.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.29.0</code></td>
<td><code>-Python-3.5.1</code></td>
<td><code>foss/2016a</code></td>
</tr>
<tr>
<td><code>0.30.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.30.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>0.30.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.31.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>fosscuda/2018a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.31.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>foss/2018a</code>, <code>fosscuda/2018a</code></td>
</tr>
<tr>
<td><code>0.31.1</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>fosscuda/2018b</code></td>
</tr>
<tr>
<td><code>0.31.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>fosscuda/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="wien2k">WIEN2k<a class="headerlink" href="#wien2k" title="Permanent link">&para;</a></h3>
<p>The program package WIEN2k allows to perform electronic structure calculations of solids using density functional theory (DFT). It is based on the full-potential (linearized) augmented plane-wave ((L)APW) + local orbitals (lo) method, one among the most accurate schemes for band structure calculations. WIEN2k is an all-electron scheme including relativistic effects and has many features.</p>
<p><em>homepage</em>: <a href="http://www.wien2k.at/">http://www.wien2k.at/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17.1</code></td>
<td><code>foss/2018a</code>, <code>gimkl/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>18.1</code></td>
<td><code>foss/2018a</code>, <code>gimkl/2017a</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>19.1</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>19.2</code></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>21.1</code></td>
<td><code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="wildmagic">WildMagic<a class="headerlink" href="#wildmagic" title="Permanent link">&para;</a></h3>
<p>Wild Magic 5.17</p>
<p><em>homepage</em>: <a href="https://www.geometrictools.com/index.html">https://www.geometrictools.com/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.17</code></td>
<td><code>GCCcore/10.3.0</code>, <code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="winnowmap">Winnowmap<a class="headerlink" href="#winnowmap" title="Permanent link">&para;</a></h3>
<p>Winnowmap is a long-read mapping algorithm, and a result of our exploration  into superior minimizer sampling techniques.</p>
<p><em>homepage</em>: <a href="https://github.com/marbl/Winnowmap">https://github.com/marbl/Winnowmap</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wisecondorx">WisecondorX<a class="headerlink" href="#wisecondorx" title="Permanent link">&para;</a></h3>
<p>WisecondorX -- an evolved WISECONDOR</p>
<p><em>homepage</em>: <a href="https://github.com/CenterForMedicalGeneticsGhent/WisecondorX">https://github.com/CenterForMedicalGeneticsGhent/WisecondorX</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.6</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="wisexome">WISExome<a class="headerlink" href="#wisexome" title="Permanent link">&para;</a></h3>
<p>A within-sample comparison approach to detect copy number variations in whole exome sequencing data</p>
<p><em>homepage</em>: <a href="https://github.com/VUmcCGP/wisexome">https://github.com/VUmcCGP/wisexome</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180517</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="wkhtmltopdf">wkhtmltopdf<a class="headerlink" href="#wkhtmltopdf" title="Permanent link">&para;</a></h3>
<p>wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render   HTML into PDF and various image formats using the Qt WebKit rendering engine. These run entirely headless   and do not require a display or display service.</p>
<p><em>homepage</em>: <a href="http://wkhtmltopdf.org/">http://wkhtmltopdf.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.12.3</code></td>
<td><code>-Linux-x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="worker">worker<a class="headerlink" href="#worker" title="Permanent link">&para;</a></h3>
<p>The Worker framework has been developed to help deal with parameter exploration experiments  that would otherwise result in many jobs, forcing the user resort to scripting to retain her sanity;  see also https://vscentrum.be/neutral/documentation/cluster-doc/running-jobs/worker-framework.</p>
<p><em>homepage</em>: <a href="https://github.com/gjbex/worker">https://github.com/gjbex/worker</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.4</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.6.5</code></td>
<td></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.6.6</code></td>
<td></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.6.7</code></td>
<td><code>-intel-2016b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.7</code></td>
<td><code>-intel-2017a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.7</code></td>
<td><code>-intel-2017b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.8</code></td>
<td><code>-intel-2018a</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.8</code></td>
<td><code>-intel-2018b</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.6.11</code></td>
<td></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.6.12</code></td>
<td></td>
<td><code>foss/2019a</code>, <code>foss/2021b</code></td>
</tr>
<tr>
<td><code>1.6.13</code></td>
<td></td>
<td><code>iimpi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="wpebackend-fdo">wpebackend-fdo<a class="headerlink" href="#wpebackend-fdo" title="Permanent link">&para;</a></h3>
<p>WPE WebKit allows embedders to create simple and performant  systems based on Web platform technologies. It is a WebKit port designed with  flexibility and hardware acceleration in mind, leveraging common 3D graphics  APIs for best performance.</p>
<p><em>homepage</em>: <a href="https://wpewebkit.org/">https://wpewebkit.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.13.1</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wps">WPS<a class="headerlink" href="#wps" title="Permanent link">&para;</a></h3>
<p>WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is  a next-generation mesoscale numerical weather prediction system designed to serve both operational  forecasting and atmospheric research needs.</p>
<p><em>homepage</em>: <a href="http://www.wrf-model.org">http://www.wrf-model.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.9.1</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.1</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.2</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.3.1</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.4</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="wrf">WRF<a class="headerlink" href="#wrf" title="Permanent link">&para;</a></h3>
<p>The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale  numerical weather prediction system designed to serve both operational forecasting and atmospheric  research needs.</p>
<p><em>homepage</em>: <a href="https://www.wrf-model.org">https://www.wrf-model.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.8.0</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.9.1.1</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2020a</code>, <code>foss/2020b</code>, <code>intel/2020a</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>4.0.1</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.0.2</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.1.3</code></td>
<td><code>-dm+sm</code></td>
<td><code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.1.3</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2019b</code>, <code>foss/2020a</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>4.3</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.4</code></td>
<td><code>-dmpar</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="wrf-fire">WRF-Fire<a class="headerlink" href="#wrf-fire" title="Permanent link">&para;</a></h3>
<p>WRF-Fire combines the Weather Research and Forecasting model (WRF) with a fire code implementing a  surface fire behavior model, called SFIRE, based on semi-empirical formulas calculate the rate of spread of the fire  line (the interface between burning and unignited fuel) based on fuel properties, wind velocities from WRF, and  terrain slope. The fire spread is implemented by the level set method.</p>
<p><em>homepage</em>: <a href="http://www.openwfm.org/wiki/WRF-SFIRE">http://www.openwfm.org/wiki/WRF-SFIRE</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20170221</code></td>
<td><code>-dmpar</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="wrf-python">wrf-python<a class="headerlink" href="#wrf-python" title="Permanent link">&para;</a></h3>
<p>A collection of diagnostic and interpolation routines for use with output from  the Weather Research and Forecasting (WRF-ARW) Model.</p>
<p><em>homepage</em>: <a href="https://wrf-python.readthedocs.io">https://wrf-python.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.3.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>intel/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="wtdbg2">wtdbg2<a class="headerlink" href="#wtdbg2" title="Permanent link">&para;</a></h3>
<p>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio  or Oxford Nanopore Technologies (ONT). It assembles raw reads without error  correction and then builds the consensus from intermediate assembly output. </p>
<p><em>homepage</em>: <a href="https://github.com/ruanjue/wtdbg2">https://github.com/ruanjue/wtdbg2</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3</code></td>
<td><code>GCC/7.3.0-2.30</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="wxpropertygrid">wxPropertyGrid<a class="headerlink" href="#wxpropertygrid" title="Permanent link">&para;</a></h3>
<p>wxPropertyGrid is a property sheet control for wxWidgets. In other words, it is  a specialized two-column grid for editing properties such as strings, numbers, flagsets, string arrays, and colours.</p>
<p><em>homepage</em>: <a href="http://wxpropgrid.sourceforge.net/">http://wxpropgrid.sourceforge.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.15</code></td>
<td><code>GCC/4.9.2</code></td>
</tr>
</tbody>
</table>
<h3 id="wxpython">wxPython<a class="headerlink" href="#wxpython" title="Permanent link">&para;</a></h3>
<p>Wraps the wxWidgets C++ toolkit and provides access to the user interface portions of the wxWidgets  API, enabling Python applications to have a native GUI on Windows, Macs or Unix systems, with a native look and feel  and requiring very little (if any) platform specific code.</p>
<p><em>homepage</em>: <a href="https://www.wxpython.org/">https://www.wxpython.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.2.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>3.0.2.0</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>3.0.2.0</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>3.0.2.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>3.0.2.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>4.0.4</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>4.0.4</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>4.0.7.post2</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>4.1.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="wxwidgets">wxWidgets<a class="headerlink" href="#wxwidgets" title="Permanent link">&para;</a></h3>
<p>wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI.</p>
<p><em>homepage</em>: <a href="https://www.wxwidgets.org">https://www.wxwidgets.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0.3</code></td>
<td><code>foss/2018a</code></td>
</tr>
<tr>
<td><code>3.0.4</code></td>
<td><code>GCC/8.2.0-2.31.1</code></td>
</tr>
<tr>
<td><code>3.1.3</code></td>
<td><code>GCC/8.3.0</code></td>
</tr>
<tr>
<td><code>3.1.4</code></td>
<td><code>GCC/10.2.0</code></td>
</tr>
<tr>
<td><code>3.1.5</code></td>
<td><code>GCC/10.3.0</code>, <code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCC/11.2.0</code></td>
</tr>
<tr>
<td><code>3.2.1</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
<tr>
<td><code>3.2.2.1</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="x">X<a class="headerlink" href="#x" title="Permanent link">&para;</a></h2>
<p><a href="#x11">X11</a> - <a href="#x13as">x13as</a> - <a href="#x264">x264</a> - <a href="#x265">x265</a> - <a href="#xalt">XALT</a> - <a href="#xarray">xarray</a> - <a href="#xbitmaps">xbitmaps</a> - <a href="#xcb-proto">xcb-proto</a> - <a href="#xcb-util">xcb-util</a> - <a href="#xcb-util-image">xcb-util-image</a> - <a href="#xcb-util-keysyms">xcb-util-keysyms</a> - <a href="#xcb-util-renderutil">xcb-util-renderutil</a> - <a href="#xcb-util-wm">xcb-util-wm</a> - <a href="#xcell">xCell</a> - <a href="#xcfun">XCFun</a> - <a href="#xcrysden">XCrySDen</a> - <a href="#xerces-c++">Xerces-C++</a> - <a href="#xesmf">xESMF</a> - <a href="#xextproto">xextproto</a> - <a href="#xf86vidmodeproto">xf86vidmodeproto</a> - <a href="#xgboost">XGBoost</a> - <a href="#xgrafix">XGrafix</a> - <a href="#xineramaproto">xineramaproto</a> - <a href="#xkeyboardconfig">XKeyboardConfig</a> - <a href="#xlsxwriter">XlsxWriter</a> - <a href="#xmds2">XMDS2</a> - <a href="#xmipp">Xmipp</a> - <a href="#xmitgcm">xmitgcm</a> - <a href="#xml-compile">XML-Compile</a> - <a href="#xml-libxml">XML-LibXML</a> - <a href="#xml-parser">XML-Parser</a> - <a href="#xmlf90">xmlf90</a> - <a href="#xmlsec">XMLSec</a> - <a href="#xmlstarlet">XMLStarlet</a> - <a href="#xonsh">xonsh</a> - <a href="#xorg-macros">xorg-macros</a> - <a href="#xplor-nih">XPLOR-NIH</a> - <a href="#xprop">xprop</a> - <a href="#xproto">xproto</a> - <a href="#xsd">XSD</a> - <a href="#xtandem">XTandem</a> - <a href="#xtb">xtb</a> - <a href="#xtensor">xtensor</a> - <a href="#xtrans">xtrans</a> - <a href="#xvfb">Xvfb</a> - <a href="#xxd">xxd</a> - <a href="#xxhash">xxHash</a> - <a href="#xz">XZ</a></p>
<h3 id="x11">X11<a class="headerlink" href="#x11" title="Permanent link">&para;</a></h3>
<p>The X Window System (X11) is a windowing system for bitmap displays</p>
<p><em>homepage</em>: <a href="https://www.x.org">https://www.x.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160819</code></td>
<td><code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20170129</code></td>
<td><code>GCCcore/6.3.0</code>, <code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>20170314</code></td>
<td><code>GCCcore/6.3.0</code></td>
</tr>
<tr>
<td><code>20171023</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20180131</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20180604</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>20190311</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>20190717</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20200222</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20201008</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20210518</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>20210802</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>20220504</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>20221110</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="x13as">x13as<a class="headerlink" href="#x13as" title="Permanent link">&para;</a></h3>
<p>X-13ARIMA-SEATS is seasonal adjustment software produced, distributed,  and maintained by the Census Bureau.  Features of X-13ARIMA-SEATS include:     - Extensive time series modeling and model selection capabilities for        linear regression models with ARIMA errors (regARIMA models);     - The capability to generate ARIMA model-based seasonal adjustment        using a version of the SEATS software originally developed by Victor        Gmez and Agustn Maravall at the Bank of Spain, as well as nonparametric        adjustments from the X-11 procedure;     - Diagnostics of the quality and stability of the adjustments        achieved under the options selected;     - The ability to efficiently process many series at once. </p>
<p><em>homepage</em>: <a href="https://www.census.gov/data/software/x13as.About_X-13.html">https://www.census.gov/data/software/x13as.About_X-13.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1-1-b59</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="x264">x264<a class="headerlink" href="#x264" title="Permanent link">&para;</a></h3>
<p>x264 is a free software library and application for encoding video streams  into the H.264/MPEG-4 AVC compression format, and is released under the  terms of the GNU GPL. </p>
<p><em>homepage</em>: <a href="https://www.videolan.org/developers/x264.html">https://www.videolan.org/developers/x264.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20160114</code></td>
<td><code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20160430</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>20160614</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>20170406</code></td>
<td><code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>20170721</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20170913</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>20171217</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20180128</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20180325</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>20181203</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>20190413</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>20190925</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>20191217</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>20201026</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>20210414</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>20210613</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>20220620</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>20230226</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="x265">x265<a class="headerlink" href="#x265" title="Permanent link">&para;</a></h3>
<p>x265 is a free software library and application for encoding video streams  into the H.265 AVC compression format, and is released under the terms of  the GNU GPL. </p>
<p><em>homepage</em>: <a href="https://x265.org/">https://x265.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.4</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>2.5</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.6</code></td>
<td><code>GCCcore/6.4.0</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.7</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.9</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>3.2</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.5</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xalt">XALT<a class="headerlink" href="#xalt" title="Permanent link">&para;</a></h3>
<p>XALT 2 is a tool to allow a site to track user executables   and library usage on a cluster. When installed it can tell a site what are   the top executables by Node-Hours or by the number of users or the number   of times it is run. XALT 2 also tracks library usage as well. XALT 2 can   also track package use by R, MATLAB or Python. It tracks both MPI and   non-MPI programs.</p>
<p><em>homepage</em>: <a href="https://github.com/xalt/xalt">https://github.com/xalt/xalt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.8.4</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="xarray">xarray<a class="headerlink" href="#xarray" title="Permanent link">&para;</a></h3>
<p>xarray (formerly xray) is an open source project and Python package that aims to bring  the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the  core pandas data structures.</p>
<p><em>homepage</em>: <a href="https://github.com/pydata/xarray">https://github.com/pydata/xarray</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.9.5</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.5</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.9.6</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.10.0</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.10.3</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.10.4</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.10.8</code></td>
<td><code>-Python-3.6.4</code></td>
<td><code>intel/2018a</code></td>
</tr>
<tr>
<td><code>0.12.1</code></td>
<td><code>-Python-3.6.6</code></td>
<td><code>foss/2018b</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>0.13.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>0.15.1</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>0.16.1</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
<tr>
<td><code>0.16.2</code></td>
<td></td>
<td><code>foss/2020b</code>, <code>fosscuda/2020b</code>, <code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.19.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>0.20.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021b</code></td>
</tr>
<tr>
<td><code>2022.6.0</code></td>
<td></td>
<td><code>foss/2022a</code>, <code>intel/2022a</code></td>
</tr>
<tr>
<td><code>2022.9.0</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="xbitmaps">xbitmaps<a class="headerlink" href="#xbitmaps" title="Permanent link">&para;</a></h3>
<p>provides bitmaps for x</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-proto">xcb-proto<a class="headerlink" href="#xcb-proto" title="Permanent link">&para;</a></h3>
<p>The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.</p>
<p><em>homepage</em>: <a href="https://xcb.freedesktop.org/">https://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.11</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.13</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-util">xcb-util<a class="headerlink" href="#xcb-util" title="Permanent link">&para;</a></h3>
<p>The xcb-util package provides additional extensions to the XCB library,  many that were previously found in Xlib, but are not part of core X protocol</p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-util-image">xcb-util-image<a class="headerlink" href="#xcb-util-image" title="Permanent link">&para;</a></h3>
<p>The xcb-util-image package provides additional extensions to the XCB library.</p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-util-keysyms">xcb-util-keysyms<a class="headerlink" href="#xcb-util-keysyms" title="Permanent link">&para;</a></h3>
<p>The xcb-util-keysyms package contains a library for  handling standard X key constants and conversion to/from keycodes.</p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.0</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-util-renderutil">xcb-util-renderutil<a class="headerlink" href="#xcb-util-renderutil" title="Permanent link">&para;</a></h3>
<p>The xcb-util-renderutil package provides additional extensions to the XCB library.</p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.9</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcb-util-wm">xcb-util-wm<a class="headerlink" href="#xcb-util-wm" title="Permanent link">&para;</a></h3>
<p>The xcb-util-wm package contains libraries which provide client and window-manager helpers  for EWMH and ICCCM.</p>
<p><em>homepage</em>: <a href="http://xcb.freedesktop.org/">http://xcb.freedesktop.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.4.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcell">xCell<a class="headerlink" href="#xcell" title="Permanent link">&para;</a></h3>
<p>xCell is a gene signatures-based method learned from thousands of pure cell types from various sources.</p>
<p><em>homepage</em>: <a href="https://github.com/dviraran/xCell">https://github.com/dviraran/xCell</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.12</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="xcfun">XCFun<a class="headerlink" href="#xcfun" title="Permanent link">&para;</a></h3>
<p>XCFun is a library of DFT exchange-correlation (XC) functionals.  It is based on automatic differentiation and can therefore generate arbitrary order  derivatives of these functionals. </p>
<p><em>homepage</em>: <a href="http://dftlibs.org/xcfun/">http://dftlibs.org/xcfun/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.0</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.1.1</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>20180122</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>20190127</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>foss/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="xcrysden">XCrySDen<a class="headerlink" href="#xcrysden" title="Permanent link">&para;</a></h3>
<p>XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. It also possesses some tools for analysis of properties in reciprocal space such as interactive selection of k-paths in the Brillouin zone for the band-structure plots, and visualisation of Fermi surfaces. </p>
<p><em>homepage</em>: <a href="http://www.xcrysden.org/">http://www.xcrysden.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.60</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.6.2</code></td>
<td><code>foss/2019b</code>, <code>foss/2020b</code>, <code>foss/2022a</code>, <code>intel/2019b</code>, <code>intel/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="xerces-c">Xerces-C++<a class="headerlink" href="#xerces-c" title="Permanent link">&para;</a></h3>
<p>Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs.</p>
<p><em>homepage</em>: <a href="https://xerces.apache.org/xerces-c/">https://xerces.apache.org/xerces-c/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.1.4</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>3.2.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>3.2.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>3.2.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>3.2.4</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xesmf">xESMF<a class="headerlink" href="#xesmf" title="Permanent link">&para;</a></h3>
<p>xESMF: Universal Regridder for Geospatial Data</p>
<p><em>homepage</em>: <a href="https://xesmf.readthedocs.io">https://xesmf.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.0</code></td>
<td></td>
<td><code>intel/2020b</code></td>
</tr>
<tr>
<td><code>0.3.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="xextproto">xextproto<a class="headerlink" href="#xextproto" title="Permanent link">&para;</a></h3>
<p>XExtProto protocol headers.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.3.0</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="xf86vidmodeproto">xf86vidmodeproto<a class="headerlink" href="#xf86vidmodeproto" title="Permanent link">&para;</a></h3>
<p>X11 XFree86 video mode extension protocol headers.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.3.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xgboost">XGBoost<a class="headerlink" href="#xgboost" title="Permanent link">&para;</a></h3>
<p>XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.</p>
<p><em>homepage</em>: <a href="https://github.com/dmlc/xgboost">https://github.com/dmlc/xgboost</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6a2</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.6a2</code></td>
<td><code>-Python-3.5.2</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>0.6a2</code></td>
<td><code>-Python-3.6.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>0.72.1</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.90</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>1.2.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>1.7.2</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>20171120</code></td>
<td><code>-Java-1.8.0_152</code></td>
<td><code>intel/2017b</code></td>
</tr>
</tbody>
</table>
<h3 id="xgrafix">XGrafix<a class="headerlink" href="#xgrafix" title="Permanent link">&para;</a></h3>
<p>A graphics library and controller for physics simulation programs. 3-d surface plots, scatter plots, 2-d line plots.</p>
<p><em>homepage</em>: <a href="https://ptsg.egr.msu.edu/">https://ptsg.egr.msu.edu/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.41</code></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xineramaproto">xineramaproto<a class="headerlink" href="#xineramaproto" title="Permanent link">&para;</a></h3>
<p>X protocol and ancillary headers for xinerama</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xkeyboardconfig">XKeyboardConfig<a class="headerlink" href="#xkeyboardconfig" title="Permanent link">&para;</a></h3>
<p>The non-arch keyboard configuration database for X Window.  The goal is to provide the consistent, well-structured,  frequently released open source of X keyboard configuration data  for X Window System implementations (free, open source and commercial).  The project is targeted to XKB-based systems.</p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/XKeyboardConfig/">http://www.freedesktop.org/wiki/Software/XKeyboardConfig/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.17</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xlsxwriter">XlsxWriter<a class="headerlink" href="#xlsxwriter" title="Permanent link">&para;</a></h3>
<p>A Python module for creating Excel XLSX files</p>
<p><em>homepage</em>: <a href="https://xlsxwriter.readthedocs.io/">https://xlsxwriter.readthedocs.io/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>3.0.2</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>3.0.8</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xmds2">XMDS2<a class="headerlink" href="#xmds2" title="Permanent link">&para;</a></h3>
<p>The purpose of XMDS2 is to simplify the process of creating simulations that  solve systems of initial-value first-order partial and ordinary differential equations.</p>
<p><em>homepage</em>: <a href="http://www.xmds.org/">http://www.xmds.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.2.3</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="xmipp">Xmipp<a class="headerlink" href="#xmipp" title="Permanent link">&para;</a></h3>
<p>Scipion is an image processing framework to obtain 3D models of macromolecular complexes using Electron Microscopy (3DEM). It integrates several software packages and presents an unified interface for both biologists and developers. Scipion allows to execute workflows combining different software tools, while taking care of formats and conversions. Additionally, all steps are tracked and can be reproduced later on.  </p>
<p><em>homepage</em>: <a href="https://github.com/I2PC/scipion/wiki">https://github.com/I2PC/scipion/wiki</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.19.04-Apollo</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>foss/2019a</code>, <code>fosscuda/2019a</code></td>
</tr>
<tr>
<td><code>3.22.07-Helios</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
<tr>
<td><code>3.22.07-Helios</code></td>
<td><code>-CUDA-11.7.0</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="xmitgcm">xmitgcm<a class="headerlink" href="#xmitgcm" title="Permanent link">&para;</a></h3>
<p>xmitgcm is a python package for reading MITgcm binary  MDS files into xarray data structures. By storing data in dask arrays,  xmitgcm enables parallel, out-of-core analysis of MITgcm output data.</p>
<p><em>homepage</em>: <a href="https://github.com/MITgcm/xmitgcm">https://github.com/MITgcm/xmitgcm</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.2</code></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="xml-compile">XML-Compile<a class="headerlink" href="#xml-compile" title="Permanent link">&para;</a></h3>
<p>Perl module for compilation based XML processing</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/XML::Compile">https://metacpan.org/pod/XML::Compile</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.63</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xml-libxml">XML-LibXML<a class="headerlink" href="#xml-libxml" title="Permanent link">&para;</a></h3>
<p>Perl binding for libxml2</p>
<p><em>homepage</em>: <a href="https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod">https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0132</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.0132</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.0132</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>foss/2017b</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>2.0132</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.0132</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.0200</code></td>
<td><code>-Perl-5.28.1</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
<tr>
<td><code>2.0201</code></td>
<td></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>2.0205</code></td>
<td></td>
<td><code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>2.0206</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0207</code></td>
<td></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xml-parser">XML-Parser<a class="headerlink" href="#xml-parser" title="Permanent link">&para;</a></h3>
<p>This is a Perl extension interface to James Clark's XML parser, expat.</p>
<p><em>homepage</em>: <a href="https://search.cpan.org/~toddr/XML-Parser-2.46/">https://search.cpan.org/~toddr/XML-Parser-2.46/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.41</code></td>
<td><code>-Perl-5.20.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.44</code></td>
<td><code>-Perl-5.22.1</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2.44_01</code></td>
<td><code>-Perl-5.24.0</code></td>
<td><code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016b</code>, <code>gimkl/2017a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2.44_01</code></td>
<td><code>-Perl-5.24.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>2.44_01</code></td>
<td><code>-Perl-5.26.0</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.44_01</code></td>
<td><code>-Perl-5.26.1</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>2.44_01</code></td>
<td><code>-Perl-5.28.0</code></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>2.46</code></td>
<td><code>-Perl-5.32.1</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>2.46</code></td>
<td><code>-Perl-5.34.1</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xmlf90">xmlf90<a class="headerlink" href="#xmlf90" title="Permanent link">&para;</a></h3>
<p>xmlf90 is a basic XML parsing library written in Fortran.</p>
<p><em>homepage</em>: <a href="https://launchpad.net/xmlf90">https://launchpad.net/xmlf90</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.5.3</code></td>
<td><code>foss/2016b</code>, <code>foss/2017a</code></td>
</tr>
<tr>
<td><code>1.5.4</code></td>
<td><code>GCC/10.2.0</code>, <code>GCC/10.3.0</code>, <code>GCC/11.2.0</code>, <code>iccifort/2019.5.281</code>, <code>iccifort/2020.4.304</code>, <code>intel-compilers/2021.2.0</code>, <code>intel-compilers/2021.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xmlsec">XMLSec<a class="headerlink" href="#xmlsec" title="Permanent link">&para;</a></h3>
<p>XML Security Library is a C library based on LibXML2,  supporting major XML security standards.</p>
<p><em>homepage</em>: <a href="https://www.aleksey.com/xmlsec/index.html">https://www.aleksey.com/xmlsec/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.26</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xmlstarlet">XMLStarlet<a class="headerlink" href="#xmlstarlet" title="Permanent link">&para;</a></h3>
<p>Command line XML tool</p>
<p><em>homepage</em>: <a href="http://xmlstar.sourceforge.net">http://xmlstar.sourceforge.net</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.6.1</code></td>
<td><code>GCCcore/6.4.0</code>, <code>foss/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xonsh">xonsh<a class="headerlink" href="#xonsh" title="Permanent link">&para;</a></h3>
<p>Xonsh is a Python-ish, BASHwards-looking shell language and command prompt.</p>
<p><em>homepage</em>: <a href="https://xon.sh/">https://xon.sh/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.3.2</code></td>
<td><code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xorg-macros">xorg-macros<a class="headerlink" href="#xorg-macros" title="Permanent link">&para;</a></h3>
<p>X.org macros utilities.</p>
<p><em>homepage</em>: <a href="https://cgit.freedesktop.org/xorg/util/macros">https://cgit.freedesktop.org/xorg/util/macros</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.19.0</code></td>
<td><code>foss/2016a</code>, <code>foss/2016b</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.19.1</code></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.19.2</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.19.3</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xplor-nih">XPLOR-NIH<a class="headerlink" href="#xplor-nih" title="Permanent link">&para;</a></h3>
<p>A System for X-ray Crystallography and NMR</p>
<p><em>homepage</em>: <a href="https://nmr.cit.nih.gov/xplor-nih/">https://nmr.cit.nih.gov/xplor-nih/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4</code></td>
<td><code>-Linux_x86_64</code></td>
<td><code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="xprop">xprop<a class="headerlink" href="#xprop" title="Permanent link">&para;</a></h3>
<p>The xprop utility is for displaying window and font properties in an X server.  One window or font is selected using the command line arguments or possibly  in the case of a window, by clicking on the desired window. A list of  properties is then given, possibly with formatting information.</p>
<p><em>homepage</em>: <a href="https://www.x.org/wiki/">https://www.x.org/wiki/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.2</code></td>
<td></td>
<td><code>GCCcore/5.4.0</code>, <code>GCCcore/6.4.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.2.2</code></td>
<td><code>-X11-20180131</code></td>
<td><code>GCCcore/6.4.0</code></td>
</tr>
<tr>
<td><code>1.2.3</code></td>
<td></td>
<td><code>GCCcore/7.3.0</code></td>
</tr>
<tr>
<td><code>1.2.4</code></td>
<td></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.2.5</code></td>
<td></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xproto">xproto<a class="headerlink" href="#xproto" title="Permanent link">&para;</a></h3>
<p>X protocol and ancillary headers</p>
<p><em>homepage</em>: <a href="https://www.freedesktop.org/wiki/Software/xlibs">https://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.0.28</code></td>
<td><code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.29</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>7.0.31</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xsd">XSD<a class="headerlink" href="#xsd" title="Permanent link">&para;</a></h3>
<p>CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.</p>
<p><em>homepage</em>: <a href="https://www.codesynthesis.com/products/xsd/">https://www.codesynthesis.com/products/xsd/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.0.0</code></td>
<td><code>GCCcore/8.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xtandem">XTandem<a class="headerlink" href="#xtandem" title="Permanent link">&para;</a></h3>
<p>X!Tandem open source is software that can match tandem mass  spectra with peptide sequences, in a process that has come to be known as  protein identification.</p>
<p><em>homepage</em>: <a href="https://www.thegpm.org/tandem">https://www.thegpm.org/tandem</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>17.02.01.4</code></td>
<td><code>GCC/6.4.0-2.28</code>, <code>iccifort/2017.4.196-GCC-6.4.0-2.28</code></td>
</tr>
</tbody>
</table>
<h3 id="xtb">xtb<a class="headerlink" href="#xtb" title="Permanent link">&para;</a></h3>
<p>xtb - An extended tight-binding semi-empirical program package. </p>
<p><em>homepage</em>: <a href="https://xtb-docs.readthedocs.io">https://xtb-docs.readthedocs.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.2.2-hotfix</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>6.2.3</code></td>
<td></td>
<td><code>foss/2019b</code></td>
</tr>
<tr>
<td><code>6.4.1</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>intel/2021a</code></td>
</tr>
<tr>
<td><code>6.5.0</code></td>
<td></td>
<td><code>foss/2021b</code></td>
</tr>
<tr>
<td><code>6.5.1</code></td>
<td></td>
<td><code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="xtensor">xtensor<a class="headerlink" href="#xtensor" title="Permanent link">&para;</a></h3>
<p>xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions.</p>
<p><em>homepage</em>: <a href="https://github.com/xtensor-stack/xtensor">https://github.com/xtensor-stack/xtensor</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.24.0</code></td>
<td><code>foss/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="xtrans">xtrans<a class="headerlink" href="#xtrans" title="Permanent link">&para;</a></h3>
<p>xtrans includes a number of routines to make X implementations transport-independent;  at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet. </p>
<p><em>homepage</em>: <a href="http://www.freedesktop.org/wiki/Software/xlibs">http://www.freedesktop.org/wiki/Software/xlibs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.5</code></td>
<td><code>GCCcore/11.3.0</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016a</code></td>
</tr>
</tbody>
</table>
<h3 id="xvfb">Xvfb<a class="headerlink" href="#xvfb" title="Permanent link">&para;</a></h3>
<p>Xvfb is an X server that can run on machines with no display hardware and no physical input devices.  It emulates a dumb framebuffer using virtual memory.</p>
<p><em>homepage</em>: <a href="https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml">https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20.8</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>1.20.9</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.20.11</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.20.13</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>21.1.3</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>21.1.6</code></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xxd">xxd<a class="headerlink" href="#xxd" title="Permanent link">&para;</a></h3>
<p>xxd is part of the VIM package and this will only install xxd, not vim! xxd converts to/from hexdumps of binary files.</p>
<p><em>homepage</em>: <a href="https://www.vim.org">https://www.vim.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>8.2.4220</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xxhash">xxHash<a class="headerlink" href="#xxhash" title="Permanent link">&para;</a></h3>
<p>xxHash is an extremely fast non-cryptographic hash algorithm, working at RAM speed limit.</p>
<p><em>homepage</em>: <a href="https://cyan4973.github.io/xxHash">https://cyan4973.github.io/xxHash</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.8.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="xz">XZ<a class="headerlink" href="#xz" title="Permanent link">&para;</a></h3>
<p>xz: XZ utilities</p>
<p><em>homepage</em>: <a href="https://tukaani.org/xz/">https://tukaani.org/xz/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.0.5</code></td>
<td></td>
<td><code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>5.2.0</code></td>
<td></td>
<td><code>GCC/4.9.2</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td></td>
<td><code>GCC/4.9.2</code>, <code>GCC/5.4.0-2.26</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/5.4.0</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>intel/2016a</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>5.2.2</code></td>
<td><code>-gettext-0.19.7</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>5.2.3</code></td>
<td></td>
<td><code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>gimkl/2017a</code></td>
</tr>
<tr>
<td><code>5.2.4</code></td>
<td></td>
<td><code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.2.0</code></td>
</tr>
<tr>
<td><code>5.2.5</code></td>
<td></td>
<td><code>GCCcore/10.1.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>5.2.7</code></td>
<td></td>
<td><code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>
<h2 id="y">Y<a class="headerlink" href="#y" title="Permanent link">&para;</a></h2>
<p><a href="#yacs">YACS</a> - <a href="#yade">Yade</a> - <a href="#yaff">yaff</a> - <a href="#yambo">Yambo</a> - <a href="#yaml-cpp">yaml-cpp</a> - <a href="#yank">YANK</a> - <a href="#yaps">YAPS</a> - <a href="#yasm">Yasm</a> - <a href="#yaxt">YAXT</a> - <a href="#yices">Yices</a> - <a href="#yoda">YODA</a></p>
<h3 id="yacs">YACS<a class="headerlink" href="#yacs" title="Permanent link">&para;</a></h3>
<p>YACS was created as a lightweight library to define and manage system configurations, such as those commonly found in software designed for scientific experimentation. These "configurations" typically cover concepts like hyperparameters used in training a machine learning model or configurable model hyperparameters, such as the depth of a convolutional neural network.</p>
<p><em>homepage</em>: <a href="https://github.com/rbgirshick/yacs">https://github.com/rbgirshick/yacs</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.1.8</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/8.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="yade">Yade<a class="headerlink" href="#yade" title="Permanent link">&para;</a></h3>
<p>Yade is an extensible open-source framework for discrete numerical models,   focused on Discrete Element Method. The computation parts are written in c++ using flexible object model,   allowing independent implementation of new alogrithms and interfaces.   Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. </p>
<p><em>homepage</em>: <a href="https://yade-dem.org/doc/">https://yade-dem.org/doc/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.20.0</code></td>
<td><code>-Python-2.7.11</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>2016.06a</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2017.01a</code></td>
<td><code>-Python-2.7.12</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>2018.02b</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>intel/2018a</code></td>
</tr>
</tbody>
</table>
<h3 id="yaff">yaff<a class="headerlink" href="#yaff" title="Permanent link">&para;</a></h3>
<p>Yaff stands for 'Yet another force field'. It is a pythonic force-field code.</p>
<p><em>homepage</em>: <a href="https://molmod.github.io/yaff/">https://molmod.github.io/yaff/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.develop.2.15</code></td>
<td><code>-Python-2.7.12-HDF5-1.8.18</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>1.1.2</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.1.3</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>intel/2017a</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-2.7.14</code></td>
<td><code>foss/2018a</code>, <code>intel/2017b</code>, <code>intel/2018a</code></td>
</tr>
<tr>
<td><code>1.4.2</code></td>
<td><code>-Python-3.6.3</code></td>
<td><code>intel/2017b</code></td>
</tr>
<tr>
<td><code>1.4.5</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>-Python-2.7.15</code></td>
<td><code>intel/2018b</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.7.2</code></td>
<td><code>intel/2019a</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.7.4</code></td>
<td><code>foss/2019b</code>, <code>intel/2019b</code></td>
</tr>
<tr>
<td><code>1.6.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code>, <code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="yambo">Yambo<a class="headerlink" href="#yambo" title="Permanent link">&para;</a></h3>
<p>Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.  Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf.</p>
<p><em>homepage</em>: <a href="http://www.yambo-code.org">http://www.yambo-code.org</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.4.2</code></td>
<td><code>intel/2016.02-GCC-4.9</code></td>
</tr>
<tr>
<td><code>5.0.4</code></td>
<td><code>intel/2021a</code></td>
</tr>
</tbody>
</table>
<h3 id="yaml-cpp">yaml-cpp<a class="headerlink" href="#yaml-cpp" title="Permanent link">&para;</a></h3>
<p>yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec</p>
<p><em>homepage</em>: <a href="https://github.com/jbeder/yaml-cpp">https://github.com/jbeder/yaml-cpp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.6.3</code></td>
<td><code>GCCcore/8.3.0</code></td>
</tr>
<tr>
<td><code>0.7.0</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="yank">YANK<a class="headerlink" href="#yank" title="Permanent link">&para;</a></h3>
<p>A GPU-accelerated Python framework for exploring algorithms for alchemical free energy calculations</p>
<p><em>homepage</em>: <a href="http://getyank.org/">http://getyank.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.25.2</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>intel/2020a</code></td>
</tr>
</tbody>
</table>
<h3 id="yaps">YAPS<a class="headerlink" href="#yaps" title="Permanent link">&para;</a></h3>
<p>YAPS - Yet Another Positioning Solver</p>
<p><em>homepage</em>: <a href="https://github.com/baktoft/yaps">https://github.com/baktoft/yaps</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.1.0</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.1.0</code></td>
<td><code>-R-3.6.0</code></td>
<td><code>intel/2019a</code></td>
</tr>
</tbody>
</table>
<h3 id="yasm">Yasm<a class="headerlink" href="#yasm" title="Permanent link">&para;</a></h3>
<p>Yasm: Complete rewrite of the NASM assembler with BSD license</p>
<p><em>homepage</em>: <a href="http://www.tortall.net/projects/yasm/">http://www.tortall.net/projects/yasm/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code>, <code>foss/2016a</code>, <code>foss/2016b</code>, <code>foss/2017a</code>, <code>gimkl/2.11.5</code>, <code>gimkl/2017a</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017a</code></td>
</tr>
</tbody>
</table>
<h3 id="yaxt">YAXT<a class="headerlink" href="#yaxt" title="Permanent link">&para;</a></h3>
<p>Yet Another eXchange Tool</p>
<p><em>homepage</em>: <a href="https://www.dkrz.de/redmine/projects/yaxt">https://www.dkrz.de/redmine/projects/yaxt</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.1</code></td>
<td><code>intel/2016b</code>, <code>intel/2017a</code>, <code>intel/2017b</code></td>
</tr>
<tr>
<td><code>0.6.0</code></td>
<td><code>intel/2018a</code>, <code>intel/2018b</code>, <code>iomkl/2018b</code></td>
</tr>
<tr>
<td><code>0.6.2</code></td>
<td><code>foss/2018b</code>, <code>gompi/2019b</code>, <code>iimpi/2019b</code></td>
</tr>
<tr>
<td><code>0.9.0</code></td>
<td><code>gompi/2020b</code></td>
</tr>
<tr>
<td><code>0.9.1</code></td>
<td><code>gompi/2021a</code></td>
</tr>
<tr>
<td><code>0.9.2</code></td>
<td><code>iimpi/2021b</code></td>
</tr>
<tr>
<td><code>0.9.2.1</code></td>
<td><code>gompi/2021b</code></td>
</tr>
</tbody>
</table>
<h3 id="yices">Yices<a class="headerlink" href="#yices" title="Permanent link">&para;</a></h3>
<p>Yices 2 is an SMT solver that decides the satisfiability of formulas containing uninterpreted  function symbols with equality, real and integer arithmetic, bitvectors, scalar types, and tuples. Yices 2 supports  both linear and nonlinear arithmetic.</p>
<p><em>homepage</em>: <a href="https://yices.csl.sri.com/index.html">https://yices.csl.sri.com/index.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.6.2</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
</tbody>
</table>
<h3 id="yoda">YODA<a class="headerlink" href="#yoda" title="Permanent link">&para;</a></h3>
<p>Yet more Objects for (High Energy Physics) Data Analysis </p>
<p><em>homepage</em>: <a href="https://yoda.hepforge.org/">https://yoda.hepforge.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.9.7</code></td>
<td><code>GCC/11.3.0</code></td>
</tr>
</tbody>
</table>
<h2 id="z">Z<a class="headerlink" href="#z" title="Permanent link">&para;</a></h2>
<p><a href="#z3">Z3</a> - <a href="#zarr">zarr</a> - <a href="#zeromq">ZeroMQ</a> - <a href="#zfp">zfp</a> - <a href="#zgoubi">Zgoubi</a> - <a href="#zinger">zingeR</a> - <a href="#zip">Zip</a> - <a href="#zlib">zlib</a> - <a href="#zlib-ng">zlib-ng</a> - <a href="#zlibbioc">zlibbioc</a> - <a href="#zopfli">Zopfli</a> - <a href="#zpaq">ZPAQ</a> - <a href="#zsh">zsh</a> - <a href="#zstd">zstd</a></p>
<h3 id="z3">Z3<a class="headerlink" href="#z3" title="Permanent link">&para;</a></h3>
<p>Z3 is a theorem prover from Microsoft Research. </p>
<p><em>homepage</em>: <a href="https://github.com/Z3Prover/z3">https://github.com/Z3Prover/z3</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.8.9</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.8.10</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.8.11</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>4.8.12</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>4.8.16</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
<tr>
<td><code>4.10.2</code></td>
<td><code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zarr">zarr<a class="headerlink" href="#zarr" title="Permanent link">&para;</a></h3>
<p>Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays,  designed for use in parallel computing.</p>
<p><em>homepage</em>: <a href="https://zarr.readthedocs.io/en/stable/">https://zarr.readthedocs.io/en/stable/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.1.4</code></td>
<td><code>-Python-2.7.13</code></td>
<td><code>foss/2017a</code></td>
</tr>
<tr>
<td><code>2.4.0</code></td>
<td><code>-Python-3.8.2</code></td>
<td><code>foss/2020a</code></td>
</tr>
<tr>
<td><code>2.8.1</code></td>
<td></td>
<td><code>foss/2020b</code></td>
</tr>
<tr>
<td><code>2.10.1</code></td>
<td></td>
<td><code>foss/2021a</code></td>
</tr>
<tr>
<td><code>2.13.3</code></td>
<td></td>
<td><code>foss/2021b</code>, <code>foss/2022a</code></td>
</tr>
</tbody>
</table>
<h3 id="zeromq">ZeroMQ<a class="headerlink" href="#zeromq" title="Permanent link">&para;</a></h3>
<p>ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.  It gives you sockets that carry atomic messages across various transports like in-process,  inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,  pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered  products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous  message-processing tasks. It has a score of language APIs and runs on most operating systems.</p>
<p><em>homepage</em>: <a href="https://www.zeromq.org/">https://www.zeromq.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>4.1.4</code></td>
<td><code>foss/2016a</code>, <code>intel/2016a</code></td>
</tr>
<tr>
<td><code>4.1.5</code></td>
<td><code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.2.0</code></td>
<td><code>foss/2016b</code>, <code>intel/2016b</code></td>
</tr>
<tr>
<td><code>4.2.2</code></td>
<td><code>foss/2017a</code>, <code>foss/2017b</code>, <code>fosscuda/2017b</code>, <code>intel/2017a</code>, <code>intel/2017b</code>, <code>intelcuda/2017b</code></td>
</tr>
<tr>
<td><code>4.2.5</code></td>
<td><code>foss/2018a</code>, <code>foss/2018b</code>, <code>fosscuda/2018b</code>, <code>intel/2018a</code>, <code>intel/2018b</code></td>
</tr>
<tr>
<td><code>4.3.2</code></td>
<td><code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>4.3.3</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>4.3.4</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zfp">zfp<a class="headerlink" href="#zfp" title="Permanent link">&para;</a></h3>
<p>zfp is a compressed format for representing multidimensional floating-point and integer arrays. zfp provides compressed-array classes that support high throughput read and write random access to individual array elements. zfp also supports serial and parallel (OpenMP and CUDA) compression of whole arrays, e.g., for applications that read and write large data sets to and from disk.</p>
<p><em>homepage</em>: <a href="https://github.com/LLNL/zfp">https://github.com/LLNL/zfp</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>0.5.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.0.0</code></td>
<td><code>GCCcore/10.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zgoubi">Zgoubi<a class="headerlink" href="#zgoubi" title="Permanent link">&para;</a></h3>
<p>Zgoubi is a ray-tracing code in use for charged particle beam dynamics simulations.  It can simulate beam dynamics in a large variety of machines and optical systems.</p>
<p><em>homepage</em>: <a href="https://zgoubi.sourceforge.io">https://zgoubi.sourceforge.io</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>6.0.2</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zinger">zingeR<a class="headerlink" href="#zinger" title="Permanent link">&para;</a></h3>
<p>Zero-Inflated Negative binomial Gene Expression in R</p>
<p><em>homepage</em>: <a href="https://github.com/statOmics/zingeR">https://github.com/statOmics/zingeR</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>20180131</code></td>
<td><code>-R-3.5.1</code></td>
<td><code>foss/2018b</code></td>
</tr>
</tbody>
</table>
<h3 id="zip">Zip<a class="headerlink" href="#zip" title="Permanent link">&para;</a></h3>
<p>Zip is a compression and file packaging/archive utility. Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives have been portability and other-than-MSDOS functionality</p>
<p><em>homepage</em>: <a href="http://www.info-zip.org/Zip.html">http://www.info-zip.org/Zip.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>3.0</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/11.3.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zlib">zlib<a class="headerlink" href="#zlib" title="Permanent link">&para;</a></h3>
<p>zlib is designed to be a free, general-purpose, legally unencumbered -- that is,  not covered by any patents -- lossless data-compression library for use on virtually any  computer hardware and operating system.</p>
<p><em>homepage</em>: <a href="http://www.zlib.net/">http://www.zlib.net/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.2.7</code></td>
<td><code>GCC/4.8.1</code>, <code>GCC/4.8.2</code></td>
</tr>
<tr>
<td><code>1.2.8</code></td>
<td><code>GCC/4.8.2</code>, <code>GCC/4.8.3</code>, <code>GCC/4.8.4</code>, <code>GCC/4.9.2</code>, <code>GCC/4.9.2-binutils-2.25</code>, <code>GCC/4.9.3</code>, <code>GCC/4.9.3-2.25</code>, <code>GCC/4.9.3-binutils-2.25</code>, <code>GCC/5.1.0-binutils-2.25</code>, <code>GCCcore/4.9.2</code>, <code>GCCcore/4.9.3</code>, <code>GCCcore/4.9.4</code>, <code>GCCcore/5.3.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/6.1.0</code>, <code>GCCcore/6.2.0</code>, <code>GCCcore/6.3.0</code>, <code>GNU/4.9.3-2.25</code>, <code>foss/2016.04</code>, <code>foss/2016a</code>, <code>gimkl/2.11.5</code>, <code>intel/2016.02-GCC-4.9</code>, <code>intel/2016a</code>, <code>intel/2016b</code>, <code>intel/2017.01</code>, <code>iomkl/2016.07</code>, <code>iomkl/2016.09-GCC-4.9.3-2.25</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.2.10</code></td>
<td><code>system</code></td>
</tr>
<tr>
<td><code>1.2.11</code></td>
<td><code>FCC/4.5.0</code>, <code>GCCcore/10.1.0</code>, <code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code>, <code>GCCcore/11.1.0</code>, <code>GCCcore/11.2.0</code>, <code>GCCcore/5.4.0</code>, <code>GCCcore/5.5.0</code>, <code>GCCcore/6.3.0</code>, <code>GCCcore/6.4.0</code>, <code>GCCcore/7.1.0</code>, <code>GCCcore/7.2.0</code>, <code>GCCcore/7.3.0</code>, <code>GCCcore/7.4.0</code>, <code>GCCcore/8.1.0</code>, <code>GCCcore/8.2.0</code>, <code>GCCcore/8.3.0</code>, <code>GCCcore/8.4.0</code>, <code>GCCcore/9.1.0</code>, <code>GCCcore/9.2.0</code>, <code>GCCcore/9.3.0</code>, <code>GCCcore/9.4.0</code>, <code>GCCcore/system</code>, <code>gimkl/2017a</code>, <code>system</code></td>
</tr>
<tr>
<td><code>1.2.12</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.1.0</code>, <code>GCCcore/12.2.0</code>, <code>GCCcore/9.5.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="zlib-ng">zlib-ng<a class="headerlink" href="#zlib-ng" title="Permanent link">&para;</a></h3>
<p>zlib data compression library for the next generation systems</p>
<p><em>homepage</em>: <a href="https://github.com/zlib-ng/zlib-ng">https://github.com/zlib-ng/zlib-ng</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>2.0.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>2.0.6</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zlibbioc">zlibbioc<a class="headerlink" href="#zlibbioc" title="Permanent link">&para;</a></h3>
<p>This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have  these available via other means.</p>
<p><em>homepage</em>: <a href="https://bioconductor.org/packages/release/bioc/html/zlibbioc.html">https://bioconductor.org/packages/release/bioc/html/zlibbioc.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>versionsuffix</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.18.0</code></td>
<td><code>-R-3.2.3</code></td>
<td><code>intel/2016a</code></td>
</tr>
<tr>
<td><code>1.20.0</code></td>
<td><code>-R-3.3.1</code></td>
<td><code>intel/2016b</code></td>
</tr>
</tbody>
</table>
<h3 id="zopfli">Zopfli<a class="headerlink" href="#zopfli" title="Permanent link">&para;</a></h3>
<p>Zopfli Compression Algorithm is a compression library programmed in C to perform very good, but slow, deflate or zlib compression.</p>
<p><em>homepage</em>: <a href="https://github.com/google/zopfli">https://github.com/google/zopfli</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.0.3</code></td>
<td><code>GCCcore/10.2.0</code>, <code>GCCcore/10.3.0</code></td>
</tr>
</tbody>
</table>
<h3 id="zpaq">ZPAQ<a class="headerlink" href="#zpaq" title="Permanent link">&para;</a></h3>
<p>zpaq is a free and open source (GPL v3) incremental,  journaling command-line archiver for Windows, Linux and Mac OS/X</p>
<p><em>homepage</em>: <a href="http://mattmahoney.net/dc/zpaq.html">http://mattmahoney.net/dc/zpaq.html</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>7.00</code></td>
<td><code>GCC/4.8.2</code></td>
</tr>
</tbody>
</table>
<h3 id="zsh">zsh<a class="headerlink" href="#zsh" title="Permanent link">&para;</a></h3>
<p>Zsh is a shell designed for interactive use, although it is also a powerful scripting language.</p>
<p><em>homepage</em>: <a href="https://www.zsh.org/">https://www.zsh.org/</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>5.1.1</code></td>
<td><code>GNU/4.9.3-2.25</code></td>
</tr>
<tr>
<td><code>5.2</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>5.8</code></td>
<td><code>GCC/8.3.0</code>, <code>system</code></td>
</tr>
</tbody>
</table>
<h3 id="zstd">zstd<a class="headerlink" href="#zstd" title="Permanent link">&para;</a></h3>
<p>Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.</p>
<p><em>homepage</em>: <a href="https://facebook.github.io/zstd">https://facebook.github.io/zstd</a></p>
<table>
<thead>
<tr>
<th>version</th>
<th>toolchain</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>1.3.4</code></td>
<td><code>foss/2016b</code></td>
</tr>
<tr>
<td><code>1.4.0</code></td>
<td><code>GCCcore/7.3.0</code>, <code>GCCcore/8.2.0</code>, <code>foss/2018b</code></td>
</tr>
<tr>
<td><code>1.4.4</code></td>
<td><code>GCCcore/8.3.0</code>, <code>GCCcore/9.3.0</code></td>
</tr>
<tr>
<td><code>1.4.5</code></td>
<td><code>GCCcore/10.2.0</code></td>
</tr>
<tr>
<td><code>1.4.9</code></td>
<td><code>GCCcore/10.3.0</code></td>
</tr>
<tr>
<td><code>1.5.0</code></td>
<td><code>GCCcore/11.2.0</code></td>
</tr>
<tr>
<td><code>1.5.2</code></td>
<td><code>GCCcore/11.3.0</code>, <code>GCCcore/12.2.0</code></td>
</tr>
</tbody>
</table>

  <hr>
<div class="md-source-file">
  <small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">March 20, 2023</span>
      
    
  </small>
</div>


  




                
              </article>
            </div>
          
          
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
            Back to top
          </button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://twitter.com/easy_build" target="_blank" rel="noopener" title="twitter.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://www.youtube.com/c/easybuilders" target="_blank" rel="noopener" title="www.youtube.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://docs.easybuild.io" target="_blank" rel="noopener" title="docs.easybuild.io" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M96 0C43 0 0 43 0 96v320c0 53 43 96 96 96h320c17.7 0 32-14.3 32-32s-14.3-32-32-32v-64c17.7 0 32-14.3 32-32V32c0-17.7-14.3-32-32-32H96zm0 384h256v64H96c-17.7 0-32-14.3-32-32s14.3-32 32-32zm32-240c0-8.8 7.2-16 16-16h192c8.8 0 16 7.2 16 16s-7.2 16-16 16H144c-8.8 0-16-7.2-16-16zm16 48h192c8.8 0 16 7.2 16 16s-7.2 16-16 16H144c-8.8 0-16-7.2-16-16s7.2-16 16-16z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.instant", "navigation.top", "navigation.tracking", "toc.follow", "toc.integrate", "search.highlight", "search.suggest"], "search": "../../assets/javascripts/workers/search.208ed371.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.b4d07000.min.js"></script>
      
        <script src="../../js/asciinema-player-2.6.1.js"></script>
      
    
  </body>
</html>