---
search:
  boost: 0.5
---
# snippy

Snippy finds SNPs between a haploid reference genome and your NGS sequence  reads. It will find both substitutions (snps) and insertions/deletions (indels). Rapid haploid variant calling and core genome alignment.

*homepage*: <https://github.com/tseemann/snippy>

version | versionsuffix | toolchain
--------|---------------|----------
``4.4.1`` | ``-Perl-5.28.0`` | ``foss/2018b``
``4.6.0`` |  | ``GCC/10.2.0``
``4.6.0`` | ``-Java-13-Python-3.8.2`` | ``GCC/9.3.0``
``4.6.0`` | ``-R-4.1.2`` | ``foss/2021b``


*(quick links: [(all)](../index.md) - [0](../0/index.md) - [a](../a/index.md) - [b](../b/index.md) - [c](../c/index.md) - [d](../d/index.md) - [e](../e/index.md) - [f](../f/index.md) - [g](../g/index.md) - [h](../h/index.md) - [i](../i/index.md) - [j](../j/index.md) - [k](../k/index.md) - [l](../l/index.md) - [m](../m/index.md) - [n](../n/index.md) - [o](../o/index.md) - [p](../p/index.md) - [q](../q/index.md) - [r](../r/index.md) - [s](../s/index.md) - [t](../t/index.md) - [u](../u/index.md) - [v](../v/index.md) - [w](../w/index.md) - [x](../x/index.md) - [y](../y/index.md) - [z](../z/index.md))*

