# Trinity

Trinity represents a novel method for the efficient and robust de novo reconstruction  of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,  Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.

*homepage*: <https://trinityrnaseq.github.io>

version | versionsuffix | toolchain
--------|---------------|----------
``2.10.0`` | ``-Python-3.7.4`` | ``foss/2019b``
``2.15.1`` |  | ``foss/2021b``
``2.15.1`` |  | ``foss/2022a``
``2.2.0`` |  | ``foss/2016a``
``2.4.0`` |  | ``foss/2017a``
``2.5.1`` |  | ``intel/2017a``
``2.5.1`` |  | ``intel/2017b``
``2.6.6`` |  | ``intel/2018a``
``2.8.4`` |  | ``foss/2018b``
``2.8.5`` | ``-Java-11`` | ``GCC/8.3.0``
``2.8.5`` |  | ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.9.1`` | ``-Python-2.7.16`` | ``foss/2019b``
``2.9.1`` | ``-Python-3.7.4`` | ``foss/2019b``
``2.9.1`` |  | ``foss/2020b``
