---
search:
  boost: 0.5
---
# Mikado

Mikado is a lightweight Python3 pipeline to identify  the most useful or “best” set of transcripts from multiple transcript  assemblies. Our approach leverages transcript assemblies generated  by multiple methods to define expressed loci, assign a representative  transcript and return a set of gene models that selects against transcripts  that are chimeric, fragmented or with short or disrupted CDS.

*homepage*: <https://github.com/EI-CoreBioinformatics/mikado>

version | toolchain
--------|----------
``2.3.4`` | ``foss/2022b``


*(quick links: [(all)](../index.md) - [0](../0/index.md) - [a](../a/index.md) - [b](../b/index.md) - [c](../c/index.md) - [d](../d/index.md) - [e](../e/index.md) - [f](../f/index.md) - [g](../g/index.md) - [h](../h/index.md) - [i](../i/index.md) - [j](../j/index.md) - [k](../k/index.md) - [l](../l/index.md) - [m](../m/index.md) - [n](../n/index.md) - [o](../o/index.md) - [p](../p/index.md) - [q](../q/index.md) - [r](../r/index.md) - [s](../s/index.md) - [t](../t/index.md) - [u](../u/index.md) - [v](../v/index.md) - [w](../w/index.md) - [x](../x/index.md) - [y](../y/index.md) - [z](../z/index.md))*

