---
search:
  boost: 0.5
---
# minimap2

Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.

*homepage*: <https://github.com/lh3/minimap2>

version | toolchain
--------|----------
``2.22`` | ``GCCcore/11.2.0``
``2.24`` | ``GCCcore/11.2.0``
``2.24`` | ``GCCcore/11.3.0``
``2.26`` | ``GCCcore/12.2.0``
``2.26`` | ``GCCcore/12.3.0``
``2.28`` | ``GCCcore/13.2.0``
``2.29`` | ``GCCcore/13.3.0``
``2.30`` | ``GCCcore/14.2.0``
``2.30`` | ``GCCcore/14.3.0``


*(quick links: [(all)](../index.md) - [0](../0/index.md) - [a](../a/index.md) - [b](../b/index.md) - [c](../c/index.md) - [d](../d/index.md) - [e](../e/index.md) - [f](../f/index.md) - [g](../g/index.md) - [h](../h/index.md) - [i](../i/index.md) - [j](../j/index.md) - [k](../k/index.md) - [l](../l/index.md) - [m](../m/index.md) - [n](../n/index.md) - [o](../o/index.md) - [p](../p/index.md) - [q](../q/index.md) - [r](../r/index.md) - [s](../s/index.md) - [t](../t/index.md) - [u](../u/index.md) - [v](../v/index.md) - [w](../w/index.md) - [x](../x/index.md) - [y](../y/index.md) - [z](../z/index.md))*

